{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#RIC3 trainee project\n",
    "###Code retreived from the Global Parkinson's Genetics Program (GP2)-Bioinformatics-course, Sara Bandres-Ciga, Hampton Leonard, Mary Makarious and other members of the International Parkinson's Disease Genomics Consortium (IPDGC)\n",
    "###Modified by Kajsa Brolin, Manuela Tan\n",
    "\n",
    "General info:\n",
    "When setting up the cloud environment, use the \"Legacy Python/R (default prior to January 14, 2020)\". For unknown reason, the current default GATK 4.1 4.1... environment will not run properly. Errors occur when using plink (at least it did in January 2021) stating that the disk space is running out no matter how high you go (for me, this occured at the step of updating sex info despite setting disk size to 1 TB). Setting disk size to 500 GB has been enough for me. \n",
    "\n",
    "If this is your first time running a Terra Notebook, you should run either the Py3 - Start Here (or the R - Start Here you want to use R instead). I would also highly recommend to go trough the GP2 Introduction. Using Terra to Access Data and Perform Analyses : https://github.com/GP2-TNC-WG/GP2-Bioinformatics-course/blob/master/Introduction.md#4 \n",
    "\n",
    "QC of the plink files has been performed (with minor modifications) according to the GP2 tutorial from: https://github.com/GP2-TNC-WG/GP2-Bioinformatics-course/blob/master/Module_I.md. Plotting the PCA results and creating the covariate file has however been done in R).\n",
    "\n",
    "Once the QC is done, the rest is similar to the other IPDGC trainee scripts.\n",
    "\n",
    "Please inform us if something doesn't work or if there is a better way of doing things! Not experts and want to learn :) "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Getting Started\n",
    "Run the cells below to set up libraries and functions.\n",
    "\n",
    "We will also set up the billing project for making queries and set a few path variables to pull data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Set up libraries\n",
    "# Use the os package to interact with the environment\n",
    "import os\n",
    "\n",
    "# Bring in Pandas for Dataframe functionality\n",
    "import pandas as pd\n",
    "pd.options.mode.chained_assignment = None  # default='warn'\n",
    "\n",
    "# numpy for basics\n",
    "import numpy as np\n",
    "\n",
    "# Use StringIO for working with file contents\n",
    "from io import StringIO\n",
    "\n",
    "# Enable IPython to display matplotlib graphs\n",
    "import matplotlib.pyplot as plt\n",
    "%matplotlib inline\n",
    "\n",
    "# Enable interaction with the FireCloud API\n",
    "from firecloud import api as fapi\n",
    "\n",
    "# Import the iPython HTML rendering for displaying links to Google Cloud Console\n",
    "from IPython.core.display import display, HTML\n",
    "\n",
    "# Import urllib modules for building URLs to Google Cloud Console\n",
    "import urllib.parse\n",
    "\n",
    "# BigQuery for querying data\n",
    "from google.cloud import bigquery\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Set up billing project and data path variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ipdgctraineeproject-ric3\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/gatk\n"
     ]
    }
   ],
   "source": [
    "BILLING_PROJECT_ID = os.environ['GOOGLE_PROJECT']\n",
    "WORKSPACE_NAMESPACE = os.environ['WORKSPACE_NAMESPACE']\n",
    "WORKSPACE_NAME = os.environ['WORKSPACE_NAME']\n",
    "WORKSPACE_BUCKET = os.environ['WORKSPACE_BUCKET']\n",
    "\n",
    "WORKSPACE_ATTRIBUTES = fapi.get_workspace(WORKSPACE_NAMESPACE, WORKSPACE_NAME).json().get('workspace',{}).get('attributes',{})\n",
    "\n",
    "GS_RELEASE_PATH = 'gs://amp-pd-data/releases/2019_v1release_1015'\n",
    "GS_CLINICAL_RELEASE_PATH = f'{GS_RELEASE_PATH}/clinical'\n",
    "\n",
    "GS_WGS_RELEASE_PATH = 'gs://amp-pd-genomics/releases/2019_v1release_1015/wgs'\n",
    "GS_WGS_RELEASE_PLINK_PATH = os.path.join(GS_WGS_RELEASE_PATH, 'plink')\n",
    "GS_WGS_RELEASE_GATK_PATH = os.path.join(GS_WGS_RELEASE_PATH, 'gatk')\n",
    "\n",
    "BQ_RELEASE_DATASET = 'amp-pd-research.2019_v1release_1015'\n",
    "\n",
    "\n",
    "print(BILLING_PROJECT_ID)\n",
    "print(GS_CLINICAL_RELEASE_PATH)\n",
    "print(GS_WGS_RELEASE_PLINK_PATH)\n",
    "print(GS_WGS_RELEASE_GATK_PATH)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "##Some useful functions (which actually needs to be run in order for the rest to work)\n",
    "#Utility routine for printing a shell command before executing it\n",
    "def shell_do(command):\n",
    "    print(f'Executing: {command}')\n",
    "    !$command\n",
    "\n",
    "# Utility routines for reading files from Google Cloud Storage\n",
    "def gcs_read_file(path):\n",
    "    \"\"\"Return the contents of a file in GCS\"\"\"\n",
    "    contents = !gsutil -u {BILLING_PROJECT_ID} cat {path}\n",
    "    return '\\n'.join(contents)\n",
    "    \n",
    "def gcs_read_csv(path, sep=None):\n",
    "    \"\"\"Return a DataFrame from the contents of a delimited file in GCS\"\"\"\n",
    "    return pd.read_csv(StringIO(gcs_read_file(path)), sep=sep, engine='python')\n",
    "\n",
    "# Utility routine for display a message and a link\n",
    "def display_html_link(description, link_text, url):\n",
    "    html = f'''\n",
    "    <p>\n",
    "    </p>\n",
    "    <p>\n",
    "    {description}\n",
    "    <a target=_blank href=\"{url}\">{link_text}</a>.\n",
    "    </p>\n",
    "    '''\n",
    "\n",
    "    display(HTML(html))\n",
    "    \n",
    "# Utility routine for displaying a message and link to Cloud Console\n",
    "def link_to_cloud_console_gcs(description, link_text, gcs_path):\n",
    "    url = '{}?{}'.format(\n",
    "        os.path.join('https://console.cloud.google.com/storage/browser',\n",
    "                     gcs_path.replace(\"gs://\",\"\")),\n",
    "        urllib.parse.urlencode({'userProject': BILLING_PROJECT_ID}))\n",
    "\n",
    "    display_html_link(description, link_text, url)\n",
    "    \n",
    "# Get the data from a query\n",
    "def bq_query(query):\n",
    "    \"\"\"Return the contents of a query against BigQuery\"\"\"\n",
    "    return pd.read_gbq(\n",
    "        query,\n",
    "        project_id=BILLING_PROJECT_ID,\n",
    "        dialect='standard')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Querying Clinical Data\n",
    "Let's look at all the available clinical tables in BigQuery. The following query will tell you the available dataset ID's, names, row count, and size."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Downloading: 100%|██████████| 33/33 [00:00<00:00, 122.35rows/s]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>project_id</th>\n",
       "      <th>dataset_id</th>\n",
       "      <th>table_id</th>\n",
       "      <th>row_count</th>\n",
       "      <th>size_bytes</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Biospecimen_analyses_CSF_abeta_tau_ptau</td>\n",
       "      <td>9385</td>\n",
       "      <td>458495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Biospecimen_analyses_CSF_beta_glucocerebrosidase</td>\n",
       "      <td>278</td>\n",
       "      <td>24342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Biospecimen_analyses_SomaLogic_plasma</td>\n",
       "      <td>26100</td>\n",
       "      <td>2273980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Biospecimen_analyses_other</td>\n",
       "      <td>6105</td>\n",
       "      <td>559618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Caffeine_history</td>\n",
       "      <td>1222</td>\n",
       "      <td>55586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>DTI</td>\n",
       "      <td>1180</td>\n",
       "      <td>132014</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>DaTSCAN_SBR</td>\n",
       "      <td>1976</td>\n",
       "      <td>106071</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>DaTSCAN_visual_interpretation</td>\n",
       "      <td>842</td>\n",
       "      <td>27048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Demographics</td>\n",
       "      <td>4298</td>\n",
       "      <td>413191</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Enrollment</td>\n",
       "      <td>4280</td>\n",
       "      <td>447363</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Epworth_Sleepiness_Scale</td>\n",
       "      <td>9353</td>\n",
       "      <td>2614392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Family_History_PD</td>\n",
       "      <td>4255</td>\n",
       "      <td>191512</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MDS_UPDRS_Part_I</td>\n",
       "      <td>15971</td>\n",
       "      <td>4455979</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MDS_UPDRS_Part_II</td>\n",
       "      <td>16887</td>\n",
       "      <td>4063524</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MDS_UPDRS_Part_III</td>\n",
       "      <td>18624</td>\n",
       "      <td>10595517</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MDS_UPDRS_Part_IV</td>\n",
       "      <td>10137</td>\n",
       "      <td>1330342</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MMSE</td>\n",
       "      <td>914</td>\n",
       "      <td>243770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MOCA</td>\n",
       "      <td>9470</td>\n",
       "      <td>2774678</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>MRI</td>\n",
       "      <td>1163</td>\n",
       "      <td>47510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Modified_Schwab___England_ADL</td>\n",
       "      <td>13127</td>\n",
       "      <td>447501</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>PDQ_39</td>\n",
       "      <td>2993</td>\n",
       "      <td>1373064</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>PD_Medical_History</td>\n",
       "      <td>17900</td>\n",
       "      <td>1188904</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>REM_Sleep_Behavior_Disorder_Questionnaire_Mayo</td>\n",
       "      <td>2928</td>\n",
       "      <td>352365</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>REM_Sleep_Behavior_Disorder_Questionnaire_Stia...</td>\n",
       "      <td>7674</td>\n",
       "      <td>2288703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>Smoking_and_alcohol_history</td>\n",
       "      <td>2796</td>\n",
       "      <td>303897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>UPDRS</td>\n",
       "      <td>668</td>\n",
       "      <td>345588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>UPSIT</td>\n",
       "      <td>4741</td>\n",
       "      <td>322539</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>amp_pd_case_control</td>\n",
       "      <td>4298</td>\n",
       "      <td>343991</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>amp_pd_participants</td>\n",
       "      <td>4298</td>\n",
       "      <td>164495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>duplicate_subjects</td>\n",
       "      <td>24</td>\n",
       "      <td>665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>rna_sample_inventory</td>\n",
       "      <td>8356</td>\n",
       "      <td>326023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>wgs_gatk_joint_genotyping_samples</td>\n",
       "      <td>3074</td>\n",
       "      <td>74500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>amp-pd-research</td>\n",
       "      <td>2019_v1release_1015</td>\n",
       "      <td>wgs_sample_inventory</td>\n",
       "      <td>3941</td>\n",
       "      <td>107446</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         project_id           dataset_id  \\\n",
       "0   amp-pd-research  2019_v1release_1015   \n",
       "1   amp-pd-research  2019_v1release_1015   \n",
       "2   amp-pd-research  2019_v1release_1015   \n",
       "3   amp-pd-research  2019_v1release_1015   \n",
       "4   amp-pd-research  2019_v1release_1015   \n",
       "5   amp-pd-research  2019_v1release_1015   \n",
       "6   amp-pd-research  2019_v1release_1015   \n",
       "7   amp-pd-research  2019_v1release_1015   \n",
       "8   amp-pd-research  2019_v1release_1015   \n",
       "9   amp-pd-research  2019_v1release_1015   \n",
       "10  amp-pd-research  2019_v1release_1015   \n",
       "11  amp-pd-research  2019_v1release_1015   \n",
       "12  amp-pd-research  2019_v1release_1015   \n",
       "13  amp-pd-research  2019_v1release_1015   \n",
       "14  amp-pd-research  2019_v1release_1015   \n",
       "15  amp-pd-research  2019_v1release_1015   \n",
       "16  amp-pd-research  2019_v1release_1015   \n",
       "17  amp-pd-research  2019_v1release_1015   \n",
       "18  amp-pd-research  2019_v1release_1015   \n",
       "19  amp-pd-research  2019_v1release_1015   \n",
       "20  amp-pd-research  2019_v1release_1015   \n",
       "21  amp-pd-research  2019_v1release_1015   \n",
       "22  amp-pd-research  2019_v1release_1015   \n",
       "23  amp-pd-research  2019_v1release_1015   \n",
       "24  amp-pd-research  2019_v1release_1015   \n",
       "25  amp-pd-research  2019_v1release_1015   \n",
       "26  amp-pd-research  2019_v1release_1015   \n",
       "27  amp-pd-research  2019_v1release_1015   \n",
       "28  amp-pd-research  2019_v1release_1015   \n",
       "29  amp-pd-research  2019_v1release_1015   \n",
       "30  amp-pd-research  2019_v1release_1015   \n",
       "31  amp-pd-research  2019_v1release_1015   \n",
       "32  amp-pd-research  2019_v1release_1015   \n",
       "\n",
       "                                             table_id  row_count  size_bytes  \n",
       "0             Biospecimen_analyses_CSF_abeta_tau_ptau       9385      458495  \n",
       "1    Biospecimen_analyses_CSF_beta_glucocerebrosidase        278       24342  \n",
       "2               Biospecimen_analyses_SomaLogic_plasma      26100     2273980  \n",
       "3                          Biospecimen_analyses_other       6105      559618  \n",
       "4                                    Caffeine_history       1222       55586  \n",
       "5                                                 DTI       1180      132014  \n",
       "6                                         DaTSCAN_SBR       1976      106071  \n",
       "7                       DaTSCAN_visual_interpretation        842       27048  \n",
       "8                                        Demographics       4298      413191  \n",
       "9                                          Enrollment       4280      447363  \n",
       "10                           Epworth_Sleepiness_Scale       9353     2614392  \n",
       "11                                  Family_History_PD       4255      191512  \n",
       "12                                   MDS_UPDRS_Part_I      15971     4455979  \n",
       "13                                  MDS_UPDRS_Part_II      16887     4063524  \n",
       "14                                 MDS_UPDRS_Part_III      18624    10595517  \n",
       "15                                  MDS_UPDRS_Part_IV      10137     1330342  \n",
       "16                                               MMSE        914      243770  \n",
       "17                                               MOCA       9470     2774678  \n",
       "18                                                MRI       1163       47510  \n",
       "19                      Modified_Schwab___England_ADL      13127      447501  \n",
       "20                                             PDQ_39       2993     1373064  \n",
       "21                                 PD_Medical_History      17900     1188904  \n",
       "22     REM_Sleep_Behavior_Disorder_Questionnaire_Mayo       2928      352365  \n",
       "23  REM_Sleep_Behavior_Disorder_Questionnaire_Stia...       7674     2288703  \n",
       "24                        Smoking_and_alcohol_history       2796      303897  \n",
       "25                                              UPDRS        668      345588  \n",
       "26                                              UPSIT       4741      322539  \n",
       "27                                amp_pd_case_control       4298      343991  \n",
       "28                                amp_pd_participants       4298      164495  \n",
       "29                                 duplicate_subjects         24         665  \n",
       "30                               rna_sample_inventory       8356      326023  \n",
       "31                  wgs_gatk_joint_genotyping_samples       3074       74500  \n",
       "32                               wgs_sample_inventory       3941      107446  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clinical_tables = f\"\"\"\n",
    "\n",
    "SELECT \n",
    "project_id, dataset_id, table_id, row_count, size_bytes \n",
    "\n",
    "FROM `{BQ_RELEASE_DATASET}.__TABLES__`\n",
    "\n",
    "\"\"\"\n",
    "\n",
    "\n",
    "bq_query(clinical_tables)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Downloading: 100%|██████████| 4298/4298 [00:00<00:00, 16025.27rows/s]\n"
     ]
    }
   ],
   "source": [
    "demographics = f\"\"\"\n",
    "\n",
    "SELECT \n",
    "participant_id, age_at_baseline, sex, race\n",
    "\n",
    "FROM `{BQ_RELEASE_DATASET}.Demographics`\n",
    "\n",
    "\"\"\"\n",
    "\n",
    "\n",
    "demo = bq_query(demographics)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 4298 entries, 0 to 4297\n",
      "Data columns (total 4 columns):\n",
      "participant_id     4298 non-null object\n",
      "age_at_baseline    4298 non-null int64\n",
      "sex                4298 non-null object\n",
      "race               4297 non-null object\n",
      "dtypes: int64(1), object(3)\n",
      "memory usage: 134.4+ KB\n"
     ]
    }
   ],
   "source": [
    "demo.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>participant_id</th>\n",
       "      <th>age_at_baseline</th>\n",
       "      <th>sex</th>\n",
       "      <th>race</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PP-41564</td>\n",
       "      <td>22</td>\n",
       "      <td>Male</td>\n",
       "      <td>White</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PD-PDZV843ATF</td>\n",
       "      <td>24</td>\n",
       "      <td>Male</td>\n",
       "      <td>White</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PD-PDCK871NBR</td>\n",
       "      <td>24</td>\n",
       "      <td>Male</td>\n",
       "      <td>White</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PP-51718</td>\n",
       "      <td>25</td>\n",
       "      <td>Male</td>\n",
       "      <td>White</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP-56267</td>\n",
       "      <td>25</td>\n",
       "      <td>Male</td>\n",
       "      <td>White</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  participant_id  age_at_baseline   sex   race\n",
       "0       PP-41564               22  Male  White\n",
       "1  PD-PDZV843ATF               24  Male  White\n",
       "2  PD-PDCK871NBR               24  Male  White\n",
       "3       PP-51718               25  Male  White\n",
       "4       PP-56267               25  Male  White"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "demo.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "*Important: clinical duplicates*\n",
    "In the clinical data there remain samples that have been determined to be genetically identical (either twins or simply sample duplication across studies).\n",
    "\n",
    "The samples with lower coverage were removed from the WGS data. The two columns of the duplicated sample table below refer to the IDs of samples deleted from the WGS data (but still in the clinical data) and the IDs of samples that remain in the WGS data, respectively.\n",
    "\n",
    "It is very important before running analyses using just the clinical data to investigate duplicates or you may be including the same sample twice. In addition, some of the duplicated samples from different studies have differing diagnoses."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Downloading: 100%|██████████| 24/24 [00:00<00:00, 202.72rows/s]\n"
     ]
    }
   ],
   "source": [
    "dups = f\"\"\"\n",
    "\n",
    "SELECT \n",
    "*\n",
    "\n",
    "FROM `{BQ_RELEASE_DATASET}.duplicate_subjects`\n",
    "\n",
    "\"\"\"\n",
    "\n",
    "\n",
    "duplicates = bq_query(dups)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>deleted_participant_id</th>\n",
       "      <th>wgs_participant_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>BF-1091</td>\n",
       "      <td>PD-PDEB612LPE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>BF-1098</td>\n",
       "      <td>PD-PDUA781LH0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>BF-1125</td>\n",
       "      <td>PD-PDWY520TTB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>HB-PD_INVFA733YEF</td>\n",
       "      <td>BF-1130</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>HB-PD_INVDZ260AW1</td>\n",
       "      <td>PD-PDFK551GW2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>HB-PD_INVEP649ZE5</td>\n",
       "      <td>PD-PDGJ885TBH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>HB-PD_INVME305HYH</td>\n",
       "      <td>PD-PDPV262JLC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>HB-PD_INVWE905KPJ</td>\n",
       "      <td>PD-PDKF434UCX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>HB-PD_INVNE231ZUD</td>\n",
       "      <td>PD-PDBR042ZHH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>PD-PDBW494GHE</td>\n",
       "      <td>BF-1100</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>HB-PD_INVRU618MM4</td>\n",
       "      <td>PD-PDCE881ME6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>HB-PD_INVXC454FPX</td>\n",
       "      <td>PD-PDDW009MK9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>PD-PDGU974LXR</td>\n",
       "      <td>PP-3451</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>HB-PD_INVXA119CHN</td>\n",
       "      <td>PD-PDGX770XYG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>PD-PDJV686AAB</td>\n",
       "      <td>BF-1088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>PD-PDKV712AWZ</td>\n",
       "      <td>PP-3615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>PD-PDTN576BY4</td>\n",
       "      <td>BF-1132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>PD-PDUH505FEZ</td>\n",
       "      <td>PP-3612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>PD-PDVF405WVV</td>\n",
       "      <td>PP-3307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>PP-50463</td>\n",
       "      <td>PD-PDZT881XTK</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>PP-50621</td>\n",
       "      <td>PD-PDBD199NY6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>PP-51632</td>\n",
       "      <td>PD-PDAC481KNB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>PP-53639</td>\n",
       "      <td>PD-PDYP676BLT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>PP-54916</td>\n",
       "      <td>PD-PDMU756PTF</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   deleted_participant_id wgs_participant_id\n",
       "0                 BF-1091      PD-PDEB612LPE\n",
       "1                 BF-1098      PD-PDUA781LH0\n",
       "2                 BF-1125      PD-PDWY520TTB\n",
       "3       HB-PD_INVFA733YEF            BF-1130\n",
       "4       HB-PD_INVDZ260AW1      PD-PDFK551GW2\n",
       "5       HB-PD_INVEP649ZE5      PD-PDGJ885TBH\n",
       "6       HB-PD_INVME305HYH      PD-PDPV262JLC\n",
       "7       HB-PD_INVWE905KPJ      PD-PDKF434UCX\n",
       "8       HB-PD_INVNE231ZUD      PD-PDBR042ZHH\n",
       "9           PD-PDBW494GHE            BF-1100\n",
       "10      HB-PD_INVRU618MM4      PD-PDCE881ME6\n",
       "11      HB-PD_INVXC454FPX      PD-PDDW009MK9\n",
       "12          PD-PDGU974LXR            PP-3451\n",
       "13      HB-PD_INVXA119CHN      PD-PDGX770XYG\n",
       "14          PD-PDJV686AAB            BF-1088\n",
       "15          PD-PDKV712AWZ            PP-3615\n",
       "16          PD-PDTN576BY4            BF-1132\n",
       "17          PD-PDUH505FEZ            PP-3612\n",
       "18          PD-PDVF405WVV            PP-3307\n",
       "19               PP-50463      PD-PDZT881XTK\n",
       "20               PP-50621      PD-PDBD199NY6\n",
       "21               PP-51632      PD-PDAC481KNB\n",
       "22               PP-53639      PD-PDYP676BLT\n",
       "23               PP-54916      PD-PDMU756PTF"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "duplicates"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Querying Variant Data\n",
    "\n",
    "Information below is retreived from the GP2 introduction to work on Terra. We will not practice here but the infortmaion can be good to have read trough. \n",
    "\n",
    "\"Now we will practice querying variant data. You should be more careful when querying this data, as it is much larger than the clinical datasets and might get expensive if you make multiple naive queries. In most cases, you should be querying from the gatk_passing_variants table, as this has been reformated to make queries less expensive. More info on the gatk_passing_variants is in the AMP-PD Getting Started notebooks:\n",
    "\n",
    "The passing_variants table includes output from:\n",
    "\n",
    "- The Broad Institute's joint genotyping workflow\n",
    "- Variants annotated with the Variant Effect Predictor (VEP)\n",
    "- The resulting VCFs from the above process were imported to BigQuery using the Variant Transforms pipeline. \n",
    "\n",
    "Further, the table was compacted and reshaped with:\n",
    "\n",
    "- Variant calls matching the reference (genotype 0, 0) moved to a column (hom_ref_call)\n",
    "- This column contbains only the sample identifiers, read statistics are omitted\n",
    "- Variant calls which were no-calls (genotype -1, -1) moved to a column (no_call)\n",
    "- This column contains only the sample identifiers read statistics are omitted\n",
    "- Variants not matching FILTER = PASS have been omitted\n",
    "Best practice is to target your variant queries. This means looking at the specific regions or variants you're interested in rather than querying the whole datatset.\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Pulling Data from Google Storage Buckets"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Querying is a quick and easy way to pull the data you're interested in, but you can also read certain datasets directly from the Google Cloud Storage Buckets.\n",
    "You can submit the command below to the shell using our path variables to look at the Google Bucket locations for the clinical data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executing: gsutil -u ipdgctraineeproject-ric3 ls gs://amp-pd-data/releases/2019_v1release_1015/clinical\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_CSF_abeta_tau_ptau.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_CSF_abeta_tau_ptau_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_CSF_beta_glucocerebrosidase.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_CSF_beta_glucocerebrosidase_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_SomaLogic_plasma.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_SomaLogic_plasma_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_other.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Biospecimen_analyses_other_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Caffeine_history.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Caffeine_history_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DTI.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DTI_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DaTSCAN_SBR.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DaTSCAN_SBR_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DaTSCAN_visual_interpretation.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/DaTSCAN_visual_interpretation_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Demographics.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Demographics_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Enrollment.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Enrollment_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Epworth_Sleepiness_Scale.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Epworth_Sleepiness_Scale_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Family_History_PD.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Family_History_PD_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_I.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_II.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_III.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_III_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_II_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_IV.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_IV_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MDS_UPDRS_Part_I_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MMSE.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MMSE_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MOCA.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MOCA_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MRI.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/MRI_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Modified_Schwab___England_ADL.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Modified_Schwab___England_ADL_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/PDQ_39.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/PDQ_39_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/PD_Medical_History.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/PD_Medical_History_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/REM_Sleep_Behavior_Disorder_Questionnaire_Mayo.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/REM_Sleep_Behavior_Disorder_Questionnaire_Mayo_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/REM_Sleep_Behavior_Disorder_Questionnaire_Stiasny_Kolster.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/REM_Sleep_Behavior_Disorder_Questionnaire_Stiasny_Kolster_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Smoking_and_alcohol_history.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/Smoking_and_alcohol_history_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/UPDRS.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/UPDRS_dictionary.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/UPSIT.csv\n",
      "gs://amp-pd-data/releases/2019_v1release_1015/clinical/UPSIT_dictionary.csv\n"
     ]
    }
   ],
   "source": [
    "shell_do(f'gsutil -u {BILLING_PROJECT_ID} ls {GS_CLINICAL_RELEASE_PATH}')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's read in the Demographics CSV"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "demographics_df = gcs_read_csv(os.path.join(GS_CLINICAL_RELEASE_PATH, 'Demographics.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 4298 entries, 0 to 4297\n",
      "Data columns (total 9 columns):\n",
      "participant_id           4298 non-null object\n",
      "GUID                     2688 non-null object\n",
      "visit_name               4298 non-null object\n",
      "visit_month              4298 non-null int64\n",
      "age_at_baseline          4298 non-null int64\n",
      "sex                      4298 non-null object\n",
      "ethnicity                4297 non-null object\n",
      "race                     4297 non-null object\n",
      "education_level_years    4297 non-null object\n",
      "dtypes: int64(2), object(7)\n",
      "memory usage: 302.3+ KB\n"
     ]
    }
   ],
   "source": [
    "demographics_df.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>participant_id</th>\n",
       "      <th>GUID</th>\n",
       "      <th>visit_name</th>\n",
       "      <th>visit_month</th>\n",
       "      <th>age_at_baseline</th>\n",
       "      <th>sex</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>race</th>\n",
       "      <th>education_level_years</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PD-PDNE299YPT</td>\n",
       "      <td>PDNE299YPT</td>\n",
       "      <td>M0</td>\n",
       "      <td>0</td>\n",
       "      <td>74</td>\n",
       "      <td>Male</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>White</td>\n",
       "      <td>Greater than 16 years</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PD-PDYW828VAV</td>\n",
       "      <td>PDYW828VAV</td>\n",
       "      <td>M0</td>\n",
       "      <td>0</td>\n",
       "      <td>57</td>\n",
       "      <td>Female</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>White</td>\n",
       "      <td>12-16 years</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PD-PDWV958JPG</td>\n",
       "      <td>PDWV958JPG</td>\n",
       "      <td>M0</td>\n",
       "      <td>0</td>\n",
       "      <td>68</td>\n",
       "      <td>Male</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>White</td>\n",
       "      <td>Greater than 16 years</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PD-PDRF387ZV3</td>\n",
       "      <td>PDRF387ZV3</td>\n",
       "      <td>M0</td>\n",
       "      <td>0</td>\n",
       "      <td>68</td>\n",
       "      <td>Male</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>White</td>\n",
       "      <td>12-16 years</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PD-PDAC268KWV</td>\n",
       "      <td>PDAC268KWV</td>\n",
       "      <td>M0</td>\n",
       "      <td>0</td>\n",
       "      <td>69</td>\n",
       "      <td>Female</td>\n",
       "      <td>Not Hispanic or Latino</td>\n",
       "      <td>White</td>\n",
       "      <td>Greater than 16 years</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  participant_id        GUID visit_name  visit_month  age_at_baseline     sex  \\\n",
       "0  PD-PDNE299YPT  PDNE299YPT         M0            0               74    Male   \n",
       "1  PD-PDYW828VAV  PDYW828VAV         M0            0               57  Female   \n",
       "2  PD-PDWV958JPG  PDWV958JPG         M0            0               68    Male   \n",
       "3  PD-PDRF387ZV3  PDRF387ZV3         M0            0               68    Male   \n",
       "4  PD-PDAC268KWV  PDAC268KWV         M0            0               69  Female   \n",
       "\n",
       "                ethnicity   race  education_level_years  \n",
       "0  Not Hispanic or Latino  White  Greater than 16 years  \n",
       "1  Not Hispanic or Latino  White            12-16 years  \n",
       "2  Not Hispanic or Latino  White  Greater than 16 years  \n",
       "3  Not Hispanic or Latino  White            12-16 years  \n",
       "4  Not Hispanic or Latino  White  Greater than 16 years  "
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "demographics_df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Some data formats aren't queryable, such as CRAMS, VCF files, and FASTQs\n",
    "\n",
    "To use these data formats, we just need to copy the files to our cluster\n",
    "First let's look at the locations in the cloud for these files:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executing: gsutil -u ipdgctraineeproject-ric3 ls gs://amp-pd-genomics/releases/2019_v1release_1015/wgs\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/wgs_samples.csv\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/gatk/\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink2/\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/topmed/\n"
     ]
    }
   ],
   "source": [
    "shell_do(f'gsutil -u {BILLING_PROJECT_ID} ls {GS_WGS_RELEASE_PATH}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executing: gsutil -u ipdgctraineeproject-ric3 ls gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs*\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.bed\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.bim\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.fam\n",
      "gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.log\n"
     ]
    }
   ],
   "source": [
    "shell_do(f'gsutil -u {BILLING_PROJECT_ID} ls {GS_WGS_RELEASE_PLINK_PATH}/bfile/all_vcfs*')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we can practice copying data from the cloud storage to our cluster. We'll need the Plink binary files for our analysis in a bit, so let's copy those.\n",
    "\n",
    "First let's make a directory to copy it to."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "data_temp\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "mkdir ~/bin\n",
    "mkdir ~/bin/data_temp\n",
    "cd ~/bin/\n",
    "ls"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Copying the vcf plink files:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executing: gsutil -mu ipdgctraineeproject-ric3 cp gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs* ~/bin/data_temp\n",
      "Copying gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.bed...\n",
      "==> NOTE: You are downloading one or more large file(s), which would\n",
      "run significantly faster if you enabled sliced object downloads. This\n",
      "feature is enabled by default but requires that compiled crcmod be\n",
      "installed (see \"gsutil help crcmod\").\n",
      "\n",
      "Copying gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.bim...\n",
      "Copying gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.fam...\n",
      "Copying gs://amp-pd-genomics/releases/2019_v1release_1015/wgs/plink/bfile/all_vcfs.log...\n",
      "\\ [4/4 files][ 21.4 GiB/ 21.4 GiB] 100% Done     0.0 B/s                        \n",
      "Operation completed over 4 objects/21.4 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "shell_do(f'gsutil -mu {BILLING_PROJECT_ID} cp {GS_WGS_RELEASE_PLINK_PATH}/bfile/all_vcfs* ~/bin/data_temp')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "all_vcfs.bed\n",
      "all_vcfs.bim\n",
      "all_vcfs.fam\n",
      "all_vcfs.log\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/bin/data_temp\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BF-1001\tBF-1001\t0\t0\t0\t-9\n",
      "BF-1002\tBF-1002\t0\t0\t0\t-9\n",
      "BF-1003\tBF-1003\t0\t0\t0\t-9\n",
      "BF-1004\tBF-1004\t0\t0\t0\t-9\n",
      "BF-1005\tBF-1005\t0\t0\t0\t-9\n",
      "BF-1006\tBF-1006\t0\t0\t0\t-9\n",
      "BF-1008\tBF-1008\t0\t0\t0\t-9\n",
      "BF-1009\tBF-1009\t0\t0\t0\t-9\n",
      "BF-1010\tBF-1010\t0\t0\t0\t-9\n",
      "BF-1011\tBF-1011\t0\t0\t0\t-9\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/bin/data_temp\n",
    "head all_vcfs.fam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "--> We need to update the fam file"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Using the Plink binary files that we copied to our cluster previously, we'll do a Plink case/control association test. For this, we will be using the fisher flag, which performs Fisher's exact test. Fisher's test is used to determine if there are nonrandom associations between two variables. Our variables here are the presence of a Parkinon's Disease diagnosis (1 if yes, 0 if no), and the major and minor allele distribution for the tested variant. So for every variant in our data, we are testing whether the distribution of alleles is significantly different between the PD group and the non PD group.\n",
    "\n",
    "To do this you'll need a few extra things:\n",
    "\n",
    "Plink installed on your cluster\n",
    "Phenotype file\n",
    "Sex info file\n",
    "Covariate file\n",
    "Installing your own packages\n",
    "You can get package files ready for installation in different ways.\n",
    "Get the files from online, upload them to the workspace bucket, copy them to the cluster.\n",
    "Download straight from a code cell using commands like git clone.\n",
    "You will then follow the regular instructions for installation.\n",
    "We are going to use wget to grab Plink 1.9 for Linux from http://s3.amazonaws.com"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plink is not installed\n",
      "Archive:  plink_linux_x86_64_20190304.zip\n",
      "  inflating: plink                   \n",
      "  inflating: LICENSE                 \n",
      "  inflating: toy.ped                 \n",
      "  inflating: toy.map                 \n",
      "  inflating: prettify                \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2021-02-02 08:19:38--  http://s3.amazonaws.com/plink1-assets/plink_linux_x86_64_20190304.zip\n",
      "Resolving s3.amazonaws.com (s3.amazonaws.com)... 52.217.107.182\n",
      "Connecting to s3.amazonaws.com (s3.amazonaws.com)|52.217.107.182|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 8708135 (8.3M) [application/zip]\n",
      "Saving to: ‘plink_linux_x86_64_20190304.zip’\n",
      "\n",
      "     0K .......... .......... .......... .......... ..........  0%  839K 10s\n",
      "    50K .......... .......... .......... .......... ..........  1% 1.64M 8s\n",
      "   100K .......... .......... .......... .......... ..........  1%  160M 5s\n",
      "   150K .......... .......... .......... .......... ..........  2% 26.7M 4s\n",
      "   200K .......... .......... .......... .......... ..........  2% 1.73M 4s\n",
      "   250K .......... .......... .......... .......... ..........  3% 79.5M 3s\n",
      "   300K .......... .......... .......... .......... ..........  4% 57.2M 3s\n",
      "   350K .......... .......... .......... .......... ..........  4% 38.8M 2s\n",
      "   400K .......... .......... .......... .......... ..........  5% 1.81M 3s\n",
      "   450K .......... .......... .......... .......... ..........  5%  109M 2s\n",
      "   500K .......... .......... .......... .......... ..........  6%  149M 2s\n",
      "   550K .......... .......... .......... .......... ..........  7%  239M 2s\n",
      "   600K .......... .......... .......... .......... ..........  7%  125M 2s\n",
      "   650K .......... .......... .......... .......... ..........  8%  103M 2s\n",
      "   700K .......... .......... .......... .......... ..........  8%  268M 2s\n",
      "   750K .......... .......... .......... .......... ..........  9%  262M 1s\n",
      "   800K .......... .......... .......... .......... ..........  9% 57.5M 1s\n",
      "   850K .......... .......... .......... .......... .......... 10% 1.81M 2s\n",
      "   900K .......... .......... .......... .......... .......... 11%  139M 1s\n",
      "   950K .......... .......... .......... .......... .......... 11%  161M 1s\n",
      "  1000K .......... .......... .......... .......... .......... 12%  213M 1s\n",
      "  1050K .......... .......... .......... .......... .......... 12%  266M 1s\n",
      "  1100K .......... .......... .......... .......... .......... 13%  267M 1s\n",
      "  1150K .......... .......... .......... .......... .......... 14%  234M 1s\n",
      "  1200K .......... .......... .......... .......... .......... 14%  112M 1s\n",
      "  1250K .......... .......... .......... .......... .......... 15%  121M 1s\n",
      "  1300K .......... .......... .......... .......... .......... 15%  257M 1s\n",
      "  1350K .......... .......... .......... .......... .......... 16%  228M 1s\n",
      "  1400K .......... .......... .......... .......... .......... 17%  265M 1s\n",
      "  1450K .......... .......... .......... .......... .......... 17%  182M 1s\n",
      "  1500K .......... .......... .......... .......... .......... 18%  200M 1s\n",
      "  1550K .......... .......... .......... .......... .......... 18%  248M 1s\n",
      "  1600K .......... .......... .......... .......... .......... 19%  258M 1s\n",
      "  1650K .......... .......... .......... .......... .......... 19%  233M 1s\n",
      "  1700K .......... .......... .......... .......... .......... 20%  244M 1s\n",
      "  1750K .......... .......... .......... .......... .......... 21% 2.80M 1s\n",
      "  1800K .......... .......... .......... .......... .......... 21% 5.50M 1s\n",
      "  1850K .......... .......... .......... .......... .......... 22%  153M 1s\n",
      "  1900K .......... .......... .......... .......... .......... 22%  188M 1s\n",
      "  1950K .......... .......... .......... .......... .......... 23%  210M 1s\n",
      "  2000K .......... .......... .......... .......... .......... 24%  194M 1s\n",
      "  2050K .......... .......... .......... .......... .......... 24% 82.2M 1s\n",
      "  2100K .......... .......... .......... .......... .......... 25%  217M 1s\n",
      "  2150K .......... .......... .......... .......... .......... 25%  219M 1s\n",
      "  2200K .......... .......... .......... .......... .......... 26%  213M 1s\n",
      "  2250K .......... .......... .......... .......... .......... 27%  213M 1s\n",
      "  2300K .......... .......... .......... .......... .......... 27%  156M 1s\n",
      "  2350K .......... .......... .......... .......... .......... 28%  217M 1s\n",
      "  2400K .......... .......... .......... .......... .......... 28%  211M 1s\n",
      "  2450K .......... .......... .......... .......... .......... 29%  243M 1s\n",
      "  2500K .......... .......... .......... .......... .......... 29%  149M 0s\n",
      "  2550K .......... .......... .......... .......... .......... 30%  184M 0s\n",
      "  2600K .......... .......... .......... .......... .......... 31%  216M 0s\n",
      "  2650K .......... .......... .......... .......... .......... 31%  218M 0s\n",
      "  2700K .......... .......... .......... .......... .......... 32%  225M 0s\n",
      "  2750K .......... .......... .......... .......... .......... 32%  272M 0s\n",
      "  2800K .......... .......... .......... .......... .......... 33%  252M 0s\n",
      "  2850K .......... .......... .......... .......... .......... 34%  260M 0s\n",
      "  2900K .......... .......... .......... .......... .......... 34%  198M 0s\n",
      "  2950K .......... .......... .......... .......... .......... 35%  181M 0s\n",
      "  3000K .......... .......... .......... .......... .......... 35%  236M 0s\n",
      "  3050K .......... .......... .......... .......... .......... 36%  274M 0s\n",
      "  3100K .......... .......... .......... .......... .......... 37%  229M 0s\n",
      "  3150K .......... .......... .......... .......... .......... 37%  261M 0s\n",
      "  3200K .......... .......... .......... .......... .......... 38%  265M 0s\n",
      "  3250K .......... .......... .......... .......... .......... 38%  263M 0s\n",
      "  3300K .......... .......... .......... .......... .......... 39%  227M 0s\n",
      "  3350K .......... .......... .......... .......... .......... 39%  257M 0s\n",
      "  3400K .......... .......... .......... .......... .......... 40%  266M 0s\n",
      "  3450K .......... .......... .......... .......... .......... 41%  270M 0s\n",
      "  3500K .......... .......... .......... .......... .......... 41% 3.67M 0s\n",
      "  3550K .......... .......... .......... .......... .......... 42%  231M 0s\n",
      "  3600K .......... .......... .......... .......... .......... 42% 5.69M 0s\n",
      "  3650K .......... .......... .......... .......... .......... 43%  170M 0s\n",
      "  3700K .......... .......... .......... .......... .......... 44%  248M 0s\n",
      "  3750K .......... .......... .......... .......... .......... 44%  146M 0s\n",
      "  3800K .......... .......... .......... .......... .......... 45%  233M 0s\n",
      "  3850K .......... .......... .......... .......... .......... 45%  219M 0s\n",
      "  3900K .......... .......... .......... .......... .......... 46%  215M 0s\n",
      "  3950K .......... .......... .......... .......... .......... 47%  170M 0s\n",
      "  4000K .......... .......... .......... .......... .......... 47%  234M 0s\n",
      "  4050K .......... .......... .......... .......... .......... 48%  173M 0s\n",
      "  4100K .......... .......... .......... .......... .......... 48%  200M 0s\n",
      "  4150K .......... .......... .......... .......... .......... 49%  210M 0s\n",
      "  4200K .......... .......... .......... .......... .......... 49%  226M 0s\n",
      "  4250K .......... .......... .......... .......... .......... 50%  232M 0s\n",
      "  4300K .......... .......... .......... .......... .......... 51%  268M 0s\n",
      "  4350K .......... .......... .......... .......... .......... 51%  230M 0s\n",
      "  4400K .......... .......... .......... .......... .......... 52%  170M 0s\n",
      "  4450K .......... .......... .......... .......... .......... 52%  258M 0s\n",
      "  4500K .......... .......... .......... .......... .......... 53%  226M 0s\n",
      "  4550K .......... .......... .......... .......... .......... 54%  186M 0s\n",
      "  4600K .......... .......... .......... .......... .......... 54%  209M 0s\n",
      "  4650K .......... .......... .......... .......... .......... 55%  226M 0s\n",
      "  4700K .......... .......... .......... .......... .......... 55%  208M 0s\n",
      "  4750K .......... .......... .......... .......... .......... 56%  197M 0s\n",
      "  4800K .......... .......... .......... .......... .......... 57%  219M 0s\n",
      "  4850K .......... .......... .......... .......... .......... 57%  193M 0s\n",
      "  4900K .......... .......... .......... .......... .......... 58%  254M 0s\n",
      "  4950K .......... .......... .......... .......... .......... 58%  182M 0s\n",
      "  5000K .......... .......... .......... .......... .......... 59%  236M 0s\n",
      "  5050K .......... .......... .......... .......... .......... 59%  206M 0s\n",
      "  5100K .......... .......... .......... .......... .......... 60%  113M 0s\n",
      "  5150K .......... .......... .......... .......... .......... 61%  194M 0s\n",
      "  5200K .......... .......... .......... .......... .......... 61%  209M 0s\n",
      "  5250K .......... .......... .......... .......... .......... 62%  219M 0s\n",
      "  5300K .......... .......... .......... .......... .......... 62%  212M 0s\n",
      "  5350K .......... .......... .......... .......... .......... 63%  185M 0s\n",
      "  5400K .......... .......... .......... .......... .......... 64%  215M 0s\n",
      "  5450K .......... .......... .......... .......... .......... 64%  218M 0s\n",
      "  5500K .......... .......... .......... .......... .......... 65%  216M 0s\n",
      "  5550K .......... .......... .......... .......... .......... 65%  200M 0s\n",
      "  5600K .......... .......... .......... .......... .......... 66%  230M 0s\n",
      "  5650K .......... .......... .......... .......... .......... 67%  236M 0s\n",
      "  5700K .......... .......... .......... .......... .......... 67%  270M 0s\n",
      "  5750K .......... .......... .......... .......... .......... 68%  222M 0s\n",
      "  5800K .......... .......... .......... .......... .......... 68%  268M 0s\n",
      "  5850K .......... .......... .......... .......... .......... 69%  268M 0s\n",
      "  5900K .......... .......... .......... .......... .......... 69%  271M 0s\n",
      "  5950K .......... .......... .......... .......... .......... 70%  235M 0s\n",
      "  6000K .......... .......... .......... .......... .......... 71%  277M 0s\n",
      "  6050K .......... .......... .......... .......... .......... 71%  275M 0s\n",
      "  6100K .......... .......... .......... .......... .......... 72%  267M 0s\n",
      "  6150K .......... .......... .......... .......... .......... 72%  238M 0s\n",
      "  6200K .......... .......... .......... .......... .......... 73%  260M 0s\n",
      "  6250K .......... .......... .......... .......... .......... 74% 5.24M 0s\n",
      "  6300K .......... .......... .......... .......... .......... 74%  230M 0s\n",
      "  6350K .......... .......... .......... .......... .......... 75%  194M 0s\n",
      "  6400K .......... .......... .......... .......... .......... 75%  217M 0s\n",
      "  6450K .......... .......... .......... .......... .......... 76%  195M 0s\n",
      "  6500K .......... .......... .......... .......... .......... 77%  252M 0s\n",
      "  6550K .......... .......... .......... .......... .......... 77%  234M 0s\n",
      "  6600K .......... .......... .......... .......... .......... 78% 6.78M 0s\n",
      "  6650K .......... .......... .......... .......... .......... 78%  184M 0s\n",
      "  6700K .......... .......... .......... .......... .......... 79%  220M 0s\n",
      "  6750K .......... .......... .......... .......... .......... 79%  208M 0s\n",
      "  6800K .......... .......... .......... .......... .......... 80%  184M 0s\n",
      "  6850K .......... .......... .......... .......... .......... 81%  205M 0s\n",
      "  6900K .......... .......... .......... .......... .......... 81%  207M 0s\n",
      "  6950K .......... .......... .......... .......... .......... 82%  217M 0s\n",
      "  7000K .......... .......... .......... .......... .......... 82%  222M 0s\n",
      "  7050K .......... .......... .......... .......... .......... 83%  164M 0s\n",
      "  7100K .......... .......... .......... .......... .......... 84%  222M 0s\n",
      "  7150K .......... .......... .......... .......... .......... 84%  239M 0s\n",
      "  7200K .......... .......... .......... .......... .......... 85%  211M 0s\n",
      "  7250K .......... .......... .......... .......... .......... 85%  183M 0s\n",
      "  7300K .......... .......... .......... .......... .......... 86%  272M 0s\n",
      "  7350K .......... .......... .......... .......... .......... 87% 99.6M 0s\n",
      "  7400K .......... .......... .......... .......... .......... 87%  217M 0s\n",
      "  7450K .......... .......... .......... .......... .......... 88%  247M 0s\n",
      "  7500K .......... .......... .......... .......... .......... 88%  181M 0s\n",
      "  7550K .......... .......... .......... .......... .......... 89%  232M 0s\n",
      "  7600K .......... .......... .......... .......... .......... 89%  250M 0s\n",
      "  7650K .......... .......... .......... .......... .......... 90%  203M 0s\n",
      "  7700K .......... .......... .......... .......... .......... 91%  228M 0s\n",
      "  7750K .......... .......... .......... .......... .......... 91%  240M 0s\n",
      "  7800K .......... .......... .......... .......... .......... 92%  246M 0s\n",
      "  7850K .......... .......... .......... .......... .......... 92%  186M 0s\n",
      "  7900K .......... .......... .......... .......... .......... 93%  246M 0s\n",
      "  7950K .......... .......... .......... .......... .......... 94%  252M 0s\n",
      "  8000K .......... .......... .......... .......... .......... 94%  213M 0s\n",
      "  8050K .......... .......... .......... .......... .......... 95%  233M 0s\n",
      "  8100K .......... .......... .......... .......... .......... 95%  258M 0s\n",
      "  8150K .......... .......... .......... .......... .......... 96% 1.54M 0s\n",
      "  8200K .......... .......... .......... .......... .......... 97%  147M 0s\n",
      "  8250K .......... .......... .......... .......... .......... 97%  145M 0s\n",
      "  8300K .......... .......... .......... .......... .......... 98%  206M 0s\n",
      "  8350K .......... .......... .......... .......... .......... 98%  257M 0s\n",
      "  8400K .......... .......... .......... .......... .......... 99%  198M 0s\n",
      "  8450K .......... .......... .......... .......... .......... 99%  191M 0s\n",
      "  8500K ....                                                  100% 7702G=0.3s\n",
      "\n",
      "2021-02-02 08:19:39 (26.8 MB/s) - ‘plink_linux_x86_64_20190304.zip’ saved [8708135/8708135]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "if test -e ~/bin/plink; then\n",
    "\n",
    "echo \"Plink is already installed in ~/bin\"\n",
    "else\n",
    "echo \"Plink is not installed\"\n",
    "cd ~/bin/\n",
    "\n",
    "wget http://s3.amazonaws.com/plink1-assets/plink_linux_x86_64_20190304.zip \n",
    "\n",
    "unzip -o plink_linux_x86_64_20190304.zip\n",
    "\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\r\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\r\n",
      "\r\n",
      "  plink <input flag(s)...> [command flag(s)...] [other flag(s)...]\r\n",
      "  plink --help [flag name(s)...]\r\n",
      "\r\n",
      "Commands include --make-bed, --recode, --flip-scan, --merge-list,\r\n",
      "--write-snplist, --list-duplicate-vars, --freqx, --missing, --test-mishap,\r\n",
      "--hardy, --mendel, --ibc, --impute-sex, --indep-pairphase, --r2, --show-tags,\r\n",
      "--blocks, --distance, --genome, --homozyg, --make-rel, --make-grm-gz,\r\n",
      "--rel-cutoff, --cluster, --pca, --neighbour, --ibs-test, --regress-distance,\r\n",
      "--model, --bd, --gxe, --logistic, --dosage, --lasso, --test-missing,\r\n",
      "--make-perm-pheno, --tdt, --qfam, --annotate, --clump, --gene-report,\r\n",
      "--meta-analysis, --epistasis, --fast-epistasis, and --score.\r\n",
      "\r\n",
      "'plink --help | more' describes all functions (warning: long).\r\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we need our phenotype and covariate files. I'm going to just query a few data fields like we did before. I'm just going to grab diagnosis and sex for this quick example.\n",
    "\n",
    "Tip: Get to know the cohorts before running analyses, cohorts like PPMI have study arms that are enriched for certain variants and you may need to exclude those samples\n",
    "\n",
    "We need to grab sex information to update the Plink fam file, as it currently does not have sex coded into it. Chi squared tests or Fisher's tests like Plink's association tests don't use covariates, but Plink still needs sex info or it sets those samples to missing. We'll grab the sex info from the demographics file and use it to update the Plink fam file.\n",
    "\n",
    "Will also grab phenotype information and update the plink fam file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Downloading: 100%|██████████| 4298/4298 [00:00<00:00, 23411.36rows/s]\n"
     ]
    }
   ],
   "source": [
    "covariates = f\"\"\"\n",
    "\n",
    "SELECT \n",
    "participant_id, sex, age_at_baseline \n",
    "\n",
    "FROM `{BQ_RELEASE_DATASET}.Demographics`\n",
    "\n",
    "\"\"\"\n",
    "\n",
    "\n",
    "covs = bq_query(covariates)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>participant_id</th>\n",
       "      <th>sex</th>\n",
       "      <th>age_at_baseline</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PP-41564</td>\n",
       "      <td>Male</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PD-PDZV843ATF</td>\n",
       "      <td>Male</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PD-PDCK871NBR</td>\n",
       "      <td>Male</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PP-51718</td>\n",
       "      <td>Male</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP-56267</td>\n",
       "      <td>Male</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  participant_id   sex  age_at_baseline\n",
       "0       PP-41564  Male               22\n",
       "1  PD-PDZV843ATF  Male               24\n",
       "2  PD-PDCK871NBR  Male               24\n",
       "3       PP-51718  Male               25\n",
       "4       PP-56267  Male               25"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "covs.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There is a case control table available that includes baseline and most recent diagnosis. Healthy samples are controls, any Parkinson's disease or idiopathic Parkinson's diagnsoses are classified as cases, and any other related diagnoses are classified as other."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Downloading: 100%|██████████| 4298/4298 [00:00<00:00, 14032.50rows/s]\n"
     ]
    }
   ],
   "source": [
    "phenotype = f\"\"\"\n",
    "\n",
    "SELECT \n",
    "* \n",
    "\n",
    "FROM `{BQ_RELEASE_DATASET}.amp_pd_case_control`\n",
    "\n",
    "\"\"\"\n",
    "\n",
    "pheno = bq_query(phenotype)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>participant_id</th>\n",
       "      <th>diagnosis_at_baseline</th>\n",
       "      <th>diagnosis_latest</th>\n",
       "      <th>case_control_other_at_baseline</th>\n",
       "      <th>case_control_other_latest</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PD-PDAB597LKL</td>\n",
       "      <td>Parkinsonism</td>\n",
       "      <td>Parkinsonism</td>\n",
       "      <td>Other</td>\n",
       "      <td>Other</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PD-PDCT406CN9</td>\n",
       "      <td>Parkinsonism</td>\n",
       "      <td>Parkinsonism</td>\n",
       "      <td>Other</td>\n",
       "      <td>Other</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PP-54461</td>\n",
       "      <td>Prodromal motor PD</td>\n",
       "      <td>Idiopathic PD</td>\n",
       "      <td>Other</td>\n",
       "      <td>Case</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PP-54215</td>\n",
       "      <td>Prodromal motor PD</td>\n",
       "      <td>Idiopathic PD</td>\n",
       "      <td>Other</td>\n",
       "      <td>Case</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP-18567</td>\n",
       "      <td>Prodromal motor PD</td>\n",
       "      <td>Idiopathic PD</td>\n",
       "      <td>Other</td>\n",
       "      <td>Case</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  participant_id diagnosis_at_baseline diagnosis_latest  \\\n",
       "0  PD-PDAB597LKL          Parkinsonism     Parkinsonism   \n",
       "1  PD-PDCT406CN9          Parkinsonism     Parkinsonism   \n",
       "2       PP-54461    Prodromal motor PD    Idiopathic PD   \n",
       "3       PP-54215    Prodromal motor PD    Idiopathic PD   \n",
       "4       PP-18567    Prodromal motor PD    Idiopathic PD   \n",
       "\n",
       "  case_control_other_at_baseline case_control_other_latest  \n",
       "0                          Other                     Other  \n",
       "1                          Other                     Other  \n",
       "2                          Other                      Case  \n",
       "3                          Other                      Case  \n",
       "4                          Other                      Case  "
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pheno.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To use Plink, we need to get our sex info file and phenotype file in Plink format, which means 2 columns of IDs followed by the other information."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "covsp = covs\n",
    "covsp['IID'] = covsp['participant_id']\n",
    "covsp = covsp[['participant_id', 'IID', 'sex', 'age_at_baseline']]\n",
    "covsp['sex'] = covsp['sex'].astype(str)\n",
    "covsp.sex[(covsp.sex == \"Male\")] = 1\n",
    "covsp.sex[(covsp.sex == \"Female\")] = 2\n",
    "covsp.columns = ['FID','IID', 'sex', 'age_at_baseline']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>FID</th>\n",
       "      <th>IID</th>\n",
       "      <th>sex</th>\n",
       "      <th>age_at_baseline</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PP-41564</td>\n",
       "      <td>PP-41564</td>\n",
       "      <td>1</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PD-PDZV843ATF</td>\n",
       "      <td>PD-PDZV843ATF</td>\n",
       "      <td>1</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PD-PDCK871NBR</td>\n",
       "      <td>PD-PDCK871NBR</td>\n",
       "      <td>1</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PP-51718</td>\n",
       "      <td>PP-51718</td>\n",
       "      <td>1</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP-56267</td>\n",
       "      <td>PP-56267</td>\n",
       "      <td>1</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "             FID            IID sex  age_at_baseline\n",
       "0       PP-41564       PP-41564   1               22\n",
       "1  PD-PDZV843ATF  PD-PDZV843ATF   1               24\n",
       "2  PD-PDCK871NBR  PD-PDCK871NBR   1               24\n",
       "3       PP-51718       PP-51718   1               25\n",
       "4       PP-56267       PP-56267   1               25"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "covsp.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "phenop = pheno\n",
    "phenop['IID'] = phenop['participant_id']\n",
    "phenop = phenop[['participant_id', 'IID', 'case_control_other_latest']]\n",
    "phenop.columns = ['FID','IID', 'PHENO']\n",
    "phenop = phenop[(phenop.PHENO != 'Other')]\n",
    "phenop.PHENO[(phenop.PHENO == \"Case\")] = 2\n",
    "phenop.PHENO[(phenop.PHENO == \"Control\")] = 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>FID</th>\n",
       "      <th>IID</th>\n",
       "      <th>PHENO</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PP-54461</td>\n",
       "      <td>PP-54461</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PP-54215</td>\n",
       "      <td>PP-54215</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP-18567</td>\n",
       "      <td>PP-18567</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>PP-40615</td>\n",
       "      <td>PP-40615</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>PP-40612</td>\n",
       "      <td>PP-40612</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        FID       IID PHENO\n",
       "2  PP-54461  PP-54461     2\n",
       "3  PP-54215  PP-54215     2\n",
       "4  PP-18567  PP-18567     2\n",
       "5  PP-40615  PP-40615     2\n",
       "6  PP-40612  PP-40612     2"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "phenop.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's write these to tab files so we can use them with Plink"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('plink_test_covs.tab', 'w') as f:\n",
    "    f.write(covsp.to_csv(index=False, sep='\\t'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('plink_test_pheno.tab', 'w') as f:\n",
    "    f.write(phenop.to_csv(index=False, sep='\\t'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we'll use Plink's --update-sex command to update the fam file with the information we queried from the demographics file.(It will be a mix of running plink in python vs bash below, sorry abut that)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executing: ~/bin/plink --bfile ~/bin/data_temp/all_vcfs             --update-sex /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit/plink_test_covs.tab             --make-bed             --out ~/bin/data_temp/plink_updated_sex             --threads 2\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/jupyter-user/bin/data_temp/plink_updated_sex.log.\n",
      "Options in effect:\n",
      "  --bfile /home/jupyter-user/bin/data_temp/all_vcfs\n",
      "  --make-bed\n",
      "  --out /home/jupyter-user/bin/data_temp/plink_updated_sex\n",
      "  --threads 2\n",
      "  --update-sex /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit/plink_test_covs.tab\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "3074 people (0 males, 0 females, 3074 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/jupyter-user/bin/data_temp/plink_updated_sex.nosex .\n",
      "--update-sex: 3074 people updated, 1225 IDs not present.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 3074 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 14673747 het. haploid genotypes present (see\n",
      "/home/jupyter-user/bin/data_temp/plink_updated_sex.hh ); many commands treat\n",
      "these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993972.\n",
      "28791969 variants and 3074 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to /home/jupyter-user/bin/data_temp/plink_updated_sex.bed +\n",
      "/home/jupyter-user/bin/data_temp/plink_updated_sex.bim +\n",
      "/home/jupyter-user/bin/data_temp/plink_updated_sex.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "shell_do(f'~/bin/plink --bfile ~/bin/data_temp/all_vcfs \\\n",
    "            --update-sex /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit/plink_test_covs.tab \\\n",
    "            --make-bed \\\n",
    "            --out ~/bin/data_temp/plink_updated_sex \\\n",
    "            --threads 2')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Update phenotypes and remove individuals with missing phenotypes. \n",
    "It might be that this is not needed if you use the --pheno flag and a file with information on phenotypes for your analyses but I choose to add it here"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Information from plink website:\n",
    "https://www.cog-genomics.org/plink/1.9/input#pheno_encoding\n",
    "Case/control phenotypes are expected to be encoded as 1=unaffected (control), 2=affected (case); 0 is accepted as an alternate missing value encoding. If you use the --1 flag, 0 is interpreted as unaffected status instead, while 1 maps to affected. This also forces phenotypes to be interpreted as case/control.\n",
    "\n",
    "Case/control phenotype generation\n",
    "--make-pheno <filename> <value>\n",
    "\n",
    "Given a text file listing family and individual IDs in the first two columns, \"--make-pheno [filename] '*'\" designates all samples listed in the named file as cases, and all other samples as controls.\n",
    "\n",
    "If the named file has a third column, and a value other than '*' is given, --make-pheno will designate all samples with third column entry equal to the given value as cases, all other samples mentioned in the file as controls, and all samples missing from the file as having missing phenotypes.\n",
    "\n",
    "Use the plink_test_pheno.tab for phenotype info and remove individuals with missing phenotypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_updated_pheno.log.\n",
      "Options in effect:\n",
      "  --bfile /home/jupyter-user/bin/data_temp/plink_updated_sex\n",
      "  --keep plink_test_pheno.tab\n",
      "  --make-bed\n",
      "  --make-pheno plink_test_pheno.tab 2\n",
      "  --out AMPPD_updated_pheno\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "3074 people (1721 males, 1353 females) loaded from .fam.\n",
      "--make-pheno: 2835 phenotype values set.\n",
      "--keep: 2835 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see AMPPD_updated_pheno.hh );\n",
      "many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate in remaining samples is 0.993967.\n",
      "28791969 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--make-bed to AMPPD_updated_pheno.bed + AMPPD_updated_pheno.bim +\n",
      "AMPPD_updated_pheno.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile ~/bin/data_temp/plink_updated_sex --make-pheno plink_test_pheno.tab 2 --keep plink_test_pheno.tab --make-bed --out AMPPD_updated_pheno"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "From GP2 Introduction to Terra course:\n",
    "\"Best practice would be to perform variant QC beforehand, such as removing missing by case/control and filtering controls by Hardy-Weinberg equilibrium. No variant QC has been done to these plink files other than removing variants that failed GATK pipeline QC, so that specific choices could be left up to individual researchers.\"\n",
    "\n",
    "We decided to run a GWAS QC according to GP2s instructions"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##Sample QC - Genotyping call rates"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Calculate genotyping call rates per sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to call_rates.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_updated_pheno\n",
      "  --missing\n",
      "  --out call_rates\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see call_rates.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "--missing: Sample missing data report written to call_rates.imiss, and\n",
      "variant-based missing data report written to call_rates.lmiss.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_updated_pheno --missing --out call_rates"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Remove call rate outliers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_call_rate.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_updated_pheno\n",
      "  --make-bed\n",
      "  --mind 0.05\n",
      "  --out AMPPD_call_rate\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "0 people removed due to missing genotype data (--mind).\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see AMPPD_call_rate.hh );\n",
      "many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "28791969 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--make-bed to AMPPD_call_rate.bed + AMPPD_call_rate.bim + AMPPD_call_rate.fam\n",
      "... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_updated_pheno --mind 0.05 --make-bed --out AMPPD_call_rate"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTES:\n",
    "\n",
    "All call rates outliers are in CALL_RATE_OUTLIERS.txt\n",
    "call_rates.imiss and F_MISS - 1 = callrate\n",
    "All call rates saved in CALL_RATES_ALL_SAMPLES.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Sample QC - Heterozygosity"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruning.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_updated_pheno\n",
      "  --geno 0.01\n",
      "  --indep-pairwise 50 5 0.5\n",
      "  --maf 0.05\n",
      "  --out pruning\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see pruning.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "907891 variants removed due to missing genotype data (--geno).\n",
      "20980257 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "6903821 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "Pruned 440824 variants from chromosome 1, leaving 66482.\n",
      "Pruned 483200 variants from chromosome 2, leaving 71022.\n",
      "Pruned 414807 variants from chromosome 3, leaving 59944.\n",
      "Pruned 431946 variants from chromosome 4, leaving 59947.\n",
      "Pruned 374443 variants from chromosome 5, leaving 52358.\n",
      "Pruned 392015 variants from chromosome 6, leaving 54834.\n",
      "Pruned 345760 variants from chromosome 7, leaving 52196.\n",
      "Pruned 318142 variants from chromosome 8, leaving 44936.\n",
      "Pruned 253900 variants from chromosome 9, leaving 46443.\n",
      "Pruned 299603 variants from chromosome 10, leaving 46508.\n",
      "Pruned 291086 variants from chromosome 11, leaving 41573.\n",
      "Pruned 279667 variants from chromosome 12, leaving 41618.\n",
      "Pruned 215979 variants from chromosome 13, leaving 31091.\n",
      "Pruned 188658 variants from chromosome 14, leaving 29441.\n",
      "Pruned 166403 variants from chromosome 15, leaving 28463.\n",
      "Pruned 174555 variants from chromosome 16, leaving 32539.\n",
      "Pruned 152569 variants from chromosome 17, leaving 27614.\n",
      "Pruned 164894 variants from chromosome 18, leaving 26201.\n",
      "Pruned 134964 variants from chromosome 19, leaving 23930.\n",
      "Pruned 126931 variants from chromosome 20, leaving 22099.\n",
      "Pruned 87203 variants from chromosome 21, leaving 20024.\n",
      "Pruned 78294 variants from chromosome 22, leaving 16434.\n",
      "Pruned 167835 variants from chromosome 23, leaving 21398.\n",
      "Pruned 2722 variants from chromosome 24, leaving 326.\n",
      "Pruning complete.  5986400 of 6903821 variants removed.\n",
      "Marker lists written to pruning.prune.in and pruning.prune.out .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruned_data.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_updated_pheno\n",
      "  --extract pruning.prune.in\n",
      "  --make-bed\n",
      "  --out pruned_data\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "--extract: 917421 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 700164 het. haploid genotypes present (see pruned_data.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.99968.\n",
      "917421 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--make-bed to pruned_data.bed + pruned_data.bim + pruned_data.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to prunedHet.log.\n",
      "Options in effect:\n",
      "  --bfile pruned_data\n",
      "  --het\n",
      "  --out prunedHet\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "917421 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 700164 het. haploid genotypes present (see prunedHet.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.99968.\n",
      "917421 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--het: 895697 variants scanned, report written to prunedHet.het .\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_updated_pheno --geno 0.01 --maf 0.05 --indep-pairwise 50 5 0.5 --out pruning\n",
    "!~/bin/plink --bfile AMPPD_updated_pheno --extract pruning.prune.in --make-bed --out pruned_data\n",
    "!~/bin/plink --bfile pruned_data --het --out prunedHet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "awk '{if ($6 <= -0.15) print $0 }' prunedHet.het > outliers1.txt\n",
    "awk '{if ($6 >= 0.15) print $0 }' prunedHet.het > outliers2.txt\n",
    "cat outliers1.txt outliers2.txt > HETEROZYGOSITY_OUTLIERS.txt\n",
    "\n",
    "cut -f 1,2 HETEROZYGOSITY_OUTLIERS.txt > all_outliers.txt\n",
    "\n",
    "mv prunedHet.het HETEROZYGOSITY_DATA.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_heterozyg.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_call_rate\n",
      "  --make-bed\n",
      "  --out AMPPD_after_heterozyg\n",
      "  --remove all_outliers.txt\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "--remove: 2835 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see AMPPD_after_heterozyg.hh\n",
      "); many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "28791969 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--make-bed to AMPPD_after_heterozyg.bed + AMPPD_after_heterozyg.bim +\n",
      "AMPPD_after_heterozyg.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_call_rate --remove all_outliers.txt --make-bed --out AMPPD_after_heterozyg"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTES:\n",
    "\n",
    "--het from LD pruned data > use F cut-off of -0.15 and <- 0.15 for inclusion\n",
    "outliers stored here -> all_outliers.txt\n",
    "all heterozygosity is stored here -> HETEROZYGOSITY_DATA.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Sample QC - Gender checking"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to gender_check1.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_heterozyg\n",
      "  --check-sex 0.25 0.75\n",
      "  --maf 0.05\n",
      "  --out gender_check1\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see gender_check1.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "21346980 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "7444989 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--check-sex: 205764 Xchr and 0 Ychr variant(s) scanned, no problems detected.\n",
      "Report written to gender_check1.sexcheck .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to gender_check2.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_heterozyg\n",
      "  --check-sex 0.25 0.75\n",
      "  --chr 23\n",
      "  --from-bp 2781479\n",
      "  --geno 0.05\n",
      "  --hwe 1E-5\n",
      "  --maf 0.05\n",
      "  --out gender_check2\n",
      "  --to-bp 155701382\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "937718 out of 28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2835 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 2827837 het. haploid genotypes present (see gender_check2.hh ); many\n",
      "commands treat these as missing.\n",
      "Total genotyping rate is 0.994533.\n",
      "18077 variants removed due to missing genotype data (--geno).\n",
      "Warning: --hwe observation counts vary by more than 10%, due to the X\n",
      "chromosome.  You may want to use a less stringent --hwe p-value threshold for X\n",
      "chromosome variants.\n",
      "--hwe: 6645 variants removed due to Hardy-Weinberg exact test.\n",
      "719853 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "193143 variants and 2835 people pass filters and QC.\n",
      "Among remaining phenotypes, 1735 are cases and 1100 are controls.\n",
      "--check-sex: 193143 Xchr and 0 Ychr variant(s) scanned, 1 problem detected.\n",
      "Report written to gender_check2.sexcheck .\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_heterozyg --check-sex 0.25 0.75 --maf 0.05 --out gender_check1\n",
    "!~/bin/plink --bfile AMPPD_after_heterozyg --chr 23 --from-bp 2781479 --to-bp 155701382 --maf 0.05 --geno 0.05 --hwe 1E-5 --check-sex  0.25 0.75 --out gender_check2 "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTES:\n",
    "\n",
    "Gender failures are stored in GENDER_FAILURES.txt\n",
    "Gender checks are stored in GENDER_CHECK1.txt and GENDER_CHECK2.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "grep \"PROBLEM\" gender_check1.sexcheck > problems1.txt\n",
    "grep \"PROBLEM\" gender_check2.sexcheck > problems2.txt\n",
    "cat problems1.txt problems2.txt > GENDER_FAILURES.txt\n",
    "cut -f 1,2 GENDER_FAILURES.txt > samples_to_remove.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_heterozyg\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender\n",
      "  --remove samples_to_remove.txt\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2835 people (1577 males, 1258 females) loaded from .fam.\n",
      "2835 phenotype values loaded from .fam.\n",
      "--remove: 2834 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2834 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see AMPPD_after_gender.hh );\n",
      "many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate in remaining samples is 0.993967.\n",
      "28791969 variants and 2834 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.\n",
      "--make-bed to AMPPD_after_gender.bed + AMPPD_after_gender.bim +\n",
      "AMPPD_after_gender.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_heterozyg --remove samples_to_remove.txt --make-bed --out AMPPD_after_gender"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Sample QC - Ancestry"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTES\n",
    "No ancestry outliers -> based on Hapmap3 PCA plot, should be near combined CEU/TSI\n",
    "Keep in mind that this comparison with hapmap is based on the number of SNPs that overlap between your input dataset and hapmap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTE! You need to upload the Hapmap files manually to your data folder (make sure that the gemone build is the same!)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.fam...\n",
      "/ [0 files][    0.0 B/ 21.4 KiB]                                                \r",
      "/ [1 files][ 21.4 KiB/ 21.4 KiB]                                                \r\n",
      "Operation completed over 1 objects/21.4 KiB.                                     \n",
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bim...\n",
      "/ [0 files][    0.0 B/ 29.3 MiB]                                                \r",
      "/ [1 files][ 29.3 MiB/ 29.3 MiB]                                                \r\n",
      "Operation completed over 1 objects/29.3 MiB.                                     \n",
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bed...\n",
      "/ [0 files][    0.0 B/168.0 MiB]                                                \r",
      "==> NOTE: You are downloading one or more large file(s), which would\n",
      "run significantly faster if you enabled sliced object downloads. This\n",
      "feature is enabled by default but requires that compiled crcmod be\n",
      "installed (see \"gsutil help crcmod\").\n",
      "\n",
      "-\r",
      "- [0 files][102.9 MiB/168.0 MiB]                                                \r",
      "\\\r",
      "\\ [1 files][168.0 MiB/168.0 MiB]                                                \r\n",
      "Operation completed over 1 objects/168.0 MiB.                                    \n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.fam /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bim /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bed /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "=1.13.0                    HAPMAP_GRCh38.bim\r\n",
      "all_outliers.txt           HAPMAP_GRCh38.fam\r\n",
      "AMPPD_after_gender.bed     HETEROZYGOSITY_DATA.txt\r\n",
      "AMPPD_after_gender.bim     HETEROZYGOSITY_OUTLIERS.txt\r\n",
      "AMPPD_after_gender.fam     outliers1.txt\r\n",
      "AMPPD_after_gender.hh      outliers2.txt\r\n",
      "AMPPD_after_gender.log     plink_test_covs.tab\r\n",
      "AMPPD_after_heterozyg.bed  plink_test_pheno.tab\r\n",
      "AMPPD_after_heterozyg.bim  problems1.txt\r\n",
      "AMPPD_after_heterozyg.fam  problems2.txt\r\n",
      "AMPPD_after_heterozyg.hh   pruned_data.bed\r\n",
      "AMPPD_after_heterozyg.log  pruned_data.bim\r\n",
      "AMPPD_call_rate.bed        pruned_data.fam\r\n",
      "AMPPD_call_rate.bim        pruned_data.hh\r\n",
      "AMPPD_call_rate.fam        pruned_data.log\r\n",
      "AMPPD_call_rate.hh         prunedHet.hh\r\n",
      "AMPPD_call_rate.log        prunedHet.log\r\n",
      "AMPPD_updated_pheno.bed    pruning.hh\r\n",
      "AMPPD_updated_pheno.bim    pruning.log\r\n",
      "AMPPD_updated_pheno.fam    pruning.prune.in\r\n",
      "AMPPD_updated_pheno.hh     pruning.prune.out\r\n",
      "AMPPD_updated_pheno.log    Py3 - Clinical - load a CSV from Cloud Storage.ipynb\r\n",
      "call_rates.hh              Py3 - Clinical - load a table from BigQuery.ipynb\r\n",
      "call_rates.imiss           Py3 - Query a saved Data Explorer cohort.ipynb\r\n",
      "call_rates.lmiss           Py3 - Saving Notebook Results.ipynb\r\n",
      "call_rates.log             Py3 - Start Here.ipynb\r\n",
      "\u001b[0m\u001b[01;34mfacets\u001b[0m/                    Py3 - WGS -  Query Variants.ipynb\r\n",
      "gender_check1.hh           Py3 - WGS - Start Here.ipynb\r\n",
      "gender_check1.log          RIC3-Py-cleaned.ipynb\r\n",
      "gender_check1.sexcheck     RIC3-Py.ipynb\r\n",
      "gender_check2.hh           RIC3-Py_New.ipynb\r\n",
      "gender_check2.log          RIC3-R.ipynb\r\n",
      "gender_check2.sexcheck     R - Start Here.ipynb\r\n",
      "GENDER_FAILURES.txt        samples_to_remove.txt\r\n",
      "HAPMAP_GRCh38.bed\r\n"
     ]
    }
   ],
   "source": [
    "ls "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to hapmap3_bin_snplis.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender\n",
      "  --bmerge HAPMAP_GRCh38\n",
      "  --make-bed\n",
      "  --out hapmap3_bin_snplis\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "2834 people loaded from AMPPD_after_gender.fam.\n",
      "640 people to be merged from HAPMAP_GRCh38.fam.\n",
      "Of these, 640 are new, while 0 are present in the base dataset.\n",
      "Warning: Multiple positions seen for variant 'rs2234167'.\n",
      "Warning: Multiple positions seen for variant 'rs11573991'.\n",
      "Warning: Multiple positions seen for variant 'rs11573979'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs9753403'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs2228367'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs1640558'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs3976557'.\n",
      "28791969 markers loaded from AMPPD_after_gender.bim.\n",
      "1100686 markers to be merged from HAPMAP_GRCh38.bim.\n",
      "Of these, 9352 are new, while 1091334 are present in the base dataset.\n",
      "82 more multiple-position warnings: see log file.\n",
      "Error: 1647 variants with 3+ alleles present.\n",
      "* If you believe this is due to strand inconsistency, try --flip with\n",
      "  hapmap3_bin_snplis-merge.missnp.\n",
      "  (Warning: if this seems to work, strand errors involving SNPs with A/T or C/G\n",
      "  alleles probably remain in your data.  If LD between nearby SNPs is high,\n",
      "  --flip-scan should detect them.)\n",
      "* If you are dealing with genuine multiallelic variants, we recommend exporting\n",
      "  that subset of the data to VCF (via e.g. '--recode vcf'), merging with\n",
      "  another tool/script, and then importing the result; PLINK is not yet suited\n",
      "  to handling them.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to after_gender3.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender\n",
      "  --flip hapmap3_bin_snplis-merge.missnp\n",
      "  --make-bed\n",
      "  --out after_gender3\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2834 people (1577 males, 1257 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "--flip: 1647 SNPs flipped.\n",
      "Warning: 263 variants had at least one non-A/C/G/T allele name.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2834 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see after_gender3.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993967.\n",
      "28791969 variants and 2834 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.\n",
      "--make-bed to after_gender3.bed + after_gender3.bim + after_gender3.fam ...\n",
      "101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to hapmap3_bin_snplis.log.\n",
      "Options in effect:\n",
      "  --bfile after_gender3\n",
      "  --bmerge HAPMAP_GRCh38\n",
      "  --make-bed\n",
      "  --out hapmap3_bin_snplis\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "2834 people loaded from after_gender3.fam.\n",
      "640 people to be merged from HAPMAP_GRCh38.fam.\n",
      "Of these, 640 are new, while 0 are present in the base dataset.\n",
      "Warning: Multiple positions seen for variant 'rs2234167'.\n",
      "Warning: Multiple positions seen for variant 'rs11573991'.\n",
      "Warning: Multiple positions seen for variant 'rs11573979'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs9753403'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs2228367'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs1640558'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs3976557'.\n",
      "28791969 markers loaded from after_gender3.bim.\n",
      "1100686 markers to be merged from HAPMAP_GRCh38.bim.\n",
      "Of these, 9352 are new, while 1091334 are present in the base dataset.\n",
      "82 more multiple-position warnings: see log file.\n",
      "Error: 535 variants with 3+ alleles present.\n",
      "* If you believe this is due to strand inconsistency, try --flip with\n",
      "  hapmap3_bin_snplis-merge.missnp.\n",
      "  (Warning: if this seems to work, strand errors involving SNPs with A/T or C/G\n",
      "  alleles probably remain in your data.  If LD between nearby SNPs is high,\n",
      "  --flip-scan should detect them.)\n",
      "* If you are dealing with genuine multiallelic variants, we recommend exporting\n",
      "  that subset of the data to VCF (via e.g. '--recode vcf'), merging with\n",
      "  another tool/script, and then importing the result; PLINK is not yet suited\n",
      "  to handling them.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to after_gender4.log.\n",
      "Options in effect:\n",
      "  --bfile after_gender3\n",
      "  --exclude hapmap3_bin_snplis-merge.missnp\n",
      "  --make-bed\n",
      "  --out after_gender4\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2834 people (1577 males, 1257 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "--exclude: 28791434 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2834 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see after_gender4.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.993968.\n",
      "28791434 variants and 2834 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.\n",
      "--make-bed to after_gender4.bed + after_gender4.bim + after_gender4.fam ...\n",
      "101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to hapmap3_bin_snplis.log.\n",
      "Options in effect:\n",
      "  --bfile after_gender4\n",
      "  --bmerge HAPMAP_GRCh38\n",
      "  --make-bed\n",
      "  --out hapmap3_bin_snplis\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "2834 people loaded from after_gender4.fam.\n",
      "640 people to be merged from HAPMAP_GRCh38.fam.\n",
      "Of these, 640 are new, while 0 are present in the base dataset.\n",
      "Warning: Multiple positions seen for variant 'rs2234167'.\n",
      "Warning: Multiple positions seen for variant 'rs11573991'.\n",
      "Warning: Multiple positions seen for variant 'rs11573979'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs9753403'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs2228367'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs1640558'.\n",
      "Warning: Multiple chromosomes seen for variant 'rs3976557'.\n",
      "28791434 markers loaded from after_gender4.bim.\n",
      "1100686 markers to be merged from HAPMAP_GRCh38.bim.\n",
      "Of these, 9887 are new, while 1090799 are present in the base dataset.\n",
      "79 more multiple-position warnings: see log file.\n",
      "Warning: Variants 'rs55744711' and 'rs11240767' have the same position.\n",
      "Warning: Variants 'rs307368' and 'rs28475450' have the same position.\n",
      "Warning: Variants 'rs34186100' and 'rs2765017' have the same position.\n",
      "2430 more same-position warnings: see log file.\n",
      "Performing 2-pass merge (3474 people, 22634847/28801321 variants per pass).\n",
      "Pass 1 complete.                              \n",
      "Merged fileset written to hapmap3_bin_snplis-merge.bed +\n",
      "hapmap3_bin_snplis-merge.bim + hapmap3_bin_snplis-merge.fam .\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "28801321 variants loaded from .bim file.\n",
      "3474 people (1895 males, 1579 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 3474 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see hapmap3_bin_snplis.hh );\n",
      "many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.817621.\n",
      "28801321 variants and 3474 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.  (640\n",
      "phenotypes are missing.)\n",
      "--make-bed to hapmap3_bin_snplis.bed + hapmap3_bin_snplis.bim +\n",
      "hapmap3_bin_snplis.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruning.log.\n",
      "Options in effect:\n",
      "  --bfile hapmap3_bin_snplis\n",
      "  --geno 0.01\n",
      "  --indep-pairwise 50 5 0.5\n",
      "  --maf 0.05\n",
      "  --out pruning\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28801321 variants loaded from .bim file.\n",
      "3474 people (1895 males, 1579 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 3474 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 13444975 het. haploid genotypes present (see pruning.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.817621.\n",
      "27712567 variants removed due to missing genotype data (--geno).\n",
      "203907 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "884847 variants and 3474 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.  (640\n",
      "phenotypes are missing.)\n",
      "Pruned 50115 variants from chromosome 1, leaving 21672.\n",
      "Pruned 50891 variants from chromosome 2, leaving 21502.\n",
      "Pruned 42611 variants from chromosome 3, leaving 18616.\n",
      "Pruned 37661 variants from chromosome 4, leaving 16800.\n",
      "Pruned 39952 variants from chromosome 5, leaving 17321.\n",
      "Pruned 42334 variants from chromosome 6, leaving 17607.\n",
      "Pruned 33726 variants from chromosome 7, leaving 15396.\n",
      "Pruned 33188 variants from chromosome 8, leaving 14639.\n",
      "Pruned 27844 variants from chromosome 9, leaving 13295.\n",
      "Pruned 32869 variants from chromosome 10, leaving 14695.\n",
      "Pruned 32387 variants from chromosome 11, leaving 13581.\n",
      "Pruned 30574 variants from chromosome 12, leaving 14009.\n",
      "Pruned 23294 variants from chromosome 13, leaving 10582.\n",
      "Pruned 19706 variants from chromosome 14, leaving 9423.\n",
      "Pruned 17522 variants from chromosome 15, leaving 8861.\n",
      "Pruned 18249 variants from chromosome 16, leaving 9822.\n",
      "Pruned 14830 variants from chromosome 17, leaving 8348.\n",
      "Pruned 17921 variants from chromosome 18, leaving 8872.\n",
      "Pruned 10027 variants from chromosome 19, leaving 6089.\n",
      "Pruned 15150 variants from chromosome 20, leaving 7835.\n",
      "Pruned 8176 variants from chromosome 21, leaving 4352.\n",
      "Pruned 7926 variants from chromosome 22, leaving 4577.\n",
      "Pruning complete.  606953 of 884847 variants removed.\n",
      "Marker lists written to pruning.prune.in and pruning.prune.out .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruned_data.log.\n",
      "Options in effect:\n",
      "  --bfile hapmap3_bin_snplis\n",
      "  --extract pruning.prune.in\n",
      "  --make-bed\n",
      "  --out pruned_data\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28801321 variants loaded from .bim file.\n",
      "3474 people (1895 males, 1579 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "--extract: 277894 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 3474 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999717.\n",
      "277894 variants and 3474 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.  (640\n",
      "phenotypes are missing.)\n",
      "--make-bed to pruned_data.bed + pruned_data.bim + pruned_data.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to prunedHet.log.\n",
      "Options in effect:\n",
      "  --bfile pruned_data\n",
      "  --het\n",
      "  --out prunedHet\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "277894 variants loaded from .bim file.\n",
      "3474 people (1895 males, 1579 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 3474 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999717.\n",
      "277894 variants and 3474 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.  (640\n",
      "phenotypes are missing.)\n",
      "--het: 277894 variants scanned, report written to prunedHet.het .\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender --bmerge HAPMAP_GRCh38 --out hapmap3_bin_snplis --make-bed\n",
    "!~/bin/plink --bfile AMPPD_after_gender --flip hapmap3_bin_snplis-merge.missnp --make-bed --out after_gender3\n",
    "!~/bin/plink --bfile after_gender3 --bmerge HAPMAP_GRCh38 --out hapmap3_bin_snplis --make-bed\n",
    "!~/bin/plink --bfile after_gender3 --exclude hapmap3_bin_snplis-merge.missnp --out after_gender4 --make-bed\n",
    "!~/bin/plink --bfile after_gender4 --bmerge HAPMAP_GRCh38 --out hapmap3_bin_snplis --make-bed\n",
    "!~/bin/plink --bfile hapmap3_bin_snplis --geno 0.01 --maf 0.05 --indep-pairwise 50 5 0.5 --out pruning\n",
    "!~/bin/plink --bfile hapmap3_bin_snplis --extract pruning.prune.in --make-bed --out pruned_data\n",
    "!~/bin/plink --bfile pruned_data --het --out prunedHet "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pca.log.\n",
      "Options in effect:\n",
      "  --bfile pruned_data\n",
      "  --geno 0.01\n",
      "  --make-bed\n",
      "  --out pca\n",
      "  --pca\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "277894 variants loaded from .bim file.\n",
      "3474 people (1895 males, 1579 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "Using up to 15 threads (change this with --threads).\n",
      "Before main variant filters, 3474 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999717.\n",
      "0 variants removed due to missing genotype data (--geno).\n",
      "277894 variants and 3474 people pass filters and QC.\n",
      "Among remaining phenotypes, 1734 are cases and 1100 are controls.  (640\n",
      "phenotypes are missing.)\n",
      "Relationship matrix calculation complete.\n",
      "--pca: Results saved to pca.eigenval and pca.eigenvec .\n",
      "--make-bed to pca.bed + pca.bim + pca.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile pruned_data --geno 0.01 --out pca --make-bed --pca"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Files needed for the next step needs to be manually uploaded to data. These were downloaded from: https://github.com/neurogenetics/GWAS-pipeline/blob/master/ancestry_comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/eur_add.txt...\n",
      "/ [0 files][    0.0 B/  389.0 B]                                                \r",
      "/ [1 files][  389.0 B/  389.0 B]                                                \r\n",
      "Operation completed over 1 objects/389.0 B.                                      \n",
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/asia_add.txt...\n",
      "/ [0 files][    0.0 B/  493.0 B]                                                \r",
      "/ [1 files][  493.0 B/  493.0 B]                                                \r\n",
      "Operation completed over 1 objects/493.0 B.                                      \n",
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/afri_add.txt...\n",
      "/ [0 files][    0.0 B/  395.0 B]                                                \r",
      "/ [1 files][  395.0 B/  395.0 B]                                                \r\n",
      "Operation completed over 1 objects/395.0 B.                                      \n",
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/PCA_in_R.R...\n",
      "/ [0 files][    0.0 B/  3.3 KiB]                                                \r",
      "/ [1 files][  3.3 KiB/  3.3 KiB]                                                \r\n",
      "Operation completed over 1 objects/3.3 KiB.                                      \n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/eur_add.txt /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/asia_add.txt /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/afri_add.txt /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/PCA_in_R.R /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "grep \"EUROPE\" pca.eigenvec > eur.txt\n",
    "grep \"ASIA\" pca.eigenvec > asia.txt\n",
    "grep \"AFRICA\" pca.eigenvec > afri.txt\n",
    "grep -v -f eur.txt pca.eigenvec | grep -v -f asia.txt | grep -v -f afri.txt > new_samples.txt\n",
    "cut -d \" \" -f 3 AMPPD_after_gender.fam > new_samples_add.txt\n",
    "paste new_samples_add.txt new_samples.txt > new_samples2.txt\n",
    "paste eur_add.txt eur.txt > euro.txt\n",
    "paste asia_add.txt asia.txt > asiao.txt\n",
    "paste afri_add.txt afri.txt > afrio.txt\n",
    "cat new_samples2.txt euro.txt asiao.txt afrio.txt > pca.eigenvec2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I have used R to plot the PCA and filter the individuals according to the code below. Have not set it up to run directly in this notebook though. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "%%R\n",
    "##1) PCA\n",
    "\n",
    "#Running PCA script from: https://github.com/neurogenetics/GWAS-pipeline/tree/master/ancestry_comparison \n",
    "\n",
    "all_data <- read.table(\"pca.eigenvec2\", header = F, fill=T)\n",
    "all_data$V1[all_data$V1 == 0] <- 4\n",
    "all_data$V1[all_data$V1 == 1] <- 5\n",
    "\n",
    "attach(all_data)\n",
    "all_data <- all_data[order(V1),] \n",
    "\n",
    "pdf(\"raw_hapmap_plot.pdf\")\n",
    "plot(all_data$V4,all_data$V5,pch=16,col=all_data$V1,xlab=\"PC1\",ylab=\"PC2\")\n",
    "legend(\"bottomleft\",col=c(\"green3\",\"red\",\"blue\",\"cyan\"),pch=16,c(\"Africa\",\"Asia\",\"New samples\",\"Europe\"))\n",
    "dev.off()\n",
    "\n",
    "project <- Sys.getenv('WORKSPACE_NAMESPACE')\n",
    "workspace <- Sys.getenv('WORKSPACE_NAME')\n",
    "bucket <- Sys.getenv('WORKSPACE_BUCKET')\n",
    "\n",
    "euros <- read.table(\"euro.txt\", header = F)\n",
    "asians <- read.table(\"asiao.txt\", header = F)\n",
    "africans <- read.table(\"afrio.txt\", header = F)\n",
    "\n",
    "eurosLowC1 <- mean(euros$V4) - (6*sd(euros$V4))\n",
    "eurosHighC1 <- mean(euros$V4) + (6*sd(euros$V4))\n",
    "eurosLowC2 <- mean(euros$V5) - (6*sd(euros$V5))\n",
    "eurosHighC2 <- mean(euros$V5) + (6*sd(euros$V5))\n",
    "\n",
    "asiansLowC1 <- mean(asians$V4) - (6*sd(asians$V4))\n",
    "asiansHighC1 <- mean(asians$V4) + (6*sd(asians$V4))\n",
    "asiansLowC2 <- mean(asians$V5) - (6*sd(asians$V5))\n",
    "asiansHighC2 <- mean(asians$V5) + (6*sd(asians$V5))\n",
    "\n",
    "africansLowC1 <- mean(africans$V4) - (6*sd(africans$V4))\n",
    "africansHighC1 <- mean(africans$V4) + (6*sd(africans$V4))\n",
    "africansLowC2 <- mean(africans$V5) - (6*sd(africans$V5))\n",
    "africansHighC2 <- mean(africans$V5) + (6*sd(africans$V5))\n",
    "\n",
    "temp2 = all_data[(all_data$V4>=eurosLowC1),]\n",
    "temp3 = temp2[(temp2$V4<=eurosHighC1),]\n",
    "temp4 = temp3[(temp3$V5>=eurosLowC2),]\n",
    "EURO = temp4[(temp4$V5<=eurosHighC2),]\n",
    "\n",
    "pdf(\"EURO_confirmation_plot.pdf\")\n",
    "plot(EURO$V4,EURO$V5,pch=16,col=EURO$V1)\n",
    "legend(\"topright\",col=c(\"blue\",\"cyan\"),pch=16,c(\"New samples\",\"Europe\"))\n",
    "dev.off()\n",
    "\n",
    "temp2 = all_data[(all_data$V4>=asiansLowC1),]\n",
    "temp3 = temp2[(temp2$V4<=asiansHighC1),]\n",
    "temp4 = temp3[(temp3$V5>=asiansLowC2),]\n",
    "ASIA = temp4[(temp4$V5<=asiansHighC2),]\n",
    "\n",
    "pdf(\"ASIA_confirmation_plot.pdf\")\n",
    "plot(ASIA$V4,ASIA$V5,pch=16,col=ASIA$V1)\n",
    "legend(\"topright\",col=c(\"blue\",\"red\"),pch=16,c(\"New samples\",\"Asia\"))\n",
    "dev.off()\n",
    "\n",
    "temp2 = all_data[(all_data$V4>=africansLowC1),]\n",
    "temp3 = temp2[(temp2$V4<=africansHighC1),]\n",
    "temp4 = temp3[(temp3$V5>=africansLowC2),]\n",
    "AFRICA = temp4[(temp4$V5<=africansHighC2),]\n",
    "\n",
    "pdf(\"AFRICA_confirmation_plot.pdf\")\n",
    "plot(AFRICA$V4,AFRICA$V5,pch=16,col=AFRICA$V1)\n",
    "legend(\"topleft\",col=c(\"blue\",\"green3\"),pch=16,c(\"New samples\",\"Africa\"))\n",
    "dev.off()\n",
    "\n",
    "library(dplyr)\n",
    "\n",
    "main_data2 <- all_data[ ! all_data %in% EURO ]\n",
    "\n",
    "mixed_race1 <- anti_join(all_data, EURO)\n",
    "mixed_race2 <- anti_join(mixed_race1, ASIA)\n",
    "mixed_race3 <- anti_join(mixed_race2, AFRICA)\n",
    "\n",
    "total <- rbind(euros,asians,africans,mixed_race3)\n",
    "\n",
    "pdf(\"mixed_race_confirmation_plot.pdf\")\n",
    "plot(total$V4,total$V5,pch=16,col=total$V1)\n",
    "legend(\"topleft\",col=c(\"green3\",\"red\",\"blue\",\"black\"),pch=16,c(\"Africa\",\"Asia\",\"Mixed race samples\",\"Europe\"))\n",
    "dev.off()\n",
    "\n",
    "final_euro = subset(EURO, EURO$V1 != 5)\n",
    "final_euro2 =  data.frame(final_euro$V2,final_euro$V3)\n",
    "\n",
    "final_asia = subset(ASIA, ASIA$V1 != 2)\n",
    "final_asia2 =  data.frame(final_asia$V2,final_asia$V3)\n",
    "\n",
    "final_africa = subset(AFRICA, AFRICA$V1 != 3)\n",
    "final_africa2 =  data.frame(final_africa$V2,final_africa$V3)\n",
    "\n",
    "mixed_race4 =  data.frame(mixed_race3$V2,mixed_race3$V3)\n",
    "\n",
    "write.table(final_euro2,file = \"PCA_filtered_europeans.txt\", quote = FALSE,row.names=F,col.names = F)\n",
    "write.table(final_asia2,file = \"PCA_filtered_asians.txt\", quote = FALSE,row.names=F,col.names = F)\n",
    "write.table(final_africa2,file = \"PCA_filtered_africans.txt\", quote = FALSE,row.names=F,col.names = F)\n",
    "write.table(mixed_race4,file = \"PCA_filtered_mixed_race.txt\", quote = FALSE,row.names=F,col.names = F)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "=1.13.0\r\n",
      "afri_add.txt\r\n",
      "AFRICA_confirmation_plot.pdf\r\n",
      "afrio.txt\r\n",
      "afri.txt\r\n",
      "after_gender3.bed\r\n",
      "after_gender3.bim\r\n",
      "after_gender3.fam\r\n",
      "after_gender3.hh\r\n",
      "after_gender3.log\r\n",
      "after_gender4.bed\r\n",
      "after_gender4.bim\r\n",
      "after_gender4.fam\r\n",
      "after_gender4.hh\r\n",
      "after_gender4.log\r\n",
      "all_outliers.txt\r\n",
      "AMPPD_after_gender.bed\r\n",
      "AMPPD_after_gender.bim\r\n",
      "AMPPD_after_gender.fam\r\n",
      "AMPPD_after_gender.hh\r\n",
      "AMPPD_after_gender.log\r\n",
      "AMPPD_after_heterozyg.bed\r\n",
      "AMPPD_after_heterozyg.bim\r\n",
      "AMPPD_after_heterozyg.fam\r\n",
      "AMPPD_after_heterozyg.hh\r\n",
      "AMPPD_after_heterozyg.log\r\n",
      "AMPPD_call_rate.bed\r\n",
      "AMPPD_call_rate.bim\r\n",
      "AMPPD_call_rate.fam\r\n",
      "AMPPD_call_rate.hh\r\n",
      "AMPPD_call_rate.log\r\n",
      "AMPPD_updated_pheno.bed\r\n",
      "AMPPD_updated_pheno.bim\r\n",
      "AMPPD_updated_pheno.fam\r\n",
      "AMPPD_updated_pheno.hh\r\n",
      "AMPPD_updated_pheno.log\r\n",
      "asia_add.txt\r\n",
      "ASIA_confirmation_plot.pdf\r\n",
      "asiao.txt\r\n",
      "asia.txt\r\n",
      "call_rates.hh\r\n",
      "call_rates.imiss\r\n",
      "call_rates.lmiss\r\n",
      "call_rates.log\r\n",
      "eur_add.txt\r\n",
      "EURO_confirmation_plot.pdf\r\n",
      "euro.txt\r\n",
      "eur.txt\r\n",
      "\u001b[0m\u001b[01;34mfacets\u001b[0m/\r\n",
      "gender_check1.hh\r\n",
      "gender_check1.log\r\n",
      "gender_check1.sexcheck\r\n",
      "gender_check2.hh\r\n",
      "gender_check2.log\r\n",
      "gender_check2.sexcheck\r\n",
      "GENDER_FAILURES.txt\r\n",
      "hapmap3_bin_snplis.bed\r\n",
      "hapmap3_bin_snplis.bim\r\n",
      "hapmap3_bin_snplis.fam\r\n",
      "hapmap3_bin_snplis.hh\r\n",
      "hapmap3_bin_snplis.log\r\n",
      "hapmap3_bin_snplis-merge.missnp\r\n",
      "HAPMAP_GRCh38.bed\r\n",
      "HAPMAP_GRCh38.bim\r\n",
      "HAPMAP_GRCh38.fam\r\n",
      "HETEROZYGOSITY_DATA.txt\r\n",
      "HETEROZYGOSITY_OUTLIERS.txt\r\n",
      "mixed_race_confirmation_plot.pdf\r\n",
      "new_samples2.txt\r\n",
      "new_samples_add.txt\r\n",
      "new_samples.txt\r\n",
      "outliers1.txt\r\n",
      "outliers2.txt\r\n",
      "pca.bed\r\n",
      "pca.bim\r\n",
      "pca.eigenval\r\n",
      "pca.eigenvec\r\n",
      "pca.eigenvec2\r\n",
      "pca.fam\r\n",
      "PCA_filtered_africans.txt\r\n",
      "PCA_filtered_asians.txt\r\n",
      "PCA_filtered_europeans.txt\r\n",
      "PCA_filtered_mixed_race.txt\r\n",
      "PCA_in_R.R\r\n",
      "pca.log\r\n",
      "plink_test_covs.tab\r\n",
      "plink_test_pheno.tab\r\n",
      "problems1.txt\r\n",
      "problems2.txt\r\n",
      "pruned_data.bed\r\n",
      "pruned_data.bim\r\n",
      "pruned_data.fam\r\n",
      "pruned_data.hh\r\n",
      "pruned_data.log\r\n",
      "prunedHet.het\r\n",
      "prunedHet.hh\r\n",
      "prunedHet.log\r\n",
      "pruning.hh\r\n",
      "pruning.log\r\n",
      "pruning.prune.in\r\n",
      "pruning.prune.out\r\n",
      "Py3 - Clinical - load a CSV from Cloud Storage.ipynb\r\n",
      "Py3 - Clinical - load a table from BigQuery.ipynb\r\n",
      "Py3 - Query a saved Data Explorer cohort.ipynb\r\n",
      "Py3 - Saving Notebook Results.ipynb\r\n",
      "Py3 - Start Here.ipynb\r\n",
      "Py3 - WGS -  Query Variants.ipynb\r\n",
      "Py3 - WGS - Start Here.ipynb\r\n",
      "raw_hapmap_plot.pdf\r\n",
      "RIC3-Py-cleaned.ipynb\r\n",
      "RIC3-Py.ipynb\r\n",
      "RIC3-Py_New.ipynb\r\n",
      "RIC3-R.ipynb\r\n",
      "R - Start Here.ipynb\r\n",
      "samples_to_remove.txt\r\n"
     ]
    }
   ],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTES:\n",
    "\n",
    "Extract population of interest. In this case europeans, but can be any populations that is present in base comparison dataset\n",
    "This creates several plots and lists based on genetic ancestry"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender\n",
      "  --keep PCA_filtered_europeans.txt\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2834 people (1577 males, 1257 females) loaded from .fam.\n",
      "2834 phenotype values loaded from .fam.\n",
      "--keep: 2688 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2688 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 12794296 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap.hh ); many commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate in remaining samples is 0.99401.\n",
      "28791969 variants and 2688 people pass filters and QC.\n",
      "Among remaining phenotypes, 1638 are cases and 1050 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender --keep PCA_filtered_europeans.txt --make-bed --out AMPPD_after_gender_heterozyg_hapmap"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cat PCA_filtered_asians.txt PCA_filtered_africans.txt PCA_filtered_mixed_race.txt > hapmap_outliers33.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download the plots from the PCA to you data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://./raw_hapmap_plot.pdf [Content-Type=application/pdf]...\n",
      "/ [1 files][ 24.5 KiB/ 24.5 KiB]                                                \n",
      "Operation completed over 1 objects/24.5 KiB.                                     \n",
      "Copying file://./EURO_confirmation_plot.pdf [Content-Type=application/pdf]...\n",
      "/ [1 files][ 24.3 KiB/ 24.3 KiB]                                                \n",
      "Operation completed over 1 objects/24.3 KiB.                                     \n",
      "Copying file://./ASIA_confirmation_plot.pdf [Content-Type=application/pdf]...\n",
      "/ [1 files][  6.6 KiB/  6.6 KiB]                                                \n",
      "Operation completed over 1 objects/6.6 KiB.                                      \n",
      "Copying file://./AFRICA_confirmation_plot.pdf [Content-Type=application/pdf]...\n",
      "/ [1 files][  6.1 KiB/  6.1 KiB]                                                \n",
      "Operation completed over 1 objects/6.1 KiB.                                      \n",
      "Copying file://./mixed_race_confirmation_plot.pdf [Content-Type=application/pdf]...\n",
      "/ [1 files][  9.6 KiB/  9.6 KiB]                                                \n",
      "Operation completed over 1 objects/9.6 KiB.                                      \n"
     ]
    }
   ],
   "source": [
    "!gsutil cp ./raw_hapmap_plot.pdf gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./EURO_confirmation_plot.pdf gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./ASIA_confirmation_plot.pdf gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AFRICA_confirmation_plot.pdf gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./mixed_race_confirmation_plot.pdf gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To check that the files have been downloaded:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/=1.13.0\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AFRICA_confirmation_plot.pdf\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMC.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMCWald.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Fp.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.MadsonBrowning.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Skat.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.SkatO.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Zeggini.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB.log\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMC.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMCWald.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Fp.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.MadsonBrowning.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Skat.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.SkatO.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Zeggini.assoc\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.log\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/ASIA_confirmation_plot.pdf\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/EURO_confirmation_plot.pdf\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bed\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.bim\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/HAPMAP_GRCh38.fam\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/PCA_in_R.R\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/RIC3.WGS.trimmed.annotation_KB.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/RIC3_WGS_AMP_PD.assoc.fisher\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/RIC3_WGS_AMP_PD.log\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/RIC3_WGS_AMP_PD_logreg.assoc.logistic\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/RIC3_WGS_AMP_PD_logreg.log\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/afri_add.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/asia_add.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/eur_add.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/exclusion_regions_hg38.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/mixed_race_confirmation_plot.pdf\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/raw_hapmap_plot.pdf\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/refFlat.txt\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/test_sample_ids.csv\r\n",
      "gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/notebooks/\r\n"
     ]
    }
   ],
   "source": [
    "!gsutil ls gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Sample QC - Relatedness\n",
    "\n",
    "NOTES:\n",
    "\n",
    "PIHAT threshold of 0.125\n",
    "GCTA needs to be installed!!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruning.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap\n",
      "  --geno 0.01\n",
      "  --indep-pairwise 50 5 0.5\n",
      "  --maf 0.05\n",
      "  --out pruning\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2688 people (1503 males, 1185 females) loaded from .fam.\n",
      "2688 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2688 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 12794296 het. haploid genotypes present (see pruning.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.99401.\n",
      "891874 variants removed due to missing genotype data (--geno).\n",
      "21029807 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "6870288 variants and 2688 people pass filters and QC.\n",
      "Among remaining phenotypes, 1638 are cases and 1050 are controls.\n",
      "Pruned 439003 variants from chromosome 1, leaving 65560.\n",
      "Pruned 482188 variants from chromosome 2, leaving 70097.\n",
      "Pruned 413498 variants from chromosome 3, leaving 59097.\n",
      "Pruned 429707 variants from chromosome 4, leaving 59236.\n",
      "Pruned 373108 variants from chromosome 5, leaving 51786.\n",
      "Pruned 390802 variants from chromosome 6, leaving 54171.\n",
      "Pruned 344589 variants from chromosome 7, leaving 51526.\n",
      "Pruned 316883 variants from chromosome 8, leaving 44302.\n",
      "Pruned 252754 variants from chromosome 9, leaving 45790.\n",
      "Pruned 299083 variants from chromosome 10, leaving 45902.\n",
      "Pruned 290268 variants from chromosome 11, leaving 41098.\n",
      "Pruned 278806 variants from chromosome 12, leaving 41038.\n",
      "Pruned 214967 variants from chromosome 13, leaving 30764.\n",
      "Pruned 187786 variants from chromosome 14, leaving 28979.\n",
      "Pruned 165533 variants from chromosome 15, leaving 27904.\n",
      "Pruned 174271 variants from chromosome 16, leaving 31939.\n",
      "Pruned 152400 variants from chromosome 17, leaving 27274.\n",
      "Pruned 163960 variants from chromosome 18, leaving 25744.\n",
      "Pruned 134418 variants from chromosome 19, leaving 23498.\n",
      "Pruned 126511 variants from chromosome 20, leaving 21775.\n",
      "Pruned 86981 variants from chromosome 21, leaving 19743.\n",
      "Pruned 78404 variants from chromosome 22, leaving 16176.\n",
      "Pruned 166723 variants from chromosome 23, leaving 21178.\n",
      "Pruned 2752 variants from chromosome 24, leaving 316.\n",
      "Pruning complete.  5965395 of 6870288 variants removed.\n",
      "Marker lists written to pruning.prune.in and pruning.prune.out .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to pruned_data.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap\n",
      "  --extract pruning.prune.in\n",
      "  --make-bed\n",
      "  --out pruned_data\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2688 people (1503 males, 1185 females) loaded from .fam.\n",
      "2688 phenotype values loaded from .fam.\n",
      "--extract: 904893 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2688 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 657313 het. haploid genotypes present (see pruned_data.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999702.\n",
      "904893 variants and 2688 people pass filters and QC.\n",
      "Among remaining phenotypes, 1638 are cases and 1050 are controls.\n",
      "--make-bed to pruned_data.bed + pruned_data.bim + pruned_data.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to prunedHet.log.\n",
      "Options in effect:\n",
      "  --bfile pruned_data\n",
      "  --het\n",
      "  --out prunedHet\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "904893 variants loaded from .bim file.\n",
      "2688 people (1503 males, 1185 females) loaded from .fam.\n",
      "2688 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2688 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 657313 het. haploid genotypes present (see prunedHet.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999702.\n",
      "904893 variants and 2688 people pass filters and QC.\n",
      "Among remaining phenotypes, 1638 are cases and 1050 are controls.\n",
      "--het: 883399 variants scanned, report written to prunedHet.het .\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap --geno 0.01 --maf 0.05 --indep-pairwise 50 5 0.5 --out pruning\n",
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap --extract pruning.prune.in --make-bed --out pruned_data\n",
    "!~/bin/plink --bfile pruned_data --het --out prunedHet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GCTA is not installed\n",
      "Archive:  gcta_1.93.2beta.zip\n",
      "   creating: gcta_1.93.2beta/\n",
      "  inflating: gcta_1.93.2beta/gcta64  \n",
      "  inflating: gcta_1.93.2beta/.DS_Store  \n",
      "  inflating: gcta_1.93.2beta/test.bim  \n",
      "  inflating: gcta_1.93.2beta/MIT_License.txt  \n",
      "  inflating: gcta_1.93.2beta/test.fam  \n",
      "  inflating: gcta_1.93.2beta/README.txt  \n",
      "  inflating: gcta_1.93.2beta/test.bed  \n",
      "  inflating: gcta_1.93.2beta/test.phen  \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2021-02-02 12:49:24--  https://cnsgenomics.com/software/gcta/bin/gcta_1.93.2beta.zip\n",
      "Resolving cnsgenomics.com (cnsgenomics.com)... 203.101.226.237\n",
      "Connecting to cnsgenomics.com (cnsgenomics.com)|203.101.226.237|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 12054383 (11M) [application/zip]\n",
      "Saving to: ‘gcta_1.93.2beta.zip’\n",
      "\n",
      "     0K .......... .......... .......... .......... ..........  0%  124K 95s\n",
      "    50K .......... .......... .......... .......... ..........  0%  248K 71s\n",
      "   100K .......... .......... .......... .......... ..........  1%  157M 47s\n",
      "   150K .......... .......... .......... .......... ..........  1%  248K 47s\n",
      "   200K .......... .......... .......... .......... ..........  2%  126M 37s\n",
      "   250K .......... .......... .......... .......... ..........  2%  169M 31s\n",
      "   300K .......... .......... .......... .......... ..........  2%  125M 26s\n",
      "   350K .......... .......... .......... .......... ..........  3% 1.16M 24s\n",
      "   400K .......... .......... .......... .......... ..........  3%  315K 25s\n",
      "   450K .......... .......... .......... .......... ..........  4%  146M 23s\n",
      "   500K .......... .......... .......... .......... ..........  4% 98.4M 21s\n",
      "   550K .......... .......... .......... .......... ..........  5%  133M 19s\n",
      "   600K .......... .......... .......... .......... ..........  5%  144M 17s\n",
      "   650K .......... .......... .......... .......... ..........  5%  197M 16s\n",
      "   700K .......... .......... .......... .......... ..........  6%  107M 15s\n",
      "   750K .......... .......... .......... .......... ..........  6% 1.20M 14s\n",
      "   800K .......... .......... .......... .......... ..........  7% 1.16M 14s\n",
      "   850K .......... .......... .......... .......... ..........  7%  428K 15s\n",
      "   900K .......... .......... .......... .......... ..........  8%  158M 14s\n",
      "   950K .......... .......... .......... .......... ..........  8%  119M 13s\n",
      "  1000K .......... .......... .......... .......... ..........  8%  171M 12s\n",
      "  1050K .......... .......... .......... .......... ..........  9%  135M 12s\n",
      "  1100K .......... .......... .......... .......... ..........  9% 88.3M 11s\n",
      "  1150K .......... .......... .......... .......... .......... 10%  148M 11s\n",
      "  1200K .......... .......... .......... .......... .......... 10%  145M 10s\n",
      "  1250K .......... .......... .......... .......... .......... 11%  174M 10s\n",
      "  1300K .......... .......... .......... .......... .......... 11%  200M 9s\n",
      "  1350K .......... .......... .......... .......... .......... 11%  195M 9s\n",
      "  1400K .......... .......... .......... .......... .......... 12%  153M 9s\n",
      "  1450K .......... .......... .......... .......... .......... 12%  194M 8s\n",
      "  1500K .......... .......... .......... .......... .......... 13%  196M 8s\n",
      "  1550K .......... .......... .......... .......... .......... 13%  215M 8s\n",
      "  1600K .......... .......... .......... .......... .......... 14% 1.26M 8s\n",
      "  1650K .......... .......... .......... .......... .......... 14%  144M 7s\n",
      "  1700K .......... .......... .......... .......... .......... 14%  316K 8s\n",
      "  1750K .......... .......... .......... .......... .......... 15%  133M 8s\n",
      "  1800K .......... .......... .......... .......... .......... 15%  141M 8s\n",
      "  1850K .......... .......... .......... .......... .......... 16%  166M 7s\n",
      "  1900K .......... .......... .......... .......... .......... 16%  170M 7s\n",
      "  1950K .......... .......... .......... .......... .......... 16%  117M 7s\n",
      "  2000K .......... .......... .......... .......... .......... 17%  138M 7s\n",
      "  2050K .......... .......... .......... .......... .......... 17%  119M 7s\n",
      "  2100K .......... .......... .......... .......... .......... 18%  133M 6s\n",
      "  2150K .......... .......... .......... .......... .......... 18%  131M 6s\n",
      "  2200K .......... .......... .......... .......... .......... 19%  106M 6s\n",
      "  2250K .......... .......... .......... .......... .......... 19%  139M 6s\n",
      "  2300K .......... .......... .......... .......... .......... 19%  105M 6s\n",
      "  2350K .......... .......... .......... .......... .......... 20%  124M 6s\n",
      "  2400K .......... .......... .......... .......... .......... 20% 89.0M 5s\n",
      "  2450K .......... .......... .......... .......... .......... 21%  131M 5s\n",
      "  2500K .......... .......... .......... .......... .......... 21%  129M 5s\n",
      "  2550K .......... .......... .......... .......... .......... 22%  142M 5s\n",
      "  2600K .......... .......... .......... .......... .......... 22%  117M 5s\n",
      "  2650K .......... .......... .......... .......... .......... 22%  157M 5s\n",
      "  2700K .......... .......... .......... .......... .......... 23%  152M 5s\n",
      "  2750K .......... .......... .......... .......... .......... 23%  155M 5s\n",
      "  2800K .......... .......... .......... .......... .......... 24%  134M 4s\n",
      "  2850K .......... .......... .......... .......... .......... 24%  165M 4s\n",
      "  2900K .......... .......... .......... .......... .......... 25%  171M 4s\n",
      "  2950K .......... .......... .......... .......... .......... 25%  170M 4s\n",
      "  3000K .......... .......... .......... .......... .......... 25%  163M 4s\n",
      "  3050K .......... .......... .......... .......... .......... 26%  173M 4s\n",
      "  3100K .......... .......... .......... .......... .......... 26%  172M 4s\n",
      "  3150K .......... .......... .......... .......... .......... 27%  177M 4s\n",
      "  3200K .......... .......... .......... .......... .......... 27% 1.47M 4s\n",
      "  3250K .......... .......... .......... .......... .......... 28% 99.5M 4s\n",
      "  3300K .......... .......... .......... .......... .......... 28%  148M 4s\n",
      "  3350K .......... .......... .......... .......... .......... 28%  138M 4s\n",
      "  3400K .......... .......... .......... .......... .......... 29%  126M 4s\n",
      "  3450K .......... .......... .......... .......... .......... 29%  319K 4s\n",
      "  3500K .......... .......... .......... .......... .......... 30%  113M 4s\n",
      "  3550K .......... .......... .......... .......... .......... 30%  110M 4s\n",
      "  3600K .......... .......... .......... .......... .......... 31%  118M 4s\n",
      "  3650K .......... .......... .......... .......... .......... 31%  148M 4s\n",
      "  3700K .......... .......... .......... .......... .......... 31% 77.9M 3s\n",
      "  3750K .......... .......... .......... .......... .......... 32%  144M 3s\n",
      "  3800K .......... .......... .......... .......... .......... 32%  133M 3s\n",
      "  3850K .......... .......... .......... .......... .......... 33%  149M 3s\n",
      "  3900K .......... .......... .......... .......... .......... 33%  160M 3s\n",
      "  3950K .......... .......... .......... .......... .......... 33%  161M 3s\n",
      "  4000K .......... .......... .......... .......... .......... 34%  124M 3s\n",
      "  4050K .......... .......... .......... .......... .......... 34%  163M 3s\n",
      "  4100K .......... .......... .......... .......... .......... 35%  137M 3s\n",
      "  4150K .......... .......... .......... .......... .......... 35%  167M 3s\n",
      "  4200K .......... .......... .......... .......... .......... 36%  169M 3s\n",
      "  4250K .......... .......... .......... .......... .......... 36%  171M 3s\n",
      "  4300K .......... .......... .......... .......... .......... 36%  195M 3s\n",
      "  4350K .......... .......... .......... .......... .......... 37%  176M 3s\n",
      "  4400K .......... .......... .......... .......... .......... 37%  161M 3s\n",
      "  4450K .......... .......... .......... .......... .......... 38%  197M 3s\n",
      "  4500K .......... .......... .......... .......... .......... 38%  186M 3s\n",
      "  4550K .......... .......... .......... .......... .......... 39%  180M 3s\n",
      "  4600K .......... .......... .......... .......... .......... 39%  158M 2s\n",
      "  4650K .......... .......... .......... .......... .......... 39%  173M 2s\n",
      "  4700K .......... .......... .......... .......... .......... 40%  194M 2s\n",
      "  4750K .......... .......... .......... .......... .......... 40%  184M 2s\n",
      "  4800K .......... .......... .......... .......... .......... 41%  159M 2s\n",
      "  4850K .......... .......... .......... .......... .......... 41%  174M 2s\n",
      "  4900K .......... .......... .......... .......... .......... 42%  184M 2s\n",
      "  4950K .......... .......... .......... .......... .......... 42%  157M 2s\n",
      "  5000K .......... .......... .......... .......... .......... 42%  130M 2s\n",
      "  5050K .......... .......... .......... .......... .......... 43%  158M 2s\n",
      "  5100K .......... .......... .......... .......... .......... 43%  182M 2s\n",
      "  5150K .......... .......... .......... .......... .......... 44%  171M 2s\n",
      "  5200K .......... .......... .......... .......... .......... 44%  128M 2s\n",
      "  5250K .......... .......... .......... .......... .......... 45%  169M 2s\n",
      "  5300K .......... .......... .......... .......... .......... 45%  154M 2s\n",
      "  5350K .......... .......... .......... .......... .......... 45%  190M 2s\n",
      "  5400K .......... .......... .......... .......... .......... 46%  178M 2s\n",
      "  5450K .......... .......... .......... .......... .......... 46%  155M 2s\n",
      "  5500K .......... .......... .......... .......... .......... 47%  173M 2s\n",
      "  5550K .......... .......... .......... .......... .......... 47%  198M 2s\n",
      "  5600K .......... .......... .......... .......... .......... 47%  162M 2s\n",
      "  5650K .......... .......... .......... .......... .......... 48%  178M 2s\n",
      "  5700K .......... .......... .......... .......... .......... 48%  183M 2s\n",
      "  5750K .......... .......... .......... .......... .......... 49%  176M 2s\n",
      "  5800K .......... .......... .......... .......... .......... 49%  178M 2s\n",
      "  5850K .......... .......... .......... .......... .......... 50%  209M 2s\n",
      "  5900K .......... .......... .......... .......... .......... 50% 1.74M 2s\n",
      "  5950K .......... .......... .......... .......... .......... 50%  200M 2s\n",
      "  6000K .......... .......... .......... .......... .......... 51%  146M 2s\n",
      "  6050K .......... .......... .......... .......... .......... 51%  145M 2s\n",
      "  6100K .......... .......... .......... .......... .......... 52%  181M 2s\n",
      "  6150K .......... .......... .......... .......... .......... 52%  197M 1s\n",
      "  6200K .......... .......... .......... .......... .......... 53%  145M 1s\n",
      "  6250K .......... .......... .......... .......... .......... 53%  171M 1s\n",
      "  6300K .......... .......... .......... .......... .......... 53%  170M 1s\n",
      "  6350K .......... .......... .......... .......... .......... 54%  169M 1s\n",
      "  6400K .......... .......... .......... .......... .......... 54% 1.24M 1s\n",
      "  6450K .......... .......... .......... .......... .......... 55%  146M 1s\n",
      "  6500K .......... .......... .......... .......... .......... 55%  429K 1s\n",
      "  6550K .......... .......... .......... .......... .......... 56%  116M 1s\n",
      "  6600K .......... .......... .......... .......... .......... 56%  159M 1s\n",
      "  6650K .......... .......... .......... .......... .......... 56%  162M 1s\n",
      "  6700K .......... .......... .......... .......... .......... 57%  168M 1s\n",
      "  6750K .......... .......... .......... .......... .......... 57%  189M 1s\n",
      "  6800K .......... .......... .......... .......... .......... 58%  126M 1s\n",
      "  6850K .......... .......... .......... .......... .......... 58%  165M 1s\n",
      "  6900K .......... .......... .......... .......... .......... 59%  181M 1s\n",
      "  6950K .......... .......... .......... .......... .......... 59%  171M 1s\n",
      "  7000K .......... .......... .......... .......... .......... 59%  156M 1s\n",
      "  7050K .......... .......... .......... .......... .......... 60%  177M 1s\n",
      "  7100K .......... .......... .......... .......... .......... 60%  174M 1s\n",
      "  7150K .......... .......... .......... .......... .......... 61%  172M 1s\n",
      "  7200K .......... .......... .......... .......... .......... 61%  137M 1s\n",
      "  7250K .......... .......... .......... .......... .......... 62%  174M 1s\n",
      "  7300K .......... .......... .......... .......... .......... 62%  170M 1s\n",
      "  7350K .......... .......... .......... .......... .......... 62%  185M 1s\n",
      "  7400K .......... .......... .......... .......... .......... 63%  168M 1s\n",
      "  7450K .......... .......... .......... .......... .......... 63%  156M 1s\n",
      "  7500K .......... .......... .......... .......... .......... 64%  179M 1s\n",
      "  7550K .......... .......... .......... .......... .......... 64%  114M 1s\n",
      "  7600K .......... .......... .......... .......... .......... 64% 68.5M 1s\n",
      "  7650K .......... .......... .......... .......... .......... 65% 59.2M 1s\n",
      "  7700K .......... .......... .......... .......... .......... 65%  112M 1s\n",
      "  7750K .......... .......... .......... .......... .......... 66%  108M 1s\n",
      "  7800K .......... .......... .......... .......... .......... 66% 78.1M 1s\n",
      "  7850K .......... .......... .......... .......... .......... 67% 77.9M 1s\n",
      "  7900K .......... .......... .......... .......... .......... 67% 90.9M 1s\n",
      "  7950K .......... .......... .......... .......... .......... 67% 71.4M 1s\n",
      "  8000K .......... .......... .......... .......... .......... 68% 72.0M 1s\n",
      "  8050K .......... .......... .......... .......... .......... 68% 78.8M 1s\n",
      "  8100K .......... .......... .......... .......... .......... 69%  105M 1s\n",
      "  8150K .......... .......... .......... .......... .......... 69% 57.5M 1s\n",
      "  8200K .......... .......... .......... .......... .......... 70% 79.8M 1s\n",
      "  8250K .......... .......... .......... .......... .......... 70%  102M 1s\n",
      "  8300K .......... .......... .......... .......... .......... 70% 97.1M 1s\n",
      "  8350K .......... .......... .......... .......... .......... 71%  150M 1s\n",
      "  8400K .......... .......... .......... .......... .......... 71% 96.9M 1s\n",
      "  8450K .......... .......... .......... .......... .......... 72%  166M 1s\n",
      "  8500K .......... .......... .......... .......... .......... 72%  112M 1s\n",
      "  8550K .......... .......... .......... .......... .......... 73%  152M 1s\n",
      "  8600K .......... .......... .......... .......... .......... 73%  139M 1s\n",
      "  8650K .......... .......... .......... .......... .......... 73%  166M 1s\n",
      "  8700K .......... .......... .......... .......... .......... 74%  160M 1s\n",
      "  8750K .......... .......... .......... .......... .......... 74%  158M 1s\n",
      "  8800K .......... .......... .......... .......... .......... 75% 53.0M 1s\n",
      "  8850K .......... .......... .......... .......... .......... 75% 2.12M 1s\n",
      "  8900K .......... .......... .......... .......... .......... 76%  168M 1s\n",
      "  8950K .......... .......... .......... .......... .......... 76%  197M 1s\n",
      "  9000K .......... .......... .......... .......... .......... 76%  179M 1s\n",
      "  9050K .......... .......... .......... .......... .......... 77%  204M 1s\n",
      "  9100K .......... .......... .......... .......... .......... 77%  208M 1s\n",
      "  9150K .......... .......... .......... .......... .......... 78%  214M 1s\n",
      "  9200K .......... .......... .......... .......... .......... 78% 1.20M 1s\n",
      "  9250K .......... .......... .......... .......... .......... 79%  180M 1s\n",
      "  9300K .......... .......... .......... .......... .......... 79%  204M 0s\n",
      "  9350K .......... .......... .......... .......... .......... 79%  211M 0s\n",
      "  9400K .......... .......... .......... .......... .......... 80%  182M 0s\n",
      "  9450K .......... .......... .......... .......... .......... 80%  206M 0s\n",
      "  9500K .......... .......... .......... .......... .......... 81%  433K 0s\n",
      "  9550K .......... .......... .......... .......... .......... 81% 99.7M 0s\n",
      "  9600K .......... .......... .......... .......... .......... 81%  130M 0s\n",
      "  9650K .......... .......... .......... .......... .......... 82%  172M 0s\n",
      "  9700K .......... .......... .......... .......... .......... 82%  178M 0s\n",
      "  9750K .......... .......... .......... .......... .......... 83%  186M 0s\n",
      "  9800K .......... .......... .......... .......... .......... 83% 31.5M 0s\n",
      "  9850K .......... .......... .......... .......... .......... 84%  169M 0s\n",
      "  9900K .......... .......... .......... .......... .......... 84%  158M 0s\n",
      "  9950K .......... .......... .......... .......... .......... 84%  181M 0s\n",
      " 10000K .......... .......... .......... .......... .......... 85%  108M 0s\n",
      " 10050K .......... .......... .......... .......... .......... 85%  142M 0s\n",
      " 10100K .......... .......... .......... .......... .......... 86%  102M 0s\n",
      " 10150K .......... .......... .......... .......... .......... 86% 78.1M 0s\n",
      " 10200K .......... .......... .......... .......... .......... 87% 55.6M 0s\n",
      " 10250K .......... .......... .......... .......... .......... 87%  177M 0s\n",
      " 10300K .......... .......... .......... .......... .......... 87%  190M 0s\n",
      " 10350K .......... .......... .......... .......... .......... 88% 97.9M 0s\n",
      " 10400K .......... .......... .......... .......... .......... 88% 86.9M 0s\n",
      " 10450K .......... .......... .......... .......... .......... 89% 77.1M 0s\n",
      " 10500K .......... .......... .......... .......... .......... 89% 96.1M 0s\n",
      " 10550K .......... .......... .......... .......... .......... 90%  106M 0s\n",
      " 10600K .......... .......... .......... .......... .......... 90%  103M 0s\n",
      " 10650K .......... .......... .......... .......... .......... 90% 96.2M 0s\n",
      " 10700K .......... .......... .......... .......... .......... 91%  108M 0s\n",
      " 10750K .......... .......... .......... .......... .......... 91% 73.7M 0s\n",
      " 10800K .......... .......... .......... .......... .......... 92% 92.6M 0s\n",
      " 10850K .......... .......... .......... .......... .......... 92% 83.3M 0s\n",
      " 10900K .......... .......... .......... .......... .......... 93%  128M 0s\n",
      " 10950K .......... .......... .......... .......... .......... 93% 61.8M 0s\n",
      " 11000K .......... .......... .......... .......... .......... 93%  127M 0s\n",
      " 11050K .......... .......... .......... .......... .......... 94%  143M 0s\n",
      " 11100K .......... .......... .......... .......... .......... 94% 99.4M 0s\n",
      " 11150K .......... .......... .......... .......... .......... 95% 83.3M 0s\n",
      " 11200K .......... .......... .......... .......... .......... 95% 68.4M 0s\n",
      " 11250K .......... .......... .......... .......... .......... 95%  107M 0s\n",
      " 11300K .......... .......... .......... .......... .......... 96% 81.4M 0s\n",
      " 11350K .......... .......... .......... .......... .......... 96%  129M 0s\n",
      " 11400K .......... .......... .......... .......... .......... 97%  134M 0s\n",
      " 11450K .......... .......... .......... .......... .......... 97%  175M 0s\n",
      " 11500K .......... .......... .......... .......... .......... 98%  100M 0s\n",
      " 11550K .......... .......... .......... .......... .......... 98%  142M 0s\n",
      " 11600K .......... .......... .......... .......... .......... 98%  156M 0s\n",
      " 11650K .......... .......... .......... .......... .......... 99%  116M 0s\n",
      " 11700K .......... .......... .......... .......... .......... 99%  156M 0s\n",
      " 11750K .......... .......... .                               100%  167M=2.0s\n",
      "\n",
      "2021-02-02 12:49:27 (5.64 MB/s) - ‘gcta_1.93.2beta.zip’ saved [12054383/12054383]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "if test -e ~/bin/gcta64; then\n",
    "\n",
    "echo \"GCTA is already installed in ~/bin\"\n",
    "else\n",
    "echo \"GCTA is not installed\"\n",
    "cd ~/bin/\n",
    "\n",
    "wget https://cnsgenomics.com/software/gcta/bin/gcta_1.93.2beta.zip\n",
    "\n",
    "unzip -o gcta_1.93.2beta.zip\n",
    "\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0;32m\u001b[0m*******************************************************************\r\n",
      "\u001b[0;32m\u001b[0m* Genome-wide Complex Trait Analysis (GCTA)\r\n",
      "\u001b[0;32m\u001b[0m* version 1.93.2 beta Linux\r\n",
      "\u001b[0;32m\u001b[0m* (C) 2010-present, Jian Yang, The University of Queensland\r\n",
      "\u001b[0;32m\u001b[0m* Please report bugs to Jian Yang <jian.yang.qt@gmail.com>\r\n",
      "\u001b[0;32m\u001b[0m*******************************************************************\r\n",
      "\u001b[0;32mAnalysis started \u001b[0mat 12:49:39 UTC on Tue Feb 02 2021.\r\n",
      "\u001b[0;32m\u001b[0mHostname: saturn-6ebc025c-7384-4edf-91c9-1b2092cda50d-m\r\n",
      "\u001b[0;32m\u001b[0m\r\n",
      "\u001b[0;32m\u001b[0mError: no analysis has been launched by the option(s)\r\n",
      "\u001b[0;32m\u001b[0mPlease see online documentation at http://cnsgenomics.com/software/gcta\r\n"
     ]
    }
   ],
   "source": [
    "!~/bin/gcta_1.93.2beta/gcta64"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0;32m\u001b[0m*******************************************************************\n",
      "\u001b[0;32m\u001b[0m* Genome-wide Complex Trait Analysis (GCTA)\n",
      "\u001b[0;32m\u001b[0m* version 1.93.2 beta Linux\n",
      "\u001b[0;32m\u001b[0m* (C) 2010-present, Jian Yang, The University of Queensland\n",
      "\u001b[0;32m\u001b[0m* Please report bugs to Jian Yang <jian.yang.qt@gmail.com>\n",
      "\u001b[0;32m\u001b[0m*******************************************************************\n",
      "\u001b[0;32mAnalysis started \u001b[0mat 12:49:46 UTC on Tue Feb 02 2021.\n",
      "\u001b[0;32m\u001b[0mHostname: saturn-6ebc025c-7384-4edf-91c9-1b2092cda50d-m\n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32m\u001b[0mOptions: \n",
      "\u001b[0;32m\u001b[0m \n",
      "--bfile pruned_data \n",
      "--make-grm \n",
      "--out GRM_matrix \n",
      "--autosome \n",
      "--maf 0.05 \n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32m\u001b[0mNote: GRM is computed using the SNPs on the autosome.\n",
      "\u001b[0;32m\u001b[0mReading PLINK FAM file from [pruned_data.fam]...\n",
      "\u001b[0;32m\u001b[0m2688 individuals to be included from FAM file.\n",
      "2688 individuals to be included. 1503 males, 1185 females, 0 unknown.\n",
      "\u001b[0;32m\u001b[0mReading PLINK BIM file from [pruned_data.bim]...\n",
      "\u001b[0;32m\u001b[0m904893 SNPs to be included from BIM file(s).\n",
      "\u001b[0;32m\u001b[0m883399 SNPs to be included from valid chromosome number\n",
      "Threshold to filter variants: MAF > 0.050000.\n",
      "\u001b[0;32m\u001b[0mComputing the genetic relationship matrix (GRM) v2 ...\n",
      "\u001b[0;32m\u001b[0mSubset 1/1, no. subject 1-2688\n",
      "  \u001b[0;32m\u001b[0m2688 samples, 883399 markers, 3614016 GRM elements\n",
      "\u001b[0;32m\u001b[0mIDs for the GRM file has been saved in the file [GRM_matrix.grm.id]\n",
      "Computing GRM...\n",
      "  \u001b[0;32m\u001b[0m100% finished in 184.4 sec\n",
      "883399 SNPs have been processed.\n",
      "  Used 883399 valid SNPs.\n",
      "\u001b[0;32m\u001b[0mThe GRM computation is completed.\n",
      "\u001b[0;32m\u001b[0mSaving GRM...\n",
      "\u001b[0;32m\u001b[0mGRM has been saved in the file [GRM_matrix.grm.bin]\n",
      "\u001b[0;32m\u001b[0mNumber of SNPs in each pair of individuals has been saved in the file [GRM_matrix.grm.N.bin]\n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32mAnalysis finished \u001b[0mat 12:52:52 UTC on Tue Feb 02 2021\n",
      "Overall computational time: 3 minutes 5 sec.\n",
      "\u001b[0;32m\u001b[0m*******************************************************************\n",
      "\u001b[0;32m\u001b[0m* Genome-wide Complex Trait Analysis (GCTA)\n",
      "\u001b[0;32m\u001b[0m* version 1.93.2 beta Linux\n",
      "\u001b[0;32m\u001b[0m* (C) 2010-present, Jian Yang, The University of Queensland\n",
      "\u001b[0;32m\u001b[0m* Please report bugs to Jian Yang <jian.yang.qt@gmail.com>\n",
      "\u001b[0;32m\u001b[0m*******************************************************************\n",
      "\u001b[0;32mAnalysis started \u001b[0mat 12:52:53 UTC on Tue Feb 02 2021.\n",
      "\u001b[0;32m\u001b[0mHostname: saturn-6ebc025c-7384-4edf-91c9-1b2092cda50d-m\n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32m\u001b[0mOptions: \n",
      "\u001b[0;32m\u001b[0m \n",
      "--grm-cutoff 0.125 \n",
      "--grm GRM_matrix \n",
      "--out GRM_matrix_0125 \n",
      "--make-grm \n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32m\u001b[0mPruning the GRM with a cutoff of 0.125000...\n",
      "\u001b[0;32m\u001b[0mTotal number of parts to proceed: 1\n",
      "    \u001b[0;32m\u001b[0mProcessing part 1\n",
      "\u001b[0;32m\u001b[0mAfter pruning the GRM, there are 2576 individuals (112 individuals removed).\n",
      "    \u001b[0;32m\u001b[0mPruned unrelated IDs have been saved to GRM_matrix_0125.grm.id\n",
      "\u001b[0;32m\u001b[0mPruning GRM values, total parts 1\n",
      "    \u001b[0;32m\u001b[0mProcessing part 1\n",
      "    \u001b[0;32m\u001b[0mGRM values has been saved to [GRM_matrix_0125.grm.bin]\n",
      "\u001b[0;32m\u001b[0mPruning number of SNPs to calculate GRM, total parts 1\n",
      "    \u001b[0;32m\u001b[0mProcessing part 1\n",
      "    \u001b[0;32m\u001b[0mNumber of SNPs has been saved to [GRM_matrix_0125.grm.N.bin]\n",
      "\u001b[0;32m\u001b[0m\n",
      "\u001b[0;32mAnalysis finished \u001b[0mat 12:52:53 UTC on Tue Feb 02 2021\n",
      "Overall computational time: 0.09 sec.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/gcta_1.93.2beta/gcta64 --bfile pruned_data --make-grm --out GRM_matrix --autosome --maf 0.05 \n",
    "!~/bin/gcta_1.93.2beta/gcta64 --grm-cutoff 0.125 --grm GRM_matrix --out GRM_matrix_0125 --make-grm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap_pihat.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap\n",
      "  --keep GRM_matrix_0125.grm.id\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap_pihat\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2688 people (1503 males, 1185 females) loaded from .fam.\n",
      "2688 phenotype values loaded from .fam.\n",
      "--keep: 2576 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 12388611 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.hh ); many commands treat these as\n",
      "missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate in remaining samples is 0.99401.\n",
      "28791969 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap_pihat.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap --keep GRM_matrix_0125.grm.id --make-bed --out AMPPD_after_gender_heterozyg_hapmap_pihat"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cut -f 1,2 AMPPD_after_gender_heterozyg_hapmap_pihat.fam > IDs_before_relatedness_filter.txt\n",
    "cut -f 1,2 AMPPD_after_gender_heterozyg_hapmap_pihat.fam > IDs_after_relatedness_filter.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Variant QC - Missingness per variant"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap_pihat_mind.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat\n",
      "  --geno 0.05\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28791969 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 12388611 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.hh ); many commands treat these\n",
      "as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.99401.\n",
      "567232 variants removed due to missing genotype data (--geno).\n",
      "28224737 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat --make-bed --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind --geno 0.05"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "grep \"(--geno)\" AMPPD_after_gender_heterozyg_hapmap_pihat_mind.log > MISSINGNESS_SNPS.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Missingness by case control P > 1E-4 # needs case control status"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to missing_snps.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind\n",
      "  --out missing_snps\n",
      "  --test-missing\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28224737 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 11013082 het. haploid genotypes present (see missing_snps.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999631.\n",
      "28224737 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "Writing --test-missing report to missing_snps.missing ... done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind --test-missing --out missing_snps "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "awk '{if ($5 <= 0.0001) print $2 }' missing_snps.missing > missing_snps_1E4.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind\n",
      "  --exclude missing_snps_1E4.txt\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28224737 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "--exclude: 28220119 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 9966687 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.hh ); many commands\n",
      "treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999635.\n",
      "28220119 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind --exclude missing_snps_1E4.txt --make-bed --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "sort -u missing_snps_1E4.txt > VARIANT_TEST_MISSING_SNPS.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to missing_hap.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1\n",
      "  --out missing_hap\n",
      "  --test-mishap\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28220119 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 9966687 het. haploid genotypes present (see missing_hap.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999635.\n",
      "28220119 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--test-mishap: 705087 loci checked (27515032 skipped).\n",
      "Report written to missing_hap.missing.hap .\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1 --test-mishap --out missing_hap "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "awk '{if ($8 <= 0.0001) print $9 }' missing_hap.missing.hap > missing_haps_1E4.txt\n",
    "sed 's/|/\\\n",
    "/g' missing_haps_1E4.txt > missing_haps_1E4_final.txt\n",
    "\n",
    "sort -u missing_haps_1E4_final.txt > HAPLOTYPE_TEST_MISSING_SNPS.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1\n",
      "  --exclude missing_haps_1E4_final.txt\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "28220119 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "--exclude: 27938327 variants remaining.\n",
      "Warning: At least 368364 duplicate IDs in --exclude file.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 9966687 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.hh ); many commands\n",
      "treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999685.\n",
      "27938327 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1 --exclude missing_haps_1E4_final.txt --make-bed --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Variant QC - HWE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to HWE_snps.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2\n",
      "  --filter-controls\n",
      "  --hwe 1E-4\n",
      "  --out HWE_snps\n",
      "  --write-snplist\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "27938327 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "1615 people removed due to case/control status (--filter-controls).\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 961 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 3039503 het. haploid genotypes present (see HWE_snps.hh ); many\n",
      "commands treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate in remaining samples is 0.999688.\n",
      "Warning: --hwe observation counts vary by more than 10%, due to the X\n",
      "chromosome.  You may want to use a less stringent --hwe p-value threshold for X\n",
      "chromosome variants.\n",
      "--hwe: 196555 variants removed due to Hardy-Weinberg exact test.\n",
      "27741772 variants and 961 people pass filters and QC.\n",
      "Among remaining phenotypes, 0 are cases and 961 are controls.\n",
      "List of variant IDs written to HWE_snps.snplist .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2\n",
      "  --extract HWE_snps.snplist\n",
      "  --make-bed\n",
      "  --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "27938327 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "--extract: 27741772 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Warning: 9699472 het. haploid genotypes present (see\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.hh ); many commands\n",
      "treat these as missing.\n",
      "Warning: Nonmissing nonmale Y chromosome genotype(s) present; many commands\n",
      "treat these as missing.\n",
      "Total genotyping rate is 0.999728.\n",
      "27741772 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.bed +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.bim +\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2 --filter-controls --hwe 1E-4 --write-snplist --out HWE_snps\n",
    "!~/bin/plink --bfile AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2 --extract HWE_snps.snplist --make-bed --out AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "mv AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.bim AMPPD_FILTERED.bim\n",
    "mv AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.bed AMPPD_FILTERED.bed\n",
    "mv AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.fam AMPPD_FILTERED.fam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#Run PCA on only the AMP-PD to get PCs for the covariate file. \n",
    "Download file with long distance high LD-regions to exclude (can alter the PCA if kept)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/exclusion_regions_hg38.txt...\n",
      "/ [0 files][    0.0 B/  466.0 B]                                                \r",
      "/ [1 files][  466.0 B/  466.0 B]                                                \r\n",
      "Operation completed over 1 objects/466.0 B.                                      \n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/exclusion_regions_hg38.txt /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to EXAMPLE_UNIMPUTED.log.\n",
      "Options in effect:\n",
      "  --autosome\n",
      "  --bfile AMPPD_FILTERED\n",
      "  --exclude range exclusion_regions_hg38.txt\n",
      "  --geno 0.05\n",
      "  --hwe 1E-6\n",
      "  --maf 0.01\n",
      "  --make-bed\n",
      "  --out EXAMPLE_UNIMPUTED\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "26799696 out of 27741772 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "--exclude range: 560772 variants excluded.\n",
      "--exclude range: 26238924 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999737.\n",
      "0 variants removed due to missing genotype data (--geno).\n",
      "--hwe: 0 variants removed due to Hardy-Weinberg exact test.\n",
      "17143631 variants removed due to minor allele threshold(s)\n",
      "(--maf/--max-maf/--mac/--max-mac).\n",
      "9095293 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to EXAMPLE_UNIMPUTED.bed + EXAMPLE_UNIMPUTED.bim +\n",
      "EXAMPLE_UNIMPUTED.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to prune.log.\n",
      "Options in effect:\n",
      "  --bfile EXAMPLE_UNIMPUTED\n",
      "  --indep-pairwise 50 5 0.5\n",
      "  --out prune\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "9095293 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999532.\n",
      "9095293 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "Pruned 538047 variants from chromosome 1, leaving 166171.\n",
      "Pruned 570956 variants from chromosome 2, leaving 174244.\n",
      "Pruned 496048 variants from chromosome 3, leaving 147885.\n",
      "Pruned 532219 variants from chromosome 4, leaving 152796.\n",
      "Pruned 443439 variants from chromosome 5, leaving 134161.\n",
      "Pruned 424034 variants from chromosome 6, leaving 124110.\n",
      "Pruned 419429 variants from chromosome 7, leaving 128411.\n",
      "Pruned 366784 variants from chromosome 8, leaving 109300.\n",
      "Pruned 306234 variants from chromosome 9, leaving 98035.\n",
      "Pruned 362406 variants from chromosome 10, leaving 110197.\n",
      "Pruned 346613 variants from chromosome 11, leaving 104854.\n",
      "Pruned 338486 variants from chromosome 12, leaving 103303.\n",
      "Pruned 265083 variants from chromosome 13, leaving 78397.\n",
      "Pruned 233839 variants from chromosome 14, leaving 71985.\n",
      "Pruned 203255 variants from chromosome 15, leaving 65181.\n",
      "Pruned 213935 variants from chromosome 16, leaving 73628.\n",
      "Pruned 183284 variants from chromosome 17, leaving 62979.\n",
      "Pruned 202905 variants from chromosome 18, leaving 63484.\n",
      "Pruned 161366 variants from chromosome 19, leaving 54300.\n",
      "Pruned 152428 variants from chromosome 20, leaving 49839.\n",
      "Pruned 99563 variants from chromosome 21, leaving 32775.\n",
      "Pruned 96170 variants from chromosome 22, leaving 32735.\n",
      "Pruning complete.  6956523 of 9095293 variants removed.\n",
      "Marker lists written to prune.prune.in and prune.prune.out .\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to prune.log.\n",
      "Options in effect:\n",
      "  --bfile EXAMPLE_UNIMPUTED\n",
      "  --extract prune.prune.in\n",
      "  --make-bed\n",
      "  --out prune\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "9095293 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "--extract: 2138770 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999165.\n",
      "2138770 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to prune.bed + prune.bim + prune.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to EXAMPLE_UNIMPUTED.PCA.log.\n",
      "Options in effect:\n",
      "  --bfile prune\n",
      "  --out EXAMPLE_UNIMPUTED.PCA\n",
      "  --pca\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "2138770 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using up to 15 threads (change this with --threads).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999165.\n",
      "2138770 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "Relationship matrix calculation complete.\n",
      "--pca: Results saved to EXAMPLE_UNIMPUTED.PCA.eigenval and\n",
      "EXAMPLE_UNIMPUTED.PCA.eigenvec .\n"
     ]
    }
   ],
   "source": [
    "# Make sure to use high-quality SNPs and exclude regions of long-range linkage disequilibrium (LD): exclusion_regions_hg38.txt\n",
    "!~/bin/plink --bfile AMPPD_FILTERED --exclude range exclusion_regions_hg38.txt --autosome --maf 0.01 --geno 0.05 --hwe 1E-6 --make-bed --out EXAMPLE_UNIMPUTED\n",
    "\n",
    "# Prune out unnecessary SNPs \n",
    "!~/bin/plink --bfile EXAMPLE_UNIMPUTED --indep-pairwise 50 5 0.5 --out prune \n",
    "\n",
    "# Keep only pruned SNPs\n",
    "!~/bin/plink --bfile EXAMPLE_UNIMPUTED --extract prune.prune.in --make-bed --out prune \n",
    "\n",
    "# Generate PCs \n",
    "!~/bin/plink --bfile prune --pca --out EXAMPLE_UNIMPUTED.PCA"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Creating covariate file: \n",
    "Move over to R and run the code below (do not have the energy to try to do it in python)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Creating covar file\n",
    "\n",
    "#### Download the necessary packages \n",
    "if (!require(tidyverse)) install.packages('tidyverse')\n",
    "if (!require(data.table)) install.packages('data.table')\n",
    "if (!require(dplyr)) install.packages('dplyr')\n",
    "if (!require(plyr)) install.packages('plyr')\n",
    "if (!require(ggplot2)) install.packages('ggplot2')\n",
    "if (!require(qqman)) install.packages('qqman')\n",
    "\n",
    "#### Load the necessary packages \n",
    "library(tidyverse)\n",
    "library(data.table)\n",
    "library(dplyr)\n",
    "library(plyr)\n",
    "library(ggplot2)\n",
    "library(qqman)\n",
    "\n",
    "#### Read in the PCA Eigenvalues and Eigenvectors \n",
    "eigenvec <- read.delim(\"EXAMPLE_UNIMPUTED.PCA.eigenvec\", sep =\" \", header = F, stringsAsFactors = F)\n",
    "head(eigenvec)\n",
    "\n",
    "#### Read in the .fam file\n",
    "fam <- fread(\"AMPPD_FILTERED.fam\", header = F)\n",
    "head(fam)\n",
    "\n",
    "#### Read in the covar file (age)\n",
    "covs <- fread(\"plink_test_covs.tab\", header = T)\n",
    "head(covs)\n",
    "\n",
    "#### Rename the columns \n",
    "colnames(fam) <- c(\"FID\", \"IID\", \"PAT\", \"MAT\", \"SEX\", \"PHENO\")\n",
    "colnames(eigenvec) <- c(\"FID\", \"IID\", \"PC1\", \"PC2\", \"PC3\", \"PC4\", \"PC5\", \"PC6\", \"PC7\", \"PC8\", \"PC9\", \"PC10\", \"PC11\", \"PC12\", \"PC13\", \"PC14\", \"PC15\", \"PC16\", \"PC17\", \"PC18\",\"PC19\",\"PC20\")\n",
    "head(fam)\n",
    "head(eigenvec)\n",
    "\n",
    "#Add age to the fam file\n",
    "fam$AGE = covs$age_at_baseline[match(fam$IID, covs$IID)]\n",
    "head(fam)\n",
    "\n",
    "#### Drop the IID column to prevent double columns\n",
    "eigenvec$FID <- NULL\n",
    "head (eigenvec)\n",
    "\n",
    "#### Combine the files\n",
    "combined <- join(fam, eigenvec, by = c(\"IID\"=\"IID\"))\n",
    "head(combined)\n",
    "\n",
    "#### Save the file \n",
    "write.table(combined, file = \"covariateFile.txt\", row.names=FALSE, na=\"\", quote = FALSE, sep=\"\\t\")\n",
    "\n",
    "#### Check that the files seems ok\n",
    "TEST <- fread(\"covariateFile.txt\", header = T) #should the header be removed??\n",
    "head(TEST)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### RIC3 analyses\n",
    "\n",
    "Positions for RIC3:\n",
    "#hg19 (Only needed for IPDGC data)\n",
    "chr11:8127597 - 8190602 \n",
    "8,127,603-8,190,602\n",
    "#hg38 (What we need for the AMP-PD data)\n",
    "chr11:8106056-8169055 (Ensemble)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1) Extract the RIC3 region from the filtered data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to AMPPD_FILTERED.RIC3.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_FILTERED\n",
      "  --chr 11\n",
      "  --from-bp 8106056\n",
      "  --make-bed\n",
      "  --out AMPPD_FILTERED.RIC3\n",
      "  --to-bp 8169055\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "773 out of 27741772 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values loaded from .fam.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999747.\n",
      "773 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "--make-bed to AMPPD_FILTERED.RIC3.bed + AMPPD_FILTERED.RIC3.bim +\n",
      "AMPPD_FILTERED.RIC3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_FILTERED --chr 11 -from-bp 8106056 --to-bp 8169055 --make-bed --out AMPPD_FILTERED.RIC3"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "2) Fisher exact test and logistic regression"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Since I added the phenotypes to the plink file, I shouldn't need to set the --pheno? (Doing it anyway)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to RIC3_WGS_AMP_PD.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_FILTERED.RIC3\n",
      "  --fisher\n",
      "  --out RIC3_WGS_AMP_PD\n",
      "  --pheno plink_test_pheno.tab\n",
      "\n",
      "Note: --fisher flag deprecated.  Use '--assoc fisher' or '--model fisher'.\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "773 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values present after --pheno.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999747.\n",
      "773 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "Writing C/C --assoc report to RIC3_WGS_AMP_PD.assoc.fisher ...\n",
      "done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_FILTERED.RIC3 --pheno plink_test_pheno.tab --fisher --out RIC3_WGS_AMP_PD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to RIC3_WGS_AMP_PD_logreg.log.\n",
      "Options in effect:\n",
      "  --bfile AMPPD_FILTERED.RIC3\n",
      "  --ci 0.95\n",
      "  --covar covariateFile.txt\n",
      "  --covar-name SEX, AGE, PC1, PC2, PC3, PC4, PC5, PC6, PC7, PC8, PC9, PC10\n",
      "  --logistic\n",
      "  --out RIC3_WGS_AMP_PD_logreg\n",
      "  --pheno plink_test_pheno.tab\n",
      "\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\n",
      "773 variants loaded from .bim file.\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\n",
      "2576 phenotype values present after --pheno.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "--covar: 12 out of 27 covariates loaded.\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999747.\n",
      "773 variants and 2576 people pass filters and QC.\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\n",
      "Writing logistic model association results to\n",
      "RIC3_WGS_AMP_PD_logreg.assoc.logistic ... 66done.\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_FILTERED.RIC3 --logistic --pheno plink_test_pheno.tab --covar covariateFile.txt --covar-name SEX, AGE, PC1, PC2, PC3, PC4, PC5, PC6, PC7, PC8, PC9, PC10 --out RIC3_WGS_AMP_PD_logreg --ci 0.95"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Downloading files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://./RIC3_WGS_AMP_PD.assoc.fisher [Content-Type=application/octet-stream]...\n",
      "/ [1 files][ 63.8 KiB/ 63.8 KiB]                                                \n",
      "Operation completed over 1 objects/63.8 KiB.                                     \n",
      "Copying file://./RIC3_WGS_AMP_PD.log [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  1.0 KiB/  1.0 KiB]                                                \n",
      "Operation completed over 1 objects/1.0 KiB.                                      \n",
      "Copying file://./RIC3_WGS_AMP_PD_logreg.assoc.logistic [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  1.1 MiB/  1.1 MiB]                                                \n",
      "Operation completed over 1 objects/1.1 MiB.                                      \n",
      "Copying file://./RIC3_WGS_AMP_PD_logreg.log [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  1.1 KiB/  1.1 KiB]                                                \n",
      "Operation completed over 1 objects/1.1 KiB.                                      \n"
     ]
    }
   ],
   "source": [
    "!gsutil cp ./RIC3_WGS_AMP_PD.assoc.fisher gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./RIC3_WGS_AMP_PD.log gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./RIC3_WGS_AMP_PD_logreg.assoc.logistic gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./RIC3_WGS_AMP_PD_logreg.log gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3) Annotation "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\r\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\r\n",
      "Logging to AMPPD_FILTERED.RIC3.log.\r\n",
      "Options in effect:\r\n",
      "  --bfile AMPPD_FILTERED.RIC3\r\n",
      "  --out AMPPD_FILTERED.RIC3\r\n",
      "  --recode vcf-fid\r\n",
      "\r\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\r\n",
      "773 variants loaded from .bim file.\r\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\r\n",
      "2576 phenotype values loaded from .fam.\r\n",
      "Using 1 thread (no multithreaded calculations invoked).\r\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\r\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\r\n",
      "Total genotyping rate is 0.999747.\r\n",
      "773 variants and 2576 people pass filters and QC.\r\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\r\n",
      "--recode vcf-fid to AMPPD_FILTERED.RIC3.vcf ... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\r\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_FILTERED.RIC3 --recode 'vcf-fid' --out AMPPD_FILTERED.RIC3"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Have had some issues with installing rvtests, if the cell below doesn't work, this might\n",
    "\n",
    "%%bash\n",
    "\n",
    "if test -e ~/bin/rvtests; then\n",
    "\n",
    "echo \"rvtests is already installed in ~/bin\"\n",
    "else\n",
    "echo \"rvtests is not installed\"\n",
    "cd ~/bin/\n",
    "\n",
    "wget https://github.com/zhanxw/rvtests/releases/download/v2.1.0/rvtests_linux64.tar.gz\n",
    "\n",
    "tar -xzvf rvtests_linux64.tar.gz \n",
    "\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rvtests is already installed in ~/bin\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "if test -e ~/bin/rvtests; then\n",
    "\n",
    "echo \"rvtests is already installed in ~/bin\"\n",
    "else\n",
    "echo \"rvtests is not installed\"\n",
    "cd ~/bin/\n",
    "\n",
    "git clone https://github.com/zhanxw/rvtests\n",
    "\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ~/bin/rvtests\n",
    "make"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "annovar\n",
      "annovar.latest.tar.gz\n",
      "data_temp\n",
      "example\n",
      "executable\n",
      "gcta_1.93.2beta\n",
      "gcta_1.93.2beta.zip\n",
      "LICENSE\n",
      "plink\n",
      "plink_linux_x86_64_20190304.zip\n",
      "prettify\n",
      "README.md\n",
      "rvtests\n",
      "rvtests_linux64.tar.gz\n",
      "toy.map\n",
      "toy.ped\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/bin/\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "rm AMPPD_FILTERED.RIC3.vcf.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "export PATH=$PATH:~/bin/rvtests/third/tabix-0.2.6/\n",
    "bgzip AMPPD_FILTERED.RIC3.vcf\n",
    "tabix -f -p vcf AMPPD_FILTERED.RIC3.vcf.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Important! You need to add your own download link after registration on the annovar website: https://www.openbioinformatics.org/annovar/annovar_download_form.php"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "annovar is not installed\n",
      "annovar/\n",
      "annovar/example/\n",
      "annovar/example/ex1.avinput\n",
      "annovar/example/example.simple_region\n",
      "annovar/example/example.tab_region\n",
      "annovar/example/ex2.vcf\n",
      "annovar/example/grantham.matrix\n",
      "annovar/example/snplist.txt\n",
      "annovar/example/README\n",
      "annovar/example/gene_xref.txt\n",
      "annovar/example/gene_fullxref.txt\n",
      "annovar/humandb/\n",
      "annovar/humandb/hg19_refGene.txt\n",
      "annovar/humandb/hg19_refGeneMrna.fa\n",
      "annovar/humandb/hg19_refGeneVersion.txt\n",
      "annovar/humandb/hg19_refGeneWithVer.txt\n",
      "annovar/humandb/hg19_refGeneWithVerMrna.fa\n",
      "annovar/humandb/hg19_example_db_generic.txt\n",
      "annovar/humandb/hg19_example_db_gff3.txt\n",
      "annovar/humandb/GRCh37_MT_ensGene.txt\n",
      "annovar/humandb/GRCh37_MT_ensGeneMrna.fa\n",
      "annovar/humandb/hg19_MT_ensGene.txt\n",
      "annovar/humandb/hg19_MT_ensGeneMrna.fa\n",
      "annovar/humandb/genometrax-sample-files-gff/\n",
      "annovar/humandb/genometrax-sample-files-gff/list\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_chip_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_common_snp_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_cosmic_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_cpg_islands_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_dbnsfp_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_disease_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_dnase_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_drug_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_evs_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_gwas_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_hgmd_common_snp_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_hgmd_disease_genes_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_hgmd_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_hgmdimputed_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_miRNA_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_microsatellites_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_omim_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_pathway_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_pgx_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_ptms_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_snps_dbsnp_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_snps_ensembl_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_transfac_sites_featuretype_hg19.gff\n",
      "annovar/humandb/genometrax-sample-files-gff/sample_tss_featuretype_hg19.gff\n",
      "annovar/annotate_variation.pl\n",
      "annovar/table_annovar.pl\n",
      "annovar/convert2annovar.pl\n",
      "annovar/retrieve_seq_from_fasta.pl\n",
      "annovar/coding_change.pl\n",
      "annovar/variants_reduction.pl\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2021-02-02 15:48:20--  http://www.openbioinformatics.org/annovar/download/0wgxR2rIVP/annovar.latest.tar.gz\n",
      "Resolving www.openbioinformatics.org (www.openbioinformatics.org)... 67.205.156.247\n",
      "Connecting to www.openbioinformatics.org (www.openbioinformatics.org)|67.205.156.247|:80... connected.\n",
      "HTTP request sent, awaiting response... 301 Moved Permanently\n",
      "Location: https://www.openbioinformatics.org/annovar/download/0wgxR2rIVP/annovar.latest.tar.gz [following]\n",
      "--2021-02-02 15:48:20--  https://www.openbioinformatics.org/annovar/download/0wgxR2rIVP/annovar.latest.tar.gz\n",
      "Connecting to www.openbioinformatics.org (www.openbioinformatics.org)|67.205.156.247|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 141660305 (135M) [application/x-gzip]\n",
      "Saving to: ‘annovar.latest.tar.gz’\n",
      "\n",
      "     0K .......... .......... .......... .......... ..........  0%  919K 2m31s\n",
      "    50K .......... .......... .......... .......... ..........  0% 1.81M 1m53s\n",
      "   100K .......... .......... .......... .......... ..........  0%  137M 75s\n",
      "   150K .......... .......... .......... .......... ..........  0% 1.85M 75s\n",
      "   200K .......... .......... .......... .......... ..........  0% 88.2M 60s\n",
      "   250K .......... .......... .......... .......... ..........  0%  164M 50s\n",
      "   300K .......... .......... .......... .......... ..........  0%  208M 43s\n",
      "   350K .......... .......... .......... .......... ..........  0%  208M 38s\n",
      "   400K .......... .......... .......... .......... ..........  0% 1.87M 42s\n",
      "   450K .......... .......... .......... .......... ..........  0%  155M 37s\n",
      "   500K .......... .......... .......... .......... ..........  0%  186M 34s\n",
      "   550K .......... .......... .......... .......... ..........  0%  174M 31s\n",
      "   600K .......... .......... .......... .......... ..........  0%  157M 29s\n",
      "   650K .......... .......... .......... .......... ..........  0%  183M 27s\n",
      "   700K .......... .......... .......... .......... ..........  0%  169M 25s\n",
      "   750K .......... .......... .......... .......... ..........  0%  161M 24s\n",
      "   800K .......... .......... .......... .......... ..........  0% 1.91M 26s\n",
      "   850K .......... .......... .......... .......... ..........  0%  159M 25s\n",
      "   900K .......... .......... .......... .......... ..........  0%  172M 24s\n",
      "   950K .......... .......... .......... .......... ..........  0%  192M 23s\n",
      "  1000K .......... .......... .......... .......... ..........  0%  164M 21s\n",
      "  1050K .......... .......... .......... .......... ..........  0%  213M 21s\n",
      "  1100K .......... .......... .......... .......... ..........  0%  210M 20s\n",
      "  1150K .......... .......... .......... .......... ..........  0%  207M 19s\n",
      "  1200K .......... .......... .......... .......... ..........  0%  181M 18s\n",
      "  1250K .......... .......... .......... .......... ..........  0%  175M 17s\n",
      "  1300K .......... .......... .......... .......... ..........  0%  192M 17s\n",
      "  1350K .......... .......... .......... .......... ..........  1%  186M 16s\n",
      "  1400K .......... .......... .......... .......... ..........  1%  165M 16s\n",
      "  1450K .......... .......... .......... .......... ..........  1%  216M 15s\n",
      "  1500K .......... .......... .......... .......... ..........  1%  204M 15s\n",
      "  1550K .......... .......... .......... .......... ..........  1% 2.12M 16s\n",
      "  1600K .......... .......... .......... .......... ..........  1%  128M 16s\n",
      "  1650K .......... .......... .......... .......... ..........  1% 29.5M 15s\n",
      "  1700K .......... .......... .......... .......... ..........  1%  198M 15s\n",
      "  1750K .......... .......... .......... .......... ..........  1%  205M 15s\n",
      "  1800K .......... .......... .......... .......... ..........  1%  214M 14s\n",
      "  1850K .......... .......... .......... .......... ..........  1%  180M 14s\n",
      "  1900K .......... .......... .......... .......... ..........  1%  212M 14s\n",
      "  1950K .......... .......... .......... .......... ..........  1%  210M 13s\n",
      "  2000K .......... .......... .......... .......... ..........  1%  150M 13s\n",
      "  2050K .......... .......... .......... .......... ..........  1%  210M 13s\n",
      "  2100K .......... .......... .......... .......... ..........  1%  213M 12s\n",
      "  2150K .......... .......... .......... .......... ..........  1%  184M 12s\n",
      "  2200K .......... .......... .......... .......... ..........  1%  185M 12s\n",
      "  2250K .......... .......... .......... .......... ..........  1%  204M 12s\n",
      "  2300K .......... .......... .......... .......... ..........  1%  215M 11s\n",
      "  2350K .......... .......... .......... .......... ..........  1%  206M 11s\n",
      "  2400K .......... .......... .......... .......... ..........  1%  171M 11s\n",
      "  2450K .......... .......... .......... .......... ..........  1%  210M 11s\n",
      "  2500K .......... .......... .......... .......... ..........  1%  189M 10s\n",
      "  2550K .......... .......... .......... .......... ..........  1%  212M 10s\n",
      "  2600K .......... .......... .......... .......... ..........  1%  189M 10s\n",
      "  2650K .......... .......... .......... .......... ..........  1%  208M 10s\n",
      "  2700K .......... .......... .......... .......... ..........  1%  199M 10s\n",
      "  2750K .......... .......... .......... .......... ..........  2%  205M 10s\n",
      "  2800K .......... .......... .......... .......... ..........  2%  181M 9s\n",
      "  2850K .......... .......... .......... .......... ..........  2%  210M 9s\n",
      "  2900K .......... .......... .......... .......... ..........  2%  204M 9s\n",
      "  2950K .......... .......... .......... .......... ..........  2% 2.60M 10s\n",
      "  3000K .......... .......... .......... .......... ..........  2% 33.5M 10s\n",
      "  3050K .......... .......... .......... .......... ..........  2% 18.3M 10s\n",
      "  3100K .......... .......... .......... .......... ..........  2%  177M 9s\n",
      "  3150K .......... .......... .......... .......... ..........  2%  214M 9s\n",
      "  3200K .......... .......... .......... .......... ..........  2%  165M 9s\n",
      "  3250K .......... .......... .......... .......... ..........  2%  209M 9s\n",
      "  3300K .......... .......... .......... .......... ..........  2%  188M 9s\n",
      "  3350K .......... .......... .......... .......... ..........  2%  194M 9s\n",
      "  3400K .......... .......... .......... .......... ..........  2%  173M 9s\n",
      "  3450K .......... .......... .......... .......... ..........  2%  205M 9s\n",
      "  3500K .......... .......... .......... .......... ..........  2%  188M 8s\n",
      "  3550K .......... .......... .......... .......... ..........  2%  189M 8s\n",
      "  3600K .......... .......... .......... .......... ..........  2%  166M 8s\n",
      "  3650K .......... .......... .......... .......... ..........  2%  215M 8s\n",
      "  3700K .......... .......... .......... .......... ..........  2%  176M 8s\n",
      "  3750K .......... .......... .......... .......... ..........  2%  186M 8s\n",
      "  3800K .......... .......... .......... .......... ..........  2%  180M 8s\n",
      "  3850K .......... .......... .......... .......... ..........  2%  211M 8s\n",
      "  3900K .......... .......... .......... .......... ..........  2%  193M 8s\n",
      "  3950K .......... .......... .......... .......... ..........  2%  214M 8s\n",
      "  4000K .......... .......... .......... .......... ..........  2%  172M 7s\n",
      "  4050K .......... .......... .......... .......... ..........  2%  216M 7s\n",
      "  4100K .......... .......... .......... .......... ..........  2%  194M 7s\n",
      "  4150K .......... .......... .......... .......... ..........  3%  209M 7s\n",
      "  4200K .......... .......... .......... .......... ..........  3%  186M 7s\n",
      "  4250K .......... .......... .......... .......... ..........  3%  207M 7s\n",
      "  4300K .......... .......... .......... .......... ..........  3%  196M 7s\n",
      "  4350K .......... .......... .......... .......... ..........  3%  209M 7s\n",
      "  4400K .......... .......... .......... .......... ..........  3%  173M 7s\n",
      "  4450K .......... .......... .......... .......... ..........  3%  212M 7s\n",
      "  4500K .......... .......... .......... .......... ..........  3%  202M 7s\n",
      "  4550K .......... .......... .......... .......... ..........  3%  213M 7s\n",
      "  4600K .......... .......... .......... .......... ..........  3%  138M 7s\n",
      "  4650K .......... .......... .......... .......... ..........  3%  212M 7s\n",
      "  4700K .......... .......... .......... .......... ..........  3%  211M 6s\n",
      "  4750K .......... .......... .......... .......... ..........  3%  203M 6s\n",
      "  4800K .......... .......... .......... .......... ..........  3%  173M 6s\n",
      "  4850K .......... .......... .......... .......... ..........  3%  207M 6s\n",
      "  4900K .......... .......... .......... .......... ..........  3%  183M 6s\n",
      "  4950K .......... .......... .......... .......... ..........  3%  182M 6s\n",
      "  5000K .......... .......... .......... .......... ..........  3%  160M 6s\n",
      "  5050K .......... .......... .......... .......... ..........  3%  219M 6s\n",
      "  5100K .......... .......... .......... .......... ..........  3%  210M 6s\n",
      "  5150K .......... .......... .......... .......... ..........  3% 3.91M 6s\n",
      "  5200K .......... .......... .......... .......... ..........  3% 31.3M 6s\n",
      "  5250K .......... .......... .......... .......... ..........  3%  186M 6s\n",
      "  5300K .......... .......... .......... .......... ..........  3%  181M 6s\n",
      "  5350K .......... .......... .......... .......... ..........  3%  153M 6s\n",
      "  5400K .......... .......... .......... .......... ..........  3%  165M 6s\n",
      "  5450K .......... .......... .......... .......... ..........  3%  190M 6s\n",
      "  5500K .......... .......... .......... .......... ..........  4%  176M 6s\n",
      "  5550K .......... .......... .......... .......... ..........  4%  146M 6s\n",
      "  5600K .......... .......... .......... .......... ..........  4%  131M 6s\n",
      "  5650K .......... .......... .......... .......... ..........  4%  182M 6s\n",
      "  5700K .......... .......... .......... .......... ..........  4%  205M 6s\n",
      "  5750K .......... .......... .......... .......... ..........  4%  212M 6s\n",
      "  5800K .......... .......... .......... .......... ..........  4%  186M 6s\n",
      "  5850K .......... .......... .......... .......... ..........  4%  203M 6s\n",
      "  5900K .......... .......... .......... .......... ..........  4%  207M 6s\n",
      "  5950K .......... .......... .......... .......... ..........  4%  203M 6s\n",
      "  6000K .......... .......... .......... .......... ..........  4%  175M 5s\n",
      "  6050K .......... .......... .......... .......... ..........  4%  202M 5s\n",
      "  6100K .......... .......... .......... .......... ..........  4%  193M 5s\n",
      "  6150K .......... .......... .......... .......... ..........  4%  216M 5s\n",
      "  6200K .......... .......... .......... .......... ..........  4%  182M 5s\n",
      "  6250K .......... .......... .......... .......... ..........  4%  212M 5s\n",
      "  6300K .......... .......... .......... .......... ..........  4%  213M 5s\n",
      "  6350K .......... .......... .......... .......... ..........  4%  208M 5s\n",
      "  6400K .......... .......... .......... .......... ..........  4%  176M 5s\n",
      "  6450K .......... .......... .......... .......... ..........  4%  207M 5s\n",
      "  6500K .......... .......... .......... .......... ..........  4%  208M 5s\n",
      "  6550K .......... .......... .......... .......... ..........  4%  207M 5s\n",
      "  6600K .......... .......... .......... .......... ..........  4%  178M 5s\n",
      "  6650K .......... .......... .......... .......... ..........  4%  207M 5s\n",
      "  6700K .......... .......... .......... .......... ..........  4%  201M 5s\n",
      "  6750K .......... .......... .......... .......... ..........  4% 2.67M 5s\n",
      "  6800K .......... .......... .......... .......... ..........  4%  133M 5s\n",
      "  6850K .......... .......... .......... .......... ..........  4% 48.4M 5s\n",
      "  6900K .......... .......... .......... .......... ..........  5% 67.9M 5s\n",
      "  6950K .......... .......... .......... .......... ..........  5% 60.1M 5s\n",
      "  7000K .......... .......... .......... .......... ..........  5%  137M 5s\n",
      "  7050K .......... .......... .......... .......... ..........  5%  186M 5s\n",
      "  7100K .......... .......... .......... .......... ..........  5%  192M 5s\n",
      "  7150K .......... .......... .......... .......... ..........  5%  209M 5s\n",
      "  7200K .......... .......... .......... .......... ..........  5%  166M 5s\n",
      "  7250K .......... .......... .......... .......... ..........  5%  209M 5s\n",
      "  7300K .......... .......... .......... .......... ..........  5%  208M 5s\n",
      "  7350K .......... .......... .......... .......... ..........  5%  202M 5s\n",
      "  7400K .......... .......... .......... .......... ..........  5%  177M 5s\n",
      "  7450K .......... .......... .......... .......... ..........  5%  201M 5s\n",
      "  7500K .......... .......... .......... .......... ..........  5%  211M 5s\n",
      "  7550K .......... .......... .......... .......... ..........  5%  211M 5s\n",
      "  7600K .......... .......... .......... .......... ..........  5%  171M 5s\n",
      "  7650K .......... .......... .......... .......... ..........  5%  209M 5s\n",
      "  7700K .......... .......... .......... .......... ..........  5%  186M 5s\n",
      "  7750K .......... .......... .......... .......... ..........  5%  208M 5s\n",
      "  7800K .......... .......... .......... .......... ..........  5%  184M 5s\n",
      "  7850K .......... .......... .......... .......... ..........  5%  204M 5s\n",
      "  7900K .......... .......... .......... .......... ..........  5%  215M 5s\n",
      "  7950K .......... .......... .......... .......... ..........  5%  206M 5s\n",
      "  8000K .......... .......... .......... .......... ..........  5%  164M 5s\n",
      "  8050K .......... .......... .......... .......... ..........  5%  211M 5s\n",
      "  8100K .......... .......... .......... .......... ..........  5%  201M 4s\n",
      "  8150K .......... .......... .......... .......... ..........  5%  211M 4s\n",
      "  8200K .......... .......... .......... .......... ..........  5%  176M 4s\n",
      "  8250K .......... .......... .......... .......... ..........  5%  210M 4s\n",
      "  8300K .......... .......... .......... .......... ..........  6%  211M 4s\n",
      "  8350K .......... .......... .......... .......... ..........  6%  208M 4s\n",
      "  8400K .......... .......... .......... .......... ..........  6%  177M 4s\n",
      "  8450K .......... .......... .......... .......... ..........  6%  198M 4s\n",
      "  8500K .......... .......... .......... .......... ..........  6%  198M 4s\n",
      "  8550K .......... .......... .......... .......... ..........  6%  209M 4s\n",
      "  8600K .......... .......... .......... .......... ..........  6%  186M 4s\n",
      "  8650K .......... .......... .......... .......... ..........  6%  217M 4s\n",
      "  8700K .......... .......... .......... .......... ..........  6%  198M 4s\n",
      "  8750K .......... .......... .......... .......... ..........  6%  211M 4s\n",
      "  8800K .......... .......... .......... .......... ..........  6%  168M 4s\n",
      "  8850K .......... .......... .......... .......... ..........  6%  215M 4s\n",
      "  8900K .......... .......... .......... .......... ..........  6%  215M 4s\n",
      "  8950K .......... .......... .......... .......... ..........  6%  206M 4s\n",
      "  9000K .......... .......... .......... .......... ..........  6%  188M 4s\n",
      "  9050K .......... .......... .......... .......... ..........  6%  212M 4s\n",
      "  9100K .......... .......... .......... .......... ..........  6% 3.50M 4s\n",
      "  9150K .......... .......... .......... .......... ..........  6% 80.2M 4s\n",
      "  9200K .......... .......... .......... .......... ..........  6%  108M 4s\n",
      "  9250K .......... .......... .......... .......... ..........  6%  169M 4s\n",
      "  9300K .......... .......... .......... .......... ..........  6%  187M 4s\n",
      "  9350K .......... .......... .......... .......... ..........  6%  168M 4s\n",
      "  9400K .......... .......... .......... .......... ..........  6%  168M 4s\n",
      "  9450K .......... .......... .......... .......... ..........  6%  178M 4s\n",
      "  9500K .......... .......... .......... .......... ..........  6%  168M 4s\n",
      "  9550K .......... .......... .......... .......... ..........  6%  163M 4s\n",
      "  9600K .......... .......... .......... .......... ..........  6%  155M 4s\n",
      "  9650K .......... .......... .......... .......... ..........  7%  212M 4s\n",
      "  9700K .......... .......... .......... .......... ..........  7%  207M 4s\n",
      "  9750K .......... .......... .......... .......... ..........  7%  216M 4s\n",
      "  9800K .......... .......... .......... .......... ..........  7%  188M 4s\n",
      "  9850K .......... .......... .......... .......... ..........  7%  207M 4s\n",
      "  9900K .......... .......... .......... .......... ..........  7%  205M 4s\n",
      "  9950K .......... .......... .......... .......... ..........  7%  210M 4s\n",
      " 10000K .......... .......... .......... .......... ..........  7%  175M 4s\n",
      " 10050K .......... .......... .......... .......... ..........  7%  212M 4s\n",
      " 10100K .......... .......... .......... .......... ..........  7%  204M 4s\n",
      " 10150K .......... .......... .......... .......... ..........  7%  211M 4s\n",
      " 10200K .......... .......... .......... .......... ..........  7%  187M 4s\n",
      " 10250K .......... .......... .......... .......... ..........  7%  169M 4s\n",
      " 10300K .......... .......... .......... .......... ..........  7%  207M 4s\n",
      " 10350K .......... .......... .......... .......... ..........  7%  214M 4s\n",
      " 10400K .......... .......... .......... .......... ..........  7%  171M 4s\n",
      " 10450K .......... .......... .......... .......... ..........  7%  213M 4s\n",
      " 10500K .......... .......... .......... .......... ..........  7%  210M 4s\n",
      " 10550K .......... .......... .......... .......... ..........  7%  214M 4s\n",
      " 10600K .......... .......... .......... .......... ..........  7%  182M 4s\n",
      " 10650K .......... .......... .......... .......... ..........  7%  213M 4s\n",
      " 10700K .......... .......... .......... .......... ..........  7% 2.71M 4s\n",
      " 10750K .......... .......... .......... .......... ..........  7% 16.0M 4s\n",
      " 10800K .......... .......... .......... .......... ..........  7%  136M 4s\n",
      " 10850K .......... .......... .......... .......... ..........  7%  193M 4s\n",
      " 10900K .......... .......... .......... .......... ..........  7%  209M 4s\n",
      " 10950K .......... .......... .......... .......... ..........  7%  206M 4s\n",
      " 11000K .......... .......... .......... .......... ..........  7%  182M 4s\n",
      " 11050K .......... .......... .......... .......... ..........  8%  205M 4s\n",
      " 11100K .......... .......... .......... .......... ..........  8%  189M 4s\n",
      " 11150K .......... .......... .......... .......... ..........  8%  209M 4s\n",
      " 11200K .......... .......... .......... .......... ..........  8%  172M 4s\n",
      " 11250K .......... .......... .......... .......... ..........  8%  210M 4s\n",
      " 11300K .......... .......... .......... .......... ..........  8%  206M 4s\n",
      " 11350K .......... .......... .......... .......... ..........  8%  213M 4s\n",
      " 11400K .......... .......... .......... .......... ..........  8%  183M 4s\n",
      " 11450K .......... .......... .......... .......... ..........  8%  216M 4s\n",
      " 11500K .......... .......... .......... .......... ..........  8%  211M 4s\n",
      " 11550K .......... .......... .......... .......... ..........  8%  205M 4s\n",
      " 11600K .......... .......... .......... .......... ..........  8%  174M 4s\n",
      " 11650K .......... .......... .......... .......... ..........  8%  206M 4s\n",
      " 11700K .......... .......... .......... .......... ..........  8%  213M 4s\n",
      " 11750K .......... .......... .......... .......... ..........  8%  217M 4s\n",
      " 11800K .......... .......... .......... .......... ..........  8%  178M 4s\n",
      " 11850K .......... .......... .......... .......... ..........  8%  216M 4s\n",
      " 11900K .......... .......... .......... .......... ..........  8%  197M 4s\n",
      " 11950K .......... .......... .......... .......... ..........  8%  217M 4s\n",
      " 12000K .......... .......... .......... .......... ..........  8%  172M 4s\n",
      " 12050K .......... .......... .......... .......... ..........  8%  210M 3s\n",
      " 12100K .......... .......... .......... .......... ..........  8%  214M 3s\n",
      " 12150K .......... .......... .......... .......... ..........  8% 95.4M 3s\n",
      " 12200K .......... .......... .......... .......... ..........  8%  146M 3s\n",
      " 12250K .......... .......... .......... .......... ..........  8%  190M 3s\n",
      " 12300K .......... .......... .......... .......... ..........  8%  183M 3s\n",
      " 12350K .......... .......... .......... .......... ..........  8%  176M 3s\n",
      " 12400K .......... .......... .......... .......... ..........  8%  147M 3s\n",
      " 12450K .......... .......... .......... .......... ..........  9%  163M 3s\n",
      " 12500K .......... .......... .......... .......... ..........  9%  162M 3s\n",
      " 12550K .......... .......... .......... .......... ..........  9%  188M 3s\n",
      " 12600K .......... .......... .......... .......... ..........  9%  167M 3s\n",
      " 12650K .......... .......... .......... .......... ..........  9%  179M 3s\n",
      " 12700K .......... .......... .......... .......... ..........  9%  195M 3s\n",
      " 12750K .......... .......... .......... .......... ..........  9%  199M 3s\n",
      " 12800K .......... .......... .......... .......... ..........  9%  155M 3s\n",
      " 12850K .......... .......... .......... .......... ..........  9%  190M 3s\n",
      " 12900K .......... .......... .......... .......... ..........  9%  190M 3s\n",
      " 12950K .......... .......... .......... .......... ..........  9%  193M 3s\n",
      " 13000K .......... .......... .......... .......... ..........  9% 4.03M 3s\n",
      " 13050K .......... .......... .......... .......... ..........  9% 64.5M 3s\n",
      " 13100K .......... .......... .......... .......... ..........  9%  155M 3s\n",
      " 13150K .......... .......... .......... .......... ..........  9%  170M 3s\n",
      " 13200K .......... .......... .......... .......... ..........  9%  143M 3s\n",
      " 13250K .......... .......... .......... .......... ..........  9%  175M 3s\n",
      " 13300K .......... .......... .......... .......... ..........  9%  202M 3s\n",
      " 13350K .......... .......... .......... .......... ..........  9%  216M 3s\n",
      " 13400K .......... .......... .......... .......... ..........  9%  167M 3s\n",
      " 13450K .......... .......... .......... .......... ..........  9%  185M 3s\n",
      " 13500K .......... .......... .......... .......... ..........  9%  177M 3s\n",
      " 13550K .......... .......... .......... .......... ..........  9%  142M 3s\n",
      " 13600K .......... .......... .......... .......... ..........  9%  118M 3s\n",
      " 13650K .......... .......... .......... .......... ..........  9%  200M 3s\n",
      " 13700K .......... .......... .......... .......... ..........  9%  207M 3s\n",
      " 13750K .......... .......... .......... .......... ..........  9%  210M 3s\n",
      " 13800K .......... .......... .......... .......... .......... 10%  181M 3s\n",
      " 13850K .......... .......... .......... .......... .......... 10%  204M 3s\n",
      " 13900K .......... .......... .......... .......... .......... 10%  205M 3s\n",
      " 13950K .......... .......... .......... .......... .......... 10%  208M 3s\n",
      " 14000K .......... .......... .......... .......... .......... 10%  165M 3s\n",
      " 14050K .......... .......... .......... .......... .......... 10%  203M 3s\n",
      " 14100K .......... .......... .......... .......... .......... 10%  213M 3s\n",
      " 14150K .......... .......... .......... .......... .......... 10%  208M 3s\n",
      " 14200K .......... .......... .......... .......... .......... 10%  185M 3s\n",
      " 14250K .......... .......... .......... .......... .......... 10%  197M 3s\n",
      " 14300K .......... .......... .......... .......... .......... 10%  205M 3s\n",
      " 14350K .......... .......... .......... .......... .......... 10%  211M 3s\n",
      " 14400K .......... .......... .......... .......... .......... 10% 7.40M 3s\n",
      " 14450K .......... .......... .......... .......... .......... 10%  189M 3s\n",
      " 14500K .......... .......... .......... .......... .......... 10%  209M 3s\n",
      " 14550K .......... .......... .......... .......... .......... 10%  191M 3s\n",
      " 14600K .......... .......... .......... .......... .......... 10%  176M 3s\n",
      " 14650K .......... .......... .......... .......... .......... 10%  207M 3s\n",
      " 14700K .......... .......... .......... .......... .......... 10%  201M 3s\n",
      " 14750K .......... .......... .......... .......... .......... 10%  215M 3s\n",
      " 14800K .......... .......... .......... .......... .......... 10%  173M 3s\n",
      " 14850K .......... .......... .......... .......... .......... 10%  189M 3s\n",
      " 14900K .......... .......... .......... .......... .......... 10%  179M 3s\n",
      " 14950K .......... .......... .......... .......... .......... 10%  205M 3s\n",
      " 15000K .......... .......... .......... .......... .......... 10%  184M 3s\n",
      " 15050K .......... .......... .......... .......... .......... 10%  214M 3s\n",
      " 15100K .......... .......... .......... .......... .......... 10%  214M 3s\n",
      " 15150K .......... .......... .......... .......... .......... 10%  196M 3s\n",
      " 15200K .......... .......... .......... .......... .......... 11%  176M 3s\n",
      " 15250K .......... .......... .......... .......... .......... 11%  210M 3s\n",
      " 15300K .......... .......... .......... .......... .......... 11%  215M 3s\n",
      " 15350K .......... .......... .......... .......... .......... 11%  213M 3s\n",
      " 15400K .......... .......... .......... .......... .......... 11%  185M 3s\n",
      " 15450K .......... .......... .......... .......... .......... 11%  214M 3s\n",
      " 15500K .......... .......... .......... .......... .......... 11%  202M 3s\n",
      " 15550K .......... .......... .......... .......... .......... 11%  212M 3s\n",
      " 15600K .......... .......... .......... .......... .......... 11%  159M 3s\n",
      " 15650K .......... .......... .......... .......... .......... 11%  169M 3s\n",
      " 15700K .......... .......... .......... .......... .......... 11%  177M 3s\n",
      " 15750K .......... .......... .......... .......... .......... 11%  202M 3s\n",
      " 15800K .......... .......... .......... .......... .......... 11% 8.16M 3s\n",
      " 15850K .......... .......... .......... .......... .......... 11%  150M 3s\n",
      " 15900K .......... .......... .......... .......... .......... 11%  172M 3s\n",
      " 15950K .......... .......... .......... .......... .......... 11%  173M 3s\n",
      " 16000K .......... .......... .......... .......... .......... 11%  146M 3s\n",
      " 16050K .......... .......... .......... .......... .......... 11%  197M 3s\n",
      " 16100K .......... .......... .......... .......... .......... 11%  202M 3s\n",
      " 16150K .......... .......... .......... .......... .......... 11%  204M 3s\n",
      " 16200K .......... .......... .......... .......... .......... 11%  181M 3s\n",
      " 16250K .......... .......... .......... .......... .......... 11%  171M 3s\n",
      " 16300K .......... .......... .......... .......... .......... 11%  129M 3s\n",
      " 16350K .......... .......... .......... .......... .......... 11%  179M 3s\n",
      " 16400K .......... .......... .......... .......... .......... 11%  159M 3s\n",
      " 16450K .......... .......... .......... .......... .......... 11%  190M 3s\n",
      " 16500K .......... .......... .......... .......... .......... 11%  180M 3s\n",
      " 16550K .......... .......... .......... .......... .......... 11%  194M 3s\n",
      " 16600K .......... .......... .......... .......... .......... 12%  188M 3s\n",
      " 16650K .......... .......... .......... .......... .......... 12%  213M 3s\n",
      " 16700K .......... .......... .......... .......... .......... 12%  217M 3s\n",
      " 16750K .......... .......... .......... .......... .......... 12%  204M 3s\n",
      " 16800K .......... .......... .......... .......... .......... 12%  181M 3s\n",
      " 16850K .......... .......... .......... .......... .......... 12%  212M 3s\n",
      " 16900K .......... .......... .......... .......... .......... 12%  199M 3s\n",
      " 16950K .......... .......... .......... .......... .......... 12%  210M 3s\n",
      " 17000K .......... .......... .......... .......... .......... 12% 7.61M 3s\n",
      " 17050K .......... .......... .......... .......... .......... 12% 81.4M 3s\n",
      " 17100K .......... .......... .......... .......... .......... 12%  174M 3s\n",
      " 17150K .......... .......... .......... .......... .......... 12%  194M 3s\n",
      " 17200K .......... .......... .......... .......... .......... 12%  148M 3s\n",
      " 17250K .......... .......... .......... .......... .......... 12%  171M 3s\n",
      " 17300K .......... .......... .......... .......... .......... 12%  210M 3s\n",
      " 17350K .......... .......... .......... .......... .......... 12%  187M 3s\n",
      " 17400K .......... .......... .......... .......... .......... 12%  142M 3s\n",
      " 17450K .......... .......... .......... .......... .......... 12%  189M 3s\n",
      " 17500K .......... .......... .......... .......... .......... 12%  204M 3s\n",
      " 17550K .......... .......... .......... .......... .......... 12%  199M 3s\n",
      " 17600K .......... .......... .......... .......... .......... 12%  171M 3s\n",
      " 17650K .......... .......... .......... .......... .......... 12%  208M 3s\n",
      " 17700K .......... .......... .......... .......... .......... 12%  213M 3s\n",
      " 17750K .......... .......... .......... .......... .......... 12% 4.86M 3s\n",
      " 17800K .......... .......... .......... .......... .......... 12% 19.3M 3s\n",
      " 17850K .......... .......... .......... .......... .......... 12%  150M 3s\n",
      " 17900K .......... .......... .......... .......... .......... 12%  163M 3s\n",
      " 17950K .......... .......... .......... .......... .......... 13%  186M 3s\n",
      " 18000K .......... .......... .......... .......... .......... 13%  151M 3s\n",
      " 18050K .......... .......... .......... .......... .......... 13%  168M 3s\n",
      " 18100K .......... .......... .......... .......... .......... 13%  170M 3s\n",
      " 18150K .......... .......... .......... .......... .......... 13%  202M 3s\n",
      " 18200K .......... .......... .......... .......... .......... 13%  187M 3s\n",
      " 18250K .......... .......... .......... .......... .......... 13%  210M 3s\n",
      " 18300K .......... .......... .......... .......... .......... 13%  142M 3s\n",
      " 18350K .......... .......... .......... .......... .......... 13%  206M 3s\n",
      " 18400K .......... .......... .......... .......... .......... 13%  181M 3s\n",
      " 18450K .......... .......... .......... .......... .......... 13%  219M 3s\n",
      " 18500K .......... .......... .......... .......... .......... 13%  210M 3s\n",
      " 18550K .......... .......... .......... .......... .......... 13%  187M 3s\n",
      " 18600K .......... .......... .......... .......... .......... 13%  188M 3s\n",
      " 18650K .......... .......... .......... .......... .......... 13%  213M 3s\n",
      " 18700K .......... .......... .......... .......... .......... 13%  215M 3s\n",
      " 18750K .......... .......... .......... .......... .......... 13%  167M 3s\n",
      " 18800K .......... .......... .......... .......... .......... 13%  178M 3s\n",
      " 18850K .......... .......... .......... .......... .......... 13%  202M 3s\n",
      " 18900K .......... .......... .......... .......... .......... 13%  180M 3s\n",
      " 18950K .......... .......... .......... .......... .......... 13%  210M 3s\n",
      " 19000K .......... .......... .......... .......... .......... 13%  186M 3s\n",
      " 19050K .......... .......... .......... .......... .......... 13%  203M 3s\n",
      " 19100K .......... .......... .......... .......... .......... 13%  199M 3s\n",
      " 19150K .......... .......... .......... .......... .......... 13%  207M 3s\n",
      " 19200K .......... .......... .......... .......... .......... 13%  178M 3s\n",
      " 19250K .......... .......... .......... .......... .......... 13%  214M 3s\n",
      " 19300K .......... .......... .......... .......... .......... 13%  209M 3s\n",
      " 19350K .......... .......... .......... .......... .......... 14%  208M 3s\n",
      " 19400K .......... .......... .......... .......... .......... 14%  193M 3s\n",
      " 19450K .......... .......... .......... .......... .......... 14%  219M 3s\n",
      " 19500K .......... .......... .......... .......... .......... 14%  177M 3s\n",
      " 19550K .......... .......... .......... .......... .......... 14%  214M 3s\n",
      " 19600K .......... .......... .......... .......... .......... 14%  149M 3s\n",
      " 19650K .......... .......... .......... .......... .......... 14%  198M 3s\n",
      " 19700K .......... .......... .......... .......... .......... 14%  209M 3s\n",
      " 19750K .......... .......... .......... .......... .......... 14%  145M 3s\n",
      " 19800K .......... .......... .......... .......... .......... 14%  174M 3s\n",
      " 19850K .......... .......... .......... .......... .......... 14%  162M 2s\n",
      " 19900K .......... .......... .......... .......... .......... 14%  149M 2s\n",
      " 19950K .......... .......... .......... .......... .......... 14%  135M 2s\n",
      " 20000K .......... .......... .......... .......... .......... 14%  148M 2s\n",
      " 20050K .......... .......... .......... .......... .......... 14%  192M 2s\n",
      " 20100K .......... .......... .......... .......... .......... 14%  211M 2s\n",
      " 20150K .......... .......... .......... .......... .......... 14%  215M 2s\n",
      " 20200K .......... .......... .......... .......... .......... 14%  172M 2s\n",
      " 20250K .......... .......... .......... .......... .......... 14%  194M 2s\n",
      " 20300K .......... .......... .......... .......... .......... 14%  214M 2s\n",
      " 20350K .......... .......... .......... .......... .......... 14%  198M 2s\n",
      " 20400K .......... .......... .......... .......... .......... 14%  177M 2s\n",
      " 20450K .......... .......... .......... .......... .......... 14% 4.57M 2s\n",
      " 20500K .......... .......... .......... .......... .......... 14%  119M 2s\n",
      " 20550K .......... .......... .......... .......... .......... 14%  180M 2s\n",
      " 20600K .......... .......... .......... .......... .......... 14%  185M 2s\n",
      " 20650K .......... .......... .......... .......... .......... 14%  176M 2s\n",
      " 20700K .......... .......... .......... .......... .......... 14%  169M 2s\n",
      " 20750K .......... .......... .......... .......... .......... 15%  176M 2s\n",
      " 20800K .......... .......... .......... .......... .......... 15%  172M 2s\n",
      " 20850K .......... .......... .......... .......... .......... 15%  169M 2s\n",
      " 20900K .......... .......... .......... .......... .......... 15%  194M 2s\n",
      " 20950K .......... .......... .......... .......... .......... 15%  189M 2s\n",
      " 21000K .......... .......... .......... .......... .......... 15%  164M 2s\n",
      " 21050K .......... .......... .......... .......... .......... 15%  194M 2s\n",
      " 21100K .......... .......... .......... .......... .......... 15%  185M 2s\n",
      " 21150K .......... .......... .......... .......... .......... 15%  143M 2s\n",
      " 21200K .......... .......... .......... .......... .......... 15%  129M 2s\n",
      " 21250K .......... .......... .......... .......... .......... 15%  205M 2s\n",
      " 21300K .......... .......... .......... .......... .......... 15%  214M 2s\n",
      " 21350K .......... .......... .......... .......... .......... 15%  199M 2s\n",
      " 21400K .......... .......... .......... .......... .......... 15%  160M 2s\n",
      " 21450K .......... .......... .......... .......... .......... 15%  213M 2s\n",
      " 21500K .......... .......... .......... .......... .......... 15%  208M 2s\n",
      " 21550K .......... .......... .......... .......... .......... 15%  210M 2s\n",
      " 21600K .......... .......... .......... .......... .......... 15% 7.57M 2s\n",
      " 21650K .......... .......... .......... .......... .......... 15% 20.3M 2s\n",
      " 21700K .......... .......... .......... .......... .......... 15%  153M 2s\n",
      " 21750K .......... .......... .......... .......... .......... 15%  175M 2s\n",
      " 21800K .......... .......... .......... .......... .......... 15%  185M 2s\n",
      " 21850K .......... .......... .......... .......... .......... 15%  211M 2s\n",
      " 21900K .......... .......... .......... .......... .......... 15%  174M 2s\n",
      " 21950K .......... .......... .......... .......... .......... 15%  197M 2s\n",
      " 22000K .......... .......... .......... .......... .......... 15%  177M 2s\n",
      " 22050K .......... .......... .......... .......... .......... 15%  202M 2s\n",
      " 22100K .......... .......... .......... .......... .......... 16%  201M 2s\n",
      " 22150K .......... .......... .......... .......... .......... 16%  212M 2s\n",
      " 22200K .......... .......... .......... .......... .......... 16%  181M 2s\n",
      " 22250K .......... .......... .......... .......... .......... 16%  215M 2s\n",
      " 22300K .......... .......... .......... .......... .......... 16%  209M 2s\n",
      " 22350K .......... .......... .......... .......... .......... 16%  202M 2s\n",
      " 22400K .......... .......... .......... .......... .......... 16%  178M 2s\n",
      " 22450K .......... .......... .......... .......... .......... 16%  173M 2s\n",
      " 22500K .......... .......... .......... .......... .......... 16%  199M 2s\n",
      " 22550K .......... .......... .......... .......... .......... 16%  210M 2s\n",
      " 22600K .......... .......... .......... .......... .......... 16%  187M 2s\n",
      " 22650K .......... .......... .......... .......... .......... 16%  214M 2s\n",
      " 22700K .......... .......... .......... .......... .......... 16%  214M 2s\n",
      " 22750K .......... .......... .......... .......... .......... 16%  208M 2s\n",
      " 22800K .......... .......... .......... .......... .......... 16%  171M 2s\n",
      " 22850K .......... .......... .......... .......... .......... 16%  193M 2s\n",
      " 22900K .......... .......... .......... .......... .......... 16%  206M 2s\n",
      " 22950K .......... .......... .......... .......... .......... 16%  203M 2s\n",
      " 23000K .......... .......... .......... .......... .......... 16%  191M 2s\n",
      " 23050K .......... .......... .......... .......... .......... 16%  217M 2s\n",
      " 23100K .......... .......... .......... .......... .......... 16%  199M 2s\n",
      " 23150K .......... .......... .......... .......... .......... 16%  207M 2s\n",
      " 23200K .......... .......... .......... .......... .......... 16%  175M 2s\n",
      " 23250K .......... .......... .......... .......... .......... 16%  210M 2s\n",
      " 23300K .......... .......... .......... .......... .......... 16% 10.7M 2s\n",
      " 23350K .......... .......... .......... .......... .......... 16%  141M 2s\n",
      " 23400K .......... .......... .......... .......... .......... 16%  121M 2s\n",
      " 23450K .......... .......... .......... .......... .......... 16%  139M 2s\n",
      " 23500K .......... .......... .......... .......... .......... 17%  140M 2s\n",
      " 23550K .......... .......... .......... .......... .......... 17%  157M 2s\n",
      " 23600K .......... .......... .......... .......... .......... 17% 86.6M 2s\n",
      " 23650K .......... .......... .......... .......... .......... 17%  136M 2s\n",
      " 23700K .......... .......... .......... .......... .......... 17%  138M 2s\n",
      " 23750K .......... .......... .......... .......... .......... 17%  142M 2s\n",
      " 23800K .......... .......... .......... .......... .......... 17%  124M 2s\n",
      " 23850K .......... .......... .......... .......... .......... 17%  165M 2s\n",
      " 23900K .......... .......... .......... .......... .......... 17%  165M 2s\n",
      " 23950K .......... .......... .......... .......... .......... 17%  138M 2s\n",
      " 24000K .......... .......... .......... .......... .......... 17%  133M 2s\n",
      " 24050K .......... .......... .......... .......... .......... 17%  163M 2s\n",
      " 24100K .......... .......... .......... .......... .......... 17%  159M 2s\n",
      " 24150K .......... .......... .......... .......... .......... 17%  157M 2s\n",
      " 24200K .......... .......... .......... .......... .......... 17%  137M 2s\n",
      " 24250K .......... .......... .......... .......... .......... 17%  160M 2s\n",
      " 24300K .......... .......... .......... .......... .......... 17%  182M 2s\n",
      " 24350K .......... .......... .......... .......... .......... 17%  197M 2s\n",
      " 24400K .......... .......... .......... .......... .......... 17% 8.95M 2s\n",
      " 24450K .......... .......... .......... .......... .......... 17%  181M 2s\n",
      " 24500K .......... .......... .......... .......... .......... 17%  208M 2s\n",
      " 24550K .......... .......... .......... .......... .......... 17%  207M 2s\n",
      " 24600K .......... .......... .......... .......... .......... 17%  161M 2s\n",
      " 24650K .......... .......... .......... .......... .......... 17%  176M 2s\n",
      " 24700K .......... .......... .......... .......... .......... 17%  196M 2s\n",
      " 24750K .......... .......... .......... .......... .......... 17%  169M 2s\n",
      " 24800K .......... .......... .......... .......... .......... 17%  155M 2s\n",
      " 24850K .......... .......... .......... .......... .......... 17%  217M 2s\n",
      " 24900K .......... .......... .......... .......... .......... 18%  217M 2s\n",
      " 24950K .......... .......... .......... .......... .......... 18%  188M 2s\n",
      " 25000K .......... .......... .......... .......... .......... 18%  186M 2s\n",
      " 25050K .......... .......... .......... .......... .......... 18%  208M 2s\n",
      " 25100K .......... .......... .......... .......... .......... 18% 4.45M 2s\n",
      " 25150K .......... .......... .......... .......... .......... 18%  185M 2s\n",
      " 25200K .......... .......... .......... .......... .......... 18% 85.6M 2s\n",
      " 25250K .......... .......... .......... .......... .......... 18% 64.2M 2s\n",
      " 25300K .......... .......... .......... .......... .......... 18% 95.1M 2s\n",
      " 25350K .......... .......... .......... .......... .......... 18% 70.9M 2s\n",
      " 25400K .......... .......... .......... .......... .......... 18%  132M 2s\n",
      " 25450K .......... .......... .......... .......... .......... 18%  171M 2s\n",
      " 25500K .......... .......... .......... .......... .......... 18%  187M 2s\n",
      " 25550K .......... .......... .......... .......... .......... 18%  201M 2s\n",
      " 25600K .......... .......... .......... .......... .......... 18%  179M 2s\n",
      " 25650K .......... .......... .......... .......... .......... 18%  211M 2s\n",
      " 25700K .......... .......... .......... .......... .......... 18%  211M 2s\n",
      " 25750K .......... .......... .......... .......... .......... 18%  208M 2s\n",
      " 25800K .......... .......... .......... .......... .......... 18%  187M 2s\n",
      " 25850K .......... .......... .......... .......... .......... 18%  217M 2s\n",
      " 25900K .......... .......... .......... .......... .......... 18%  173M 2s\n",
      " 25950K .......... .......... .......... .......... .......... 18%  199M 2s\n",
      " 26000K .......... .......... .......... .......... .......... 18%  180M 2s\n",
      " 26050K .......... .......... .......... .......... .......... 18%  202M 2s\n",
      " 26100K .......... .......... .......... .......... .......... 18%  210M 2s\n",
      " 26150K .......... .......... .......... .......... .......... 18%  212M 2s\n",
      " 26200K .......... .......... .......... .......... .......... 18%  185M 2s\n",
      " 26250K .......... .......... .......... .......... .......... 19%  215M 2s\n",
      " 26300K .......... .......... .......... .......... .......... 19%  177M 2s\n",
      " 26350K .......... .......... .......... .......... .......... 19%  214M 2s\n",
      " 26400K .......... .......... .......... .......... .......... 19%  176M 2s\n",
      " 26450K .......... .......... .......... .......... .......... 19%  206M 2s\n",
      " 26500K .......... .......... .......... .......... .......... 19%  205M 2s\n",
      " 26550K .......... .......... .......... .......... .......... 19%  210M 2s\n",
      " 26600K .......... .......... .......... .......... .......... 19%  165M 2s\n",
      " 26650K .......... .......... .......... .......... .......... 19%  211M 2s\n",
      " 26700K .......... .......... .......... .......... .......... 19%  139M 2s\n",
      " 26750K .......... .......... .......... .......... .......... 19%  212M 2s\n",
      " 26800K .......... .......... .......... .......... .......... 19%  178M 2s\n",
      " 26850K .......... .......... .......... .......... .......... 19%  203M 2s\n",
      " 26900K .......... .......... .......... .......... .......... 19%  209M 2s\n",
      " 26950K .......... .......... .......... .......... .......... 19%  207M 2s\n",
      " 27000K .......... .......... .......... .......... .......... 19%  190M 2s\n",
      " 27050K .......... .......... .......... .......... .......... 19%  212M 2s\n",
      " 27100K .......... .......... .......... .......... .......... 19% 70.4M 2s\n",
      " 27150K .......... .......... .......... .......... .......... 19%  117M 2s\n",
      " 27200K .......... .......... .......... .......... .......... 19% 87.5M 2s\n",
      " 27250K .......... .......... .......... .......... .......... 19%  199M 2s\n",
      " 27300K .......... .......... .......... .......... .......... 19%  187M 2s\n",
      " 27350K .......... .......... .......... .......... .......... 19%  169M 2s\n",
      " 27400K .......... .......... .......... .......... .......... 19%  137M 2s\n",
      " 27450K .......... .......... .......... .......... .......... 19%  153M 2s\n",
      " 27500K .......... .......... .......... .......... .......... 19%  199M 2s\n",
      " 27550K .......... .......... .......... .......... .......... 19%  211M 2s\n",
      " 27600K .......... .......... .......... .......... .......... 19%  157M 2s\n",
      " 27650K .......... .......... .......... .......... .......... 20%  181M 2s\n",
      " 27700K .......... .......... .......... .......... .......... 20%  177M 2s\n",
      " 27750K .......... .......... .......... .......... .......... 20%  206M 2s\n",
      " 27800K .......... .......... .......... .......... .......... 20%  189M 2s\n",
      " 27850K .......... .......... .......... .......... .......... 20%  214M 2s\n",
      " 27900K .......... .......... .......... .......... .......... 20%  191M 2s\n",
      " 27950K .......... .......... .......... .......... .......... 20%  212M 2s\n",
      " 28000K .......... .......... .......... .......... .......... 20% 5.14M 2s\n",
      " 28050K .......... .......... .......... .......... .......... 20%  137M 2s\n",
      " 28100K .......... .......... .......... .......... .......... 20%  181M 2s\n",
      " 28150K .......... .......... .......... .......... .......... 20%  171M 2s\n",
      " 28200K .......... .......... .......... .......... .......... 20%  173M 2s\n",
      " 28250K .......... .......... .......... .......... .......... 20%  212M 2s\n",
      " 28300K .......... .......... .......... .......... .......... 20%  194M 2s\n",
      " 28350K .......... .......... .......... .......... .......... 20%  188M 2s\n",
      " 28400K .......... .......... .......... .......... .......... 20%  119M 2s\n",
      " 28450K .......... .......... .......... .......... .......... 20%  177M 2s\n",
      " 28500K .......... .......... .......... .......... .......... 20%  191M 2s\n",
      " 28550K .......... .......... .......... .......... .......... 20%  166M 2s\n",
      " 28600K .......... .......... .......... .......... .......... 20%  152M 2s\n",
      " 28650K .......... .......... .......... .......... .......... 20%  169M 2s\n",
      " 28700K .......... .......... .......... .......... .......... 20%  191M 2s\n",
      " 28750K .......... .......... .......... .......... .......... 20%  211M 2s\n",
      " 28800K .......... .......... .......... .......... .......... 20%  173M 2s\n",
      " 28850K .......... .......... .......... .......... .......... 20%  212M 2s\n",
      " 28900K .......... .......... .......... .......... .......... 20%  208M 2s\n",
      " 28950K .......... .......... .......... .......... .......... 20%  176M 2s\n",
      " 29000K .......... .......... .......... .......... .......... 20%  188M 2s\n",
      " 29050K .......... .......... .......... .......... .......... 21% 5.56M 2s\n",
      " 29100K .......... .......... .......... .......... .......... 21%  175M 2s\n",
      " 29150K .......... .......... .......... .......... .......... 21%  165M 2s\n",
      " 29200K .......... .......... .......... .......... .......... 21%  153M 2s\n",
      " 29250K .......... .......... .......... .......... .......... 21%  180M 2s\n",
      " 29300K .......... .......... .......... .......... .......... 21%  178M 2s\n",
      " 29350K .......... .......... .......... .......... .......... 21%  212M 2s\n",
      " 29400K .......... .......... .......... .......... .......... 21%  182M 2s\n",
      " 29450K .......... .......... .......... .......... .......... 21%  212M 2s\n",
      " 29500K .......... .......... .......... .......... .......... 21%  214M 2s\n",
      " 29550K .......... .......... .......... .......... .......... 21%  206M 2s\n",
      " 29600K .......... .......... .......... .......... .......... 21%  178M 2s\n",
      " 29650K .......... .......... .......... .......... .......... 21%  204M 2s\n",
      " 29700K .......... .......... .......... .......... .......... 21%  210M 2s\n",
      " 29750K .......... .......... .......... .......... .......... 21%  211M 2s\n",
      " 29800K .......... .......... .......... .......... .......... 21%  183M 2s\n",
      " 29850K .......... .......... .......... .......... .......... 21%  209M 2s\n",
      " 29900K .......... .......... .......... .......... .......... 21%  206M 2s\n",
      " 29950K .......... .......... .......... .......... .......... 21%  210M 2s\n",
      " 30000K .......... .......... .......... .......... .......... 21%  172M 2s\n",
      " 30050K .......... .......... .......... .......... .......... 21%  214M 2s\n",
      " 30100K .......... .......... .......... .......... .......... 21%  216M 2s\n",
      " 30150K .......... .......... .......... .......... .......... 21%  199M 2s\n",
      " 30200K .......... .......... .......... .......... .......... 21%  187M 2s\n",
      " 30250K .......... .......... .......... .......... .......... 21%  212M 2s\n",
      " 30300K .......... .......... .......... .......... .......... 21%  217M 2s\n",
      " 30350K .......... .......... .......... .......... .......... 21%  217M 2s\n",
      " 30400K .......... .......... .......... .......... .......... 22%  166M 2s\n",
      " 30450K .......... .......... .......... .......... .......... 22%  218M 2s\n",
      " 30500K .......... .......... .......... .......... .......... 22%  212M 2s\n",
      " 30550K .......... .......... .......... .......... .......... 22%  193M 2s\n",
      " 30600K .......... .......... .......... .......... .......... 22%  183M 2s\n",
      " 30650K .......... .......... .......... .......... .......... 22%  206M 2s\n",
      " 30700K .......... .......... .......... .......... .......... 22%  212M 2s\n",
      " 30750K .......... .......... .......... .......... .......... 22%  163M 2s\n",
      " 30800K .......... .......... .......... .......... .......... 22% 11.0M 2s\n",
      " 30850K .......... .......... .......... .......... .......... 22%  152M 2s\n",
      " 30900K .......... .......... .......... .......... .......... 22%  183M 2s\n",
      " 30950K .......... .......... .......... .......... .......... 22%  209M 2s\n",
      " 31000K .......... .......... .......... .......... .......... 22%  121M 2s\n",
      " 31050K .......... .......... .......... .......... .......... 22%  186M 2s\n",
      " 31100K .......... .......... .......... .......... .......... 22%  190M 2s\n",
      " 31150K .......... .......... .......... .......... .......... 22% 80.9M 2s\n",
      " 31200K .......... .......... .......... .......... .......... 22%  159M 2s\n",
      " 31250K .......... .......... .......... .......... .......... 22%  161M 2s\n",
      " 31300K .......... .......... .......... .......... .......... 22%  180M 2s\n",
      " 31350K .......... .......... .......... .......... .......... 22%  169M 2s\n",
      " 31400K .......... .......... .......... .......... .......... 22%  131M 2s\n",
      " 31450K .......... .......... .......... .......... .......... 22%  206M 2s\n",
      " 31500K .......... .......... .......... .......... .......... 22%  209M 2s\n",
      " 31550K .......... .......... .......... .......... .......... 22%  199M 2s\n",
      " 31600K .......... .......... .......... .......... .......... 22%  166M 2s\n",
      " 31650K .......... .......... .......... .......... .......... 22%  209M 2s\n",
      " 31700K .......... .......... .......... .......... .......... 22%  217M 2s\n",
      " 31750K .......... .......... .......... .......... .......... 22%  194M 2s\n",
      " 31800K .......... .......... .......... .......... .......... 23%  182M 2s\n",
      " 31850K .......... .......... .......... .......... .......... 23% 7.88M 2s\n",
      " 31900K .......... .......... .......... .......... .......... 23%  167M 2s\n",
      " 31950K .......... .......... .......... .......... .......... 23%  182M 2s\n",
      " 32000K .......... .......... .......... .......... .......... 23%  156M 2s\n",
      " 32050K .......... .......... .......... .......... .......... 23%  217M 2s\n",
      " 32100K .......... .......... .......... .......... .......... 23%  210M 2s\n",
      " 32150K .......... .......... .......... .......... .......... 23%  184M 2s\n",
      " 32200K .......... .......... .......... .......... .......... 23%  138M 2s\n",
      " 32250K .......... .......... .......... .......... .......... 23%  186M 2s\n",
      " 32300K .......... .......... .......... .......... .......... 23%  192M 2s\n",
      " 32350K .......... .......... .......... .......... .......... 23%  193M 2s\n",
      " 32400K .......... .......... .......... .......... .......... 23%  153M 2s\n",
      " 32450K .......... .......... .......... .......... .......... 23%  171M 2s\n",
      " 32500K .......... .......... .......... .......... .......... 23%  199M 2s\n",
      " 32550K .......... .......... .......... .......... .......... 23%  201M 2s\n",
      " 32600K .......... .......... .......... .......... .......... 23%  192M 2s\n",
      " 32650K .......... .......... .......... .......... .......... 23%  212M 2s\n",
      " 32700K .......... .......... .......... .......... .......... 23%  167M 2s\n",
      " 32750K .......... .......... .......... .......... .......... 23%  206M 2s\n",
      " 32800K .......... .......... .......... .......... .......... 23% 4.59M 2s\n",
      " 32850K .......... .......... .......... .......... .......... 23% 43.4M 2s\n",
      " 32900K .......... .......... .......... .......... .......... 23%  143M 2s\n",
      " 32950K .......... .......... .......... .......... .......... 23%  123M 2s\n",
      " 33000K .......... .......... .......... .......... .......... 23%  128M 2s\n",
      " 33050K .......... .......... .......... .......... .......... 23%  161M 2s\n",
      " 33100K .......... .......... .......... .......... .......... 23%  149M 2s\n",
      " 33150K .......... .......... .......... .......... .......... 23%  151M 2s\n",
      " 33200K .......... .......... .......... .......... .......... 24%  164M 2s\n",
      " 33250K .......... .......... .......... .......... .......... 24%  183M 2s\n",
      " 33300K .......... .......... .......... .......... .......... 24%  185M 2s\n",
      " 33350K .......... .......... .......... .......... .......... 24%  194M 2s\n",
      " 33400K .......... .......... .......... .......... .......... 24%  174M 2s\n",
      " 33450K .......... .......... .......... .......... .......... 24%  191M 2s\n",
      " 33500K .......... .......... .......... .......... .......... 24%  186M 2s\n",
      " 33550K .......... .......... .......... .......... .......... 24%  200M 2s\n",
      " 33600K .......... .......... .......... .......... .......... 24%  164M 2s\n",
      " 33650K .......... .......... .......... .......... .......... 24%  191M 2s\n",
      " 33700K .......... .......... .......... .......... .......... 24%  174M 2s\n",
      " 33750K .......... .......... .......... .......... .......... 24%  191M 2s\n",
      " 33800K .......... .......... .......... .......... .......... 24%  173M 2s\n",
      " 33850K .......... .......... .......... .......... .......... 24%  196M 2s\n",
      " 33900K .......... .......... .......... .......... .......... 24%  143M 2s\n",
      " 33950K .......... .......... .......... .......... .......... 24%  202M 2s\n",
      " 34000K .......... .......... .......... .......... .......... 24%  173M 2s\n",
      " 34050K .......... .......... .......... .......... .......... 24%  211M 2s\n",
      " 34100K .......... .......... .......... .......... .......... 24%  197M 2s\n",
      " 34150K .......... .......... .......... .......... .......... 24%  140M 2s\n",
      " 34200K .......... .......... .......... .......... .......... 24%  172M 2s\n",
      " 34250K .......... .......... .......... .......... .......... 24%  205M 2s\n",
      " 34300K .......... .......... .......... .......... .......... 24%  198M 2s\n",
      " 34350K .......... .......... .......... .......... .......... 24%  167M 2s\n",
      " 34400K .......... .......... .......... .......... .......... 24%  165M 2s\n",
      " 34450K .......... .......... .......... .......... .......... 24%  199M 2s\n",
      " 34500K .......... .......... .......... .......... .......... 24%  197M 2s\n",
      " 34550K .......... .......... .......... .......... .......... 25%  175M 2s\n",
      " 34600K .......... .......... .......... .......... .......... 25%  144M 2s\n",
      " 34650K .......... .......... .......... .......... .......... 25% 66.5M 2s\n",
      " 34700K .......... .......... .......... .......... .......... 25%  115M 2s\n",
      " 34750K .......... .......... .......... .......... .......... 25%  150M 2s\n",
      " 34800K .......... .......... .......... .......... .......... 25%  126M 2s\n",
      " 34850K .......... .......... .......... .......... .......... 25%  149M 2s\n",
      " 34900K .......... .......... .......... .......... .......... 25%  203M 2s\n",
      " 34950K .......... .......... .......... .......... .......... 25%  162M 2s\n",
      " 35000K .......... .......... .......... .......... .......... 25%  143M 2s\n",
      " 35050K .......... .......... .......... .......... .......... 25%  168M 2s\n",
      " 35100K .......... .......... .......... .......... .......... 25%  133M 2s\n",
      " 35150K .......... .......... .......... .......... .......... 25%  174M 2s\n",
      " 35200K .......... .......... .......... .......... .......... 25%  161M 2s\n",
      " 35250K .......... .......... .......... .......... .......... 25%  197M 2s\n",
      " 35300K .......... .......... .......... .......... .......... 25%  200M 2s\n",
      " 35350K .......... .......... .......... .......... .......... 25%  161M 2s\n",
      " 35400K .......... .......... .......... .......... .......... 25%  179M 2s\n",
      " 35450K .......... .......... .......... .......... .......... 25%  190M 2s\n",
      " 35500K .......... .......... .......... .......... .......... 25%  185M 2s\n",
      " 35550K .......... .......... .......... .......... .......... 25%  168M 2s\n",
      " 35600K .......... .......... .......... .......... .......... 25%  167M 2s\n",
      " 35650K .......... .......... .......... .......... .......... 25% 5.44M 2s\n",
      " 35700K .......... .......... .......... .......... .......... 25%  144M 2s\n",
      " 35750K .......... .......... .......... .......... .......... 25%  131M 2s\n",
      " 35800K .......... .......... .......... .......... .......... 25%  114M 2s\n",
      " 35850K .......... .......... .......... .......... .......... 25%  153M 2s\n",
      " 35900K .......... .......... .......... .......... .......... 25%  142M 2s\n",
      " 35950K .......... .......... .......... .......... .......... 26%  182M 2s\n",
      " 36000K .......... .......... .......... .......... .......... 26%  130M 2s\n",
      " 36050K .......... .......... .......... .......... .......... 26%  133M 2s\n",
      " 36100K .......... .......... .......... .......... .......... 26%  163M 2s\n",
      " 36150K .......... .......... .......... .......... .......... 26%  182M 2s\n",
      " 36200K .......... .......... .......... .......... .......... 26%  171M 2s\n",
      " 36250K .......... .......... .......... .......... .......... 26%  198M 2s\n",
      " 36300K .......... .......... .......... .......... .......... 26%  159M 2s\n",
      " 36350K .......... .......... .......... .......... .......... 26%  175M 2s\n",
      " 36400K .......... .......... .......... .......... .......... 26%  162M 2s\n",
      " 36450K .......... .......... .......... .......... .......... 26%  196M 2s\n",
      " 36500K .......... .......... .......... .......... .......... 26%  196M 2s\n",
      " 36550K .......... .......... .......... .......... .......... 26%  161M 2s\n",
      " 36600K .......... .......... .......... .......... .......... 26%  155M 2s\n",
      " 36650K .......... .......... .......... .......... .......... 26%  194M 2s\n",
      " 36700K .......... .......... .......... .......... .......... 26% 8.07M 2s\n",
      " 36750K .......... .......... .......... .......... .......... 26% 25.5M 2s\n",
      " 36800K .......... .......... .......... .......... .......... 26% 21.6M 2s\n",
      " 36850K .......... .......... .......... .......... .......... 26%  194M 2s\n",
      " 36900K .......... .......... .......... .......... .......... 26%  202M 2s\n",
      " 36950K .......... .......... .......... .......... .......... 26%  212M 2s\n",
      " 37000K .......... .......... .......... .......... .......... 26%  170M 2s\n",
      " 37050K .......... .......... .......... .......... .......... 26%  213M 2s\n",
      " 37100K .......... .......... .......... .......... .......... 26%  209M 2s\n",
      " 37150K .......... .......... .......... .......... .......... 26%  215M 2s\n",
      " 37200K .......... .......... .......... .......... .......... 26%  169M 2s\n",
      " 37250K .......... .......... .......... .......... .......... 26%  174M 2s\n",
      " 37300K .......... .......... .......... .......... .......... 26%  211M 2s\n",
      " 37350K .......... .......... .......... .......... .......... 27%  209M 2s\n",
      " 37400K .......... .......... .......... .......... .......... 27%  188M 2s\n",
      " 37450K .......... .......... .......... .......... .......... 27%  200M 2s\n",
      " 37500K .......... .......... .......... .......... .......... 27%  177M 2s\n",
      " 37550K .......... .......... .......... .......... .......... 27%  215M 2s\n",
      " 37600K .......... .......... .......... .......... .......... 27%  173M 2s\n",
      " 37650K .......... .......... .......... .......... .......... 27%  220M 2s\n",
      " 37700K .......... .......... .......... .......... .......... 27%  209M 2s\n",
      " 37750K .......... .......... .......... .......... .......... 27%  179M 2s\n",
      " 37800K .......... .......... .......... .......... .......... 27%  185M 2s\n",
      " 37850K .......... .......... .......... .......... .......... 27%  209M 2s\n",
      " 37900K .......... .......... .......... .......... .......... 27%  205M 2s\n",
      " 37950K .......... .......... .......... .......... .......... 27%  148M 2s\n",
      " 38000K .......... .......... .......... .......... .......... 27%  162M 2s\n",
      " 38050K .......... .......... .......... .......... .......... 27%  205M 2s\n",
      " 38100K .......... .......... .......... .......... .......... 27%  203M 2s\n",
      " 38150K .......... .......... .......... .......... .......... 27%  203M 2s\n",
      " 38200K .......... .......... .......... .......... .......... 27%  164M 2s\n",
      " 38250K .......... .......... .......... .......... .......... 27%  205M 2s\n",
      " 38300K .......... .......... .......... .......... .......... 27%  194M 2s\n",
      " 38350K .......... .......... .......... .......... .......... 27%  204M 2s\n",
      " 38400K .......... .......... .......... .......... .......... 27% 17.0M 2s\n",
      " 38450K .......... .......... .......... .......... .......... 27%  157M 2s\n",
      " 38500K .......... .......... .......... .......... .......... 27%  163M 2s\n",
      " 38550K .......... .......... .......... .......... .......... 27%  144M 2s\n",
      " 38600K .......... .......... .......... .......... .......... 27%  174M 2s\n",
      " 38650K .......... .......... .......... .......... .......... 27%  159M 2s\n",
      " 38700K .......... .......... .......... .......... .......... 28%  130M 2s\n",
      " 38750K .......... .......... .......... .......... .......... 28%  198M 2s\n",
      " 38800K .......... .......... .......... .......... .......... 28%  148M 2s\n",
      " 38850K .......... .......... .......... .......... .......... 28%  188M 2s\n",
      " 38900K .......... .......... .......... .......... .......... 28%  172M 2s\n",
      " 38950K .......... .......... .......... .......... .......... 28%  191M 2s\n",
      " 39000K .......... .......... .......... .......... .......... 28%  193M 2s\n",
      " 39050K .......... .......... .......... .......... .......... 28%  214M 2s\n",
      " 39100K .......... .......... .......... .......... .......... 28%  211M 2s\n",
      " 39150K .......... .......... .......... .......... .......... 28%  166M 2s\n",
      " 39200K .......... .......... .......... .......... .......... 28%  176M 2s\n",
      " 39250K .......... .......... .......... .......... .......... 28%  212M 2s\n",
      " 39300K .......... .......... .......... .......... .......... 28%  211M 2s\n",
      " 39350K .......... .......... .......... .......... .......... 28% 6.97M 2s\n",
      " 39400K .......... .......... .......... .......... .......... 28% 65.1M 2s\n",
      " 39450K .......... .......... .......... .......... .......... 28%  164M 2s\n",
      " 39500K .......... .......... .......... .......... .......... 28%  214M 2s\n",
      " 39550K .......... .......... .......... .......... .......... 28%  222M 2s\n",
      " 39600K .......... .......... .......... .......... .......... 28%  160M 2s\n",
      " 39650K .......... .......... .......... .......... .......... 28%  210M 2s\n",
      " 39700K .......... .......... .......... .......... .......... 28%  207M 2s\n",
      " 39750K .......... .......... .......... .......... .......... 28%  216M 2s\n",
      " 39800K .......... .......... .......... .......... .......... 28%  182M 2s\n",
      " 39850K .......... .......... .......... .......... .......... 28%  224M 2s\n",
      " 39900K .......... .......... .......... .......... .......... 28%  189M 2s\n",
      " 39950K .......... .......... .......... .......... .......... 28% 4.37M 2s\n",
      " 40000K .......... .......... .......... .......... .......... 28%  182M 2s\n",
      " 40050K .......... .......... .......... .......... .......... 28%  194M 2s\n",
      " 40100K .......... .......... .......... .......... .......... 29%  176M 2s\n",
      " 40150K .......... .......... .......... .......... .......... 29%  190M 2s\n",
      " 40200K .......... .......... .......... .......... .......... 29%  186M 2s\n",
      " 40250K .......... .......... .......... .......... .......... 29%  227M 2s\n",
      " 40300K .......... .......... .......... .......... .......... 29%  174M 2s\n",
      " 40350K .......... .......... .......... .......... .......... 29%  176M 2s\n",
      " 40400K .......... .......... .......... .......... .......... 29%  145M 2s\n",
      " 40450K .......... .......... .......... .......... .......... 29%  175M 2s\n",
      " 40500K .......... .......... .......... .......... .......... 29%  190M 2s\n",
      " 40550K .......... .......... .......... .......... .......... 29%  179M 2s\n",
      " 40600K .......... .......... .......... .......... .......... 29%  184M 2s\n",
      " 40650K .......... .......... .......... .......... .......... 29%  220M 2s\n",
      " 40700K .......... .......... .......... .......... .......... 29%  212M 2s\n",
      " 40750K .......... .......... .......... .......... .......... 29%  219M 2s\n",
      " 40800K .......... .......... .......... .......... .......... 29%  175M 2s\n",
      " 40850K .......... .......... .......... .......... .......... 29%  218M 2s\n",
      " 40900K .......... .......... .......... .......... .......... 29%  216M 2s\n",
      " 40950K .......... .......... .......... .......... .......... 29%  213M 2s\n",
      " 41000K .......... .......... .......... .......... .......... 29%  170M 2s\n",
      " 41050K .......... .......... .......... .......... .......... 29%  211M 2s\n",
      " 41100K .......... .......... .......... .......... .......... 29%  218M 2s\n",
      " 41150K .......... .......... .......... .......... .......... 29%  212M 2s\n",
      " 41200K .......... .......... .......... .......... .......... 29%  178M 2s\n",
      " 41250K .......... .......... .......... .......... .......... 29%  216M 2s\n",
      " 41300K .......... .......... .......... .......... .......... 29%  220M 2s\n",
      " 41350K .......... .......... .......... .......... .......... 29%  219M 2s\n",
      " 41400K .......... .......... .......... .......... .......... 29%  194M 2s\n",
      " 41450K .......... .......... .......... .......... .......... 29%  214M 2s\n",
      " 41500K .......... .......... .......... .......... .......... 30%  225M 2s\n",
      " 41550K .......... .......... .......... .......... .......... 30%  219M 2s\n",
      " 41600K .......... .......... .......... .......... .......... 30%  184M 2s\n",
      " 41650K .......... .......... .......... .......... .......... 30%  223M 2s\n",
      " 41700K .......... .......... .......... .......... .......... 30%  225M 2s\n",
      " 41750K .......... .......... .......... .......... .......... 30%  208M 2s\n",
      " 41800K .......... .......... .......... .......... .......... 30%  199M 2s\n",
      " 41850K .......... .......... .......... .......... .......... 30%  211M 2s\n",
      " 41900K .......... .......... .......... .......... .......... 30%  220M 2s\n",
      " 41950K .......... .......... .......... .......... .......... 30%  214M 2s\n",
      " 42000K .......... .......... .......... .......... .......... 30%  182M 2s\n",
      " 42050K .......... .......... .......... .......... .......... 30%  210M 2s\n",
      " 42100K .......... .......... .......... .......... .......... 30% 21.2M 2s\n",
      " 42150K .......... .......... .......... .......... .......... 30%  166M 2s\n",
      " 42200K .......... .......... .......... .......... .......... 30%  171M 2s\n",
      " 42250K .......... .......... .......... .......... .......... 30%  150M 2s\n",
      " 42300K .......... .......... .......... .......... .......... 30%  175M 2s\n",
      " 42350K .......... .......... .......... .......... .......... 30%  201M 2s\n",
      " 42400K .......... .......... .......... .......... .......... 30%  164M 1s\n",
      " 42450K .......... .......... .......... .......... .......... 30%  220M 1s\n",
      " 42500K .......... .......... .......... .......... .......... 30%  184M 1s\n",
      " 42550K .......... .......... .......... .......... .......... 30%  190M 1s\n",
      " 42600K .......... .......... .......... .......... .......... 30%  195M 1s\n",
      " 42650K .......... .......... .......... .......... .......... 30%  221M 1s\n",
      " 42700K .......... .......... .......... .......... .......... 30%  215M 1s\n",
      " 42750K .......... .......... .......... .......... .......... 30% 4.96M 2s\n",
      " 42800K .......... .......... .......... .......... .......... 30%  132M 2s\n",
      " 42850K .......... .......... .......... .......... .......... 31%  181M 2s\n",
      " 42900K .......... .......... .......... .......... .......... 31%  207M 2s\n",
      " 42950K .......... .......... .......... .......... .......... 31%  195M 1s\n",
      " 43000K .......... .......... .......... .......... .......... 31%  160M 1s\n",
      " 43050K .......... .......... .......... .......... .......... 31%  180M 1s\n",
      " 43100K .......... .......... .......... .......... .......... 31%  201M 1s\n",
      " 43150K .......... .......... .......... .......... .......... 31%  179M 1s\n",
      " 43200K .......... .......... .......... .......... .......... 31%  159M 1s\n",
      " 43250K .......... .......... .......... .......... .......... 31%  196M 1s\n",
      " 43300K .......... .......... .......... .......... .......... 31%  213M 1s\n",
      " 43350K .......... .......... .......... .......... .......... 31%  170M 1s\n",
      " 43400K .......... .......... .......... .......... .......... 31%  163M 1s\n",
      " 43450K .......... .......... .......... .......... .......... 31%  184M 1s\n",
      " 43500K .......... .......... .......... .......... .......... 31%  199M 1s\n",
      " 43550K .......... .......... .......... .......... .......... 31%  185M 1s\n",
      " 43600K .......... .......... .......... .......... .......... 31%  155M 1s\n",
      " 43650K .......... .......... .......... .......... .......... 31%  213M 1s\n",
      " 43700K .......... .......... .......... .......... .......... 31%  219M 1s\n",
      " 43750K .......... .......... .......... .......... .......... 31%  205M 1s\n",
      " 43800K .......... .......... .......... .......... .......... 31%  189M 1s\n",
      " 43850K .......... .......... .......... .......... .......... 31%  220M 1s\n",
      " 43900K .......... .......... .......... .......... .......... 31%  213M 1s\n",
      " 43950K .......... .......... .......... .......... .......... 31%  211M 1s\n",
      " 44000K .......... .......... .......... .......... .......... 31%  180M 1s\n",
      " 44050K .......... .......... .......... .......... .......... 31% 6.62M 1s\n",
      " 44100K .......... .......... .......... .......... .......... 31% 22.1M 1s\n",
      " 44150K .......... .......... .......... .......... .......... 31%  174M 1s\n",
      " 44200K .......... .......... .......... .......... .......... 31%  184M 1s\n",
      " 44250K .......... .......... .......... .......... .......... 32%  218M 1s\n",
      " 44300K .......... .......... .......... .......... .......... 32%  217M 1s\n",
      " 44350K .......... .......... .......... .......... .......... 32%  227M 1s\n",
      " 44400K .......... .......... .......... .......... .......... 32%  173M 1s\n",
      " 44450K .......... .......... .......... .......... .......... 32%  218M 1s\n",
      " 44500K .......... .......... .......... .......... .......... 32%  217M 1s\n",
      " 44550K .......... .......... .......... .......... .......... 32%  207M 1s\n",
      " 44600K .......... .......... .......... .......... .......... 32%  189M 1s\n",
      " 44650K .......... .......... .......... .......... .......... 32%  209M 1s\n",
      " 44700K .......... .......... .......... .......... .......... 32%  214M 1s\n",
      " 44750K .......... .......... .......... .......... .......... 32%  193M 1s\n",
      " 44800K .......... .......... .......... .......... .......... 32%  173M 1s\n",
      " 44850K .......... .......... .......... .......... .......... 32%  217M 1s\n",
      " 44900K .......... .......... .......... .......... .......... 32%  216M 1s\n",
      " 44950K .......... .......... .......... .......... .......... 32%  208M 1s\n",
      " 45000K .......... .......... .......... .......... .......... 32%  194M 1s\n",
      " 45050K .......... .......... .......... .......... .......... 32%  212M 1s\n",
      " 45100K .......... .......... .......... .......... .......... 32%  218M 1s\n",
      " 45150K .......... .......... .......... .......... .......... 32%  173M 1s\n",
      " 45200K .......... .......... .......... .......... .......... 32%  184M 1s\n",
      " 45250K .......... .......... .......... .......... .......... 32%  214M 1s\n",
      " 45300K .......... .......... .......... .......... .......... 32%  218M 1s\n",
      " 45350K .......... .......... .......... .......... .......... 32%  219M 1s\n",
      " 45400K .......... .......... .......... .......... .......... 32%  158M 1s\n",
      " 45450K .......... .......... .......... .......... .......... 32%  214M 1s\n",
      " 45500K .......... .......... .......... .......... .......... 32%  223M 1s\n",
      " 45550K .......... .......... .......... .......... .......... 32%  198M 1s\n",
      " 45600K .......... .......... .......... .......... .......... 32%  182M 1s\n",
      " 45650K .......... .......... .......... .......... .......... 33% 12.4M 1s\n",
      " 45700K .......... .......... .......... .......... .......... 33%  190M 1s\n",
      " 45750K .......... .......... .......... .......... .......... 33%  185M 1s\n",
      " 45800K .......... .......... .......... .......... .......... 33%  153M 1s\n",
      " 45850K .......... .......... .......... .......... .......... 33%  190M 1s\n",
      " 45900K .......... .......... .......... .......... .......... 33%  168M 1s\n",
      " 45950K .......... .......... .......... .......... .......... 33%  190M 1s\n",
      " 46000K .......... .......... .......... .......... .......... 33%  151M 1s\n",
      " 46050K .......... .......... .......... .......... .......... 33%  190M 1s\n",
      " 46100K .......... .......... .......... .......... .......... 33%  209M 1s\n",
      " 46150K .......... .......... .......... .......... .......... 33%  136M 1s\n",
      " 46200K .......... .......... .......... .......... .......... 33%  150M 1s\n",
      " 46250K .......... .......... .......... .......... .......... 33%  197M 1s\n",
      " 46300K .......... .......... .......... .......... .......... 33%  208M 1s\n",
      " 46350K .......... .......... .......... .......... .......... 33%  147M 1s\n",
      " 46400K .......... .......... .......... .......... .......... 33%  157M 1s\n",
      " 46450K .......... .......... .......... .......... .......... 33%  221M 1s\n",
      " 46500K .......... .......... .......... .......... .......... 33%  207M 1s\n",
      " 46550K .......... .......... .......... .......... .......... 33%  221M 1s\n",
      " 46600K .......... .......... .......... .......... .......... 33%  158M 1s\n",
      " 46650K .......... .......... .......... .......... .......... 33%  214M 1s\n",
      " 46700K .......... .......... .......... .......... .......... 33%  215M 1s\n",
      " 46750K .......... .......... .......... .......... .......... 33%  213M 1s\n",
      " 46800K .......... .......... .......... .......... .......... 33%  176M 1s\n",
      " 46850K .......... .......... .......... .......... .......... 33% 6.38M 1s\n",
      " 46900K .......... .......... .......... .......... .......... 33%  108M 1s\n",
      " 46950K .......... .......... .......... .......... .......... 33%  212M 1s\n",
      " 47000K .......... .......... .......... .......... .......... 34%  173M 1s\n",
      " 47050K .......... .......... .......... .......... .......... 34%  187M 1s\n",
      " 47100K .......... .......... .......... .......... .......... 34%  157M 1s\n",
      " 47150K .......... .......... .......... .......... .......... 34%  161M 1s\n",
      " 47200K .......... .......... .......... .......... .......... 34%  150M 1s\n",
      " 47250K .......... .......... .......... .......... .......... 34%  211M 1s\n",
      " 47300K .......... .......... .......... .......... .......... 34%  223M 1s\n",
      " 47350K .......... .......... .......... .......... .......... 34%  209M 1s\n",
      " 47400K .......... .......... .......... .......... .......... 34%  195M 1s\n",
      " 47450K .......... .......... .......... .......... .......... 34%  205M 1s\n",
      " 47500K .......... .......... .......... .......... .......... 34%  215M 1s\n",
      " 47550K .......... .......... .......... .......... .......... 34%  186M 1s\n",
      " 47600K .......... .......... .......... .......... .......... 34%  179M 1s\n",
      " 47650K .......... .......... .......... .......... .......... 34% 5.65M 1s\n",
      " 47700K .......... .......... .......... .......... .......... 34% 24.5M 1s\n",
      " 47750K .......... .......... .......... .......... .......... 34%  150M 1s\n",
      " 47800K .......... .......... .......... .......... .......... 34%  175M 1s\n",
      " 47850K .......... .......... .......... .......... .......... 34%  166M 1s\n",
      " 47900K .......... .......... .......... .......... .......... 34%  191M 1s\n",
      " 47950K .......... .......... .......... .......... .......... 34%  192M 1s\n",
      " 48000K .......... .......... .......... .......... .......... 34%  163M 1s\n",
      " 48050K .......... .......... .......... .......... .......... 34%  103M 1s\n",
      " 48100K .......... .......... .......... .......... .......... 34%  194M 1s\n",
      " 48150K .......... .......... .......... .......... .......... 34%  178M 1s\n",
      " 48200K .......... .......... .......... .......... .......... 34%  179M 1s\n",
      " 48250K .......... .......... .......... .......... .......... 34%  200M 1s\n",
      " 48300K .......... .......... .......... .......... .......... 34%  216M 1s\n",
      " 48350K .......... .......... .......... .......... .......... 34%  211M 1s\n",
      " 48400K .......... .......... .......... .......... .......... 35%  174M 1s\n",
      " 48450K .......... .......... .......... .......... .......... 35%  219M 1s\n",
      " 48500K .......... .......... .......... .......... .......... 35%  178M 1s\n",
      " 48550K .......... .......... .......... .......... .......... 35%  207M 1s\n",
      " 48600K .......... .......... .......... .......... .......... 35%  182M 1s\n",
      " 48650K .......... .......... .......... .......... .......... 35%  212M 1s\n",
      " 48700K .......... .......... .......... .......... .......... 35%  218M 1s\n",
      " 48750K .......... .......... .......... .......... .......... 35%  203M 1s\n",
      " 48800K .......... .......... .......... .......... .......... 35%  178M 1s\n",
      " 48850K .......... .......... .......... .......... .......... 35%  210M 1s\n",
      " 48900K .......... .......... .......... .......... .......... 35%  212M 1s\n",
      " 48950K .......... .......... .......... .......... .......... 35%  211M 1s\n",
      " 49000K .......... .......... .......... .......... .......... 35%  192M 1s\n",
      " 49050K .......... .......... .......... .......... .......... 35%  204M 1s\n",
      " 49100K .......... .......... .......... .......... .......... 35%  214M 1s\n",
      " 49150K .......... .......... .......... .......... .......... 35%  207M 1s\n",
      " 49200K .......... .......... .......... .......... .......... 35%  178M 1s\n",
      " 49250K .......... .......... .......... .......... .......... 35%  207M 1s\n",
      " 49300K .......... .......... .......... .......... .......... 35%  212M 1s\n",
      " 49350K .......... .......... .......... .......... .......... 35%  214M 1s\n",
      " 49400K .......... .......... .......... .......... .......... 35%  186M 1s\n",
      " 49450K .......... .......... .......... .......... .......... 35%  216M 1s\n",
      " 49500K .......... .......... .......... .......... .......... 35%  211M 1s\n",
      " 49550K .......... .......... .......... .......... .......... 35% 17.8M 1s\n",
      " 49600K .......... .......... .......... .......... .......... 35%  142M 1s\n",
      " 49650K .......... .......... .......... .......... .......... 35%  133M 1s\n",
      " 49700K .......... .......... .......... .......... .......... 35%  152M 1s\n",
      " 49750K .......... .......... .......... .......... .......... 35%  171M 1s\n",
      " 49800K .......... .......... .......... .......... .......... 36%  162M 1s\n",
      " 49850K .......... .......... .......... .......... .......... 36%  194M 1s\n",
      " 49900K .......... .......... .......... .......... .......... 36%  191M 1s\n",
      " 49950K .......... .......... .......... .......... .......... 36%  170M 1s\n",
      " 50000K .......... .......... .......... .......... .......... 36%  173M 1s\n",
      " 50050K .......... .......... .......... .......... .......... 36%  215M 1s\n",
      " 50100K .......... .......... .......... .......... .......... 36%  183M 1s\n",
      " 50150K .......... .......... .......... .......... .......... 36%  218M 1s\n",
      " 50200K .......... .......... .......... .......... .......... 36%  184M 1s\n",
      " 50250K .......... .......... .......... .......... .......... 36%  210M 1s\n",
      " 50300K .......... .......... .......... .......... .......... 36%  198M 1s\n",
      " 50350K .......... .......... .......... .......... .......... 36%  210M 1s\n",
      " 50400K .......... .......... .......... .......... .......... 36% 5.71M 1s\n",
      " 50450K .......... .......... .......... .......... .......... 36% 92.2M 1s\n",
      " 50500K .......... .......... .......... .......... .......... 36%  173M 1s\n",
      " 50550K .......... .......... .......... .......... .......... 36%  177M 1s\n",
      " 50600K .......... .......... .......... .......... .......... 36%  166M 1s\n",
      " 50650K .......... .......... .......... .......... .......... 36%  207M 1s\n",
      " 50700K .......... .......... .......... .......... .......... 36%  181M 1s\n",
      " 50750K .......... .......... .......... .......... .......... 36%  170M 1s\n",
      " 50800K .......... .......... .......... .......... .......... 36%  160M 1s\n",
      " 50850K .......... .......... .......... .......... .......... 36%  207M 1s\n",
      " 50900K .......... .......... .......... .......... .......... 36%  139M 1s\n",
      " 50950K .......... .......... .......... .......... .......... 36%  186M 1s\n",
      " 51000K .......... .......... .......... .......... .......... 36%  180M 1s\n",
      " 51050K .......... .......... .......... .......... .......... 36%  197M 1s\n",
      " 51100K .......... .......... .......... .......... .......... 36%  202M 1s\n",
      " 51150K .......... .......... .......... .......... .......... 37%  155M 1s\n",
      " 51200K .......... .......... .......... .......... .......... 37%  175M 1s\n",
      " 51250K .......... .......... .......... .......... .......... 37%  212M 1s\n",
      " 51300K .......... .......... .......... .......... .......... 37%  212M 1s\n",
      " 51350K .......... .......... .......... .......... .......... 37%  214M 1s\n",
      " 51400K .......... .......... .......... .......... .......... 37%  186M 1s\n",
      " 51450K .......... .......... .......... .......... .......... 37%  217M 1s\n",
      " 51500K .......... .......... .......... .......... .......... 37%  202M 1s\n",
      " 51550K .......... .......... .......... .......... .......... 37%  220M 1s\n",
      " 51600K .......... .......... .......... .......... .......... 37%  181M 1s\n",
      " 51650K .......... .......... .......... .......... .......... 37% 6.56M 1s\n",
      " 51700K .......... .......... .......... .......... .......... 37%  144M 1s\n",
      " 51750K .......... .......... .......... .......... .......... 37% 99.1M 1s\n",
      " 51800K .......... .......... .......... .......... .......... 37% 47.8M 1s\n",
      " 51850K .......... .......... .......... .......... .......... 37%  166M 1s\n",
      " 51900K .......... .......... .......... .......... .......... 37%  179M 1s\n",
      " 51950K .......... .......... .......... .......... .......... 37%  178M 1s\n",
      " 52000K .......... .......... .......... .......... .......... 37%  183M 1s\n",
      " 52050K .......... .......... .......... .......... .......... 37%  221M 1s\n",
      " 52100K .......... .......... .......... .......... .......... 37%  182M 1s\n",
      " 52150K .......... .......... .......... .......... .......... 37%  207M 1s\n",
      " 52200K .......... .......... .......... .......... .......... 37%  193M 1s\n",
      " 52250K .......... .......... .......... .......... .......... 37%  207M 1s\n",
      " 52300K .......... .......... .......... .......... .......... 37%  221M 1s\n",
      " 52350K .......... .......... .......... .......... .......... 37%  215M 1s\n",
      " 52400K .......... .......... .......... .......... .......... 37%  180M 1s\n",
      " 52450K .......... .......... .......... .......... .......... 37%  212M 1s\n",
      " 52500K .......... .......... .......... .......... .......... 37%  183M 1s\n",
      " 52550K .......... .......... .......... .......... .......... 38%  215M 1s\n",
      " 52600K .......... .......... .......... .......... .......... 38%  193M 1s\n",
      " 52650K .......... .......... .......... .......... .......... 38%  220M 1s\n",
      " 52700K .......... .......... .......... .......... .......... 38%  217M 1s\n",
      " 52750K .......... .......... .......... .......... .......... 38%  218M 1s\n",
      " 52800K .......... .......... .......... .......... .......... 38%  172M 1s\n",
      " 52850K .......... .......... .......... .......... .......... 38%  155M 1s\n",
      " 52900K .......... .......... .......... .......... .......... 38%  213M 1s\n",
      " 52950K .......... .......... .......... .......... .......... 38%  211M 1s\n",
      " 53000K .......... .......... .......... .......... .......... 38%  192M 1s\n",
      " 53050K .......... .......... .......... .......... .......... 38%  202M 1s\n",
      " 53100K .......... .......... .......... .......... .......... 38%  212M 1s\n",
      " 53150K .......... .......... .......... .......... .......... 38%  217M 1s\n",
      " 53200K .......... .......... .......... .......... .......... 38% 10.4M 1s\n",
      " 53250K .......... .......... .......... .......... .......... 38%  158M 1s\n",
      " 53300K .......... .......... .......... .......... .......... 38%  155M 1s\n",
      " 53350K .......... .......... .......... .......... .......... 38%  201M 1s\n",
      " 53400K .......... .......... .......... .......... .......... 38%  181M 1s\n",
      " 53450K .......... .......... .......... .......... .......... 38%  167M 1s\n",
      " 53500K .......... .......... .......... .......... .......... 38%  187M 1s\n",
      " 53550K .......... .......... .......... .......... .......... 38%  194M 1s\n",
      " 53600K .......... .......... .......... .......... .......... 38%  156M 1s\n",
      " 53650K .......... .......... .......... .......... .......... 38%  192M 1s\n",
      " 53700K .......... .......... .......... .......... .......... 38%  181M 1s\n",
      " 53750K .......... .......... .......... .......... .......... 38%  180M 1s\n",
      " 53800K .......... .......... .......... .......... .......... 38%  174M 1s\n",
      " 53850K .......... .......... .......... .......... .......... 38%  212M 1s\n",
      " 53900K .......... .......... .......... .......... .......... 38%  167M 1s\n",
      " 53950K .......... .......... .......... .......... .......... 39%  184M 1s\n",
      " 54000K .......... .......... .......... .......... .......... 39%  174M 1s\n",
      " 54050K .......... .......... .......... .......... .......... 39%  218M 1s\n",
      " 54100K .......... .......... .......... .......... .......... 39%  216M 1s\n",
      " 54150K .......... .......... .......... .......... .......... 39%  215M 1s\n",
      " 54200K .......... .......... .......... .......... .......... 39%  187M 1s\n",
      " 54250K .......... .......... .......... .......... .......... 39%  204M 1s\n",
      " 54300K .......... .......... .......... .......... .......... 39%  213M 1s\n",
      " 54350K .......... .......... .......... .......... .......... 39%  212M 1s\n",
      " 54400K .......... .......... .......... .......... .......... 39% 5.71M 1s\n",
      " 54450K .......... .......... .......... .......... .......... 39%  169M 1s\n",
      " 54500K .......... .......... .......... .......... .......... 39%  186M 1s\n",
      " 54550K .......... .......... .......... .......... .......... 39%  178M 1s\n",
      " 54600K .......... .......... .......... .......... .......... 39%  155M 1s\n",
      " 54650K .......... .......... .......... .......... .......... 39%  182M 1s\n",
      " 54700K .......... .......... .......... .......... .......... 39%  170M 1s\n",
      " 54750K .......... .......... .......... .......... .......... 39%  210M 1s\n",
      " 54800K .......... .......... .......... .......... .......... 39%  178M 1s\n",
      " 54850K .......... .......... .......... .......... .......... 39%  205M 1s\n",
      " 54900K .......... .......... .......... .......... .......... 39%  218M 1s\n",
      " 54950K .......... .......... .......... .......... .......... 39%  209M 1s\n",
      " 55000K .......... .......... .......... .......... .......... 39%  190M 1s\n",
      " 55050K .......... .......... .......... .......... .......... 39%  207M 1s\n",
      " 55100K .......... .......... .......... .......... .......... 39%  218M 1s\n",
      " 55150K .......... .......... .......... .......... .......... 39%  212M 1s\n",
      " 55200K .......... .......... .......... .......... .......... 39%  182M 1s\n",
      " 55250K .......... .......... .......... .......... .......... 39%  200M 1s\n",
      " 55300K .......... .......... .......... .......... .......... 40%  208M 1s\n",
      " 55350K .......... .......... .......... .......... .......... 40%  215M 1s\n",
      " 55400K .......... .......... .......... .......... .......... 40% 6.33M 1s\n",
      " 55450K .......... .......... .......... .......... .......... 40% 98.8M 1s\n",
      " 55500K .......... .......... .......... .......... .......... 40% 69.6M 1s\n",
      " 55550K .......... .......... .......... .......... .......... 40% 98.3M 1s\n",
      " 55600K .......... .......... .......... .......... .......... 40%  147M 1s\n",
      " 55650K .......... .......... .......... .......... .......... 40%  178M 1s\n",
      " 55700K .......... .......... .......... .......... .......... 40%  191M 1s\n",
      " 55750K .......... .......... .......... .......... .......... 40%  178M 1s\n",
      " 55800K .......... .......... .......... .......... .......... 40%  167M 1s\n",
      " 55850K .......... .......... .......... .......... .......... 40%  197M 1s\n",
      " 55900K .......... .......... .......... .......... .......... 40%  203M 1s\n",
      " 55950K .......... .......... .......... .......... .......... 40%  175M 1s\n",
      " 56000K .......... .......... .......... .......... .......... 40%  175M 1s\n",
      " 56050K .......... .......... .......... .......... .......... 40%  204M 1s\n",
      " 56100K .......... .......... .......... .......... .......... 40%  217M 1s\n",
      " 56150K .......... .......... .......... .......... .......... 40%  209M 1s\n",
      " 56200K .......... .......... .......... .......... .......... 40%  176M 1s\n",
      " 56250K .......... .......... .......... .......... .......... 40%  214M 1s\n",
      " 56300K .......... .......... .......... .......... .......... 40%  207M 1s\n",
      " 56350K .......... .......... .......... .......... .......... 40%  212M 1s\n",
      " 56400K .......... .......... .......... .......... .......... 40%  179M 1s\n",
      " 56450K .......... .......... .......... .......... .......... 40%  210M 1s\n",
      " 56500K .......... .......... .......... .......... .......... 40%  218M 1s\n",
      " 56550K .......... .......... .......... .......... .......... 40%  217M 1s\n",
      " 56600K .......... .......... .......... .......... .......... 40%  196M 1s\n",
      " 56650K .......... .......... .......... .......... .......... 40%  215M 1s\n",
      " 56700K .......... .......... .......... .......... .......... 41%  214M 1s\n",
      " 56750K .......... .......... .......... .......... .......... 41%  203M 1s\n",
      " 56800K .......... .......... .......... .......... .......... 41%  180M 1s\n",
      " 56850K .......... .......... .......... .......... .......... 41%  214M 1s\n",
      " 56900K .......... .......... .......... .......... .......... 41%  218M 1s\n",
      " 56950K .......... .......... .......... .......... .......... 41%  207M 1s\n",
      " 57000K .......... .......... .......... .......... .......... 41%  188M 1s\n",
      " 57050K .......... .......... .......... .......... .......... 41%  217M 1s\n",
      " 57100K .......... .......... .......... .......... .......... 41%  214M 1s\n",
      " 57150K .......... .......... .......... .......... .......... 41% 10.6M 1s\n",
      " 57200K .......... .......... .......... .......... .......... 41%  145M 1s\n",
      " 57250K .......... .......... .......... .......... .......... 41%  129M 1s\n",
      " 57300K .......... .......... .......... .......... .......... 41%  162M 1s\n",
      " 57350K .......... .......... .......... .......... .......... 41%  176M 1s\n",
      " 57400K .......... .......... .......... .......... .......... 41%  164M 1s\n",
      " 57450K .......... .......... .......... .......... .......... 41%  132M 1s\n",
      " 57500K .......... .......... .......... .......... .......... 41%  178M 1s\n",
      " 57550K .......... .......... .......... .......... .......... 41%  208M 1s\n",
      " 57600K .......... .......... .......... .......... .......... 41%  144M 1s\n",
      " 57650K .......... .......... .......... .......... .......... 41%  217M 1s\n",
      " 57700K .......... .......... .......... .......... .......... 41%  208M 1s\n",
      " 57750K .......... .......... .......... .......... .......... 41%  217M 1s\n",
      " 57800K .......... .......... .......... .......... .......... 41%  193M 1s\n",
      " 57850K .......... .......... .......... .......... .......... 41%  213M 1s\n",
      " 57900K .......... .......... .......... .......... .......... 41%  219M 1s\n",
      " 57950K .......... .......... .......... .......... .......... 41%  214M 1s\n",
      " 58000K .......... .......... .......... .......... .......... 41%  182M 1s\n",
      " 58050K .......... .......... .......... .......... .......... 41%  214M 1s\n",
      " 58100K .......... .......... .......... .......... .......... 42% 6.24M 1s\n",
      " 58150K .......... .......... .......... .......... .......... 42%  176M 1s\n",
      " 58200K .......... .......... .......... .......... .......... 42%  134M 1s\n",
      " 58250K .......... .......... .......... .......... .......... 42%  178M 1s\n",
      " 58300K .......... .......... .......... .......... .......... 42%  167M 1s\n",
      " 58350K .......... .......... .......... .......... .......... 42%  213M 1s\n",
      " 58400K .......... .......... .......... .......... .......... 42%  140M 1s\n",
      " 58450K .......... .......... .......... .......... .......... 42%  186M 1s\n",
      " 58500K .......... .......... .......... .......... .......... 42%  176M 1s\n",
      " 58550K .......... .......... .......... .......... .......... 42%  187M 1s\n",
      " 58600K .......... .......... .......... .......... .......... 42%  161M 1s\n",
      " 58650K .......... .......... .......... .......... .......... 42%  181M 1s\n",
      " 58700K .......... .......... .......... .......... .......... 42%  194M 1s\n",
      " 58750K .......... .......... .......... .......... .......... 42%  201M 1s\n",
      " 58800K .......... .......... .......... .......... .......... 42%  156M 1s\n",
      " 58850K .......... .......... .......... .......... .......... 42%  209M 1s\n",
      " 58900K .......... .......... .......... .......... .......... 42%  218M 1s\n",
      " 58950K .......... .......... .......... .......... .......... 42%  210M 1s\n",
      " 59000K .......... .......... .......... .......... .......... 42%  193M 1s\n",
      " 59050K .......... .......... .......... .......... .......... 42%  199M 1s\n",
      " 59100K .......... .......... .......... .......... .......... 42%  145M 1s\n",
      " 59150K .......... .......... .......... .......... .......... 42%  211M 1s\n",
      " 59200K .......... .......... .......... .......... .......... 42%  185M 1s\n",
      " 59250K .......... .......... .......... .......... .......... 42%  225M 1s\n",
      " 59300K .......... .......... .......... .......... .......... 42%  220M 1s\n",
      " 59350K .......... .......... .......... .......... .......... 42%  208M 1s\n",
      " 59400K .......... .......... .......... .......... .......... 42%  190M 1s\n",
      " 59450K .......... .......... .......... .......... .......... 43% 6.47M 1s\n",
      " 59500K .......... .......... .......... .......... .......... 43% 39.6M 1s\n",
      " 59550K .......... .......... .......... .......... .......... 43%  161M 1s\n",
      " 59600K .......... .......... .......... .......... .......... 43%  162M 1s\n",
      " 59650K .......... .......... .......... .......... .......... 43%  165M 1s\n",
      " 59700K .......... .......... .......... .......... .......... 43%  180M 1s\n",
      " 59750K .......... .......... .......... .......... .......... 43%  185M 1s\n",
      " 59800K .......... .......... .......... .......... .......... 43%  193M 1s\n",
      " 59850K .......... .......... .......... .......... .......... 43%  162M 1s\n",
      " 59900K .......... .......... .......... .......... .......... 43%  165M 1s\n",
      " 59950K .......... .......... .......... .......... .......... 43%  161M 1s\n",
      " 60000K .......... .......... .......... .......... .......... 43%  170M 1s\n",
      " 60050K .......... .......... .......... .......... .......... 43%  210M 1s\n",
      " 60100K .......... .......... .......... .......... .......... 43%  208M 1s\n",
      " 60150K .......... .......... .......... .......... .......... 43%  213M 1s\n",
      " 60200K .......... .......... .......... .......... .......... 43%  192M 1s\n",
      " 60250K .......... .......... .......... .......... .......... 43%  210M 1s\n",
      " 60300K .......... .......... .......... .......... .......... 43%  166M 1s\n",
      " 60350K .......... .......... .......... .......... .......... 43%  218M 1s\n",
      " 60400K .......... .......... .......... .......... .......... 43%  176M 1s\n",
      " 60450K .......... .......... .......... .......... .......... 43%  217M 1s\n",
      " 60500K .......... .......... .......... .......... .......... 43%  199M 1s\n",
      " 60550K .......... .......... .......... .......... .......... 43%  209M 1s\n",
      " 60600K .......... .......... .......... .......... .......... 43%  191M 1s\n",
      " 60650K .......... .......... .......... .......... .......... 43%  216M 1s\n",
      " 60700K .......... .......... .......... .......... .......... 43%  216M 1s\n",
      " 60750K .......... .......... .......... .......... .......... 43%  212M 1s\n",
      " 60800K .......... .......... .......... .......... .......... 43%  169M 1s\n",
      " 60850K .......... .......... .......... .......... .......... 44% 9.64M 1s\n",
      " 60900K .......... .......... .......... .......... .......... 44%  152M 1s\n",
      " 60950K .......... .......... .......... .......... .......... 44%  150M 1s\n",
      " 61000K .......... .......... .......... .......... .......... 44%  150M 1s\n",
      " 61050K .......... .......... .......... .......... .......... 44%  186M 1s\n",
      " 61100K .......... .......... .......... .......... .......... 44%  178M 1s\n",
      " 61150K .......... .......... .......... .......... .......... 44%  199M 1s\n",
      " 61200K .......... .......... .......... .......... .......... 44%  171M 1s\n",
      " 61250K .......... .......... .......... .......... .......... 44%  191M 1s\n",
      " 61300K .......... .......... .......... .......... .......... 44%  171M 1s\n",
      " 61350K .......... .......... .......... .......... .......... 44%  206M 1s\n",
      " 61400K .......... .......... .......... .......... .......... 44%  158M 1s\n",
      " 61450K .......... .......... .......... .......... .......... 44%  188M 1s\n",
      " 61500K .......... .......... .......... .......... .......... 44%  215M 1s\n",
      " 61550K .......... .......... .......... .......... .......... 44%  222M 1s\n",
      " 61600K .......... .......... .......... .......... .......... 44%  180M 1s\n",
      " 61650K .......... .......... .......... .......... .......... 44%  215M 1s\n",
      " 61700K .......... .......... .......... .......... .......... 44%  218M 1s\n",
      " 61750K .......... .......... .......... .......... .......... 44%  214M 1s\n",
      " 61800K .......... .......... .......... .......... .......... 44%  192M 1s\n",
      " 61850K .......... .......... .......... .......... .......... 44%  217M 1s\n",
      " 61900K .......... .......... .......... .......... .......... 44%  197M 1s\n",
      " 61950K .......... .......... .......... .......... .......... 44%  216M 1s\n",
      " 62000K .......... .......... .......... .......... .......... 44%  169M 1s\n",
      " 62050K .......... .......... .......... .......... .......... 44%  219M 1s\n",
      " 62100K .......... .......... .......... .......... .......... 44% 6.02M 1s\n",
      " 62150K .......... .......... .......... .......... .......... 44%  172M 1s\n",
      " 62200K .......... .......... .......... .......... .......... 44%  141M 1s\n",
      " 62250K .......... .......... .......... .......... .......... 45%  168M 1s\n",
      " 62300K .......... .......... .......... .......... .......... 45%  171M 1s\n",
      " 62350K .......... .......... .......... .......... .......... 45%  179M 1s\n",
      " 62400K .......... .......... .......... .......... .......... 45%  160M 1s\n",
      " 62450K .......... .......... .......... .......... .......... 45%  168M 1s\n",
      " 62500K .......... .......... .......... .......... .......... 45%  209M 1s\n",
      " 62550K .......... .......... .......... .......... .......... 45%  209M 1s\n",
      " 62600K .......... .......... .......... .......... .......... 45%  199M 1s\n",
      " 62650K .......... .......... .......... .......... .......... 45%  191M 1s\n",
      " 62700K .......... .......... .......... .......... .......... 45%  210M 1s\n",
      " 62750K .......... .......... .......... .......... .......... 45%  218M 1s\n",
      " 62800K .......... .......... .......... .......... .......... 45%  182M 1s\n",
      " 62850K .......... .......... .......... .......... .......... 45%  218M 1s\n",
      " 62900K .......... .......... .......... .......... .......... 45%  215M 1s\n",
      " 62950K .......... .......... .......... .......... .......... 45%  216M 1s\n",
      " 63000K .......... .......... .......... .......... .......... 45%  186M 1s\n",
      " 63050K .......... .......... .......... .......... .......... 45%  220M 1s\n",
      " 63100K .......... .......... .......... .......... .......... 45%  219M 1s\n",
      " 63150K .......... .......... .......... .......... .......... 45% 5.55M 1s\n",
      " 63200K .......... .......... .......... .......... .......... 45%  149M 1s\n",
      " 63250K .......... .......... .......... .......... .......... 45%  159M 1s\n",
      " 63300K .......... .......... .......... .......... .......... 45%  215M 1s\n",
      " 63350K .......... .......... .......... .......... .......... 45%  174M 1s\n",
      " 63400K .......... .......... .......... .......... .......... 45%  170M 1s\n",
      " 63450K .......... .......... .......... .......... .......... 45%  195M 1s\n",
      " 63500K .......... .......... .......... .......... .......... 45%  198M 1s\n",
      " 63550K .......... .......... .......... .......... .......... 45%  221M 1s\n",
      " 63600K .......... .......... .......... .......... .......... 46%  167M 1s\n",
      " 63650K .......... .......... .......... .......... .......... 46%  203M 1s\n",
      " 63700K .......... .......... .......... .......... .......... 46%  217M 1s\n",
      " 63750K .......... .......... .......... .......... .......... 46%  212M 1s\n",
      " 63800K .......... .......... .......... .......... .......... 46%  194M 1s\n",
      " 63850K .......... .......... .......... .......... .......... 46%  208M 1s\n",
      " 63900K .......... .......... .......... .......... .......... 46%  211M 1s\n",
      " 63950K .......... .......... .......... .......... .......... 46%  217M 1s\n",
      " 64000K .......... .......... .......... .......... .......... 46%  177M 1s\n",
      " 64050K .......... .......... .......... .......... .......... 46%  228M 1s\n",
      " 64100K .......... .......... .......... .......... .......... 46%  212M 1s\n",
      " 64150K .......... .......... .......... .......... .......... 46%  214M 1s\n",
      " 64200K .......... .......... .......... .......... .......... 46%  186M 1s\n",
      " 64250K .......... .......... .......... .......... .......... 46%  210M 1s\n",
      " 64300K .......... .......... .......... .......... .......... 46%  212M 1s\n",
      " 64350K .......... .......... .......... .......... .......... 46%  216M 1s\n",
      " 64400K .......... .......... .......... .......... .......... 46%  175M 1s\n",
      " 64450K .......... .......... .......... .......... .......... 46%  219M 1s\n",
      " 64500K .......... .......... .......... .......... .......... 46%  212M 1s\n",
      " 64550K .......... .......... .......... .......... .......... 46%  217M 1s\n",
      " 64600K .......... .......... .......... .......... .......... 46%  191M 1s\n",
      " 64650K .......... .......... .......... .......... .......... 46%  222M 1s\n",
      " 64700K .......... .......... .......... .......... .......... 46%  213M 1s\n",
      " 64750K .......... .......... .......... .......... .......... 46% 9.56M 1s\n",
      " 64800K .......... .......... .......... .......... .......... 46%  104M 1s\n",
      " 64850K .......... .......... .......... .......... .......... 46%  160M 1s\n",
      " 64900K .......... .......... .......... .......... .......... 46%  179M 1s\n",
      " 64950K .......... .......... .......... .......... .......... 46%  181M 1s\n",
      " 65000K .......... .......... .......... .......... .......... 47%  146M 1s\n",
      " 65050K .......... .......... .......... .......... .......... 47%  179M 1s\n",
      " 65100K .......... .......... .......... .......... .......... 47%  180M 1s\n",
      " 65150K .......... .......... .......... .......... .......... 47%  189M 1s\n",
      " 65200K .......... .......... .......... .......... .......... 47%  183M 1s\n",
      " 65250K .......... .......... .......... .......... .......... 47%  213M 1s\n",
      " 65300K .......... .......... .......... .......... .......... 47%  221M 1s\n",
      " 65350K .......... .......... .......... .......... .......... 47%  215M 1s\n",
      " 65400K .......... .......... .......... .......... .......... 47%  192M 1s\n",
      " 65450K .......... .......... .......... .......... .......... 47%  215M 1s\n",
      " 65500K .......... .......... .......... .......... .......... 47%  213M 1s\n",
      " 65550K .......... .......... .......... .......... .......... 47%  205M 1s\n",
      " 65600K .......... .......... .......... .......... .......... 47%  177M 1s\n",
      " 65650K .......... .......... .......... .......... .......... 47%  214M 1s\n",
      " 65700K .......... .......... .......... .......... .......... 47%  214M 1s\n",
      " 65750K .......... .......... .......... .......... .......... 47% 6.70M 1s\n",
      " 65800K .......... .......... .......... .......... .......... 47% 78.4M 1s\n",
      " 65850K .......... .......... .......... .......... .......... 47%  202M 1s\n",
      " 65900K .......... .......... .......... .......... .......... 47%  186M 1s\n",
      " 65950K .......... .......... .......... .......... .......... 47%  215M 1s\n",
      " 66000K .......... .......... .......... .......... .......... 47%  149M 1s\n",
      " 66050K .......... .......... .......... .......... .......... 47%  160M 1s\n",
      " 66100K .......... .......... .......... .......... .......... 47%  159M 1s\n",
      " 66150K .......... .......... .......... .......... .......... 47%  208M 1s\n",
      " 66200K .......... .......... .......... .......... .......... 47%  164M 1s\n",
      " 66250K .......... .......... .......... .......... .......... 47%  183M 1s\n",
      " 66300K .......... .......... .......... .......... .......... 47%  175M 1s\n",
      " 66350K .......... .......... .......... .......... .......... 47%  178M 1s\n",
      " 66400K .......... .......... .......... .......... .......... 48%  140M 1s\n",
      " 66450K .......... .......... .......... .......... .......... 48%  199M 1s\n",
      " 66500K .......... .......... .......... .......... .......... 48%  209M 1s\n",
      " 66550K .......... .......... .......... .......... .......... 48%  213M 1s\n",
      " 66600K .......... .......... .......... .......... .......... 48%  197M 1s\n",
      " 66650K .......... .......... .......... .......... .......... 48%  222M 1s\n",
      " 66700K .......... .......... .......... .......... .......... 48%  215M 1s\n",
      " 66750K .......... .......... .......... .......... .......... 48%  192M 1s\n",
      " 66800K .......... .......... .......... .......... .......... 48%  177M 1s\n",
      " 66850K .......... .......... .......... .......... .......... 48%  213M 1s\n",
      " 66900K .......... .......... .......... .......... .......... 48%  215M 1s\n",
      " 66950K .......... .......... .......... .......... .......... 48%  208M 1s\n",
      " 67000K .......... .......... .......... .......... .......... 48% 5.38M 1s\n",
      " 67050K .......... .......... .......... .......... .......... 48% 86.6M 1s\n",
      " 67100K .......... .......... .......... .......... .......... 48%  151M 1s\n",
      " 67150K .......... .......... .......... .......... .......... 48%  164M 1s\n",
      " 67200K .......... .......... .......... .......... .......... 48%  165M 1s\n",
      " 67250K .......... .......... .......... .......... .......... 48%  215M 1s\n",
      " 67300K .......... .......... .......... .......... .......... 48%  212M 1s\n",
      " 67350K .......... .......... .......... .......... .......... 48%  217M 1s\n",
      " 67400K .......... .......... .......... .......... .......... 48%  193M 1s\n",
      " 67450K .......... .......... .......... .......... .......... 48%  209M 1s\n",
      " 67500K .......... .......... .......... .......... .......... 48%  214M 1s\n",
      " 67550K .......... .......... .......... .......... .......... 48%  221M 1s\n",
      " 67600K .......... .......... .......... .......... .......... 48%  178M 1s\n",
      " 67650K .......... .......... .......... .......... .......... 48%  215M 1s\n",
      " 67700K .......... .......... .......... .......... .......... 48%  208M 1s\n",
      " 67750K .......... .......... .......... .......... .......... 49%  223M 1s\n",
      " 67800K .......... .......... .......... .......... .......... 49%  179M 1s\n",
      " 67850K .......... .......... .......... .......... .......... 49%  216M 1s\n",
      " 67900K .......... .......... .......... .......... .......... 49%  211M 1s\n",
      " 67950K .......... .......... .......... .......... .......... 49%  211M 1s\n",
      " 68000K .......... .......... .......... .......... .......... 49%  185M 1s\n",
      " 68050K .......... .......... .......... .......... .......... 49%  222M 1s\n",
      " 68100K .......... .......... .......... .......... .......... 49%  221M 1s\n",
      " 68150K .......... .......... .......... .......... .......... 49%  216M 1s\n",
      " 68200K .......... .......... .......... .......... .......... 49%  189M 1s\n",
      " 68250K .......... .......... .......... .......... .......... 49%  218M 1s\n",
      " 68300K .......... .......... .......... .......... .......... 49%  221M 1s\n",
      " 68350K .......... .......... .......... .......... .......... 49%  204M 1s\n",
      " 68400K .......... .......... .......... .......... .......... 49% 10.9M 1s\n",
      " 68450K .......... .......... .......... .......... .......... 49% 91.9M 1s\n",
      " 68500K .......... .......... .......... .......... .......... 49%  156M 1s\n",
      " 68550K .......... .......... .......... .......... .......... 49%  179M 1s\n",
      " 68600K .......... .......... .......... .......... .......... 49%  190M 1s\n",
      " 68650K .......... .......... .......... .......... .......... 49%  212M 1s\n",
      " 68700K .......... .......... .......... .......... .......... 49%  210M 1s\n",
      " 68750K .......... .......... .......... .......... .......... 49%  169M 1s\n",
      " 68800K .......... .......... .......... .......... .......... 49%  143M 1s\n",
      " 68850K .......... .......... .......... .......... .......... 49%  176M 1s\n",
      " 68900K .......... .......... .......... .......... .......... 49%  178M 1s\n",
      " 68950K .......... .......... .......... .......... .......... 49%  190M 1s\n",
      " 69000K .......... .......... .......... .......... .......... 49%  190M 1s\n",
      " 69050K .......... .......... .......... .......... .......... 49%  204M 1s\n",
      " 69100K .......... .......... .......... .......... .......... 49%  203M 1s\n",
      " 69150K .......... .......... .......... .......... .......... 50%  205M 1s\n",
      " 69200K .......... .......... .......... .......... .......... 50%  171M 1s\n",
      " 69250K .......... .......... .......... .......... .......... 50%  215M 1s\n",
      " 69300K .......... .......... .......... .......... .......... 50%  205M 1s\n",
      " 69350K .......... .......... .......... .......... .......... 50%  210M 1s\n",
      " 69400K .......... .......... .......... .......... .......... 50%  187M 1s\n",
      " 69450K .......... .......... .......... .......... .......... 50%  211M 1s\n",
      " 69500K .......... .......... .......... .......... .......... 50%  215M 1s\n",
      " 69550K .......... .......... .......... .......... .......... 50%  218M 1s\n",
      " 69600K .......... .......... .......... .......... .......... 50%  174M 1s\n",
      " 69650K .......... .......... .......... .......... .......... 50% 5.84M 1s\n",
      " 69700K .......... .......... .......... .......... .......... 50%  136M 1s\n",
      " 69750K .......... .......... .......... .......... .......... 50%  160M 1s\n",
      " 69800K .......... .......... .......... .......... .......... 50%  178M 1s\n",
      " 69850K .......... .......... .......... .......... .......... 50%  167M 1s\n",
      " 69900K .......... .......... .......... .......... .......... 50%  198M 1s\n",
      " 69950K .......... .......... .......... .......... .......... 50% 99.5M 1s\n",
      " 70000K .......... .......... .......... .......... .......... 50%  139M 1s\n",
      " 70050K .......... .......... .......... .......... .......... 50%  216M 1s\n",
      " 70100K .......... .......... .......... .......... .......... 50%  211M 1s\n",
      " 70150K .......... .......... .......... .......... .......... 50%  212M 1s\n",
      " 70200K .......... .......... .......... .......... .......... 50%  196M 1s\n",
      " 70250K .......... .......... .......... .......... .......... 50%  215M 1s\n",
      " 70300K .......... .......... .......... .......... .......... 50%  214M 1s\n",
      " 70350K .......... .......... .......... .......... .......... 50%  212M 1s\n",
      " 70400K .......... .......... .......... .......... .......... 50%  180M 1s\n",
      " 70450K .......... .......... .......... .......... .......... 50%  216M 1s\n",
      " 70500K .......... .......... .......... .......... .......... 50%  219M 1s\n",
      " 70550K .......... .......... .......... .......... .......... 51%  215M 1s\n",
      " 70600K .......... .......... .......... .......... .......... 51% 6.58M 1s\n",
      " 70650K .......... .......... .......... .......... .......... 51% 46.6M 1s\n",
      " 70700K .......... .......... .......... .......... .......... 51%  160M 1s\n",
      " 70750K .......... .......... .......... .......... .......... 51%  160M 1s\n",
      " 70800K .......... .......... .......... .......... .......... 51%  113M 1s\n",
      " 70850K .......... .......... .......... .......... .......... 51%  165M 1s\n",
      " 70900K .......... .......... .......... .......... .......... 51%  191M 1s\n",
      " 70950K .......... .......... .......... .......... .......... 51%  171M 1s\n",
      " 71000K .......... .......... .......... .......... .......... 51%  161M 1s\n",
      " 71050K .......... .......... .......... .......... .......... 51%  184M 1s\n",
      " 71100K .......... .......... .......... .......... .......... 51%  184M 1s\n",
      " 71150K .......... .......... .......... .......... .......... 51%  196M 1s\n",
      " 71200K .......... .......... .......... .......... .......... 51%  176M 1s\n",
      " 71250K .......... .......... .......... .......... .......... 51%  216M 1s\n",
      " 71300K .......... .......... .......... .......... .......... 51%  217M 1s\n",
      " 71350K .......... .......... .......... .......... .......... 51%  222M 1s\n",
      " 71400K .......... .......... .......... .......... .......... 51%  196M 1s\n",
      " 71450K .......... .......... .......... .......... .......... 51%  201M 1s\n",
      " 71500K .......... .......... .......... .......... .......... 51%  215M 1s\n",
      " 71550K .......... .......... .......... .......... .......... 51%  203M 1s\n",
      " 71600K .......... .......... .......... .......... .......... 51%  186M 1s\n",
      " 71650K .......... .......... .......... .......... .......... 51%  216M 1s\n",
      " 71700K .......... .......... .......... .......... .......... 51%  218M 1s\n",
      " 71750K .......... .......... .......... .......... .......... 51%  220M 1s\n",
      " 71800K .......... .......... .......... .......... .......... 51%  191M 1s\n",
      " 71850K .......... .......... .......... .......... .......... 51%  214M 1s\n",
      " 71900K .......... .......... .......... .......... .......... 52%  195M 1s\n",
      " 71950K .......... .......... .......... .......... .......... 52%  213M 1s\n",
      " 72000K .......... .......... .......... .......... .......... 52%  184M 1s\n",
      " 72050K .......... .......... .......... .......... .......... 52%  208M 1s\n",
      " 72100K .......... .......... .......... .......... .......... 52%  217M 1s\n",
      " 72150K .......... .......... .......... .......... .......... 52%  209M 1s\n",
      " 72200K .......... .......... .......... .......... .......... 52%  191M 1s\n",
      " 72250K .......... .......... .......... .......... .......... 52%  212M 1s\n",
      " 72300K .......... .......... .......... .......... .......... 52%  171M 1s\n",
      " 72350K .......... .......... .......... .......... .......... 52%  198M 1s\n",
      " 72400K .......... .......... .......... .......... .......... 52% 9.38M 1s\n",
      " 72450K .......... .......... .......... .......... .......... 52%  131M 1s\n",
      " 72500K .......... .......... .......... .......... .......... 52%  120M 1s\n",
      " 72550K .......... .......... .......... .......... .......... 52%  129M 1s\n",
      " 72600K .......... .......... .......... .......... .......... 52%  145M 1s\n",
      " 72650K .......... .......... .......... .......... .......... 52%  194M 1s\n",
      " 72700K .......... .......... .......... .......... .......... 52%  214M 1s\n",
      " 72750K .......... .......... .......... .......... .......... 52%  170M 1s\n",
      " 72800K .......... .......... .......... .......... .......... 52%  163M 1s\n",
      " 72850K .......... .......... .......... .......... .......... 52%  178M 1s\n",
      " 72900K .......... .......... .......... .......... .......... 52%  193M 1s\n",
      " 72950K .......... .......... .......... .......... .......... 52%  191M 1s\n",
      " 73000K .......... .......... .......... .......... .......... 52%  190M 1s\n",
      " 73050K .......... .......... .......... .......... .......... 52%  213M 1s\n",
      " 73100K .......... .......... .......... .......... .......... 52%  216M 1s\n",
      " 73150K .......... .......... .......... .......... .......... 52%  214M 1s\n",
      " 73200K .......... .......... .......... .......... .......... 52%  180M 1s\n",
      " 73250K .......... .......... .......... .......... .......... 52%  216M 1s\n",
      " 73300K .......... .......... .......... .......... .......... 53% 7.02M 1s\n",
      " 73350K .......... .......... .......... .......... .......... 53%  143M 1s\n",
      " 73400K .......... .......... .......... .......... .......... 53%  146M 1s\n",
      " 73450K .......... .......... .......... .......... .......... 53%  202M 1s\n",
      " 73500K .......... .......... .......... .......... .......... 53%  187M 1s\n",
      " 73550K .......... .......... .......... .......... .......... 53%  189M 1s\n",
      " 73600K .......... .......... .......... .......... .......... 53%  149M 1s\n",
      " 73650K .......... .......... .......... .......... .......... 53%  177M 1s\n",
      " 73700K .......... .......... .......... .......... .......... 53%  164M 1s\n",
      " 73750K .......... .......... .......... .......... .......... 53%  167M 1s\n",
      " 73800K .......... .......... .......... .......... .......... 53%  171M 1s\n",
      " 73850K .......... .......... .......... .......... .......... 53%  165M 1s\n",
      " 73900K .......... .......... .......... .......... .......... 53%  207M 1s\n",
      " 73950K .......... .......... .......... .......... .......... 53%  173M 1s\n",
      " 74000K .......... .......... .......... .......... .......... 53%  167M 1s\n",
      " 74050K .......... .......... .......... .......... .......... 53%  197M 1s\n",
      " 74100K .......... .......... .......... .......... .......... 53%  188M 1s\n",
      " 74150K .......... .......... .......... .......... .......... 53%  203M 1s\n",
      " 74200K .......... .......... .......... .......... .......... 53%  193M 1s\n",
      " 74250K .......... .......... .......... .......... .......... 53%  216M 1s\n",
      " 74300K .......... .......... .......... .......... .......... 53%  213M 1s\n",
      " 74350K .......... .......... .......... .......... .......... 53%  210M 1s\n",
      " 74400K .......... .......... .......... .......... .......... 53%  182M 1s\n",
      " 74450K .......... .......... .......... .......... .......... 53%  168M 1s\n",
      " 74500K .......... .......... .......... .......... .......... 53%  220M 1s\n",
      " 74550K .......... .......... .......... .......... .......... 53%  209M 1s\n",
      " 74600K .......... .......... .......... .......... .......... 53% 5.94M 1s\n",
      " 74650K .......... .......... .......... .......... .......... 53% 69.6M 1s\n",
      " 74700K .......... .......... .......... .......... .......... 54% 42.6M 1s\n",
      " 74750K .......... .......... .......... .......... .......... 54%  135M 1s\n",
      " 74800K .......... .......... .......... .......... .......... 54%  159M 1s\n",
      " 74850K .......... .......... .......... .......... .......... 54%  212M 1s\n",
      " 74900K .......... .......... .......... .......... .......... 54%  200M 1s\n",
      " 74950K .......... .......... .......... .......... .......... 54%  165M 1s\n",
      " 75000K .......... .......... .......... .......... .......... 54%  160M 1s\n",
      " 75050K .......... .......... .......... .......... .......... 54%  173M 1s\n",
      " 75100K .......... .......... .......... .......... .......... 54%  202M 1s\n",
      " 75150K .......... .......... .......... .......... .......... 54%  205M 1s\n",
      " 75200K .......... .......... .......... .......... .......... 54%  175M 1s\n",
      " 75250K .......... .......... .......... .......... .......... 54%  216M 1s\n",
      " 75300K .......... .......... .......... .......... .......... 54%  213M 1s\n",
      " 75350K .......... .......... .......... .......... .......... 54%  220M 1s\n",
      " 75400K .......... .......... .......... .......... .......... 54%  141M 1s\n",
      " 75450K .......... .......... .......... .......... .......... 54%  216M 1s\n",
      " 75500K .......... .......... .......... .......... .......... 54%  212M 1s\n",
      " 75550K .......... .......... .......... .......... .......... 54%  210M 1s\n",
      " 75600K .......... .......... .......... .......... .......... 54%  178M 1s\n",
      " 75650K .......... .......... .......... .......... .......... 54%  201M 1s\n",
      " 75700K .......... .......... .......... .......... .......... 54%  216M 1s\n",
      " 75750K .......... .......... .......... .......... .......... 54%  219M 1s\n",
      " 75800K .......... .......... .......... .......... .......... 54%  184M 1s\n",
      " 75850K .......... .......... .......... .......... .......... 54%  218M 1s\n",
      " 75900K .......... .......... .......... .......... .......... 54%  213M 1s\n",
      " 75950K .......... .......... .......... .......... .......... 54%  198M 1s\n",
      " 76000K .......... .......... .......... .......... .......... 54%  178M 1s\n",
      " 76050K .......... .......... .......... .......... .......... 55%  207M 1s\n",
      " 76100K .......... .......... .......... .......... .......... 55%  210M 1s\n",
      " 76150K .......... .......... .......... .......... .......... 55% 15.0M 1s\n",
      " 76200K .......... .......... .......... .......... .......... 55%  144M 1s\n",
      " 76250K .......... .......... .......... .......... .......... 55%  166M 1s\n",
      " 76300K .......... .......... .......... .......... .......... 55%  189M 1s\n",
      " 76350K .......... .......... .......... .......... .......... 55%  176M 1s\n",
      " 76400K .......... .......... .......... .......... .......... 55%  157M 1s\n",
      " 76450K .......... .......... .......... .......... .......... 55%  201M 1s\n",
      " 76500K .......... .......... .......... .......... .......... 55%  191M 1s\n",
      " 76550K .......... .......... .......... .......... .......... 55%  181M 1s\n",
      " 76600K .......... .......... .......... .......... .......... 55%  127M 1s\n",
      " 76650K .......... .......... .......... .......... .......... 55%  189M 1s\n",
      " 76700K .......... .......... .......... .......... .......... 55%  184M 1s\n",
      " 76750K .......... .......... .......... .......... .......... 55%  195M 1s\n",
      " 76800K .......... .......... .......... .......... .......... 55%  178M 1s\n",
      " 76850K .......... .......... .......... .......... .......... 55%  171M 1s\n",
      " 76900K .......... .......... .......... .......... .......... 55%  192M 1s\n",
      " 76950K .......... .......... .......... .......... .......... 55%  209M 1s\n",
      " 77000K .......... .......... .......... .......... .......... 55%  191M 1s\n",
      " 77050K .......... .......... .......... .......... .......... 55%  219M 1s\n",
      " 77100K .......... .......... .......... .......... .......... 55%  220M 1s\n",
      " 77150K .......... .......... .......... .......... .......... 55%  203M 1s\n",
      " 77200K .......... .......... .......... .......... .......... 55%  175M 1s\n",
      " 77250K .......... .......... .......... .......... .......... 55%  210M 1s\n",
      " 77300K .......... .......... .......... .......... .......... 55%  210M 1s\n",
      " 77350K .......... .......... .......... .......... .......... 55% 5.39M 1s\n",
      " 77400K .......... .......... .......... .......... .......... 55%  172M 1s\n",
      " 77450K .......... .......... .......... .......... .......... 56%  220M 1s\n",
      " 77500K .......... .......... .......... .......... .......... 56%  174M 1s\n",
      " 77550K .......... .......... .......... .......... .......... 56%  173M 1s\n",
      " 77600K .......... .......... .......... .......... .......... 56%  133M 1s\n",
      " 77650K .......... .......... .......... .......... .......... 56%  191M 1s\n",
      " 77700K .......... .......... .......... .......... .......... 56%  183M 1s\n",
      " 77750K .......... .......... .......... .......... .......... 56%  184M 1s\n",
      " 77800K .......... .......... .......... .......... .......... 56%  173M 1s\n",
      " 77850K .......... .......... .......... .......... .......... 56%  181M 1s\n",
      " 77900K .......... .......... .......... .......... .......... 56%  184M 1s\n",
      " 77950K .......... .......... .......... .......... .......... 56%  184M 1s\n",
      " 78000K .......... .......... .......... .......... .......... 56%  177M 1s\n",
      " 78050K .......... .......... .......... .......... .......... 56%  206M 1s\n",
      " 78100K .......... .......... .......... .......... .......... 56%  218M 1s\n",
      " 78150K .......... .......... .......... .......... .......... 56%  207M 1s\n",
      " 78200K .......... .......... .......... .......... .......... 56%  191M 1s\n",
      " 78250K .......... .......... .......... .......... .......... 56%  198M 1s\n",
      " 78300K .......... .......... .......... .......... .......... 56%  216M 1s\n",
      " 78350K .......... .......... .......... .......... .......... 56%  213M 1s\n",
      " 78400K .......... .......... .......... .......... .......... 56%  177M 1s\n",
      " 78450K .......... .......... .......... .......... .......... 56% 7.58M 1s\n",
      " 78500K .......... .......... .......... .......... .......... 56%  127M 1s\n",
      " 78550K .......... .......... .......... .......... .......... 56% 71.3M 1s\n",
      " 78600K .......... .......... .......... .......... .......... 56% 16.9M 1s\n",
      " 78650K .......... .......... .......... .......... .......... 56%  196M 1s\n",
      " 78700K .......... .......... .......... .......... .......... 56%  211M 1s\n",
      " 78750K .......... .......... .......... .......... .......... 56%  152M 1s\n",
      " 78800K .......... .......... .......... .......... .......... 56%  138M 1s\n",
      " 78850K .......... .......... .......... .......... .......... 57% 85.6M 1s\n",
      " 78900K .......... .......... .......... .......... .......... 57%  144M 1s\n",
      " 78950K .......... .......... .......... .......... .......... 57%  192M 1s\n",
      " 79000K .......... .......... .......... .......... .......... 57%  170M 1s\n",
      " 79050K .......... .......... .......... .......... .......... 57%  215M 1s\n",
      " 79100K .......... .......... .......... .......... .......... 57%  198M 1s\n",
      " 79150K .......... .......... .......... .......... .......... 57%  208M 1s\n",
      " 79200K .......... .......... .......... .......... .......... 57%  174M 1s\n",
      " 79250K .......... .......... .......... .......... .......... 57%  199M 1s\n",
      " 79300K .......... .......... .......... .......... .......... 57%  213M 1s\n",
      " 79350K .......... .......... .......... .......... .......... 57%  208M 1s\n",
      " 79400K .......... .......... .......... .......... .......... 57%  189M 1s\n",
      " 79450K .......... .......... .......... .......... .......... 57%  181M 1s\n",
      " 79500K .......... .......... .......... .......... .......... 57%  220M 1s\n",
      " 79550K .......... .......... .......... .......... .......... 57%  203M 1s\n",
      " 79600K .......... .......... .......... .......... .......... 57%  169M 1s\n",
      " 79650K .......... .......... .......... .......... .......... 57%  210M 1s\n",
      " 79700K .......... .......... .......... .......... .......... 57%  209M 1s\n",
      " 79750K .......... .......... .......... .......... .......... 57%  205M 1s\n",
      " 79800K .......... .......... .......... .......... .......... 57%  181M 1s\n",
      " 79850K .......... .......... .......... .......... .......... 57%  211M 1s\n",
      " 79900K .......... .......... .......... .......... .......... 57%  218M 1s\n",
      " 79950K .......... .......... .......... .......... .......... 57%  205M 1s\n",
      " 80000K .......... .......... .......... .......... .......... 57%  172M 1s\n",
      " 80050K .......... .......... .......... .......... .......... 57% 15.6M 1s\n",
      " 80100K .......... .......... .......... .......... .......... 57%  160M 1s\n",
      " 80150K .......... .......... .......... .......... .......... 57%  147M 1s\n",
      " 80200K .......... .......... .......... .......... .......... 58%  150M 1s\n",
      " 80250K .......... .......... .......... .......... .......... 58%  192M 1s\n",
      " 80300K .......... .......... .......... .......... .......... 58%  211M 1s\n",
      " 80350K .......... .......... .......... .......... .......... 58%  196M 1s\n",
      " 80400K .......... .......... .......... .......... .......... 58%  130M 1s\n",
      " 80450K .......... .......... .......... .......... .......... 58%  169M 1s\n",
      " 80500K .......... .......... .......... .......... .......... 58%  174M 1s\n",
      " 80550K .......... .......... .......... .......... .......... 58%  196M 1s\n",
      " 80600K .......... .......... .......... .......... .......... 58%  155M 1s\n",
      " 80650K .......... .......... .......... .......... .......... 58%  213M 1s\n",
      " 80700K .......... .......... .......... .......... .......... 58%  217M 1s\n",
      " 80750K .......... .......... .......... .......... .......... 58%  207M 1s\n",
      " 80800K .......... .......... .......... .......... .......... 58%  183M 1s\n",
      " 80850K .......... .......... .......... .......... .......... 58%  197M 1s\n",
      " 80900K .......... .......... .......... .......... .......... 58%  204M 1s\n",
      " 80950K .......... .......... .......... .......... .......... 58%  204M 1s\n",
      " 81000K .......... .......... .......... .......... .......... 58%  187M 1s\n",
      " 81050K .......... .......... .......... .......... .......... 58%  214M 1s\n",
      " 81100K .......... .......... .......... .......... .......... 58% 7.17M 1s\n",
      " 81150K .......... .......... .......... .......... .......... 58% 94.5M 1s\n",
      " 81200K .......... .......... .......... .......... .......... 58%  119M 1s\n",
      " 81250K .......... .......... .......... .......... .......... 58%  192M 1s\n",
      " 81300K .......... .......... .......... .......... .......... 58%  195M 1s\n",
      " 81350K .......... .......... .......... .......... .......... 58%  208M 1s\n",
      " 81400K .......... .......... .......... .......... .......... 58%  151M 1s\n",
      " 81450K .......... .......... .......... .......... .......... 58%  206M 1s\n",
      " 81500K .......... .......... .......... .......... .......... 58%  196M 1s\n",
      " 81550K .......... .......... .......... .......... .......... 58%  209M 1s\n",
      " 81600K .......... .......... .......... .......... .......... 59% 4.03M 1s\n",
      " 81650K .......... .......... .......... .......... .......... 59% 86.3M 1s\n",
      " 81700K .......... .......... .......... .......... .......... 59% 19.0M 1s\n",
      " 81750K .......... .......... .......... .......... .......... 59%  171M 1s\n",
      " 81800K .......... .......... .......... .......... .......... 59%  189M 1s\n",
      " 81850K .......... .......... .......... .......... .......... 59%  210M 1s\n",
      " 81900K .......... .......... .......... .......... .......... 59%  208M 1s\n",
      " 81950K .......... .......... .......... .......... .......... 59%  219M 1s\n",
      " 82000K .......... .......... .......... .......... .......... 59%  159M 1s\n",
      " 82050K .......... .......... .......... .......... .......... 59%  208M 1s\n",
      " 82100K .......... .......... .......... .......... .......... 59%  212M 1s\n",
      " 82150K .......... .......... .......... .......... .......... 59%  219M 1s\n",
      " 82200K .......... .......... .......... .......... .......... 59%  185M 1s\n",
      " 82250K .......... .......... .......... .......... .......... 59%  214M 1s\n",
      " 82300K .......... .......... .......... .......... .......... 59%  219M 1s\n",
      " 82350K .......... .......... .......... .......... .......... 59%  212M 1s\n",
      " 82400K .......... .......... .......... .......... .......... 59%  176M 1s\n",
      " 82450K .......... .......... .......... .......... .......... 59%  212M 1s\n",
      " 82500K .......... .......... .......... .......... .......... 59%  197M 1s\n",
      " 82550K .......... .......... .......... .......... .......... 59%  208M 1s\n",
      " 82600K .......... .......... .......... .......... .......... 59%  180M 1s\n",
      " 82650K .......... .......... .......... .......... .......... 59%  213M 1s\n",
      " 82700K .......... .......... .......... .......... .......... 59%  180M 1s\n",
      " 82750K .......... .......... .......... .......... .......... 59%  191M 1s\n",
      " 82800K .......... .......... .......... .......... .......... 59%  162M 1s\n",
      " 82850K .......... .......... .......... .......... .......... 59%  209M 1s\n",
      " 82900K .......... .......... .......... .......... .......... 59%  218M 1s\n",
      " 82950K .......... .......... .......... .......... .......... 59%  217M 1s\n",
      " 83000K .......... .......... .......... .......... .......... 60%  163M 1s\n",
      " 83050K .......... .......... .......... .......... .......... 60%  218M 1s\n",
      " 83100K .......... .......... .......... .......... .......... 60%  208M 1s\n",
      " 83150K .......... .......... .......... .......... .......... 60%  194M 1s\n",
      " 83200K .......... .......... .......... .......... .......... 60%  179M 1s\n",
      " 83250K .......... .......... .......... .......... .......... 60%  216M 1s\n",
      " 83300K .......... .......... .......... .......... .......... 60%  205M 1s\n",
      " 83350K .......... .......... .......... .......... .......... 60%  190M 1s\n",
      " 83400K .......... .......... .......... .......... .......... 60%  194M 1s\n",
      " 83450K .......... .......... .......... .......... .......... 60%  205M 1s\n",
      " 83500K .......... .......... .......... .......... .......... 60%  204M 1s\n",
      " 83550K .......... .......... .......... .......... .......... 60%  175M 1s\n",
      " 83600K .......... .......... .......... .......... .......... 60%  153M 1s\n",
      " 83650K .......... .......... .......... .......... .......... 60%  194M 1s\n",
      " 83700K .......... .......... .......... .......... .......... 60%  214M 1s\n",
      " 83750K .......... .......... .......... .......... .......... 60%  174M 1s\n",
      " 83800K .......... .......... .......... .......... .......... 60%  164M 1s\n",
      " 83850K .......... .......... .......... .......... .......... 60%  193M 1s\n",
      " 83900K .......... .......... .......... .......... .......... 60%  199M 1s\n",
      " 83950K .......... .......... .......... .......... .......... 60%  218M 1s\n",
      " 84000K .......... .......... .......... .......... .......... 60%  170M 1s\n",
      " 84050K .......... .......... .......... .......... .......... 60%  154M 1s\n",
      " 84100K .......... .......... .......... .......... .......... 60%  152M 1s\n",
      " 84150K .......... .......... .......... .......... .......... 60%  211M 1s\n",
      " 84200K .......... .......... .......... .......... .......... 60%  185M 1s\n",
      " 84250K .......... .......... .......... .......... .......... 60%  122M 1s\n",
      " 84300K .......... .......... .......... .......... .......... 60% 68.8M 1s\n",
      " 84350K .......... .......... .......... .......... .......... 61% 76.9M 1s\n",
      " 84400K .......... .......... .......... .......... .......... 61%  126M 1s\n",
      " 84450K .......... .......... .......... .......... .......... 61%  208M 1s\n",
      " 84500K .......... .......... .......... .......... .......... 61%  207M 1s\n",
      " 84550K .......... .......... .......... .......... .......... 61%  186M 1s\n",
      " 84600K .......... .......... .......... .......... .......... 61%  179M 1s\n",
      " 84650K .......... .......... .......... .......... .......... 61%  189M 1s\n",
      " 84700K .......... .......... .......... .......... .......... 61% 6.03M 1s\n",
      " 84750K .......... .......... .......... .......... .......... 61%  166M 1s\n",
      " 84800K .......... .......... .......... .......... .......... 61% 95.3M 1s\n",
      " 84850K .......... .......... .......... .......... .......... 61% 55.9M 1s\n",
      " 84900K .......... .......... .......... .......... .......... 61%  153M 1s\n",
      " 84950K .......... .......... .......... .......... .......... 61%  162M 1s\n",
      " 85000K .......... .......... .......... .......... .......... 61%  177M 1s\n",
      " 85050K .......... .......... .......... .......... .......... 61%  197M 1s\n",
      " 85100K .......... .......... .......... .......... .......... 61%  191M 1s\n",
      " 85150K .......... .......... .......... .......... .......... 61%  213M 1s\n",
      " 85200K .......... .......... .......... .......... .......... 61%  179M 1s\n",
      " 85250K .......... .......... .......... .......... .......... 61%  191M 1s\n",
      " 85300K .......... .......... .......... .......... .......... 61%  199M 1s\n",
      " 85350K .......... .......... .......... .......... .......... 61% 5.91M 1s\n",
      " 85400K .......... .......... .......... .......... .......... 61% 18.9M 1s\n",
      " 85450K .......... .......... .......... .......... .......... 61%  176M 1s\n",
      " 85500K .......... .......... .......... .......... .......... 61%  172M 1s\n",
      " 85550K .......... .......... .......... .......... .......... 61%  152M 1s\n",
      " 85600K .......... .......... .......... .......... .......... 61%  187M 1s\n",
      " 85650K .......... .......... .......... .......... .......... 61%  208M 1s\n",
      " 85700K .......... .......... .......... .......... .......... 61%  220M 1s\n",
      " 85750K .......... .......... .......... .......... .......... 62%  221M 1s\n",
      " 85800K .......... .......... .......... .......... .......... 62%  187M 1s\n",
      " 85850K .......... .......... .......... .......... .......... 62%  220M 1s\n",
      " 85900K .......... .......... .......... .......... .......... 62%  199M 1s\n",
      " 85950K .......... .......... .......... .......... .......... 62%  180M 1s\n",
      " 86000K .......... .......... .......... .......... .......... 62%  170M 1s\n",
      " 86050K .......... .......... .......... .......... .......... 62%  200M 1s\n",
      " 86100K .......... .......... .......... .......... .......... 62%  217M 1s\n",
      " 86150K .......... .......... .......... .......... .......... 62%  220M 1s\n",
      " 86200K .......... .......... .......... .......... .......... 62%  158M 1s\n",
      " 86250K .......... .......... .......... .......... .......... 62%  123M 1s\n",
      " 86300K .......... .......... .......... .......... .......... 62%  165M 1s\n",
      " 86350K .......... .......... .......... .......... .......... 62%  208M 1s\n",
      " 86400K .......... .......... .......... .......... .......... 62%  173M 1s\n",
      " 86450K .......... .......... .......... .......... .......... 62%  213M 1s\n",
      " 86500K .......... .......... .......... .......... .......... 62%  214M 1s\n",
      " 86550K .......... .......... .......... .......... .......... 62%  209M 1s\n",
      " 86600K .......... .......... .......... .......... .......... 62%  187M 1s\n",
      " 86650K .......... .......... .......... .......... .......... 62%  227M 1s\n",
      " 86700K .......... .......... .......... .......... .......... 62%  208M 1s\n",
      " 86750K .......... .......... .......... .......... .......... 62%  221M 1s\n",
      " 86800K .......... .......... .......... .......... .......... 62%  176M 1s\n",
      " 86850K .......... .......... .......... .......... .......... 62%  183M 1s\n",
      " 86900K .......... .......... .......... .......... .......... 62%  179M 1s\n",
      " 86950K .......... .......... .......... .......... .......... 62%  197M 1s\n",
      " 87000K .......... .......... .......... .......... .......... 62%  172M 1s\n",
      " 87050K .......... .......... .......... .......... .......... 62%  169M 1s\n",
      " 87100K .......... .......... .......... .......... .......... 62%  219M 1s\n",
      " 87150K .......... .......... .......... .......... .......... 63%  222M 1s\n",
      " 87200K .......... .......... .......... .......... .......... 63%  177M 1s\n",
      " 87250K .......... .......... .......... .......... .......... 63%  225M 1s\n",
      " 87300K .......... .......... .......... .......... .......... 63%  219M 1s\n",
      " 87350K .......... .......... .......... .......... .......... 63%  207M 1s\n",
      " 87400K .......... .......... .......... .......... .......... 63%  195M 1s\n",
      " 87450K .......... .......... .......... .......... .......... 63%  214M 1s\n",
      " 87500K .......... .......... .......... .......... .......... 63%  207M 1s\n",
      " 87550K .......... .......... .......... .......... .......... 63%  184M 1s\n",
      " 87600K .......... .......... .......... .......... .......... 63%  182M 1s\n",
      " 87650K .......... .......... .......... .......... .......... 63%  205M 1s\n",
      " 87700K .......... .......... .......... .......... .......... 63%  218M 1s\n",
      " 87750K .......... .......... .......... .......... .......... 63% 93.5M 1s\n",
      " 87800K .......... .......... .......... .......... .......... 63% 69.9M 1s\n",
      " 87850K .......... .......... .......... .......... .......... 63% 23.9M 1s\n",
      " 87900K .......... .......... .......... .......... .......... 63%  129M 1s\n",
      " 87950K .......... .......... .......... .......... .......... 63%  167M 1s\n",
      " 88000K .......... .......... .......... .......... .......... 63%  160M 1s\n",
      " 88050K .......... .......... .......... .......... .......... 63%  197M 1s\n",
      " 88100K .......... .......... .......... .......... .......... 63%  210M 1s\n",
      " 88150K .......... .......... .......... .......... .......... 63%  217M 1s\n",
      " 88200K .......... .......... .......... .......... .......... 63%  170M 1s\n",
      " 88250K .......... .......... .......... .......... .......... 63% 6.67M 1s\n",
      " 88300K .......... .......... .......... .......... .......... 63%  170M 1s\n",
      " 88350K .......... .......... .......... .......... .......... 63%  171M 1s\n",
      " 88400K .......... .......... .......... .......... .......... 63%  166M 1s\n",
      " 88450K .......... .......... .......... .......... .......... 63%  170M 1s\n",
      " 88500K .......... .......... .......... .......... .......... 64%  213M 1s\n",
      " 88550K .......... .......... .......... .......... .......... 64%  111M 1s\n",
      " 88600K .......... .......... .......... .......... .......... 64% 67.8M 1s\n",
      " 88650K .......... .......... .......... .......... .......... 64%  198M 1s\n",
      " 88700K .......... .......... .......... .......... .......... 64%  207M 1s\n",
      " 88750K .......... .......... .......... .......... .......... 64%  174M 1s\n",
      " 88800K .......... .......... .......... .......... .......... 64%  108M 1s\n",
      " 88850K .......... .......... .......... .......... .......... 64%  167M 1s\n",
      " 88900K .......... .......... .......... .......... .......... 64%  198M 1s\n",
      " 88950K .......... .......... .......... .......... .......... 64% 77.0M 1s\n",
      " 89000K .......... .......... .......... .......... .......... 64%  149M 1s\n",
      " 89050K .......... .......... .......... .......... .......... 64%  178M 1s\n",
      " 89100K .......... .......... .......... .......... .......... 64%  209M 1s\n",
      " 89150K .......... .......... .......... .......... .......... 64%  212M 1s\n",
      " 89200K .......... .......... .......... .......... .......... 64% 59.3M 1s\n",
      " 89250K .......... .......... .......... .......... .......... 64% 73.6M 1s\n",
      " 89300K .......... .......... .......... .......... .......... 64%  208M 1s\n",
      " 89350K .......... .......... .......... .......... .......... 64%  215M 1s\n",
      " 89400K .......... .......... .......... .......... .......... 64%  180M 1s\n",
      " 89450K .......... .......... .......... .......... .......... 64%  210M 1s\n",
      " 89500K .......... .......... .......... .......... .......... 64%  192M 1s\n",
      " 89550K .......... .......... .......... .......... .......... 64%  197M 1s\n",
      " 89600K .......... .......... .......... .......... .......... 64%  150M 1s\n",
      " 89650K .......... .......... .......... .......... .......... 64%  220M 1s\n",
      " 89700K .......... .......... .......... .......... .......... 64% 9.09M 1s\n",
      " 89750K .......... .......... .......... .......... .......... 64% 24.2M 1s\n",
      " 89800K .......... .......... .......... .......... .......... 64%  182M 1s\n",
      " 89850K .......... .......... .......... .......... .......... 64%  211M 1s\n",
      " 89900K .......... .......... .......... .......... .......... 65%  200M 1s\n",
      " 89950K .......... .......... .......... .......... .......... 65%  200M 1s\n",
      " 90000K .......... .......... .......... .......... .......... 65%  162M 1s\n",
      " 90050K .......... .......... .......... .......... .......... 65%  194M 1s\n",
      " 90100K .......... .......... .......... .......... .......... 65%  198M 1s\n",
      " 90150K .......... .......... .......... .......... .......... 65%  167M 1s\n",
      " 90200K .......... .......... .......... .......... .......... 65%  137M 1s\n",
      " 90250K .......... .......... .......... .......... .......... 65%  165M 1s\n",
      " 90300K .......... .......... .......... .......... .......... 65%  160M 1s\n",
      " 90350K .......... .......... .......... .......... .......... 65%  160M 1s\n",
      " 90400K .......... .......... .......... .......... .......... 65%  131M 1s\n",
      " 90450K .......... .......... .......... .......... .......... 65%  214M 1s\n",
      " 90500K .......... .......... .......... .......... .......... 65%  202M 1s\n",
      " 90550K .......... .......... .......... .......... .......... 65%  206M 1s\n",
      " 90600K .......... .......... .......... .......... .......... 65%  189M 1s\n",
      " 90650K .......... .......... .......... .......... .......... 65%  210M 1s\n",
      " 90700K .......... .......... .......... .......... .......... 65%  207M 1s\n",
      " 90750K .......... .......... .......... .......... .......... 65%  207M 1s\n",
      " 90800K .......... .......... .......... .......... .......... 65%  171M 1s\n",
      " 90850K .......... .......... .......... .......... .......... 65%  216M 1s\n",
      " 90900K .......... .......... .......... .......... .......... 65%  213M 1s\n",
      " 90950K .......... .......... .......... .......... .......... 65%  217M 1s\n",
      " 91000K .......... .......... .......... .......... .......... 65%  192M 1s\n",
      " 91050K .......... .......... .......... .......... .......... 65%  182M 1s\n",
      " 91100K .......... .......... .......... .......... .......... 65%  217M 1s\n",
      " 91150K .......... .......... .......... .......... .......... 65% 16.3M 1s\n",
      " 91200K .......... .......... .......... .......... .......... 65%  149M 1s\n",
      " 91250K .......... .......... .......... .......... .......... 65%  162M 1s\n",
      " 91300K .......... .......... .......... .......... .......... 66%  211M 1s\n",
      " 91350K .......... .......... .......... .......... .......... 66% 24.6M 1s\n",
      " 91400K .......... .......... .......... .......... .......... 66%  114M 1s\n",
      " 91450K .......... .......... .......... .......... .......... 66%  176M 1s\n",
      " 91500K .......... .......... .......... .......... .......... 66%  189M 1s\n",
      " 91550K .......... .......... .......... .......... .......... 66%  199M 1s\n",
      " 91600K .......... .......... .......... .......... .......... 66%  165M 1s\n",
      " 91650K .......... .......... .......... .......... .......... 66%  194M 1s\n",
      " 91700K .......... .......... .......... .......... .......... 66%  197M 1s\n",
      " 91750K .......... .......... .......... .......... .......... 66%  187M 1s\n",
      " 91800K .......... .......... .......... .......... .......... 66%  177M 1s\n",
      " 91850K .......... .......... .......... .......... .......... 66%  204M 1s\n",
      " 91900K .......... .......... .......... .......... .......... 66%  219M 1s\n",
      " 91950K .......... .......... .......... .......... .......... 66%  209M 1s\n",
      " 92000K .......... .......... .......... .......... .......... 66%  177M 1s\n",
      " 92050K .......... .......... .......... .......... .......... 66%  187M 1s\n",
      " 92100K .......... .......... .......... .......... .......... 66%  178M 1s\n",
      " 92150K .......... .......... .......... .......... .......... 66%  192M 1s\n",
      " 92200K .......... .......... .......... .......... .......... 66%  171M 1s\n",
      " 92250K .......... .......... .......... .......... .......... 66%  180M 1s\n",
      " 92300K .......... .......... .......... .......... .......... 66%  213M 1s\n",
      " 92350K .......... .......... .......... .......... .......... 66%  209M 1s\n",
      " 92400K .......... .......... .......... .......... .......... 66%  173M 1s\n",
      " 92450K .......... .......... .......... .......... .......... 66% 7.17M 1s\n",
      " 92500K .......... .......... .......... .......... .......... 66%  174M 1s\n",
      " 92550K .......... .......... .......... .......... .......... 66%  213M 1s\n",
      " 92600K .......... .......... .......... .......... .......... 66%  169M 1s\n",
      " 92650K .......... .......... .......... .......... .......... 67%  201M 1s\n",
      " 92700K .......... .......... .......... .......... .......... 67%  213M 1s\n",
      " 92750K .......... .......... .......... .......... .......... 67%  179M 1s\n",
      " 92800K .......... .......... .......... .......... .......... 67%  166M 1s\n",
      " 92850K .......... .......... .......... .......... .......... 67%  216M 1s\n",
      " 92900K .......... .......... .......... .......... .......... 67%  143M 1s\n",
      " 92950K .......... .......... .......... .......... .......... 67%  187M 1s\n",
      " 93000K .......... .......... .......... .......... .......... 67%  167M 1s\n",
      " 93050K .......... .......... .......... .......... .......... 67%  159M 1s\n",
      " 93100K .......... .......... .......... .......... .......... 67%  166M 1s\n",
      " 93150K .......... .......... .......... .......... .......... 67%  182M 1s\n",
      " 93200K .......... .......... .......... .......... .......... 67%  176M 1s\n",
      " 93250K .......... .......... .......... .......... .......... 67%  210M 1s\n",
      " 93300K .......... .......... .......... .......... .......... 67%  216M 1s\n",
      " 93350K .......... .......... .......... .......... .......... 67%  213M 1s\n",
      " 93400K .......... .......... .......... .......... .......... 67% 7.07M 1s\n",
      " 93450K .......... .......... .......... .......... .......... 67% 67.3M 1s\n",
      " 93500K .......... .......... .......... .......... .......... 67% 45.1M 1s\n",
      " 93550K .......... .......... .......... .......... .......... 67%  208M 1s\n",
      " 93600K .......... .......... .......... .......... .......... 67%  175M 1s\n",
      " 93650K .......... .......... .......... .......... .......... 67%  142M 1s\n",
      " 93700K .......... .......... .......... .......... .......... 67%  195M 1s\n",
      " 93750K .......... .......... .......... .......... .......... 67%  193M 1s\n",
      " 93800K .......... .......... .......... .......... .......... 67%  181M 1s\n",
      " 93850K .......... .......... .......... .......... .......... 67%  204M 1s\n",
      " 93900K .......... .......... .......... .......... .......... 67%  172M 1s\n",
      " 93950K .......... .......... .......... .......... .......... 67%  191M 1s\n",
      " 94000K .......... .......... .......... .......... .......... 67%  163M 1s\n",
      " 94050K .......... .......... .......... .......... .......... 68%  189M 1s\n",
      " 94100K .......... .......... .......... .......... .......... 68%  204M 1s\n",
      " 94150K .......... .......... .......... .......... .......... 68%  197M 1s\n",
      " 94200K .......... .......... .......... .......... .......... 68%  162M 1s\n",
      " 94250K .......... .......... .......... .......... .......... 68%  199M 1s\n",
      " 94300K .......... .......... .......... .......... .......... 68%  193M 1s\n",
      " 94350K .......... .......... .......... .......... .......... 68%  214M 1s\n",
      " 94400K .......... .......... .......... .......... .......... 68%  151M 1s\n",
      " 94450K .......... .......... .......... .......... .......... 68%  171M 1s\n",
      " 94500K .......... .......... .......... .......... .......... 68%  181M 1s\n",
      " 94550K .......... .......... .......... .......... .......... 68%  215M 1s\n",
      " 94600K .......... .......... .......... .......... .......... 68%  189M 1s\n",
      " 94650K .......... .......... .......... .......... .......... 68%  211M 1s\n",
      " 94700K .......... .......... .......... .......... .......... 68%  212M 1s\n",
      " 94750K .......... .......... .......... .......... .......... 68%  215M 1s\n",
      " 94800K .......... .......... .......... .......... .......... 68%  180M 1s\n",
      " 94850K .......... .......... .......... .......... .......... 68%  219M 1s\n",
      " 94900K .......... .......... .......... .......... .......... 68%  166M 1s\n",
      " 94950K .......... .......... .......... .......... .......... 68%  210M 1s\n",
      " 95000K .......... .......... .......... .......... .......... 68%  187M 1s\n",
      " 95050K .......... .......... .......... .......... .......... 68%  185M 1s\n",
      " 95100K .......... .......... .......... .......... .......... 68%  205M 1s\n",
      " 95150K .......... .......... .......... .......... .......... 68% 9.18M 1s\n",
      " 95200K .......... .......... .......... .......... .......... 68%  135M 1s\n",
      " 95250K .......... .......... .......... .......... .......... 68%  113M 1s\n",
      " 95300K .......... .......... .......... .......... .......... 68%  227M 1s\n",
      " 95350K .......... .......... .......... .......... .......... 68%  181M 1s\n",
      " 95400K .......... .......... .......... .......... .......... 68%  174M 1s\n",
      " 95450K .......... .......... .......... .......... .......... 69%  183M 1s\n",
      " 95500K .......... .......... .......... .......... .......... 69%  177M 1s\n",
      " 95550K .......... .......... .......... .......... .......... 69%  182M 1s\n",
      " 95600K .......... .......... .......... .......... .......... 69%  156M 1s\n",
      " 95650K .......... .......... .......... .......... .......... 69%  189M 1s\n",
      " 95700K .......... .......... .......... .......... .......... 69%  169M 1s\n",
      " 95750K .......... .......... .......... .......... .......... 69%  189M 1s\n",
      " 95800K .......... .......... .......... .......... .......... 69%  187M 1s\n",
      " 95850K .......... .......... .......... .......... .......... 69%  215M 1s\n",
      " 95900K .......... .......... .......... .......... .......... 69%  209M 1s\n",
      " 95950K .......... .......... .......... .......... .......... 69%  173M 1s\n",
      " 96000K .......... .......... .......... .......... .......... 69%  151M 1s\n",
      " 96050K .......... .......... .......... .......... .......... 69%  212M 1s\n",
      " 96100K .......... .......... .......... .......... .......... 69%  215M 1s\n",
      " 96150K .......... .......... .......... .......... .......... 69%  210M 1s\n",
      " 96200K .......... .......... .......... .......... .......... 69%  184M 1s\n",
      " 96250K .......... .......... .......... .......... .......... 69% 8.37M 1s\n",
      " 96300K .......... .......... .......... .......... .......... 69%  120M 1s\n",
      " 96350K .......... .......... .......... .......... .......... 69%  166M 1s\n",
      " 96400K .......... .......... .......... .......... .......... 69%  182M 1s\n",
      " 96450K .......... .......... .......... .......... .......... 69%  207M 1s\n",
      " 96500K .......... .......... .......... .......... .......... 69%  171M 1s\n",
      " 96550K .......... .......... .......... .......... .......... 69%  196M 1s\n",
      " 96600K .......... .......... .......... .......... .......... 69%  186M 1s\n",
      " 96650K .......... .......... .......... .......... .......... 69%  200M 1s\n",
      " 96700K .......... .......... .......... .......... .......... 69%  135M 1s\n",
      " 96750K .......... .......... .......... .......... .......... 69% 72.7M 1s\n",
      " 96800K .......... .......... .......... .......... .......... 70%  150M 1s\n",
      " 96850K .......... .......... .......... .......... .......... 70%  189M 1s\n",
      " 96900K .......... .......... .......... .......... .......... 70%  199M 1s\n",
      " 96950K .......... .......... .......... .......... .......... 70%  180M 1s\n",
      " 97000K .......... .......... .......... .......... .......... 70%  147M 1s\n",
      " 97050K .......... .......... .......... .......... .......... 70%  222M 1s\n",
      " 97100K .......... .......... .......... .......... .......... 70%  173M 1s\n",
      " 97150K .......... .......... .......... .......... .......... 70%  180M 1s\n",
      " 97200K .......... .......... .......... .......... .......... 70%  164M 1s\n",
      " 97250K .......... .......... .......... .......... .......... 70%  208M 1s\n",
      " 97300K .......... .......... .......... .......... .......... 70%  214M 1s\n",
      " 97350K .......... .......... .......... .......... .......... 70%  177M 1s\n",
      " 97400K .......... .......... .......... .......... .......... 70%  173M 1s\n",
      " 97450K .......... .......... .......... .......... .......... 70%  207M 1s\n",
      " 97500K .......... .......... .......... .......... .......... 70%  214M 1s\n",
      " 97550K .......... .......... .......... .......... .......... 70%  227M 1s\n",
      " 97600K .......... .......... .......... .......... .......... 70%  170M 1s\n",
      " 97650K .......... .......... .......... .......... .......... 70%  221M 1s\n",
      " 97700K .......... .......... .......... .......... .......... 70% 6.87M 1s\n",
      " 97750K .......... .......... .......... .......... .......... 70%  179M 1s\n",
      " 97800K .......... .......... .......... .......... .......... 70% 89.0M 1s\n",
      " 97850K .......... .......... .......... .......... .......... 70%  168M 1s\n",
      " 97900K .......... .......... .......... .......... .......... 70%  170M 1s\n",
      " 97950K .......... .......... .......... .......... .......... 70%  181M 1s\n",
      " 98000K .......... .......... .......... .......... .......... 70%  172M 1s\n",
      " 98050K .......... .......... .......... .......... .......... 70%  164M 1s\n",
      " 98100K .......... .......... .......... .......... .......... 70%  193M 1s\n",
      " 98150K .......... .......... .......... .......... .......... 70%  208M 1s\n",
      " 98200K .......... .......... .......... .......... .......... 71%  167M 1s\n",
      " 98250K .......... .......... .......... .......... .......... 71%  212M 1s\n",
      " 98300K .......... .......... .......... .......... .......... 71%  184M 1s\n",
      " 98350K .......... .......... .......... .......... .......... 71%  204M 1s\n",
      " 98400K .......... .......... .......... .......... .......... 71%  178M 1s\n",
      " 98450K .......... .......... .......... .......... .......... 71%  209M 1s\n",
      " 98500K .......... .......... .......... .......... .......... 71%  202M 1s\n",
      " 98550K .......... .......... .......... .......... .......... 71%  196M 1s\n",
      " 98600K .......... .......... .......... .......... .......... 71%  176M 1s\n",
      " 98650K .......... .......... .......... .......... .......... 71%  212M 1s\n",
      " 98700K .......... .......... .......... .......... .......... 71%  211M 1s\n",
      " 98750K .......... .......... .......... .......... .......... 71%  213M 1s\n",
      " 98800K .......... .......... .......... .......... .......... 71%  156M 1s\n",
      " 98850K .......... .......... .......... .......... .......... 71%  207M 1s\n",
      " 98900K .......... .......... .......... .......... .......... 71%  219M 1s\n",
      " 98950K .......... .......... .......... .......... .......... 71%  208M 1s\n",
      " 99000K .......... .......... .......... .......... .......... 71% 8.37M 1s\n",
      " 99050K .......... .......... .......... .......... .......... 71%  189M 1s\n",
      " 99100K .......... .......... .......... .......... .......... 71%  170M 1s\n",
      " 99150K .......... .......... .......... .......... .......... 71%  178M 1s\n",
      " 99200K .......... .......... .......... .......... .......... 71%  177M 1s\n",
      " 99250K .......... .......... .......... .......... .......... 71%  217M 0s\n",
      " 99300K .......... .......... .......... .......... .......... 71%  210M 0s\n",
      " 99350K .......... .......... .......... .......... .......... 71%  160M 0s\n",
      " 99400K .......... .......... .......... .......... .......... 71%  151M 0s\n",
      " 99450K .......... .......... .......... .......... .......... 71%  175M 0s\n",
      " 99500K .......... .......... .......... .......... .......... 71%  215M 0s\n",
      " 99550K .......... .......... .......... .......... .......... 71%  196M 0s\n",
      " 99600K .......... .......... .......... .......... .......... 72%  152M 0s\n",
      " 99650K .......... .......... .......... .......... .......... 72%  174M 0s\n",
      " 99700K .......... .......... .......... .......... .......... 72%  205M 0s\n",
      " 99750K .......... .......... .......... .......... .......... 72%  169M 0s\n",
      " 99800K .......... .......... .......... .......... .......... 72%  144M 0s\n",
      " 99850K .......... .......... .......... .......... .......... 72%  180M 0s\n",
      " 99900K .......... .......... .......... .......... .......... 72%  201M 0s\n",
      " 99950K .......... .......... .......... .......... .......... 72%  197M 0s\n",
      "100000K .......... .......... .......... .......... .......... 72%  169M 0s\n",
      "100050K .......... .......... .......... .......... .......... 72%  216M 0s\n",
      "100100K .......... .......... .......... .......... .......... 72%  155M 0s\n",
      "100150K .......... .......... .......... .......... .......... 72%  152M 0s\n",
      "100200K .......... .......... .......... .......... .......... 72%  182M 0s\n",
      "100250K .......... .......... .......... .......... .......... 72%  216M 0s\n",
      "100300K .......... .......... .......... .......... .......... 72%  222M 0s\n",
      "100350K .......... .......... .......... .......... .......... 72%  211M 0s\n",
      "100400K .......... .......... .......... .......... .......... 72%  179M 0s\n",
      "100450K .......... .......... .......... .......... .......... 72%  205M 0s\n",
      "100500K .......... .......... .......... .......... .......... 72%  204M 0s\n",
      "100550K .......... .......... .......... .......... .......... 72%  228M 0s\n",
      "100600K .......... .......... .......... .......... .......... 72% 7.52M 0s\n",
      "100650K .......... .......... .......... .......... .......... 72%  163M 0s\n",
      "100700K .......... .......... .......... .......... .......... 72%  164M 0s\n",
      "100750K .......... .......... .......... .......... .......... 72%  178M 0s\n",
      "100800K .......... .......... .......... .......... .......... 72%  132M 0s\n",
      "100850K .......... .......... .......... .......... .......... 72%  183M 0s\n",
      "100900K .......... .......... .......... .......... .......... 72%  199M 0s\n",
      "100950K .......... .......... .......... .......... .......... 73%  203M 0s\n",
      "101000K .......... .......... .......... .......... .......... 73%  182M 0s\n",
      "101050K .......... .......... .......... .......... .......... 73%  193M 0s\n",
      "101100K .......... .......... .......... .......... .......... 73%  169M 0s\n",
      "101150K .......... .......... .......... .......... .......... 73%  186M 0s\n",
      "101200K .......... .......... .......... .......... .......... 73%  160M 0s\n",
      "101250K .......... .......... .......... .......... .......... 73%  208M 0s\n",
      "101300K .......... .......... .......... .......... .......... 73%  195M 0s\n",
      "101350K .......... .......... .......... .......... .......... 73%  186M 0s\n",
      "101400K .......... .......... .......... .......... .......... 73%  175M 0s\n",
      "101450K .......... .......... .......... .......... .......... 73%  207M 0s\n",
      "101500K .......... .......... .......... .......... .......... 73%  214M 0s\n",
      "101550K .......... .......... .......... .......... .......... 73%  212M 0s\n",
      "101600K .......... .......... .......... .......... .......... 73%  171M 0s\n",
      "101650K .......... .......... .......... .......... .......... 73%  183M 0s\n",
      "101700K .......... .......... .......... .......... .......... 73% 8.02M 0s\n",
      "101750K .......... .......... .......... .......... .......... 73%  157M 0s\n",
      "101800K .......... .......... .......... .......... .......... 73%  132M 0s\n",
      "101850K .......... .......... .......... .......... .......... 73%  133M 0s\n",
      "101900K .......... .......... .......... .......... .......... 73%  180M 0s\n",
      "101950K .......... .......... .......... .......... .......... 73%  166M 0s\n",
      "102000K .......... .......... .......... .......... .......... 73%  183M 0s\n",
      "102050K .......... .......... .......... .......... .......... 73%  164M 0s\n",
      "102100K .......... .......... .......... .......... .......... 73%  181M 0s\n",
      "102150K .......... .......... .......... .......... .......... 73%  175M 0s\n",
      "102200K .......... .......... .......... .......... .......... 73%  166M 0s\n",
      "102250K .......... .......... .......... .......... .......... 73%  215M 0s\n",
      "102300K .......... .......... .......... .......... .......... 73%  191M 0s\n",
      "102350K .......... .......... .......... .......... .......... 74%  222M 0s\n",
      "102400K .......... .......... .......... .......... .......... 74%  168M 0s\n",
      "102450K .......... .......... .......... .......... .......... 74%  202M 0s\n",
      "102500K .......... .......... .......... .......... .......... 74%  215M 0s\n",
      "102550K .......... .......... .......... .......... .......... 74%  152M 0s\n",
      "102600K .......... .......... .......... .......... .......... 74%  175M 0s\n",
      "102650K .......... .......... .......... .......... .......... 74%  218M 0s\n",
      "102700K .......... .......... .......... .......... .......... 74%  203M 0s\n",
      "102750K .......... .......... .......... .......... .......... 74%  204M 0s\n",
      "102800K .......... .......... .......... .......... .......... 74%  178M 0s\n",
      "102850K .......... .......... .......... .......... .......... 74%  188M 0s\n",
      "102900K .......... .......... .......... .......... .......... 74%  184M 0s\n",
      "102950K .......... .......... .......... .......... .......... 74%  179M 0s\n",
      "103000K .......... .......... .......... .......... .......... 74%  176M 0s\n",
      "103050K .......... .......... .......... .......... .......... 74%  211M 0s\n",
      "103100K .......... .......... .......... .......... .......... 74%  217M 0s\n",
      "103150K .......... .......... .......... .......... .......... 74%  213M 0s\n",
      "103200K .......... .......... .......... .......... .......... 74%  173M 0s\n",
      "103250K .......... .......... .......... .......... .......... 74%  213M 0s\n",
      "103300K .......... .......... .......... .......... .......... 74%  210M 0s\n",
      "103350K .......... .......... .......... .......... .......... 74%  199M 0s\n",
      "103400K .......... .......... .......... .......... .......... 74% 7.65M 0s\n",
      "103450K .......... .......... .......... .......... .......... 74%  107M 0s\n",
      "103500K .......... .......... .......... .......... .......... 74%  205M 0s\n",
      "103550K .......... .......... .......... .......... .......... 74%  186M 0s\n",
      "103600K .......... .......... .......... .......... .......... 74%  179M 0s\n",
      "103650K .......... .......... .......... .......... .......... 74%  172M 0s\n",
      "103700K .......... .......... .......... .......... .......... 74%  154M 0s\n",
      "103750K .......... .......... .......... .......... .......... 75% 98.5M 0s\n",
      "103800K .......... .......... .......... .......... .......... 75%  174M 0s\n",
      "103850K .......... .......... .......... .......... .......... 75%  190M 0s\n",
      "103900K .......... .......... .......... .......... .......... 75%  186M 0s\n",
      "103950K .......... .......... .......... .......... .......... 75%  127M 0s\n",
      "104000K .......... .......... .......... .......... .......... 75%  169M 0s\n",
      "104050K .......... .......... .......... .......... .......... 75%  168M 0s\n",
      "104100K .......... .......... .......... .......... .......... 75%  148M 0s\n",
      "104150K .......... .......... .......... .......... .......... 75%  211M 0s\n",
      "104200K .......... .......... .......... .......... .......... 75%  186M 0s\n",
      "104250K .......... .......... .......... .......... .......... 75%  220M 0s\n",
      "104300K .......... .......... .......... .......... .......... 75%  213M 0s\n",
      "104350K .......... .......... .......... .......... .......... 75%  213M 0s\n",
      "104400K .......... .......... .......... .......... .......... 75%  171M 0s\n",
      "104450K .......... .......... .......... .......... .......... 75%  211M 0s\n",
      "104500K .......... .......... .......... .......... .......... 75%  201M 0s\n",
      "104550K .......... .......... .......... .......... .......... 75%  209M 0s\n",
      "104600K .......... .......... .......... .......... .......... 75% 8.62M 0s\n",
      "104650K .......... .......... .......... .......... .......... 75%  116M 0s\n",
      "104700K .......... .......... .......... .......... .......... 75%  129M 0s\n",
      "104750K .......... .......... .......... .......... .......... 75%  178M 0s\n",
      "104800K .......... .......... .......... .......... .......... 75%  129M 0s\n",
      "104850K .......... .......... .......... .......... .......... 75%  170M 0s\n",
      "104900K .......... .......... .......... .......... .......... 75%  123M 0s\n",
      "104950K .......... .......... .......... .......... .......... 75%  119M 0s\n",
      "105000K .......... .......... .......... .......... .......... 75%  162M 0s\n",
      "105050K .......... .......... .......... .......... .......... 75%  135M 0s\n",
      "105100K .......... .......... .......... .......... .......... 76%  128M 0s\n",
      "105150K .......... .......... .......... .......... .......... 76%  188M 0s\n",
      "105200K .......... .......... .......... .......... .......... 76%  131M 0s\n",
      "105250K .......... .......... .......... .......... .......... 76%  184M 0s\n",
      "105300K .......... .......... .......... .......... .......... 76%  143M 0s\n",
      "105350K .......... .......... .......... .......... .......... 76%  181M 0s\n",
      "105400K .......... .......... .......... .......... .......... 76%  169M 0s\n",
      "105450K .......... .......... .......... .......... .......... 76%  197M 0s\n",
      "105500K .......... .......... .......... .......... .......... 76%  175M 0s\n",
      "105550K .......... .......... .......... .......... .......... 76%  113M 0s\n",
      "105600K .......... .......... .......... .......... .......... 76%  151M 0s\n",
      "105650K .......... .......... .......... .......... .......... 76%  186M 0s\n",
      "105700K .......... .......... .......... .......... .......... 76%  196M 0s\n",
      "105750K .......... .......... .......... .......... .......... 76%  199M 0s\n",
      "105800K .......... .......... .......... .......... .......... 76%  169M 0s\n",
      "105850K .......... .......... .......... .......... .......... 76%  172M 0s\n",
      "105900K .......... .......... .......... .......... .......... 76%  187M 0s\n",
      "105950K .......... .......... .......... .......... .......... 76%  192M 0s\n",
      "106000K .......... .......... .......... .......... .......... 76%  161M 0s\n",
      "106050K .......... .......... .......... .......... .......... 76%  187M 0s\n",
      "106100K .......... .......... .......... .......... .......... 76%  188M 0s\n",
      "106150K .......... .......... .......... .......... .......... 76% 8.80M 0s\n",
      "106200K .......... .......... .......... .......... .......... 76%  148M 0s\n",
      "106250K .......... .......... .......... .......... .......... 76%  197M 0s\n",
      "106300K .......... .......... .......... .......... .......... 76%  175M 0s\n",
      "106350K .......... .......... .......... .......... .......... 76%  177M 0s\n",
      "106400K .......... .......... .......... .......... .......... 76%  133M 0s\n",
      "106450K .......... .......... .......... .......... .......... 76%  178M 0s\n",
      "106500K .......... .......... .......... .......... .......... 77%  180M 0s\n",
      "106550K .......... .......... .......... .......... .......... 77%  193M 0s\n",
      "106600K .......... .......... .......... .......... .......... 77%  176M 0s\n",
      "106650K .......... .......... .......... .......... .......... 77%  204M 0s\n",
      "106700K .......... .......... .......... .......... .......... 77%  182M 0s\n",
      "106750K .......... .......... .......... .......... .......... 77%  186M 0s\n",
      "106800K .......... .......... .......... .......... .......... 77%  165M 0s\n",
      "106850K .......... .......... .......... .......... .......... 77%  191M 0s\n",
      "106900K .......... .......... .......... .......... .......... 77%  177M 0s\n",
      "106950K .......... .......... .......... .......... .......... 77%  211M 0s\n",
      "107000K .......... .......... .......... .......... .......... 77%  187M 0s\n",
      "107050K .......... .......... .......... .......... .......... 77%  217M 0s\n",
      "107100K .......... .......... .......... .......... .......... 77%  217M 0s\n",
      "107150K .......... .......... .......... .......... .......... 77%  213M 0s\n",
      "107200K .......... .......... .......... .......... .......... 77%  174M 0s\n",
      "107250K .......... .......... .......... .......... .......... 77%  184M 0s\n",
      "107300K .......... .......... .......... .......... .......... 77%  211M 0s\n",
      "107350K .......... .......... .......... .......... .......... 77% 8.64M 0s\n",
      "107400K .......... .......... .......... .......... .......... 77%  132M 0s\n",
      "107450K .......... .......... .......... .......... .......... 77%  153M 0s\n",
      "107500K .......... .......... .......... .......... .......... 77%  131M 0s\n",
      "107550K .......... .......... .......... .......... .......... 77%  170M 0s\n",
      "107600K .......... .......... .......... .......... .......... 77%  158M 0s\n",
      "107650K .......... .......... .......... .......... .......... 77%  178M 0s\n",
      "107700K .......... .......... .......... .......... .......... 77%  182M 0s\n",
      "107750K .......... .......... .......... .......... .......... 77%  188M 0s\n",
      "107800K .......... .......... .......... .......... .......... 77%  168M 0s\n",
      "107850K .......... .......... .......... .......... .......... 77%  199M 0s\n",
      "107900K .......... .......... .......... .......... .......... 78%  209M 0s\n",
      "107950K .......... .......... .......... .......... .......... 78%  170M 0s\n",
      "108000K .......... .......... .......... .......... .......... 78%  160M 0s\n",
      "108050K .......... .......... .......... .......... .......... 78%  219M 0s\n",
      "108100K .......... .......... .......... .......... .......... 78%  206M 0s\n",
      "108150K .......... .......... .......... .......... .......... 78%  190M 0s\n",
      "108200K .......... .......... .......... .......... .......... 78%  177M 0s\n",
      "108250K .......... .......... .......... .......... .......... 78%  206M 0s\n",
      "108300K .......... .......... .......... .......... .......... 78%  204M 0s\n",
      "108350K .......... .......... .......... .......... .......... 78%  202M 0s\n",
      "108400K .......... .......... .......... .......... .......... 78%  165M 0s\n",
      "108450K .......... .......... .......... .......... .......... 78%  216M 0s\n",
      "108500K .......... .......... .......... .......... .......... 78%  211M 0s\n",
      "108550K .......... .......... .......... .......... .......... 78%  121M 0s\n",
      "108600K .......... .......... .......... .......... .......... 78%  156M 0s\n",
      "108650K .......... .......... .......... .......... .......... 78%  211M 0s\n",
      "108700K .......... .......... .......... .......... .......... 78%  195M 0s\n",
      "108750K .......... .......... .......... .......... .......... 78%  166M 0s\n",
      "108800K .......... .......... .......... .......... .......... 78%  174M 0s\n",
      "108850K .......... .......... .......... .......... .......... 78% 7.08M 0s\n",
      "108900K .......... .......... .......... .......... .......... 78% 95.3M 0s\n",
      "108950K .......... .......... .......... .......... .......... 78%  210M 0s\n",
      "109000K .......... .......... .......... .......... .......... 78%  155M 0s\n",
      "109050K .......... .......... .......... .......... .......... 78%  166M 0s\n",
      "109100K .......... .......... .......... .......... .......... 78%  168M 0s\n",
      "109150K .......... .......... .......... .......... .......... 78%  143M 0s\n",
      "109200K .......... .......... .......... .......... .......... 78%  170M 0s\n",
      "109250K .......... .......... .......... .......... .......... 79%  172M 0s\n",
      "109300K .......... .......... .......... .......... .......... 79%  185M 0s\n",
      "109350K .......... .......... .......... .......... .......... 79%  186M 0s\n",
      "109400K .......... .......... .......... .......... .......... 79%  178M 0s\n",
      "109450K .......... .......... .......... .......... .......... 79%  180M 0s\n",
      "109500K .......... .......... .......... .......... .......... 79%  184M 0s\n",
      "109550K .......... .......... .......... .......... .......... 79%  182M 0s\n",
      "109600K .......... .......... .......... .......... .......... 79%  147M 0s\n",
      "109650K .......... .......... .......... .......... .......... 79%  165M 0s\n",
      "109700K .......... .......... .......... .......... .......... 79%  166M 0s\n",
      "109750K .......... .......... .......... .......... .......... 79%  169M 0s\n",
      "109800K .......... .......... .......... .......... .......... 79%  165M 0s\n",
      "109850K .......... .......... .......... .......... .......... 79%  170M 0s\n",
      "109900K .......... .......... .......... .......... .......... 79%  169M 0s\n",
      "109950K .......... .......... .......... .......... .......... 79%  176M 0s\n",
      "110000K .......... .......... .......... .......... .......... 79%  142M 0s\n",
      "110050K .......... .......... .......... .......... .......... 79%  175M 0s\n",
      "110100K .......... .......... .......... .......... .......... 79%  205M 0s\n",
      "110150K .......... .......... .......... .......... .......... 79% 9.80M 0s\n",
      "110200K .......... .......... .......... .......... .......... 79%  139M 0s\n",
      "110250K .......... .......... .......... .......... .......... 79%  137M 0s\n",
      "110300K .......... .......... .......... .......... .......... 79%  172M 0s\n",
      "110350K .......... .......... .......... .......... .......... 79%  213M 0s\n",
      "110400K .......... .......... .......... .......... .......... 79%  140M 0s\n",
      "110450K .......... .......... .......... .......... .......... 79%  185M 0s\n",
      "110500K .......... .......... .......... .......... .......... 79%  192M 0s\n",
      "110550K .......... .......... .......... .......... .......... 79%  203M 0s\n",
      "110600K .......... .......... .......... .......... .......... 79%  184M 0s\n",
      "110650K .......... .......... .......... .......... .......... 80%  218M 0s\n",
      "110700K .......... .......... .......... .......... .......... 80%  193M 0s\n",
      "110750K .......... .......... .......... .......... .......... 80%  182M 0s\n",
      "110800K .......... .......... .......... .......... .......... 80%  178M 0s\n",
      "110850K .......... .......... .......... .......... .......... 80%  196M 0s\n",
      "110900K .......... .......... .......... .......... .......... 80%  190M 0s\n",
      "110950K .......... .......... .......... .......... .......... 80%  217M 0s\n",
      "111000K .......... .......... .......... .......... .......... 80%  171M 0s\n",
      "111050K .......... .......... .......... .......... .......... 80%  217M 0s\n",
      "111100K .......... .......... .......... .......... .......... 80%  213M 0s\n",
      "111150K .......... .......... .......... .......... .......... 80%  213M 0s\n",
      "111200K .......... .......... .......... .......... .......... 80% 44.4M 0s\n",
      "111250K .......... .......... .......... .......... .......... 80%  140M 0s\n",
      "111300K .......... .......... .......... .......... .......... 80% 81.1M 0s\n",
      "111350K .......... .......... .......... .......... .......... 80%  164M 0s\n",
      "111400K .......... .......... .......... .......... .......... 80%  166M 0s\n",
      "111450K .......... .......... .......... .......... .......... 80%  215M 0s\n",
      "111500K .......... .......... .......... .......... .......... 80%  195M 0s\n",
      "111550K .......... .......... .......... .......... .......... 80%  211M 0s\n",
      "111600K .......... .......... .......... .......... .......... 80%  182M 0s\n",
      "111650K .......... .......... .......... .......... .......... 80%  200M 0s\n",
      "111700K .......... .......... .......... .......... .......... 80%  213M 0s\n",
      "111750K .......... .......... .......... .......... .......... 80% 9.45M 0s\n",
      "111800K .......... .......... .......... .......... .......... 80%  142M 0s\n",
      "111850K .......... .......... .......... .......... .......... 80%  163M 0s\n",
      "111900K .......... .......... .......... .......... .......... 80%  238M 0s\n",
      "111950K .......... .......... .......... .......... .......... 80%  201M 0s\n",
      "112000K .......... .......... .......... .......... .......... 80%  149M 0s\n",
      "112050K .......... .......... .......... .......... .......... 81%  190M 0s\n",
      "112100K .......... .......... .......... .......... .......... 81%  184M 0s\n",
      "112150K .......... .......... .......... .......... .......... 81%  150M 0s\n",
      "112200K .......... .......... .......... .......... .......... 81%  175M 0s\n",
      "112250K .......... .......... .......... .......... .......... 81%  173M 0s\n",
      "112300K .......... .......... .......... .......... .......... 81%  152M 0s\n",
      "112350K .......... .......... .......... .......... .......... 81% 99.3M 0s\n",
      "112400K .......... .......... .......... .......... .......... 81%  128M 0s\n",
      "112450K .......... .......... .......... .......... .......... 81%  172M 0s\n",
      "112500K .......... .......... .......... .......... .......... 81%  191M 0s\n",
      "112550K .......... .......... .......... .......... .......... 81%  197M 0s\n",
      "112600K .......... .......... .......... .......... .......... 81%  161M 0s\n",
      "112650K .......... .......... .......... .......... .......... 81%  216M 0s\n",
      "112700K .......... .......... .......... .......... .......... 81%  206M 0s\n",
      "112750K .......... .......... .......... .......... .......... 81%  214M 0s\n",
      "112800K .......... .......... .......... .......... .......... 81% 7.54M 0s\n",
      "112850K .......... .......... .......... .......... .......... 81%  115M 0s\n",
      "112900K .......... .......... .......... .......... .......... 81%  120M 0s\n",
      "112950K .......... .......... .......... .......... .......... 81%  171M 0s\n",
      "113000K .......... .......... .......... .......... .......... 81%  159M 0s\n",
      "113050K .......... .......... .......... .......... .......... 81%  159M 0s\n",
      "113100K .......... .......... .......... .......... .......... 81%  156M 0s\n",
      "113150K .......... .......... .......... .......... .......... 81%  177M 0s\n",
      "113200K .......... .......... .......... .......... .......... 81%  170M 0s\n",
      "113250K .......... .......... .......... .......... .......... 81%  209M 0s\n",
      "113300K .......... .......... .......... .......... .......... 81%  192M 0s\n",
      "113350K .......... .......... .......... .......... .......... 81%  172M 0s\n",
      "113400K .......... .......... .......... .......... .......... 82%  179M 0s\n",
      "113450K .......... .......... .......... .......... .......... 82%  193M 0s\n",
      "113500K .......... .......... .......... .......... .......... 82%  190M 0s\n",
      "113550K .......... .......... .......... .......... .......... 82%  170M 0s\n",
      "113600K .......... .......... .......... .......... .......... 82%  172M 0s\n",
      "113650K .......... .......... .......... .......... .......... 82%  197M 0s\n",
      "113700K .......... .......... .......... .......... .......... 82%  201M 0s\n",
      "113750K .......... .......... .......... .......... .......... 82%  214M 0s\n",
      "113800K .......... .......... .......... .......... .......... 82%  161M 0s\n",
      "113850K .......... .......... .......... .......... .......... 82%  198M 0s\n",
      "113900K .......... .......... .......... .......... .......... 82%  208M 0s\n",
      "113950K .......... .......... .......... .......... .......... 82%  167M 0s\n",
      "114000K .......... .......... .......... .......... .......... 82%  142M 0s\n",
      "114050K .......... .......... .......... .......... .......... 82%  175M 0s\n",
      "114100K .......... .......... .......... .......... .......... 82%  169M 0s\n",
      "114150K .......... .......... .......... .......... .......... 82%  178M 0s\n",
      "114200K .......... .......... .......... .......... .......... 82%  188M 0s\n",
      "114250K .......... .......... .......... .......... .......... 82%  197M 0s\n",
      "114300K .......... .......... .......... .......... .......... 82%  211M 0s\n",
      "114350K .......... .......... .......... .......... .......... 82%  216M 0s\n",
      "114400K .......... .......... .......... .......... .......... 82%  169M 0s\n",
      "114450K .......... .......... .......... .......... .......... 82%  185M 0s\n",
      "114500K .......... .......... .......... .......... .......... 82% 8.91M 0s\n",
      "114550K .......... .......... .......... .......... .......... 82%  156M 0s\n",
      "114600K .......... .......... .......... .......... .......... 82%  174M 0s\n",
      "114650K .......... .......... .......... .......... .......... 82%  200M 0s\n",
      "114700K .......... .......... .......... .......... .......... 82%  207M 0s\n",
      "114750K .......... .......... .......... .......... .......... 82%  216M 0s\n",
      "114800K .......... .......... .......... .......... .......... 83%  178M 0s\n",
      "114850K .......... .......... .......... .......... .......... 83%  219M 0s\n",
      "114900K .......... .......... .......... .......... .......... 83%  177M 0s\n",
      "114950K .......... .......... .......... .......... .......... 83%  127M 0s\n",
      "115000K .......... .......... .......... .......... .......... 83%  158M 0s\n",
      "115050K .......... .......... .......... .......... .......... 83%  208M 0s\n",
      "115100K .......... .......... .......... .......... .......... 83%  191M 0s\n",
      "115150K .......... .......... .......... .......... .......... 83%  182M 0s\n",
      "115200K .......... .......... .......... .......... .......... 83%  156M 0s\n",
      "115250K .......... .......... .......... .......... .......... 83%  191M 0s\n",
      "115300K .......... .......... .......... .......... .......... 83%  216M 0s\n",
      "115350K .......... .......... .......... .......... .......... 83%  186M 0s\n",
      "115400K .......... .......... .......... .......... .......... 83%  163M 0s\n",
      "115450K .......... .......... .......... .......... .......... 83%  165M 0s\n",
      "115500K .......... .......... .......... .......... .......... 83%  219M 0s\n",
      "115550K .......... .......... .......... .......... .......... 83%  188M 0s\n",
      "115600K .......... .......... .......... .......... .......... 83%  174M 0s\n",
      "115650K .......... .......... .......... .......... .......... 83% 7.59M 0s\n",
      "115700K .......... .......... .......... .......... .......... 83%  146M 0s\n",
      "115750K .......... .......... .......... .......... .......... 83%  166M 0s\n",
      "115800K .......... .......... .......... .......... .......... 83%  149M 0s\n",
      "115850K .......... .......... .......... .......... .......... 83%  160M 0s\n",
      "115900K .......... .......... .......... .......... .......... 83%  163M 0s\n",
      "115950K .......... .......... .......... .......... .......... 83%  165M 0s\n",
      "116000K .......... .......... .......... .......... .......... 83%  136M 0s\n",
      "116050K .......... .......... .......... .......... .......... 83%  177M 0s\n",
      "116100K .......... .......... .......... .......... .......... 83%  197M 0s\n",
      "116150K .......... .......... .......... .......... .......... 83%  191M 0s\n",
      "116200K .......... .......... .......... .......... .......... 84%  173M 0s\n",
      "116250K .......... .......... .......... .......... .......... 84%  199M 0s\n",
      "116300K .......... .......... .......... .......... .......... 84%  193M 0s\n",
      "116350K .......... .......... .......... .......... .......... 84%  183M 0s\n",
      "116400K .......... .......... .......... .......... .......... 84%  178M 0s\n",
      "116450K .......... .......... .......... .......... .......... 84%  208M 0s\n",
      "116500K .......... .......... .......... .......... .......... 84%  218M 0s\n",
      "116550K .......... .......... .......... .......... .......... 84%  216M 0s\n",
      "116600K .......... .......... .......... .......... .......... 84%  188M 0s\n",
      "116650K .......... .......... .......... .......... .......... 84%  216M 0s\n",
      "116700K .......... .......... .......... .......... .......... 84%  201M 0s\n",
      "116750K .......... .......... .......... .......... .......... 84% 56.8M 0s\n",
      "116800K .......... .......... .......... .......... .......... 84% 94.8M 0s\n",
      "116850K .......... .......... .......... .......... .......... 84%  179M 0s\n",
      "116900K .......... .......... .......... .......... .......... 84%  209M 0s\n",
      "116950K .......... .......... .......... .......... .......... 84%  212M 0s\n",
      "117000K .......... .......... .......... .......... .......... 84%  175M 0s\n",
      "117050K .......... .......... .......... .......... .......... 84%  213M 0s\n",
      "117100K .......... .......... .......... .......... .......... 84%  209M 0s\n",
      "117150K .......... .......... .......... .......... .......... 84%  215M 0s\n",
      "117200K .......... .......... .......... .......... .......... 84%  178M 0s\n",
      "117250K .......... .......... .......... .......... .......... 84%  216M 0s\n",
      "117300K .......... .......... .......... .......... .......... 84% 9.29M 0s\n",
      "117350K .......... .......... .......... .......... .......... 84%  158M 0s\n",
      "117400K .......... .......... .......... .......... .......... 84%  143M 0s\n",
      "117450K .......... .......... .......... .......... .......... 84%  211M 0s\n",
      "117500K .......... .......... .......... .......... .......... 84%  193M 0s\n",
      "117550K .......... .......... .......... .......... .......... 85%  207M 0s\n",
      "117600K .......... .......... .......... .......... .......... 85%  181M 0s\n",
      "117650K .......... .......... .......... .......... .......... 85%  197M 0s\n",
      "117700K .......... .......... .......... .......... .......... 85%  170M 0s\n",
      "117750K .......... .......... .......... .......... .......... 85%  177M 0s\n",
      "117800K .......... .......... .......... .......... .......... 85%  170M 0s\n",
      "117850K .......... .......... .......... .......... .......... 85%  211M 0s\n",
      "117900K .......... .......... .......... .......... .......... 85%  168M 0s\n",
      "117950K .......... .......... .......... .......... .......... 85%  103M 0s\n",
      "118000K .......... .......... .......... .......... .......... 85%  170M 0s\n",
      "118050K .......... .......... .......... .......... .......... 85%  142M 0s\n",
      "118100K .......... .......... .......... .......... .......... 85%  203M 0s\n",
      "118150K .......... .......... .......... .......... .......... 85%  148M 0s\n",
      "118200K .......... .......... .......... .......... .......... 85%  169M 0s\n",
      "118250K .......... .......... .......... .......... .......... 85%  211M 0s\n",
      "118300K .......... .......... .......... .......... .......... 85%  172M 0s\n",
      "118350K .......... .......... .......... .......... .......... 85%  147M 0s\n",
      "118400K .......... .......... .......... .......... .......... 85%  139M 0s\n",
      "118450K .......... .......... .......... .......... .......... 85%  169M 0s\n",
      "118500K .......... .......... .......... .......... .......... 85% 7.70M 0s\n",
      "118550K .......... .......... .......... .......... .......... 85%  131M 0s\n",
      "118600K .......... .......... .......... .......... .......... 85%  114M 0s\n",
      "118650K .......... .......... .......... .......... .......... 85%  117M 0s\n",
      "118700K .......... .......... .......... .......... .......... 85% 89.5M 0s\n",
      "118750K .......... .......... .......... .......... .......... 85%  141M 0s\n",
      "118800K .......... .......... .......... .......... .......... 85%  110M 0s\n",
      "118850K .......... .......... .......... .......... .......... 85%  132M 0s\n",
      "118900K .......... .......... .......... .......... .......... 85%  134M 0s\n",
      "118950K .......... .......... .......... .......... .......... 86%  213M 0s\n",
      "119000K .......... .......... .......... .......... .......... 86%  194M 0s\n",
      "119050K .......... .......... .......... .......... .......... 86%  191M 0s\n",
      "119100K .......... .......... .......... .......... .......... 86%  174M 0s\n",
      "119150K .......... .......... .......... .......... .......... 86%  175M 0s\n",
      "119200K .......... .......... .......... .......... .......... 86%  141M 0s\n",
      "119250K .......... .......... .......... .......... .......... 86%  175M 0s\n",
      "119300K .......... .......... .......... .......... .......... 86%  171M 0s\n",
      "119350K .......... .......... .......... .......... .......... 86%  135M 0s\n",
      "119400K .......... .......... .......... .......... .......... 86%  112M 0s\n",
      "119450K .......... .......... .......... .......... .......... 86%  152M 0s\n",
      "119500K .......... .......... .......... .......... .......... 86%  214M 0s\n",
      "119550K .......... .......... .......... .......... .......... 86%  164M 0s\n",
      "119600K .......... .......... .......... .......... .......... 86%  141M 0s\n",
      "119650K .......... .......... .......... .......... .......... 86%  179M 0s\n",
      "119700K .......... .......... .......... .......... .......... 86%  206M 0s\n",
      "119750K .......... .......... .......... .......... .......... 86%  212M 0s\n",
      "119800K .......... .......... .......... .......... .......... 86%  187M 0s\n",
      "119850K .......... .......... .......... .......... .......... 86%  222M 0s\n",
      "119900K .......... .......... .......... .......... .......... 86%  207M 0s\n",
      "119950K .......... .......... .......... .......... .......... 86%  223M 0s\n",
      "120000K .......... .......... .......... .......... .......... 86%  179M 0s\n",
      "120050K .......... .......... .......... .......... .......... 86%  212M 0s\n",
      "120100K .......... .......... .......... .......... .......... 86%  181M 0s\n",
      "120150K .......... .......... .......... .......... .......... 86% 12.1M 0s\n",
      "120200K .......... .......... .......... .......... .......... 86%  135M 0s\n",
      "120250K .......... .......... .......... .......... .......... 86%  150M 0s\n",
      "120300K .......... .......... .......... .......... .......... 86%  180M 0s\n",
      "120350K .......... .......... .......... .......... .......... 87%  177M 0s\n",
      "120400K .......... .......... .......... .......... .......... 87% 76.0M 0s\n",
      "120450K .......... .......... .......... .......... .......... 87%  114M 0s\n",
      "120500K .......... .......... .......... .......... .......... 87%  132M 0s\n",
      "120550K .......... .......... .......... .......... .......... 87%  110M 0s\n",
      "120600K .......... .......... .......... .......... .......... 87%  153M 0s\n",
      "120650K .......... .......... .......... .......... .......... 87%  144M 0s\n",
      "120700K .......... .......... .......... .......... .......... 87%  179M 0s\n",
      "120750K .......... .......... .......... .......... .......... 87%  165M 0s\n",
      "120800K .......... .......... .......... .......... .......... 87%  149M 0s\n",
      "120850K .......... .......... .......... .......... .......... 87%  204M 0s\n",
      "120900K .......... .......... .......... .......... .......... 87%  181M 0s\n",
      "120950K .......... .......... .......... .......... .......... 87%  213M 0s\n",
      "121000K .......... .......... .......... .......... .......... 87%  193M 0s\n",
      "121050K .......... .......... .......... .......... .......... 87%  216M 0s\n",
      "121100K .......... .......... .......... .......... .......... 87%  207M 0s\n",
      "121150K .......... .......... .......... .......... .......... 87%  209M 0s\n",
      "121200K .......... .......... .......... .......... .......... 87%  182M 0s\n",
      "121250K .......... .......... .......... .......... .......... 87%  215M 0s\n",
      "121300K .......... .......... .......... .......... .......... 87%  207M 0s\n",
      "121350K .......... .......... .......... .......... .......... 87%  209M 0s\n",
      "121400K .......... .......... .......... .......... .......... 87% 8.12M 0s\n",
      "121450K .......... .......... .......... .......... .......... 87%  161M 0s\n",
      "121500K .......... .......... .......... .......... .......... 87%  196M 0s\n",
      "121550K .......... .......... .......... .......... .......... 87%  177M 0s\n",
      "121600K .......... .......... .......... .......... .......... 87%  172M 0s\n",
      "121650K .......... .......... .......... .......... .......... 87%  204M 0s\n",
      "121700K .......... .......... .......... .......... .......... 88%  192M 0s\n",
      "121750K .......... .......... .......... .......... .......... 88%  202M 0s\n",
      "121800K .......... .......... .......... .......... .......... 88%  170M 0s\n",
      "121850K .......... .......... .......... .......... .......... 88%  193M 0s\n",
      "121900K .......... .......... .......... .......... .......... 88%  197M 0s\n",
      "121950K .......... .......... .......... .......... .......... 88%  198M 0s\n",
      "122000K .......... .......... .......... .......... .......... 88%  149M 0s\n",
      "122050K .......... .......... .......... .......... .......... 88%  217M 0s\n",
      "122100K .......... .......... .......... .......... .......... 88%  213M 0s\n",
      "122150K .......... .......... .......... .......... .......... 88%  213M 0s\n",
      "122200K .......... .......... .......... .......... .......... 88%  158M 0s\n",
      "122250K .......... .......... .......... .......... .......... 88% 31.8M 0s\n",
      "122300K .......... .......... .......... .......... .......... 88%  144M 0s\n",
      "122350K .......... .......... .......... .......... .......... 88%  185M 0s\n",
      "122400K .......... .......... .......... .......... .......... 88%  127M 0s\n",
      "122450K .......... .......... .......... .......... .......... 88%  195M 0s\n",
      "122500K .......... .......... .......... .......... .......... 88%  187M 0s\n",
      "122550K .......... .......... .......... .......... .......... 88%  181M 0s\n",
      "122600K .......... .......... .......... .......... .......... 88%  174M 0s\n",
      "122650K .......... .......... .......... .......... .......... 88%  213M 0s\n",
      "122700K .......... .......... .......... .......... .......... 88%  218M 0s\n",
      "122750K .......... .......... .......... .......... .......... 88%  181M 0s\n",
      "122800K .......... .......... .......... .......... .......... 88%  171M 0s\n",
      "122850K .......... .......... .......... .......... .......... 88% 9.09M 0s\n",
      "122900K .......... .......... .......... .......... .......... 88%  148M 0s\n",
      "122950K .......... .......... .......... .......... .......... 88%  166M 0s\n",
      "123000K .......... .......... .......... .......... .......... 88%  155M 0s\n",
      "123050K .......... .......... .......... .......... .......... 88%  172M 0s\n",
      "123100K .......... .......... .......... .......... .......... 89%  186M 0s\n",
      "123150K .......... .......... .......... .......... .......... 89%  173M 0s\n",
      "123200K .......... .......... .......... .......... .......... 89%  164M 0s\n",
      "123250K .......... .......... .......... .......... .......... 89%  196M 0s\n",
      "123300K .......... .......... .......... .......... .......... 89%  176M 0s\n",
      "123350K .......... .......... .......... .......... .......... 89%  185M 0s\n",
      "123400K .......... .......... .......... .......... .......... 89%  164M 0s\n",
      "123450K .......... .......... .......... .......... .......... 89%  214M 0s\n",
      "123500K .......... .......... .......... .......... .......... 89%  185M 0s\n",
      "123550K .......... .......... .......... .......... .......... 89%  218M 0s\n",
      "123600K .......... .......... .......... .......... .......... 89%  163M 0s\n",
      "123650K .......... .......... .......... .......... .......... 89%  180M 0s\n",
      "123700K .......... .......... .......... .......... .......... 89%  204M 0s\n",
      "123750K .......... .......... .......... .......... .......... 89%  219M 0s\n",
      "123800K .......... .......... .......... .......... .......... 89%  187M 0s\n",
      "123850K .......... .......... .......... .......... .......... 89%  225M 0s\n",
      "123900K .......... .......... .......... .......... .......... 89%  204M 0s\n",
      "123950K .......... .......... .......... .......... .......... 89%  222M 0s\n",
      "124000K .......... .......... .......... .......... .......... 89%  169M 0s\n",
      "124050K .......... .......... .......... .......... .......... 89%  217M 0s\n",
      "124100K .......... .......... .......... .......... .......... 89% 7.66M 0s\n",
      "124150K .......... .......... .......... .......... .......... 89%  174M 0s\n",
      "124200K .......... .......... .......... .......... .......... 89%  148M 0s\n",
      "124250K .......... .......... .......... .......... .......... 89%  134M 0s\n",
      "124300K .......... .......... .......... .......... .......... 89%  159M 0s\n",
      "124350K .......... .......... .......... .......... .......... 89%  176M 0s\n",
      "124400K .......... .......... .......... .......... .......... 89%  156M 0s\n",
      "124450K .......... .......... .......... .......... .......... 89%  181M 0s\n",
      "124500K .......... .......... .......... .......... .......... 90%  201M 0s\n",
      "124550K .......... .......... .......... .......... .......... 90%  196M 0s\n",
      "124600K .......... .......... .......... .......... .......... 90%  186M 0s\n",
      "124650K .......... .......... .......... .......... .......... 90%  183M 0s\n",
      "124700K .......... .......... .......... .......... .......... 90%  195M 0s\n",
      "124750K .......... .......... .......... .......... .......... 90%  210M 0s\n",
      "124800K .......... .......... .......... .......... .......... 90%  174M 0s\n",
      "124850K .......... .......... .......... .......... .......... 90%  146M 0s\n",
      "124900K .......... .......... .......... .......... .......... 90%  208M 0s\n",
      "124950K .......... .......... .......... .......... .......... 90%  214M 0s\n",
      "125000K .......... .......... .......... .......... .......... 90%  187M 0s\n",
      "125050K .......... .......... .......... .......... .......... 90%  220M 0s\n",
      "125100K .......... .......... .......... .......... .......... 90%  231M 0s\n",
      "125150K .......... .......... .......... .......... .......... 90%  190M 0s\n",
      "125200K .......... .......... .......... .......... .......... 90%  111M 0s\n",
      "125250K .......... .......... .......... .......... .......... 90%  199M 0s\n",
      "125300K .......... .......... .......... .......... .......... 90%  200M 0s\n",
      "125350K .......... .......... .......... .......... .......... 90%  174M 0s\n",
      "125400K .......... .......... .......... .......... .......... 90%  175M 0s\n",
      "125450K .......... .......... .......... .......... .......... 90%  209M 0s\n",
      "125500K .......... .......... .......... .......... .......... 90%  207M 0s\n",
      "125550K .......... .......... .......... .......... .......... 90%  198M 0s\n",
      "125600K .......... .......... .......... .......... .......... 90%  174M 0s\n",
      "125650K .......... .......... .......... .......... .......... 90%  187M 0s\n",
      "125700K .......... .......... .......... .......... .......... 90%  214M 0s\n",
      "125750K .......... .......... .......... .......... .......... 90% 9.19M 0s\n",
      "125800K .......... .......... .......... .......... .......... 90%  184M 0s\n",
      "125850K .......... .......... .......... .......... .......... 91%  179M 0s\n",
      "125900K .......... .......... .......... .......... .......... 91%  197M 0s\n",
      "125950K .......... .......... .......... .......... .......... 91%  212M 0s\n",
      "126000K .......... .......... .......... .......... .......... 91%  169M 0s\n",
      "126050K .......... .......... .......... .......... .......... 91%  193M 0s\n",
      "126100K .......... .......... .......... .......... .......... 91%  177M 0s\n",
      "126150K .......... .......... .......... .......... .......... 91%  178M 0s\n",
      "126200K .......... .......... .......... .......... .......... 91% 88.5M 0s\n",
      "126250K .......... .......... .......... .......... .......... 91%  184M 0s\n",
      "126300K .......... .......... .......... .......... .......... 91%  200M 0s\n",
      "126350K .......... .......... .......... .......... .......... 91%  185M 0s\n",
      "126400K .......... .......... .......... .......... .......... 91%  166M 0s\n",
      "126450K .......... .......... .......... .......... .......... 91%  201M 0s\n",
      "126500K .......... .......... .......... .......... .......... 91%  213M 0s\n",
      "126550K .......... .......... .......... .......... .......... 91%  170M 0s\n",
      "126600K .......... .......... .......... .......... .......... 91%  165M 0s\n",
      "126650K .......... .......... .......... .......... .......... 91%  199M 0s\n",
      "126700K .......... .......... .......... .......... .......... 91%  201M 0s\n",
      "126750K .......... .......... .......... .......... .......... 91%  208M 0s\n",
      "126800K .......... .......... .......... .......... .......... 91%  178M 0s\n",
      "126850K .......... .......... .......... .......... .......... 91%  212M 0s\n",
      "126900K .......... .......... .......... .......... .......... 91% 6.93M 0s\n",
      "126950K .......... .......... .......... .......... .......... 91%  169M 0s\n",
      "127000K .......... .......... .......... .......... .......... 91%  150M 0s\n",
      "127050K .......... .......... .......... .......... .......... 91%  148M 0s\n",
      "127100K .......... .......... .......... .......... .......... 91%  179M 0s\n",
      "127150K .......... .......... .......... .......... .......... 91%  196M 0s\n",
      "127200K .......... .......... .......... .......... .......... 91%  168M 0s\n",
      "127250K .......... .......... .......... .......... .......... 92%  181M 0s\n",
      "127300K .......... .......... .......... .......... .......... 92%  174M 0s\n",
      "127350K .......... .......... .......... .......... .......... 92%  215M 0s\n",
      "127400K .......... .......... .......... .......... .......... 92%  193M 0s\n",
      "127450K .......... .......... .......... .......... .......... 92%  190M 0s\n",
      "127500K .......... .......... .......... .......... .......... 92%  167M 0s\n",
      "127550K .......... .......... .......... .......... .......... 92%  220M 0s\n",
      "127600K .......... .......... .......... .......... .......... 92%  181M 0s\n",
      "127650K .......... .......... .......... .......... .......... 92%  218M 0s\n",
      "127700K .......... .......... .......... .......... .......... 92%  217M 0s\n",
      "127750K .......... .......... .......... .......... .......... 92%  156M 0s\n",
      "127800K .......... .......... .......... .......... .......... 92%  190M 0s\n",
      "127850K .......... .......... .......... .......... .......... 92%  215M 0s\n",
      "127900K .......... .......... .......... .......... .......... 92% 46.0M 0s\n",
      "127950K .......... .......... .......... .......... .......... 92%  163M 0s\n",
      "128000K .......... .......... .......... .......... .......... 92%  139M 0s\n",
      "128050K .......... .......... .......... .......... .......... 92%  177M 0s\n",
      "128100K .......... .......... .......... .......... .......... 92%  191M 0s\n",
      "128150K .......... .......... .......... .......... .......... 92%  197M 0s\n",
      "128200K .......... .......... .......... .......... .......... 92%  165M 0s\n",
      "128250K .......... .......... .......... .......... .......... 92%  190M 0s\n",
      "128300K .......... .......... .......... .......... .......... 92%  199M 0s\n",
      "128350K .......... .......... .......... .......... .......... 92%  197M 0s\n",
      "128400K .......... .......... .......... .......... .......... 92%  164M 0s\n",
      "128450K .......... .......... .......... .......... .......... 92%  175M 0s\n",
      "128500K .......... .......... .......... .......... .......... 92%  201M 0s\n",
      "128550K .......... .......... .......... .......... .......... 92% 11.1M 0s\n",
      "128600K .......... .......... .......... .......... .......... 92%  126M 0s\n",
      "128650K .......... .......... .......... .......... .......... 93%  186M 0s\n",
      "128700K .......... .......... .......... .......... .......... 93%  208M 0s\n",
      "128750K .......... .......... .......... .......... .......... 93%  210M 0s\n",
      "128800K .......... .......... .......... .......... .......... 93%  158M 0s\n",
      "128850K .......... .......... .......... .......... .......... 93%  199M 0s\n",
      "128900K .......... .......... .......... .......... .......... 93%  194M 0s\n",
      "128950K .......... .......... .......... .......... .......... 93%  192M 0s\n",
      "129000K .......... .......... .......... .......... .......... 93%  166M 0s\n",
      "129050K .......... .......... .......... .......... .......... 93%  185M 0s\n",
      "129100K .......... .......... .......... .......... .......... 93%  198M 0s\n",
      "129150K .......... .......... .......... .......... .......... 93%  150M 0s\n",
      "129200K .......... .......... .......... .......... .......... 93%  148M 0s\n",
      "129250K .......... .......... .......... .......... .......... 93%  214M 0s\n",
      "129300K .......... .......... .......... .......... .......... 93%  211M 0s\n",
      "129350K .......... .......... .......... .......... .......... 93%  200M 0s\n",
      "129400K .......... .......... .......... .......... .......... 93%  174M 0s\n",
      "129450K .......... .......... .......... .......... .......... 93%  200M 0s\n",
      "129500K .......... .......... .......... .......... .......... 93%  193M 0s\n",
      "129550K .......... .......... .......... .......... .......... 93%  212M 0s\n",
      "129600K .......... .......... .......... .......... .......... 93%  178M 0s\n",
      "129650K .......... .......... .......... .......... .......... 93%  159M 0s\n",
      "129700K .......... .......... .......... .......... .......... 93%  199M 0s\n",
      "129750K .......... .......... .......... .......... .......... 93% 6.65M 0s\n",
      "129800K .......... .......... .......... .......... .......... 93%  119M 0s\n",
      "129850K .......... .......... .......... .......... .......... 93%  176M 0s\n",
      "129900K .......... .......... .......... .......... .......... 93%  179M 0s\n",
      "129950K .......... .......... .......... .......... .......... 93%  190M 0s\n",
      "130000K .......... .......... .......... .......... .......... 94%  166M 0s\n",
      "130050K .......... .......... .......... .......... .......... 94%  178M 0s\n",
      "130100K .......... .......... .......... .......... .......... 94%  162M 0s\n",
      "130150K .......... .......... .......... .......... .......... 94%  202M 0s\n",
      "130200K .......... .......... .......... .......... .......... 94%  167M 0s\n",
      "130250K .......... .......... .......... .......... .......... 94%  188M 0s\n",
      "130300K .......... .......... .......... .......... .......... 94%  190M 0s\n",
      "130350K .......... .......... .......... .......... .......... 94%  210M 0s\n",
      "130400K .......... .......... .......... .......... .......... 94%  171M 0s\n",
      "130450K .......... .......... .......... .......... .......... 94%  210M 0s\n",
      "130500K .......... .......... .......... .......... .......... 94%  218M 0s\n",
      "130550K .......... .......... .......... .......... .......... 94%  222M 0s\n",
      "130600K .......... .......... .......... .......... .......... 94%  141M 0s\n",
      "130650K .......... .......... .......... .......... .......... 94%  195M 0s\n",
      "130700K .......... .......... .......... .......... .......... 94% 67.9M 0s\n",
      "130750K .......... .......... .......... .......... .......... 94%  133M 0s\n",
      "130800K .......... .......... .......... .......... .......... 94%  138M 0s\n",
      "130850K .......... .......... .......... .......... .......... 94%  182M 0s\n",
      "130900K .......... .......... .......... .......... .......... 94%  203M 0s\n",
      "130950K .......... .......... .......... .......... .......... 94%  145M 0s\n",
      "131000K .......... .......... .......... .......... .......... 94%  189M 0s\n",
      "131050K .......... .......... .......... .......... .......... 94%  191M 0s\n",
      "131100K .......... .......... .......... .......... .......... 94%  184M 0s\n",
      "131150K .......... .......... .......... .......... .......... 94%  201M 0s\n",
      "131200K .......... .......... .......... .......... .......... 94%  175M 0s\n",
      "131250K .......... .......... .......... .......... .......... 94%  216M 0s\n",
      "131300K .......... .......... .......... .......... .......... 94%  215M 0s\n",
      "131350K .......... .......... .......... .......... .......... 94%  165M 0s\n",
      "131400K .......... .......... .......... .......... .......... 95%  189M 0s\n",
      "131450K .......... .......... .......... .......... .......... 95% 13.2M 0s\n",
      "131500K .......... .......... .......... .......... .......... 95%  187M 0s\n",
      "131550K .......... .......... .......... .......... .......... 95%  207M 0s\n",
      "131600K .......... .......... .......... .......... .......... 95%  129M 0s\n",
      "131650K .......... .......... .......... .......... .......... 95%  213M 0s\n",
      "131700K .......... .......... .......... .......... .......... 95% 26.9M 0s\n",
      "131750K .......... .......... .......... .......... .......... 95%  157M 0s\n",
      "131800K .......... .......... .......... .......... .......... 95%  146M 0s\n",
      "131850K .......... .......... .......... .......... .......... 95%  154M 0s\n",
      "131900K .......... .......... .......... .......... .......... 95%  179M 0s\n",
      "131950K .......... .......... .......... .......... .......... 95%  178M 0s\n",
      "132000K .......... .......... .......... .......... .......... 95%  136M 0s\n",
      "132050K .......... .......... .......... .......... .......... 95%  175M 0s\n",
      "132100K .......... .......... .......... .......... .......... 95%  162M 0s\n",
      "132150K .......... .......... .......... .......... .......... 95%  133M 0s\n",
      "132200K .......... .......... .......... .......... .......... 95%  164M 0s\n",
      "132250K .......... .......... .......... .......... .......... 95%  211M 0s\n",
      "132300K .......... .......... .......... .......... .......... 95%  188M 0s\n",
      "132350K .......... .......... .......... .......... .......... 95%  206M 0s\n",
      "132400K .......... .......... .......... .......... .......... 95%  178M 0s\n",
      "132450K .......... .......... .......... .......... .......... 95%  213M 0s\n",
      "132500K .......... .......... .......... .......... .......... 95%  222M 0s\n",
      "132550K .......... .......... .......... .......... .......... 95%  207M 0s\n",
      "132600K .......... .......... .......... .......... .......... 95%  189M 0s\n",
      "132650K .......... .......... .......... .......... .......... 95% 8.49M 0s\n",
      "132700K .......... .......... .......... .......... .......... 95%  148M 0s\n",
      "132750K .......... .......... .......... .......... .......... 95%  205M 0s\n",
      "132800K .......... .......... .......... .......... .......... 96%  153M 0s\n",
      "132850K .......... .......... .......... .......... .......... 96%  190M 0s\n",
      "132900K .......... .......... .......... .......... .......... 96%  212M 0s\n",
      "132950K .......... .......... .......... .......... .......... 96%  190M 0s\n",
      "133000K .......... .......... .......... .......... .......... 96%  153M 0s\n",
      "133050K .......... .......... .......... .......... .......... 96%  188M 0s\n",
      "133100K .......... .......... .......... .......... .......... 96%  202M 0s\n",
      "133150K .......... .......... .......... .......... .......... 96%  208M 0s\n",
      "133200K .......... .......... .......... .......... .......... 96%  156M 0s\n",
      "133250K .......... .......... .......... .......... .......... 96%  181M 0s\n",
      "133300K .......... .......... .......... .......... .......... 96%  196M 0s\n",
      "133350K .......... .......... .......... .......... .......... 96%  216M 0s\n",
      "133400K .......... .......... .......... .......... .......... 96%  188M 0s\n",
      "133450K .......... .......... .......... .......... .......... 96% 31.0M 0s\n",
      "133500K .......... .......... .......... .......... .......... 96%  179M 0s\n",
      "133550K .......... .......... .......... .......... .......... 96%  129M 0s\n",
      "133600K .......... .......... .......... .......... .......... 96% 86.8M 0s\n",
      "133650K .......... .......... .......... .......... .......... 96%  161M 0s\n",
      "133700K .......... .......... .......... .......... .......... 96%  165M 0s\n",
      "133750K .......... .......... .......... .......... .......... 96%  180M 0s\n",
      "133800K .......... .......... .......... .......... .......... 96%  170M 0s\n",
      "133850K .......... .......... .......... .......... .......... 96%  184M 0s\n",
      "133900K .......... .......... .......... .......... .......... 96%  218M 0s\n",
      "133950K .......... .......... .......... .......... .......... 96%  190M 0s\n",
      "134000K .......... .......... .......... .......... .......... 96%  174M 0s\n",
      "134050K .......... .......... .......... .......... .......... 96%  212M 0s\n",
      "134100K .......... .......... .......... .......... .......... 96%  211M 0s\n",
      "134150K .......... .......... .......... .......... .......... 97%  219M 0s\n",
      "134200K .......... .......... .......... .......... .......... 97%  184M 0s\n",
      "134250K .......... .......... .......... .......... .......... 97%  203M 0s\n",
      "134300K .......... .......... .......... .......... .......... 97% 15.3M 0s\n",
      "134350K .......... .......... .......... .......... .......... 97%  166M 0s\n",
      "134400K .......... .......... .......... .......... .......... 97%  161M 0s\n",
      "134450K .......... .......... .......... .......... .......... 97%  210M 0s\n",
      "134500K .......... .......... .......... .......... .......... 97%  212M 0s\n",
      "134550K .......... .......... .......... .......... .......... 97%  197M 0s\n",
      "134600K .......... .......... .......... .......... .......... 97% 28.3M 0s\n",
      "134650K .......... .......... .......... .......... .......... 97%  189M 0s\n",
      "134700K .......... .......... .......... .......... .......... 97%  175M 0s\n",
      "134750K .......... .......... .......... .......... .......... 97%  177M 0s\n",
      "134800K .......... .......... .......... .......... .......... 97%  151M 0s\n",
      "134850K .......... .......... .......... .......... .......... 97%  198M 0s\n",
      "134900K .......... .......... .......... .......... .......... 97%  193M 0s\n",
      "134950K .......... .......... .......... .......... .......... 97%  196M 0s\n",
      "135000K .......... .......... .......... .......... .......... 97%  181M 0s\n",
      "135050K .......... .......... .......... .......... .......... 97%  200M 0s\n",
      "135100K .......... .......... .......... .......... .......... 97%  170M 0s\n",
      "135150K .......... .......... .......... .......... .......... 97%  150M 0s\n",
      "135200K .......... .......... .......... .......... .......... 97%  177M 0s\n",
      "135250K .......... .......... .......... .......... .......... 97%  210M 0s\n",
      "135300K .......... .......... .......... .......... .......... 97%  203M 0s\n",
      "135350K .......... .......... .......... .......... .......... 97% 7.31M 0s\n",
      "135400K .......... .......... .......... .......... .......... 97%  130M 0s\n",
      "135450K .......... .......... .......... .......... .......... 97% 84.8M 0s\n",
      "135500K .......... .......... .......... .......... .......... 97%  163M 0s\n",
      "135550K .......... .......... .......... .......... .......... 98%  179M 0s\n",
      "135600K .......... .......... .......... .......... .......... 98%  158M 0s\n",
      "135650K .......... .......... .......... .......... .......... 98%  188M 0s\n",
      "135700K .......... .......... .......... .......... .......... 98%  182M 0s\n",
      "135750K .......... .......... .......... .......... .......... 98%  212M 0s\n",
      "135800K .......... .......... .......... .......... .......... 98%  178M 0s\n",
      "135850K .......... .......... .......... .......... .......... 98%  151M 0s\n",
      "135900K .......... .......... .......... .......... .......... 98%  142M 0s\n",
      "135950K .......... .......... .......... .......... .......... 98%  209M 0s\n",
      "136000K .......... .......... .......... .......... .......... 98%  173M 0s\n",
      "136050K .......... .......... .......... .......... .......... 98%  187M 0s\n",
      "136100K .......... .......... .......... .......... .......... 98%  215M 0s\n",
      "136150K .......... .......... .......... .......... .......... 98%  189M 0s\n",
      "136200K .......... .......... .......... .......... .......... 98%  175M 0s\n",
      "136250K .......... .......... .......... .......... .......... 98%  186M 0s\n",
      "136300K .......... .......... .......... .......... .......... 98%  165M 0s\n",
      "136350K .......... .......... .......... .......... .......... 98%  143M 0s\n",
      "136400K .......... .......... .......... .......... .......... 98%  150M 0s\n",
      "136450K .......... .......... .......... .......... .......... 98%  200M 0s\n",
      "136500K .......... .......... .......... .......... .......... 98%  201M 0s\n",
      "136550K .......... .......... .......... .......... .......... 98%  197M 0s\n",
      "136600K .......... .......... .......... .......... .......... 98%  126M 0s\n",
      "136650K .......... .......... .......... .......... .......... 98%  199M 0s\n",
      "136700K .......... .......... .......... .......... .......... 98%  197M 0s\n",
      "136750K .......... .......... .......... .......... .......... 98%  173M 0s\n",
      "136800K .......... .......... .......... .......... .......... 98%  142M 0s\n",
      "136850K .......... .......... .......... .......... .......... 98%  207M 0s\n",
      "136900K .......... .......... .......... .......... .......... 98%  211M 0s\n",
      "136950K .......... .......... .......... .......... .......... 99%  202M 0s\n",
      "137000K .......... .......... .......... .......... .......... 99%  174M 0s\n",
      "137050K .......... .......... .......... .......... .......... 99%  214M 0s\n",
      "137100K .......... .......... .......... .......... .......... 99% 12.5M 0s\n",
      "137150K .......... .......... .......... .......... .......... 99%  180M 0s\n",
      "137200K .......... .......... .......... .......... .......... 99%  149M 0s\n",
      "137250K .......... .......... .......... .......... .......... 99%  210M 0s\n",
      "137300K .......... .......... .......... .......... .......... 99%  206M 0s\n",
      "137350K .......... .......... .......... .......... .......... 99% 23.6M 0s\n",
      "137400K .......... .......... .......... .......... .......... 99%  112M 0s\n",
      "137450K .......... .......... .......... .......... .......... 99%  168M 0s\n",
      "137500K .......... .......... .......... .......... .......... 99%  194M 0s\n",
      "137550K .......... .......... .......... .......... .......... 99%  193M 0s\n",
      "137600K .......... .......... .......... .......... .......... 99%  155M 0s\n",
      "137650K .......... .......... .......... .......... .......... 99%  172M 0s\n",
      "137700K .......... .......... .......... .......... .......... 99%  191M 0s\n",
      "137750K .......... .......... .......... .......... .......... 99%  193M 0s\n",
      "137800K .......... .......... .......... .......... .......... 99%  182M 0s\n",
      "137850K .......... .......... .......... .......... .......... 99%  178M 0s\n",
      "137900K .......... .......... .......... .......... .......... 99%  229M 0s\n",
      "137950K .......... .......... .......... .......... .......... 99%  189M 0s\n",
      "138000K .......... .......... .......... .......... .......... 99%  184M 0s\n",
      "138050K .......... .......... .......... .......... .......... 99%  138M 0s\n",
      "138100K .......... .......... .......... .......... .......... 99%  186M 0s\n",
      "138150K .......... .......... .......... .......... .......... 99%  208M 0s\n",
      "138200K .......... .......... .......... .......... .......... 99%  200M 0s\n",
      "138250K .......... .......... .......... .......... .......... 99% 8.84M 0s\n",
      "138300K .......... .......... .......... ..........           100%  152M=1.6s\n",
      "\n",
      "2021-02-02 15:48:22 (81.9 MB/s) - ‘annovar.latest.tar.gz’ saved [141660305/141660305]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "if test -e ~/bin/annovar; then\n",
    "\n",
    "echo \"annovar is already installed in ~/bin\"\n",
    "else\n",
    "echo \"annovar is not installed\"\n",
    "cd ~/bin/\n",
    "\n",
    "wget LINK_TO_ANNOVAR_DOWNLOAD.tar.gz\n",
    "\n",
    "tar xvfz annovar.latest.tar.gz\n",
    "\n",
    "fi"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download needed files:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGene.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneMrna.fa.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_refGeneVersion.txt.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://hgdownload.cse.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_ensGene.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_ensGeneMrna.fa.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_exac03.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_avsnp147.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_avsnp147.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_dbnsfp30a.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_dbnsfp30a.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_gnomad211_genome.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_gnomad211_genome.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_ljb26_all.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_ljb26_all.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n",
      "NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_clinvar_20190305.txt.gz ... OK\n",
      "NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_clinvar_20190305.txt.idx.gz ... OK\n",
      "NOTICE: Uncompressing downloaded files\n",
      "NOTICE: Finished downloading annotation files for hg38 build version, with files saved at the 'humandb' directory\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/bin/annovar/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar refGene humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb cytoBand humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar ensGene humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar exac03 humandb/ \n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar avsnp147 humandb/ \n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar dbnsfp30a humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar gnomad211_genome humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar ljb26_all humandb/\n",
    "perl annotate_variation.pl -buildver hg38 -downdb -webfrom annovar clinvar_20190305 humandb/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "annovar_downdb.log\n",
      "genometrax-sample-files-gff\n",
      "GRCh37_MT_ensGeneMrna.fa\n",
      "GRCh37_MT_ensGene.txt\n",
      "hg19_example_db_generic.txt\n",
      "hg19_example_db_gff3.txt\n",
      "hg19_MT_ensGeneMrna.fa\n",
      "hg19_MT_ensGene.txt\n",
      "hg19_refGeneMrna.fa\n",
      "hg19_refGene.txt\n",
      "hg19_refGeneVersion.txt\n",
      "hg19_refGeneWithVerMrna.fa\n",
      "hg19_refGeneWithVer.txt\n",
      "hg38_avsnp147.txt\n",
      "hg38_avsnp147.txt.idx\n",
      "hg38_clinvar_20190305.txt\n",
      "hg38_clinvar_20190305.txt.idx\n",
      "hg38_cytoBand.txt\n",
      "hg38_dbnsfp30a.txt\n",
      "hg38_dbnsfp30a.txt.idx\n",
      "hg38_ensGeneMrna.fa\n",
      "hg38_ensGene.txt\n",
      "hg38_exac03.txt\n",
      "hg38_exac03.txt.idx\n",
      "hg38_gnomad211_genome.txt\n",
      "hg38_gnomad211_genome.txt.idx\n",
      "hg38_ljb26_all.txt\n",
      "hg38_ljb26_all.txt.idx\n",
      "hg38_refGeneMrna.fa\n",
      "hg38_refGene.txt\n",
      "hg38_refGeneVersion.txt\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/bin/annovar/humandb/\n",
    "ls"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There is something that isn't correct in the input vcf I think since I get >1000 columns with \"Otherinfo\" when using the vcf file. I decided to change to format to a \"annovar formated file\" with the code below and everything appears to look fine\n",
    "\n",
    "#The following code generates a file with >1000 \"otherinfo\" columns, there is something wrong with the vcf (after bgzip and tabix), hence I converted it to the \"annovar format\" instead\n",
    "\n",
    "%%bash\n",
    "perl ~/bin/annovar/table_annovar.pl AMPPD_FILTERED.RIC3.vcf.gz ~/bin/annovar/humandb/ -buildver hg38 \\\n",
    "--thread 16 \\\n",
    "-out RIC3_WGS.annovar \\\n",
    "-remove -protocol refGene,ljb26_all,gnomad211_genome,clinvar_20190305 \\\n",
    "-operation g,f,f,f \\\n",
    "-nastring . \\\n",
    "-vcfinput"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Finished reading 780 lines from VCF file\n",
      "NOTICE: A total of 773 locus in VCF file passed QC threshold, representing 719 SNPs (477 transitions and 242 transversions) and 50 indels/substitutions\n",
      "NOTICE: Finished writing allele frequencies based on 1852144 SNP genotypes (1228752 transitions and 623392 transversions) and 128800 indels/substitutions for 2576 samples\n",
      "WARNING: 4 invalid alternative alleles found in input file\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "perl ~/bin/annovar/convert2annovar.pl -format vcf4 -allsample -withfreq AMPPD_FILTERED.RIC3.vcf.gz > AMPPD_FILTERED.RIC3_Annovar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: the --polish argument is set ON automatically (use --nopolish to change this behavior)\n",
      "-----------------------------------------------------------------\n",
      "NOTICE: Processing operation=g protocol=refGene\n",
      "\n",
      "NOTICE: Running with system command <annotate_variation.pl -geneanno -buildver hg38 -dbtype refGene -outfile RIC3_WGS.annovar.refGene -exonsort -nofirstcodondel AMPPD_FILTERED.RIC3_Annovar /home/jupyter-user/bin/annovar/humandb/ -thread 16>\n",
      "NOTICE: Output files are written to RIC3_WGS.annovar.refGene.variant_function, RIC3_WGS.annovar.refGene.exonic_variant_function\n",
      "NOTICE: the queryfile AMPPD_FILTERED.RIC3_Annovar contains 773 lines\n",
      "NOTICE: threading is disabled for gene-based annotation on file with less than 1000000 input lines\n",
      "NOTICE: Reading gene annotation from /home/jupyter-user/bin/annovar/humandb/hg38_refGene.txt ... Done with 82500 transcripts (including 20366 without coding sequence annotation) for 28265 unique genes\n",
      "NOTICE: Processing next batch with 773 unique variants in 773 input lines\n",
      "NOTICE: Reading FASTA sequences from /home/jupyter-user/bin/annovar/humandb/hg38_refGeneMrna.fa ... Done with 9 sequences\n",
      "WARNING: A total of 591 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "NOTICE: Running with system command <coding_change.pl  RIC3_WGS.annovar.refGene.exonic_variant_function.orig /home/jupyter-user/bin/annovar/humandb//hg38_refGene.txt /home/jupyter-user/bin/annovar/humandb//hg38_refGeneMrna.fa -alltranscript -out RIC3_WGS.annovar.refGene.fa -newevf RIC3_WGS.annovar.refGene.exonic_variant_function>\n",
      "-----------------------------------------------------------------\n",
      "NOTICE: Processing operation=f protocol=ljb26_all\n",
      "NOTICE: Finished reading 25 column headers for '-dbtype ljb26_all'\n",
      "\n",
      "NOTICE: Running system command <annotate_variation.pl -filter -dbtype ljb26_all -buildver hg38 -outfile RIC3_WGS.annovar AMPPD_FILTERED.RIC3_Annovar /home/jupyter-user/bin/annovar/humandb/ -thread 16 -otherinfo>\n",
      "NOTICE: the --dbtype ljb26_all is assumed to be in generic ANNOVAR database format\n",
      "NOTICE: Output file with variants matching filtering criteria is written to RIC3_WGS.annovar.hg38_ljb26_all_dropped, and output file with other variants is written to RIC3_WGS.annovar.hg38_ljb26_all_filtered\n",
      "NOTICE: the queryfile AMPPD_FILTERED.RIC3_Annovar contains 773 lines\n",
      "NOTICE: Creating new threads for query line 1 to 49\n",
      "NOTICE: Creating new threads for query line 50 to 98\n",
      "NOTICE: Creating new threads for query line 99 to 147\n",
      "NOTICE: Creating new threads for query line 148 to 196\n",
      "NOTICE: Creating new threads for query line 197 to 245\n",
      "NOTICE: Creating new threads for query line 246 to 294\n",
      "NOTICE: Creating new threads for query line 295 to 343\n",
      "NOTICE: Creating new threads for query line 344 to 392\n",
      "NOTICE: Creating new threads for query line 393 to 441\n",
      "NOTICE: Creating new threads for query line 442 to 490\n",
      "NOTICE: Creating new threads for query line 491 to 539\n",
      "NOTICE: Creating new threads for query line 540 to 588\n",
      "NOTICE: Creating new threads for query line 589 to 637\n",
      "NOTICE: Creating new threads for query line 638 to 686\n",
      "NOTICE: Creating new threads for query line 687 to 735\n",
      "NOTICE: Creating new threads for query line 736 to 773\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 4\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 1\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 2\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 1\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 38 unique variants in 38 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 2\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 4\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 555944 and the number of bins to be scanned is 3\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_ljb26_all.txt...Done\n",
      "-----------------------------------------------------------------\n",
      "NOTICE: Processing operation=f protocol=gnomad211_genome\n",
      "NOTICE: Finished reading 17 column headers for '-dbtype gnomad211_genome'\n",
      "\n",
      "NOTICE: Running system command <annotate_variation.pl -filter -dbtype gnomad211_genome -buildver hg38 -outfile RIC3_WGS.annovar AMPPD_FILTERED.RIC3_Annovar /home/jupyter-user/bin/annovar/humandb/ -thread 16 -otherinfo>\n",
      "NOTICE: Output file with variants matching filtering criteria is written to RIC3_WGS.annovar.hg38_gnomad211_genome_dropped, and output file with other variants is written to RIC3_WGS.annovar.hg38_gnomad211_genome_filtered\n",
      "NOTICE: the queryfile AMPPD_FILTERED.RIC3_Annovar contains 773 lines\n",
      "NOTICE: Creating new threads for query line 1 to 49\n",
      "NOTICE: Creating new threads for query line 50 to 98\n",
      "NOTICE: Creating new threads for query line 99 to 147\n",
      "NOTICE: Creating new threads for query line 148 to 196\n",
      "NOTICE: Creating new threads for query line 197 to 245\n",
      "NOTICE: Creating new threads for query line 246 to 294\n",
      "NOTICE: Creating new threads for query line 295 to 343\n",
      "NOTICE: Creating new threads for query line 344 to 392\n",
      "NOTICE: Creating new threads for query line 393 to 441\n",
      "NOTICE: Creating new threads for query line 442 to 490\n",
      "NOTICE: Creating new threads for query line 491 to 539\n",
      "NOTICE: Creating new threads for query line 540 to 588\n",
      "NOTICE: Creating new threads for query line 589 to 637\n",
      "NOTICE: Creating new threads for query line 638 to 686\n",
      "NOTICE: Creating new threads for query line 687 to 735\n",
      "NOTICE: Creating new threads for query line 736 to 773\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 27\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 30\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 38 unique variants in 38 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 26\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 30\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 29\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 26\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 30\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 27\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 31\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 29\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 31\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 23\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 33\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 33\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 25\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 28076758 and the number of bins to be scanned is 28\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_gnomad211_genome.txt...Done\n",
      "-----------------------------------------------------------------\n",
      "NOTICE: Processing operation=f protocol=clinvar_20190305\n",
      "NOTICE: Finished reading 5 column headers for '-dbtype clinvar_20190305'\n",
      "\n",
      "NOTICE: Running system command <annotate_variation.pl -filter -dbtype clinvar_20190305 -buildver hg38 -outfile RIC3_WGS.annovar AMPPD_FILTERED.RIC3_Annovar /home/jupyter-user/bin/annovar/humandb/ -thread 16 -otherinfo>\n",
      "NOTICE: the --dbtype clinvar_20190305 is assumed to be in generic ANNOVAR database format\n",
      "NOTICE: Output file with variants matching filtering criteria is written to RIC3_WGS.annovar.hg38_clinvar_20190305_dropped, and output file with other variants is written to RIC3_WGS.annovar.hg38_clinvar_20190305_filtered\n",
      "NOTICE: the queryfile AMPPD_FILTERED.RIC3_Annovar contains 773 lines\n",
      "NOTICE: Creating new threads for query line 1 to 49\n",
      "NOTICE: Creating new threads for query line 50 to 98\n",
      "NOTICE: Creating new threads for query line 99 to 147\n",
      "NOTICE: Creating new threads for query line 148 to 196\n",
      "NOTICE: Creating new threads for query line 197 to 245\n",
      "NOTICE: Creating new threads for query line 246 to 294\n",
      "NOTICE: Creating new threads for query line 295 to 343\n",
      "NOTICE: Creating new threads for query line 344 to 392\n",
      "NOTICE: Creating new threads for query line 393 to 441\n",
      "NOTICE: Creating new threads for query line 442 to 490\n",
      "NOTICE: Creating new threads for query line 491 to 539\n",
      "NOTICE: Creating new threads for query line 540 to 588\n",
      "NOTICE: Creating new threads for query line 589 to 637\n",
      "NOTICE: Creating new threads for query line 638 to 686\n",
      "NOTICE: Creating new threads for query line 687 to 735\n",
      "NOTICE: Creating new threads for query line 736 to 773\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 1\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 49 unique variants in 49 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "NOTICE: Processing next batch with 38 unique variants in 38 input lines\n",
      "NOTICE: Database index loaded. Total number of bins is 46038 and the number of bins to be scanned is 0\n",
      "NOTICE: Scanning filter database /home/jupyter-user/bin/annovar/humandb/hg38_clinvar_20190305.txt...Done\n",
      "-----------------------------------------------------------------\n",
      "NOTICE: Multianno output file is written to RIC3_WGS.annovar.hg38_multianno.txt\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "perl ~/bin/annovar/table_annovar.pl AMPPD_FILTERED.RIC3_Annovar ~/bin/annovar/humandb/ -buildver hg38 \\\n",
    "--thread 16 \\\n",
    "-out RIC3_WGS.annovar \\\n",
    "-remove -protocol refGene,ljb26_all,gnomad211_genome,clinvar_20190305 \\\n",
    "-operation g,f,f,f \\\n",
    "-nastring ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "head -1 RIC3_WGS.annovar.hg38_multianno.txt > header.txt\n",
    "colct=\"$(wc -w header.txt| cut -f1 -d' ')\"\n",
    "cut -f1-$colct RIC3_WGS.annovar.hg38_multianno.txt > RIC3.WGS.trimmed.annotation_KB.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://./RIC3.WGS.trimmed.annotation_KB.txt [Content-Type=text/plain]...\n",
      "/ [1 files][166.5 KiB/166.5 KiB]                                                \n",
      "Operation completed over 1 objects/166.5 KiB.                                    \n"
     ]
    }
   ],
   "source": [
    "#download the annotation file\n",
    "!gsutil cp ./RIC3.WGS.trimmed.annotation_KB.txt gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 4) Burden analysis\n",
    "##### Generating list of coding variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "awk '$6==\"exonic\" {print}' RIC3.WGS.trimmed.annotation_KB.txt > RIC3.WGS.trimmed.annotation_KB.coding.variants.txt\n",
    "awk '{print $1\" \"$2\" \"$2\" \"$7}' RIC3.WGS.trimmed.annotation_KB.coding.variants.txt > RIC3.WGS.trimmed.annotation_KB.SNPs.txt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b6.9 64-bit (4 Mar 2019)            www.cog-genomics.org/plink/1.9/\r\n",
      "(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3\r\n",
      "Logging to RIC3_CODING_KB.log.\r\n",
      "Options in effect:\r\n",
      "  --bfile AMPPD_FILTERED.RIC3\r\n",
      "  --extract range RIC3.WGS.trimmed.annotation_KB.coding.variants.txt\r\n",
      "  --out RIC3_CODING_KB\r\n",
      "  --recode vcf-fid\r\n",
      "\r\n",
      "60402 MB RAM detected; reserving 30201 MB for main workspace.\r\n",
      "773 variants loaded from .bim file.\r\n",
      "2576 people (1455 males, 1121 females) loaded from .fam.\r\n",
      "2576 phenotype values loaded from .fam.\r\n",
      "--extract range: 754 variants excluded.\r\n",
      "--extract range: 19 variants remaining.\r\n",
      "Using 1 thread (no multithreaded calculations invoked).\r\n",
      "Before main variant filters, 2576 founders and 0 nonfounders present.\r\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\r\n",
      "19 variants and 2576 people pass filters and QC.\r\n",
      "Among remaining phenotypes, 1615 are cases and 961 are controls.\r\n",
      "--recode vcf-fid to RIC3_CODING_KB.vcf ... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\r\n"
     ]
    }
   ],
   "source": [
    "!~/bin/plink --bfile AMPPD_FILTERED.RIC3 --extract range RIC3.WGS.trimmed.annotation_KB.coding.variants.txt --recode 'vcf-fid'  --out RIC3_CODING_KB"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "MAF < 0.03\n",
    "#### ALL VARIANTS "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##Upload needed file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Copying gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/refFlat.txt...\n",
      "/ [0 files][    0.0 B/ 21.2 MiB]                                                \r",
      "/ [1 files][ 21.2 MiB/ 21.2 MiB]                                                \r\n",
      "Operation completed over 1 objects/21.2 MiB.                                     \n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "gsutil cp gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf/refFlat.txt /home/jupyter-user/notebooks/ipdgctraineeproject-ric3/edit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Thank you for using rvtests (version: 20190205, git: c86e589efef15382603300dc7f4c3394c82d69b8)\n",
      "  For documentations, refer to http://zhanxw.github.io/rvtests/\n",
      "  For questions and comments, plase send to Xiaowei Zhan <zhanxw@umich.edu>\n",
      "  For bugs and feature requests, please submit at: https://github.com/zhanxw/rvtests/issues\n",
      "\n",
      "RVTESTS finished successfully\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The following parameters are available.  Ones with \"[]\" are in effect:\n",
      "\n",
      "Available Options\n",
      "      Basic Input/Output: --inVcf [AMPPD_FILTERED.RIC3.vcf.gz], --inBgen []\n",
      "                          --inBgenSample [], --inKgg []\n",
      "                          --out [AMP_PD_BURDEN.RIC3.WGS.maf003_KB], --outputRaw\n",
      "       Specify Covariate: --covar [covariateFile.txt], --covar-name [SEX,AGE,PC1\n",
      "                         PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10], --sex\n",
      "       Specify Phenotype: --pheno [covariateFile.txt], --inverseNormal\n",
      "                          --useResidualAsPhenotype, --mpheno []\n",
      "                          --pheno-name [PHENO], --qtl, --multiplePheno []\n",
      "        Specify Genotype: --dosage [], --multipleAllele\n",
      "    Chromosome X Options: --xLabel [], --xParRegion []\n",
      "           People Filter: --peopleIncludeID [], --peopleIncludeFile []\n",
      "                          --peopleExcludeID [], --peopleExcludeFile []\n",
      "             Site Filter: --rangeList [], --rangeFile [], --siteFile []\n",
      "                          --siteDepthMin [], --siteDepthMax [], --siteMACMin []\n",
      "                          --annoType []\n",
      "         Genotype Filter: --indvDepthMin [], --indvDepthMax [], --indvQualMin []\n",
      "       Association Model: --single [], --burden [cmc,zeggini,mb,fp,cmcWald]\n",
      "                          --vt [], --kernel [skat,skato], --meta []\n",
      "     Family-based Models: --kinship [], --xHemiKinship [], --kinshipEigen []\n",
      "                          --xHemiKinshipEigen [], --boltPlink []\n",
      "                          --boltPlinkNoCheck\n",
      "          Grouping Unit : --geneFile [refFlat.txt], --gene [], --setList []\n",
      "                          --setFile [], --set []\n",
      "        Frequency Cutoff: --freqUpper [0.03], --freqLower []\n",
      "            Missing Data: --impute [], --imputePheno, --imputeCov\n",
      "    Conditional Analysis: --condition []\n",
      "     Auxiliary Functions: --noweb [true], --hide-covar, --numThread []\n",
      "                          --outputID, --help\n",
      "\n",
      "Effective Options\n",
      "    --inVcf AMPPD_FILTERED.RIC3.vcf.gz\n",
      "    --out AMP_PD_BURDEN.RIC3.WGS.maf003_KB\n",
      "    --covar covariateFile.txt\n",
      "    --covar-name SEX,AGE,PC1,PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10\n",
      "    --pheno covariateFile.txt\n",
      "    --pheno-name PHENO\n",
      "    --burden cmc,zeggini,mb,fp,cmcWald\n",
      "    --kernel skat,skato\n",
      "    --geneFile refFlat.txt\n",
      "    --freqUpper 0.03\n",
      "    --noweb\n",
      "\n",
      "[INFO]\tProgram version: 20190205\n",
      "[INFO]\tAnalysis started at: Fri Feb  5 14:49:59 2021\n",
      "[INFO]\tLoaded [ 2576 ] samples from genotype files\n",
      "[INFO]\tLoaded [ 2576 ] sample phenotypes\n",
      "[INFO]\tBegin to read covariate file\n",
      "[INFO]\tLoaded 1455 male, 1121 female and 0 sex-unknown samples from covariateFile.txt\n",
      "[INFO]\tLoaded 1615 cases, 961 controls, and 0 missing phenotypes\n",
      "[WARN]\t-- Enabling binary phenotype mode -- \n",
      "[INFO]\tAnalysis begins with [ 2576 ] samples...\n",
      "[INFO]\tMadsonBrowning test significance will be evaluated using 10000 permutations at alpha = 0.05\n",
      "[INFO]\tSKAT test significance will be evaluated using 10000 permutations at alpha = 0.05 weight = Beta[beta1 = 1.00, beta2 = 25.00]\n",
      "[INFO]\tSKAT-O test significance will be evaluated using weight = Beta[beta1 = 1.00, beta2 = 25.00]\n",
      "[INFO]\tLoaded [ 28307 ] genes.\n",
      "[INFO]\tImpute missing genotype to mean (by default)\n",
      "[INFO]\tSet upper minor allele frequency limit to 0.03\n",
      "[INFO]\tAnalysis started\n",
      "[INFO]\tGene ZNF639 has 0 variants, skipping\n",
      "[INFO]\tGene WASH7P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-11 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138A has 0 variants, skipping\n",
      "[INFO]\tGene FAM138F has 0 variants, skipping\n",
      "[INFO]\tGene FAM138C has 0 variants, skipping\n",
      "[INFO]\tGene LOC729737 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138D has 0 variants, skipping\n",
      "[INFO]\tGene PTX3 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRZ1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132287 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928626 has 0 variants, skipping\n",
      "[INFO]\tGene FAM87B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00115 has 0 variants, skipping\n",
      "[INFO]\tGene MADD has 0 variants, skipping\n",
      "[INFO]\tGene FAM41C has 0 variants, skipping\n",
      "[INFO]\tGene LINC02593 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB9 has 0 variants, skipping\n",
      "[INFO]\tGene GNL3 has 0 variants, skipping\n",
      "[INFO]\tGene MESD has 0 variants, skipping\n",
      "[INFO]\tGene PHF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR512-1 has 0 variants, skipping\n",
      "[INFO]\tGene TSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH12 has 0 variants, skipping\n",
      "[INFO]\tGene WDSUB1 has 0 variants, skipping\n",
      "[INFO]\tGene ADPGK has 0 variants, skipping\n",
      "[INFO]\tGene THOC5 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNG has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288175 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01342 has 0 variants, skipping\n",
      "[INFO]\tGene PHF1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM18 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD has 0 variants, skipping\n",
      "[INFO]\tGene PXK has 0 variants, skipping\n",
      "[INFO]\tGene NT5C2 has 0 variants, skipping\n",
      "[INFO]\tGene SERF1B has 0 variants, skipping\n",
      "[INFO]\tGene ASCC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF24 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A5P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6726 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD167 has 0 variants, skipping\n",
      "[INFO]\tGene ECSIT has 0 variants, skipping\n",
      "[INFO]\tGene PUSL1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM243 has 0 variants, skipping\n",
      "[INFO]\tGene NUP85 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6727 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6808 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC6 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF13 has 0 variants, skipping\n",
      "[INFO]\tGene EED has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF14 has 0 variants, skipping\n",
      "[INFO]\tGene URI1 has 0 variants, skipping\n",
      "[INFO]\tGene HRK has 0 variants, skipping\n",
      "[INFO]\tGene SNX3 has 0 variants, skipping\n",
      "[INFO]\tGene CRADD has 0 variants, skipping\n",
      "[INFO]\tGene SNX4 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK1 has 0 variants, skipping\n",
      "[INFO]\tGene STK31 has 0 variants, skipping\n",
      "[INFO]\tGene PANX2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01770 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H11A has 0 variants, skipping\n",
      "[INFO]\tGene CEP78 has 0 variants, skipping\n",
      "[INFO]\tGene PCM1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL12 has 0 variants, skipping\n",
      "[INFO]\tGene AP1G2 has 0 variants, skipping\n",
      "[INFO]\tGene SMC2 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378586 has 0 variants, skipping\n",
      "[INFO]\tGene PAX6 has 0 variants, skipping\n",
      "[INFO]\tGene ARFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL2 has 0 variants, skipping\n",
      "[INFO]\tGene HPS4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM91A1 has 0 variants, skipping\n",
      "[INFO]\tGene KANK2 has 0 variants, skipping\n",
      "[INFO]\tGene RBPJL has 0 variants, skipping\n",
      "[INFO]\tGene MMP23A has 0 variants, skipping\n",
      "[INFO]\tGene SPNS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B4 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2M has 0 variants, skipping\n",
      "[INFO]\tGene UFM1 has 0 variants, skipping\n",
      "[INFO]\tGene NME8 has 0 variants, skipping\n",
      "[INFO]\tGene STAT3 has 0 variants, skipping\n",
      "[INFO]\tGene PSPH has 0 variants, skipping\n",
      "[INFO]\tGene DPCD has 0 variants, skipping\n",
      "[INFO]\tGene DYDC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTCFL has 0 variants, skipping\n",
      "[INFO]\tGene GABRB3 has 0 variants, skipping\n",
      "[INFO]\tGene FDFT1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM9 has 0 variants, skipping\n",
      "[INFO]\tGene DDX55 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL4 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL2 has 0 variants, skipping\n",
      "[INFO]\tGene DDAH2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPI has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D1 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf35 has 0 variants, skipping\n",
      "[INFO]\tGene ERC2 has 0 variants, skipping\n",
      "[INFO]\tGene TANGO2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2G has 0 variants, skipping\n",
      "[INFO]\tGene RNFT1 has 0 variants, skipping\n",
      "[INFO]\tGene RHD has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378591 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC7 has 0 variants, skipping\n",
      "[INFO]\tGene GANAB has 0 variants, skipping\n",
      "[INFO]\tGene TMC5 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP91 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF260 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCZ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPH3AL has 0 variants, skipping\n",
      "[INFO]\tGene FAAP20 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA3 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC18 has 0 variants, skipping\n",
      "[INFO]\tGene PARP15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129534 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO48 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf68 has 0 variants, skipping\n",
      "[INFO]\tGene MED29 has 0 variants, skipping\n",
      "[INFO]\tGene BRAF has 0 variants, skipping\n",
      "[INFO]\tGene DMXL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R3 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SNX13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996583 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD12B has 0 variants, skipping\n",
      "[INFO]\tGene DCTN2 has 0 variants, skipping\n",
      "[INFO]\tGene IPO8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM104B has 0 variants, skipping\n",
      "[INFO]\tGene C11orf91 has 0 variants, skipping\n",
      "[INFO]\tGene TTK has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3E has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEE1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM16-DT has 0 variants, skipping\n",
      "[INFO]\tGene ODF2L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4251 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD10 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN3 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA4 has 0 variants, skipping\n",
      "[INFO]\tGene RGS7 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF1 has 0 variants, skipping\n",
      "[INFO]\tGene SGTA has 0 variants, skipping\n",
      "[INFO]\tGene RGR has 0 variants, skipping\n",
      "[INFO]\tGene COPB1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM8B has 0 variants, skipping\n",
      "[INFO]\tGene PLEK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF226 has 0 variants, skipping\n",
      "[INFO]\tGene CELF4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3CL has 0 variants, skipping\n",
      "[INFO]\tGene CAMKK2 has 0 variants, skipping\n",
      "[INFO]\tGene GOSR2 has 0 variants, skipping\n",
      "[INFO]\tGene AURKB has 0 variants, skipping\n",
      "[INFO]\tGene PIGT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01134 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01777 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01646 has 0 variants, skipping\n",
      "[INFO]\tGene DAZ3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4689 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2B has 0 variants, skipping\n",
      "[INFO]\tGene TGOLN2 has 0 variants, skipping\n",
      "[INFO]\tGene GSG1 has 0 variants, skipping\n",
      "[INFO]\tGene SUN1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPA1 has 0 variants, skipping\n",
      "[INFO]\tGene EMSY has 0 variants, skipping\n",
      "[INFO]\tGene TMEM62 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LPGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WWTR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CELF1 has 0 variants, skipping\n",
      "[INFO]\tGene RFX3 has 0 variants, skipping\n",
      "[INFO]\tGene WDR49 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4252 has 0 variants, skipping\n",
      "[INFO]\tGene BARX2 has 0 variants, skipping\n",
      "[INFO]\tGene ASL has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AA1 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNCRIP has 0 variants, skipping\n",
      "[INFO]\tGene SH3YL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC5 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4 has 0 variants, skipping\n",
      "[INFO]\tGene RGMA has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK1 has 0 variants, skipping\n",
      "[INFO]\tGene PI15 has 0 variants, skipping\n",
      "[INFO]\tGene PCF11 has 0 variants, skipping\n",
      "[INFO]\tGene MBTPS2 has 0 variants, skipping\n",
      "[INFO]\tGene DKK3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RPN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4306 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3196 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3194 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPG11 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB4B has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2A has 0 variants, skipping\n",
      "[INFO]\tGene TUBB4A has 0 variants, skipping\n",
      "[INFO]\tGene SPRR1A has 0 variants, skipping\n",
      "[INFO]\tGene ERMARD has 0 variants, skipping\n",
      "[INFO]\tGene ALG13 has 0 variants, skipping\n",
      "[INFO]\tGene DLC1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF24 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH3 has 0 variants, skipping\n",
      "[INFO]\tGene SGSM1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF12 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4 has 0 variants, skipping\n",
      "[INFO]\tGene GAD2 has 0 variants, skipping\n",
      "[INFO]\tGene VOPP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA7 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31C1 has 0 variants, skipping\n",
      "[INFO]\tGene USHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLMAP has 0 variants, skipping\n",
      "[INFO]\tGene SIDT1 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7C has 0 variants, skipping\n",
      "[INFO]\tGene ASB8 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF391 has 0 variants, skipping\n",
      "[INFO]\tGene NTPCR has 0 variants, skipping\n",
      "[INFO]\tGene IQCD has 0 variants, skipping\n",
      "[INFO]\tGene ETS1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1AR has 0 variants, skipping\n",
      "[INFO]\tGene SAMD4B has 0 variants, skipping\n",
      "[INFO]\tGene GULP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM205 has 0 variants, skipping\n",
      "[INFO]\tGene PAX5 has 0 variants, skipping\n",
      "[INFO]\tGene MCTS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLG3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC148 has 0 variants, skipping\n",
      "[INFO]\tGene RERE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD128 has 0 variants, skipping\n",
      "[INFO]\tGene CLPSL2 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD20 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GDS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6728 has 0 variants, skipping\n",
      "[INFO]\tGene ENO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM248 has 0 variants, skipping\n",
      "[INFO]\tGene GSPT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR74 has 0 variants, skipping\n",
      "[INFO]\tGene MEAF6 has 0 variants, skipping\n",
      "[INFO]\tGene ATG13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR34AHG has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02606 has 0 variants, skipping\n",
      "[INFO]\tGene SYK has 0 variants, skipping\n",
      "[INFO]\tGene KNOP1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP92 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CD1E has 0 variants, skipping\n",
      "[INFO]\tGene NADK has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1B-RDH14 has 0 variants, skipping\n",
      "[INFO]\tGene KRTDAP has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD8 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR2 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM114A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5697 has 0 variants, skipping\n",
      "[INFO]\tGene NSD2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG3 has 0 variants, skipping\n",
      "[INFO]\tGene DEDD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF10 has 0 variants, skipping\n",
      "[INFO]\tGene FCAR has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-9 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-7 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-8 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP3 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC7 has 0 variants, skipping\n",
      "[INFO]\tGene USP33 has 0 variants, skipping\n",
      "[INFO]\tGene NTNG1 has 0 variants, skipping\n",
      "[INFO]\tGene LMTK2 has 0 variants, skipping\n",
      "[INFO]\tGene PRLH has 0 variants, skipping\n",
      "[INFO]\tGene UBR5 has 0 variants, skipping\n",
      "[INFO]\tGene ABCD2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL13 has 0 variants, skipping\n",
      "[INFO]\tGene APLP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOC has 0 variants, skipping\n",
      "[INFO]\tGene CCNJ has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A2 has 0 variants, skipping\n",
      "[INFO]\tGene RABAC1 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP3 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene DHX16 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2D has 0 variants, skipping\n",
      "[INFO]\tGene MTOR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B4 has 0 variants, skipping\n",
      "[INFO]\tGene VRK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01647 has 0 variants, skipping\n",
      "[INFO]\tGene FANCC has 0 variants, skipping\n",
      "[INFO]\tGene RGS10 has 0 variants, skipping\n",
      "[INFO]\tGene ITSN1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4F has 0 variants, skipping\n",
      "[INFO]\tGene FBXO44 has 0 variants, skipping\n",
      "[INFO]\tGene VIPR2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D14 has 0 variants, skipping\n",
      "[INFO]\tGene NUFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene HACE1 has 0 variants, skipping\n",
      "[INFO]\tGene CGN has 0 variants, skipping\n",
      "[INFO]\tGene GHR has 0 variants, skipping\n",
      "[INFO]\tGene MBNL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN6 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP44 has 0 variants, skipping\n",
      "[INFO]\tGene NPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP2 has 0 variants, skipping\n",
      "[INFO]\tGene FER has 0 variants, skipping\n",
      "[INFO]\tGene RNU5E-1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PF has 0 variants, skipping\n",
      "[INFO]\tGene AP3M1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP63 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GAP has 0 variants, skipping\n",
      "[INFO]\tGene MIR6729 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ISY1-RAB43 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7846 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC163 has 0 variants, skipping\n",
      "[INFO]\tGene CSAG1 has 0 variants, skipping\n",
      "[INFO]\tGene MLC1 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR2 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2A has 0 variants, skipping\n",
      "[INFO]\tGene USP17L28 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf109 has 0 variants, skipping\n",
      "[INFO]\tGene PRG2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA59B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA59A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927572 has 0 variants, skipping\n",
      "[INFO]\tGene SNX19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6730 has 0 variants, skipping\n",
      "[INFO]\tGene FAM120B has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01784 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB8 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB9 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM234 has 0 variants, skipping\n",
      "[INFO]\tGene LARS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX30 has 0 variants, skipping\n",
      "[INFO]\tGene FURIN has 0 variants, skipping\n",
      "[INFO]\tGene LZTS2 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN10 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF36P has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF34P has 0 variants, skipping\n",
      "[INFO]\tGene COLGALT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8 has 0 variants, skipping\n",
      "[INFO]\tGene RNF8 has 0 variants, skipping\n",
      "[INFO]\tGene CAPS has 0 variants, skipping\n",
      "[INFO]\tGene BRSK2 has 0 variants, skipping\n",
      "[INFO]\tGene KAZN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1D has 0 variants, skipping\n",
      "[INFO]\tGene DLG2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM51-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB4 has 0 variants, skipping\n",
      "[INFO]\tGene PAK3 has 0 variants, skipping\n",
      "[INFO]\tGene PAK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR142 has 0 variants, skipping\n",
      "[INFO]\tGene MIR141 has 0 variants, skipping\n",
      "[INFO]\tGene MIR140 has 0 variants, skipping\n",
      "[INFO]\tGene MIR138-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR137 has 0 variants, skipping\n",
      "[INFO]\tGene MIR135A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR135A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR130A has 0 variants, skipping\n",
      "[INFO]\tGene MIR128-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR323B has 0 variants, skipping\n",
      "[INFO]\tGene TGS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA21B has 0 variants, skipping\n",
      "[INFO]\tGene ROR2 has 0 variants, skipping\n",
      "[INFO]\tGene NTHL1 has 0 variants, skipping\n",
      "[INFO]\tGene VTI1A has 0 variants, skipping\n",
      "[INFO]\tGene CCNA1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170 has 0 variants, skipping\n",
      "[INFO]\tGene NFIC has 0 variants, skipping\n",
      "[INFO]\tGene OSCAR has 0 variants, skipping\n",
      "[INFO]\tGene FLJ37453 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF559 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN3 has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4L2 has 0 variants, skipping\n",
      "[INFO]\tGene APTX has 0 variants, skipping\n",
      "[INFO]\tGene ACSL6 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2C has 0 variants, skipping\n",
      "[INFO]\tGene C4BPB has 0 variants, skipping\n",
      "[INFO]\tGene SORBS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1B has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C8 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY6 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH4 has 0 variants, skipping\n",
      "[INFO]\tGene KEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene TLK1 has 0 variants, skipping\n",
      "[INFO]\tGene MPDZ has 0 variants, skipping\n",
      "[INFO]\tGene SPATA21 has 0 variants, skipping\n",
      "[INFO]\tGene BARD1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC182 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf135 has 0 variants, skipping\n",
      "[INFO]\tGene KIF27 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K1 has 0 variants, skipping\n",
      "[INFO]\tGene FYB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01772 has 0 variants, skipping\n",
      "[INFO]\tGene CROCCP3 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-18 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-4 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01783 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3675 has 0 variants, skipping\n",
      "[INFO]\tGene ST18 has 0 variants, skipping\n",
      "[INFO]\tGene CROCCP2 has 0 variants, skipping\n",
      "[INFO]\tGene MST1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ESPNP has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376805 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A11 has 0 variants, skipping\n",
      "[INFO]\tGene TAOK3 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM108 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B1 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOSIN has 0 variants, skipping\n",
      "[INFO]\tGene OXNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP3 has 0 variants, skipping\n",
      "[INFO]\tGene GSN has 0 variants, skipping\n",
      "[INFO]\tGene KLHL10 has 0 variants, skipping\n",
      "[INFO]\tGene F2R has 0 variants, skipping\n",
      "[INFO]\tGene GGCX has 0 variants, skipping\n",
      "[INFO]\tGene VKORC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD73B has 0 variants, skipping\n",
      "[INFO]\tGene GPM6A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3972 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA8 has 0 variants, skipping\n",
      "[INFO]\tGene XKRY has 0 variants, skipping\n",
      "[INFO]\tGene WRNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD2B has 0 variants, skipping\n",
      "[INFO]\tGene KDM4C has 0 variants, skipping\n",
      "[INFO]\tGene CYB561D2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D20 has 0 variants, skipping\n",
      "[INFO]\tGene MADD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUSB has 0 variants, skipping\n",
      "[INFO]\tGene CHN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM14A has 0 variants, skipping\n",
      "[INFO]\tGene MX1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A4 has 0 variants, skipping\n",
      "[INFO]\tGene DTNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01654 has 0 variants, skipping\n",
      "[INFO]\tGene PDP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIP has 0 variants, skipping\n",
      "[INFO]\tGene KLC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIK1B has 0 variants, skipping\n",
      "[INFO]\tGene LPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX19B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4695 has 0 variants, skipping\n",
      "[INFO]\tGene CES5A has 0 variants, skipping\n",
      "[INFO]\tGene TWIST2 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT3 has 0 variants, skipping\n",
      "[INFO]\tGene KLK8 has 0 variants, skipping\n",
      "[INFO]\tGene EMC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP4 has 0 variants, skipping\n",
      "[INFO]\tGene NPRL3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2F has 0 variants, skipping\n",
      "[INFO]\tGene FAM131B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP65 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7L has 0 variants, skipping\n",
      "[INFO]\tGene DCAF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506730 has 0 variants, skipping\n",
      "[INFO]\tGene PPIC has 0 variants, skipping\n",
      "[INFO]\tGene PPIB has 0 variants, skipping\n",
      "[INFO]\tGene PDE3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-4 has 0 variants, skipping\n",
      "[INFO]\tGene FZD4-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02740 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRCD has 0 variants, skipping\n",
      "[INFO]\tGene CTDP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378614 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF40B has 0 variants, skipping\n",
      "[INFO]\tGene EBF4 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPS14P3 has 0 variants, skipping\n",
      "[INFO]\tGene TJP1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA1 has 0 variants, skipping\n",
      "[INFO]\tGene PSD3 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD8 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL2 has 0 variants, skipping\n",
      "[INFO]\tGene TWF2 has 0 variants, skipping\n",
      "[INFO]\tGene MGLL has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01757 has 0 variants, skipping\n",
      "[INFO]\tGene DOHH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01141 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF40A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6084 has 0 variants, skipping\n",
      "[INFO]\tGene PINK1-AS has 0 variants, skipping\n",
      "[INFO]\tGene GABBR1 has 0 variants, skipping\n",
      "[INFO]\tGene LRBA has 0 variants, skipping\n",
      "[INFO]\tGene PCID2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LF has 0 variants, skipping\n",
      "[INFO]\tGene C18orf54 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN6 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA5 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3A has 0 variants, skipping\n",
      "[INFO]\tGene HTR4 has 0 variants, skipping\n",
      "[INFO]\tGene HTR2C has 0 variants, skipping\n",
      "[INFO]\tGene HTR2B has 0 variants, skipping\n",
      "[INFO]\tGene HTR1F has 0 variants, skipping\n",
      "[INFO]\tGene HTR1E has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A3 has 0 variants, skipping\n",
      "[INFO]\tGene STPG2 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF575 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK4 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L3 has 0 variants, skipping\n",
      "[INFO]\tGene TBXA2R has 0 variants, skipping\n",
      "[INFO]\tGene LMO3 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RB has 0 variants, skipping\n",
      "[INFO]\tGene TNXB has 0 variants, skipping\n",
      "[INFO]\tGene STK40 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF17 has 0 variants, skipping\n",
      "[INFO]\tGene PROM2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LG has 0 variants, skipping\n",
      "[INFO]\tGene CELF3 has 0 variants, skipping\n",
      "[INFO]\tGene EDN1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI27 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOP has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3B2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM162B has 0 variants, skipping\n",
      "[INFO]\tGene C10orf120 has 0 variants, skipping\n",
      "[INFO]\tGene MUC21 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10B has 0 variants, skipping\n",
      "[INFO]\tGene NME1-NME2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC53 has 0 variants, skipping\n",
      "[INFO]\tGene PDXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene GAPVD1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2C_2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A6 has 0 variants, skipping\n",
      "[INFO]\tGene RALGPS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNPK has 0 variants, skipping\n",
      "[INFO]\tGene CCDC196 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC1L has 0 variants, skipping\n",
      "[INFO]\tGene CDC14B has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R10 has 0 variants, skipping\n",
      "[INFO]\tGene SYT14 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC2 has 0 variants, skipping\n",
      "[INFO]\tGene NME9 has 0 variants, skipping\n",
      "[INFO]\tGene SYCE1L has 0 variants, skipping\n",
      "[INFO]\tGene JCAD has 0 variants, skipping\n",
      "[INFO]\tGene GSG1L has 0 variants, skipping\n",
      "[INFO]\tGene GALT has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01635 has 0 variants, skipping\n",
      "[INFO]\tGene ASNS has 0 variants, skipping\n",
      "[INFO]\tGene HP1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene FIP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4418 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF133 has 0 variants, skipping\n",
      "[INFO]\tGene CPD has 0 variants, skipping\n",
      "[INFO]\tGene OSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6127 has 0 variants, skipping\n",
      "[INFO]\tGene ALG9 has 0 variants, skipping\n",
      "[INFO]\tGene NAA60 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC11 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4684 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4253 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf58 has 0 variants, skipping\n",
      "[INFO]\tGene INVS has 0 variants, skipping\n",
      "[INFO]\tGene TP53I11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3115 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71F1 has 0 variants, skipping\n",
      "[INFO]\tGene RABL2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF436-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP2 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL2 has 0 variants, skipping\n",
      "[INFO]\tGene NFIX has 0 variants, skipping\n",
      "[INFO]\tGene KLF7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928163 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD2 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACYP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX3Y has 0 variants, skipping\n",
      "[INFO]\tGene SNCAIP has 0 variants, skipping\n",
      "[INFO]\tGene ZFP69 has 0 variants, skipping\n",
      "[INFO]\tGene GBP3 has 0 variants, skipping\n",
      "[INFO]\tGene GLYR1 has 0 variants, skipping\n",
      "[INFO]\tGene EBF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378F has 0 variants, skipping\n",
      "[INFO]\tGene AK2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF738 has 0 variants, skipping\n",
      "[INFO]\tGene STATH has 0 variants, skipping\n",
      "[INFO]\tGene PRKACB has 0 variants, skipping\n",
      "[INFO]\tGene MAP2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGXB3 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHKG1 has 0 variants, skipping\n",
      "[INFO]\tGene KNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280C has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC10 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD10 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280D has 0 variants, skipping\n",
      "[INFO]\tGene EOLA1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH23 has 0 variants, skipping\n",
      "[INFO]\tGene HCK has 0 variants, skipping\n",
      "[INFO]\tGene SPATS2L has 0 variants, skipping\n",
      "[INFO]\tGene RCAN3AS has 0 variants, skipping\n",
      "[INFO]\tGene STIM1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T5 has 0 variants, skipping\n",
      "[INFO]\tGene CD276 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02774 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC19 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3GL has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMKK1 has 0 variants, skipping\n",
      "[INFO]\tGene HAGHL has 0 variants, skipping\n",
      "[INFO]\tGene GARNL3 has 0 variants, skipping\n",
      "[INFO]\tGene HSDL2 has 0 variants, skipping\n",
      "[INFO]\tGene GON4L has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP12 has 0 variants, skipping\n",
      "[INFO]\tGene IL18 has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC2 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5F has 0 variants, skipping\n",
      "[INFO]\tGene SH3GLB1 has 0 variants, skipping\n",
      "[INFO]\tGene KCND1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIOBP has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A3 has 0 variants, skipping\n",
      "[INFO]\tGene HSF2BP has 0 variants, skipping\n",
      "[INFO]\tGene TPPP has 0 variants, skipping\n",
      "[INFO]\tGene SKIV2L has 0 variants, skipping\n",
      "[INFO]\tGene RIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF254 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A2 has 0 variants, skipping\n",
      "[INFO]\tGene POLDIP3 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD3 has 0 variants, skipping\n",
      "[INFO]\tGene RANGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB48 has 0 variants, skipping\n",
      "[INFO]\tGene EDIL3 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP1 has 0 variants, skipping\n",
      "[INFO]\tGene AHSG has 0 variants, skipping\n",
      "[INFO]\tGene DAZAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL19 has 0 variants, skipping\n",
      "[INFO]\tGene DHCR24 has 0 variants, skipping\n",
      "[INFO]\tGene IST1 has 0 variants, skipping\n",
      "[INFO]\tGene TATDN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A46 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL3 has 0 variants, skipping\n",
      "[INFO]\tGene UTY has 0 variants, skipping\n",
      "[INFO]\tGene ADAMDEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125A has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1A has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00173 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD54 has 0 variants, skipping\n",
      "[INFO]\tGene MIR17HG has 0 variants, skipping\n",
      "[INFO]\tGene AKAP17A has 0 variants, skipping\n",
      "[INFO]\tGene CD8A has 0 variants, skipping\n",
      "[INFO]\tGene TATDN1 has 0 variants, skipping\n",
      "[INFO]\tGene BACH1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF6 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC20A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6731 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4425 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRB has 0 variants, skipping\n",
      "[INFO]\tGene NUP54 has 0 variants, skipping\n",
      "[INFO]\tGene NKIRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF707 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61G-DT has 0 variants, skipping\n",
      "[INFO]\tGene OLFM3 has 0 variants, skipping\n",
      "[INFO]\tGene TLX3 has 0 variants, skipping\n",
      "[INFO]\tGene CACHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB26 has 0 variants, skipping\n",
      "[INFO]\tGene ZDBF2 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR3 has 0 variants, skipping\n",
      "[INFO]\tGene WNK3 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39A has 0 variants, skipping\n",
      "[INFO]\tGene WDR38 has 0 variants, skipping\n",
      "[INFO]\tGene USF1 has 0 variants, skipping\n",
      "[INFO]\tGene TPCN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646471 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf90 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2C has 0 variants, skipping\n",
      "[INFO]\tGene RNASE4 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HC1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SRI has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3917 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB6 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5D has 0 variants, skipping\n",
      "[INFO]\tGene HPCAL4 has 0 variants, skipping\n",
      "[INFO]\tGene NRBF2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC33A1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A7 has 0 variants, skipping\n",
      "[INFO]\tGene ETF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692A1 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK6 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL5 has 0 variants, skipping\n",
      "[INFO]\tGene IQCA1 has 0 variants, skipping\n",
      "[INFO]\tGene IL20RA has 0 variants, skipping\n",
      "[INFO]\tGene PALD1 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEB has 0 variants, skipping\n",
      "[INFO]\tGene LRFN2 has 0 variants, skipping\n",
      "[INFO]\tGene RIMKLB has 0 variants, skipping\n",
      "[INFO]\tGene HEG1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB6 has 0 variants, skipping\n",
      "[INFO]\tGene NUMB has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC17 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150C has 0 variants, skipping\n",
      "[INFO]\tGene LIN54 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928728 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf94 has 0 variants, skipping\n",
      "[INFO]\tGene DGKB has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNPH1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1586 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF14 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4C has 0 variants, skipping\n",
      "[INFO]\tGene FGGY has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8K has 0 variants, skipping\n",
      "[INFO]\tGene AGER has 0 variants, skipping\n",
      "[INFO]\tGene ARL3 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P20 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM262 has 0 variants, skipping\n",
      "[INFO]\tGene PIF1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene RXRA has 0 variants, skipping\n",
      "[INFO]\tGene ORC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANO10 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1L has 0 variants, skipping\n",
      "[INFO]\tGene CARD19 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TF has 0 variants, skipping\n",
      "[INFO]\tGene LINC02574 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF140 has 0 variants, skipping\n",
      "[INFO]\tGene AAMDC has 0 variants, skipping\n",
      "[INFO]\tGene GPALPP1 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PDHX has 0 variants, skipping\n",
      "[INFO]\tGene TLL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA1 has 0 variants, skipping\n",
      "[INFO]\tGene GATAD2A has 0 variants, skipping\n",
      "[INFO]\tGene OAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NTM has 0 variants, skipping\n",
      "[INFO]\tGene LILRA1 has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC5OS has 0 variants, skipping\n",
      "[INFO]\tGene DICER1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC20 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC51 has 0 variants, skipping\n",
      "[INFO]\tGene USP15 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18C has 0 variants, skipping\n",
      "[INFO]\tGene CETN3 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0513 has 0 variants, skipping\n",
      "[INFO]\tGene COX4I1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIE has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC8 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf112 has 0 variants, skipping\n",
      "[INFO]\tGene CBLB has 0 variants, skipping\n",
      "[INFO]\tGene RAD51B has 0 variants, skipping\n",
      "[INFO]\tGene KCNS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF5 has 0 variants, skipping\n",
      "[INFO]\tGene DPP3 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf52 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1328 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA73A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA73B has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MF-PTCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRNAU1AP has 0 variants, skipping\n",
      "[INFO]\tGene OR4F6 has 0 variants, skipping\n",
      "[INFO]\tGene DLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1468 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP7P has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA61 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA16A has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL6 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL4 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01715 has 0 variants, skipping\n",
      "[INFO]\tGene TMC6 has 0 variants, skipping\n",
      "[INFO]\tGene LAYN has 0 variants, skipping\n",
      "[INFO]\tGene PTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene KAT6A has 0 variants, skipping\n",
      "[INFO]\tGene DEGS1 has 0 variants, skipping\n",
      "[INFO]\tGene TOGARAM2 has 0 variants, skipping\n",
      "[INFO]\tGene COL3A1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP26 has 0 variants, skipping\n",
      "[INFO]\tGene SNCB has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM163A has 0 variants, skipping\n",
      "[INFO]\tGene FBXW11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP23 has 0 variants, skipping\n",
      "[INFO]\tGene SOX5 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL1 has 0 variants, skipping\n",
      "[INFO]\tGene CUL4B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01756 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01648 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01778 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K7CL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385A has 0 variants, skipping\n",
      "[INFO]\tGene SLCO6A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103C has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE16 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3K has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF9 has 0 variants, skipping\n",
      "[INFO]\tGene ENOX1 has 0 variants, skipping\n",
      "[INFO]\tGene AR has 0 variants, skipping\n",
      "[INFO]\tGene CXCR4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3D has 0 variants, skipping\n",
      "[INFO]\tGene EIF3C has 0 variants, skipping\n",
      "[INFO]\tGene KATNA1 has 0 variants, skipping\n",
      "[INFO]\tGene KRR1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX12 has 0 variants, skipping\n",
      "[INFO]\tGene SOX15 has 0 variants, skipping\n",
      "[INFO]\tGene SOX10 has 0 variants, skipping\n",
      "[INFO]\tGene ATE1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R5 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM160B2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01226 has 0 variants, skipping\n",
      "[INFO]\tGene RPP21 has 0 variants, skipping\n",
      "[INFO]\tGene MCM7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS11 has 0 variants, skipping\n",
      "[INFO]\tGene WDR97 has 0 variants, skipping\n",
      "[INFO]\tGene PDPR has 0 variants, skipping\n",
      "[INFO]\tGene SDHC has 0 variants, skipping\n",
      "[INFO]\tGene SCIN has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4254 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT22 has 0 variants, skipping\n",
      "[INFO]\tGene HABP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A6 has 0 variants, skipping\n",
      "[INFO]\tGene NRM has 0 variants, skipping\n",
      "[INFO]\tGene MIR5585 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP1 has 0 variants, skipping\n",
      "[INFO]\tGene KNG1 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM2 has 0 variants, skipping\n",
      "[INFO]\tGene COX11 has 0 variants, skipping\n",
      "[INFO]\tGene TAMM41 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL8 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR9LP has 0 variants, skipping\n",
      "[INFO]\tGene C4A has 0 variants, skipping\n",
      "[INFO]\tGene USP44 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DOB has 0 variants, skipping\n",
      "[INFO]\tGene ZYX has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK1 has 0 variants, skipping\n",
      "[INFO]\tGene ST7L has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4F has 0 variants, skipping\n",
      "[INFO]\tGene PHF7 has 0 variants, skipping\n",
      "[INFO]\tGene RCVRN has 0 variants, skipping\n",
      "[INFO]\tGene SDHD has 0 variants, skipping\n",
      "[INFO]\tGene RCN2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS23 has 0 variants, skipping\n",
      "[INFO]\tGene CHST11 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH13 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEC has 0 variants, skipping\n",
      "[INFO]\tGene RAB5IF has 0 variants, skipping\n",
      "[INFO]\tGene CLIP3 has 0 variants, skipping\n",
      "[INFO]\tGene BVES has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8OS has 0 variants, skipping\n",
      "[INFO]\tGene MPG has 0 variants, skipping\n",
      "[INFO]\tGene ACOT9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCZ has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK11 has 0 variants, skipping\n",
      "[INFO]\tGene S100PBP has 0 variants, skipping\n",
      "[INFO]\tGene MAFA has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6L2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A3 has 0 variants, skipping\n",
      "[INFO]\tGene CAPS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL24A1 has 0 variants, skipping\n",
      "[INFO]\tGene VEZT has 0 variants, skipping\n",
      "[INFO]\tGene CP has 0 variants, skipping\n",
      "[INFO]\tGene NUP93 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP3B has 0 variants, skipping\n",
      "[INFO]\tGene NRCAM has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1 has 0 variants, skipping\n",
      "[INFO]\tGene EVL has 0 variants, skipping\n",
      "[INFO]\tGene PRODH has 0 variants, skipping\n",
      "[INFO]\tGene NAT9 has 0 variants, skipping\n",
      "[INFO]\tGene XYLB has 0 variants, skipping\n",
      "[INFO]\tGene TMEM70 has 0 variants, skipping\n",
      "[INFO]\tGene SCARB1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH1 has 0 variants, skipping\n",
      "[INFO]\tGene DCX has 0 variants, skipping\n",
      "[INFO]\tGene VIT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3605 has 0 variants, skipping\n",
      "[INFO]\tGene MED23 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26C has 0 variants, skipping\n",
      "[INFO]\tGene MT1X has 0 variants, skipping\n",
      "[INFO]\tGene APBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MT1H has 0 variants, skipping\n",
      "[INFO]\tGene MT1F has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC5 has 0 variants, skipping\n",
      "[INFO]\tGene MT1G has 0 variants, skipping\n",
      "[INFO]\tGene UBE4B has 0 variants, skipping\n",
      "[INFO]\tGene CSMD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L8 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF8 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOU has 0 variants, skipping\n",
      "[INFO]\tGene CYRIB has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653160 has 0 variants, skipping\n",
      "[INFO]\tGene HFE has 0 variants, skipping\n",
      "[INFO]\tGene PAM has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC3 has 0 variants, skipping\n",
      "[INFO]\tGene PHKB has 0 variants, skipping\n",
      "[INFO]\tGene COL9A3 has 0 variants, skipping\n",
      "[INFO]\tGene COL9A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN20 has 0 variants, skipping\n",
      "[INFO]\tGene HAGH has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLC has 0 variants, skipping\n",
      "[INFO]\tGene FTO has 0 variants, skipping\n",
      "[INFO]\tGene IRF8 has 0 variants, skipping\n",
      "[INFO]\tGene CD53 has 0 variants, skipping\n",
      "[INFO]\tGene AQP9 has 0 variants, skipping\n",
      "[INFO]\tGene BIN1 has 0 variants, skipping\n",
      "[INFO]\tGene STRBP has 0 variants, skipping\n",
      "[INFO]\tGene DISC2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A8 has 0 variants, skipping\n",
      "[INFO]\tGene EPS15 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF12 has 0 variants, skipping\n",
      "[INFO]\tGene MSRA has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A4 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF7 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L3 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM126B has 0 variants, skipping\n",
      "[INFO]\tGene IKZF1 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE2 has 0 variants, skipping\n",
      "[INFO]\tGene GDAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY10 has 0 variants, skipping\n",
      "[INFO]\tGene APP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63C has 0 variants, skipping\n",
      "[INFO]\tGene SZT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC21 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4255 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01137 has 0 variants, skipping\n",
      "[INFO]\tGene MINK1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene SDCCAG8 has 0 variants, skipping\n",
      "[INFO]\tGene TLE1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6732 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB109B has 0 variants, skipping\n",
      "[INFO]\tGene CAST has 0 variants, skipping\n",
      "[INFO]\tGene PWWP3B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF736 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL10 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN5 has 0 variants, skipping\n",
      "[INFO]\tGene VMP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCALD has 0 variants, skipping\n",
      "[INFO]\tGene FLNC has 0 variants, skipping\n",
      "[INFO]\tGene CXorf56 has 0 variants, skipping\n",
      "[INFO]\tGene DNMBP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF346 has 0 variants, skipping\n",
      "[INFO]\tGene DUOX2 has 0 variants, skipping\n",
      "[INFO]\tGene ANO4 has 0 variants, skipping\n",
      "[INFO]\tGene CSF3 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH16A1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1A has 0 variants, skipping\n",
      "[INFO]\tGene TTC26 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAIR1 has 0 variants, skipping\n",
      "[INFO]\tGene THPO has 0 variants, skipping\n",
      "[INFO]\tGene ETNK1 has 0 variants, skipping\n",
      "[INFO]\tGene PARL has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A10 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA4 has 0 variants, skipping\n",
      "[INFO]\tGene CSGALNACT2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2N1 has 0 variants, skipping\n",
      "[INFO]\tGene STRADB has 0 variants, skipping\n",
      "[INFO]\tGene SERINC2 has 0 variants, skipping\n",
      "[INFO]\tGene USP49 has 0 variants, skipping\n",
      "[INFO]\tGene CYYR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL4 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PAK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01343 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01685 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2BP-COX16 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP14 has 0 variants, skipping\n",
      "[INFO]\tGene GJA9-MYCBP has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378663 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC9 has 0 variants, skipping\n",
      "[INFO]\tGene SEC31A has 0 variants, skipping\n",
      "[INFO]\tGene SLU7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM120C has 0 variants, skipping\n",
      "[INFO]\tGene BEST1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR59 has 0 variants, skipping\n",
      "[INFO]\tGene ZCWPW2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT2P1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIEL has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA55 has 0 variants, skipping\n",
      "[INFO]\tGene IDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ID3 has 0 variants, skipping\n",
      "[INFO]\tGene ABI2 has 0 variants, skipping\n",
      "[INFO]\tGene LST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF197 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929536 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX1 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC9 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC115995503 has 0 variants, skipping\n",
      "[INFO]\tGene TRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS3 has 0 variants, skipping\n",
      "[INFO]\tGene FREM1 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PET100 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111B has 0 variants, skipping\n",
      "[INFO]\tGene TBPL2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN6 has 0 variants, skipping\n",
      "[INFO]\tGene TEPP has 0 variants, skipping\n",
      "[INFO]\tGene NSUN4 has 0 variants, skipping\n",
      "[INFO]\tGene OR7C1 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1B has 0 variants, skipping\n",
      "[INFO]\tGene SS18L1 has 0 variants, skipping\n",
      "[INFO]\tGene PYHIN1 has 0 variants, skipping\n",
      "[INFO]\tGene INKA2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4B has 0 variants, skipping\n",
      "[INFO]\tGene KRT72 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2026 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF7 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA3 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP18 has 0 variants, skipping\n",
      "[INFO]\tGene SRR has 0 variants, skipping\n",
      "[INFO]\tGene SENP6 has 0 variants, skipping\n",
      "[INFO]\tGene FLCN has 0 variants, skipping\n",
      "[INFO]\tGene TRIM16L has 0 variants, skipping\n",
      "[INFO]\tGene DYM has 0 variants, skipping\n",
      "[INFO]\tGene NFYC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A12 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1671 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A6 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0895 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF761 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF91 has 0 variants, skipping\n",
      "[INFO]\tGene PIGF has 0 variants, skipping\n",
      "[INFO]\tGene PIGC has 0 variants, skipping\n",
      "[INFO]\tGene PIGA has 0 variants, skipping\n",
      "[INFO]\tGene TSTD1 has 0 variants, skipping\n",
      "[INFO]\tGene USP54 has 0 variants, skipping\n",
      "[INFO]\tGene DACT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLFNL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SCMH1 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf48 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG6 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE9 has 0 variants, skipping\n",
      "[INFO]\tGene C1QC has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD6 has 0 variants, skipping\n",
      "[INFO]\tGene JADE1 has 0 variants, skipping\n",
      "[INFO]\tGene DGUOK has 0 variants, skipping\n",
      "[INFO]\tGene PUS7L has 0 variants, skipping\n",
      "[INFO]\tGene KDM5C has 0 variants, skipping\n",
      "[INFO]\tGene PSME3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC11 has 0 variants, skipping\n",
      "[INFO]\tGene SPART has 0 variants, skipping\n",
      "[INFO]\tGene ZEB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC13 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH5 has 0 variants, skipping\n",
      "[INFO]\tGene KLF11 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPA has 0 variants, skipping\n",
      "[INFO]\tGene CBX4 has 0 variants, skipping\n",
      "[INFO]\tGene CHST1 has 0 variants, skipping\n",
      "[INFO]\tGene AP5S1 has 0 variants, skipping\n",
      "[INFO]\tGene KRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA10 has 0 variants, skipping\n",
      "[INFO]\tGene MRGBP has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3L has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1B has 0 variants, skipping\n",
      "[INFO]\tGene ANO5 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB6 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP30 has 0 variants, skipping\n",
      "[INFO]\tGene IL36B has 0 variants, skipping\n",
      "[INFO]\tGene SLC50A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339539 has 0 variants, skipping\n",
      "[INFO]\tGene MTG2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK33 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6733 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2A has 0 variants, skipping\n",
      "[INFO]\tGene CFAP410 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R2 has 0 variants, skipping\n",
      "[INFO]\tGene PITX1 has 0 variants, skipping\n",
      "[INFO]\tGene PIP has 0 variants, skipping\n",
      "[INFO]\tGene PI4KB has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CG has 0 variants, skipping\n",
      "[INFO]\tGene PIM1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C3 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2B has 0 variants, skipping\n",
      "[INFO]\tGene CCL3L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6734 has 0 variants, skipping\n",
      "[INFO]\tGene ERO1A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF14 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1A1 has 0 variants, skipping\n",
      "[INFO]\tGene ITM2C has 0 variants, skipping\n",
      "[INFO]\tGene COMMD5 has 0 variants, skipping\n",
      "[INFO]\tGene SZT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6735 has 0 variants, skipping\n",
      "[INFO]\tGene RPL14 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D2A has 0 variants, skipping\n",
      "[INFO]\tGene CENPQ has 0 variants, skipping\n",
      "[INFO]\tGene CEP55 has 0 variants, skipping\n",
      "[INFO]\tGene SHQ1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPO has 0 variants, skipping\n",
      "[INFO]\tGene TSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAC2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM33 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF10L has 0 variants, skipping\n",
      "[INFO]\tGene RFWD3 has 0 variants, skipping\n",
      "[INFO]\tGene DARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLF2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3E has 0 variants, skipping\n",
      "[INFO]\tGene KDM4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM44 has 0 variants, skipping\n",
      "[INFO]\tGene ODF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF213 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM3 has 0 variants, skipping\n",
      "[INFO]\tGene CTSC has 0 variants, skipping\n",
      "[INFO]\tGene CENPE has 0 variants, skipping\n",
      "[INFO]\tGene LKAAEAR1 has 0 variants, skipping\n",
      "[INFO]\tGene FSD2 has 0 variants, skipping\n",
      "[INFO]\tGene SCP2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2 has 0 variants, skipping\n",
      "[INFO]\tGene SALL2 has 0 variants, skipping\n",
      "[INFO]\tGene HHEX has 0 variants, skipping\n",
      "[INFO]\tGene PRKACA has 0 variants, skipping\n",
      "[INFO]\tGene DSC1 has 0 variants, skipping\n",
      "[INFO]\tGene PGS1 has 0 variants, skipping\n",
      "[INFO]\tGene GRM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A10 has 0 variants, skipping\n",
      "[INFO]\tGene RHNO1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR9 has 0 variants, skipping\n",
      "[INFO]\tGene ANK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6079 has 0 variants, skipping\n",
      "[INFO]\tGene SNX30 has 0 variants, skipping\n",
      "[INFO]\tGene CEP83 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL2 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC9 has 0 variants, skipping\n",
      "[INFO]\tGene QRICH1 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF112 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL2 has 0 variants, skipping\n",
      "[INFO]\tGene RNFT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX22 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC2C has 0 variants, skipping\n",
      "[INFO]\tGene PUM1 has 0 variants, skipping\n",
      "[INFO]\tGene CROCC has 0 variants, skipping\n",
      "[INFO]\tGene ERI3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA110 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF4L has 0 variants, skipping\n",
      "[INFO]\tGene RAD51D has 0 variants, skipping\n",
      "[INFO]\tGene RAB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5584 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD145 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5F-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5D-1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC6 has 0 variants, skipping\n",
      "[INFO]\tGene TMED7-TICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L10 has 0 variants, skipping\n",
      "[INFO]\tGene USP19 has 0 variants, skipping\n",
      "[INFO]\tGene MLST8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD160 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD55 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38B has 0 variants, skipping\n",
      "[INFO]\tGene KAT6B has 0 variants, skipping\n",
      "[INFO]\tGene RPA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM107 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E3 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD5 has 0 variants, skipping\n",
      "[INFO]\tGene DHX8 has 0 variants, skipping\n",
      "[INFO]\tGene MFGE8 has 0 variants, skipping\n",
      "[INFO]\tGene SDR16C5 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB4 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01144 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7 has 0 variants, skipping\n",
      "[INFO]\tGene RASAL1 has 0 variants, skipping\n",
      "[INFO]\tGene SHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR542 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1224 has 0 variants, skipping\n",
      "[INFO]\tGene MIR421 has 0 variants, skipping\n",
      "[INFO]\tGene MIR671 has 0 variants, skipping\n",
      "[INFO]\tGene MIR658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR657 has 0 variants, skipping\n",
      "[INFO]\tGene MIR411 has 0 variants, skipping\n",
      "[INFO]\tGene MIR654 has 0 variants, skipping\n",
      "[INFO]\tGene MIR653 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449B has 0 variants, skipping\n",
      "[INFO]\tGene MIR663A has 0 variants, skipping\n",
      "[INFO]\tGene MIR548D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR661 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR647 has 0 variants, skipping\n",
      "[INFO]\tGene MIR646 has 0 variants, skipping\n",
      "[INFO]\tGene LEF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR637 has 0 variants, skipping\n",
      "[INFO]\tGene MIR634 has 0 variants, skipping\n",
      "[INFO]\tGene MIR632 has 0 variants, skipping\n",
      "[INFO]\tGene MIR33B has 0 variants, skipping\n",
      "[INFO]\tGene MIR630 has 0 variants, skipping\n",
      "[INFO]\tGene MIR627 has 0 variants, skipping\n",
      "[INFO]\tGene MIR626 has 0 variants, skipping\n",
      "[INFO]\tGene MIR625 has 0 variants, skipping\n",
      "[INFO]\tGene MIR624 has 0 variants, skipping\n",
      "[INFO]\tGene MIR619 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548C has 0 variants, skipping\n",
      "[INFO]\tGene MIR616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR612 has 0 variants, skipping\n",
      "[INFO]\tGene MIR610 has 0 variants, skipping\n",
      "[INFO]\tGene MIR605 has 0 variants, skipping\n",
      "[INFO]\tGene MIR602 has 0 variants, skipping\n",
      "[INFO]\tGene MIR601 has 0 variants, skipping\n",
      "[INFO]\tGene MIR598 has 0 variants, skipping\n",
      "[INFO]\tGene LARGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6839 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6836 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6775 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB3 has 0 variants, skipping\n",
      "[INFO]\tGene TPM3 has 0 variants, skipping\n",
      "[INFO]\tGene MDFIC2 has 0 variants, skipping\n",
      "[INFO]\tGene SPEM3 has 0 variants, skipping\n",
      "[INFO]\tGene CMTR2 has 0 variants, skipping\n",
      "[INFO]\tGene LHX2 has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A6 has 0 variants, skipping\n",
      "[INFO]\tGene UBE4A has 0 variants, skipping\n",
      "[INFO]\tGene SLIT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15AP10 has 0 variants, skipping\n",
      "[INFO]\tGene SSC5D has 0 variants, skipping\n",
      "[INFO]\tGene MKLN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPDPFL has 0 variants, skipping\n",
      "[INFO]\tGene PHF14 has 0 variants, skipping\n",
      "[INFO]\tGene MYO6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF429 has 0 variants, skipping\n",
      "[INFO]\tGene MTSS1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929626 has 0 variants, skipping\n",
      "[INFO]\tGene AEBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD6 has 0 variants, skipping\n",
      "[INFO]\tGene SERHL2 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG5 has 0 variants, skipping\n",
      "[INFO]\tGene KYNU has 0 variants, skipping\n",
      "[INFO]\tGene SGO1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3D has 0 variants, skipping\n",
      "[INFO]\tGene AIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01398 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCLAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02719 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02760 has 0 variants, skipping\n",
      "[INFO]\tGene LDAH has 0 variants, skipping\n",
      "[INFO]\tGene MSTO1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2-SMG1P6 has 0 variants, skipping\n",
      "[INFO]\tGene DERL1 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCM has 0 variants, skipping\n",
      "[INFO]\tGene CCNC has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2B has 0 variants, skipping\n",
      "[INFO]\tGene SCML4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM249 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP has 0 variants, skipping\n",
      "[INFO]\tGene DTX2 has 0 variants, skipping\n",
      "[INFO]\tGene GNL1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG5 has 0 variants, skipping\n",
      "[INFO]\tGene GNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00853 has 0 variants, skipping\n",
      "[INFO]\tGene FUOM has 0 variants, skipping\n",
      "[INFO]\tGene HLA-F has 0 variants, skipping\n",
      "[INFO]\tGene CCDC77 has 0 variants, skipping\n",
      "[INFO]\tGene STARD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF573 has 0 variants, skipping\n",
      "[INFO]\tGene GGA1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MF has 0 variants, skipping\n",
      "[INFO]\tGene RREB1 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02265 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01205 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01389 has 0 variants, skipping\n",
      "[INFO]\tGene SKINT1L has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG1 has 0 variants, skipping\n",
      "[INFO]\tGene IHO1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf71 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6L has 0 variants, skipping\n",
      "[INFO]\tGene FAM135A has 0 variants, skipping\n",
      "[INFO]\tGene CFLAR has 0 variants, skipping\n",
      "[INFO]\tGene TIA1 has 0 variants, skipping\n",
      "[INFO]\tGene HCN3 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0C has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B1 has 0 variants, skipping\n",
      "[INFO]\tGene GIGYF1 has 0 variants, skipping\n",
      "[INFO]\tGene CSE1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF518B has 0 variants, skipping\n",
      "[INFO]\tGene CLASP2 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6500 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01562 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3IP has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM18 has 0 variants, skipping\n",
      "[INFO]\tGene ANKAR has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8G has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL3 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP5 has 0 variants, skipping\n",
      "[INFO]\tGene RFC3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC5 has 0 variants, skipping\n",
      "[INFO]\tGene FARSB has 0 variants, skipping\n",
      "[INFO]\tGene ABHD10 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18A has 0 variants, skipping\n",
      "[INFO]\tGene FSD1L has 0 variants, skipping\n",
      "[INFO]\tGene SGMS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM6 has 0 variants, skipping\n",
      "[INFO]\tGene TATDN3 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCR10 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1A has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LMBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTROB has 0 variants, skipping\n",
      "[INFO]\tGene HOXA11-AS has 0 variants, skipping\n",
      "[INFO]\tGene CYP4Z2P has 0 variants, skipping\n",
      "[INFO]\tGene MFSD11 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene P3H1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP1 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD1C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF567 has 0 variants, skipping\n",
      "[INFO]\tGene NOSIP has 0 variants, skipping\n",
      "[INFO]\tGene MYL5 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR2 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD6-FXYD2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7B has 0 variants, skipping\n",
      "[INFO]\tGene PID1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB1 has 0 variants, skipping\n",
      "[INFO]\tGene GIT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD1 has 0 variants, skipping\n",
      "[INFO]\tGene VDAC2 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD2 has 0 variants, skipping\n",
      "[INFO]\tGene MRE11 has 0 variants, skipping\n",
      "[INFO]\tGene TPM1 has 0 variants, skipping\n",
      "[INFO]\tGene AICDA has 0 variants, skipping\n",
      "[INFO]\tGene MGRN1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS1A has 0 variants, skipping\n",
      "[INFO]\tGene FLAD1 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIB1 has 0 variants, skipping\n",
      "[INFO]\tGene MCTP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND11 has 0 variants, skipping\n",
      "[INFO]\tGene GJB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAT has 0 variants, skipping\n",
      "[INFO]\tGene SHISA2 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF9B has 0 variants, skipping\n",
      "[INFO]\tGene ETV2 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DS1 has 0 variants, skipping\n",
      "[INFO]\tGene NACA has 0 variants, skipping\n",
      "[INFO]\tGene LIF has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9 has 0 variants, skipping\n",
      "[INFO]\tGene LCP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIG4 has 0 variants, skipping\n",
      "[INFO]\tGene LIG3 has 0 variants, skipping\n",
      "[INFO]\tGene LCT has 0 variants, skipping\n",
      "[INFO]\tGene LIFR has 0 variants, skipping\n",
      "[INFO]\tGene FANCI has 0 variants, skipping\n",
      "[INFO]\tGene CCSER2 has 0 variants, skipping\n",
      "[INFO]\tGene SARAF has 0 variants, skipping\n",
      "[INFO]\tGene LPP has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507564 has 0 variants, skipping\n",
      "[INFO]\tGene BEX2 has 0 variants, skipping\n",
      "[INFO]\tGene IFI16 has 0 variants, skipping\n",
      "[INFO]\tGene PODN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01771 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A3P1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK17 has 0 variants, skipping\n",
      "[INFO]\tGene PDE8A has 0 variants, skipping\n",
      "[INFO]\tGene MS4A13 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-6 has 0 variants, skipping\n",
      "[INFO]\tGene NKD2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2F1 has 0 variants, skipping\n",
      "[INFO]\tGene RILPL1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1B has 0 variants, skipping\n",
      "[INFO]\tGene PIMREG has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA10 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA5 has 0 variants, skipping\n",
      "[INFO]\tGene NPTN-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF248 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM138 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL37 has 0 variants, skipping\n",
      "[INFO]\tGene PBDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C3A has 0 variants, skipping\n",
      "[INFO]\tGene PPHLN1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRES1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4781 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC191 has 0 variants, skipping\n",
      "[INFO]\tGene RNF7 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNC1 has 0 variants, skipping\n",
      "[INFO]\tGene GGCT has 0 variants, skipping\n",
      "[INFO]\tGene PTPN7 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S1 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL2 has 0 variants, skipping\n",
      "[INFO]\tGene TPMT has 0 variants, skipping\n",
      "[INFO]\tGene DDIT3 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN1 has 0 variants, skipping\n",
      "[INFO]\tGene MROH7 has 0 variants, skipping\n",
      "[INFO]\tGene MROH7-TTC4 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY2 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L3 has 0 variants, skipping\n",
      "[INFO]\tGene TK2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMCC1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT3 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507634 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4422HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4422 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01753 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01755 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01767 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929935 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L5 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L29 has 0 variants, skipping\n",
      "[INFO]\tGene PFKP has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERE has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A12 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB5 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM94 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL43 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01135 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01358 has 0 variants, skipping\n",
      "[INFO]\tGene HSD52 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5B has 0 variants, skipping\n",
      "[INFO]\tGene SBK3 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L4 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD14 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI2-DT has 0 variants, skipping\n",
      "[INFO]\tGene TMEM139-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RCHY1 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01614 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf136 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3H has 0 variants, skipping\n",
      "[INFO]\tGene MIR4711 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926944 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01748 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926964 has 0 variants, skipping\n",
      "[INFO]\tGene NFIA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATC1L has 0 variants, skipping\n",
      "[INFO]\tGene MGC34796 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF20B has 0 variants, skipping\n",
      "[INFO]\tGene HSPB11 has 0 variants, skipping\n",
      "[INFO]\tGene SUGP2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB3 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT2 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ANG has 0 variants, skipping\n",
      "[INFO]\tGene AMFR has 0 variants, skipping\n",
      "[INFO]\tGene MIR3116-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3116-2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP5 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS12 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1D has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K3 has 0 variants, skipping\n",
      "[INFO]\tGene RGS5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A7 has 0 variants, skipping\n",
      "[INFO]\tGene GALR3 has 0 variants, skipping\n",
      "[INFO]\tGene CILP has 0 variants, skipping\n",
      "[INFO]\tGene HGS has 0 variants, skipping\n",
      "[INFO]\tGene STXBP6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM193A has 0 variants, skipping\n",
      "[INFO]\tGene TTLL6 has 0 variants, skipping\n",
      "[INFO]\tGene KHSRP has 0 variants, skipping\n",
      "[INFO]\tGene BCAS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01739 has 0 variants, skipping\n",
      "[INFO]\tGene FAM238C has 0 variants, skipping\n",
      "[INFO]\tGene TRMT11 has 0 variants, skipping\n",
      "[INFO]\tGene TMED2-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR6068 has 0 variants, skipping\n",
      "[INFO]\tGene HOATZ has 0 variants, skipping\n",
      "[INFO]\tGene PTX4 has 0 variants, skipping\n",
      "[INFO]\tGene RPGRIP1L has 0 variants, skipping\n",
      "[INFO]\tGene ESR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR591 has 0 variants, skipping\n",
      "[INFO]\tGene MIR589 has 0 variants, skipping\n",
      "[INFO]\tGene MIR587 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU2L has 0 variants, skipping\n",
      "[INFO]\tGene DOCK8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5196 has 0 variants, skipping\n",
      "[INFO]\tGene TAPBPL has 0 variants, skipping\n",
      "[INFO]\tGene MIB2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL25 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2U has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8R has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4794 has 0 variants, skipping\n",
      "[INFO]\tGene HYDIN has 0 variants, skipping\n",
      "[INFO]\tGene ERAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GNA12 has 0 variants, skipping\n",
      "[INFO]\tGene CDC27 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF11 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01359 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3671 has 0 variants, skipping\n",
      "[INFO]\tGene COLGALT1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM37 has 0 variants, skipping\n",
      "[INFO]\tGene CADPS2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP36 has 0 variants, skipping\n",
      "[INFO]\tGene RCC1 has 0 variants, skipping\n",
      "[INFO]\tGene DPF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GCOM1 has 0 variants, skipping\n",
      "[INFO]\tGene NFKB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3117 has 0 variants, skipping\n",
      "[INFO]\tGene PTK2 has 0 variants, skipping\n",
      "[INFO]\tGene LARP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289561 has 0 variants, skipping\n",
      "[INFO]\tGene HAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGED4 has 0 variants, skipping\n",
      "[INFO]\tGene TTC29 has 0 variants, skipping\n",
      "[INFO]\tGene FZD6 has 0 variants, skipping\n",
      "[INFO]\tGene SLAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM252 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene MON2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC197 has 0 variants, skipping\n",
      "[INFO]\tGene COA1 has 0 variants, skipping\n",
      "[INFO]\tGene TIAM1 has 0 variants, skipping\n",
      "[INFO]\tGene STK32A has 0 variants, skipping\n",
      "[INFO]\tGene RANBP3L has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G1 has 0 variants, skipping\n",
      "[INFO]\tGene ASB7 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H11B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB12 has 0 variants, skipping\n",
      "[INFO]\tGene GNG12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESCO2 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2B has 0 variants, skipping\n",
      "[INFO]\tGene CT45A1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP8 has 0 variants, skipping\n",
      "[INFO]\tGene CEP164 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1262 has 0 variants, skipping\n",
      "[INFO]\tGene CTNND1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF550 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01707 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01758 has 0 variants, skipping\n",
      "[INFO]\tGene PLCD1 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAE has 0 variants, skipping\n",
      "[INFO]\tGene BCAR4 has 0 variants, skipping\n",
      "[INFO]\tGene TAZ has 0 variants, skipping\n",
      "[INFO]\tGene GALNT5 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB has 0 variants, skipping\n",
      "[INFO]\tGene PIN1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC113 has 0 variants, skipping\n",
      "[INFO]\tGene RALGDS has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF1 has 0 variants, skipping\n",
      "[INFO]\tGene COX15 has 0 variants, skipping\n",
      "[INFO]\tGene PISD has 0 variants, skipping\n",
      "[INFO]\tGene AQP7 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1C has 0 variants, skipping\n",
      "[INFO]\tGene KATNAL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01788 has 0 variants, skipping\n",
      "[INFO]\tGene EPN1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F2 has 0 variants, skipping\n",
      "[INFO]\tGene IL6ST has 0 variants, skipping\n",
      "[INFO]\tGene SUMF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD1 has 0 variants, skipping\n",
      "[INFO]\tGene NEGR1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02238 has 0 variants, skipping\n",
      "[INFO]\tGene FAM169A has 0 variants, skipping\n",
      "[INFO]\tGene ERICH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VARS2 has 0 variants, skipping\n",
      "[INFO]\tGene TYW3 has 0 variants, skipping\n",
      "[INFO]\tGene CBY1 has 0 variants, skipping\n",
      "[INFO]\tGene INSC has 0 variants, skipping\n",
      "[INFO]\tGene CFTR has 0 variants, skipping\n",
      "[INFO]\tGene SHMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHA has 0 variants, skipping\n",
      "[INFO]\tGene RRAD has 0 variants, skipping\n",
      "[INFO]\tGene RBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB27B has 0 variants, skipping\n",
      "[INFO]\tGene RAB5A has 0 variants, skipping\n",
      "[INFO]\tGene RAB1A has 0 variants, skipping\n",
      "[INFO]\tGene PSMB8 has 0 variants, skipping\n",
      "[INFO]\tGene DLSTP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45B has 0 variants, skipping\n",
      "[INFO]\tGene SNHG29 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927342 has 0 variants, skipping\n",
      "[INFO]\tGene BMF has 0 variants, skipping\n",
      "[INFO]\tGene TMEM139 has 0 variants, skipping\n",
      "[INFO]\tGene CHD9 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02567 has 0 variants, skipping\n",
      "[INFO]\tGene EFEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7156 has 0 variants, skipping\n",
      "[INFO]\tGene COL11A2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF5 has 0 variants, skipping\n",
      "[INFO]\tGene PRCP has 0 variants, skipping\n",
      "[INFO]\tGene CDC14A has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1C has 0 variants, skipping\n",
      "[INFO]\tGene CYTH1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8T has 0 variants, skipping\n",
      "[INFO]\tGene ATF7-NPFF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF655 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf226 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC3 has 0 variants, skipping\n",
      "[INFO]\tGene MGC27382 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB1 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1C has 0 variants, skipping\n",
      "[INFO]\tGene HPCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MTPAP has 0 variants, skipping\n",
      "[INFO]\tGene FAR2 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf40 has 0 variants, skipping\n",
      "[INFO]\tGene ECT2 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM23 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927434 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF6 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42SE1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAPD1 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB3 has 0 variants, skipping\n",
      "[INFO]\tGene LMCD1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC16 has 0 variants, skipping\n",
      "[INFO]\tGene RMC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A11 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS1B has 0 variants, skipping\n",
      "[INFO]\tGene SCAF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01362 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01361 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01725 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927560 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX5 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS2 has 0 variants, skipping\n",
      "[INFO]\tGene C8B has 0 variants, skipping\n",
      "[INFO]\tGene MYO16 has 0 variants, skipping\n",
      "[INFO]\tGene SATB2 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPD3 has 0 variants, skipping\n",
      "[INFO]\tGene UCKL1 has 0 variants, skipping\n",
      "[INFO]\tGene FASLG has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01461 has 0 variants, skipping\n",
      "[INFO]\tGene MGARP has 0 variants, skipping\n",
      "[INFO]\tGene LNX1 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A3 has 0 variants, skipping\n",
      "[INFO]\tGene COX4I2 has 0 variants, skipping\n",
      "[INFO]\tGene RABGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene POM121 has 0 variants, skipping\n",
      "[INFO]\tGene RARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF879 has 0 variants, skipping\n",
      "[INFO]\tGene POMT1 has 0 variants, skipping\n",
      "[INFO]\tGene MICB has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD has 0 variants, skipping\n",
      "[INFO]\tGene LOC646626 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH2B has 0 variants, skipping\n",
      "[INFO]\tGene NCR1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM11 has 0 variants, skipping\n",
      "[INFO]\tGene RGS13 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIGA2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4A has 0 variants, skipping\n",
      "[INFO]\tGene RPS9 has 0 variants, skipping\n",
      "[INFO]\tGene MBP has 0 variants, skipping\n",
      "[INFO]\tGene CRAT has 0 variants, skipping\n",
      "[INFO]\tGene IFT81 has 0 variants, skipping\n",
      "[INFO]\tGene BBS9 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD11 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH2 has 0 variants, skipping\n",
      "[INFO]\tGene DBI has 0 variants, skipping\n",
      "[INFO]\tGene RCBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7856 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN11 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA3P has 0 variants, skipping\n",
      "[INFO]\tGene SLX4IP has 0 variants, skipping\n",
      "[INFO]\tGene MPP1 has 0 variants, skipping\n",
      "[INFO]\tGene APPL2 has 0 variants, skipping\n",
      "[INFO]\tGene EDA2R has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01140 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01364 has 0 variants, skipping\n",
      "[INFO]\tGene PKN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8A has 0 variants, skipping\n",
      "[INFO]\tGene FBRS has 0 variants, skipping\n",
      "[INFO]\tGene TSFM has 0 variants, skipping\n",
      "[INFO]\tGene PPARGC1B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF512 has 0 variants, skipping\n",
      "[INFO]\tGene KYAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP45 has 0 variants, skipping\n",
      "[INFO]\tGene ST7 has 0 variants, skipping\n",
      "[INFO]\tGene CPAMD8 has 0 variants, skipping\n",
      "[INFO]\tGene EPOP has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR2 has 0 variants, skipping\n",
      "[INFO]\tGene WWTR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR156 has 0 variants, skipping\n",
      "[INFO]\tGene SNTB1 has 0 variants, skipping\n",
      "[INFO]\tGene DCTPP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIO has 0 variants, skipping\n",
      "[INFO]\tGene FAM214B has 0 variants, skipping\n",
      "[INFO]\tGene BRD9 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GBP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8C-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A11 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN8P4 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHIP has 0 variants, skipping\n",
      "[INFO]\tGene PSRC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3G has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC25 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01763 has 0 variants, skipping\n",
      "[INFO]\tGene RFC2 has 0 variants, skipping\n",
      "[INFO]\tGene DIS3L has 0 variants, skipping\n",
      "[INFO]\tGene NOL4 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6 has 0 variants, skipping\n",
      "[INFO]\tGene PEPD has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161A has 0 variants, skipping\n",
      "[INFO]\tGene IL15RA has 0 variants, skipping\n",
      "[INFO]\tGene MIXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513C has 0 variants, skipping\n",
      "[INFO]\tGene PDE9A has 0 variants, skipping\n",
      "[INFO]\tGene PRLR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF664-RFLNA has 0 variants, skipping\n",
      "[INFO]\tGene GC has 0 variants, skipping\n",
      "[INFO]\tGene MUC1 has 0 variants, skipping\n",
      "[INFO]\tGene TSC2 has 0 variants, skipping\n",
      "[INFO]\tGene LTB has 0 variants, skipping\n",
      "[INFO]\tGene LARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADNP has 0 variants, skipping\n",
      "[INFO]\tGene ASXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MED13L has 0 variants, skipping\n",
      "[INFO]\tGene IL32 has 0 variants, skipping\n",
      "[INFO]\tGene RNF170 has 0 variants, skipping\n",
      "[INFO]\tGene PMFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C has 0 variants, skipping\n",
      "[INFO]\tGene ATG2B has 0 variants, skipping\n",
      "[INFO]\tGene DET1 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA66 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL2 has 0 variants, skipping\n",
      "[INFO]\tGene SHOC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGT has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF33 has 0 variants, skipping\n",
      "[INFO]\tGene USP28 has 0 variants, skipping\n",
      "[INFO]\tGene PLCL1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B15 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene ATG4B has 0 variants, skipping\n",
      "[INFO]\tGene RABGGTB has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L3 has 0 variants, skipping\n",
      "[INFO]\tGene ENAH has 0 variants, skipping\n",
      "[INFO]\tGene PACC1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF9 has 0 variants, skipping\n",
      "[INFO]\tGene HAO2 has 0 variants, skipping\n",
      "[INFO]\tGene PQBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACYP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOK4 has 0 variants, skipping\n",
      "[INFO]\tGene REEP6 has 0 variants, skipping\n",
      "[INFO]\tGene AKAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378G has 0 variants, skipping\n",
      "[INFO]\tGene CCN5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM186B has 0 variants, skipping\n",
      "[INFO]\tGene MUC5AC has 0 variants, skipping\n",
      "[INFO]\tGene WNT8A has 0 variants, skipping\n",
      "[INFO]\tGene CABP1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX59 has 0 variants, skipping\n",
      "[INFO]\tGene TTN has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB38 has 0 variants, skipping\n",
      "[INFO]\tGene ZZZ3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928118 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23B has 0 variants, skipping\n",
      "[INFO]\tGene GZF1 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFA has 0 variants, skipping\n",
      "[INFO]\tGene CLIC6 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01760 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01650 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01761 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02607 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01787 has 0 variants, skipping\n",
      "[INFO]\tGene PNCK has 0 variants, skipping\n",
      "[INFO]\tGene SMTNL2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3H has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K19 has 0 variants, skipping\n",
      "[INFO]\tGene CARF has 0 variants, skipping\n",
      "[INFO]\tGene DPYD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01930 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2682 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01776 has 0 variants, skipping\n",
      "[INFO]\tGene GLT8D2 has 0 variants, skipping\n",
      "[INFO]\tGene CHML has 0 variants, skipping\n",
      "[INFO]\tGene GCSAM has 0 variants, skipping\n",
      "[INFO]\tGene DNAH10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129620 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8F has 0 variants, skipping\n",
      "[INFO]\tGene ACSF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01708 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548N has 0 variants, skipping\n",
      "[INFO]\tGene PEX5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF658 has 0 variants, skipping\n",
      "[INFO]\tGene APTR has 0 variants, skipping\n",
      "[INFO]\tGene STX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A7 has 0 variants, skipping\n",
      "[INFO]\tGene TMC7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF37A has 0 variants, skipping\n",
      "[INFO]\tGene MAP1B has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB8 has 0 variants, skipping\n",
      "[INFO]\tGene PI3 has 0 variants, skipping\n",
      "[INFO]\tGene PHKA1 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT13 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL38 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCAD1 has 0 variants, skipping\n",
      "[INFO]\tGene REXO4 has 0 variants, skipping\n",
      "[INFO]\tGene RTCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SORBS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR553 has 0 variants, skipping\n",
      "[INFO]\tGene GRM5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKMY1 has 0 variants, skipping\n",
      "[INFO]\tGene ILK has 0 variants, skipping\n",
      "[INFO]\tGene LINC01349 has 0 variants, skipping\n",
      "[INFO]\tGene SASH1 has 0 variants, skipping\n",
      "[INFO]\tGene ICA1 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A7 has 0 variants, skipping\n",
      "[INFO]\tGene MNT has 0 variants, skipping\n",
      "[INFO]\tGene CCNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MGST1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRB1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNBIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102606465 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928370 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01307 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX2 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA1P5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC32 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B5 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2B has 0 variants, skipping\n",
      "[INFO]\tGene MIA has 0 variants, skipping\n",
      "[INFO]\tGene NCAPH2 has 0 variants, skipping\n",
      "[INFO]\tGene CARD10 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF232 has 0 variants, skipping\n",
      "[INFO]\tGene MYBBP1A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF229 has 0 variants, skipping\n",
      "[INFO]\tGene UBP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2S2 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01348 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP3 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA6 has 0 variants, skipping\n",
      "[INFO]\tGene PHRF1 has 0 variants, skipping\n",
      "[INFO]\tGene EPS15L1 has 0 variants, skipping\n",
      "[INFO]\tGene PRODH2 has 0 variants, skipping\n",
      "[INFO]\tGene TC2N has 0 variants, skipping\n",
      "[INFO]\tGene MSH6 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P4 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L5 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CYBA has 0 variants, skipping\n",
      "[INFO]\tGene COL4A3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF20L1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTK1 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A39 has 0 variants, skipping\n",
      "[INFO]\tGene ATL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN19 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129138 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01676 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01677 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01661 has 0 variants, skipping\n",
      "[INFO]\tGene UFSP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX5 has 0 variants, skipping\n",
      "[INFO]\tGene ELP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7852 has 0 variants, skipping\n",
      "[INFO]\tGene VAV3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRB has 0 variants, skipping\n",
      "[INFO]\tGene GALC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF267 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf54 has 0 variants, skipping\n",
      "[INFO]\tGene NMRAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDR2 has 0 variants, skipping\n",
      "[INFO]\tGene CDO1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42 has 0 variants, skipping\n",
      "[INFO]\tGene CDC25C has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2D has 0 variants, skipping\n",
      "[INFO]\tGene CDC25A has 0 variants, skipping\n",
      "[INFO]\tGene CDK11B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC103 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN4 has 0 variants, skipping\n",
      "[INFO]\tGene CHN1 has 0 variants, skipping\n",
      "[INFO]\tGene SNTG1 has 0 variants, skipping\n",
      "[INFO]\tGene PROCA1 has 0 variants, skipping\n",
      "[INFO]\tGene COX7A2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA42 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFSD has 0 variants, skipping\n",
      "[INFO]\tGene IRAG1 has 0 variants, skipping\n",
      "[INFO]\tGene ERLIN2 has 0 variants, skipping\n",
      "[INFO]\tGene STRADA has 0 variants, skipping\n",
      "[INFO]\tGene ETFDH has 0 variants, skipping\n",
      "[INFO]\tGene ENOX2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2H1 has 0 variants, skipping\n",
      "[INFO]\tGene APOA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC25 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN3 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene APLNR has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAB2B has 0 variants, skipping\n",
      "[INFO]\tGene GCC2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT10 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1H has 0 variants, skipping\n",
      "[INFO]\tGene LGMN has 0 variants, skipping\n",
      "[INFO]\tGene MXRA7 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF92 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD12 has 0 variants, skipping\n",
      "[INFO]\tGene RALYL has 0 variants, skipping\n",
      "[INFO]\tGene LDHB has 0 variants, skipping\n",
      "[INFO]\tGene LAIR2 has 0 variants, skipping\n",
      "[INFO]\tGene ISYNA1 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10-NBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MARF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF692 has 0 variants, skipping\n",
      "[INFO]\tGene INTS8 has 0 variants, skipping\n",
      "[INFO]\tGene DUOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1A has 0 variants, skipping\n",
      "[INFO]\tGene NOX1 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1HG has 0 variants, skipping\n",
      "[INFO]\tGene SIRPD has 0 variants, skipping\n",
      "[INFO]\tGene C20orf85 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT4 has 0 variants, skipping\n",
      "[INFO]\tGene DRAM2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF3 has 0 variants, skipping\n",
      "[INFO]\tGene PTH2 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD2 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR4 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE1 has 0 variants, skipping\n",
      "[INFO]\tGene GMCL1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE5A has 0 variants, skipping\n",
      "[INFO]\tGene LCE3E has 0 variants, skipping\n",
      "[INFO]\tGene LCE3C has 0 variants, skipping\n",
      "[INFO]\tGene LCE3B has 0 variants, skipping\n",
      "[INFO]\tGene CDK20 has 0 variants, skipping\n",
      "[INFO]\tGene LCE3A has 0 variants, skipping\n",
      "[INFO]\tGene LCE2D has 0 variants, skipping\n",
      "[INFO]\tGene LCE2C has 0 variants, skipping\n",
      "[INFO]\tGene LCE2A has 0 variants, skipping\n",
      "[INFO]\tGene ARNT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP21 has 0 variants, skipping\n",
      "[INFO]\tGene CLN6 has 0 variants, skipping\n",
      "[INFO]\tGene NMRK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF416 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf42 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB104A has 0 variants, skipping\n",
      "[INFO]\tGene MIR600 has 0 variants, skipping\n",
      "[INFO]\tGene OPN4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS36 has 0 variants, skipping\n",
      "[INFO]\tGene SEC11C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01397 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02586 has 0 variants, skipping\n",
      "[INFO]\tGene GOSR1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2L has 0 variants, skipping\n",
      "[INFO]\tGene PMAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INO80E has 0 variants, skipping\n",
      "[INFO]\tGene ZWILCH has 0 variants, skipping\n",
      "[INFO]\tGene YJU2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf33 has 0 variants, skipping\n",
      "[INFO]\tGene CYMP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNAR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMUB2 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS2 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT2B has 0 variants, skipping\n",
      "[INFO]\tGene FAS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3G has 0 variants, skipping\n",
      "[INFO]\tGene BZW1 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL5 has 0 variants, skipping\n",
      "[INFO]\tGene AP2A2 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL2 has 0 variants, skipping\n",
      "[INFO]\tGene DENND4A has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A1 has 0 variants, skipping\n",
      "[INFO]\tGene SPOP has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB8 has 0 variants, skipping\n",
      "[INFO]\tGene LRRFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2C has 0 variants, skipping\n",
      "[INFO]\tGene DMGDH has 0 variants, skipping\n",
      "[INFO]\tGene CYP2D7 has 0 variants, skipping\n",
      "[INFO]\tGene PIFO has 0 variants, skipping\n",
      "[INFO]\tGene FAM181A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM2 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RUSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01160 has 0 variants, skipping\n",
      "[INFO]\tGene DHX36 has 0 variants, skipping\n",
      "[INFO]\tGene FLT3LG has 0 variants, skipping\n",
      "[INFO]\tGene INKA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928718 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene DFFB has 0 variants, skipping\n",
      "[INFO]\tGene KCND3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene KCND3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01750 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4256 has 0 variants, skipping\n",
      "[INFO]\tGene CCT5 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F1 has 0 variants, skipping\n",
      "[INFO]\tGene USF3 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAP has 0 variants, skipping\n",
      "[INFO]\tGene PRH1-TAS2R14 has 0 variants, skipping\n",
      "[INFO]\tGene PTBP3 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF11 has 0 variants, skipping\n",
      "[INFO]\tGene TFPT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01356 has 0 variants, skipping\n",
      "[INFO]\tGene SMTN has 0 variants, skipping\n",
      "[INFO]\tGene MIR4739 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00639 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45C has 0 variants, skipping\n",
      "[INFO]\tGene FMO9P has 0 variants, skipping\n",
      "[INFO]\tGene MBD4 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP57 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643441 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH10 has 0 variants, skipping\n",
      "[INFO]\tGene RBM6 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N4C has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2A has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L7 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC13 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD3 has 0 variants, skipping\n",
      "[INFO]\tGene PHTF1 has 0 variants, skipping\n",
      "[INFO]\tGene APH1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF13B has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC6A has 0 variants, skipping\n",
      "[INFO]\tGene GLS2 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA13 has 0 variants, skipping\n",
      "[INFO]\tGene CHRDL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCHCR1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IRF5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF526 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF21B has 0 variants, skipping\n",
      "[INFO]\tGene GRB7 has 0 variants, skipping\n",
      "[INFO]\tGene CIB2 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD9L has 0 variants, skipping\n",
      "[INFO]\tGene LIPK has 0 variants, skipping\n",
      "[INFO]\tGene CPNE4 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNP1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI2 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNC2 has 0 variants, skipping\n",
      "[INFO]\tGene TNNC1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC3B has 0 variants, skipping\n",
      "[INFO]\tGene CLDN5 has 0 variants, skipping\n",
      "[INFO]\tGene TMPO has 0 variants, skipping\n",
      "[INFO]\tGene TMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN8 has 0 variants, skipping\n",
      "[INFO]\tGene SETD9 has 0 variants, skipping\n",
      "[INFO]\tGene MBD1 has 0 variants, skipping\n",
      "[INFO]\tGene STAM has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP3 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC21 has 0 variants, skipping\n",
      "[INFO]\tGene AFTPH has 0 variants, skipping\n",
      "[INFO]\tGene ZNF542P has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-1HG has 0 variants, skipping\n",
      "[INFO]\tGene TASOR2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB4 has 0 variants, skipping\n",
      "[INFO]\tGene GSTZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1L has 0 variants, skipping\n",
      "[INFO]\tGene AP4S1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02861 has 0 variants, skipping\n",
      "[INFO]\tGene KLK2 has 0 variants, skipping\n",
      "[INFO]\tGene COL25A1 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL4 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB1 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH1 has 0 variants, skipping\n",
      "[INFO]\tGene C2 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB6 has 0 variants, skipping\n",
      "[INFO]\tGene NGF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM26 has 0 variants, skipping\n",
      "[INFO]\tGene FBLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM218 has 0 variants, skipping\n",
      "[INFO]\tGene CROCC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01649 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928977 has 0 variants, skipping\n",
      "[INFO]\tGene RAB5C has 0 variants, skipping\n",
      "[INFO]\tGene RAB5B has 0 variants, skipping\n",
      "[INFO]\tGene TPBG has 0 variants, skipping\n",
      "[INFO]\tGene CYP46A1 has 0 variants, skipping\n",
      "[INFO]\tGene PGRMC1 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE4 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01762 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3B1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM34 has 0 variants, skipping\n",
      "[INFO]\tGene ASB12 has 0 variants, skipping\n",
      "[INFO]\tGene ASB14 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC12 has 0 variants, skipping\n",
      "[INFO]\tGene ATRIP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF783 has 0 variants, skipping\n",
      "[INFO]\tGene SSB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929099 has 0 variants, skipping\n",
      "[INFO]\tGene TP53RK has 0 variants, skipping\n",
      "[INFO]\tGene INTS4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL4L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01525 has 0 variants, skipping\n",
      "[INFO]\tGene NFAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996263 has 0 variants, skipping\n",
      "[INFO]\tGene EHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene GPS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP4 has 0 variants, skipping\n",
      "[INFO]\tGene NEK6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS41 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378933 has 0 variants, skipping\n",
      "[INFO]\tGene IL17REL has 0 variants, skipping\n",
      "[INFO]\tGene WARS2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene WARS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01780 has 0 variants, skipping\n",
      "[INFO]\tGene HAO2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51I2 has 0 variants, skipping\n",
      "[INFO]\tGene HSD3BP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00622 has 0 variants, skipping\n",
      "[INFO]\tGene DAAM2 has 0 variants, skipping\n",
      "[INFO]\tGene ELN has 0 variants, skipping\n",
      "[INFO]\tGene DNM1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC16 has 0 variants, skipping\n",
      "[INFO]\tGene BBS4 has 0 variants, skipping\n",
      "[INFO]\tGene PFN1P2 has 0 variants, skipping\n",
      "[INFO]\tGene RFX5 has 0 variants, skipping\n",
      "[INFO]\tGene RBPJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR128-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR374A has 0 variants, skipping\n",
      "[INFO]\tGene MIR155 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200C has 0 variants, skipping\n",
      "[INFO]\tGene MIR650 has 0 variants, skipping\n",
      "[INFO]\tGene MIR622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR615 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-19 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00623 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-20 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-4 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL3 has 0 variants, skipping\n",
      "[INFO]\tGene PARP6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996318 has 0 variants, skipping\n",
      "[INFO]\tGene OBSL1 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P5 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644634 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1CP has 0 variants, skipping\n",
      "[INFO]\tGene MIEF1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM72C has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101060524 has 0 variants, skipping\n",
      "[INFO]\tGene DRD5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927429 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A2 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-15 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF3 has 0 variants, skipping\n",
      "[INFO]\tGene EML2 has 0 variants, skipping\n",
      "[INFO]\tGene DBNDD2 has 0 variants, skipping\n",
      "[INFO]\tGene PKP4 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC5 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1A has 0 variants, skipping\n",
      "[INFO]\tGene FAM72D has 0 variants, skipping\n",
      "[INFO]\tGene LINC01145 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996740 has 0 variants, skipping\n",
      "[INFO]\tGene FGF13 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-14 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF9 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC19 has 0 variants, skipping\n",
      "[INFO]\tGene GPX4 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO4 has 0 variants, skipping\n",
      "[INFO]\tGene RARG has 0 variants, skipping\n",
      "[INFO]\tGene CCDC74A has 0 variants, skipping\n",
      "[INFO]\tGene ABCB1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF25P has 0 variants, skipping\n",
      "[INFO]\tGene BDKRB2 has 0 variants, skipping\n",
      "[INFO]\tGene HHIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR89A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF888 has 0 variants, skipping\n",
      "[INFO]\tGene NKX3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF266 has 0 variants, skipping\n",
      "[INFO]\tGene PEX5L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6736 has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG1 has 0 variants, skipping\n",
      "[INFO]\tGene APBB1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2D has 0 variants, skipping\n",
      "[INFO]\tGene GSS has 0 variants, skipping\n",
      "[INFO]\tGene NR3C1 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE5 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL3 has 0 variants, skipping\n",
      "[INFO]\tGene GNRHR2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8S has 0 variants, skipping\n",
      "[INFO]\tGene LIX1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOF has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01719 has 0 variants, skipping\n",
      "[INFO]\tGene ARL17B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4P2 has 0 variants, skipping\n",
      "[INFO]\tGene HYDIN2 has 0 variants, skipping\n",
      "[INFO]\tGene SV2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC728989 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-8 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2C has 0 variants, skipping\n",
      "[INFO]\tGene PDIA3P1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW12 has 0 variants, skipping\n",
      "[INFO]\tGene FLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC10 has 0 variants, skipping\n",
      "[INFO]\tGene ZPBP has 0 variants, skipping\n",
      "[INFO]\tGene CD200 has 0 variants, skipping\n",
      "[INFO]\tGene NTRK2 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT2 has 0 variants, skipping\n",
      "[INFO]\tGene FYN has 0 variants, skipping\n",
      "[INFO]\tGene EFR3A has 0 variants, skipping\n",
      "[INFO]\tGene STOML1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF286A has 0 variants, skipping\n",
      "[INFO]\tGene IYD has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L2-SCNM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO2-KLHL23 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A has 0 variants, skipping\n",
      "[INFO]\tGene TYW5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM165 has 0 variants, skipping\n",
      "[INFO]\tGene FUBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCER1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM28 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL5 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HINT1 has 0 variants, skipping\n",
      "[INFO]\tGene STYXL1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO11 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H4 has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ has 0 variants, skipping\n",
      "[INFO]\tGene METTL23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00624 has 0 variants, skipping\n",
      "[INFO]\tGene ERLIN1 has 0 variants, skipping\n",
      "[INFO]\tGene SOST has 0 variants, skipping\n",
      "[INFO]\tGene LY6K has 0 variants, skipping\n",
      "[INFO]\tGene ZNF7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF460 has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD1 has 0 variants, skipping\n",
      "[INFO]\tGene AK4 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED9 has 0 variants, skipping\n",
      "[INFO]\tGene PPARG has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF205-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CKS1B has 0 variants, skipping\n",
      "[INFO]\tGene DGCR5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC-PINT has 0 variants, skipping\n",
      "[INFO]\tGene SNHG12 has 0 variants, skipping\n",
      "[INFO]\tGene PBX3 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM3A has 0 variants, skipping\n",
      "[INFO]\tGene VSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927468 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5087 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6077 has 0 variants, skipping\n",
      "[INFO]\tGene FN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-3 has 0 variants, skipping\n",
      "[INFO]\tGene MZB1 has 0 variants, skipping\n",
      "[INFO]\tGene HK1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT8 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2A has 0 variants, skipping\n",
      "[INFO]\tGene PCCA has 0 variants, skipping\n",
      "[INFO]\tGene PRIMPOL has 0 variants, skipping\n",
      "[INFO]\tGene RNF167 has 0 variants, skipping\n",
      "[INFO]\tGene PNO1 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene GPI has 0 variants, skipping\n",
      "[INFO]\tGene STX3 has 0 variants, skipping\n",
      "[INFO]\tGene USO1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A35 has 0 variants, skipping\n",
      "[INFO]\tGene BMX has 0 variants, skipping\n",
      "[INFO]\tGene SPATA1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD7 has 0 variants, skipping\n",
      "[INFO]\tGene NKTR has 0 variants, skipping\n",
      "[INFO]\tGene PIGG has 0 variants, skipping\n",
      "[INFO]\tGene LOC653513 has 0 variants, skipping\n",
      "[INFO]\tGene DOK1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf47 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN7 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A1 has 0 variants, skipping\n",
      "[INFO]\tGene RND3 has 0 variants, skipping\n",
      "[INFO]\tGene NABP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00869 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF518A has 0 variants, skipping\n",
      "[INFO]\tGene AZIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4F has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS5 has 0 variants, skipping\n",
      "[INFO]\tGene CADM2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE8B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371433 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN6 has 0 variants, skipping\n",
      "[INFO]\tGene ANK3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13C has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL5B has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene APH1A has 0 variants, skipping\n",
      "[INFO]\tGene ZPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene CMPK1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM131A has 0 variants, skipping\n",
      "[INFO]\tGene DSE has 0 variants, skipping\n",
      "[INFO]\tGene KCP has 0 variants, skipping\n",
      "[INFO]\tGene FOXO3 has 0 variants, skipping\n",
      "[INFO]\tGene GATD3A has 0 variants, skipping\n",
      "[INFO]\tGene C16orf87 has 0 variants, skipping\n",
      "[INFO]\tGene DLX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00461 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF224 has 0 variants, skipping\n",
      "[INFO]\tGene MOV10L1 has 0 variants, skipping\n",
      "[INFO]\tGene COPS7A has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DOA has 0 variants, skipping\n",
      "[INFO]\tGene FKBP7 has 0 variants, skipping\n",
      "[INFO]\tGene KAAG1 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6878 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3J has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN25 has 0 variants, skipping\n",
      "[INFO]\tGene OR2F1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP12 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EPC1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL7 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8O has 0 variants, skipping\n",
      "[INFO]\tGene MYH4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF821 has 0 variants, skipping\n",
      "[INFO]\tGene USP48 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3B has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA1 has 0 variants, skipping\n",
      "[INFO]\tGene LARP1B has 0 variants, skipping\n",
      "[INFO]\tGene POMK has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10 has 0 variants, skipping\n",
      "[INFO]\tGene SERPING1 has 0 variants, skipping\n",
      "[INFO]\tGene ENO1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL6 has 0 variants, skipping\n",
      "[INFO]\tGene IQCB1 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L22 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L21 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L20 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L18 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L19 has 0 variants, skipping\n",
      "[INFO]\tGene RNF34 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L17 has 0 variants, skipping\n",
      "[INFO]\tGene BTRC has 0 variants, skipping\n",
      "[INFO]\tGene USP17L13 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L11 has 0 variants, skipping\n",
      "[INFO]\tGene GALK2 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK6 has 0 variants, skipping\n",
      "[INFO]\tGene FAR1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPM has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PITRM1 has 0 variants, skipping\n",
      "[INFO]\tGene TSBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF556 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN9 has 0 variants, skipping\n",
      "[INFO]\tGene CD1A has 0 variants, skipping\n",
      "[INFO]\tGene CCT6A has 0 variants, skipping\n",
      "[INFO]\tGene CCNF has 0 variants, skipping\n",
      "[INFO]\tGene CCND3 has 0 variants, skipping\n",
      "[INFO]\tGene CCND2 has 0 variants, skipping\n",
      "[INFO]\tGene DST has 0 variants, skipping\n",
      "[INFO]\tGene CEP97 has 0 variants, skipping\n",
      "[INFO]\tGene CTSB has 0 variants, skipping\n",
      "[INFO]\tGene CANX has 0 variants, skipping\n",
      "[INFO]\tGene NXPE3 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf173 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724788 has 0 variants, skipping\n",
      "[INFO]\tGene CRYAB has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724770 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724159 has 0 variants, skipping\n",
      "[INFO]\tGene FAM243A has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF687-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK3 has 0 variants, skipping\n",
      "[INFO]\tGene DMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM121B has 0 variants, skipping\n",
      "[INFO]\tGene C17orf99 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT11 has 0 variants, skipping\n",
      "[INFO]\tGene CHST15 has 0 variants, skipping\n",
      "[INFO]\tGene ARNT2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C2 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3B has 0 variants, skipping\n",
      "[INFO]\tGene ESRRG has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1191 has 0 variants, skipping\n",
      "[INFO]\tGene NAV1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAKD has 0 variants, skipping\n",
      "[INFO]\tGene TSEN34 has 0 variants, skipping\n",
      "[INFO]\tGene MIR554 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD5 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf40 has 0 variants, skipping\n",
      "[INFO]\tGene CBSL has 0 variants, skipping\n",
      "[INFO]\tGene BLZF1 has 0 variants, skipping\n",
      "[INFO]\tGene CBS has 0 variants, skipping\n",
      "[INFO]\tGene CCDC66 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB2 has 0 variants, skipping\n",
      "[INFO]\tGene TDRKH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110384692 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3B has 0 variants, skipping\n",
      "[INFO]\tGene AP2S1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT5 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF18P has 0 variants, skipping\n",
      "[INFO]\tGene MYO19 has 0 variants, skipping\n",
      "[INFO]\tGene FLG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRF has 0 variants, skipping\n",
      "[INFO]\tGene RPL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00302 has 0 variants, skipping\n",
      "[INFO]\tGene ADK has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF12 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3A has 0 variants, skipping\n",
      "[INFO]\tGene USP40 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf14 has 0 variants, skipping\n",
      "[INFO]\tGene SETMAR has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928009 has 0 variants, skipping\n",
      "[INFO]\tGene DCHS2 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPPS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928034 has 0 variants, skipping\n",
      "[INFO]\tGene FGD4 has 0 variants, skipping\n",
      "[INFO]\tGene CAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GREB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIB1 has 0 variants, skipping\n",
      "[INFO]\tGene LIPT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8083 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF419 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC343052 has 0 variants, skipping\n",
      "[INFO]\tGene RAB44 has 0 variants, skipping\n",
      "[INFO]\tGene BCS1L has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZPR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6737 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN14 has 0 variants, skipping\n",
      "[INFO]\tGene NEK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3 has 0 variants, skipping\n",
      "[INFO]\tGene OSGEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF44 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01060 has 0 variants, skipping\n",
      "[INFO]\tGene ENKUR has 0 variants, skipping\n",
      "[INFO]\tGene STAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMLHE has 0 variants, skipping\n",
      "[INFO]\tGene FAM78B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5698 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1 has 0 variants, skipping\n",
      "[INFO]\tGene HCLS1 has 0 variants, skipping\n",
      "[INFO]\tGene HDLBP has 0 variants, skipping\n",
      "[INFO]\tGene CYC1 has 0 variants, skipping\n",
      "[INFO]\tGene FLG has 0 variants, skipping\n",
      "[INFO]\tGene FOXO1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2E3 has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF4 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM174C has 0 variants, skipping\n",
      "[INFO]\tGene CDC37L1 has 0 variants, skipping\n",
      "[INFO]\tGene ATG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM3 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf72 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF619 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12E has 0 variants, skipping\n",
      "[INFO]\tGene PRM2 has 0 variants, skipping\n",
      "[INFO]\tGene GPRC6A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA58B has 0 variants, skipping\n",
      "[INFO]\tGene REV1 has 0 variants, skipping\n",
      "[INFO]\tGene IL6R-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAF8 has 0 variants, skipping\n",
      "[INFO]\tGene TEX49 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG6 has 0 variants, skipping\n",
      "[INFO]\tGene HYKK has 0 variants, skipping\n",
      "[INFO]\tGene MKS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF8 has 0 variants, skipping\n",
      "[INFO]\tGene TP73 has 0 variants, skipping\n",
      "[INFO]\tGene GLMP has 0 variants, skipping\n",
      "[INFO]\tGene LARP4 has 0 variants, skipping\n",
      "[INFO]\tGene CDK1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC20B has 0 variants, skipping\n",
      "[INFO]\tGene GLS has 0 variants, skipping\n",
      "[INFO]\tGene EPHX2 has 0 variants, skipping\n",
      "[INFO]\tGene DMPK has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF75A has 0 variants, skipping\n",
      "[INFO]\tGene CPNE1 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE1 has 0 variants, skipping\n",
      "[INFO]\tGene CHFR has 0 variants, skipping\n",
      "[INFO]\tGene NR2C2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF420 has 0 variants, skipping\n",
      "[INFO]\tGene DEDD has 0 variants, skipping\n",
      "[INFO]\tGene ZNF875 has 0 variants, skipping\n",
      "[INFO]\tGene WDR20 has 0 variants, skipping\n",
      "[INFO]\tGene EHMT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6J1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724488 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4258 has 0 variants, skipping\n",
      "[INFO]\tGene NFIL3 has 0 variants, skipping\n",
      "[INFO]\tGene FHIT has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01082 has 0 variants, skipping\n",
      "[INFO]\tGene MINAR2 has 0 variants, skipping\n",
      "[INFO]\tGene MOK has 0 variants, skipping\n",
      "[INFO]\tGene ZNF74 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT has 0 variants, skipping\n",
      "[INFO]\tGene DPP10 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene CADM3 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL5 has 0 variants, skipping\n",
      "[INFO]\tGene CD8B2 has 0 variants, skipping\n",
      "[INFO]\tGene MYO7A has 0 variants, skipping\n",
      "[INFO]\tGene MYO5A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC30 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA4B has 0 variants, skipping\n",
      "[INFO]\tGene OR10G6 has 0 variants, skipping\n",
      "[INFO]\tGene RET has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM126A has 0 variants, skipping\n",
      "[INFO]\tGene AKAP10 has 0 variants, skipping\n",
      "[INFO]\tGene F11R has 0 variants, skipping\n",
      "[INFO]\tGene LINC02751 has 0 variants, skipping\n",
      "[INFO]\tGene HSPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB2 has 0 variants, skipping\n",
      "[INFO]\tGene OAT has 0 variants, skipping\n",
      "[INFO]\tGene SRSF7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM240A has 0 variants, skipping\n",
      "[INFO]\tGene MAS1L has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN15 has 0 variants, skipping\n",
      "[INFO]\tGene CLNK has 0 variants, skipping\n",
      "[INFO]\tGene GNG2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1MT has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF35-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184B has 0 variants, skipping\n",
      "[INFO]\tGene THBS3 has 0 variants, skipping\n",
      "[INFO]\tGene MUSK has 0 variants, skipping\n",
      "[INFO]\tGene PAN2 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1B has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL26 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC14 has 0 variants, skipping\n",
      "[INFO]\tGene THOC3 has 0 variants, skipping\n",
      "[INFO]\tGene CMSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC115 has 0 variants, skipping\n",
      "[INFO]\tGene NAA38 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12C has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX2 has 0 variants, skipping\n",
      "[INFO]\tGene SSH2 has 0 variants, skipping\n",
      "[INFO]\tGene ATG16L2 has 0 variants, skipping\n",
      "[INFO]\tGene IL18BP has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A2 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1C has 0 variants, skipping\n",
      "[INFO]\tGene BCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D5 has 0 variants, skipping\n",
      "[INFO]\tGene OR1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2 has 0 variants, skipping\n",
      "[INFO]\tGene OTX1 has 0 variants, skipping\n",
      "[INFO]\tGene PARPBP has 0 variants, skipping\n",
      "[INFO]\tGene ERI3 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA2 has 0 variants, skipping\n",
      "[INFO]\tGene MOBP has 0 variants, skipping\n",
      "[INFO]\tGene IFFO1 has 0 variants, skipping\n",
      "[INFO]\tGene USP5 has 0 variants, skipping\n",
      "[INFO]\tGene DDR1 has 0 variants, skipping\n",
      "[INFO]\tGene COQ8A has 0 variants, skipping\n",
      "[INFO]\tGene TAF1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR555 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P4 has 0 variants, skipping\n",
      "[INFO]\tGene ASH1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TOP3A has 0 variants, skipping\n",
      "[INFO]\tGene RB1CC1 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRF1L has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG1 has 0 variants, skipping\n",
      "[INFO]\tGene STRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM20 has 0 variants, skipping\n",
      "[INFO]\tGene CASP8AP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC7 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L2 has 0 variants, skipping\n",
      "[INFO]\tGene TSIX has 0 variants, skipping\n",
      "[INFO]\tGene LAS1L has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN5 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01714 has 0 variants, skipping\n",
      "[INFO]\tGene RNF207-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSTO2P has 0 variants, skipping\n",
      "[INFO]\tGene SMAGP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF451 has 0 variants, skipping\n",
      "[INFO]\tGene AMMECR1 has 0 variants, skipping\n",
      "[INFO]\tGene SOHLH1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX23 has 0 variants, skipping\n",
      "[INFO]\tGene TJP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM189A2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP29 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA26A has 0 variants, skipping\n",
      "[INFO]\tGene LALBA has 0 variants, skipping\n",
      "[INFO]\tGene GTDC1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB2 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1A1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP9 has 0 variants, skipping\n",
      "[INFO]\tGene TP53I3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3R2 has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA26B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA4 has 0 variants, skipping\n",
      "[INFO]\tGene EMC8 has 0 variants, skipping\n",
      "[INFO]\tGene MTERF4 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT1A has 0 variants, skipping\n",
      "[INFO]\tGene SP140L has 0 variants, skipping\n",
      "[INFO]\tGene GLT8D1 has 0 variants, skipping\n",
      "[INFO]\tGene SMU1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6738 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2B has 0 variants, skipping\n",
      "[INFO]\tGene IMPG1 has 0 variants, skipping\n",
      "[INFO]\tGene CDIN1 has 0 variants, skipping\n",
      "[INFO]\tGene INF2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX25 has 0 variants, skipping\n",
      "[INFO]\tGene SRPK2 has 0 variants, skipping\n",
      "[INFO]\tGene RACGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF184 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1 has 0 variants, skipping\n",
      "[INFO]\tGene STRA6 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B15 has 0 variants, skipping\n",
      "[INFO]\tGene BRS3 has 0 variants, skipping\n",
      "[INFO]\tGene BRDT has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB44 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC8 has 0 variants, skipping\n",
      "[INFO]\tGene DMAC2 has 0 variants, skipping\n",
      "[INFO]\tGene PHIP has 0 variants, skipping\n",
      "[INFO]\tGene WDR70 has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L has 0 variants, skipping\n",
      "[INFO]\tGene WDR6 has 0 variants, skipping\n",
      "[INFO]\tGene PEX26 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL1A has 0 variants, skipping\n",
      "[INFO]\tGene DAZAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMF1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D26 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4RL2 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf38 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4RL1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO2 has 0 variants, skipping\n",
      "[INFO]\tGene NRBP2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC7 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf51B has 0 variants, skipping\n",
      "[INFO]\tGene CNIH2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A5 has 0 variants, skipping\n",
      "[INFO]\tGene CYP20A1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG2 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK1 has 0 variants, skipping\n",
      "[INFO]\tGene PANDAR has 0 variants, skipping\n",
      "[INFO]\tGene RPL13 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A5 has 0 variants, skipping\n",
      "[INFO]\tGene DUS2 has 0 variants, skipping\n",
      "[INFO]\tGene SIVA1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2CA has 0 variants, skipping\n",
      "[INFO]\tGene DLD has 0 variants, skipping\n",
      "[INFO]\tGene CPOX has 0 variants, skipping\n",
      "[INFO]\tGene DDB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928177 has 0 variants, skipping\n",
      "[INFO]\tGene DHFR has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS12 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene SAP30L has 0 variants, skipping\n",
      "[INFO]\tGene ARMC9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF558 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC1 has 0 variants, skipping\n",
      "[INFO]\tGene CABIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF513 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK2 has 0 variants, skipping\n",
      "[INFO]\tGene SCRN1 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM20 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MMS19 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A2 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI has 0 variants, skipping\n",
      "[INFO]\tGene TFDP2 has 0 variants, skipping\n",
      "[INFO]\tGene TESK1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD3 has 0 variants, skipping\n",
      "[INFO]\tGene CYCSP52 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF595 has 0 variants, skipping\n",
      "[INFO]\tGene IDH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAK has 0 variants, skipping\n",
      "[INFO]\tGene LINC00324 has 0 variants, skipping\n",
      "[INFO]\tGene EWSAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR8 has 0 variants, skipping\n",
      "[INFO]\tGene SMN2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC13 has 0 variants, skipping\n",
      "[INFO]\tGene SEPSECS has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371458 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF160 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01704 has 0 variants, skipping\n",
      "[INFO]\tGene POLD2 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf58 has 0 variants, skipping\n",
      "[INFO]\tGene DHPS has 0 variants, skipping\n",
      "[INFO]\tGene TTC27 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL24 has 0 variants, skipping\n",
      "[INFO]\tGene EVPL has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC28 has 0 variants, skipping\n",
      "[INFO]\tGene MUC3A has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE3 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFF has 0 variants, skipping\n",
      "[INFO]\tGene LSM5 has 0 variants, skipping\n",
      "[INFO]\tGene LSM4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G15 has 0 variants, skipping\n",
      "[INFO]\tGene CADM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCAP31 has 0 variants, skipping\n",
      "[INFO]\tGene REPS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL11A1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S6 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J4 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12G has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161B has 0 variants, skipping\n",
      "[INFO]\tGene LATS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6506 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG28 has 0 variants, skipping\n",
      "[INFO]\tGene NEPRO has 0 variants, skipping\n",
      "[INFO]\tGene MIR4259 has 0 variants, skipping\n",
      "[INFO]\tGene ATG4D has 0 variants, skipping\n",
      "[INFO]\tGene CLTC has 0 variants, skipping\n",
      "[INFO]\tGene PRELID1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN10 has 0 variants, skipping\n",
      "[INFO]\tGene DROSHA has 0 variants, skipping\n",
      "[INFO]\tGene EIF3K has 0 variants, skipping\n",
      "[INFO]\tGene STK39 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729867 has 0 variants, skipping\n",
      "[INFO]\tGene PATE3 has 0 variants, skipping\n",
      "[INFO]\tGene WWOX has 0 variants, skipping\n",
      "[INFO]\tGene FOXP2 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1D has 0 variants, skipping\n",
      "[INFO]\tGene EIF6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01535 has 0 variants, skipping\n",
      "[INFO]\tGene CCND1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287049 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723623 has 0 variants, skipping\n",
      "[INFO]\tGene ACACA has 0 variants, skipping\n",
      "[INFO]\tGene PPRC1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMLG has 0 variants, skipping\n",
      "[INFO]\tGene CAMK4 has 0 variants, skipping\n",
      "[INFO]\tGene CALM2 has 0 variants, skipping\n",
      "[INFO]\tGene CALCR has 0 variants, skipping\n",
      "[INFO]\tGene CALCA has 0 variants, skipping\n",
      "[INFO]\tGene CALB2 has 0 variants, skipping\n",
      "[INFO]\tGene CA5A has 0 variants, skipping\n",
      "[INFO]\tGene CA1 has 0 variants, skipping\n",
      "[INFO]\tGene C9 has 0 variants, skipping\n",
      "[INFO]\tGene C5AR1 has 0 variants, skipping\n",
      "[INFO]\tGene C5 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf147 has 0 variants, skipping\n",
      "[INFO]\tGene IFT20 has 0 variants, skipping\n",
      "[INFO]\tGene CARD16 has 0 variants, skipping\n",
      "[INFO]\tGene CTSA has 0 variants, skipping\n",
      "[INFO]\tGene WDR77 has 0 variants, skipping\n",
      "[INFO]\tGene RAB25 has 0 variants, skipping\n",
      "[INFO]\tGene S100A16 has 0 variants, skipping\n",
      "[INFO]\tGene MMP11 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1C has 0 variants, skipping\n",
      "[INFO]\tGene DPP6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928372 has 0 variants, skipping\n",
      "[INFO]\tGene HERC1 has 0 variants, skipping\n",
      "[INFO]\tGene PPAN has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN2 has 0 variants, skipping\n",
      "[INFO]\tGene BAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL27 has 0 variants, skipping\n",
      "[INFO]\tGene GMPR2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2H has 0 variants, skipping\n",
      "[INFO]\tGene ACSM5 has 0 variants, skipping\n",
      "[INFO]\tGene BBIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIPOL1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP4 has 0 variants, skipping\n",
      "[INFO]\tGene BCO1 has 0 variants, skipping\n",
      "[INFO]\tGene SPA17 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT7 has 0 variants, skipping\n",
      "[INFO]\tGene CALML5 has 0 variants, skipping\n",
      "[INFO]\tGene COQ3 has 0 variants, skipping\n",
      "[INFO]\tGene SIX4 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC5 has 0 variants, skipping\n",
      "[INFO]\tGene CLPSL1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC8L1 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHB has 0 variants, skipping\n",
      "[INFO]\tGene PRDM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5187 has 0 variants, skipping\n",
      "[INFO]\tGene STING1 has 0 variants, skipping\n",
      "[INFO]\tGene RPGR has 0 variants, skipping\n",
      "[INFO]\tGene HEPH has 0 variants, skipping\n",
      "[INFO]\tGene NXF2 has 0 variants, skipping\n",
      "[INFO]\tGene MRNIP has 0 variants, skipping\n",
      "[INFO]\tGene ZXDC has 0 variants, skipping\n",
      "[INFO]\tGene CILK1 has 0 variants, skipping\n",
      "[INFO]\tGene VSX1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF22 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A45 has 0 variants, skipping\n",
      "[INFO]\tGene MECR has 0 variants, skipping\n",
      "[INFO]\tGene KLK15 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC18 has 0 variants, skipping\n",
      "[INFO]\tGene WEE1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf67 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2012 has 0 variants, skipping\n",
      "[INFO]\tGene RPL31P11 has 0 variants, skipping\n",
      "[INFO]\tGene AMD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF17 has 0 variants, skipping\n",
      "[INFO]\tGene STARD3NL has 0 variants, skipping\n",
      "[INFO]\tGene GATB has 0 variants, skipping\n",
      "[INFO]\tGene CLHC1 has 0 variants, skipping\n",
      "[INFO]\tGene CS has 0 variants, skipping\n",
      "[INFO]\tGene AIP has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4654 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF227 has 0 variants, skipping\n",
      "[INFO]\tGene STRA8 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf50 has 0 variants, skipping\n",
      "[INFO]\tGene USP12 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML2A has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF6 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA13 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM59 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1L1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT10 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC92 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM19 has 0 variants, skipping\n",
      "[INFO]\tGene HBS1L has 0 variants, skipping\n",
      "[INFO]\tGene PMP2 has 0 variants, skipping\n",
      "[INFO]\tGene PMM1 has 0 variants, skipping\n",
      "[INFO]\tGene PML has 0 variants, skipping\n",
      "[INFO]\tGene PMCH has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422212 has 0 variants, skipping\n",
      "[INFO]\tGene LENG8 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP7 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF9 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene LCN12 has 0 variants, skipping\n",
      "[INFO]\tGene RNF180 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9C2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC52-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440700 has 0 variants, skipping\n",
      "[INFO]\tGene OXR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T7 has 0 variants, skipping\n",
      "[INFO]\tGene DDX39B has 0 variants, skipping\n",
      "[INFO]\tGene TMCO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB6 has 0 variants, skipping\n",
      "[INFO]\tGene PART1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01675 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG1 has 0 variants, skipping\n",
      "[INFO]\tGene RLN2 has 0 variants, skipping\n",
      "[INFO]\tGene MTOR has 0 variants, skipping\n",
      "[INFO]\tGene TEX15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01363 has 0 variants, skipping\n",
      "[INFO]\tGene SH3D19 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN3 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB2 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1A1 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP5 has 0 variants, skipping\n",
      "[INFO]\tGene FPGS has 0 variants, skipping\n",
      "[INFO]\tGene ETV1 has 0 variants, skipping\n",
      "[INFO]\tGene RAPSN has 0 variants, skipping\n",
      "[INFO]\tGene RGPD5 has 0 variants, skipping\n",
      "[INFO]\tGene MARK2 has 0 variants, skipping\n",
      "[INFO]\tGene H3-3B has 0 variants, skipping\n",
      "[INFO]\tGene H1-6 has 0 variants, skipping\n",
      "[INFO]\tGene COPS8 has 0 variants, skipping\n",
      "[INFO]\tGene BLCAP has 0 variants, skipping\n",
      "[INFO]\tGene GLO1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD8 has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45G has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3A has 0 variants, skipping\n",
      "[INFO]\tGene JTB has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCQ has 0 variants, skipping\n",
      "[INFO]\tGene GAREM1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO32 has 0 variants, skipping\n",
      "[INFO]\tGene RSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255B2 has 0 variants, skipping\n",
      "[INFO]\tGene IRAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TBXAS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC8 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS11 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR557 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505918 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928565 has 0 variants, skipping\n",
      "[INFO]\tGene TNK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00626 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00970 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928596 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1D has 0 variants, skipping\n",
      "[INFO]\tGene TTC13 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB3BP has 0 variants, skipping\n",
      "[INFO]\tGene RC3H1 has 0 variants, skipping\n",
      "[INFO]\tGene TNR has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A9 has 0 variants, skipping\n",
      "[INFO]\tGene IFT43 has 0 variants, skipping\n",
      "[INFO]\tGene ATF2 has 0 variants, skipping\n",
      "[INFO]\tGene SWI5 has 0 variants, skipping\n",
      "[INFO]\tGene DERPC has 0 variants, skipping\n",
      "[INFO]\tGene LIG1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H14 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF5 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL1 has 0 variants, skipping\n",
      "[INFO]\tGene SV2A has 0 variants, skipping\n",
      "[INFO]\tGene STUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MUTYH has 0 variants, skipping\n",
      "[INFO]\tGene MVP has 0 variants, skipping\n",
      "[INFO]\tGene FAM153CP has 0 variants, skipping\n",
      "[INFO]\tGene CREM has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene RHOG has 0 variants, skipping\n",
      "[INFO]\tGene RHOA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM102 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC40 has 0 variants, skipping\n",
      "[INFO]\tGene TTI2 has 0 variants, skipping\n",
      "[INFO]\tGene EMC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3119-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3119-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01681 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01142 has 0 variants, skipping\n",
      "[INFO]\tGene GORAB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RDM1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-E has 0 variants, skipping\n",
      "[INFO]\tGene MNX1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-B has 0 variants, skipping\n",
      "[INFO]\tGene DEF8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1295A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1295B has 0 variants, skipping\n",
      "[INFO]\tGene VWA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCMT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf153 has 0 variants, skipping\n",
      "[INFO]\tGene NRDC has 0 variants, skipping\n",
      "[INFO]\tGene NASP has 0 variants, skipping\n",
      "[INFO]\tGene TEX28 has 0 variants, skipping\n",
      "[INFO]\tGene CNPPD1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT8 has 0 variants, skipping\n",
      "[INFO]\tGene TWSG1 has 0 variants, skipping\n",
      "[INFO]\tGene JPH1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL9A1 has 0 variants, skipping\n",
      "[INFO]\tGene CLPB has 0 variants, skipping\n",
      "[INFO]\tGene DNM3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene QRFPR has 0 variants, skipping\n",
      "[INFO]\tGene SPATA32 has 0 variants, skipping\n",
      "[INFO]\tGene CDYL2 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA2L has 0 variants, skipping\n",
      "[INFO]\tGene ZG16 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf46 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM266 has 0 variants, skipping\n",
      "[INFO]\tGene TARS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506023 has 0 variants, skipping\n",
      "[INFO]\tGene MPZ has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC8 has 0 variants, skipping\n",
      "[INFO]\tGene DLG4 has 0 variants, skipping\n",
      "[INFO]\tGene MB has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA12 has 0 variants, skipping\n",
      "[INFO]\tGene HELB has 0 variants, skipping\n",
      "[INFO]\tGene RIPK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF430 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRNP has 0 variants, skipping\n",
      "[INFO]\tGene GNAL has 0 variants, skipping\n",
      "[INFO]\tGene AIFM3 has 0 variants, skipping\n",
      "[INFO]\tGene NOP16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01355 has 0 variants, skipping\n",
      "[INFO]\tGene UGP2 has 0 variants, skipping\n",
      "[INFO]\tGene GUCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD81 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD47 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD80 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD79 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD78 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA103 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD77 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP94 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD75 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD74 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF717 has 0 variants, skipping\n",
      "[INFO]\tGene OPALIN has 0 variants, skipping\n",
      "[INFO]\tGene LRIG2-DT has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2 has 0 variants, skipping\n",
      "[INFO]\tGene WARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400499 has 0 variants, skipping\n",
      "[INFO]\tGene DUS4L-BCAP29 has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L has 0 variants, skipping\n",
      "[INFO]\tGene STAC3 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L3B has 0 variants, skipping\n",
      "[INFO]\tGene ITGB4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF205 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1B1 has 0 variants, skipping\n",
      "[INFO]\tGene IMMT has 0 variants, skipping\n",
      "[INFO]\tGene LUZP4 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D19 has 0 variants, skipping\n",
      "[INFO]\tGene FAM160B1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT80 has 0 variants, skipping\n",
      "[INFO]\tGene TOP3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01657 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K4 has 0 variants, skipping\n",
      "[INFO]\tGene PLD3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCG2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB14 has 0 variants, skipping\n",
      "[INFO]\tGene WIPI2 has 0 variants, skipping\n",
      "[INFO]\tGene ICA1L has 0 variants, skipping\n",
      "[INFO]\tGene TTC23 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC130 has 0 variants, skipping\n",
      "[INFO]\tGene GDF15 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC2 has 0 variants, skipping\n",
      "[INFO]\tGene LITAF has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP39 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01645 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01741 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD5 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB4 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EXO1 has 0 variants, skipping\n",
      "[INFO]\tGene MTFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC16 has 0 variants, skipping\n",
      "[INFO]\tGene FTX has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL2P has 0 variants, skipping\n",
      "[INFO]\tGene RASAL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4424 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL6 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD28 has 0 variants, skipping\n",
      "[INFO]\tGene HERC4 has 0 variants, skipping\n",
      "[INFO]\tGene ST14 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2L has 0 variants, skipping\n",
      "[INFO]\tGene SMG7 has 0 variants, skipping\n",
      "[INFO]\tGene FMC1-LUC7L2 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW3 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3F has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC9 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB3 has 0 variants, skipping\n",
      "[INFO]\tGene DMTN has 0 variants, skipping\n",
      "[INFO]\tGene ATG7 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT44 has 0 variants, skipping\n",
      "[INFO]\tGene STH has 0 variants, skipping\n",
      "[INFO]\tGene NKAPL has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF38B has 0 variants, skipping\n",
      "[INFO]\tGene SCML1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB5 has 0 variants, skipping\n",
      "[INFO]\tGene ADD1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3121 has 0 variants, skipping\n",
      "[INFO]\tGene AASDH has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5B has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH4 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1614-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX17 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF6 has 0 variants, skipping\n",
      "[INFO]\tGene DDTL has 0 variants, skipping\n",
      "[INFO]\tGene TDRKH has 0 variants, skipping\n",
      "[INFO]\tGene LCN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01732 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01699 has 0 variants, skipping\n",
      "[INFO]\tGene MITF has 0 variants, skipping\n",
      "[INFO]\tGene WDR44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF461 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL7 has 0 variants, skipping\n",
      "[INFO]\tGene PBRM1 has 0 variants, skipping\n",
      "[INFO]\tGene APOM has 0 variants, skipping\n",
      "[INFO]\tGene LINC01344 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00272 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf65 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK2 has 0 variants, skipping\n",
      "[INFO]\tGene CTPS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01686 has 0 variants, skipping\n",
      "[INFO]\tGene DMXL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01688 has 0 variants, skipping\n",
      "[INFO]\tGene SCN3B has 0 variants, skipping\n",
      "[INFO]\tGene OOEP has 0 variants, skipping\n",
      "[INFO]\tGene BANP has 0 variants, skipping\n",
      "[INFO]\tGene LOC647070 has 0 variants, skipping\n",
      "[INFO]\tGene PDIA6 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC3 has 0 variants, skipping\n",
      "[INFO]\tGene MTERF3 has 0 variants, skipping\n",
      "[INFO]\tGene DERL2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1H has 0 variants, skipping\n",
      "[INFO]\tGene SNX20 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E2A has 0 variants, skipping\n",
      "[INFO]\tGene LAMC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8N has 0 variants, skipping\n",
      "[INFO]\tGene C19orf48 has 0 variants, skipping\n",
      "[INFO]\tGene MICA has 0 variants, skipping\n",
      "[INFO]\tGene TBATA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF43 has 0 variants, skipping\n",
      "[INFO]\tGene MZF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF35 has 0 variants, skipping\n",
      "[INFO]\tGene SMG7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABR has 0 variants, skipping\n",
      "[INFO]\tGene TTTY15 has 0 variants, skipping\n",
      "[INFO]\tGene ABTB2 has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF511 has 0 variants, skipping\n",
      "[INFO]\tGene ISL2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNB1 has 0 variants, skipping\n",
      "[INFO]\tGene UIMC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD3 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD2 has 0 variants, skipping\n",
      "[INFO]\tGene RCC2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM13 has 0 variants, skipping\n",
      "[INFO]\tGene GPR63 has 0 variants, skipping\n",
      "[INFO]\tGene ADTRP has 0 variants, skipping\n",
      "[INFO]\tGene UNC93A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD33 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR2 has 0 variants, skipping\n",
      "[INFO]\tGene MMP17 has 0 variants, skipping\n",
      "[INFO]\tGene RAB4B has 0 variants, skipping\n",
      "[INFO]\tGene HSFX1 has 0 variants, skipping\n",
      "[INFO]\tGene RARB has 0 variants, skipping\n",
      "[INFO]\tGene TAOK2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf80 has 0 variants, skipping\n",
      "[INFO]\tGene MYLPF has 0 variants, skipping\n",
      "[INFO]\tGene LINC01633 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM220 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM5 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC138 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-279B7.1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01350 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR3 has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene DISP1 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CES1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KB1 has 0 variants, skipping\n",
      "[INFO]\tGene RTEL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644090 has 0 variants, skipping\n",
      "[INFO]\tGene PINLYP has 0 variants, skipping\n",
      "[INFO]\tGene EYS has 0 variants, skipping\n",
      "[INFO]\tGene MINDY1 has 0 variants, skipping\n",
      "[INFO]\tGene CIB1 has 0 variants, skipping\n",
      "[INFO]\tGene PACERR has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01036 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01037 has 0 variants, skipping\n",
      "[INFO]\tGene IL27RA has 0 variants, skipping\n",
      "[INFO]\tGene AKAP7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01680 has 0 variants, skipping\n",
      "[INFO]\tGene MIER2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4426 has 0 variants, skipping\n",
      "[INFO]\tGene GNPAT has 0 variants, skipping\n",
      "[INFO]\tGene MAF has 0 variants, skipping\n",
      "[INFO]\tGene LLGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC5A has 0 variants, skipping\n",
      "[INFO]\tGene CDHR3 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNFX1 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA18B has 0 variants, skipping\n",
      "[INFO]\tGene FGFRL1 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA1 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1L has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL has 0 variants, skipping\n",
      "[INFO]\tGene MIR1278 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01031 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01724 has 0 variants, skipping\n",
      "[INFO]\tGene CELSR3 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBIS has 0 variants, skipping\n",
      "[INFO]\tGene KIF1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OBP2A has 0 variants, skipping\n",
      "[INFO]\tGene SEC61A2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1C has 0 variants, skipping\n",
      "[INFO]\tGene SYTL3 has 0 variants, skipping\n",
      "[INFO]\tGene DTNB has 0 variants, skipping\n",
      "[INFO]\tGene GATD3B has 0 variants, skipping\n",
      "[INFO]\tGene ECPAS has 0 variants, skipping\n",
      "[INFO]\tGene DIS3 has 0 variants, skipping\n",
      "[INFO]\tGene BATF3 has 0 variants, skipping\n",
      "[INFO]\tGene PXMP2 has 0 variants, skipping\n",
      "[INFO]\tGene DISC1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB103B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF395 has 0 variants, skipping\n",
      "[INFO]\tGene CYTL1 has 0 variants, skipping\n",
      "[INFO]\tGene MYNN has 0 variants, skipping\n",
      "[INFO]\tGene LENEP has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5D has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5C has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6B has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN7 has 0 variants, skipping\n",
      "[INFO]\tGene MACROH2A2 has 0 variants, skipping\n",
      "[INFO]\tGene MARK1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP10 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF19 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A1HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPL42 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01222 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01221 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00862 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10 has 0 variants, skipping\n",
      "[INFO]\tGene MEIKIN has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CD226 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX2 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene AZIN2 has 0 variants, skipping\n",
      "[INFO]\tGene NOD2 has 0 variants, skipping\n",
      "[INFO]\tGene VBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP4 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEFL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02709 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINI2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6H has 0 variants, skipping\n",
      "[INFO]\tGene THAP5 has 0 variants, skipping\n",
      "[INFO]\tGene DYSF has 0 variants, skipping\n",
      "[INFO]\tGene CNST has 0 variants, skipping\n",
      "[INFO]\tGene LBHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MGST3 has 0 variants, skipping\n",
      "[INFO]\tGene MEOX1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM2 has 0 variants, skipping\n",
      "[INFO]\tGene LRP2 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH2L has 0 variants, skipping\n",
      "[INFO]\tGene MROH3P has 0 variants, skipping\n",
      "[INFO]\tGene NGLY1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO2A has 0 variants, skipping\n",
      "[INFO]\tGene IKBIP has 0 variants, skipping\n",
      "[INFO]\tGene GJD2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD14 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MID2 has 0 variants, skipping\n",
      "[INFO]\tGene CARD9 has 0 variants, skipping\n",
      "[INFO]\tGene BAG6 has 0 variants, skipping\n",
      "[INFO]\tGene COBL has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K8 has 0 variants, skipping\n",
      "[INFO]\tGene SPIB has 0 variants, skipping\n",
      "[INFO]\tGene HMOX2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB40 has 0 variants, skipping\n",
      "[INFO]\tGene AIF1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA3 has 0 variants, skipping\n",
      "[INFO]\tGene PPAN-P2RY11 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10P7 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5191 has 0 variants, skipping\n",
      "[INFO]\tGene HES6 has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TREM1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST12 has 0 variants, skipping\n",
      "[INFO]\tGene WSB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6739 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70H has 0 variants, skipping\n",
      "[INFO]\tGene SEC23B has 0 variants, skipping\n",
      "[INFO]\tGene AMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6740 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B2 has 0 variants, skipping\n",
      "[INFO]\tGene CPTP has 0 variants, skipping\n",
      "[INFO]\tGene ALK has 0 variants, skipping\n",
      "[INFO]\tGene PTPRVP has 0 variants, skipping\n",
      "[INFO]\tGene ELF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCCS has 0 variants, skipping\n",
      "[INFO]\tGene CTSG has 0 variants, skipping\n",
      "[INFO]\tGene GDA has 0 variants, skipping\n",
      "[INFO]\tGene LINC00997 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00673 has 0 variants, skipping\n",
      "[INFO]\tGene SBF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FEZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL5 has 0 variants, skipping\n",
      "[INFO]\tGene FENDRR has 0 variants, skipping\n",
      "[INFO]\tGene MIR4288 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4262 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4257 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4319 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3198-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320E has 0 variants, skipping\n",
      "[INFO]\tGene CPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CPM has 0 variants, skipping\n",
      "[INFO]\tGene CPE has 0 variants, skipping\n",
      "[INFO]\tGene CPB2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4EP has 0 variants, skipping\n",
      "[INFO]\tGene KRTCAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA12 has 0 variants, skipping\n",
      "[INFO]\tGene GSC has 0 variants, skipping\n",
      "[INFO]\tGene SPACA3 has 0 variants, skipping\n",
      "[INFO]\tGene TPPP2 has 0 variants, skipping\n",
      "[INFO]\tGene MALT1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RN has 0 variants, skipping\n",
      "[INFO]\tGene CBR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103B1 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRV1 has 0 variants, skipping\n",
      "[INFO]\tGene F5 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3IP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM3 has 0 variants, skipping\n",
      "[INFO]\tGene GPHN has 0 variants, skipping\n",
      "[INFO]\tGene GRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DACH1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3622A has 0 variants, skipping\n",
      "[INFO]\tGene MIR23C has 0 variants, skipping\n",
      "[INFO]\tGene JDP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3909 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01353 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01136 has 0 variants, skipping\n",
      "[INFO]\tGene GRID1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM168A has 0 variants, skipping\n",
      "[INFO]\tGene PMPCA has 0 variants, skipping\n",
      "[INFO]\tGene KANK1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA77 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00260 has 0 variants, skipping\n",
      "[INFO]\tGene CYLD has 0 variants, skipping\n",
      "[INFO]\tGene ADD3 has 0 variants, skipping\n",
      "[INFO]\tGene AGMO has 0 variants, skipping\n",
      "[INFO]\tGene NEO1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE1B has 0 variants, skipping\n",
      "[INFO]\tGene JMJD7-PLA2G4B has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF275 has 0 variants, skipping\n",
      "[INFO]\tGene ARRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB5 has 0 variants, skipping\n",
      "[INFO]\tGene CALCRL has 0 variants, skipping\n",
      "[INFO]\tGene CCNY has 0 variants, skipping\n",
      "[INFO]\tGene CMTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00303 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS6 has 0 variants, skipping\n",
      "[INFO]\tGene JADE2 has 0 variants, skipping\n",
      "[INFO]\tGene BOC has 0 variants, skipping\n",
      "[INFO]\tGene DDX60L has 0 variants, skipping\n",
      "[INFO]\tGene RABL2B has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B7 has 0 variants, skipping\n",
      "[INFO]\tGene TJAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM20 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM21 has 0 variants, skipping\n",
      "[INFO]\tGene CEP19 has 0 variants, skipping\n",
      "[INFO]\tGene DAB1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO1 has 0 variants, skipping\n",
      "[INFO]\tGene EPDR1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-C has 0 variants, skipping\n",
      "[INFO]\tGene RNF146 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS3 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1A has 0 variants, skipping\n",
      "[INFO]\tGene RHOH has 0 variants, skipping\n",
      "[INFO]\tGene CDH22 has 0 variants, skipping\n",
      "[INFO]\tGene ARL13B has 0 variants, skipping\n",
      "[INFO]\tGene GMEB1 has 0 variants, skipping\n",
      "[INFO]\tGene PSTPIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG2 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAV has 0 variants, skipping\n",
      "[INFO]\tGene PWWP2A has 0 variants, skipping\n",
      "[INFO]\tGene NCSTN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF334 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725023 has 0 variants, skipping\n",
      "[INFO]\tGene LCORL has 0 variants, skipping\n",
      "[INFO]\tGene PUF60 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284581 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF443 has 0 variants, skipping\n",
      "[INFO]\tGene GPA33 has 0 variants, skipping\n",
      "[INFO]\tGene PRKD3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103021295 has 0 variants, skipping\n",
      "[INFO]\tGene GPR141 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122B has 0 variants, skipping\n",
      "[INFO]\tGene THAP8 has 0 variants, skipping\n",
      "[INFO]\tGene PKN2 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6L has 0 variants, skipping\n",
      "[INFO]\tGene OTUB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDE1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS50 has 0 variants, skipping\n",
      "[INFO]\tGene TANC2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO33 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1C has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6769B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF66 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD5 has 0 variants, skipping\n",
      "[INFO]\tGene SPHK1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSA has 0 variants, skipping\n",
      "[INFO]\tGene USP6 has 0 variants, skipping\n",
      "[INFO]\tGene RNF10 has 0 variants, skipping\n",
      "[INFO]\tGene BLM has 0 variants, skipping\n",
      "[INFO]\tGene BCKDHB has 0 variants, skipping\n",
      "[INFO]\tGene BCHE has 0 variants, skipping\n",
      "[INFO]\tGene ATP7B has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG7 has 0 variants, skipping\n",
      "[INFO]\tGene URGCP-MRPS24 has 0 variants, skipping\n",
      "[INFO]\tGene MIER1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF195 has 0 variants, skipping\n",
      "[INFO]\tGene LIAS has 0 variants, skipping\n",
      "[INFO]\tGene CAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23A has 0 variants, skipping\n",
      "[INFO]\tGene PDIA5 has 0 variants, skipping\n",
      "[INFO]\tGene TOGARAM1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD3 has 0 variants, skipping\n",
      "[INFO]\tGene BCAS3 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR8 has 0 variants, skipping\n",
      "[INFO]\tGene GIN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDHR2 has 0 variants, skipping\n",
      "[INFO]\tGene SWT1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL6 has 0 variants, skipping\n",
      "[INFO]\tGene GPAM has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST8 has 0 variants, skipping\n",
      "[INFO]\tGene AZI2 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1A has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN10 has 0 variants, skipping\n",
      "[INFO]\tGene SQLE has 0 variants, skipping\n",
      "[INFO]\tGene TIMM10 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS2 has 0 variants, skipping\n",
      "[INFO]\tGene PJVK has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B2CHG has 0 variants, skipping\n",
      "[INFO]\tGene APBB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC148696 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01717 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01774 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01696 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01698 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY3AP has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A22 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPD has 0 variants, skipping\n",
      "[INFO]\tGene MIR5193 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A8 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPA1 has 0 variants, skipping\n",
      "[INFO]\tGene AGO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4260 has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE2 has 0 variants, skipping\n",
      "[INFO]\tGene RB1-DT has 0 variants, skipping\n",
      "[INFO]\tGene BANK1 has 0 variants, skipping\n",
      "[INFO]\tGene DEF6 has 0 variants, skipping\n",
      "[INFO]\tGene BCORL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK14 has 0 variants, skipping\n",
      "[INFO]\tGene MTBP has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A23 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf93 has 0 variants, skipping\n",
      "[INFO]\tGene SDF2L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00595 has 0 variants, skipping\n",
      "[INFO]\tGene PACRGL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF846 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM26 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene APOC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX4 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD5 has 0 variants, skipping\n",
      "[INFO]\tGene KAT14 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FUS has 0 variants, skipping\n",
      "[INFO]\tGene METTL21A has 0 variants, skipping\n",
      "[INFO]\tGene RNF6 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND8 has 0 variants, skipping\n",
      "[INFO]\tGene NIN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01693 has 0 variants, skipping\n",
      "[INFO]\tGene PKM has 0 variants, skipping\n",
      "[INFO]\tGene LOC91548 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH8 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT4 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene LPGAT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA9 has 0 variants, skipping\n",
      "[INFO]\tGene PRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene STK26 has 0 variants, skipping\n",
      "[INFO]\tGene INTS7 has 0 variants, skipping\n",
      "[INFO]\tGene IL31RA has 0 variants, skipping\n",
      "[INFO]\tGene AHCYL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21P28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02608 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGED4B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA16B has 0 variants, skipping\n",
      "[INFO]\tGene CD300A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF382 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY3 has 0 variants, skipping\n",
      "[INFO]\tGene SSX2IP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM176B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS28 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR2 has 0 variants, skipping\n",
      "[INFO]\tGene SACM1L has 0 variants, skipping\n",
      "[INFO]\tGene DEXI has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP200 has 0 variants, skipping\n",
      "[INFO]\tGene REM1 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS5 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN2 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B7 has 0 variants, skipping\n",
      "[INFO]\tGene GCFC2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A6 has 0 variants, skipping\n",
      "[INFO]\tGene ANGEL2 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4C has 0 variants, skipping\n",
      "[INFO]\tGene AKIRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP92 has 0 variants, skipping\n",
      "[INFO]\tGene IFFO2 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6 has 0 variants, skipping\n",
      "[INFO]\tGene ELAPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR155 has 0 variants, skipping\n",
      "[INFO]\tGene UNC45B has 0 variants, skipping\n",
      "[INFO]\tGene CASP7 has 0 variants, skipping\n",
      "[INFO]\tGene RIBC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX17 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHB2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L4 has 0 variants, skipping\n",
      "[INFO]\tGene ARFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KC1 has 0 variants, skipping\n",
      "[INFO]\tGene PROX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00538 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND12 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L2 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL4 has 0 variants, skipping\n",
      "[INFO]\tGene RPL34 has 0 variants, skipping\n",
      "[INFO]\tGene IL12RB2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0040 has 0 variants, skipping\n",
      "[INFO]\tGene CEP131 has 0 variants, skipping\n",
      "[INFO]\tGene NEU1 has 0 variants, skipping\n",
      "[INFO]\tGene NCF2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB2 has 0 variants, skipping\n",
      "[INFO]\tGene CALCOCO2 has 0 variants, skipping\n",
      "[INFO]\tGene AP3S2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS31 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPR has 0 variants, skipping\n",
      "[INFO]\tGene ANO1 has 0 variants, skipping\n",
      "[INFO]\tGene PPEF1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM15 has 0 variants, skipping\n",
      "[INFO]\tGene SGIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITSN2 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf60 has 0 variants, skipping\n",
      "[INFO]\tGene CPA4 has 0 variants, skipping\n",
      "[INFO]\tGene RCBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHN1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT6B1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01653 has 0 variants, skipping\n",
      "[INFO]\tGene KIFAP3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARF1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM41B has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F3 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01710 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1-DT has 0 variants, skipping\n",
      "[INFO]\tGene NUDT17 has 0 variants, skipping\n",
      "[INFO]\tGene STYX has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CA has 0 variants, skipping\n",
      "[INFO]\tGene MIA2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KTN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506388 has 0 variants, skipping\n",
      "[INFO]\tGene ATF7IP2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8H has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A1 has 0 variants, skipping\n",
      "[INFO]\tGene CBFB has 0 variants, skipping\n",
      "[INFO]\tGene BRMS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36B has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6ATAC35P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01352 has 0 variants, skipping\n",
      "[INFO]\tGene HLX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2D has 0 variants, skipping\n",
      "[INFO]\tGene LINC01655 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02257 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02474 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GYS1 has 0 variants, skipping\n",
      "[INFO]\tGene F3 has 0 variants, skipping\n",
      "[INFO]\tGene GRSF1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPG has 0 variants, skipping\n",
      "[INFO]\tGene PROZ has 0 variants, skipping\n",
      "[INFO]\tGene CAPG has 0 variants, skipping\n",
      "[INFO]\tGene FSBP has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1BP1L has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20H has 0 variants, skipping\n",
      "[INFO]\tGene VDR has 0 variants, skipping\n",
      "[INFO]\tGene FAM177B has 0 variants, skipping\n",
      "[INFO]\tGene IARS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPP8 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf54 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927164 has 0 variants, skipping\n",
      "[INFO]\tGene DAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY10 has 0 variants, skipping\n",
      "[INFO]\tGene NCR3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM164 has 0 variants, skipping\n",
      "[INFO]\tGene ESRP1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP3 has 0 variants, skipping\n",
      "[INFO]\tGene CDK19 has 0 variants, skipping\n",
      "[INFO]\tGene MORN1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLT1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4742 has 0 variants, skipping\n",
      "[INFO]\tGene MDGA2 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB3 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A3 has 0 variants, skipping\n",
      "[INFO]\tGene CLASP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD2L has 0 variants, skipping\n",
      "[INFO]\tGene PYCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MATR3 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN5 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA3 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH1 has 0 variants, skipping\n",
      "[INFO]\tGene BPGM has 0 variants, skipping\n",
      "[INFO]\tGene SREK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6741 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF106 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM216 has 0 variants, skipping\n",
      "[INFO]\tGene MZT2B has 0 variants, skipping\n",
      "[INFO]\tGene TVP23A has 0 variants, skipping\n",
      "[INFO]\tGene ACBD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB18 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC42 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1A2 has 0 variants, skipping\n",
      "[INFO]\tGene GPAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ATPAF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene HVCN1 has 0 variants, skipping\n",
      "[INFO]\tGene USP22 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC4 has 0 variants, skipping\n",
      "[INFO]\tGene SRP19 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKB-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLA has 0 variants, skipping\n",
      "[INFO]\tGene USP17L25 has 0 variants, skipping\n",
      "[INFO]\tGene PDE7A has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K13 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD4 has 0 variants, skipping\n",
      "[INFO]\tGene RNF144A has 0 variants, skipping\n",
      "[INFO]\tGene IGFN1 has 0 variants, skipping\n",
      "[INFO]\tGene LDLR has 0 variants, skipping\n",
      "[INFO]\tGene MLLT10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF678 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF847P has 0 variants, skipping\n",
      "[INFO]\tGene DDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CARNMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC-ROR has 0 variants, skipping\n",
      "[INFO]\tGene HDDC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF345 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5008 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985355 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3620 has 0 variants, skipping\n",
      "[INFO]\tGene RAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A7 has 0 variants, skipping\n",
      "[INFO]\tGene MOG has 0 variants, skipping\n",
      "[INFO]\tGene CFB has 0 variants, skipping\n",
      "[INFO]\tGene SBK2 has 0 variants, skipping\n",
      "[INFO]\tGene PODNL1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4B has 0 variants, skipping\n",
      "[INFO]\tGene IBA57-DT has 0 variants, skipping\n",
      "[INFO]\tGene OBSCN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BECN1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM32 has 0 variants, skipping\n",
      "[INFO]\tGene FHL5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-8 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP36 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF131 has 0 variants, skipping\n",
      "[INFO]\tGene OARD1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM136A has 0 variants, skipping\n",
      "[INFO]\tGene ABCD4 has 0 variants, skipping\n",
      "[INFO]\tGene PNISR has 0 variants, skipping\n",
      "[INFO]\tGene MIR6742 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4666A has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S5 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S6 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S7 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S8 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S10 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S11 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S12 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S13 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S14 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S15 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S16 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S17 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S9 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP5P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373159 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11A-SCG5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927478 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01682 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01736 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM17 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01737 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF568 has 0 variants, skipping\n",
      "[INFO]\tGene FBF1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C has 0 variants, skipping\n",
      "[INFO]\tGene WASHC3 has 0 variants, skipping\n",
      "[INFO]\tGene NOX4 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927604 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS12 has 0 variants, skipping\n",
      "[INFO]\tGene CASZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3C has 0 variants, skipping\n",
      "[INFO]\tGene CSNKA2IP has 0 variants, skipping\n",
      "[INFO]\tGene SMIM28 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP4B has 0 variants, skipping\n",
      "[INFO]\tGene SCN4B has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF25 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A2 has 0 variants, skipping\n",
      "[INFO]\tGene POGK has 0 variants, skipping\n",
      "[INFO]\tGene F9 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD2P2 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAX-DISC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00582 has 0 variants, skipping\n",
      "[INFO]\tGene APOC1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD2 has 0 variants, skipping\n",
      "[INFO]\tGene PNKD has 0 variants, skipping\n",
      "[INFO]\tGene RWDD3 has 0 variants, skipping\n",
      "[INFO]\tGene SYF2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD2 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN3 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM4 has 0 variants, skipping\n",
      "[INFO]\tGene PSME3 has 0 variants, skipping\n",
      "[INFO]\tGene IFI35 has 0 variants, skipping\n",
      "[INFO]\tGene DISC1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01745 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01744 has 0 variants, skipping\n",
      "[INFO]\tGene EGF has 0 variants, skipping\n",
      "[INFO]\tGene MIR4427 has 0 variants, skipping\n",
      "[INFO]\tGene AMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4671 has 0 variants, skipping\n",
      "[INFO]\tGene COA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A9 has 0 variants, skipping\n",
      "[INFO]\tGene FLNB has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A11 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene A1CF has 0 variants, skipping\n",
      "[INFO]\tGene SH2D6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927787 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01132 has 0 variants, skipping\n",
      "[INFO]\tGene PP2672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927851 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC4 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1L has 0 variants, skipping\n",
      "[INFO]\tGene SNORA14B has 0 variants, skipping\n",
      "[INFO]\tGene LIMS4 has 0 variants, skipping\n",
      "[INFO]\tGene IRF4 has 0 variants, skipping\n",
      "[INFO]\tGene ASCC1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3L has 0 variants, skipping\n",
      "[INFO]\tGene HTR2A has 0 variants, skipping\n",
      "[INFO]\tGene NPR2 has 0 variants, skipping\n",
      "[INFO]\tGene ARSA has 0 variants, skipping\n",
      "[INFO]\tGene FCGBP has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K2 has 0 variants, skipping\n",
      "[INFO]\tGene KTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR89B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4753 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7A has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2LI1 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS4 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND10 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM33 has 0 variants, skipping\n",
      "[INFO]\tGene GMNN has 0 variants, skipping\n",
      "[INFO]\tGene TRIM24 has 0 variants, skipping\n",
      "[INFO]\tGene STMN3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5B has 0 variants, skipping\n",
      "[INFO]\tGene IFT52 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD2 has 0 variants, skipping\n",
      "[INFO]\tGene FCHO1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPIND1 has 0 variants, skipping\n",
      "[INFO]\tGene HBG2 has 0 variants, skipping\n",
      "[INFO]\tGene HADHB has 0 variants, skipping\n",
      "[INFO]\tGene HADHA has 0 variants, skipping\n",
      "[INFO]\tGene MIR1537 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6872 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6778 has 0 variants, skipping\n",
      "[INFO]\tGene MDM1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS3 has 0 variants, skipping\n",
      "[INFO]\tGene GCM2 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CRLF1 has 0 variants, skipping\n",
      "[INFO]\tGene TBRG4 has 0 variants, skipping\n",
      "[INFO]\tGene CCPG1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH1 has 0 variants, skipping\n",
      "[INFO]\tGene ART1 has 0 variants, skipping\n",
      "[INFO]\tGene ARR3 has 0 variants, skipping\n",
      "[INFO]\tGene C1RL has 0 variants, skipping\n",
      "[INFO]\tGene RBP5 has 0 variants, skipping\n",
      "[INFO]\tGene DAXX has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130331 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS7P5 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN2 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1B has 0 variants, skipping\n",
      "[INFO]\tGene PTPN22 has 0 variants, skipping\n",
      "[INFO]\tGene NMT2 has 0 variants, skipping\n",
      "[INFO]\tGene H2AW has 0 variants, skipping\n",
      "[INFO]\tGene MIR190B has 0 variants, skipping\n",
      "[INFO]\tGene TP53 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8Q has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A has 0 variants, skipping\n",
      "[INFO]\tGene NFU1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF431 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR31HG has 0 variants, skipping\n",
      "[INFO]\tGene LAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5W2 has 0 variants, skipping\n",
      "[INFO]\tGene OR11L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3123 has 0 variants, skipping\n",
      "[INFO]\tGene SMN1 has 0 variants, skipping\n",
      "[INFO]\tGene SWAP70 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML2B has 0 variants, skipping\n",
      "[INFO]\tGene PUM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132202 has 0 variants, skipping\n",
      "[INFO]\tGene CES3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01347 has 0 variants, skipping\n",
      "[INFO]\tGene BCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF185 has 0 variants, skipping\n",
      "[INFO]\tGene UNKL has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5L has 0 variants, skipping\n",
      "[INFO]\tGene UBE2V1 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4677 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS7 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS3 has 0 variants, skipping\n",
      "[INFO]\tGene LBR has 0 variants, skipping\n",
      "[INFO]\tGene H3C7 has 0 variants, skipping\n",
      "[INFO]\tGene PSG3 has 0 variants, skipping\n",
      "[INFO]\tGene SSX5 has 0 variants, skipping\n",
      "[INFO]\tGene SSX3 has 0 variants, skipping\n",
      "[INFO]\tGene FABP7 has 0 variants, skipping\n",
      "[INFO]\tGene MEA1 has 0 variants, skipping\n",
      "[INFO]\tGene EAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TPRKB has 0 variants, skipping\n",
      "[INFO]\tGene SPDL1 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA100 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928068 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF26B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01743 has 0 variants, skipping\n",
      "[INFO]\tGene MC2R has 0 variants, skipping\n",
      "[INFO]\tGene CORO1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01341 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf83 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR72 has 0 variants, skipping\n",
      "[INFO]\tGene RNF144B has 0 variants, skipping\n",
      "[INFO]\tGene SPDYA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF438 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1L has 0 variants, skipping\n",
      "[INFO]\tGene MCM10 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA4 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF670-ZNF695 has 0 variants, skipping\n",
      "[INFO]\tGene MICALL2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ39095 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF436 has 0 variants, skipping\n",
      "[INFO]\tGene GPBAR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3916 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD2 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1A has 0 variants, skipping\n",
      "[INFO]\tGene SMIM33 has 0 variants, skipping\n",
      "[INFO]\tGene GCSAML-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SKP2 has 0 variants, skipping\n",
      "[INFO]\tGene HIP1R has 0 variants, skipping\n",
      "[INFO]\tGene ZNF570 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L1P has 0 variants, skipping\n",
      "[INFO]\tGene DPP9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M1P has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A4 has 0 variants, skipping\n",
      "[INFO]\tGene OSM has 0 variants, skipping\n",
      "[INFO]\tGene MTX2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH16 has 0 variants, skipping\n",
      "[INFO]\tGene OPCML has 0 variants, skipping\n",
      "[INFO]\tGene LYPD9P has 0 variants, skipping\n",
      "[INFO]\tGene CDKL4 has 0 variants, skipping\n",
      "[INFO]\tGene EGFR has 0 variants, skipping\n",
      "[INFO]\tGene TMEM183A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3124 has 0 variants, skipping\n",
      "[INFO]\tGene BICDL2 has 0 variants, skipping\n",
      "[INFO]\tGene CSAG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02564 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723376 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DMA has 0 variants, skipping\n",
      "[INFO]\tGene HAX1 has 0 variants, skipping\n",
      "[INFO]\tGene ECI2 has 0 variants, skipping\n",
      "[INFO]\tGene TAB1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAME has 0 variants, skipping\n",
      "[INFO]\tGene TOMM40 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5699 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT1 has 0 variants, skipping\n",
      "[INFO]\tGene GPAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927762 has 0 variants, skipping\n",
      "[INFO]\tGene HRAS has 0 variants, skipping\n",
      "[INFO]\tGene IDI2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNP1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00200 has 0 variants, skipping\n",
      "[INFO]\tGene ADARB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00700 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6072 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00701 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02645 has 0 variants, skipping\n",
      "[INFO]\tGene SPI1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1C has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E2 has 0 variants, skipping\n",
      "[INFO]\tGene LCE1C has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A12 has 0 variants, skipping\n",
      "[INFO]\tGene F11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF138 has 0 variants, skipping\n",
      "[INFO]\tGene NAT1 has 0 variants, skipping\n",
      "[INFO]\tGene HDX has 0 variants, skipping\n",
      "[INFO]\tGene GCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD142 has 0 variants, skipping\n",
      "[INFO]\tGene PACS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLK7 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4A has 0 variants, skipping\n",
      "[INFO]\tGene GRM4 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO3B has 0 variants, skipping\n",
      "[INFO]\tGene WDFY4 has 0 variants, skipping\n",
      "[INFO]\tGene ISG20L2 has 0 variants, skipping\n",
      "[INFO]\tGene DTYMK has 0 variants, skipping\n",
      "[INFO]\tGene USP46-DT has 0 variants, skipping\n",
      "[INFO]\tGene SFTA3 has 0 variants, skipping\n",
      "[INFO]\tGene MCRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL4 has 0 variants, skipping\n",
      "[INFO]\tGene AKT1S1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L1 has 0 variants, skipping\n",
      "[INFO]\tGene GPSM1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2F has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2 has 0 variants, skipping\n",
      "[INFO]\tGene PITRM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02669 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02639 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6078 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927964 has 0 variants, skipping\n",
      "[INFO]\tGene MIR339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00705 has 0 variants, skipping\n",
      "[INFO]\tGene RNF135 has 0 variants, skipping\n",
      "[INFO]\tGene NIT1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1E2 has 0 variants, skipping\n",
      "[INFO]\tGene CYB561D1 has 0 variants, skipping\n",
      "[INFO]\tGene OSTC has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C6P has 0 variants, skipping\n",
      "[INFO]\tGene ITGA6 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C8P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02561 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC2-KCTD14 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF41 has 0 variants, skipping\n",
      "[INFO]\tGene DPH1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB49 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX3 has 0 variants, skipping\n",
      "[INFO]\tGene RGMB has 0 variants, skipping\n",
      "[INFO]\tGene SPIDR has 0 variants, skipping\n",
      "[INFO]\tGene AHNAK has 0 variants, skipping\n",
      "[INFO]\tGene FKBP6 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA5 has 0 variants, skipping\n",
      "[INFO]\tGene ATF7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF15 has 0 variants, skipping\n",
      "[INFO]\tGene WIPI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM118A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX2 has 0 variants, skipping\n",
      "[INFO]\tGene SKA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3155A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3155B has 0 variants, skipping\n",
      "[INFO]\tGene COMMD9 has 0 variants, skipping\n",
      "[INFO]\tGene UGGT1 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD2B has 0 variants, skipping\n",
      "[INFO]\tGene TRMT2B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02649 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02656 has 0 variants, skipping\n",
      "[INFO]\tGene KCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3A1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN15 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIA has 0 variants, skipping\n",
      "[INFO]\tGene PRKCQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02648 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00706 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A7 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD129 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM52B has 0 variants, skipping\n",
      "[INFO]\tGene PLB1 has 0 variants, skipping\n",
      "[INFO]\tGene TLK2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00708 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02676 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928272 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00709 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02670 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-DT has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R1B has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2A has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00710 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC4 has 0 variants, skipping\n",
      "[INFO]\tGene PCMTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANTXR2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF19 has 0 variants, skipping\n",
      "[INFO]\tGene TENT4A has 0 variants, skipping\n",
      "[INFO]\tGene CLTCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCGN has 0 variants, skipping\n",
      "[INFO]\tGene ELOB has 0 variants, skipping\n",
      "[INFO]\tGene SSR3 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PLS3 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2B has 0 variants, skipping\n",
      "[INFO]\tGene NRON has 0 variants, skipping\n",
      "[INFO]\tGene MIR371B has 0 variants, skipping\n",
      "[INFO]\tGene PROSER2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEK has 0 variants, skipping\n",
      "[INFO]\tGene PLD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AK has 0 variants, skipping\n",
      "[INFO]\tGene EVX2 has 0 variants, skipping\n",
      "[INFO]\tGene PHTF2 has 0 variants, skipping\n",
      "[INFO]\tGene SUPV3L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Q has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL3 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83A has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN3 has 0 variants, skipping\n",
      "[INFO]\tGene PLAGL1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2C has 0 variants, skipping\n",
      "[INFO]\tGene FBXL22 has 0 variants, skipping\n",
      "[INFO]\tGene KSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM8 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L11 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB112 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928453 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4293 has 0 variants, skipping\n",
      "[INFO]\tGene AQP8 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC5 has 0 variants, skipping\n",
      "[INFO]\tGene XIAP has 0 variants, skipping\n",
      "[INFO]\tGene BIRC2 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC3 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR6 has 0 variants, skipping\n",
      "[INFO]\tGene DDX51 has 0 variants, skipping\n",
      "[INFO]\tGene LYNX1 has 0 variants, skipping\n",
      "[INFO]\tGene PLRG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR758 has 0 variants, skipping\n",
      "[INFO]\tGene UROC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYRFL has 0 variants, skipping\n",
      "[INFO]\tGene DRC3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3A has 0 variants, skipping\n",
      "[INFO]\tGene CINP has 0 variants, skipping\n",
      "[INFO]\tGene SSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT71 has 0 variants, skipping\n",
      "[INFO]\tGene RNF14 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1CP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1C1 has 0 variants, skipping\n",
      "[INFO]\tGene SPG7 has 0 variants, skipping\n",
      "[INFO]\tGene MORN2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAP30 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM2 has 0 variants, skipping\n",
      "[INFO]\tGene TACC1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf162 has 0 variants, skipping\n",
      "[INFO]\tGene CIRBP has 0 variants, skipping\n",
      "[INFO]\tGene XRRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD46 has 0 variants, skipping\n",
      "[INFO]\tGene LOXHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25C has 0 variants, skipping\n",
      "[INFO]\tGene LANCL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF3 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD65 has 0 variants, skipping\n",
      "[INFO]\tGene FAM240C has 0 variants, skipping\n",
      "[INFO]\tGene RAD21L1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4M has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSGALNACT1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3A has 0 variants, skipping\n",
      "[INFO]\tGene IL1R1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL55 has 0 variants, skipping\n",
      "[INFO]\tGene FAXC has 0 variants, skipping\n",
      "[INFO]\tGene GNB3 has 0 variants, skipping\n",
      "[INFO]\tGene ENO2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928834 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4675 has 0 variants, skipping\n",
      "[INFO]\tGene CST8 has 0 variants, skipping\n",
      "[INFO]\tGene ANKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene NEBL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02643 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6G has 0 variants, skipping\n",
      "[INFO]\tGene PCNA has 0 variants, skipping\n",
      "[INFO]\tGene PDE6D has 0 variants, skipping\n",
      "[INFO]\tGene PCK1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG14 has 0 variants, skipping\n",
      "[INFO]\tGene RPS19BP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130992 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP1 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF5 has 0 variants, skipping\n",
      "[INFO]\tGene SCHIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100499489 has 0 variants, skipping\n",
      "[INFO]\tGene POTEA has 0 variants, skipping\n",
      "[INFO]\tGene DMD has 0 variants, skipping\n",
      "[INFO]\tGene ERLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene WWC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376453 has 0 variants, skipping\n",
      "[INFO]\tGene MAOA has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN4 has 0 variants, skipping\n",
      "[INFO]\tGene VPS16 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A2 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP98 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF264 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR195 has 0 variants, skipping\n",
      "[INFO]\tGene LINCADL has 0 variants, skipping\n",
      "[INFO]\tGene PSMC3IP has 0 variants, skipping\n",
      "[INFO]\tGene MYH10 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA8 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF31 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene WSB1 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1 has 0 variants, skipping\n",
      "[INFO]\tGene OTX2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23 has 0 variants, skipping\n",
      "[INFO]\tGene ULK3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01516 has 0 variants, skipping\n",
      "[INFO]\tGene GPR158-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00836 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929073 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO2 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX has 0 variants, skipping\n",
      "[INFO]\tGene NAA40 has 0 variants, skipping\n",
      "[INFO]\tGene ECE1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929117 has 0 variants, skipping\n",
      "[INFO]\tGene FAM238A has 0 variants, skipping\n",
      "[INFO]\tGene FAM238B has 0 variants, skipping\n",
      "[INFO]\tGene PWWP3A has 0 variants, skipping\n",
      "[INFO]\tGene IRGM has 0 variants, skipping\n",
      "[INFO]\tGene MEF2B has 0 variants, skipping\n",
      "[INFO]\tGene CHAT has 0 variants, skipping\n",
      "[INFO]\tGene PARG has 0 variants, skipping\n",
      "[INFO]\tGene MFSD2A has 0 variants, skipping\n",
      "[INFO]\tGene PEG3 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L1 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF540 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA4 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF484 has 0 variants, skipping\n",
      "[INFO]\tGene GRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD1 has 0 variants, skipping\n",
      "[INFO]\tGene ITCH has 0 variants, skipping\n",
      "[INFO]\tGene ATG10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A18 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene POU1F1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT8 has 0 variants, skipping\n",
      "[INFO]\tGene CYREN has 0 variants, skipping\n",
      "[INFO]\tGene CCDC110 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD4 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7A has 0 variants, skipping\n",
      "[INFO]\tGene UBXN10 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf216 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF684 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR875 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF580 has 0 variants, skipping\n",
      "[INFO]\tGene GLYCTK has 0 variants, skipping\n",
      "[INFO]\tGene VLDLR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD86 has 0 variants, skipping\n",
      "[INFO]\tGene OBI1 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A3 has 0 variants, skipping\n",
      "[INFO]\tGene WFIKKN2 has 0 variants, skipping\n",
      "[INFO]\tGene ADRM1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP8 has 0 variants, skipping\n",
      "[INFO]\tGene XG has 0 variants, skipping\n",
      "[INFO]\tGene CNTN5 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7F2 has 0 variants, skipping\n",
      "[INFO]\tGene RCC1L has 0 variants, skipping\n",
      "[INFO]\tGene TNKS1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR642A has 0 variants, skipping\n",
      "[INFO]\tGene MID1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A6P has 0 variants, skipping\n",
      "[INFO]\tGene ARMC4P1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA5 has 0 variants, skipping\n",
      "[INFO]\tGene RIF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A4 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL4 has 0 variants, skipping\n",
      "[INFO]\tGene SERBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30B has 0 variants, skipping\n",
      "[INFO]\tGene GPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LY9 has 0 variants, skipping\n",
      "[INFO]\tGene CLK2 has 0 variants, skipping\n",
      "[INFO]\tGene FABP3 has 0 variants, skipping\n",
      "[INFO]\tGene ECHS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCS2 has 0 variants, skipping\n",
      "[INFO]\tGene MFNG has 0 variants, skipping\n",
      "[INFO]\tGene SNORD130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8086 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02652 has 0 variants, skipping\n",
      "[INFO]\tGene WAC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01517 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00837 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5A has 0 variants, skipping\n",
      "[INFO]\tGene SVIL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB10 has 0 variants, skipping\n",
      "[INFO]\tGene NRXN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR604 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929279 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7162 has 0 variants, skipping\n",
      "[INFO]\tGene KALRN has 0 variants, skipping\n",
      "[INFO]\tGene SVIL2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02644 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR2 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf56 has 0 variants, skipping\n",
      "[INFO]\tGene PRR30 has 0 variants, skipping\n",
      "[INFO]\tGene RPH3A has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF285 has 0 variants, skipping\n",
      "[INFO]\tGene TECRL has 0 variants, skipping\n",
      "[INFO]\tGene TSPO2 has 0 variants, skipping\n",
      "[INFO]\tGene GBP6 has 0 variants, skipping\n",
      "[INFO]\tGene SCYL2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8J has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ15 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ13 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ10 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNF1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA5 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA3 has 0 variants, skipping\n",
      "[INFO]\tGene CD82 has 0 variants, skipping\n",
      "[INFO]\tGene JUP has 0 variants, skipping\n",
      "[INFO]\tGene JUNB has 0 variants, skipping\n",
      "[INFO]\tGene JUN has 0 variants, skipping\n",
      "[INFO]\tGene JAG2 has 0 variants, skipping\n",
      "[INFO]\tGene IVD has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD16 has 0 variants, skipping\n",
      "[INFO]\tGene RNF19A has 0 variants, skipping\n",
      "[INFO]\tGene PLPP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102031319 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929431 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF7 has 0 variants, skipping\n",
      "[INFO]\tGene MSRB3 has 0 variants, skipping\n",
      "[INFO]\tGene STAC has 0 variants, skipping\n",
      "[INFO]\tGene SSX2 has 0 variants, skipping\n",
      "[INFO]\tGene SSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA86 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMCH1 has 0 variants, skipping\n",
      "[INFO]\tGene IL18RAP has 0 variants, skipping\n",
      "[INFO]\tGene PLIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CD96 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM4B has 0 variants, skipping\n",
      "[INFO]\tGene CHDH has 0 variants, skipping\n",
      "[INFO]\tGene MUC4 has 0 variants, skipping\n",
      "[INFO]\tGene PPT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00838 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18B has 0 variants, skipping\n",
      "[INFO]\tGene IL1RL1 has 0 variants, skipping\n",
      "[INFO]\tGene NLK has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R4 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS21 has 0 variants, skipping\n",
      "[INFO]\tGene USP25 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC4 has 0 variants, skipping\n",
      "[INFO]\tGene NUP153 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFB has 0 variants, skipping\n",
      "[INFO]\tGene VCL has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA1 has 0 variants, skipping\n",
      "[INFO]\tGene VCX has 0 variants, skipping\n",
      "[INFO]\tGene MIR3611 has 0 variants, skipping\n",
      "[INFO]\tGene BEND3 has 0 variants, skipping\n",
      "[INFO]\tGene KIF6 has 0 variants, skipping\n",
      "[INFO]\tGene SYT16 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF18 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A26 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BL has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5C has 0 variants, skipping\n",
      "[INFO]\tGene MREG has 0 variants, skipping\n",
      "[INFO]\tGene CZIB has 0 variants, skipping\n",
      "[INFO]\tGene ULK4 has 0 variants, skipping\n",
      "[INFO]\tGene RIC8B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4683 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT5 has 0 variants, skipping\n",
      "[INFO]\tGene MTIF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D3 has 0 variants, skipping\n",
      "[INFO]\tGene UNK has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B10 has 0 variants, skipping\n",
      "[INFO]\tGene CBFA2T2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRK has 0 variants, skipping\n",
      "[INFO]\tGene SIGIRR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33BP1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD6 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene DDI2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIEN1 has 0 variants, skipping\n",
      "[INFO]\tGene LLPH has 0 variants, skipping\n",
      "[INFO]\tGene GINS4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF6 has 0 variants, skipping\n",
      "[INFO]\tGene IFT122 has 0 variants, skipping\n",
      "[INFO]\tGene DPF3 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00667 has 0 variants, skipping\n",
      "[INFO]\tGene UCA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG17 has 0 variants, skipping\n",
      "[INFO]\tGene PCA3 has 0 variants, skipping\n",
      "[INFO]\tGene HULC has 0 variants, skipping\n",
      "[INFO]\tGene RNU11 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA10 has 0 variants, skipping\n",
      "[INFO]\tGene EMBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX2-OT has 0 variants, skipping\n",
      "[INFO]\tGene HCG22 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf37-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PNMT has 0 variants, skipping\n",
      "[INFO]\tGene UBAC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC190 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN5 has 0 variants, skipping\n",
      "[INFO]\tGene GMDS-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR876 has 0 variants, skipping\n",
      "[INFO]\tGene INTS9 has 0 variants, skipping\n",
      "[INFO]\tGene THEG has 0 variants, skipping\n",
      "[INFO]\tGene IL23A has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A3 has 0 variants, skipping\n",
      "[INFO]\tGene CD320 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH12 has 0 variants, skipping\n",
      "[INFO]\tGene RSF1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD2 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR6 has 0 variants, skipping\n",
      "[INFO]\tGene HUWE1 has 0 variants, skipping\n",
      "[INFO]\tGene COBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B7P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00999 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3BP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441666 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00839 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF37BP has 0 variants, skipping\n",
      "[INFO]\tGene IL6R has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33B has 0 variants, skipping\n",
      "[INFO]\tGene PRPF4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP2 has 0 variants, skipping\n",
      "[INFO]\tGene TXN has 0 variants, skipping\n",
      "[INFO]\tGene MST1R has 0 variants, skipping\n",
      "[INFO]\tGene LOC283028 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378269 has 0 variants, skipping\n",
      "[INFO]\tGene SCN8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02623 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01264 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF782 has 0 variants, skipping\n",
      "[INFO]\tGene ACE has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3J has 0 variants, skipping\n",
      "[INFO]\tGene CCDC58 has 0 variants, skipping\n",
      "[INFO]\tGene TRABD has 0 variants, skipping\n",
      "[INFO]\tGene PIGP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf141 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02658 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00619 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00840 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00841 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM72-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FADS2 has 0 variants, skipping\n",
      "[INFO]\tGene CD8B has 0 variants, skipping\n",
      "[INFO]\tGene RSU1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724323 has 0 variants, skipping\n",
      "[INFO]\tGene IPCEF1 has 0 variants, skipping\n",
      "[INFO]\tGene PLP2 has 0 variants, skipping\n",
      "[INFO]\tGene PLN has 0 variants, skipping\n",
      "[INFO]\tGene PLIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLB2 has 0 variants, skipping\n",
      "[INFO]\tGene ECE2 has 0 variants, skipping\n",
      "[INFO]\tGene OAS2 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM2 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP4 has 0 variants, skipping\n",
      "[INFO]\tGene SP4 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS35 has 0 variants, skipping\n",
      "[INFO]\tGene STK32B has 0 variants, skipping\n",
      "[INFO]\tGene DUSP22 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC4 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP6 has 0 variants, skipping\n",
      "[INFO]\tGene MT1HL1 has 0 variants, skipping\n",
      "[INFO]\tGene ECSCR has 0 variants, skipping\n",
      "[INFO]\tGene HIP1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PAWR has 0 variants, skipping\n",
      "[INFO]\tGene MCM3 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0319 has 0 variants, skipping\n",
      "[INFO]\tGene TPBGL has 0 variants, skipping\n",
      "[INFO]\tGene MPC1L has 0 variants, skipping\n",
      "[INFO]\tGene MICU1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R17 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN14 has 0 variants, skipping\n",
      "[INFO]\tGene EVPLL has 0 variants, skipping\n",
      "[INFO]\tGene VANGL1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX13 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD1 has 0 variants, skipping\n",
      "[INFO]\tGene PARGP1 has 0 variants, skipping\n",
      "[INFO]\tGene HEMK1 has 0 variants, skipping\n",
      "[INFO]\tGene UBAC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74C-1 has 0 variants, skipping\n",
      "[INFO]\tGene ELOC has 0 variants, skipping\n",
      "[INFO]\tGene COA8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A9 has 0 variants, skipping\n",
      "[INFO]\tGene CAPNS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM1 has 0 variants, skipping\n",
      "[INFO]\tGene ENOSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF302 has 0 variants, skipping\n",
      "[INFO]\tGene ANTXRLP1 has 0 variants, skipping\n",
      "[INFO]\tGene CUTA has 0 variants, skipping\n",
      "[INFO]\tGene SORCS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25BP has 0 variants, skipping\n",
      "[INFO]\tGene CDH12 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1P33 has 0 variants, skipping\n",
      "[INFO]\tGene CRHR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927699 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2P1 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1P2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD2B has 0 variants, skipping\n",
      "[INFO]\tGene SURF4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2I has 0 variants, skipping\n",
      "[INFO]\tGene RNF130 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB6 has 0 variants, skipping\n",
      "[INFO]\tGene GFM2 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF281 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR68 has 0 variants, skipping\n",
      "[INFO]\tGene IFT27 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD35 has 0 variants, skipping\n",
      "[INFO]\tGene M1AP has 0 variants, skipping\n",
      "[INFO]\tGene FUT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107001062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378292 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP6 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC9 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-5 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CTSLP2 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724593 has 0 variants, skipping\n",
      "[INFO]\tGene HEPACAM2 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP12P has 0 variants, skipping\n",
      "[INFO]\tGene UTS2R has 0 variants, skipping\n",
      "[INFO]\tGene AFF2 has 0 variants, skipping\n",
      "[INFO]\tGene STIM2 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP4 has 0 variants, skipping\n",
      "[INFO]\tGene IGBP1 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT3 has 0 variants, skipping\n",
      "[INFO]\tGene IFRD1 has 0 variants, skipping\n",
      "[INFO]\tGene PELI1 has 0 variants, skipping\n",
      "[INFO]\tGene RCN3 has 0 variants, skipping\n",
      "[INFO]\tGene CBX8 has 0 variants, skipping\n",
      "[INFO]\tGene AMOTL1 has 0 variants, skipping\n",
      "[INFO]\tGene FXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE28 has 0 variants, skipping\n",
      "[INFO]\tGene FAM209A has 0 variants, skipping\n",
      "[INFO]\tGene SPTB has 0 variants, skipping\n",
      "[INFO]\tGene MYH2 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2D has 0 variants, skipping\n",
      "[INFO]\tGene PPA2 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM2 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAP30L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DYRK1A has 0 variants, skipping\n",
      "[INFO]\tGene PFKM has 0 variants, skipping\n",
      "[INFO]\tGene CD36 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP22-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PASK has 0 variants, skipping\n",
      "[INFO]\tGene ADCK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4294 has 0 variants, skipping\n",
      "[INFO]\tGene FAM170B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSLN has 0 variants, skipping\n",
      "[INFO]\tGene STRN4 has 0 variants, skipping\n",
      "[INFO]\tGene MRM2 has 0 variants, skipping\n",
      "[INFO]\tGene NRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf71-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JPH3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC8A has 0 variants, skipping\n",
      "[INFO]\tGene PNLDC1 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKG has 0 variants, skipping\n",
      "[INFO]\tGene FBXO42 has 0 variants, skipping\n",
      "[INFO]\tGene LIN37 has 0 variants, skipping\n",
      "[INFO]\tGene RIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF183 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AB1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG1 has 0 variants, skipping\n",
      "[INFO]\tGene PDIA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCN2A has 0 variants, skipping\n",
      "[INFO]\tGene PRR20C has 0 variants, skipping\n",
      "[INFO]\tGene PRR20B has 0 variants, skipping\n",
      "[INFO]\tGene MYB has 0 variants, skipping\n",
      "[INFO]\tGene LTBP3 has 0 variants, skipping\n",
      "[INFO]\tGene DENND4C has 0 variants, skipping\n",
      "[INFO]\tGene PCBP3 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1A has 0 variants, skipping\n",
      "[INFO]\tGene GFI1B has 0 variants, skipping\n",
      "[INFO]\tGene FAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC31 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9 has 0 variants, skipping\n",
      "[INFO]\tGene RNF13 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLRB1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23B has 0 variants, skipping\n",
      "[INFO]\tGene TSHB has 0 variants, skipping\n",
      "[INFO]\tGene WDFY3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA2 has 0 variants, skipping\n",
      "[INFO]\tGene CYTOR has 0 variants, skipping\n",
      "[INFO]\tGene TBCB has 0 variants, skipping\n",
      "[INFO]\tGene PRDM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193A has 0 variants, skipping\n",
      "[INFO]\tGene MIR136 has 0 variants, skipping\n",
      "[INFO]\tGene MIR190A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74C-2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCD has 0 variants, skipping\n",
      "[INFO]\tGene FAM21EP has 0 variants, skipping\n",
      "[INFO]\tGene FRG2 has 0 variants, skipping\n",
      "[INFO]\tGene DPH2 has 0 variants, skipping\n",
      "[INFO]\tGene ASMTL has 0 variants, skipping\n",
      "[INFO]\tGene PDHB has 0 variants, skipping\n",
      "[INFO]\tGene SP7 has 0 variants, skipping\n",
      "[INFO]\tGene KIF21A has 0 variants, skipping\n",
      "[INFO]\tGene UCHL5 has 0 variants, skipping\n",
      "[INFO]\tGene SGMS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN6 has 0 variants, skipping\n",
      "[INFO]\tGene SPTY2D1OS has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN2 has 0 variants, skipping\n",
      "[INFO]\tGene CCM2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724719 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INAVA has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378305 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM247 has 0 variants, skipping\n",
      "[INFO]\tGene C3AR1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL4A has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF1 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF39 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM51 has 0 variants, skipping\n",
      "[INFO]\tGene NPLOC4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3C has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC12 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH4 has 0 variants, skipping\n",
      "[INFO]\tGene POMT2 has 0 variants, skipping\n",
      "[INFO]\tGene DPP7 has 0 variants, skipping\n",
      "[INFO]\tGene VPREB3 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABT1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6IP has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC8 has 0 variants, skipping\n",
      "[INFO]\tGene GREM1 has 0 variants, skipping\n",
      "[INFO]\tGene IL19 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD3 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC2 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEA has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF860 has 0 variants, skipping\n",
      "[INFO]\tGene RTKN has 0 variants, skipping\n",
      "[INFO]\tGene ARMC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378311 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F1 has 0 variants, skipping\n",
      "[INFO]\tGene UVSSA has 0 variants, skipping\n",
      "[INFO]\tGene MIR3924 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF10 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C2 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133CP has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA1 has 0 variants, skipping\n",
      "[INFO]\tGene AHRR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01553 has 0 variants, skipping\n",
      "[INFO]\tGene GBA3 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF3 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf22 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT2 has 0 variants, skipping\n",
      "[INFO]\tGene MOB1A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC87 has 0 variants, skipping\n",
      "[INFO]\tGene EDEM2 has 0 variants, skipping\n",
      "[INFO]\tGene PANK4 has 0 variants, skipping\n",
      "[INFO]\tGene CER1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB22 has 0 variants, skipping\n",
      "[INFO]\tGene WDR46 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM7 has 0 variants, skipping\n",
      "[INFO]\tGene FCMR has 0 variants, skipping\n",
      "[INFO]\tGene TXNL4B has 0 variants, skipping\n",
      "[INFO]\tGene LMO2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00845 has 0 variants, skipping\n",
      "[INFO]\tGene RELCH has 0 variants, skipping\n",
      "[INFO]\tGene YEATS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MED15 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM3 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23D1 has 0 variants, skipping\n",
      "[INFO]\tGene GLUL has 0 variants, skipping\n",
      "[INFO]\tGene EGR4 has 0 variants, skipping\n",
      "[INFO]\tGene EGR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283045 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3F has 0 variants, skipping\n",
      "[INFO]\tGene ZNF331 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN2L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM235 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP90 has 0 variants, skipping\n",
      "[INFO]\tGene TPGS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR203A has 0 variants, skipping\n",
      "[INFO]\tGene PM20D2 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX6 has 0 variants, skipping\n",
      "[INFO]\tGene BOD1 has 0 variants, skipping\n",
      "[INFO]\tGene BNIPL has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378330 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02671 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01515 has 0 variants, skipping\n",
      "[INFO]\tGene TRAK1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928961 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7151 has 0 variants, skipping\n",
      "[INFO]\tGene PTRH2 has 0 variants, skipping\n",
      "[INFO]\tGene DESI2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS36 has 0 variants, skipping\n",
      "[INFO]\tGene TAF9B has 0 variants, skipping\n",
      "[INFO]\tGene MRPL48 has 0 variants, skipping\n",
      "[INFO]\tGene PPME1 has 0 variants, skipping\n",
      "[INFO]\tGene FMO2 has 0 variants, skipping\n",
      "[INFO]\tGene MYD88 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P5 has 0 variants, skipping\n",
      "[INFO]\tGene ROBO2 has 0 variants, skipping\n",
      "[INFO]\tGene COG6 has 0 variants, skipping\n",
      "[INFO]\tGene XPO5 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9B has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM21 has 0 variants, skipping\n",
      "[INFO]\tGene UCK1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRA1 has 0 variants, skipping\n",
      "[INFO]\tGene GKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP5 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO2 has 0 variants, skipping\n",
      "[INFO]\tGene DNM3 has 0 variants, skipping\n",
      "[INFO]\tGene ANO9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS22 has 0 variants, skipping\n",
      "[INFO]\tGene COQ6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR573 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA1 has 0 variants, skipping\n",
      "[INFO]\tGene ECI2-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01619 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01138 has 0 variants, skipping\n",
      "[INFO]\tGene KRT23 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8D has 0 variants, skipping\n",
      "[INFO]\tGene MRPS2 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1L4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR61 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD98 has 0 variants, skipping\n",
      "[INFO]\tGene H3-4 has 0 variants, skipping\n",
      "[INFO]\tGene CDC7 has 0 variants, skipping\n",
      "[INFO]\tGene AXIN1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM187 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX3 has 0 variants, skipping\n",
      "[INFO]\tGene NAA10 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX2 has 0 variants, skipping\n",
      "[INFO]\tGene NRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYN3 has 0 variants, skipping\n",
      "[INFO]\tGene DPH7 has 0 variants, skipping\n",
      "[INFO]\tGene UCN has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2V2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2N has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2I has 0 variants, skipping\n",
      "[INFO]\tGene CCNG1 has 0 variants, skipping\n",
      "[INFO]\tGene ENO4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928994 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF763 has 0 variants, skipping\n",
      "[INFO]\tGene PALLD has 0 variants, skipping\n",
      "[INFO]\tGene ZNF446 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF806 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKN has 0 variants, skipping\n",
      "[INFO]\tGene EWSR1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG2 has 0 variants, skipping\n",
      "[INFO]\tGene LERFS has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB12 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL7 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15B has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4IP1 has 0 variants, skipping\n",
      "[INFO]\tGene GATM has 0 variants, skipping\n",
      "[INFO]\tGene C17orf113 has 0 variants, skipping\n",
      "[INFO]\tGene MOCS1 has 0 variants, skipping\n",
      "[INFO]\tGene MME has 0 variants, skipping\n",
      "[INFO]\tGene LMLN2 has 0 variants, skipping\n",
      "[INFO]\tGene PVALEF has 0 variants, skipping\n",
      "[INFO]\tGene TTC3 has 0 variants, skipping\n",
      "[INFO]\tGene TYSND1 has 0 variants, skipping\n",
      "[INFO]\tGene ME1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC3A2 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP4 has 0 variants, skipping\n",
      "[INFO]\tGene MDM4 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MEST has 0 variants, skipping\n",
      "[INFO]\tGene MDM2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K3 has 0 variants, skipping\n",
      "[INFO]\tGene MDK has 0 variants, skipping\n",
      "[INFO]\tGene ADAM11 has 0 variants, skipping\n",
      "[INFO]\tGene CD46 has 0 variants, skipping\n",
      "[INFO]\tGene COCH has 0 variants, skipping\n",
      "[INFO]\tGene RHEX has 0 variants, skipping\n",
      "[INFO]\tGene HEXD has 0 variants, skipping\n",
      "[INFO]\tGene KCTD16 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR3A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM10 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02622 has 0 variants, skipping\n",
      "[INFO]\tGene COX19 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3B2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP47 has 0 variants, skipping\n",
      "[INFO]\tGene RAD17 has 0 variants, skipping\n",
      "[INFO]\tGene MYOF has 0 variants, skipping\n",
      "[INFO]\tGene GNAS has 0 variants, skipping\n",
      "[INFO]\tGene SP110 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN2 has 0 variants, skipping\n",
      "[INFO]\tGene MYL1 has 0 variants, skipping\n",
      "[INFO]\tGene NR6A1 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH23-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2NL has 0 variants, skipping\n",
      "[INFO]\tGene MIR7152 has 0 variants, skipping\n",
      "[INFO]\tGene APLP2 has 0 variants, skipping\n",
      "[INFO]\tGene HES5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM230 has 0 variants, skipping\n",
      "[INFO]\tGene FAM171A1 has 0 variants, skipping\n",
      "[INFO]\tGene THEMIS has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L15 has 0 variants, skipping\n",
      "[INFO]\tGene PAK4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110A has 0 variants, skipping\n",
      "[INFO]\tGene POLDIP2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK10 has 0 variants, skipping\n",
      "[INFO]\tGene CCL28 has 0 variants, skipping\n",
      "[INFO]\tGene HPS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF5 has 0 variants, skipping\n",
      "[INFO]\tGene S100A8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR595 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4676 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC20 has 0 variants, skipping\n",
      "[INFO]\tGene LIPN has 0 variants, skipping\n",
      "[INFO]\tGene AQP12B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF219 has 0 variants, skipping\n",
      "[INFO]\tGene OPRM1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF5A has 0 variants, skipping\n",
      "[INFO]\tGene PRUNE1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11F has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIPL2 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB4 has 0 variants, skipping\n",
      "[INFO]\tGene NTMT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B has 0 variants, skipping\n",
      "[INFO]\tGene LYPD6 has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS54 has 0 variants, skipping\n",
      "[INFO]\tGene RSPRY1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3C has 0 variants, skipping\n",
      "[INFO]\tGene IFNGR2 has 0 variants, skipping\n",
      "[INFO]\tGene RANGRF has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC13 has 0 variants, skipping\n",
      "[INFO]\tGene IL16 has 0 variants, skipping\n",
      "[INFO]\tGene DKC1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC22 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSF1R has 0 variants, skipping\n",
      "[INFO]\tGene PHF19 has 0 variants, skipping\n",
      "[INFO]\tGene CENPJ has 0 variants, skipping\n",
      "[INFO]\tGene ZNF718 has 0 variants, skipping\n",
      "[INFO]\tGene FAM227A has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF121 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P4 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1P3 has 0 variants, skipping\n",
      "[INFO]\tGene GRB14 has 0 variants, skipping\n",
      "[INFO]\tGene GP5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGB1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2 has 0 variants, skipping\n",
      "[INFO]\tGene GNG4 has 0 variants, skipping\n",
      "[INFO]\tGene GPC3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL32 has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP has 0 variants, skipping\n",
      "[INFO]\tGene CRLF2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL4 has 0 variants, skipping\n",
      "[INFO]\tGene DNM2 has 0 variants, skipping\n",
      "[INFO]\tGene ATL2 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1A has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM69 has 0 variants, skipping\n",
      "[INFO]\tGene ISCU has 0 variants, skipping\n",
      "[INFO]\tGene PEX11A has 0 variants, skipping\n",
      "[INFO]\tGene LRRN3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137C has 0 variants, skipping\n",
      "[INFO]\tGene FOXN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR300 has 0 variants, skipping\n",
      "[INFO]\tGene PANTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A3 has 0 variants, skipping\n",
      "[INFO]\tGene ASZ1 has 0 variants, skipping\n",
      "[INFO]\tGene RGSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4L1 has 0 variants, skipping\n",
      "[INFO]\tGene TWNK has 0 variants, skipping\n",
      "[INFO]\tGene HPSE2 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC13-AS has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN12 has 0 variants, skipping\n",
      "[INFO]\tGene DPY30 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP44 has 0 variants, skipping\n",
      "[INFO]\tGene IRF3 has 0 variants, skipping\n",
      "[INFO]\tGene POLM has 0 variants, skipping\n",
      "[INFO]\tGene MRPL44 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3E has 0 variants, skipping\n",
      "[INFO]\tGene MIR3714 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3690 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR26 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4735 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4715 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4712 has 0 variants, skipping\n",
      "[INFO]\tGene MIR203B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3198-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4701 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4673 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4669 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4665 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4656 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4638 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4632 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378I has 0 variants, skipping\n",
      "[INFO]\tGene MIR4516 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4513 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2392 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4505 has 0 variants, skipping\n",
      "[INFO]\tGene RBM39 has 0 variants, skipping\n",
      "[INFO]\tGene BTG4 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP17 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD11 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P1-BPTFP1-KPNA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene POU6F1 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723439 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1G has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929165 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD172 has 0 variants, skipping\n",
      "[INFO]\tGene FAM219B has 0 variants, skipping\n",
      "[INFO]\tGene NUTM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929234 has 0 variants, skipping\n",
      "[INFO]\tGene MIR606 has 0 variants, skipping\n",
      "[INFO]\tGene LRMDA has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378367 has 0 variants, skipping\n",
      "[INFO]\tGene PKN1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCG has 0 variants, skipping\n",
      "[INFO]\tGene FCRL1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F3 has 0 variants, skipping\n",
      "[INFO]\tGene HGF has 0 variants, skipping\n",
      "[INFO]\tGene HRNR has 0 variants, skipping\n",
      "[INFO]\tGene TIAM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM149A has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L2 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene DLG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBIP has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00856 has 0 variants, skipping\n",
      "[INFO]\tGene MDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM27 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPHL has 0 variants, skipping\n",
      "[INFO]\tGene TRPM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729815 has 0 variants, skipping\n",
      "[INFO]\tGene PXN has 0 variants, skipping\n",
      "[INFO]\tGene USP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02679 has 0 variants, skipping\n",
      "[INFO]\tGene BEND3P3 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P21 has 0 variants, skipping\n",
      "[INFO]\tGene MBL1P has 0 variants, skipping\n",
      "[INFO]\tGene PLS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM254-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM254 has 0 variants, skipping\n",
      "[INFO]\tGene MTCL1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB4 has 0 variants, skipping\n",
      "[INFO]\tGene CETP has 0 variants, skipping\n",
      "[INFO]\tGene FZD5 has 0 variants, skipping\n",
      "[INFO]\tGene CHIT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130698 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRDL1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929574 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02655 has 0 variants, skipping\n",
      "[INFO]\tGene DMKN has 0 variants, skipping\n",
      "[INFO]\tGene NRG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02650 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM8 has 0 variants, skipping\n",
      "[INFO]\tGene STOX2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20C has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM19 has 0 variants, skipping\n",
      "[INFO]\tGene RNPEP has 0 variants, skipping\n",
      "[INFO]\tGene AGFG1 has 0 variants, skipping\n",
      "[INFO]\tGene LMBRD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01519 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02647 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929662 has 0 variants, skipping\n",
      "[INFO]\tGene GRID1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX46 has 0 variants, skipping\n",
      "[INFO]\tGene RBM17 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R16A has 0 variants, skipping\n",
      "[INFO]\tGene COX14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP19 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD5 has 0 variants, skipping\n",
      "[INFO]\tGene FATE1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCD2 has 0 variants, skipping\n",
      "[INFO]\tGene EAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP2 has 0 variants, skipping\n",
      "[INFO]\tGene SARNP has 0 variants, skipping\n",
      "[INFO]\tGene DIDO1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00863 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM245A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4678 has 0 variants, skipping\n",
      "[INFO]\tGene HES7 has 0 variants, skipping\n",
      "[INFO]\tGene DHX29 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD11 has 0 variants, skipping\n",
      "[INFO]\tGene CFL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene QPRT has 0 variants, skipping\n",
      "[INFO]\tGene SMIM5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP25 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC4 has 0 variants, skipping\n",
      "[INFO]\tGene AMELY has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-3 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX2 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R5 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4679-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4679-1 has 0 variants, skipping\n",
      "[INFO]\tGene WDPCP has 0 variants, skipping\n",
      "[INFO]\tGene SRRT has 0 variants, skipping\n",
      "[INFO]\tGene FAM219A has 0 variants, skipping\n",
      "[INFO]\tGene ARGFX has 0 variants, skipping\n",
      "[INFO]\tGene MYPOP has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9C has 0 variants, skipping\n",
      "[INFO]\tGene PRORP has 0 variants, skipping\n",
      "[INFO]\tGene FLJ37201 has 0 variants, skipping\n",
      "[INFO]\tGene TAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01374 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02653 has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_008559 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378430 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00502 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT9P1 has 0 variants, skipping\n",
      "[INFO]\tGene PIANP has 0 variants, skipping\n",
      "[INFO]\tGene HECTD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM65 has 0 variants, skipping\n",
      "[INFO]\tGene ADSL has 0 variants, skipping\n",
      "[INFO]\tGene ADRB3 has 0 variants, skipping\n",
      "[INFO]\tGene ADRB2 has 0 variants, skipping\n",
      "[INFO]\tGene SGCA has 0 variants, skipping\n",
      "[INFO]\tGene ADA has 0 variants, skipping\n",
      "[INFO]\tGene HECTD2 has 0 variants, skipping\n",
      "[INFO]\tGene TNKS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS39 has 0 variants, skipping\n",
      "[INFO]\tGene VAC14 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB130B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14A-ACY1 has 0 variants, skipping\n",
      "[INFO]\tGene MARK2P9 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK1 has 0 variants, skipping\n",
      "[INFO]\tGene RTL8A has 0 variants, skipping\n",
      "[INFO]\tGene PRR22 has 0 variants, skipping\n",
      "[INFO]\tGene AP3M2 has 0 variants, skipping\n",
      "[INFO]\tGene PFN1 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP3 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1C has 0 variants, skipping\n",
      "[INFO]\tGene AATK has 0 variants, skipping\n",
      "[INFO]\tGene PDCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCL1B has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1A has 0 variants, skipping\n",
      "[INFO]\tGene KLK4 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCG1 has 0 variants, skipping\n",
      "[INFO]\tGene OXA1L has 0 variants, skipping\n",
      "[INFO]\tGene OMD has 0 variants, skipping\n",
      "[INFO]\tGene NUCB2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS9D1 has 0 variants, skipping\n",
      "[INFO]\tGene ROR1 has 0 variants, skipping\n",
      "[INFO]\tGene NRF1 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM1 has 0 variants, skipping\n",
      "[INFO]\tGene NME4 has 0 variants, skipping\n",
      "[INFO]\tGene CLOCK has 0 variants, skipping\n",
      "[INFO]\tGene SNU13 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIL1 has 0 variants, skipping\n",
      "[INFO]\tGene IER2 has 0 variants, skipping\n",
      "[INFO]\tGene MINPP1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26A has 0 variants, skipping\n",
      "[INFO]\tGene WTAP has 0 variants, skipping\n",
      "[INFO]\tGene FRA10AC1 has 0 variants, skipping\n",
      "[INFO]\tGene LACC1 has 0 variants, skipping\n",
      "[INFO]\tGene PIPSL has 0 variants, skipping\n",
      "[INFO]\tGene PGPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GP2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF667 has 0 variants, skipping\n",
      "[INFO]\tGene UAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13D has 0 variants, skipping\n",
      "[INFO]\tGene TMX2 has 0 variants, skipping\n",
      "[INFO]\tGene GVQW3 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC6 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPDL has 0 variants, skipping\n",
      "[INFO]\tGene MOV10 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM34 has 0 variants, skipping\n",
      "[INFO]\tGene KDM1A has 0 variants, skipping\n",
      "[INFO]\tGene ADNP2 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB3 has 0 variants, skipping\n",
      "[INFO]\tGene CALU has 0 variants, skipping\n",
      "[INFO]\tGene RTL5 has 0 variants, skipping\n",
      "[INFO]\tGene SBK1 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR6 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS35L has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC3 has 0 variants, skipping\n",
      "[INFO]\tGene BMP2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX3-2 has 0 variants, skipping\n",
      "[INFO]\tGene BAK1 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE has 0 variants, skipping\n",
      "[INFO]\tGene APBA2 has 0 variants, skipping\n",
      "[INFO]\tGene TEK has 0 variants, skipping\n",
      "[INFO]\tGene MIR3157 has 0 variants, skipping\n",
      "[INFO]\tGene SUGCT has 0 variants, skipping\n",
      "[INFO]\tGene OR9G4 has 0 variants, skipping\n",
      "[INFO]\tGene OR9Q2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M8 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACR has 0 variants, skipping\n",
      "[INFO]\tGene ACLY has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4Y has 0 variants, skipping\n",
      "[INFO]\tGene WLS has 0 variants, skipping\n",
      "[INFO]\tGene BLNK has 0 variants, skipping\n",
      "[INFO]\tGene TEX14 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL3 has 0 variants, skipping\n",
      "[INFO]\tGene MALRD1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN8 has 0 variants, skipping\n",
      "[INFO]\tGene AJAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CX3CL1 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723665 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf50 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP19-SLIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268354 has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L1BP has 0 variants, skipping\n",
      "[INFO]\tGene PRSS47 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LETMD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4787 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4779 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4756 has 0 variants, skipping\n",
      "[INFO]\tGene MIR499B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4741 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4749 has 0 variants, skipping\n",
      "[INFO]\tGene ZBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF630 has 0 variants, skipping\n",
      "[INFO]\tGene NR1D2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMK1 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE7 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE3 has 0 variants, skipping\n",
      "[INFO]\tGene SPZ1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106B has 0 variants, skipping\n",
      "[INFO]\tGene FOXP3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC22 has 0 variants, skipping\n",
      "[INFO]\tGene SPON2 has 0 variants, skipping\n",
      "[INFO]\tGene BDNF has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO1 has 0 variants, skipping\n",
      "[INFO]\tGene FES has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL4 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP4 has 0 variants, skipping\n",
      "[INFO]\tGene ECM2 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-4 has 0 variants, skipping\n",
      "[INFO]\tGene DTX1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00866 has 0 variants, skipping\n",
      "[INFO]\tGene GRAP2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRF1 has 0 variants, skipping\n",
      "[INFO]\tGene COL8A2 has 0 variants, skipping\n",
      "[INFO]\tGene CD80 has 0 variants, skipping\n",
      "[INFO]\tGene CCR8 has 0 variants, skipping\n",
      "[INFO]\tGene CBX2 has 0 variants, skipping\n",
      "[INFO]\tGene RGN has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AY has 0 variants, skipping\n",
      "[INFO]\tGene VCY has 0 variants, skipping\n",
      "[INFO]\tGene CDY1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3F has 0 variants, skipping\n",
      "[INFO]\tGene LGALS7B has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R4 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2D has 0 variants, skipping\n",
      "[INFO]\tGene GRK6 has 0 variants, skipping\n",
      "[INFO]\tGene CD209 has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO2 has 0 variants, skipping\n",
      "[INFO]\tGene MYO18B has 0 variants, skipping\n",
      "[INFO]\tGene SPATA22 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L3 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A14 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf24 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R9B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4685 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-H has 0 variants, skipping\n",
      "[INFO]\tGene STX16 has 0 variants, skipping\n",
      "[INFO]\tGene CCL3L3 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT7L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6507 has 0 variants, skipping\n",
      "[INFO]\tGene SENP8 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE2B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM44 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01475 has 0 variants, skipping\n",
      "[INFO]\tGene EFHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP112 has 0 variants, skipping\n",
      "[INFO]\tGene TNNT2 has 0 variants, skipping\n",
      "[INFO]\tGene RXRB has 0 variants, skipping\n",
      "[INFO]\tGene TSC1 has 0 variants, skipping\n",
      "[INFO]\tGene DNMBP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC4H2 has 0 variants, skipping\n",
      "[INFO]\tGene NENF has 0 variants, skipping\n",
      "[INFO]\tGene RACGAP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM99A has 0 variants, skipping\n",
      "[INFO]\tGene NR1I3 has 0 variants, skipping\n",
      "[INFO]\tGene THAP9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCLL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF343 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22C has 0 variants, skipping\n",
      "[INFO]\tGene ASB11 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR3 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1L has 0 variants, skipping\n",
      "[INFO]\tGene PIGB has 0 variants, skipping\n",
      "[INFO]\tGene CHST10 has 0 variants, skipping\n",
      "[INFO]\tGene STX8 has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA12 has 0 variants, skipping\n",
      "[INFO]\tGene SYBU has 0 variants, skipping\n",
      "[INFO]\tGene GDAP1 has 0 variants, skipping\n",
      "[INFO]\tGene APOL1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1 has 0 variants, skipping\n",
      "[INFO]\tGene OLMALINC has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB13 has 0 variants, skipping\n",
      "[INFO]\tGene GRP has 0 variants, skipping\n",
      "[INFO]\tGene GYPC has 0 variants, skipping\n",
      "[INFO]\tGene EVX1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC45A has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A5 has 0 variants, skipping\n",
      "[INFO]\tGene TLX1NB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01514 has 0 variants, skipping\n",
      "[INFO]\tGene LBX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02681 has 0 variants, skipping\n",
      "[INFO]\tGene YY1AP1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8I2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A3 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene IGFLR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3158-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3158-2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC4 has 0 variants, skipping\n",
      "[INFO]\tGene PDE12 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD2 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA6 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C2 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf15 has 0 variants, skipping\n",
      "[INFO]\tGene RBM46 has 0 variants, skipping\n",
      "[INFO]\tGene CLDND1 has 0 variants, skipping\n",
      "[INFO]\tGene DUX4 has 0 variants, skipping\n",
      "[INFO]\tGene TFF1 has 0 variants, skipping\n",
      "[INFO]\tGene UBD has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP5 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2A has 0 variants, skipping\n",
      "[INFO]\tGene PRDX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD2 has 0 variants, skipping\n",
      "[INFO]\tGene CNBD2 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBID has 0 variants, skipping\n",
      "[INFO]\tGene MGAM has 0 variants, skipping\n",
      "[INFO]\tGene TIRAP has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A4 has 0 variants, skipping\n",
      "[INFO]\tGene TESMIN has 0 variants, skipping\n",
      "[INFO]\tGene HNF4A has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A3 has 0 variants, skipping\n",
      "[INFO]\tGene RPARP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SGK1 has 0 variants, skipping\n",
      "[INFO]\tGene SGSM3 has 0 variants, skipping\n",
      "[INFO]\tGene LY75-CD302 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP42-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY3-GNMT has 0 variants, skipping\n",
      "[INFO]\tGene BORCS7-ASMT has 0 variants, skipping\n",
      "[INFO]\tGene YIPF5 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO38 has 0 variants, skipping\n",
      "[INFO]\tGene CIITA has 0 variants, skipping\n",
      "[INFO]\tGene CADPS has 0 variants, skipping\n",
      "[INFO]\tGene PIGH has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2G has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF12-TNFSF13 has 0 variants, skipping\n",
      "[INFO]\tGene NOB1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP19 has 0 variants, skipping\n",
      "[INFO]\tGene TTC32 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6B has 0 variants, skipping\n",
      "[INFO]\tGene NDOR1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB29 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC74B has 0 variants, skipping\n",
      "[INFO]\tGene R3HCC1L has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2I1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF660-ZNF197 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM69 has 0 variants, skipping\n",
      "[INFO]\tGene VTA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD11 has 0 variants, skipping\n",
      "[INFO]\tGene TRMO has 0 variants, skipping\n",
      "[INFO]\tGene SHISA5 has 0 variants, skipping\n",
      "[INFO]\tGene JKAMP has 0 variants, skipping\n",
      "[INFO]\tGene CCDC174 has 0 variants, skipping\n",
      "[INFO]\tGene GSKIP has 0 variants, skipping\n",
      "[INFO]\tGene OSER1 has 0 variants, skipping\n",
      "[INFO]\tGene COX16 has 0 variants, skipping\n",
      "[INFO]\tGene BRF1 has 0 variants, skipping\n",
      "[INFO]\tGene HACD3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM217B has 0 variants, skipping\n",
      "[INFO]\tGene CDV3 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZB has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IMP4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR7 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT13 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR936 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4482 has 0 variants, skipping\n",
      "[INFO]\tGene HIKESHI has 0 variants, skipping\n",
      "[INFO]\tGene CFAP58-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02620 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0586 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02627 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02624 has 0 variants, skipping\n",
      "[INFO]\tGene SMKR1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRO has 0 variants, skipping\n",
      "[INFO]\tGene CXorf49B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01435 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM177 has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389831 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM135 has 0 variants, skipping\n",
      "[INFO]\tGene NDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM36 has 0 variants, skipping\n",
      "[INFO]\tGene ADD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM52 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF383 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A14 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM166B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4680 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF75D has 0 variants, skipping\n",
      "[INFO]\tGene FXYD4 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP6 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPC has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2A has 0 variants, skipping\n",
      "[INFO]\tGene PIGR has 0 variants, skipping\n",
      "[INFO]\tGene MIR6715B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6715A has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2GP has 0 variants, skipping\n",
      "[INFO]\tGene TRMT6 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B6 has 0 variants, skipping\n",
      "[INFO]\tGene PELO has 0 variants, skipping\n",
      "[INFO]\tGene COPZ1 has 0 variants, skipping\n",
      "[INFO]\tGene APIP has 0 variants, skipping\n",
      "[INFO]\tGene MRPL4 has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L2 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1 has 0 variants, skipping\n",
      "[INFO]\tGene TASP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0895L has 0 variants, skipping\n",
      "[INFO]\tGene NUDT6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103344931 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA87 has 0 variants, skipping\n",
      "[INFO]\tGene PRR13 has 0 variants, skipping\n",
      "[INFO]\tGene GPNMB has 0 variants, skipping\n",
      "[INFO]\tGene XPO7 has 0 variants, skipping\n",
      "[INFO]\tGene TSNARE1 has 0 variants, skipping\n",
      "[INFO]\tGene CWC27 has 0 variants, skipping\n",
      "[INFO]\tGene DHH has 0 variants, skipping\n",
      "[INFO]\tGene GBA2 has 0 variants, skipping\n",
      "[INFO]\tGene CWC22 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4483 has 0 variants, skipping\n",
      "[INFO]\tGene GBA has 0 variants, skipping\n",
      "[INFO]\tGene LDB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC186 has 0 variants, skipping\n",
      "[INFO]\tGene SFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW9 has 0 variants, skipping\n",
      "[INFO]\tGene PSD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLF1 has 0 variants, skipping\n",
      "[INFO]\tGene TCHP has 0 variants, skipping\n",
      "[INFO]\tGene FYTTD1 has 0 variants, skipping\n",
      "[INFO]\tGene RTL6 has 0 variants, skipping\n",
      "[INFO]\tGene MED10 has 0 variants, skipping\n",
      "[INFO]\tGene MRI1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC18 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47E has 0 variants, skipping\n",
      "[INFO]\tGene E2F3 has 0 variants, skipping\n",
      "[INFO]\tGene ELF5 has 0 variants, skipping\n",
      "[INFO]\tGene PLK5 has 0 variants, skipping\n",
      "[INFO]\tGene PARN has 0 variants, skipping\n",
      "[INFO]\tGene NTRK3 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIB has 0 variants, skipping\n",
      "[INFO]\tGene IL5RA has 0 variants, skipping\n",
      "[INFO]\tGene NUSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PELI3 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP3 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene BBS12 has 0 variants, skipping\n",
      "[INFO]\tGene SAA1 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL4 has 0 variants, skipping\n",
      "[INFO]\tGene CLINT1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC14 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4A has 0 variants, skipping\n",
      "[INFO]\tGene FAM53B has 0 variants, skipping\n",
      "[INFO]\tGene TTI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY1 has 0 variants, skipping\n",
      "[INFO]\tGene MLF1 has 0 variants, skipping\n",
      "[INFO]\tGene DCP1A has 0 variants, skipping\n",
      "[INFO]\tGene DIXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A24 has 0 variants, skipping\n",
      "[INFO]\tGene TERB1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM2 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD6 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927692 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02626 has 0 variants, skipping\n",
      "[INFO]\tGene CASKIN1 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSAD has 0 variants, skipping\n",
      "[INFO]\tGene HMCN2 has 0 variants, skipping\n",
      "[INFO]\tGene SYNM has 0 variants, skipping\n",
      "[INFO]\tGene SNORD158 has 0 variants, skipping\n",
      "[INFO]\tGene RAD23A has 0 variants, skipping\n",
      "[INFO]\tGene HSPB3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SRPX has 0 variants, skipping\n",
      "[INFO]\tGene RPL22L1 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf49 has 0 variants, skipping\n",
      "[INFO]\tGene MSL3 has 0 variants, skipping\n",
      "[INFO]\tGene CGB5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3663HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3663 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAS has 0 variants, skipping\n",
      "[INFO]\tGene OXCT1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00867 has 0 variants, skipping\n",
      "[INFO]\tGene KLF8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA19 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T6 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC17 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP13 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7B has 0 variants, skipping\n",
      "[INFO]\tGene BAHCC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4681 has 0 variants, skipping\n",
      "[INFO]\tGene NFASC has 0 variants, skipping\n",
      "[INFO]\tGene DAZ4 has 0 variants, skipping\n",
      "[INFO]\tGene GNAT3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXRED2 has 0 variants, skipping\n",
      "[INFO]\tGene DGLUCY has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1197 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1306 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302E has 0 variants, skipping\n",
      "[INFO]\tGene SNAPIN has 0 variants, skipping\n",
      "[INFO]\tGene ELDR has 0 variants, skipping\n",
      "[INFO]\tGene MIR663B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1181 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND1 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH5 has 0 variants, skipping\n",
      "[INFO]\tGene ATRAID has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4682 has 0 variants, skipping\n",
      "[INFO]\tGene RBM41 has 0 variants, skipping\n",
      "[INFO]\tGene WDR11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM13A has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A8 has 0 variants, skipping\n",
      "[INFO]\tGene HSH2D has 0 variants, skipping\n",
      "[INFO]\tGene PGBD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB33 has 0 variants, skipping\n",
      "[INFO]\tGene FGB has 0 variants, skipping\n",
      "[INFO]\tGene MMP2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ACP2 has 0 variants, skipping\n",
      "[INFO]\tGene FHL2 has 0 variants, skipping\n",
      "[INFO]\tGene OPA1 has 0 variants, skipping\n",
      "[INFO]\tGene A4GNT has 0 variants, skipping\n",
      "[INFO]\tGene TRO has 0 variants, skipping\n",
      "[INFO]\tGene LOC339685 has 0 variants, skipping\n",
      "[INFO]\tGene MIS18A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC93 has 0 variants, skipping\n",
      "[INFO]\tGene PUS7 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM234B has 0 variants, skipping\n",
      "[INFO]\tGene ATE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASPH has 0 variants, skipping\n",
      "[INFO]\tGene CCL20 has 0 variants, skipping\n",
      "[INFO]\tGene SCO1 has 0 variants, skipping\n",
      "[INFO]\tGene RO60 has 0 variants, skipping\n",
      "[INFO]\tGene INO80 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP8 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3I has 0 variants, skipping\n",
      "[INFO]\tGene PURB has 0 variants, skipping\n",
      "[INFO]\tGene GABRG3 has 0 variants, skipping\n",
      "[INFO]\tGene PSIP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM14 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3F has 0 variants, skipping\n",
      "[INFO]\tGene GK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF670 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC102A has 0 variants, skipping\n",
      "[INFO]\tGene MACIR has 0 variants, skipping\n",
      "[INFO]\tGene UCN2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF502 has 0 variants, skipping\n",
      "[INFO]\tGene PIGS has 0 variants, skipping\n",
      "[INFO]\tGene TIGD7 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF682 has 0 variants, skipping\n",
      "[INFO]\tGene LDHAL6B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF101 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB9 has 0 variants, skipping\n",
      "[INFO]\tGene BMERB1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-4 has 0 variants, skipping\n",
      "[INFO]\tGene BCOR has 0 variants, skipping\n",
      "[INFO]\tGene DEFB123 has 0 variants, skipping\n",
      "[INFO]\tGene PMP22 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A7 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB119 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL1 has 0 variants, skipping\n",
      "[INFO]\tGene UBA1 has 0 variants, skipping\n",
      "[INFO]\tGene SFTA1P has 0 variants, skipping\n",
      "[INFO]\tGene LOC646214 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24B-CUZD1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24B has 0 variants, skipping\n",
      "[INFO]\tGene NSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene CHIC1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM23 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF9 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L has 0 variants, skipping\n",
      "[INFO]\tGene H1-5 has 0 variants, skipping\n",
      "[INFO]\tGene H1-4 has 0 variants, skipping\n",
      "[INFO]\tGene H1-3 has 0 variants, skipping\n",
      "[INFO]\tGene H1-2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-0 has 0 variants, skipping\n",
      "[INFO]\tGene GZMM has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H1 has 0 variants, skipping\n",
      "[INFO]\tGene GSC2 has 0 variants, skipping\n",
      "[INFO]\tGene GRPR has 0 variants, skipping\n",
      "[INFO]\tGene PDIA3 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L2 has 0 variants, skipping\n",
      "[INFO]\tGene NFIB has 0 variants, skipping\n",
      "[INFO]\tGene DSN1 has 0 variants, skipping\n",
      "[INFO]\tGene BHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B5 has 0 variants, skipping\n",
      "[INFO]\tGene MLH3 has 0 variants, skipping\n",
      "[INFO]\tGene NBEA has 0 variants, skipping\n",
      "[INFO]\tGene LIMK2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC62 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4296 has 0 variants, skipping\n",
      "[INFO]\tGene TEX36-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283038 has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1-DT has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1 has 0 variants, skipping\n",
      "[INFO]\tGene STKLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A14 has 0 variants, skipping\n",
      "[INFO]\tGene SENP7 has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NFILZ has 0 variants, skipping\n",
      "[INFO]\tGene OTOGL has 0 variants, skipping\n",
      "[INFO]\tGene FAM120AOS has 0 variants, skipping\n",
      "[INFO]\tGene MCRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene FANK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00601 has 0 variants, skipping\n",
      "[INFO]\tGene LEPR has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6E has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD4 has 0 variants, skipping\n",
      "[INFO]\tGene ERAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728158 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A4A has 0 variants, skipping\n",
      "[INFO]\tGene HINFP has 0 variants, skipping\n",
      "[INFO]\tGene GRTP1 has 0 variants, skipping\n",
      "[INFO]\tGene TKFC has 0 variants, skipping\n",
      "[INFO]\tGene PRH1 has 0 variants, skipping\n",
      "[INFO]\tGene USP16 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1 has 0 variants, skipping\n",
      "[INFO]\tGene CDYL has 0 variants, skipping\n",
      "[INFO]\tGene KIF2A has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ2 has 0 variants, skipping\n",
      "[INFO]\tGene GHRL has 0 variants, skipping\n",
      "[INFO]\tGene ACP6 has 0 variants, skipping\n",
      "[INFO]\tGene TACO1 has 0 variants, skipping\n",
      "[INFO]\tGene AS-PTPRE has 0 variants, skipping\n",
      "[INFO]\tGene MIR455 has 0 variants, skipping\n",
      "[INFO]\tGene MIR532 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01163 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02667 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4297 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf143 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE7P has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01164 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINH1 has 0 variants, skipping\n",
      "[INFO]\tGene XRN1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX2 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC1 has 0 variants, skipping\n",
      "[INFO]\tGene GPM6B has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5L has 0 variants, skipping\n",
      "[INFO]\tGene HP has 0 variants, skipping\n",
      "[INFO]\tGene CBLIF has 0 variants, skipping\n",
      "[INFO]\tGene CNGA2 has 0 variants, skipping\n",
      "[INFO]\tGene DPF1 has 0 variants, skipping\n",
      "[INFO]\tGene MED24 has 0 variants, skipping\n",
      "[INFO]\tGene MTDH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01166 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01167 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01168 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB4 has 0 variants, skipping\n",
      "[INFO]\tGene POTEH has 0 variants, skipping\n",
      "[INFO]\tGene MIR202HG has 0 variants, skipping\n",
      "[INFO]\tGene TBCD has 0 variants, skipping\n",
      "[INFO]\tGene TULP4 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00184 has 0 variants, skipping\n",
      "[INFO]\tGene MPPE1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM35 has 0 variants, skipping\n",
      "[INFO]\tGene NOS1AP has 0 variants, skipping\n",
      "[INFO]\tGene SART3 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3944 has 0 variants, skipping\n",
      "[INFO]\tGene FADS3 has 0 variants, skipping\n",
      "[INFO]\tGene NR1D1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM22 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1C has 0 variants, skipping\n",
      "[INFO]\tGene SCART1 has 0 variants, skipping\n",
      "[INFO]\tGene CUX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC117779438 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1109 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE10A has 0 variants, skipping\n",
      "[INFO]\tGene TUT7 has 0 variants, skipping\n",
      "[INFO]\tGene AGPS has 0 variants, skipping\n",
      "[INFO]\tGene PNMA5 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01001 has 0 variants, skipping\n",
      "[INFO]\tGene APOH has 0 variants, skipping\n",
      "[INFO]\tGene APOE has 0 variants, skipping\n",
      "[INFO]\tGene APOC3 has 0 variants, skipping\n",
      "[INFO]\tGene AMHR2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPA has 0 variants, skipping\n",
      "[INFO]\tGene DVL1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP5 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF13B has 0 variants, skipping\n",
      "[INFO]\tGene SLC35C2 has 0 variants, skipping\n",
      "[INFO]\tGene ERP44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF365 has 0 variants, skipping\n",
      "[INFO]\tGene CMTR1 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC4 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L24 has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP3 has 0 variants, skipping\n",
      "[INFO]\tGene THEM4 has 0 variants, skipping\n",
      "[INFO]\tGene TGM7 has 0 variants, skipping\n",
      "[INFO]\tGene STAT1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD4 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN2 has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR210HG has 0 variants, skipping\n",
      "[INFO]\tGene LOC143666 has 0 variants, skipping\n",
      "[INFO]\tGene KIF21B has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1B has 0 variants, skipping\n",
      "[INFO]\tGene DCT has 0 variants, skipping\n",
      "[INFO]\tGene CNN3 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A6 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR4 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI3K has 0 variants, skipping\n",
      "[INFO]\tGene ZFR has 0 variants, skipping\n",
      "[INFO]\tGene IER3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene MDH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC171391 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL3 has 0 variants, skipping\n",
      "[INFO]\tGene NNMT has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA52 has 0 variants, skipping\n",
      "[INFO]\tGene MRAS has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D31 has 0 variants, skipping\n",
      "[INFO]\tGene IDE has 0 variants, skipping\n",
      "[INFO]\tGene SMIM21 has 0 variants, skipping\n",
      "[INFO]\tGene CNGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2B has 0 variants, skipping\n",
      "[INFO]\tGene CNGA1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCNKB has 0 variants, skipping\n",
      "[INFO]\tGene CLCN5 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P14 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL9 has 0 variants, skipping\n",
      "[INFO]\tGene RGS8 has 0 variants, skipping\n",
      "[INFO]\tGene ATG14 has 0 variants, skipping\n",
      "[INFO]\tGene WDTC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6744 has 0 variants, skipping\n",
      "[INFO]\tGene TOLLIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBRSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4B has 0 variants, skipping\n",
      "[INFO]\tGene FAM13C has 0 variants, skipping\n",
      "[INFO]\tGene IGSF1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F12 has 0 variants, skipping\n",
      "[INFO]\tGene ANGEL1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCH has 0 variants, skipping\n",
      "[INFO]\tGene MIR4298 has 0 variants, skipping\n",
      "[INFO]\tGene LIPJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR7847 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01150 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCB1 has 0 variants, skipping\n",
      "[INFO]\tGene SFI1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF11 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34C has 0 variants, skipping\n",
      "[INFO]\tGene NKX1-2 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM114 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A15 has 0 variants, skipping\n",
      "[INFO]\tGene NACAD has 0 variants, skipping\n",
      "[INFO]\tGene PLAUR has 0 variants, skipping\n",
      "[INFO]\tGene KRTCAP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF615 has 0 variants, skipping\n",
      "[INFO]\tGene MMP14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD131 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01219 has 0 variants, skipping\n",
      "[INFO]\tGene JAM3 has 0 variants, skipping\n",
      "[INFO]\tGene LPXN has 0 variants, skipping\n",
      "[INFO]\tGene UBAP2L has 0 variants, skipping\n",
      "[INFO]\tGene MET has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF40 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX6 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C3 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK4 has 0 variants, skipping\n",
      "[INFO]\tGene CD81-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SUN2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN2 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM5 has 0 variants, skipping\n",
      "[INFO]\tGene FIBCD1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85C has 0 variants, skipping\n",
      "[INFO]\tGene OCA2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1DN has 0 variants, skipping\n",
      "[INFO]\tGene TMEM259 has 0 variants, skipping\n",
      "[INFO]\tGene ACP5 has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF17 has 0 variants, skipping\n",
      "[INFO]\tGene BMP4 has 0 variants, skipping\n",
      "[INFO]\tGene BCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA54 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM100 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1AIP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T2 has 0 variants, skipping\n",
      "[INFO]\tGene REST has 0 variants, skipping\n",
      "[INFO]\tGene ADAR has 0 variants, skipping\n",
      "[INFO]\tGene ENO3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR644A has 0 variants, skipping\n",
      "[INFO]\tGene OR7E12P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927708 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM9 has 0 variants, skipping\n",
      "[INFO]\tGene ABI1 has 0 variants, skipping\n",
      "[INFO]\tGene ART5 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF2 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SOCS3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCSH has 0 variants, skipping\n",
      "[INFO]\tGene DPP4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF3 has 0 variants, skipping\n",
      "[INFO]\tGene CLU has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NRP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM41 has 0 variants, skipping\n",
      "[INFO]\tGene DLST has 0 variants, skipping\n",
      "[INFO]\tGene CHEK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1298 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441204 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A18AS has 0 variants, skipping\n",
      "[INFO]\tGene IGF2 has 0 variants, skipping\n",
      "[INFO]\tGene EZH1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2A has 0 variants, skipping\n",
      "[INFO]\tGene H2AJ has 0 variants, skipping\n",
      "[INFO]\tGene PP2D1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA11 has 0 variants, skipping\n",
      "[INFO]\tGene LPO has 0 variants, skipping\n",
      "[INFO]\tGene MIR105-2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K9 has 0 variants, skipping\n",
      "[INFO]\tGene CALD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4687 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene SYT6 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4D has 0 variants, skipping\n",
      "[INFO]\tGene HAUS4 has 0 variants, skipping\n",
      "[INFO]\tGene PMS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA13 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF6 has 0 variants, skipping\n",
      "[INFO]\tGene HBBP1 has 0 variants, skipping\n",
      "[INFO]\tGene BGLT3 has 0 variants, skipping\n",
      "[INFO]\tGene USP39 has 0 variants, skipping\n",
      "[INFO]\tGene TEX13A has 0 variants, skipping\n",
      "[INFO]\tGene OLFM5P has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD42 has 0 variants, skipping\n",
      "[INFO]\tGene EYA2 has 0 variants, skipping\n",
      "[INFO]\tGene SREBF2 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF536 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM207A has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL7A has 0 variants, skipping\n",
      "[INFO]\tGene ACTL7B has 0 variants, skipping\n",
      "[INFO]\tGene SMR3B has 0 variants, skipping\n",
      "[INFO]\tGene CFHR4 has 0 variants, skipping\n",
      "[INFO]\tGene EDDM3A has 0 variants, skipping\n",
      "[INFO]\tGene FGL2 has 0 variants, skipping\n",
      "[INFO]\tGene NMU has 0 variants, skipping\n",
      "[INFO]\tGene ME3 has 0 variants, skipping\n",
      "[INFO]\tGene CTPS1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf27A has 0 variants, skipping\n",
      "[INFO]\tGene CENPS-CORT has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3L has 0 variants, skipping\n",
      "[INFO]\tGene RPL19 has 0 variants, skipping\n",
      "[INFO]\tGene REEP2 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1I2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF212 has 0 variants, skipping\n",
      "[INFO]\tGene GVINP1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD1 has 0 variants, skipping\n",
      "[INFO]\tGene AUH has 0 variants, skipping\n",
      "[INFO]\tGene MSR1 has 0 variants, skipping\n",
      "[INFO]\tGene MSN has 0 variants, skipping\n",
      "[INFO]\tGene MSMB has 0 variants, skipping\n",
      "[INFO]\tGene MTAP has 0 variants, skipping\n",
      "[INFO]\tGene TOMM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506258 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene GGT1 has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF2 has 0 variants, skipping\n",
      "[INFO]\tGene PCLAF has 0 variants, skipping\n",
      "[INFO]\tGene JADE3 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC136 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN1 has 0 variants, skipping\n",
      "[INFO]\tGene PMEL has 0 variants, skipping\n",
      "[INFO]\tGene NPL has 0 variants, skipping\n",
      "[INFO]\tGene SYNPR has 0 variants, skipping\n",
      "[INFO]\tGene TESPA1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5E1P has 0 variants, skipping\n",
      "[INFO]\tGene LOC283299 has 0 variants, skipping\n",
      "[INFO]\tGene CASC23 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH11 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2B has 0 variants, skipping\n",
      "[INFO]\tGene TUB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G2 has 0 variants, skipping\n",
      "[INFO]\tGene PBX4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A3 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC102B has 0 variants, skipping\n",
      "[INFO]\tGene DYTN has 0 variants, skipping\n",
      "[INFO]\tGene MEX3A has 0 variants, skipping\n",
      "[INFO]\tGene FMN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA2 has 0 variants, skipping\n",
      "[INFO]\tGene LRP5L has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP299 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724784 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D2A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIM1 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM1 has 0 variants, skipping\n",
      "[INFO]\tGene AUNIP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5691 has 0 variants, skipping\n",
      "[INFO]\tGene KRT8P41 has 0 variants, skipping\n",
      "[INFO]\tGene TP53BP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF257 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA23 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644656 has 0 variants, skipping\n",
      "[INFO]\tGene ZAP70 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23 has 0 variants, skipping\n",
      "[INFO]\tGene COMT has 0 variants, skipping\n",
      "[INFO]\tGene BRCA1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH2 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf204 has 0 variants, skipping\n",
      "[INFO]\tGene AGO3 has 0 variants, skipping\n",
      "[INFO]\tGene URGCP has 0 variants, skipping\n",
      "[INFO]\tGene MED9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928008 has 0 variants, skipping\n",
      "[INFO]\tGene CAND1.11 has 0 variants, skipping\n",
      "[INFO]\tGene RRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF2RA has 0 variants, skipping\n",
      "[INFO]\tGene COMMD7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4485 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf43 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA11 has 0 variants, skipping\n",
      "[INFO]\tGene CRACDL has 0 variants, skipping\n",
      "[INFO]\tGene FREM2 has 0 variants, skipping\n",
      "[INFO]\tGene GADL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12D has 0 variants, skipping\n",
      "[INFO]\tGene C1orf174 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL18 has 0 variants, skipping\n",
      "[INFO]\tGene CYP27A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP26A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP24A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP11A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP7A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4B1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4A11 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A5 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2J2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2E1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD97 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928053 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL3 has 0 variants, skipping\n",
      "[INFO]\tGene NRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8070 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF99 has 0 variants, skipping\n",
      "[INFO]\tGene OC90 has 0 variants, skipping\n",
      "[INFO]\tGene API5 has 0 variants, skipping\n",
      "[INFO]\tGene POLG has 0 variants, skipping\n",
      "[INFO]\tGene HNF1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02547 has 0 variants, skipping\n",
      "[INFO]\tGene RAD9B has 0 variants, skipping\n",
      "[INFO]\tGene ASAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA8 has 0 variants, skipping\n",
      "[INFO]\tGene CLUH has 0 variants, skipping\n",
      "[INFO]\tGene ACTB has 0 variants, skipping\n",
      "[INFO]\tGene POFUT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPP25L has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRB has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB5 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL6 has 0 variants, skipping\n",
      "[INFO]\tGene DOP1A has 0 variants, skipping\n",
      "[INFO]\tGene EBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene SESN1 has 0 variants, skipping\n",
      "[INFO]\tGene SIAE has 0 variants, skipping\n",
      "[INFO]\tGene PGM3 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF200 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372977 has 0 variants, skipping\n",
      "[INFO]\tGene NPPB has 0 variants, skipping\n",
      "[INFO]\tGene NPAT has 0 variants, skipping\n",
      "[INFO]\tGene NPTX2 has 0 variants, skipping\n",
      "[INFO]\tGene NPTX1 has 0 variants, skipping\n",
      "[INFO]\tGene TCTN1 has 0 variants, skipping\n",
      "[INFO]\tGene CA10 has 0 variants, skipping\n",
      "[INFO]\tGene OPRL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAA80 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267881 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC4 has 0 variants, skipping\n",
      "[INFO]\tGene CUL2 has 0 variants, skipping\n",
      "[INFO]\tGene GFAP has 0 variants, skipping\n",
      "[INFO]\tGene PYDC5 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645188 has 0 variants, skipping\n",
      "[INFO]\tGene EML4 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN15 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN12 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP45 has 0 variants, skipping\n",
      "[INFO]\tGene NARF has 0 variants, skipping\n",
      "[INFO]\tGene MYO1F has 0 variants, skipping\n",
      "[INFO]\tGene BAMBI has 0 variants, skipping\n",
      "[INFO]\tGene DEFA1B has 0 variants, skipping\n",
      "[INFO]\tGene RAC3 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf95 has 0 variants, skipping\n",
      "[INFO]\tGene FDPS has 0 variants, skipping\n",
      "[INFO]\tGene GDE1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507006 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM129 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927168 has 0 variants, skipping\n",
      "[INFO]\tGene ABI3BP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02548 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02545 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02683 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB9 has 0 variants, skipping\n",
      "[INFO]\tGene EOMES has 0 variants, skipping\n",
      "[INFO]\tGene ARMH2 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45A has 0 variants, skipping\n",
      "[INFO]\tGene CTSS has 0 variants, skipping\n",
      "[INFO]\tGene CRABP2 has 0 variants, skipping\n",
      "[INFO]\tGene DHX33 has 0 variants, skipping\n",
      "[INFO]\tGene MIR561 has 0 variants, skipping\n",
      "[INFO]\tGene GET1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D15 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF396 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF432 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7 has 0 variants, skipping\n",
      "[INFO]\tGene CCNE1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF79 has 0 variants, skipping\n",
      "[INFO]\tGene CCNI has 0 variants, skipping\n",
      "[INFO]\tGene NBDY has 0 variants, skipping\n",
      "[INFO]\tGene TRIM16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6073 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY5 has 0 variants, skipping\n",
      "[INFO]\tGene IBTK has 0 variants, skipping\n",
      "[INFO]\tGene SNORD147 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E14P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM6 has 0 variants, skipping\n",
      "[INFO]\tGene RECQL5 has 0 variants, skipping\n",
      "[INFO]\tGene MTARC2 has 0 variants, skipping\n",
      "[INFO]\tGene VMO1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL12B has 0 variants, skipping\n",
      "[INFO]\tGene CTSE has 0 variants, skipping\n",
      "[INFO]\tGene PALM3 has 0 variants, skipping\n",
      "[INFO]\tGene NKIRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRQ has 0 variants, skipping\n",
      "[INFO]\tGene GLI1 has 0 variants, skipping\n",
      "[INFO]\tGene NEU4 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM3 has 0 variants, skipping\n",
      "[INFO]\tGene TLR10 has 0 variants, skipping\n",
      "[INFO]\tGene GSR has 0 variants, skipping\n",
      "[INFO]\tGene MEPCE has 0 variants, skipping\n",
      "[INFO]\tGene COPS4 has 0 variants, skipping\n",
      "[INFO]\tGene AP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF16 has 0 variants, skipping\n",
      "[INFO]\tGene TUNAR has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01611 has 0 variants, skipping\n",
      "[INFO]\tGene THRA1/BTR has 0 variants, skipping\n",
      "[INFO]\tGene ENC1 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG1IP has 0 variants, skipping\n",
      "[INFO]\tGene METAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF623 has 0 variants, skipping\n",
      "[INFO]\tGene SAA3P has 0 variants, skipping\n",
      "[INFO]\tGene FBXL12 has 0 variants, skipping\n",
      "[INFO]\tGene MTIF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3159 has 0 variants, skipping\n",
      "[INFO]\tGene XIRP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFY has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP2 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DENND5B has 0 variants, skipping\n",
      "[INFO]\tGene STRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF415 has 0 variants, skipping\n",
      "[INFO]\tGene ATG2A has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01094 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00707 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1B has 0 variants, skipping\n",
      "[INFO]\tGene NEK9 has 0 variants, skipping\n",
      "[INFO]\tGene TPP2 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4486 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126784 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB17 has 0 variants, skipping\n",
      "[INFO]\tGene COLQ has 0 variants, skipping\n",
      "[INFO]\tGene LILRA4 has 0 variants, skipping\n",
      "[INFO]\tGene IL36RN has 0 variants, skipping\n",
      "[INFO]\tGene H2AP has 0 variants, skipping\n",
      "[INFO]\tGene HYAL4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4694 has 0 variants, skipping\n",
      "[INFO]\tGene GPSM3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKEF1 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK9 has 0 variants, skipping\n",
      "[INFO]\tGene ARID5A has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM117 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM68 has 0 variants, skipping\n",
      "[INFO]\tGene PREPL has 0 variants, skipping\n",
      "[INFO]\tGene ITM2A has 0 variants, skipping\n",
      "[INFO]\tGene MCFD2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R13L has 0 variants, skipping\n",
      "[INFO]\tGene DHX15 has 0 variants, skipping\n",
      "[INFO]\tGene DHX9 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8054 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14C has 0 variants, skipping\n",
      "[INFO]\tGene ANO3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIO3 has 0 variants, skipping\n",
      "[INFO]\tGene DHODH has 0 variants, skipping\n",
      "[INFO]\tGene DHCR7 has 0 variants, skipping\n",
      "[INFO]\tGene DECR1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK2A has 0 variants, skipping\n",
      "[INFO]\tGene AMOTL2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2A has 0 variants, skipping\n",
      "[INFO]\tGene CDK2AP2 has 0 variants, skipping\n",
      "[INFO]\tGene MLPH has 0 variants, skipping\n",
      "[INFO]\tGene FLYWCH1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8087 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00678 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf86 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02546 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01616 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY4 has 0 variants, skipping\n",
      "[INFO]\tGene CHID1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF1A has 0 variants, skipping\n",
      "[INFO]\tGene RARA has 0 variants, skipping\n",
      "[INFO]\tGene TKT has 0 variants, skipping\n",
      "[INFO]\tGene PTGS2 has 0 variants, skipping\n",
      "[INFO]\tGene PAX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PAUPAR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506675 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA88 has 0 variants, skipping\n",
      "[INFO]\tGene IDNK has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B3P has 0 variants, skipping\n",
      "[INFO]\tGene CEMIP2 has 0 variants, skipping\n",
      "[INFO]\tGene CEMIP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00294 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM251 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf43 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CADM1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO3-DT has 0 variants, skipping\n",
      "[INFO]\tGene TFG has 0 variants, skipping\n",
      "[INFO]\tGene OAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ1 has 0 variants, skipping\n",
      "[INFO]\tGene DRG1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB7 has 0 variants, skipping\n",
      "[INFO]\tGene MYO9B has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB1 has 0 variants, skipping\n",
      "[INFO]\tGene IREB2 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD17 has 0 variants, skipping\n",
      "[INFO]\tGene E2F8 has 0 variants, skipping\n",
      "[INFO]\tGene NARS2 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN7 has 0 variants, skipping\n",
      "[INFO]\tGene ARG1 has 0 variants, skipping\n",
      "[INFO]\tGene RRNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1343 has 0 variants, skipping\n",
      "[INFO]\tGene GPR34 has 0 variants, skipping\n",
      "[INFO]\tGene CIAPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene DUS1L has 0 variants, skipping\n",
      "[INFO]\tGene SNORD164 has 0 variants, skipping\n",
      "[INFO]\tGene PTGR2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP51A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC124 has 0 variants, skipping\n",
      "[INFO]\tGene ISOC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3973 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928510 has 0 variants, skipping\n",
      "[INFO]\tGene ASPDH has 0 variants, skipping\n",
      "[INFO]\tGene OR2J3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02790 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02762 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01493 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01499 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPKP3 has 0 variants, skipping\n",
      "[INFO]\tGene OR52I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E8 has 0 variants, skipping\n",
      "[INFO]\tGene PJA1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4A has 0 variants, skipping\n",
      "[INFO]\tGene TFB2M has 0 variants, skipping\n",
      "[INFO]\tGene MIR670 has 0 variants, skipping\n",
      "[INFO]\tGene MIR670HG has 0 variants, skipping\n",
      "[INFO]\tGene EVI5 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EYA1 has 0 variants, skipping\n",
      "[INFO]\tGene CD59 has 0 variants, skipping\n",
      "[INFO]\tGene CD44 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP5 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP3 has 0 variants, skipping\n",
      "[INFO]\tGene ORM2 has 0 variants, skipping\n",
      "[INFO]\tGene CIR1 has 0 variants, skipping\n",
      "[INFO]\tGene NBAS has 0 variants, skipping\n",
      "[INFO]\tGene SNCA has 0 variants, skipping\n",
      "[INFO]\tGene CEROX1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD5 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN3 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02685 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02687 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7154 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507384 has 0 variants, skipping\n",
      "[INFO]\tGene COX7B2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD7A has 0 variants, skipping\n",
      "[INFO]\tGene RAET1L has 0 variants, skipping\n",
      "[INFO]\tGene PTPRS has 0 variants, skipping\n",
      "[INFO]\tGene ALPL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM71 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD2 has 0 variants, skipping\n",
      "[INFO]\tGene EHD3 has 0 variants, skipping\n",
      "[INFO]\tGene ERVW-1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2E has 0 variants, skipping\n",
      "[INFO]\tGene SOX8 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A41 has 0 variants, skipping\n",
      "[INFO]\tGene AKIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928894 has 0 variants, skipping\n",
      "[INFO]\tGene NPY5R has 0 variants, skipping\n",
      "[INFO]\tGene RPL12 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPE has 0 variants, skipping\n",
      "[INFO]\tGene LCK has 0 variants, skipping\n",
      "[INFO]\tGene KIF25 has 0 variants, skipping\n",
      "[INFO]\tGene JUND has 0 variants, skipping\n",
      "[INFO]\tGene ITGAD has 0 variants, skipping\n",
      "[INFO]\tGene MIR4688 has 0 variants, skipping\n",
      "[INFO]\tGene LOC408186 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA8 has 0 variants, skipping\n",
      "[INFO]\tGene CNR1 has 0 variants, skipping\n",
      "[INFO]\tGene TYW1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM2 has 0 variants, skipping\n",
      "[INFO]\tGene GMIP has 0 variants, skipping\n",
      "[INFO]\tGene YEATS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIS3L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3160-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3160-2 has 0 variants, skipping\n",
      "[INFO]\tGene ART3 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB1 has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD3 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA8 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2D has 0 variants, skipping\n",
      "[INFO]\tGene SETBP1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN13 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL2 has 0 variants, skipping\n",
      "[INFO]\tGene JAML has 0 variants, skipping\n",
      "[INFO]\tGene CKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene ASB1 has 0 variants, skipping\n",
      "[INFO]\tGene TDP1 has 0 variants, skipping\n",
      "[INFO]\tGene MCC has 0 variants, skipping\n",
      "[INFO]\tGene TIMM8A has 0 variants, skipping\n",
      "[INFO]\tGene HSDL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALCAM has 0 variants, skipping\n",
      "[INFO]\tGene RAET1E has 0 variants, skipping\n",
      "[INFO]\tGene MS4A15 has 0 variants, skipping\n",
      "[INFO]\tGene ATP4A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268355 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SECISBP2 has 0 variants, skipping\n",
      "[INFO]\tGene BRD4 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB4 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D3 has 0 variants, skipping\n",
      "[INFO]\tGene DAG1 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD67 has 0 variants, skipping\n",
      "[INFO]\tGene LRP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AMN1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS4 has 0 variants, skipping\n",
      "[INFO]\tGene JAZF1 has 0 variants, skipping\n",
      "[INFO]\tGene SVOPL has 0 variants, skipping\n",
      "[INFO]\tGene EBF3 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC107 has 0 variants, skipping\n",
      "[INFO]\tGene TEX50 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6745 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G3 has 0 variants, skipping\n",
      "[INFO]\tGene GPC4 has 0 variants, skipping\n",
      "[INFO]\tGene FABP6 has 0 variants, skipping\n",
      "[INFO]\tGene FABP5 has 0 variants, skipping\n",
      "[INFO]\tGene FABP1 has 0 variants, skipping\n",
      "[INFO]\tGene FABP4 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL1 has 0 variants, skipping\n",
      "[INFO]\tGene GRID2 has 0 variants, skipping\n",
      "[INFO]\tGene GPX5 has 0 variants, skipping\n",
      "[INFO]\tGene CHKB has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA1 has 0 variants, skipping\n",
      "[INFO]\tGene CRKL has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3 has 0 variants, skipping\n",
      "[INFO]\tGene CRK has 0 variants, skipping\n",
      "[INFO]\tGene CEBPD has 0 variants, skipping\n",
      "[INFO]\tGene COX6A2 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC5 has 0 variants, skipping\n",
      "[INFO]\tGene IMMP1L has 0 variants, skipping\n",
      "[INFO]\tGene SPRR3 has 0 variants, skipping\n",
      "[INFO]\tGene LUZP2 has 0 variants, skipping\n",
      "[INFO]\tGene SGPL1 has 0 variants, skipping\n",
      "[INFO]\tGene CTLA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCD5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP8 has 0 variants, skipping\n",
      "[INFO]\tGene KMT5B has 0 variants, skipping\n",
      "[INFO]\tGene HTR3B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM196 has 0 variants, skipping\n",
      "[INFO]\tGene RIN1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J1 has 0 variants, skipping\n",
      "[INFO]\tGene PAAF1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1E has 0 variants, skipping\n",
      "[INFO]\tGene CCKBR has 0 variants, skipping\n",
      "[INFO]\tGene FANK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A13 has 0 variants, skipping\n",
      "[INFO]\tGene MYC has 0 variants, skipping\n",
      "[INFO]\tGene MYF6 has 0 variants, skipping\n",
      "[INFO]\tGene XCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPF has 0 variants, skipping\n",
      "[INFO]\tGene SORD has 0 variants, skipping\n",
      "[INFO]\tGene HSF2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS5 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS8 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS3 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1B has 0 variants, skipping\n",
      "[INFO]\tGene DRP2 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR3 has 0 variants, skipping\n",
      "[INFO]\tGene TRDMT1 has 0 variants, skipping\n",
      "[INFO]\tGene GCKR has 0 variants, skipping\n",
      "[INFO]\tGene NCOA7 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14 has 0 variants, skipping\n",
      "[INFO]\tGene NUTF2 has 0 variants, skipping\n",
      "[INFO]\tGene PHYH has 0 variants, skipping\n",
      "[INFO]\tGene SFXN1 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX3 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3161 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376526 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1D1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT1 has 0 variants, skipping\n",
      "[INFO]\tGene MLKL has 0 variants, skipping\n",
      "[INFO]\tGene LOC440040 has 0 variants, skipping\n",
      "[INFO]\tGene TFEB has 0 variants, skipping\n",
      "[INFO]\tGene OR4C45 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646813 has 0 variants, skipping\n",
      "[INFO]\tGene CCNL2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51HP has 0 variants, skipping\n",
      "[INFO]\tGene RAD52 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC31 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E5P has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6128 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AK4P has 0 variants, skipping\n",
      "[INFO]\tGene HELZ has 0 variants, skipping\n",
      "[INFO]\tGene JOSD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF14 has 0 variants, skipping\n",
      "[INFO]\tGene RAB27A has 0 variants, skipping\n",
      "[INFO]\tGene RAB4A has 0 variants, skipping\n",
      "[INFO]\tGene POU4F2 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL4 has 0 variants, skipping\n",
      "[INFO]\tGene DXO has 0 variants, skipping\n",
      "[INFO]\tGene FARP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR122 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L2 has 0 variants, skipping\n",
      "[INFO]\tGene EDN3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC47A2 has 0 variants, skipping\n",
      "[INFO]\tGene TMX2-CTNND1 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GLT1D1 has 0 variants, skipping\n",
      "[INFO]\tGene ATL3 has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST1 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6B has 0 variants, skipping\n",
      "[INFO]\tGene MAGED1 has 0 variants, skipping\n",
      "[INFO]\tGene TRA2B has 0 variants, skipping\n",
      "[INFO]\tGene DYNLL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB4 has 0 variants, skipping\n",
      "[INFO]\tGene CALCOCO1 has 0 variants, skipping\n",
      "[INFO]\tGene TLE3 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP91-CNTF has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283194 has 0 variants, skipping\n",
      "[INFO]\tGene GSTCD has 0 variants, skipping\n",
      "[INFO]\tGene CXCR2 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKB has 0 variants, skipping\n",
      "[INFO]\tGene DUSP23 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111A-DT has 0 variants, skipping\n",
      "[INFO]\tGene AZGP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATR has 0 variants, skipping\n",
      "[INFO]\tGene HARS1 has 0 variants, skipping\n",
      "[INFO]\tGene BID has 0 variants, skipping\n",
      "[INFO]\tGene ROBO4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR87 has 0 variants, skipping\n",
      "[INFO]\tGene OR10V2P has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22A has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE16 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC8 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7C has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CB has 0 variants, skipping\n",
      "[INFO]\tGene MIR6503 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC6 has 0 variants, skipping\n",
      "[INFO]\tGene DPEP2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7 has 0 variants, skipping\n",
      "[INFO]\tGene MAX has 0 variants, skipping\n",
      "[INFO]\tGene GP6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00301 has 0 variants, skipping\n",
      "[INFO]\tGene FAAH has 0 variants, skipping\n",
      "[INFO]\tGene CDK10 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A2 has 0 variants, skipping\n",
      "[INFO]\tGene IL21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF233 has 0 variants, skipping\n",
      "[INFO]\tGene SAP130 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4488 has 0 variants, skipping\n",
      "[INFO]\tGene RPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927495 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP0P2 has 0 variants, skipping\n",
      "[INFO]\tGene THSD4 has 0 variants, skipping\n",
      "[INFO]\tGene REV3L has 0 variants, skipping\n",
      "[INFO]\tGene RPL5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF829 has 0 variants, skipping\n",
      "[INFO]\tGene MIR611 has 0 variants, skipping\n",
      "[INFO]\tGene CD28 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6746 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY6-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4725 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K6 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3654 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6747 has 0 variants, skipping\n",
      "[INFO]\tGene PTGR1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf81 has 0 variants, skipping\n",
      "[INFO]\tGene ADA2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM39 has 0 variants, skipping\n",
      "[INFO]\tGene POP1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf98 has 0 variants, skipping\n",
      "[INFO]\tGene SHBG has 0 variants, skipping\n",
      "[INFO]\tGene POMC has 0 variants, skipping\n",
      "[INFO]\tGene PON3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA57 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-3 has 0 variants, skipping\n",
      "[INFO]\tGene CSPG5 has 0 variants, skipping\n",
      "[INFO]\tGene OR52W1 has 0 variants, skipping\n",
      "[INFO]\tGene CLK3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL2-BSCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6748 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR2 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6514 has 0 variants, skipping\n",
      "[INFO]\tGene WDR89 has 0 variants, skipping\n",
      "[INFO]\tGene APC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369332 has 0 variants, skipping\n",
      "[INFO]\tGene GLOD5 has 0 variants, skipping\n",
      "[INFO]\tGene ADAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF20 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MACC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROGDI has 0 variants, skipping\n",
      "[INFO]\tGene FKBP14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD25 has 0 variants, skipping\n",
      "[INFO]\tGene DANCR has 0 variants, skipping\n",
      "[INFO]\tGene LINC00339 has 0 variants, skipping\n",
      "[INFO]\tGene WT1-AS has 0 variants, skipping\n",
      "[INFO]\tGene PTENP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD7 has 0 variants, skipping\n",
      "[INFO]\tGene COPB2 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L6 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA3 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRQ has 0 variants, skipping\n",
      "[INFO]\tGene ARAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MYEOV has 0 variants, skipping\n",
      "[INFO]\tGene NEU3 has 0 variants, skipping\n",
      "[INFO]\tGene SATL1 has 0 variants, skipping\n",
      "[INFO]\tGene RAG1 has 0 variants, skipping\n",
      "[INFO]\tGene PER3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene MMS22L has 0 variants, skipping\n",
      "[INFO]\tGene MECOM has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF18 has 0 variants, skipping\n",
      "[INFO]\tGene BMP8B has 0 variants, skipping\n",
      "[INFO]\tGene BMP6 has 0 variants, skipping\n",
      "[INFO]\tGene BNC1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR5 has 0 variants, skipping\n",
      "[INFO]\tGene NNAT has 0 variants, skipping\n",
      "[INFO]\tGene PCSK2 has 0 variants, skipping\n",
      "[INFO]\tGene CD40 has 0 variants, skipping\n",
      "[INFO]\tGene RPS29 has 0 variants, skipping\n",
      "[INFO]\tGene CD79A has 0 variants, skipping\n",
      "[INFO]\tGene CD72 has 0 variants, skipping\n",
      "[INFO]\tGene CD69 has 0 variants, skipping\n",
      "[INFO]\tGene CD63 has 0 variants, skipping\n",
      "[INFO]\tGene CD58 has 0 variants, skipping\n",
      "[INFO]\tGene RPL8 has 0 variants, skipping\n",
      "[INFO]\tGene STAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CUL5 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02743 has 0 variants, skipping\n",
      "[INFO]\tGene DBNL has 0 variants, skipping\n",
      "[INFO]\tGene HFM1 has 0 variants, skipping\n",
      "[INFO]\tGene RFLNB has 0 variants, skipping\n",
      "[INFO]\tGene SELENOV has 0 variants, skipping\n",
      "[INFO]\tGene ANKDD1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02735 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-1 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKB has 0 variants, skipping\n",
      "[INFO]\tGene ITGB8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7155 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1237 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM4 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKMY2 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP37 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106C has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A5 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf25 has 0 variants, skipping\n",
      "[INFO]\tGene GID4 has 0 variants, skipping\n",
      "[INFO]\tGene DDA1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT9 has 0 variants, skipping\n",
      "[INFO]\tGene CAMKV has 0 variants, skipping\n",
      "[INFO]\tGene LYNX1-SLURP2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX50 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1B has 0 variants, skipping\n",
      "[INFO]\tGene DBNDD1 has 0 variants, skipping\n",
      "[INFO]\tGene METRN has 0 variants, skipping\n",
      "[INFO]\tGene LRRC61 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE4 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A7 has 0 variants, skipping\n",
      "[INFO]\tGene APC has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-2HG has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL7 has 0 variants, skipping\n",
      "[INFO]\tGene PXDN has 0 variants, skipping\n",
      "[INFO]\tGene SIN3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6750 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6749 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2-SNX15 has 0 variants, skipping\n",
      "[INFO]\tGene SASS6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6879 has 0 variants, skipping\n",
      "[INFO]\tGene CDK18 has 0 variants, skipping\n",
      "[INFO]\tGene LNCOG has 0 variants, skipping\n",
      "[INFO]\tGene DLEU2 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7 has 0 variants, skipping\n",
      "[INFO]\tGene OR5L2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX14 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP27 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL49 has 0 variants, skipping\n",
      "[INFO]\tGene MYRIP has 0 variants, skipping\n",
      "[INFO]\tGene DENND1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC728975 has 0 variants, skipping\n",
      "[INFO]\tGene NTN4 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A20P has 0 variants, skipping\n",
      "[INFO]\tGene PAK5 has 0 variants, skipping\n",
      "[INFO]\tGene RRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene AHCY has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD5 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR5 has 0 variants, skipping\n",
      "[INFO]\tGene BCKDK has 0 variants, skipping\n",
      "[INFO]\tGene TRAIP has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ3 has 0 variants, skipping\n",
      "[INFO]\tGene TXNL4A has 0 variants, skipping\n",
      "[INFO]\tGene DIABLO has 0 variants, skipping\n",
      "[INFO]\tGene RPE has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1 has 0 variants, skipping\n",
      "[INFO]\tGene BCAP29 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOB has 0 variants, skipping\n",
      "[INFO]\tGene PRL has 0 variants, skipping\n",
      "[INFO]\tGene SYNRG has 0 variants, skipping\n",
      "[INFO]\tGene FBXW7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4690 has 0 variants, skipping\n",
      "[INFO]\tGene RELA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4489 has 0 variants, skipping\n",
      "[INFO]\tGene MIR570 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPLL has 0 variants, skipping\n",
      "[INFO]\tGene UMOD has 0 variants, skipping\n",
      "[INFO]\tGene PCDH1 has 0 variants, skipping\n",
      "[INFO]\tGene AXL has 0 variants, skipping\n",
      "[INFO]\tGene NELFE has 0 variants, skipping\n",
      "[INFO]\tGene RBBP8 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARA3 has 0 variants, skipping\n",
      "[INFO]\tGene U2SURP has 0 variants, skipping\n",
      "[INFO]\tGene APEH has 0 variants, skipping\n",
      "[INFO]\tGene C1S has 0 variants, skipping\n",
      "[INFO]\tGene MTERF2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLPH3L has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2 has 0 variants, skipping\n",
      "[INFO]\tGene LGI2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1S has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2B has 0 variants, skipping\n",
      "[INFO]\tGene MAST2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF654 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13F has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC15 has 0 variants, skipping\n",
      "[INFO]\tGene TBL1XR1 has 0 variants, skipping\n",
      "[INFO]\tGene SF3A1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF8 has 0 variants, skipping\n",
      "[INFO]\tGene TAP1 has 0 variants, skipping\n",
      "[INFO]\tGene STX1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724064 has 0 variants, skipping\n",
      "[INFO]\tGene FLII has 0 variants, skipping\n",
      "[INFO]\tGene DUSP7 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP6 has 0 variants, skipping\n",
      "[INFO]\tGene MFF has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928069 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM6 has 0 variants, skipping\n",
      "[INFO]\tGene PICALM has 0 variants, skipping\n",
      "[INFO]\tGene SF3A2 has 0 variants, skipping\n",
      "[INFO]\tGene IMPG2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF20 has 0 variants, skipping\n",
      "[INFO]\tGene ISLR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A10 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS8 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12D has 0 variants, skipping\n",
      "[INFO]\tGene PEX14 has 0 variants, skipping\n",
      "[INFO]\tGene SETX has 0 variants, skipping\n",
      "[INFO]\tGene RBM14-RBM4 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf56 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AX has 0 variants, skipping\n",
      "[INFO]\tGene EDA has 0 variants, skipping\n",
      "[INFO]\tGene MEGF6 has 0 variants, skipping\n",
      "[INFO]\tGene ECH1 has 0 variants, skipping\n",
      "[INFO]\tGene CELSR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A1 has 0 variants, skipping\n",
      "[INFO]\tGene F8A3 has 0 variants, skipping\n",
      "[INFO]\tGene STOML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3163 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6860 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D4 has 0 variants, skipping\n",
      "[INFO]\tGene TTC21B has 0 variants, skipping\n",
      "[INFO]\tGene LRCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130987 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL3 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS6 has 0 variants, skipping\n",
      "[INFO]\tGene DOK3 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC11 has 0 variants, skipping\n",
      "[INFO]\tGene SNPH has 0 variants, skipping\n",
      "[INFO]\tGene RFNG has 0 variants, skipping\n",
      "[INFO]\tGene SRSF3 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF2 has 0 variants, skipping\n",
      "[INFO]\tGene HES2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MRRF has 0 variants, skipping\n",
      "[INFO]\tGene ABCC9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6752 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN3 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNTL1 has 0 variants, skipping\n",
      "[INFO]\tGene MXD1 has 0 variants, skipping\n",
      "[INFO]\tGene SURF2 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2GP has 0 variants, skipping\n",
      "[INFO]\tGene ACADVL has 0 variants, skipping\n",
      "[INFO]\tGene FAM86C2P has 0 variants, skipping\n",
      "[INFO]\tGene FBXO46 has 0 variants, skipping\n",
      "[INFO]\tGene PRPSAP2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR48 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724951 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4691 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1T1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6753 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R42 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO3 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF433 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF48 has 0 variants, skipping\n",
      "[INFO]\tGene DIMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A8 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R18 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM17 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC152 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB2 has 0 variants, skipping\n",
      "[INFO]\tGene AHI1 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL5 has 0 variants, skipping\n",
      "[INFO]\tGene MAJIN has 0 variants, skipping\n",
      "[INFO]\tGene CNOT8 has 0 variants, skipping\n",
      "[INFO]\tGene NUMBL has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP2-CSNK1E has 0 variants, skipping\n",
      "[INFO]\tGene EYA4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF333 has 0 variants, skipping\n",
      "[INFO]\tGene MYOT has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN4 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf202 has 0 variants, skipping\n",
      "[INFO]\tGene LAT has 0 variants, skipping\n",
      "[INFO]\tGene CYB561 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1958 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC92B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC6 has 0 variants, skipping\n",
      "[INFO]\tGene LRCOL1 has 0 variants, skipping\n",
      "[INFO]\tGene TLX1 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO3 has 0 variants, skipping\n",
      "[INFO]\tGene LXN has 0 variants, skipping\n",
      "[INFO]\tGene PAK6 has 0 variants, skipping\n",
      "[INFO]\tGene AMBRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF850 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A8 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1B has 0 variants, skipping\n",
      "[INFO]\tGene INTS13 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG4 has 0 variants, skipping\n",
      "[INFO]\tGene ALG8 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM8 has 0 variants, skipping\n",
      "[INFO]\tGene NOC4L has 0 variants, skipping\n",
      "[INFO]\tGene KCTD15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM38A has 0 variants, skipping\n",
      "[INFO]\tGene DDX54 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE21 has 0 variants, skipping\n",
      "[INFO]\tGene KXD1 has 0 variants, skipping\n",
      "[INFO]\tGene PVRIG has 0 variants, skipping\n",
      "[INFO]\tGene NABP2 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf26 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3164 has 0 variants, skipping\n",
      "[INFO]\tGene LOC338694 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC157 has 0 variants, skipping\n",
      "[INFO]\tGene OR51J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01488 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH15 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30E has 0 variants, skipping\n",
      "[INFO]\tGene LINC02584 has 0 variants, skipping\n",
      "[INFO]\tGene KLK13 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD3B has 0 variants, skipping\n",
      "[INFO]\tGene YPEL3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC3 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR645 has 0 variants, skipping\n",
      "[INFO]\tGene OGFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02878 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS4 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene CAV2 has 0 variants, skipping\n",
      "[INFO]\tGene CERS1 has 0 variants, skipping\n",
      "[INFO]\tGene MLX has 0 variants, skipping\n",
      "[INFO]\tGene STOM has 0 variants, skipping\n",
      "[INFO]\tGene POP5 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L9P has 0 variants, skipping\n",
      "[INFO]\tGene GNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene KNL1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP12 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN14 has 0 variants, skipping\n",
      "[INFO]\tGene MATK has 0 variants, skipping\n",
      "[INFO]\tGene TAF1C has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128494 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3165 has 0 variants, skipping\n",
      "[INFO]\tGene EML1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP1L has 0 variants, skipping\n",
      "[INFO]\tGene ELK4 has 0 variants, skipping\n",
      "[INFO]\tGene ELANE has 0 variants, skipping\n",
      "[INFO]\tGene GNAQ has 0 variants, skipping\n",
      "[INFO]\tGene CELA1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5A has 0 variants, skipping\n",
      "[INFO]\tGene GNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E has 0 variants, skipping\n",
      "[INFO]\tGene GNA15 has 0 variants, skipping\n",
      "[INFO]\tGene GNA11 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A2 has 0 variants, skipping\n",
      "[INFO]\tGene EHHADH has 0 variants, skipping\n",
      "[INFO]\tGene GML has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B2 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC4 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC12 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT5H has 0 variants, skipping\n",
      "[INFO]\tGene ALG6 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL27A has 0 variants, skipping\n",
      "[INFO]\tGene ACTA1 has 0 variants, skipping\n",
      "[INFO]\tGene MEPE has 0 variants, skipping\n",
      "[INFO]\tGene GIGYF2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM156A has 0 variants, skipping\n",
      "[INFO]\tGene MIF4GD has 0 variants, skipping\n",
      "[INFO]\tGene DYRK1B has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4692 has 0 variants, skipping\n",
      "[INFO]\tGene SDHB has 0 variants, skipping\n",
      "[INFO]\tGene RBP3 has 0 variants, skipping\n",
      "[INFO]\tGene RARRES1 has 0 variants, skipping\n",
      "[INFO]\tGene RARS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2B has 0 variants, skipping\n",
      "[INFO]\tGene FLOT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287837 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF808 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC8 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP2 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf43 has 0 variants, skipping\n",
      "[INFO]\tGene APEX1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNS has 0 variants, skipping\n",
      "[INFO]\tGene DCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LETM2 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene THSD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11E has 0 variants, skipping\n",
      "[INFO]\tGene OGN has 0 variants, skipping\n",
      "[INFO]\tGene CNPY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF114 has 0 variants, skipping\n",
      "[INFO]\tGene COX6B2 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK1 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GCK has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AL has 0 variants, skipping\n",
      "[INFO]\tGene MIR623 has 0 variants, skipping\n",
      "[INFO]\tGene MIR620 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2B has 0 variants, skipping\n",
      "[INFO]\tGene YWHAG has 0 variants, skipping\n",
      "[INFO]\tGene WBP1 has 0 variants, skipping\n",
      "[INFO]\tGene VAX2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4696 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC3B has 0 variants, skipping\n",
      "[INFO]\tGene ATG16L1 has 0 variants, skipping\n",
      "[INFO]\tGene TOPORS has 0 variants, skipping\n",
      "[INFO]\tGene DKK2 has 0 variants, skipping\n",
      "[INFO]\tGene DKK4 has 0 variants, skipping\n",
      "[INFO]\tGene DKKL1 has 0 variants, skipping\n",
      "[INFO]\tGene BBC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB11 has 0 variants, skipping\n",
      "[INFO]\tGene CACYBP has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN13 has 0 variants, skipping\n",
      "[INFO]\tGene TAF5L has 0 variants, skipping\n",
      "[INFO]\tGene SNORD15A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD15B has 0 variants, skipping\n",
      "[INFO]\tGene AAGAB has 0 variants, skipping\n",
      "[INFO]\tGene RGS7BP has 0 variants, skipping\n",
      "[INFO]\tGene GLE1 has 0 variants, skipping\n",
      "[INFO]\tGene GHDC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369391 has 0 variants, skipping\n",
      "[INFO]\tGene PRIM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM180B has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC283214 has 0 variants, skipping\n",
      "[INFO]\tGene UVRAG-DT has 0 variants, skipping\n",
      "[INFO]\tGene MUCL1 has 0 variants, skipping\n",
      "[INFO]\tGene THAP12 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL5 has 0 variants, skipping\n",
      "[INFO]\tGene ERBIN has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2EP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928837 has 0 variants, skipping\n",
      "[INFO]\tGene TMEFF2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCK has 0 variants, skipping\n",
      "[INFO]\tGene DUSP26 has 0 variants, skipping\n",
      "[INFO]\tGene RPL15 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646029 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3A has 0 variants, skipping\n",
      "[INFO]\tGene CNTN1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL2P-SEC16B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN32 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF746 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928896 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A5 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5579 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4300HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4300 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1B has 0 variants, skipping\n",
      "[INFO]\tGene FDXR has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIDEAS has 0 variants, skipping\n",
      "[INFO]\tGene ATG4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70E has 0 variants, skipping\n",
      "[INFO]\tGene RAB30-DT has 0 variants, skipping\n",
      "[INFO]\tGene GBGT1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506282 has 0 variants, skipping\n",
      "[INFO]\tGene PCF11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC15 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK16 has 0 variants, skipping\n",
      "[INFO]\tGene PHF21A has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2 has 0 variants, skipping\n",
      "[INFO]\tGene IFT88 has 0 variants, skipping\n",
      "[INFO]\tGene CENPK has 0 variants, skipping\n",
      "[INFO]\tGene MDP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX5-GPRASP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFA has 0 variants, skipping\n",
      "[INFO]\tGene NAPA has 0 variants, skipping\n",
      "[INFO]\tGene NAPG has 0 variants, skipping\n",
      "[INFO]\tGene SNAP23 has 0 variants, skipping\n",
      "[INFO]\tGene SOHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRS has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF5 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL2 has 0 variants, skipping\n",
      "[INFO]\tGene CT55 has 0 variants, skipping\n",
      "[INFO]\tGene PIGX has 0 variants, skipping\n",
      "[INFO]\tGene C1orf56 has 0 variants, skipping\n",
      "[INFO]\tGene TIPIN has 0 variants, skipping\n",
      "[INFO]\tGene SSH3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS7 has 0 variants, skipping\n",
      "[INFO]\tGene GCM1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB121 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1L2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO7-PAM16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6755 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B14 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B11 has 0 variants, skipping\n",
      "[INFO]\tGene TUG1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK4 has 0 variants, skipping\n",
      "[INFO]\tGene GET1-SH3BGR has 0 variants, skipping\n",
      "[INFO]\tGene OR7E2P has 0 variants, skipping\n",
      "[INFO]\tGene CX3CR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A21 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02753 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3166 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A10 has 0 variants, skipping\n",
      "[INFO]\tGene GRM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC91 has 0 variants, skipping\n",
      "[INFO]\tGene OR4S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N5 has 0 variants, skipping\n",
      "[INFO]\tGene HELT has 0 variants, skipping\n",
      "[INFO]\tGene RPL28 has 0 variants, skipping\n",
      "[INFO]\tGene ITPK1 has 0 variants, skipping\n",
      "[INFO]\tGene RBKS has 0 variants, skipping\n",
      "[INFO]\tGene LHPP has 0 variants, skipping\n",
      "[INFO]\tGene TRIM53AP has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51EP has 0 variants, skipping\n",
      "[INFO]\tGene FANCL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC81 has 0 variants, skipping\n",
      "[INFO]\tGene DSTN has 0 variants, skipping\n",
      "[INFO]\tGene INTS14 has 0 variants, skipping\n",
      "[INFO]\tGene ETNPPL has 0 variants, skipping\n",
      "[INFO]\tGene DISC1FP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283710 has 0 variants, skipping\n",
      "[INFO]\tGene MICU2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB1 has 0 variants, skipping\n",
      "[INFO]\tGene USF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA9 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA32 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A13 has 0 variants, skipping\n",
      "[INFO]\tGene COX5A has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3R1 has 0 variants, skipping\n",
      "[INFO]\tGene MRO has 0 variants, skipping\n",
      "[INFO]\tGene SSTR5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAST4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548L has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369438 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1A has 0 variants, skipping\n",
      "[INFO]\tGene SRSF8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929295 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129203 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L2 has 0 variants, skipping\n",
      "[INFO]\tGene JRKL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EZR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL4 has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02720 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02728 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02763 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02696 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02759 has 0 variants, skipping\n",
      "[INFO]\tGene CCN1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13I has 0 variants, skipping\n",
      "[INFO]\tGene FKBP9 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB11 has 0 variants, skipping\n",
      "[INFO]\tGene PGR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3920 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A2 has 0 variants, skipping\n",
      "[INFO]\tGene RGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNJL has 0 variants, skipping\n",
      "[INFO]\tGene DCAF17 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR3 has 0 variants, skipping\n",
      "[INFO]\tGene DHDDS has 0 variants, skipping\n",
      "[INFO]\tGene TAF7L has 0 variants, skipping\n",
      "[INFO]\tGene LONRF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928424 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723838 has 0 variants, skipping\n",
      "[INFO]\tGene CEP85 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM265 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC7A has 0 variants, skipping\n",
      "[INFO]\tGene HIBCH has 0 variants, skipping\n",
      "[INFO]\tGene PRR23B has 0 variants, skipping\n",
      "[INFO]\tGene INAFM1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC60 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOV has 0 variants, skipping\n",
      "[INFO]\tGene FLNA has 0 variants, skipping\n",
      "[INFO]\tGene TMX3 has 0 variants, skipping\n",
      "[INFO]\tGene DYDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL32 has 0 variants, skipping\n",
      "[INFO]\tGene NCLN has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB13 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4693 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02552 has 0 variants, skipping\n",
      "[INFO]\tGene MANEAL has 0 variants, skipping\n",
      "[INFO]\tGene SPEF2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC13 has 0 variants, skipping\n",
      "[INFO]\tGene SPICE1 has 0 variants, skipping\n",
      "[INFO]\tGene PORCN has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9 has 0 variants, skipping\n",
      "[INFO]\tGene DLL3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP15 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548J has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B3-SLCO1B7 has 0 variants, skipping\n",
      "[INFO]\tGene STX7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF841 has 0 variants, skipping\n",
      "[INFO]\tGene CASP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene AURKA has 0 variants, skipping\n",
      "[INFO]\tGene DENND1C has 0 variants, skipping\n",
      "[INFO]\tGene RPS20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928535 has 0 variants, skipping\n",
      "[INFO]\tGene GNAO1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT7 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA5 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H7 has 0 variants, skipping\n",
      "[INFO]\tGene OR4Q2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA5L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM273 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730098 has 0 variants, skipping\n",
      "[INFO]\tGene EMG1 has 0 variants, skipping\n",
      "[INFO]\tGene NAPEPLD has 0 variants, skipping\n",
      "[INFO]\tGene HHAT has 0 variants, skipping\n",
      "[INFO]\tGene MIR335 has 0 variants, skipping\n",
      "[INFO]\tGene MIR384 has 0 variants, skipping\n",
      "[INFO]\tGene MIR338 has 0 variants, skipping\n",
      "[INFO]\tGene MIR346 has 0 variants, skipping\n",
      "[INFO]\tGene MIR324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR345 has 0 variants, skipping\n",
      "[INFO]\tGene MIR331 has 0 variants, skipping\n",
      "[INFO]\tGene MIR148B has 0 variants, skipping\n",
      "[INFO]\tGene MIR133B has 0 variants, skipping\n",
      "[INFO]\tGene MIR135B has 0 variants, skipping\n",
      "[INFO]\tGene MIR326 has 0 variants, skipping\n",
      "[INFO]\tGene MIR337 has 0 variants, skipping\n",
      "[INFO]\tGene MIR342 has 0 variants, skipping\n",
      "[INFO]\tGene MIR330 has 0 variants, skipping\n",
      "[INFO]\tGene MIR340 has 0 variants, skipping\n",
      "[INFO]\tGene MIR383 has 0 variants, skipping\n",
      "[INFO]\tGene MIR382 has 0 variants, skipping\n",
      "[INFO]\tGene MIR381 has 0 variants, skipping\n",
      "[INFO]\tGene MIR379 has 0 variants, skipping\n",
      "[INFO]\tGene MIR377 has 0 variants, skipping\n",
      "[INFO]\tGene COL10A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01087 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF84 has 0 variants, skipping\n",
      "[INFO]\tGene DOK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF566 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC181 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP70 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02550 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4491 has 0 variants, skipping\n",
      "[INFO]\tGene IRX4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132078 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728196 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN4 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF4 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA8 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS55 has 0 variants, skipping\n",
      "[INFO]\tGene ADRB1 has 0 variants, skipping\n",
      "[INFO]\tGene PSEN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2C has 0 variants, skipping\n",
      "[INFO]\tGene CDK16 has 0 variants, skipping\n",
      "[INFO]\tGene S100A5 has 0 variants, skipping\n",
      "[INFO]\tGene S100A4 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT1 has 0 variants, skipping\n",
      "[INFO]\tGene SORT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A4 has 0 variants, skipping\n",
      "[INFO]\tGene S100A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A2 has 0 variants, skipping\n",
      "[INFO]\tGene RYK has 0 variants, skipping\n",
      "[INFO]\tGene WWP2 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT14 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB2-C11orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP2B has 0 variants, skipping\n",
      "[INFO]\tGene CHTOP has 0 variants, skipping\n",
      "[INFO]\tGene BRINP3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM8B has 0 variants, skipping\n",
      "[INFO]\tGene NIF3L1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132686 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928847 has 0 variants, skipping\n",
      "[INFO]\tGene REEP1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C4 has 0 variants, skipping\n",
      "[INFO]\tGene SPTAN1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24OS has 0 variants, skipping\n",
      "[INFO]\tGene NCAM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP4 has 0 variants, skipping\n",
      "[INFO]\tGene PCOTH has 0 variants, skipping\n",
      "[INFO]\tGene CARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF532 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERB has 0 variants, skipping\n",
      "[INFO]\tGene DNAI4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF552 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3B has 0 variants, skipping\n",
      "[INFO]\tGene AGMAT has 0 variants, skipping\n",
      "[INFO]\tGene MMRN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLTM has 0 variants, skipping\n",
      "[INFO]\tGene PTCD2 has 0 variants, skipping\n",
      "[INFO]\tGene REXO5 has 0 variants, skipping\n",
      "[INFO]\tGene PLET1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCG has 0 variants, skipping\n",
      "[INFO]\tGene XPC has 0 variants, skipping\n",
      "[INFO]\tGene WNT11 has 0 variants, skipping\n",
      "[INFO]\tGene WNT7A has 0 variants, skipping\n",
      "[INFO]\tGene VIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC4 has 0 variants, skipping\n",
      "[INFO]\tGene TSN has 0 variants, skipping\n",
      "[INFO]\tGene TRPC6 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF6 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF4 has 0 variants, skipping\n",
      "[INFO]\tGene NETO2 has 0 variants, skipping\n",
      "[INFO]\tGene ALPG has 0 variants, skipping\n",
      "[INFO]\tGene CPVL has 0 variants, skipping\n",
      "[INFO]\tGene PLVAP has 0 variants, skipping\n",
      "[INFO]\tGene SCRT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC6 has 0 variants, skipping\n",
      "[INFO]\tGene EPPK1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF208 has 0 variants, skipping\n",
      "[INFO]\tGene PUS3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB33B has 0 variants, skipping\n",
      "[INFO]\tGene IP6K2 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12H has 0 variants, skipping\n",
      "[INFO]\tGene COPA has 0 variants, skipping\n",
      "[INFO]\tGene SPSB2 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14B has 0 variants, skipping\n",
      "[INFO]\tGene FAM222B has 0 variants, skipping\n",
      "[INFO]\tGene EPHA7 has 0 variants, skipping\n",
      "[INFO]\tGene USP7 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF599 has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132A has 0 variants, skipping\n",
      "[INFO]\tGene TTC12 has 0 variants, skipping\n",
      "[INFO]\tGene HPF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928940 has 0 variants, skipping\n",
      "[INFO]\tGene HNMT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC34 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB4 has 0 variants, skipping\n",
      "[INFO]\tGene HLF has 0 variants, skipping\n",
      "[INFO]\tGene CYP27B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369507 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00900 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205A has 0 variants, skipping\n",
      "[INFO]\tGene PRKRA has 0 variants, skipping\n",
      "[INFO]\tGene TXNIP has 0 variants, skipping\n",
      "[INFO]\tGene LNCOC1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM72B has 0 variants, skipping\n",
      "[INFO]\tGene RSPH9 has 0 variants, skipping\n",
      "[INFO]\tGene BEND6 has 0 variants, skipping\n",
      "[INFO]\tGene PYGO1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF23 has 0 variants, skipping\n",
      "[INFO]\tGene TARS2 has 0 variants, skipping\n",
      "[INFO]\tGene CFC1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652768 has 0 variants, skipping\n",
      "[INFO]\tGene RC3H2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD5 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13B has 0 variants, skipping\n",
      "[INFO]\tGene DHFR2 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2R1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM156B has 0 variants, skipping\n",
      "[INFO]\tGene ELP4 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A31 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC3A has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP28 has 0 variants, skipping\n",
      "[INFO]\tGene TACC2 has 0 variants, skipping\n",
      "[INFO]\tGene CSK has 0 variants, skipping\n",
      "[INFO]\tGene CEP104 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT4-ECE2 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf62 has 0 variants, skipping\n",
      "[INFO]\tGene MGST2 has 0 variants, skipping\n",
      "[INFO]\tGene NPR3 has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6L has 0 variants, skipping\n",
      "[INFO]\tGene CNOT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA9 has 0 variants, skipping\n",
      "[INFO]\tGene CDX2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131626 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF83 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR4 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929089 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388813 has 0 variants, skipping\n",
      "[INFO]\tGene ECI1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6716 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4492 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF292 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX10 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP64 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN12L has 0 variants, skipping\n",
      "[INFO]\tGene FFAR4 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS13 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LRTM1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71B has 0 variants, skipping\n",
      "[INFO]\tGene DYNLRB2 has 0 variants, skipping\n",
      "[INFO]\tGene SNCG has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14D has 0 variants, skipping\n",
      "[INFO]\tGene TOR4A has 0 variants, skipping\n",
      "[INFO]\tGene TRMT1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D1A has 0 variants, skipping\n",
      "[INFO]\tGene SNRK has 0 variants, skipping\n",
      "[INFO]\tGene C17orf78 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12F has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB21 has 0 variants, skipping\n",
      "[INFO]\tGene DTL has 0 variants, skipping\n",
      "[INFO]\tGene FAAH2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD150 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF589 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6756 has 0 variants, skipping\n",
      "[INFO]\tGene PEMT has 0 variants, skipping\n",
      "[INFO]\tGene CHAD has 0 variants, skipping\n",
      "[INFO]\tGene USP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C8 has 0 variants, skipping\n",
      "[INFO]\tGene SPAST has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724301 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984399 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM4 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC649133 has 0 variants, skipping\n",
      "[INFO]\tGene ETV3L has 0 variants, skipping\n",
      "[INFO]\tGene ZYG11A has 0 variants, skipping\n",
      "[INFO]\tGene GPR179 has 0 variants, skipping\n",
      "[INFO]\tGene SAE1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D3C has 0 variants, skipping\n",
      "[INFO]\tGene TOM1 has 0 variants, skipping\n",
      "[INFO]\tGene TOM1L1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP3 has 0 variants, skipping\n",
      "[INFO]\tGene CHAF1A has 0 variants, skipping\n",
      "[INFO]\tGene PIGK has 0 variants, skipping\n",
      "[INFO]\tGene RNF31 has 0 variants, skipping\n",
      "[INFO]\tGene RGL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929227 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929208 has 0 variants, skipping\n",
      "[INFO]\tGene CXADR has 0 variants, skipping\n",
      "[INFO]\tGene CYP2D6 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL5 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA9 has 0 variants, skipping\n",
      "[INFO]\tGene DMBT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT19 has 0 variants, skipping\n",
      "[INFO]\tGene PDXP-DT has 0 variants, skipping\n",
      "[INFO]\tGene GPR1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LIN9 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D24 has 0 variants, skipping\n",
      "[INFO]\tGene NIPBL has 0 variants, skipping\n",
      "[INFO]\tGene LOC341056 has 0 variants, skipping\n",
      "[INFO]\tGene INPP4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14E has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14C has 0 variants, skipping\n",
      "[INFO]\tGene COL19A1 has 0 variants, skipping\n",
      "[INFO]\tGene COL16A1 has 0 variants, skipping\n",
      "[INFO]\tGene SCG2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4493 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00210 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HAV1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHD1L has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R10 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R7 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R13 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R8 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD7 has 0 variants, skipping\n",
      "[INFO]\tGene PTK7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929340 has 0 variants, skipping\n",
      "[INFO]\tGene HERC6 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF239 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507283 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD3 has 0 variants, skipping\n",
      "[INFO]\tGene GCLM has 0 variants, skipping\n",
      "[INFO]\tGene BCAR3 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL8 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4 has 0 variants, skipping\n",
      "[INFO]\tGene PHB2 has 0 variants, skipping\n",
      "[INFO]\tGene FMO5 has 0 variants, skipping\n",
      "[INFO]\tGene ARL13A has 0 variants, skipping\n",
      "[INFO]\tGene ANXA7 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX11B has 0 variants, skipping\n",
      "[INFO]\tGene DYRK4 has 0 variants, skipping\n",
      "[INFO]\tGene SIM2 has 0 variants, skipping\n",
      "[INFO]\tGene TARDBP has 0 variants, skipping\n",
      "[INFO]\tGene SIX6 has 0 variants, skipping\n",
      "[INFO]\tGene SHOC2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX42 has 0 variants, skipping\n",
      "[INFO]\tGene STT3A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT8B has 0 variants, skipping\n",
      "[INFO]\tGene XRCC4 has 0 variants, skipping\n",
      "[INFO]\tGene WNT2 has 0 variants, skipping\n",
      "[INFO]\tGene XPO1 has 0 variants, skipping\n",
      "[INFO]\tGene HYMAI has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A25 has 0 variants, skipping\n",
      "[INFO]\tGene EMX2OS has 0 variants, skipping\n",
      "[INFO]\tGene ISG20 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF606 has 0 variants, skipping\n",
      "[INFO]\tGene CEP95 has 0 variants, skipping\n",
      "[INFO]\tGene GLCE has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385B has 0 variants, skipping\n",
      "[INFO]\tGene SMUG1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2R1 has 0 variants, skipping\n",
      "[INFO]\tGene FYCO1 has 0 variants, skipping\n",
      "[INFO]\tGene GSEC has 0 variants, skipping\n",
      "[INFO]\tGene CYP26B1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA3 has 0 variants, skipping\n",
      "[INFO]\tGene SCO2 has 0 variants, skipping\n",
      "[INFO]\tGene GATD1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM7A-DT has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR761 has 0 variants, skipping\n",
      "[INFO]\tGene MTA3 has 0 variants, skipping\n",
      "[INFO]\tGene CTRB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929427 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3167 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02098 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6090 has 0 variants, skipping\n",
      "[INFO]\tGene FLYWCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929538 has 0 variants, skipping\n",
      "[INFO]\tGene CD22 has 0 variants, skipping\n",
      "[INFO]\tGene PRC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR323A has 0 variants, skipping\n",
      "[INFO]\tGene PROM1 has 0 variants, skipping\n",
      "[INFO]\tGene NECAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LUM has 0 variants, skipping\n",
      "[INFO]\tGene C16orf90 has 0 variants, skipping\n",
      "[INFO]\tGene LOC399975 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01395 has 0 variants, skipping\n",
      "[INFO]\tGene EXO5 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R35 has 0 variants, skipping\n",
      "[INFO]\tGene IFNL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF488 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22A has 0 variants, skipping\n",
      "[INFO]\tGene PIH1D1 has 0 variants, skipping\n",
      "[INFO]\tGene RADX has 0 variants, skipping\n",
      "[INFO]\tGene BCL11A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5C has 0 variants, skipping\n",
      "[INFO]\tGene FNBP4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM189A1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM127 has 0 variants, skipping\n",
      "[INFO]\tGene ODR4 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD8 has 0 variants, skipping\n",
      "[INFO]\tGene BORA has 0 variants, skipping\n",
      "[INFO]\tGene SNX22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00167 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF669 has 0 variants, skipping\n",
      "[INFO]\tGene ALKAL2 has 0 variants, skipping\n",
      "[INFO]\tGene BCAS4 has 0 variants, skipping\n",
      "[INFO]\tGene MXI1 has 0 variants, skipping\n",
      "[INFO]\tGene EI24 has 0 variants, skipping\n",
      "[INFO]\tGene USP45 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8052 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02551 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF454 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2 has 0 variants, skipping\n",
      "[INFO]\tGene PCYOX1L has 0 variants, skipping\n",
      "[INFO]\tGene FARS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929653 has 0 variants, skipping\n",
      "[INFO]\tGene NTM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF836 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA4 has 0 variants, skipping\n",
      "[INFO]\tGene NTM-IT has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02724 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4697 has 0 variants, skipping\n",
      "[INFO]\tGene MUC7 has 0 variants, skipping\n",
      "[INFO]\tGene UNC119 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD153 has 0 variants, skipping\n",
      "[INFO]\tGene CGNL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL26L1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC3 has 0 variants, skipping\n",
      "[INFO]\tGene EFNB1 has 0 variants, skipping\n",
      "[INFO]\tGene MSL1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507548 has 0 variants, skipping\n",
      "[INFO]\tGene AGTR2 has 0 variants, skipping\n",
      "[INFO]\tGene AGTR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44 has 0 variants, skipping\n",
      "[INFO]\tGene MTFR1L has 0 variants, skipping\n",
      "[INFO]\tGene TMUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MFRP has 0 variants, skipping\n",
      "[INFO]\tGene AFF1 has 0 variants, skipping\n",
      "[INFO]\tGene ESM1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS27A has 0 variants, skipping\n",
      "[INFO]\tGene ABCA4 has 0 variants, skipping\n",
      "[INFO]\tGene PAH has 0 variants, skipping\n",
      "[INFO]\tGene NRXN2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD8 has 0 variants, skipping\n",
      "[INFO]\tGene CALM1 has 0 variants, skipping\n",
      "[INFO]\tGene TINF2 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL2 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP69B has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATS2 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCB has 0 variants, skipping\n",
      "[INFO]\tGene ABCA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF789 has 0 variants, skipping\n",
      "[INFO]\tGene COG3 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL2 has 0 variants, skipping\n",
      "[INFO]\tGene CALN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A9 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC9 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R3 has 0 variants, skipping\n",
      "[INFO]\tGene GINS3 has 0 variants, skipping\n",
      "[INFO]\tGene PROK2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB4 has 0 variants, skipping\n",
      "[INFO]\tGene MKKS has 0 variants, skipping\n",
      "[INFO]\tGene KLHL9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM107B has 0 variants, skipping\n",
      "[INFO]\tGene GDPD2 has 0 variants, skipping\n",
      "[INFO]\tGene HNF1A has 0 variants, skipping\n",
      "[INFO]\tGene GBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF69 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL2 has 0 variants, skipping\n",
      "[INFO]\tGene WASH8P has 0 variants, skipping\n",
      "[INFO]\tGene PRMT9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC574538 has 0 variants, skipping\n",
      "[INFO]\tGene CLNS1A has 0 variants, skipping\n",
      "[INFO]\tGene TNNT3 has 0 variants, skipping\n",
      "[INFO]\tGene EPN2 has 0 variants, skipping\n",
      "[INFO]\tGene EXPH5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723544 has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369595 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1290 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1285-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1285-1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR5 has 0 variants, skipping\n",
      "[INFO]\tGene ZRSR2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR91 has 0 variants, skipping\n",
      "[INFO]\tGene TCF25 has 0 variants, skipping\n",
      "[INFO]\tGene EFR3B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF410 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F11 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSP1 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00942 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3649 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00940 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene AARD has 0 variants, skipping\n",
      "[INFO]\tGene MYOC has 0 variants, skipping\n",
      "[INFO]\tGene RTCA has 0 variants, skipping\n",
      "[INFO]\tGene PCBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CSTB has 0 variants, skipping\n",
      "[INFO]\tGene COL1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LYVE1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAFD1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL1 has 0 variants, skipping\n",
      "[INFO]\tGene JPT1 has 0 variants, skipping\n",
      "[INFO]\tGene NDEL1 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAMR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN31 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf89 has 0 variants, skipping\n",
      "[INFO]\tGene TCF4 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7B has 0 variants, skipping\n",
      "[INFO]\tGene ANKUB1 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131107 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129307 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3I has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF706 has 0 variants, skipping\n",
      "[INFO]\tGene SCFD1 has 0 variants, skipping\n",
      "[INFO]\tGene KLF13 has 0 variants, skipping\n",
      "[INFO]\tGene GINS2 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1D has 0 variants, skipping\n",
      "[INFO]\tGene SAR1B has 0 variants, skipping\n",
      "[INFO]\tGene PLLP has 0 variants, skipping\n",
      "[INFO]\tGene C1QA has 0 variants, skipping\n",
      "[INFO]\tGene EIF3L has 0 variants, skipping\n",
      "[INFO]\tGene NIP7 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPE has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CEPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEANC2 has 0 variants, skipping\n",
      "[INFO]\tGene HPGD has 0 variants, skipping\n",
      "[INFO]\tGene DENND4B has 0 variants, skipping\n",
      "[INFO]\tGene FLACC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT3H has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02417 has 0 variants, skipping\n",
      "[INFO]\tGene THCAT155 has 0 variants, skipping\n",
      "[INFO]\tGene THRB has 0 variants, skipping\n",
      "[INFO]\tGene CCND2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PADI4 has 0 variants, skipping\n",
      "[INFO]\tGene NUPR1 has 0 variants, skipping\n",
      "[INFO]\tGene GMEB2 has 0 variants, skipping\n",
      "[INFO]\tGene OSTF1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC33 has 0 variants, skipping\n",
      "[INFO]\tGene ORC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B8 has 0 variants, skipping\n",
      "[INFO]\tGene OR7C2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX1 has 0 variants, skipping\n",
      "[INFO]\tGene CABP4 has 0 variants, skipping\n",
      "[INFO]\tGene ESRRB has 0 variants, skipping\n",
      "[INFO]\tGene FAM43A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02443 has 0 variants, skipping\n",
      "[INFO]\tGene MYL4 has 0 variants, skipping\n",
      "[INFO]\tGene CRPPA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA120 has 0 variants, skipping\n",
      "[INFO]\tGene KYAT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AT4 has 0 variants, skipping\n",
      "[INFO]\tGene LYAR has 0 variants, skipping\n",
      "[INFO]\tGene KIF3A has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA4 has 0 variants, skipping\n",
      "[INFO]\tGene CD27-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK17 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02570 has 0 variants, skipping\n",
      "[INFO]\tGene PARP12 has 0 variants, skipping\n",
      "[INFO]\tGene ACD has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH1 has 0 variants, skipping\n",
      "[INFO]\tGene TCIRG1 has 0 variants, skipping\n",
      "[INFO]\tGene AFG1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC02737 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02746 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02711 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02732 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF778 has 0 variants, skipping\n",
      "[INFO]\tGene MED18 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12B has 0 variants, skipping\n",
      "[INFO]\tGene ACADM has 0 variants, skipping\n",
      "[INFO]\tGene EFTUD2 has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA11 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA3 has 0 variants, skipping\n",
      "[INFO]\tGene TM6SF1 has 0 variants, skipping\n",
      "[INFO]\tGene CHAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MRS2 has 0 variants, skipping\n",
      "[INFO]\tGene KIDINS220 has 0 variants, skipping\n",
      "[INFO]\tGene NIFK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5A has 0 variants, skipping\n",
      "[INFO]\tGene INHBC has 0 variants, skipping\n",
      "[INFO]\tGene TNIK has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369632 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13P5 has 0 variants, skipping\n",
      "[INFO]\tGene DSTNP2 has 0 variants, skipping\n",
      "[INFO]\tGene PROSER3 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU7-1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf57 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200CHG has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA12 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1C has 0 variants, skipping\n",
      "[INFO]\tGene MRPL40 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR4 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT36 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF9 has 0 variants, skipping\n",
      "[INFO]\tGene KRT37 has 0 variants, skipping\n",
      "[INFO]\tGene PEA15 has 0 variants, skipping\n",
      "[INFO]\tGene C1RL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELF4 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4B has 0 variants, skipping\n",
      "[INFO]\tGene FUZ has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD3-TMEFF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A1 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC4-TTLL3 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP85L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF544 has 0 variants, skipping\n",
      "[INFO]\tGene SYT12 has 0 variants, skipping\n",
      "[INFO]\tGene KIF2C has 0 variants, skipping\n",
      "[INFO]\tGene RUSC2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724562 has 0 variants, skipping\n",
      "[INFO]\tGene CA8 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP4 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM66C has 0 variants, skipping\n",
      "[INFO]\tGene FAM86FP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02449 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00937 has 0 variants, skipping\n",
      "[INFO]\tGene CDH11 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM267 has 0 variants, skipping\n",
      "[INFO]\tGene CMC1 has 0 variants, skipping\n",
      "[INFO]\tGene L3HYPDH has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2A has 0 variants, skipping\n",
      "[INFO]\tGene PTAR1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL22 has 0 variants, skipping\n",
      "[INFO]\tGene MFFP2 has 0 variants, skipping\n",
      "[INFO]\tGene TMED8 has 0 variants, skipping\n",
      "[INFO]\tGene TFB1M has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD1L has 0 variants, skipping\n",
      "[INFO]\tGene WDR45 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9C1 has 0 variants, skipping\n",
      "[INFO]\tGene LY86-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene A2M-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GNB1 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1 has 0 variants, skipping\n",
      "[INFO]\tGene BEST2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB8 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ2A has 0 variants, skipping\n",
      "[INFO]\tGene BAZ1A has 0 variants, skipping\n",
      "[INFO]\tGene PLK4 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15L2 has 0 variants, skipping\n",
      "[INFO]\tGene A2MP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00987 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642846 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02367 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928030 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728715 has 0 variants, skipping\n",
      "[INFO]\tGene DDX12P has 0 variants, skipping\n",
      "[INFO]\tGene LOC374443 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646588 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A4 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02470 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4C has 0 variants, skipping\n",
      "[INFO]\tGene GRK2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP32 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD3 has 0 variants, skipping\n",
      "[INFO]\tGene DCTD has 0 variants, skipping\n",
      "[INFO]\tGene ITPA has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2L has 0 variants, skipping\n",
      "[INFO]\tGene PHF8 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNND2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02617 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L5 has 0 variants, skipping\n",
      "[INFO]\tGene GANC has 0 variants, skipping\n",
      "[INFO]\tGene WASHC2A has 0 variants, skipping\n",
      "[INFO]\tGene ERG has 0 variants, skipping\n",
      "[INFO]\tGene F8A1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP188 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02446 has 0 variants, skipping\n",
      "[INFO]\tGene KLRA1P has 0 variants, skipping\n",
      "[INFO]\tGene CRELD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN2 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO1R has 0 variants, skipping\n",
      "[INFO]\tGene ZBBX has 0 variants, skipping\n",
      "[INFO]\tGene ZNF81 has 0 variants, skipping\n",
      "[INFO]\tGene PRH1-PRR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440084 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01252 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1A has 0 variants, skipping\n",
      "[INFO]\tGene CCP110 has 0 variants, skipping\n",
      "[INFO]\tGene NAGLU has 0 variants, skipping\n",
      "[INFO]\tGene RPL22 has 0 variants, skipping\n",
      "[INFO]\tGene PDE5A has 0 variants, skipping\n",
      "[INFO]\tGene MIR614 has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR7P1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1H has 0 variants, skipping\n",
      "[INFO]\tGene APOLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF665 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF525 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01559 has 0 variants, skipping\n",
      "[INFO]\tGene MCUR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM39B has 0 variants, skipping\n",
      "[INFO]\tGene PLBD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene QPCTL has 0 variants, skipping\n",
      "[INFO]\tGene SYTL5 has 0 variants, skipping\n",
      "[INFO]\tGene NAPSA has 0 variants, skipping\n",
      "[INFO]\tGene CREB5 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF6 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC9 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT21 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA7 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN5 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01489 has 0 variants, skipping\n",
      "[INFO]\tGene RERG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1R2 has 0 variants, skipping\n",
      "[INFO]\tGene LUZP6 has 0 variants, skipping\n",
      "[INFO]\tGene EXT1 has 0 variants, skipping\n",
      "[INFO]\tGene ETFA has 0 variants, skipping\n",
      "[INFO]\tGene MPV17L has 0 variants, skipping\n",
      "[INFO]\tGene BSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene GASK1B has 0 variants, skipping\n",
      "[INFO]\tGene SMOC2 has 0 variants, skipping\n",
      "[INFO]\tGene UPP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A1 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC8 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf85 has 0 variants, skipping\n",
      "[INFO]\tGene MIR544A has 0 variants, skipping\n",
      "[INFO]\tGene MIR574 has 0 variants, skipping\n",
      "[INFO]\tGene SKP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02378 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3974 has 0 variants, skipping\n",
      "[INFO]\tGene RERGL has 0 variants, skipping\n",
      "[INFO]\tGene GNRH1 has 0 variants, skipping\n",
      "[INFO]\tGene DDC has 0 variants, skipping\n",
      "[INFO]\tGene HTT has 0 variants, skipping\n",
      "[INFO]\tGene RNF157 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02468 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1D has 0 variants, skipping\n",
      "[INFO]\tGene BTAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE3 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD10 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF39 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518C has 0 variants, skipping\n",
      "[INFO]\tGene KCNH5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80E has 0 variants, skipping\n",
      "[INFO]\tGene DMBX1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP21 has 0 variants, skipping\n",
      "[INFO]\tGene CERS5 has 0 variants, skipping\n",
      "[INFO]\tGene CMAS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8A has 0 variants, skipping\n",
      "[INFO]\tGene WT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF483 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369691 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928441 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT3 has 0 variants, skipping\n",
      "[INFO]\tGene MDS2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM10 has 0 variants, skipping\n",
      "[INFO]\tGene NAPB has 0 variants, skipping\n",
      "[INFO]\tGene COA6 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP6 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene EXD2 has 0 variants, skipping\n",
      "[INFO]\tGene DOK2 has 0 variants, skipping\n",
      "[INFO]\tGene CHM has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505841 has 0 variants, skipping\n",
      "[INFO]\tGene SUV39H2 has 0 variants, skipping\n",
      "[INFO]\tGene FUT8 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA3 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG16 has 0 variants, skipping\n",
      "[INFO]\tGene PALM2AKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL6 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNE has 0 variants, skipping\n",
      "[INFO]\tGene POLK has 0 variants, skipping\n",
      "[INFO]\tGene BORCS8-MEF2B has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP10 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D21 has 0 variants, skipping\n",
      "[INFO]\tGene COQ10B has 0 variants, skipping\n",
      "[INFO]\tGene LEO1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4302 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NAB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF593 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf87 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC75B has 0 variants, skipping\n",
      "[INFO]\tGene IFNAR2 has 0 variants, skipping\n",
      "[INFO]\tGene ERVFRD-1 has 0 variants, skipping\n",
      "[INFO]\tGene DUOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP15 has 0 variants, skipping\n",
      "[INFO]\tGene TYMS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF215 has 0 variants, skipping\n",
      "[INFO]\tGene NOD1 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729291 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD5 has 0 variants, skipping\n",
      "[INFO]\tGene ATG9A has 0 variants, skipping\n",
      "[INFO]\tGene CYFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506606 has 0 variants, skipping\n",
      "[INFO]\tGene OVCH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AUTS2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNBL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAB1 has 0 variants, skipping\n",
      "[INFO]\tGene STIL has 0 variants, skipping\n",
      "[INFO]\tGene RTF2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM54 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF341 has 0 variants, skipping\n",
      "[INFO]\tGene NAIP has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645485 has 0 variants, skipping\n",
      "[INFO]\tGene GNA14 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FCRLA has 0 variants, skipping\n",
      "[INFO]\tGene CD84 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH19 has 0 variants, skipping\n",
      "[INFO]\tGene GPRASP2 has 0 variants, skipping\n",
      "[INFO]\tGene OVOS2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP58 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ13224 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02387 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRB has 0 variants, skipping\n",
      "[INFO]\tGene DENND5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOT1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02422 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf74 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF569 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf25 has 0 variants, skipping\n",
      "[INFO]\tGene FAM187B has 0 variants, skipping\n",
      "[INFO]\tGene TTC9B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF583 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF565 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF560 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM91 has 0 variants, skipping\n",
      "[INFO]\tGene BEND7 has 0 variants, skipping\n",
      "[INFO]\tGene LMX1B has 0 variants, skipping\n",
      "[INFO]\tGene LOX has 0 variants, skipping\n",
      "[INFO]\tGene RNF5 has 0 variants, skipping\n",
      "[INFO]\tGene RLN1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD2 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3B has 0 variants, skipping\n",
      "[INFO]\tGene DDX52 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE1 has 0 variants, skipping\n",
      "[INFO]\tGene CRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02471 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAA1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX1 has 0 variants, skipping\n",
      "[INFO]\tGene BBX has 0 variants, skipping\n",
      "[INFO]\tGene TREX1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2L has 0 variants, skipping\n",
      "[INFO]\tGene PRR4 has 0 variants, skipping\n",
      "[INFO]\tGene SNF8 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP12 has 0 variants, skipping\n",
      "[INFO]\tGene PXMP4 has 0 variants, skipping\n",
      "[INFO]\tGene SP140 has 0 variants, skipping\n",
      "[INFO]\tGene CHP1 has 0 variants, skipping\n",
      "[INFO]\tGene XPOT has 0 variants, skipping\n",
      "[INFO]\tGene HRH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02400 has 0 variants, skipping\n",
      "[INFO]\tGene YAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7851 has 0 variants, skipping\n",
      "[INFO]\tGene IFNGR1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL58 has 0 variants, skipping\n",
      "[INFO]\tGene HGSNAT has 0 variants, skipping\n",
      "[INFO]\tGene GPAT4 has 0 variants, skipping\n",
      "[INFO]\tGene CTSF has 0 variants, skipping\n",
      "[INFO]\tGene MYOM1 has 0 variants, skipping\n",
      "[INFO]\tGene MYH13 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE5 has 0 variants, skipping\n",
      "[INFO]\tGene NOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02402 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02451 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02450 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02461 has 0 variants, skipping\n",
      "[INFO]\tGene GRAPL has 0 variants, skipping\n",
      "[INFO]\tGene NSDHL has 0 variants, skipping\n",
      "[INFO]\tGene DPEP3 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG4 has 0 variants, skipping\n",
      "[INFO]\tGene GPX3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF397 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABI3 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA8P1 has 0 variants, skipping\n",
      "[INFO]\tGene CAB39 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00938 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF189 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF177 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288798 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4698 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369747 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02156 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf127 has 0 variants, skipping\n",
      "[INFO]\tGene ACY1 has 0 variants, skipping\n",
      "[INFO]\tGene MPPED2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF7 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK1 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3E has 0 variants, skipping\n",
      "[INFO]\tGene INPPL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRG1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTRL has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT24 has 0 variants, skipping\n",
      "[INFO]\tGene LRP10 has 0 variants, skipping\n",
      "[INFO]\tGene PAX7 has 0 variants, skipping\n",
      "[INFO]\tGene UNC50 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC4 has 0 variants, skipping\n",
      "[INFO]\tGene PAX3 has 0 variants, skipping\n",
      "[INFO]\tGene SPCS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10C1 has 0 variants, skipping\n",
      "[INFO]\tGene ZP4 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERG has 0 variants, skipping\n",
      "[INFO]\tGene C6orf47 has 0 variants, skipping\n",
      "[INFO]\tGene HMHB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF7 has 0 variants, skipping\n",
      "[INFO]\tGene GRHL3 has 0 variants, skipping\n",
      "[INFO]\tGene LSM2 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC2 has 0 variants, skipping\n",
      "[INFO]\tGene HAMP has 0 variants, skipping\n",
      "[INFO]\tGene CCAR2 has 0 variants, skipping\n",
      "[INFO]\tGene MAZ has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene GATAD1 has 0 variants, skipping\n",
      "[INFO]\tGene BRINP2 has 0 variants, skipping\n",
      "[INFO]\tGene POLD4 has 0 variants, skipping\n",
      "[INFO]\tGene HES4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40C has 0 variants, skipping\n",
      "[INFO]\tGene AP1M1 has 0 variants, skipping\n",
      "[INFO]\tGene MCHR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928841 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6505 has 0 variants, skipping\n",
      "[INFO]\tGene OR4P4 has 0 variants, skipping\n",
      "[INFO]\tGene AVL9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR1291 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2B has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P3 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POC5 has 0 variants, skipping\n",
      "[INFO]\tGene RNF24 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS5 has 0 variants, skipping\n",
      "[INFO]\tGene RUBCNL has 0 variants, skipping\n",
      "[INFO]\tGene CERCAM has 0 variants, skipping\n",
      "[INFO]\tGene SETD4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS27 has 0 variants, skipping\n",
      "[INFO]\tGene SIMC1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF220 has 0 variants, skipping\n",
      "[INFO]\tGene SMG8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927267 has 0 variants, skipping\n",
      "[INFO]\tGene COLCA2 has 0 variants, skipping\n",
      "[INFO]\tGene EMC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO18A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100335030 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM159 has 0 variants, skipping\n",
      "[INFO]\tGene REL has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K2 has 0 variants, skipping\n",
      "[INFO]\tGene LARP6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM13B has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B3 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MRM3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL11 has 0 variants, skipping\n",
      "[INFO]\tGene INTS10 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS10 has 0 variants, skipping\n",
      "[INFO]\tGene CEP72 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TBCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT6 has 0 variants, skipping\n",
      "[INFO]\tGene ASPM has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724265 has 0 variants, skipping\n",
      "[INFO]\tGene BCDIN3D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB6 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02396 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02395 has 0 variants, skipping\n",
      "[INFO]\tGene SS18L2 has 0 variants, skipping\n",
      "[INFO]\tGene RSL24D1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL9 has 0 variants, skipping\n",
      "[INFO]\tGene BIN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GPN3 has 0 variants, skipping\n",
      "[INFO]\tGene ARPP21 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB1 has 0 variants, skipping\n",
      "[INFO]\tGene RAI1 has 0 variants, skipping\n",
      "[INFO]\tGene PTER has 0 variants, skipping\n",
      "[INFO]\tGene SMCP has 0 variants, skipping\n",
      "[INFO]\tGene CKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene GRINA has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM87B has 0 variants, skipping\n",
      "[INFO]\tGene SENCR has 0 variants, skipping\n",
      "[INFO]\tGene CCDC169-SOHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A47 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-2 has 0 variants, skipping\n",
      "[INFO]\tGene CLUL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1293 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-11 has 0 variants, skipping\n",
      "[INFO]\tGene MARK3 has 0 variants, skipping\n",
      "[INFO]\tGene PDS5A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM192 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL3 has 0 variants, skipping\n",
      "[INFO]\tGene PLPBP has 0 variants, skipping\n",
      "[INFO]\tGene DUSP10 has 0 variants, skipping\n",
      "[INFO]\tGene FZD10 has 0 variants, skipping\n",
      "[INFO]\tGene DDX20 has 0 variants, skipping\n",
      "[INFO]\tGene NFYC has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA8 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF1 has 0 variants, skipping\n",
      "[INFO]\tGene HSF4 has 0 variants, skipping\n",
      "[INFO]\tGene MPEG1 has 0 variants, skipping\n",
      "[INFO]\tGene PRB1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOP1B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD133 has 0 variants, skipping\n",
      "[INFO]\tGene VSNL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC16 has 0 variants, skipping\n",
      "[INFO]\tGene TYMSOS has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR4 has 0 variants, skipping\n",
      "[INFO]\tGene OSCP1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM2 has 0 variants, skipping\n",
      "[INFO]\tGene SFMBT1 has 0 variants, skipping\n",
      "[INFO]\tGene UMODL1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1B has 0 variants, skipping\n",
      "[INFO]\tGene ACTL6A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA3 has 0 variants, skipping\n",
      "[INFO]\tGene SART1 has 0 variants, skipping\n",
      "[INFO]\tGene MUL1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL24 has 0 variants, skipping\n",
      "[INFO]\tGene RNF128 has 0 variants, skipping\n",
      "[INFO]\tGene CARS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHPF has 0 variants, skipping\n",
      "[INFO]\tGene BLVRA has 0 variants, skipping\n",
      "[INFO]\tGene CHRAC1 has 0 variants, skipping\n",
      "[INFO]\tGene NEMP2 has 0 variants, skipping\n",
      "[INFO]\tGene CCM2 has 0 variants, skipping\n",
      "[INFO]\tGene BAX has 0 variants, skipping\n",
      "[INFO]\tGene BAD has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MPL has 0 variants, skipping\n",
      "[INFO]\tGene MYRF has 0 variants, skipping\n",
      "[INFO]\tGene ERC1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT78 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24B has 0 variants, skipping\n",
      "[INFO]\tGene ARID3B has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13A has 0 variants, skipping\n",
      "[INFO]\tGene APOL2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM9 has 0 variants, skipping\n",
      "[INFO]\tGene GRIFIN has 0 variants, skipping\n",
      "[INFO]\tGene LENG9 has 0 variants, skipping\n",
      "[INFO]\tGene ASNSD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E47P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00592 has 0 variants, skipping\n",
      "[INFO]\tGene KRT7-AS has 0 variants, skipping\n",
      "[INFO]\tGene KRT87P has 0 variants, skipping\n",
      "[INFO]\tGene PCTP has 0 variants, skipping\n",
      "[INFO]\tGene KRT8 has 0 variants, skipping\n",
      "[INFO]\tGene KRT4 has 0 variants, skipping\n",
      "[INFO]\tGene IPO5 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT73-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1G has 0 variants, skipping\n",
      "[INFO]\tGene BAAT has 0 variants, skipping\n",
      "[INFO]\tGene LINGO2 has 0 variants, skipping\n",
      "[INFO]\tGene TANK has 0 variants, skipping\n",
      "[INFO]\tGene ZNF607 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205C has 0 variants, skipping\n",
      "[INFO]\tGene LOC283335 has 0 variants, skipping\n",
      "[INFO]\tGene C4BPA has 0 variants, skipping\n",
      "[INFO]\tGene TSPO has 0 variants, skipping\n",
      "[INFO]\tGene BLVRB has 0 variants, skipping\n",
      "[INFO]\tGene BCKDHA has 0 variants, skipping\n",
      "[INFO]\tGene AMY2A has 0 variants, skipping\n",
      "[INFO]\tGene BDKRB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6757 has 0 variants, skipping\n",
      "[INFO]\tGene MYEF2 has 0 variants, skipping\n",
      "[INFO]\tGene NBN has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3614 has 0 variants, skipping\n",
      "[INFO]\tGene MIR28 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR9 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf62 has 0 variants, skipping\n",
      "[INFO]\tGene ISCA1 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT1L has 0 variants, skipping\n",
      "[INFO]\tGene SHCBP1L has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRNDE has 0 variants, skipping\n",
      "[INFO]\tGene PCSK9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652999 has 0 variants, skipping\n",
      "[INFO]\tGene IMP3 has 0 variants, skipping\n",
      "[INFO]\tGene ANPEP has 0 variants, skipping\n",
      "[INFO]\tGene RRP12 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT4 has 0 variants, skipping\n",
      "[INFO]\tGene CISTR has 0 variants, skipping\n",
      "[INFO]\tGene GNG11 has 0 variants, skipping\n",
      "[INFO]\tGene GMFB has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ12825 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240735 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240734 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02381 has 0 variants, skipping\n",
      "[INFO]\tGene TEX38 has 0 variants, skipping\n",
      "[INFO]\tGene FAM186A has 0 variants, skipping\n",
      "[INFO]\tGene NFIA has 0 variants, skipping\n",
      "[INFO]\tGene VPS37A has 0 variants, skipping\n",
      "[INFO]\tGene JAGN1 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724050 has 0 variants, skipping\n",
      "[INFO]\tGene MED13 has 0 variants, skipping\n",
      "[INFO]\tGene THRAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MED12 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf159 has 0 variants, skipping\n",
      "[INFO]\tGene GNRHR has 0 variants, skipping\n",
      "[INFO]\tGene CSF2RB has 0 variants, skipping\n",
      "[INFO]\tGene GM2A has 0 variants, skipping\n",
      "[INFO]\tGene ADAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARA5 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP41 has 0 variants, skipping\n",
      "[INFO]\tGene SREK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP57L1 has 0 variants, skipping\n",
      "[INFO]\tGene RELL2 has 0 variants, skipping\n",
      "[INFO]\tGene COX18 has 0 variants, skipping\n",
      "[INFO]\tGene TCAIM has 0 variants, skipping\n",
      "[INFO]\tGene RABL3 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf38 has 0 variants, skipping\n",
      "[INFO]\tGene FAM126B has 0 variants, skipping\n",
      "[INFO]\tGene RTP5 has 0 variants, skipping\n",
      "[INFO]\tGene CES4A has 0 variants, skipping\n",
      "[INFO]\tGene PRTG has 0 variants, skipping\n",
      "[INFO]\tGene LMBR1L has 0 variants, skipping\n",
      "[INFO]\tGene AKT3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00337 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW3P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM143 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280B has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S1-RDH5 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHIA has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD4 has 0 variants, skipping\n",
      "[INFO]\tGene CORO7 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM198B has 0 variants, skipping\n",
      "[INFO]\tGene POSTN has 0 variants, skipping\n",
      "[INFO]\tGene TMEM158 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT10 has 0 variants, skipping\n",
      "[INFO]\tGene STS has 0 variants, skipping\n",
      "[INFO]\tGene TCOF1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL9 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H13 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3H has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369781 has 0 variants, skipping\n",
      "[INFO]\tGene RBM14 has 0 variants, skipping\n",
      "[INFO]\tGene LTF has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A7 has 0 variants, skipping\n",
      "[INFO]\tGene ACSF2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD16A has 0 variants, skipping\n",
      "[INFO]\tGene TMED1 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD18 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3G has 0 variants, skipping\n",
      "[INFO]\tGene RTN3 has 0 variants, skipping\n",
      "[INFO]\tGene INSL3 has 0 variants, skipping\n",
      "[INFO]\tGene URM1 has 0 variants, skipping\n",
      "[INFO]\tGene DACT3 has 0 variants, skipping\n",
      "[INFO]\tGene KPNB1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAGA has 0 variants, skipping\n",
      "[INFO]\tGene CHI3L1 has 0 variants, skipping\n",
      "[INFO]\tGene CHGA has 0 variants, skipping\n",
      "[INFO]\tGene CHD4 has 0 variants, skipping\n",
      "[INFO]\tGene CHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105C has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4C has 0 variants, skipping\n",
      "[INFO]\tGene DAOA has 0 variants, skipping\n",
      "[INFO]\tGene C7orf57 has 0 variants, skipping\n",
      "[INFO]\tGene AFDN has 0 variants, skipping\n",
      "[INFO]\tGene PTPRC has 0 variants, skipping\n",
      "[INFO]\tGene SNORD59B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD59A has 0 variants, skipping\n",
      "[INFO]\tGene CWC25 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37B has 0 variants, skipping\n",
      "[INFO]\tGene FCN2 has 0 variants, skipping\n",
      "[INFO]\tGene WARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADARB1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD2 has 0 variants, skipping\n",
      "[INFO]\tGene TACR1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8 has 0 variants, skipping\n",
      "[INFO]\tGene RDH8 has 0 variants, skipping\n",
      "[INFO]\tGene CALY has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN4 has 0 variants, skipping\n",
      "[INFO]\tGene PODXL2 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG11B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354B has 0 variants, skipping\n",
      "[INFO]\tGene TECTB has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC6 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51C has 0 variants, skipping\n",
      "[INFO]\tGene TMEM185A has 0 variants, skipping\n",
      "[INFO]\tGene ARMC6 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene BEAN1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGPS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC63 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID4B has 0 variants, skipping\n",
      "[INFO]\tGene LRP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene HSCB has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPL has 0 variants, skipping\n",
      "[INFO]\tGene ADCK5 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM14 has 0 variants, skipping\n",
      "[INFO]\tGene MCEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene OGFRL1 has 0 variants, skipping\n",
      "[INFO]\tGene FILIP1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCG has 0 variants, skipping\n",
      "[INFO]\tGene NBPF20 has 0 variants, skipping\n",
      "[INFO]\tGene NOMO3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA7 has 0 variants, skipping\n",
      "[INFO]\tGene SAA2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2C1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf86 has 0 variants, skipping\n",
      "[INFO]\tGene FARP1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGR has 0 variants, skipping\n",
      "[INFO]\tGene LCN10 has 0 variants, skipping\n",
      "[INFO]\tGene CD93 has 0 variants, skipping\n",
      "[INFO]\tGene PBK has 0 variants, skipping\n",
      "[INFO]\tGene CRB1 has 0 variants, skipping\n",
      "[INFO]\tGene FECH has 0 variants, skipping\n",
      "[INFO]\tGene RBL1 has 0 variants, skipping\n",
      "[INFO]\tGene DERL3 has 0 variants, skipping\n",
      "[INFO]\tGene PPWD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIZ has 0 variants, skipping\n",
      "[INFO]\tGene JRK has 0 variants, skipping\n",
      "[INFO]\tGene VPS45 has 0 variants, skipping\n",
      "[INFO]\tGene TIMD4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6758 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF27 has 0 variants, skipping\n",
      "[INFO]\tGene LRP8 has 0 variants, skipping\n",
      "[INFO]\tGene KNSTRN has 0 variants, skipping\n",
      "[INFO]\tGene TP53INP1 has 0 variants, skipping\n",
      "[INFO]\tGene TBL1Y has 0 variants, skipping\n",
      "[INFO]\tGene SAMD3 has 0 variants, skipping\n",
      "[INFO]\tGene NF1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP57 has 0 variants, skipping\n",
      "[INFO]\tGene UBIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT4 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK2 has 0 variants, skipping\n",
      "[INFO]\tGene SAC3D1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR171 has 0 variants, skipping\n",
      "[INFO]\tGene DGKH has 0 variants, skipping\n",
      "[INFO]\tGene MED21 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927583 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF12 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene UEVLD has 0 variants, skipping\n",
      "[INFO]\tGene LIM2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6759 has 0 variants, skipping\n",
      "[INFO]\tGene PSCA has 0 variants, skipping\n",
      "[INFO]\tGene UBXN8 has 0 variants, skipping\n",
      "[INFO]\tGene REEP5 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283387 has 0 variants, skipping\n",
      "[INFO]\tGene RTL1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB9 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC10A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02403 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02388 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506869 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A18 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP100 has 0 variants, skipping\n",
      "[INFO]\tGene WDR53 has 0 variants, skipping\n",
      "[INFO]\tGene GEN1 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5KL1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF12 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC23 has 0 variants, skipping\n",
      "[INFO]\tGene VPS52 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SELL has 0 variants, skipping\n",
      "[INFO]\tGene RNF123 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN16 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF10-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR6125 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB11 has 0 variants, skipping\n",
      "[INFO]\tGene CARHSP1 has 0 variants, skipping\n",
      "[INFO]\tGene POLE3 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01465 has 0 variants, skipping\n",
      "[INFO]\tGene RXYLT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPN has 0 variants, skipping\n",
      "[INFO]\tGene PTGER2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGDS has 0 variants, skipping\n",
      "[INFO]\tGene PTGDR has 0 variants, skipping\n",
      "[INFO]\tGene PTAFR has 0 variants, skipping\n",
      "[INFO]\tGene SSTR4 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG2 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR2 has 0 variants, skipping\n",
      "[INFO]\tGene PANK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Z has 0 variants, skipping\n",
      "[INFO]\tGene CLCA1 has 0 variants, skipping\n",
      "[INFO]\tGene AP3S1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02389 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02231 has 0 variants, skipping\n",
      "[INFO]\tGene ERF has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369187 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02454 has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6074 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02425 has 0 variants, skipping\n",
      "[INFO]\tGene LLPH-DT has 0 variants, skipping\n",
      "[INFO]\tGene C12orf4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6502 has 0 variants, skipping\n",
      "[INFO]\tGene LMCD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724421 has 0 variants, skipping\n",
      "[INFO]\tGene DST-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP3L has 0 variants, skipping\n",
      "[INFO]\tGene DAAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02408 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02421 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01479 has 0 variants, skipping\n",
      "[INFO]\tGene HCST has 0 variants, skipping\n",
      "[INFO]\tGene UBL7 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02384 has 0 variants, skipping\n",
      "[INFO]\tGene SCOC has 0 variants, skipping\n",
      "[INFO]\tGene GORAB has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70G has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507250 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2 has 0 variants, skipping\n",
      "[INFO]\tGene SMC1B has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM7 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130075 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND6 has 0 variants, skipping\n",
      "[INFO]\tGene CTSK has 0 variants, skipping\n",
      "[INFO]\tGene MIR1279 has 0 variants, skipping\n",
      "[INFO]\tGene ART4 has 0 variants, skipping\n",
      "[INFO]\tGene TMPPE has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A2 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3913-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3913-2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF487 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC2 has 0 variants, skipping\n",
      "[INFO]\tGene HCRTR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928002 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD12 has 0 variants, skipping\n",
      "[INFO]\tGene PIKFYVE has 0 variants, skipping\n",
      "[INFO]\tGene MORC2 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM243B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A24 has 0 variants, skipping\n",
      "[INFO]\tGene SYT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF641 has 0 variants, skipping\n",
      "[INFO]\tGene GAB4 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNA has 0 variants, skipping\n",
      "[INFO]\tGene PIN4 has 0 variants, skipping\n",
      "[INFO]\tGene PIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CB has 0 variants, skipping\n",
      "[INFO]\tGene MOGS has 0 variants, skipping\n",
      "[INFO]\tGene MIR7977 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM7 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM181 has 0 variants, skipping\n",
      "[INFO]\tGene MRS2P2 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC11 has 0 variants, skipping\n",
      "[INFO]\tGene TRHDE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02444 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM2 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130268 has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX3X has 0 variants, skipping\n",
      "[INFO]\tGene PTPA has 0 variants, skipping\n",
      "[INFO]\tGene OGA has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC10 has 0 variants, skipping\n",
      "[INFO]\tGene TADA1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF510 has 0 variants, skipping\n",
      "[INFO]\tGene EHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene LMBRD2 has 0 variants, skipping\n",
      "[INFO]\tGene CAND1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA4 has 0 variants, skipping\n",
      "[INFO]\tGene SARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD5 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02424 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L3 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1C has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG11 has 0 variants, skipping\n",
      "[INFO]\tGene ELF3 has 0 variants, skipping\n",
      "[INFO]\tGene MVK has 0 variants, skipping\n",
      "[INFO]\tGene RD3 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf97 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01490 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf73 has 0 variants, skipping\n",
      "[INFO]\tGene MIR617 has 0 variants, skipping\n",
      "[INFO]\tGene ITGBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4699 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM19 has 0 variants, skipping\n",
      "[INFO]\tGene TTC7A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF207 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3D has 0 variants, skipping\n",
      "[INFO]\tGene CANT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2B has 0 variants, skipping\n",
      "[INFO]\tGene GALM has 0 variants, skipping\n",
      "[INFO]\tGene PIH1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM43 has 0 variants, skipping\n",
      "[INFO]\tGene PLD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM45B has 0 variants, skipping\n",
      "[INFO]\tGene IFTAP has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02426 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC59 has 0 variants, skipping\n",
      "[INFO]\tGene LCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF133 has 0 variants, skipping\n",
      "[INFO]\tGene ADAD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9B1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN8 has 0 variants, skipping\n",
      "[INFO]\tGene L1CAM has 0 variants, skipping\n",
      "[INFO]\tGene GDNF has 0 variants, skipping\n",
      "[INFO]\tGene COL5A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369879 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02258 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN9P has 0 variants, skipping\n",
      "[INFO]\tGene MYO1C has 0 variants, skipping\n",
      "[INFO]\tGene H3-2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L22 has 0 variants, skipping\n",
      "[INFO]\tGene STAR has 0 variants, skipping\n",
      "[INFO]\tGene CD14 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02458 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4 has 0 variants, skipping\n",
      "[INFO]\tGene POC1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH4A has 0 variants, skipping\n",
      "[INFO]\tGene MTX3 has 0 variants, skipping\n",
      "[INFO]\tGene MAD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene IMPDH1 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP20 has 0 variants, skipping\n",
      "[INFO]\tGene HMG20A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF23 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02399 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02392 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00615 has 0 variants, skipping\n",
      "[INFO]\tGene SOX21 has 0 variants, skipping\n",
      "[INFO]\tGene MMP23B has 0 variants, skipping\n",
      "[INFO]\tGene DENND3 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928617 has 0 variants, skipping\n",
      "[INFO]\tGene CGB1 has 0 variants, skipping\n",
      "[INFO]\tGene NCDN has 0 variants, skipping\n",
      "[INFO]\tGene LINC02397 has 0 variants, skipping\n",
      "[INFO]\tGene SGPP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02412 has 0 variants, skipping\n",
      "[INFO]\tGene PREB has 0 variants, skipping\n",
      "[INFO]\tGene SOCS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A44 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928731 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369911 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25G has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1A has 0 variants, skipping\n",
      "[INFO]\tGene BEX5 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX6 has 0 variants, skipping\n",
      "[INFO]\tGene ARSL has 0 variants, skipping\n",
      "[INFO]\tGene SPRING1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS48 has 0 variants, skipping\n",
      "[INFO]\tGene CEP83-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5700 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7844 has 0 variants, skipping\n",
      "[INFO]\tGene KRT19P2 has 0 variants, skipping\n",
      "[INFO]\tGene MORN4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4G has 0 variants, skipping\n",
      "[INFO]\tGene FOXP1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB1B has 0 variants, skipping\n",
      "[INFO]\tGene C19orf44 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106A has 0 variants, skipping\n",
      "[INFO]\tGene RABEP1 has 0 variants, skipping\n",
      "[INFO]\tGene NBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1A has 0 variants, skipping\n",
      "[INFO]\tGene CT45A7 has 0 variants, skipping\n",
      "[INFO]\tGene THY1 has 0 variants, skipping\n",
      "[INFO]\tGene PTH2R has 0 variants, skipping\n",
      "[INFO]\tGene SHH has 0 variants, skipping\n",
      "[INFO]\tGene GALNS has 0 variants, skipping\n",
      "[INFO]\tGene AAR2 has 0 variants, skipping\n",
      "[INFO]\tGene SZRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3685 has 0 variants, skipping\n",
      "[INFO]\tGene COPS2 has 0 variants, skipping\n",
      "[INFO]\tGene VIPAS39 has 0 variants, skipping\n",
      "[INFO]\tGene SCAP has 0 variants, skipping\n",
      "[INFO]\tGene ADAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF423 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM1P5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF883 has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1C1L has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369921 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02410 has 0 variants, skipping\n",
      "[INFO]\tGene RPP30 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP57 has 0 variants, skipping\n",
      "[INFO]\tGene POLL has 0 variants, skipping\n",
      "[INFO]\tGene ADH6 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC10 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF7 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD2 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52 has 0 variants, skipping\n",
      "[INFO]\tGene CTSV has 0 variants, skipping\n",
      "[INFO]\tGene PAEP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF214 has 0 variants, skipping\n",
      "[INFO]\tGene NXT1 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4495 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4303 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A7P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02453 has 0 variants, skipping\n",
      "[INFO]\tGene INSM2 has 0 variants, skipping\n",
      "[INFO]\tGene HINT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL3 has 0 variants, skipping\n",
      "[INFO]\tGene ACCS has 0 variants, skipping\n",
      "[INFO]\tGene NIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA53 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GAL1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5 has 0 variants, skipping\n",
      "[INFO]\tGene ADAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MTFMT has 0 variants, skipping\n",
      "[INFO]\tGene B9D1 has 0 variants, skipping\n",
      "[INFO]\tGene STAT6 has 0 variants, skipping\n",
      "[INFO]\tGene GPN1 has 0 variants, skipping\n",
      "[INFO]\tGene LONP2 has 0 variants, skipping\n",
      "[INFO]\tGene LRTM2 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10L has 0 variants, skipping\n",
      "[INFO]\tGene TEX33 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC1A has 0 variants, skipping\n",
      "[INFO]\tGene TBCK has 0 variants, skipping\n",
      "[INFO]\tGene CIART has 0 variants, skipping\n",
      "[INFO]\tGene PRKDC has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12B has 0 variants, skipping\n",
      "[INFO]\tGene C1orf198 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928937 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13A has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1827 has 0 variants, skipping\n",
      "[INFO]\tGene EPSTI1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA8 has 0 variants, skipping\n",
      "[INFO]\tGene C4B_2 has 0 variants, skipping\n",
      "[INFO]\tGene CYCS has 0 variants, skipping\n",
      "[INFO]\tGene PKLR has 0 variants, skipping\n",
      "[INFO]\tGene APAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3D has 0 variants, skipping\n",
      "[INFO]\tGene TRPC4AP has 0 variants, skipping\n",
      "[INFO]\tGene CLLU1 has 0 variants, skipping\n",
      "[INFO]\tGene GRB10 has 0 variants, skipping\n",
      "[INFO]\tGene EMCN has 0 variants, skipping\n",
      "[INFO]\tGene CTAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AQ has 0 variants, skipping\n",
      "[INFO]\tGene GJB3 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL11 has 0 variants, skipping\n",
      "[INFO]\tGene CHPF2 has 0 variants, skipping\n",
      "[INFO]\tGene TASOR has 0 variants, skipping\n",
      "[INFO]\tGene CCDC153 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB3 has 0 variants, skipping\n",
      "[INFO]\tGene SOX1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS6 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB10 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB4 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA10 has 0 variants, skipping\n",
      "[INFO]\tGene NEB has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00485 has 0 variants, skipping\n",
      "[INFO]\tGene LSMEM1 has 0 variants, skipping\n",
      "[INFO]\tGene FHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929058 has 0 variants, skipping\n",
      "[INFO]\tGene SEPHS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369945 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02401 has 0 variants, skipping\n",
      "[INFO]\tGene NPC1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRP2 has 0 variants, skipping\n",
      "[INFO]\tGene TREH has 0 variants, skipping\n",
      "[INFO]\tGene TTC41P has 0 variants, skipping\n",
      "[INFO]\tGene MIR3652 has 0 variants, skipping\n",
      "[INFO]\tGene SPAAR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF462 has 0 variants, skipping\n",
      "[INFO]\tGene NWD1 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC5 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM67 has 0 variants, skipping\n",
      "[INFO]\tGene DCK has 0 variants, skipping\n",
      "[INFO]\tGene DBH has 0 variants, skipping\n",
      "[INFO]\tGene DDX25 has 0 variants, skipping\n",
      "[INFO]\tGene LOC414300 has 0 variants, skipping\n",
      "[INFO]\tGene TOX3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF26 has 0 variants, skipping\n",
      "[INFO]\tGene REPS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE27 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC6 has 0 variants, skipping\n",
      "[INFO]\tGene KCCAT198 has 0 variants, skipping\n",
      "[INFO]\tGene CASC18 has 0 variants, skipping\n",
      "[INFO]\tGene ADAL has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287944 has 0 variants, skipping\n",
      "[INFO]\tGene THNSL2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505978 has 0 variants, skipping\n",
      "[INFO]\tGene TST has 0 variants, skipping\n",
      "[INFO]\tGene PSMD11 has 0 variants, skipping\n",
      "[INFO]\tGene WNK2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB30 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY7 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6 has 0 variants, skipping\n",
      "[INFO]\tGene FGR has 0 variants, skipping\n",
      "[INFO]\tGene FGF3 has 0 variants, skipping\n",
      "[INFO]\tGene AQR has 0 variants, skipping\n",
      "[INFO]\tGene SECISBP2L has 0 variants, skipping\n",
      "[INFO]\tGene CCDC178 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49D2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728739 has 0 variants, skipping\n",
      "[INFO]\tGene ETV7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01498 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP251 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B3 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF6 has 0 variants, skipping\n",
      "[INFO]\tGene SORBS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF256 has 0 variants, skipping\n",
      "[INFO]\tGene RCL1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST4 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR6 has 0 variants, skipping\n",
      "[INFO]\tGene USP30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY3 has 0 variants, skipping\n",
      "[INFO]\tGene ALAD has 0 variants, skipping\n",
      "[INFO]\tGene LINC01486 has 0 variants, skipping\n",
      "[INFO]\tGene BET1 has 0 variants, skipping\n",
      "[INFO]\tGene PBXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF225 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF223 has 0 variants, skipping\n",
      "[INFO]\tGene CEP68 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC14 has 0 variants, skipping\n",
      "[INFO]\tGene EVI2A has 0 variants, skipping\n",
      "[INFO]\tGene FAM222A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPST2 has 0 variants, skipping\n",
      "[INFO]\tGene DTHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DAP3 has 0 variants, skipping\n",
      "[INFO]\tGene CERT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM250 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf76 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL8 has 0 variants, skipping\n",
      "[INFO]\tGene OXLD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSACC has 0 variants, skipping\n",
      "[INFO]\tGene ADIG has 0 variants, skipping\n",
      "[INFO]\tGene ODAPH has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-1HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPS25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4474 has 0 variants, skipping\n",
      "[INFO]\tGene GLTPD2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01638 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01405 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369980 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6760 has 0 variants, skipping\n",
      "[INFO]\tGene CTTN has 0 variants, skipping\n",
      "[INFO]\tGene CRYAA2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6761 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128966 has 0 variants, skipping\n",
      "[INFO]\tGene COG5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3657 has 0 variants, skipping\n",
      "[INFO]\tGene NFXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6861 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF174 has 0 variants, skipping\n",
      "[INFO]\tGene PTH has 0 variants, skipping\n",
      "[INFO]\tGene THSD7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-1 has 0 variants, skipping\n",
      "[INFO]\tGene DNA2 has 0 variants, skipping\n",
      "[INFO]\tGene CLTA has 0 variants, skipping\n",
      "[INFO]\tGene NABP1-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCEPR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF596 has 0 variants, skipping\n",
      "[INFO]\tGene PSPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7106 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA11 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA4 has 0 variants, skipping\n",
      "[INFO]\tGene EGR3 has 0 variants, skipping\n",
      "[INFO]\tGene SERF2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6762 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF6 has 0 variants, skipping\n",
      "[INFO]\tGene LHX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPF2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02459 has 0 variants, skipping\n",
      "[INFO]\tGene TBX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4472-2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L7 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L30 has 0 variants, skipping\n",
      "[INFO]\tGene UTP14C has 0 variants, skipping\n",
      "[INFO]\tGene TRIB2 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2A has 0 variants, skipping\n",
      "[INFO]\tGene ADAM12 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506551 has 0 variants, skipping\n",
      "[INFO]\tGene CR1 has 0 variants, skipping\n",
      "[INFO]\tGene TESC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11A has 0 variants, skipping\n",
      "[INFO]\tGene KAT5 has 0 variants, skipping\n",
      "[INFO]\tGene METTL21C has 0 variants, skipping\n",
      "[INFO]\tGene SH3KBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS17 has 0 variants, skipping\n",
      "[INFO]\tGene RNMT has 0 variants, skipping\n",
      "[INFO]\tGene MRPS16 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA13 has 0 variants, skipping\n",
      "[INFO]\tGene TPPP3 has 0 variants, skipping\n",
      "[INFO]\tGene THAP4 has 0 variants, skipping\n",
      "[INFO]\tGene HDDC2 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO2B has 0 variants, skipping\n",
      "[INFO]\tGene FCF1 has 0 variants, skipping\n",
      "[INFO]\tGene MED31 has 0 variants, skipping\n",
      "[INFO]\tGene CUTC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02423 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02439 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370024 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00934 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF90 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF772 has 0 variants, skipping\n",
      "[INFO]\tGene TENM1 has 0 variants, skipping\n",
      "[INFO]\tGene BICDL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT4 has 0 variants, skipping\n",
      "[INFO]\tGene RBM10 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF11A has 0 variants, skipping\n",
      "[INFO]\tGene CFAP221 has 0 variants, skipping\n",
      "[INFO]\tGene PUS10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4498 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4-1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB43 has 0 variants, skipping\n",
      "[INFO]\tGene DNTTIP1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAV has 0 variants, skipping\n",
      "[INFO]\tGene SCRIB has 0 variants, skipping\n",
      "[INFO]\tGene GNA13 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4700 has 0 variants, skipping\n",
      "[INFO]\tGene BUD13 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPA-DT has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_009911 has 0 variants, skipping\n",
      "[INFO]\tGene ELK3 has 0 variants, skipping\n",
      "[INFO]\tGene ELK1 has 0 variants, skipping\n",
      "[INFO]\tGene LDB1 has 0 variants, skipping\n",
      "[INFO]\tGene KIN has 0 variants, skipping\n",
      "[INFO]\tGene KIF4A has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM2A has 0 variants, skipping\n",
      "[INFO]\tGene YJEFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF577 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF445 has 0 variants, skipping\n",
      "[INFO]\tGene NUP42 has 0 variants, skipping\n",
      "[INFO]\tGene ALOXE3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7107 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6F-LY6G6D has 0 variants, skipping\n",
      "[INFO]\tGene PDE4C has 0 variants, skipping\n",
      "[INFO]\tGene LOXL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP21A1P has 0 variants, skipping\n",
      "[INFO]\tGene GLRA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506691 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO1C has 0 variants, skipping\n",
      "[INFO]\tGene GSDMB has 0 variants, skipping\n",
      "[INFO]\tGene NAA16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101593348 has 0 variants, skipping\n",
      "[INFO]\tGene GGACT has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3B has 0 variants, skipping\n",
      "[INFO]\tGene KHNYN has 0 variants, skipping\n",
      "[INFO]\tGene CLIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF726 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH8 has 0 variants, skipping\n",
      "[INFO]\tGene GON7 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3 has 0 variants, skipping\n",
      "[INFO]\tGene ENKD1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28A has 0 variants, skipping\n",
      "[INFO]\tGene KCND3 has 0 variants, skipping\n",
      "[INFO]\tGene CDON has 0 variants, skipping\n",
      "[INFO]\tGene SF1 has 0 variants, skipping\n",
      "[INFO]\tGene PRUNE2 has 0 variants, skipping\n",
      "[INFO]\tGene PAXX has 0 variants, skipping\n",
      "[INFO]\tGene TBX15 has 0 variants, skipping\n",
      "[INFO]\tGene LAX1 has 0 variants, skipping\n",
      "[INFO]\tGene DDR2 has 0 variants, skipping\n",
      "[INFO]\tGene STK16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4304 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8072 has 0 variants, skipping\n",
      "[INFO]\tGene CWF19L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3908 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G7 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP40 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA9B has 0 variants, skipping\n",
      "[INFO]\tGene GAGE10 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE13 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2B has 0 variants, skipping\n",
      "[INFO]\tGene PAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR7 has 0 variants, skipping\n",
      "[INFO]\tGene ESR2 has 0 variants, skipping\n",
      "[INFO]\tGene STAMBP has 0 variants, skipping\n",
      "[INFO]\tGene ING4 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB43 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA8 has 0 variants, skipping\n",
      "[INFO]\tGene DHX38 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKE has 0 variants, skipping\n",
      "[INFO]\tGene MIR6880 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5188 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00939 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02359 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02350 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02347 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996671 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00944 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00943 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02372 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02405 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02376 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02411 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02393 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00507 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02441 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02369 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02368 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3612 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128276 has 0 variants, skipping\n",
      "[INFO]\tGene NBEAL1 has 0 variants, skipping\n",
      "[INFO]\tGene AMDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MPIG6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02419 has 0 variants, skipping\n",
      "[INFO]\tGene FZD10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A10 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R21 has 0 variants, skipping\n",
      "[INFO]\tGene POMZP3 has 0 variants, skipping\n",
      "[INFO]\tGene PXT1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR3 has 0 variants, skipping\n",
      "[INFO]\tGene INTS3 has 0 variants, skipping\n",
      "[INFO]\tGene FILIP1L has 0 variants, skipping\n",
      "[INFO]\tGene FAM3C has 0 variants, skipping\n",
      "[INFO]\tGene NSA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF557 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A3 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01257 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02415 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02370 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02414 has 0 variants, skipping\n",
      "[INFO]\tGene METTL5 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF26 has 0 variants, skipping\n",
      "[INFO]\tGene GABRD has 0 variants, skipping\n",
      "[INFO]\tGene SNORA49 has 0 variants, skipping\n",
      "[INFO]\tGene EP400P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02361 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928416 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6763 has 0 variants, skipping\n",
      "[INFO]\tGene SP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN22 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928530 has 0 variants, skipping\n",
      "[INFO]\tGene HMBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF76 has 0 variants, skipping\n",
      "[INFO]\tGene COPS3 has 0 variants, skipping\n",
      "[INFO]\tGene PI16 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3F has 0 variants, skipping\n",
      "[INFO]\tGene CENPL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF135 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC3 has 0 variants, skipping\n",
      "[INFO]\tGene ATF5 has 0 variants, skipping\n",
      "[INFO]\tGene CFD has 0 variants, skipping\n",
      "[INFO]\tGene ERP27 has 0 variants, skipping\n",
      "[INFO]\tGene MPL has 0 variants, skipping\n",
      "[INFO]\tGene MOS has 0 variants, skipping\n",
      "[INFO]\tGene ANXA6 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA5 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA4 has 0 variants, skipping\n",
      "[INFO]\tGene IATPR has 0 variants, skipping\n",
      "[INFO]\tGene PRRX1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF34 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM80 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT6 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B4 has 0 variants, skipping\n",
      "[INFO]\tGene NSMAF has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L4 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf67 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4L has 0 variants, skipping\n",
      "[INFO]\tGene NDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AWAT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A6 has 0 variants, skipping\n",
      "[INFO]\tGene PRR3 has 0 variants, skipping\n",
      "[INFO]\tGene AIFM1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYZ has 0 variants, skipping\n",
      "[INFO]\tGene MYO15B has 0 variants, skipping\n",
      "[INFO]\tGene GSG1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM230C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00417 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02481 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR375 has 0 variants, skipping\n",
      "[INFO]\tGene MIR373 has 0 variants, skipping\n",
      "[INFO]\tGene MIR372 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00408 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984132 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928697 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00421 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3D21 has 0 variants, skipping\n",
      "[INFO]\tGene PALB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00350 has 0 variants, skipping\n",
      "[INFO]\tGene ORAI2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1A has 0 variants, skipping\n",
      "[INFO]\tGene NSUN6 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4C has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3A has 0 variants, skipping\n",
      "[INFO]\tGene SDK1 has 0 variants, skipping\n",
      "[INFO]\tGene OASL has 0 variants, skipping\n",
      "[INFO]\tGene GSPT1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM4 has 0 variants, skipping\n",
      "[INFO]\tGene CCT2 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN5A has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE3 has 0 variants, skipping\n",
      "[INFO]\tGene HARS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-2HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC01072 has 0 variants, skipping\n",
      "[INFO]\tGene DNPEP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4499 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6F has 0 variants, skipping\n",
      "[INFO]\tGene OR10V1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AK2 has 0 variants, skipping\n",
      "[INFO]\tGene SREBF1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM76A has 0 variants, skipping\n",
      "[INFO]\tGene OXSM has 0 variants, skipping\n",
      "[INFO]\tGene FGFR1OP2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCR has 0 variants, skipping\n",
      "[INFO]\tGene ACOT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC22 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB16 has 0 variants, skipping\n",
      "[INFO]\tGene ASPA has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-3 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE4 has 0 variants, skipping\n",
      "[INFO]\tGene MELK has 0 variants, skipping\n",
      "[INFO]\tGene STPG3 has 0 variants, skipping\n",
      "[INFO]\tGene BANF2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM38B has 0 variants, skipping\n",
      "[INFO]\tGene C19orf73 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNG has 0 variants, skipping\n",
      "[INFO]\tGene C22orf15 has 0 variants, skipping\n",
      "[INFO]\tGene SFTA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR656 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00367 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01046 has 0 variants, skipping\n",
      "[INFO]\tGene MIPEPP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00539 has 0 variants, skipping\n",
      "[INFO]\tGene GRK6P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00424 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00621 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00327 has 0 variants, skipping\n",
      "[INFO]\tGene UGCG has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM223 has 0 variants, skipping\n",
      "[INFO]\tGene MND1 has 0 variants, skipping\n",
      "[INFO]\tGene SRP68 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A19P has 0 variants, skipping\n",
      "[INFO]\tGene THTPA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF576 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL15 has 0 variants, skipping\n",
      "[INFO]\tGene CRELD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2276 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00566 has 0 variants, skipping\n",
      "[INFO]\tGene MCMDC2 has 0 variants, skipping\n",
      "[INFO]\tGene NAPRT has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC6 has 0 variants, skipping\n",
      "[INFO]\tGene MMD2 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01088 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00463 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01053 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF8 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN3 has 0 variants, skipping\n",
      "[INFO]\tGene STK32C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00415 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY2 has 0 variants, skipping\n",
      "[INFO]\tGene RXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene USP12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP12-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00412 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD102 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA27 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZWINT has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT2 has 0 variants, skipping\n",
      "[INFO]\tGene PLUT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00543 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8 has 0 variants, skipping\n",
      "[INFO]\tGene TBX20 has 0 variants, skipping\n",
      "[INFO]\tGene OLR1 has 0 variants, skipping\n",
      "[INFO]\tGene AMN has 0 variants, skipping\n",
      "[INFO]\tGene PAN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGR has 0 variants, skipping\n",
      "[INFO]\tGene RPS24 has 0 variants, skipping\n",
      "[INFO]\tGene RPS23 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1B has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D2 has 0 variants, skipping\n",
      "[INFO]\tGene URB2 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00297 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00572 has 0 variants, skipping\n",
      "[INFO]\tGene MUCL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00365 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00384 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00426 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01058 has 0 variants, skipping\n",
      "[INFO]\tGene USB1 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP300 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRU has 0 variants, skipping\n",
      "[INFO]\tGene LINC00398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00545 has 0 variants, skipping\n",
      "[INFO]\tGene TEX26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3GAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5A has 0 variants, skipping\n",
      "[INFO]\tGene EEF1DP3 has 0 variants, skipping\n",
      "[INFO]\tGene FRY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP2 has 0 variants, skipping\n",
      "[INFO]\tGene NKG7 has 0 variants, skipping\n",
      "[INFO]\tGene KRT33A has 0 variants, skipping\n",
      "[INFO]\tGene TCEANC has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG3 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10P1 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L2-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXM1 has 0 variants, skipping\n",
      "[INFO]\tGene STARD13-AS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02344 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02343 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00457 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF780A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00445 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A8 has 0 variants, skipping\n",
      "[INFO]\tGene CHRND has 0 variants, skipping\n",
      "[INFO]\tGene FAM166C has 0 variants, skipping\n",
      "[INFO]\tGene C17orf100 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPB has 0 variants, skipping\n",
      "[INFO]\tGene SPART-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FHL3 has 0 variants, skipping\n",
      "[INFO]\tGene FGL1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM22 has 0 variants, skipping\n",
      "[INFO]\tGene AGGF1 has 0 variants, skipping\n",
      "[INFO]\tGene BSDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5 has 0 variants, skipping\n",
      "[INFO]\tGene USP47 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO34 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN12 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00547 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPPIN-WFDC6 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC5 has 0 variants, skipping\n",
      "[INFO]\tGene DCN has 0 variants, skipping\n",
      "[INFO]\tGene OS9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00571 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF18 has 0 variants, skipping\n",
      "[INFO]\tGene PERP has 0 variants, skipping\n",
      "[INFO]\tGene PRSS22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00437 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00366 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF15 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP1 has 0 variants, skipping\n",
      "[INFO]\tGene FREM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RUBCN has 0 variants, skipping\n",
      "[INFO]\tGene SBNO2 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene KNTC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4305 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00332 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00548 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00598 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR621 has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3168 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929140 has 0 variants, skipping\n",
      "[INFO]\tGene YME1L1 has 0 variants, skipping\n",
      "[INFO]\tGene P3H3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E37P has 0 variants, skipping\n",
      "[INFO]\tGene HEATR4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5006 has 0 variants, skipping\n",
      "[INFO]\tGene VWA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBASH3B has 0 variants, skipping\n",
      "[INFO]\tGene PF4 has 0 variants, skipping\n",
      "[INFO]\tGene CTTNBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC3 has 0 variants, skipping\n",
      "[INFO]\tGene USH1G has 0 variants, skipping\n",
      "[INFO]\tGene LINC02341 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01050 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00428 has 0 variants, skipping\n",
      "[INFO]\tGene DGKA has 0 variants, skipping\n",
      "[INFO]\tGene PSMC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00400 has 0 variants, skipping\n",
      "[INFO]\tGene NUP58 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF8 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K17 has 0 variants, skipping\n",
      "[INFO]\tGene POTEF has 0 variants, skipping\n",
      "[INFO]\tGene GGT5 has 0 variants, skipping\n",
      "[INFO]\tGene HRCT1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB28 has 0 variants, skipping\n",
      "[INFO]\tGene ENOX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene URB1 has 0 variants, skipping\n",
      "[INFO]\tGene FCHSD2 has 0 variants, skipping\n",
      "[INFO]\tGene WNK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00390 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8079 has 0 variants, skipping\n",
      "[INFO]\tGene SERP2 has 0 variants, skipping\n",
      "[INFO]\tGene PTCRA has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00330 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929259 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD9 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCTA has 0 variants, skipping\n",
      "[INFO]\tGene ABCG4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA31B has 0 variants, skipping\n",
      "[INFO]\tGene CRB3 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN1 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3G has 0 variants, skipping\n",
      "[INFO]\tGene AK5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01055 has 0 variants, skipping\n",
      "[INFO]\tGene ERVMER34-1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST5 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG16 has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS19 has 0 variants, skipping\n",
      "[INFO]\tGene CPB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCEH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00563 has 0 variants, skipping\n",
      "[INFO]\tGene GPR149 has 0 variants, skipping\n",
      "[INFO]\tGene OLFM1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01198 has 0 variants, skipping\n",
      "[INFO]\tGene HTR2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00562 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED3 has 0 variants, skipping\n",
      "[INFO]\tGene ASB17 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A9 has 0 variants, skipping\n",
      "[INFO]\tGene MED4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP10 has 0 variants, skipping\n",
      "[INFO]\tGene BCL9L has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5E has 0 variants, skipping\n",
      "[INFO]\tGene CDH18 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00462 has 0 variants, skipping\n",
      "[INFO]\tGene SPO11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370203 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1B has 0 variants, skipping\n",
      "[INFO]\tGene USP17L3 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF7 has 0 variants, skipping\n",
      "[INFO]\tGene EBPL has 0 variants, skipping\n",
      "[INFO]\tGene RGS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE10P has 0 variants, skipping\n",
      "[INFO]\tGene GAGE7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507412 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00641 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723729 has 0 variants, skipping\n",
      "[INFO]\tGene BBOX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ST13P4 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD7 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1B2 has 0 variants, skipping\n",
      "[INFO]\tGene C13orf42 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645177 has 0 variants, skipping\n",
      "[INFO]\tGene DAP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5693 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC120 has 0 variants, skipping\n",
      "[INFO]\tGene GPANK1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4703 has 0 variants, skipping\n",
      "[INFO]\tGene ECT2L has 0 variants, skipping\n",
      "[INFO]\tGene PROSER1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL4 has 0 variants, skipping\n",
      "[INFO]\tGene AQP11 has 0 variants, skipping\n",
      "[INFO]\tGene MYPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929657 has 0 variants, skipping\n",
      "[INFO]\tGene PLAA has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A42 has 0 variants, skipping\n",
      "[INFO]\tGene RNPS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS31P5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02333 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND10 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P3 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR759 has 0 variants, skipping\n",
      "[INFO]\tGene NAAA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00558 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5007 has 0 variants, skipping\n",
      "[INFO]\tGene AK9 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A6A has 0 variants, skipping\n",
      "[INFO]\tGene NCAPG has 0 variants, skipping\n",
      "[INFO]\tGene KIF13B has 0 variants, skipping\n",
      "[INFO]\tGene EDDM3B has 0 variants, skipping\n",
      "[INFO]\tGene KCNK15 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02338 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00374 has 0 variants, skipping\n",
      "[INFO]\tGene MATN2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR27 has 0 variants, skipping\n",
      "[INFO]\tGene COL8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD2 has 0 variants, skipping\n",
      "[INFO]\tGene AGTRAP has 0 variants, skipping\n",
      "[INFO]\tGene LZTFL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF543 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf66 has 0 variants, skipping\n",
      "[INFO]\tGene KRBA2 has 0 variants, skipping\n",
      "[INFO]\tGene MTNR1B has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00434 has 0 variants, skipping\n",
      "[INFO]\tGene RGL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00378 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00358 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01075 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00459 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00448 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00376 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00395 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E156P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00355 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01052 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548X2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4704 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00364 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00550 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00383 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00348 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN17 has 0 variants, skipping\n",
      "[INFO]\tGene MON1B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0754 has 0 variants, skipping\n",
      "[INFO]\tGene MLXIP has 0 variants, skipping\n",
      "[INFO]\tGene ENPP4 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOD1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA6 has 0 variants, skipping\n",
      "[INFO]\tGene FNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDS5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00392 has 0 variants, skipping\n",
      "[INFO]\tGene SRPRB has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00381 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00347 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE11P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01078 has 0 variants, skipping\n",
      "[INFO]\tGene ERO1B has 0 variants, skipping\n",
      "[INFO]\tGene CIP2A has 0 variants, skipping\n",
      "[INFO]\tGene LMO7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF554 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF300 has 0 variants, skipping\n",
      "[INFO]\tGene CEP43 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2H has 0 variants, skipping\n",
      "[INFO]\tGene ANTXRL has 0 variants, skipping\n",
      "[INFO]\tGene PLCH2 has 0 variants, skipping\n",
      "[INFO]\tGene TENT4B has 0 variants, skipping\n",
      "[INFO]\tGene NQO1 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7DN-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00561 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3P11 has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTR has 0 variants, skipping\n",
      "[INFO]\tGene KNCN has 0 variants, skipping\n",
      "[INFO]\tGene SCEL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD4L has 0 variants, skipping\n",
      "[INFO]\tGene MIR3665 has 0 variants, skipping\n",
      "[INFO]\tGene ZIK1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01069 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC780529 has 0 variants, skipping\n",
      "[INFO]\tGene RNF20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00331 has 0 variants, skipping\n",
      "[INFO]\tGene KLK11 has 0 variants, skipping\n",
      "[INFO]\tGene NPM3 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA2B has 0 variants, skipping\n",
      "[INFO]\tGene RBM26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01038 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00382 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00564 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00333 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00430 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4500HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC00397 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00373 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00433 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00560 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01047 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00440 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01040 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00353 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00559 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01049 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00410 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00379 has 0 variants, skipping\n",
      "[INFO]\tGene RNF150 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC5-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AS has 0 variants, skipping\n",
      "[INFO]\tGene GPC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00363 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB10 has 0 variants, skipping\n",
      "[INFO]\tGene ECE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927248 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927284 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00557 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A8 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC14 has 0 variants, skipping\n",
      "[INFO]\tGene RPL31 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM6 has 0 variants, skipping\n",
      "[INFO]\tGene HPSE has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13G has 0 variants, skipping\n",
      "[INFO]\tGene CLDN10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC3-DT has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD3 has 0 variants, skipping\n",
      "[INFO]\tGene SPARC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00359 has 0 variants, skipping\n",
      "[INFO]\tGene CRMP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00456 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY3 has 0 variants, skipping\n",
      "[INFO]\tGene STARD9 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP31 has 0 variants, skipping\n",
      "[INFO]\tGene CASKIN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3170 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf201 has 0 variants, skipping\n",
      "[INFO]\tGene NUBPL has 0 variants, skipping\n",
      "[INFO]\tGene STK24-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF6 has 0 variants, skipping\n",
      "[INFO]\tGene GDF7 has 0 variants, skipping\n",
      "[INFO]\tGene ATF4 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB4 has 0 variants, skipping\n",
      "[INFO]\tGene MAT2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF787 has 0 variants, skipping\n",
      "[INFO]\tGene HRH2 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC9 has 0 variants, skipping\n",
      "[INFO]\tGene SC5D has 0 variants, skipping\n",
      "[INFO]\tGene ZNF671 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GLB2 has 0 variants, skipping\n",
      "[INFO]\tGene CEP70 has 0 variants, skipping\n",
      "[INFO]\tGene TEX29 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5G has 0 variants, skipping\n",
      "[INFO]\tGene RHEBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-6 has 0 variants, skipping\n",
      "[INFO]\tGene UCMA has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1C has 0 variants, skipping\n",
      "[INFO]\tGene DARS1 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49C has 0 variants, skipping\n",
      "[INFO]\tGene C4B has 0 variants, skipping\n",
      "[INFO]\tGene PDCD4 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9-DT has 0 variants, skipping\n",
      "[INFO]\tGene UBAC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FKSG29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01232 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00449 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01039 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927437 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00554 has 0 variants, skipping\n",
      "[INFO]\tGene ATG12 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR150 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13D has 0 variants, skipping\n",
      "[INFO]\tGene ANO6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF780B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF835 has 0 variants, skipping\n",
      "[INFO]\tGene STUM has 0 variants, skipping\n",
      "[INFO]\tGene SMAD2 has 0 variants, skipping\n",
      "[INFO]\tGene RAE1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC25 has 0 variants, skipping\n",
      "[INFO]\tGene AURKC has 0 variants, skipping\n",
      "[INFO]\tGene PAPSS2 has 0 variants, skipping\n",
      "[INFO]\tGene PURG has 0 variants, skipping\n",
      "[INFO]\tGene PCCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF629 has 0 variants, skipping\n",
      "[INFO]\tGene NVL has 0 variants, skipping\n",
      "[INFO]\tGene LOC642414 has 0 variants, skipping\n",
      "[INFO]\tGene XKR7 has 0 variants, skipping\n",
      "[INFO]\tGene NMS has 0 variants, skipping\n",
      "[INFO]\tGene IQCE has 0 variants, skipping\n",
      "[INFO]\tGene PAX4 has 0 variants, skipping\n",
      "[INFO]\tGene NALCN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00411 has 0 variants, skipping\n",
      "[INFO]\tGene CHURC1-FNTB has 0 variants, skipping\n",
      "[INFO]\tGene BUD23 has 0 variants, skipping\n",
      "[INFO]\tGene PIGV has 0 variants, skipping\n",
      "[INFO]\tGene GRXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM3 has 0 variants, skipping\n",
      "[INFO]\tGene MMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K11 has 0 variants, skipping\n",
      "[INFO]\tGene CTR9 has 0 variants, skipping\n",
      "[INFO]\tGene RNF182 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6M1 has 0 variants, skipping\n",
      "[INFO]\tGene LMTK3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2681 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4705 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00958 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01116 has 0 variants, skipping\n",
      "[INFO]\tGene MROCKI has 0 variants, skipping\n",
      "[INFO]\tGene LINC01133 has 0 variants, skipping\n",
      "[INFO]\tGene MNX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00974 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1G-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00637 has 0 variants, skipping\n",
      "[INFO]\tGene DNM3OS has 0 variants, skipping\n",
      "[INFO]\tGene WTAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene HOTAIRM1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM147 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM20 has 0 variants, skipping\n",
      "[INFO]\tGene ZAR1L has 0 variants, skipping\n",
      "[INFO]\tGene METTL21EP has 0 variants, skipping\n",
      "[INFO]\tGene DAOA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00343 has 0 variants, skipping\n",
      "[INFO]\tGene SRGN has 0 variants, skipping\n",
      "[INFO]\tGene PREP has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1549L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00551 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00443 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD31B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1267 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene ETV4 has 0 variants, skipping\n",
      "[INFO]\tGene MYO16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00399 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00676 has 0 variants, skipping\n",
      "[INFO]\tGene TAF6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00396 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370361 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF549 has 0 variants, skipping\n",
      "[INFO]\tGene CNEP1R1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC57 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN2 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL6 has 0 variants, skipping\n",
      "[INFO]\tGene PGPEP1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR8073 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS15 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L3 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf71 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF316 has 0 variants, skipping\n",
      "[INFO]\tGene WHRN has 0 variants, skipping\n",
      "[INFO]\tGene LINC00567 has 0 variants, skipping\n",
      "[INFO]\tGene KIF15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00431 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00368 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP7 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF10 has 0 variants, skipping\n",
      "[INFO]\tGene SAXO2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC4 has 0 variants, skipping\n",
      "[INFO]\tGene F8 has 0 variants, skipping\n",
      "[INFO]\tGene F7 has 0 variants, skipping\n",
      "[INFO]\tGene F13A1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101060553 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11A has 0 variants, skipping\n",
      "[INFO]\tGene UPK3BL1 has 0 variants, skipping\n",
      "[INFO]\tGene KREMEN2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX52 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC29 has 0 variants, skipping\n",
      "[INFO]\tGene RFLNA has 0 variants, skipping\n",
      "[INFO]\tGene SMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A2 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1B has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP58 has 0 variants, skipping\n",
      "[INFO]\tGene NANOGP8 has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL1 has 0 variants, skipping\n",
      "[INFO]\tGene FBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene H3Y1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD3OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02337 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00354 has 0 variants, skipping\n",
      "[INFO]\tGene SOX1-OT has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506016 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01070 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928730 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01043 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01044 has 0 variants, skipping\n",
      "[INFO]\tGene GLMN has 0 variants, skipping\n",
      "[INFO]\tGene BTD has 0 variants, skipping\n",
      "[INFO]\tGene KIF5C has 0 variants, skipping\n",
      "[INFO]\tGene KIF5B has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ3 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL23 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53BP2 has 0 variants, skipping\n",
      "[INFO]\tGene TNP2 has 0 variants, skipping\n",
      "[INFO]\tGene TIE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFF2 has 0 variants, skipping\n",
      "[INFO]\tGene TECTA has 0 variants, skipping\n",
      "[INFO]\tGene TAL2 has 0 variants, skipping\n",
      "[INFO]\tGene STAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1E1 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB3 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT2A has 0 variants, skipping\n",
      "[INFO]\tGene GPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELF2 has 0 variants, skipping\n",
      "[INFO]\tGene CFI has 0 variants, skipping\n",
      "[INFO]\tGene TMCC2 has 0 variants, skipping\n",
      "[INFO]\tGene F10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD17 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R16B has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1C has 0 variants, skipping\n",
      "[INFO]\tGene FAM120A has 0 variants, skipping\n",
      "[INFO]\tGene RMDN1 has 0 variants, skipping\n",
      "[INFO]\tGene AGTPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA1 has 0 variants, skipping\n",
      "[INFO]\tGene SHF has 0 variants, skipping\n",
      "[INFO]\tGene EME2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288966 has 0 variants, skipping\n",
      "[INFO]\tGene TPO has 0 variants, skipping\n",
      "[INFO]\tGene AMZ2 has 0 variants, skipping\n",
      "[INFO]\tGene GRTP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT5 has 0 variants, skipping\n",
      "[INFO]\tGene KRT2 has 0 variants, skipping\n",
      "[INFO]\tGene TPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC4 has 0 variants, skipping\n",
      "[INFO]\tGene THAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF8 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00552 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00454 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00565 has 0 variants, skipping\n",
      "[INFO]\tGene OSR2 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA2 has 0 variants, skipping\n",
      "[INFO]\tGene NPFF has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L1 has 0 variants, skipping\n",
      "[INFO]\tGene SALL1 has 0 variants, skipping\n",
      "[INFO]\tGene TFIP11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AR has 0 variants, skipping\n",
      "[INFO]\tGene MIR4502 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS3 has 0 variants, skipping\n",
      "[INFO]\tGene GCC1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRA has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P11 has 0 variants, skipping\n",
      "[INFO]\tGene POT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG1 has 0 variants, skipping\n",
      "[INFO]\tGene NBR2 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG9 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01054 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A9 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC3 has 0 variants, skipping\n",
      "[INFO]\tGene FADD has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100508046 has 0 variants, skipping\n",
      "[INFO]\tGene ACIN1 has 0 variants, skipping\n",
      "[INFO]\tGene XYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P18 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P17 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P22 has 0 variants, skipping\n",
      "[INFO]\tGene POTEG has 0 variants, skipping\n",
      "[INFO]\tGene POTEH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929572 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1B has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA79B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD126 has 0 variants, skipping\n",
      "[INFO]\tGene RPPH1 has 0 variants, skipping\n",
      "[INFO]\tGene PGC has 0 variants, skipping\n",
      "[INFO]\tGene LOC254028 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3B has 0 variants, skipping\n",
      "[INFO]\tGene ECRP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929718 has 0 variants, skipping\n",
      "[INFO]\tGene FAIM2 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A5 has 0 variants, skipping\n",
      "[INFO]\tGene TKTL1 has 0 variants, skipping\n",
      "[INFO]\tGene TFEC has 0 variants, skipping\n",
      "[INFO]\tGene RHOQ has 0 variants, skipping\n",
      "[INFO]\tGene SEPHS2 has 0 variants, skipping\n",
      "[INFO]\tGene SNW1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A3 has 0 variants, skipping\n",
      "[INFO]\tGene DKK1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT3 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT2 has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6717 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD9 has 0 variants, skipping\n",
      "[INFO]\tGene YEATS4 has 0 variants, skipping\n",
      "[INFO]\tGene PEX6 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA4 has 0 variants, skipping\n",
      "[INFO]\tGene RPL35A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD9 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAB has 0 variants, skipping\n",
      "[INFO]\tGene ATRN has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B2 has 0 variants, skipping\n",
      "[INFO]\tGene INS has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370401 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02332 has 0 variants, skipping\n",
      "[INFO]\tGene GPR161 has 0 variants, skipping\n",
      "[INFO]\tGene FAM114A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR107 has 0 variants, skipping\n",
      "[INFO]\tGene ORC2 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf39 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G7 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H5 has 0 variants, skipping\n",
      "[INFO]\tGene TERT has 0 variants, skipping\n",
      "[INFO]\tGene DBF4 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RE has 0 variants, skipping\n",
      "[INFO]\tGene PRMT5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL38 has 0 variants, skipping\n",
      "[INFO]\tGene TSPOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACACB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926933 has 0 variants, skipping\n",
      "[INFO]\tGene THADA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4707 has 0 variants, skipping\n",
      "[INFO]\tGene LRRN1 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPA1 has 0 variants, skipping\n",
      "[INFO]\tGene IL13 has 0 variants, skipping\n",
      "[INFO]\tGene PLCH1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A17 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2B has 0 variants, skipping\n",
      "[INFO]\tGene RTBDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF610 has 0 variants, skipping\n",
      "[INFO]\tGene SPP1 has 0 variants, skipping\n",
      "[INFO]\tGene IAH1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF19B has 0 variants, skipping\n",
      "[INFO]\tGene GAR1 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1B has 0 variants, skipping\n",
      "[INFO]\tGene C8orf37 has 0 variants, skipping\n",
      "[INFO]\tGene SYP has 0 variants, skipping\n",
      "[INFO]\tGene PEF1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf35 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724814 has 0 variants, skipping\n",
      "[INFO]\tGene CST9 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FETUB has 0 variants, skipping\n",
      "[INFO]\tGene GPR160 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3G has 0 variants, skipping\n",
      "[INFO]\tGene ACBD4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC3A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf53 has 0 variants, skipping\n",
      "[INFO]\tGene CENPA has 0 variants, skipping\n",
      "[INFO]\tGene ITGB2 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP10 has 0 variants, skipping\n",
      "[INFO]\tGene CASP9 has 0 variants, skipping\n",
      "[INFO]\tGene GALP has 0 variants, skipping\n",
      "[INFO]\tGene MIR7703 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A4 has 0 variants, skipping\n",
      "[INFO]\tGene POR has 0 variants, skipping\n",
      "[INFO]\tGene IPO4 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf89 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD8-MDP1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F2 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK3 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D5 has 0 variants, skipping\n",
      "[INFO]\tGene RPA3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A1 has 0 variants, skipping\n",
      "[INFO]\tGene MED30 has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2B has 0 variants, skipping\n",
      "[INFO]\tGene INTS6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927045 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF408 has 0 variants, skipping\n",
      "[INFO]\tGene KAZN has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724219 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN3 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02588 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02294 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02293 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4307HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4307 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728755 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00645 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3171 has 0 variants, skipping\n",
      "[INFO]\tGene FOXG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INHBE has 0 variants, skipping\n",
      "[INFO]\tGene LINC02282 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3B has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3C has 0 variants, skipping\n",
      "[INFO]\tGene BUB1B-PAK6 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4O has 0 variants, skipping\n",
      "[INFO]\tGene G2E3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NTSR1 has 0 variants, skipping\n",
      "[INFO]\tGene NTRK1 has 0 variants, skipping\n",
      "[INFO]\tGene NTF3 has 0 variants, skipping\n",
      "[INFO]\tGene NT5E has 0 variants, skipping\n",
      "[INFO]\tGene IFT140 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM4A has 0 variants, skipping\n",
      "[INFO]\tGene SETD1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506071 has 0 variants, skipping\n",
      "[INFO]\tGene PDE3B has 0 variants, skipping\n",
      "[INFO]\tGene NCOA5 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND15 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPG has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927124 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02313 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GSDMD has 0 variants, skipping\n",
      "[INFO]\tGene SNORA89 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP5 has 0 variants, skipping\n",
      "[INFO]\tGene MIGA1 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF283 has 0 variants, skipping\n",
      "[INFO]\tGene BCL6B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB12 has 0 variants, skipping\n",
      "[INFO]\tGene SRP54-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGBP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927178 has 0 variants, skipping\n",
      "[INFO]\tGene RGP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP4 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP1 has 0 variants, skipping\n",
      "[INFO]\tGene TIAL1 has 0 variants, skipping\n",
      "[INFO]\tGene THRSP has 0 variants, skipping\n",
      "[INFO]\tGene THOP1 has 0 variants, skipping\n",
      "[INFO]\tGene THRA has 0 variants, skipping\n",
      "[INFO]\tGene THBS4 has 0 variants, skipping\n",
      "[INFO]\tGene THBS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D2 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB5 has 0 variants, skipping\n",
      "[INFO]\tGene SDK2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP3 has 0 variants, skipping\n",
      "[INFO]\tGene FAH has 0 variants, skipping\n",
      "[INFO]\tGene FANCA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA101B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00609 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1287 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL25 has 0 variants, skipping\n",
      "[INFO]\tGene BEX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF426 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4503 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AG1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00517 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF597 has 0 variants, skipping\n",
      "[INFO]\tGene BEST3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370457 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4G has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF18 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN21 has 0 variants, skipping\n",
      "[INFO]\tGene CDC45 has 0 variants, skipping\n",
      "[INFO]\tGene PLAU has 0 variants, skipping\n",
      "[INFO]\tGene METTL18 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC6 has 0 variants, skipping\n",
      "[INFO]\tGene PLAT has 0 variants, skipping\n",
      "[INFO]\tGene SCIMP has 0 variants, skipping\n",
      "[INFO]\tGene HDGF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF326 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288846 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM24 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1200 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1238 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1236 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1178 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2113 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf40 has 0 variants, skipping\n",
      "[INFO]\tGene NOS3 has 0 variants, skipping\n",
      "[INFO]\tGene SGO2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02315 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1F has 0 variants, skipping\n",
      "[INFO]\tGene CENPW has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02302 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927418 has 0 variants, skipping\n",
      "[INFO]\tGene IL34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD127 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2A has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE8 has 0 variants, skipping\n",
      "[INFO]\tGene COL21A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00871 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Y has 0 variants, skipping\n",
      "[INFO]\tGene LINC00648 has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL2 has 0 variants, skipping\n",
      "[INFO]\tGene LRR1 has 0 variants, skipping\n",
      "[INFO]\tGene CABLES1 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT5B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB32 has 0 variants, skipping\n",
      "[INFO]\tGene RBAK has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6076 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01588 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01599 has 0 variants, skipping\n",
      "[INFO]\tGene RBM24 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4504 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD1 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1L has 0 variants, skipping\n",
      "[INFO]\tGene TOMM40L has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A1 has 0 variants, skipping\n",
      "[INFO]\tGene ALX3 has 0 variants, skipping\n",
      "[INFO]\tGene NOVA1 has 0 variants, skipping\n",
      "[INFO]\tGene MCAM has 0 variants, skipping\n",
      "[INFO]\tGene KHK has 0 variants, skipping\n",
      "[INFO]\tGene HTR6 has 0 variants, skipping\n",
      "[INFO]\tGene BRD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370489 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6P1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS53 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00519 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00640 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02310 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD6 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ODF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723604 has 0 variants, skipping\n",
      "[INFO]\tGene CD2 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927620 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5580 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D4 has 0 variants, skipping\n",
      "[INFO]\tGene PPARGC1A has 0 variants, skipping\n",
      "[INFO]\tGene ELP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4308 has 0 variants, skipping\n",
      "[INFO]\tGene CHTF8 has 0 variants, skipping\n",
      "[INFO]\tGene TOX2 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN5 has 0 variants, skipping\n",
      "[INFO]\tGene GPS2 has 0 variants, skipping\n",
      "[INFO]\tGene GCSAML has 0 variants, skipping\n",
      "[INFO]\tGene DSG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRRAP has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12A has 0 variants, skipping\n",
      "[INFO]\tGene FAS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ID1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP3 has 0 variants, skipping\n",
      "[INFO]\tGene LIMA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927690 has 0 variants, skipping\n",
      "[INFO]\tGene SIX3 has 0 variants, skipping\n",
      "[INFO]\tGene OTX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCP1B has 0 variants, skipping\n",
      "[INFO]\tGene ASTE1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8E has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H18 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN4 has 0 variants, skipping\n",
      "[INFO]\tGene NOL4L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00216 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4 has 0 variants, skipping\n",
      "[INFO]\tGene USP14 has 0 variants, skipping\n",
      "[INFO]\tGene PRR11 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106A has 0 variants, skipping\n",
      "[INFO]\tGene SIKE1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF703 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP3 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6D has 0 variants, skipping\n",
      "[INFO]\tGene ATRX has 0 variants, skipping\n",
      "[INFO]\tGene PCSK4 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK2 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene CPED1 has 0 variants, skipping\n",
      "[INFO]\tGene NREP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF304 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF571 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01500 has 0 variants, skipping\n",
      "[INFO]\tGene SS18 has 0 variants, skipping\n",
      "[INFO]\tGene NRXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF649 has 0 variants, skipping\n",
      "[INFO]\tGene CPLANE1 has 0 variants, skipping\n",
      "[INFO]\tGene RABGGTA has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5586 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC9 has 0 variants, skipping\n",
      "[INFO]\tGene CDK7 has 0 variants, skipping\n",
      "[INFO]\tGene MYO3A has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02322 has 0 variants, skipping\n",
      "[INFO]\tGene LDHAL6A has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM272 has 0 variants, skipping\n",
      "[INFO]\tGene DDX56 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01303 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ22447 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150A has 0 variants, skipping\n",
      "[INFO]\tGene OPTN has 0 variants, skipping\n",
      "[INFO]\tGene SMC1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00643 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00644 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A7 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A8 has 0 variants, skipping\n",
      "[INFO]\tGene HEXIM1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM1 has 0 variants, skipping\n",
      "[INFO]\tGene CSDC2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF30 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR2 has 0 variants, skipping\n",
      "[INFO]\tGene ERFL has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H1 has 0 variants, skipping\n",
      "[INFO]\tGene CYBC1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP8 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSP has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOB has 0 variants, skipping\n",
      "[INFO]\tGene ABCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX21P has 0 variants, skipping\n",
      "[INFO]\tGene SLCO5A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723809 has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A2 has 0 variants, skipping\n",
      "[INFO]\tGene UBL3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10A has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7855 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4706 has 0 variants, skipping\n",
      "[INFO]\tGene PRRX2 has 0 variants, skipping\n",
      "[INFO]\tGene PCYOX1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506321 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4708 has 0 variants, skipping\n",
      "[INFO]\tGene HMGXB4 has 0 variants, skipping\n",
      "[INFO]\tGene FUT8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GINS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNN2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPF has 0 variants, skipping\n",
      "[INFO]\tGene HMGN1 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L has 0 variants, skipping\n",
      "[INFO]\tGene HDAC1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL23 has 0 variants, skipping\n",
      "[INFO]\tGene SCD has 0 variants, skipping\n",
      "[INFO]\tGene MIR5694 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4B has 0 variants, skipping\n",
      "[INFO]\tGene LRRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX19 has 0 variants, skipping\n",
      "[INFO]\tGene CRYAA has 0 variants, skipping\n",
      "[INFO]\tGene COL5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SCAPER has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996664 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE19 has 0 variants, skipping\n",
      "[INFO]\tGene ERI2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB3 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R3 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289511 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506358 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD169 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646548 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM21P1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR15 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM20P1 has 0 variants, skipping\n",
      "[INFO]\tGene CSKMT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC158 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928075 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01269 has 0 variants, skipping\n",
      "[INFO]\tGene SMG5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD56B has 0 variants, skipping\n",
      "[INFO]\tGene DDAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145474 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4D has 0 variants, skipping\n",
      "[INFO]\tGene KLHL41 has 0 variants, skipping\n",
      "[INFO]\tGene MYH1 has 0 variants, skipping\n",
      "[INFO]\tGene OLFM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB42 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7843 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF627 has 0 variants, skipping\n",
      "[INFO]\tGene SDF2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN3A has 0 variants, skipping\n",
      "[INFO]\tGene ENPEP has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5A has 0 variants, skipping\n",
      "[INFO]\tGene CSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM245 has 0 variants, skipping\n",
      "[INFO]\tGene TEX48 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928123 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF507 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS6 has 0 variants, skipping\n",
      "[INFO]\tGene GNPDA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928143 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB16 has 0 variants, skipping\n",
      "[INFO]\tGene PRX has 0 variants, skipping\n",
      "[INFO]\tGene HSPA9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02274 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC1 has 0 variants, skipping\n",
      "[INFO]\tGene RAVER2 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAXD has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC11 has 0 variants, skipping\n",
      "[INFO]\tGene ETNK2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM62 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP64 has 0 variants, skipping\n",
      "[INFO]\tGene VPS35 has 0 variants, skipping\n",
      "[INFO]\tGene BEND4 has 0 variants, skipping\n",
      "[INFO]\tGene SORL1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RL2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL5 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL6 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPC has 0 variants, skipping\n",
      "[INFO]\tGene SNRPB2 has 0 variants, skipping\n",
      "[INFO]\tGene SOD3 has 0 variants, skipping\n",
      "[INFO]\tGene SOAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP25 has 0 variants, skipping\n",
      "[INFO]\tGene CCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCE1 has 0 variants, skipping\n",
      "[INFO]\tGene SCTR has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD3 has 0 variants, skipping\n",
      "[INFO]\tGene OGDHL has 0 variants, skipping\n",
      "[INFO]\tGene MAPK12 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX2 has 0 variants, skipping\n",
      "[INFO]\tGene CST4 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM9 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN10 has 0 variants, skipping\n",
      "[INFO]\tGene ESRP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01220 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724153 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1 has 0 variants, skipping\n",
      "[INFO]\tGene DLG5 has 0 variants, skipping\n",
      "[INFO]\tGene AP2M1 has 0 variants, skipping\n",
      "[INFO]\tGene VASH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RYR3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01629 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF541 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02288 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02289 has 0 variants, skipping\n",
      "[INFO]\tGene WSCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ATMIN has 0 variants, skipping\n",
      "[INFO]\tGene BICD2 has 0 variants, skipping\n",
      "[INFO]\tGene PES1 has 0 variants, skipping\n",
      "[INFO]\tGene COPG2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA1 has 0 variants, skipping\n",
      "[INFO]\tGene MAML1 has 0 variants, skipping\n",
      "[INFO]\tGene KLC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02527 has 0 variants, skipping\n",
      "[INFO]\tGene GLA has 0 variants, skipping\n",
      "[INFO]\tGene TTC6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC198 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX4 has 0 variants, skipping\n",
      "[INFO]\tGene MEG3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370586 has 0 variants, skipping\n",
      "[INFO]\tGene WDR4 has 0 variants, skipping\n",
      "[INFO]\tGene PTEN has 0 variants, skipping\n",
      "[INFO]\tGene BROX has 0 variants, skipping\n",
      "[INFO]\tGene GARS1-DT has 0 variants, skipping\n",
      "[INFO]\tGene DIO2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO22 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01061 has 0 variants, skipping\n",
      "[INFO]\tGene CHASERR has 0 variants, skipping\n",
      "[INFO]\tGene SNORA79 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928504 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02308 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01467 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02311 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02301 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02329 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00911 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02328 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01148 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02330 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01147 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01146 has 0 variants, skipping\n",
      "[INFO]\tGene PDHA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6B has 0 variants, skipping\n",
      "[INFO]\tGene PCBD1 has 0 variants, skipping\n",
      "[INFO]\tGene OCRL has 0 variants, skipping\n",
      "[INFO]\tGene CLVS2 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT8 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAALAD2 has 0 variants, skipping\n",
      "[INFO]\tGene MED6 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MAFB has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1C has 0 variants, skipping\n",
      "[INFO]\tGene FAM221B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN23 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71F2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-7 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP157 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF2 has 0 variants, skipping\n",
      "[INFO]\tGene DDO has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK1 has 0 variants, skipping\n",
      "[INFO]\tGene WIF1 has 0 variants, skipping\n",
      "[INFO]\tGene UTP11 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKNMT has 0 variants, skipping\n",
      "[INFO]\tGene COQ4 has 0 variants, skipping\n",
      "[INFO]\tGene NOP58 has 0 variants, skipping\n",
      "[INFO]\tGene TMA7 has 0 variants, skipping\n",
      "[INFO]\tGene POMP has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC9 has 0 variants, skipping\n",
      "[INFO]\tGene SSUH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC7 has 0 variants, skipping\n",
      "[INFO]\tGene VEZF1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370619 has 0 variants, skipping\n",
      "[INFO]\tGene RSKR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928909 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370622 has 0 variants, skipping\n",
      "[INFO]\tGene TPRN has 0 variants, skipping\n",
      "[INFO]\tGene GNE has 0 variants, skipping\n",
      "[INFO]\tGene PMVK has 0 variants, skipping\n",
      "[INFO]\tGene YKT6 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID3A has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D1 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D2 has 0 variants, skipping\n",
      "[INFO]\tGene AIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene IL12RB1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP12 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAZ has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11B has 0 variants, skipping\n",
      "[INFO]\tGene DEFB104B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF181 has 0 variants, skipping\n",
      "[INFO]\tGene RFX8 has 0 variants, skipping\n",
      "[INFO]\tGene GATA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A1 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT2 has 0 variants, skipping\n",
      "[INFO]\tGene NRG3 has 0 variants, skipping\n",
      "[INFO]\tGene UPF3A has 0 variants, skipping\n",
      "[INFO]\tGene OR8S1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP6 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf115 has 0 variants, skipping\n",
      "[INFO]\tGene SETD7 has 0 variants, skipping\n",
      "[INFO]\tGene KDM7A has 0 variants, skipping\n",
      "[INFO]\tGene APOL3 has 0 variants, skipping\n",
      "[INFO]\tGene APOL4 has 0 variants, skipping\n",
      "[INFO]\tGene APOL5 has 0 variants, skipping\n",
      "[INFO]\tGene APOL6 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP16 has 0 variants, skipping\n",
      "[INFO]\tGene EEPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ASXL3 has 0 variants, skipping\n",
      "[INFO]\tGene HID1 has 0 variants, skipping\n",
      "[INFO]\tGene CMIP has 0 variants, skipping\n",
      "[INFO]\tGene KRT73 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02287 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD7 has 0 variants, skipping\n",
      "[INFO]\tGene ITPK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC7 has 0 variants, skipping\n",
      "[INFO]\tGene PCNX3 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY4 has 0 variants, skipping\n",
      "[INFO]\tGene UNC79 has 0 variants, skipping\n",
      "[INFO]\tGene AP5M1 has 0 variants, skipping\n",
      "[INFO]\tGene VRTN has 0 variants, skipping\n",
      "[INFO]\tGene UBA6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM181A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RD3L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4506 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC1L has 0 variants, skipping\n",
      "[INFO]\tGene NFYB has 0 variants, skipping\n",
      "[INFO]\tGene CNOT7 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD4 has 0 variants, skipping\n",
      "[INFO]\tGene TBX21 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ14 has 0 variants, skipping\n",
      "[INFO]\tGene RPA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNX12 has 0 variants, skipping\n",
      "[INFO]\tGene GPR132 has 0 variants, skipping\n",
      "[INFO]\tGene STRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG2 has 0 variants, skipping\n",
      "[INFO]\tGene FGD3 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA2 has 0 variants, skipping\n",
      "[INFO]\tGene BRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02767 has 0 variants, skipping\n",
      "[INFO]\tGene PLD5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC112 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA13P has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2B has 0 variants, skipping\n",
      "[INFO]\tGene GSC-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A52 has 0 variants, skipping\n",
      "[INFO]\tGene H4C15 has 0 variants, skipping\n",
      "[INFO]\tGene CDK3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3173 has 0 variants, skipping\n",
      "[INFO]\tGene PTPMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX4 has 0 variants, skipping\n",
      "[INFO]\tGene MED11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02292 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN4 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB131B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02318 has 0 variants, skipping\n",
      "[INFO]\tGene POLD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984703 has 0 variants, skipping\n",
      "[INFO]\tGene DERA has 0 variants, skipping\n",
      "[INFO]\tGene TEX10 has 0 variants, skipping\n",
      "[INFO]\tGene H19 has 0 variants, skipping\n",
      "[INFO]\tGene XACT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730202 has 0 variants, skipping\n",
      "[INFO]\tGene TDGF1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene LAG3 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02299 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00618 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02304 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02325 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02291 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02312 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01550 has 0 variants, skipping\n",
      "[INFO]\tGene BCL6 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6764 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A6 has 0 variants, skipping\n",
      "[INFO]\tGene KDM6A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM45 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2C has 0 variants, skipping\n",
      "[INFO]\tGene PTPRE has 0 variants, skipping\n",
      "[INFO]\tGene LINC00523 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA1 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL3 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD8 has 0 variants, skipping\n",
      "[INFO]\tGene LGR5 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH3 has 0 variants, skipping\n",
      "[INFO]\tGene IQCN has 0 variants, skipping\n",
      "[INFO]\tGene SRCIN1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02702 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02745 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD112 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-31 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1193 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376B has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1185-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1185-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR381HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF713 has 0 variants, skipping\n",
      "[INFO]\tGene CLECL1 has 0 variants, skipping\n",
      "[INFO]\tGene MEG9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02285 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00524 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02314 has 0 variants, skipping\n",
      "[INFO]\tGene DIO3OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00239 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK1 has 0 variants, skipping\n",
      "[INFO]\tGene DGKG has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A2 has 0 variants, skipping\n",
      "[INFO]\tGene CELF6 has 0 variants, skipping\n",
      "[INFO]\tGene ASMT has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3 has 0 variants, skipping\n",
      "[INFO]\tGene FRS2 has 0 variants, skipping\n",
      "[INFO]\tGene FRS3 has 0 variants, skipping\n",
      "[INFO]\tGene SNURF has 0 variants, skipping\n",
      "[INFO]\tGene NES has 0 variants, skipping\n",
      "[INFO]\tGene CAPN9 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXB1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP8 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO7 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO8 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO5 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD21 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO43 has 0 variants, skipping\n",
      "[INFO]\tGene ANKZF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIPEP has 0 variants, skipping\n",
      "[INFO]\tGene YDJC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4309 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02323 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00677 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00605 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378183 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA28 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N2 has 0 variants, skipping\n",
      "[INFO]\tGene NCK2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF28 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0753 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1FNB has 0 variants, skipping\n",
      "[INFO]\tGene ACCSL has 0 variants, skipping\n",
      "[INFO]\tGene OR2V2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CB has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CA has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2B has 0 variants, skipping\n",
      "[INFO]\tGene QKI has 0 variants, skipping\n",
      "[INFO]\tGene SKOR2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02691 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02280 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4710 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC3 has 0 variants, skipping\n",
      "[INFO]\tGene STK38L has 0 variants, skipping\n",
      "[INFO]\tGene NPNT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF26 has 0 variants, skipping\n",
      "[INFO]\tGene RAB34 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00638 has 0 variants, skipping\n",
      "[INFO]\tGene HGC6.3 has 0 variants, skipping\n",
      "[INFO]\tGene ARSB has 0 variants, skipping\n",
      "[INFO]\tGene LINC02298 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6765 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB2 has 0 variants, skipping\n",
      "[INFO]\tGene NDST2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23C-CDRT4 has 0 variants, skipping\n",
      "[INFO]\tGene CUZD1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP24-1 has 0 variants, skipping\n",
      "[INFO]\tGene GIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PGP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF732 has 0 variants, skipping\n",
      "[INFO]\tGene PARVG has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507437 has 0 variants, skipping\n",
      "[INFO]\tGene CD19 has 0 variants, skipping\n",
      "[INFO]\tGene PHETA1 has 0 variants, skipping\n",
      "[INFO]\tGene AJUBA has 0 variants, skipping\n",
      "[INFO]\tGene RAI14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370697 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8071-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8071-2 has 0 variants, skipping\n",
      "[INFO]\tGene ELK2AP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4539 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4507 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4538 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4537 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00226 has 0 variants, skipping\n",
      "[INFO]\tGene NEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene C8A has 0 variants, skipping\n",
      "[INFO]\tGene GSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene HBG1 has 0 variants, skipping\n",
      "[INFO]\tGene HBA1 has 0 variants, skipping\n",
      "[INFO]\tGene GDF5 has 0 variants, skipping\n",
      "[INFO]\tGene CRX has 0 variants, skipping\n",
      "[INFO]\tGene WRN has 0 variants, skipping\n",
      "[INFO]\tGene KCTD13 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF3 has 0 variants, skipping\n",
      "[INFO]\tGene FCER2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM170B has 0 variants, skipping\n",
      "[INFO]\tGene C5orf63 has 0 variants, skipping\n",
      "[INFO]\tGene CHD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF813 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F4 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN4 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf19 has 0 variants, skipping\n",
      "[INFO]\tGene HHATL has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D4 has 0 variants, skipping\n",
      "[INFO]\tGene SLFNL1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5B1 has 0 variants, skipping\n",
      "[INFO]\tGene NPPA has 0 variants, skipping\n",
      "[INFO]\tGene AKTIP has 0 variants, skipping\n",
      "[INFO]\tGene TUSC3 has 0 variants, skipping\n",
      "[INFO]\tGene CELF5 has 0 variants, skipping\n",
      "[INFO]\tGene CORO2B has 0 variants, skipping\n",
      "[INFO]\tGene LYZ has 0 variants, skipping\n",
      "[INFO]\tGene LPL has 0 variants, skipping\n",
      "[INFO]\tGene LIPC has 0 variants, skipping\n",
      "[INFO]\tGene LHCGR has 0 variants, skipping\n",
      "[INFO]\tGene SGCB has 0 variants, skipping\n",
      "[INFO]\tGene SGCG has 0 variants, skipping\n",
      "[INFO]\tGene LEP has 0 variants, skipping\n",
      "[INFO]\tGene LCAT has 0 variants, skipping\n",
      "[INFO]\tGene KRT9 has 0 variants, skipping\n",
      "[INFO]\tGene KRT18 has 0 variants, skipping\n",
      "[INFO]\tGene KRT12 has 0 variants, skipping\n",
      "[INFO]\tGene KIT has 0 variants, skipping\n",
      "[INFO]\tGene RBM18 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE8 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2013 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5B2 has 0 variants, skipping\n",
      "[INFO]\tGene KLC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRKD2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA4A has 0 variants, skipping\n",
      "[INFO]\tGene PDK1 has 0 variants, skipping\n",
      "[INFO]\tGene S100A13 has 0 variants, skipping\n",
      "[INFO]\tGene TANC1 has 0 variants, skipping\n",
      "[INFO]\tGene POLI has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene KPTN has 0 variants, skipping\n",
      "[INFO]\tGene CFC1 has 0 variants, skipping\n",
      "[INFO]\tGene DUS4L has 0 variants, skipping\n",
      "[INFO]\tGene CUX2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CNTFR has 0 variants, skipping\n",
      "[INFO]\tGene CNR2 has 0 variants, skipping\n",
      "[INFO]\tGene CCR3 has 0 variants, skipping\n",
      "[INFO]\tGene CHEK2P2 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P9 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3C has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8CP has 0 variants, skipping\n",
      "[INFO]\tGene NBEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF470 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01193 has 0 variants, skipping\n",
      "[INFO]\tGene FAM30C has 0 variants, skipping\n",
      "[INFO]\tGene CXADRP2 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02203 has 0 variants, skipping\n",
      "[INFO]\tGene IGHV1OR15-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724760 has 0 variants, skipping\n",
      "[INFO]\tGene IGHV1OR15-3 has 0 variants, skipping\n",
      "[INFO]\tGene PRANCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927079 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1268A has 0 variants, skipping\n",
      "[INFO]\tGene REREP3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8EP has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P7 has 0 variants, skipping\n",
      "[INFO]\tGene RPS21 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8IP has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283683 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM2 has 0 variants, skipping\n",
      "[INFO]\tGene GAPDH has 0 variants, skipping\n",
      "[INFO]\tGene GMCL2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S3B has 0 variants, skipping\n",
      "[INFO]\tGene ABCG8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM275 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH18A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCT4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC19A has 0 variants, skipping\n",
      "[INFO]\tGene ALPK1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP7 has 0 variants, skipping\n",
      "[INFO]\tGene MMP3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8DP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4508 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN4 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN2 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN3 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN1 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOVA2 has 0 variants, skipping\n",
      "[INFO]\tGene CCN3 has 0 variants, skipping\n",
      "[INFO]\tGene NME3 has 0 variants, skipping\n",
      "[INFO]\tGene NME2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A9P has 0 variants, skipping\n",
      "[INFO]\tGene NEXN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G10 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD107 has 0 variants, skipping\n",
      "[INFO]\tGene PWARSN has 0 variants, skipping\n",
      "[INFO]\tGene PWAR5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD64 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD108 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD109B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD109A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-30 has 0 variants, skipping\n",
      "[INFO]\tGene IPW has 0 variants, skipping\n",
      "[INFO]\tGene PWAR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-36 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-22 has 0 variants, skipping\n",
      "[INFO]\tGene PWAR4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-32 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-33 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-35 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-37 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-38 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-39 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-40 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-41 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-42 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-45 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-47 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-48 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64B has 0 variants, skipping\n",
      "[INFO]\tGene BHLHA9 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6C has 0 variants, skipping\n",
      "[INFO]\tGene MTF2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF6 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAM has 0 variants, skipping\n",
      "[INFO]\tGene MOB2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT7 has 0 variants, skipping\n",
      "[INFO]\tGene KLF10 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F1 has 0 variants, skipping\n",
      "[INFO]\tGene NUGGC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF98 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00929 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1M has 0 variants, skipping\n",
      "[INFO]\tGene GPR4 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK1 has 0 variants, skipping\n",
      "[INFO]\tGene ETFBKMT has 0 variants, skipping\n",
      "[INFO]\tGene PHPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF4B has 0 variants, skipping\n",
      "[INFO]\tGene F10 has 0 variants, skipping\n",
      "[INFO]\tGene KLRF2 has 0 variants, skipping\n",
      "[INFO]\tGene YARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928042 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289656 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6IPP2 has 0 variants, skipping\n",
      "[INFO]\tGene VAX1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53C has 0 variants, skipping\n",
      "[INFO]\tGene SEMA7A has 0 variants, skipping\n",
      "[INFO]\tGene KLHL8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF136 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130111 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf62 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D2B has 0 variants, skipping\n",
      "[INFO]\tGene ITPRID2 has 0 variants, skipping\n",
      "[INFO]\tGene COX20 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1B has 0 variants, skipping\n",
      "[INFO]\tGene CHRM4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM1 has 0 variants, skipping\n",
      "[INFO]\tGene CGB3 has 0 variants, skipping\n",
      "[INFO]\tGene CGA has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P1 has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P2 has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P3 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02249 has 0 variants, skipping\n",
      "[INFO]\tGene IER3 has 0 variants, skipping\n",
      "[INFO]\tGene CGREF1 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGA has 0 variants, skipping\n",
      "[INFO]\tGene CGRRF1 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P50 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288203 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02256 has 0 variants, skipping\n",
      "[INFO]\tGene ASH2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369728 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6C has 0 variants, skipping\n",
      "[INFO]\tGene CEP76 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288637 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P10 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02352 has 0 variants, skipping\n",
      "[INFO]\tGene IL33 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5B has 0 variants, skipping\n",
      "[INFO]\tGene COLCA1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF5 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO25 has 0 variants, skipping\n",
      "[INFO]\tGene PC has 0 variants, skipping\n",
      "[INFO]\tGene FAM209B has 0 variants, skipping\n",
      "[INFO]\tGene TRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene THYN1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR7 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL13 has 0 variants, skipping\n",
      "[INFO]\tGene FAM32A has 0 variants, skipping\n",
      "[INFO]\tGene GCDH has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131315 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO5B has 0 variants, skipping\n",
      "[INFO]\tGene MKRN2OS has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KL1 has 0 variants, skipping\n",
      "[INFO]\tGene CD99L2 has 0 variants, skipping\n",
      "[INFO]\tGene CRISPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SESN2 has 0 variants, skipping\n",
      "[INFO]\tGene PARP9 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A8 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOO has 0 variants, skipping\n",
      "[INFO]\tGene RAMAC has 0 variants, skipping\n",
      "[INFO]\tGene TEX101 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf68 has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ2 has 0 variants, skipping\n",
      "[INFO]\tGene ERP29 has 0 variants, skipping\n",
      "[INFO]\tGene RING1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1233-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1233-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02252 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928174 has 0 variants, skipping\n",
      "[INFO]\tGene AIFM2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32AP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3942 has 0 variants, skipping\n",
      "[INFO]\tGene DPH6-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4510 has 0 variants, skipping\n",
      "[INFO]\tGene OR52J3 has 0 variants, skipping\n",
      "[INFO]\tGene HEPN1 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145845 has 0 variants, skipping\n",
      "[INFO]\tGene CRYM has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA4 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8063 has 0 variants, skipping\n",
      "[INFO]\tGene PTGS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01852 has 0 variants, skipping\n",
      "[INFO]\tGene WIZ has 0 variants, skipping\n",
      "[INFO]\tGene SMAD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370941 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF234 has 0 variants, skipping\n",
      "[INFO]\tGene BACE1 has 0 variants, skipping\n",
      "[INFO]\tGene RPAIN has 0 variants, skipping\n",
      "[INFO]\tGene FEZF1 has 0 variants, skipping\n",
      "[INFO]\tGene SRP14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCSER1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31C2 has 0 variants, skipping\n",
      "[INFO]\tGene REEP4 has 0 variants, skipping\n",
      "[INFO]\tGene VCP has 0 variants, skipping\n",
      "[INFO]\tGene MYBL2 has 0 variants, skipping\n",
      "[INFO]\tGene CBL has 0 variants, skipping\n",
      "[INFO]\tGene CBFA2T3 has 0 variants, skipping\n",
      "[INFO]\tGene CALML3 has 0 variants, skipping\n",
      "[INFO]\tGene CALM3 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL42P5 has 0 variants, skipping\n",
      "[INFO]\tGene OR6S1 has 0 variants, skipping\n",
      "[INFO]\tGene TDG has 0 variants, skipping\n",
      "[INFO]\tGene CTIF has 0 variants, skipping\n",
      "[INFO]\tGene RNF40 has 0 variants, skipping\n",
      "[INFO]\tGene SHTN1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD5 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF71 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370943 has 0 variants, skipping\n",
      "[INFO]\tGene UTP15 has 0 variants, skipping\n",
      "[INFO]\tGene PSTPIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR35 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A8 has 0 variants, skipping\n",
      "[INFO]\tGene EARS2 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A30 has 0 variants, skipping\n",
      "[INFO]\tGene HMX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE5 has 0 variants, skipping\n",
      "[INFO]\tGene QDPR has 0 variants, skipping\n",
      "[INFO]\tGene TDRD15 has 0 variants, skipping\n",
      "[INFO]\tGene RASAL2 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF6 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK3 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4310 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370792 has 0 variants, skipping\n",
      "[INFO]\tGene EHD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD4 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC14 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf210 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF714 has 0 variants, skipping\n",
      "[INFO]\tGene SPC24 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD10 has 0 variants, skipping\n",
      "[INFO]\tGene ISM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF428 has 0 variants, skipping\n",
      "[INFO]\tGene FAM216B has 0 variants, skipping\n",
      "[INFO]\tGene KRT80 has 0 variants, skipping\n",
      "[INFO]\tGene KRT40 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD4A has 0 variants, skipping\n",
      "[INFO]\tGene SETDB1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRR has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM50B has 0 variants, skipping\n",
      "[INFO]\tGene DAGLA has 0 variants, skipping\n",
      "[INFO]\tGene BMP1 has 0 variants, skipping\n",
      "[INFO]\tGene IDS has 0 variants, skipping\n",
      "[INFO]\tGene MAN2A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA10 has 0 variants, skipping\n",
      "[INFO]\tGene VPS72 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB10 has 0 variants, skipping\n",
      "[INFO]\tGene ETDA has 0 variants, skipping\n",
      "[INFO]\tGene ETDB has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B2 has 0 variants, skipping\n",
      "[INFO]\tGene HRC has 0 variants, skipping\n",
      "[INFO]\tGene HPN has 0 variants, skipping\n",
      "[INFO]\tGene HPD has 0 variants, skipping\n",
      "[INFO]\tGene HOXD10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB5 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB1 has 0 variants, skipping\n",
      "[INFO]\tGene HPCA has 0 variants, skipping\n",
      "[INFO]\tGene HOXA4 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPK has 0 variants, skipping\n",
      "[INFO]\tGene CLIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SP1 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene USP4 has 0 variants, skipping\n",
      "[INFO]\tGene CD83 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF621 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49D1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP20 has 0 variants, skipping\n",
      "[INFO]\tGene SNX10 has 0 variants, skipping\n",
      "[INFO]\tGene CD79B has 0 variants, skipping\n",
      "[INFO]\tGene DAW1 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT112 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO2 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf74 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB1 has 0 variants, skipping\n",
      "[INFO]\tGene LDHC has 0 variants, skipping\n",
      "[INFO]\tGene ASPHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SALL4 has 0 variants, skipping\n",
      "[INFO]\tGene GJC2 has 0 variants, skipping\n",
      "[INFO]\tGene JPH2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63C has 0 variants, skipping\n",
      "[INFO]\tGene C18orf63 has 0 variants, skipping\n",
      "[INFO]\tGene NR2E1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9B has 0 variants, skipping\n",
      "[INFO]\tGene ATG5 has 0 variants, skipping\n",
      "[INFO]\tGene GPR6 has 0 variants, skipping\n",
      "[INFO]\tGene THEMIS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC90B has 0 variants, skipping\n",
      "[INFO]\tGene ACSM1 has 0 variants, skipping\n",
      "[INFO]\tGene UROS has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER2 has 0 variants, skipping\n",
      "[INFO]\tGene NCF4 has 0 variants, skipping\n",
      "[INFO]\tGene PEDS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAT1 has 0 variants, skipping\n",
      "[INFO]\tGene STRCP1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER2P1 has 0 variants, skipping\n",
      "[INFO]\tGene CD1D has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP3 has 0 variants, skipping\n",
      "[INFO]\tGene SERF2-C15ORF63 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1282 has 0 variants, skipping\n",
      "[INFO]\tGene MPP5 has 0 variants, skipping\n",
      "[INFO]\tGene PIN4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287387 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100419583 has 0 variants, skipping\n",
      "[INFO]\tGene LIN52 has 0 variants, skipping\n",
      "[INFO]\tGene SORD2P has 0 variants, skipping\n",
      "[INFO]\tGene PLAC8 has 0 variants, skipping\n",
      "[INFO]\tGene IMPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MARK4 has 0 variants, skipping\n",
      "[INFO]\tGene DDX19A has 0 variants, skipping\n",
      "[INFO]\tGene TOP2B has 0 variants, skipping\n",
      "[INFO]\tGene TOP2A has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928414 has 0 variants, skipping\n",
      "[INFO]\tGene CNRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2 has 0 variants, skipping\n",
      "[INFO]\tGene SP6 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN2P46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA41B has 0 variants, skipping\n",
      "[INFO]\tGene C3orf62 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370802 has 0 variants, skipping\n",
      "[INFO]\tGene DCBLD1 has 0 variants, skipping\n",
      "[INFO]\tGene BIVM has 0 variants, skipping\n",
      "[INFO]\tGene GCA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01491 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30A has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-3 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA7 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL5A has 0 variants, skipping\n",
      "[INFO]\tGene MRPL57 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN18 has 0 variants, skipping\n",
      "[INFO]\tGene CAPRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MD has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF4P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4716 has 0 variants, skipping\n",
      "[INFO]\tGene EMC3 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P1 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf58 has 0 variants, skipping\n",
      "[INFO]\tGene UXT has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf78 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC150 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA5 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRP3 has 0 variants, skipping\n",
      "[INFO]\tGene SURF6 has 0 variants, skipping\n",
      "[INFO]\tGene GNMT has 0 variants, skipping\n",
      "[INFO]\tGene CELSR1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRD1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF13P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4713HG has 0 variants, skipping\n",
      "[INFO]\tGene ZFR2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D1 has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2C has 0 variants, skipping\n",
      "[INFO]\tGene XPO6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4713 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7973-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7973-1 has 0 variants, skipping\n",
      "[INFO]\tGene PADI2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422556 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D7-LOC100130357 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A9 has 0 variants, skipping\n",
      "[INFO]\tGene GORASP2 has 0 variants, skipping\n",
      "[INFO]\tGene GALK1 has 0 variants, skipping\n",
      "[INFO]\tGene GAA has 0 variants, skipping\n",
      "[INFO]\tGene LINC02490 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370829 has 0 variants, skipping\n",
      "[INFO]\tGene STRN3 has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD2 has 0 variants, skipping\n",
      "[INFO]\tGene MACROH2A1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF4-CCPG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR628 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR1 has 0 variants, skipping\n",
      "[INFO]\tGene LARP4B has 0 variants, skipping\n",
      "[INFO]\tGene TIMP3 has 0 variants, skipping\n",
      "[INFO]\tGene S100G has 0 variants, skipping\n",
      "[INFO]\tGene CH25H has 0 variants, skipping\n",
      "[INFO]\tGene LINC02716 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02800 has 0 variants, skipping\n",
      "[INFO]\tGene MYRF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MMGT1 has 0 variants, skipping\n",
      "[INFO]\tGene OLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG5 has 0 variants, skipping\n",
      "[INFO]\tGene NAA35 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145783 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP17A1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-11 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL17 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00926 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01413 has 0 variants, skipping\n",
      "[INFO]\tGene RTL4 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX31 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283665 has 0 variants, skipping\n",
      "[INFO]\tGene LIPC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AB4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928725 has 0 variants, skipping\n",
      "[INFO]\tGene SATB1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC1 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD11 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPB has 0 variants, skipping\n",
      "[INFO]\tGene SIGMAR1 has 0 variants, skipping\n",
      "[INFO]\tGene RORA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PHYKPL has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF1 has 0 variants, skipping\n",
      "[INFO]\tGene LNX2 has 0 variants, skipping\n",
      "[INFO]\tGene RORA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02349 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN5 has 0 variants, skipping\n",
      "[INFO]\tGene PSPN has 0 variants, skipping\n",
      "[INFO]\tGene CDC25B has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928907 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984784 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8067 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6085 has 0 variants, skipping\n",
      "[INFO]\tGene MGC15885 has 0 variants, skipping\n",
      "[INFO]\tGene HSPG2 has 0 variants, skipping\n",
      "[INFO]\tGene EP400 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14A has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR33 has 0 variants, skipping\n",
      "[INFO]\tGene SHFL has 0 variants, skipping\n",
      "[INFO]\tGene MEN1 has 0 variants, skipping\n",
      "[INFO]\tGene TPM1-AS has 0 variants, skipping\n",
      "[INFO]\tGene SGCE has 0 variants, skipping\n",
      "[INFO]\tGene ADAP2 has 0 variants, skipping\n",
      "[INFO]\tGene VEPH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02568 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP91 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9B has 0 variants, skipping\n",
      "[INFO]\tGene GMPPA has 0 variants, skipping\n",
      "[INFO]\tGene USP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR1B has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR1A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH9A1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP4B has 0 variants, skipping\n",
      "[INFO]\tGene PSMD10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928988 has 0 variants, skipping\n",
      "[INFO]\tGene A4GALT has 0 variants, skipping\n",
      "[INFO]\tGene MTERF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A8 has 0 variants, skipping\n",
      "[INFO]\tGene RBP7 has 0 variants, skipping\n",
      "[INFO]\tGene PANX3 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf34 has 0 variants, skipping\n",
      "[INFO]\tGene METAP1D has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD7 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR4 has 0 variants, skipping\n",
      "[INFO]\tGene ATM has 0 variants, skipping\n",
      "[INFO]\tGene MIR1272 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB3 has 0 variants, skipping\n",
      "[INFO]\tGene LGR4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA24B has 0 variants, skipping\n",
      "[INFO]\tGene RNU5A-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5B-1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM222 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC23 has 0 variants, skipping\n",
      "[INFO]\tGene ACADS has 0 variants, skipping\n",
      "[INFO]\tGene NAT2 has 0 variants, skipping\n",
      "[INFO]\tGene A2M has 0 variants, skipping\n",
      "[INFO]\tGene MIA3 has 0 variants, skipping\n",
      "[INFO]\tGene SAP18 has 0 variants, skipping\n",
      "[INFO]\tGene ACO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13E has 0 variants, skipping\n",
      "[INFO]\tGene MIR4511 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4311 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A16 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23IP has 0 variants, skipping\n",
      "[INFO]\tGene CRYL1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNB2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA14 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4512 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18A has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K1 has 0 variants, skipping\n",
      "[INFO]\tGene AIRE has 0 variants, skipping\n",
      "[INFO]\tGene LINC01169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02206 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723493 has 0 variants, skipping\n",
      "[INFO]\tGene CCNP has 0 variants, skipping\n",
      "[INFO]\tGene COQ8B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0319L has 0 variants, skipping\n",
      "[INFO]\tGene IQCH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AG1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929076 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA7L has 0 variants, skipping\n",
      "[INFO]\tGene WDR62 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4312 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA10 has 0 variants, skipping\n",
      "[INFO]\tGene CDH3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145694 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC114 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2B has 0 variants, skipping\n",
      "[INFO]\tGene CYP19A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMED2 has 0 variants, skipping\n",
      "[INFO]\tGene PNRC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00593 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02205 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02204 has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA3 has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1A has 0 variants, skipping\n",
      "[INFO]\tGene THSD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THSD4-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRH has 0 variants, skipping\n",
      "[INFO]\tGene PTPRG has 0 variants, skipping\n",
      "[INFO]\tGene TRIM67 has 0 variants, skipping\n",
      "[INFO]\tGene RAB24 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF584 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD10 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB6 has 0 variants, skipping\n",
      "[INFO]\tGene ELOF1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4M1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2BP has 0 variants, skipping\n",
      "[INFO]\tGene LRIF1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1B has 0 variants, skipping\n",
      "[INFO]\tGene DRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ACER3 has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2 has 0 variants, skipping\n",
      "[INFO]\tGene SKA3 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP4 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2C has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2B has 0 variants, skipping\n",
      "[INFO]\tGene TERF1 has 0 variants, skipping\n",
      "[INFO]\tGene TEF has 0 variants, skipping\n",
      "[INFO]\tGene TEC has 0 variants, skipping\n",
      "[INFO]\tGene TEAD3 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD4 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA has 0 variants, skipping\n",
      "[INFO]\tGene TCF21 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA3 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA2 has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L1 has 0 variants, skipping\n",
      "[INFO]\tGene TBP has 0 variants, skipping\n",
      "[INFO]\tGene TBCE has 0 variants, skipping\n",
      "[INFO]\tGene TBCC has 0 variants, skipping\n",
      "[INFO]\tGene TFAM has 0 variants, skipping\n",
      "[INFO]\tGene HEXA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM202-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02259 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3B has 0 variants, skipping\n",
      "[INFO]\tGene GIPC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MASTL has 0 variants, skipping\n",
      "[INFO]\tGene OR5M3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01541 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8B has 0 variants, skipping\n",
      "[INFO]\tGene PIK3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP41 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK10 has 0 variants, skipping\n",
      "[INFO]\tGene HGH1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC4 has 0 variants, skipping\n",
      "[INFO]\tGene NCKIPSD has 0 variants, skipping\n",
      "[INFO]\tGene LOC283731 has 0 variants, skipping\n",
      "[INFO]\tGene CCNI2 has 0 variants, skipping\n",
      "[INFO]\tGene TAPBP has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K7 has 0 variants, skipping\n",
      "[INFO]\tGene TAF9 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN has 0 variants, skipping\n",
      "[INFO]\tGene TAF4 has 0 variants, skipping\n",
      "[INFO]\tGene TAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TAC1 has 0 variants, skipping\n",
      "[INFO]\tGene TBXT has 0 variants, skipping\n",
      "[INFO]\tGene SYT5 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAP46 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02255 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1260B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3162 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6881 has 0 variants, skipping\n",
      "[INFO]\tGene MCPH1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1A has 0 variants, skipping\n",
      "[INFO]\tGene PKD2L2 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH2 has 0 variants, skipping\n",
      "[INFO]\tGene CES2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR180 has 0 variants, skipping\n",
      "[INFO]\tGene ZACN has 0 variants, skipping\n",
      "[INFO]\tGene MRPL52 has 0 variants, skipping\n",
      "[INFO]\tGene IL21R has 0 variants, skipping\n",
      "[INFO]\tGene LINC00857 has 0 variants, skipping\n",
      "[INFO]\tGene NORAD has 0 variants, skipping\n",
      "[INFO]\tGene LINC01124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6882 has 0 variants, skipping\n",
      "[INFO]\tGene IL22RA2 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf86 has 0 variants, skipping\n",
      "[INFO]\tGene PTDSS2 has 0 variants, skipping\n",
      "[INFO]\tGene ARPP19 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD7 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR631 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP3 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD5 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN5 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT13 has 0 variants, skipping\n",
      "[INFO]\tGene IL6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A2 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4313 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN3 has 0 variants, skipping\n",
      "[INFO]\tGene NACA2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC137 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929439 has 0 variants, skipping\n",
      "[INFO]\tGene TAX1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TYRO3P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF250 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC2 has 0 variants, skipping\n",
      "[INFO]\tGene JHY has 0 variants, skipping\n",
      "[INFO]\tGene ZNF141 has 0 variants, skipping\n",
      "[INFO]\tGene PAOX has 0 variants, skipping\n",
      "[INFO]\tGene EZR has 0 variants, skipping\n",
      "[INFO]\tGene FAM160A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3713 has 0 variants, skipping\n",
      "[INFO]\tGene CUL4A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88C has 0 variants, skipping\n",
      "[INFO]\tGene STARD13 has 0 variants, skipping\n",
      "[INFO]\tGene UBASH3A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00597 has 0 variants, skipping\n",
      "[INFO]\tGene CHN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929457 has 0 variants, skipping\n",
      "[INFO]\tGene TOLLIP has 0 variants, skipping\n",
      "[INFO]\tGene LIPG has 0 variants, skipping\n",
      "[INFO]\tGene GDAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC91450 has 0 variants, skipping\n",
      "[INFO]\tGene EPGN has 0 variants, skipping\n",
      "[INFO]\tGene WDR47 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54L has 0 variants, skipping\n",
      "[INFO]\tGene LUZP1 has 0 variants, skipping\n",
      "[INFO]\tGene DBF4B has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA2 has 0 variants, skipping\n",
      "[INFO]\tGene IL27 has 0 variants, skipping\n",
      "[INFO]\tGene SPESP1 has 0 variants, skipping\n",
      "[INFO]\tGene GSX1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646938 has 0 variants, skipping\n",
      "[INFO]\tGene CCN6 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RAD9A has 0 variants, skipping\n",
      "[INFO]\tGene COL5A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129540 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMED3 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFS has 0 variants, skipping\n",
      "[INFO]\tGene PRR7 has 0 variants, skipping\n",
      "[INFO]\tGene ST20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM54 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00927 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929586 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5572 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4514 has 0 variants, skipping\n",
      "[INFO]\tGene COX7A2L has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB37 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01583 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R3 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN14 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX16 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS7P1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC73 has 0 variants, skipping\n",
      "[INFO]\tGene VCPKMT has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2P2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724034 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC727751 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L17P has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF749 has 0 variants, skipping\n",
      "[INFO]\tGene INIP has 0 variants, skipping\n",
      "[INFO]\tGene C10orf105 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A6 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A3 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB107B has 0 variants, skipping\n",
      "[INFO]\tGene INCENP has 0 variants, skipping\n",
      "[INFO]\tGene SMARCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC338963 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P17 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG21 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA15 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1E has 0 variants, skipping\n",
      "[INFO]\tGene CSN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373289 has 0 variants, skipping\n",
      "[INFO]\tGene GUK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4515 has 0 variants, skipping\n",
      "[INFO]\tGene DUT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF865 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF17 has 0 variants, skipping\n",
      "[INFO]\tGene EFHB has 0 variants, skipping\n",
      "[INFO]\tGene CBLN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM232 has 0 variants, skipping\n",
      "[INFO]\tGene CCNH has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS13 has 0 variants, skipping\n",
      "[INFO]\tGene TCAP has 0 variants, skipping\n",
      "[INFO]\tGene EFL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P41 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440300 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P7 has 0 variants, skipping\n",
      "[INFO]\tGene NBAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103171574 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2E has 0 variants, skipping\n",
      "[INFO]\tGene LINC00460 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L5P has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00933 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAND2P has 0 variants, skipping\n",
      "[INFO]\tGene OR3A2 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D2 has 0 variants, skipping\n",
      "[INFO]\tGene OPHN1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7706 has 0 variants, skipping\n",
      "[INFO]\tGene FMO1 has 0 variants, skipping\n",
      "[INFO]\tGene ORC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1276 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AP has 0 variants, skipping\n",
      "[INFO]\tGene ARID1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01584 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370954 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724452 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF3 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L3 has 0 variants, skipping\n",
      "[INFO]\tGene CLN5 has 0 variants, skipping\n",
      "[INFO]\tGene POLH has 0 variants, skipping\n",
      "[INFO]\tGene NTRK3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP4 has 0 variants, skipping\n",
      "[INFO]\tGene NCK1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SETD6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01586 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3529 has 0 variants, skipping\n",
      "[INFO]\tGene BCO2 has 0 variants, skipping\n",
      "[INFO]\tGene MLEC has 0 variants, skipping\n",
      "[INFO]\tGene MIR6766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-3HG has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00928 has 0 variants, skipping\n",
      "[INFO]\tGene WASF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5ME has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5094 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5009 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3174 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF710-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RESF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNL3L has 0 variants, skipping\n",
      "[INFO]\tGene IDH2-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP3 has 0 variants, skipping\n",
      "[INFO]\tGene CNDP2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP133 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB105A has 0 variants, skipping\n",
      "[INFO]\tGene SND1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB114 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL13P has 0 variants, skipping\n",
      "[INFO]\tGene C1orf52 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF853 has 0 variants, skipping\n",
      "[INFO]\tGene NETO1 has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1B has 0 variants, skipping\n",
      "[INFO]\tGene CRTC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01585 has 0 variants, skipping\n",
      "[INFO]\tGene OR56A1 has 0 variants, skipping\n",
      "[INFO]\tGene DBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705A has 0 variants, skipping\n",
      "[INFO]\tGene CYP27C1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB7B has 0 variants, skipping\n",
      "[INFO]\tGene CCNQ has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G9 has 0 variants, skipping\n",
      "[INFO]\tGene CYTIP has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SUCO has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP22 has 0 variants, skipping\n",
      "[INFO]\tGene PRC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP126 has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1B has 0 variants, skipping\n",
      "[INFO]\tGene VPS33B-DT has 0 variants, skipping\n",
      "[INFO]\tGene NAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CRAT37 has 0 variants, skipping\n",
      "[INFO]\tGene LOC104613533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00930 has 0 variants, skipping\n",
      "[INFO]\tGene ASB9P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02207 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01580 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01581 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440311 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00924 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1L has 0 variants, skipping\n",
      "[INFO]\tGene SPATA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927310 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00923 has 0 variants, skipping\n",
      "[INFO]\tGene NUP62CL has 0 variants, skipping\n",
      "[INFO]\tGene LINC02251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01582 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02351 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4714 has 0 variants, skipping\n",
      "[INFO]\tGene LUNAR1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH1 has 0 variants, skipping\n",
      "[INFO]\tGene EML6 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN2A has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01213 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF4P has 0 variants, skipping\n",
      "[INFO]\tGene HSP90B2P has 0 variants, skipping\n",
      "[INFO]\tGene HSPA4L has 0 variants, skipping\n",
      "[INFO]\tGene MPDU1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA18 has 0 variants, skipping\n",
      "[INFO]\tGene RAD23B has 0 variants, skipping\n",
      "[INFO]\tGene PRRC1 has 0 variants, skipping\n",
      "[INFO]\tGene AFF3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT85 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P46 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400464 has 0 variants, skipping\n",
      "[INFO]\tGene RBM25 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA41 has 0 variants, skipping\n",
      "[INFO]\tGene CERS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL2NB has 0 variants, skipping\n",
      "[INFO]\tGene DLAT has 0 variants, skipping\n",
      "[INFO]\tGene PRKXP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723335 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927751 has 0 variants, skipping\n",
      "[INFO]\tGene H1-8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507472 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02348 has 0 variants, skipping\n",
      "[INFO]\tGene FOLH1 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A4E has 0 variants, skipping\n",
      "[INFO]\tGene FAM25E has 0 variants, skipping\n",
      "[INFO]\tGene OR4F13P has 0 variants, skipping\n",
      "[INFO]\tGene FAM138E has 0 variants, skipping\n",
      "[INFO]\tGene WASH3P has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-4 has 0 variants, skipping\n",
      "[INFO]\tGene SP100 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIZ has 0 variants, skipping\n",
      "[INFO]\tGene MRPL9 has 0 variants, skipping\n",
      "[INFO]\tGene ANO3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH19 has 0 variants, skipping\n",
      "[INFO]\tGene SPEGNB has 0 variants, skipping\n",
      "[INFO]\tGene EAPP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM242 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOS has 0 variants, skipping\n",
      "[INFO]\tGene PDP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A10 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3E has 0 variants, skipping\n",
      "[INFO]\tGene NISCH has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1B has 0 variants, skipping\n",
      "[INFO]\tGene RPS14 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA2 has 0 variants, skipping\n",
      "[INFO]\tGene LTA has 0 variants, skipping\n",
      "[INFO]\tGene CD4 has 0 variants, skipping\n",
      "[INFO]\tGene NCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene VRK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389895 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene COA3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH17 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB11 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQA2 has 0 variants, skipping\n",
      "[INFO]\tGene UBB has 0 variants, skipping\n",
      "[INFO]\tGene WWC1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM184A has 0 variants, skipping\n",
      "[INFO]\tGene RWDD4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD4A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC36 has 0 variants, skipping\n",
      "[INFO]\tGene PGK1 has 0 variants, skipping\n",
      "[INFO]\tGene PLG has 0 variants, skipping\n",
      "[INFO]\tGene PGAM2 has 0 variants, skipping\n",
      "[INFO]\tGene DYRK3 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM4 has 0 variants, skipping\n",
      "[INFO]\tGene AOAH has 0 variants, skipping\n",
      "[INFO]\tGene LASP1 has 0 variants, skipping\n",
      "[INFO]\tGene TWF1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT4H1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT1 has 0 variants, skipping\n",
      "[INFO]\tGene SRF has 0 variants, skipping\n",
      "[INFO]\tGene WNT2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724720 has 0 variants, skipping\n",
      "[INFO]\tGene OR51S1 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL has 0 variants, skipping\n",
      "[INFO]\tGene SMIM41 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM40 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM38 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2C has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3B has 0 variants, skipping\n",
      "[INFO]\tGene NXF2B has 0 variants, skipping\n",
      "[INFO]\tGene BCL11B has 0 variants, skipping\n",
      "[INFO]\tGene STAG3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12J has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L6 has 0 variants, skipping\n",
      "[INFO]\tGene WFS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL2 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MAT1A has 0 variants, skipping\n",
      "[INFO]\tGene WASF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF317 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D30 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD4 has 0 variants, skipping\n",
      "[INFO]\tGene OSMR has 0 variants, skipping\n",
      "[INFO]\tGene FDX1 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene IRX5 has 0 variants, skipping\n",
      "[INFO]\tGene CHD3 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE18 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA2 has 0 variants, skipping\n",
      "[INFO]\tGene ACSBG2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP5 has 0 variants, skipping\n",
      "[INFO]\tGene GH2 has 0 variants, skipping\n",
      "[INFO]\tGene ETHE1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D13 has 0 variants, skipping\n",
      "[INFO]\tGene OR14A16 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D5 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1LW has 0 variants, skipping\n",
      "[INFO]\tGene CCDC33 has 0 variants, skipping\n",
      "[INFO]\tGene CENPU has 0 variants, skipping\n",
      "[INFO]\tGene PSMA5 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB8 has 0 variants, skipping\n",
      "[INFO]\tGene VPS51 has 0 variants, skipping\n",
      "[INFO]\tGene MYL9 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH1 has 0 variants, skipping\n",
      "[INFO]\tGene TNK2 has 0 variants, skipping\n",
      "[INFO]\tGene PITX2 has 0 variants, skipping\n",
      "[INFO]\tGene EXD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR572 has 0 variants, skipping\n",
      "[INFO]\tGene WASIR2 has 0 variants, skipping\n",
      "[INFO]\tGene PHKG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM234A has 0 variants, skipping\n",
      "[INFO]\tGene GIMD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRTOMT has 0 variants, skipping\n",
      "[INFO]\tGene RNF225 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf27 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA1 has 0 variants, skipping\n",
      "[INFO]\tGene TBRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134368 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W3 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C9 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C13 has 0 variants, skipping\n",
      "[INFO]\tGene OR5A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP37 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00235 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5587 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3176 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11L2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SETDB2-PHF11 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf57 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287175 has 0 variants, skipping\n",
      "[INFO]\tGene COA7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371038 has 0 variants, skipping\n",
      "[INFO]\tGene MDH1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA52 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02825 has 0 variants, skipping\n",
      "[INFO]\tGene STARD6 has 0 variants, skipping\n",
      "[INFO]\tGene RHOQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CD177 has 0 variants, skipping\n",
      "[INFO]\tGene PLXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LMF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG1 has 0 variants, skipping\n",
      "[INFO]\tGene CD55 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD8 has 0 variants, skipping\n",
      "[INFO]\tGene USP21 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM237 has 0 variants, skipping\n",
      "[INFO]\tGene MYL3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02789 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02714 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371046 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD3 has 0 variants, skipping\n",
      "[INFO]\tGene IGFALS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00254 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02124 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA64 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA78 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC20 has 0 variants, skipping\n",
      "[INFO]\tGene GDF10 has 0 variants, skipping\n",
      "[INFO]\tGene RPL3L has 0 variants, skipping\n",
      "[INFO]\tGene OR8D2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8D1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD11 has 0 variants, skipping\n",
      "[INFO]\tGene PTRH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRCP has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD7 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf16 has 0 variants, skipping\n",
      "[INFO]\tGene EQTN has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF23 has 0 variants, skipping\n",
      "[INFO]\tGene FGF22 has 0 variants, skipping\n",
      "[INFO]\tGene GDF3 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF117 has 0 variants, skipping\n",
      "[INFO]\tGene CELA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF18 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF56 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511B2 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD60 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4F has 0 variants, skipping\n",
      "[INFO]\tGene TUT4 has 0 variants, skipping\n",
      "[INFO]\tGene FOXRED1 has 0 variants, skipping\n",
      "[INFO]\tGene PPBP has 0 variants, skipping\n",
      "[INFO]\tGene PPAT has 0 variants, skipping\n",
      "[INFO]\tGene POLE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC106660606 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA17P has 0 variants, skipping\n",
      "[INFO]\tGene DMTF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6767 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM53 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6768 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517B has 0 variants, skipping\n",
      "[INFO]\tGene LOC652276 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42627 has 0 variants, skipping\n",
      "[INFO]\tGene ERVK13-1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3C has 0 variants, skipping\n",
      "[INFO]\tGene NONO has 0 variants, skipping\n",
      "[INFO]\tGene PRSS30P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929613 has 0 variants, skipping\n",
      "[INFO]\tGene LCA5 has 0 variants, skipping\n",
      "[INFO]\tGene HMMR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128770 has 0 variants, skipping\n",
      "[INFO]\tGene MMP25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF213-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NHP2 has 0 variants, skipping\n",
      "[INFO]\tGene MUC12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP16P has 0 variants, skipping\n",
      "[INFO]\tGene OR1F2P has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD29 has 0 variants, skipping\n",
      "[INFO]\tGene LARP7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00921 has 0 variants, skipping\n",
      "[INFO]\tGene SLC40A1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6126 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E2B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01569 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM236 has 0 variants, skipping\n",
      "[INFO]\tGene SSX4B has 0 variants, skipping\n",
      "[INFO]\tGene PDCD10 has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD5 has 0 variants, skipping\n",
      "[INFO]\tGene MUC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6769A has 0 variants, skipping\n",
      "[INFO]\tGene IQUB has 0 variants, skipping\n",
      "[INFO]\tGene TAFA5 has 0 variants, skipping\n",
      "[INFO]\tGene SAMM50 has 0 variants, skipping\n",
      "[INFO]\tGene BRI3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13D has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L13 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS40A has 0 variants, skipping\n",
      "[INFO]\tGene AUP1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf84 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF10 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L3 has 0 variants, skipping\n",
      "[INFO]\tGene NAGPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PI4K2A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01570 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8065 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM131 has 0 variants, skipping\n",
      "[INFO]\tGene CASS4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130283 has 0 variants, skipping\n",
      "[INFO]\tGene PPP5C has 0 variants, skipping\n",
      "[INFO]\tGene FBXW10 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01195 has 0 variants, skipping\n",
      "[INFO]\tGene SPX has 0 variants, skipping\n",
      "[INFO]\tGene PM20D1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723753 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01290 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2K has 0 variants, skipping\n",
      "[INFO]\tGene RNF214 has 0 variants, skipping\n",
      "[INFO]\tGene LTB4R2 has 0 variants, skipping\n",
      "[INFO]\tGene BEGAIN has 0 variants, skipping\n",
      "[INFO]\tGene HOMEZ has 0 variants, skipping\n",
      "[INFO]\tGene ZNF687 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371083 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927131 has 0 variants, skipping\n",
      "[INFO]\tGene INSR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF8 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGC has 0 variants, skipping\n",
      "[INFO]\tGene NFS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF516 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4718 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ERVV-2 has 0 variants, skipping\n",
      "[INFO]\tGene PDK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02185 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02186 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3L4 has 0 variants, skipping\n",
      "[INFO]\tGene POMGNT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02130 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC11A2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288162 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-3 has 0 variants, skipping\n",
      "[INFO]\tGene DEK has 0 variants, skipping\n",
      "[INFO]\tGene SARDH has 0 variants, skipping\n",
      "[INFO]\tGene C1orf94 has 0 variants, skipping\n",
      "[INFO]\tGene NXNL2 has 0 variants, skipping\n",
      "[INFO]\tGene DUS3L has 0 variants, skipping\n",
      "[INFO]\tGene NECAB1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1972-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1972-2 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene FGF17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505915 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P6-NPIPP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100190986 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK4 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK1 has 0 variants, skipping\n",
      "[INFO]\tGene MAS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1C has 0 variants, skipping\n",
      "[INFO]\tGene ATOH1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOX2A has 0 variants, skipping\n",
      "[INFO]\tGene PPM1J has 0 variants, skipping\n",
      "[INFO]\tGene P4HA3 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A12 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf72 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf197 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF652 has 0 variants, skipping\n",
      "[INFO]\tGene CERKL has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723692 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20A has 0 variants, skipping\n",
      "[INFO]\tGene PRECSIT has 0 variants, skipping\n",
      "[INFO]\tGene NOP56 has 0 variants, skipping\n",
      "[INFO]\tGene DOLK has 0 variants, skipping\n",
      "[INFO]\tGene GLTP has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC15 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6P1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf36 has 0 variants, skipping\n",
      "[INFO]\tGene ADM2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723385 has 0 variants, skipping\n",
      "[INFO]\tGene SCN9A has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN7A has 0 variants, skipping\n",
      "[INFO]\tGene CLEC11A has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB4 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC1 has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1 has 0 variants, skipping\n",
      "[INFO]\tGene GASK1A has 0 variants, skipping\n",
      "[INFO]\tGene MSMO1 has 0 variants, skipping\n",
      "[INFO]\tGene NDP has 0 variants, skipping\n",
      "[INFO]\tGene NCF1 has 0 variants, skipping\n",
      "[INFO]\tGene NFYA has 0 variants, skipping\n",
      "[INFO]\tGene TICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF500 has 0 variants, skipping\n",
      "[INFO]\tGene OTOA has 0 variants, skipping\n",
      "[INFO]\tGene KCNH3 has 0 variants, skipping\n",
      "[INFO]\tGene VAV3 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B4 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC5 has 0 variants, skipping\n",
      "[INFO]\tGene ARVCF has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA2 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ARTN has 0 variants, skipping\n",
      "[INFO]\tGene KRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TAGAP has 0 variants, skipping\n",
      "[INFO]\tGene CIB3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB118 has 0 variants, skipping\n",
      "[INFO]\tGene IP6K3 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT15 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT5 has 0 variants, skipping\n",
      "[INFO]\tGene OR1J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AC2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C3 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX4 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKH has 0 variants, skipping\n",
      "[INFO]\tGene SCGB3A2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR101 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1A has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4B has 0 variants, skipping\n",
      "[INFO]\tGene ASS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT1 has 0 variants, skipping\n",
      "[INFO]\tGene FBH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29P1 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3680-2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927814 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF211 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200A has 0 variants, skipping\n",
      "[INFO]\tGene OTOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE7 has 0 variants, skipping\n",
      "[INFO]\tGene PPOX has 0 variants, skipping\n",
      "[INFO]\tGene BGLAP has 0 variants, skipping\n",
      "[INFO]\tGene PMEPA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440346 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD13B has 0 variants, skipping\n",
      "[INFO]\tGene CD151 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF4 has 0 variants, skipping\n",
      "[INFO]\tGene BICD1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS29 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730183 has 0 variants, skipping\n",
      "[INFO]\tGene PI4K2B has 0 variants, skipping\n",
      "[INFO]\tGene DUSP9 has 0 variants, skipping\n",
      "[INFO]\tGene PSME2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1273H has 0 variants, skipping\n",
      "[INFO]\tGene LINC01567 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA6 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB1 has 0 variants, skipping\n",
      "[INFO]\tGene HASPIN has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP17-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-8 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-5 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP15 has 0 variants, skipping\n",
      "[INFO]\tGene F11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF185 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P1 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC554206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02175 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548W has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE1-DT has 0 variants, skipping\n",
      "[INFO]\tGene IL21R-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NGRN has 0 variants, skipping\n",
      "[INFO]\tGene MYCL has 0 variants, skipping\n",
      "[INFO]\tGene TAAR2 has 0 variants, skipping\n",
      "[INFO]\tGene INTS11 has 0 variants, skipping\n",
      "[INFO]\tGene WAPL has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128079 has 0 variants, skipping\n",
      "[INFO]\tGene WDR3 has 0 variants, skipping\n",
      "[INFO]\tGene VLDLR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF845 has 0 variants, skipping\n",
      "[INFO]\tGene GSAP has 0 variants, skipping\n",
      "[INFO]\tGene DDX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6862-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6862-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD2 has 0 variants, skipping\n",
      "[INFO]\tGene TBCA has 0 variants, skipping\n",
      "[INFO]\tGene SNRPA has 0 variants, skipping\n",
      "[INFO]\tGene SMS has 0 variants, skipping\n",
      "[INFO]\tGene SULT2B1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP23 has 0 variants, skipping\n",
      "[INFO]\tGene CACFD1 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF559-ZNF177 has 0 variants, skipping\n",
      "[INFO]\tGene GLRB has 0 variants, skipping\n",
      "[INFO]\tGene GHRHR has 0 variants, skipping\n",
      "[INFO]\tGene GART has 0 variants, skipping\n",
      "[INFO]\tGene MIR4721 has 0 variants, skipping\n",
      "[INFO]\tGene HTATSF1 has 0 variants, skipping\n",
      "[INFO]\tGene UGDH has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4517 has 0 variants, skipping\n",
      "[INFO]\tGene PGM1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC1 has 0 variants, skipping\n",
      "[INFO]\tGene WSCD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29P2 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC606724 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC4 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1A-SULT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1B-SULT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388242 has 0 variants, skipping\n",
      "[INFO]\tGene LOC613038 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR3B has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA7 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA5 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA3 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A40 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG12 has 0 variants, skipping\n",
      "[INFO]\tGene NSFL1C has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA12 has 0 variants, skipping\n",
      "[INFO]\tGene SULF2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIPTOSP has 0 variants, skipping\n",
      "[INFO]\tGene FBXO21 has 0 variants, skipping\n",
      "[INFO]\tGene RBMX has 0 variants, skipping\n",
      "[INFO]\tGene CXorf65 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642361 has 0 variants, skipping\n",
      "[INFO]\tGene MUSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12B has 0 variants, skipping\n",
      "[INFO]\tGene ACER2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM102B has 0 variants, skipping\n",
      "[INFO]\tGene CDRT1 has 0 variants, skipping\n",
      "[INFO]\tGene LCTL has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM has 0 variants, skipping\n",
      "[INFO]\tGene DRG2 has 0 variants, skipping\n",
      "[INFO]\tGene ALS2 has 0 variants, skipping\n",
      "[INFO]\tGene LZTS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928595 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD9 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P5 has 0 variants, skipping\n",
      "[INFO]\tGene IL4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80C has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO2 has 0 variants, skipping\n",
      "[INFO]\tGene AHSA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4518 has 0 variants, skipping\n",
      "[INFO]\tGene TRDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF689 has 0 variants, skipping\n",
      "[INFO]\tGene CASP12 has 0 variants, skipping\n",
      "[INFO]\tGene GSTO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA30 has 0 variants, skipping\n",
      "[INFO]\tGene BLID has 0 variants, skipping\n",
      "[INFO]\tGene COP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP295NL has 0 variants, skipping\n",
      "[INFO]\tGene HELLS has 0 variants, skipping\n",
      "[INFO]\tGene MIR4519 has 0 variants, skipping\n",
      "[INFO]\tGene MIR762HG has 0 variants, skipping\n",
      "[INFO]\tGene PTPN9 has 0 variants, skipping\n",
      "[INFO]\tGene DDX10 has 0 variants, skipping\n",
      "[INFO]\tGene DDX6 has 0 variants, skipping\n",
      "[INFO]\tGene DMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX5 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH2 has 0 variants, skipping\n",
      "[INFO]\tGene PALM has 0 variants, skipping\n",
      "[INFO]\tGene REG3A has 0 variants, skipping\n",
      "[INFO]\tGene NAV3 has 0 variants, skipping\n",
      "[INFO]\tGene PYCARD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF320 has 0 variants, skipping\n",
      "[INFO]\tGene RDH11 has 0 variants, skipping\n",
      "[INFO]\tGene C1R has 0 variants, skipping\n",
      "[INFO]\tGene ADCY3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02190 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2KP has 0 variants, skipping\n",
      "[INFO]\tGene YBX3P1 has 0 variants, skipping\n",
      "[INFO]\tGene CLUHP3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P4 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3D has 0 variants, skipping\n",
      "[INFO]\tGene LOC390705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369266 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724135 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A10P has 0 variants, skipping\n",
      "[INFO]\tGene DCTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC93622 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG8 has 0 variants, skipping\n",
      "[INFO]\tGene SACS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369212 has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929479 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP7P13 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2MP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01566 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2DP has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3HP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02167 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26P1 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf180 has 0 variants, skipping\n",
      "[INFO]\tGene SLIRP has 0 variants, skipping\n",
      "[INFO]\tGene UPF2 has 0 variants, skipping\n",
      "[INFO]\tGene CST9L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF222 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD148 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02192 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02133 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02134 has 0 variants, skipping\n",
      "[INFO]\tGene CCT8 has 0 variants, skipping\n",
      "[INFO]\tGene PCP2 has 0 variants, skipping\n",
      "[INFO]\tGene ESPNL has 0 variants, skipping\n",
      "[INFO]\tGene ADAM3B has 0 variants, skipping\n",
      "[INFO]\tGene THG1L has 0 variants, skipping\n",
      "[INFO]\tGene PCMTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A36 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6771 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927272 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene SAFB has 0 variants, skipping\n",
      "[INFO]\tGene S100A10 has 0 variants, skipping\n",
      "[INFO]\tGene S100A9 has 0 variants, skipping\n",
      "[INFO]\tGene SLPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR3181 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02168 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AI has 0 variants, skipping\n",
      "[INFO]\tGene LINC02127 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01571 has 0 variants, skipping\n",
      "[INFO]\tGene PPCS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00919 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02180 has 0 variants, skipping\n",
      "[INFO]\tGene CASC22 has 0 variants, skipping\n",
      "[INFO]\tGene CASC16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371267 has 0 variants, skipping\n",
      "[INFO]\tGene RECK has 0 variants, skipping\n",
      "[INFO]\tGene CTAG1A has 0 variants, skipping\n",
      "[INFO]\tGene MS4A6E has 0 variants, skipping\n",
      "[INFO]\tGene LIPH has 0 variants, skipping\n",
      "[INFO]\tGene TENT5D has 0 variants, skipping\n",
      "[INFO]\tGene TMEM123 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF6 has 0 variants, skipping\n",
      "[INFO]\tGene FHAD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723373 has 0 variants, skipping\n",
      "[INFO]\tGene YIF1B has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L12 has 0 variants, skipping\n",
      "[INFO]\tGene CENPN has 0 variants, skipping\n",
      "[INFO]\tGene RDH12 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL14 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf66 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF664 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1L2 has 0 variants, skipping\n",
      "[INFO]\tGene AMDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CWF19L2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP42 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL4 has 0 variants, skipping\n",
      "[INFO]\tGene OR51E1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP3 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3D has 0 variants, skipping\n",
      "[INFO]\tGene FTO-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927480 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA6 has 0 variants, skipping\n",
      "[INFO]\tGene MMP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MED27 has 0 variants, skipping\n",
      "[INFO]\tGene ACY3 has 0 variants, skipping\n",
      "[INFO]\tGene ARAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CES1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CES1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF404 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283856 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL22 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP434H168 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3935 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K13 has 0 variants, skipping\n",
      "[INFO]\tGene OR1I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A3 has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIB has 0 variants, skipping\n",
      "[INFO]\tGene TDRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1L has 0 variants, skipping\n",
      "[INFO]\tGene ZAR1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1DP has 0 variants, skipping\n",
      "[INFO]\tGene MT1IP has 0 variants, skipping\n",
      "[INFO]\tGene GPR33 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F16 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AE1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A15 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A16 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R13B has 0 variants, skipping\n",
      "[INFO]\tGene STIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RIOX2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB10 has 0 variants, skipping\n",
      "[INFO]\tGene COPG1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3C has 0 variants, skipping\n",
      "[INFO]\tGene FAM170A has 0 variants, skipping\n",
      "[INFO]\tGene TOX has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD4 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2B has 0 variants, skipping\n",
      "[INFO]\tGene POLR2A has 0 variants, skipping\n",
      "[INFO]\tGene PNLIP has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR18 has 0 variants, skipping\n",
      "[INFO]\tGene ASB9 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF3 has 0 variants, skipping\n",
      "[INFO]\tGene UXS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6772 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAL has 0 variants, skipping\n",
      "[INFO]\tGene KRT33B has 0 variants, skipping\n",
      "[INFO]\tGene KRT35 has 0 variants, skipping\n",
      "[INFO]\tGene KRT32 has 0 variants, skipping\n",
      "[INFO]\tGene FTSJ1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE5 has 0 variants, skipping\n",
      "[INFO]\tGene KCND2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD3 has 0 variants, skipping\n",
      "[INFO]\tGene TREML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8084 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6783 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6751 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF84-DT has 0 variants, skipping\n",
      "[INFO]\tGene SEH1L has 0 variants, skipping\n",
      "[INFO]\tGene MPST has 0 variants, skipping\n",
      "[INFO]\tGene PAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene PREX2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX21 has 0 variants, skipping\n",
      "[INFO]\tGene MROH8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50A has 0 variants, skipping\n",
      "[INFO]\tGene APOOP5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02141 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02126 has 0 variants, skipping\n",
      "[INFO]\tGene LOC647264 has 0 variants, skipping\n",
      "[INFO]\tGene H2BU1 has 0 variants, skipping\n",
      "[INFO]\tGene SYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene BEAN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13C has 0 variants, skipping\n",
      "[INFO]\tGene EFNA4 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR3 has 0 variants, skipping\n",
      "[INFO]\tGene E2F1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID3A has 0 variants, skipping\n",
      "[INFO]\tGene DLX6 has 0 variants, skipping\n",
      "[INFO]\tGene DLX5 has 0 variants, skipping\n",
      "[INFO]\tGene DLX3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA3 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM3 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1LI2 has 0 variants, skipping\n",
      "[INFO]\tGene NGDN has 0 variants, skipping\n",
      "[INFO]\tGene LOC106699570 has 0 variants, skipping\n",
      "[INFO]\tGene SSH1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F5 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC8 has 0 variants, skipping\n",
      "[INFO]\tGene AK3 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT3 has 0 variants, skipping\n",
      "[INFO]\tGene CHGB has 0 variants, skipping\n",
      "[INFO]\tGene IGSF9 has 0 variants, skipping\n",
      "[INFO]\tGene RGS6 has 0 variants, skipping\n",
      "[INFO]\tGene TINAGL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF213 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF529 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A14 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1614 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA5 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA10 has 0 variants, skipping\n",
      "[INFO]\tGene MEAK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF253 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-8 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF15 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELL2 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PUDP has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF9 has 0 variants, skipping\n",
      "[INFO]\tGene OPRPN has 0 variants, skipping\n",
      "[INFO]\tGene VASH2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505942 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984813 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD6B has 0 variants, skipping\n",
      "[INFO]\tGene TECR has 0 variants, skipping\n",
      "[INFO]\tGene HRG has 0 variants, skipping\n",
      "[INFO]\tGene HLCS has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10C has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10D has 0 variants, skipping\n",
      "[INFO]\tGene FBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D1 has 0 variants, skipping\n",
      "[INFO]\tGene RTF1 has 0 variants, skipping\n",
      "[INFO]\tGene RNGTT has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ-SCHIP1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A2 has 0 variants, skipping\n",
      "[INFO]\tGene CCL27 has 0 variants, skipping\n",
      "[INFO]\tGene SRCAP has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP2 has 0 variants, skipping\n",
      "[INFO]\tGene CLPX has 0 variants, skipping\n",
      "[INFO]\tGene FTCD has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A4 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD10 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6773 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRD has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A1 has 0 variants, skipping\n",
      "[INFO]\tGene DPM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF418 has 0 variants, skipping\n",
      "[INFO]\tGene EZH2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2W has 0 variants, skipping\n",
      "[INFO]\tGene SPATA48 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMM has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRF has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF5 has 0 variants, skipping\n",
      "[INFO]\tGene HTATIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB6 has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL2 has 0 variants, skipping\n",
      "[INFO]\tGene STX18 has 0 variants, skipping\n",
      "[INFO]\tGene PIGW has 0 variants, skipping\n",
      "[INFO]\tGene MIR1538 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A9 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM29 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf58 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13H has 0 variants, skipping\n",
      "[INFO]\tGene HACD4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIP has 0 variants, skipping\n",
      "[INFO]\tGene PXDNL has 0 variants, skipping\n",
      "[INFO]\tGene ADHFE1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA46 has 0 variants, skipping\n",
      "[INFO]\tGene THEM5 has 0 variants, skipping\n",
      "[INFO]\tGene ICE1 has 0 variants, skipping\n",
      "[INFO]\tGene PDXDC2P-NPIPB14P has 0 variants, skipping\n",
      "[INFO]\tGene LOC400541 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM268 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371328 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P7 has 0 variants, skipping\n",
      "[INFO]\tGene CLGN has 0 variants, skipping\n",
      "[INFO]\tGene SPAG9 has 0 variants, skipping\n",
      "[INFO]\tGene HACD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506083 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf74 has 0 variants, skipping\n",
      "[INFO]\tGene NHSL1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB10 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM270 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK3 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD111B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD111 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4A has 0 variants, skipping\n",
      "[INFO]\tGene TCF15 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK1 has 0 variants, skipping\n",
      "[INFO]\tGene TBX6 has 0 variants, skipping\n",
      "[INFO]\tGene DAB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC117 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNB has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02136 has 0 variants, skipping\n",
      "[INFO]\tGene CEP120 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371335 has 0 variants, skipping\n",
      "[INFO]\tGene TAT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD71 has 0 variants, skipping\n",
      "[INFO]\tGene IL12B has 0 variants, skipping\n",
      "[INFO]\tGene IL9R has 0 variants, skipping\n",
      "[INFO]\tGene IL7R has 0 variants, skipping\n",
      "[INFO]\tGene IL3RA has 0 variants, skipping\n",
      "[INFO]\tGene ANKFY1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01572 has 0 variants, skipping\n",
      "[INFO]\tGene HCCAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01568 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928035 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5RL has 0 variants, skipping\n",
      "[INFO]\tGene LOC283922 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376772 has 0 variants, skipping\n",
      "[INFO]\tGene DGKZ has 0 variants, skipping\n",
      "[INFO]\tGene ARPC3 has 0 variants, skipping\n",
      "[INFO]\tGene LMLN has 0 variants, skipping\n",
      "[INFO]\tGene UBTF has 0 variants, skipping\n",
      "[INFO]\tGene NSUN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21 has 0 variants, skipping\n",
      "[INFO]\tGene CUBN has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506281 has 0 variants, skipping\n",
      "[INFO]\tGene LEXM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF609 has 0 variants, skipping\n",
      "[INFO]\tGene NRSN2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT34 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ12 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA4 has 0 variants, skipping\n",
      "[INFO]\tGene ILF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM5 has 0 variants, skipping\n",
      "[INFO]\tGene COL20A1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC15 has 0 variants, skipping\n",
      "[INFO]\tGene KATNBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM29 has 0 variants, skipping\n",
      "[INFO]\tGene CCR2 has 0 variants, skipping\n",
      "[INFO]\tGene CASK has 0 variants, skipping\n",
      "[INFO]\tGene CYBRD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4719 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf116 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM15 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D2 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALSL has 0 variants, skipping\n",
      "[INFO]\tGene LTB4R has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2H1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMSN1 has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFTRR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01229 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724084 has 0 variants, skipping\n",
      "[INFO]\tGene CYP7B1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01227 has 0 variants, skipping\n",
      "[INFO]\tGene ARLNC1 has 0 variants, skipping\n",
      "[INFO]\tGene RMST has 0 variants, skipping\n",
      "[INFO]\tGene BBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF1 has 0 variants, skipping\n",
      "[INFO]\tGene LBX1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1B has 0 variants, skipping\n",
      "[INFO]\tGene DEFA1 has 0 variants, skipping\n",
      "[INFO]\tGene ERV3-1-ZNF117 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT9 has 0 variants, skipping\n",
      "[INFO]\tGene AOC4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928985 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF471 has 0 variants, skipping\n",
      "[INFO]\tGene PYGM has 0 variants, skipping\n",
      "[INFO]\tGene MIR4720 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR7854 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6504 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129617 has 0 variants, skipping\n",
      "[INFO]\tGene IFT46 has 0 variants, skipping\n",
      "[INFO]\tGene MARCOL has 0 variants, skipping\n",
      "[INFO]\tGene RFC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8058 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928417 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724163 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GAL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO3A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC654780 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TARBP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02139 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5093 has 0 variants, skipping\n",
      "[INFO]\tGene PDE11A has 0 variants, skipping\n",
      "[INFO]\tGene SAMD15 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1910 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32E has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928557 has 0 variants, skipping\n",
      "[INFO]\tGene HEXIM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6774 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01081 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02135 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00917 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC4 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ30679 has 0 variants, skipping\n",
      "[INFO]\tGene LMF2 has 0 variants, skipping\n",
      "[INFO]\tGene RMND5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02189 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02188 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02181 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928708 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928682 has 0 variants, skipping\n",
      "[INFO]\tGene PELP1 has 0 variants, skipping\n",
      "[INFO]\tGene FCHO2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928659 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928737 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88A has 0 variants, skipping\n",
      "[INFO]\tGene ATF7IP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724467 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400553 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128882 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCB has 0 variants, skipping\n",
      "[INFO]\tGene C1orf167-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI1 has 0 variants, skipping\n",
      "[INFO]\tGene RAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4722 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289580 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339059 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF8 has 0 variants, skipping\n",
      "[INFO]\tGene EYA3 has 0 variants, skipping\n",
      "[INFO]\tGene EFCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L2 has 0 variants, skipping\n",
      "[INFO]\tGene TCN1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGIS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927793 has 0 variants, skipping\n",
      "[INFO]\tGene FANCM has 0 variants, skipping\n",
      "[INFO]\tGene HACL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02138 has 0 variants, skipping\n",
      "[INFO]\tGene CISD3 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf185 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371414 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927817 has 0 variants, skipping\n",
      "[INFO]\tGene CSPP1 has 0 variants, skipping\n",
      "[INFO]\tGene SSU72 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM208 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD68 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT18 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM120A has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP7A has 0 variants, skipping\n",
      "[INFO]\tGene GBF1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS9D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF276 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT76 has 0 variants, skipping\n",
      "[INFO]\tGene TGFBRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene AFG3L1P has 0 variants, skipping\n",
      "[INFO]\tGene RPS4X has 0 variants, skipping\n",
      "[INFO]\tGene RPS3A has 0 variants, skipping\n",
      "[INFO]\tGene URAHP has 0 variants, skipping\n",
      "[INFO]\tGene FAM157C has 0 variants, skipping\n",
      "[INFO]\tGene RTTN has 0 variants, skipping\n",
      "[INFO]\tGene MPI has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P3 has 0 variants, skipping\n",
      "[INFO]\tGene SRPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SRL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC169 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A5 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM4 has 0 variants, skipping\n",
      "[INFO]\tGene GFUS has 0 variants, skipping\n",
      "[INFO]\tGene HPS3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAML1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1B has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B1 has 0 variants, skipping\n",
      "[INFO]\tGene HRH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK6 has 0 variants, skipping\n",
      "[INFO]\tGene PEG10 has 0 variants, skipping\n",
      "[INFO]\tGene ADCYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL2 has 0 variants, skipping\n",
      "[INFO]\tGene CHEK2 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA10 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT16H has 0 variants, skipping\n",
      "[INFO]\tGene ICE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RIOX1 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT12 has 0 variants, skipping\n",
      "[INFO]\tGene MANEA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF322 has 0 variants, skipping\n",
      "[INFO]\tGene YRDC has 0 variants, skipping\n",
      "[INFO]\tGene QTRT2 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP2P1 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD2 has 0 variants, skipping\n",
      "[INFO]\tGene PECAM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723971 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA17 has 0 variants, skipping\n",
      "[INFO]\tGene RP1 has 0 variants, skipping\n",
      "[INFO]\tGene GARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTU2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf49 has 0 variants, skipping\n",
      "[INFO]\tGene BUD31 has 0 variants, skipping\n",
      "[INFO]\tGene MMEL1 has 0 variants, skipping\n",
      "[INFO]\tGene NOSTRIN has 0 variants, skipping\n",
      "[INFO]\tGene UCN3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL53 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP33 has 0 variants, skipping\n",
      "[INFO]\tGene ALPK2 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene FAXDC2 has 0 variants, skipping\n",
      "[INFO]\tGene COG8 has 0 variants, skipping\n",
      "[INFO]\tGene CORIN has 0 variants, skipping\n",
      "[INFO]\tGene UCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02091 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371430 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM244 has 0 variants, skipping\n",
      "[INFO]\tGene EGR2 has 0 variants, skipping\n",
      "[INFO]\tGene GPD2 has 0 variants, skipping\n",
      "[INFO]\tGene DBIL5P has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A9 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927727 has 0 variants, skipping\n",
      "[INFO]\tGene IL4I1 has 0 variants, skipping\n",
      "[INFO]\tGene RAN has 0 variants, skipping\n",
      "[INFO]\tGene BATF2 has 0 variants, skipping\n",
      "[INFO]\tGene MDFI has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3183 has 0 variants, skipping\n",
      "[INFO]\tGene DOK5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF208 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC116 has 0 variants, skipping\n",
      "[INFO]\tGene TP63 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GZMH has 0 variants, skipping\n",
      "[INFO]\tGene NTAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371485 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927839 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD91B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD91A has 0 variants, skipping\n",
      "[INFO]\tGene LOC284009 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF497 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB45 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6776 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371592 has 0 variants, skipping\n",
      "[INFO]\tGene MTA1 has 0 variants, skipping\n",
      "[INFO]\tGene NMI has 0 variants, skipping\n",
      "[INFO]\tGene MTMR4 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927911 has 0 variants, skipping\n",
      "[INFO]\tGene NEK4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288728 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf3 has 0 variants, skipping\n",
      "[INFO]\tGene CPS1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A4P has 0 variants, skipping\n",
      "[INFO]\tGene MYZAP has 0 variants, skipping\n",
      "[INFO]\tGene OPA3 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB1 has 0 variants, skipping\n",
      "[INFO]\tGene OIP5 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX5-TAX1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPZOS has 0 variants, skipping\n",
      "[INFO]\tGene FAM189B has 0 variants, skipping\n",
      "[INFO]\tGene PKMYT1 has 0 variants, skipping\n",
      "[INFO]\tGene OTC has 0 variants, skipping\n",
      "[INFO]\tGene PRND has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1C has 0 variants, skipping\n",
      "[INFO]\tGene PICK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01975 has 0 variants, skipping\n",
      "[INFO]\tGene AMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTC1 has 0 variants, skipping\n",
      "[INFO]\tGene ABL2 has 0 variants, skipping\n",
      "[INFO]\tGene ABL1 has 0 variants, skipping\n",
      "[INFO]\tGene CWH43 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A17 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24C has 0 variants, skipping\n",
      "[INFO]\tGene LGALS4 has 0 variants, skipping\n",
      "[INFO]\tGene IRF6 has 0 variants, skipping\n",
      "[INFO]\tGene GZMA has 0 variants, skipping\n",
      "[INFO]\tGene GPR19 has 0 variants, skipping\n",
      "[INFO]\tGene SFN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01996 has 0 variants, skipping\n",
      "[INFO]\tGene TYMP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101559451 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288069 has 0 variants, skipping\n",
      "[INFO]\tGene CLCC1 has 0 variants, skipping\n",
      "[INFO]\tGene BBS2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K6 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC10 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1B has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1D has 0 variants, skipping\n",
      "[INFO]\tGene RECQL has 0 variants, skipping\n",
      "[INFO]\tGene MEI4 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ2B has 0 variants, skipping\n",
      "[INFO]\tGene MS4A7 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A1 has 0 variants, skipping\n",
      "[INFO]\tGene POFUT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6864 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6865 has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO2 has 0 variants, skipping\n",
      "[INFO]\tGene PTK6 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA1 has 0 variants, skipping\n",
      "[INFO]\tGene NARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130950 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF4 has 0 variants, skipping\n",
      "[INFO]\tGene LHX6 has 0 variants, skipping\n",
      "[INFO]\tGene POPDC3 has 0 variants, skipping\n",
      "[INFO]\tGene NMBR has 0 variants, skipping\n",
      "[INFO]\tGene LOC339166 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC97 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4520-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4520-2 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15P1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP298 has 0 variants, skipping\n",
      "[INFO]\tGene SNX6 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12P2 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK38 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEK has 0 variants, skipping\n",
      "[INFO]\tGene RNASEK-C17orf49 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR497HG has 0 variants, skipping\n",
      "[INFO]\tGene NEFL has 0 variants, skipping\n",
      "[INFO]\tGene KDM3A has 0 variants, skipping\n",
      "[INFO]\tGene BRICD5 has 0 variants, skipping\n",
      "[INFO]\tGene GJE1 has 0 variants, skipping\n",
      "[INFO]\tGene SURF1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT6H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF711 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA6 has 0 variants, skipping\n",
      "[INFO]\tGene PTHLH has 0 variants, skipping\n",
      "[INFO]\tGene PTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA4 has 0 variants, skipping\n",
      "[INFO]\tGene NPHS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTPAL has 0 variants, skipping\n",
      "[INFO]\tGene RANBP10 has 0 variants, skipping\n",
      "[INFO]\tGene REG1CP has 0 variants, skipping\n",
      "[INFO]\tGene SNHG6 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL3 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY2 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM256-PLSCR3 has 0 variants, skipping\n",
      "[INFO]\tGene THOC6 has 0 variants, skipping\n",
      "[INFO]\tGene IRX1 has 0 variants, skipping\n",
      "[INFO]\tGene IRX3 has 0 variants, skipping\n",
      "[INFO]\tGene IRX6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM43 has 0 variants, skipping\n",
      "[INFO]\tGene FSD1 has 0 variants, skipping\n",
      "[INFO]\tGene BRCC3 has 0 variants, skipping\n",
      "[INFO]\tGene SECTM1 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP7 has 0 variants, skipping\n",
      "[INFO]\tGene SENP3-EIF4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA48 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA67 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L2 has 0 variants, skipping\n",
      "[INFO]\tGene CA2 has 0 variants, skipping\n",
      "[INFO]\tGene GBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA16 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PATZ1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL2 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL29P2 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC3L has 0 variants, skipping\n",
      "[INFO]\tGene H2AB2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTBL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB20 has 0 variants, skipping\n",
      "[INFO]\tGene BPNT2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2R2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM260 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF1 has 0 variants, skipping\n",
      "[INFO]\tGene OSGEP has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA21 has 0 variants, skipping\n",
      "[INFO]\tGene VPS54 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A3 has 0 variants, skipping\n",
      "[INFO]\tGene VIP has 0 variants, skipping\n",
      "[INFO]\tGene VIM has 0 variants, skipping\n",
      "[INFO]\tGene VGF has 0 variants, skipping\n",
      "[INFO]\tGene VEGFB has 0 variants, skipping\n",
      "[INFO]\tGene VDAC1 has 0 variants, skipping\n",
      "[INFO]\tGene VASP has 0 variants, skipping\n",
      "[INFO]\tGene UVRAG has 0 variants, skipping\n",
      "[INFO]\tGene USP1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4314 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6883 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4521 has 0 variants, skipping\n",
      "[INFO]\tGene HS1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene XPO4 has 0 variants, skipping\n",
      "[INFO]\tGene CASR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128288 has 0 variants, skipping\n",
      "[INFO]\tGene GZMB has 0 variants, skipping\n",
      "[INFO]\tGene GYG1 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF16 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN11 has 0 variants, skipping\n",
      "[INFO]\tGene RCSD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR150 has 0 variants, skipping\n",
      "[INFO]\tGene MIR149 has 0 variants, skipping\n",
      "[INFO]\tGene MIR146A has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR138-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR191 has 0 variants, skipping\n",
      "[INFO]\tGene MIR153-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR152 has 0 variants, skipping\n",
      "[INFO]\tGene MIR145 has 0 variants, skipping\n",
      "[INFO]\tGene MIR144 has 0 variants, skipping\n",
      "[INFO]\tGene MIR143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928266 has 0 variants, skipping\n",
      "[INFO]\tGene YBX3 has 0 variants, skipping\n",
      "[INFO]\tGene CST7 has 0 variants, skipping\n",
      "[INFO]\tGene DGKD has 0 variants, skipping\n",
      "[INFO]\tGene DGKE has 0 variants, skipping\n",
      "[INFO]\tGene LRRC18 has 0 variants, skipping\n",
      "[INFO]\tGene OFD1 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1R1 has 0 variants, skipping\n",
      "[INFO]\tGene UBN1 has 0 variants, skipping\n",
      "[INFO]\tGene STAG1 has 0 variants, skipping\n",
      "[INFO]\tGene EFS has 0 variants, skipping\n",
      "[INFO]\tGene PRSS16 has 0 variants, skipping\n",
      "[INFO]\tGene BOP1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC6B has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC9 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D12 has 0 variants, skipping\n",
      "[INFO]\tGene MAGOH2P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM220-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42SE2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL1 has 0 variants, skipping\n",
      "[INFO]\tGene DTWD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRM has 0 variants, skipping\n",
      "[INFO]\tGene PSMG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00670 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128006 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1269B has 0 variants, skipping\n",
      "[INFO]\tGene TROAP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02093 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H3 has 0 variants, skipping\n",
      "[INFO]\tGene COX10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGC12916 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928475 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT7 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT8 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4731 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM256 has 0 variants, skipping\n",
      "[INFO]\tGene VXN has 0 variants, skipping\n",
      "[INFO]\tGene AKNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA10IP has 0 variants, skipping\n",
      "[INFO]\tGene CXorf58 has 0 variants, skipping\n",
      "[INFO]\tGene SNX32 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC43 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02087 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD163 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF165 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD49B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD49A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene USP32P1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D27P has 0 variants, skipping\n",
      "[INFO]\tGene SHLD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284191 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02090 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG11A has 0 variants, skipping\n",
      "[INFO]\tGene DEFB103A has 0 variants, skipping\n",
      "[INFO]\tGene C11orf24 has 0 variants, skipping\n",
      "[INFO]\tGene CCL22 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL18 has 0 variants, skipping\n",
      "[INFO]\tGene CCL17 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC4 has 0 variants, skipping\n",
      "[INFO]\tGene OR6A2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6D has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A12 has 0 variants, skipping\n",
      "[INFO]\tGene SMCR2 has 0 variants, skipping\n",
      "[INFO]\tGene RAI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6777 has 0 variants, skipping\n",
      "[INFO]\tGene DNTT has 0 variants, skipping\n",
      "[INFO]\tGene PRAC2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF32 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF121 has 0 variants, skipping\n",
      "[INFO]\tGene REPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene BZW2 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL7 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02076 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P5 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P1 has 0 variants, skipping\n",
      "[INFO]\tGene USP32P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144B has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPA has 0 variants, skipping\n",
      "[INFO]\tGene CDH7 has 0 variants, skipping\n",
      "[INFO]\tGene CDC34 has 0 variants, skipping\n",
      "[INFO]\tGene CD81 has 0 variants, skipping\n",
      "[INFO]\tGene CD74 has 0 variants, skipping\n",
      "[INFO]\tGene CCNG2 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX3 has 0 variants, skipping\n",
      "[INFO]\tGene CASP3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMP has 0 variants, skipping\n",
      "[INFO]\tGene CETN2 has 0 variants, skipping\n",
      "[INFO]\tGene CALR has 0 variants, skipping\n",
      "[INFO]\tGene TDO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3B-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3B-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC79999 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388436 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3C has 0 variants, skipping\n",
      "[INFO]\tGene EPN2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PARK7 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX2B has 0 variants, skipping\n",
      "[INFO]\tGene FSTL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L6P has 0 variants, skipping\n",
      "[INFO]\tGene SIRT6 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF6 has 0 variants, skipping\n",
      "[INFO]\tGene MGME1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144CP has 0 variants, skipping\n",
      "[INFO]\tGene USP32P3 has 0 variants, skipping\n",
      "[INFO]\tGene PHGR1 has 0 variants, skipping\n",
      "[INFO]\tGene NOS2P3 has 0 variants, skipping\n",
      "[INFO]\tGene ARL4A has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02088 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287072 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01563 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGB has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G5C has 0 variants, skipping\n",
      "[INFO]\tGene UBBP4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E5 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ36000 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371703 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4522 has 0 variants, skipping\n",
      "[INFO]\tGene NRTN has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P5 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28B has 0 variants, skipping\n",
      "[INFO]\tGene LEKR1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01992 has 0 variants, skipping\n",
      "[INFO]\tGene PYY2 has 0 variants, skipping\n",
      "[INFO]\tGene PPY2P has 0 variants, skipping\n",
      "[INFO]\tGene KRT18P55 has 0 variants, skipping\n",
      "[INFO]\tGene OR6Q1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4723 has 0 variants, skipping\n",
      "[INFO]\tGene SGK2 has 0 variants, skipping\n",
      "[INFO]\tGene COL12A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL45 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4X1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI44L has 0 variants, skipping\n",
      "[INFO]\tGene MMAA has 0 variants, skipping\n",
      "[INFO]\tGene MEF2D has 0 variants, skipping\n",
      "[INFO]\tGene TGM3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC4 has 0 variants, skipping\n",
      "[INFO]\tGene SYNC has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3 has 0 variants, skipping\n",
      "[INFO]\tGene MC1R has 0 variants, skipping\n",
      "[INFO]\tGene MATN3 has 0 variants, skipping\n",
      "[INFO]\tGene MATN1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1A has 0 variants, skipping\n",
      "[INFO]\tGene MAN2A1 has 0 variants, skipping\n",
      "[INFO]\tGene HADH has 0 variants, skipping\n",
      "[INFO]\tGene GYG2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD42B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD42A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD4B has 0 variants, skipping\n",
      "[INFO]\tGene ALPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR451B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4732 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927018 has 0 variants, skipping\n",
      "[INFO]\tGene E2F5 has 0 variants, skipping\n",
      "[INFO]\tGene E2F4 has 0 variants, skipping\n",
      "[INFO]\tGene GATA1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F29 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B21 has 0 variants, skipping\n",
      "[INFO]\tGene CDHR4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4523 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOA has 0 variants, skipping\n",
      "[INFO]\tGene AGL has 0 variants, skipping\n",
      "[INFO]\tGene TSPY1 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF2P1-LRRC37BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1P2 has 0 variants, skipping\n",
      "[INFO]\tGene SUZ12P1 has 0 variants, skipping\n",
      "[INFO]\tGene RRM1 has 0 variants, skipping\n",
      "[INFO]\tGene DPRXP4 has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984974 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646030 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4733 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC166 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10B has 0 variants, skipping\n",
      "[INFO]\tGene FMNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4724 has 0 variants, skipping\n",
      "[INFO]\tGene SYDE1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371730 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf67 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC8 has 0 variants, skipping\n",
      "[INFO]\tGene ARGFXP2 has 0 variants, skipping\n",
      "[INFO]\tGene MYL6B has 0 variants, skipping\n",
      "[INFO]\tGene MYH11 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS11 has 0 variants, skipping\n",
      "[INFO]\tGene DNLZ has 0 variants, skipping\n",
      "[INFO]\tGene TTC28 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC3 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR8 has 0 variants, skipping\n",
      "[INFO]\tGene PHGDH has 0 variants, skipping\n",
      "[INFO]\tGene PLK2 has 0 variants, skipping\n",
      "[INFO]\tGene AA06 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01989 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF668 has 0 variants, skipping\n",
      "[INFO]\tGene AGRN has 0 variants, skipping\n",
      "[INFO]\tGene SAMD12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51L3-RFFL has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985033 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD7 has 0 variants, skipping\n",
      "[INFO]\tGene CPA5 has 0 variants, skipping\n",
      "[INFO]\tGene RNF41 has 0 variants, skipping\n",
      "[INFO]\tGene MPRIP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A8P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF534 has 0 variants, skipping\n",
      "[INFO]\tGene CCL15-CCL14 has 0 variants, skipping\n",
      "[INFO]\tGene PBX2 has 0 variants, skipping\n",
      "[INFO]\tGene PBX1 has 0 variants, skipping\n",
      "[INFO]\tGene OCLN has 0 variants, skipping\n",
      "[INFO]\tGene CCL4L1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC4 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM47 has 0 variants, skipping\n",
      "[INFO]\tGene CERS6 has 0 variants, skipping\n",
      "[INFO]\tGene RPS5 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6 has 0 variants, skipping\n",
      "[INFO]\tGene NPY2R has 0 variants, skipping\n",
      "[INFO]\tGene RRP9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A6 has 0 variants, skipping\n",
      "[INFO]\tGene KRT75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR943 has 0 variants, skipping\n",
      "[INFO]\tGene MIR940 has 0 variants, skipping\n",
      "[INFO]\tGene LHX1-DT has 0 variants, skipping\n",
      "[INFO]\tGene STK24 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3C has 0 variants, skipping\n",
      "[INFO]\tGene FAT4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S1 has 0 variants, skipping\n",
      "[INFO]\tGene PTK2B has 0 variants, skipping\n",
      "[INFO]\tGene EFNB2 has 0 variants, skipping\n",
      "[INFO]\tGene ARGLU1 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT61B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA90 has 0 variants, skipping\n",
      "[INFO]\tGene EPM2A has 0 variants, skipping\n",
      "[INFO]\tGene RNF103 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA9 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL33 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA7 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3B has 0 variants, skipping\n",
      "[INFO]\tGene SPAG7 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL14 has 0 variants, skipping\n",
      "[INFO]\tGene APBA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378J has 0 variants, skipping\n",
      "[INFO]\tGene YWHAEP7 has 0 variants, skipping\n",
      "[INFO]\tGene SPAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF284 has 0 variants, skipping\n",
      "[INFO]\tGene ANLN has 0 variants, skipping\n",
      "[INFO]\tGene CBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5J has 0 variants, skipping\n",
      "[INFO]\tGene STAT5A has 0 variants, skipping\n",
      "[INFO]\tGene SRPRA has 0 variants, skipping\n",
      "[INFO]\tGene OPN5 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371763 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4734 has 0 variants, skipping\n",
      "[INFO]\tGene WDR75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4726 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287808 has 0 variants, skipping\n",
      "[INFO]\tGene DTD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4727 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA21B has 0 variants, skipping\n",
      "[INFO]\tGene GRM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6779 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00672 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02079 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A11P has 0 variants, skipping\n",
      "[INFO]\tGene RDM1P5 has 0 variants, skipping\n",
      "[INFO]\tGene MTX1 has 0 variants, skipping\n",
      "[INFO]\tGene STAT5B has 0 variants, skipping\n",
      "[INFO]\tGene CORO6 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A5 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3E has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L2 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT11 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC61 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4728 has 0 variants, skipping\n",
      "[INFO]\tGene ST13 has 0 variants, skipping\n",
      "[INFO]\tGene SKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL39 has 0 variants, skipping\n",
      "[INFO]\tGene H2BS1 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CROT has 0 variants, skipping\n",
      "[INFO]\tGene GPR142 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K4 has 0 variants, skipping\n",
      "[INFO]\tGene AMBP has 0 variants, skipping\n",
      "[INFO]\tGene ALPP has 0 variants, skipping\n",
      "[INFO]\tGene HERC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6884 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD124 has 0 variants, skipping\n",
      "[INFO]\tGene VCAN has 0 variants, skipping\n",
      "[INFO]\tGene LMF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6866 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF716 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6867 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE1 has 0 variants, skipping\n",
      "[INFO]\tGene RARA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RDX has 0 variants, skipping\n",
      "[INFO]\tGene CTCF has 0 variants, skipping\n",
      "[INFO]\tGene KLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHUK has 0 variants, skipping\n",
      "[INFO]\tGene PRKCI has 0 variants, skipping\n",
      "[INFO]\tGene HSPA4 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505782 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6510 has 0 variants, skipping\n",
      "[INFO]\tGene UPF3B has 0 variants, skipping\n",
      "[INFO]\tGene KRT42P has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A14 has 0 variants, skipping\n",
      "[INFO]\tGene IMMP2L has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C3B has 0 variants, skipping\n",
      "[INFO]\tGene SPNS3 has 0 variants, skipping\n",
      "[INFO]\tGene ANK1 has 0 variants, skipping\n",
      "[INFO]\tGene YIF1A has 0 variants, skipping\n",
      "[INFO]\tGene ABO has 0 variants, skipping\n",
      "[INFO]\tGene SMIM7 has 0 variants, skipping\n",
      "[INFO]\tGene GPR146 has 0 variants, skipping\n",
      "[INFO]\tGene MAP11 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL2 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML3 has 0 variants, skipping\n",
      "[INFO]\tGene CDR2L has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R4 has 0 variants, skipping\n",
      "[INFO]\tGene MTRR has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5010 has 0 variants, skipping\n",
      "[INFO]\tGene EVI5L has 0 variants, skipping\n",
      "[INFO]\tGene MRAP has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AD has 0 variants, skipping\n",
      "[INFO]\tGene IL10RA has 0 variants, skipping\n",
      "[INFO]\tGene ADCY9 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY8 has 0 variants, skipping\n",
      "[INFO]\tGene GPHA2 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3C has 0 variants, skipping\n",
      "[INFO]\tGene ACOT12 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5K has 0 variants, skipping\n",
      "[INFO]\tGene MADCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6780A has 0 variants, skipping\n",
      "[INFO]\tGene RAMP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IDI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6781 has 0 variants, skipping\n",
      "[INFO]\tGene MPND has 0 variants, skipping\n",
      "[INFO]\tGene MIR1261 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00671 has 0 variants, skipping\n",
      "[INFO]\tGene DDX41 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3L-AARSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNTN has 0 variants, skipping\n",
      "[INFO]\tGene NOMO1 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG2 has 0 variants, skipping\n",
      "[INFO]\tGene CRY1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929767 has 0 variants, skipping\n",
      "[INFO]\tGene DTX4 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA4 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC14 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC7 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC11 has 0 variants, skipping\n",
      "[INFO]\tGene EPG5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF146 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC200 has 0 variants, skipping\n",
      "[INFO]\tGene RNU2-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00910 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378A has 0 variants, skipping\n",
      "[INFO]\tGene MIR2117 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO4 has 0 variants, skipping\n",
      "[INFO]\tGene METRNL has 0 variants, skipping\n",
      "[INFO]\tGene FAM76B has 0 variants, skipping\n",
      "[INFO]\tGene C10orf82 has 0 variants, skipping\n",
      "[INFO]\tGene TCP10L has 0 variants, skipping\n",
      "[INFO]\tGene SAMD8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00309 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5D has 0 variants, skipping\n",
      "[INFO]\tGene WFDC12 has 0 variants, skipping\n",
      "[INFO]\tGene RDH14 has 0 variants, skipping\n",
      "[INFO]\tGene HAL has 0 variants, skipping\n",
      "[INFO]\tGene BRSK1 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8 has 0 variants, skipping\n",
      "[INFO]\tGene TOM1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF85 has 0 variants, skipping\n",
      "[INFO]\tGene FAM215A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01976 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf64 has 0 variants, skipping\n",
      "[INFO]\tGene LACRT has 0 variants, skipping\n",
      "[INFO]\tGene ANKS4B has 0 variants, skipping\n",
      "[INFO]\tGene ASB3 has 0 variants, skipping\n",
      "[INFO]\tGene EDARADD has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371789 has 0 variants, skipping\n",
      "[INFO]\tGene ZFX has 0 variants, skipping\n",
      "[INFO]\tGene RPIA has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM217 has 0 variants, skipping\n",
      "[INFO]\tGene WDCP has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6782 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAF11 has 0 variants, skipping\n",
      "[INFO]\tGene EBP has 0 variants, skipping\n",
      "[INFO]\tGene GNB5 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT2 has 0 variants, skipping\n",
      "[INFO]\tGene AVIL has 0 variants, skipping\n",
      "[INFO]\tGene NFRKB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01180 has 0 variants, skipping\n",
      "[INFO]\tGene COASY has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4C has 0 variants, skipping\n",
      "[INFO]\tGene SINHCAF has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD1 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP2 has 0 variants, skipping\n",
      "[INFO]\tGene NFKB1 has 0 variants, skipping\n",
      "[INFO]\tGene EMX2 has 0 variants, skipping\n",
      "[INFO]\tGene FCGRT has 0 variants, skipping\n",
      "[INFO]\tGene DOT1L has 0 variants, skipping\n",
      "[INFO]\tGene MRPL36 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6784 has 0 variants, skipping\n",
      "[INFO]\tGene DPYD has 0 variants, skipping\n",
      "[INFO]\tGene NACC1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26B has 0 variants, skipping\n",
      "[INFO]\tGene STX1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371795 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339192 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4315-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4315-2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A4P has 0 variants, skipping\n",
      "[INFO]\tGene ARL17A has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02210 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02210-CRHR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLTP has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene AQP2 has 0 variants, skipping\n",
      "[INFO]\tGene APRT has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF236 has 0 variants, skipping\n",
      "[INFO]\tGene NSFP1 has 0 variants, skipping\n",
      "[INFO]\tGene WRAP53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF407 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf91 has 0 variants, skipping\n",
      "[INFO]\tGene ASB15 has 0 variants, skipping\n",
      "[INFO]\tGene DCBLD2 has 0 variants, skipping\n",
      "[INFO]\tGene SETSIP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5089 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B4 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC65 has 0 variants, skipping\n",
      "[INFO]\tGene THCAT158 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAE has 0 variants, skipping\n",
      "[INFO]\tGene MRPL45P2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2U1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724532 has 0 variants, skipping\n",
      "[INFO]\tGene SP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA20 has 0 variants, skipping\n",
      "[INFO]\tGene PEX12 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927166 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG7 has 0 variants, skipping\n",
      "[INFO]\tGene NLE1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3185 has 0 variants, skipping\n",
      "[INFO]\tGene EXT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371814 has 0 variants, skipping\n",
      "[INFO]\tGene AP1G1 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ40194 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6129 has 0 variants, skipping\n",
      "[INFO]\tGene ICOSLG has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724596 has 0 variants, skipping\n",
      "[INFO]\tGene BCL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02075 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288866 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6165 has 0 variants, skipping\n",
      "[INFO]\tGene RIC1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC11A has 0 variants, skipping\n",
      "[INFO]\tGene PICART1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3 has 0 variants, skipping\n",
      "[INFO]\tGene POLD3 has 0 variants, skipping\n",
      "[INFO]\tGene MTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01969 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM92-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND1B has 0 variants, skipping\n",
      "[INFO]\tGene FOSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20E has 0 variants, skipping\n",
      "[INFO]\tGene PRR20D has 0 variants, skipping\n",
      "[INFO]\tGene FN3KRP has 0 variants, skipping\n",
      "[INFO]\tGene NUP43 has 0 variants, skipping\n",
      "[INFO]\tGene TTC9C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371824 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM19 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY2 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf49 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP3 has 0 variants, skipping\n",
      "[INFO]\tGene SOD2 has 0 variants, skipping\n",
      "[INFO]\tGene DZANK1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX2 has 0 variants, skipping\n",
      "[INFO]\tGene NPY4R2 has 0 variants, skipping\n",
      "[INFO]\tGene LTK has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB4 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB3 has 0 variants, skipping\n",
      "[INFO]\tGene MAL has 0 variants, skipping\n",
      "[INFO]\tGene MAGOH has 0 variants, skipping\n",
      "[INFO]\tGene IARS2 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf116 has 0 variants, skipping\n",
      "[INFO]\tGene RAVER1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL16 has 0 variants, skipping\n",
      "[INFO]\tGene SCN2B has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA3 has 0 variants, skipping\n",
      "[INFO]\tGene PTN has 0 variants, skipping\n",
      "[INFO]\tGene FBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS8 has 0 variants, skipping\n",
      "[INFO]\tGene CRISP3 has 0 variants, skipping\n",
      "[INFO]\tGene LAMC3 has 0 variants, skipping\n",
      "[INFO]\tGene RPS27L has 0 variants, skipping\n",
      "[INFO]\tGene DPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1I1 has 0 variants, skipping\n",
      "[INFO]\tGene TRA2A has 0 variants, skipping\n",
      "[INFO]\tGene ENOPH1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8059 has 0 variants, skipping\n",
      "[INFO]\tGene TOB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53INP2 has 0 variants, skipping\n",
      "[INFO]\tGene RHCE has 0 variants, skipping\n",
      "[INFO]\tGene ARMC3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02071 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02072 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02073 has 0 variants, skipping\n",
      "[INFO]\tGene GJD4 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01982 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02089 has 0 variants, skipping\n",
      "[INFO]\tGene LRATD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM167A has 0 variants, skipping\n",
      "[INFO]\tGene TCTE3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO36 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC50 has 0 variants, skipping\n",
      "[INFO]\tGene LYG1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R2 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98C has 0 variants, skipping\n",
      "[INFO]\tGene RNF151 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD3 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LB has 0 variants, skipping\n",
      "[INFO]\tGene CLBA1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9C has 0 variants, skipping\n",
      "[INFO]\tGene ZFP42 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MTVR2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF126P1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1B has 0 variants, skipping\n",
      "[INFO]\tGene BCAS2 has 0 variants, skipping\n",
      "[INFO]\tGene SMNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927557 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927539 has 0 variants, skipping\n",
      "[INFO]\tGene GGT6 has 0 variants, skipping\n",
      "[INFO]\tGene MECP2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927666 has 0 variants, skipping\n",
      "[INFO]\tGene MSX2P1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT14 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ11 has 0 variants, skipping\n",
      "[INFO]\tGene TSPOAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4736 has 0 variants, skipping\n",
      "[INFO]\tGene DES has 0 variants, skipping\n",
      "[INFO]\tGene CENPF has 0 variants, skipping\n",
      "[INFO]\tGene MAGEL2 has 0 variants, skipping\n",
      "[INFO]\tGene FMN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3680-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-1 has 0 variants, skipping\n",
      "[INFO]\tGene PBX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D3 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A3 has 0 variants, skipping\n",
      "[INFO]\tGene ABAT has 0 variants, skipping\n",
      "[INFO]\tGene PPM1E has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A5 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK7 has 0 variants, skipping\n",
      "[INFO]\tGene NEMF has 0 variants, skipping\n",
      "[INFO]\tGene STOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH5 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4729 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01476 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF574 has 0 variants, skipping\n",
      "[INFO]\tGene H3-3A has 0 variants, skipping\n",
      "[INFO]\tGene TRIR has 0 variants, skipping\n",
      "[INFO]\tGene KAT7 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P1-DHX40P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNFT1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4737 has 0 variants, skipping\n",
      "[INFO]\tGene ORM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371849 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653653 has 0 variants, skipping\n",
      "[INFO]\tGene CASP4LP has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA20 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01999 has 0 variants, skipping\n",
      "[INFO]\tGene LRATD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM209 has 0 variants, skipping\n",
      "[INFO]\tGene MIR216A has 0 variants, skipping\n",
      "[INFO]\tGene MIR217 has 0 variants, skipping\n",
      "[INFO]\tGene MIR215 has 0 variants, skipping\n",
      "[INFO]\tGene MIR214 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927855 has 0 variants, skipping\n",
      "[INFO]\tGene TBX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P2 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF676 has 0 variants, skipping\n",
      "[INFO]\tGene ATF6B has 0 variants, skipping\n",
      "[INFO]\tGene TBL1X has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A4 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK1 has 0 variants, skipping\n",
      "[INFO]\tGene PKIB has 0 variants, skipping\n",
      "[INFO]\tGene UHRF2 has 0 variants, skipping\n",
      "[INFO]\tGene PRLHR has 0 variants, skipping\n",
      "[INFO]\tGene GTF2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729683 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A31 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC70 has 0 variants, skipping\n",
      "[INFO]\tGene MXD3 has 0 variants, skipping\n",
      "[INFO]\tGene INO80B has 0 variants, skipping\n",
      "[INFO]\tGene SF3B5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL3 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L1 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS15 has 0 variants, skipping\n",
      "[INFO]\tGene RNF17 has 0 variants, skipping\n",
      "[INFO]\tGene TCAM1P has 0 variants, skipping\n",
      "[INFO]\tGene SUOX has 0 variants, skipping\n",
      "[INFO]\tGene TAC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR29-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL11 has 0 variants, skipping\n",
      "[INFO]\tGene SEC16A has 0 variants, skipping\n",
      "[INFO]\tGene SERTM1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD104 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50C has 0 variants, skipping\n",
      "[INFO]\tGene PPDPF has 0 variants, skipping\n",
      "[INFO]\tGene TNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene TGFA has 0 variants, skipping\n",
      "[INFO]\tGene PCNX1 has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5047 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6080 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM1P1 has 0 variants, skipping\n",
      "[INFO]\tGene AMZ2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507002 has 0 variants, skipping\n",
      "[INFO]\tGene TP53I13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02563 has 0 variants, skipping\n",
      "[INFO]\tGene NALCN has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA7 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF6 has 0 variants, skipping\n",
      "[INFO]\tGene CCL19 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC7 has 0 variants, skipping\n",
      "[INFO]\tGene CCL7 has 0 variants, skipping\n",
      "[INFO]\tGene S100B has 0 variants, skipping\n",
      "[INFO]\tGene S100A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928021 has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AA2 has 0 variants, skipping\n",
      "[INFO]\tGene GORASP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA38B has 0 variants, skipping\n",
      "[INFO]\tGene PSG6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00674 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378948 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSDE1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL5 has 0 variants, skipping\n",
      "[INFO]\tGene SCYL1 has 0 variants, skipping\n",
      "[INFO]\tGene CEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO3B has 0 variants, skipping\n",
      "[INFO]\tGene LY6E has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2A has 0 variants, skipping\n",
      "[INFO]\tGene UGT3A1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT3A2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD60 has 0 variants, skipping\n",
      "[INFO]\tGene MIR635 has 0 variants, skipping\n",
      "[INFO]\tGene FEV has 0 variants, skipping\n",
      "[INFO]\tGene LINC01482 has 0 variants, skipping\n",
      "[INFO]\tGene SLA has 0 variants, skipping\n",
      "[INFO]\tGene MON1A has 0 variants, skipping\n",
      "[INFO]\tGene MAN2C1 has 0 variants, skipping\n",
      "[INFO]\tGene HEXA has 0 variants, skipping\n",
      "[INFO]\tGene ABCA9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4524A has 0 variants, skipping\n",
      "[INFO]\tGene PRO1804 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM131L has 0 variants, skipping\n",
      "[INFO]\tGene KCTD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371880 has 0 variants, skipping\n",
      "[INFO]\tGene SCHLAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01497 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01028 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASC17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01152 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723517 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4530 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02097 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02003 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01128 has 0 variants, skipping\n",
      "[INFO]\tGene HOTAIR has 0 variants, skipping\n",
      "[INFO]\tGene RGPD3 has 0 variants, skipping\n",
      "[INFO]\tGene CARNS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00469 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02092 has 0 variants, skipping\n",
      "[INFO]\tGene MYH14 has 0 variants, skipping\n",
      "[INFO]\tGene MGC16275 has 0 variants, skipping\n",
      "[INFO]\tGene PERM1 has 0 variants, skipping\n",
      "[INFO]\tGene LYST has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130520 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928343 has 0 variants, skipping\n",
      "[INFO]\tGene TBCEL has 0 variants, skipping\n",
      "[INFO]\tGene MSX2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3615 has 0 variants, skipping\n",
      "[INFO]\tGene GRXCR2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM163B has 0 variants, skipping\n",
      "[INFO]\tGene SCX has 0 variants, skipping\n",
      "[INFO]\tGene CPN2 has 0 variants, skipping\n",
      "[INFO]\tGene BICC1 has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1C has 0 variants, skipping\n",
      "[INFO]\tGene NPY1R has 0 variants, skipping\n",
      "[INFO]\tGene HID1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL5 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP7 has 0 variants, skipping\n",
      "[INFO]\tGene QRSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM8 has 0 variants, skipping\n",
      "[INFO]\tGene GRAP has 0 variants, skipping\n",
      "[INFO]\tGene KLHL26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287042 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A4 has 0 variants, skipping\n",
      "[INFO]\tGene LCLAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP214 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6785 has 0 variants, skipping\n",
      "[INFO]\tGene AK8 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3D has 0 variants, skipping\n",
      "[INFO]\tGene NBPF7 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene LFNG has 0 variants, skipping\n",
      "[INFO]\tGene LAMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LECT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4738 has 0 variants, skipping\n",
      "[INFO]\tGene MCAT has 0 variants, skipping\n",
      "[INFO]\tGene TEN1-CDK3 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC8 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA5 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1B2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF157-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF587B has 0 variants, skipping\n",
      "[INFO]\tGene MS4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF12 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370579 has 0 variants, skipping\n",
      "[INFO]\tGene BRPF1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105274304 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4463 has 0 variants, skipping\n",
      "[INFO]\tGene MIR636 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1A has 0 variants, skipping\n",
      "[INFO]\tGene CERS3 has 0 variants, skipping\n",
      "[INFO]\tGene CHAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00868 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371899 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6516 has 0 variants, skipping\n",
      "[INFO]\tGene RNF216 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400622 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01987 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01973 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B8 has 0 variants, skipping\n",
      "[INFO]\tGene AFMID has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01993 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928674 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA21 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN20 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A27 has 0 variants, skipping\n",
      "[INFO]\tGene CHURC1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD3-BMI1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02078 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01977 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01979 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01978 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF807P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1268B has 0 variants, skipping\n",
      "[INFO]\tGene PRY2 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF1 has 0 variants, skipping\n",
      "[INFO]\tGene DTNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGT1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4730 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3E has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928855 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400627 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1250 has 0 variants, skipping\n",
      "[INFO]\tGene GK has 0 variants, skipping\n",
      "[INFO]\tGene CAND2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF155 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371925 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG16-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00482 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN26 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4740 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3186 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2K has 0 variants, skipping\n",
      "[INFO]\tGene OR10K1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10R2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR95 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370295 has 0 variants, skipping\n",
      "[INFO]\tGene EBI3 has 0 variants, skipping\n",
      "[INFO]\tGene G3BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SBSN has 0 variants, skipping\n",
      "[INFO]\tGene RGS9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6786 has 0 variants, skipping\n",
      "[INFO]\tGene D2HGDH has 0 variants, skipping\n",
      "[INFO]\tGene TMEM168 has 0 variants, skipping\n",
      "[INFO]\tGene NOP14 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN1 has 0 variants, skipping\n",
      "[INFO]\tGene TOR2A has 0 variants, skipping\n",
      "[INFO]\tGene KLHL6 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA6 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF8 has 0 variants, skipping\n",
      "[INFO]\tGene VHL has 0 variants, skipping\n",
      "[INFO]\tGene AFP has 0 variants, skipping\n",
      "[INFO]\tGene SNORD134 has 0 variants, skipping\n",
      "[INFO]\tGene GLP1R has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene TRH has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6787 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01970 has 0 variants, skipping\n",
      "[INFO]\tGene PEX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TFDP1 has 0 variants, skipping\n",
      "[INFO]\tGene TEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4525 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929552 has 0 variants, skipping\n",
      "[INFO]\tGene PSG9 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP87 has 0 variants, skipping\n",
      "[INFO]\tGene WNT4 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR5 has 0 variants, skipping\n",
      "[INFO]\tGene MAEA has 0 variants, skipping\n",
      "[INFO]\tGene GCH1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT6 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX1-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730100 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SVIP has 0 variants, skipping\n",
      "[INFO]\tGene MESP1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS33B has 0 variants, skipping\n",
      "[INFO]\tGene SMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene SAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFE3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC39 has 0 variants, skipping\n",
      "[INFO]\tGene ARL16 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1L5 has 0 variants, skipping\n",
      "[INFO]\tGene WAC has 0 variants, skipping\n",
      "[INFO]\tGene DNAH11 has 0 variants, skipping\n",
      "[INFO]\tGene OR6N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C76 has 0 variants, skipping\n",
      "[INFO]\tGene RPRD1A has 0 variants, skipping\n",
      "[INFO]\tGene CUL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCCA-DT has 0 variants, skipping\n",
      "[INFO]\tGene EREG has 0 variants, skipping\n",
      "[INFO]\tGene MSX1 has 0 variants, skipping\n",
      "[INFO]\tGene NMUR1 has 0 variants, skipping\n",
      "[INFO]\tGene MTF1 has 0 variants, skipping\n",
      "[INFO]\tGene MT3 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM22P has 0 variants, skipping\n",
      "[INFO]\tGene LTV1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD13 has 0 variants, skipping\n",
      "[INFO]\tGene MAEL has 0 variants, skipping\n",
      "[INFO]\tGene LACTB has 0 variants, skipping\n",
      "[INFO]\tGene WDR73 has 0 variants, skipping\n",
      "[INFO]\tGene WDR41 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP62 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF268 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf56 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH4A1 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR3 has 0 variants, skipping\n",
      "[INFO]\tGene SULT2A1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP1 has 0 variants, skipping\n",
      "[INFO]\tGene STX5 has 0 variants, skipping\n",
      "[INFO]\tGene STRN has 0 variants, skipping\n",
      "[INFO]\tGene UBOX5 has 0 variants, skipping\n",
      "[INFO]\tGene COL2A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM186 has 0 variants, skipping\n",
      "[INFO]\tGene MXRA5 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF13 has 0 variants, skipping\n",
      "[INFO]\tGene SPEF1 has 0 variants, skipping\n",
      "[INFO]\tGene MPC2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA5 has 0 variants, skipping\n",
      "[INFO]\tGene ETDC has 0 variants, skipping\n",
      "[INFO]\tGene LRRC27 has 0 variants, skipping\n",
      "[INFO]\tGene FBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene ZGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene NCK1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf42 has 0 variants, skipping\n",
      "[INFO]\tGene TSTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A3 has 0 variants, skipping\n",
      "[INFO]\tGene PHB has 0 variants, skipping\n",
      "[INFO]\tGene PGF has 0 variants, skipping\n",
      "[INFO]\tGene PGD has 0 variants, skipping\n",
      "[INFO]\tGene PGAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PFN2 has 0 variants, skipping\n",
      "[INFO]\tGene TUFM has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985728 has 0 variants, skipping\n",
      "[INFO]\tGene EOGT has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E3 has 0 variants, skipping\n",
      "[INFO]\tGene CAAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2A has 0 variants, skipping\n",
      "[INFO]\tGene RSRC2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L9P has 0 variants, skipping\n",
      "[INFO]\tGene RNF43 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPC has 0 variants, skipping\n",
      "[INFO]\tGene PRELP has 0 variants, skipping\n",
      "[INFO]\tGene HMGN5 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE41 has 0 variants, skipping\n",
      "[INFO]\tGene SKIL has 0 variants, skipping\n",
      "[INFO]\tGene LOC112577516 has 0 variants, skipping\n",
      "[INFO]\tGene RYR2 has 0 variants, skipping\n",
      "[INFO]\tGene STX10 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L4 has 0 variants, skipping\n",
      "[INFO]\tGene NLRX1 has 0 variants, skipping\n",
      "[INFO]\tGene GYPE has 0 variants, skipping\n",
      "[INFO]\tGene ALG14 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRC1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN7 has 0 variants, skipping\n",
      "[INFO]\tGene CNMD has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R32 has 0 variants, skipping\n",
      "[INFO]\tGene CDH10 has 0 variants, skipping\n",
      "[INFO]\tGene ALG10B has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD50 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPB has 0 variants, skipping\n",
      "[INFO]\tGene VANGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR2 has 0 variants, skipping\n",
      "[INFO]\tGene CAV3 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN3 has 0 variants, skipping\n",
      "[INFO]\tGene MAMLD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS15 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene SPEN has 0 variants, skipping\n",
      "[INFO]\tGene EFHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A3 has 0 variants, skipping\n",
      "[INFO]\tGene DACT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287036 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM7 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF5 has 0 variants, skipping\n",
      "[INFO]\tGene HJURP has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL1 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK1P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8078 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AD1 has 0 variants, skipping\n",
      "[INFO]\tGene QSOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01925 has 0 variants, skipping\n",
      "[INFO]\tGene GDPGP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12B has 0 variants, skipping\n",
      "[INFO]\tGene AGRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01904 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00470 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN8 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN7 has 0 variants, skipping\n",
      "[INFO]\tGene CBX3P2 has 0 variants, skipping\n",
      "[INFO]\tGene GAS8 has 0 variants, skipping\n",
      "[INFO]\tGene GLYAT has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01895 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3D has 0 variants, skipping\n",
      "[INFO]\tGene RAB6D has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6718 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01892 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L26 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E1B has 0 variants, skipping\n",
      "[INFO]\tGene PSTK has 0 variants, skipping\n",
      "[INFO]\tGene NUB1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP4 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3976HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3976 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373311 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR199B has 0 variants, skipping\n",
      "[INFO]\tGene MIR374B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01089 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM221 has 0 variants, skipping\n",
      "[INFO]\tGene MGP has 0 variants, skipping\n",
      "[INFO]\tGene RHBG has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01387 has 0 variants, skipping\n",
      "[INFO]\tGene UOX has 0 variants, skipping\n",
      "[INFO]\tGene TSSK3 has 0 variants, skipping\n",
      "[INFO]\tGene ADD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927188 has 0 variants, skipping\n",
      "[INFO]\tGene PRH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100192426 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF223 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf12 has 0 variants, skipping\n",
      "[INFO]\tGene NTAQ1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01254 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01887 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927410 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM167B has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL3 has 0 variants, skipping\n",
      "[INFO]\tGene TEX51 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6788 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01928 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01255 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7153 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723996 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf61 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371998 has 0 variants, skipping\n",
      "[INFO]\tGene LEPROT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01882 has 0 variants, skipping\n",
      "[INFO]\tGene CENPO has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5190 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4526 has 0 variants, skipping\n",
      "[INFO]\tGene INHA has 0 variants, skipping\n",
      "[INFO]\tGene UPK1A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM50 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD2 has 0 variants, skipping\n",
      "[INFO]\tGene PKIG has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F35P has 0 variants, skipping\n",
      "[INFO]\tGene CXADRP3 has 0 variants, skipping\n",
      "[INFO]\tGene POTEC has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01443 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01444 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644669 has 0 variants, skipping\n",
      "[INFO]\tGene SCEL has 0 variants, skipping\n",
      "[INFO]\tGene PRKD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A1HG has 0 variants, skipping\n",
      "[INFO]\tGene SEC24D has 0 variants, skipping\n",
      "[INFO]\tGene LGR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927571 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1F has 0 variants, skipping\n",
      "[INFO]\tGene OR5AP2 has 0 variants, skipping\n",
      "[INFO]\tGene AMT has 0 variants, skipping\n",
      "[INFO]\tGene CDHR5 has 0 variants, skipping\n",
      "[INFO]\tGene RBFA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM241 has 0 variants, skipping\n",
      "[INFO]\tGene RTL8B has 0 variants, skipping\n",
      "[INFO]\tGene RTL8C has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC3 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDXK has 0 variants, skipping\n",
      "[INFO]\tGene YARS1 has 0 variants, skipping\n",
      "[INFO]\tGene KMO has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3A has 0 variants, skipping\n",
      "[INFO]\tGene RAB2A has 0 variants, skipping\n",
      "[INFO]\tGene PZP has 0 variants, skipping\n",
      "[INFO]\tGene PYGL has 0 variants, skipping\n",
      "[INFO]\tGene TTC39C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AMER3 has 0 variants, skipping\n",
      "[INFO]\tGene APOC2 has 0 variants, skipping\n",
      "[INFO]\tGene CITED1 has 0 variants, skipping\n",
      "[INFO]\tGene LLGL1 has 0 variants, skipping\n",
      "[INFO]\tGene LBP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01915 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372028 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01894 has 0 variants, skipping\n",
      "[INFO]\tGene NR5A2 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM6 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF13 has 0 variants, skipping\n",
      "[INFO]\tGene OAZ3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01543 has 0 variants, skipping\n",
      "[INFO]\tGene SESN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8057 has 0 variants, skipping\n",
      "[INFO]\tGene ALPK3 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372038 has 0 variants, skipping\n",
      "[INFO]\tGene ADAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAS has 0 variants, skipping\n",
      "[INFO]\tGene SULF1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf105 has 0 variants, skipping\n",
      "[INFO]\tGene DSG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A2 has 0 variants, skipping\n",
      "[INFO]\tGene DSG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABRAXAS1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CEP250 has 0 variants, skipping\n",
      "[INFO]\tGene SNX5 has 0 variants, skipping\n",
      "[INFO]\tGene RELT has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1E has 0 variants, skipping\n",
      "[INFO]\tGene GJD3 has 0 variants, skipping\n",
      "[INFO]\tGene PANK1 has 0 variants, skipping\n",
      "[INFO]\tGene OXER1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOS has 0 variants, skipping\n",
      "[INFO]\tGene HUS1B has 0 variants, skipping\n",
      "[INFO]\tGene RPP38 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H2 has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS1 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB1I1 has 0 variants, skipping\n",
      "[INFO]\tGene NPM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF271P has 0 variants, skipping\n",
      "[INFO]\tGene KRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3975 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3929 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1 has 0 variants, skipping\n",
      "[INFO]\tGene VARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEM1 has 0 variants, skipping\n",
      "[INFO]\tGene AIMP2 has 0 variants, skipping\n",
      "[INFO]\tGene TYRO3 has 0 variants, skipping\n",
      "[INFO]\tGene TSG101 has 0 variants, skipping\n",
      "[INFO]\tGene SEC16B has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC22 has 0 variants, skipping\n",
      "[INFO]\tGene HROB has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372071 has 0 variants, skipping\n",
      "[INFO]\tGene NAE1 has 0 variants, skipping\n",
      "[INFO]\tGene GJB2 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL1 has 0 variants, skipping\n",
      "[INFO]\tGene ETV5 has 0 variants, skipping\n",
      "[INFO]\tGene ESRRA has 0 variants, skipping\n",
      "[INFO]\tGene ERH has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372069 has 0 variants, skipping\n",
      "[INFO]\tGene SVEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK4 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf18 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG1 has 0 variants, skipping\n",
      "[INFO]\tGene SST has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A1 has 0 variants, skipping\n",
      "[INFO]\tGene PRIM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3R1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS3 has 0 variants, skipping\n",
      "[INFO]\tGene PENK has 0 variants, skipping\n",
      "[INFO]\tGene TARM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB14 has 0 variants, skipping\n",
      "[INFO]\tGene DELE1 has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL3 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MAMSTR has 0 variants, skipping\n",
      "[INFO]\tGene C10orf62 has 0 variants, skipping\n",
      "[INFO]\tGene DALRD3 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H3 has 0 variants, skipping\n",
      "[INFO]\tGene APPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2BP has 0 variants, skipping\n",
      "[INFO]\tGene FITM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF697 has 0 variants, skipping\n",
      "[INFO]\tGene SYCN has 0 variants, skipping\n",
      "[INFO]\tGene MYO5B has 0 variants, skipping\n",
      "[INFO]\tGene CCRL2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6V1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12A has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4A has 0 variants, skipping\n",
      "[INFO]\tGene PYM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372068 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA111 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4318 has 0 variants, skipping\n",
      "[INFO]\tGene MIR924HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR5583-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5583-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01902 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01477 has 0 variants, skipping\n",
      "[INFO]\tGene KC6 has 0 variants, skipping\n",
      "[INFO]\tGene FKTN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF765-ZNF761 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00907 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01478 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01601 has 0 variants, skipping\n",
      "[INFO]\tGene MCMBP has 0 variants, skipping\n",
      "[INFO]\tGene TESK2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC14 has 0 variants, skipping\n",
      "[INFO]\tGene TARBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF165 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C5 has 0 variants, skipping\n",
      "[INFO]\tGene SET has 0 variants, skipping\n",
      "[INFO]\tGene FAM124B has 0 variants, skipping\n",
      "[INFO]\tGene FAM228B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0825 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf20 has 0 variants, skipping\n",
      "[INFO]\tGene INS-IGF2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR92 has 0 variants, skipping\n",
      "[INFO]\tGene DHX30 has 0 variants, skipping\n",
      "[INFO]\tGene HAS3 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM5 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG4 has 0 variants, skipping\n",
      "[INFO]\tGene TAF8 has 0 variants, skipping\n",
      "[INFO]\tGene HINT3 has 0 variants, skipping\n",
      "[INFO]\tGene NUP88 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4527HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4743 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129878 has 0 variants, skipping\n",
      "[INFO]\tGene TCP10L3 has 0 variants, skipping\n",
      "[INFO]\tGene KY has 0 variants, skipping\n",
      "[INFO]\tGene MIR4744 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF3 has 0 variants, skipping\n",
      "[INFO]\tGene SPRN has 0 variants, skipping\n",
      "[INFO]\tGene MIR1539 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF6 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA17 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG22 has 0 variants, skipping\n",
      "[INFO]\tGene H2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4320 has 0 variants, skipping\n",
      "[INFO]\tGene MYCN has 0 variants, skipping\n",
      "[INFO]\tGene PRG4 has 0 variants, skipping\n",
      "[INFO]\tGene MCCC1 has 0 variants, skipping\n",
      "[INFO]\tGene GFM1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX30 has 0 variants, skipping\n",
      "[INFO]\tGene CDH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01630 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4528 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01919 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01917 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA37 has 0 variants, skipping\n",
      "[INFO]\tGene INPP4B has 0 variants, skipping\n",
      "[INFO]\tGene STX19 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T35 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T27 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS53 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01929 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG4 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A9 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A10 has 0 variants, skipping\n",
      "[INFO]\tGene CASP8 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F3 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM15 has 0 variants, skipping\n",
      "[INFO]\tGene SMC4 has 0 variants, skipping\n",
      "[INFO]\tGene TCF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4529 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01415 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01416 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642484 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01905 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01539 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH1 has 0 variants, skipping\n",
      "[INFO]\tGene NPM1 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02565 has 0 variants, skipping\n",
      "[INFO]\tGene SNX1 has 0 variants, skipping\n",
      "[INFO]\tGene CMPK2 has 0 variants, skipping\n",
      "[INFO]\tGene THEGL has 0 variants, skipping\n",
      "[INFO]\tGene MRPL14 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP3 has 0 variants, skipping\n",
      "[INFO]\tGene UCHL1 has 0 variants, skipping\n",
      "[INFO]\tGene TPH1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF3 has 0 variants, skipping\n",
      "[INFO]\tGene IL4R has 0 variants, skipping\n",
      "[INFO]\tGene PLP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG1 has 0 variants, skipping\n",
      "[INFO]\tGene HECW2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3591 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA108 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01926 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24A has 0 variants, skipping\n",
      "[INFO]\tGene PSME1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT12 has 0 variants, skipping\n",
      "[INFO]\tGene OACYLP has 0 variants, skipping\n",
      "[INFO]\tGene ITIH4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01544 has 0 variants, skipping\n",
      "[INFO]\tGene LSAMP has 0 variants, skipping\n",
      "[INFO]\tGene LRPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP6 has 0 variants, skipping\n",
      "[INFO]\tGene LRP4 has 0 variants, skipping\n",
      "[INFO]\tGene MROH6 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHD1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR139 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2A has 0 variants, skipping\n",
      "[INFO]\tGene RNASE10 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC73 has 0 variants, skipping\n",
      "[INFO]\tGene LRP2BP has 0 variants, skipping\n",
      "[INFO]\tGene ARV1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00515 has 0 variants, skipping\n",
      "[INFO]\tGene PICSAR has 0 variants, skipping\n",
      "[INFO]\tGene TAC3 has 0 variants, skipping\n",
      "[INFO]\tGene MASP1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR14 has 0 variants, skipping\n",
      "[INFO]\tGene WDR54 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10A has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP3A has 0 variants, skipping\n",
      "[INFO]\tGene C3orf85 has 0 variants, skipping\n",
      "[INFO]\tGene CDH4 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP14 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM115 has 0 variants, skipping\n",
      "[INFO]\tGene DAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00305 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01924 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01538 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5011 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643542 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01903 has 0 variants, skipping\n",
      "[INFO]\tGene FAM221A has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372179 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf194 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01909 has 0 variants, skipping\n",
      "[INFO]\tGene LIVAR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01899 has 0 variants, skipping\n",
      "[INFO]\tGene MLH1 has 0 variants, skipping\n",
      "[INFO]\tGene GYPA has 0 variants, skipping\n",
      "[INFO]\tGene LOC541473 has 0 variants, skipping\n",
      "[INFO]\tGene SPEG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AV has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02582 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01922 has 0 variants, skipping\n",
      "[INFO]\tGene GRM2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01898 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01893 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927989 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00683 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01927 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01879 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF236-DT has 0 variants, skipping\n",
      "[INFO]\tGene CA7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01029 has 0 variants, skipping\n",
      "[INFO]\tGene CDK8 has 0 variants, skipping\n",
      "[INFO]\tGene CD164 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R3 has 0 variants, skipping\n",
      "[INFO]\tGene SLURP1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L2B has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR3L has 0 variants, skipping\n",
      "[INFO]\tGene LOC284240 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284241 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF35 has 0 variants, skipping\n",
      "[INFO]\tGene RBFADN has 0 variants, skipping\n",
      "[INFO]\tGene PARD6G-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRSAM1 has 0 variants, skipping\n",
      "[INFO]\tGene WASH5P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01002 has 0 variants, skipping\n",
      "[INFO]\tGene PSG7 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL36 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf72 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN11A has 0 variants, skipping\n",
      "[INFO]\tGene PNP has 0 variants, skipping\n",
      "[INFO]\tGene NF2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C has 0 variants, skipping\n",
      "[INFO]\tGene MIR4745 has 0 variants, skipping\n",
      "[INFO]\tGene AZU1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3187 has 0 variants, skipping\n",
      "[INFO]\tGene CRHR2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E1 has 0 variants, skipping\n",
      "[INFO]\tGene TFR2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN2 has 0 variants, skipping\n",
      "[INFO]\tGene STMN1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMC2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3D has 0 variants, skipping\n",
      "[INFO]\tGene BAG2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG3 has 0 variants, skipping\n",
      "[INFO]\tGene PMPCB has 0 variants, skipping\n",
      "[INFO]\tGene CIRBP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM11B has 0 variants, skipping\n",
      "[INFO]\tGene NRGN has 0 variants, skipping\n",
      "[INFO]\tGene PPP5D1 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1909 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288123 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HIRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR65 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1227 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6789 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4321 has 0 variants, skipping\n",
      "[INFO]\tGene CD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7108 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01775 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7850 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPD has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6A has 0 variants, skipping\n",
      "[INFO]\tGene UTP20 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF19 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2S has 0 variants, skipping\n",
      "[INFO]\tGene LARGE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996351 has 0 variants, skipping\n",
      "[INFO]\tGene CACTIN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF765 has 0 variants, skipping\n",
      "[INFO]\tGene NMRK2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD37 has 0 variants, skipping\n",
      "[INFO]\tGene MIS12 has 0 variants, skipping\n",
      "[INFO]\tGene STAC2 has 0 variants, skipping\n",
      "[INFO]\tGene LAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD2L has 0 variants, skipping\n",
      "[INFO]\tGene DTX3 has 0 variants, skipping\n",
      "[INFO]\tGene EML3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4746 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B1 has 0 variants, skipping\n",
      "[INFO]\tGene FTCDNL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDY2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-3HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4747 has 0 variants, skipping\n",
      "[INFO]\tGene RFX1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDF2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4B1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A8 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB7 has 0 variants, skipping\n",
      "[INFO]\tGene TGDS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF600 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC154 has 0 variants, skipping\n",
      "[INFO]\tGene TTC9 has 0 variants, skipping\n",
      "[INFO]\tGene BICRAL has 0 variants, skipping\n",
      "[INFO]\tGene KRT6B has 0 variants, skipping\n",
      "[INFO]\tGene FEM1C has 0 variants, skipping\n",
      "[INFO]\tGene SMCO4 has 0 variants, skipping\n",
      "[INFO]\tGene HMCES has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF3 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928844 has 0 variants, skipping\n",
      "[INFO]\tGene MCM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128568 has 0 variants, skipping\n",
      "[INFO]\tGene FAM204A has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6885 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6790 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF384 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3940 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM240 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2A1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1F has 0 variants, skipping\n",
      "[INFO]\tGene KMT5A has 0 variants, skipping\n",
      "[INFO]\tGene SMIM34A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6791 has 0 variants, skipping\n",
      "[INFO]\tGene KIF13A has 0 variants, skipping\n",
      "[INFO]\tGene C20orf194 has 0 variants, skipping\n",
      "[INFO]\tGene CD6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK9 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE4P has 0 variants, skipping\n",
      "[INFO]\tGene FLJ25758 has 0 variants, skipping\n",
      "[INFO]\tGene CD244 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF723 has 0 variants, skipping\n",
      "[INFO]\tGene DNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene TECPR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID4A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC19 has 0 variants, skipping\n",
      "[INFO]\tGene CASD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6792 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0100 has 0 variants, skipping\n",
      "[INFO]\tGene NET1 has 0 variants, skipping\n",
      "[INFO]\tGene COL13A1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCKSL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4GP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC121 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA2P2 has 0 variants, skipping\n",
      "[INFO]\tGene S100A12 has 0 variants, skipping\n",
      "[INFO]\tGene S100A11 has 0 variants, skipping\n",
      "[INFO]\tGene RTN2 has 0 variants, skipping\n",
      "[INFO]\tGene DLL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE6 has 0 variants, skipping\n",
      "[INFO]\tGene RHEB has 0 variants, skipping\n",
      "[INFO]\tGene RBL2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP4 has 0 variants, skipping\n",
      "[INFO]\tGene NHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRCAP has 0 variants, skipping\n",
      "[INFO]\tGene NHLH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPNS2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2B has 0 variants, skipping\n",
      "[INFO]\tGene RAB11B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4999 has 0 variants, skipping\n",
      "[INFO]\tGene RNF4 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1SW has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PO has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1E1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP16 has 0 variants, skipping\n",
      "[INFO]\tGene MRC2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL6 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf70 has 0 variants, skipping\n",
      "[INFO]\tGene RAB12 has 0 variants, skipping\n",
      "[INFO]\tGene LACTBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF426-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF561-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM179 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF812P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505555 has 0 variants, skipping\n",
      "[INFO]\tGene OGG1 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGD has 0 variants, skipping\n",
      "[INFO]\tGene ABRACL has 0 variants, skipping\n",
      "[INFO]\tGene MID1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNE2 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5589 has 0 variants, skipping\n",
      "[INFO]\tGene C3P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD105 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD105B has 0 variants, skipping\n",
      "[INFO]\tGene RBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1841 has 0 variants, skipping\n",
      "[INFO]\tGene RGS11 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG4 has 0 variants, skipping\n",
      "[INFO]\tGene ECD has 0 variants, skipping\n",
      "[INFO]\tGene MIR4322 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB7 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF157 has 0 variants, skipping\n",
      "[INFO]\tGene METAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FIGN has 0 variants, skipping\n",
      "[INFO]\tGene TCP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4748 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6793 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL8 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF14 has 0 variants, skipping\n",
      "[INFO]\tGene AK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6886 has 0 variants, skipping\n",
      "[INFO]\tGene FAM227B has 0 variants, skipping\n",
      "[INFO]\tGene ARID2 has 0 variants, skipping\n",
      "[INFO]\tGene STPG4 has 0 variants, skipping\n",
      "[INFO]\tGene TYROBP has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372273 has 0 variants, skipping\n",
      "[INFO]\tGene OXTR has 0 variants, skipping\n",
      "[INFO]\tGene GIP has 0 variants, skipping\n",
      "[INFO]\tGene GHSR has 0 variants, skipping\n",
      "[INFO]\tGene GBP2 has 0 variants, skipping\n",
      "[INFO]\tGene FLT3 has 0 variants, skipping\n",
      "[INFO]\tGene VWA3B has 0 variants, skipping\n",
      "[INFO]\tGene AKT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7974 has 0 variants, skipping\n",
      "[INFO]\tGene IFT22 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1E2 has 0 variants, skipping\n",
      "[INFO]\tGene PWP1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1P10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF833P has 0 variants, skipping\n",
      "[INFO]\tGene CEBPG has 0 variants, skipping\n",
      "[INFO]\tGene SPDEF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF433-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723360 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF625-ZNF20 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289333 has 0 variants, skipping\n",
      "[INFO]\tGene PSAP has 0 variants, skipping\n",
      "[INFO]\tGene PRR36 has 0 variants, skipping\n",
      "[INFO]\tGene CTH has 0 variants, skipping\n",
      "[INFO]\tGene SNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene BPNT1 has 0 variants, skipping\n",
      "[INFO]\tGene IRF9 has 0 variants, skipping\n",
      "[INFO]\tGene PTH1R has 0 variants, skipping\n",
      "[INFO]\tGene L1TD1 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene NUAK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD135 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD41 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00703 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5684 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6794 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC19 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC141 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG1 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L12 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5695 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6515 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3177 has 0 variants, skipping\n",
      "[INFO]\tGene CCL11 has 0 variants, skipping\n",
      "[INFO]\tGene CCL5 has 0 variants, skipping\n",
      "[INFO]\tGene EVC2 has 0 variants, skipping\n",
      "[INFO]\tGene FRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM135B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF691 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L5 has 0 variants, skipping\n",
      "[INFO]\tGene RFC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284454 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF679 has 0 variants, skipping\n",
      "[INFO]\tGene EPRS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD5 has 0 variants, skipping\n",
      "[INFO]\tGene MISP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1199 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507373 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01841 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01842 has 0 variants, skipping\n",
      "[INFO]\tGene MIR639 has 0 variants, skipping\n",
      "[INFO]\tGene SCG3 has 0 variants, skipping\n",
      "[INFO]\tGene AGXT has 0 variants, skipping\n",
      "[INFO]\tGene CLEC17A has 0 variants, skipping\n",
      "[INFO]\tGene CXCR3 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA104 has 0 variants, skipping\n",
      "[INFO]\tGene H3C4 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB3 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6795 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1470 has 0 variants, skipping\n",
      "[INFO]\tGene LZIC has 0 variants, skipping\n",
      "[INFO]\tGene AGO1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F24P has 0 variants, skipping\n",
      "[INFO]\tGene FLI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM229A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00661 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00905 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01855 has 0 variants, skipping\n",
      "[INFO]\tGene FLG2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC183 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02749 has 0 variants, skipping\n",
      "[INFO]\tGene UTP6 has 0 variants, skipping\n",
      "[INFO]\tGene PRAC1 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene NIFK has 0 variants, skipping\n",
      "[INFO]\tGene WNK4 has 0 variants, skipping\n",
      "[INFO]\tGene NNT has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MG has 0 variants, skipping\n",
      "[INFO]\tGene GRK4 has 0 variants, skipping\n",
      "[INFO]\tGene OSGIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00221 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2U1 has 0 variants, skipping\n",
      "[INFO]\tGene GLT6D1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL8 has 0 variants, skipping\n",
      "[INFO]\tGene CDK14 has 0 variants, skipping\n",
      "[INFO]\tGene CENPP has 0 variants, skipping\n",
      "[INFO]\tGene SRP14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA118 has 0 variants, skipping\n",
      "[INFO]\tGene COTL1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNO has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1256 has 0 variants, skipping\n",
      "[INFO]\tGene PKN3 has 0 variants, skipping\n",
      "[INFO]\tGene BISPR has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT77 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CD99 has 0 variants, skipping\n",
      "[INFO]\tGene MIF has 0 variants, skipping\n",
      "[INFO]\tGene MKI67 has 0 variants, skipping\n",
      "[INFO]\tGene MSH3 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2B has 0 variants, skipping\n",
      "[INFO]\tGene OR8J3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC119B has 0 variants, skipping\n",
      "[INFO]\tGene SCRG1 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAP has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA68 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112577592 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725254 has 0 variants, skipping\n",
      "[INFO]\tGene KIFC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729966 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3189 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF514 has 0 variants, skipping\n",
      "[INFO]\tGene PCNX4 has 0 variants, skipping\n",
      "[INFO]\tGene ACADSB has 0 variants, skipping\n",
      "[INFO]\tGene LINC00513 has 0 variants, skipping\n",
      "[INFO]\tGene FAM177A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR18 has 0 variants, skipping\n",
      "[INFO]\tGene ARAF has 0 variants, skipping\n",
      "[INFO]\tGene BTG2 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM5 has 0 variants, skipping\n",
      "[INFO]\tGene UPK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF175 has 0 variants, skipping\n",
      "[INFO]\tGene EPC2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS38 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM34 has 0 variants, skipping\n",
      "[INFO]\tGene PDYN has 0 variants, skipping\n",
      "[INFO]\tGene FLT4 has 0 variants, skipping\n",
      "[INFO]\tGene INKA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS7 has 0 variants, skipping\n",
      "[INFO]\tGene ARSG has 0 variants, skipping\n",
      "[INFO]\tGene FAM104A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00663 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF826P has 0 variants, skipping\n",
      "[INFO]\tGene RLF has 0 variants, skipping\n",
      "[INFO]\tGene RGS17 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA6L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00664 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB2A has 0 variants, skipping\n",
      "[INFO]\tGene TUBG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400682 has 0 variants, skipping\n",
      "[INFO]\tGene TIFA has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929124 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01233 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01785 has 0 variants, skipping\n",
      "[INFO]\tGene NAGS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01859 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01858 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO5P1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA10 has 0 variants, skipping\n",
      "[INFO]\tGene FEZ1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505851 has 0 variants, skipping\n",
      "[INFO]\tGene FCRLB has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927151 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100420587 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724908 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00906 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724958 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284395 has 0 variants, skipping\n",
      "[INFO]\tGene MRC1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK5 has 0 variants, skipping\n",
      "[INFO]\tGene PTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGOHB has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN2 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01791 has 0 variants, skipping\n",
      "[INFO]\tGene THEG5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01782 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400684 has 0 variants, skipping\n",
      "[INFO]\tGene XKR9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA68B has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA13 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA12 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN2A has 0 variants, skipping\n",
      "[INFO]\tGene ETAA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA11 has 0 variants, skipping\n",
      "[INFO]\tGene DHX40 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2A has 0 variants, skipping\n",
      "[INFO]\tGene LEF1 has 0 variants, skipping\n",
      "[INFO]\tGene FIGLA has 0 variants, skipping\n",
      "[INFO]\tGene VMA21 has 0 variants, skipping\n",
      "[INFO]\tGene CT83 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1B2P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM130 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2B3P has 0 variants, skipping\n",
      "[INFO]\tGene UNC80 has 0 variants, skipping\n",
      "[INFO]\tGene STK25 has 0 variants, skipping\n",
      "[INFO]\tGene APMAP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01801 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00904 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01838 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BUB3 has 0 variants, skipping\n",
      "[INFO]\tGene LNPEP has 0 variants, skipping\n",
      "[INFO]\tGene SERGEF has 0 variants, skipping\n",
      "[INFO]\tGene HPN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN23 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA5 has 0 variants, skipping\n",
      "[INFO]\tGene UBC has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1549 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE26 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL10 has 0 variants, skipping\n",
      "[INFO]\tGene ING2 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF9 has 0 variants, skipping\n",
      "[INFO]\tGene IL13RA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00312 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC66 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01531 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PRG3 has 0 variants, skipping\n",
      "[INFO]\tGene NDC80 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL13 has 0 variants, skipping\n",
      "[INFO]\tGene TSSC4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM147-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01766 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF180 has 0 variants, skipping\n",
      "[INFO]\tGene UPK1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH16 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR4 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA2 has 0 variants, skipping\n",
      "[INFO]\tGene NEFM has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-1 has 0 variants, skipping\n",
      "[INFO]\tGene RRP7A has 0 variants, skipping\n",
      "[INFO]\tGene MMADHC has 0 variants, skipping\n",
      "[INFO]\tGene LINC01529 has 0 variants, skipping\n",
      "[INFO]\tGene PIBF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372383 has 0 variants, skipping\n",
      "[INFO]\tGene SIK1 has 0 variants, skipping\n",
      "[INFO]\tGene TPH2 has 0 variants, skipping\n",
      "[INFO]\tGene MED20 has 0 variants, skipping\n",
      "[INFO]\tGene NKILA has 0 variants, skipping\n",
      "[INFO]\tGene FAT3 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf71 has 0 variants, skipping\n",
      "[INFO]\tGene STK19 has 0 variants, skipping\n",
      "[INFO]\tGene ASMTL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134317 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728752 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF529-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0D1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01534 has 0 variants, skipping\n",
      "[INFO]\tGene STMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSG4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF790-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ING1 has 0 variants, skipping\n",
      "[INFO]\tGene EPYC has 0 variants, skipping\n",
      "[INFO]\tGene CREBRF has 0 variants, skipping\n",
      "[INFO]\tGene MIR328 has 0 variants, skipping\n",
      "[INFO]\tGene LOC104968399 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376731 has 0 variants, skipping\n",
      "[INFO]\tGene LSS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01391 has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ-SCHIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IRAIN has 0 variants, skipping\n",
      "[INFO]\tGene SPATA19 has 0 variants, skipping\n",
      "[INFO]\tGene PDILT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF793-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF571-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX37 has 0 variants, skipping\n",
      "[INFO]\tGene NANP has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D1 has 0 variants, skipping\n",
      "[INFO]\tGene GGN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF585A has 0 variants, skipping\n",
      "[INFO]\tGene DAND5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644554 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD152 has 0 variants, skipping\n",
      "[INFO]\tGene LDB2 has 0 variants, skipping\n",
      "[INFO]\tGene GAPT has 0 variants, skipping\n",
      "[INFO]\tGene SPRED3 has 0 variants, skipping\n",
      "[INFO]\tGene STK11 has 0 variants, skipping\n",
      "[INFO]\tGene SOD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A2 has 0 variants, skipping\n",
      "[INFO]\tGene SHOX has 0 variants, skipping\n",
      "[INFO]\tGene SELE has 0 variants, skipping\n",
      "[INFO]\tGene PSEN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372397 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF330 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1B has 0 variants, skipping\n",
      "[INFO]\tGene CYP2S1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP2 has 0 variants, skipping\n",
      "[INFO]\tGene ROS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA6 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA4 has 0 variants, skipping\n",
      "[INFO]\tGene FMOD has 0 variants, skipping\n",
      "[INFO]\tGene DDB1 has 0 variants, skipping\n",
      "[INFO]\tGene FMO4 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT4 has 0 variants, skipping\n",
      "[INFO]\tGene NCS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF5 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD175 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129935 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS17A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM169 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6719 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2A has 0 variants, skipping\n",
      "[INFO]\tGene UBA7 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC49A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6796 has 0 variants, skipping\n",
      "[INFO]\tGene RALA has 0 variants, skipping\n",
      "[INFO]\tGene PHF2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN14 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A1 has 0 variants, skipping\n",
      "[INFO]\tGene SSPN has 0 variants, skipping\n",
      "[INFO]\tGene PARP10 has 0 variants, skipping\n",
      "[INFO]\tGene SDS has 0 variants, skipping\n",
      "[INFO]\tGene SF3B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIA-RAB4B has 0 variants, skipping\n",
      "[INFO]\tGene RAB4B-EGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM8 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2T1P has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2G1P has 0 variants, skipping\n",
      "[INFO]\tGene CYP2B7P has 0 variants, skipping\n",
      "[INFO]\tGene NPTN has 0 variants, skipping\n",
      "[INFO]\tGene S100A14 has 0 variants, skipping\n",
      "[INFO]\tGene CLK4 has 0 variants, skipping\n",
      "[INFO]\tGene CLTRN has 0 variants, skipping\n",
      "[INFO]\tGene DECR2 has 0 variants, skipping\n",
      "[INFO]\tGene RHOJ has 0 variants, skipping\n",
      "[INFO]\tGene NUP205 has 0 variants, skipping\n",
      "[INFO]\tGene UPP2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-9 has 0 variants, skipping\n",
      "[INFO]\tGene NINL has 0 variants, skipping\n",
      "[INFO]\tGene AHSP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01480 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A2 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL2A has 0 variants, skipping\n",
      "[INFO]\tGene SNAP29 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505585 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D4A has 0 variants, skipping\n",
      "[INFO]\tGene GFOD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4323 has 0 variants, skipping\n",
      "[INFO]\tGene STX6 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR1 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD2B has 0 variants, skipping\n",
      "[INFO]\tGene OTOF has 0 variants, skipping\n",
      "[INFO]\tGene CLCNKA has 0 variants, skipping\n",
      "[INFO]\tGene ILVBL has 0 variants, skipping\n",
      "[INFO]\tGene MIR8077 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930071 has 0 variants, skipping\n",
      "[INFO]\tGene LIPE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BARHL2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR108 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1D has 0 variants, skipping\n",
      "[INFO]\tGene PSG8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf58 has 0 variants, skipping\n",
      "[INFO]\tGene STAP2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6N2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284344 has 0 variants, skipping\n",
      "[INFO]\tGene PRG1 has 0 variants, skipping\n",
      "[INFO]\tGene TWIST1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAP has 0 variants, skipping\n",
      "[INFO]\tGene CDC73 has 0 variants, skipping\n",
      "[INFO]\tGene NEBL has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A3 has 0 variants, skipping\n",
      "[INFO]\tGene DFFA has 0 variants, skipping\n",
      "[INFO]\tGene ALLC has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505715 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928063 has 0 variants, skipping\n",
      "[INFO]\tGene SNUPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC100379224 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A6 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC47A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184C has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP1L has 0 variants, skipping\n",
      "[INFO]\tGene PTPN6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4531 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7A has 0 variants, skipping\n",
      "[INFO]\tGene MIR8085 has 0 variants, skipping\n",
      "[INFO]\tGene CCR4 has 0 variants, skipping\n",
      "[INFO]\tGene AP1M2 has 0 variants, skipping\n",
      "[INFO]\tGene FSTL3 has 0 variants, skipping\n",
      "[INFO]\tGene PURA has 0 variants, skipping\n",
      "[INFO]\tGene AKAP8 has 0 variants, skipping\n",
      "[INFO]\tGene ZMPSTE24 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene APOC4-APOC2 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG4 has 0 variants, skipping\n",
      "[INFO]\tGene TESC has 0 variants, skipping\n",
      "[INFO]\tGene RMND1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT57 has 0 variants, skipping\n",
      "[INFO]\tGene FEZF2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR1 has 0 variants, skipping\n",
      "[INFO]\tGene GID8 has 0 variants, skipping\n",
      "[INFO]\tGene TRMU has 0 variants, skipping\n",
      "[INFO]\tGene PAK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM45A has 0 variants, skipping\n",
      "[INFO]\tGene TTC22 has 0 variants, skipping\n",
      "[INFO]\tGene UACA has 0 variants, skipping\n",
      "[INFO]\tGene DDT has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6088 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF107 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM46 has 0 variants, skipping\n",
      "[INFO]\tGene TCF3 has 0 variants, skipping\n",
      "[INFO]\tGene EML2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AHR has 0 variants, skipping\n",
      "[INFO]\tGene CYP2W1 has 0 variants, skipping\n",
      "[INFO]\tGene CHD7 has 0 variants, skipping\n",
      "[INFO]\tGene DM1-AS has 0 variants, skipping\n",
      "[INFO]\tGene FLOT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF417 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC93429 has 0 variants, skipping\n",
      "[INFO]\tGene VIL1 has 0 variants, skipping\n",
      "[INFO]\tGene AGK has 0 variants, skipping\n",
      "[INFO]\tGene CCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F6 has 0 variants, skipping\n",
      "[INFO]\tGene DACT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS34 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-E has 0 variants, skipping\n",
      "[INFO]\tGene USP17L1 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L8 has 0 variants, skipping\n",
      "[INFO]\tGene TENT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3190 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3191 has 0 variants, skipping\n",
      "[INFO]\tGene MANSC4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP52 has 0 variants, skipping\n",
      "[INFO]\tGene NAPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD23 has 0 variants, skipping\n",
      "[INFO]\tGene NOP53-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG5 has 0 variants, skipping\n",
      "[INFO]\tGene PSD has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A12 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A13 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A4 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A7 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A11 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A9 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A6 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A8 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A10 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A14 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C4 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C3 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDY1B has 0 variants, skipping\n",
      "[INFO]\tGene MANBAL has 0 variants, skipping\n",
      "[INFO]\tGene SPAG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP6NL has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD3 has 0 variants, skipping\n",
      "[INFO]\tGene RALB has 0 variants, skipping\n",
      "[INFO]\tGene CARD8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CST11 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A16 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX24 has 0 variants, skipping\n",
      "[INFO]\tGene ASF1A has 0 variants, skipping\n",
      "[INFO]\tGene FOCAD has 0 variants, skipping\n",
      "[INFO]\tGene SNORD76 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL3 has 0 variants, skipping\n",
      "[INFO]\tGene MESTIT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01139 has 0 variants, skipping\n",
      "[INFO]\tGene ASPSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC1P has 0 variants, skipping\n",
      "[INFO]\tGene TIMM50 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105447645 has 0 variants, skipping\n",
      "[INFO]\tGene PAG1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO6 has 0 variants, skipping\n",
      "[INFO]\tGene HEMGN has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1G has 0 variants, skipping\n",
      "[INFO]\tGene NUCB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HELQ has 0 variants, skipping\n",
      "[INFO]\tGene SAP25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF675 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR1 has 0 variants, skipping\n",
      "[INFO]\tGene ACMSD has 0 variants, skipping\n",
      "[INFO]\tGene MIR6798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101059948 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-G2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-G1 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB4 has 0 variants, skipping\n",
      "[INFO]\tGene H4C14 has 0 variants, skipping\n",
      "[INFO]\tGene CHST14 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB3 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC12 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928295 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AJ1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14C has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOQ has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST1 has 0 variants, skipping\n",
      "[INFO]\tGene KL has 0 variants, skipping\n",
      "[INFO]\tGene SNORD32A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD33 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD35A has 0 variants, skipping\n",
      "[INFO]\tGene ZIC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD35B has 0 variants, skipping\n",
      "[INFO]\tGene MORC3 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR939 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOI has 0 variants, skipping\n",
      "[INFO]\tGene MIR5088 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG6 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1D has 0 variants, skipping\n",
      "[INFO]\tGene MIR6799 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHJ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6800 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1E has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A32 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL19 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP29 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4750 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR4751 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP35 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400710 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-B2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-B1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-D has 0 variants, skipping\n",
      "[INFO]\tGene TOPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMF1 has 0 variants, skipping\n",
      "[INFO]\tGene NAPSB has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-F has 0 variants, skipping\n",
      "[INFO]\tGene C19orf33 has 0 variants, skipping\n",
      "[INFO]\tGene RMI1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01869 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3C has 0 variants, skipping\n",
      "[INFO]\tGene TMEM225B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372441 has 0 variants, skipping\n",
      "[INFO]\tGene TLE2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1143 has 0 variants, skipping\n",
      "[INFO]\tGene KLKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM160 has 0 variants, skipping\n",
      "[INFO]\tGene G2E3 has 0 variants, skipping\n",
      "[INFO]\tGene STMN4 has 0 variants, skipping\n",
      "[INFO]\tGene SGO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C6 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC28 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928517 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC17P has 0 variants, skipping\n",
      "[INFO]\tGene MIR8074 has 0 variants, skipping\n",
      "[INFO]\tGene FSHB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01872 has 0 variants, skipping\n",
      "[INFO]\tGene RPS16 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB2 has 0 variants, skipping\n",
      "[INFO]\tGene PHF23 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A6 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP30 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01530 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA6P-AS has 0 variants, skipping\n",
      "[INFO]\tGene MATN4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF649-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF350-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD8 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF616 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6801 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM42 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF528-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DUOX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF137P has 0 variants, skipping\n",
      "[INFO]\tGene MSI1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF321P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF816-ZNF321P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF702P has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1C2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R16 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R4 has 0 variants, skipping\n",
      "[INFO]\tGene RFFL has 0 variants, skipping\n",
      "[INFO]\tGene YLPM1 has 0 variants, skipping\n",
      "[INFO]\tGene SF3A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK4 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH3 has 0 variants, skipping\n",
      "[INFO]\tGene GNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A27P has 0 variants, skipping\n",
      "[INFO]\tGene RAD51AP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAAP100 has 0 variants, skipping\n",
      "[INFO]\tGene TARS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPM3P9 has 0 variants, skipping\n",
      "[INFO]\tGene CLCF1 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF45 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284379 has 0 variants, skipping\n",
      "[INFO]\tGene MIR512-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR515-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519E has 0 variants, skipping\n",
      "[INFO]\tGene COMMD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR523 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518F has 0 variants, skipping\n",
      "[INFO]\tGene MIR521-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518E has 0 variants, skipping\n",
      "[INFO]\tGene MIR518A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR521-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR522 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1283-2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCA1L has 0 variants, skipping\n",
      "[INFO]\tGene LSMEM2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54L2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX1 has 0 variants, skipping\n",
      "[INFO]\tGene OVGP1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBP has 0 variants, skipping\n",
      "[INFO]\tGene DENND11 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17 has 0 variants, skipping\n",
      "[INFO]\tGene MYO15A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX5 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC2 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFC has 0 variants, skipping\n",
      "[INFO]\tGene WNT1 has 0 variants, skipping\n",
      "[INFO]\tGene VAV1 has 0 variants, skipping\n",
      "[INFO]\tGene TCHHL1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B28 has 0 variants, skipping\n",
      "[INFO]\tGene FAM99B has 0 variants, skipping\n",
      "[INFO]\tGene DMAC2L has 0 variants, skipping\n",
      "[INFO]\tGene ERCC3 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4752 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC5 has 0 variants, skipping\n",
      "[INFO]\tGene IGDCC4 has 0 variants, skipping\n",
      "[INFO]\tGene BDNF-AS has 0 variants, skipping\n",
      "[INFO]\tGene SMPX has 0 variants, skipping\n",
      "[INFO]\tGene FPGT-TNNI3K has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A1 has 0 variants, skipping\n",
      "[INFO]\tGene LENG8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DX1 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00899 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8061 has 0 variants, skipping\n",
      "[INFO]\tGene CEP44 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINI1 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL3 has 0 variants, skipping\n",
      "[INFO]\tGene LILRP2 has 0 variants, skipping\n",
      "[INFO]\tGene BRD7 has 0 variants, skipping\n",
      "[INFO]\tGene MYLIP has 0 variants, skipping\n",
      "[INFO]\tGene SAP30BP has 0 variants, skipping\n",
      "[INFO]\tGene PYCARD has 0 variants, skipping\n",
      "[INFO]\tGene SGK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928804 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133A has 0 variants, skipping\n",
      "[INFO]\tGene GP1BA has 0 variants, skipping\n",
      "[INFO]\tGene MMP12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00659 has 0 variants, skipping\n",
      "[INFO]\tGene NOL8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR137HG has 0 variants, skipping\n",
      "[INFO]\tGene SCYL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF468 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7975 has 0 variants, skipping\n",
      "[INFO]\tGene PTCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM98 has 0 variants, skipping\n",
      "[INFO]\tGene CST3 has 0 variants, skipping\n",
      "[INFO]\tGene KDSR has 0 variants, skipping\n",
      "[INFO]\tGene DLX4 has 0 variants, skipping\n",
      "[INFO]\tGene FUT5 has 0 variants, skipping\n",
      "[INFO]\tGene FUT4 has 0 variants, skipping\n",
      "[INFO]\tGene FTH1 has 0 variants, skipping\n",
      "[INFO]\tGene FRK has 0 variants, skipping\n",
      "[INFO]\tGene MIR6804 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6803 has 0 variants, skipping\n",
      "[INFO]\tGene CD24 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51P4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6805 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF683 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD157 has 0 variants, skipping\n",
      "[INFO]\tGene IL7 has 0 variants, skipping\n",
      "[INFO]\tGene HYPK has 0 variants, skipping\n",
      "[INFO]\tGene GNAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SOSTDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF521 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR9 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC5 has 0 variants, skipping\n",
      "[INFO]\tGene NELFB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01864 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF485 has 0 variants, skipping\n",
      "[INFO]\tGene TACR3 has 0 variants, skipping\n",
      "[INFO]\tGene PTGFR has 0 variants, skipping\n",
      "[INFO]\tGene PTGIR has 0 variants, skipping\n",
      "[INFO]\tGene PTGER4 has 0 variants, skipping\n",
      "[INFO]\tGene TACR2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C2 has 0 variants, skipping\n",
      "[INFO]\tGene SERF1A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2O has 0 variants, skipping\n",
      "[INFO]\tGene TCL1A has 0 variants, skipping\n",
      "[INFO]\tGene PGM5 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF582-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA1 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7C has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF4 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN3 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1522 has 0 variants, skipping\n",
      "[INFO]\tGene PEG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR1B has 0 variants, skipping\n",
      "[INFO]\tGene SPR has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF460-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE4A has 0 variants, skipping\n",
      "[INFO]\tGene DONSON has 0 variants, skipping\n",
      "[INFO]\tGene PLCB3 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM13 has 0 variants, skipping\n",
      "[INFO]\tGene POC1A has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A5 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR3 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPD1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1AP1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2CP5 has 0 variants, skipping\n",
      "[INFO]\tGene RESP18 has 0 variants, skipping\n",
      "[INFO]\tGene CD48 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128398 has 0 variants, skipping\n",
      "[INFO]\tGene SLC48A1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF2 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene CD109 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP39A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF8-ERVK3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ERVK3-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372480 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6806 has 0 variants, skipping\n",
      "[INFO]\tGene A1BG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4754 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02560 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL18 has 0 variants, skipping\n",
      "[INFO]\tGene MZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPBD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf226 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO2 has 0 variants, skipping\n",
      "[INFO]\tGene RELL1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2M has 0 variants, skipping\n",
      "[INFO]\tGene PPM1N has 0 variants, skipping\n",
      "[INFO]\tGene PCGF3 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA2B has 0 variants, skipping\n",
      "[INFO]\tGene HERPUD1 has 0 variants, skipping\n",
      "[INFO]\tGene TEDC1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT3 has 0 variants, skipping\n",
      "[INFO]\tGene SPTY2D1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN23 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC21 has 0 variants, skipping\n",
      "[INFO]\tGene INO80C has 0 variants, skipping\n",
      "[INFO]\tGene ISCA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM219 has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP82 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705G has 0 variants, skipping\n",
      "[INFO]\tGene C6orf132 has 0 variants, skipping\n",
      "[INFO]\tGene CBY3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD31 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL2B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD33B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM212 has 0 variants, skipping\n",
      "[INFO]\tGene DAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM255A has 0 variants, skipping\n",
      "[INFO]\tGene PPT1 has 0 variants, skipping\n",
      "[INFO]\tGene GDI2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LD has 0 variants, skipping\n",
      "[INFO]\tGene TBK1 has 0 variants, skipping\n",
      "[INFO]\tGene YBX1 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01606 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY3 has 0 variants, skipping\n",
      "[INFO]\tGene CLK1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX46 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP11 has 0 variants, skipping\n",
      "[INFO]\tGene PTENP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP16 has 0 variants, skipping\n",
      "[INFO]\tGene JARID2 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01561 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB12 has 0 variants, skipping\n",
      "[INFO]\tGene ATRNL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CMA1 has 0 variants, skipping\n",
      "[INFO]\tGene DTWD2 has 0 variants, skipping\n",
      "[INFO]\tGene BECN2 has 0 variants, skipping\n",
      "[INFO]\tGene NSF has 0 variants, skipping\n",
      "[INFO]\tGene NCAPG2 has 0 variants, skipping\n",
      "[INFO]\tGene MYOM2 has 0 variants, skipping\n",
      "[INFO]\tGene MTG1 has 0 variants, skipping\n",
      "[INFO]\tGene MTSS2 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY1 has 0 variants, skipping\n",
      "[INFO]\tGene FDXACB1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC5 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD6 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23B-AGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene NOLC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4686 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4651 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4437 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4423 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH9 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2-AS has 0 variants, skipping\n",
      "[INFO]\tGene PKDCC has 0 variants, skipping\n",
      "[INFO]\tGene AP5B1 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3A has 0 variants, skipping\n",
      "[INFO]\tGene CD160 has 0 variants, skipping\n",
      "[INFO]\tGene UPK1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3184 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC7A has 0 variants, skipping\n",
      "[INFO]\tGene CAPN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L has 0 variants, skipping\n",
      "[INFO]\tGene IGF1R has 0 variants, skipping\n",
      "[INFO]\tGene C11orf80 has 0 variants, skipping\n",
      "[INFO]\tGene MIR641 has 0 variants, skipping\n",
      "[INFO]\tGene MIR640 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP69 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAH has 0 variants, skipping\n",
      "[INFO]\tGene WNT10B has 0 variants, skipping\n",
      "[INFO]\tGene YY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF251 has 0 variants, skipping\n",
      "[INFO]\tGene MFHAS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR50 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L6 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11B has 0 variants, skipping\n",
      "[INFO]\tGene MYH9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR575 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP0 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1322 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1277 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1269A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1266 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548G has 0 variants, skipping\n",
      "[INFO]\tGene MIR1253 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L1 has 0 variants, skipping\n",
      "[INFO]\tGene H6PD has 0 variants, skipping\n",
      "[INFO]\tGene CMTM5 has 0 variants, skipping\n",
      "[INFO]\tGene SHPK has 0 variants, skipping\n",
      "[INFO]\tGene GPR83 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6C has 0 variants, skipping\n",
      "[INFO]\tGene SUMO3 has 0 variants, skipping\n",
      "[INFO]\tGene YBEY has 0 variants, skipping\n",
      "[INFO]\tGene CFAP54 has 0 variants, skipping\n",
      "[INFO]\tGene MIR487A has 0 variants, skipping\n",
      "[INFO]\tGene MDH1B has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36 has 0 variants, skipping\n",
      "[INFO]\tGene WNT9B has 0 variants, skipping\n",
      "[INFO]\tGene WNT9A has 0 variants, skipping\n",
      "[INFO]\tGene SCGB3A1 has 0 variants, skipping\n",
      "[INFO]\tGene RFT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF28 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CDC37 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF22 has 0 variants, skipping\n",
      "[INFO]\tGene CNTF has 0 variants, skipping\n",
      "[INFO]\tGene MIR633 has 0 variants, skipping\n",
      "[INFO]\tGene PAX1 has 0 variants, skipping\n",
      "[INFO]\tGene PA2G4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF625 has 0 variants, skipping\n",
      "[INFO]\tGene ASGR2 has 0 variants, skipping\n",
      "[INFO]\tGene CWC15 has 0 variants, skipping\n",
      "[INFO]\tGene RBM48 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK1N has 0 variants, skipping\n",
      "[INFO]\tGene PRSS50 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS13 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M2 has 0 variants, skipping\n",
      "[INFO]\tGene AMH has 0 variants, skipping\n",
      "[INFO]\tGene NUF2 has 0 variants, skipping\n",
      "[INFO]\tGene COX8A has 0 variants, skipping\n",
      "[INFO]\tGene PLK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD5 has 0 variants, skipping\n",
      "[INFO]\tGene TNKS2 has 0 variants, skipping\n",
      "[INFO]\tGene CDH9 has 0 variants, skipping\n",
      "[INFO]\tGene ALX1 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A3 has 0 variants, skipping\n",
      "[INFO]\tGene DENND5A has 0 variants, skipping\n",
      "[INFO]\tGene MIR618 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT1 has 0 variants, skipping\n",
      "[INFO]\tGene CFP has 0 variants, skipping\n",
      "[INFO]\tGene PF4V1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL2 has 0 variants, skipping\n",
      "[INFO]\tGene PRIMA1 has 0 variants, skipping\n",
      "[INFO]\tGene MVB12B has 0 variants, skipping\n",
      "[INFO]\tGene HELZ2 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF10 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD2 has 0 variants, skipping\n",
      "[INFO]\tGene PPCDC has 0 variants, skipping\n",
      "[INFO]\tGene TICRR has 0 variants, skipping\n",
      "[INFO]\tGene TRIM6 has 0 variants, skipping\n",
      "[INFO]\tGene PROX2 has 0 variants, skipping\n",
      "[INFO]\tGene GALE has 0 variants, skipping\n",
      "[INFO]\tGene IL17C has 0 variants, skipping\n",
      "[INFO]\tGene DCPS has 0 variants, skipping\n",
      "[INFO]\tGene WDR37 has 0 variants, skipping\n",
      "[INFO]\tGene ALG3 has 0 variants, skipping\n",
      "[INFO]\tGene TAB2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP97 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D3 has 0 variants, skipping\n",
      "[INFO]\tGene PPARA has 0 variants, skipping\n",
      "[INFO]\tGene C1orf54 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45GIP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3M has 0 variants, skipping\n",
      "[INFO]\tGene ZNF805 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2C has 0 variants, skipping\n",
      "[INFO]\tGene FAM71E2 has 0 variants, skipping\n",
      "[INFO]\tGene RASL12 has 0 variants, skipping\n",
      "[INFO]\tGene PLCZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF773 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L2 has 0 variants, skipping\n",
      "[INFO]\tGene COQ7 has 0 variants, skipping\n",
      "[INFO]\tGene LIPM has 0 variants, skipping\n",
      "[INFO]\tGene DAZ2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM204 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1J has 0 variants, skipping\n",
      "[INFO]\tGene IP6K1 has 0 variants, skipping\n",
      "[INFO]\tGene PMF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A7 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH8 has 0 variants, skipping\n",
      "[INFO]\tGene NRL has 0 variants, skipping\n",
      "[INFO]\tGene NRAP has 0 variants, skipping\n",
      "[INFO]\tGene NSMF has 0 variants, skipping\n",
      "[INFO]\tGene ST20-MTHFS has 0 variants, skipping\n",
      "[INFO]\tGene OR6C65 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K4 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H15 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H14 has 0 variants, skipping\n",
      "[INFO]\tGene SENP5 has 0 variants, skipping\n",
      "[INFO]\tGene UPRT has 0 variants, skipping\n",
      "[INFO]\tGene FBXL4 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL3 has 0 variants, skipping\n",
      "[INFO]\tGene SUFU has 0 variants, skipping\n",
      "[INFO]\tGene NIPA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL20 has 0 variants, skipping\n",
      "[INFO]\tGene NRDE2 has 0 variants, skipping\n",
      "[INFO]\tGene IWS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D1 has 0 variants, skipping\n",
      "[INFO]\tGene GPN2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37C has 0 variants, skipping\n",
      "[INFO]\tGene AKIRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM3 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61G has 0 variants, skipping\n",
      "[INFO]\tGene CECR2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP4 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G5 has 0 variants, skipping\n",
      "[INFO]\tGene PKP3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF788P has 0 variants, skipping\n",
      "[INFO]\tGene H2BC3 has 0 variants, skipping\n",
      "[INFO]\tGene CLSPN has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF124 has 0 variants, skipping\n",
      "[INFO]\tGene EBAG9 has 0 variants, skipping\n",
      "[INFO]\tGene MAN2B2 has 0 variants, skipping\n",
      "[INFO]\tGene NDST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN4 has 0 variants, skipping\n",
      "[INFO]\tGene GPX8 has 0 variants, skipping\n",
      "[INFO]\tGene CKAP2L has 0 variants, skipping\n",
      "[INFO]\tGene DYRK2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS14 has 0 variants, skipping\n",
      "[INFO]\tGene ATN1 has 0 variants, skipping\n",
      "[INFO]\tGene PSLNR has 0 variants, skipping\n",
      "[INFO]\tGene AP3D1 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC106 has 0 variants, skipping\n",
      "[INFO]\tGene FAM216A has 0 variants, skipping\n",
      "[INFO]\tGene PEDS1-UBE2V1 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH3 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS12 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM269 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36 has 0 variants, skipping\n",
      "[INFO]\tGene ARL9 has 0 variants, skipping\n",
      "[INFO]\tGene LTA4H has 0 variants, skipping\n",
      "[INFO]\tGene GTSF1L has 0 variants, skipping\n",
      "[INFO]\tGene FSTL5 has 0 variants, skipping\n",
      "[INFO]\tGene HSPE1-MOB4 has 0 variants, skipping\n",
      "[INFO]\tGene STARD4 has 0 variants, skipping\n",
      "[INFO]\tGene DYNAP has 0 variants, skipping\n",
      "[INFO]\tGene RAB26 has 0 variants, skipping\n",
      "[INFO]\tGene SHD has 0 variants, skipping\n",
      "[INFO]\tGene PHETA2 has 0 variants, skipping\n",
      "[INFO]\tGene HBM has 0 variants, skipping\n",
      "[INFO]\tGene LMNA has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX8 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23D2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM210 has 0 variants, skipping\n",
      "[INFO]\tGene APPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB10 has 0 variants, skipping\n",
      "[INFO]\tGene PGGT1B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP46 has 0 variants, skipping\n",
      "[INFO]\tGene STK10 has 0 variants, skipping\n",
      "[INFO]\tGene MSL2 has 0 variants, skipping\n",
      "[INFO]\tGene RPTOR has 0 variants, skipping\n",
      "[INFO]\tGene TMTC3 has 0 variants, skipping\n",
      "[INFO]\tGene SDR39U1 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2L has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5L has 0 variants, skipping\n",
      "[INFO]\tGene RAB23 has 0 variants, skipping\n",
      "[INFO]\tGene TTC1 has 0 variants, skipping\n",
      "[INFO]\tGene XAF1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC5 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17B has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC11 has 0 variants, skipping\n",
      "[INFO]\tGene DSCC1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf49 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A23 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM109 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD3 has 0 variants, skipping\n",
      "[INFO]\tGene PCK2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236A has 0 variants, skipping\n",
      "[INFO]\tGene AHCYL2 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE1 has 0 variants, skipping\n",
      "[INFO]\tGene CST1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM32 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC58 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT9B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A13 has 0 variants, skipping\n",
      "[INFO]\tGene TEX35 has 0 variants, skipping\n",
      "[INFO]\tGene APOA4 has 0 variants, skipping\n",
      "[INFO]\tGene MMADHC-DT has 0 variants, skipping\n",
      "[INFO]\tGene HAS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00423 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1A3P has 0 variants, skipping\n",
      "[INFO]\tGene OR2L13 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CD86 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLA has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC4 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP6 has 0 variants, skipping\n",
      "[INFO]\tGene TERB2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB35 has 0 variants, skipping\n",
      "[INFO]\tGene IL10RB has 0 variants, skipping\n",
      "[INFO]\tGene NOS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL15 has 0 variants, skipping\n",
      "[INFO]\tGene SPHKAP has 0 variants, skipping\n",
      "[INFO]\tGene SON has 0 variants, skipping\n",
      "[INFO]\tGene ZNF764 has 0 variants, skipping\n",
      "[INFO]\tGene FOXS1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP5 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12 has 0 variants, skipping\n",
      "[INFO]\tGene FGF11 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF221 has 0 variants, skipping\n",
      "[INFO]\tGene PADI1 has 0 variants, skipping\n",
      "[INFO]\tGene COMP has 0 variants, skipping\n",
      "[INFO]\tGene COL7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLK has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN3 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP77 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2K has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC9 has 0 variants, skipping\n",
      "[INFO]\tGene JMY has 0 variants, skipping\n",
      "[INFO]\tGene CALHM4 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2B has 0 variants, skipping\n",
      "[INFO]\tGene CDK5 has 0 variants, skipping\n",
      "[INFO]\tGene POLRMT has 0 variants, skipping\n",
      "[INFO]\tGene TOB1 has 0 variants, skipping\n",
      "[INFO]\tGene SPHK2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC11 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA7 has 0 variants, skipping\n",
      "[INFO]\tGene MPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRE has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRL has 0 variants, skipping\n",
      "[INFO]\tGene PCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRA has 0 variants, skipping\n",
      "[INFO]\tGene PDE6H has 0 variants, skipping\n",
      "[INFO]\tGene PAX9 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6C has 0 variants, skipping\n",
      "[INFO]\tGene CLIP4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20DC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCP4 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CC has 0 variants, skipping\n",
      "[INFO]\tGene G3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene SIK2 has 0 variants, skipping\n",
      "[INFO]\tGene CDA has 0 variants, skipping\n",
      "[INFO]\tGene DOCK3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12 has 0 variants, skipping\n",
      "[INFO]\tGene MITD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A53 has 0 variants, skipping\n",
      "[INFO]\tGene DDX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPID has 0 variants, skipping\n",
      "[INFO]\tGene IHH has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE4 has 0 variants, skipping\n",
      "[INFO]\tGene AOPEP has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A3 has 0 variants, skipping\n",
      "[INFO]\tGene TONSL has 0 variants, skipping\n",
      "[INFO]\tGene ACO1 has 0 variants, skipping\n",
      "[INFO]\tGene INSM1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA5 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C68 has 0 variants, skipping\n",
      "[INFO]\tGene NSG2 has 0 variants, skipping\n",
      "[INFO]\tGene GLOD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM64 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC4-KLRK1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU2 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF138 has 0 variants, skipping\n",
      "[INFO]\tGene TULP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH7A1 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST3 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER1 has 0 variants, skipping\n",
      "[INFO]\tGene TTF1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18A has 0 variants, skipping\n",
      "[INFO]\tGene CENPS has 0 variants, skipping\n",
      "[INFO]\tGene RBMY2EP has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD27 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440570 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY2 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4E has 0 variants, skipping\n",
      "[INFO]\tGene NOS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESF1 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS12 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1I has 0 variants, skipping\n",
      "[INFO]\tGene FGF6 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAL4 has 0 variants, skipping\n",
      "[INFO]\tGene CRTC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00628 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA10 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG1 has 0 variants, skipping\n",
      "[INFO]\tGene OCM has 0 variants, skipping\n",
      "[INFO]\tGene LEFTY1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302B has 0 variants, skipping\n",
      "[INFO]\tGene SAA4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2G2 has 0 variants, skipping\n",
      "[INFO]\tGene ERVH48-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSLTR2 has 0 variants, skipping\n",
      "[INFO]\tGene NAA50 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP28 has 0 variants, skipping\n",
      "[INFO]\tGene TSR2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DRAIC has 0 variants, skipping\n",
      "[INFO]\tGene POC1B has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP has 0 variants, skipping\n",
      "[INFO]\tGene MIR3666 has 0 variants, skipping\n",
      "[INFO]\tGene GLI2 has 0 variants, skipping\n",
      "[INFO]\tGene MORC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01874 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01875 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373352 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01115 has 0 variants, skipping\n",
      "[INFO]\tGene SNTG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS26 has 0 variants, skipping\n",
      "[INFO]\tGene GPR45 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH2 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH3 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR176 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX1 has 0 variants, skipping\n",
      "[INFO]\tGene CA5B has 0 variants, skipping\n",
      "[INFO]\tGene RNF139 has 0 variants, skipping\n",
      "[INFO]\tGene HPS5 has 0 variants, skipping\n",
      "[INFO]\tGene POLG2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC63 has 0 variants, skipping\n",
      "[INFO]\tGene PRAF2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF2 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01250 has 0 variants, skipping\n",
      "[INFO]\tGene MSH2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01304 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373394 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01249 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01248 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01810 has 0 variants, skipping\n",
      "[INFO]\tGene SILC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7158 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400940 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01247 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01246 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7515HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR7515 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01857 has 0 variants, skipping\n",
      "[INFO]\tGene CHORDC1P4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929452 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506274 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929551 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01814 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC has 0 variants, skipping\n",
      "[INFO]\tGene FUCA1 has 0 variants, skipping\n",
      "[INFO]\tGene FTL has 0 variants, skipping\n",
      "[INFO]\tGene FSHR has 0 variants, skipping\n",
      "[INFO]\tGene FXN has 0 variants, skipping\n",
      "[INFO]\tGene FH has 0 variants, skipping\n",
      "[INFO]\tGene MS4A2 has 0 variants, skipping\n",
      "[INFO]\tGene FBN1 has 0 variants, skipping\n",
      "[INFO]\tGene HACD1 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM6 has 0 variants, skipping\n",
      "[INFO]\tGene VAT1L has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A9 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene XK has 0 variants, skipping\n",
      "[INFO]\tGene CLDN9 has 0 variants, skipping\n",
      "[INFO]\tGene HPS6 has 0 variants, skipping\n",
      "[INFO]\tGene HHIPL2 has 0 variants, skipping\n",
      "[INFO]\tGene SHCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR212 has 0 variants, skipping\n",
      "[INFO]\tGene MIR208B has 0 variants, skipping\n",
      "[INFO]\tGene MIR211 has 0 variants, skipping\n",
      "[INFO]\tGene MIR210 has 0 variants, skipping\n",
      "[INFO]\tGene MIR216B has 0 variants, skipping\n",
      "[INFO]\tGene MIR205 has 0 variants, skipping\n",
      "[INFO]\tGene MIR301B has 0 variants, skipping\n",
      "[INFO]\tGene MIR204 has 0 variants, skipping\n",
      "[INFO]\tGene MIR760 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO2A has 0 variants, skipping\n",
      "[INFO]\tGene DDX59-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCMAP-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00993 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOC has 0 variants, skipping\n",
      "[INFO]\tGene AARSD1 has 0 variants, skipping\n",
      "[INFO]\tGene CD38 has 0 variants, skipping\n",
      "[INFO]\tGene CD37 has 0 variants, skipping\n",
      "[INFO]\tGene CD34 has 0 variants, skipping\n",
      "[INFO]\tGene CD33 has 0 variants, skipping\n",
      "[INFO]\tGene CD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929882 has 0 variants, skipping\n",
      "[INFO]\tGene MYCBPAP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80B has 0 variants, skipping\n",
      "[INFO]\tGene ODC1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL832P has 0 variants, skipping\n",
      "[INFO]\tGene PDCD11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01954 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ33534 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR3 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR2 has 0 variants, skipping\n",
      "[INFO]\tGene CALB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00570 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD9 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC57 has 0 variants, skipping\n",
      "[INFO]\tGene NSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX17 has 0 variants, skipping\n",
      "[INFO]\tGene RNF25 has 0 variants, skipping\n",
      "[INFO]\tGene NOC3L has 0 variants, skipping\n",
      "[INFO]\tGene RHBDF1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB17 has 0 variants, skipping\n",
      "[INFO]\tGene TTC36 has 0 variants, skipping\n",
      "[INFO]\tGene OTOL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSF5 has 0 variants, skipping\n",
      "[INFO]\tGene A3GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548S has 0 variants, skipping\n",
      "[INFO]\tGene MIR3681HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3681 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3125 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506474 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00276 has 0 variants, skipping\n",
      "[INFO]\tGene RHCG has 0 variants, skipping\n",
      "[INFO]\tGene KCNH7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR151A has 0 variants, skipping\n",
      "[INFO]\tGene GNL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01804 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf98 has 0 variants, skipping\n",
      "[INFO]\tGene ASPG has 0 variants, skipping\n",
      "[INFO]\tGene GP9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO4 has 0 variants, skipping\n",
      "[INFO]\tGene CD274 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01376 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4757 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00954 has 0 variants, skipping\n",
      "[INFO]\tGene DLK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928222 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIQK has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985856 has 0 variants, skipping\n",
      "[INFO]\tGene HS1BP3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01822 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01884 has 0 variants, skipping\n",
      "[INFO]\tGene BRD1 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC30 has 0 variants, skipping\n",
      "[INFO]\tGene SYNDIG1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51AP2 has 0 variants, skipping\n",
      "[INFO]\tGene YAP1 has 0 variants, skipping\n",
      "[INFO]\tGene POGZ has 0 variants, skipping\n",
      "[INFO]\tGene CT45A8 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC187 has 0 variants, skipping\n",
      "[INFO]\tGene SMG6 has 0 variants, skipping\n",
      "[INFO]\tGene CNTLN has 0 variants, skipping\n",
      "[INFO]\tGene MUC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC171 has 0 variants, skipping\n",
      "[INFO]\tGene SKA2 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KA has 0 variants, skipping\n",
      "[INFO]\tGene ASRGL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC27-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01381 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF37 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT5 has 0 variants, skipping\n",
      "[INFO]\tGene RAB22A has 0 variants, skipping\n",
      "[INFO]\tGene FBXO27 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM4 has 0 variants, skipping\n",
      "[INFO]\tGene ARL5B has 0 variants, skipping\n",
      "[INFO]\tGene SMOX has 0 variants, skipping\n",
      "[INFO]\tGene TRANK1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA3 has 0 variants, skipping\n",
      "[INFO]\tGene LGI1 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf17 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT3 has 0 variants, skipping\n",
      "[INFO]\tGene CPLX1 has 0 variants, skipping\n",
      "[INFO]\tGene UTP14A has 0 variants, skipping\n",
      "[INFO]\tGene CPLX2 has 0 variants, skipping\n",
      "[INFO]\tGene STOML3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31E1 has 0 variants, skipping\n",
      "[INFO]\tGene RBCK1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSLTR1 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN4 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1B1 has 0 variants, skipping\n",
      "[INFO]\tGene HECA has 0 variants, skipping\n",
      "[INFO]\tGene DBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CUTALP has 0 variants, skipping\n",
      "[INFO]\tGene HAUS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL34-DT has 0 variants, skipping\n",
      "[INFO]\tGene TINCR has 0 variants, skipping\n",
      "[INFO]\tGene AGBL5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND5 has 0 variants, skipping\n",
      "[INFO]\tGene CDK12 has 0 variants, skipping\n",
      "[INFO]\tGene FOLR1 has 0 variants, skipping\n",
      "[INFO]\tGene PIEZO1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB37 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPTSSB has 0 variants, skipping\n",
      "[INFO]\tGene TRIM71 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C2 has 0 variants, skipping\n",
      "[INFO]\tGene COPS9 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM6-TRIM34 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B7 has 0 variants, skipping\n",
      "[INFO]\tGene CISH has 0 variants, skipping\n",
      "[INFO]\tGene SNX8 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZW10 has 0 variants, skipping\n",
      "[INFO]\tGene LRAT has 0 variants, skipping\n",
      "[INFO]\tGene MTA2 has 0 variants, skipping\n",
      "[INFO]\tGene TIAF1 has 0 variants, skipping\n",
      "[INFO]\tGene VAPB has 0 variants, skipping\n",
      "[INFO]\tGene XPR1 has 0 variants, skipping\n",
      "[INFO]\tGene PDE2A has 0 variants, skipping\n",
      "[INFO]\tGene SMAD5 has 0 variants, skipping\n",
      "[INFO]\tGene CNP has 0 variants, skipping\n",
      "[INFO]\tGene ZIC5 has 0 variants, skipping\n",
      "[INFO]\tGene WNT3A has 0 variants, skipping\n",
      "[INFO]\tGene SCRT2 has 0 variants, skipping\n",
      "[INFO]\tGene PSKH2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A16 has 0 variants, skipping\n",
      "[INFO]\tGene CA4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01460 has 0 variants, skipping\n",
      "[INFO]\tGene PNPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM7 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC13 has 0 variants, skipping\n",
      "[INFO]\tGene ADSS2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP8L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4263 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505736 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505716 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ31356 has 0 variants, skipping\n",
      "[INFO]\tGene MYH8 has 0 variants, skipping\n",
      "[INFO]\tGene MYH6 has 0 variants, skipping\n",
      "[INFO]\tGene MYH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00320 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD92 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD53 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD53B has 0 variants, skipping\n",
      "[INFO]\tGene CHMP5 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM144 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374389 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA10B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01936 has 0 variants, skipping\n",
      "[INFO]\tGene CCT3 has 0 variants, skipping\n",
      "[INFO]\tGene FHOD1 has 0 variants, skipping\n",
      "[INFO]\tGene N6AMT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3B has 0 variants, skipping\n",
      "[INFO]\tGene ACRV1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01946 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF142 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A48-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOK has 0 variants, skipping\n",
      "[INFO]\tGene KAT8 has 0 variants, skipping\n",
      "[INFO]\tGene ESPN has 0 variants, skipping\n",
      "[INFO]\tGene NRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene IL36G has 0 variants, skipping\n",
      "[INFO]\tGene GKN1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR45B has 0 variants, skipping\n",
      "[INFO]\tGene IRGC has 0 variants, skipping\n",
      "[INFO]\tGene CPXM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR558 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4765 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00486 has 0 variants, skipping\n",
      "[INFO]\tGene FAM168B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4430 has 0 variants, skipping\n",
      "[INFO]\tGene CRHBP has 0 variants, skipping\n",
      "[INFO]\tGene CR2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA6 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01317 has 0 variants, skipping\n",
      "[INFO]\tGene MYADML has 0 variants, skipping\n",
      "[INFO]\tGene LINC01318 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01320 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AD has 0 variants, skipping\n",
      "[INFO]\tGene CRIM1-DT has 0 variants, skipping\n",
      "[INFO]\tGene PTPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MED16 has 0 variants, skipping\n",
      "[INFO]\tGene JRKL has 0 variants, skipping\n",
      "[INFO]\tGene EXTL3 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf18 has 0 variants, skipping\n",
      "[INFO]\tGene ERVV-1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM199 has 0 variants, skipping\n",
      "[INFO]\tGene EME1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G3 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf89 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf77 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF785 has 0 variants, skipping\n",
      "[INFO]\tGene KCNU1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00211 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNN3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CYP1B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO5A has 0 variants, skipping\n",
      "[INFO]\tGene GPRASP1 has 0 variants, skipping\n",
      "[INFO]\tGene EHBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01883 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02613 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2D has 0 variants, skipping\n",
      "[INFO]\tGene SMR3A has 0 variants, skipping\n",
      "[INFO]\tGene LOC375196 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K3-DT has 0 variants, skipping\n",
      "[INFO]\tGene AGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene DAZL has 0 variants, skipping\n",
      "[INFO]\tGene ORC1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT31 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01913 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01914 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01819 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02580 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01126 has 0 variants, skipping\n",
      "[INFO]\tGene RFX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02476 has 0 variants, skipping\n",
      "[INFO]\tGene CSHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01833 has 0 variants, skipping\n",
      "[INFO]\tGene SIX3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01121 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01820 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02583 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01119 has 0 variants, skipping\n",
      "[INFO]\tGene KRAS has 0 variants, skipping\n",
      "[INFO]\tGene HBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520F has 0 variants, skipping\n",
      "[INFO]\tGene VCY1B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPCAM-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR559 has 0 variants, skipping\n",
      "[INFO]\tGene TBX10 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD23 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM190 has 0 variants, skipping\n",
      "[INFO]\tGene RCE1 has 0 variants, skipping\n",
      "[INFO]\tGene STT3A has 0 variants, skipping\n",
      "[INFO]\tGene C12orf77 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF473 has 0 variants, skipping\n",
      "[INFO]\tGene MMACHC has 0 variants, skipping\n",
      "[INFO]\tGene LINC01867 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3682 has 0 variants, skipping\n",
      "[INFO]\tGene PEBP4 has 0 variants, skipping\n",
      "[INFO]\tGene ACHE has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP32 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE3 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK11 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF622 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC46 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF662 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN7 has 0 variants, skipping\n",
      "[INFO]\tGene AXDND1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C75 has 0 variants, skipping\n",
      "[INFO]\tGene FPGT has 0 variants, skipping\n",
      "[INFO]\tGene FOXB2 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4B has 0 variants, skipping\n",
      "[INFO]\tGene FAM47C has 0 variants, skipping\n",
      "[INFO]\tGene OR4F15 has 0 variants, skipping\n",
      "[INFO]\tGene PRORSD1P has 0 variants, skipping\n",
      "[INFO]\tGene MNDA has 0 variants, skipping\n",
      "[INFO]\tGene MIR217HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC01813 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129434 has 0 variants, skipping\n",
      "[INFO]\tGene SHPRH has 0 variants, skipping\n",
      "[INFO]\tGene USP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01795 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01122 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4432HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4432 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01185 has 0 variants, skipping\n",
      "[INFO]\tGene DBT has 0 variants, skipping\n",
      "[INFO]\tGene CCER2 has 0 variants, skipping\n",
      "[INFO]\tGene VCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339803 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4E2 has 0 variants, skipping\n",
      "[INFO]\tGene AHSA2P has 0 variants, skipping\n",
      "[INFO]\tGene XIRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70B has 0 variants, skipping\n",
      "[INFO]\tGene SPARCL1 has 0 variants, skipping\n",
      "[INFO]\tGene C1D has 0 variants, skipping\n",
      "[INFO]\tGene MIR5192 has 0 variants, skipping\n",
      "[INFO]\tGene KPRP has 0 variants, skipping\n",
      "[INFO]\tGene CHI3L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929372 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf76 has 0 variants, skipping\n",
      "[INFO]\tGene MTCP1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF5 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL13 has 0 variants, skipping\n",
      "[INFO]\tGene ERFE has 0 variants, skipping\n",
      "[INFO]\tGene ABCC4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132215 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP20 has 0 variants, skipping\n",
      "[INFO]\tGene DBIL5P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCT7 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ8 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ4 has 0 variants, skipping\n",
      "[INFO]\tGene XBP1 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4433B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4433A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927402 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4434 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02579 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01800 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02245 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4778 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01873 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01798 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01799 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01628 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01828 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01829 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927661 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01812 has 0 variants, skipping\n",
      "[INFO]\tGene GPR143 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD2B has 0 variants, skipping\n",
      "[INFO]\tGene AURKAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO3 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA5 has 0 variants, skipping\n",
      "[INFO]\tGene PLAGL2 has 0 variants, skipping\n",
      "[INFO]\tGene PLAG1 has 0 variants, skipping\n",
      "[INFO]\tGene EFEMP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3126 has 0 variants, skipping\n",
      "[INFO]\tGene PARVB has 0 variants, skipping\n",
      "[INFO]\tGene RNF224 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36C has 0 variants, skipping\n",
      "[INFO]\tGene CLDN8 has 0 variants, skipping\n",
      "[INFO]\tGene HENMT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01816 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf92 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01224 has 0 variants, skipping\n",
      "[INFO]\tGene EXOG has 0 variants, skipping\n",
      "[INFO]\tGene ABCA12 has 0 variants, skipping\n",
      "[INFO]\tGene GFOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01354 has 0 variants, skipping\n",
      "[INFO]\tGene TRHR has 0 variants, skipping\n",
      "[INFO]\tGene TGFA-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LEFTY2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01143 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E91P has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC9B has 0 variants, skipping\n",
      "[INFO]\tGene TOMM20L has 0 variants, skipping\n",
      "[INFO]\tGene CPT2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL7B has 0 variants, skipping\n",
      "[INFO]\tGene METTL15 has 0 variants, skipping\n",
      "[INFO]\tGene OIT3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47B has 0 variants, skipping\n",
      "[INFO]\tGene SNX31 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF366 has 0 variants, skipping\n",
      "[INFO]\tGene MIER3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM60 has 0 variants, skipping\n",
      "[INFO]\tGene RNF168 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM42 has 0 variants, skipping\n",
      "[INFO]\tGene TEX44 has 0 variants, skipping\n",
      "[INFO]\tGene WBP2NL has 0 variants, skipping\n",
      "[INFO]\tGene LRRN4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC185 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD6L has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF10 has 0 variants, skipping\n",
      "[INFO]\tGene SDE2 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP1 has 0 variants, skipping\n",
      "[INFO]\tGene EXD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF781 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD4 has 0 variants, skipping\n",
      "[INFO]\tGene SPRED1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF709 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf50 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF579 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf65 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A2 has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD2 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1NB has 0 variants, skipping\n",
      "[INFO]\tGene CBLL2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39B has 0 variants, skipping\n",
      "[INFO]\tGene RASEF has 0 variants, skipping\n",
      "[INFO]\tGene C9orf163 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0D2 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL27 has 0 variants, skipping\n",
      "[INFO]\tGene BMT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP48 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM81B has 0 variants, skipping\n",
      "[INFO]\tGene BTNL9 has 0 variants, skipping\n",
      "[INFO]\tGene SRFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf45 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1K has 0 variants, skipping\n",
      "[INFO]\tGene TEX55 has 0 variants, skipping\n",
      "[INFO]\tGene SCFD2 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02729 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4497 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4490 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4465 has 0 variants, skipping\n",
      "[INFO]\tGene PLXND1 has 0 variants, skipping\n",
      "[INFO]\tGene DAAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZER1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR564 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4775 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4496 has 0 variants, skipping\n",
      "[INFO]\tGene PRADC1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPSL has 0 variants, skipping\n",
      "[INFO]\tGene JCHAIN has 0 variants, skipping\n",
      "[INFO]\tGene SPATA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1P1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8B has 0 variants, skipping\n",
      "[INFO]\tGene ULK2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf12 has 0 variants, skipping\n",
      "[INFO]\tGene MELTF has 0 variants, skipping\n",
      "[INFO]\tGene PNMA1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB7 has 0 variants, skipping\n",
      "[INFO]\tGene TMCC3 has 0 variants, skipping\n",
      "[INFO]\tGene CD247 has 0 variants, skipping\n",
      "[INFO]\tGene DGUOK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VCX3A has 0 variants, skipping\n",
      "[INFO]\tGene VCX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR297 has 0 variants, skipping\n",
      "[INFO]\tGene ANKLE1 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO4 has 0 variants, skipping\n",
      "[INFO]\tGene AWAT2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS37 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM89 has 0 variants, skipping\n",
      "[INFO]\tGene MISP has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB41 has 0 variants, skipping\n",
      "[INFO]\tGene STX11 has 0 variants, skipping\n",
      "[INFO]\tGene INO80B-WBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SETD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR889 has 0 variants, skipping\n",
      "[INFO]\tGene MIR147A has 0 variants, skipping\n",
      "[INFO]\tGene MIR139 has 0 variants, skipping\n",
      "[INFO]\tGene MIR892B has 0 variants, skipping\n",
      "[INFO]\tGene MIR874 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450B has 0 variants, skipping\n",
      "[INFO]\tGene MIR199A1 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC has 0 variants, skipping\n",
      "[INFO]\tGene CHORDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01291 has 0 variants, skipping\n",
      "[INFO]\tGene CDK2AP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5000 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927884 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927967 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01851 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-H has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927948 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374820 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCC1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB3 has 0 variants, skipping\n",
      "[INFO]\tGene DDOST has 0 variants, skipping\n",
      "[INFO]\tGene DCC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4264 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01815 has 0 variants, skipping\n",
      "[INFO]\tGene DHFRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01809 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA4 has 0 variants, skipping\n",
      "[INFO]\tGene PWWP2B has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6 has 0 variants, skipping\n",
      "[INFO]\tGene HS2ST1 has 0 variants, skipping\n",
      "[INFO]\tGene GBP5 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01964 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724579 has 0 variants, skipping\n",
      "[INFO]\tGene NBEAL2 has 0 variants, skipping\n",
      "[INFO]\tGene CRISP2 has 0 variants, skipping\n",
      "[INFO]\tGene TCFL5 has 0 variants, skipping\n",
      "[INFO]\tGene BPTF has 0 variants, skipping\n",
      "[INFO]\tGene CDY2B has 0 variants, skipping\n",
      "[INFO]\tGene OSGIN2 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H6 has 0 variants, skipping\n",
      "[INFO]\tGene NGFR has 0 variants, skipping\n",
      "[INFO]\tGene NEK2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL28 has 0 variants, skipping\n",
      "[INFO]\tGene GIPC2 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF586 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R9A has 0 variants, skipping\n",
      "[INFO]\tGene FTSJ3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6071 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284950 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD94 has 0 variants, skipping\n",
      "[INFO]\tGene QSER1 has 0 variants, skipping\n",
      "[INFO]\tGene CA3 has 0 variants, skipping\n",
      "[INFO]\tGene BMI1 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3A has 0 variants, skipping\n",
      "[INFO]\tGene HSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4771-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4771-2 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB561A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4780 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928371 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK3-DT has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927050 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P4 has 0 variants, skipping\n",
      "[INFO]\tGene GGT8P has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133920 has 0 variants, skipping\n",
      "[INFO]\tGene FAM95A has 0 variants, skipping\n",
      "[INFO]\tGene LOC442028 has 0 variants, skipping\n",
      "[INFO]\tGene INSRR has 0 variants, skipping\n",
      "[INFO]\tGene C3orf49 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2CP has 0 variants, skipping\n",
      "[INFO]\tGene HYI has 0 variants, skipping\n",
      "[INFO]\tGene MRPS35 has 0 variants, skipping\n",
      "[INFO]\tGene STARD7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA112 has 0 variants, skipping\n",
      "[INFO]\tGene OR4M2 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373496 has 0 variants, skipping\n",
      "[INFO]\tGene MCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN11 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372440 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2B2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC2 has 0 variants, skipping\n",
      "[INFO]\tGene PROSER2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2A has 0 variants, skipping\n",
      "[INFO]\tGene CNNM3-DT has 0 variants, skipping\n",
      "[INFO]\tGene EPAS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927053 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN5 has 0 variants, skipping\n",
      "[INFO]\tGene FTCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506076 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506123 has 0 variants, skipping\n",
      "[INFO]\tGene TGM6 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf92 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10C has 0 variants, skipping\n",
      "[INFO]\tGene LINC02611 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00304 has 0 variants, skipping\n",
      "[INFO]\tGene ILF3-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR662 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX3 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST3B1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO16 has 0 variants, skipping\n",
      "[INFO]\tGene HMBS has 0 variants, skipping\n",
      "[INFO]\tGene TSKU has 0 variants, skipping\n",
      "[INFO]\tGene SMIM18 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF19 has 0 variants, skipping\n",
      "[INFO]\tGene GPRACR has 0 variants, skipping\n",
      "[INFO]\tGene DCLK2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01104 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01849 has 0 variants, skipping\n",
      "[INFO]\tGene AMER1 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL2 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L10 has 0 variants, skipping\n",
      "[INFO]\tGene ALS2CL has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5696 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF394 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01127 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4X has 0 variants, skipping\n",
      "[INFO]\tGene IFT74 has 0 variants, skipping\n",
      "[INFO]\tGene LSM12 has 0 variants, skipping\n",
      "[INFO]\tGene SRP72 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB34 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC43 has 0 variants, skipping\n",
      "[INFO]\tGene CUEDC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4772 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA11 has 0 variants, skipping\n",
      "[INFO]\tGene RETN has 0 variants, skipping\n",
      "[INFO]\tGene FABP2 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01796 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01935 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287010 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01831 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01102 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01103 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01114 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01159 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01918 has 0 variants, skipping\n",
      "[INFO]\tGene NGEF has 0 variants, skipping\n",
      "[INFO]\tGene LOC285000 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLA has 0 variants, skipping\n",
      "[INFO]\tGene OR4K5 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K15 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F5 has 0 variants, skipping\n",
      "[INFO]\tGene OR13G1 has 0 variants, skipping\n",
      "[INFO]\tGene ILDR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AU has 0 variants, skipping\n",
      "[INFO]\tGene LINC01886 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A38 has 0 variants, skipping\n",
      "[INFO]\tGene ASB6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01594 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C2P1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL2 has 0 variants, skipping\n",
      "[INFO]\tGene CABP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM229B has 0 variants, skipping\n",
      "[INFO]\tGene GCC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VTCN1 has 0 variants, skipping\n",
      "[INFO]\tGene CABP2 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA1 has 0 variants, skipping\n",
      "[INFO]\tGene PHC2 has 0 variants, skipping\n",
      "[INFO]\tGene PHC1 has 0 variants, skipping\n",
      "[INFO]\tGene ECM1 has 0 variants, skipping\n",
      "[INFO]\tGene E4F1 has 0 variants, skipping\n",
      "[INFO]\tGene DVL3 has 0 variants, skipping\n",
      "[INFO]\tGene RBMX2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2J1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00161 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193BHG has 0 variants, skipping\n",
      "[INFO]\tGene MGAT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF22-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4265 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4266 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268219 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4S2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K5 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM118B has 0 variants, skipping\n",
      "[INFO]\tGene CERS4 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS3-LOC440895 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440895 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288570 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01123 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4267 has 0 variants, skipping\n",
      "[INFO]\tGene TMED5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF667-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507334 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01106 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD2B has 0 variants, skipping\n",
      "[INFO]\tGene AHNAK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373553 has 0 variants, skipping\n",
      "[INFO]\tGene ALAS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD132 has 0 variants, skipping\n",
      "[INFO]\tGene ACOXL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AIF1L has 0 variants, skipping\n",
      "[INFO]\tGene NUDC has 0 variants, skipping\n",
      "[INFO]\tGene NFATC3 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2R has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC13 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF3 has 0 variants, skipping\n",
      "[INFO]\tGene COG4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373562 has 0 variants, skipping\n",
      "[INFO]\tGene CHAC2 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF8 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42351 has 0 variants, skipping\n",
      "[INFO]\tGene PET117 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC1B has 0 variants, skipping\n",
      "[INFO]\tGene LRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A11 has 0 variants, skipping\n",
      "[INFO]\tGene UROD has 0 variants, skipping\n",
      "[INFO]\tGene UMPS has 0 variants, skipping\n",
      "[INFO]\tGene TYR has 0 variants, skipping\n",
      "[INFO]\tGene TTR has 0 variants, skipping\n",
      "[INFO]\tGene TTPA has 0 variants, skipping\n",
      "[INFO]\tGene TSHR has 0 variants, skipping\n",
      "[INFO]\tGene TRIM23 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC12 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene HYOU1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138B has 0 variants, skipping\n",
      "[INFO]\tGene WASH2P has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP7 has 0 variants, skipping\n",
      "[INFO]\tGene TLR4 has 0 variants, skipping\n",
      "[INFO]\tGene TLR3 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE3 has 0 variants, skipping\n",
      "[INFO]\tGene MFN2 has 0 variants, skipping\n",
      "[INFO]\tGene R3HCC1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL11B has 0 variants, skipping\n",
      "[INFO]\tGene ATXN3L has 0 variants, skipping\n",
      "[INFO]\tGene SCRN2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4782 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01191 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100499194 has 0 variants, skipping\n",
      "[INFO]\tGene TARP has 0 variants, skipping\n",
      "[INFO]\tGene POTEE has 0 variants, skipping\n",
      "[INFO]\tGene DPP10-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene DPP10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A11 has 0 variants, skipping\n",
      "[INFO]\tGene MDFIC has 0 variants, skipping\n",
      "[INFO]\tGene RNF212B has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985939 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927709 has 0 variants, skipping\n",
      "[INFO]\tGene EVI2B has 0 variants, skipping\n",
      "[INFO]\tGene STEAP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD3 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAX has 0 variants, skipping\n",
      "[INFO]\tGene TRIP13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01101 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00908 has 0 variants, skipping\n",
      "[INFO]\tGene CASC15 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01826 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985820 has 0 variants, skipping\n",
      "[INFO]\tGene KARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01941 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM35B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287896 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS7 has 0 variants, skipping\n",
      "[INFO]\tGene HSBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene BET1L has 0 variants, skipping\n",
      "[INFO]\tGene SYT17 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB11 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT12 has 0 variants, skipping\n",
      "[INFO]\tGene NODAL has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX5 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD3 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4783 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373609 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA4 has 0 variants, skipping\n",
      "[INFO]\tGene MAG has 0 variants, skipping\n",
      "[INFO]\tGene MAFK has 0 variants, skipping\n",
      "[INFO]\tGene MAFG has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCKS has 0 variants, skipping\n",
      "[INFO]\tGene M6PR has 0 variants, skipping\n",
      "[INFO]\tGene EPCAM has 0 variants, skipping\n",
      "[INFO]\tGene TACSTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D1A has 0 variants, skipping\n",
      "[INFO]\tGene LYN has 0 variants, skipping\n",
      "[INFO]\tGene LY75 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB6 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD4 has 0 variants, skipping\n",
      "[INFO]\tGene CRTAM has 0 variants, skipping\n",
      "[INFO]\tGene GPCPD1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RC has 0 variants, skipping\n",
      "[INFO]\tGene PARM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927881 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01854 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02572 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC9P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01856 has 0 variants, skipping\n",
      "[INFO]\tGene HCAR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAR2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MED15P9 has 0 variants, skipping\n",
      "[INFO]\tGene CEP152 has 0 variants, skipping\n",
      "[INFO]\tGene RAG2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR183 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF844 has 0 variants, skipping\n",
      "[INFO]\tGene CDK4 has 0 variants, skipping\n",
      "[INFO]\tGene CD40LG has 0 variants, skipping\n",
      "[INFO]\tGene FAR2P2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F62P has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A10 has 0 variants, skipping\n",
      "[INFO]\tGene UNG has 0 variants, skipping\n",
      "[INFO]\tGene DBN1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F30P has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK12 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01120 has 0 variants, skipping\n",
      "[INFO]\tGene NOC2LP2 has 0 variants, skipping\n",
      "[INFO]\tGene CTRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4784 has 0 variants, skipping\n",
      "[INFO]\tGene GINM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF653 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-4 has 0 variants, skipping\n",
      "[INFO]\tGene POTEKP has 0 variants, skipping\n",
      "[INFO]\tGene IRAG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF9 has 0 variants, skipping\n",
      "[INFO]\tGene CHD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01945 has 0 variants, skipping\n",
      "[INFO]\tGene FAM201B has 0 variants, skipping\n",
      "[INFO]\tGene KIF4B has 0 variants, skipping\n",
      "[INFO]\tGene USP20 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5IF1 has 0 variants, skipping\n",
      "[INFO]\tGene NME1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT5A has 0 variants, skipping\n",
      "[INFO]\tGene PWP2 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40B has 0 variants, skipping\n",
      "[INFO]\tGene MMP8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132E has 0 variants, skipping\n",
      "[INFO]\tGene PRPSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR302C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928273 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01832 has 0 variants, skipping\n",
      "[INFO]\tGene YY1P2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1 has 0 variants, skipping\n",
      "[INFO]\tGene ERN2 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7157 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928386 has 0 variants, skipping\n",
      "[INFO]\tGene MST1L has 0 variants, skipping\n",
      "[INFO]\tGene C5AR2 has 0 variants, skipping\n",
      "[INFO]\tGene APEX2 has 0 variants, skipping\n",
      "[INFO]\tGene FHOD3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2C has 0 variants, skipping\n",
      "[INFO]\tGene BTK has 0 variants, skipping\n",
      "[INFO]\tGene BRCA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01412 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373656 has 0 variants, skipping\n",
      "[INFO]\tGene TEX41 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1P2 has 0 variants, skipping\n",
      "[INFO]\tGene PROC has 0 variants, skipping\n",
      "[INFO]\tGene PKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TICAM1 has 0 variants, skipping\n",
      "[INFO]\tGene TRNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM50A has 0 variants, skipping\n",
      "[INFO]\tGene PDE7B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01931 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01817 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01920 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02612 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929319 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4773-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4773-2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1217 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF561 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF562 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN29 has 0 variants, skipping\n",
      "[INFO]\tGene FAM81A has 0 variants, skipping\n",
      "[INFO]\tGene DCST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF362 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC27 has 0 variants, skipping\n",
      "[INFO]\tGene CST6 has 0 variants, skipping\n",
      "[INFO]\tGene CST2 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G2 has 0 variants, skipping\n",
      "[INFO]\tGene CSH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK14 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CREBL2 has 0 variants, skipping\n",
      "[INFO]\tGene CNN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNGA3 has 0 variants, skipping\n",
      "[INFO]\tGene CPN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNGB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR2 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN3 has 0 variants, skipping\n",
      "[INFO]\tGene CCR1 has 0 variants, skipping\n",
      "[INFO]\tGene CLPTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LRP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6X1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM66 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3J has 0 variants, skipping\n",
      "[INFO]\tGene MPPED1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO6 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB11 has 0 variants, skipping\n",
      "[INFO]\tGene RAPH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01876 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46875 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3A_B has 0 variants, skipping\n",
      "[INFO]\tGene FAM133DP has 0 variants, skipping\n",
      "[INFO]\tGene CENPX has 0 variants, skipping\n",
      "[INFO]\tGene P3H2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC148-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AXIN2 has 0 variants, skipping\n",
      "[INFO]\tGene USP9Y has 0 variants, skipping\n",
      "[INFO]\tGene PKP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC14B has 0 variants, skipping\n",
      "[INFO]\tGene TENM3 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA3 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6888 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM128 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF337 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL35 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643072 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD2 has 0 variants, skipping\n",
      "[INFO]\tGene SIDT2 has 0 variants, skipping\n",
      "[INFO]\tGene LETM1 has 0 variants, skipping\n",
      "[INFO]\tGene LDOC1 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA6 has 0 variants, skipping\n",
      "[INFO]\tGene KLK9 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02478 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF737 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4785 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929512 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01806 has 0 variants, skipping\n",
      "[INFO]\tGene AHCTF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A6 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL1 has 0 variants, skipping\n",
      "[INFO]\tGene HEY2 has 0 variants, skipping\n",
      "[INFO]\tGene HEY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF212 has 0 variants, skipping\n",
      "[INFO]\tGene XRN2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1E1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI3 has 0 variants, skipping\n",
      "[INFO]\tGene CRNKL1 has 0 variants, skipping\n",
      "[INFO]\tGene GOT2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132B has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RITA1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM45 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2C has 0 variants, skipping\n",
      "[INFO]\tGene CCDC142 has 0 variants, skipping\n",
      "[INFO]\tGene PARM1 has 0 variants, skipping\n",
      "[INFO]\tGene AADAT has 0 variants, skipping\n",
      "[INFO]\tGene TFRC has 0 variants, skipping\n",
      "[INFO]\tGene PADI3 has 0 variants, skipping\n",
      "[INFO]\tGene USPL1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929532 has 0 variants, skipping\n",
      "[INFO]\tGene NXF3 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR88 has 0 variants, skipping\n",
      "[INFO]\tGene BCAN has 0 variants, skipping\n",
      "[INFO]\tGene PRSS57 has 0 variants, skipping\n",
      "[INFO]\tGene MIR944 has 0 variants, skipping\n",
      "[INFO]\tGene SOX6 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2 has 0 variants, skipping\n",
      "[INFO]\tGene NCR2 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70F has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929633 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP12 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM2 has 0 variants, skipping\n",
      "[INFO]\tGene ELP5 has 0 variants, skipping\n",
      "[INFO]\tGene GNAZ has 0 variants, skipping\n",
      "[INFO]\tGene ZCWPW1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF358 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD4 has 0 variants, skipping\n",
      "[INFO]\tGene SRBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506124 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf42 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN3 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR5 has 0 variants, skipping\n",
      "[INFO]\tGene TTC21B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724058 has 0 variants, skipping\n",
      "[INFO]\tGene KLKB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF624 has 0 variants, skipping\n",
      "[INFO]\tGene TBL2 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SENP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4774 has 0 variants, skipping\n",
      "[INFO]\tGene CERS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM61 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA13 has 0 variants, skipping\n",
      "[INFO]\tGene BZW1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3K has 0 variants, skipping\n",
      "[INFO]\tGene LRRC7 has 0 variants, skipping\n",
      "[INFO]\tGene MAFG-DT has 0 variants, skipping\n",
      "[INFO]\tGene MYO3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926913 has 0 variants, skipping\n",
      "[INFO]\tGene VDAC3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26 has 0 variants, skipping\n",
      "[INFO]\tGene DCXR has 0 variants, skipping\n",
      "[INFO]\tGene AK6 has 0 variants, skipping\n",
      "[INFO]\tGene FOXK2 has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXA3 has 0 variants, skipping\n",
      "[INFO]\tGene HUS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRP8 has 0 variants, skipping\n",
      "[INFO]\tGene UFL1 has 0 variants, skipping\n",
      "[INFO]\tGene FEM1B has 0 variants, skipping\n",
      "[INFO]\tGene BFAR has 0 variants, skipping\n",
      "[INFO]\tGene POLR2E has 0 variants, skipping\n",
      "[INFO]\tGene GPR42 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK1 has 0 variants, skipping\n",
      "[INFO]\tGene CKLF has 0 variants, skipping\n",
      "[INFO]\tGene FIG4 has 0 variants, skipping\n",
      "[INFO]\tGene TECPR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A3 has 0 variants, skipping\n",
      "[INFO]\tGene NOL9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DLX2-DT has 0 variants, skipping\n",
      "[INFO]\tGene GPR182 has 0 variants, skipping\n",
      "[INFO]\tGene COPE has 0 variants, skipping\n",
      "[INFO]\tGene RTRAF has 0 variants, skipping\n",
      "[INFO]\tGene SBDS has 0 variants, skipping\n",
      "[INFO]\tGene NMD3 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R1A has 0 variants, skipping\n",
      "[INFO]\tGene PERCC1 has 0 variants, skipping\n",
      "[INFO]\tGene BMPER has 0 variants, skipping\n",
      "[INFO]\tGene LOC112694756 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01960 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01305 has 0 variants, skipping\n",
      "[INFO]\tGene WDR81 has 0 variants, skipping\n",
      "[INFO]\tGene MROH9 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF4 has 0 variants, skipping\n",
      "[INFO]\tGene HIRA has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC74B has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA11 has 0 variants, skipping\n",
      "[INFO]\tGene CYLC1 has 0 variants, skipping\n",
      "[INFO]\tGene CRY2 has 0 variants, skipping\n",
      "[INFO]\tGene TPM4 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401021 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf53 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7704 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01117 has 0 variants, skipping\n",
      "[INFO]\tGene NUP35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4444-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4444-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130691 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6512 has 0 variants, skipping\n",
      "[INFO]\tGene GAP43 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385C has 0 variants, skipping\n",
      "[INFO]\tGene DGKI has 0 variants, skipping\n",
      "[INFO]\tGene OBI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NADSYN1 has 0 variants, skipping\n",
      "[INFO]\tGene BACE2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A1AP has 0 variants, skipping\n",
      "[INFO]\tGene ASF1B has 0 variants, skipping\n",
      "[INFO]\tGene ANTXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR11 has 0 variants, skipping\n",
      "[INFO]\tGene PVR has 0 variants, skipping\n",
      "[INFO]\tGene TRAF1 has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC1B has 0 variants, skipping\n",
      "[INFO]\tGene SCLY has 0 variants, skipping\n",
      "[INFO]\tGene MTFP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL51 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL27 has 0 variants, skipping\n",
      "[INFO]\tGene RNF181 has 0 variants, skipping\n",
      "[INFO]\tGene RSC1A1 has 0 variants, skipping\n",
      "[INFO]\tGene RELB has 0 variants, skipping\n",
      "[INFO]\tGene MCL1 has 0 variants, skipping\n",
      "[INFO]\tGene MACC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUMF1 has 0 variants, skipping\n",
      "[INFO]\tGene BUB1 has 0 variants, skipping\n",
      "[INFO]\tGene BST2 has 0 variants, skipping\n",
      "[INFO]\tGene BST1 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD4 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD4 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP3L has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H7A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2T has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927055 has 0 variants, skipping\n",
      "[INFO]\tGene SUV39H1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7L1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO9A has 0 variants, skipping\n",
      "[INFO]\tGene EGOT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4B has 0 variants, skipping\n",
      "[INFO]\tGene ACSM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01934 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM1 has 0 variants, skipping\n",
      "[INFO]\tGene BDP1 has 0 variants, skipping\n",
      "[INFO]\tGene CBX3 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP10 has 0 variants, skipping\n",
      "[INFO]\tGene MYBL1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1LI1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR99A has 0 variants, skipping\n",
      "[INFO]\tGene CEP162 has 0 variants, skipping\n",
      "[INFO]\tGene CRLS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRADD has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12A has 0 variants, skipping\n",
      "[INFO]\tGene FBXW4 has 0 variants, skipping\n",
      "[INFO]\tGene FST has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD2 has 0 variants, skipping\n",
      "[INFO]\tGene DLX1 has 0 variants, skipping\n",
      "[INFO]\tGene SIM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AE1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373782 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01473 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN2 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNC1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3GAP2 has 0 variants, skipping\n",
      "[INFO]\tGene QPCT has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01090 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR1245A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1245B has 0 variants, skipping\n",
      "[INFO]\tGene SNORC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF790 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A10 has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B7 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP7 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNB has 0 variants, skipping\n",
      "[INFO]\tGene OSGEPL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP74 has 0 variants, skipping\n",
      "[INFO]\tGene ENG has 0 variants, skipping\n",
      "[INFO]\tGene ZEB2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A18 has 0 variants, skipping\n",
      "[INFO]\tGene ORC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101059915 has 0 variants, skipping\n",
      "[INFO]\tGene NGF has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS8 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS4 has 0 variants, skipping\n",
      "[INFO]\tGene NFX1 has 0 variants, skipping\n",
      "[INFO]\tGene SMDT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01821 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01790 has 0 variants, skipping\n",
      "[INFO]\tGene HECW2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130452 has 0 variants, skipping\n",
      "[INFO]\tGene HAND2 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN3 has 0 variants, skipping\n",
      "[INFO]\tGene GMPPB has 0 variants, skipping\n",
      "[INFO]\tGene NR0B2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR3 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR6 has 0 variants, skipping\n",
      "[INFO]\tGene PIEZO2 has 0 variants, skipping\n",
      "[INFO]\tGene MT1JP has 0 variants, skipping\n",
      "[INFO]\tGene ZFAS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFC has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene GNPDA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105B has 0 variants, skipping\n",
      "[INFO]\tGene SAMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP4 has 0 variants, skipping\n",
      "[INFO]\tGene WAS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01923 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01877 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG3 has 0 variants, skipping\n",
      "[INFO]\tGene RIN3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL18A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH5A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM40 has 0 variants, skipping\n",
      "[INFO]\tGene AOX3P-AOX2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01792 has 0 variants, skipping\n",
      "[INFO]\tGene LTO1 has 0 variants, skipping\n",
      "[INFO]\tGene MED19 has 0 variants, skipping\n",
      "[INFO]\tGene ESX1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP5 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene BEND5 has 0 variants, skipping\n",
      "[INFO]\tGene CMKLR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375556 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02429 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927278 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM231 has 0 variants, skipping\n",
      "[INFO]\tGene GK5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02664 has 0 variants, skipping\n",
      "[INFO]\tGene CFLAR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCN9 has 0 variants, skipping\n",
      "[INFO]\tGene BMP5 has 0 variants, skipping\n",
      "[INFO]\tGene AQP6 has 0 variants, skipping\n",
      "[INFO]\tGene AQP5 has 0 variants, skipping\n",
      "[INFO]\tGene ARF6 has 0 variants, skipping\n",
      "[INFO]\tGene ARF5 has 0 variants, skipping\n",
      "[INFO]\tGene ARL4D has 0 variants, skipping\n",
      "[INFO]\tGene ARF3 has 0 variants, skipping\n",
      "[INFO]\tGene ARF4 has 0 variants, skipping\n",
      "[INFO]\tGene ARF1 has 0 variants, skipping\n",
      "[INFO]\tGene AREG has 0 variants, skipping\n",
      "[INFO]\tGene ARCN1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2012-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD70 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD70B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD11B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD11 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2B has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW has 0 variants, skipping\n",
      "[INFO]\tGene FUT2 has 0 variants, skipping\n",
      "[INFO]\tGene FGG has 0 variants, skipping\n",
      "[INFO]\tGene FGA has 0 variants, skipping\n",
      "[INFO]\tGene CYP11B2 has 0 variants, skipping\n",
      "[INFO]\tGene FBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP11B1 has 0 variants, skipping\n",
      "[INFO]\tGene F2 has 0 variants, skipping\n",
      "[INFO]\tGene KERA has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A3 has 0 variants, skipping\n",
      "[INFO]\tGene RER1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT10 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM11A has 0 variants, skipping\n",
      "[INFO]\tGene PSAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA4 has 0 variants, skipping\n",
      "[INFO]\tGene ALAS2 has 0 variants, skipping\n",
      "[INFO]\tGene HPGDS has 0 variants, skipping\n",
      "[INFO]\tGene MOCS3 has 0 variants, skipping\n",
      "[INFO]\tGene BMP10 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG5 has 0 variants, skipping\n",
      "[INFO]\tGene DHDH has 0 variants, skipping\n",
      "[INFO]\tGene SMPDL3B has 0 variants, skipping\n",
      "[INFO]\tGene SPINK4 has 0 variants, skipping\n",
      "[INFO]\tGene RND1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL2 has 0 variants, skipping\n",
      "[INFO]\tGene VENTX has 0 variants, skipping\n",
      "[INFO]\tGene SRPX2 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT2 has 0 variants, skipping\n",
      "[INFO]\tGene VASH1 has 0 variants, skipping\n",
      "[INFO]\tGene GCSHP3 has 0 variants, skipping\n",
      "[INFO]\tGene RPP40 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCL has 0 variants, skipping\n",
      "[INFO]\tGene SNORD51 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA41 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02731 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P9 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2B has 0 variants, skipping\n",
      "[INFO]\tGene GRIK3 has 0 variants, skipping\n",
      "[INFO]\tGene CIC has 0 variants, skipping\n",
      "[INFO]\tGene MIR3130-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3130-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7845 has 0 variants, skipping\n",
      "[INFO]\tGene MYOSLID-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-4 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF3 has 0 variants, skipping\n",
      "[INFO]\tGene RICTOR has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507443 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM40 has 0 variants, skipping\n",
      "[INFO]\tGene IDH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM185A has 0 variants, skipping\n",
      "[INFO]\tGene SLC35C1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927960 has 0 variants, skipping\n",
      "[INFO]\tGene RYBP has 0 variants, skipping\n",
      "[INFO]\tGene C7orf33 has 0 variants, skipping\n",
      "[INFO]\tGene PIDD1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR3 has 0 variants, skipping\n",
      "[INFO]\tGene SYCE2 has 0 variants, skipping\n",
      "[INFO]\tGene URAD has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01878 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4776-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4776-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01953 has 0 variants, skipping\n",
      "[INFO]\tGene MYCNUT has 0 variants, skipping\n",
      "[INFO]\tGene GATA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4438 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2L-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOX4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70I has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A2 has 0 variants, skipping\n",
      "[INFO]\tGene ASXL2 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene EHF has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724849 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00607 has 0 variants, skipping\n",
      "[INFO]\tGene GPR84 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01963 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01280 has 0 variants, skipping\n",
      "[INFO]\tGene DVL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373876 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01921 has 0 variants, skipping\n",
      "[INFO]\tGene DIRC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC3 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA115 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6513 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6810 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD8 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF2T has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D3 has 0 variants, skipping\n",
      "[INFO]\tGene RBAK-RBAKDN has 0 variants, skipping\n",
      "[INFO]\tGene VCX3B has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9500 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00608 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129175 has 0 variants, skipping\n",
      "[INFO]\tGene TERF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A7 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA8 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3131 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2A has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L9 has 0 variants, skipping\n",
      "[INFO]\tGene AAK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A1 has 0 variants, skipping\n",
      "[INFO]\tGene SPC25 has 0 variants, skipping\n",
      "[INFO]\tGene NEK1 has 0 variants, skipping\n",
      "[INFO]\tGene MVB12A has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02418 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL2 has 0 variants, skipping\n",
      "[INFO]\tGene NPY4R has 0 variants, skipping\n",
      "[INFO]\tGene MIR3132 has 0 variants, skipping\n",
      "[INFO]\tGene NELL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN30 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236B has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01803 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4268 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3B has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1 has 0 variants, skipping\n",
      "[INFO]\tGene RABL6 has 0 variants, skipping\n",
      "[INFO]\tGene MCTP1 has 0 variants, skipping\n",
      "[INFO]\tGene CT75 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB4Y has 0 variants, skipping\n",
      "[INFO]\tGene LIPF has 0 variants, skipping\n",
      "[INFO]\tGene SOX14 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3F has 0 variants, skipping\n",
      "[INFO]\tGene TPST1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB9 has 0 variants, skipping\n",
      "[INFO]\tGene SGCZ has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC10 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC8 has 0 variants, skipping\n",
      "[INFO]\tGene CDK13 has 0 variants, skipping\n",
      "[INFO]\tGene VIRMA has 0 variants, skipping\n",
      "[INFO]\tGene UBXN11 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4439 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646736 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5702 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF695 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP99 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5703 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC729968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01807 has 0 variants, skipping\n",
      "[INFO]\tGene XKR5 has 0 variants, skipping\n",
      "[INFO]\tGene GLP2R has 0 variants, skipping\n",
      "[INFO]\tGene TGM5 has 0 variants, skipping\n",
      "[INFO]\tGene CD163 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP11 has 0 variants, skipping\n",
      "[INFO]\tGene KLF4 has 0 variants, skipping\n",
      "[INFO]\tGene SOS2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM124A has 0 variants, skipping\n",
      "[INFO]\tGene PYCR3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A37 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLM1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD2 has 0 variants, skipping\n",
      "[INFO]\tGene LGSN has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC2 has 0 variants, skipping\n",
      "[INFO]\tGene TBX3 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP3 has 0 variants, skipping\n",
      "[INFO]\tGene BCCIP has 0 variants, skipping\n",
      "[INFO]\tGene COA4 has 0 variants, skipping\n",
      "[INFO]\tGene CEND1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR7 has 0 variants, skipping\n",
      "[INFO]\tGene SCAND1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP62 has 0 variants, skipping\n",
      "[INFO]\tGene TM7SF3 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN11 has 0 variants, skipping\n",
      "[INFO]\tGene MAN2B1 has 0 variants, skipping\n",
      "[INFO]\tGene OMP has 0 variants, skipping\n",
      "[INFO]\tGene PCSK5 has 0 variants, skipping\n",
      "[INFO]\tGene CUL9 has 0 variants, skipping\n",
      "[INFO]\tGene DDN has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01907 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC12 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM74B has 0 variants, skipping\n",
      "[INFO]\tGene MRLN has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHO1 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4777 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA75 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD82 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00471 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC24 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL10 has 0 variants, skipping\n",
      "[INFO]\tGene SEBOX has 0 variants, skipping\n",
      "[INFO]\tGene MIR1471 has 0 variants, skipping\n",
      "[INFO]\tGene MIR562 has 0 variants, skipping\n",
      "[INFO]\tGene ECEL1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC17 has 0 variants, skipping\n",
      "[INFO]\tGene STARD10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5001 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6D has 0 variants, skipping\n",
      "[INFO]\tGene ARL5C has 0 variants, skipping\n",
      "[INFO]\tGene STT3B has 0 variants, skipping\n",
      "[INFO]\tGene POP4 has 0 variants, skipping\n",
      "[INFO]\tGene COX8C has 0 variants, skipping\n",
      "[INFO]\tGene COPRS has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928881 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC151 has 0 variants, skipping\n",
      "[INFO]\tGene RBSN has 0 variants, skipping\n",
      "[INFO]\tGene UMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX3 has 0 variants, skipping\n",
      "[INFO]\tGene NSL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA5 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA6 has 0 variants, skipping\n",
      "[INFO]\tGene ENAM has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4H has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR3 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100286922 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGN has 0 variants, skipping\n",
      "[INFO]\tGene RNF145 has 0 variants, skipping\n",
      "[INFO]\tGene TTC23L has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT2 has 0 variants, skipping\n",
      "[INFO]\tGene MSL3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01891 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01173 has 0 variants, skipping\n",
      "[INFO]\tGene LONRF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF80 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC17 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF614 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD53 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC93463 has 0 variants, skipping\n",
      "[INFO]\tGene RAD50 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSP2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIF has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL11 has 0 variants, skipping\n",
      "[INFO]\tGene SRI-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COMETT has 0 variants, skipping\n",
      "[INFO]\tGene PLIN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6811 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM2A has 0 variants, skipping\n",
      "[INFO]\tGene MEP1B has 0 variants, skipping\n",
      "[INFO]\tGene TRAK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1B1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2F-SCLY has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02610 has 0 variants, skipping\n",
      "[INFO]\tGene SUZ12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01107 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373759 has 0 variants, skipping\n",
      "[INFO]\tGene HPX has 0 variants, skipping\n",
      "[INFO]\tGene LINC01940 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4440 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4441 has 0 variants, skipping\n",
      "[INFO]\tGene MGC16025 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4269 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2467 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC150935 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4786 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A17 has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN10-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC285191 has 0 variants, skipping\n",
      "[INFO]\tGene GPT2 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM171 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC18 has 0 variants, skipping\n",
      "[INFO]\tGene RBM15B has 0 variants, skipping\n",
      "[INFO]\tGene SLC9B2 has 0 variants, skipping\n",
      "[INFO]\tGene ORAI1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B3 has 0 variants, skipping\n",
      "[INFO]\tGene HCN4 has 0 variants, skipping\n",
      "[INFO]\tGene FRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene INSL5 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1B has 0 variants, skipping\n",
      "[INFO]\tGene PDK4 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFB has 0 variants, skipping\n",
      "[INFO]\tGene C11orf21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3133 has 0 variants, skipping\n",
      "[INFO]\tGene BAG4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH9 has 0 variants, skipping\n",
      "[INFO]\tGene BOK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN7 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1H has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R8 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3A has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CC has 0 variants, skipping\n",
      "[INFO]\tGene POU3F1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01237 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285097 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01880 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01881 has 0 variants, skipping\n",
      "[INFO]\tGene TCHH has 0 variants, skipping\n",
      "[INFO]\tGene NRSN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372493 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1A-SDCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289473 has 0 variants, skipping\n",
      "[INFO]\tGene LBX2 has 0 variants, skipping\n",
      "[INFO]\tGene CBX7 has 0 variants, skipping\n",
      "[INFO]\tGene PDYN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRH has 0 variants, skipping\n",
      "[INFO]\tGene GNLY has 0 variants, skipping\n",
      "[INFO]\tGene SNORD119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1292 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA51 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD56 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD57 has 0 variants, skipping\n",
      "[INFO]\tGene FNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIME1 has 0 variants, skipping\n",
      "[INFO]\tGene FCAMR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996637 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723730 has 0 variants, skipping\n",
      "[INFO]\tGene OVCH2 has 0 variants, skipping\n",
      "[INFO]\tGene CABCOCO1 has 0 variants, skipping\n",
      "[INFO]\tGene MT2A has 0 variants, skipping\n",
      "[INFO]\tGene UBOX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L2 has 0 variants, skipping\n",
      "[INFO]\tGene NOM1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA2 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC16 has 0 variants, skipping\n",
      "[INFO]\tGene JPH4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01730 has 0 variants, skipping\n",
      "[INFO]\tGene NANOGNB has 0 variants, skipping\n",
      "[INFO]\tGene PRNT has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2A has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1B has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1D has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2B has 0 variants, skipping\n",
      "[INFO]\tGene ADH7 has 0 variants, skipping\n",
      "[INFO]\tGene ID2 has 0 variants, skipping\n",
      "[INFO]\tGene PCNA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00658 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643406 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00654 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01729 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA2 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf23 has 0 variants, skipping\n",
      "[INFO]\tGene KIF2B has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14B has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2E has 0 variants, skipping\n",
      "[INFO]\tGene MCM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAIM has 0 variants, skipping\n",
      "[INFO]\tGene CASC20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01713 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01428 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01751 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01706 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8062 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU105B has 0 variants, skipping\n",
      "[INFO]\tGene GNGT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CA6 has 0 variants, skipping\n",
      "[INFO]\tGene TAF11 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB10 has 0 variants, skipping\n",
      "[INFO]\tGene PGLS has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C3 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD2 has 0 variants, skipping\n",
      "[INFO]\tGene FRAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6870 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929413 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00687 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01722 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102606466 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01723 has 0 variants, skipping\n",
      "[INFO]\tGene ISM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01605 has 0 variants, skipping\n",
      "[INFO]\tGene THOC7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS25 has 0 variants, skipping\n",
      "[INFO]\tGene BRMS1L has 0 variants, skipping\n",
      "[INFO]\tGene MXRA8 has 0 variants, skipping\n",
      "[INFO]\tGene FMO6P has 0 variants, skipping\n",
      "[INFO]\tGene CCDC82 has 0 variants, skipping\n",
      "[INFO]\tGene ASB13 has 0 variants, skipping\n",
      "[INFO]\tGene SNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC16 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC9 has 0 variants, skipping\n",
      "[INFO]\tGene REX1BD has 0 variants, skipping\n",
      "[INFO]\tGene LOC613266 has 0 variants, skipping\n",
      "[INFO]\tGene HEXB has 0 variants, skipping\n",
      "[INFO]\tGene CKM has 0 variants, skipping\n",
      "[INFO]\tGene CKB has 0 variants, skipping\n",
      "[INFO]\tGene CD3G has 0 variants, skipping\n",
      "[INFO]\tGene CLTB has 0 variants, skipping\n",
      "[INFO]\tGene SNORD17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A2 has 0 variants, skipping\n",
      "[INFO]\tGene MOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene NPVF has 0 variants, skipping\n",
      "[INFO]\tGene LINC00851 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A11 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf122 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2V1 has 0 variants, skipping\n",
      "[INFO]\tGene FDX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3192 has 0 variants, skipping\n",
      "[INFO]\tGene DAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00652 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00653 has 0 variants, skipping\n",
      "[INFO]\tGene NID2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRK1 has 0 variants, skipping\n",
      "[INFO]\tGene CASC3 has 0 variants, skipping\n",
      "[INFO]\tGene COG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00237 has 0 variants, skipping\n",
      "[INFO]\tGene AIM2 has 0 variants, skipping\n",
      "[INFO]\tGene KIZ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01727 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01726 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01432 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01427 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284788 has 0 variants, skipping\n",
      "[INFO]\tGene GPR35 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01747 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00656 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01431 has 0 variants, skipping\n",
      "[INFO]\tGene CST13P has 0 variants, skipping\n",
      "[INFO]\tGene PRDM11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01721 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150B has 0 variants, skipping\n",
      "[INFO]\tGene SYNDIG1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926889 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SSNA1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASET2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF770 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J4 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF337-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372582 has 0 variants, skipping\n",
      "[INFO]\tGene FAM182B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926935 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01733 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134868 has 0 variants, skipping\n",
      "[INFO]\tGene FAM182A has 0 variants, skipping\n",
      "[INFO]\tGene MIR663AHG has 0 variants, skipping\n",
      "[INFO]\tGene FRG1CP has 0 variants, skipping\n",
      "[INFO]\tGene FRG1DP has 0 variants, skipping\n",
      "[INFO]\tGene FRG2EP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379477 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1EP has 0 variants, skipping\n",
      "[INFO]\tGene FAM242B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4477A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4477B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01597 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1BP has 0 variants, skipping\n",
      "[INFO]\tGene MLLT10P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUMA1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB122 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00028 has 0 variants, skipping\n",
      "[INFO]\tGene FGF7 has 0 variants, skipping\n",
      "[INFO]\tGene FGF9 has 0 variants, skipping\n",
      "[INFO]\tGene HM13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3193 has 0 variants, skipping\n",
      "[INFO]\tGene ABALON has 0 variants, skipping\n",
      "[INFO]\tGene REC114 has 0 variants, skipping\n",
      "[INFO]\tGene LEUTX has 0 variants, skipping\n",
      "[INFO]\tGene ZNF701 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC3 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK7 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY26P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1825 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929698 has 0 variants, skipping\n",
      "[INFO]\tGene NOL4L-DT has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A8 has 0 variants, skipping\n",
      "[INFO]\tGene TERF2IP has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A10 has 0 variants, skipping\n",
      "[INFO]\tGene DLL4 has 0 variants, skipping\n",
      "[INFO]\tGene DPM3 has 0 variants, skipping\n",
      "[INFO]\tGene SGTB has 0 variants, skipping\n",
      "[INFO]\tGene ING3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR27 has 0 variants, skipping\n",
      "[INFO]\tGene DDX49 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA4P has 0 variants, skipping\n",
      "[INFO]\tGene CDH5 has 0 variants, skipping\n",
      "[INFO]\tGene CDX1 has 0 variants, skipping\n",
      "[INFO]\tGene CD52 has 0 variants, skipping\n",
      "[INFO]\tGene FEN1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF341-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RALY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4755 has 0 variants, skipping\n",
      "[INFO]\tGene OR2Y1 has 0 variants, skipping\n",
      "[INFO]\tGene MED8 has 0 variants, skipping\n",
      "[INFO]\tGene BSPRY has 0 variants, skipping\n",
      "[INFO]\tGene FOLR3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71E1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM16 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB3P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1205 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1229 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4261 has 0 variants, skipping\n",
      "[INFO]\tGene TSKS has 0 variants, skipping\n",
      "[INFO]\tGene AVPI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TADA2A has 0 variants, skipping\n",
      "[INFO]\tGene TDRD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1289-1 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT7 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC3 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD4B has 0 variants, skipping\n",
      "[INFO]\tGene STAM2 has 0 variants, skipping\n",
      "[INFO]\tGene THBD has 0 variants, skipping\n",
      "[INFO]\tGene TGM1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK6 has 0 variants, skipping\n",
      "[INFO]\tGene NTN1 has 0 variants, skipping\n",
      "[INFO]\tGene KAT2A has 0 variants, skipping\n",
      "[INFO]\tGene CHKA has 0 variants, skipping\n",
      "[INFO]\tGene CYFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130373 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA21 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA16 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA14 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA10 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA8 has 0 variants, skipping\n",
      "[INFO]\tGene FNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN8 has 0 variants, skipping\n",
      "[INFO]\tGene GAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SAG has 0 variants, skipping\n",
      "[INFO]\tGene RHO has 0 variants, skipping\n",
      "[INFO]\tGene RYR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM201 has 0 variants, skipping\n",
      "[INFO]\tGene NR5A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRY has 0 variants, skipping\n",
      "[INFO]\tGene POP7 has 0 variants, skipping\n",
      "[INFO]\tGene RIDA has 0 variants, skipping\n",
      "[INFO]\tGene TIMM17B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00489 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287792 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF626 has 0 variants, skipping\n",
      "[INFO]\tGene SMPDL3A has 0 variants, skipping\n",
      "[INFO]\tGene SUB1 has 0 variants, skipping\n",
      "[INFO]\tGene FASTK has 0 variants, skipping\n",
      "[INFO]\tGene FCRL6 has 0 variants, skipping\n",
      "[INFO]\tGene MSH2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC149684 has 0 variants, skipping\n",
      "[INFO]\tGene TCTN2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71D has 0 variants, skipping\n",
      "[INFO]\tGene DRICH1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71E has 0 variants, skipping\n",
      "[INFO]\tGene SNORA60 has 0 variants, skipping\n",
      "[INFO]\tGene GAREM2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548O2 has 0 variants, skipping\n",
      "[INFO]\tGene DLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01734 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01370 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD154 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128988 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RORA has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6871 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF467 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927159 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32A has 0 variants, skipping\n",
      "[INFO]\tGene WDR43 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN6 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D9 has 0 variants, skipping\n",
      "[INFO]\tGene MT1B has 0 variants, skipping\n",
      "[INFO]\tGene MT1A has 0 variants, skipping\n",
      "[INFO]\tGene ATP9A has 0 variants, skipping\n",
      "[INFO]\tGene GSTM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01728 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D28 has 0 variants, skipping\n",
      "[INFO]\tGene OSER1-DT has 0 variants, skipping\n",
      "[INFO]\tGene FPR3 has 0 variants, skipping\n",
      "[INFO]\tGene FPR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3646 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01430 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01620 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC47 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A20 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD2 has 0 variants, skipping\n",
      "[INFO]\tGene EML4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPGRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01260 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR12 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF735 has 0 variants, skipping\n",
      "[INFO]\tGene SYS1-DBNDD2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6812 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372631 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3617 has 0 variants, skipping\n",
      "[INFO]\tGene LACTB2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX264 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM3 has 0 variants, skipping\n",
      "[INFO]\tGene CGGBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ2 has 0 variants, skipping\n",
      "[INFO]\tGene ETFB has 0 variants, skipping\n",
      "[INFO]\tGene SSR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372633 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF663P has 0 variants, skipping\n",
      "[INFO]\tGene MKRN7P has 0 variants, skipping\n",
      "[INFO]\tGene PRTFDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5B has 0 variants, skipping\n",
      "[INFO]\tGene B9D2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD2 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM121 has 0 variants, skipping\n",
      "[INFO]\tGene RFXAP has 0 variants, skipping\n",
      "[INFO]\tGene REN has 0 variants, skipping\n",
      "[INFO]\tGene HEATR5A has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP5 has 0 variants, skipping\n",
      "[INFO]\tGene ELP6 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC3 has 0 variants, skipping\n",
      "[INFO]\tGene CLPS has 0 variants, skipping\n",
      "[INFO]\tGene CEL has 0 variants, skipping\n",
      "[INFO]\tGene CDH8 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPE has 0 variants, skipping\n",
      "[INFO]\tGene COL6A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131496 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01754 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01522 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01523 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00494 has 0 variants, skipping\n",
      "[INFO]\tGene CSE1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM271 has 0 variants, skipping\n",
      "[INFO]\tGene AP5Z1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOM has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372653 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHB9 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA9 has 0 variants, skipping\n",
      "[INFO]\tGene SETDB2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01273 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01270 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01271 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA5B has 0 variants, skipping\n",
      "[INFO]\tGene ANTKMT has 0 variants, skipping\n",
      "[INFO]\tGene OSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-5 has 0 variants, skipping\n",
      "[INFO]\tGene ADNP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SYS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF19 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01429 has 0 variants, skipping\n",
      "[INFO]\tGene TREM2 has 0 variants, skipping\n",
      "[INFO]\tGene BARHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01524 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927770 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372672 has 0 variants, skipping\n",
      "[INFO]\tGene MIR767 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01441 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01440 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01716 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFD has 0 variants, skipping\n",
      "[INFO]\tGene BMP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4325 has 0 variants, skipping\n",
      "[INFO]\tGene RTP4 has 0 variants, skipping\n",
      "[INFO]\tGene NPFFR1 has 0 variants, skipping\n",
      "[INFO]\tGene TNMD has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF563 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01742 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR1288 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1L-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01711 has 0 variants, skipping\n",
      "[INFO]\tGene STX16-NPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372695 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927932 has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIEF2 has 0 variants, skipping\n",
      "[INFO]\tGene CDK15 has 0 variants, skipping\n",
      "[INFO]\tGene SLMO2-ATP5E has 0 variants, skipping\n",
      "[INFO]\tGene PDXP has 0 variants, skipping\n",
      "[INFO]\tGene GDF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNRH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR646HG has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928048 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4533 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01718 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128310 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1257 has 0 variants, skipping\n",
      "[INFO]\tGene FBLN7 has 0 variants, skipping\n",
      "[INFO]\tGene SBF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4758 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1HG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR520D has 0 variants, skipping\n",
      "[INFO]\tGene OGFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP21A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA117 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01749 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00029 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01056 has 0 variants, skipping\n",
      "[INFO]\tGene HAR1B has 0 variants, skipping\n",
      "[INFO]\tGene HAR1A has 0 variants, skipping\n",
      "[INFO]\tGene FLJ16779 has 0 variants, skipping\n",
      "[INFO]\tGene NPY has 0 variants, skipping\n",
      "[INFO]\tGene RPLP2 has 0 variants, skipping\n",
      "[INFO]\tGene LHB has 0 variants, skipping\n",
      "[INFO]\tGene NQO2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4326 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130587 has 0 variants, skipping\n",
      "[INFO]\tGene CDSN has 0 variants, skipping\n",
      "[INFO]\tGene MIR3135B has 0 variants, skipping\n",
      "[INFO]\tGene EMX1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F6 has 0 variants, skipping\n",
      "[INFO]\tGene MHENCR has 0 variants, skipping\n",
      "[INFO]\tGene SPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene RTEL1-TNFRSF6B has 0 variants, skipping\n",
      "[INFO]\tGene FBXO30 has 0 variants, skipping\n",
      "[INFO]\tGene XIRP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB46-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02764 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD3 has 0 variants, skipping\n",
      "[INFO]\tGene FTHL17 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL32 has 0 variants, skipping\n",
      "[INFO]\tGene CDH20 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS21 has 0 variants, skipping\n",
      "[INFO]\tGene AGXT2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHAC2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHAC1 has 0 variants, skipping\n",
      "[INFO]\tGene FZD8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1914 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34A has 0 variants, skipping\n",
      "[INFO]\tGene UCKL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6813 has 0 variants, skipping\n",
      "[INFO]\tGene PEX7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00266-1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13A1 has 0 variants, skipping\n",
      "[INFO]\tGene STX4 has 0 variants, skipping\n",
      "[INFO]\tGene NPTXR has 0 variants, skipping\n",
      "[INFO]\tGene CBX6 has 0 variants, skipping\n",
      "[INFO]\tGene GCAT has 0 variants, skipping\n",
      "[INFO]\tGene OR2T4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01678 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01669 has 0 variants, skipping\n",
      "[INFO]\tGene TOPAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene FRGCA has 0 variants, skipping\n",
      "[INFO]\tGene LOC106780825 has 0 variants, skipping\n",
      "[INFO]\tGene NKRF has 0 variants, skipping\n",
      "[INFO]\tGene MIR22HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR8069-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8069-2 has 0 variants, skipping\n",
      "[INFO]\tGene ULK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR549A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3648-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3648-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL23-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN2 has 0 variants, skipping\n",
      "[INFO]\tGene GHRLOS has 0 variants, skipping\n",
      "[INFO]\tGene SNHG7 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT4P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379514 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01667 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD49 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01674 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F29P has 0 variants, skipping\n",
      "[INFO]\tGene SRSF3P2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3120 has 0 variants, skipping\n",
      "[INFO]\tGene SAMSN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02246 has 0 variants, skipping\n",
      "[INFO]\tGene SCG5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR99AHG has 0 variants, skipping\n",
      "[INFO]\tGene SNORD74B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01549 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf91-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene CHODL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548XHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548X has 0 variants, skipping\n",
      "[INFO]\tGene LINC01683 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02573 has 0 variants, skipping\n",
      "[INFO]\tGene DTD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS40B has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST6 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM151B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01425 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01687 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00308 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6130 has 0 variants, skipping\n",
      "[INFO]\tGene D21S2088E has 0 variants, skipping\n",
      "[INFO]\tGene LINC01689 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01684 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01692 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00158 has 0 variants, skipping\n",
      "[INFO]\tGene DHX58 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM48 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548K has 0 variants, skipping\n",
      "[INFO]\tGene PRSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM4 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD9 has 0 variants, skipping\n",
      "[INFO]\tGene BTG3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF720 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4759 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00113 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00314 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01697 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01695 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM156 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf88 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00189 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD19 has 0 variants, skipping\n",
      "[INFO]\tGene CRBN has 0 variants, skipping\n",
      "[INFO]\tGene BACH1-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene BACH1-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene MROH2A has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00307 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4327 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS7 has 0 variants, skipping\n",
      "[INFO]\tGene RAB39A has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F2 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA3 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH8 has 0 variants, skipping\n",
      "[INFO]\tGene SMAP2 has 0 variants, skipping\n",
      "[INFO]\tGene FRYL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC42 has 0 variants, skipping\n",
      "[INFO]\tGene OR52R1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N4 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52K2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00159 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80A has 0 variants, skipping\n",
      "[INFO]\tGene URB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE12 has 0 variants, skipping\n",
      "[INFO]\tGene PAXBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PAXBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf62-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01690 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00945 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01548 has 0 variants, skipping\n",
      "[INFO]\tGene GABPA has 0 variants, skipping\n",
      "[INFO]\tGene IL10RB-DT has 0 variants, skipping\n",
      "[INFO]\tGene TTC21A has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1C1 has 0 variants, skipping\n",
      "[INFO]\tGene AMER2 has 0 variants, skipping\n",
      "[INFO]\tGene RTL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02525 has 0 variants, skipping\n",
      "[INFO]\tGene MELTF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6501 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL5 has 0 variants, skipping\n",
      "[INFO]\tGene THAP11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10D has 0 variants, skipping\n",
      "[INFO]\tGene SELENON has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928126 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA9 has 0 variants, skipping\n",
      "[INFO]\tGene EEA1 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00160 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPB has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506403 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928269 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01436 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133286 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF398 has 0 variants, skipping\n",
      "[INFO]\tGene NRN1L has 0 variants, skipping\n",
      "[INFO]\tGene ABCF3 has 0 variants, skipping\n",
      "[INFO]\tGene STRAP has 0 variants, skipping\n",
      "[INFO]\tGene FAM107A has 0 variants, skipping\n",
      "[INFO]\tGene ERG28 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00174 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C46 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T10 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T12 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C16 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M7 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M3 has 0 variants, skipping\n",
      "[INFO]\tGene TTC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB110 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR4 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8L has 0 variants, skipping\n",
      "[INFO]\tGene TRIML1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR10 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E3 has 0 variants, skipping\n",
      "[INFO]\tGene PIGQ has 0 variants, skipping\n",
      "[INFO]\tGene LINC01423 has 0 variants, skipping\n",
      "[INFO]\tGene NCCRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD7 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00114 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1755 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928398 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400867 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01700 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1B has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEPSIN has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6508 has 0 variants, skipping\n",
      "[INFO]\tGene PECR has 0 variants, skipping\n",
      "[INFO]\tGene IMPACT has 0 variants, skipping\n",
      "[INFO]\tGene AMPH has 0 variants, skipping\n",
      "[INFO]\tGene MIR412 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM15 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4760 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00922 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3197 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1B has 0 variants, skipping\n",
      "[INFO]\tGene PLAC4 has 0 variants, skipping\n",
      "[INFO]\tGene PODXL has 0 variants, skipping\n",
      "[INFO]\tGene GSDME has 0 variants, skipping\n",
      "[INFO]\tGene LINC00111 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00479 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6814 has 0 variants, skipping\n",
      "[INFO]\tGene TRIB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1 has 0 variants, skipping\n",
      "[INFO]\tGene USP30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA91 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF295-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFA has 0 variants, skipping\n",
      "[INFO]\tGene UMODL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT7 has 0 variants, skipping\n",
      "[INFO]\tGene AOC1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL30 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA3 has 0 variants, skipping\n",
      "[INFO]\tGene FZD2 has 0 variants, skipping\n",
      "[INFO]\tGene FRZB has 0 variants, skipping\n",
      "[INFO]\tGene FPR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930094 has 0 variants, skipping\n",
      "[INFO]\tGene NME6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01671 has 0 variants, skipping\n",
      "[INFO]\tGene DELEC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00271 has 0 variants, skipping\n",
      "[INFO]\tGene PYGO2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPTSSA has 0 variants, skipping\n",
      "[INFO]\tGene DTX3L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF681 has 0 variants, skipping\n",
      "[INFO]\tGene CSTL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01668 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692B has 0 variants, skipping\n",
      "[INFO]\tGene PTPRJ has 0 variants, skipping\n",
      "[INFO]\tGene HECW1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSPL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC173 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00322 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01679 has 0 variants, skipping\n",
      "[INFO]\tGene MYOSLID has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6070 has 0 variants, skipping\n",
      "[INFO]\tGene AATBC has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A1 has 0 variants, skipping\n",
      "[INFO]\tGene ENY2 has 0 variants, skipping\n",
      "[INFO]\tGene KLK6 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGS has 0 variants, skipping\n",
      "[INFO]\tGene TRPM2-AS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02575 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM42 has 0 variants, skipping\n",
      "[INFO]\tGene PRR15L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01424 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL1 has 0 variants, skipping\n",
      "[INFO]\tGene LMO1 has 0 variants, skipping\n",
      "[INFO]\tGene SAYSD1 has 0 variants, skipping\n",
      "[INFO]\tGene GCNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01547 has 0 variants, skipping\n",
      "[INFO]\tGene SSR4P1 has 0 variants, skipping\n",
      "[INFO]\tGene ETS2 has 0 variants, skipping\n",
      "[INFO]\tGene CALML3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00334 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00316 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CBLC has 0 variants, skipping\n",
      "[INFO]\tGene HBZ has 0 variants, skipping\n",
      "[INFO]\tGene MIR6815 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01694 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM202 has 0 variants, skipping\n",
      "[INFO]\tGene UNCX has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928796 has 0 variants, skipping\n",
      "[INFO]\tGene MTRF1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD159 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD118 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF587 has 0 variants, skipping\n",
      "[INFO]\tGene ATOH8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B4 has 0 variants, skipping\n",
      "[INFO]\tGene DDIAS has 0 variants, skipping\n",
      "[INFO]\tGene OR1G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E2 has 0 variants, skipping\n",
      "[INFO]\tGene USP24 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C15 has 0 variants, skipping\n",
      "[INFO]\tGene PUS1 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX53 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A4 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA1 has 0 variants, skipping\n",
      "[INFO]\tGene COPS7B has 0 variants, skipping\n",
      "[INFO]\tGene ANKDD1B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS30 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF12A has 0 variants, skipping\n",
      "[INFO]\tGene SPG21 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3C has 0 variants, skipping\n",
      "[INFO]\tGene RSRC1 has 0 variants, skipping\n",
      "[INFO]\tGene SCG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1FP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723769 has 0 variants, skipping\n",
      "[INFO]\tGene LHX1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD62P1-PARP4P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01665 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY1P1 has 0 variants, skipping\n",
      "[INFO]\tGene CECR7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01664 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CECR3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF611 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC9 has 0 variants, skipping\n",
      "[INFO]\tGene ALYREF has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL6 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM5 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF38 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00528 has 0 variants, skipping\n",
      "[INFO]\tGene MIR648 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01634 has 0 variants, skipping\n",
      "[INFO]\tGene RMI2 has 0 variants, skipping\n",
      "[INFO]\tGene MTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00106 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00243 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT3P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725072 has 0 variants, skipping\n",
      "[INFO]\tGene MTARC1 has 0 variants, skipping\n",
      "[INFO]\tGene RUSF1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD3 has 0 variants, skipping\n",
      "[INFO]\tGene HM13 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01311 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128108 has 0 variants, skipping\n",
      "[INFO]\tGene WDR25 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00895 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK5 has 0 variants, skipping\n",
      "[INFO]\tGene SEPT5-GP1BB has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC5 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR4761 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH2 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS16 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL31 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF12 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN2 has 0 variants, skipping\n",
      "[INFO]\tGene PRAG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM120B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA24 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN18 has 0 variants, skipping\n",
      "[INFO]\tGene NPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3618 has 0 variants, skipping\n",
      "[INFO]\tGene TMX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6816 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA77B has 0 variants, skipping\n",
      "[INFO]\tGene LOC284865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00896 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1286 has 0 variants, skipping\n",
      "[INFO]\tGene SPOUT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM170B has 0 variants, skipping\n",
      "[INFO]\tGene POM121L4P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191A has 0 variants, skipping\n",
      "[INFO]\tGene TEX36 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01637 has 0 variants, skipping\n",
      "[INFO]\tGene FABP9 has 0 variants, skipping\n",
      "[INFO]\tGene AVEN has 0 variants, skipping\n",
      "[INFO]\tGene THAP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3FP has 0 variants, skipping\n",
      "[INFO]\tGene EIF5AL1 has 0 variants, skipping\n",
      "[INFO]\tGene RRS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR649 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX6P has 0 variants, skipping\n",
      "[INFO]\tGene BCRP2 has 0 variants, skipping\n",
      "[INFO]\tGene FNTA has 0 variants, skipping\n",
      "[INFO]\tGene POM121L8P has 0 variants, skipping\n",
      "[INFO]\tGene RPL37A has 0 variants, skipping\n",
      "[INFO]\tGene RPL37 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KAP2 has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47B1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf72 has 0 variants, skipping\n",
      "[INFO]\tGene SCN4A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01210 has 0 variants, skipping\n",
      "[INFO]\tGene CCR10 has 0 variants, skipping\n",
      "[INFO]\tGene TIMMDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NRN1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1A has 0 variants, skipping\n",
      "[INFO]\tGene PRAMENP has 0 variants, skipping\n",
      "[INFO]\tGene STARD3 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC4 has 0 variants, skipping\n",
      "[INFO]\tGene LL22NC03-63E9.3 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L1P has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST4 has 0 variants, skipping\n",
      "[INFO]\tGene CD68 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02556 has 0 variants, skipping\n",
      "[INFO]\tGene CES5AP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01659 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT14 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS2 has 0 variants, skipping\n",
      "[INFO]\tGene HIC1 has 0 variants, skipping\n",
      "[INFO]\tGene REG1B has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP11 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1F has 0 variants, skipping\n",
      "[INFO]\tGene NANOG has 0 variants, skipping\n",
      "[INFO]\tGene MRM1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTTP2 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF791 has 0 variants, skipping\n",
      "[INFO]\tGene DICER1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CALR3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF501 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CENPBD1 has 0 variants, skipping\n",
      "[INFO]\tGene DRC1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DPEP2NB has 0 variants, skipping\n",
      "[INFO]\tGene POM121L9P has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1L-ADORA2A has 0 variants, skipping\n",
      "[INFO]\tGene B3GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX15 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC14A has 0 variants, skipping\n",
      "[INFO]\tGene BCRP3 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L10P has 0 variants, skipping\n",
      "[INFO]\tGene PRKAB2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CDS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM7 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC62 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6817 has 0 variants, skipping\n",
      "[INFO]\tGene CSH2 has 0 variants, skipping\n",
      "[INFO]\tGene PHKA2 has 0 variants, skipping\n",
      "[INFO]\tGene MYMX has 0 variants, skipping\n",
      "[INFO]\tGene RASAL3 has 0 variants, skipping\n",
      "[INFO]\tGene COL1A2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00299 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2E has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507599 has 0 variants, skipping\n",
      "[INFO]\tGene MIR668 has 0 variants, skipping\n",
      "[INFO]\tGene MIATNB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01422 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD66 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02554 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40AL has 0 variants, skipping\n",
      "[INFO]\tGene MIR3199-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3199-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5739 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5D2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32B has 0 variants, skipping\n",
      "[INFO]\tGene IPO7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191B has 0 variants, skipping\n",
      "[INFO]\tGene BOD1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD125 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL1S has 0 variants, skipping\n",
      "[INFO]\tGene NID1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP12A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6818 has 0 variants, skipping\n",
      "[INFO]\tGene LIF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC91370 has 0 variants, skipping\n",
      "[INFO]\tGene DUXA has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1656 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372990 has 0 variants, skipping\n",
      "[INFO]\tGene RAB41 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB136 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB135 has 0 variants, skipping\n",
      "[INFO]\tGene SDC4P has 0 variants, skipping\n",
      "[INFO]\tGene RTKN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3200 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM72 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985544 has 0 variants, skipping\n",
      "[INFO]\tGene MORC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF274 has 0 variants, skipping\n",
      "[INFO]\tGene HERC5 has 0 variants, skipping\n",
      "[INFO]\tGene RAB14 has 0 variants, skipping\n",
      "[INFO]\tGene TGM2 has 0 variants, skipping\n",
      "[INFO]\tGene HAND2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01521 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L12 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L10 has 0 variants, skipping\n",
      "[INFO]\tGene CBX5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3R2 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD9 has 0 variants, skipping\n",
      "[INFO]\tGene MACO1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D13 has 0 variants, skipping\n",
      "[INFO]\tGene BBS7 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf106 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02868 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02870 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF414 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02558 has 0 variants, skipping\n",
      "[INFO]\tGene AP1B1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129484 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF3 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL3S has 0 variants, skipping\n",
      "[INFO]\tGene LOC339666 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01640 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4764 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50B has 0 variants, skipping\n",
      "[INFO]\tGene LARGE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01399 has 0 variants, skipping\n",
      "[INFO]\tGene FRRS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR31 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV4 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6069 has 0 variants, skipping\n",
      "[INFO]\tGene PDIK1L has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF700 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF5 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2D has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6819 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373021 has 0 variants, skipping\n",
      "[INFO]\tGene NCF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LL22NC01-81G9.3 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA8L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF799 has 0 variants, skipping\n",
      "[INFO]\tGene TMED10 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAQ has 0 variants, skipping\n",
      "[INFO]\tGene CKAP4 has 0 variants, skipping\n",
      "[INFO]\tGene EBNA1BP2 has 0 variants, skipping\n",
      "[INFO]\tGene PKIA has 0 variants, skipping\n",
      "[INFO]\tGene RAB40B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF644 has 0 variants, skipping\n",
      "[INFO]\tGene SYDE2 has 0 variants, skipping\n",
      "[INFO]\tGene ABRAXAS2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM161A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506271 has 0 variants, skipping\n",
      "[INFO]\tGene ACOD1 has 0 variants, skipping\n",
      "[INFO]\tGene FCN3 has 0 variants, skipping\n",
      "[INFO]\tGene AP3B1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2G1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR659 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6820 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4534 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF496 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA92 has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF4 has 0 variants, skipping\n",
      "[INFO]\tGene CSPG4 has 0 variants, skipping\n",
      "[INFO]\tGene CTRL has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373031 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene QARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02871 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD83B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD83A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD139 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD43 has 0 variants, skipping\n",
      "[INFO]\tGene HBB has 0 variants, skipping\n",
      "[INFO]\tGene HBD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506472 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1H1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF3B has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A2 has 0 variants, skipping\n",
      "[INFO]\tGene DIO2 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR3 has 0 variants, skipping\n",
      "[INFO]\tGene STX17 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A4 has 0 variants, skipping\n",
      "[INFO]\tGene DCANP1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf47 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD140 has 0 variants, skipping\n",
      "[INFO]\tGene KIFBP has 0 variants, skipping\n",
      "[INFO]\tGene TCTN3 has 0 variants, skipping\n",
      "[INFO]\tGene EP300-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6889 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM74 has 0 variants, skipping\n",
      "[INFO]\tGene NATD1 has 0 variants, skipping\n",
      "[INFO]\tGene NADK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373044 has 0 variants, skipping\n",
      "[INFO]\tGene CD1C has 0 variants, skipping\n",
      "[INFO]\tGene CD1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR30C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29C has 0 variants, skipping\n",
      "[INFO]\tGene MIR106B has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR371A has 0 variants, skipping\n",
      "[INFO]\tGene MIR370 has 0 variants, skipping\n",
      "[INFO]\tGene MIR369 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376C has 0 variants, skipping\n",
      "[INFO]\tGene MIR367 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302D has 0 variants, skipping\n",
      "[INFO]\tGene MIR365B has 0 variants, skipping\n",
      "[INFO]\tGene MIR365A has 0 variants, skipping\n",
      "[INFO]\tGene MIR363 has 0 variants, skipping\n",
      "[INFO]\tGene MIR362 has 0 variants, skipping\n",
      "[INFO]\tGene MIR361 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR130B has 0 variants, skipping\n",
      "[INFO]\tGene MIR296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR99B has 0 variants, skipping\n",
      "[INFO]\tGene MIR301A has 0 variants, skipping\n",
      "[INFO]\tGene MIR299 has 0 variants, skipping\n",
      "[INFO]\tGene MIR34B has 0 variants, skipping\n",
      "[INFO]\tGene MIR34C has 0 variants, skipping\n",
      "[INFO]\tGene MIR219A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR101-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302A has 0 variants, skipping\n",
      "[INFO]\tGene MIR200A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00634 has 0 variants, skipping\n",
      "[INFO]\tGene SOBP has 0 variants, skipping\n",
      "[INFO]\tGene KIF26B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA6-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929829 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1202 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01315 has 0 variants, skipping\n",
      "[INFO]\tGene SERHL has 0 variants, skipping\n",
      "[INFO]\tGene FAM174B has 0 variants, skipping\n",
      "[INFO]\tGene RRP7BP has 0 variants, skipping\n",
      "[INFO]\tGene RNU12 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB105B has 0 variants, skipping\n",
      "[INFO]\tGene EFL1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT1 has 0 variants, skipping\n",
      "[INFO]\tGene AQP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRUB2 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01639 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01656 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00207 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00229 has 0 variants, skipping\n",
      "[INFO]\tGene SMC5 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927551 has 0 variants, skipping\n",
      "[INFO]\tGene NUP50-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373064 has 0 variants, skipping\n",
      "[INFO]\tGene IL12A has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2F has 0 variants, skipping\n",
      "[INFO]\tGene ANO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4762 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730668 has 0 variants, skipping\n",
      "[INFO]\tGene PRR34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642648 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7BHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4763 has 0 variants, skipping\n",
      "[INFO]\tGene PHLPP2 has 0 variants, skipping\n",
      "[INFO]\tGene GTSE1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SQSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF776 has 0 variants, skipping\n",
      "[INFO]\tGene OR10K2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01644 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00898 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3201 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284933 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4535 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01310 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6821 has 0 variants, skipping\n",
      "[INFO]\tGene GPER1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL16 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9 has 0 variants, skipping\n",
      "[INFO]\tGene GYPB has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG5 has 0 variants, skipping\n",
      "[INFO]\tGene RPS28 has 0 variants, skipping\n",
      "[INFO]\tGene NEK10 has 0 variants, skipping\n",
      "[INFO]\tGene INSIG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM95 has 0 variants, skipping\n",
      "[INFO]\tGene ADSS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC7B has 0 variants, skipping\n",
      "[INFO]\tGene MCCC2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB7A has 0 variants, skipping\n",
      "[INFO]\tGene CHKB-CPT1B has 0 variants, skipping\n",
      "[INFO]\tGene CHKB-DT has 0 variants, skipping\n",
      "[INFO]\tGene TRHDE has 0 variants, skipping\n",
      "[INFO]\tGene G6PD has 0 variants, skipping\n",
      "[INFO]\tGene ABCC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373100 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP82 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF4 has 0 variants, skipping\n",
      "[INFO]\tGene CAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf25 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF4B has 0 variants, skipping\n",
      "[INFO]\tGene IBA57 has 0 variants, skipping\n",
      "[INFO]\tGene PVT1 has 0 variants, skipping\n",
      "[INFO]\tGene JPX has 0 variants, skipping\n",
      "[INFO]\tGene APOO has 0 variants, skipping\n",
      "[INFO]\tGene LBX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDC3 has 0 variants, skipping\n",
      "[INFO]\tGene IPO13 has 0 variants, skipping\n",
      "[INFO]\tGene RPS19 has 0 variants, skipping\n",
      "[INFO]\tGene RERE has 0 variants, skipping\n",
      "[INFO]\tGene RFX7 has 0 variants, skipping\n",
      "[INFO]\tGene FCER1G has 0 variants, skipping\n",
      "[INFO]\tGene LRRC32 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K12 has 0 variants, skipping\n",
      "[INFO]\tGene PLCD4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC391322 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEC has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6P2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM72 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AC1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13H1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02692 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284412 has 0 variants, skipping\n",
      "[INFO]\tGene OR13F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13D1 has 0 variants, skipping\n",
      "[INFO]\tGene VAC14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5A has 0 variants, skipping\n",
      "[INFO]\tGene NCBP3 has 0 variants, skipping\n",
      "[INFO]\tGene TAX1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A14 has 0 variants, skipping\n",
      "[INFO]\tGene CRACR2A has 0 variants, skipping\n",
      "[INFO]\tGene IFIT1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP11C has 0 variants, skipping\n",
      "[INFO]\tGene GRB2 has 0 variants, skipping\n",
      "[INFO]\tGene USP50 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01986 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01266 has 0 variants, skipping\n",
      "[INFO]\tGene AIMP1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf47 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A3 has 0 variants, skipping\n",
      "[INFO]\tGene AP4E1 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF3 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR934 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4P1 has 0 variants, skipping\n",
      "[INFO]\tGene HMMR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK11 has 0 variants, skipping\n",
      "[INFO]\tGene LNCNEF has 0 variants, skipping\n",
      "[INFO]\tGene ROCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130207 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR1-DT has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D2 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGC has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE40-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4790 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927394 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM5 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A5 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMC6 has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB3 has 0 variants, skipping\n",
      "[INFO]\tGene MOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10B has 0 variants, skipping\n",
      "[INFO]\tGene TSLP has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIL has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R26 has 0 variants, skipping\n",
      "[INFO]\tGene CEP350 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL16 has 0 variants, skipping\n",
      "[INFO]\tGene ABITRAM has 0 variants, skipping\n",
      "[INFO]\tGene RNF125 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD5 has 0 variants, skipping\n",
      "[INFO]\tGene SARS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT3 has 0 variants, skipping\n",
      "[INFO]\tGene EMC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIDECP1 has 0 variants, skipping\n",
      "[INFO]\tGene AP2B1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCRB1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND10P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF213-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00852 has 0 variants, skipping\n",
      "[INFO]\tGene SPRTN has 0 variants, skipping\n",
      "[INFO]\tGene NFE2 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG5 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6F has 0 variants, skipping\n",
      "[INFO]\tGene XPA has 0 variants, skipping\n",
      "[INFO]\tGene PSMA7 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf70 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378B has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B2-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00606 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP91 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL2 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR3 has 0 variants, skipping\n",
      "[INFO]\tGene ANO8 has 0 variants, skipping\n",
      "[INFO]\tGene H3C14 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCD3 has 0 variants, skipping\n",
      "[INFO]\tGene MYO7B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF480 has 0 variants, skipping\n",
      "[INFO]\tGene DHX35 has 0 variants, skipping\n",
      "[INFO]\tGene DUXB has 0 variants, skipping\n",
      "[INFO]\tGene SNORA7A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02022 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA93 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00620 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01267 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02011 has 0 variants, skipping\n",
      "[INFO]\tGene PSG8 has 0 variants, skipping\n",
      "[INFO]\tGene PGBP has 0 variants, skipping\n",
      "[INFO]\tGene LRGUK has 0 variants, skipping\n",
      "[INFO]\tGene CCDC18 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG4 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS16 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4270 has 0 variants, skipping\n",
      "[INFO]\tGene BTLA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF134 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF132 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT3 has 0 variants, skipping\n",
      "[INFO]\tGene CALML4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284930 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00887 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR563 has 0 variants, skipping\n",
      "[INFO]\tGene FAM224A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3129 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00690 has 0 variants, skipping\n",
      "[INFO]\tGene PLCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG11 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD7 has 0 variants, skipping\n",
      "[INFO]\tGene NXF1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB13 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PD has 0 variants, skipping\n",
      "[INFO]\tGene TRIM38 has 0 variants, skipping\n",
      "[INFO]\tGene TADA3 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN4 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB18 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIH has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A9 has 0 variants, skipping\n",
      "[INFO]\tGene MERTK has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376975 has 0 variants, skipping\n",
      "[INFO]\tGene RERG has 0 variants, skipping\n",
      "[INFO]\tGene TRIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SATB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4791 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3135A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3659HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3065 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927829 has 0 variants, skipping\n",
      "[INFO]\tGene VENTXP7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TK1 has 0 variants, skipping\n",
      "[INFO]\tGene JAM2 has 0 variants, skipping\n",
      "[INFO]\tGene BYSL has 0 variants, skipping\n",
      "[INFO]\tGene SARM1 has 0 variants, skipping\n",
      "[INFO]\tGene H3-5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED6 has 0 variants, skipping\n",
      "[INFO]\tGene THRB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RARB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TDP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4442 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00692 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM239 has 0 variants, skipping\n",
      "[INFO]\tGene SYMPK has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1270 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01981 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00693 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01985 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN9 has 0 variants, skipping\n",
      "[INFO]\tGene PIGN has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AY has 0 variants, skipping\n",
      "[INFO]\tGene PHC3 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1A has 0 variants, skipping\n",
      "[INFO]\tGene EOLA2 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX4 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN12 has 0 variants, skipping\n",
      "[INFO]\tGene PYDC2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A3 has 0 variants, skipping\n",
      "[INFO]\tGene PRF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZGPAT has 0 variants, skipping\n",
      "[INFO]\tGene C7orf65 has 0 variants, skipping\n",
      "[INFO]\tGene RPTN has 0 variants, skipping\n",
      "[INFO]\tGene MTREX has 0 variants, skipping\n",
      "[INFO]\tGene ERAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI3 has 0 variants, skipping\n",
      "[INFO]\tGene MAVS has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928135 has 0 variants, skipping\n",
      "[INFO]\tGene MED25 has 0 variants, skipping\n",
      "[INFO]\tGene AK7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02033 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP3 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705E has 0 variants, skipping\n",
      "[INFO]\tGene TP53AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene A2ML1 has 0 variants, skipping\n",
      "[INFO]\tGene PRORY has 0 variants, skipping\n",
      "[INFO]\tGene TSPY10 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8M has 0 variants, skipping\n",
      "[INFO]\tGene SPIN2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC63 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM33 has 0 variants, skipping\n",
      "[INFO]\tGene LOC152048 has 0 variants, skipping\n",
      "[INFO]\tGene UTP18 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBX4 has 0 variants, skipping\n",
      "[INFO]\tGene ACAA1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4ENIF1 has 0 variants, skipping\n",
      "[INFO]\tGene FNBP1L has 0 variants, skipping\n",
      "[INFO]\tGene MLN has 0 variants, skipping\n",
      "[INFO]\tGene MIR100HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC00689 has 0 variants, skipping\n",
      "[INFO]\tGene CASC8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-2HG has 0 variants, skipping\n",
      "[INFO]\tGene FAM225A has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B3P has 0 variants, skipping\n",
      "[INFO]\tGene FBXL19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HNF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZEB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NECAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT18 has 0 variants, skipping\n",
      "[INFO]\tGene GP1BB has 0 variants, skipping\n",
      "[INFO]\tGene SRP54 has 0 variants, skipping\n",
      "[INFO]\tGene CDC5L has 0 variants, skipping\n",
      "[INFO]\tGene CD70 has 0 variants, skipping\n",
      "[INFO]\tGene MIR503HG has 0 variants, skipping\n",
      "[INFO]\tGene CASP6 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RASA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR218-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6822 has 0 variants, skipping\n",
      "[INFO]\tGene PIR has 0 variants, skipping\n",
      "[INFO]\tGene FOXE3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA62 has 0 variants, skipping\n",
      "[INFO]\tGene ADCYAP1R1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1S has 0 variants, skipping\n",
      "[INFO]\tGene KPNA3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LTBR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2CB has 0 variants, skipping\n",
      "[INFO]\tGene SGSM2 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB107A has 0 variants, skipping\n",
      "[INFO]\tGene DEFB128 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB116 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB115 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB113 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM8 has 0 variants, skipping\n",
      "[INFO]\tGene GPR39 has 0 variants, skipping\n",
      "[INFO]\tGene GCLC has 0 variants, skipping\n",
      "[INFO]\tGene MIR1231 has 0 variants, skipping\n",
      "[INFO]\tGene CRTAC1 has 0 variants, skipping\n",
      "[INFO]\tGene VIPR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH18 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf131 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR425 has 0 variants, skipping\n",
      "[INFO]\tGene MIR424 has 0 variants, skipping\n",
      "[INFO]\tGene MIR423 has 0 variants, skipping\n",
      "[INFO]\tGene MIR422A has 0 variants, skipping\n",
      "[INFO]\tGene MIR196B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02158 has 0 variants, skipping\n",
      "[INFO]\tGene NUP107 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D3A has 0 variants, skipping\n",
      "[INFO]\tGene BRWD3 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDF4 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G2 has 0 variants, skipping\n",
      "[INFO]\tGene GDF11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01988 has 0 variants, skipping\n",
      "[INFO]\tGene TNC has 0 variants, skipping\n",
      "[INFO]\tGene HTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20HL1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20HL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF197-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1P-OR7E38P has 0 variants, skipping\n",
      "[INFO]\tGene RBM7 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE1A has 0 variants, skipping\n",
      "[INFO]\tGene H4C9 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR3 has 0 variants, skipping\n",
      "[INFO]\tGene RPF1 has 0 variants, skipping\n",
      "[INFO]\tGene LARS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT11 has 0 variants, skipping\n",
      "[INFO]\tGene TCN2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00458 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02009 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA7B has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC11 has 0 variants, skipping\n",
      "[INFO]\tGene GJC1 has 0 variants, skipping\n",
      "[INFO]\tGene CFL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARB2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS46P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS42P has 0 variants, skipping\n",
      "[INFO]\tGene UNC5B has 0 variants, skipping\n",
      "[INFO]\tGene CRYGD has 0 variants, skipping\n",
      "[INFO]\tGene NRADDP has 0 variants, skipping\n",
      "[INFO]\tGene REC8 has 0 variants, skipping\n",
      "[INFO]\tGene KIF9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51V1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13J has 0 variants, skipping\n",
      "[INFO]\tGene TNKS has 0 variants, skipping\n",
      "[INFO]\tGene IRGQ has 0 variants, skipping\n",
      "[INFO]\tGene SNORD146 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2B has 0 variants, skipping\n",
      "[INFO]\tGene GSTA4 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP5 has 0 variants, skipping\n",
      "[INFO]\tGene FCF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene NPS has 0 variants, skipping\n",
      "[INFO]\tGene KLK3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2115 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF143 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6823 has 0 variants, skipping\n",
      "[INFO]\tGene MIR711 has 0 variants, skipping\n",
      "[INFO]\tGene PNOC has 0 variants, skipping\n",
      "[INFO]\tGene SNORA94 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6824 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4793 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02585 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18B1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM166A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF618 has 0 variants, skipping\n",
      "[INFO]\tGene MYADM has 0 variants, skipping\n",
      "[INFO]\tGene CFAP70 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC6B has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR586 has 0 variants, skipping\n",
      "[INFO]\tGene GPR151 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF804A has 0 variants, skipping\n",
      "[INFO]\tGene DND1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1182 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10A has 0 variants, skipping\n",
      "[INFO]\tGene P3R3URF-PIK3R3 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH4 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6890 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN5 has 0 variants, skipping\n",
      "[INFO]\tGene SIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB5 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF4 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN9 has 0 variants, skipping\n",
      "[INFO]\tGene ISY1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene IL18R1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA12 has 0 variants, skipping\n",
      "[INFO]\tGene AOC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4271 has 0 variants, skipping\n",
      "[INFO]\tGene CMC2 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene SASH3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11A has 0 variants, skipping\n",
      "[INFO]\tGene THSD7A has 0 variants, skipping\n",
      "[INFO]\tGene ARC has 0 variants, skipping\n",
      "[INFO]\tGene RPRD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB1 has 0 variants, skipping\n",
      "[INFO]\tGene BSN-DT has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2B has 0 variants, skipping\n",
      "[INFO]\tGene ACTG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00662 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf88B has 0 variants, skipping\n",
      "[INFO]\tGene RBM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5787 has 0 variants, skipping\n",
      "[INFO]\tGene PHF11 has 0 variants, skipping\n",
      "[INFO]\tGene CDH17 has 0 variants, skipping\n",
      "[INFO]\tGene BNC2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD4 has 0 variants, skipping\n",
      "[INFO]\tGene PMF1-BGLAP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC10B has 0 variants, skipping\n",
      "[INFO]\tGene USP27X has 0 variants, skipping\n",
      "[INFO]\tGene NELL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3941 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF818P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00589 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01018 has 0 variants, skipping\n",
      "[INFO]\tGene CARS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM89B has 0 variants, skipping\n",
      "[INFO]\tGene IQCF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D4 has 0 variants, skipping\n",
      "[INFO]\tGene YBX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00696 has 0 variants, skipping\n",
      "[INFO]\tGene PSENEN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929054 has 0 variants, skipping\n",
      "[INFO]\tGene CREBBP has 0 variants, skipping\n",
      "[INFO]\tGene LPIN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLYCTK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA2 has 0 variants, skipping\n",
      "[INFO]\tGene PJA2 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM70 has 0 variants, skipping\n",
      "[INFO]\tGene RAB15 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3C has 0 variants, skipping\n",
      "[INFO]\tGene SMIM27 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf15 has 0 variants, skipping\n",
      "[INFO]\tGene METTL17 has 0 variants, skipping\n",
      "[INFO]\tGene GCGR has 0 variants, skipping\n",
      "[INFO]\tGene GBE1 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD6 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D23 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA8 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L4 has 0 variants, skipping\n",
      "[INFO]\tGene GATA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD136 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD69 has 0 variants, skipping\n",
      "[INFO]\tGene UFD1 has 0 variants, skipping\n",
      "[INFO]\tGene SSPOP has 0 variants, skipping\n",
      "[INFO]\tGene PLCD3 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8064 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM43B has 0 variants, skipping\n",
      "[INFO]\tGene IFIT2 has 0 variants, skipping\n",
      "[INFO]\tGene ID4 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG2 has 0 variants, skipping\n",
      "[INFO]\tGene RASD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN8OS has 0 variants, skipping\n",
      "[INFO]\tGene ACAD9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38C has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN11 has 0 variants, skipping\n",
      "[INFO]\tGene TIFAB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF862 has 0 variants, skipping\n",
      "[INFO]\tGene SI has 0 variants, skipping\n",
      "[INFO]\tGene ESRG has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02017 has 0 variants, skipping\n",
      "[INFO]\tGene TXK has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF4 has 0 variants, skipping\n",
      "[INFO]\tGene ERC2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3938 has 0 variants, skipping\n",
      "[INFO]\tGene SHKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QB has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377102 has 0 variants, skipping\n",
      "[INFO]\tGene ARF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6A-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985115 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C4 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD1 has 0 variants, skipping\n",
      "[INFO]\tGene DRC7 has 0 variants, skipping\n",
      "[INFO]\tGene SLA2 has 0 variants, skipping\n",
      "[INFO]\tGene CD5L has 0 variants, skipping\n",
      "[INFO]\tGene FLNB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM17 has 0 variants, skipping\n",
      "[INFO]\tGene TUBD1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH11X has 0 variants, skipping\n",
      "[INFO]\tGene ME2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC6A has 0 variants, skipping\n",
      "[INFO]\tGene CITED2 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf44-SGK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHODL has 0 variants, skipping\n",
      "[INFO]\tGene ZHX1-C8orf76 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA4 has 0 variants, skipping\n",
      "[INFO]\tGene CUL7 has 0 variants, skipping\n",
      "[INFO]\tGene CUL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339902 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BB has 0 variants, skipping\n",
      "[INFO]\tGene SERINC5 has 0 variants, skipping\n",
      "[INFO]\tGene LMX1A has 0 variants, skipping\n",
      "[INFO]\tGene PTPRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00698 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THOC7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAANT1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00994 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K5 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF704 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF10 has 0 variants, skipping\n",
      "[INFO]\tGene FGD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FZR1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBE1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF2 has 0 variants, skipping\n",
      "[INFO]\tGene NAGPA has 0 variants, skipping\n",
      "[INFO]\tGene FAM8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP15 has 0 variants, skipping\n",
      "[INFO]\tGene MAML3 has 0 variants, skipping\n",
      "[INFO]\tGene PDE11A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377143 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4272 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377146 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3136 has 0 variants, skipping\n",
      "[INFO]\tGene RBM28 has 0 variants, skipping\n",
      "[INFO]\tGene KIF18B has 0 variants, skipping\n",
      "[INFO]\tGene RPE65 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4B has 0 variants, skipping\n",
      "[INFO]\tGene GRHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK1 has 0 variants, skipping\n",
      "[INFO]\tGene QRICH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT172 has 0 variants, skipping\n",
      "[INFO]\tGene TES has 0 variants, skipping\n",
      "[INFO]\tGene LINC00877 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00870 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377162 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP4 has 0 variants, skipping\n",
      "[INFO]\tGene RNF217 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927296 has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02005 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86DP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02018 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00960 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4273 has 0 variants, skipping\n",
      "[INFO]\tGene MRFAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02077 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927374 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3923 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02050 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02027 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02008 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00971 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02025 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA95 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5688 has 0 variants, skipping\n",
      "[INFO]\tGene CADM2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02070 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00506 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4795 has 0 variants, skipping\n",
      "[INFO]\tGene SNX18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF880 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00879 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8060 has 0 variants, skipping\n",
      "[INFO]\tGene AACS has 0 variants, skipping\n",
      "[INFO]\tGene GFPT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF837 has 0 variants, skipping\n",
      "[INFO]\tGene FAM30A has 0 variants, skipping\n",
      "[INFO]\tGene NPFFR2 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645202 has 0 variants, skipping\n",
      "[INFO]\tGene HP09053 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3921 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30CP has 0 variants, skipping\n",
      "[INFO]\tGene SRRM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR576 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92B has 0 variants, skipping\n",
      "[INFO]\tGene MIR552 has 0 variants, skipping\n",
      "[INFO]\tGene MIR551A has 0 variants, skipping\n",
      "[INFO]\tGene MIR487B has 0 variants, skipping\n",
      "[INFO]\tGene MIR376A2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH6A1 has 0 variants, skipping\n",
      "[INFO]\tGene GFI1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A9 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM172BP has 0 variants, skipping\n",
      "[INFO]\tGene TAPT1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71C has 0 variants, skipping\n",
      "[INFO]\tGene PCNP has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL3P4 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00310 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00265 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02085 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929411 has 0 variants, skipping\n",
      "[INFO]\tGene NPHS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AB has 0 variants, skipping\n",
      "[INFO]\tGene MROH1 has 0 variants, skipping\n",
      "[INFO]\tGene AP3B2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCO has 0 variants, skipping\n",
      "[INFO]\tGene DUBR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929579 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01990 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00635 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00636 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01215 has 0 variants, skipping\n",
      "[INFO]\tGene TAL1 has 0 variants, skipping\n",
      "[INFO]\tGene TULP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH2 has 0 variants, skipping\n",
      "[INFO]\tGene RSAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4445 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPT1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3J-DT has 0 variants, skipping\n",
      "[INFO]\tGene SMIM22 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF444 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02042 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C8G has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02044 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8076 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF530 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ANOS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK3 has 0 variants, skipping\n",
      "[INFO]\tGene MX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4446 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RG has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B2 has 0 variants, skipping\n",
      "[INFO]\tGene HPRT1 has 0 variants, skipping\n",
      "[INFO]\tGene IDUA has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR890 has 0 variants, skipping\n",
      "[INFO]\tGene OR51F2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA7 has 0 variants, skipping\n",
      "[INFO]\tGene TAT has 0 variants, skipping\n",
      "[INFO]\tGene MIR568 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4796 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD155 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00903 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4447 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00901 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02024 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374060 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926968 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA5 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP31-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RADIL has 0 variants, skipping\n",
      "[INFO]\tGene CAB39L has 0 variants, skipping\n",
      "[INFO]\tGene FAM193B has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF2 has 0 variants, skipping\n",
      "[INFO]\tGene ROBO1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02049 has 0 variants, skipping\n",
      "[INFO]\tGene MBL2 has 0 variants, skipping\n",
      "[INFO]\tGene LENG1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL12 has 0 variants, skipping\n",
      "[INFO]\tGene GNPTAB has 0 variants, skipping\n",
      "[INFO]\tGene MIR5682 has 0 variants, skipping\n",
      "[INFO]\tGene RLIM has 0 variants, skipping\n",
      "[INFO]\tGene TRPV2 has 0 variants, skipping\n",
      "[INFO]\tGene TELO2 has 0 variants, skipping\n",
      "[INFO]\tGene VAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723582 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH6A has 0 variants, skipping\n",
      "[INFO]\tGene CLPTM1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A10 has 0 variants, skipping\n",
      "[INFO]\tGene WNT5B has 0 variants, skipping\n",
      "[INFO]\tGene OR51E2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB1 has 0 variants, skipping\n",
      "[INFO]\tGene GJA9 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC34 has 0 variants, skipping\n",
      "[INFO]\tGene ILKAP has 0 variants, skipping\n",
      "[INFO]\tGene AKNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC02035 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7110 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2 has 0 variants, skipping\n",
      "[INFO]\tGene ADI1 has 0 variants, skipping\n",
      "[INFO]\tGene TG has 0 variants, skipping\n",
      "[INFO]\tGene MYLK-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC189 has 0 variants, skipping\n",
      "[INFO]\tGene METTL4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4316 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5002 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6083 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM213 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF843 has 0 variants, skipping\n",
      "[INFO]\tGene RNF26 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS24 has 0 variants, skipping\n",
      "[INFO]\tGene ARID3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR544B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5092 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374312 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02614 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86JP has 0 variants, skipping\n",
      "[INFO]\tGene CNBP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf220 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC149 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC37-DT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC24 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210P1 has 0 variants, skipping\n",
      "[INFO]\tGene ANO7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02016 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01471 has 0 variants, skipping\n",
      "[INFO]\tGene LDHA has 0 variants, skipping\n",
      "[INFO]\tGene MIR6825 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7976 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC10B has 0 variants, skipping\n",
      "[INFO]\tGene WFDC13 has 0 variants, skipping\n",
      "[INFO]\tGene SYT3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF5 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SP5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC90246 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01565 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653712 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM200A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC45 has 0 variants, skipping\n",
      "[INFO]\tGene PGA3 has 0 variants, skipping\n",
      "[INFO]\tGene PTGDR2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT5 has 0 variants, skipping\n",
      "[INFO]\tGene MED1 has 0 variants, skipping\n",
      "[INFO]\tGene TREML1 has 0 variants, skipping\n",
      "[INFO]\tGene HLTF has 0 variants, skipping\n",
      "[INFO]\tGene MIR6826 has 0 variants, skipping\n",
      "[INFO]\tGene RPL32P3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA7B has 0 variants, skipping\n",
      "[INFO]\tGene TET3 has 0 variants, skipping\n",
      "[INFO]\tGene TM7SF2 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB131A has 0 variants, skipping\n",
      "[INFO]\tGene TMCC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02014 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86HP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02021 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A4P2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R1 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF18 has 0 variants, skipping\n",
      "[INFO]\tGene MCIDAS has 0 variants, skipping\n",
      "[INFO]\tGene STMND1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14C36 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM22 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61A1 has 0 variants, skipping\n",
      "[INFO]\tGene HILPDA has 0 variants, skipping\n",
      "[INFO]\tGene NME7 has 0 variants, skipping\n",
      "[INFO]\tGene PYCR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339874 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA58 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5704 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3-ACAD11 has 0 variants, skipping\n",
      "[INFO]\tGene DPAGT1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM108-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCD2OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02000 has 0 variants, skipping\n",
      "[INFO]\tGene FCHSD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02004 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6827 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4788 has 0 variants, skipping\n",
      "[INFO]\tGene LY6L has 0 variants, skipping\n",
      "[INFO]\tGene TCP11X1 has 0 variants, skipping\n",
      "[INFO]\tGene TENM2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L27 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD14 has 0 variants, skipping\n",
      "[INFO]\tGene ELL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIGZ has 0 variants, skipping\n",
      "[INFO]\tGene TASL has 0 variants, skipping\n",
      "[INFO]\tGene DCSTAMP has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP29-1 has 0 variants, skipping\n",
      "[INFO]\tGene MLNR has 0 variants, skipping\n",
      "[INFO]\tGene GFRA3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR32 has 0 variants, skipping\n",
      "[INFO]\tGene GPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3677 has 0 variants, skipping\n",
      "[INFO]\tGene BPESC1 has 0 variants, skipping\n",
      "[INFO]\tGene PISRT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMID1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507291 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTF2 has 0 variants, skipping\n",
      "[INFO]\tGene IRS4 has 0 variants, skipping\n",
      "[INFO]\tGene MORN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02618 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51 has 0 variants, skipping\n",
      "[INFO]\tGene DDX53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF232-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLASRP has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4E has 0 variants, skipping\n",
      "[INFO]\tGene POLR2G has 0 variants, skipping\n",
      "[INFO]\tGene PPL has 0 variants, skipping\n",
      "[INFO]\tGene POU4F3 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507389 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289361 has 0 variants, skipping\n",
      "[INFO]\tGene EFHC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCHS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCSRLR has 0 variants, skipping\n",
      "[INFO]\tGene FAM178B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02010 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02032 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02046 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN2B has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF263 has 0 variants, skipping\n",
      "[INFO]\tGene HLTF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728392 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene NOL6 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN21 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646903 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374313 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01214 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT1 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2L has 0 variants, skipping\n",
      "[INFO]\tGene SVIL has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928105 has 0 variants, skipping\n",
      "[INFO]\tGene NKD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5186 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AADACP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928166 has 0 variants, skipping\n",
      "[INFO]\tGene RFX4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA33 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14EP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02006 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INSIG2 has 0 variants, skipping\n",
      "[INFO]\tGene CBY2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01487 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP61 has 0 variants, skipping\n",
      "[INFO]\tGene PLCH1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMELESS has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL21 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TIPARP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00886 has 0 variants, skipping\n",
      "[INFO]\tGene PA2G4P4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00880 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02029 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00881 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHC has 0 variants, skipping\n",
      "[INFO]\tGene VPS33A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4458 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996447 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA5 has 0 variants, skipping\n",
      "[INFO]\tGene CTSZ has 0 variants, skipping\n",
      "[INFO]\tGene CTSW has 0 variants, skipping\n",
      "[INFO]\tGene CTSO has 0 variants, skipping\n",
      "[INFO]\tGene MCU has 0 variants, skipping\n",
      "[INFO]\tGene C5orf64 has 0 variants, skipping\n",
      "[INFO]\tGene GPX1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3C has 0 variants, skipping\n",
      "[INFO]\tGene USP17L2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2KMT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3919 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG30 has 0 variants, skipping\n",
      "[INFO]\tGene IL12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01100 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM59-IFT80 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC52 has 0 variants, skipping\n",
      "[INFO]\tGene OR9Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR9I1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC55 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM198 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL10 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA7 has 0 variants, skipping\n",
      "[INFO]\tGene OBP2B has 0 variants, skipping\n",
      "[INFO]\tGene RHOD has 0 variants, skipping\n",
      "[INFO]\tGene TMEM97 has 0 variants, skipping\n",
      "[INFO]\tGene LATS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02067 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01192 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1263 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01324 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01322 has 0 variants, skipping\n",
      "[INFO]\tGene SSR1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374194 has 0 variants, skipping\n",
      "[INFO]\tGene RABEPK has 0 variants, skipping\n",
      "[INFO]\tGene RFX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01327 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN6 has 0 variants, skipping\n",
      "[INFO]\tGene MANSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC77P has 0 variants, skipping\n",
      "[INFO]\tGene EGFEM1P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02082 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF9 has 0 variants, skipping\n",
      "[INFO]\tGene NCMAP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf189 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD6 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128164 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6828 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB134 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR569 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM212-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATIC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02068 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106C has 0 variants, skipping\n",
      "[INFO]\tGene STK3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC59 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO6 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD1 has 0 variants, skipping\n",
      "[INFO]\tGene ASH1L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM176A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR10 has 0 variants, skipping\n",
      "[INFO]\tGene BEX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H15 has 0 variants, skipping\n",
      "[INFO]\tGene USE1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGRKT has 0 variants, skipping\n",
      "[INFO]\tGene CISD1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG2 has 0 variants, skipping\n",
      "[INFO]\tGene BRK1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2D has 0 variants, skipping\n",
      "[INFO]\tGene NLGN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4789 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01209 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02015 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01014 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928739 has 0 variants, skipping\n",
      "[INFO]\tGene TLR5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-4 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf49 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF823 has 0 variants, skipping\n",
      "[INFO]\tGene GNG14 has 0 variants, skipping\n",
      "[INFO]\tGene PEX5L-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02053 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928882 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724604 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01994 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01995 has 0 variants, skipping\n",
      "[INFO]\tGene STBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00888 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63E has 0 variants, skipping\n",
      "[INFO]\tGene KLHL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK2 has 0 variants, skipping\n",
      "[INFO]\tGene LCE6A has 0 variants, skipping\n",
      "[INFO]\tGene FBXO17 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOX2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4448 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OLAH has 0 variants, skipping\n",
      "[INFO]\tGene BRIX1 has 0 variants, skipping\n",
      "[INFO]\tGene BRF2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA4 has 0 variants, skipping\n",
      "[INFO]\tGene MEP1A has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L1 has 0 variants, skipping\n",
      "[INFO]\tGene LYL1 has 0 variants, skipping\n",
      "[INFO]\tGene CD180 has 0 variants, skipping\n",
      "[INFO]\tGene BCAM has 0 variants, skipping\n",
      "[INFO]\tGene LPA has 0 variants, skipping\n",
      "[INFO]\tGene LOXL1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf71 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM13 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf73 has 0 variants, skipping\n",
      "[INFO]\tGene DENR has 0 variants, skipping\n",
      "[INFO]\tGene TLN1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf100 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD66 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY6 has 0 variants, skipping\n",
      "[INFO]\tGene MMAB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01840 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02069 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM39A has 0 variants, skipping\n",
      "[INFO]\tGene EHHADH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A9 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NMRAL2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02020 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02052 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1248 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA81 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02043 has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AKT1 has 0 variants, skipping\n",
      "[INFO]\tGene GRK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929106 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRH has 0 variants, skipping\n",
      "[INFO]\tGene ADM has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR410 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131635 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01991 has 0 variants, skipping\n",
      "[INFO]\tGene LPP-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42393 has 0 variants, skipping\n",
      "[INFO]\tGene LPP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF831 has 0 variants, skipping\n",
      "[INFO]\tGene GNG10 has 0 variants, skipping\n",
      "[INFO]\tGene P3H2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-I has 0 variants, skipping\n",
      "[INFO]\tGene OSTN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A1 has 0 variants, skipping\n",
      "[INFO]\tGene TENT5C has 0 variants, skipping\n",
      "[INFO]\tGene FOXC1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXF2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXF1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGC2889 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYS has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK11A has 0 variants, skipping\n",
      "[INFO]\tGene OPA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02038 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02026 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02028 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02036 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02037 has 0 variants, skipping\n",
      "[INFO]\tGene LCE4A has 0 variants, skipping\n",
      "[INFO]\tGene LCE1F has 0 variants, skipping\n",
      "[INFO]\tGene LCE1E has 0 variants, skipping\n",
      "[INFO]\tGene LINC00884 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WASF3 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC16A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01972 has 0 variants, skipping\n",
      "[INFO]\tGene GKN2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM61 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3137 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PROKR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374297 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12B has 0 variants, skipping\n",
      "[INFO]\tGene WDR83 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01983 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6829 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00885 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19-TCTEX1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01063 has 0 variants, skipping\n",
      "[INFO]\tGene TAF15 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK6 has 0 variants, skipping\n",
      "[INFO]\tGene POM121C has 0 variants, skipping\n",
      "[INFO]\tGene PCNX2 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN2 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR543 has 0 variants, skipping\n",
      "[INFO]\tGene MIR873 has 0 variants, skipping\n",
      "[INFO]\tGene MIR199A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR196A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR665 has 0 variants, skipping\n",
      "[INFO]\tGene MIR187 has 0 variants, skipping\n",
      "[INFO]\tGene MIR183 has 0 variants, skipping\n",
      "[INFO]\tGene MIR182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR885 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181C has 0 variants, skipping\n",
      "[INFO]\tGene MIR744 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30D has 0 variants, skipping\n",
      "[INFO]\tGene MIR34A has 0 variants, skipping\n",
      "[INFO]\tGene MIR10B has 0 variants, skipping\n",
      "[INFO]\tGene MIR10A has 0 variants, skipping\n",
      "[INFO]\tGene MIR708 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR148A has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR129-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR208A has 0 variants, skipping\n",
      "[INFO]\tGene MTFR2 has 0 variants, skipping\n",
      "[INFO]\tGene PRCC has 0 variants, skipping\n",
      "[INFO]\tGene CCL26 has 0 variants, skipping\n",
      "[INFO]\tGene PKDREJ has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02012 has 0 variants, skipping\n",
      "[INFO]\tGene LOC220729 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR922 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf158 has 0 variants, skipping\n",
      "[INFO]\tGene POLA1 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFR has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984282 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA8 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD7 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB4B has 0 variants, skipping\n",
      "[INFO]\tGene LMLN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18DP has 0 variants, skipping\n",
      "[INFO]\tGene FAM157A has 0 variants, skipping\n",
      "[INFO]\tGene JMJD7 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA4 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf232 has 0 variants, skipping\n",
      "[INFO]\tGene KDM1B has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A7-CYP3A51P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652931 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642131 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK4-TEX40 has 0 variants, skipping\n",
      "[INFO]\tGene FAAHP1 has 0 variants, skipping\n",
      "[INFO]\tGene SQOR has 0 variants, skipping\n",
      "[INFO]\tGene GPT has 0 variants, skipping\n",
      "[INFO]\tGene GPR31 has 0 variants, skipping\n",
      "[INFO]\tGene GRK5 has 0 variants, skipping\n",
      "[INFO]\tGene MAFIP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM269-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF876P has 0 variants, skipping\n",
      "[INFO]\tGene KRT19 has 0 variants, skipping\n",
      "[INFO]\tGene KRT15 has 0 variants, skipping\n",
      "[INFO]\tGene KRT13 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA11P has 0 variants, skipping\n",
      "[INFO]\tGene MTMR11 has 0 variants, skipping\n",
      "[INFO]\tGene AARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374338 has 0 variants, skipping\n",
      "[INFO]\tGene APBA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129917 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC5 has 0 variants, skipping\n",
      "[INFO]\tGene CTF1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM172A has 0 variants, skipping\n",
      "[INFO]\tGene OVCA2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM210B has 0 variants, skipping\n",
      "[INFO]\tGene H2BC11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374344 has 0 variants, skipping\n",
      "[INFO]\tGene TMED11P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130872 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM255B has 0 variants, skipping\n",
      "[INFO]\tGene MED22 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS19 has 0 variants, skipping\n",
      "[INFO]\tGene DQX1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132C has 0 variants, skipping\n",
      "[INFO]\tGene DMAC1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA22 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf120 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4800 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF638 has 0 variants, skipping\n",
      "[INFO]\tGene HPR has 0 variants, skipping\n",
      "[INFO]\tGene NECAB2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLIM4 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B11 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7A has 0 variants, skipping\n",
      "[INFO]\tGene H2BE1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS4Y1 has 0 variants, skipping\n",
      "[INFO]\tGene MAOB has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR2 has 0 variants, skipping\n",
      "[INFO]\tGene TGM4 has 0 variants, skipping\n",
      "[INFO]\tGene HTT-AS has 0 variants, skipping\n",
      "[INFO]\tGene SERPINC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00955 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02171 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86EP has 0 variants, skipping\n",
      "[INFO]\tGene GCG has 0 variants, skipping\n",
      "[INFO]\tGene STX18-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene STX18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928279 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01396 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928306 has 0 variants, skipping\n",
      "[INFO]\tGene MUS81 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01587 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02495 has 0 variants, skipping\n",
      "[INFO]\tGene MUC5B has 0 variants, skipping\n",
      "[INFO]\tGene XAB2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD7B has 0 variants, skipping\n",
      "[INFO]\tGene LHX9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02482 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP3 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP14P has 0 variants, skipping\n",
      "[INFO]\tGene AGAP13P has 0 variants, skipping\n",
      "[INFO]\tGene CLEC3A has 0 variants, skipping\n",
      "[INFO]\tGene PGM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129931 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02447 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4798 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4274 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB12 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB11 has 0 variants, skipping\n",
      "[INFO]\tGene PYROXD2 has 0 variants, skipping\n",
      "[INFO]\tGene COX7B has 0 variants, skipping\n",
      "[INFO]\tGene COX7A1 has 0 variants, skipping\n",
      "[INFO]\tGene COX6B1 has 0 variants, skipping\n",
      "[INFO]\tGene COX5B has 0 variants, skipping\n",
      "[INFO]\tGene SLC31A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC31A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02517 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE4 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf88 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L6P has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3138 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02270 has 0 variants, skipping\n",
      "[INFO]\tGene VHLL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01097 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01096 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5091 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01182 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01085 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB2-DT has 0 variants, skipping\n",
      "[INFO]\tGene GALR1 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12I has 0 variants, skipping\n",
      "[INFO]\tGene GAGE6 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2A has 0 variants, skipping\n",
      "[INFO]\tGene KRT20 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA6 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA3 has 0 variants, skipping\n",
      "[INFO]\tGene TAPT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02493 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA75B has 0 variants, skipping\n",
      "[INFO]\tGene KANK4 has 0 variants, skipping\n",
      "[INFO]\tGene KIF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370687 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375131 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370708 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BAHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7978 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374516 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505912 has 0 variants, skipping\n",
      "[INFO]\tGene KCNV1 has 0 variants, skipping\n",
      "[INFO]\tGene BEX3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC2 has 0 variants, skipping\n",
      "[INFO]\tGene ST20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02194 has 0 variants, skipping\n",
      "[INFO]\tGene PPTC7 has 0 variants, skipping\n",
      "[INFO]\tGene SEPSECS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929161 has 0 variants, skipping\n",
      "[INFO]\tGene ALX4 has 0 variants, skipping\n",
      "[INFO]\tGene ENSA has 0 variants, skipping\n",
      "[INFO]\tGene STIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF2 has 0 variants, skipping\n",
      "[INFO]\tGene CSTA has 0 variants, skipping\n",
      "[INFO]\tGene CSN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02261 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4275 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02364 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02472 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02497 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02506 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377651 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02353 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928622 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02484 has 0 variants, skipping\n",
      "[INFO]\tGene LOC439933 has 0 variants, skipping\n",
      "[INFO]\tGene SCNM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255B1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02505 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4801 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf48 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02513 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01258 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02278 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2E has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4D has 0 variants, skipping\n",
      "[INFO]\tGene TMEM229A has 0 variants, skipping\n",
      "[INFO]\tGene DENND2A has 0 variants, skipping\n",
      "[INFO]\tGene FAM184B has 0 variants, skipping\n",
      "[INFO]\tGene PDGFD has 0 variants, skipping\n",
      "[INFO]\tGene MYH7 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf229 has 0 variants, skipping\n",
      "[INFO]\tGene AGA has 0 variants, skipping\n",
      "[INFO]\tGene LOC401127 has 0 variants, skipping\n",
      "[INFO]\tGene UGDH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INAFM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC344967 has 0 variants, skipping\n",
      "[INFO]\tGene TMED4 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC159 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4802 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643802 has 0 variants, skipping\n",
      "[INFO]\tGene UCHL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC2 has 0 variants, skipping\n",
      "[INFO]\tGene NEXN has 0 variants, skipping\n",
      "[INFO]\tGene CRTAP has 0 variants, skipping\n",
      "[INFO]\tGene VTI1B has 0 variants, skipping\n",
      "[INFO]\tGene SRPK3 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4717 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4709 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4660 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374428 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02383 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02475 has 0 variants, skipping\n",
      "[INFO]\tGene LSM7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS28 has 0 variants, skipping\n",
      "[INFO]\tGene SEC31B has 0 variants, skipping\n",
      "[INFO]\tGene METTL1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB8A has 0 variants, skipping\n",
      "[INFO]\tGene MIR8053 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927179 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927157 has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02480 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF350 has 0 variants, skipping\n",
      "[INFO]\tGene GNB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4449 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA26 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01618 has 0 variants, skipping\n",
      "[INFO]\tGene LNX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506444 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21P44 has 0 variants, skipping\n",
      "[INFO]\tGene PATL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02283 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02260 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AADAC has 0 variants, skipping\n",
      "[INFO]\tGene CACNG7 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF5 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD4BP has 0 variants, skipping\n",
      "[INFO]\tGene RGS9BP has 0 variants, skipping\n",
      "[INFO]\tGene UBR7 has 0 variants, skipping\n",
      "[INFO]\tGene CBR1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA6 has 0 variants, skipping\n",
      "[INFO]\tGene CAV1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXE1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02619 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AG1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02271 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF100 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01093 has 0 variants, skipping\n",
      "[INFO]\tGene TRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02232 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927237 has 0 variants, skipping\n",
      "[INFO]\tGene PROX1 has 0 variants, skipping\n",
      "[INFO]\tGene UBA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11GP has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4G has 0 variants, skipping\n",
      "[INFO]\tGene LOC550113 has 0 variants, skipping\n",
      "[INFO]\tGene SYT14P1 has 0 variants, skipping\n",
      "[INFO]\tGene FTLP10 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11BNL has 0 variants, skipping\n",
      "[INFO]\tGene RASL11A has 0 variants, skipping\n",
      "[INFO]\tGene GNS has 0 variants, skipping\n",
      "[INFO]\tGene GOT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377267 has 0 variants, skipping\n",
      "[INFO]\tGene EDC3 has 0 variants, skipping\n",
      "[INFO]\tGene PSG1 has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S2AP has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S2BP has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS2 has 0 variants, skipping\n",
      "[INFO]\tGene THOC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996842 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP8 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO3 has 0 variants, skipping\n",
      "[INFO]\tGene CBR4 has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD14 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ2 has 0 variants, skipping\n",
      "[INFO]\tGene VNN1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02499 has 0 variants, skipping\n",
      "[INFO]\tGene DGKQ has 0 variants, skipping\n",
      "[INFO]\tGene POTEJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR147B has 0 variants, skipping\n",
      "[INFO]\tGene PPBPP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A6 has 0 variants, skipping\n",
      "[INFO]\tGene MTM1 has 0 variants, skipping\n",
      "[INFO]\tGene NUBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02562 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02483 has 0 variants, skipping\n",
      "[INFO]\tGene EPB42 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO2L has 0 variants, skipping\n",
      "[INFO]\tGene SNX27 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC2 has 0 variants, skipping\n",
      "[INFO]\tGene CRACR2B has 0 variants, skipping\n",
      "[INFO]\tGene NXN has 0 variants, skipping\n",
      "[INFO]\tGene HIF3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4450 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AH has 0 variants, skipping\n",
      "[INFO]\tGene POLR1D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD161 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX2 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG15 has 0 variants, skipping\n",
      "[INFO]\tGene MGC70870 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00989 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR64 has 0 variants, skipping\n",
      "[INFO]\tGene MMP28 has 0 variants, skipping\n",
      "[INFO]\tGene FKRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00575 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD6 has 0 variants, skipping\n",
      "[INFO]\tGene TIPRL has 0 variants, skipping\n",
      "[INFO]\tGene PYDC1 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1 has 0 variants, skipping\n",
      "[INFO]\tGene UST has 0 variants, skipping\n",
      "[INFO]\tGene SNORD143 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD144 has 0 variants, skipping\n",
      "[INFO]\tGene EDEM3 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM4 has 0 variants, skipping\n",
      "[INFO]\tGene STON1 has 0 variants, skipping\n",
      "[INFO]\tGene STON1-GTF2A1L has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928978 has 0 variants, skipping\n",
      "[INFO]\tGene ADPGK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4451 has 0 variants, skipping\n",
      "[INFO]\tGene BTC has 0 variants, skipping\n",
      "[INFO]\tGene PATL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4452 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLA2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP1 has 0 variants, skipping\n",
      "[INFO]\tGene AFF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996750 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPA2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF169 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1K-DT has 0 variants, skipping\n",
      "[INFO]\tGene PIGY-DT has 0 variants, skipping\n",
      "[INFO]\tGene ARFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LBHD2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4C has 0 variants, skipping\n",
      "[INFO]\tGene FAM13A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFSWAP has 0 variants, skipping\n",
      "[INFO]\tGene SNCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCPRESS2 has 0 variants, skipping\n",
      "[INFO]\tGene EPOR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929210 has 0 variants, skipping\n",
      "[INFO]\tGene GAN has 0 variants, skipping\n",
      "[INFO]\tGene LINC02114 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63B has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1B-DT has 0 variants, skipping\n",
      "[INFO]\tGene ALOX5AP has 0 variants, skipping\n",
      "[INFO]\tGene ACOXL has 0 variants, skipping\n",
      "[INFO]\tGene LINC02267 has 0 variants, skipping\n",
      "[INFO]\tGene STPG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4ATAC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377342 has 0 variants, skipping\n",
      "[INFO]\tGene OR52M1 has 0 variants, skipping\n",
      "[INFO]\tGene CORO1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3684 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507053 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC78 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC49 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929353 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA101A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01216 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8066 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255A has 0 variants, skipping\n",
      "[INFO]\tGene FLJ20021 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP7 has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D3 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377621 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2E1 has 0 variants, skipping\n",
      "[INFO]\tGene MLANA has 0 variants, skipping\n",
      "[INFO]\tGene UBA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF169 has 0 variants, skipping\n",
      "[INFO]\tGene FAM200B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723704 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A10 has 0 variants, skipping\n",
      "[INFO]\tGene OR6Y1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD12 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA6 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TET2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRM2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF38-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BPHL has 0 variants, skipping\n",
      "[INFO]\tGene SRA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929577 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP1-GIMAP5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02173 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSD has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929621 has 0 variants, skipping\n",
      "[INFO]\tGene MMP9 has 0 variants, skipping\n",
      "[INFO]\tGene DLX2 has 0 variants, skipping\n",
      "[INFO]\tGene HBQ1 has 0 variants, skipping\n",
      "[INFO]\tGene HBE1 has 0 variants, skipping\n",
      "[INFO]\tGene HAS2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-1 has 0 variants, skipping\n",
      "[INFO]\tGene GJA10 has 0 variants, skipping\n",
      "[INFO]\tGene RALY has 0 variants, skipping\n",
      "[INFO]\tGene SLC8B1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC170 has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN4 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7D4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H3 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF729 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-A has 0 variants, skipping\n",
      "[INFO]\tGene EIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00526 has 0 variants, skipping\n",
      "[INFO]\tGene FOSB has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF11B has 0 variants, skipping\n",
      "[INFO]\tGene RASSF7 has 0 variants, skipping\n",
      "[INFO]\tGene BIK has 0 variants, skipping\n",
      "[INFO]\tGene PANCR has 0 variants, skipping\n",
      "[INFO]\tGene TMEM52 has 0 variants, skipping\n",
      "[INFO]\tGene PRTN3 has 0 variants, skipping\n",
      "[INFO]\tGene KLK10 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS8 has 0 variants, skipping\n",
      "[INFO]\tGene MANCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC440982 has 0 variants, skipping\n",
      "[INFO]\tGene MKX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3664 has 0 variants, skipping\n",
      "[INFO]\tGene CA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01520 has 0 variants, skipping\n",
      "[INFO]\tGene LNCAROD has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0002 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01495 has 0 variants, skipping\n",
      "[INFO]\tGene ETS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507144 has 0 variants, skipping\n",
      "[INFO]\tGene CYYR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02366 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6852 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5588 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf46 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2861 has 0 variants, skipping\n",
      "[INFO]\tGene TMC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5P3 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM79 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR567 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927318 has 0 variants, skipping\n",
      "[INFO]\tGene ILRUN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1912 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927378 has 0 variants, skipping\n",
      "[INFO]\tGene RP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6887 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6875 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6868 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6797 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6754 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB24 has 0 variants, skipping\n",
      "[INFO]\tGene TFCP2 has 0 variants, skipping\n",
      "[INFO]\tGene CCL21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF814 has 0 variants, skipping\n",
      "[INFO]\tGene H4-16 has 0 variants, skipping\n",
      "[INFO]\tGene KRT74 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf50 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A48 has 0 variants, skipping\n",
      "[INFO]\tGene RABEP2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIPC has 0 variants, skipping\n",
      "[INFO]\tGene MIR765 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1243 has 0 variants, skipping\n",
      "[INFO]\tGene DRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L2 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC10 has 0 variants, skipping\n",
      "[INFO]\tGene PARP4 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM2 has 0 variants, skipping\n",
      "[INFO]\tGene NPC2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8082 has 0 variants, skipping\n",
      "[INFO]\tGene TFDP3 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf78 has 0 variants, skipping\n",
      "[INFO]\tGene AMBN has 0 variants, skipping\n",
      "[INFO]\tGene PIPOX has 0 variants, skipping\n",
      "[INFO]\tGene GDF9 has 0 variants, skipping\n",
      "[INFO]\tGene VAV2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC4 has 0 variants, skipping\n",
      "[INFO]\tGene LCN15 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR9 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB3 has 0 variants, skipping\n",
      "[INFO]\tGene RXRG has 0 variants, skipping\n",
      "[INFO]\tGene RP2 has 0 variants, skipping\n",
      "[INFO]\tGene RORB has 0 variants, skipping\n",
      "[INFO]\tGene WWP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF546 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02262 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02263 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01378 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02264 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00488 has 0 variants, skipping\n",
      "[INFO]\tGene CHIAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729218 has 0 variants, skipping\n",
      "[INFO]\tGene METTL14-DT has 0 variants, skipping\n",
      "[INFO]\tGene PAICS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929762 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01365 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02502 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A3 has 0 variants, skipping\n",
      "[INFO]\tGene PP12613 has 0 variants, skipping\n",
      "[INFO]\tGene NPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES has 0 variants, skipping\n",
      "[INFO]\tGene GMFG has 0 variants, skipping\n",
      "[INFO]\tGene IL21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CETN4P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01091 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02516 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP2L has 0 variants, skipping\n",
      "[INFO]\tGene MIR2054 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA8 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNG has 0 variants, skipping\n",
      "[INFO]\tGene TLN2 has 0 variants, skipping\n",
      "[INFO]\tGene TMIE has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18A has 0 variants, skipping\n",
      "[INFO]\tGene GLIS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL14 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH10 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02615 has 0 variants, skipping\n",
      "[INFO]\tGene SCARF2 has 0 variants, skipping\n",
      "[INFO]\tGene VSX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02466 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02465 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02479 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01256 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927359 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02462 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02485 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00613 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02511 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02510 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02172 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00616 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00499 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377448 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY4B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927516 has 0 variants, skipping\n",
      "[INFO]\tGene SCOC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR51T1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B6 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E6 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E4 has 0 variants, skipping\n",
      "[INFO]\tGene INSL6 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC18P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02432 has 0 variants, skipping\n",
      "[INFO]\tGene BCLAF3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377623 has 0 variants, skipping\n",
      "[INFO]\tGene FABP12 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM179B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3139 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377458 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927636 has 0 variants, skipping\n",
      "[INFO]\tGene IGHMBP2 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP6 has 0 variants, skipping\n",
      "[INFO]\tGene IFNW1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNB1 has 0 variants, skipping\n",
      "[INFO]\tGene HHIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02266 has 0 variants, skipping\n",
      "[INFO]\tGene IL10 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01095 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7849 has 0 variants, skipping\n",
      "[INFO]\tGene SPATS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4799 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377480 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02355 has 0 variants, skipping\n",
      "[INFO]\tGene GJA4 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1A2 has 0 variants, skipping\n",
      "[INFO]\tGene EDN2 has 0 variants, skipping\n",
      "[INFO]\tGene GFPT1 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC2 has 0 variants, skipping\n",
      "[INFO]\tGene RALBP1 has 0 variants, skipping\n",
      "[INFO]\tGene UTS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRY has 0 variants, skipping\n",
      "[INFO]\tGene MRPL10 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0B has 0 variants, skipping\n",
      "[INFO]\tGene LOC729558 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD73A has 0 variants, skipping\n",
      "[INFO]\tGene CCNA2 has 0 variants, skipping\n",
      "[INFO]\tGene CBR3 has 0 variants, skipping\n",
      "[INFO]\tGene NAGK has 0 variants, skipping\n",
      "[INFO]\tGene MIR3140 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4453HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4453 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729870 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100419170 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724776 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02433 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM198B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3688-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3688-2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA12B has 0 variants, skipping\n",
      "[INFO]\tGene GRHPR has 0 variants, skipping\n",
      "[INFO]\tGene FZD4 has 0 variants, skipping\n",
      "[INFO]\tGene GNG3 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM10B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02233 has 0 variants, skipping\n",
      "[INFO]\tGene TTC34 has 0 variants, skipping\n",
      "[INFO]\tGene FMO3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928052 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1C1 has 0 variants, skipping\n",
      "[INFO]\tGene GK3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01179 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928131 has 0 variants, skipping\n",
      "[INFO]\tGene VPREB1 has 0 variants, skipping\n",
      "[INFO]\tGene ISX has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A has 0 variants, skipping\n",
      "[INFO]\tGene GNGT1 has 0 variants, skipping\n",
      "[INFO]\tGene RGL4 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R30 has 0 variants, skipping\n",
      "[INFO]\tGene GJB1 has 0 variants, skipping\n",
      "[INFO]\tGene REEP3 has 0 variants, skipping\n",
      "[INFO]\tGene DACH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02275 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02382 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6082 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02174 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441052 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930370 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548T has 0 variants, skipping\n",
      "[INFO]\tGene FAM83G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF727 has 0 variants, skipping\n",
      "[INFO]\tGene AMY2B has 0 variants, skipping\n",
      "[INFO]\tGene NMB has 0 variants, skipping\n",
      "[INFO]\tGene LINC02269 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02268 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4276 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928551 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM225 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928590 has 0 variants, skipping\n",
      "[INFO]\tGene VMAC has 0 variants, skipping\n",
      "[INFO]\tGene RCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM50A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02509 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01098 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01099 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00290 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02500 has 0 variants, skipping\n",
      "[INFO]\tGene TEMN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TENM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1305 has 0 variants, skipping\n",
      "[INFO]\tGene EID3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A11P has 0 variants, skipping\n",
      "[INFO]\tGene WWC2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389247 has 0 variants, skipping\n",
      "[INFO]\tGene CPZ has 0 variants, skipping\n",
      "[INFO]\tGene PPIG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02363 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02362 has 0 variants, skipping\n",
      "[INFO]\tGene ALG14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02427 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02365 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4B has 0 variants, skipping\n",
      "[INFO]\tGene IGFBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A3 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA5 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD5 has 0 variants, skipping\n",
      "[INFO]\tGene SLED1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3945HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3945 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4455 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02436 has 0 variants, skipping\n",
      "[INFO]\tGene MAGIX has 0 variants, skipping\n",
      "[INFO]\tGene FAM47E-STBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377590 has 0 variants, skipping\n",
      "[INFO]\tGene RNH1 has 0 variants, skipping\n",
      "[INFO]\tGene AMACR has 0 variants, skipping\n",
      "[INFO]\tGene ACSL5 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ38576 has 0 variants, skipping\n",
      "[INFO]\tGene FGF19 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR82 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02374 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339975 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02514 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02492 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRD1 has 0 variants, skipping\n",
      "[INFO]\tGene OXGR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01262 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01596 has 0 variants, skipping\n",
      "[INFO]\tGene DBET has 0 variants, skipping\n",
      "[INFO]\tGene DDX39A has 0 variants, skipping\n",
      "[INFO]\tGene GPR78 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692A2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5591 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT5 has 0 variants, skipping\n",
      "[INFO]\tGene CA12 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6D has 0 variants, skipping\n",
      "[INFO]\tGene PP7080 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4456 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996325 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506688 has 0 variants, skipping\n",
      "[INFO]\tGene SIN3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4635 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-3080P12.3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF5A has 0 variants, skipping\n",
      "[INFO]\tGene HPDL has 0 variants, skipping\n",
      "[INFO]\tGene RAX2 has 0 variants, skipping\n",
      "[INFO]\tGene XCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1P3 has 0 variants, skipping\n",
      "[INFO]\tGene AURKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4457 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01511 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6075 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4277 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02116 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2194D22.4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506858 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374620 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01019 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02162 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01017 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01020 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02121 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2297D10.2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF768 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD55 has 0 variants, skipping\n",
      "[INFO]\tGene LCE1D has 0 variants, skipping\n",
      "[INFO]\tGene LCE1B has 0 variants, skipping\n",
      "[INFO]\tGene LCE1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02102 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02236 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4278 has 0 variants, skipping\n",
      "[INFO]\tGene LOC442132 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02123 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RD has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2F has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG5 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA7 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL7 has 0 variants, skipping\n",
      "[INFO]\tGene CCR5AS has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B12 has 0 variants, skipping\n",
      "[INFO]\tGene OR8A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02226 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4458HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02199 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4636 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD123 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02112 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02221 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC15 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6131 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02212 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02213 has 0 variants, skipping\n",
      "[INFO]\tGene OCM2 has 0 variants, skipping\n",
      "[INFO]\tGene OAS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01194 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02220 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD170 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD141A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD141B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM12 has 0 variants, skipping\n",
      "[INFO]\tGene DNALI1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130744 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4637 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02149 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2350J17.1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR887 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401176 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02150 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929524 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02111 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02217 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02218 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02223 has 0 variants, skipping\n",
      "[INFO]\tGene DAGLB has 0 variants, skipping\n",
      "[INFO]\tGene NCOA3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02241 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02146 has 0 variants, skipping\n",
      "[INFO]\tGene MYADML2 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD1 has 0 variants, skipping\n",
      "[INFO]\tGene PMCHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02239 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02228 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02211 has 0 variants, skipping\n",
      "[INFO]\tGene RP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02103 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929645 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02109 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02064 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374704 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4279 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL4 has 0 variants, skipping\n",
      "[INFO]\tGene AAMP has 0 variants, skipping\n",
      "[INFO]\tGene LZTS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02120 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02160 has 0 variants, skipping\n",
      "[INFO]\tGene PDPN has 0 variants, skipping\n",
      "[INFO]\tGene MAGED2 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf38 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL8 has 0 variants, skipping\n",
      "[INFO]\tGene RNF114 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2E has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF3-AMACR has 0 variants, skipping\n",
      "[INFO]\tGene SYCE1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM149B1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN10A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506406 has 0 variants, skipping\n",
      "[INFO]\tGene MIR580 has 0 variants, skipping\n",
      "[INFO]\tGene LTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL29 has 0 variants, skipping\n",
      "[INFO]\tGene NIPBL-DT has 0 variants, skipping\n",
      "[INFO]\tGene TRPV5 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374727 has 0 variants, skipping\n",
      "[INFO]\tGene GDNF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02110 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02119 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TULP3 has 0 variants, skipping\n",
      "[INFO]\tGene LIFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3650 has 0 variants, skipping\n",
      "[INFO]\tGene OSMR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01265 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf88 has 0 variants, skipping\n",
      "[INFO]\tGene RORC has 0 variants, skipping\n",
      "[INFO]\tGene ISG15 has 0 variants, skipping\n",
      "[INFO]\tGene MARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02104 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00603 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD72 has 0 variants, skipping\n",
      "[INFO]\tGene GARRE1 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926960 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD12 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ32255 has 0 variants, skipping\n",
      "[INFO]\tGene LOC648987 has 0 variants, skipping\n",
      "[INFO]\tGene LOC153684 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132356 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506639 has 0 variants, skipping\n",
      "[INFO]\tGene NIM1K has 0 variants, skipping\n",
      "[INFO]\tGene DEFB130A has 0 variants, skipping\n",
      "[INFO]\tGene COL28A1 has 0 variants, skipping\n",
      "[INFO]\tGene PIGY has 0 variants, skipping\n",
      "[INFO]\tGene FGF10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02224 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS30-DT has 0 variants, skipping\n",
      "[INFO]\tGene GSTA3 has 0 variants, skipping\n",
      "[INFO]\tGene STIMATE-MUSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM21 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02106 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642366 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02118 has 0 variants, skipping\n",
      "[INFO]\tGene TPT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RILP has 0 variants, skipping\n",
      "[INFO]\tGene LOC257396 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02105 has 0 variants, skipping\n",
      "[INFO]\tGene MIR581 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01033 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467080 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467081 has 0 variants, skipping\n",
      "[INFO]\tGene NTN5 has 0 variants, skipping\n",
      "[INFO]\tGene CCN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449C has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5687 has 0 variants, skipping\n",
      "[INFO]\tGene RNF138P1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51A7 has 0 variants, skipping\n",
      "[INFO]\tGene OR51A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD50A has 0 variants, skipping\n",
      "[INFO]\tGene PTTG3P has 0 variants, skipping\n",
      "[INFO]\tGene ARSI has 0 variants, skipping\n",
      "[INFO]\tGene MCRS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPIRE1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM140 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ31104 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01948 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM22 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0003 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928505 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02225 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02101 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AE2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02108 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM19 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF479 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH7 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD6 has 0 variants, skipping\n",
      "[INFO]\tGene IRX2 has 0 variants, skipping\n",
      "[INFO]\tGene IDI2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXQ1 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2N2 has 0 variants, skipping\n",
      "[INFO]\tGene GNG8 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR6L has 0 variants, skipping\n",
      "[INFO]\tGene SMIM15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02057 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928651 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf64-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OGDH has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF18 has 0 variants, skipping\n",
      "[INFO]\tGene IPO11-LRRC70 has 0 variants, skipping\n",
      "[INFO]\tGene GABRE has 0 variants, skipping\n",
      "[INFO]\tGene LOC100303749 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02229 has 0 variants, skipping\n",
      "[INFO]\tGene CCNDBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAST4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02242 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928858 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02219 has 0 variants, skipping\n",
      "[INFO]\tGene HSPE1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02198 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50D has 0 variants, skipping\n",
      "[INFO]\tGene OLFM4 has 0 variants, skipping\n",
      "[INFO]\tGene IFI44 has 0 variants, skipping\n",
      "[INFO]\tGene DHX34 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A13 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A7 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1A2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF3R has 0 variants, skipping\n",
      "[INFO]\tGene CSF2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF1 has 0 variants, skipping\n",
      "[INFO]\tGene CRH has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928924 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13B-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13B-1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2B has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02197 has 0 variants, skipping\n",
      "[INFO]\tGene FAM78A has 0 variants, skipping\n",
      "[INFO]\tGene MICALL1 has 0 variants, skipping\n",
      "[INFO]\tGene IPP has 0 variants, skipping\n",
      "[INFO]\tGene KDM8 has 0 variants, skipping\n",
      "[INFO]\tGene ELF1 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28A has 0 variants, skipping\n",
      "[INFO]\tGene THAP9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441081 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646652 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20G has 0 variants, skipping\n",
      "[INFO]\tGene KIFC2 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL3 has 0 variants, skipping\n",
      "[INFO]\tGene OCLNP1 has 0 variants, skipping\n",
      "[INFO]\tGene PMCHL2 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4803 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102503427 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02056 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4804 has 0 variants, skipping\n",
      "[INFO]\tGene USP8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379030 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36C has 0 variants, skipping\n",
      "[INFO]\tGene LOC340090 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02230 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01386 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3BL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01335 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01333 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01331 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD110 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD99 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441086 has 0 variants, skipping\n",
      "[INFO]\tGene MYMK has 0 variants, skipping\n",
      "[INFO]\tGene AJM1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-4 has 0 variants, skipping\n",
      "[INFO]\tGene CGB8 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPH has 0 variants, skipping\n",
      "[INFO]\tGene LOC441087 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929109 has 0 variants, skipping\n",
      "[INFO]\tGene ABCE1 has 0 variants, skipping\n",
      "[INFO]\tGene NCRUPAR has 0 variants, skipping\n",
      "[INFO]\tGene SNORA47 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOG has 0 variants, skipping\n",
      "[INFO]\tGene GABBR2 has 0 variants, skipping\n",
      "[INFO]\tGene KLLN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929154 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL3 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf16 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC6 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP35 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPU has 0 variants, skipping\n",
      "[INFO]\tGene CBWD3 has 0 variants, skipping\n",
      "[INFO]\tGene S100A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01455 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA6 has 0 variants, skipping\n",
      "[INFO]\tGene OR5R1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR921 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644936 has 0 variants, skipping\n",
      "[INFO]\tGene CRSP8P has 0 variants, skipping\n",
      "[INFO]\tGene GRHL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01337 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D4 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRH has 0 variants, skipping\n",
      "[INFO]\tGene CKMT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3977 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01338 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA18 has 0 variants, skipping\n",
      "[INFO]\tGene CABYR has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8A2 has 0 variants, skipping\n",
      "[INFO]\tGene VCAN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF22P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02059 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4280 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01949 has 0 variants, skipping\n",
      "[INFO]\tGene HJV has 0 variants, skipping\n",
      "[INFO]\tGene LOC644285 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02488 has 0 variants, skipping\n",
      "[INFO]\tGene ARPIN has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM55 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02060 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MICU3 has 0 variants, skipping\n",
      "[INFO]\tGene PELATON has 0 variants, skipping\n",
      "[INFO]\tGene CDH26 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3660 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01339 has 0 variants, skipping\n",
      "[INFO]\tGene LOC731157 has 0 variants, skipping\n",
      "[INFO]\tGene SGSH has 0 variants, skipping\n",
      "[INFO]\tGene PLIN2 has 0 variants, skipping\n",
      "[INFO]\tGene FUT9 has 0 variants, skipping\n",
      "[INFO]\tGene MMP13 has 0 variants, skipping\n",
      "[INFO]\tGene IGDCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00909 has 0 variants, skipping\n",
      "[INFO]\tGene UICLM has 0 variants, skipping\n",
      "[INFO]\tGene MIR2052HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR2277 has 0 variants, skipping\n",
      "[INFO]\tGene ESS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929710 has 0 variants, skipping\n",
      "[INFO]\tGene MIR583 has 0 variants, skipping\n",
      "[INFO]\tGene USP9X has 0 variants, skipping\n",
      "[INFO]\tGene ADAM10 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK7 has 0 variants, skipping\n",
      "[INFO]\tGene ASTL has 0 variants, skipping\n",
      "[INFO]\tGene DEFB108B has 0 variants, skipping\n",
      "[INFO]\tGene RSPH10B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3609 has 0 variants, skipping\n",
      "[INFO]\tGene TUB has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01340 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02234 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01846 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289230 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02113 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133050 has 0 variants, skipping\n",
      "[INFO]\tGene COX7C has 0 variants, skipping\n",
      "[INFO]\tGene MIR548P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00492 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00491 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02867 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1OT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNK2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02609 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643327 has 0 variants, skipping\n",
      "[INFO]\tGene NFYCP2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf21 has 0 variants, skipping\n",
      "[INFO]\tGene AMMECR1L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM47 has 0 variants, skipping\n",
      "[INFO]\tGene SOX7 has 0 variants, skipping\n",
      "[INFO]\tGene RTP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02801 has 0 variants, skipping\n",
      "[INFO]\tGene ACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY2 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE9 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS5 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02115 has 0 variants, skipping\n",
      "[INFO]\tGene RAB9BP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01950 has 0 variants, skipping\n",
      "[INFO]\tGene KDM6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01023 has 0 variants, skipping\n",
      "[INFO]\tGene RDH5 has 0 variants, skipping\n",
      "[INFO]\tGene OVAAL has 0 variants, skipping\n",
      "[INFO]\tGene LOC285638 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01848 has 0 variants, skipping\n",
      "[INFO]\tGene CEP290 has 0 variants, skipping\n",
      "[INFO]\tGene SLURP2 has 0 variants, skipping\n",
      "[INFO]\tGene STARD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK8 has 0 variants, skipping\n",
      "[INFO]\tGene NREP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927023 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02200 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467216 has 0 variants, skipping\n",
      "[INFO]\tGene PCARE has 0 variants, skipping\n",
      "[INFO]\tGene AHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CIB4 has 0 variants, skipping\n",
      "[INFO]\tGene SFMBT2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH8A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCK has 0 variants, skipping\n",
      "[INFO]\tGene HAO1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT2 has 0 variants, skipping\n",
      "[INFO]\tGene AP2A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927078 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01957 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927100 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467217 has 0 variants, skipping\n",
      "[INFO]\tGene METTL7A has 0 variants, skipping\n",
      "[INFO]\tGene DNHD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00649 has 0 variants, skipping\n",
      "[INFO]\tGene MIR325 has 0 variants, skipping\n",
      "[INFO]\tGene MIR129-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR185 has 0 variants, skipping\n",
      "[INFO]\tGene MIR126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR184 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927190 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02214 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00992 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02147 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02148 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02215 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379143 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61B has 0 variants, skipping\n",
      "[INFO]\tGene CBX1 has 0 variants, skipping\n",
      "[INFO]\tGene AFG3L2 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA0 has 0 variants, skipping\n",
      "[INFO]\tGene GPR75 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5706 has 0 variants, skipping\n",
      "[INFO]\tGene SDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467226 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf45 has 0 variants, skipping\n",
      "[INFO]\tGene SHARPIN has 0 variants, skipping\n",
      "[INFO]\tGene MGC32805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02201 has 0 variants, skipping\n",
      "[INFO]\tGene LGR6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379152 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24A has 0 variants, skipping\n",
      "[INFO]\tGene TAFA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM182 has 0 variants, skipping\n",
      "[INFO]\tGene SNED1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927421 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02240 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02039 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927488 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01309 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01740 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01781 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02428 has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB1-DT has 0 variants, skipping\n",
      "[INFO]\tGene FADS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3IL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01184 has 0 variants, skipping\n",
      "[INFO]\tGene ENGASE has 0 variants, skipping\n",
      "[INFO]\tGene C16orf96 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4633 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4460 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCR has 0 variants, skipping\n",
      "[INFO]\tGene RPL41 has 0 variants, skipping\n",
      "[INFO]\tGene MGMT has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2A2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT5 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA9 has 0 variants, skipping\n",
      "[INFO]\tGene HIBADH has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB9 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH4 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP97D1 has 0 variants, skipping\n",
      "[INFO]\tGene IL11RA has 0 variants, skipping\n",
      "[INFO]\tGene MIR6830 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3936HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3936 has 0 variants, skipping\n",
      "[INFO]\tGene ATPSCKMT has 0 variants, skipping\n",
      "[INFO]\tGene TH2LCRR has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8L1 has 0 variants, skipping\n",
      "[INFO]\tGene LCT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1294 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1208 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379176 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC160 has 0 variants, skipping\n",
      "[INFO]\tGene ADM5 has 0 variants, skipping\n",
      "[INFO]\tGene KLF15 has 0 variants, skipping\n",
      "[INFO]\tGene OAZ2 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1289-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379183 has 0 variants, skipping\n",
      "[INFO]\tGene CDH15 has 0 variants, skipping\n",
      "[INFO]\tGene CDH6 has 0 variants, skipping\n",
      "[INFO]\tGene PLK1 has 0 variants, skipping\n",
      "[INFO]\tGene PITX3 has 0 variants, skipping\n",
      "[INFO]\tGene PROCR has 0 variants, skipping\n",
      "[INFO]\tGene PEX16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644215 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3661 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546229 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01843 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00544 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM5 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692C1 has 0 variants, skipping\n",
      "[INFO]\tGene FA2H has 0 variants, skipping\n",
      "[INFO]\tGene FBXL21P has 0 variants, skipping\n",
      "[INFO]\tGene TRPC7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PEAK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP9B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379192 has 0 variants, skipping\n",
      "[INFO]\tGene NPY6R has 0 variants, skipping\n",
      "[INFO]\tGene OR2L3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2F2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2D3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130172 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL2 has 0 variants, skipping\n",
      "[INFO]\tGene LONP1 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD63B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD63 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379194 has 0 variants, skipping\n",
      "[INFO]\tGene IGIP has 0 variants, skipping\n",
      "[INFO]\tGene DIO1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW2 has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PXYLP1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXK1 has 0 variants, skipping\n",
      "[INFO]\tGene MSS51 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA45 has 0 variants, skipping\n",
      "[INFO]\tGene HES3 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf66 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf53 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74A has 0 variants, skipping\n",
      "[INFO]\tGene CHD5 has 0 variants, skipping\n",
      "[INFO]\tGene TUFT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM126A has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK13 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM220A has 0 variants, skipping\n",
      "[INFO]\tGene PSD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MALINC1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929719 has 0 variants, skipping\n",
      "[INFO]\tGene ANKHD1-EIF4EBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SP8 has 0 variants, skipping\n",
      "[INFO]\tGene DHX57 has 0 variants, skipping\n",
      "[INFO]\tGene GCSH has 0 variants, skipping\n",
      "[INFO]\tGene MIR6831 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3655 has 0 variants, skipping\n",
      "[INFO]\tGene RAF1 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-2 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-3 has 0 variants, skipping\n",
      "[INFO]\tGene VNN3 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926905 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB17P has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB18P has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB19P has 0 variants, skipping\n",
      "[INFO]\tGene KAZALD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM170A has 0 variants, skipping\n",
      "[INFO]\tGene INPP5D has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB8P has 0 variants, skipping\n",
      "[INFO]\tGene RPS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD8 has 0 variants, skipping\n",
      "[INFO]\tGene RGS2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL has 0 variants, skipping\n",
      "[INFO]\tGene RGS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABRA has 0 variants, skipping\n",
      "[INFO]\tGene LOC729080 has 0 variants, skipping\n",
      "[INFO]\tGene CLP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TH has 0 variants, skipping\n",
      "[INFO]\tGene TGFBI has 0 variants, skipping\n",
      "[INFO]\tGene TAF10 has 0 variants, skipping\n",
      "[INFO]\tGene STK4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01844 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO4 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG6 has 0 variants, skipping\n",
      "[INFO]\tGene ILF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5197 has 0 variants, skipping\n",
      "[INFO]\tGene C13orf46 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723899 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene MED28 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKC has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHO2 has 0 variants, skipping\n",
      "[INFO]\tGene PROS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2B-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene STK32A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP8B1 has 0 variants, skipping\n",
      "[INFO]\tGene NOTO has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5F1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546294 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC2-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CKLF-CMTM1 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5100 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO28 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1D has 0 variants, skipping\n",
      "[INFO]\tGene POLB has 0 variants, skipping\n",
      "[INFO]\tGene SCCPDH has 0 variants, skipping\n",
      "[INFO]\tGene NUDT13 has 0 variants, skipping\n",
      "[INFO]\tGene CDC40 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5-ARHGAP8 has 0 variants, skipping\n",
      "[INFO]\tGene FKBPL has 0 variants, skipping\n",
      "[INFO]\tGene ITM2B has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP3 has 0 variants, skipping\n",
      "[INFO]\tGene CHP2 has 0 variants, skipping\n",
      "[INFO]\tGene LY96 has 0 variants, skipping\n",
      "[INFO]\tGene COX6C has 0 variants, skipping\n",
      "[INFO]\tGene COX6A1 has 0 variants, skipping\n",
      "[INFO]\tGene EBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf131 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ1B has 0 variants, skipping\n",
      "[INFO]\tGene NDST1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE22 has 0 variants, skipping\n",
      "[INFO]\tGene GOT1L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF572 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF786 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG has 0 variants, skipping\n",
      "[INFO]\tGene WDR93 has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6499 has 0 variants, skipping\n",
      "[INFO]\tGene CLMAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652758 has 0 variants, skipping\n",
      "[INFO]\tGene CENPB has 0 variants, skipping\n",
      "[INFO]\tGene IBSP has 0 variants, skipping\n",
      "[INFO]\tGene SFPQ has 0 variants, skipping\n",
      "[INFO]\tGene LINC01933 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01470 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00920 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A12P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01861 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3141 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378H has 0 variants, skipping\n",
      "[INFO]\tGene C11orf96 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB3 has 0 variants, skipping\n",
      "[INFO]\tGene PUM3 has 0 variants, skipping\n",
      "[INFO]\tGene IL26 has 0 variants, skipping\n",
      "[INFO]\tGene LSG1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF8 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP161 has 0 variants, skipping\n",
      "[INFO]\tGene SETD1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724404 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02227 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP3 has 0 variants, skipping\n",
      "[INFO]\tGene SELPLG has 0 variants, skipping\n",
      "[INFO]\tGene SELP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02202 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01932 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285626 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01847 has 0 variants, skipping\n",
      "[INFO]\tGene CHERP has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF20 has 0 variants, skipping\n",
      "[INFO]\tGene TUBAL3 has 0 variants, skipping\n",
      "[INFO]\tGene CLUAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02159 has 0 variants, skipping\n",
      "[INFO]\tGene ATG9B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF680 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01202 has 0 variants, skipping\n",
      "[INFO]\tGene BTG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPA1 has 0 variants, skipping\n",
      "[INFO]\tGene JOSD2 has 0 variants, skipping\n",
      "[INFO]\tGene MATN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546299 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01947 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927908 has 0 variants, skipping\n",
      "[INFO]\tGene CTB-178M22.2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR585 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G1 has 0 variants, skipping\n",
      "[INFO]\tGene HEIH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01187 has 0 variants, skipping\n",
      "[INFO]\tGene CALR4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128059 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01366 has 0 variants, skipping\n",
      "[INFO]\tGene TRIML2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70J has 0 variants, skipping\n",
      "[INFO]\tGene MIR3912 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH4 has 0 variants, skipping\n",
      "[INFO]\tGene PIM3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37D has 0 variants, skipping\n",
      "[INFO]\tGene CAD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288254 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01944 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5003 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100268168 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC179 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8056 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01942 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01484 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01485 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01411 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4634 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01951 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996385 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507387 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643201 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37B has 0 variants, skipping\n",
      "[INFO]\tGene VPS4B has 0 variants, skipping\n",
      "[INFO]\tGene ZFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC16 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E1 has 0 variants, skipping\n",
      "[INFO]\tGene ATOX1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4281 has 0 variants, skipping\n",
      "[INFO]\tGene VKORC1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01574 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF12 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM5 has 0 variants, skipping\n",
      "[INFO]\tGene C6 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP9 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF13 has 0 variants, skipping\n",
      "[INFO]\tGene TAF7 has 0 variants, skipping\n",
      "[INFO]\tGene OR2H2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB5 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC1B has 0 variants, skipping\n",
      "[INFO]\tGene HEBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS18 has 0 variants, skipping\n",
      "[INFO]\tGene LOC202181 has 0 variants, skipping\n",
      "[INFO]\tGene FAM153A has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986794 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728554 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128340 has 0 variants, skipping\n",
      "[INFO]\tGene RHOU has 0 variants, skipping\n",
      "[INFO]\tGene ARL4C has 0 variants, skipping\n",
      "[INFO]\tGene AACSP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL15 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC1 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2B has 0 variants, skipping\n",
      "[INFO]\tGene HOPX has 0 variants, skipping\n",
      "[INFO]\tGene TMEM183B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF747 has 0 variants, skipping\n",
      "[INFO]\tGene CDK6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377763 has 0 variants, skipping\n",
      "[INFO]\tGene RRM2B has 0 variants, skipping\n",
      "[INFO]\tGene STAT4 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE9 has 0 variants, skipping\n",
      "[INFO]\tGene CD163L1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996419 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02222 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF506 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00847 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2D has 0 variants, skipping\n",
      "[INFO]\tGene RAB33A has 0 variants, skipping\n",
      "[INFO]\tGene CIAO1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A14 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4C has 0 variants, skipping\n",
      "[INFO]\tGene MIR8089 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01962 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD96A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD95 has 0 variants, skipping\n",
      "[INFO]\tGene CTC-338M12.4 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM52-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf80 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB39 has 0 variants, skipping\n",
      "[INFO]\tGene HIC2 has 0 variants, skipping\n",
      "[INFO]\tGene NOXO1 has 0 variants, skipping\n",
      "[INFO]\tGene IL11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF728 has 0 variants, skipping\n",
      "[INFO]\tGene MTTP has 0 variants, skipping\n",
      "[INFO]\tGene ECEL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF469 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929599 has 0 variants, skipping\n",
      "[INFO]\tGene AANAT has 0 variants, skipping\n",
      "[INFO]\tGene RPL26 has 0 variants, skipping\n",
      "[INFO]\tGene CEP295 has 0 variants, skipping\n",
      "[INFO]\tGene ACP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS2 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD5 has 0 variants, skipping\n",
      "[INFO]\tGene CD101 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL4 has 0 variants, skipping\n",
      "[INFO]\tGene ARID5B has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2C has 0 variants, skipping\n",
      "[INFO]\tGene ETV3 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPH has 0 variants, skipping\n",
      "[INFO]\tGene FICD has 0 variants, skipping\n",
      "[INFO]\tGene GPC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00266-3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285766 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927691 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6720 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01600 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02521 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4645 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB9P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927730 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01011 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927759 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02880 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY4 has 0 variants, skipping\n",
      "[INFO]\tGene VNN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130357 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724096 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02533 has 0 variants, skipping\n",
      "[INFO]\tGene KU-MEL-3 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3691 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927972 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927950 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7853 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5683 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf71 has 0 variants, skipping\n",
      "[INFO]\tGene CLMP has 0 variants, skipping\n",
      "[INFO]\tGene CAMKMT has 0 variants, skipping\n",
      "[INFO]\tGene HCN2 has 0 variants, skipping\n",
      "[INFO]\tGene BMPR2 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S5-TXNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RTP2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1E1-BLOC1S5 has 0 variants, skipping\n",
      "[INFO]\tGene WEE2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506207 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5689HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR5689 has 0 variants, skipping\n",
      "[INFO]\tGene RIIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928191 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D16 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3K has 0 variants, skipping\n",
      "[INFO]\tGene FBXO40 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02530 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A42 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ2 has 0 variants, skipping\n",
      "[INFO]\tGene NR3C2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36AL has 0 variants, skipping\n",
      "[INFO]\tGene RPL39 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB7 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD16B has 0 variants, skipping\n",
      "[INFO]\tGene SAMD10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724843 has 0 variants, skipping\n",
      "[INFO]\tGene CIT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01108 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO31 has 0 variants, skipping\n",
      "[INFO]\tGene JARID2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4639 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene S100A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLN has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928491 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374952 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP11 has 0 variants, skipping\n",
      "[INFO]\tGene SOAT2 has 0 variants, skipping\n",
      "[INFO]\tGene UTF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF282 has 0 variants, skipping\n",
      "[INFO]\tGene VAPA has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A1HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928519 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374960 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00581 has 0 variants, skipping\n",
      "[INFO]\tGene POLE4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374972 has 0 variants, skipping\n",
      "[INFO]\tGene NRP2 has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CD3E has 0 variants, skipping\n",
      "[INFO]\tGene CD3D has 0 variants, skipping\n",
      "[INFO]\tGene CCKAR has 0 variants, skipping\n",
      "[INFO]\tGene CALCB has 0 variants, skipping\n",
      "[INFO]\tGene CACNG1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A3 has 0 variants, skipping\n",
      "[INFO]\tGene NAA20 has 0 variants, skipping\n",
      "[INFO]\tGene HEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT16 has 0 variants, skipping\n",
      "[INFO]\tGene CRLF3 has 0 variants, skipping\n",
      "[INFO]\tGene RASD1 has 0 variants, skipping\n",
      "[INFO]\tGene CMAHP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928663 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC96 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2B has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6 has 0 variants, skipping\n",
      "[INFO]\tGene SMCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A3P has 0 variants, skipping\n",
      "[INFO]\tGene C12orf75 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A25 has 0 variants, skipping\n",
      "[INFO]\tGene OR13J1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285819 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374988 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00240 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100270746 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3143 has 0 variants, skipping\n",
      "[INFO]\tGene GPR17 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R10P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF204P has 0 variants, skipping\n",
      "[INFO]\tGene GUF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01012 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131289 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN12P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf61 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF192P1 has 0 variants, skipping\n",
      "[INFO]\tGene TOB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCH has 0 variants, skipping\n",
      "[INFO]\tGene CFAP206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01623 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01556 has 0 variants, skipping\n",
      "[INFO]\tGene HCG15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129636 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929006 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01015 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD32B has 0 variants, skipping\n",
      "[INFO]\tGene ENDOG has 0 variants, skipping\n",
      "[INFO]\tGene EPHA2 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM4P has 0 variants, skipping\n",
      "[INFO]\tGene HCG4 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-V has 0 variants, skipping\n",
      "[INFO]\tGene HCP5B has 0 variants, skipping\n",
      "[INFO]\tGene GPR3 has 0 variants, skipping\n",
      "[INFO]\tGene HCG4B has 0 variants, skipping\n",
      "[INFO]\tGene HCG9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD1ASP has 0 variants, skipping\n",
      "[INFO]\tGene HLA-J has 0 variants, skipping\n",
      "[INFO]\tGene GJB5 has 0 variants, skipping\n",
      "[INFO]\tGene PGA5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM31-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG17 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-L has 0 variants, skipping\n",
      "[INFO]\tGene HCG18 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM39-RPP21 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02569 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP47 has 0 variants, skipping\n",
      "[INFO]\tGene GNG13 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF581 has 0 variants, skipping\n",
      "[INFO]\tGene KLF3 has 0 variants, skipping\n",
      "[INFO]\tGene MDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IER3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG20 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4640 has 0 variants, skipping\n",
      "[INFO]\tGene HCG21 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM77 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02571 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6891 has 0 variants, skipping\n",
      "[INFO]\tGene MICA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG26 has 0 variants, skipping\n",
      "[INFO]\tGene MICB-DT has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G2-DDX39B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD117 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD84 has 0 variants, skipping\n",
      "[INFO]\tGene DDX39B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287329 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA7 has 0 variants, skipping\n",
      "[INFO]\tGene KRT6A has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-9 has 0 variants, skipping\n",
      "[INFO]\tGene HDC has 0 variants, skipping\n",
      "[INFO]\tGene ODC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6832 has 0 variants, skipping\n",
      "[INFO]\tGene CARD17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4646 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6E has 0 variants, skipping\n",
      "[INFO]\tGene HK3 has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH5-SAPCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRERF1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1G has 0 variants, skipping\n",
      "[INFO]\tGene SNORD48 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD52 has 0 variants, skipping\n",
      "[INFO]\tGene SOGA1 has 0 variants, skipping\n",
      "[INFO]\tGene XDH has 0 variants, skipping\n",
      "[INFO]\tGene C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E24 has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D9B has 0 variants, skipping\n",
      "[INFO]\tGene SGPP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF688 has 0 variants, skipping\n",
      "[INFO]\tGene MORN5 has 0 variants, skipping\n",
      "[INFO]\tGene TNXA has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10Q1 has 0 variants, skipping\n",
      "[INFO]\tGene MGA has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507547 has 0 variants, skipping\n",
      "[INFO]\tGene PPT2-EGFL8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6721 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6833 has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L3 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG23 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB6 has 0 variants, skipping\n",
      "[INFO]\tGene GAMT has 0 variants, skipping\n",
      "[INFO]\tGene PABPC5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB129 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100294145 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPB2 has 0 variants, skipping\n",
      "[INFO]\tGene HCG24 has 0 variants, skipping\n",
      "[INFO]\tGene ARSJ has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378979 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT3 has 0 variants, skipping\n",
      "[INFO]\tGene HCG25 has 0 variants, skipping\n",
      "[INFO]\tGene ZP3 has 0 variants, skipping\n",
      "[INFO]\tGene GPBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene MYG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM35A has 0 variants, skipping\n",
      "[INFO]\tGene PBOV1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6873 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD62 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6834 has 0 variants, skipping\n",
      "[INFO]\tGene WASL has 0 variants, skipping\n",
      "[INFO]\tGene USP13 has 0 variants, skipping\n",
      "[INFO]\tGene PEX11G has 0 variants, skipping\n",
      "[INFO]\tGene MIR5004 has 0 variants, skipping\n",
      "[INFO]\tGene GGNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A2 has 0 variants, skipping\n",
      "[INFO]\tGene MLXIPL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3934 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3G has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR510 has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01016 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7159 has 0 variants, skipping\n",
      "[INFO]\tGene OR5C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6835 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10-NUDT3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929243 has 0 variants, skipping\n",
      "[INFO]\tGene PATE4 has 0 variants, skipping\n",
      "[INFO]\tGene GPD1 has 0 variants, skipping\n",
      "[INFO]\tGene CDS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2C has 0 variants, skipping\n",
      "[INFO]\tGene CDK9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7111 has 0 variants, skipping\n",
      "[INFO]\tGene MIR153-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5690 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285847 has 0 variants, skipping\n",
      "[INFO]\tGene CD302 has 0 variants, skipping\n",
      "[INFO]\tGene MKX has 0 variants, skipping\n",
      "[INFO]\tGene TMED9 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A5 has 0 variants, skipping\n",
      "[INFO]\tGene RBM38 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR609 has 0 variants, skipping\n",
      "[INFO]\tGene MIR596 has 0 variants, skipping\n",
      "[INFO]\tGene RHOB has 0 variants, skipping\n",
      "[INFO]\tGene MIR3925 has 0 variants, skipping\n",
      "[INFO]\tGene DINOL has 0 variants, skipping\n",
      "[INFO]\tGene C6orf163 has 0 variants, skipping\n",
      "[INFO]\tGene PCP4L1 has 0 variants, skipping\n",
      "[INFO]\tGene BSG has 0 variants, skipping\n",
      "[INFO]\tGene MSI2 has 0 variants, skipping\n",
      "[INFO]\tGene MZT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP155 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02520 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4462 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MACF1 has 0 variants, skipping\n",
      "[INFO]\tGene APOD has 0 variants, skipping\n",
      "[INFO]\tGene APOC4 has 0 variants, skipping\n",
      "[INFO]\tGene APOA2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRN2 has 0 variants, skipping\n",
      "[INFO]\tGene MUC12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00951 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929555 has 0 variants, skipping\n",
      "[INFO]\tGene HES1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA11 has 0 variants, skipping\n",
      "[INFO]\tGene HMX2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A6OS has 0 variants, skipping\n",
      "[INFO]\tGene PDE6A has 0 variants, skipping\n",
      "[INFO]\tGene ADCY10P1 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM2 has 0 variants, skipping\n",
      "[INFO]\tGene TREML3P has 0 variants, skipping\n",
      "[INFO]\tGene IDO1 has 0 variants, skipping\n",
      "[INFO]\tGene TREML5P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01276 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4641 has 0 variants, skipping\n",
      "[INFO]\tGene NPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTD1 has 0 variants, skipping\n",
      "[INFO]\tGene CA11 has 0 variants, skipping\n",
      "[INFO]\tGene CA9 has 0 variants, skipping\n",
      "[INFO]\tGene C1QBP has 0 variants, skipping\n",
      "[INFO]\tGene BUB1B has 0 variants, skipping\n",
      "[INFO]\tGene REELD1 has 0 variants, skipping\n",
      "[INFO]\tGene USP11 has 0 variants, skipping\n",
      "[INFO]\tGene GPR25 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR22 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR3 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR20 has 0 variants, skipping\n",
      "[INFO]\tGene GPR37 has 0 variants, skipping\n",
      "[INFO]\tGene GPR15 has 0 variants, skipping\n",
      "[INFO]\tGene GPR12 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0CP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401261 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT2 has 0 variants, skipping\n",
      "[INFO]\tGene USH1C has 0 variants, skipping\n",
      "[INFO]\tGene FOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene ERMP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC15 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS20 has 0 variants, skipping\n",
      "[INFO]\tGene LAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM1 has 0 variants, skipping\n",
      "[INFO]\tGene SRARP has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6780B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10 has 0 variants, skipping\n",
      "[INFO]\tGene PIGL has 0 variants, skipping\n",
      "[INFO]\tGene GOLPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PBLD has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02537 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01512 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929705 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6C has 0 variants, skipping\n",
      "[INFO]\tGene SYCE3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4647 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD2 has 0 variants, skipping\n",
      "[INFO]\tGene TLL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375075 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF300P1 has 0 variants, skipping\n",
      "[INFO]\tGene FIRRE has 0 variants, skipping\n",
      "[INFO]\tGene TMEM65 has 0 variants, skipping\n",
      "[INFO]\tGene GET4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926915 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926898 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926934 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF605 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL23 has 0 variants, skipping\n",
      "[INFO]\tGene CPLX3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927020 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927048 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370980 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927082 has 0 variants, skipping\n",
      "[INFO]\tGene LINCMD1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIG7 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD18 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA7P has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL1 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE has 0 variants, skipping\n",
      "[INFO]\tGene MIR5685 has 0 variants, skipping\n",
      "[INFO]\tGene RPS16P5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927136 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01564 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927189 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSH has 0 variants, skipping\n",
      "[INFO]\tGene CTBS has 0 variants, skipping\n",
      "[INFO]\tGene NCAN has 0 variants, skipping\n",
      "[INFO]\tGene TRIP6 has 0 variants, skipping\n",
      "[INFO]\tGene TPR has 0 variants, skipping\n",
      "[INFO]\tGene ICOS has 0 variants, skipping\n",
      "[INFO]\tGene METTL16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM185B has 0 variants, skipping\n",
      "[INFO]\tGene DSTYK has 0 variants, skipping\n",
      "[INFO]\tGene PCLO has 0 variants, skipping\n",
      "[INFO]\tGene RFESD has 0 variants, skipping\n",
      "[INFO]\tGene FAM160A2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM210A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF451-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506188 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYC has 0 variants, skipping\n",
      "[INFO]\tGene MIR548U has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00680-GUSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00680 has 0 variants, skipping\n",
      "[INFO]\tGene MPP7 has 0 variants, skipping\n",
      "[INFO]\tGene OPN3 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX58 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT8 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723883 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02549 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928307 has 0 variants, skipping\n",
      "[INFO]\tGene EVADR has 0 variants, skipping\n",
      "[INFO]\tGene GRM1 has 0 variants, skipping\n",
      "[INFO]\tGene SACS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01626 has 0 variants, skipping\n",
      "[INFO]\tGene ETV6 has 0 variants, skipping\n",
      "[INFO]\tGene GRN has 0 variants, skipping\n",
      "[INFO]\tGene HSPA5 has 0 variants, skipping\n",
      "[INFO]\tGene APCS has 0 variants, skipping\n",
      "[INFO]\tGene APOF has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A6 has 0 variants, skipping\n",
      "[INFO]\tGene PEAR1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AB1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378949 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP27-1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110C has 0 variants, skipping\n",
      "[INFO]\tGene SH2D7 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO3 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN25 has 0 variants, skipping\n",
      "[INFO]\tGene PIRT has 0 variants, skipping\n",
      "[INFO]\tGene FAM183A has 0 variants, skipping\n",
      "[INFO]\tGene IGLON5 has 0 variants, skipping\n",
      "[INFO]\tGene YAE1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM9 has 0 variants, skipping\n",
      "[INFO]\tGene SKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPH has 0 variants, skipping\n",
      "[INFO]\tGene AMELX has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15B has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15 has 0 variants, skipping\n",
      "[INFO]\tGene PHEX has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928489 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928516 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD156 has 0 variants, skipping\n",
      "[INFO]\tGene RUSC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928540 has 0 variants, skipping\n",
      "[INFO]\tGene PATJ has 0 variants, skipping\n",
      "[INFO]\tGene LINC02540 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100144595 has 0 variants, skipping\n",
      "[INFO]\tGene CCNK has 0 variants, skipping\n",
      "[INFO]\tGene HMGN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01621 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01526 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02542 has 0 variants, skipping\n",
      "[INFO]\tGene DLK1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN1A1 has 0 variants, skipping\n",
      "[INFO]\tGene BTG1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM21 has 0 variants, skipping\n",
      "[INFO]\tGene AFF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377879 has 0 variants, skipping\n",
      "[INFO]\tGene TBX18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02535 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL1 has 0 variants, skipping\n",
      "[INFO]\tGene ERI1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K2 has 0 variants, skipping\n",
      "[INFO]\tGene MASP2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3195 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC12 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R5 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R4 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01590 has 0 variants, skipping\n",
      "[INFO]\tGene RBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928911 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928936 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929057 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4464 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4643 has 0 variants, skipping\n",
      "[INFO]\tGene CASC6 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB3 has 0 variants, skipping\n",
      "[INFO]\tGene TSG1 has 0 variants, skipping\n",
      "[INFO]\tGene MANEA-DT has 0 variants, skipping\n",
      "[INFO]\tGene KIF20A has 0 variants, skipping\n",
      "[INFO]\tGene CRAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR642B has 0 variants, skipping\n",
      "[INFO]\tGene MIR676 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3926-1 has 0 variants, skipping\n",
      "[INFO]\tGene DEGS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927314 has 0 variants, skipping\n",
      "[INFO]\tGene HCP5 has 0 variants, skipping\n",
      "[INFO]\tGene OR10T2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D11 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D10 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T33 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T29 has 0 variants, skipping\n",
      "[INFO]\tGene CHRFAM7A has 0 variants, skipping\n",
      "[INFO]\tGene CTHRC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927365 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D18 has 0 variants, skipping\n",
      "[INFO]\tGene CLMN has 0 variants, skipping\n",
      "[INFO]\tGene C1orf195 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN9-DT has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL2-DT has 0 variants, skipping\n",
      "[INFO]\tGene DHX33-DT has 0 variants, skipping\n",
      "[INFO]\tGene MCHR2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD1 has 0 variants, skipping\n",
      "[INFO]\tGene ATPAF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP1L has 0 variants, skipping\n",
      "[INFO]\tGene SETD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLBD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6869 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6743 has 0 variants, skipping\n",
      "[INFO]\tGene GPX2 has 0 variants, skipping\n",
      "[INFO]\tGene BVES-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIOS has 0 variants, skipping\n",
      "[INFO]\tGene C1orf146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377924 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02526 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02532 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC74A has 0 variants, skipping\n",
      "[INFO]\tGene CNOT6L has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPB has 0 variants, skipping\n",
      "[INFO]\tGene OSTM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2A has 0 variants, skipping\n",
      "[INFO]\tGene C9orf92 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00222 has 0 variants, skipping\n",
      "[INFO]\tGene RPL24 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA3 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD2A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC162P has 0 variants, skipping\n",
      "[INFO]\tGene ELP3 has 0 variants, skipping\n",
      "[INFO]\tGene TJP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEFF1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS20 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2D2 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40C has 0 variants, skipping\n",
      "[INFO]\tGene GSTM2P1 has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD166 has 0 variants, skipping\n",
      "[INFO]\tGene STC1 has 0 variants, skipping\n",
      "[INFO]\tGene TXN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR135 has 0 variants, skipping\n",
      "[INFO]\tGene NDST4 has 0 variants, skipping\n",
      "[INFO]\tGene NYX has 0 variants, skipping\n",
      "[INFO]\tGene TLNRD1 has 0 variants, skipping\n",
      "[INFO]\tGene GH1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02518 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02541 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02534 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1260A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287467 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT8 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAPL2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL3 has 0 variants, skipping\n",
      "[INFO]\tGene USP18 has 0 variants, skipping\n",
      "[INFO]\tGene APLN has 0 variants, skipping\n",
      "[INFO]\tGene FZD3 has 0 variants, skipping\n",
      "[INFO]\tGene ZUP1 has 0 variants, skipping\n",
      "[INFO]\tGene GREB1L has 0 variants, skipping\n",
      "[INFO]\tGene LPIN3 has 0 variants, skipping\n",
      "[INFO]\tGene IFI6 has 0 variants, skipping\n",
      "[INFO]\tGene ITK has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927919 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377967 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2QL1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD7P3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOKP3 has 0 variants, skipping\n",
      "[INFO]\tGene GAK has 0 variants, skipping\n",
      "[INFO]\tGene MIR548B has 0 variants, skipping\n",
      "[INFO]\tGene LOC285762 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377975 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3144 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T3 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT6 has 0 variants, skipping\n",
      "[INFO]\tGene TRDN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63A has 0 variants, skipping\n",
      "[INFO]\tGene RNF217-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MEFV has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126584 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02523 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene HRH4 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM12 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM13 has 0 variants, skipping\n",
      "[INFO]\tGene CHST6 has 0 variants, skipping\n",
      "[INFO]\tGene CARD18 has 0 variants, skipping\n",
      "[INFO]\tGene BARX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF708 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA17 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene IL22RA1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR588 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene UBALD2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM169B has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS12 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4D has 0 variants, skipping\n",
      "[INFO]\tGene EP300 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM35 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN19 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP11 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT18 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PKP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH11Y has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-8 has 0 variants, skipping\n",
      "[INFO]\tGene DPRX has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AJ1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01013 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR3P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD101 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD100 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00326 has 0 variants, skipping\n",
      "[INFO]\tGene GIPC3 has 0 variants, skipping\n",
      "[INFO]\tGene TARID has 0 variants, skipping\n",
      "[INFO]\tGene LINC01312 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA1P7 has 0 variants, skipping\n",
      "[INFO]\tGene CT69 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01010 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928304 has 0 variants, skipping\n",
      "[INFO]\tGene OR4L1 has 0 variants, skipping\n",
      "[INFO]\tGene SERTM2 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A18 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC69 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT3 has 0 variants, skipping\n",
      "[INFO]\tGene OR10P1 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR9 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX43 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF827 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A2 has 0 variants, skipping\n",
      "[INFO]\tGene BSCL2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644135 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM68 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928429 has 0 variants, skipping\n",
      "[INFO]\tGene NHEG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723649 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02539 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02528 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3145 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46906 has 0 variants, skipping\n",
      "[INFO]\tGene CRTC3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01625 has 0 variants, skipping\n",
      "[INFO]\tGene FILNC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507477 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103352541 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3668 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A2 has 0 variants, skipping\n",
      "[INFO]\tGene CT66 has 0 variants, skipping\n",
      "[INFO]\tGene CCT8L2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC153910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01277 has 0 variants, skipping\n",
      "[INFO]\tGene RSU1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC89 has 0 variants, skipping\n",
      "[INFO]\tGene HEPACAM has 0 variants, skipping\n",
      "[INFO]\tGene DOK6 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF6 has 0 variants, skipping\n",
      "[INFO]\tGene TPSG1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOP3 has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB2 has 0 variants, skipping\n",
      "[INFO]\tGene F12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA98 has 0 variants, skipping\n",
      "[INFO]\tGene TPSAB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928661 has 0 variants, skipping\n",
      "[INFO]\tGene KATNBL1P6 has 0 variants, skipping\n",
      "[INFO]\tGene UST-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2-RAB5IF has 0 variants, skipping\n",
      "[INFO]\tGene TALDO1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS18P9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM131C has 0 variants, skipping\n",
      "[INFO]\tGene LOC645967 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378052 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1K has 0 variants, skipping\n",
      "[INFO]\tGene MIR505 has 0 variants, skipping\n",
      "[INFO]\tGene MIR503 has 0 variants, skipping\n",
      "[INFO]\tGene MIR502 has 0 variants, skipping\n",
      "[INFO]\tGene MIR501 has 0 variants, skipping\n",
      "[INFO]\tGene MIR500A has 0 variants, skipping\n",
      "[INFO]\tGene MIR499A has 0 variants, skipping\n",
      "[INFO]\tGene MIR519A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520H has 0 variants, skipping\n",
      "[INFO]\tGene MIR518D has 0 variants, skipping\n",
      "[INFO]\tGene MIR520C has 0 variants, skipping\n",
      "[INFO]\tGene MIR518B has 0 variants, skipping\n",
      "[INFO]\tGene MIR520G has 0 variants, skipping\n",
      "[INFO]\tGene MIR520B has 0 variants, skipping\n",
      "[INFO]\tGene MIR525 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519D has 0 variants, skipping\n",
      "[INFO]\tGene MIR519B has 0 variants, skipping\n",
      "[INFO]\tGene MIR524 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526B has 0 variants, skipping\n",
      "[INFO]\tGene MIR520A has 0 variants, skipping\n",
      "[INFO]\tGene MIR519C has 0 variants, skipping\n",
      "[INFO]\tGene MIR515-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520E has 0 variants, skipping\n",
      "[INFO]\tGene MIR498 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181D has 0 variants, skipping\n",
      "[INFO]\tGene MIR497 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193B has 0 variants, skipping\n",
      "[INFO]\tGene MIR496 has 0 variants, skipping\n",
      "[INFO]\tGene MIR495 has 0 variants, skipping\n",
      "[INFO]\tGene MIR494 has 0 variants, skipping\n",
      "[INFO]\tGene MIR432 has 0 variants, skipping\n",
      "[INFO]\tGene MIR493 has 0 variants, skipping\n",
      "[INFO]\tGene MIR202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR146B has 0 variants, skipping\n",
      "[INFO]\tGene MIR511 has 0 variants, skipping\n",
      "[INFO]\tGene MIR491 has 0 variants, skipping\n",
      "[INFO]\tGene MIR490 has 0 variants, skipping\n",
      "[INFO]\tGene MIR489 has 0 variants, skipping\n",
      "[INFO]\tGene MIR488 has 0 variants, skipping\n",
      "[INFO]\tGene MIR486-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR485 has 0 variants, skipping\n",
      "[INFO]\tGene MIR508 has 0 variants, skipping\n",
      "[INFO]\tGene MIR507 has 0 variants, skipping\n",
      "[INFO]\tGene MIR506 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR504 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723831 has 0 variants, skipping\n",
      "[INFO]\tGene PEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1 has 0 variants, skipping\n",
      "[INFO]\tGene DGKK has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH3 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN2 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TLDC2 has 0 variants, skipping\n",
      "[INFO]\tGene RNLS has 0 variants, skipping\n",
      "[INFO]\tGene MIR1273C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378068 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD28B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4466 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3692 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7161 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3918 has 0 variants, skipping\n",
      "[INFO]\tGene OSTCP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM178A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02529 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT3 has 0 variants, skipping\n",
      "[INFO]\tGene USP26 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG8 has 0 variants, skipping\n",
      "[INFO]\tGene SOD2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene OPRK1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA29 has 0 variants, skipping\n",
      "[INFO]\tGene AIRN has 0 variants, skipping\n",
      "[INFO]\tGene UTRN has 0 variants, skipping\n",
      "[INFO]\tGene LPAL2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM238L has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC8P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4301 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378098 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZp451B082 has 0 variants, skipping\n",
      "[INFO]\tGene CAHM has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724152 has 0 variants, skipping\n",
      "[INFO]\tGene MEAT6 has 0 variants, skipping\n",
      "[INFO]\tGene FGF16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729681 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289495 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO47 has 0 variants, skipping\n",
      "[INFO]\tGene RAB19 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1913 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3939 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378123 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378127 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02538 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01558 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441179 has 0 variants, skipping\n",
      "[INFO]\tGene PKD2 has 0 variants, skipping\n",
      "[INFO]\tGene POU3F4 has 0 variants, skipping\n",
      "[INFO]\tGene PON2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378137 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929420 has 0 variants, skipping\n",
      "[INFO]\tGene SP9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929460 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02544 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929523 has 0 variants, skipping\n",
      "[INFO]\tGene FAM153B has 0 variants, skipping\n",
      "[INFO]\tGene OR4A5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00242 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00574 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724511 has 0 variants, skipping\n",
      "[INFO]\tGene LOC154449 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285804 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01624 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF816 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4644 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G3 has 0 variants, skipping\n",
      "[INFO]\tGene MMP10 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf203 has 0 variants, skipping\n",
      "[INFO]\tGene NTN3 has 0 variants, skipping\n",
      "[INFO]\tGene NTF4 has 0 variants, skipping\n",
      "[INFO]\tGene NPBWR2 has 0 variants, skipping\n",
      "[INFO]\tGene NPBWR1 has 0 variants, skipping\n",
      "[INFO]\tGene XCR1 has 0 variants, skipping\n",
      "[INFO]\tGene NPRL2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268269 has 0 variants, skipping\n",
      "[INFO]\tGene RNF111 has 0 variants, skipping\n",
      "[INFO]\tGene TRABD2B has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1E has 0 variants, skipping\n",
      "[INFO]\tGene SHOX2 has 0 variants, skipping\n",
      "[INFO]\tGene GMPR has 0 variants, skipping\n",
      "[INFO]\tGene B4GAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PIM2 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK3 has 0 variants, skipping\n",
      "[INFO]\tGene AOC2 has 0 variants, skipping\n",
      "[INFO]\tGene MNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH4 has 0 variants, skipping\n",
      "[INFO]\tGene MYL10 has 0 variants, skipping\n",
      "[INFO]\tGene GSE1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL9 has 0 variants, skipping\n",
      "[INFO]\tGene MCUB has 0 variants, skipping\n",
      "[INFO]\tGene ACOT7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928436 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC49A4 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf65 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA12A has 0 variants, skipping\n",
      "[INFO]\tGene C17orf58 has 0 variants, skipping\n",
      "[INFO]\tGene NUCB1 has 0 variants, skipping\n",
      "[INFO]\tGene NTS has 0 variants, skipping\n",
      "[INFO]\tGene CYP4V2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A34 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN54 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC9A has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP1B has 0 variants, skipping\n",
      "[INFO]\tGene ZP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC24 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184A has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J3 has 0 variants, skipping\n",
      "[INFO]\tGene RAC2 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA5 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC3L has 0 variants, skipping\n",
      "[INFO]\tGene OR13C4 has 0 variants, skipping\n",
      "[INFO]\tGene MED4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34B has 0 variants, skipping\n",
      "[INFO]\tGene SMYD4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB31 has 0 variants, skipping\n",
      "[INFO]\tGene VWDE has 0 variants, skipping\n",
      "[INFO]\tGene FBN2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX47 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA2 has 0 variants, skipping\n",
      "[INFO]\tGene NPSR1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17A has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIE has 0 variants, skipping\n",
      "[INFO]\tGene POU3F2 has 0 variants, skipping\n",
      "[INFO]\tGene MYF5 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B1 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F3 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHD has 0 variants, skipping\n",
      "[INFO]\tGene HMX3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO45 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7C has 0 variants, skipping\n",
      "[INFO]\tGene MSGN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL2 has 0 variants, skipping\n",
      "[INFO]\tGene UTP4 has 0 variants, skipping\n",
      "[INFO]\tGene SOS1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L has 0 variants, skipping\n",
      "[INFO]\tGene NOX5 has 0 variants, skipping\n",
      "[INFO]\tGene PALMD has 0 variants, skipping\n",
      "[INFO]\tGene COL6A3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL4 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD10 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR5 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5A has 0 variants, skipping\n",
      "[INFO]\tGene CETN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1B has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF5 has 0 variants, skipping\n",
      "[INFO]\tGene CPA6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49 has 0 variants, skipping\n",
      "[INFO]\tGene RNF115 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP2A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF21 has 0 variants, skipping\n",
      "[INFO]\tGene SIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF16 has 0 variants, skipping\n",
      "[INFO]\tGene SERP1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17B has 0 variants, skipping\n",
      "[INFO]\tGene IL36A has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0232 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L1 has 0 variants, skipping\n",
      "[INFO]\tGene KEL has 0 variants, skipping\n",
      "[INFO]\tGene IAPP has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM125 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL1 has 0 variants, skipping\n",
      "[INFO]\tGene CBARP has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN20 has 0 variants, skipping\n",
      "[INFO]\tGene FOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene MVD has 0 variants, skipping\n",
      "[INFO]\tGene FAM43B has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDPF1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L3 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4A has 0 variants, skipping\n",
      "[INFO]\tGene ACSM6 has 0 variants, skipping\n",
      "[INFO]\tGene EFNB3 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1G has 0 variants, skipping\n",
      "[INFO]\tGene PEX3 has 0 variants, skipping\n",
      "[INFO]\tGene PARP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP22-2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP40 has 0 variants, skipping\n",
      "[INFO]\tGene FAM228A has 0 variants, skipping\n",
      "[INFO]\tGene CDHR1 has 0 variants, skipping\n",
      "[INFO]\tGene ISLR has 0 variants, skipping\n",
      "[INFO]\tGene BLK has 0 variants, skipping\n",
      "[INFO]\tGene TPM2 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC1 has 0 variants, skipping\n",
      "[INFO]\tGene ATCAY has 0 variants, skipping\n",
      "[INFO]\tGene MAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MYL7 has 0 variants, skipping\n",
      "[INFO]\tGene AASDHPPT has 0 variants, skipping\n",
      "[INFO]\tGene CDADC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAN has 0 variants, skipping\n",
      "[INFO]\tGene AFM has 0 variants, skipping\n",
      "[INFO]\tGene CRISP1 has 0 variants, skipping\n",
      "[INFO]\tGene AEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TLE5 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1E has 0 variants, skipping\n",
      "[INFO]\tGene GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY14 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC10 has 0 variants, skipping\n",
      "[INFO]\tGene DCD has 0 variants, skipping\n",
      "[INFO]\tGene MEOX2 has 0 variants, skipping\n",
      "[INFO]\tGene IGF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC175 has 0 variants, skipping\n",
      "[INFO]\tGene PARP2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6A has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3C has 0 variants, skipping\n",
      "[INFO]\tGene FAM199X has 0 variants, skipping\n",
      "[INFO]\tGene ZNF474 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM207 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS33 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A9 has 0 variants, skipping\n",
      "[INFO]\tGene RPL3 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF1B has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3B has 0 variants, skipping\n",
      "[INFO]\tGene GAU1 has 0 variants, skipping\n",
      "[INFO]\tGene NHS has 0 variants, skipping\n",
      "[INFO]\tGene SNHG10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR155HG has 0 variants, skipping\n",
      "[INFO]\tGene RN7SK has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPAB has 0 variants, skipping\n",
      "[INFO]\tGene FAM83B has 0 variants, skipping\n",
      "[INFO]\tGene NANS has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-3 has 0 variants, skipping\n",
      "[INFO]\tGene BHMT has 0 variants, skipping\n",
      "[INFO]\tGene BGN has 0 variants, skipping\n",
      "[INFO]\tGene ICAM5 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO15 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B13 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10-AS has 0 variants, skipping\n",
      "[INFO]\tGene SRMS has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM86A has 0 variants, skipping\n",
      "[INFO]\tGene BEND2 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP6 has 0 variants, skipping\n",
      "[INFO]\tGene ERMAP has 0 variants, skipping\n",
      "[INFO]\tGene P2RX3 has 0 variants, skipping\n",
      "[INFO]\tGene DAO has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3E has 0 variants, skipping\n",
      "[INFO]\tGene UAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210L has 0 variants, skipping\n",
      "[INFO]\tGene CXCL2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL3 has 0 variants, skipping\n",
      "[INFO]\tGene RECQL4 has 0 variants, skipping\n",
      "[INFO]\tGene PDAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DARS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCC has 0 variants, skipping\n",
      "[INFO]\tGene RFXANK has 0 variants, skipping\n",
      "[INFO]\tGene UQCRHL has 0 variants, skipping\n",
      "[INFO]\tGene BCYRN1 has 0 variants, skipping\n",
      "[INFO]\tGene TERC has 0 variants, skipping\n",
      "[INFO]\tGene XIST has 0 variants, skipping\n",
      "[INFO]\tGene LINC00261 has 0 variants, skipping\n",
      "[INFO]\tGene ASGR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516A1 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA4 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM2 has 0 variants, skipping\n",
      "[INFO]\tGene FHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF44 has 0 variants, skipping\n",
      "[INFO]\tGene LMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene IFIH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A7 has 0 variants, skipping\n",
      "[INFO]\tGene AAAS has 0 variants, skipping\n",
      "[INFO]\tGene HOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM3 has 0 variants, skipping\n",
      "[INFO]\tGene PSKH1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1A has 0 variants, skipping\n",
      "[INFO]\tGene MCM5 has 0 variants, skipping\n",
      "[INFO]\tGene FZD7 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC71 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2C has 0 variants, skipping\n",
      "[INFO]\tGene MYH15 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4D has 0 variants, skipping\n",
      "[INFO]\tGene FAM214A has 0 variants, skipping\n",
      "[INFO]\tGene MRPL34 has 0 variants, skipping\n",
      "[INFO]\tGene PAPLN has 0 variants, skipping\n",
      "[INFO]\tGene MIR891B has 0 variants, skipping\n",
      "[INFO]\tGene MIR920 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A2 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF724 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14A2 has 0 variants, skipping\n",
      "[INFO]\tGene NCR3LG1 has 0 variants, skipping\n",
      "[INFO]\tGene ERN1 has 0 variants, skipping\n",
      "[INFO]\tGene EN1 has 0 variants, skipping\n",
      "[INFO]\tGene EMP3 has 0 variants, skipping\n",
      "[INFO]\tGene EMP2 has 0 variants, skipping\n",
      "[INFO]\tGene CHROMR has 0 variants, skipping\n",
      "[INFO]\tGene SAA2-SAA4 has 0 variants, skipping\n",
      "[INFO]\tGene DEPTOR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF582 has 0 variants, skipping\n",
      "[INFO]\tGene SBNO1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR83OS has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B12 has 0 variants, skipping\n",
      "[INFO]\tGene CPQ has 0 variants, skipping\n",
      "[INFO]\tGene TBX18 has 0 variants, skipping\n",
      "[INFO]\tGene PLA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3142HG has 0 variants, skipping\n",
      "[INFO]\tGene RPS27 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX1 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC2 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS3 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BPL has 0 variants, skipping\n",
      "[INFO]\tGene SUN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200C has 0 variants, skipping\n",
      "[INFO]\tGene NUPR2 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf144 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL30 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF38 has 0 variants, skipping\n",
      "[INFO]\tGene SAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SYT13 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM30 has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE has 0 variants, skipping\n",
      "[INFO]\tGene COL17A1 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX2 has 0 variants, skipping\n",
      "[INFO]\tGene USP38 has 0 variants, skipping\n",
      "[INFO]\tGene OR1F12 has 0 variants, skipping\n",
      "[INFO]\tGene SKIDA1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR153 has 0 variants, skipping\n",
      "[INFO]\tGene PLXDC2 has 0 variants, skipping\n",
      "[INFO]\tGene OXSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS4 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1E has 0 variants, skipping\n",
      "[INFO]\tGene OR1J4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1B1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF10 has 0 variants, skipping\n",
      "[INFO]\tGene LY86 has 0 variants, skipping\n",
      "[INFO]\tGene MED7 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR1A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR1B has 0 variants, skipping\n",
      "[INFO]\tGene ERV3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ACADL has 0 variants, skipping\n",
      "[INFO]\tGene POLR3D has 0 variants, skipping\n",
      "[INFO]\tGene BMP7 has 0 variants, skipping\n",
      "[INFO]\tGene RBM26 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT4 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL42 has 0 variants, skipping\n",
      "[INFO]\tGene DISP3 has 0 variants, skipping\n",
      "[INFO]\tGene PSD4 has 0 variants, skipping\n",
      "[INFO]\tGene FANCE has 0 variants, skipping\n",
      "[INFO]\tGene SCT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF70 has 0 variants, skipping\n",
      "[INFO]\tGene CFL2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRB2 has 0 variants, skipping\n",
      "[INFO]\tGene PFKL has 0 variants, skipping\n",
      "[INFO]\tGene CFAP73 has 0 variants, skipping\n",
      "[INFO]\tGene NAA30 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP3 has 0 variants, skipping\n",
      "[INFO]\tGene INSL4 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA3 has 0 variants, skipping\n",
      "[INFO]\tGene INPP1 has 0 variants, skipping\n",
      "[INFO]\tGene INHBB has 0 variants, skipping\n",
      "[INFO]\tGene INHBA has 0 variants, skipping\n",
      "[INFO]\tGene ISL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A4 has 0 variants, skipping\n",
      "[INFO]\tGene HNF4G has 0 variants, skipping\n",
      "[INFO]\tGene WFDC6 has 0 variants, skipping\n",
      "[INFO]\tGene ISM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A3 has 0 variants, skipping\n",
      "[INFO]\tGene RINL has 0 variants, skipping\n",
      "[INFO]\tGene C3orf84 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCOS has 0 variants, skipping\n",
      "[INFO]\tGene ANHX has 0 variants, skipping\n",
      "[INFO]\tGene DUSP21 has 0 variants, skipping\n",
      "[INFO]\tGene CERS2 has 0 variants, skipping\n",
      "[INFO]\tGene RRP1B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV2 has 0 variants, skipping\n",
      "[INFO]\tGene HCN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN4 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375115 has 0 variants, skipping\n",
      "[INFO]\tGene LOC442497 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT92 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM163 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83D has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN4 has 0 variants, skipping\n",
      "[INFO]\tGene LBH has 0 variants, skipping\n",
      "[INFO]\tGene OR2A4 has 0 variants, skipping\n",
      "[INFO]\tGene CPLANE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF696 has 0 variants, skipping\n",
      "[INFO]\tGene OR2J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T1 has 0 variants, skipping\n",
      "[INFO]\tGene STPG1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK13 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf82 has 0 variants, skipping\n",
      "[INFO]\tGene NNT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK6-DT has 0 variants, skipping\n",
      "[INFO]\tGene ESD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128653 has 0 variants, skipping\n",
      "[INFO]\tGene TFAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4655 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA114 has 0 variants, skipping\n",
      "[INFO]\tGene SOX18 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4648 has 0 variants, skipping\n",
      "[INFO]\tGene COL26A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR593 has 0 variants, skipping\n",
      "[INFO]\tGene MIR590 has 0 variants, skipping\n",
      "[INFO]\tGene MIR584 has 0 variants, skipping\n",
      "[INFO]\tGene MIR582 has 0 variants, skipping\n",
      "[INFO]\tGene MIR571 has 0 variants, skipping\n",
      "[INFO]\tGene MIR579 has 0 variants, skipping\n",
      "[INFO]\tGene MIR551B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129603 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD165 has 0 variants, skipping\n",
      "[INFO]\tGene CAT has 0 variants, skipping\n",
      "[INFO]\tGene RNF216P1 has 0 variants, skipping\n",
      "[INFO]\tGene RBAKDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF890P has 0 variants, skipping\n",
      "[INFO]\tGene USP42 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS13 has 0 variants, skipping\n",
      "[INFO]\tGene ANKFN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC221946 has 0 variants, skipping\n",
      "[INFO]\tGene RNF216-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6874 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF815P has 0 variants, skipping\n",
      "[INFO]\tGene SOGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A5 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2D has 0 variants, skipping\n",
      "[INFO]\tGene LOH12CR2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB2B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80D has 0 variants, skipping\n",
      "[INFO]\tGene SV2C has 0 variants, skipping\n",
      "[INFO]\tGene PMS2CL has 0 variants, skipping\n",
      "[INFO]\tGene MIR3683 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131257 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927354 has 0 variants, skipping\n",
      "[INFO]\tGene SPOPL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM211 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927391 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L2 has 0 variants, skipping\n",
      "[INFO]\tGene HESX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705D has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505938 has 0 variants, skipping\n",
      "[INFO]\tGene MGC4859 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131472 has 0 variants, skipping\n",
      "[INFO]\tGene CHPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725191 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF4 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK6 has 0 variants, skipping\n",
      "[INFO]\tGene INMT has 0 variants, skipping\n",
      "[INFO]\tGene DDX18 has 0 variants, skipping\n",
      "[INFO]\tGene KRT38 has 0 variants, skipping\n",
      "[INFO]\tGene LMO4 has 0 variants, skipping\n",
      "[INFO]\tGene BRAP has 0 variants, skipping\n",
      "[INFO]\tGene LZTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SIX2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375166 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02587 has 0 variants, skipping\n",
      "[INFO]\tGene CRPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RINT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF11 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB16 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRFS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCIN has 0 variants, skipping\n",
      "[INFO]\tGene UCHL3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM4 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf51A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-6 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A7 has 0 variants, skipping\n",
      "[INFO]\tGene CPHXL has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL1B has 0 variants, skipping\n",
      "[INFO]\tGene TEX13D has 0 variants, skipping\n",
      "[INFO]\tGene MIR3146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927668 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J5 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506098 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927769 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927811 has 0 variants, skipping\n",
      "[INFO]\tGene ERMN has 0 variants, skipping\n",
      "[INFO]\tGene RPL23P8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01162 has 0 variants, skipping\n",
      "[INFO]\tGene MEIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF7 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8C has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf16 has 0 variants, skipping\n",
      "[INFO]\tGene WDR24 has 0 variants, skipping\n",
      "[INFO]\tGene CCL13 has 0 variants, skipping\n",
      "[INFO]\tGene LCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401312 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD93 has 0 variants, skipping\n",
      "[INFO]\tGene POC1B-GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL7-DT has 0 variants, skipping\n",
      "[INFO]\tGene GAL has 0 variants, skipping\n",
      "[INFO]\tGene GOLT1B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS7 has 0 variants, skipping\n",
      "[INFO]\tGene PAM16 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS23 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS17 has 0 variants, skipping\n",
      "[INFO]\tGene FIS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65C has 0 variants, skipping\n",
      "[INFO]\tGene RPS2P32 has 0 variants, skipping\n",
      "[INFO]\tGene CHST9 has 0 variants, skipping\n",
      "[INFO]\tGene CLK2P1 has 0 variants, skipping\n",
      "[INFO]\tGene NAB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929322 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375304 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPPSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02805 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4295 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0087 has 0 variants, skipping\n",
      "[INFO]\tGene RHOC has 0 variants, skipping\n",
      "[INFO]\tGene BACH2 has 0 variants, skipping\n",
      "[INFO]\tGene OSTN has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10-HOXA9 has 0 variants, skipping\n",
      "[INFO]\tGene PINX1 has 0 variants, skipping\n",
      "[INFO]\tGene EVX1-AS has 0 variants, skipping\n",
      "[INFO]\tGene MAU2 has 0 variants, skipping\n",
      "[INFO]\tGene TNS2 has 0 variants, skipping\n",
      "[INFO]\tGene TSL has 0 variants, skipping\n",
      "[INFO]\tGene EDF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01297-DUXAP10-NBEAP6 has 0 variants, skipping\n",
      "[INFO]\tGene JAZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506497 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928168 has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD4-RWDD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646762 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NEAT1 has 0 variants, skipping\n",
      "[INFO]\tGene RSBN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375218 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550B1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP586I1420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01176 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf48 has 0 variants, skipping\n",
      "[INFO]\tGene INMT-MINDY4 has 0 variants, skipping\n",
      "[INFO]\tGene PYGB has 0 variants, skipping\n",
      "[INFO]\tGene GABRA6 has 0 variants, skipping\n",
      "[INFO]\tGene EPO has 0 variants, skipping\n",
      "[INFO]\tGene GABRA5 has 0 variants, skipping\n",
      "[INFO]\tGene DRD5 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA3 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GSK3B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130673 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2D has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550B2 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1P2 has 0 variants, skipping\n",
      "[INFO]\tGene RP9P has 0 variants, skipping\n",
      "[INFO]\tGene OR51A4 has 0 variants, skipping\n",
      "[INFO]\tGene GDF6 has 0 variants, skipping\n",
      "[INFO]\tGene KLK5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB6 has 0 variants, skipping\n",
      "[INFO]\tGene TYW1B has 0 variants, skipping\n",
      "[INFO]\tGene NPSR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401324 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930085 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506725 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928618 has 0 variants, skipping\n",
      "[INFO]\tGene TPP1 has 0 variants, skipping\n",
      "[INFO]\tGene AOAH-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1AN has 0 variants, skipping\n",
      "[INFO]\tGene ELMO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF19 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB126 has 0 variants, skipping\n",
      "[INFO]\tGene UNC93B1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM183BP has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA20B has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01197 has 0 variants, skipping\n",
      "[INFO]\tGene HAGLR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01450 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01449 has 0 variants, skipping\n",
      "[INFO]\tGene INHBA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IL31 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2G has 0 variants, skipping\n",
      "[INFO]\tGene LINC01448 has 0 variants, skipping\n",
      "[INFO]\tGene HECW1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3943 has 0 variants, skipping\n",
      "[INFO]\tGene LUARIS has 0 variants, skipping\n",
      "[INFO]\tGene MIR492 has 0 variants, skipping\n",
      "[INFO]\tGene STARD5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF648 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPA has 0 variants, skipping\n",
      "[INFO]\tGene CLDN24 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP4 has 0 variants, skipping\n",
      "[INFO]\tGene GDI1 has 0 variants, skipping\n",
      "[INFO]\tGene CCS has 0 variants, skipping\n",
      "[INFO]\tGene RASA4CP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00957 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6837 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6838 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4649 has 0 variants, skipping\n",
      "[INFO]\tGene ARL1 has 0 variants, skipping\n",
      "[INFO]\tGene COQ2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB8B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD40CL has 0 variants, skipping\n",
      "[INFO]\tGene C4orf33 has 0 variants, skipping\n",
      "[INFO]\tGene RNF112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4657 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM201A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5B has 0 variants, skipping\n",
      "[INFO]\tGene LOC730234 has 0 variants, skipping\n",
      "[INFO]\tGene MED17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730338 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD151 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01447 has 0 variants, skipping\n",
      "[INFO]\tGene DSC3 has 0 variants, skipping\n",
      "[INFO]\tGene DSC2 has 0 variants, skipping\n",
      "[INFO]\tGene PIGBOS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COL14A1 has 0 variants, skipping\n",
      "[INFO]\tGene L2HGDH has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100509620 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01446 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01445 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2A-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf34 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT4 has 0 variants, skipping\n",
      "[INFO]\tGene HARBI1 has 0 variants, skipping\n",
      "[INFO]\tGene TIGIT has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC12 has 0 variants, skipping\n",
      "[INFO]\tGene NEGR1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP9P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT11 has 0 variants, skipping\n",
      "[INFO]\tGene XKRY2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC650226 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240728 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZp434L192 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928401 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401357 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130849 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4283-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4283-2 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100653233 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3147 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF733P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724738 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287704 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287834 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF722P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01005 has 0 variants, skipping\n",
      "[INFO]\tGene THEM6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF771 has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN5 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAEP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128885 has 0 variants, skipping\n",
      "[INFO]\tGene LOC641746 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF273 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5D1 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6P3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15B-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15B-1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441242 has 0 variants, skipping\n",
      "[INFO]\tGene INTS4P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22 has 0 variants, skipping\n",
      "[INFO]\tGene VWA2 has 0 variants, skipping\n",
      "[INFO]\tGene ARSD has 0 variants, skipping\n",
      "[INFO]\tGene ACTL9 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-124K5.4 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-124K5.11 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996437 has 0 variants, skipping\n",
      "[INFO]\tGene OR6F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51I1 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf91 has 0 variants, skipping\n",
      "[INFO]\tGene ARMS2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02604 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4650-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4650-2 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P4 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01372 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723427 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3914-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3914-2 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL3 has 0 variants, skipping\n",
      "[INFO]\tGene SBDSP1 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5P2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100101148 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L3 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P7 has 0 variants, skipping\n",
      "[INFO]\tGene WDR76 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCF1B has 0 variants, skipping\n",
      "[INFO]\tGene EIF4H has 0 variants, skipping\n",
      "[INFO]\tGene GLDC has 0 variants, skipping\n",
      "[INFO]\tGene GLI3 has 0 variants, skipping\n",
      "[INFO]\tGene GJA1 has 0 variants, skipping\n",
      "[INFO]\tGene GIPR has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L7 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L8 has 0 variants, skipping\n",
      "[INFO]\tGene RELN has 0 variants, skipping\n",
      "[INFO]\tGene AQP3 has 0 variants, skipping\n",
      "[INFO]\tGene H1-10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926943 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L2 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P5 has 0 variants, skipping\n",
      "[INFO]\tGene ARID1A has 0 variants, skipping\n",
      "[INFO]\tGene PPIL4 has 0 variants, skipping\n",
      "[INFO]\tGene NCF1C has 0 variants, skipping\n",
      "[INFO]\tGene C15orf54 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14C has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5P1 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P3 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA14A has 0 variants, skipping\n",
      "[INFO]\tGene PVALB has 0 variants, skipping\n",
      "[INFO]\tGene GJA8 has 0 variants, skipping\n",
      "[INFO]\tGene GJA5 has 0 variants, skipping\n",
      "[INFO]\tGene GFER has 0 variants, skipping\n",
      "[INFO]\tGene GEM has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP7 has 0 variants, skipping\n",
      "[INFO]\tGene HBA2 has 0 variants, skipping\n",
      "[INFO]\tGene CASQ2 has 0 variants, skipping\n",
      "[INFO]\tGene CASQ1 has 0 variants, skipping\n",
      "[INFO]\tGene FDPSP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKIB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133091 has 0 variants, skipping\n",
      "[INFO]\tGene DTX2P1-UPK3BP1-PMS2P11 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM185BP has 0 variants, skipping\n",
      "[INFO]\tGene EDEM1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3C has 0 variants, skipping\n",
      "[INFO]\tGene MYLK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927243 has 0 variants, skipping\n",
      "[INFO]\tGene ING5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF852 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP17 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF113B has 0 variants, skipping\n",
      "[INFO]\tGene SLC51B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01555 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927269 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128317 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5E has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00972 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927420 has 0 variants, skipping\n",
      "[INFO]\tGene PTMS has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene MFN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP18 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS3 has 0 variants, skipping\n",
      "[INFO]\tGene TPGS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL5 has 0 variants, skipping\n",
      "[INFO]\tGene DISP2 has 0 variants, skipping\n",
      "[INFO]\tGene MSMP has 0 variants, skipping\n",
      "[INFO]\tGene DSG2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR2 has 0 variants, skipping\n",
      "[INFO]\tGene DSG1 has 0 variants, skipping\n",
      "[INFO]\tGene GAB1 has 0 variants, skipping\n",
      "[INFO]\tGene DR1 has 0 variants, skipping\n",
      "[INFO]\tGene DPT has 0 variants, skipping\n",
      "[INFO]\tGene ACKR1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIPT has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4A has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP8 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL18 has 0 variants, skipping\n",
      "[INFO]\tGene UCP3 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101409256 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF548 has 0 variants, skipping\n",
      "[INFO]\tGene FERD3L has 0 variants, skipping\n",
      "[INFO]\tGene CYP51A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAR1A has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR86 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD16 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM7P has 0 variants, skipping\n",
      "[INFO]\tGene GGNBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LTC4S has 0 variants, skipping\n",
      "[INFO]\tGene IFNL4 has 0 variants, skipping\n",
      "[INFO]\tGene TXNL1 has 0 variants, skipping\n",
      "[INFO]\tGene NDST3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02145 has 0 variants, skipping\n",
      "[INFO]\tGene COL1A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF578 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28B has 0 variants, skipping\n",
      "[INFO]\tGene CD2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF186 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A4RG has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA2B1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1 has 0 variants, skipping\n",
      "[INFO]\tGene HMOX1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM4 has 0 variants, skipping\n",
      "[INFO]\tGene CDT1 has 0 variants, skipping\n",
      "[INFO]\tGene CZ1P-ASNS has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692C2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLB has 0 variants, skipping\n",
      "[INFO]\tGene COX10 has 0 variants, skipping\n",
      "[INFO]\tGene CORT has 0 variants, skipping\n",
      "[INFO]\tGene CLIC2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5 has 0 variants, skipping\n",
      "[INFO]\tGene BIN3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA28 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927550 has 0 variants, skipping\n",
      "[INFO]\tGene MYH16 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL18 has 0 variants, skipping\n",
      "[INFO]\tGene CEP135 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D4 has 0 variants, skipping\n",
      "[INFO]\tGene NME5 has 0 variants, skipping\n",
      "[INFO]\tGene BCL10 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F8 has 0 variants, skipping\n",
      "[INFO]\tGene AZGP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375423 has 0 variants, skipping\n",
      "[INFO]\tGene CPPED1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene IMPDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K14 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA1 has 0 variants, skipping\n",
      "[INFO]\tGene GPAA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDC has 0 variants, skipping\n",
      "[INFO]\tGene GPR52 has 0 variants, skipping\n",
      "[INFO]\tGene GPR55 has 0 variants, skipping\n",
      "[INFO]\tGene TLR1 has 0 variants, skipping\n",
      "[INFO]\tGene TIPARP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM87A has 0 variants, skipping\n",
      "[INFO]\tGene GABRB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6845 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6809 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01672 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL4 has 0 variants, skipping\n",
      "[INFO]\tGene RBM43 has 0 variants, skipping\n",
      "[INFO]\tGene FAM89A has 0 variants, skipping\n",
      "[INFO]\tGene DRAXIN has 0 variants, skipping\n",
      "[INFO]\tGene IGFL1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF699 has 0 variants, skipping\n",
      "[INFO]\tGene TEX45 has 0 variants, skipping\n",
      "[INFO]\tGene PER3P1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P1 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L5P-PVRIG2P-PILRB has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L5P has 0 variants, skipping\n",
      "[INFO]\tGene PVRIG2P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6840 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB46 has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375429 has 0 variants, skipping\n",
      "[INFO]\tGene MOSMO has 0 variants, skipping\n",
      "[INFO]\tGene TRABD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF12 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L15 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B2P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD50B has 0 variants, skipping\n",
      "[INFO]\tGene GLYCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P25 has 0 variants, skipping\n",
      "[INFO]\tGene PCGEM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4653 has 0 variants, skipping\n",
      "[INFO]\tGene NUP50 has 0 variants, skipping\n",
      "[INFO]\tGene LNCPRESS1 has 0 variants, skipping\n",
      "[INFO]\tGene FREM3 has 0 variants, skipping\n",
      "[INFO]\tGene POTEM has 0 variants, skipping\n",
      "[INFO]\tGene LINC01007 has 0 variants, skipping\n",
      "[INFO]\tGene PGRMC2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIPT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4285 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100630923 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548O has 0 variants, skipping\n",
      "[INFO]\tGene GPSM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5090 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY11 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY4 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4DP has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506422 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT5 has 0 variants, skipping\n",
      "[INFO]\tGene RPL19P12 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P2 has 0 variants, skipping\n",
      "[INFO]\tGene SMOC1 has 0 variants, skipping\n",
      "[INFO]\tGene HYLS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927870 has 0 variants, skipping\n",
      "[INFO]\tGene IPO11 has 0 variants, skipping\n",
      "[INFO]\tGene RASL10A has 0 variants, skipping\n",
      "[INFO]\tGene TAF6L has 0 variants, skipping\n",
      "[INFO]\tGene MYL12A has 0 variants, skipping\n",
      "[INFO]\tGene IVNS1ABP has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01004 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CD7 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378E has 0 variants, skipping\n",
      "[INFO]\tGene MIR378D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR151B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00941 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927974 has 0 variants, skipping\n",
      "[INFO]\tGene RASIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB6 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP13 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996249 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928012 has 0 variants, skipping\n",
      "[INFO]\tGene RGS21 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT17 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT7 has 0 variants, skipping\n",
      "[INFO]\tGene PNKP has 0 variants, skipping\n",
      "[INFO]\tGene SFXN4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01393 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01392 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA25B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724434 has 0 variants, skipping\n",
      "[INFO]\tGene COQ9 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR3 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B10 has 0 variants, skipping\n",
      "[INFO]\tGene FN3K has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ3 has 0 variants, skipping\n",
      "[INFO]\tGene NEFH has 0 variants, skipping\n",
      "[INFO]\tGene ST7-OT4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6132 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-OT3 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC4 has 0 variants, skipping\n",
      "[INFO]\tGene REXO1 has 0 variants, skipping\n",
      "[INFO]\tGene LRPPRC has 0 variants, skipping\n",
      "[INFO]\tGene CFTR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX47 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF449 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM104 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45B has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SENP3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB5 has 0 variants, skipping\n",
      "[INFO]\tGene RGS22 has 0 variants, skipping\n",
      "[INFO]\tGene FZD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR433 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL6P has 0 variants, skipping\n",
      "[INFO]\tGene GOPC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375483 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928211 has 0 variants, skipping\n",
      "[INFO]\tGene IRS2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3A has 0 variants, skipping\n",
      "[INFO]\tGene POT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928283 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506682 has 0 variants, skipping\n",
      "[INFO]\tGene SND1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM238 has 0 variants, skipping\n",
      "[INFO]\tGene GET3 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01000 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23C has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF17 has 0 variants, skipping\n",
      "[INFO]\tGene FLNC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB5 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2AIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369535 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC407835 has 0 variants, skipping\n",
      "[INFO]\tGene COL15A1 has 0 variants, skipping\n",
      "[INFO]\tGene CTXND2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128325 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3928 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3922 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375504 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA3 has 0 variants, skipping\n",
      "[INFO]\tGene WDR82 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM9 has 0 variants, skipping\n",
      "[INFO]\tGene ACSBG1 has 0 variants, skipping\n",
      "[INFO]\tGene COPG2IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM4 has 0 variants, skipping\n",
      "[INFO]\tGene PACS1 has 0 variants, skipping\n",
      "[INFO]\tGene MKLN1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928782 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928807 has 0 variants, skipping\n",
      "[INFO]\tGene UCK2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ40288 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506937 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375512 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6133 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928861 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01230 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM101 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC349160 has 0 variants, skipping\n",
      "[INFO]\tGene FAM229B has 0 variants, skipping\n",
      "[INFO]\tGene LRIT2 has 0 variants, skipping\n",
      "[INFO]\tGene CPA2 has 0 variants, skipping\n",
      "[INFO]\tGene CPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4468 has 0 variants, skipping\n",
      "[INFO]\tGene GDF2 has 0 variants, skipping\n",
      "[INFO]\tGene LSM8 has 0 variants, skipping\n",
      "[INFO]\tGene CRNN has 0 variants, skipping\n",
      "[INFO]\tGene ACTL6B has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4A has 0 variants, skipping\n",
      "[INFO]\tGene MRTO4 has 0 variants, skipping\n",
      "[INFO]\tGene SRC has 0 variants, skipping\n",
      "[INFO]\tGene RIMS3 has 0 variants, skipping\n",
      "[INFO]\tGene FMC1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129148 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene RILPL2 has 0 variants, skipping\n",
      "[INFO]\tGene WEE2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK14 has 0 variants, skipping\n",
      "[INFO]\tGene INCA1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP27 has 0 variants, skipping\n",
      "[INFO]\tGene MOXD2P has 0 variants, skipping\n",
      "[INFO]\tGene TRY2P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC168 has 0 variants, skipping\n",
      "[INFO]\tGene OR6W1P has 0 variants, skipping\n",
      "[INFO]\tGene ETFRF1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00336 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6892 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC154761 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A9P has 0 variants, skipping\n",
      "[INFO]\tGene OR2A20P has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF34P has 0 variants, skipping\n",
      "[INFO]\tGene MIR197 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA2-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR196A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR298 has 0 variants, skipping\n",
      "[INFO]\tGene MIR192 has 0 variants, skipping\n",
      "[INFO]\tGene MIR675 has 0 variants, skipping\n",
      "[INFO]\tGene MIR16-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR107 has 0 variants, skipping\n",
      "[INFO]\tGene MIR106A has 0 variants, skipping\n",
      "[INFO]\tGene MIR103A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR101-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR100 has 0 variants, skipping\n",
      "[INFO]\tGene MIR98 has 0 variants, skipping\n",
      "[INFO]\tGene MIR96 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR93 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR33A has 0 variants, skipping\n",
      "[INFO]\tGene MIR24-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR24-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR31 has 0 variants, skipping\n",
      "[INFO]\tGene MIR23A has 0 variants, skipping\n",
      "[INFO]\tGene MIR30A has 0 variants, skipping\n",
      "[INFO]\tGene MIR22 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29A has 0 variants, skipping\n",
      "[INFO]\tGene MIR21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR20A has 0 variants, skipping\n",
      "[INFO]\tGene MIR27A has 0 variants, skipping\n",
      "[INFO]\tGene MIR19B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR19B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26B has 0 variants, skipping\n",
      "[INFO]\tGene MIR19A has 0 variants, skipping\n",
      "[INFO]\tGene MIR17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR16-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR15A has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7F1 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7E has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7D has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7B has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7C has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F4 has 0 variants, skipping\n",
      "[INFO]\tGene TAOK1 has 0 variants, skipping\n",
      "[INFO]\tGene BBOF1 has 0 variants, skipping\n",
      "[INFO]\tGene STN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNY5 has 0 variants, skipping\n",
      "[INFO]\tGene RNY4 has 0 variants, skipping\n",
      "[INFO]\tGene RNY3 has 0 variants, skipping\n",
      "[INFO]\tGene RNY1 has 0 variants, skipping\n",
      "[INFO]\tGene RBX1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC155060 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF767P has 0 variants, skipping\n",
      "[INFO]\tGene SLAIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPRM has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134040 has 0 variants, skipping\n",
      "[INFO]\tGene GPC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728743 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00996 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK7 has 0 variants, skipping\n",
      "[INFO]\tGene CD300H has 0 variants, skipping\n",
      "[INFO]\tGene OR9A2 has 0 variants, skipping\n",
      "[INFO]\tGene AVP has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D2 has 0 variants, skipping\n",
      "[INFO]\tGene NR1I2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3907 has 0 variants, skipping\n",
      "[INFO]\tGene MRAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LCOR has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM3AP has 0 variants, skipping\n",
      "[INFO]\tGene FABP5P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01003 has 0 variants, skipping\n",
      "[INFO]\tGene ASPN has 0 variants, skipping\n",
      "[INFO]\tGene RIT2 has 0 variants, skipping\n",
      "[INFO]\tGene GRK1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS16 has 0 variants, skipping\n",
      "[INFO]\tGene SHB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929998 has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene HTR5A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL4 has 0 variants, skipping\n",
      "[INFO]\tGene BLACE has 0 variants, skipping\n",
      "[INFO]\tGene LOC100286906 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506302 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF750 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285889 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01006 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00244 has 0 variants, skipping\n",
      "[INFO]\tGene IL13RA2 has 0 variants, skipping\n",
      "[INFO]\tGene VTN has 0 variants, skipping\n",
      "[INFO]\tGene MIR762 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1301 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1271 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320B1 has 0 variants, skipping\n",
      "[INFO]\tGene MNX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927914 has 0 variants, skipping\n",
      "[INFO]\tGene FXR2 has 0 variants, skipping\n",
      "[INFO]\tGene DAB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01022 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5707 has 0 variants, skipping\n",
      "[INFO]\tGene PARVA has 0 variants, skipping\n",
      "[INFO]\tGene UBR4 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS3P2 has 0 variants, skipping\n",
      "[INFO]\tGene FOSL2 has 0 variants, skipping\n",
      "[INFO]\tGene FOS has 0 variants, skipping\n",
      "[INFO]\tGene FOXC2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXG1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NRARP has 0 variants, skipping\n",
      "[INFO]\tGene PMM2 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM28 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP53 has 0 variants, skipping\n",
      "[INFO]\tGene FAM87A has 0 variants, skipping\n",
      "[INFO]\tGene DNAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286083 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927752 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3674 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928058 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD11-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7160 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927815 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287015 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A25 has 0 variants, skipping\n",
      "[INFO]\tGene ILDR2 has 0 variants, skipping\n",
      "[INFO]\tGene MCPH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8055 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4659A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4659B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT19 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-24F4.2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA8P has 0 variants, skipping\n",
      "[INFO]\tGene DEFA9P has 0 variants, skipping\n",
      "[INFO]\tGene DEFA10P has 0 variants, skipping\n",
      "[INFO]\tGene DEFT1P has 0 variants, skipping\n",
      "[INFO]\tGene DEFT1P2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA11P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00965 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A7P has 0 variants, skipping\n",
      "[INFO]\tGene FAM66B has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A10P has 0 variants, skipping\n",
      "[INFO]\tGene FAM66E has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM85B has 0 variants, skipping\n",
      "[INFO]\tGene ASDURF has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3I has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC157273 has 0 variants, skipping\n",
      "[INFO]\tGene BBS5 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0001 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4286 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723313 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929229 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929269 has 0 variants, skipping\n",
      "[INFO]\tGene TDH has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129129 has 0 variants, skipping\n",
      "[INFO]\tGene FAM167A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00208 has 0 variants, skipping\n",
      "[INFO]\tGene PGA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA99 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1973 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1915 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1908 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1469 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1307 has 0 variants, skipping\n",
      "[INFO]\tGene MIR664A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1281 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1252 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302F has 0 variants, skipping\n",
      "[INFO]\tGene MIR1275 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1265 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1258 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1251 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1249 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1247 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1246 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1304 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1299 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1297 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548E has 0 variants, skipping\n",
      "[INFO]\tGene MIR1207 has 0 variants, skipping\n",
      "[INFO]\tGene ASB18 has 0 variants, skipping\n",
      "[INFO]\tGene PATE2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM66D has 0 variants, skipping\n",
      "[INFO]\tGene LOC392196 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A2P has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM85A has 0 variants, skipping\n",
      "[INFO]\tGene FAM66A has 0 variants, skipping\n",
      "[INFO]\tGene LOC649352 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB109A has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A25P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506990 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729732 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3926-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC340357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00681 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725080 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF230 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P20 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00685 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF793 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548V has 0 variants, skipping\n",
      "[INFO]\tGene MOCS2 has 0 variants, skipping\n",
      "[INFO]\tGene SDR42E2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K2 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH3 has 0 variants, skipping\n",
      "[INFO]\tGene FBL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929066 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128993 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00536 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRG has 0 variants, skipping\n",
      "[INFO]\tGene TSPY4 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT6 has 0 variants, skipping\n",
      "[INFO]\tGene LZTS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467222 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02153 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929172 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCCB has 0 variants, skipping\n",
      "[INFO]\tGene CFAP36 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM26 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A12 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507071 has 0 variants, skipping\n",
      "[INFO]\tGene PSME4 has 0 variants, skipping\n",
      "[INFO]\tGene FAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MDN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN5 has 0 variants, skipping\n",
      "[INFO]\tGene SPP2 has 0 variants, skipping\n",
      "[INFO]\tGene BIN3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929237 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286059 has 0 variants, skipping\n",
      "[INFO]\tGene LOC254896 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389641 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507156 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA25 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2L has 0 variants, skipping\n",
      "[INFO]\tGene NFIA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929294 has 0 variants, skipping\n",
      "[INFO]\tGene TBX19 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02869 has 0 variants, skipping\n",
      "[INFO]\tGene CCR5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929315 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM178B has 0 variants, skipping\n",
      "[INFO]\tGene TEX13C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6841 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6876 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379331 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF367 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA4 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARPIN-AP3S2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H4 has 0 variants, skipping\n",
      "[INFO]\tGene EN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6842 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6843 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3622B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01703 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4287 has 0 variants, skipping\n",
      "[INFO]\tGene CD27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450A2 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-12 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM4B has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF113A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02099 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929470 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02209 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3148 has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene REG3G has 0 variants, skipping\n",
      "[INFO]\tGene NRG1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG1-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf51 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOT has 0 variants, skipping\n",
      "[INFO]\tGene TOB2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC75A has 0 variants, skipping\n",
      "[INFO]\tGene GRASLND has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01288 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929550 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929622 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723701 has 0 variants, skipping\n",
      "[INFO]\tGene GPR75-ASB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728024 has 0 variants, skipping\n",
      "[INFO]\tGene SYN2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10A has 0 variants, skipping\n",
      "[INFO]\tGene RNF5P1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH6 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF26 has 0 variants, skipping\n",
      "[INFO]\tGene CREBZF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF493 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO9 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS3BP has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38D has 0 variants, skipping\n",
      "[INFO]\tGene ADAM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130964 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AO has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65B has 0 variants, skipping\n",
      "[INFO]\tGene EDDM13 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D16 has 0 variants, skipping\n",
      "[INFO]\tGene KCNS3 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379393 has 0 variants, skipping\n",
      "[INFO]\tGene SCRN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929897 has 0 variants, skipping\n",
      "[INFO]\tGene IER5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC27 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC7 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4469 has 0 variants, skipping\n",
      "[INFO]\tGene ASNSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00293 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287846 has 0 variants, skipping\n",
      "[INFO]\tGene SDR9C7 has 0 variants, skipping\n",
      "[INFO]\tGene NPB has 0 variants, skipping\n",
      "[INFO]\tGene BOD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRBA1 has 0 variants, skipping\n",
      "[INFO]\tGene TNS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00472 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00520 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00313 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00691 has 0 variants, skipping\n",
      "[INFO]\tGene SATB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP52 has 0 variants, skipping\n",
      "[INFO]\tGene PSG10P has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3G has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929268 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929217 has 0 variants, skipping\n",
      "[INFO]\tGene PEX13 has 0 variants, skipping\n",
      "[INFO]\tGene PER1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507464 has 0 variants, skipping\n",
      "[INFO]\tGene ARSF has 0 variants, skipping\n",
      "[INFO]\tGene RPS7 has 0 variants, skipping\n",
      "[INFO]\tGene KITLG has 0 variants, skipping\n",
      "[INFO]\tGene SRP9 has 0 variants, skipping\n",
      "[INFO]\tGene SRM has 0 variants, skipping\n",
      "[INFO]\tGene PRB3 has 0 variants, skipping\n",
      "[INFO]\tGene PRB2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5B has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5A has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3D has 0 variants, skipping\n",
      "[INFO]\tGene PPEF2 has 0 variants, skipping\n",
      "[INFO]\tGene PPARD has 0 variants, skipping\n",
      "[INFO]\tGene POU4F1 has 0 variants, skipping\n",
      "[INFO]\tGene POU3F3 has 0 variants, skipping\n",
      "[INFO]\tGene POU2AF1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J has 0 variants, skipping\n",
      "[INFO]\tGene POLR2I has 0 variants, skipping\n",
      "[INFO]\tGene POLE has 0 variants, skipping\n",
      "[INFO]\tGene ACAA2 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAX has 0 variants, skipping\n",
      "[INFO]\tGene PIAS3 has 0 variants, skipping\n",
      "[INFO]\tGene RACK1 has 0 variants, skipping\n",
      "[INFO]\tGene SPON1 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE6 has 0 variants, skipping\n",
      "[INFO]\tGene PCNT has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929341 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01537 has 0 variants, skipping\n",
      "[INFO]\tGene NRIR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507516 has 0 variants, skipping\n",
      "[INFO]\tGene FEZ2 has 0 variants, skipping\n",
      "[INFO]\tGene HERPUD2 has 0 variants, skipping\n",
      "[INFO]\tGene TOR3A has 0 variants, skipping\n",
      "[INFO]\tGene ROBO3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU105C has 0 variants, skipping\n",
      "[INFO]\tGene SBF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375843 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA1B has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD54 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA3 has 0 variants, skipping\n",
      "[INFO]\tGene H3C13 has 0 variants, skipping\n",
      "[INFO]\tGene HMSD has 0 variants, skipping\n",
      "[INFO]\tGene SDR16C6P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929415 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00588 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929488 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286178 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929528 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01301 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02155 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR11G2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01289 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01414 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401463 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01299 has 0 variants, skipping\n",
      "[INFO]\tGene HR has 0 variants, skipping\n",
      "[INFO]\tGene LINC00967 has 0 variants, skipping\n",
      "[INFO]\tGene RRS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGH has 0 variants, skipping\n",
      "[INFO]\tGene GMPS has 0 variants, skipping\n",
      "[INFO]\tGene ARL8A has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723655 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723713 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD87 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724708 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX9 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM187A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01592 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01603 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT1P3 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7-DT has 0 variants, skipping\n",
      "[INFO]\tGene CCL24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926892 has 0 variants, skipping\n",
      "[INFO]\tGene LACTB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC392232 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926908 has 0 variants, skipping\n",
      "[INFO]\tGene RDH10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01617 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5681A has 0 variants, skipping\n",
      "[INFO]\tGene MIR5681B has 0 variants, skipping\n",
      "[INFO]\tGene TUSC7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2052 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01111 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3149 has 0 variants, skipping\n",
      "[INFO]\tGene WTIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724874 has 0 variants, skipping\n",
      "[INFO]\tGene PKIA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101241902 has 0 variants, skipping\n",
      "[INFO]\tGene PIGO has 0 variants, skipping\n",
      "[INFO]\tGene LINC01607 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927040 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389199 has 0 variants, skipping\n",
      "[INFO]\tGene TBKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5708 has 0 variants, skipping\n",
      "[INFO]\tGene IMPA1P1 has 0 variants, skipping\n",
      "[INFO]\tGene AMTN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723322 has 0 variants, skipping\n",
      "[INFO]\tGene REXO1L2P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929709 has 0 variants, skipping\n",
      "[INFO]\tGene PXN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00534 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01030 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4661 has 0 variants, skipping\n",
      "[INFO]\tGene LLCFC1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM274P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724710 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46284 has 0 variants, skipping\n",
      "[INFO]\tGene CARTPT has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00535 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R14 has 0 variants, skipping\n",
      "[INFO]\tGene CEP128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288748 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A4 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA5 has 0 variants, skipping\n",
      "[INFO]\tGene DTD2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HAV1L has 0 variants, skipping\n",
      "[INFO]\tGene C11orf52 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX3 has 0 variants, skipping\n",
      "[INFO]\tGene P4HTM has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4D has 0 variants, skipping\n",
      "[INFO]\tGene NPW has 0 variants, skipping\n",
      "[INFO]\tGene ARAP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR105-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150BHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01298 has 0 variants, skipping\n",
      "[INFO]\tGene AFDN-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724804 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927066 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF774 has 0 variants, skipping\n",
      "[INFO]\tGene FCER1A has 0 variants, skipping\n",
      "[INFO]\tGene CST5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA72 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2D has 0 variants, skipping\n",
      "[INFO]\tGene FAM83E has 0 variants, skipping\n",
      "[INFO]\tGene PTS has 0 variants, skipping\n",
      "[INFO]\tGene SAP30 has 0 variants, skipping\n",
      "[INFO]\tGene DPM2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF18 has 0 variants, skipping\n",
      "[INFO]\tGene BANF1 has 0 variants, skipping\n",
      "[INFO]\tGene GALR2 has 0 variants, skipping\n",
      "[INFO]\tGene NMUR2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD77B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4471 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7705 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42969 has 0 variants, skipping\n",
      "[INFO]\tGene NACA4P has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A20 has 0 variants, skipping\n",
      "[INFO]\tGene BLMH has 0 variants, skipping\n",
      "[INFO]\tGene APOB has 0 variants, skipping\n",
      "[INFO]\tGene LOC104054148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5680 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC72 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF23 has 0 variants, skipping\n",
      "[INFO]\tGene UBR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX9 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC13 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf61 has 0 variants, skipping\n",
      "[INFO]\tGene SLC20A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01181 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3151 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369147 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD173 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375690 has 0 variants, skipping\n",
      "[INFO]\tGene RNF126 has 0 variants, skipping\n",
      "[INFO]\tGene SKOR1 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT2 has 0 variants, skipping\n",
      "[INFO]\tGene STON2 has 0 variants, skipping\n",
      "[INFO]\tGene MIS18BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMA16 has 0 variants, skipping\n",
      "[INFO]\tGene GMDS has 0 variants, skipping\n",
      "[INFO]\tGene OR52I2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN12 has 0 variants, skipping\n",
      "[INFO]\tGene UHMK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO9 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2F has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3B has 0 variants, skipping\n",
      "[INFO]\tGene PER2 has 0 variants, skipping\n",
      "[INFO]\tGene MIF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PFN4 has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D2 has 0 variants, skipping\n",
      "[INFO]\tGene SVBP has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM9 has 0 variants, skipping\n",
      "[INFO]\tGene SPEM1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01608 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01609 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02237 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRA has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1B has 0 variants, skipping\n",
      "[INFO]\tGene SIX1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN31 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3A has 0 variants, skipping\n",
      "[INFO]\tGene S100P has 0 variants, skipping\n",
      "[INFO]\tGene AGFG2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375713 has 0 variants, skipping\n",
      "[INFO]\tGene RAD21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3610 has 0 variants, skipping\n",
      "[INFO]\tGene MZT2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD168 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927513 has 0 variants, skipping\n",
      "[INFO]\tGene MAL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AZ has 0 variants, skipping\n",
      "[INFO]\tGene DEFB127 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK5 has 0 variants, skipping\n",
      "[INFO]\tGene TPCN2 has 0 variants, skipping\n",
      "[INFO]\tGene MINAR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF425 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM88 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927543 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01151 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4663 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AA1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927588 has 0 variants, skipping\n",
      "[INFO]\tGene RNF139-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6844 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4662B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4662A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00964 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375744 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00861 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927657 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene CASC21 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00824 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00977 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3686 has 0 variants, skipping\n",
      "[INFO]\tGene FIBP has 0 variants, skipping\n",
      "[INFO]\tGene OR2C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B6 has 0 variants, skipping\n",
      "[INFO]\tGene OR1Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1F1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC146 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS14 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5194 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MGL has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A11 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH2 has 0 variants, skipping\n",
      "[INFO]\tGene UBR3 has 0 variants, skipping\n",
      "[INFO]\tGene HPYR1 has 0 variants, skipping\n",
      "[INFO]\tGene H4C11 has 0 variants, skipping\n",
      "[INFO]\tGene FDCSP has 0 variants, skipping\n",
      "[INFO]\tGene LIX1 has 0 variants, skipping\n",
      "[INFO]\tGene EID2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM92 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO39 has 0 variants, skipping\n",
      "[INFO]\tGene SBSPON has 0 variants, skipping\n",
      "[INFO]\tGene CILP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF524 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM174 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF19 has 0 variants, skipping\n",
      "[INFO]\tGene GJB4 has 0 variants, skipping\n",
      "[INFO]\tGene KIF19 has 0 variants, skipping\n",
      "[INFO]\tGene USP43 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7848 has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375773 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927822 has 0 variants, skipping\n",
      "[INFO]\tGene PRB4 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCRNA00250 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01591 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02055 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927915 has 0 variants, skipping\n",
      "[INFO]\tGene TIGAR has 0 variants, skipping\n",
      "[INFO]\tGene PEG13 has 0 variants, skipping\n",
      "[INFO]\tGene RGMB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOBOX has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ1-DT has 0 variants, skipping\n",
      "[INFO]\tGene GSTCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF878 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT2 has 0 variants, skipping\n",
      "[INFO]\tGene DHDDS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L3 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L4 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-9 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MPV17 has 0 variants, skipping\n",
      "[INFO]\tGene TENM4 has 0 variants, skipping\n",
      "[INFO]\tGene DBX1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT1 has 0 variants, skipping\n",
      "[INFO]\tGene CYS1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G2P has 0 variants, skipping\n",
      "[INFO]\tGene LYPD4 has 0 variants, skipping\n",
      "[INFO]\tGene GSTP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376306 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371049 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01300 has 0 variants, skipping\n",
      "[INFO]\tGene MROH5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4472-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00051 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2B has 0 variants, skipping\n",
      "[INFO]\tGene ASIC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928087 has 0 variants, skipping\n",
      "[INFO]\tGene HUNK has 0 variants, skipping\n",
      "[INFO]\tGene LY6E-DT has 0 variants, skipping\n",
      "[INFO]\tGene CDC42P3 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf31 has 0 variants, skipping\n",
      "[INFO]\tGene CRISPLD2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD149 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100310756 has 0 variants, skipping\n",
      "[INFO]\tGene NUBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT81 has 0 variants, skipping\n",
      "[INFO]\tGene KRT83 has 0 variants, skipping\n",
      "[INFO]\tGene CPB1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928160 has 0 variants, skipping\n",
      "[INFO]\tGene SPTA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4664 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5C has 0 variants, skipping\n",
      "[INFO]\tGene GPR173 has 0 variants, skipping\n",
      "[INFO]\tGene WDR5B has 0 variants, skipping\n",
      "[INFO]\tGene SNTG2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6846 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6847 has 0 variants, skipping\n",
      "[INFO]\tGene FAAP24 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6848 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928764 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928902 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1234 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6849 has 0 variants, skipping\n",
      "[INFO]\tGene TRERNA1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR9 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF39 has 0 variants, skipping\n",
      "[INFO]\tGene CEP89 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6850 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130027 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf167 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2A has 0 variants, skipping\n",
      "[INFO]\tGene COQ10A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF252P has 0 variants, skipping\n",
      "[INFO]\tGene TMED10P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF252P-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA3 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP6 has 0 variants, skipping\n",
      "[INFO]\tGene CPA3 has 0 variants, skipping\n",
      "[INFO]\tGene GMNC has 0 variants, skipping\n",
      "[INFO]\tGene TMEM88B has 0 variants, skipping\n",
      "[INFO]\tGene RNF222 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02303 has 0 variants, skipping\n",
      "[INFO]\tGene SYVN1 has 0 variants, skipping\n",
      "[INFO]\tGene RABIF has 0 variants, skipping\n",
      "[INFO]\tGene RAB6A has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM154 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6A has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC51A has 0 variants, skipping\n",
      "[INFO]\tGene STK36 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A3 has 0 variants, skipping\n",
      "[INFO]\tGene CHST13 has 0 variants, skipping\n",
      "[INFO]\tGene COL22A1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPD2 has 0 variants, skipping\n",
      "[INFO]\tGene USP37 has 0 variants, skipping\n",
      "[INFO]\tGene NXT2 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8 has 0 variants, skipping\n",
      "[INFO]\tGene EMC10 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA10 has 0 variants, skipping\n",
      "[INFO]\tGene HCRT has 0 variants, skipping\n",
      "[INFO]\tGene HAS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2F has 0 variants, skipping\n",
      "[INFO]\tGene C1orf87 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF766 has 0 variants, skipping\n",
      "[INFO]\tGene WFIKKN1 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPC has 0 variants, skipping\n",
      "[INFO]\tGene OR6T1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01231 has 0 variants, skipping\n",
      "[INFO]\tGene RFX3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZGLP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC37L1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MMP20 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM82 has 0 variants, skipping\n",
      "[INFO]\tGene PRR25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC184 has 0 variants, skipping\n",
      "[INFO]\tGene CKS2 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD3 has 0 variants, skipping\n",
      "[INFO]\tGene GSTO1 has 0 variants, skipping\n",
      "[INFO]\tGene MED26 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC8 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4L1 has 0 variants, skipping\n",
      "[INFO]\tGene CDNF has 0 variants, skipping\n",
      "[INFO]\tGene OVCH1 has 0 variants, skipping\n",
      "[INFO]\tGene MRFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375972 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD137 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01235 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00583 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD13 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4A22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389705 has 0 variants, skipping\n",
      "[INFO]\tGene REP15 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE40 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRHL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3152 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928120 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL20 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4473 has 0 variants, skipping\n",
      "[INFO]\tGene FOCAD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA30B has 0 variants, skipping\n",
      "[INFO]\tGene IFNA22P has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM16 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2A-DT has 0 variants, skipping\n",
      "[INFO]\tGene MN1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01239 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929563 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DMB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01241 has 0 variants, skipping\n",
      "[INFO]\tGene IFT74-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERZ has 0 variants, skipping\n",
      "[INFO]\tGene LINC00032 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01242 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01243 has 0 variants, skipping\n",
      "[INFO]\tGene SYN1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA13 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC2 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSPL2 has 0 variants, skipping\n",
      "[INFO]\tGene UFC1 has 0 variants, skipping\n",
      "[INFO]\tGene USH2A has 0 variants, skipping\n",
      "[INFO]\tGene TENT5A has 0 variants, skipping\n",
      "[INFO]\tGene DDX60 has 0 variants, skipping\n",
      "[INFO]\tGene AGO4 has 0 variants, skipping\n",
      "[INFO]\tGene TET2 has 0 variants, skipping\n",
      "[INFO]\tGene ITLN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM3 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS6 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH4 has 0 variants, skipping\n",
      "[INFO]\tGene RNPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6851 has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01251 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB8 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB125 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD121B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD121A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC83 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGB has 0 variants, skipping\n",
      "[INFO]\tGene CNTFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205BP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01118 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB5-DT has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B5P has 0 variants, skipping\n",
      "[INFO]\tGene GPR26 has 0 variants, skipping\n",
      "[INFO]\tGene VRK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4667 has 0 variants, skipping\n",
      "[INFO]\tGene NOG has 0 variants, skipping\n",
      "[INFO]\tGene BMP15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926948 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6853 has 0 variants, skipping\n",
      "[INFO]\tGene GBX1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX5 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC2 has 0 variants, skipping\n",
      "[INFO]\tGene P3H4 has 0 variants, skipping\n",
      "[INFO]\tGene MXD4 has 0 variants, skipping\n",
      "[INFO]\tGene TBL3 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10C has 0 variants, skipping\n",
      "[INFO]\tGene WRAP73 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2Z1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W5 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D6 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4475 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00477 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4540 has 0 variants, skipping\n",
      "[INFO]\tGene EBLN3P has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD3 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM95C has 0 variants, skipping\n",
      "[INFO]\tGene CLN8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF658B has 0 variants, skipping\n",
      "[INFO]\tGene GLIDR has 0 variants, skipping\n",
      "[INFO]\tGene FGF7P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC554249 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA9B has 0 variants, skipping\n",
      "[INFO]\tGene FAM95B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379252 has 0 variants, skipping\n",
      "[INFO]\tGene AQP7P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929583 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724580 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1HP has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440896 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2C has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984035 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM242F has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103908605 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927827 has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_007697 has 0 variants, skipping\n",
      "[INFO]\tGene CD164L2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379807 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723709 has 0 variants, skipping\n",
      "[INFO]\tGene FGF7P6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103908604 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132249 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER4P2-CDK2AP2P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996349 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728673 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928381 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928195 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286297 has 0 variants, skipping\n",
      "[INFO]\tGene AQP7P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00537 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1JP has 0 variants, skipping\n",
      "[INFO]\tGene FLJ43315 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00882 has 0 variants, skipping\n",
      "[INFO]\tGene TCL6 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB4 has 0 variants, skipping\n",
      "[INFO]\tGene XYLT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642929 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379450 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3P2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27B has 0 variants, skipping\n",
      "[INFO]\tGene MKRN3 has 0 variants, skipping\n",
      "[INFO]\tGene NELFA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01506 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927069 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf135-DT has 0 variants, skipping\n",
      "[INFO]\tGene SMC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP46 has 0 variants, skipping\n",
      "[INFO]\tGene PON1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF235 has 0 variants, skipping\n",
      "[INFO]\tGene H4C7 has 0 variants, skipping\n",
      "[INFO]\tGene H4C13 has 0 variants, skipping\n",
      "[INFO]\tGene H4C5 has 0 variants, skipping\n",
      "[INFO]\tGene H4C2 has 0 variants, skipping\n",
      "[INFO]\tGene H4C8 has 0 variants, skipping\n",
      "[INFO]\tGene H4C3 has 0 variants, skipping\n",
      "[INFO]\tGene H4C12 has 0 variants, skipping\n",
      "[INFO]\tGene H4C6 has 0 variants, skipping\n",
      "[INFO]\tGene H4C4 has 0 variants, skipping\n",
      "[INFO]\tGene H4C1 has 0 variants, skipping\n",
      "[INFO]\tGene H3C2 has 0 variants, skipping\n",
      "[INFO]\tGene H3C10 has 0 variants, skipping\n",
      "[INFO]\tGene H3C12 has 0 variants, skipping\n",
      "[INFO]\tGene H3C8 has 0 variants, skipping\n",
      "[INFO]\tGene H3C11 has 0 variants, skipping\n",
      "[INFO]\tGene H3C6 has 0 variants, skipping\n",
      "[INFO]\tGene H3C3 has 0 variants, skipping\n",
      "[INFO]\tGene FARSA has 0 variants, skipping\n",
      "[INFO]\tGene FAP has 0 variants, skipping\n",
      "[INFO]\tGene SOCS6 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR2 has 0 variants, skipping\n",
      "[INFO]\tGene MED14 has 0 variants, skipping\n",
      "[INFO]\tGene STK17B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G12B has 0 variants, skipping\n",
      "[INFO]\tGene H3C1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC21 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC17 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC4 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC10 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC9 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC6 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC7 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC14 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC15 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC13 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC8 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC20 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC18 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC17 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC4 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC6 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC16 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC15 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC13 has 0 variants, skipping\n",
      "[INFO]\tGene SNN has 0 variants, skipping\n",
      "[INFO]\tGene FZD9 has 0 variants, skipping\n",
      "[INFO]\tGene OBSCN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01504 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01474 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927358 has 0 variants, skipping\n",
      "[INFO]\tGene RORB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00668 has 0 variants, skipping\n",
      "[INFO]\tGene NFAM1 has 0 variants, skipping\n",
      "[INFO]\tGene XKR3 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L1 has 0 variants, skipping\n",
      "[INFO]\tGene CARNMT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP9 has 0 variants, skipping\n",
      "[INFO]\tGene GABRP has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTO1 has 0 variants, skipping\n",
      "[INFO]\tGene ALB has 0 variants, skipping\n",
      "[INFO]\tGene NR0B1 has 0 variants, skipping\n",
      "[INFO]\tGene GNA14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS36 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927450 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE13 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6B has 0 variants, skipping\n",
      "[INFO]\tGene CPXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene F8A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP25-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01507 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927502 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D5P has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376114 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927575 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388282 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389765 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927623 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS1RR has 0 variants, skipping\n",
      "[INFO]\tGene LOC440173 has 0 variants, skipping\n",
      "[INFO]\tGene AREL1 has 0 variants, skipping\n",
      "[INFO]\tGene MBTD1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSL3P has 0 variants, skipping\n",
      "[INFO]\tGene CTSLP8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC392364 has 0 variants, skipping\n",
      "[INFO]\tGene LOC497256 has 0 variants, skipping\n",
      "[INFO]\tGene GLDN has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724156 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4289 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3153 has 0 variants, skipping\n",
      "[INFO]\tGene ARSH has 0 variants, skipping\n",
      "[INFO]\tGene ZNF57 has 0 variants, skipping\n",
      "[INFO]\tGene UNQ6494 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927847 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4290HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4290 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01501 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK5 has 0 variants, skipping\n",
      "[INFO]\tGene TEX22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129316 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3910-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3910-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128076 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00475 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3651 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA84 has 0 variants, skipping\n",
      "[INFO]\tGene ELSPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4670 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128361 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929748 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD19P has 0 variants, skipping\n",
      "[INFO]\tGene LOC642943 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927954 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00702 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4291 has 0 variants, skipping\n",
      "[INFO]\tGene CHSY1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02603 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132077 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF740 has 0 variants, skipping\n",
      "[INFO]\tGene FYB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2278 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6081 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3074 has 0 variants, skipping\n",
      "[INFO]\tGene AS3MT has 0 variants, skipping\n",
      "[INFO]\tGene SERAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00092 has 0 variants, skipping\n",
      "[INFO]\tGene LOC158435 has 0 variants, skipping\n",
      "[INFO]\tGene LOC158434 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO10 has 0 variants, skipping\n",
      "[INFO]\tGene MMD has 0 variants, skipping\n",
      "[INFO]\tGene THAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441455 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132781 has 0 variants, skipping\n",
      "[INFO]\tGene PTMAP11 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18CP has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P4-STRA6LP has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P4-STRA6LP-CCDC180 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286359 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505774 has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CMC4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6854 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1A has 0 variants, skipping\n",
      "[INFO]\tGene NAMA has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928438 has 0 variants, skipping\n",
      "[INFO]\tGene STX17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR37L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM246-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00587 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01492 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928523 has 0 variants, skipping\n",
      "[INFO]\tGene SMC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376194 has 0 variants, skipping\n",
      "[INFO]\tGene PPY has 0 variants, skipping\n",
      "[INFO]\tGene KRT86 has 0 variants, skipping\n",
      "[INFO]\tGene OR4E1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8081 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01505 has 0 variants, skipping\n",
      "[INFO]\tGene LOC340512 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01509 has 0 variants, skipping\n",
      "[INFO]\tGene NOP9 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4D has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3D has 0 variants, skipping\n",
      "[INFO]\tGene ABCA6 has 0 variants, skipping\n",
      "[INFO]\tGene UBALD1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP3 has 0 variants, skipping\n",
      "[INFO]\tGene PGK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3927 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7702 has 0 variants, skipping\n",
      "[INFO]\tGene COPS5 has 0 variants, skipping\n",
      "[INFO]\tGene GCN1 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD9 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A5P has 0 variants, skipping\n",
      "[INFO]\tGene TRIM56 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4668 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS58 has 0 variants, skipping\n",
      "[INFO]\tGene RLN3 has 0 variants, skipping\n",
      "[INFO]\tGene IL22 has 0 variants, skipping\n",
      "[INFO]\tGene MIR454 has 0 variants, skipping\n",
      "[INFO]\tGene RAB9A has 0 variants, skipping\n",
      "[INFO]\tGene FAM225B has 0 variants, skipping\n",
      "[INFO]\tGene RAB9B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP298-TCP10L has 0 variants, skipping\n",
      "[INFO]\tGene RBMS2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMG2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMG1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928748 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928775 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00474 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01613 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC12 has 0 variants, skipping\n",
      "[INFO]\tGene TTC24 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T34 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf82 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4500 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4481 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4480 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4476 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4470 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4467 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA1 has 0 variants, skipping\n",
      "[INFO]\tGene FNTB has 0 variants, skipping\n",
      "[INFO]\tGene C14orf28 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIG has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene ARG2 has 0 variants, skipping\n",
      "[INFO]\tGene GSN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L3 has 0 variants, skipping\n",
      "[INFO]\tGene H3P6 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY2FP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4478 has 0 variants, skipping\n",
      "[INFO]\tGene OR1C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1A2 has 0 variants, skipping\n",
      "[INFO]\tGene NUFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD90 has 0 variants, skipping\n",
      "[INFO]\tGene RCN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR600HG has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7150 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505588 has 0 variants, skipping\n",
      "[INFO]\tGene LOC613206 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129034 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A2HG has 0 variants, skipping\n",
      "[INFO]\tGene AHCTF1 has 0 variants, skipping\n",
      "[INFO]\tGene NOXRED1 has 0 variants, skipping\n",
      "[INFO]\tGene AMOT has 0 variants, skipping\n",
      "[INFO]\tGene XKRX has 0 variants, skipping\n",
      "[INFO]\tGene TMEM215 has 0 variants, skipping\n",
      "[INFO]\tGene CDC20 has 0 variants, skipping\n",
      "[INFO]\tGene CDC6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376271 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK3 has 0 variants, skipping\n",
      "[INFO]\tGene KIF16B has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L2-PABPN1 has 0 variants, skipping\n",
      "[INFO]\tGene COA5 has 0 variants, skipping\n",
      "[INFO]\tGene CISD2 has 0 variants, skipping\n",
      "[INFO]\tGene CDC23 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC51145 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1183 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929116 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA65 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB15 has 0 variants, skipping\n",
      "[INFO]\tGene ZADH2 has 0 variants, skipping\n",
      "[INFO]\tGene EVC has 0 variants, skipping\n",
      "[INFO]\tGene GAST has 0 variants, skipping\n",
      "[INFO]\tGene DRD1 has 0 variants, skipping\n",
      "[INFO]\tGene FOLR2 has 0 variants, skipping\n",
      "[INFO]\tGene NOL7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3911 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49B has 0 variants, skipping\n",
      "[INFO]\tGene SMTNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3960 has 0 variants, skipping\n",
      "[INFO]\tGene ALG12 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC126 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723566 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4672 has 0 variants, skipping\n",
      "[INFO]\tGene RASL11B has 0 variants, skipping\n",
      "[INFO]\tGene CUEDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHF1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24-AS1-EDEM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3154 has 0 variants, skipping\n",
      "[INFO]\tGene NTSR2 has 0 variants, skipping\n",
      "[INFO]\tGene P3R3URF has 0 variants, skipping\n",
      "[INFO]\tGene EPX has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MMUT has 0 variants, skipping\n",
      "[INFO]\tGene MPO has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A14 has 0 variants, skipping\n",
      "[INFO]\tGene TTC30B has 0 variants, skipping\n",
      "[INFO]\tGene MEI1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF347 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506100 has 0 variants, skipping\n",
      "[INFO]\tGene DLG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00665 has 0 variants, skipping\n",
      "[INFO]\tGene TTN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf57 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00466 has 0 variants, skipping\n",
      "[INFO]\tGene DDRGK1 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC2 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF2 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929331 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01503 has 0 variants, skipping\n",
      "[INFO]\tGene NPR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR603 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PRRX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C9 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C19 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2B6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6855 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100272217 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6856 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3662 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3606 has 0 variants, skipping\n",
      "[INFO]\tGene KIFC1 has 0 variants, skipping\n",
      "[INFO]\tGene BCL9 has 0 variants, skipping\n",
      "[INFO]\tGene COPZ2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA2 has 0 variants, skipping\n",
      "[INFO]\tGene GTSE1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD62A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD62B has 0 variants, skipping\n",
      "[INFO]\tGene LSM6 has 0 variants, skipping\n",
      "[INFO]\tGene RTL9 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D has 0 variants, skipping\n",
      "[INFO]\tGene ADGB has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AW has 0 variants, skipping\n",
      "[INFO]\tGene WNT10A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC68 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6877 has 0 variants, skipping\n",
      "[INFO]\tGene CELP has 0 variants, skipping\n",
      "[INFO]\tGene RTCB has 0 variants, skipping\n",
      "[INFO]\tGene SNORD24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36C has 0 variants, skipping\n",
      "[INFO]\tGene TLX2 has 0 variants, skipping\n",
      "[INFO]\tGene DBH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130548 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6ATAC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02247 has 0 variants, skipping\n",
      "[INFO]\tGene COL5A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101448202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689E has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689F has 0 variants, skipping\n",
      "[INFO]\tGene LOC401557 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928525 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01502 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A6 has 0 variants, skipping\n",
      "[INFO]\tGene OR1S2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1N2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1M1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP434A062 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf68 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3 has 0 variants, skipping\n",
      "[INFO]\tGene RIC8A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4674 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01451 has 0 variants, skipping\n",
      "[INFO]\tGene HSPC324 has 0 variants, skipping\n",
      "[INFO]\tGene BRI3BP has 0 variants, skipping\n",
      "[INFO]\tGene MBTPS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA17B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA17A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128593 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6722 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ9 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC183-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4292 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF613 has 0 variants, skipping\n",
      "[INFO]\tGene CREG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4479 has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB8B has 0 variants, skipping\n",
      "[INFO]\tGene MAN1B1-DT has 0 variants, skipping\n",
      "[INFO]\tGene NOMO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3621 has 0 variants, skipping\n",
      "[INFO]\tGene SNTB2 has 0 variants, skipping\n",
      "[INFO]\tGene STPG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GCSIR has 0 variants, skipping\n",
      "[INFO]\tGene LDC1P has 0 variants, skipping\n",
      "[INFO]\tGene MIR7114 has 0 variants, skipping\n",
      "[INFO]\tGene MEG8 has 0 variants, skipping\n",
      "[INFO]\tGene PRR27 has 0 variants, skipping\n",
      "[INFO]\tGene KRT39 has 0 variants, skipping\n",
      "[INFO]\tGene RNF166 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC11 has 0 variants, skipping\n",
      "[INFO]\tGene DSPP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM258 has 0 variants, skipping\n",
      "[INFO]\tGene LOC651337 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133077 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1C2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376331 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928786 has 0 variants, skipping\n",
      "[INFO]\tGene TUBBP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928932 has 0 variants, skipping\n",
      "[INFO]\tGene FAM157B has 0 variants, skipping\n",
      "[INFO]\tGene DDX28 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100233156 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389834 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283788 has 0 variants, skipping\n",
      "[INFO]\tGene MIAT has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A8P has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R15B has 0 variants, skipping\n",
      "[INFO]\tGene DIRC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505874 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT6 has 0 variants, skipping\n",
      "[INFO]\tGene XGY2 has 0 variants, skipping\n",
      "[INFO]\tGene TYRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01546 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA48B has 0 variants, skipping\n",
      "[INFO]\tGene PRKX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389906 has 0 variants, skipping\n",
      "[INFO]\tGene FAM239B has 0 variants, skipping\n",
      "[INFO]\tGene FAM239A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928201 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373156 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4770 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR651 has 0 variants, skipping\n",
      "[INFO]\tGene NUP160 has 0 variants, skipping\n",
      "[INFO]\tGene CTXND1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM31 has 0 variants, skipping\n",
      "[INFO]\tGene TPRXL has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AX has 0 variants, skipping\n",
      "[INFO]\tGene TLR8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02154 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-600G8.3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01203 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6086 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK2 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF2 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD3 has 0 variants, skipping\n",
      "[INFO]\tGene NUCKS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD6 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E4P has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8B has 0 variants, skipping\n",
      "[INFO]\tGene SH2D4B has 0 variants, skipping\n",
      "[INFO]\tGene RRAS has 0 variants, skipping\n",
      "[INFO]\tGene RANBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PIR-FIGF has 0 variants, skipping\n",
      "[INFO]\tGene GS1-594A7.3 has 0 variants, skipping\n",
      "[INFO]\tGene INE2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AM has 0 variants, skipping\n",
      "[INFO]\tGene TMEM151A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4768 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928389 has 0 variants, skipping\n",
      "[INFO]\tGene NHS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D14 has 0 variants, skipping\n",
      "[INFO]\tGene OR14I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14K1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01456 has 0 variants, skipping\n",
      "[INFO]\tGene PPEF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1G has 0 variants, skipping\n",
      "[INFO]\tGene PHKA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CACTIN has 0 variants, skipping\n",
      "[INFO]\tGene LOC729609 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA9L has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT9 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25A has 0 variants, skipping\n",
      "[INFO]\tGene PHEX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD1-AS has 0 variants, skipping\n",
      "[INFO]\tGene ZFX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAG1 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA23 has 0 variants, skipping\n",
      "[INFO]\tGene VENTXP1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4666B has 0 variants, skipping\n",
      "[INFO]\tGene CBWD6 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB5 has 0 variants, skipping\n",
      "[INFO]\tGene PRKACG has 0 variants, skipping\n",
      "[INFO]\tGene TAB3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729930 has 0 variants, skipping\n",
      "[INFO]\tGene RAB32 has 0 variants, skipping\n",
      "[INFO]\tGene COPS6 has 0 variants, skipping\n",
      "[INFO]\tGene UQCR11 has 0 variants, skipping\n",
      "[INFO]\tGene ADIRF has 0 variants, skipping\n",
      "[INFO]\tGene FRG2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3915 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928627 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AJ2 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR15B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-2 has 0 variants, skipping\n",
      "[INFO]\tGene FHAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MID1IP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01281 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01282 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3937 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1587 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02601 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2110 has 0 variants, skipping\n",
      "[INFO]\tGene FAM102A has 0 variants, skipping\n",
      "[INFO]\tGene PINCR has 0 variants, skipping\n",
      "[INFO]\tGene MIR643 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01204 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02595 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01186 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF674-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01545 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP7 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A3 has 0 variants, skipping\n",
      "[INFO]\tGene SETD3 has 0 variants, skipping\n",
      "[INFO]\tGene BATF has 0 variants, skipping\n",
      "[INFO]\tGene AGR2 has 0 variants, skipping\n",
      "[INFO]\tGene INE1 has 0 variants, skipping\n",
      "[INFO]\tGene PYROXD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01560 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4769 has 0 variants, skipping\n",
      "[INFO]\tGene UXT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF630-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX6P has 0 variants, skipping\n",
      "[INFO]\tGene VPS4A has 0 variants, skipping\n",
      "[INFO]\tGene SSX9P has 0 variants, skipping\n",
      "[INFO]\tGene GSTA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927635 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3H has 0 variants, skipping\n",
      "[INFO]\tGene EIF3G has 0 variants, skipping\n",
      "[INFO]\tGene EIF3F has 0 variants, skipping\n",
      "[INFO]\tGene ERRFI1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC86 has 0 variants, skipping\n",
      "[INFO]\tGene MIR380 has 0 variants, skipping\n",
      "[INFO]\tGene SYP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FLICR has 0 variants, skipping\n",
      "[INFO]\tGene FAM236C has 0 variants, skipping\n",
      "[INFO]\tGene HTD2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B1 has 0 variants, skipping\n",
      "[INFO]\tGene USP27X-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBILA has 0 variants, skipping\n",
      "[INFO]\tGene MIR500B has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL5 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM31 has 0 variants, skipping\n",
      "[INFO]\tGene TCTE1 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01496 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2E2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2C2AP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11E has 0 variants, skipping\n",
      "[INFO]\tGene MIR8088 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE1B has 0 variants, skipping\n",
      "[INFO]\tGene SSX8P has 0 variants, skipping\n",
      "[INFO]\tGene SDSL has 0 variants, skipping\n",
      "[INFO]\tGene KANTR has 0 variants, skipping\n",
      "[INFO]\tGene PAQR8 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6895 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6894 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6857 has 0 variants, skipping\n",
      "[INFO]\tGene DBP has 0 variants, skipping\n",
      "[INFO]\tGene MORC4 has 0 variants, skipping\n",
      "[INFO]\tGene CDX4 has 0 variants, skipping\n",
      "[INFO]\tGene RAD21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11G has 0 variants, skipping\n",
      "[INFO]\tGene METTL3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC38 has 0 variants, skipping\n",
      "[INFO]\tGene SHE has 0 variants, skipping\n",
      "[INFO]\tGene SNORA109 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100421746 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4536-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4536-2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR5 has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01278 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF9-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene FUBP3 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD8P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00269 has 0 variants, skipping\n",
      "[INFO]\tGene ARL5A has 0 variants, skipping\n",
      "[INFO]\tGene RFK has 0 variants, skipping\n",
      "[INFO]\tGene DLG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3E has 0 variants, skipping\n",
      "[INFO]\tGene BEST4 has 0 variants, skipping\n",
      "[INFO]\tGene SNX33 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf96 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL34 has 0 variants, skipping\n",
      "[INFO]\tGene ADH5 has 0 variants, skipping\n",
      "[INFO]\tGene ADH4 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1A has 0 variants, skipping\n",
      "[INFO]\tGene FAM161B has 0 variants, skipping\n",
      "[INFO]\tGene AKAP4 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF12 has 0 variants, skipping\n",
      "[INFO]\tGene CFH has 0 variants, skipping\n",
      "[INFO]\tGene UFSP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B6 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN4 has 0 variants, skipping\n",
      "[INFO]\tGene INGX has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129291 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132741 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00891 has 0 variants, skipping\n",
      "[INFO]\tGene RPS26P11 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR has 0 variants, skipping\n",
      "[INFO]\tGene PHKA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM226B has 0 variants, skipping\n",
      "[INFO]\tGene FAM226A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX7 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K4P1 has 0 variants, skipping\n",
      "[INFO]\tGene UBR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR374C has 0 variants, skipping\n",
      "[INFO]\tGene APOBR has 0 variants, skipping\n",
      "[INFO]\tGene TTC3P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR325HG has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6A has 0 variants, skipping\n",
      "[INFO]\tGene LSM10 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4328 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1B2P has 0 variants, skipping\n",
      "[INFO]\tGene ANKLE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZZEF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2DNL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928128 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1321 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG2 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548M has 0 variants, skipping\n",
      "[INFO]\tGene BRDTP1 has 0 variants, skipping\n",
      "[INFO]\tGene MSTN has 0 variants, skipping\n",
      "[INFO]\tGene GCHFR has 0 variants, skipping\n",
      "[INFO]\tGene GATA6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G1B has 0 variants, skipping\n",
      "[INFO]\tGene RPS25 has 0 variants, skipping\n",
      "[INFO]\tGene DNTTIP2 has 0 variants, skipping\n",
      "[INFO]\tGene UBTD1 has 0 variants, skipping\n",
      "[INFO]\tGene VCPIP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAA25 has 0 variants, skipping\n",
      "[INFO]\tGene WWC2 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2L2 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD19 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM233 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36A-HNRNPH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A3 has 0 variants, skipping\n",
      "[INFO]\tGene HBEGF has 0 variants, skipping\n",
      "[INFO]\tGene NXF5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A1 has 0 variants, skipping\n",
      "[INFO]\tGene NXF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00630 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02589 has 0 variants, skipping\n",
      "[INFO]\tGene GPIHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B6 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB11 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM162A has 0 variants, skipping\n",
      "[INFO]\tGene LOC286437 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA3 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AN has 0 variants, skipping\n",
      "[INFO]\tGene NT5M has 0 variants, skipping\n",
      "[INFO]\tGene ZNF620 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH10 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928335 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1B has 0 variants, skipping\n",
      "[INFO]\tGene FAU has 0 variants, skipping\n",
      "[INFO]\tGene GTF3A has 0 variants, skipping\n",
      "[INFO]\tGene GTF2F1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AX has 0 variants, skipping\n",
      "[INFO]\tGene LINC02864 has 0 variants, skipping\n",
      "[INFO]\tGene BCAR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1303 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1226 has 0 variants, skipping\n",
      "[INFO]\tGene FAM180A has 0 variants, skipping\n",
      "[INFO]\tGene MIR764 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1283-1 has 0 variants, skipping\n",
      "[INFO]\tGene GJB7 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf34 has 0 variants, skipping\n",
      "[INFO]\tGene NRROS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928358 has 0 variants, skipping\n",
      "[INFO]\tGene PCNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD96B has 0 variants, skipping\n",
      "[INFO]\tGene TDGF1P3 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL3 has 0 variants, skipping\n",
      "[INFO]\tGene CYBB has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6A has 0 variants, skipping\n",
      "[INFO]\tGene MYSM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441155 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF154 has 0 variants, skipping\n",
      "[INFO]\tGene PSAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVRL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4329 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928437 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1264 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1911 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA35B has 0 variants, skipping\n",
      "[INFO]\tGene PLS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DANT2 has 0 variants, skipping\n",
      "[INFO]\tGene DANT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126447 has 0 variants, skipping\n",
      "[INFO]\tGene IL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01285 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928336 has 0 variants, skipping\n",
      "[INFO]\tGene GPC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA69 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01402 has 0 variants, skipping\n",
      "[INFO]\tGene NKAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene C3 has 0 variants, skipping\n",
      "[INFO]\tGene STMN2 has 0 variants, skipping\n",
      "[INFO]\tGene SKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM31 has 0 variants, skipping\n",
      "[INFO]\tGene IVL has 0 variants, skipping\n",
      "[INFO]\tGene KCNA10 has 0 variants, skipping\n",
      "[INFO]\tGene PRKX has 0 variants, skipping\n",
      "[INFO]\tGene DMWD has 0 variants, skipping\n",
      "[INFO]\tGene DEFB4A has 0 variants, skipping\n",
      "[INFO]\tGene SYCP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928402 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928495 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM74 has 0 variants, skipping\n",
      "[INFO]\tGene PRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBD2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373338 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01201 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM6 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HGFAC has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5R2 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFC has 0 variants, skipping\n",
      "[INFO]\tGene LINC00629 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN32 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK15 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL3 has 0 variants, skipping\n",
      "[INFO]\tGene F13B has 0 variants, skipping\n",
      "[INFO]\tGene GZMK has 0 variants, skipping\n",
      "[INFO]\tGene TBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370362 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02243 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191C has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506790 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF103-CHMP3 has 0 variants, skipping\n",
      "[INFO]\tGene SAGE2P has 0 variants, skipping\n",
      "[INFO]\tGene SSR2 has 0 variants, skipping\n",
      "[INFO]\tGene USP29 has 0 variants, skipping\n",
      "[INFO]\tGene RFTN1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO3 has 0 variants, skipping\n",
      "[INFO]\tGene PLIN5 has 0 variants, skipping\n",
      "[INFO]\tGene SRRD has 0 variants, skipping\n",
      "[INFO]\tGene LINC00892 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC8 has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD61 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF592 has 0 variants, skipping\n",
      "[INFO]\tGene S100A6 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD4 has 0 variants, skipping\n",
      "[INFO]\tGene BRINP1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGA has 0 variants, skipping\n",
      "[INFO]\tGene ATP11B has 0 variants, skipping\n",
      "[INFO]\tGene LOC728660 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00632 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR892C has 0 variants, skipping\n",
      "[INFO]\tGene MIR892A has 0 variants, skipping\n",
      "[INFO]\tGene MIR891A has 0 variants, skipping\n",
      "[INFO]\tGene PRDX2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517C has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB4X has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG3 has 0 variants, skipping\n",
      "[INFO]\tGene MST1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00893 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIA has 0 variants, skipping\n",
      "[INFO]\tGene LINC00850 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00894 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2114 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE8 has 0 variants, skipping\n",
      "[INFO]\tGene EPPIN has 0 variants, skipping\n",
      "[INFO]\tGene MIR4330 has 0 variants, skipping\n",
      "[INFO]\tGene GPR50-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377213 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA9 has 0 variants, skipping\n",
      "[INFO]\tGene IL17F has 0 variants, skipping\n",
      "[INFO]\tGene CSAG4 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf49 has 0 variants, skipping\n",
      "[INFO]\tGene TINAG has 0 variants, skipping\n",
      "[INFO]\tGene LSM3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM29 has 0 variants, skipping\n",
      "[INFO]\tGene PROB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373378 has 0 variants, skipping\n",
      "[INFO]\tGene USP10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373383 has 0 variants, skipping\n",
      "[INFO]\tGene LCA10 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4E has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3202-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3202-2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLL has 0 variants, skipping\n",
      "[INFO]\tGene OR2AG2 has 0 variants, skipping\n",
      "[INFO]\tGene BSX has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D2B has 0 variants, skipping\n",
      "[INFO]\tGene CH17-340M24.3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6858 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4AL1 has 0 variants, skipping\n",
      "[INFO]\tGene POTEI has 0 variants, skipping\n",
      "[INFO]\tGene FAM223A has 0 variants, skipping\n",
      "[INFO]\tGene FAM223B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354C has 0 variants, skipping\n",
      "[INFO]\tGene CXXC1 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA56 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5R1 has 0 variants, skipping\n",
      "[INFO]\tGene KAT2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-3 has 0 variants, skipping\n",
      "[INFO]\tGene TMLHE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927830 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129697 has 0 variants, skipping\n",
      "[INFO]\tGene CYRIA has 0 variants, skipping\n",
      "[INFO]\tGene TTC37 has 0 variants, skipping\n",
      "[INFO]\tGene MIR545 has 0 variants, skipping\n",
      "[INFO]\tGene MIR539 has 0 variants, skipping\n",
      "[INFO]\tGene ZFY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00278 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY23 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY23B has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00280 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY1 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY1B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY2 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY2B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY21 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY21B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY7 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY7B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY8 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY8B has 0 variants, skipping\n",
      "[INFO]\tGene PRKY has 0 variants, skipping\n",
      "[INFO]\tGene TTTY16 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00279 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY18 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY19 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY11 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY20 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y6 has 0 variants, skipping\n",
      "[INFO]\tGene UBR2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y3 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY22 has 0 variants, skipping\n",
      "[INFO]\tGene GYG2P1 has 0 variants, skipping\n",
      "[INFO]\tGene OMG has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THRB-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AU1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4X2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4X1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN12 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf32 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF33 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02882 has 0 variants, skipping\n",
      "[INFO]\tGene GIHCG has 0 variants, skipping\n",
      "[INFO]\tGene FBXO30-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00858 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00311 has 0 variants, skipping\n",
      "[INFO]\tGene PURPL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01426 has 0 variants, skipping\n",
      "[INFO]\tGene ACTE1P has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZEB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30A-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAD51-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00963 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4454 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4421 has 0 variants, skipping\n",
      "[INFO]\tGene DCST1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR219B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR638 has 0 variants, skipping\n",
      "[INFO]\tGene MIR608 has 0 variants, skipping\n",
      "[INFO]\tGene MIR607 has 0 variants, skipping\n",
      "[INFO]\tGene MIR599 has 0 variants, skipping\n",
      "[INFO]\tGene MIR597 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR592 has 0 variants, skipping\n",
      "[INFO]\tGene MIR18A has 0 variants, skipping\n",
      "[INFO]\tGene MIR770 has 0 variants, skipping\n",
      "[INFO]\tGene MIR541 has 0 variants, skipping\n",
      "[INFO]\tGene MIR888 has 0 variants, skipping\n",
      "[INFO]\tGene MIR198 has 0 variants, skipping\n",
      "[INFO]\tGene MIR877 has 0 variants, skipping\n",
      "[INFO]\tGene MIR127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR154 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4Y-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM41AY1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM41AY2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM224B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY9A has 0 variants, skipping\n",
      "[INFO]\tGene TTTY9B has 0 variants, skipping\n",
      "[INFO]\tGene OTULINL has 0 variants, skipping\n",
      "[INFO]\tGene BCORP1 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNGY has 0 variants, skipping\n",
      "[INFO]\tGene TTTY10 has 0 variants, skipping\n",
      "[INFO]\tGene SKI has 0 variants, skipping\n",
      "[INFO]\tGene TTTY13 has 0 variants, skipping\n",
      "[INFO]\tGene FBN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929148 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY6 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY6B has 0 variants, skipping\n",
      "[INFO]\tGene LORICRIN has 0 variants, skipping\n",
      "[INFO]\tGene TTTY5 has 0 variants, skipping\n",
      "[INFO]\tGene MAST1 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17A has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17C has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4C has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY3 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY3B has 0 variants, skipping\n",
      "[INFO]\tGene CSPG4P1Y has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P2Y has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P3Y has 0 variants, skipping\n",
      "[INFO]\tGene AARS2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P2-AGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P4-AGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene TTC28-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00205 has 0 variants, skipping\n",
      "[INFO]\tGene CASC2 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01554 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TARDBPP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02809 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLECL1 has 0 variants, skipping\n",
      "[INFO]\tGene MFFP3 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR86-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene PRNCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACP3 has 0 variants, skipping\n",
      "[INFO]\tGene NEU2 has 0 variants, skipping\n",
      "[INFO]\tGene NCL has 0 variants, skipping\n",
      "[INFO]\tGene CLDN34 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf81 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996720 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3B has 0 variants, skipping\n",
      "[INFO]\tGene RARRES2 has 0 variants, skipping\n",
      "[INFO]\tGene RASA1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD18 has 0 variants, skipping\n",
      "[INFO]\tGene NWD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986453 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK2 has 0 variants, skipping\n",
      "[INFO]\tGene REXO2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155B has 0 variants, skipping\n",
      "[INFO]\tGene AGAP10P has 0 variants, skipping\n",
      "[INFO]\tGene PARGP1-AGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPM has 0 variants, skipping\n",
      "[INFO]\tGene TMEM175 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA3 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K7 has 0 variants, skipping\n",
      "[INFO]\tGene GRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf38 has 0 variants, skipping\n",
      "[INFO]\tGene APELA has 0 variants, skipping\n",
      "[INFO]\tGene UPF1 has 0 variants, skipping\n",
      "[INFO]\tGene FALEC has 0 variants, skipping\n",
      "[INFO]\tGene PAGE2B has 0 variants, skipping\n",
      "[INFO]\tGene EFNA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369199 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371215 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF286B has 0 variants, skipping\n",
      "[INFO]\tGene SENP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CA5BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724020 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02883 has 0 variants, skipping\n",
      "[INFO]\tGene CASC19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927789 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984208 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369209 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS45P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS43P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS44P has 0 variants, skipping\n",
      "[INFO]\tGene BLACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR206 has 0 variants, skipping\n",
      "[INFO]\tGene MIR629 has 0 variants, skipping\n",
      "[INFO]\tGene SIRLNT has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P4 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf95 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927245 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379443 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724904 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996643 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927824 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723534 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369201 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339260 has 0 variants, skipping\n",
      "[INFO]\tGene FGD6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM214 has 0 variants, skipping\n",
      "[INFO]\tGene CALML6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64C has 0 variants, skipping\n",
      "[INFO]\tGene YOD1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA3 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724378 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378577 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984236 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984265 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371031 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376917 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996413 has 0 variants, skipping\n",
      "[INFO]\tGene POF1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374378 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378230 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377777 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985184 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985246 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984341 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984507 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985911 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986163 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986876 has 0 variants, skipping\n",
      "[INFO]\tGene ABTB1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP38-DT has 0 variants, skipping\n",
      "[INFO]\tGene C17orf102 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1915HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02688 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02783 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB44-DT has 0 variants, skipping\n",
      "[INFO]\tGene GTSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf46 has 0 variants, skipping\n",
      "[INFO]\tGene PDSS2 has 0 variants, skipping\n",
      "[INFO]\tGene NAT14 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3A has 0 variants, skipping\n",
      "[INFO]\tGene ARL8B has 0 variants, skipping\n",
      "[INFO]\tGene NPPC has 0 variants, skipping\n",
      "[INFO]\tGene ODF1 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP11 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP6 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP2B has 0 variants, skipping\n",
      "[INFO]\tGene SNHG31 has 0 variants, skipping\n",
      "[INFO]\tGene KDM3B has 0 variants, skipping\n",
      "[INFO]\tGene N4BP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2A-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDCD1LG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM99 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02693 has 0 variants, skipping\n",
      "[INFO]\tGene GAB2 has 0 variants, skipping\n",
      "[INFO]\tGene EEFSEC has 0 variants, skipping\n",
      "[INFO]\tGene ATG101 has 0 variants, skipping\n",
      "[INFO]\tGene MIIP has 0 variants, skipping\n",
      "[INFO]\tGene SPCS3 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD12 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD11 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD4 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf223 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112543491 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL46 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL4 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTA1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP25 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR935 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC32A1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P11 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8L2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF287 has 0 variants, skipping\n",
      "[INFO]\tGene TM6SF2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SAXO1 has 0 variants, skipping\n",
      "[INFO]\tGene GRID2IP has 0 variants, skipping\n",
      "[INFO]\tGene PFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ESPL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTDSS1 has 0 variants, skipping\n",
      "[INFO]\tGene STYXL2 has 0 variants, skipping\n",
      "[INFO]\tGene NOX3 has 0 variants, skipping\n",
      "[INFO]\tGene CD207 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G3 has 0 variants, skipping\n",
      "[INFO]\tGene G0S2 has 0 variants, skipping\n",
      "[INFO]\tGene BICRA has 0 variants, skipping\n",
      "[INFO]\tGene NOP53 has 0 variants, skipping\n",
      "[INFO]\tGene GPKOW has 0 variants, skipping\n",
      "[INFO]\tGene GPX7 has 0 variants, skipping\n",
      "[INFO]\tGene BRPF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF777 has 0 variants, skipping\n",
      "[INFO]\tGene DESI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PDX1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNRG has 0 variants, skipping\n",
      "[INFO]\tGene ZNF335 has 0 variants, skipping\n",
      "[INFO]\tGene PCIF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF277 has 0 variants, skipping\n",
      "[INFO]\tGene TNN has 0 variants, skipping\n",
      "[INFO]\tGene CHTF18 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15A has 0 variants, skipping\n",
      "[INFO]\tGene ZBED8 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB4 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA2 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90B1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12L2 has 0 variants, skipping\n",
      "[INFO]\tGene NHSL2 has 0 variants, skipping\n",
      "[INFO]\tGene NEXMIF has 0 variants, skipping\n",
      "[INFO]\tGene MILR1 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF674 has 0 variants, skipping\n",
      "[INFO]\tGene KATNIP has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1D has 0 variants, skipping\n",
      "[INFO]\tGene PRM1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR32 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA3 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY8 has 0 variants, skipping\n",
      "[INFO]\tGene FAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PYY has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC17 has 0 variants, skipping\n",
      "[INFO]\tGene PABPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP9 has 0 variants, skipping\n",
      "[INFO]\tGene KREMEN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS4Y2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL35 has 0 variants, skipping\n",
      "[INFO]\tGene KATNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene PDHA2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCE has 0 variants, skipping\n",
      "[INFO]\tGene PDCL has 0 variants, skipping\n",
      "[INFO]\tGene MLF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF319 has 0 variants, skipping\n",
      "[INFO]\tGene H1-9P has 0 variants, skipping\n",
      "[INFO]\tGene CDC14C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF646 has 0 variants, skipping\n",
      "[INFO]\tGene USP34 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1A has 0 variants, skipping\n",
      "[INFO]\tGene HOXD13 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA13 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB4 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDNEP1 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHIPL has 0 variants, skipping\n",
      "[INFO]\tGene THOC2 has 0 variants, skipping\n",
      "[INFO]\tGene ELL has 0 variants, skipping\n",
      "[INFO]\tGene CFHR5 has 0 variants, skipping\n",
      "[INFO]\tGene ACER1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC114841035 has 0 variants, skipping\n",
      "[INFO]\tGene HOGA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23A has 0 variants, skipping\n",
      "[INFO]\tGene INSYN1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAT2 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT61A has 0 variants, skipping\n",
      "[INFO]\tGene ZASP has 0 variants, skipping\n",
      "[INFO]\tGene FCN1 has 0 variants, skipping\n",
      "[INFO]\tGene TYK2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G5B has 0 variants, skipping\n",
      "[INFO]\tGene RBM8A has 0 variants, skipping\n",
      "[INFO]\tGene FBLN5 has 0 variants, skipping\n",
      "[INFO]\tGene CYLC2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP2 has 0 variants, skipping\n",
      "[INFO]\tGene TNS3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP3 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTRC has 0 variants, skipping\n",
      "[INFO]\tGene RIBC2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene HTATSF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGFRN has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD13 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP1L has 0 variants, skipping\n",
      "[INFO]\tGene CT62 has 0 variants, skipping\n",
      "[INFO]\tGene MGAM2 has 0 variants, skipping\n",
      "[INFO]\tGene OST4 has 0 variants, skipping\n",
      "[INFO]\tGene IL24 has 0 variants, skipping\n",
      "[INFO]\tGene MAST3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL2B has 0 variants, skipping\n",
      "[INFO]\tGene PTRHD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11L1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD24 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF721 has 0 variants, skipping\n",
      "[INFO]\tGene CITED4 has 0 variants, skipping\n",
      "[INFO]\tGene SGCD has 0 variants, skipping\n",
      "[INFO]\tGene SPRED2 has 0 variants, skipping\n",
      "[INFO]\tGene LEPROTL1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L6 has 0 variants, skipping\n",
      "[INFO]\tGene FCSK has 0 variants, skipping\n",
      "[INFO]\tGene RSL1D1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP2 has 0 variants, skipping\n",
      "[INFO]\tGene NOC2L has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB36 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40A has 0 variants, skipping\n",
      "[INFO]\tGene ITLN2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB5 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD4 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO41 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L3 has 0 variants, skipping\n",
      "[INFO]\tGene HEPHL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLF12 has 0 variants, skipping\n",
      "[INFO]\tGene CLVS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf62 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S5 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP16-1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCTAM34A has 0 variants, skipping\n",
      "[INFO]\tGene MESP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF555 has 0 variants, skipping\n",
      "[INFO]\tGene NACC2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20DC has 0 variants, skipping\n",
      "[INFO]\tGene MCM4 has 0 variants, skipping\n",
      "[INFO]\tGene HAND1 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP1L has 0 variants, skipping\n",
      "[INFO]\tGene TRPM5 has 0 variants, skipping\n",
      "[INFO]\tGene TFCP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC167 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985524 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNV2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A7 has 0 variants, skipping\n",
      "[INFO]\tGene DRGX has 0 variants, skipping\n",
      "[INFO]\tGene RNF227 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23C has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R11 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD4 has 0 variants, skipping\n",
      "[INFO]\tGene HLX has 0 variants, skipping\n",
      "[INFO]\tGene GYS2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK12 has 0 variants, skipping\n",
      "[INFO]\tGene GJA3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK13 has 0 variants, skipping\n",
      "[INFO]\tGene CGB2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC19 has 0 variants, skipping\n",
      "[INFO]\tGene MOCOS has 0 variants, skipping\n",
      "[INFO]\tGene OR5H6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C74 has 0 variants, skipping\n",
      "[INFO]\tGene REG4 has 0 variants, skipping\n",
      "[INFO]\tGene BPI has 0 variants, skipping\n",
      "[INFO]\tGene RPL17-C18orf32 has 0 variants, skipping\n",
      "[INFO]\tGene FIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG10 has 0 variants, skipping\n",
      "[INFO]\tGene NGB has 0 variants, skipping\n",
      "[INFO]\tGene PELI2 has 0 variants, skipping\n",
      "[INFO]\tGene APOA5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB106B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF77 has 0 variants, skipping\n",
      "[INFO]\tGene CENPT has 0 variants, skipping\n",
      "[INFO]\tGene NYNRIN has 0 variants, skipping\n",
      "[INFO]\tGene GPR157 has 0 variants, skipping\n",
      "[INFO]\tGene TOE1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL41 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D has 0 variants, skipping\n",
      "[INFO]\tGene TEN1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAI has 0 variants, skipping\n",
      "[INFO]\tGene TAMALIN has 0 variants, skipping\n",
      "[INFO]\tGene C10orf95 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G12A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13C has 0 variants, skipping\n",
      "[INFO]\tGene PDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC12 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA3 has 0 variants, skipping\n",
      "[INFO]\tGene RASL10B has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT9 has 0 variants, skipping\n",
      "[INFO]\tGene CASP4 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN3A has 0 variants, skipping\n",
      "[INFO]\tGene AGAP11 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH20 has 0 variants, skipping\n",
      "[INFO]\tGene CDCP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA1 has 0 variants, skipping\n",
      "[INFO]\tGene AIDA has 0 variants, skipping\n",
      "[INFO]\tGene KIF26A has 0 variants, skipping\n",
      "[INFO]\tGene OR56A5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2C has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM60 has 0 variants, skipping\n",
      "[INFO]\tGene MT4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT82 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM141 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE1 has 0 variants, skipping\n",
      "[INFO]\tGene IL9 has 0 variants, skipping\n",
      "[INFO]\tGene IL3 has 0 variants, skipping\n",
      "[INFO]\tGene C7 has 0 variants, skipping\n",
      "[INFO]\tGene IL2 has 0 variants, skipping\n",
      "[INFO]\tGene IL15 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM263 has 0 variants, skipping\n",
      "[INFO]\tGene CYHR1 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf107 has 0 variants, skipping\n",
      "[INFO]\tGene TTBK1 has 0 variants, skipping\n",
      "[INFO]\tGene NTNG2 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF594 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL7 has 0 variants, skipping\n",
      "[INFO]\tGene OR52Z1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2B has 0 variants, skipping\n",
      "[INFO]\tGene NSD3 has 0 variants, skipping\n",
      "[INFO]\tGene LRP11 has 0 variants, skipping\n",
      "[INFO]\tGene FAM222A has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS41 has 0 variants, skipping\n",
      "[INFO]\tGene IK has 0 variants, skipping\n",
      "[INFO]\tGene TSR3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987394 has 0 variants, skipping\n",
      "[INFO]\tGene RAB33B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HEAT2 has 0 variants, skipping\n",
      "[INFO]\tGene NIT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP6D1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110D has 0 variants, skipping\n",
      "[INFO]\tGene OR5B3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4B1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G3 has 0 variants, skipping\n",
      "[INFO]\tGene H3C15 has 0 variants, skipping\n",
      "[INFO]\tGene RNF141 has 0 variants, skipping\n",
      "[INFO]\tGene LRP1B has 0 variants, skipping\n",
      "[INFO]\tGene GSDMC has 0 variants, skipping\n",
      "[INFO]\tGene INA has 0 variants, skipping\n",
      "[INFO]\tGene C6orf99 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD1 has 0 variants, skipping\n",
      "[INFO]\tGene FMNL3 has 0 variants, skipping\n",
      "[INFO]\tGene CELA3A has 0 variants, skipping\n",
      "[INFO]\tGene NAMPT has 0 variants, skipping\n",
      "[INFO]\tGene INTS5 has 0 variants, skipping\n",
      "[INFO]\tGene TET1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX5 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM1 has 0 variants, skipping\n",
      "[INFO]\tGene HK2 has 0 variants, skipping\n",
      "[INFO]\tGene HTN3 has 0 variants, skipping\n",
      "[INFO]\tGene HGD has 0 variants, skipping\n",
      "[INFO]\tGene BCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB12 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC10 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL17 has 0 variants, skipping\n",
      "[INFO]\tGene RPS18 has 0 variants, skipping\n",
      "[INFO]\tGene SUDS3 has 0 variants, skipping\n",
      "[INFO]\tGene FSTL4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130449 has 0 variants, skipping\n",
      "[INFO]\tGene XKR6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02054 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2LY has 0 variants, skipping\n",
      "[INFO]\tGene LINC02860 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf44 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928093 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1C has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT6 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS7 has 0 variants, skipping\n",
      "[INFO]\tGene UBTFL1 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC2IP has 0 variants, skipping\n",
      "[INFO]\tGene DDX24 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL47 has 0 variants, skipping\n",
      "[INFO]\tGene COL27A1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB8 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF14 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEE2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC3 has 0 variants, skipping\n",
      "[INFO]\tGene MPLKIP has 0 variants, skipping\n",
      "[INFO]\tGene TMC1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14B has 0 variants, skipping\n",
      "[INFO]\tGene GRIN3B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13B has 0 variants, skipping\n",
      "[INFO]\tGene SH2D1B has 0 variants, skipping\n",
      "[INFO]\tGene FAM167A has 0 variants, skipping\n",
      "[INFO]\tGene TAAR8 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30A has 0 variants, skipping\n",
      "[INFO]\tGene RPRD1B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17C has 0 variants, skipping\n",
      "[INFO]\tGene CLDN22 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK3 has 0 variants, skipping\n",
      "[INFO]\tGene METTL14 has 0 variants, skipping\n",
      "[INFO]\tGene MED12L has 0 variants, skipping\n",
      "[INFO]\tGene EFHC2 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH2 has 0 variants, skipping\n",
      "[INFO]\tGene CD248 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A28 has 0 variants, skipping\n",
      "[INFO]\tGene QTRT1 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI2 has 0 variants, skipping\n",
      "[INFO]\tGene ARAP3 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF672 has 0 variants, skipping\n",
      "[INFO]\tGene THNSL1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4R has 0 variants, skipping\n",
      "[INFO]\tGene RTL10 has 0 variants, skipping\n",
      "[INFO]\tGene TMX1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC177 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K3 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC4 has 0 variants, skipping\n",
      "[INFO]\tGene MTLN has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14A has 0 variants, skipping\n",
      "[INFO]\tGene CCL15 has 0 variants, skipping\n",
      "[INFO]\tGene ACP4 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTB1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP4 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B2 has 0 variants, skipping\n",
      "[INFO]\tGene SUCNR1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC13 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT84 has 0 variants, skipping\n",
      "[INFO]\tGene PYURF has 0 variants, skipping\n",
      "[INFO]\tGene BSND has 0 variants, skipping\n",
      "[INFO]\tGene NAA15 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2J2 has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT2 has 0 variants, skipping\n",
      "[INFO]\tGene NMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATC1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-4 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6B has 0 variants, skipping\n",
      "[INFO]\tGene GNPTG has 0 variants, skipping\n",
      "[INFO]\tGene LYZL1 has 0 variants, skipping\n",
      "[INFO]\tGene BOK has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3A has 0 variants, skipping\n",
      "[INFO]\tGene RNF215 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15A has 0 variants, skipping\n",
      "[INFO]\tGene P4HB has 0 variants, skipping\n",
      "[INFO]\tGene RPS13 has 0 variants, skipping\n",
      "[INFO]\tGene RPS12 has 0 variants, skipping\n",
      "[INFO]\tGene OXT has 0 variants, skipping\n",
      "[INFO]\tGene RPS11 has 0 variants, skipping\n",
      "[INFO]\tGene RPS8 has 0 variants, skipping\n",
      "[INFO]\tGene OPRD1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOV has 0 variants, skipping\n",
      "[INFO]\tGene LNCRNA-ATB has 0 variants, skipping\n",
      "[INFO]\tGene OR56A3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C12 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M11 has 0 variants, skipping\n",
      "[INFO]\tGene OR9K2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N4 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F17 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF187 has 0 variants, skipping\n",
      "[INFO]\tGene LELP1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER1 has 0 variants, skipping\n",
      "[INFO]\tGene STYK1 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K15 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD3A has 0 variants, skipping\n",
      "[INFO]\tGene MFSD13A has 0 variants, skipping\n",
      "[INFO]\tGene RNF122 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC11 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL2 has 0 variants, skipping\n",
      "[INFO]\tGene NHEJ1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6C has 0 variants, skipping\n",
      "[INFO]\tGene CEP192 has 0 variants, skipping\n",
      "[INFO]\tGene LINP1 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf69 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388780 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339862 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC140 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf91 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507507 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf87 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132813 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401478 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375787 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPH has 0 variants, skipping\n",
      "[INFO]\tGene ACTR8 has 0 variants, skipping\n",
      "[INFO]\tGene KRI1 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP17 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf33 has 0 variants, skipping\n",
      "[INFO]\tGene ISOC1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB5 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN4 has 0 variants, skipping\n",
      "[INFO]\tGene TCIM has 0 variants, skipping\n",
      "[INFO]\tGene HOXB9 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R9 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB8 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR87 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA6 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR5A has 0 variants, skipping\n",
      "[INFO]\tGene CELA2A has 0 variants, skipping\n",
      "[INFO]\tGene TBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF628 has 0 variants, skipping\n",
      "[INFO]\tGene CABS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13A has 0 variants, skipping\n",
      "[INFO]\tGene ZYG11B has 0 variants, skipping\n",
      "[INFO]\tGene BBS1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX4 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf119 has 0 variants, skipping\n",
      "[INFO]\tGene GNB1L has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725021 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268173 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP15 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG7 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF486 has 0 variants, skipping\n",
      "[INFO]\tGene MAL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A13 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM3 has 0 variants, skipping\n",
      "[INFO]\tGene FMNL2 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK2 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN2 has 0 variants, skipping\n",
      "[INFO]\tGene ESCO1 has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG4B has 0 variants, skipping\n",
      "[INFO]\tGene XKR4 has 0 variants, skipping\n",
      "[INFO]\tGene MBD6 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL29 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-3 has 0 variants, skipping\n",
      "[INFO]\tGene PSG2 has 0 variants, skipping\n",
      "[INFO]\tGene NECAB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF93 has 0 variants, skipping\n",
      "[INFO]\tGene KIF18A has 0 variants, skipping\n",
      "[INFO]\tGene CABLES2 has 0 variants, skipping\n",
      "[INFO]\tGene NUAK2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BANCR has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS17 has 0 variants, skipping\n",
      "[INFO]\tGene DNER has 0 variants, skipping\n",
      "[INFO]\tGene PROKR1 has 0 variants, skipping\n",
      "[INFO]\tGene ESAM has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2LX has 0 variants, skipping\n",
      "[INFO]\tGene ARX has 0 variants, skipping\n",
      "[INFO]\tGene SMPD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268117 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268114 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268068 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267983 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB4 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB15 has 0 variants, skipping\n",
      "[INFO]\tGene RHOF has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB14 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB13 has 0 variants, skipping\n",
      "[INFO]\tGene CCL25 has 0 variants, skipping\n",
      "[INFO]\tGene MYO5C has 0 variants, skipping\n",
      "[INFO]\tGene ABCG5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLM2 has 0 variants, skipping\n",
      "[INFO]\tGene SBNO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP25 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT17 has 0 variants, skipping\n",
      "[INFO]\tGene CDH24 has 0 variants, skipping\n",
      "[INFO]\tGene SCP2D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3142 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2909 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01793 has 0 variants, skipping\n",
      "[INFO]\tGene TEDC2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA5 has 0 variants, skipping\n",
      "[INFO]\tGene PIGU has 0 variants, skipping\n",
      "[INFO]\tGene STAB2 has 0 variants, skipping\n",
      "[INFO]\tGene OPLAH has 0 variants, skipping\n",
      "[INFO]\tGene RNASEL has 0 variants, skipping\n",
      "[INFO]\tGene PPA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM117A has 0 variants, skipping\n",
      "[INFO]\tGene RAI2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02877 has 0 variants, skipping\n",
      "[INFO]\tGene DCXR-DT has 0 variants, skipping\n",
      "[INFO]\tGene COL25A1-DT has 0 variants, skipping\n",
      "[INFO]\tGene GAPLINC has 0 variants, skipping\n",
      "[INFO]\tGene THRIL has 0 variants, skipping\n",
      "[INFO]\tGene C5orf67 has 0 variants, skipping\n",
      "[INFO]\tGene GDF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH2OS has 0 variants, skipping\n",
      "[INFO]\tGene RAB1B has 0 variants, skipping\n",
      "[INFO]\tGene ARPC5L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-6 has 0 variants, skipping\n",
      "[INFO]\tGene IPPK has 0 variants, skipping\n",
      "[INFO]\tGene ATG4C has 0 variants, skipping\n",
      "[INFO]\tGene MEDAG has 0 variants, skipping\n",
      "[INFO]\tGene C8orf76 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12136 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12133 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11399 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1843 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3085 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9851 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10392 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10526 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9898 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3059 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9902-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9902-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9718 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12121 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10393 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12135 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11181 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9901 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12128 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11401 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BC has 0 variants, skipping\n",
      "[INFO]\tGene MIR10226 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S4 has 0 variants, skipping\n",
      "[INFO]\tGene MNS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPD1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR10394 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9986 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9899 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12122 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10396B has 0 variants, skipping\n",
      "[INFO]\tGene MIR10396A has 0 variants, skipping\n",
      "[INFO]\tGene MIR12114 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9900 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6529 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12115 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10397 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10522 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10523 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12118 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9983 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10398 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10524 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12131 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10525 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10399 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11400 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC71 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12123 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9903 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10400 has 0 variants, skipping\n",
      "[INFO]\tGene GHITM has 0 variants, skipping\n",
      "[INFO]\tGene UQCRQ has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12117 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12129 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9985 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12120 has 0 variants, skipping\n",
      "[INFO]\tGene DDIT4 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A7 has 0 variants, skipping\n",
      "[INFO]\tGene CA5BP1-CA5B has 0 variants, skipping\n",
      "[INFO]\tGene TEX12 has 0 variants, skipping\n",
      "[INFO]\tGene TEX13B has 0 variants, skipping\n",
      "[INFO]\tGene WDR55 has 0 variants, skipping\n",
      "[INFO]\tGene MYOPARR has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267871 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF2-H2BE1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144NL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268124 has 0 variants, skipping\n",
      "[INFO]\tGene PHAX has 0 variants, skipping\n",
      "[INFO]\tGene GPR158 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC8 has 0 variants, skipping\n",
      "[INFO]\tGene TREX2 has 0 variants, skipping\n",
      "[INFO]\tGene MC3R has 0 variants, skipping\n",
      "[INFO]\tGene CHST7 has 0 variants, skipping\n",
      "[INFO]\tGene GSK3A has 0 variants, skipping\n",
      "[INFO]\tGene TRPV1 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK6 has 0 variants, skipping\n",
      "[INFO]\tGene IFI27L2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR61 has 0 variants, skipping\n",
      "[INFO]\tGene HMCN1 has 0 variants, skipping\n",
      "[INFO]\tGene BRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTA2 has 0 variants, skipping\n",
      "[INFO]\tGene OTP has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6B has 0 variants, skipping\n",
      "[INFO]\tGene OR8D4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8J1 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD7 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXB1 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6G has 0 variants, skipping\n",
      "[INFO]\tGene MAK16 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H8 has 0 variants, skipping\n",
      "[INFO]\tGene NAA11 has 0 variants, skipping\n",
      "[INFO]\tGene KMT5C has 0 variants, skipping\n",
      "[INFO]\tGene IFNLR1 has 0 variants, skipping\n",
      "[INFO]\tGene MPV17L2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT10 has 0 variants, skipping\n",
      "[INFO]\tGene DHX37 has 0 variants, skipping\n",
      "[INFO]\tGene CNDP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM167B has 0 variants, skipping\n",
      "[INFO]\tGene EPN3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP23 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT4 has 0 variants, skipping\n",
      "[INFO]\tGene CAPNS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL35 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT6 has 0 variants, skipping\n",
      "[INFO]\tGene DHX32 has 0 variants, skipping\n",
      "[INFO]\tGene RETSAT has 0 variants, skipping\n",
      "[INFO]\tGene MRPL21 has 0 variants, skipping\n",
      "[INFO]\tGene XKR8 has 0 variants, skipping\n",
      "[INFO]\tGene PTCD3 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L5 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0408 has 0 variants, skipping\n",
      "[INFO]\tGene RAB21 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22B has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL2 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD5 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN1L has 0 variants, skipping\n",
      "[INFO]\tGene SAV1 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN2 has 0 variants, skipping\n",
      "[INFO]\tGene UCP1 has 0 variants, skipping\n",
      "[INFO]\tGene RMND5A has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL11 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L2 has 0 variants, skipping\n",
      "[INFO]\tGene AEN has 0 variants, skipping\n",
      "[INFO]\tGene CERK has 0 variants, skipping\n",
      "[INFO]\tGene ACE2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM30 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC12 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL8 has 0 variants, skipping\n",
      "[INFO]\tGene CHD6 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene MAML2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF528 has 0 variants, skipping\n",
      "[INFO]\tGene CLIP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXD has 0 variants, skipping\n",
      "[INFO]\tGene PROK1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene ECRG4 has 0 variants, skipping\n",
      "[INFO]\tGene PINK1 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK3 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB7 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K21 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS26 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC30 has 0 variants, skipping\n",
      "[INFO]\tGene NICN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A33 has 0 variants, skipping\n",
      "[INFO]\tGene COQ5 has 0 variants, skipping\n",
      "[INFO]\tGene NOA1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP10 has 0 variants, skipping\n",
      "[INFO]\tGene AASS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF527 has 0 variants, skipping\n",
      "[INFO]\tGene SPIRE2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf15 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf15 has 0 variants, skipping\n",
      "[INFO]\tGene A1BG has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN18 has 0 variants, skipping\n",
      "[INFO]\tGene RGS18 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP5 has 0 variants, skipping\n",
      "[INFO]\tGene S100Z has 0 variants, skipping\n",
      "[INFO]\tGene ZNF71-SMIM17 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PAGR1 has 0 variants, skipping\n",
      "[INFO]\tGene VILL has 0 variants, skipping\n",
      "[INFO]\tGene PRM3 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf75 has 0 variants, skipping\n",
      "[INFO]\tGene SLX4 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB133 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL17 has 0 variants, skipping\n",
      "[INFO]\tGene DAB2IP has 0 variants, skipping\n",
      "[INFO]\tGene KISS1R has 0 variants, skipping\n",
      "[INFO]\tGene USP32 has 0 variants, skipping\n",
      "[INFO]\tGene IL1F10 has 0 variants, skipping\n",
      "[INFO]\tGene MCEE has 0 variants, skipping\n",
      "[INFO]\tGene CCDC54 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM63 has 0 variants, skipping\n",
      "[INFO]\tGene CARD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30B has 0 variants, skipping\n",
      "[INFO]\tGene MIR27B has 0 variants, skipping\n",
      "[INFO]\tGene MIR23B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00102 has 0 variants, skipping\n",
      "[INFO]\tGene BACE1-AS has 0 variants, skipping\n",
      "[INFO]\tGene TAMALIN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE3D has 0 variants, skipping\n",
      "[INFO]\tGene RETNLB has 0 variants, skipping\n",
      "[INFO]\tGene OIP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCMF1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R37 has 0 variants, skipping\n",
      "[INFO]\tGene UGGT2 has 0 variants, skipping\n",
      "[INFO]\tGene PDF has 0 variants, skipping\n",
      "[INFO]\tGene RAB38 has 0 variants, skipping\n",
      "[INFO]\tGene EDAR has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLG has 0 variants, skipping\n",
      "[INFO]\tGene ANKRA2 has 0 variants, skipping\n",
      "[INFO]\tGene FRY has 0 variants, skipping\n",
      "[INFO]\tGene TPSB2 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK2B has 0 variants, skipping\n",
      "[INFO]\tGene BBS10 has 0 variants, skipping\n",
      "[INFO]\tGene TEFM has 0 variants, skipping\n",
      "[INFO]\tGene SLBP has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN6 has 0 variants, skipping\n",
      "[INFO]\tGene PLBD1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXC has 0 variants, skipping\n",
      "[INFO]\tGene EBF2 has 0 variants, skipping\n",
      "[INFO]\tGene MYCT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB1 has 0 variants, skipping\n",
      "[INFO]\tGene PGGHG has 0 variants, skipping\n",
      "[INFO]\tGene PTMA has 0 variants, skipping\n",
      "[INFO]\tGene NYAP2 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN3 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2I2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02872 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf55 has 0 variants, skipping\n",
      "[INFO]\tGene FAM218A has 0 variants, skipping\n",
      "[INFO]\tGene BIRC8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00540 has 0 variants, skipping\n",
      "[INFO]\tGene MIR186 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF329 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA7 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR5B has 0 variants, skipping\n",
      "[INFO]\tGene APBB1IP has 0 variants, skipping\n",
      "[INFO]\tGene TANGO6 has 0 variants, skipping\n",
      "[INFO]\tGene PITHD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM47 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL50 has 0 variants, skipping\n",
      "[INFO]\tGene USP53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF830 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD40 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD2 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK1B has 0 variants, skipping\n",
      "[INFO]\tGene VPS18 has 0 variants, skipping\n",
      "[INFO]\tGene CDKAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD63 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L5 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L4 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD39 has 0 variants, skipping\n",
      "[INFO]\tGene TLR8 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX1 has 0 variants, skipping\n",
      "[INFO]\tGene HKDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01720 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985946 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM14 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPM2AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPZ has 0 variants, skipping\n",
      "[INFO]\tGene MSRB2 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP6 has 0 variants, skipping\n",
      "[INFO]\tGene TPSD1 has 0 variants, skipping\n",
      "[INFO]\tGene MLYCD has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE8 has 0 variants, skipping\n",
      "[INFO]\tGene HAAO has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A15 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRB1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK1 has 0 variants, skipping\n",
      "[INFO]\tGene KISS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF3C has 0 variants, skipping\n",
      "[INFO]\tGene KDR has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC69 has 0 variants, skipping\n",
      "[INFO]\tGene LRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene HABP4 has 0 variants, skipping\n",
      "[INFO]\tGene SOX3 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN15 has 0 variants, skipping\n",
      "[INFO]\tGene SMO has 0 variants, skipping\n",
      "[INFO]\tGene SLCO2A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL8 has 0 variants, skipping\n",
      "[INFO]\tGene OR5I1 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene COG1 has 0 variants, skipping\n",
      "[INFO]\tGene SVOP has 0 variants, skipping\n",
      "[INFO]\tGene PIP4P2 has 0 variants, skipping\n",
      "[INFO]\tGene FEM1A has 0 variants, skipping\n",
      "[INFO]\tGene ELAC1 has 0 variants, skipping\n",
      "[INFO]\tGene DHTKD1 has 0 variants, skipping\n",
      "[INFO]\tGene CTTNBP2NL has 0 variants, skipping\n",
      "[INFO]\tGene ADARB2 has 0 variants, skipping\n",
      "[INFO]\tGene MYH7B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Z has 0 variants, skipping\n",
      "[INFO]\tGene TIGD3 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD4 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEH1 has 0 variants, skipping\n",
      "[INFO]\tGene RGCC has 0 variants, skipping\n",
      "[INFO]\tGene SOCS7 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN4 has 0 variants, skipping\n",
      "[INFO]\tGene SPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC150051 has 0 variants, skipping\n",
      "[INFO]\tGene CRABP1 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPCS2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL4 has 0 variants, skipping\n",
      "[INFO]\tGene FANCF has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1 has 0 variants, skipping\n",
      "[INFO]\tGene COIL has 0 variants, skipping\n",
      "[INFO]\tGene STK17A has 0 variants, skipping\n",
      "[INFO]\tGene PRKCA has 0 variants, skipping\n",
      "[INFO]\tGene WDR87BP has 0 variants, skipping\n",
      "[INFO]\tGene PARTICL has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TNF has 0 variants, skipping\n",
      "[INFO]\tGene C5orf47 has 0 variants, skipping\n",
      "[INFO]\tGene SLC11A1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1B has 0 variants, skipping\n",
      "[INFO]\tGene IL1A has 0 variants, skipping\n",
      "[INFO]\tGene PRRT1B has 0 variants, skipping\n",
      "[INFO]\tGene ICMT has 0 variants, skipping\n",
      "[INFO]\tGene PFDN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32D has 0 variants, skipping\n",
      "[INFO]\tGene PFAS has 0 variants, skipping\n",
      "[INFO]\tGene ANP32C has 0 variants, skipping\n",
      "[INFO]\tGene PREX1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPG has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB17 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF891 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF9B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT14 has 0 variants, skipping\n",
      "[INFO]\tGene TSTD3 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS56 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD61 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A-DT has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ18 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC25-GNG10 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA4 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA6 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD1 has 0 variants, skipping\n",
      "[INFO]\tGene NDN has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM6 has 0 variants, skipping\n",
      "[INFO]\tGene MYOG has 0 variants, skipping\n",
      "[INFO]\tGene MYOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP25 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG1 has 0 variants, skipping\n",
      "[INFO]\tGene SDC4 has 0 variants, skipping\n",
      "[INFO]\tGene SP2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX11 has 0 variants, skipping\n",
      "[INFO]\tGene SDC2 has 0 variants, skipping\n",
      "[INFO]\tGene SNTA1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX4 has 0 variants, skipping\n",
      "[INFO]\tGene SOX2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR12 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC5 has 0 variants, skipping\n",
      "[INFO]\tGene OTOR has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC7 has 0 variants, skipping\n",
      "[INFO]\tGene THAP10 has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLB has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA4L2 has 0 variants, skipping\n",
      "[INFO]\tGene BDH2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01615 has 0 variants, skipping\n",
      "[INFO]\tGene LUADT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505502 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF9 has 0 variants, skipping\n",
      "[INFO]\tGene SMC3 has 0 variants, skipping\n",
      "[INFO]\tGene PAPSS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHAF1B has 0 variants, skipping\n",
      "[INFO]\tGene RND2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC6 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF5 has 0 variants, skipping\n",
      "[INFO]\tGene MALL has 0 variants, skipping\n",
      "[INFO]\tGene YES1 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L10 has 0 variants, skipping\n",
      "[INFO]\tGene ASB15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSRB1 has 0 variants, skipping\n",
      "[INFO]\tGene AOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CCBE1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene CLC has 0 variants, skipping\n",
      "[INFO]\tGene IMPA2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD9 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene BMP2K has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H7B has 0 variants, skipping\n",
      "[INFO]\tGene WDR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIOX has 0 variants, skipping\n",
      "[INFO]\tGene TRIM44 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q1 has 0 variants, skipping\n",
      "[INFO]\tGene CELA3B has 0 variants, skipping\n",
      "[INFO]\tGene MMRN1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA1 has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF6 has 0 variants, skipping\n",
      "[INFO]\tGene OGFR has 0 variants, skipping\n",
      "[INFO]\tGene FTHL18 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf97 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAB1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAA2 has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS3 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf31 has 0 variants, skipping\n",
      "[INFO]\tGene PANX1 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf66 has 0 variants, skipping\n",
      "[INFO]\tGene OR9G9 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U9 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U8 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G6 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H12 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM203 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF13C has 0 variants, skipping\n",
      "[INFO]\tGene RBM33 has 0 variants, skipping\n",
      "[INFO]\tGene TENT5B has 0 variants, skipping\n",
      "[INFO]\tGene GBP4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB3 has 0 variants, skipping\n",
      "[INFO]\tGene DCLK3 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene BSN has 0 variants, skipping\n",
      "[INFO]\tGene SULT4A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF324 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF318 has 0 variants, skipping\n",
      "[INFO]\tGene FJX1 has 0 variants, skipping\n",
      "[INFO]\tGene EID1 has 0 variants, skipping\n",
      "[INFO]\tGene FRRS1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A11 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R15A has 0 variants, skipping\n",
      "[INFO]\tGene EDC4 has 0 variants, skipping\n",
      "[INFO]\tGene HEYL has 0 variants, skipping\n",
      "[INFO]\tGene FOXH1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KB2 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene BDH1 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL3 has 0 variants, skipping\n",
      "[INFO]\tGene B2M has 0 variants, skipping\n",
      "[INFO]\tGene FASN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF730 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11X2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC40 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR10 has 0 variants, skipping\n",
      "[INFO]\tGene PNRC2 has 0 variants, skipping\n",
      "[INFO]\tGene INO80D has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH5 has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8B has 0 variants, skipping\n",
      "[INFO]\tGene OR11H4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10Z1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10X1 has 0 variants, skipping\n",
      "[INFO]\tGene EMD has 0 variants, skipping\n",
      "[INFO]\tGene TOR1A has 0 variants, skipping\n",
      "[INFO]\tGene UBFD1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF21 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA7 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRA has 0 variants, skipping\n",
      "[INFO]\tGene SMG9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW5 has 0 variants, skipping\n",
      "[INFO]\tGene MYDGF has 0 variants, skipping\n",
      "[INFO]\tGene TMEM145 has 0 variants, skipping\n",
      "[INFO]\tGene CTAG1B has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2B has 0 variants, skipping\n",
      "[INFO]\tGene FUT7 has 0 variants, skipping\n",
      "[INFO]\tGene VWF has 0 variants, skipping\n",
      "[INFO]\tGene SCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene LRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LEAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL39L has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf29 has 0 variants, skipping\n",
      "[INFO]\tGene KCNB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A1 has 0 variants, skipping\n",
      "[INFO]\tGene PILRA has 0 variants, skipping\n",
      "[INFO]\tGene RAX has 0 variants, skipping\n",
      "[INFO]\tGene VSIR has 0 variants, skipping\n",
      "[INFO]\tGene ACRBP has 0 variants, skipping\n",
      "[INFO]\tGene CNFN has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT3 has 0 variants, skipping\n",
      "[INFO]\tGene MICALCL has 0 variants, skipping\n",
      "[INFO]\tGene TNS4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF38A has 0 variants, skipping\n",
      "[INFO]\tGene TIGD5 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC12 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02694 has 0 variants, skipping\n",
      "[INFO]\tGene IL25 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY12 has 0 variants, skipping\n",
      "[INFO]\tGene SBF2 has 0 variants, skipping\n",
      "[INFO]\tGene SUN5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1204 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1203 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1228 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02885 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00052 has 0 variants, skipping\n",
      "[INFO]\tGene MIR409 has 0 variants, skipping\n",
      "[INFO]\tGene MIR451A has 0 variants, skipping\n",
      "[INFO]\tGene MIR329-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR329-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449A has 0 variants, skipping\n",
      "[INFO]\tGene MIR429 has 0 variants, skipping\n",
      "[INFO]\tGene MIR448 has 0 variants, skipping\n",
      "[INFO]\tGene MIR18B has 0 variants, skipping\n",
      "[INFO]\tGene MIR20B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4524B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1180 has 0 variants, skipping\n",
      "[INFO]\tGene MIR452 has 0 variants, skipping\n",
      "[INFO]\tGene WBP11P1 has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1L has 0 variants, skipping\n",
      "[INFO]\tGene MALAT1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A2P1 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene MFF-DT has 0 variants, skipping\n",
      "[INFO]\tGene SNHG8 has 0 variants, skipping\n",
      "[INFO]\tGene ZPLD2P has 0 variants, skipping\n",
      "[INFO]\tGene MECOM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM135A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPP4-DT has 0 variants, skipping\n",
      "[INFO]\tGene HOXC5 has 0 variants, skipping\n",
      "[INFO]\tGene PHC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01287 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01170 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1671-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00511 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG1 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR7 has 0 variants, skipping\n",
      "[INFO]\tGene FTH1P3 has 0 variants, skipping\n",
      "[INFO]\tGene GBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMPO-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-2A has 0 variants, skipping\n",
      "[INFO]\tGene DLEU1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR933 has 0 variants, skipping\n",
      "[INFO]\tGene MIR924 has 0 variants, skipping\n",
      "[INFO]\tGene TSSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG19 has 0 variants, skipping\n",
      "[INFO]\tGene WSPAR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01589 has 0 variants, skipping\n",
      "[INFO]\tGene SMILR has 0 variants, skipping\n",
      "[INFO]\tGene WFDC21P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1179 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2053 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1976 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3064 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8485 has 0 variants, skipping\n",
      "[INFO]\tGene MIR577 has 0 variants, skipping\n",
      "[INFO]\tGene MIR769 has 0 variants, skipping\n",
      "[INFO]\tGene MIR660 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4767 has 0 variants, skipping\n",
      "[INFO]\tGene WWC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene APOA1-AS has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC11B has 0 variants, skipping\n",
      "[INFO]\tGene MIR718 has 0 variants, skipping\n",
      "[INFO]\tGene CLLU1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00525 has 0 variants, skipping\n",
      "[INFO]\tGene IL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730101 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00682 has 0 variants, skipping\n",
      "[INFO]\tGene GLCCI1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR2116 has 0 variants, skipping\n",
      "[INFO]\tGene KLRK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLD6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTPN has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4FP has 0 variants, skipping\n",
      "[INFO]\tGene MIR556 has 0 variants, skipping\n",
      "[INFO]\tGene CYMP has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00467 has 0 variants, skipping\n",
      "[INFO]\tGene WDR88 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7109 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6893 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02605 has 0 variants, skipping\n",
      "[INFO]\tGene HPAT5 has 0 variants, skipping\n",
      "[INFO]\tGene THORLNC has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7I has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7G has 0 variants, skipping\n",
      "[INFO]\tGene MIR1225 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200B has 0 variants, skipping\n",
      "[INFO]\tGene MIR942 has 0 variants, skipping\n",
      "[INFO]\tGene MIR224 has 0 variants, skipping\n",
      "[INFO]\tGene MIR223 has 0 variants, skipping\n",
      "[INFO]\tGene MIR222 has 0 variants, skipping\n",
      "[INFO]\tGene MIR221 has 0 variants, skipping\n",
      "[INFO]\tGene MIR938 has 0 variants, skipping\n",
      "[INFO]\tGene MIR219A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR188 has 0 variants, skipping\n",
      "[INFO]\tGene GNAS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM3AP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6124 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD89 has 0 variants, skipping\n",
      "[INFO]\tGene RMRP has 0 variants, skipping\n",
      "[INFO]\tGene SNHG5 has 0 variants, skipping\n",
      "[INFO]\tGene DSEL has 0 variants, skipping\n",
      "[INFO]\tGene THBS4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC20P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100421372 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1P4 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00518 has 0 variants, skipping\n",
      "[INFO]\tGene BNC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN10 has 0 variants, skipping\n",
      "[INFO]\tGene PARAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3978 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5571 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00514 has 0 variants, skipping\n",
      "[INFO]\tGene OR52H1 has 0 variants, skipping\n",
      "[INFO]\tGene USP51 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3178 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR9 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW4P has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4443 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4429 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4428 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3175 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf36 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf32 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf177 has 0 variants, skipping\n",
      "[INFO]\tGene IRF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene H3P4 has 0 variants, skipping\n",
      "[INFO]\tGene H2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR937 has 0 variants, skipping\n",
      "[INFO]\tGene MIR218-2 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00976 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK1IP1L has 0 variants, skipping\n",
      "[INFO]\tGene PDIA2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85B has 0 variants, skipping\n",
      "[INFO]\tGene EGFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB6 has 0 variants, skipping\n",
      "[INFO]\tGene JSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AB3 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4P1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN13 has 0 variants, skipping\n",
      "[INFO]\tGene JPT2 has 0 variants, skipping\n",
      "[INFO]\tGene IL23R has 0 variants, skipping\n",
      "[INFO]\tGene GPR62 has 0 variants, skipping\n",
      "[INFO]\tGene ASB16 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1K1 has 0 variants, skipping\n",
      "[INFO]\tGene STK35 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP8NL has 0 variants, skipping\n",
      "[INFO]\tGene GTSF1 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf78 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD22 has 0 variants, skipping\n",
      "[INFO]\tGene COMTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF519 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF492 has 0 variants, skipping\n",
      "[INFO]\tGene CABP5 has 0 variants, skipping\n",
      "[INFO]\tGene STAMBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene USP35 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL14 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L2 has 0 variants, skipping\n",
      "[INFO]\tGene DCST2 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO1 has 0 variants, skipping\n",
      "[INFO]\tGene MYORG has 0 variants, skipping\n",
      "[INFO]\tGene TAF3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF4 has 0 variants, skipping\n",
      "[INFO]\tGene FUCA2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF16 has 0 variants, skipping\n",
      "[INFO]\tGene NLN has 0 variants, skipping\n",
      "[INFO]\tGene CAPN12 has 0 variants, skipping\n",
      "[INFO]\tGene KASH5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB9 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CA has 0 variants, skipping\n",
      "[INFO]\tGene CACNG3 has 0 variants, skipping\n",
      "[INFO]\tGene WNT6 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC10A has 0 variants, skipping\n",
      "[INFO]\tGene DPH3P1 has 0 variants, skipping\n",
      "[INFO]\tGene ROMO1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10L has 0 variants, skipping\n",
      "[INFO]\tGene SSC4D has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280A has 0 variants, skipping\n",
      "[INFO]\tGene SRXN1 has 0 variants, skipping\n",
      "[INFO]\tGene NRSN1 has 0 variants, skipping\n",
      "[INFO]\tGene OCSTAMP has 0 variants, skipping\n",
      "[INFO]\tGene GCNT4 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11AUN has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD7 has 0 variants, skipping\n",
      "[INFO]\tGene TKTL2 has 0 variants, skipping\n",
      "[INFO]\tGene FSCB has 0 variants, skipping\n",
      "[INFO]\tGene C10orf126 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7DN has 0 variants, skipping\n",
      "[INFO]\tGene LINC00452 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376353 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378479 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728485 has 0 variants, skipping\n",
      "[INFO]\tGene PCSEAT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02874 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985770 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02881 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ45513 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02876 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134391 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130370 has 0 variants, skipping\n",
      "[INFO]\tGene LOC403312 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505549 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927322 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02873 has 0 variants, skipping\n",
      "[INFO]\tGene IL20RB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF516-DT has 0 variants, skipping\n",
      "[INFO]\tGene NAXE has 0 variants, skipping\n",
      "[INFO]\tGene SMCR8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371566 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724624 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929950 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929578 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371910 has 0 variants, skipping\n",
      "[INFO]\tGene DPP9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA4 has 0 variants, skipping\n",
      "[INFO]\tGene HHIP has 0 variants, skipping\n",
      "[INFO]\tGene GREM2 has 0 variants, skipping\n",
      "[INFO]\tGene SAFB2 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01297 has 0 variants, skipping\n",
      "[INFO]\tGene RS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROM1 has 0 variants, skipping\n",
      "[INFO]\tGene RLBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RHAG has 0 variants, skipping\n",
      "[INFO]\tGene PRPH2 has 0 variants, skipping\n",
      "[INFO]\tGene RB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4282 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2355 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4299 has 0 variants, skipping\n",
      "[INFO]\tGene ADO has 0 variants, skipping\n",
      "[INFO]\tGene POMGNT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA10-MAGEA5 has 0 variants, skipping\n",
      "[INFO]\tGene CRCT1 has 0 variants, skipping\n",
      "[INFO]\tGene AATF has 0 variants, skipping\n",
      "[INFO]\tGene FAM50B has 0 variants, skipping\n",
      "[INFO]\tGene LMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN17 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP4 has 0 variants, skipping\n",
      "[INFO]\tGene CD2AP has 0 variants, skipping\n",
      "[INFO]\tGene NOCT has 0 variants, skipping\n",
      "[INFO]\tGene CASP14 has 0 variants, skipping\n",
      "[INFO]\tGene CA14 has 0 variants, skipping\n",
      "[INFO]\tGene TPX2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM59L has 0 variants, skipping\n",
      "[INFO]\tGene CHIC2 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP10 has 0 variants, skipping\n",
      "[INFO]\tGene PANK3 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A3 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD3 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP6 has 0 variants, skipping\n",
      "[INFO]\tGene OCEL1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC2 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L2 has 0 variants, skipping\n",
      "[INFO]\tGene NDNF has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP25 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RB has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF296 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-3 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PRR35 has 0 variants, skipping\n",
      "[INFO]\tGene SSMEM1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC127 has 0 variants, skipping\n",
      "[INFO]\tGene OSR1 has 0 variants, skipping\n",
      "[INFO]\tGene CCSAP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC25 has 0 variants, skipping\n",
      "[INFO]\tGene ZG16B has 0 variants, skipping\n",
      "[INFO]\tGene IFI27L1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE11 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA7 has 0 variants, skipping\n",
      "[INFO]\tGene DDIT4L has 0 variants, skipping\n",
      "[INFO]\tGene OMA1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PB has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1B has 0 variants, skipping\n",
      "[INFO]\tGene COMMD3 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIP has 0 variants, skipping\n",
      "[INFO]\tGene POLA2 has 0 variants, skipping\n",
      "[INFO]\tGene PNN has 0 variants, skipping\n",
      "[INFO]\tGene CD300C has 0 variants, skipping\n",
      "[INFO]\tGene TRIM58 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1A has 0 variants, skipping\n",
      "[INFO]\tGene KATNB1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K5 has 0 variants, skipping\n",
      "[INFO]\tGene CDC123 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT1 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST2 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST3A1 has 0 variants, skipping\n",
      "[INFO]\tGene MTNR1A has 0 variants, skipping\n",
      "[INFO]\tGene TLR6 has 0 variants, skipping\n",
      "[INFO]\tGene GGA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOH has 0 variants, skipping\n",
      "[INFO]\tGene TBR1 has 0 variants, skipping\n",
      "[INFO]\tGene INTU has 0 variants, skipping\n",
      "[INFO]\tGene CRACD has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1210 has 0 variants, skipping\n",
      "[INFO]\tGene PRR12 has 0 variants, skipping\n",
      "[INFO]\tGene USP31 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF490 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF512B has 0 variants, skipping\n",
      "[INFO]\tGene GATAD2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC47 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA8B has 0 variants, skipping\n",
      "[INFO]\tGene KCNS2 has 0 variants, skipping\n",
      "[INFO]\tGene ERLNC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB3P has 0 variants, skipping\n",
      "[INFO]\tGene ZNNT1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP5 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLNL has 0 variants, skipping\n",
      "[INFO]\tGene UBLCP1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP8 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR570HG has 0 variants, skipping\n",
      "[INFO]\tGene HOXA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01419 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01410 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02273 has 0 variants, skipping\n",
      "[INFO]\tGene NALT1 has 0 variants, skipping\n",
      "[INFO]\tGene MINCR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01518 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2A has 0 variants, skipping\n",
      "[INFO]\tGene SOX9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00342 has 0 variants, skipping\n",
      "[INFO]\tGene LUCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF13P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00968 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLL2 has 0 variants, skipping\n",
      "[INFO]\tGene RSAD2 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2AIPNL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM41A has 0 variants, skipping\n",
      "[INFO]\tGene SDR42E1 has 0 variants, skipping\n",
      "[INFO]\tGene PIGM has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB47 has 0 variants, skipping\n",
      "[INFO]\tGene GPHB5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB7 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRSX has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A12 has 0 variants, skipping\n",
      "[INFO]\tGene ATOH7 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT7 has 0 variants, skipping\n",
      "[INFO]\tGene CTU1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM11 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L1 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC02875 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP17 has 0 variants, skipping\n",
      "[INFO]\tGene MUC5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00842 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01433 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00163 has 0 variants, skipping\n",
      "[INFO]\tGene TP73-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GATC has 0 variants, skipping\n",
      "[INFO]\tGene MSC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CD99P1 has 0 variants, skipping\n",
      "[INFO]\tGene MTURN has 0 variants, skipping\n",
      "[INFO]\tGene ASPRV1 has 0 variants, skipping\n",
      "[INFO]\tGene TAB3 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR56B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR56A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR56B1 has 0 variants, skipping\n",
      "[INFO]\tGene RFC4 has 0 variants, skipping\n",
      "[INFO]\tGene MALSU1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD12 has 0 variants, skipping\n",
      "[INFO]\tGene ARL11 has 0 variants, skipping\n",
      "[INFO]\tGene NXNL1 has 0 variants, skipping\n",
      "[INFO]\tGene GLI4 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND19 has 0 variants, skipping\n",
      "[INFO]\tGene NUS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINDOC has 0 variants, skipping\n",
      "[INFO]\tGene CDAN1 has 0 variants, skipping\n",
      "[INFO]\tGene CHADL has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP7-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT26 has 0 variants, skipping\n",
      "[INFO]\tGene GJC3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT27 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT28 has 0 variants, skipping\n",
      "[INFO]\tGene KRT25 has 0 variants, skipping\n",
      "[INFO]\tGene ERAS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC70 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC8 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA1 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM7 has 0 variants, skipping\n",
      "[INFO]\tGene CD300E has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R42 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC39 has 0 variants, skipping\n",
      "[INFO]\tGene GATA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5590 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5582 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5581 has 0 variants, skipping\n",
      "[INFO]\tGene MIR664B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5195 has 0 variants, skipping\n",
      "[INFO]\tGene WAKMAR2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLR has 0 variants, skipping\n",
      "[INFO]\tGene IFNG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CENATAC has 0 variants, skipping\n",
      "[INFO]\tGene CASC9 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP10 has 0 variants, skipping\n",
      "[INFO]\tGene GSX2 has 0 variants, skipping\n",
      "[INFO]\tGene OAF has 0 variants, skipping\n",
      "[INFO]\tGene MB21D2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTUM has 0 variants, skipping\n",
      "[INFO]\tGene R3HDML has 0 variants, skipping\n",
      "[INFO]\tGene SCP2D1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR119 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12L1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83C has 0 variants, skipping\n",
      "[INFO]\tGene FOXL3 has 0 variants, skipping\n",
      "[INFO]\tGene CADM4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM241B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A43 has 0 variants, skipping\n",
      "[INFO]\tGene WDR90 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf69 has 0 variants, skipping\n",
      "[INFO]\tGene STRC has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF13 has 0 variants, skipping\n",
      "[INFO]\tGene CACUL1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1L has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L5 has 0 variants, skipping\n",
      "[INFO]\tGene LIX1L has 0 variants, skipping\n",
      "[INFO]\tGene TTL has 0 variants, skipping\n",
      "[INFO]\tGene CALHM5 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA5 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG4 has 0 variants, skipping\n",
      "[INFO]\tGene CREG2 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC1 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2C has 0 variants, skipping\n",
      "[INFO]\tGene RYR3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CASC11 has 0 variants, skipping\n",
      "[INFO]\tGene ZAN has 0 variants, skipping\n",
      "[INFO]\tGene LSAMP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HNF4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOW has 0 variants, skipping\n",
      "[INFO]\tGene REG1A has 0 variants, skipping\n",
      "[INFO]\tGene RENBP has 0 variants, skipping\n",
      "[INFO]\tGene MYCNOS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01939 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA119 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD138 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3F has 0 variants, skipping\n",
      "[INFO]\tGene HAGLROS has 0 variants, skipping\n",
      "[INFO]\tGene STK11IP has 0 variants, skipping\n",
      "[INFO]\tGene SGMS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB42 has 0 variants, skipping\n",
      "[INFO]\tGene TADA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A8 has 0 variants, skipping\n",
      "[INFO]\tGene PATE1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene GASAL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM38-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT6 has 0 variants, skipping\n",
      "[INFO]\tGene PKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL3-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122A has 0 variants, skipping\n",
      "[INFO]\tGene ZFC3H1 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE1 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf118 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF564 has 0 variants, skipping\n",
      "[INFO]\tGene FOLH1B has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2C has 0 variants, skipping\n",
      "[INFO]\tGene RTP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A47 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C70 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A13 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA2 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf31 has 0 variants, skipping\n",
      "[INFO]\tGene CMBL has 0 variants, skipping\n",
      "[INFO]\tGene CYGB has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH3 has 0 variants, skipping\n",
      "[INFO]\tGene LGI3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LGI4 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC26 has 0 variants, skipping\n",
      "[INFO]\tGene TEX261 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH8 has 0 variants, skipping\n",
      "[INFO]\tGene RFTN2 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK4 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS13 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN13 has 0 variants, skipping\n",
      "[INFO]\tGene TMED6 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF440 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF491 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF441 has 0 variants, skipping\n",
      "[INFO]\tGene CLDND2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT222 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD1 has 0 variants, skipping\n",
      "[INFO]\tGene OTULIN has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL3 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF15 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM29 has 0 variants, skipping\n",
      "[INFO]\tGene SAAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ID2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMSLR has 0 variants, skipping\n",
      "[INFO]\tGene LINC02888 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02889 has 0 variants, skipping\n",
      "[INFO]\tGene SAMMSON has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VIM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00844 has 0 variants, skipping\n",
      "[INFO]\tGene MIR486-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8075 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6863 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6807 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6089 has 0 variants, skipping\n",
      "[INFO]\tGene CARMN has 0 variants, skipping\n",
      "[INFO]\tGene CRB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133315 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C6 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF9 has 0 variants, skipping\n",
      "[INFO]\tGene MARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SGF29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3679 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3678 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3667 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3659 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3619 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD3 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC1 has 0 variants, skipping\n",
      "[INFO]\tGene TEDDM1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01579 has 0 variants, skipping\n",
      "[INFO]\tGene WASIR1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD45 has 0 variants, skipping\n",
      "[INFO]\tGene KTI12 has 0 variants, skipping\n",
      "[INFO]\tGene LSM14B has 0 variants, skipping\n",
      "[INFO]\tGene IFNL1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNL2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4Q3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL54 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2D has 0 variants, skipping\n",
      "[INFO]\tGene SUGP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP22-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT6C has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R36 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724728 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375800 has 0 variants, skipping\n",
      "[INFO]\tGene COL23A1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX7 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF17 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F22 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC105 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX4 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5CL has 0 variants, skipping\n",
      "[INFO]\tGene FAM217A has 0 variants, skipping\n",
      "[INFO]\tGene NYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH10B has 0 variants, skipping\n",
      "[INFO]\tGene CGAS has 0 variants, skipping\n",
      "[INFO]\tGene VASN has 0 variants, skipping\n",
      "[INFO]\tGene FAM83F has 0 variants, skipping\n",
      "[INFO]\tGene ZBED6CL has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC7B has 0 variants, skipping\n",
      "[INFO]\tGene GLCCI1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF12 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf42 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB6 has 0 variants, skipping\n",
      "[INFO]\tGene REM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71D has 0 variants, skipping\n",
      "[INFO]\tGene RNF152 has 0 variants, skipping\n",
      "[INFO]\tGene RFX6 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS6 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC8B has 0 variants, skipping\n",
      "[INFO]\tGene APLF has 0 variants, skipping\n",
      "[INFO]\tGene CNBD1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG1 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1 has 0 variants, skipping\n",
      "[INFO]\tGene PASD1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM117B has 0 variants, skipping\n",
      "[INFO]\tGene FGD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR14L has 0 variants, skipping\n",
      "[INFO]\tGene CT47A11 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD2 has 0 variants, skipping\n",
      "[INFO]\tGene PGM2L1 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF22 has 0 variants, skipping\n",
      "[INFO]\tGene TRARG1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM59 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR4 has 0 variants, skipping\n",
      "[INFO]\tGene CPO has 0 variants, skipping\n",
      "[INFO]\tGene ATRIP-TREX1 has 0 variants, skipping\n",
      "[INFO]\tGene DIRC1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM11 has 0 variants, skipping\n",
      "[INFO]\tGene TTBK2 has 0 variants, skipping\n",
      "[INFO]\tGene MROH2B has 0 variants, skipping\n",
      "[INFO]\tGene UBN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD52 has 0 variants, skipping\n",
      "[INFO]\tGene MYHAS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00407 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00319 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A9 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF149 has 0 variants, skipping\n",
      "[INFO]\tGene RIMKLA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF547 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R2 has 0 variants, skipping\n",
      "[INFO]\tGene IPMK has 0 variants, skipping\n",
      "[INFO]\tGene LOC284898 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB106A has 0 variants, skipping\n",
      "[INFO]\tGene PARS2 has 0 variants, skipping\n",
      "[INFO]\tGene UBTD2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC30A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF585B has 0 variants, skipping\n",
      "[INFO]\tGene SYT11 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4C has 0 variants, skipping\n",
      "[INFO]\tGene ORAI3 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3AP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN3 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf45 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14EP has 0 variants, skipping\n",
      "[INFO]\tGene C12orf54 has 0 variants, skipping\n",
      "[INFO]\tGene SPIC has 0 variants, skipping\n",
      "[INFO]\tGene ACOT4 has 0 variants, skipping\n",
      "[INFO]\tGene EID2B has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L3 has 0 variants, skipping\n",
      "[INFO]\tGene MYOM3 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R3 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf39 has 0 variants, skipping\n",
      "[INFO]\tGene POTED has 0 variants, skipping\n",
      "[INFO]\tGene SPEM2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG3 has 0 variants, skipping\n",
      "[INFO]\tGene TCF23 has 0 variants, skipping\n",
      "[INFO]\tGene KRT79 has 0 variants, skipping\n",
      "[INFO]\tGene FTMT has 0 variants, skipping\n",
      "[INFO]\tGene FAM171B has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R60 has 0 variants, skipping\n",
      "[INFO]\tGene STX12 has 0 variants, skipping\n",
      "[INFO]\tGene KLC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIDN has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY13 has 0 variants, skipping\n",
      "[INFO]\tGene IFNE has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R20 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R50 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R19 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R46 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R45 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R31 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R43 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R41 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R40 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R39 has 0 variants, skipping\n",
      "[INFO]\tGene MT1M has 0 variants, skipping\n",
      "[INFO]\tGene S100A7A has 0 variants, skipping\n",
      "[INFO]\tGene NLRP14 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP9 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R38 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF800 has 0 variants, skipping\n",
      "[INFO]\tGene AGR3 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4B has 0 variants, skipping\n",
      "[INFO]\tGene PKHD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT9 has 0 variants, skipping\n",
      "[INFO]\tGene LYG2 has 0 variants, skipping\n",
      "[INFO]\tGene KLB has 0 variants, skipping\n",
      "[INFO]\tGene FAM151A has 0 variants, skipping\n",
      "[INFO]\tGene PRR18 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf51 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf54 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS1L1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM181B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC71L has 0 variants, skipping\n",
      "[INFO]\tGene OR7D2 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2C has 0 variants, skipping\n",
      "[INFO]\tGene ODF3L1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC8 has 0 variants, skipping\n",
      "[INFO]\tGene SIX5 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf60 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF792 has 0 variants, skipping\n",
      "[INFO]\tGene SWSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP11-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP8-1 has 0 variants, skipping\n",
      "[INFO]\tGene CHSY3 has 0 variants, skipping\n",
      "[INFO]\tGene PAN3 has 0 variants, skipping\n",
      "[INFO]\tGene KDF1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM241A has 0 variants, skipping\n",
      "[INFO]\tGene TRAM1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL40 has 0 variants, skipping\n",
      "[INFO]\tGene CR1L has 0 variants, skipping\n",
      "[INFO]\tGene LSINCT5 has 0 variants, skipping\n",
      "[INFO]\tGene TALAM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR205HG has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER3 has 0 variants, skipping\n",
      "[INFO]\tGene OTOP1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB8 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1G has 0 variants, skipping\n",
      "[INFO]\tGene SESTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL5 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene PILRB has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GSDMA has 0 variants, skipping\n",
      "[INFO]\tGene NEK8 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF598 has 0 variants, skipping\n",
      "[INFO]\tGene PTF1A has 0 variants, skipping\n",
      "[INFO]\tGene OTOP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4Z1 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY8 has 0 variants, skipping\n",
      "[INFO]\tGene FADS6 has 0 variants, skipping\n",
      "[INFO]\tGene TONSL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L16 has 0 variants, skipping\n",
      "[INFO]\tGene FARSA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01259 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01074 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C5-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene UC.134 has 0 variants, skipping\n",
      "[INFO]\tGene PNKY has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L17 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369306 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371855 has 0 variants, skipping\n",
      "[INFO]\tGene SCTR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930421 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422781 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374042 has 0 variants, skipping\n",
      "[INFO]\tGene HHATL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2117HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02041 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091777 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091776 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01980 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091768 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2-AS has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A4-ANKRD20A20P has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K1P1-CATSPER2 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P4-NPIPA8 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P5-LOC105376752 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P3-NPIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG14 has 0 variants, skipping\n",
      "[INFO]\tGene PWAR6 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302CHG has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWCE has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE6 has 0 variants, skipping\n",
      "[INFO]\tGene MASCRNA has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP2 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD7-DCLRE1CP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01149 has 0 variants, skipping\n",
      "[INFO]\tGene MORC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARSD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BARX1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00402 has 0 variants, skipping\n",
      "[INFO]\tGene LOC108783654 has 0 variants, skipping\n",
      "[INFO]\tGene LOC108783645 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERD has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP15-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-3 has 0 variants, skipping\n",
      "[INFO]\tGene DMBT1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP23-1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK18 has 0 variants, skipping\n",
      "[INFO]\tGene TMED7 has 0 variants, skipping\n",
      "[INFO]\tGene BMP8A has 0 variants, skipping\n",
      "[INFO]\tGene POLN has 0 variants, skipping\n",
      "[INFO]\tGene H1-7 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA12 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD37 has 0 variants, skipping\n",
      "[INFO]\tGene METTL2A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM119 has 0 variants, skipping\n",
      "[INFO]\tGene FOXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ASPHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPV has 0 variants, skipping\n",
      "[INFO]\tGene CRTC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF4 has 0 variants, skipping\n",
      "[INFO]\tGene BCDIN3D has 0 variants, skipping\n",
      "[INFO]\tGene C17orf64 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC24 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene QSOX2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-2 has 0 variants, skipping\n",
      "[INFO]\tGene OGT has 0 variants, skipping\n",
      "[INFO]\tGene CPLX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF804B has 0 variants, skipping\n",
      "[INFO]\tGene DEUP1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4A has 0 variants, skipping\n",
      "[INFO]\tGene HOTTIP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01234 has 0 variants, skipping\n",
      "[INFO]\tGene ANKK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A12 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR1P2 has 0 variants, skipping\n",
      "[INFO]\tGene GPX6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98B has 0 variants, skipping\n",
      "[INFO]\tGene KSR2 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM37 has 0 variants, skipping\n",
      "[INFO]\tGene EML5 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP26C1 has 0 variants, skipping\n",
      "[INFO]\tGene PXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A13 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2A has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FIBIN has 0 variants, skipping\n",
      "[INFO]\tGene CAPRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF784 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM81 has 0 variants, skipping\n",
      "[INFO]\tGene RPRML has 0 variants, skipping\n",
      "[INFO]\tGene FITM1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F7 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP26-1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47A has 0 variants, skipping\n",
      "[INFO]\tGene EZHIP has 0 variants, skipping\n",
      "[INFO]\tGene MBLAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MBLAC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBL4B has 0 variants, skipping\n",
      "[INFO]\tGene LRRN4CL has 0 variants, skipping\n",
      "[INFO]\tGene IER5L has 0 variants, skipping\n",
      "[INFO]\tGene IQCF2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL4 has 0 variants, skipping\n",
      "[INFO]\tGene GGPS1 has 0 variants, skipping\n",
      "[INFO]\tGene LDHD has 0 variants, skipping\n",
      "[INFO]\tGene MDGA1 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT4 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST5 has 0 variants, skipping\n",
      "[INFO]\tGene CMYA5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71A has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG3 has 0 variants, skipping\n",
      "[INFO]\tGene COG7 has 0 variants, skipping\n",
      "[INFO]\tGene PHLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GLCT has 0 variants, skipping\n",
      "[INFO]\tGene KIF24 has 0 variants, skipping\n",
      "[INFO]\tGene IDO2 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf74 has 0 variants, skipping\n",
      "[INFO]\tGene FAM151B has 0 variants, skipping\n",
      "[INFO]\tGene LYPD2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2K2 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA5 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG32 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SYPL2 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P3 has 0 variants, skipping\n",
      "[INFO]\tGene TDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene BIVM-ERCC5 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986192 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267991 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987142 has 0 variants, skipping\n",
      "[INFO]\tGene YY2 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR34 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A4 has 0 variants, skipping\n",
      "[INFO]\tGene LMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM105 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf46 has 0 variants, skipping\n",
      "[INFO]\tGene GPR152 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG17 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D4 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA1 has 0 variants, skipping\n",
      "[INFO]\tGene SALL3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB39B has 0 variants, skipping\n",
      "[INFO]\tGene OR10W1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf99 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375303 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf139 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD4 has 0 variants, skipping\n",
      "[INFO]\tGene LCNL1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB132 has 0 variants, skipping\n",
      "[INFO]\tGene TEX19 has 0 variants, skipping\n",
      "[INFO]\tGene NAIF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A6 has 0 variants, skipping\n",
      "[INFO]\tGene GNPNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2C3 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1B has 0 variants, skipping\n",
      "[INFO]\tGene RNF148 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC56 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02817 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT3 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2B has 0 variants, skipping\n",
      "[INFO]\tGene MUC19 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ44635 has 0 variants, skipping\n",
      "[INFO]\tGene PADI6 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A7 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D3 has 0 variants, skipping\n",
      "[INFO]\tGene ALKAL1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRAL has 0 variants, skipping\n",
      "[INFO]\tGene GBP7 has 0 variants, skipping\n",
      "[INFO]\tGene TEX43 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11F has 0 variants, skipping\n",
      "[INFO]\tGene RNF207 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF324B has 0 variants, skipping\n",
      "[INFO]\tGene IGFL3 has 0 variants, skipping\n",
      "[INFO]\tGene CPXM2 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD15 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER4 has 0 variants, skipping\n",
      "[INFO]\tGene QRFP has 0 variants, skipping\n",
      "[INFO]\tGene TREML4 has 0 variants, skipping\n",
      "[INFO]\tGene UTS2B has 0 variants, skipping\n",
      "[INFO]\tGene ARSK has 0 variants, skipping\n",
      "[INFO]\tGene FLJ20712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02591 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18C has 0 variants, skipping\n",
      "[INFO]\tGene GALNT10 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL17 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC2P has 0 variants, skipping\n",
      "[INFO]\tGene KCTD8 has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM17 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD4 has 0 variants, skipping\n",
      "[INFO]\tGene KLRG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM174A has 0 variants, skipping\n",
      "[INFO]\tGene LRIT3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A5 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20A has 0 variants, skipping\n",
      "[INFO]\tGene EMB has 0 variants, skipping\n",
      "[INFO]\tGene GOLT1A has 0 variants, skipping\n",
      "[INFO]\tGene FOXR2 has 0 variants, skipping\n",
      "[INFO]\tGene APOOL has 0 variants, skipping\n",
      "[INFO]\tGene LONRF2 has 0 variants, skipping\n",
      "[INFO]\tGene RSBN1L has 0 variants, skipping\n",
      "[INFO]\tGene NRK has 0 variants, skipping\n",
      "[INFO]\tGene FAM171A2 has 0 variants, skipping\n",
      "[INFO]\tGene KANK3 has 0 variants, skipping\n",
      "[INFO]\tGene TPRX1 has 0 variants, skipping\n",
      "[INFO]\tGene NKPD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83H has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1 has 0 variants, skipping\n",
      "[INFO]\tGene CA13 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-12 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-8 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-11 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-9 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-6 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-4 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF775 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD8 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP2 has 0 variants, skipping\n",
      "[INFO]\tGene LVRN has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB18 has 0 variants, skipping\n",
      "[INFO]\tGene FAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM73 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRD has 0 variants, skipping\n",
      "[INFO]\tGene CYSRT1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF175 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF660 has 0 variants, skipping\n",
      "[INFO]\tGene MIR655 has 0 variants, skipping\n",
      "[INFO]\tGene LOC114841040 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT2L6 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH6 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF710 has 0 variants, skipping\n",
      "[INFO]\tGene KIF7 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC172 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC2 has 0 variants, skipping\n",
      "[INFO]\tGene STIMATE has 0 variants, skipping\n",
      "[INFO]\tGene DPPA3 has 0 variants, skipping\n",
      "[INFO]\tGene PRR19 has 0 variants, skipping\n",
      "[INFO]\tGene POLQ has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH3 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB3 has 0 variants, skipping\n",
      "[INFO]\tGene ENHO has 0 variants, skipping\n",
      "[INFO]\tGene KLHL30 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC1 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM86B has 0 variants, skipping\n",
      "[INFO]\tGene NELFCD has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF3 has 0 variants, skipping\n",
      "[INFO]\tGene CARM1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L3 has 0 variants, skipping\n",
      "[INFO]\tGene AQP12A has 0 variants, skipping\n",
      "[INFO]\tGene SYT10 has 0 variants, skipping\n",
      "[INFO]\tGene LDLR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC80 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNC-LBCS has 0 variants, skipping\n",
      "[INFO]\tGene UST-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene COSMOC has 0 variants, skipping\n",
      "[INFO]\tGene LASTR has 0 variants, skipping\n",
      "[INFO]\tGene FAM230D has 0 variants, skipping\n",
      "[INFO]\tGene FAM230J has 0 variants, skipping\n",
      "[INFO]\tGene FAM230A has 0 variants, skipping\n",
      "[INFO]\tGene FAM230E has 0 variants, skipping\n",
      "[INFO]\tGene FAM230F has 0 variants, skipping\n",
      "[INFO]\tGene FAM230G has 0 variants, skipping\n",
      "[INFO]\tGene FAM230B has 0 variants, skipping\n",
      "[INFO]\tGene FAM230H has 0 variants, skipping\n",
      "[INFO]\tGene FAM230I has 0 variants, skipping\n",
      "[INFO]\tGene HSPB8 has 0 variants, skipping\n",
      "[INFO]\tGene GAPDHS has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-8 has 0 variants, skipping\n",
      "[INFO]\tGene OPTC has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4E has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1ANB has 0 variants, skipping\n",
      "[INFO]\tGene HCAR3 has 0 variants, skipping\n",
      "[INFO]\tGene MCM6 has 0 variants, skipping\n",
      "[INFO]\tGene LAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene MC5R has 0 variants, skipping\n",
      "[INFO]\tGene CLPP has 0 variants, skipping\n",
      "[INFO]\tGene MC4R has 0 variants, skipping\n",
      "[INFO]\tGene MAT2A has 0 variants, skipping\n",
      "[INFO]\tGene MANF has 0 variants, skipping\n",
      "[INFO]\tGene MANBA has 0 variants, skipping\n",
      "[INFO]\tGene MAN1A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR466 has 0 variants, skipping\n",
      "[INFO]\tGene GHET1 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT3 has 0 variants, skipping\n",
      "[INFO]\tGene MHRT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00504 has 0 variants, skipping\n",
      "[INFO]\tGene MAILR has 0 variants, skipping\n",
      "[INFO]\tGene USH2A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXCUT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01048 has 0 variants, skipping\n",
      "[INFO]\tGene TBCEL-TECTA has 0 variants, skipping\n",
      "[INFO]\tGene LOC116435278 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144NL has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267934 has 0 variants, skipping\n",
      "[INFO]\tGene LOC115308161 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf129 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2L has 0 variants, skipping\n",
      "[INFO]\tGene NUDT3 has 0 variants, skipping\n",
      "[INFO]\tGene H3Y2 has 0 variants, skipping\n",
      "[INFO]\tGene MEMO1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM67-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP53 has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NFE4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y2 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6-AHRR has 0 variants, skipping\n",
      "[INFO]\tGene SREBF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFDP1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI30 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM17A has 0 variants, skipping\n",
      "[INFO]\tGene TACC3 has 0 variants, skipping\n",
      "[INFO]\tGene HMG20B has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987389 has 0 variants, skipping\n",
      "[INFO]\tGene LNCDAT has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A10 has 0 variants, skipping\n",
      "[INFO]\tGene LCA5L has 0 variants, skipping\n",
      "[INFO]\tGene TFF3 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf39 has 0 variants, skipping\n",
      "[INFO]\tGene MCHR1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC134 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0930 has 0 variants, skipping\n",
      "[INFO]\tGene TTC38 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2FP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP14 has 0 variants, skipping\n",
      "[INFO]\tGene CD47 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1L has 0 variants, skipping\n",
      "[INFO]\tGene POPDC2 has 0 variants, skipping\n",
      "[INFO]\tGene COX17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A3 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf80 has 0 variants, skipping\n",
      "[INFO]\tGene NSG1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B17 has 0 variants, skipping\n",
      "[INFO]\tGene BMP3 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf54 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM43 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728178 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2R has 0 variants, skipping\n",
      "[INFO]\tGene CCDC192 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA24 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB3 has 0 variants, skipping\n",
      "[INFO]\tGene PROP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF311 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-G has 0 variants, skipping\n",
      "[INFO]\tGene TOMM6 has 0 variants, skipping\n",
      "[INFO]\tGene HCRTR2 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724646 has 0 variants, skipping\n",
      "[INFO]\tGene CCN2 has 0 variants, skipping\n",
      "[INFO]\tGene THBS2 has 0 variants, skipping\n",
      "[INFO]\tGene AMZ1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD11 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM7 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA3 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRID1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE21 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE7P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE8 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE11 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE15 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE9 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE13 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE14 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE10P has 0 variants, skipping\n",
      "[INFO]\tGene CLDN4 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE12P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE17 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3D has 0 variants, skipping\n",
      "[INFO]\tGene FAM237B has 0 variants, skipping\n",
      "[INFO]\tGene LRRD1 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHA15 has 0 variants, skipping\n",
      "[INFO]\tGene SYPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR85 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf77 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927506 has 0 variants, skipping\n",
      "[INFO]\tGene TDRP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377785 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928095 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-6 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf86 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375938 has 0 variants, skipping\n",
      "[INFO]\tGene MSC has 0 variants, skipping\n",
      "[INFO]\tGene VPS13B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375683 has 0 variants, skipping\n",
      "[INFO]\tGene IQANK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF517 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC180 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf152 has 0 variants, skipping\n",
      "[INFO]\tGene GGTA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL11 has 0 variants, skipping\n",
      "[INFO]\tGene GPR21 has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC26 has 0 variants, skipping\n",
      "[INFO]\tGene MED14OS has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZXDB has 0 variants, skipping\n",
      "[INFO]\tGene ZXDA has 0 variants, skipping\n",
      "[INFO]\tGene GPR174 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD4 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1F has 0 variants, skipping\n",
      "[INFO]\tGene LRRC41 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP3 has 0 variants, skipping\n",
      "[INFO]\tGene MR1 has 0 variants, skipping\n",
      "[INFO]\tGene IL20 has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF839 has 0 variants, skipping\n",
      "[INFO]\tGene TEX9 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF11 has 0 variants, skipping\n",
      "[INFO]\tGene PEAK1 has 0 variants, skipping\n",
      "[INFO]\tGene USP36 has 0 variants, skipping\n",
      "[INFO]\tGene IRS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF217 has 0 variants, skipping\n",
      "[INFO]\tGene IFRD2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L has 0 variants, skipping\n",
      "[INFO]\tGene GRK7 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF10 has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14EPL has 0 variants, skipping\n",
      "[INFO]\tGene AADACL4 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1C1 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA3 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18B has 0 variants, skipping\n",
      "[INFO]\tGene GFY has 0 variants, skipping\n",
      "[INFO]\tGene LSR has 0 variants, skipping\n",
      "[INFO]\tGene IL37 has 0 variants, skipping\n",
      "[INFO]\tGene TMC2 has 0 variants, skipping\n",
      "[INFO]\tGene ASIP has 0 variants, skipping\n",
      "[INFO]\tGene CT45A10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268136 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM72A has 0 variants, skipping\n",
      "[INFO]\tGene IL17D has 0 variants, skipping\n",
      "[INFO]\tGene MTMR6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF439 has 0 variants, skipping\n",
      "[INFO]\tGene CGB7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM237A has 0 variants, skipping\n",
      "[INFO]\tGene LAP3 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A1 has 0 variants, skipping\n",
      "[INFO]\tGene GHRH has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ECMXP has 0 variants, skipping\n",
      "[INFO]\tGene NBPF15 has 0 variants, skipping\n",
      "[INFO]\tGene CNGB3 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF11 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS33 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP11 has 0 variants, skipping\n",
      "[INFO]\tGene DDX27 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF6 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE2B has 0 variants, skipping\n",
      "[INFO]\tGene RBM15 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3L2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF677 has 0 variants, skipping\n",
      "[INFO]\tGene STARD7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985532 has 0 variants, skipping\n",
      "[INFO]\tGene ODAM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF608 has 0 variants, skipping\n",
      "[INFO]\tGene OR2J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5V1 has 0 variants, skipping\n",
      "[INFO]\tGene OR11A1 has 0 variants, skipping\n",
      "[INFO]\tGene SMLR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR9A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNK has 0 variants, skipping\n",
      "[INFO]\tGene MAP10 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E5 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AN1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G8 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5B has 0 variants, skipping\n",
      "[INFO]\tGene OR2A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A12 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A14 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L6 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC390877 has 0 variants, skipping\n",
      "[INFO]\tGene FAT2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52K1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51H1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G5 has 0 variants, skipping\n",
      "[INFO]\tGene PABPN1L has 0 variants, skipping\n",
      "[INFO]\tGene WDR36 has 0 variants, skipping\n",
      "[INFO]\tGene OR51G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR51L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR9G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B12 has 0 variants, skipping\n",
      "[INFO]\tGene OR5A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10D3 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT4 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE4 has 0 variants, skipping\n",
      "[INFO]\tAnalyzed [ 571 ] variants from [ 28307 ] genes/regions\n",
      "[INFO]\tAnalysis ends at: Fri Feb  5 14:50:11 2021\n",
      "[INFO]\tAnalysis took 12 seconds\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "export PATH=$PATH:~/bin/executable\n",
    "rvtest --noweb --inVcf AMPPD_FILTERED.RIC3.vcf.gz --pheno covariateFile.txt --pheno-name PHENO --covar covariateFile.txt --covar-name SEX,AGE,PC1,PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10 --kernel skat,skato --burden cmc,zeggini,mb,fp,cmcWald --geneFile refFlat.txt --freqUpper 0.03 --out AMP_PD_BURDEN.RIC3.WGS.maf003_KB"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### CODING VARIANTS "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "export PATH=$PATH:~/bin/rvtests/third/tabix-0.2.6/\n",
    "bgzip RIC3_CODING_KB.vcf\n",
    "tabix -f -p vcf RIC3_CODING_KB.vcf.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Thank you for using rvtests (version: 20190205, git: c86e589efef15382603300dc7f4c3394c82d69b8)\n",
      "  For documentations, refer to http://zhanxw.github.io/rvtests/\n",
      "  For questions and comments, plase send to Xiaowei Zhan <zhanxw@umich.edu>\n",
      "  For bugs and feature requests, please submit at: https://github.com/zhanxw/rvtests/issues\n",
      "\n",
      "RVTESTS finished successfully\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The following parameters are available.  Ones with \"[]\" are in effect:\n",
      "\n",
      "Available Options\n",
      "      Basic Input/Output: --inVcf [RIC3_CODING_KB.vcf.gz], --inBgen []\n",
      "                          --inBgenSample [], --inKgg []\n",
      "                          --out [AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING]\n",
      "                          --outputRaw\n",
      "       Specify Covariate: --covar [covariateFile.txt], --covar-name [SEX,AGE,PC1\n",
      "                         PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10], --sex\n",
      "       Specify Phenotype: --pheno [covariateFile.txt], --inverseNormal\n",
      "                          --useResidualAsPhenotype, --mpheno []\n",
      "                          --pheno-name [PHENO], --qtl, --multiplePheno []\n",
      "        Specify Genotype: --dosage [], --multipleAllele\n",
      "    Chromosome X Options: --xLabel [], --xParRegion []\n",
      "           People Filter: --peopleIncludeID [], --peopleIncludeFile []\n",
      "                          --peopleExcludeID [], --peopleExcludeFile []\n",
      "             Site Filter: --rangeList [], --rangeFile [], --siteFile []\n",
      "                          --siteDepthMin [], --siteDepthMax [], --siteMACMin []\n",
      "                          --annoType []\n",
      "         Genotype Filter: --indvDepthMin [], --indvDepthMax [], --indvQualMin []\n",
      "       Association Model: --single [], --burden [cmc,zeggini,mb,fp,cmcWald]\n",
      "                          --vt [], --kernel [skat,skato], --meta []\n",
      "     Family-based Models: --kinship [], --xHemiKinship [], --kinshipEigen []\n",
      "                          --xHemiKinshipEigen [], --boltPlink []\n",
      "                          --boltPlinkNoCheck\n",
      "          Grouping Unit : --geneFile [refFlat.txt], --gene [], --setList []\n",
      "                          --setFile [], --set []\n",
      "        Frequency Cutoff: --freqUpper [0.03], --freqLower []\n",
      "            Missing Data: --impute [], --imputePheno, --imputeCov\n",
      "    Conditional Analysis: --condition []\n",
      "     Auxiliary Functions: --noweb [true], --hide-covar, --numThread []\n",
      "                          --outputID, --help\n",
      "\n",
      "Effective Options\n",
      "    --inVcf RIC3_CODING_KB.vcf.gz\n",
      "    --out AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING\n",
      "    --covar covariateFile.txt\n",
      "    --covar-name SEX,AGE,PC1,PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10\n",
      "    --pheno covariateFile.txt\n",
      "    --pheno-name PHENO\n",
      "    --burden cmc,zeggini,mb,fp,cmcWald\n",
      "    --kernel skat,skato\n",
      "    --geneFile refFlat.txt\n",
      "    --freqUpper 0.03\n",
      "    --noweb\n",
      "\n",
      "[INFO]\tProgram version: 20190205\n",
      "[INFO]\tAnalysis started at: Fri Feb  5 14:50:26 2021\n",
      "[INFO]\tLoaded [ 2576 ] samples from genotype files\n",
      "[INFO]\tLoaded [ 2576 ] sample phenotypes\n",
      "[INFO]\tBegin to read covariate file\n",
      "[INFO]\tLoaded 1455 male, 1121 female and 0 sex-unknown samples from covariateFile.txt\n",
      "[INFO]\tLoaded 1615 cases, 961 controls, and 0 missing phenotypes\n",
      "[WARN]\t-- Enabling binary phenotype mode -- \n",
      "[INFO]\tAnalysis begins with [ 2576 ] samples...\n",
      "[INFO]\tMadsonBrowning test significance will be evaluated using 10000 permutations at alpha = 0.05\n",
      "[INFO]\tSKAT test significance will be evaluated using 10000 permutations at alpha = 0.05 weight = Beta[beta1 = 1.00, beta2 = 25.00]\n",
      "[INFO]\tSKAT-O test significance will be evaluated using weight = Beta[beta1 = 1.00, beta2 = 25.00]\n",
      "[INFO]\tLoaded [ 28307 ] genes.\n",
      "[INFO]\tImpute missing genotype to mean (by default)\n",
      "[INFO]\tSet upper minor allele frequency limit to 0.03\n",
      "[INFO]\tAnalysis started\n",
      "[INFO]\tGene ZNF639 has 0 variants, skipping\n",
      "[INFO]\tGene WASH7P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6859-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-11 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138A has 0 variants, skipping\n",
      "[INFO]\tGene FAM138F has 0 variants, skipping\n",
      "[INFO]\tGene FAM138C has 0 variants, skipping\n",
      "[INFO]\tGene LOC729737 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138D has 0 variants, skipping\n",
      "[INFO]\tGene PTX3 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRZ1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132287 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928626 has 0 variants, skipping\n",
      "[INFO]\tGene FAM87B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00115 has 0 variants, skipping\n",
      "[INFO]\tGene MADD has 0 variants, skipping\n",
      "[INFO]\tGene FAM41C has 0 variants, skipping\n",
      "[INFO]\tGene LINC02593 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB9 has 0 variants, skipping\n",
      "[INFO]\tGene GNL3 has 0 variants, skipping\n",
      "[INFO]\tGene MESD has 0 variants, skipping\n",
      "[INFO]\tGene PHF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR512-1 has 0 variants, skipping\n",
      "[INFO]\tGene TSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH12 has 0 variants, skipping\n",
      "[INFO]\tGene WDSUB1 has 0 variants, skipping\n",
      "[INFO]\tGene ADPGK has 0 variants, skipping\n",
      "[INFO]\tGene THOC5 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNG has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288175 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01342 has 0 variants, skipping\n",
      "[INFO]\tGene PHF1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM18 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD has 0 variants, skipping\n",
      "[INFO]\tGene PXK has 0 variants, skipping\n",
      "[INFO]\tGene NT5C2 has 0 variants, skipping\n",
      "[INFO]\tGene SERF1B has 0 variants, skipping\n",
      "[INFO]\tGene ASCC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF24 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A5P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6726 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD167 has 0 variants, skipping\n",
      "[INFO]\tGene ECSIT has 0 variants, skipping\n",
      "[INFO]\tGene PUSL1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM243 has 0 variants, skipping\n",
      "[INFO]\tGene NUP85 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6727 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6808 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC6 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF13 has 0 variants, skipping\n",
      "[INFO]\tGene EED has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF14 has 0 variants, skipping\n",
      "[INFO]\tGene URI1 has 0 variants, skipping\n",
      "[INFO]\tGene HRK has 0 variants, skipping\n",
      "[INFO]\tGene SNX3 has 0 variants, skipping\n",
      "[INFO]\tGene CRADD has 0 variants, skipping\n",
      "[INFO]\tGene SNX4 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK1 has 0 variants, skipping\n",
      "[INFO]\tGene STK31 has 0 variants, skipping\n",
      "[INFO]\tGene PANX2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01770 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H11A has 0 variants, skipping\n",
      "[INFO]\tGene CEP78 has 0 variants, skipping\n",
      "[INFO]\tGene PCM1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL12 has 0 variants, skipping\n",
      "[INFO]\tGene AP1G2 has 0 variants, skipping\n",
      "[INFO]\tGene SMC2 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378586 has 0 variants, skipping\n",
      "[INFO]\tGene PAX6 has 0 variants, skipping\n",
      "[INFO]\tGene ARFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL2 has 0 variants, skipping\n",
      "[INFO]\tGene HPS4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM91A1 has 0 variants, skipping\n",
      "[INFO]\tGene KANK2 has 0 variants, skipping\n",
      "[INFO]\tGene RBPJL has 0 variants, skipping\n",
      "[INFO]\tGene MMP23A has 0 variants, skipping\n",
      "[INFO]\tGene SPNS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B4 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2M has 0 variants, skipping\n",
      "[INFO]\tGene UFM1 has 0 variants, skipping\n",
      "[INFO]\tGene NME8 has 0 variants, skipping\n",
      "[INFO]\tGene STAT3 has 0 variants, skipping\n",
      "[INFO]\tGene PSPH has 0 variants, skipping\n",
      "[INFO]\tGene DPCD has 0 variants, skipping\n",
      "[INFO]\tGene DYDC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTCFL has 0 variants, skipping\n",
      "[INFO]\tGene GABRB3 has 0 variants, skipping\n",
      "[INFO]\tGene FDFT1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM9 has 0 variants, skipping\n",
      "[INFO]\tGene DDX55 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL4 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL2 has 0 variants, skipping\n",
      "[INFO]\tGene DDAH2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPI has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D1 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf35 has 0 variants, skipping\n",
      "[INFO]\tGene ERC2 has 0 variants, skipping\n",
      "[INFO]\tGene TANGO2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2G has 0 variants, skipping\n",
      "[INFO]\tGene RNFT1 has 0 variants, skipping\n",
      "[INFO]\tGene RHD has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378591 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC7 has 0 variants, skipping\n",
      "[INFO]\tGene GANAB has 0 variants, skipping\n",
      "[INFO]\tGene TMC5 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP91 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF260 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCZ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPH3AL has 0 variants, skipping\n",
      "[INFO]\tGene FAAP20 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA3 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC18 has 0 variants, skipping\n",
      "[INFO]\tGene PARP15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129534 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO48 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf68 has 0 variants, skipping\n",
      "[INFO]\tGene MED29 has 0 variants, skipping\n",
      "[INFO]\tGene BRAF has 0 variants, skipping\n",
      "[INFO]\tGene DMXL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R3 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SNX13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996583 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD12B has 0 variants, skipping\n",
      "[INFO]\tGene DCTN2 has 0 variants, skipping\n",
      "[INFO]\tGene IPO8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM104B has 0 variants, skipping\n",
      "[INFO]\tGene C11orf91 has 0 variants, skipping\n",
      "[INFO]\tGene TTK has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3E has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEE1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM16-DT has 0 variants, skipping\n",
      "[INFO]\tGene ODF2L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4251 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD10 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN3 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA4 has 0 variants, skipping\n",
      "[INFO]\tGene RGS7 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF1 has 0 variants, skipping\n",
      "[INFO]\tGene SGTA has 0 variants, skipping\n",
      "[INFO]\tGene RGR has 0 variants, skipping\n",
      "[INFO]\tGene COPB1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM8B has 0 variants, skipping\n",
      "[INFO]\tGene PLEK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF226 has 0 variants, skipping\n",
      "[INFO]\tGene CELF4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3CL has 0 variants, skipping\n",
      "[INFO]\tGene CAMKK2 has 0 variants, skipping\n",
      "[INFO]\tGene GOSR2 has 0 variants, skipping\n",
      "[INFO]\tGene AURKB has 0 variants, skipping\n",
      "[INFO]\tGene PIGT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01134 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01777 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01646 has 0 variants, skipping\n",
      "[INFO]\tGene DAZ3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4689 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2B has 0 variants, skipping\n",
      "[INFO]\tGene TGOLN2 has 0 variants, skipping\n",
      "[INFO]\tGene GSG1 has 0 variants, skipping\n",
      "[INFO]\tGene SUN1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPA1 has 0 variants, skipping\n",
      "[INFO]\tGene EMSY has 0 variants, skipping\n",
      "[INFO]\tGene TMEM62 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LPGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WWTR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CELF1 has 0 variants, skipping\n",
      "[INFO]\tGene RFX3 has 0 variants, skipping\n",
      "[INFO]\tGene WDR49 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4252 has 0 variants, skipping\n",
      "[INFO]\tGene BARX2 has 0 variants, skipping\n",
      "[INFO]\tGene ASL has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AA1 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNCRIP has 0 variants, skipping\n",
      "[INFO]\tGene SH3YL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC5 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4 has 0 variants, skipping\n",
      "[INFO]\tGene RGMA has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK1 has 0 variants, skipping\n",
      "[INFO]\tGene PI15 has 0 variants, skipping\n",
      "[INFO]\tGene PCF11 has 0 variants, skipping\n",
      "[INFO]\tGene MBTPS2 has 0 variants, skipping\n",
      "[INFO]\tGene DKK3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RPN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4306 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3196 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3194 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPG11 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB4B has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2A has 0 variants, skipping\n",
      "[INFO]\tGene TUBB4A has 0 variants, skipping\n",
      "[INFO]\tGene SPRR1A has 0 variants, skipping\n",
      "[INFO]\tGene ERMARD has 0 variants, skipping\n",
      "[INFO]\tGene ALG13 has 0 variants, skipping\n",
      "[INFO]\tGene DLC1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF24 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH3 has 0 variants, skipping\n",
      "[INFO]\tGene SGSM1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF12 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4 has 0 variants, skipping\n",
      "[INFO]\tGene GAD2 has 0 variants, skipping\n",
      "[INFO]\tGene VOPP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA7 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31C1 has 0 variants, skipping\n",
      "[INFO]\tGene USHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLMAP has 0 variants, skipping\n",
      "[INFO]\tGene SIDT1 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7C has 0 variants, skipping\n",
      "[INFO]\tGene ASB8 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF391 has 0 variants, skipping\n",
      "[INFO]\tGene NTPCR has 0 variants, skipping\n",
      "[INFO]\tGene IQCD has 0 variants, skipping\n",
      "[INFO]\tGene ETS1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1AR has 0 variants, skipping\n",
      "[INFO]\tGene SAMD4B has 0 variants, skipping\n",
      "[INFO]\tGene GULP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM205 has 0 variants, skipping\n",
      "[INFO]\tGene PAX5 has 0 variants, skipping\n",
      "[INFO]\tGene MCTS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLG3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC148 has 0 variants, skipping\n",
      "[INFO]\tGene RERE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD128 has 0 variants, skipping\n",
      "[INFO]\tGene CLPSL2 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD20 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GDS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6728 has 0 variants, skipping\n",
      "[INFO]\tGene ENO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM248 has 0 variants, skipping\n",
      "[INFO]\tGene GSPT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR74 has 0 variants, skipping\n",
      "[INFO]\tGene MEAF6 has 0 variants, skipping\n",
      "[INFO]\tGene ATG13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR34AHG has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02606 has 0 variants, skipping\n",
      "[INFO]\tGene SYK has 0 variants, skipping\n",
      "[INFO]\tGene KNOP1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP92 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CD1E has 0 variants, skipping\n",
      "[INFO]\tGene NADK has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1B-RDH14 has 0 variants, skipping\n",
      "[INFO]\tGene KRTDAP has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD8 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR2 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM114A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5697 has 0 variants, skipping\n",
      "[INFO]\tGene NSD2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG3 has 0 variants, skipping\n",
      "[INFO]\tGene DEDD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF10 has 0 variants, skipping\n",
      "[INFO]\tGene FCAR has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-9 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-7 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-8 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6-2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP3 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC7 has 0 variants, skipping\n",
      "[INFO]\tGene USP33 has 0 variants, skipping\n",
      "[INFO]\tGene NTNG1 has 0 variants, skipping\n",
      "[INFO]\tGene LMTK2 has 0 variants, skipping\n",
      "[INFO]\tGene PRLH has 0 variants, skipping\n",
      "[INFO]\tGene UBR5 has 0 variants, skipping\n",
      "[INFO]\tGene ABCD2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL13 has 0 variants, skipping\n",
      "[INFO]\tGene APLP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOC has 0 variants, skipping\n",
      "[INFO]\tGene CCNJ has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A2 has 0 variants, skipping\n",
      "[INFO]\tGene RABAC1 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP3 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene DHX16 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2D has 0 variants, skipping\n",
      "[INFO]\tGene MTOR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B4 has 0 variants, skipping\n",
      "[INFO]\tGene VRK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01647 has 0 variants, skipping\n",
      "[INFO]\tGene FANCC has 0 variants, skipping\n",
      "[INFO]\tGene RGS10 has 0 variants, skipping\n",
      "[INFO]\tGene ITSN1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4F has 0 variants, skipping\n",
      "[INFO]\tGene FBXO44 has 0 variants, skipping\n",
      "[INFO]\tGene VIPR2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D14 has 0 variants, skipping\n",
      "[INFO]\tGene NUFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene HACE1 has 0 variants, skipping\n",
      "[INFO]\tGene CGN has 0 variants, skipping\n",
      "[INFO]\tGene GHR has 0 variants, skipping\n",
      "[INFO]\tGene MBNL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN6 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP44 has 0 variants, skipping\n",
      "[INFO]\tGene NPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP2 has 0 variants, skipping\n",
      "[INFO]\tGene FER has 0 variants, skipping\n",
      "[INFO]\tGene RNU5E-1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PF has 0 variants, skipping\n",
      "[INFO]\tGene AP3M1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP63 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GAP has 0 variants, skipping\n",
      "[INFO]\tGene MIR6729 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ISY1-RAB43 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7846 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC163 has 0 variants, skipping\n",
      "[INFO]\tGene CSAG1 has 0 variants, skipping\n",
      "[INFO]\tGene MLC1 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR2 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2A has 0 variants, skipping\n",
      "[INFO]\tGene USP17L28 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf109 has 0 variants, skipping\n",
      "[INFO]\tGene PRG2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA59B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA59A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927572 has 0 variants, skipping\n",
      "[INFO]\tGene SNX19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6730 has 0 variants, skipping\n",
      "[INFO]\tGene FAM120B has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01784 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB8 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB9 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM234 has 0 variants, skipping\n",
      "[INFO]\tGene LARS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX30 has 0 variants, skipping\n",
      "[INFO]\tGene FURIN has 0 variants, skipping\n",
      "[INFO]\tGene LZTS2 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN10 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF36P has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF34P has 0 variants, skipping\n",
      "[INFO]\tGene COLGALT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8 has 0 variants, skipping\n",
      "[INFO]\tGene RNF8 has 0 variants, skipping\n",
      "[INFO]\tGene CAPS has 0 variants, skipping\n",
      "[INFO]\tGene BRSK2 has 0 variants, skipping\n",
      "[INFO]\tGene KAZN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1D has 0 variants, skipping\n",
      "[INFO]\tGene DLG2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM51-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB4 has 0 variants, skipping\n",
      "[INFO]\tGene PAK3 has 0 variants, skipping\n",
      "[INFO]\tGene PAK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR142 has 0 variants, skipping\n",
      "[INFO]\tGene MIR141 has 0 variants, skipping\n",
      "[INFO]\tGene MIR140 has 0 variants, skipping\n",
      "[INFO]\tGene MIR138-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR137 has 0 variants, skipping\n",
      "[INFO]\tGene MIR135A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR135A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR130A has 0 variants, skipping\n",
      "[INFO]\tGene MIR128-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR323B has 0 variants, skipping\n",
      "[INFO]\tGene TGS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA21B has 0 variants, skipping\n",
      "[INFO]\tGene ROR2 has 0 variants, skipping\n",
      "[INFO]\tGene NTHL1 has 0 variants, skipping\n",
      "[INFO]\tGene VTI1A has 0 variants, skipping\n",
      "[INFO]\tGene CCNA1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170 has 0 variants, skipping\n",
      "[INFO]\tGene NFIC has 0 variants, skipping\n",
      "[INFO]\tGene OSCAR has 0 variants, skipping\n",
      "[INFO]\tGene FLJ37453 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF559 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN3 has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4L2 has 0 variants, skipping\n",
      "[INFO]\tGene APTX has 0 variants, skipping\n",
      "[INFO]\tGene ACSL6 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2C has 0 variants, skipping\n",
      "[INFO]\tGene C4BPB has 0 variants, skipping\n",
      "[INFO]\tGene SORBS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1B has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C8 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY6 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH4 has 0 variants, skipping\n",
      "[INFO]\tGene KEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene TLK1 has 0 variants, skipping\n",
      "[INFO]\tGene MPDZ has 0 variants, skipping\n",
      "[INFO]\tGene SPATA21 has 0 variants, skipping\n",
      "[INFO]\tGene BARD1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC182 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf135 has 0 variants, skipping\n",
      "[INFO]\tGene KIF27 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K1 has 0 variants, skipping\n",
      "[INFO]\tGene FYB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01772 has 0 variants, skipping\n",
      "[INFO]\tGene CROCCP3 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-18 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-4 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNU1-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01783 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3675 has 0 variants, skipping\n",
      "[INFO]\tGene ST18 has 0 variants, skipping\n",
      "[INFO]\tGene CROCCP2 has 0 variants, skipping\n",
      "[INFO]\tGene MST1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ESPNP has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376805 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A11 has 0 variants, skipping\n",
      "[INFO]\tGene TAOK3 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM108 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B1 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOSIN has 0 variants, skipping\n",
      "[INFO]\tGene OXNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP3 has 0 variants, skipping\n",
      "[INFO]\tGene GSN has 0 variants, skipping\n",
      "[INFO]\tGene KLHL10 has 0 variants, skipping\n",
      "[INFO]\tGene F2R has 0 variants, skipping\n",
      "[INFO]\tGene GGCX has 0 variants, skipping\n",
      "[INFO]\tGene VKORC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD73B has 0 variants, skipping\n",
      "[INFO]\tGene GPM6A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3972 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA8 has 0 variants, skipping\n",
      "[INFO]\tGene XKRY has 0 variants, skipping\n",
      "[INFO]\tGene WRNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD2B has 0 variants, skipping\n",
      "[INFO]\tGene KDM4C has 0 variants, skipping\n",
      "[INFO]\tGene CYB561D2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D20 has 0 variants, skipping\n",
      "[INFO]\tGene MADD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUSB has 0 variants, skipping\n",
      "[INFO]\tGene CHN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM14A has 0 variants, skipping\n",
      "[INFO]\tGene MX1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A4 has 0 variants, skipping\n",
      "[INFO]\tGene DTNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01654 has 0 variants, skipping\n",
      "[INFO]\tGene PDP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIP has 0 variants, skipping\n",
      "[INFO]\tGene KLC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIK1B has 0 variants, skipping\n",
      "[INFO]\tGene LPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX19B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4695 has 0 variants, skipping\n",
      "[INFO]\tGene CES5A has 0 variants, skipping\n",
      "[INFO]\tGene TWIST2 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT3 has 0 variants, skipping\n",
      "[INFO]\tGene KLK8 has 0 variants, skipping\n",
      "[INFO]\tGene EMC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP4 has 0 variants, skipping\n",
      "[INFO]\tGene NPRL3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2F has 0 variants, skipping\n",
      "[INFO]\tGene FAM131B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP65 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7L has 0 variants, skipping\n",
      "[INFO]\tGene DCAF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506730 has 0 variants, skipping\n",
      "[INFO]\tGene PPIC has 0 variants, skipping\n",
      "[INFO]\tGene PPIB has 0 variants, skipping\n",
      "[INFO]\tGene PDE3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-4 has 0 variants, skipping\n",
      "[INFO]\tGene FZD4-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02740 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRCD has 0 variants, skipping\n",
      "[INFO]\tGene CTDP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378614 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF40B has 0 variants, skipping\n",
      "[INFO]\tGene EBF4 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPS14P3 has 0 variants, skipping\n",
      "[INFO]\tGene TJP1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA1 has 0 variants, skipping\n",
      "[INFO]\tGene PSD3 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD8 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL2 has 0 variants, skipping\n",
      "[INFO]\tGene TWF2 has 0 variants, skipping\n",
      "[INFO]\tGene MGLL has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01757 has 0 variants, skipping\n",
      "[INFO]\tGene DOHH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01141 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF40A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6084 has 0 variants, skipping\n",
      "[INFO]\tGene PINK1-AS has 0 variants, skipping\n",
      "[INFO]\tGene GABBR1 has 0 variants, skipping\n",
      "[INFO]\tGene LRBA has 0 variants, skipping\n",
      "[INFO]\tGene PCID2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LF has 0 variants, skipping\n",
      "[INFO]\tGene C18orf54 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN6 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA5 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3A has 0 variants, skipping\n",
      "[INFO]\tGene HTR4 has 0 variants, skipping\n",
      "[INFO]\tGene HTR2C has 0 variants, skipping\n",
      "[INFO]\tGene HTR2B has 0 variants, skipping\n",
      "[INFO]\tGene HTR1F has 0 variants, skipping\n",
      "[INFO]\tGene HTR1E has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A3 has 0 variants, skipping\n",
      "[INFO]\tGene STPG2 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF575 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK4 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L3 has 0 variants, skipping\n",
      "[INFO]\tGene TBXA2R has 0 variants, skipping\n",
      "[INFO]\tGene LMO3 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RB has 0 variants, skipping\n",
      "[INFO]\tGene TNXB has 0 variants, skipping\n",
      "[INFO]\tGene STK40 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF17 has 0 variants, skipping\n",
      "[INFO]\tGene PROM2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LG has 0 variants, skipping\n",
      "[INFO]\tGene CELF3 has 0 variants, skipping\n",
      "[INFO]\tGene EDN1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI27 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOP has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3B2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM162B has 0 variants, skipping\n",
      "[INFO]\tGene C10orf120 has 0 variants, skipping\n",
      "[INFO]\tGene MUC21 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10B has 0 variants, skipping\n",
      "[INFO]\tGene NME1-NME2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC53 has 0 variants, skipping\n",
      "[INFO]\tGene PDXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene GAPVD1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2C_2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A6 has 0 variants, skipping\n",
      "[INFO]\tGene RALGPS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNPK has 0 variants, skipping\n",
      "[INFO]\tGene CCDC196 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC1L has 0 variants, skipping\n",
      "[INFO]\tGene CDC14B has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R10 has 0 variants, skipping\n",
      "[INFO]\tGene SYT14 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC2 has 0 variants, skipping\n",
      "[INFO]\tGene NME9 has 0 variants, skipping\n",
      "[INFO]\tGene SYCE1L has 0 variants, skipping\n",
      "[INFO]\tGene JCAD has 0 variants, skipping\n",
      "[INFO]\tGene GSG1L has 0 variants, skipping\n",
      "[INFO]\tGene GALT has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01635 has 0 variants, skipping\n",
      "[INFO]\tGene ASNS has 0 variants, skipping\n",
      "[INFO]\tGene HP1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene FIP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4418 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF133 has 0 variants, skipping\n",
      "[INFO]\tGene CPD has 0 variants, skipping\n",
      "[INFO]\tGene OSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6127 has 0 variants, skipping\n",
      "[INFO]\tGene ALG9 has 0 variants, skipping\n",
      "[INFO]\tGene NAA60 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC11 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4684 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4253 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf58 has 0 variants, skipping\n",
      "[INFO]\tGene INVS has 0 variants, skipping\n",
      "[INFO]\tGene TP53I11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3115 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71F1 has 0 variants, skipping\n",
      "[INFO]\tGene RABL2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF436-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP2 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL2 has 0 variants, skipping\n",
      "[INFO]\tGene NFIX has 0 variants, skipping\n",
      "[INFO]\tGene KLF7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928163 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD2 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACYP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX3Y has 0 variants, skipping\n",
      "[INFO]\tGene SNCAIP has 0 variants, skipping\n",
      "[INFO]\tGene ZFP69 has 0 variants, skipping\n",
      "[INFO]\tGene GBP3 has 0 variants, skipping\n",
      "[INFO]\tGene GLYR1 has 0 variants, skipping\n",
      "[INFO]\tGene EBF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378F has 0 variants, skipping\n",
      "[INFO]\tGene AK2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF738 has 0 variants, skipping\n",
      "[INFO]\tGene STATH has 0 variants, skipping\n",
      "[INFO]\tGene PRKACB has 0 variants, skipping\n",
      "[INFO]\tGene MAP2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGXB3 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHKG1 has 0 variants, skipping\n",
      "[INFO]\tGene KNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280C has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC10 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD10 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280D has 0 variants, skipping\n",
      "[INFO]\tGene EOLA1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH23 has 0 variants, skipping\n",
      "[INFO]\tGene HCK has 0 variants, skipping\n",
      "[INFO]\tGene SPATS2L has 0 variants, skipping\n",
      "[INFO]\tGene RCAN3AS has 0 variants, skipping\n",
      "[INFO]\tGene STIM1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T5 has 0 variants, skipping\n",
      "[INFO]\tGene CD276 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02774 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC19 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3GL has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMKK1 has 0 variants, skipping\n",
      "[INFO]\tGene HAGHL has 0 variants, skipping\n",
      "[INFO]\tGene GARNL3 has 0 variants, skipping\n",
      "[INFO]\tGene HSDL2 has 0 variants, skipping\n",
      "[INFO]\tGene GON4L has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP12 has 0 variants, skipping\n",
      "[INFO]\tGene IL18 has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC2 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5F has 0 variants, skipping\n",
      "[INFO]\tGene SH3GLB1 has 0 variants, skipping\n",
      "[INFO]\tGene KCND1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIOBP has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A3 has 0 variants, skipping\n",
      "[INFO]\tGene HSF2BP has 0 variants, skipping\n",
      "[INFO]\tGene TPPP has 0 variants, skipping\n",
      "[INFO]\tGene SKIV2L has 0 variants, skipping\n",
      "[INFO]\tGene RIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF254 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A2 has 0 variants, skipping\n",
      "[INFO]\tGene POLDIP3 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD3 has 0 variants, skipping\n",
      "[INFO]\tGene RANGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB48 has 0 variants, skipping\n",
      "[INFO]\tGene EDIL3 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP1 has 0 variants, skipping\n",
      "[INFO]\tGene AHSG has 0 variants, skipping\n",
      "[INFO]\tGene DAZAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL19 has 0 variants, skipping\n",
      "[INFO]\tGene DHCR24 has 0 variants, skipping\n",
      "[INFO]\tGene IST1 has 0 variants, skipping\n",
      "[INFO]\tGene TATDN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A46 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL3 has 0 variants, skipping\n",
      "[INFO]\tGene UTY has 0 variants, skipping\n",
      "[INFO]\tGene ADAMDEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR125A has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1A has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00173 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD54 has 0 variants, skipping\n",
      "[INFO]\tGene MIR17HG has 0 variants, skipping\n",
      "[INFO]\tGene AKAP17A has 0 variants, skipping\n",
      "[INFO]\tGene CD8A has 0 variants, skipping\n",
      "[INFO]\tGene TATDN1 has 0 variants, skipping\n",
      "[INFO]\tGene BACH1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF6 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC20A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6731 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4425 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRB has 0 variants, skipping\n",
      "[INFO]\tGene NUP54 has 0 variants, skipping\n",
      "[INFO]\tGene NKIRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF707 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61G-DT has 0 variants, skipping\n",
      "[INFO]\tGene OLFM3 has 0 variants, skipping\n",
      "[INFO]\tGene TLX3 has 0 variants, skipping\n",
      "[INFO]\tGene CACHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB26 has 0 variants, skipping\n",
      "[INFO]\tGene ZDBF2 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR3 has 0 variants, skipping\n",
      "[INFO]\tGene WNK3 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39A has 0 variants, skipping\n",
      "[INFO]\tGene WDR38 has 0 variants, skipping\n",
      "[INFO]\tGene USF1 has 0 variants, skipping\n",
      "[INFO]\tGene TPCN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646471 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf90 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2C has 0 variants, skipping\n",
      "[INFO]\tGene RNASE4 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HC1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SRI has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3917 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB6 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5D has 0 variants, skipping\n",
      "[INFO]\tGene HPCAL4 has 0 variants, skipping\n",
      "[INFO]\tGene NRBF2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC33A1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A7 has 0 variants, skipping\n",
      "[INFO]\tGene ETF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692A1 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK6 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL5 has 0 variants, skipping\n",
      "[INFO]\tGene IQCA1 has 0 variants, skipping\n",
      "[INFO]\tGene IL20RA has 0 variants, skipping\n",
      "[INFO]\tGene PALD1 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEB has 0 variants, skipping\n",
      "[INFO]\tGene LRFN2 has 0 variants, skipping\n",
      "[INFO]\tGene RIMKLB has 0 variants, skipping\n",
      "[INFO]\tGene HEG1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB6 has 0 variants, skipping\n",
      "[INFO]\tGene NUMB has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC17 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150C has 0 variants, skipping\n",
      "[INFO]\tGene LIN54 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928728 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf94 has 0 variants, skipping\n",
      "[INFO]\tGene DGKB has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNPH1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1586 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF14 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4C has 0 variants, skipping\n",
      "[INFO]\tGene FGGY has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8K has 0 variants, skipping\n",
      "[INFO]\tGene AGER has 0 variants, skipping\n",
      "[INFO]\tGene ARL3 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P20 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM262 has 0 variants, skipping\n",
      "[INFO]\tGene PIF1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene RXRA has 0 variants, skipping\n",
      "[INFO]\tGene ORC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANO10 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1L has 0 variants, skipping\n",
      "[INFO]\tGene CARD19 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TF has 0 variants, skipping\n",
      "[INFO]\tGene LINC02574 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF140 has 0 variants, skipping\n",
      "[INFO]\tGene AAMDC has 0 variants, skipping\n",
      "[INFO]\tGene GPALPP1 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PDHX has 0 variants, skipping\n",
      "[INFO]\tGene TLL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA1 has 0 variants, skipping\n",
      "[INFO]\tGene GATAD2A has 0 variants, skipping\n",
      "[INFO]\tGene OAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NTM has 0 variants, skipping\n",
      "[INFO]\tGene LILRA1 has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC5OS has 0 variants, skipping\n",
      "[INFO]\tGene DICER1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC20 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC51 has 0 variants, skipping\n",
      "[INFO]\tGene USP15 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18C has 0 variants, skipping\n",
      "[INFO]\tGene CETN3 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0513 has 0 variants, skipping\n",
      "[INFO]\tGene COX4I1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIE has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC8 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf112 has 0 variants, skipping\n",
      "[INFO]\tGene CBLB has 0 variants, skipping\n",
      "[INFO]\tGene RAD51B has 0 variants, skipping\n",
      "[INFO]\tGene KCNS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF5 has 0 variants, skipping\n",
      "[INFO]\tGene DPP3 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf52 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1328 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA73A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA73B has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MF-PTCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRNAU1AP has 0 variants, skipping\n",
      "[INFO]\tGene OR4F6 has 0 variants, skipping\n",
      "[INFO]\tGene DLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1468 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP7P has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA61 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA16A has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL6 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL4 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01715 has 0 variants, skipping\n",
      "[INFO]\tGene TMC6 has 0 variants, skipping\n",
      "[INFO]\tGene LAYN has 0 variants, skipping\n",
      "[INFO]\tGene PTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene KAT6A has 0 variants, skipping\n",
      "[INFO]\tGene DEGS1 has 0 variants, skipping\n",
      "[INFO]\tGene TOGARAM2 has 0 variants, skipping\n",
      "[INFO]\tGene COL3A1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP26 has 0 variants, skipping\n",
      "[INFO]\tGene SNCB has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM163A has 0 variants, skipping\n",
      "[INFO]\tGene FBXW11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP23 has 0 variants, skipping\n",
      "[INFO]\tGene SOX5 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL1 has 0 variants, skipping\n",
      "[INFO]\tGene CUL4B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01756 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01648 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01778 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K7CL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385A has 0 variants, skipping\n",
      "[INFO]\tGene SLCO6A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD103C has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE16 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3K has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF9 has 0 variants, skipping\n",
      "[INFO]\tGene ENOX1 has 0 variants, skipping\n",
      "[INFO]\tGene AR has 0 variants, skipping\n",
      "[INFO]\tGene CXCR4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3D has 0 variants, skipping\n",
      "[INFO]\tGene EIF3C has 0 variants, skipping\n",
      "[INFO]\tGene KATNA1 has 0 variants, skipping\n",
      "[INFO]\tGene KRR1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX12 has 0 variants, skipping\n",
      "[INFO]\tGene SOX15 has 0 variants, skipping\n",
      "[INFO]\tGene SOX10 has 0 variants, skipping\n",
      "[INFO]\tGene ATE1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R5 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM160B2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01226 has 0 variants, skipping\n",
      "[INFO]\tGene RPP21 has 0 variants, skipping\n",
      "[INFO]\tGene MCM7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS11 has 0 variants, skipping\n",
      "[INFO]\tGene WDR97 has 0 variants, skipping\n",
      "[INFO]\tGene PDPR has 0 variants, skipping\n",
      "[INFO]\tGene SDHC has 0 variants, skipping\n",
      "[INFO]\tGene SCIN has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4254 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT22 has 0 variants, skipping\n",
      "[INFO]\tGene HABP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A6 has 0 variants, skipping\n",
      "[INFO]\tGene NRM has 0 variants, skipping\n",
      "[INFO]\tGene MIR5585 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP1 has 0 variants, skipping\n",
      "[INFO]\tGene KNG1 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM2 has 0 variants, skipping\n",
      "[INFO]\tGene COX11 has 0 variants, skipping\n",
      "[INFO]\tGene TAMM41 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL8 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR9LP has 0 variants, skipping\n",
      "[INFO]\tGene C4A has 0 variants, skipping\n",
      "[INFO]\tGene USP44 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DOB has 0 variants, skipping\n",
      "[INFO]\tGene ZYX has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK1 has 0 variants, skipping\n",
      "[INFO]\tGene ST7L has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4F has 0 variants, skipping\n",
      "[INFO]\tGene PHF7 has 0 variants, skipping\n",
      "[INFO]\tGene RCVRN has 0 variants, skipping\n",
      "[INFO]\tGene SDHD has 0 variants, skipping\n",
      "[INFO]\tGene RCN2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS23 has 0 variants, skipping\n",
      "[INFO]\tGene CHST11 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH13 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEC has 0 variants, skipping\n",
      "[INFO]\tGene RAB5IF has 0 variants, skipping\n",
      "[INFO]\tGene CLIP3 has 0 variants, skipping\n",
      "[INFO]\tGene BVES has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8OS has 0 variants, skipping\n",
      "[INFO]\tGene MPG has 0 variants, skipping\n",
      "[INFO]\tGene ACOT9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCZ has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK11 has 0 variants, skipping\n",
      "[INFO]\tGene S100PBP has 0 variants, skipping\n",
      "[INFO]\tGene MAFA has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6L2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A3 has 0 variants, skipping\n",
      "[INFO]\tGene CAPS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL24A1 has 0 variants, skipping\n",
      "[INFO]\tGene VEZT has 0 variants, skipping\n",
      "[INFO]\tGene CP has 0 variants, skipping\n",
      "[INFO]\tGene NUP93 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP3B has 0 variants, skipping\n",
      "[INFO]\tGene NRCAM has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1 has 0 variants, skipping\n",
      "[INFO]\tGene EVL has 0 variants, skipping\n",
      "[INFO]\tGene PRODH has 0 variants, skipping\n",
      "[INFO]\tGene NAT9 has 0 variants, skipping\n",
      "[INFO]\tGene XYLB has 0 variants, skipping\n",
      "[INFO]\tGene TMEM70 has 0 variants, skipping\n",
      "[INFO]\tGene SCARB1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH1 has 0 variants, skipping\n",
      "[INFO]\tGene DCX has 0 variants, skipping\n",
      "[INFO]\tGene VIT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3605 has 0 variants, skipping\n",
      "[INFO]\tGene MED23 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26C has 0 variants, skipping\n",
      "[INFO]\tGene MT1X has 0 variants, skipping\n",
      "[INFO]\tGene APBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MT1H has 0 variants, skipping\n",
      "[INFO]\tGene MT1F has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC5 has 0 variants, skipping\n",
      "[INFO]\tGene MT1G has 0 variants, skipping\n",
      "[INFO]\tGene UBE4B has 0 variants, skipping\n",
      "[INFO]\tGene CSMD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L8 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF8 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOU has 0 variants, skipping\n",
      "[INFO]\tGene CYRIB has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653160 has 0 variants, skipping\n",
      "[INFO]\tGene HFE has 0 variants, skipping\n",
      "[INFO]\tGene PAM has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC3 has 0 variants, skipping\n",
      "[INFO]\tGene PHKB has 0 variants, skipping\n",
      "[INFO]\tGene COL9A3 has 0 variants, skipping\n",
      "[INFO]\tGene COL9A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN20 has 0 variants, skipping\n",
      "[INFO]\tGene HAGH has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLC has 0 variants, skipping\n",
      "[INFO]\tGene FTO has 0 variants, skipping\n",
      "[INFO]\tGene IRF8 has 0 variants, skipping\n",
      "[INFO]\tGene CD53 has 0 variants, skipping\n",
      "[INFO]\tGene AQP9 has 0 variants, skipping\n",
      "[INFO]\tGene BIN1 has 0 variants, skipping\n",
      "[INFO]\tGene STRBP has 0 variants, skipping\n",
      "[INFO]\tGene DISC2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A8 has 0 variants, skipping\n",
      "[INFO]\tGene EPS15 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF12 has 0 variants, skipping\n",
      "[INFO]\tGene MSRA has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A4 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF7 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L3 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM126B has 0 variants, skipping\n",
      "[INFO]\tGene IKZF1 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE2 has 0 variants, skipping\n",
      "[INFO]\tGene GDAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY10 has 0 variants, skipping\n",
      "[INFO]\tGene APP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63C has 0 variants, skipping\n",
      "[INFO]\tGene SZT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC21 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4255 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01137 has 0 variants, skipping\n",
      "[INFO]\tGene MINK1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene SDCCAG8 has 0 variants, skipping\n",
      "[INFO]\tGene TLE1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6732 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB109B has 0 variants, skipping\n",
      "[INFO]\tGene CAST has 0 variants, skipping\n",
      "[INFO]\tGene PWWP3B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF736 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL10 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN5 has 0 variants, skipping\n",
      "[INFO]\tGene VMP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCALD has 0 variants, skipping\n",
      "[INFO]\tGene FLNC has 0 variants, skipping\n",
      "[INFO]\tGene CXorf56 has 0 variants, skipping\n",
      "[INFO]\tGene DNMBP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF346 has 0 variants, skipping\n",
      "[INFO]\tGene DUOX2 has 0 variants, skipping\n",
      "[INFO]\tGene ANO4 has 0 variants, skipping\n",
      "[INFO]\tGene CSF3 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH16A1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1A has 0 variants, skipping\n",
      "[INFO]\tGene TTC26 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAIR1 has 0 variants, skipping\n",
      "[INFO]\tGene THPO has 0 variants, skipping\n",
      "[INFO]\tGene ETNK1 has 0 variants, skipping\n",
      "[INFO]\tGene PARL has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A10 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA4 has 0 variants, skipping\n",
      "[INFO]\tGene CSGALNACT2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2N1 has 0 variants, skipping\n",
      "[INFO]\tGene STRADB has 0 variants, skipping\n",
      "[INFO]\tGene SERINC2 has 0 variants, skipping\n",
      "[INFO]\tGene USP49 has 0 variants, skipping\n",
      "[INFO]\tGene CYYR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL4 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PAK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01343 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01685 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2BP-COX16 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP14 has 0 variants, skipping\n",
      "[INFO]\tGene GJA9-MYCBP has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378663 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC9 has 0 variants, skipping\n",
      "[INFO]\tGene SEC31A has 0 variants, skipping\n",
      "[INFO]\tGene SLU7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM120C has 0 variants, skipping\n",
      "[INFO]\tGene BEST1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR59 has 0 variants, skipping\n",
      "[INFO]\tGene ZCWPW2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT2P1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIEL has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA55 has 0 variants, skipping\n",
      "[INFO]\tGene IDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ID3 has 0 variants, skipping\n",
      "[INFO]\tGene ABI2 has 0 variants, skipping\n",
      "[INFO]\tGene LST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF197 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929536 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX1 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC9 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC115995503 has 0 variants, skipping\n",
      "[INFO]\tGene TRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS3 has 0 variants, skipping\n",
      "[INFO]\tGene FREM1 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PET100 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111B has 0 variants, skipping\n",
      "[INFO]\tGene TBPL2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN6 has 0 variants, skipping\n",
      "[INFO]\tGene TEPP has 0 variants, skipping\n",
      "[INFO]\tGene NSUN4 has 0 variants, skipping\n",
      "[INFO]\tGene OR7C1 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1B has 0 variants, skipping\n",
      "[INFO]\tGene SS18L1 has 0 variants, skipping\n",
      "[INFO]\tGene PYHIN1 has 0 variants, skipping\n",
      "[INFO]\tGene INKA2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4B has 0 variants, skipping\n",
      "[INFO]\tGene KRT72 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2026 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF7 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA3 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP18 has 0 variants, skipping\n",
      "[INFO]\tGene SRR has 0 variants, skipping\n",
      "[INFO]\tGene SENP6 has 0 variants, skipping\n",
      "[INFO]\tGene FLCN has 0 variants, skipping\n",
      "[INFO]\tGene TRIM16L has 0 variants, skipping\n",
      "[INFO]\tGene DYM has 0 variants, skipping\n",
      "[INFO]\tGene NFYC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A12 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1671 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A6 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0895 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF761 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF91 has 0 variants, skipping\n",
      "[INFO]\tGene PIGF has 0 variants, skipping\n",
      "[INFO]\tGene PIGC has 0 variants, skipping\n",
      "[INFO]\tGene PIGA has 0 variants, skipping\n",
      "[INFO]\tGene TSTD1 has 0 variants, skipping\n",
      "[INFO]\tGene USP54 has 0 variants, skipping\n",
      "[INFO]\tGene DACT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLFNL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SCMH1 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf48 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG6 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE9 has 0 variants, skipping\n",
      "[INFO]\tGene C1QC has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD6 has 0 variants, skipping\n",
      "[INFO]\tGene JADE1 has 0 variants, skipping\n",
      "[INFO]\tGene DGUOK has 0 variants, skipping\n",
      "[INFO]\tGene PUS7L has 0 variants, skipping\n",
      "[INFO]\tGene KDM5C has 0 variants, skipping\n",
      "[INFO]\tGene PSME3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC11 has 0 variants, skipping\n",
      "[INFO]\tGene SPART has 0 variants, skipping\n",
      "[INFO]\tGene ZEB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC13 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH5 has 0 variants, skipping\n",
      "[INFO]\tGene KLF11 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPA has 0 variants, skipping\n",
      "[INFO]\tGene CBX4 has 0 variants, skipping\n",
      "[INFO]\tGene CHST1 has 0 variants, skipping\n",
      "[INFO]\tGene AP5S1 has 0 variants, skipping\n",
      "[INFO]\tGene KRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA10 has 0 variants, skipping\n",
      "[INFO]\tGene MRGBP has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3L has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1B has 0 variants, skipping\n",
      "[INFO]\tGene ANO5 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB6 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP30 has 0 variants, skipping\n",
      "[INFO]\tGene IL36B has 0 variants, skipping\n",
      "[INFO]\tGene SLC50A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339539 has 0 variants, skipping\n",
      "[INFO]\tGene MTG2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK33 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6733 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2A has 0 variants, skipping\n",
      "[INFO]\tGene CFAP410 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R2 has 0 variants, skipping\n",
      "[INFO]\tGene PITX1 has 0 variants, skipping\n",
      "[INFO]\tGene PIP has 0 variants, skipping\n",
      "[INFO]\tGene PI4KB has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CG has 0 variants, skipping\n",
      "[INFO]\tGene PIM1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C3 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2B has 0 variants, skipping\n",
      "[INFO]\tGene CCL3L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6734 has 0 variants, skipping\n",
      "[INFO]\tGene ERO1A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF14 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1A1 has 0 variants, skipping\n",
      "[INFO]\tGene ITM2C has 0 variants, skipping\n",
      "[INFO]\tGene COMMD5 has 0 variants, skipping\n",
      "[INFO]\tGene SZT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6735 has 0 variants, skipping\n",
      "[INFO]\tGene RPL14 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D2A has 0 variants, skipping\n",
      "[INFO]\tGene CENPQ has 0 variants, skipping\n",
      "[INFO]\tGene CEP55 has 0 variants, skipping\n",
      "[INFO]\tGene SHQ1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPO has 0 variants, skipping\n",
      "[INFO]\tGene TSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAC2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM33 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF10L has 0 variants, skipping\n",
      "[INFO]\tGene RFWD3 has 0 variants, skipping\n",
      "[INFO]\tGene DARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLF2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3E has 0 variants, skipping\n",
      "[INFO]\tGene KDM4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM44 has 0 variants, skipping\n",
      "[INFO]\tGene ODF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF213 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM3 has 0 variants, skipping\n",
      "[INFO]\tGene CTSC has 0 variants, skipping\n",
      "[INFO]\tGene CENPE has 0 variants, skipping\n",
      "[INFO]\tGene LKAAEAR1 has 0 variants, skipping\n",
      "[INFO]\tGene FSD2 has 0 variants, skipping\n",
      "[INFO]\tGene SCP2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2 has 0 variants, skipping\n",
      "[INFO]\tGene SALL2 has 0 variants, skipping\n",
      "[INFO]\tGene HHEX has 0 variants, skipping\n",
      "[INFO]\tGene PRKACA has 0 variants, skipping\n",
      "[INFO]\tGene DSC1 has 0 variants, skipping\n",
      "[INFO]\tGene PGS1 has 0 variants, skipping\n",
      "[INFO]\tGene GRM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A10 has 0 variants, skipping\n",
      "[INFO]\tGene RHNO1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR9 has 0 variants, skipping\n",
      "[INFO]\tGene ANK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6079 has 0 variants, skipping\n",
      "[INFO]\tGene SNX30 has 0 variants, skipping\n",
      "[INFO]\tGene CEP83 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL2 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC9 has 0 variants, skipping\n",
      "[INFO]\tGene QRICH1 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF112 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL2 has 0 variants, skipping\n",
      "[INFO]\tGene RNFT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX22 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC2C has 0 variants, skipping\n",
      "[INFO]\tGene PUM1 has 0 variants, skipping\n",
      "[INFO]\tGene CROCC has 0 variants, skipping\n",
      "[INFO]\tGene ERI3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA110 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF4L has 0 variants, skipping\n",
      "[INFO]\tGene RAD51D has 0 variants, skipping\n",
      "[INFO]\tGene RAB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5584 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD145 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5F-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5D-1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC6 has 0 variants, skipping\n",
      "[INFO]\tGene TMED7-TICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L10 has 0 variants, skipping\n",
      "[INFO]\tGene USP19 has 0 variants, skipping\n",
      "[INFO]\tGene MLST8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD160 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD55 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38B has 0 variants, skipping\n",
      "[INFO]\tGene KAT6B has 0 variants, skipping\n",
      "[INFO]\tGene RPA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM107 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E3 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD5 has 0 variants, skipping\n",
      "[INFO]\tGene DHX8 has 0 variants, skipping\n",
      "[INFO]\tGene MFGE8 has 0 variants, skipping\n",
      "[INFO]\tGene SDR16C5 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB4 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01144 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7 has 0 variants, skipping\n",
      "[INFO]\tGene RASAL1 has 0 variants, skipping\n",
      "[INFO]\tGene SHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR542 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1224 has 0 variants, skipping\n",
      "[INFO]\tGene MIR421 has 0 variants, skipping\n",
      "[INFO]\tGene MIR671 has 0 variants, skipping\n",
      "[INFO]\tGene MIR658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR657 has 0 variants, skipping\n",
      "[INFO]\tGene MIR411 has 0 variants, skipping\n",
      "[INFO]\tGene MIR654 has 0 variants, skipping\n",
      "[INFO]\tGene MIR653 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449B has 0 variants, skipping\n",
      "[INFO]\tGene MIR663A has 0 variants, skipping\n",
      "[INFO]\tGene MIR548D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR661 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR647 has 0 variants, skipping\n",
      "[INFO]\tGene MIR646 has 0 variants, skipping\n",
      "[INFO]\tGene LEF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR637 has 0 variants, skipping\n",
      "[INFO]\tGene MIR634 has 0 variants, skipping\n",
      "[INFO]\tGene MIR632 has 0 variants, skipping\n",
      "[INFO]\tGene MIR33B has 0 variants, skipping\n",
      "[INFO]\tGene MIR630 has 0 variants, skipping\n",
      "[INFO]\tGene MIR627 has 0 variants, skipping\n",
      "[INFO]\tGene MIR626 has 0 variants, skipping\n",
      "[INFO]\tGene MIR625 has 0 variants, skipping\n",
      "[INFO]\tGene MIR624 has 0 variants, skipping\n",
      "[INFO]\tGene MIR619 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548C has 0 variants, skipping\n",
      "[INFO]\tGene MIR616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR612 has 0 variants, skipping\n",
      "[INFO]\tGene MIR610 has 0 variants, skipping\n",
      "[INFO]\tGene MIR605 has 0 variants, skipping\n",
      "[INFO]\tGene MIR602 has 0 variants, skipping\n",
      "[INFO]\tGene MIR601 has 0 variants, skipping\n",
      "[INFO]\tGene MIR598 has 0 variants, skipping\n",
      "[INFO]\tGene LARGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6839 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6836 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6775 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB3 has 0 variants, skipping\n",
      "[INFO]\tGene TPM3 has 0 variants, skipping\n",
      "[INFO]\tGene MDFIC2 has 0 variants, skipping\n",
      "[INFO]\tGene SPEM3 has 0 variants, skipping\n",
      "[INFO]\tGene CMTR2 has 0 variants, skipping\n",
      "[INFO]\tGene LHX2 has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A6 has 0 variants, skipping\n",
      "[INFO]\tGene UBE4A has 0 variants, skipping\n",
      "[INFO]\tGene SLIT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15AP10 has 0 variants, skipping\n",
      "[INFO]\tGene SSC5D has 0 variants, skipping\n",
      "[INFO]\tGene MKLN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPDPFL has 0 variants, skipping\n",
      "[INFO]\tGene PHF14 has 0 variants, skipping\n",
      "[INFO]\tGene MYO6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF429 has 0 variants, skipping\n",
      "[INFO]\tGene MTSS1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929626 has 0 variants, skipping\n",
      "[INFO]\tGene AEBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD6 has 0 variants, skipping\n",
      "[INFO]\tGene SERHL2 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG5 has 0 variants, skipping\n",
      "[INFO]\tGene KYNU has 0 variants, skipping\n",
      "[INFO]\tGene SGO1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3D has 0 variants, skipping\n",
      "[INFO]\tGene AIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01398 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCLAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02719 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02760 has 0 variants, skipping\n",
      "[INFO]\tGene LDAH has 0 variants, skipping\n",
      "[INFO]\tGene MSTO1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2-SMG1P6 has 0 variants, skipping\n",
      "[INFO]\tGene DERL1 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCM has 0 variants, skipping\n",
      "[INFO]\tGene CCNC has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2B has 0 variants, skipping\n",
      "[INFO]\tGene SCML4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM249 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP has 0 variants, skipping\n",
      "[INFO]\tGene DTX2 has 0 variants, skipping\n",
      "[INFO]\tGene GNL1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG5 has 0 variants, skipping\n",
      "[INFO]\tGene GNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00853 has 0 variants, skipping\n",
      "[INFO]\tGene FUOM has 0 variants, skipping\n",
      "[INFO]\tGene HLA-F has 0 variants, skipping\n",
      "[INFO]\tGene CCDC77 has 0 variants, skipping\n",
      "[INFO]\tGene STARD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF573 has 0 variants, skipping\n",
      "[INFO]\tGene GGA1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MF has 0 variants, skipping\n",
      "[INFO]\tGene RREB1 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02265 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01205 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01389 has 0 variants, skipping\n",
      "[INFO]\tGene SKINT1L has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG1 has 0 variants, skipping\n",
      "[INFO]\tGene IHO1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf71 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6L has 0 variants, skipping\n",
      "[INFO]\tGene FAM135A has 0 variants, skipping\n",
      "[INFO]\tGene CFLAR has 0 variants, skipping\n",
      "[INFO]\tGene TIA1 has 0 variants, skipping\n",
      "[INFO]\tGene HCN3 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0C has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B1 has 0 variants, skipping\n",
      "[INFO]\tGene GIGYF1 has 0 variants, skipping\n",
      "[INFO]\tGene CSE1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF518B has 0 variants, skipping\n",
      "[INFO]\tGene CLASP2 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6500 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01562 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3IP has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM18 has 0 variants, skipping\n",
      "[INFO]\tGene ANKAR has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8G has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL3 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP5 has 0 variants, skipping\n",
      "[INFO]\tGene RFC3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC5 has 0 variants, skipping\n",
      "[INFO]\tGene FARSB has 0 variants, skipping\n",
      "[INFO]\tGene ABHD10 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18A has 0 variants, skipping\n",
      "[INFO]\tGene FSD1L has 0 variants, skipping\n",
      "[INFO]\tGene SGMS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM6 has 0 variants, skipping\n",
      "[INFO]\tGene TATDN3 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCR10 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1A has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LMBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTROB has 0 variants, skipping\n",
      "[INFO]\tGene HOXA11-AS has 0 variants, skipping\n",
      "[INFO]\tGene CYP4Z2P has 0 variants, skipping\n",
      "[INFO]\tGene MFSD11 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene P3H1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP1 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD1C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF567 has 0 variants, skipping\n",
      "[INFO]\tGene NOSIP has 0 variants, skipping\n",
      "[INFO]\tGene MYL5 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR2 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD6-FXYD2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7B has 0 variants, skipping\n",
      "[INFO]\tGene PID1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB1 has 0 variants, skipping\n",
      "[INFO]\tGene GIT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD1 has 0 variants, skipping\n",
      "[INFO]\tGene VDAC2 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD2 has 0 variants, skipping\n",
      "[INFO]\tGene MRE11 has 0 variants, skipping\n",
      "[INFO]\tGene TPM1 has 0 variants, skipping\n",
      "[INFO]\tGene AICDA has 0 variants, skipping\n",
      "[INFO]\tGene MGRN1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS1A has 0 variants, skipping\n",
      "[INFO]\tGene FLAD1 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIB1 has 0 variants, skipping\n",
      "[INFO]\tGene MCTP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND11 has 0 variants, skipping\n",
      "[INFO]\tGene GJB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAT has 0 variants, skipping\n",
      "[INFO]\tGene SHISA2 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF9B has 0 variants, skipping\n",
      "[INFO]\tGene ETV2 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DS1 has 0 variants, skipping\n",
      "[INFO]\tGene NACA has 0 variants, skipping\n",
      "[INFO]\tGene LIF has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9 has 0 variants, skipping\n",
      "[INFO]\tGene LCP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIG4 has 0 variants, skipping\n",
      "[INFO]\tGene LIG3 has 0 variants, skipping\n",
      "[INFO]\tGene LCT has 0 variants, skipping\n",
      "[INFO]\tGene LIFR has 0 variants, skipping\n",
      "[INFO]\tGene FANCI has 0 variants, skipping\n",
      "[INFO]\tGene CCSER2 has 0 variants, skipping\n",
      "[INFO]\tGene SARAF has 0 variants, skipping\n",
      "[INFO]\tGene LPP has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507564 has 0 variants, skipping\n",
      "[INFO]\tGene BEX2 has 0 variants, skipping\n",
      "[INFO]\tGene IFI16 has 0 variants, skipping\n",
      "[INFO]\tGene PODN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01771 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A3P1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK17 has 0 variants, skipping\n",
      "[INFO]\tGene PDE8A has 0 variants, skipping\n",
      "[INFO]\tGene MS4A13 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-6 has 0 variants, skipping\n",
      "[INFO]\tGene NKD2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2F1 has 0 variants, skipping\n",
      "[INFO]\tGene RILPL1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1B has 0 variants, skipping\n",
      "[INFO]\tGene PIMREG has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA10 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA5 has 0 variants, skipping\n",
      "[INFO]\tGene NPTN-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF248 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM138 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL37 has 0 variants, skipping\n",
      "[INFO]\tGene PBDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C3A has 0 variants, skipping\n",
      "[INFO]\tGene PPHLN1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRES1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4781 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC191 has 0 variants, skipping\n",
      "[INFO]\tGene RNF7 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNC1 has 0 variants, skipping\n",
      "[INFO]\tGene GGCT has 0 variants, skipping\n",
      "[INFO]\tGene PTPN7 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S1 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL2 has 0 variants, skipping\n",
      "[INFO]\tGene TPMT has 0 variants, skipping\n",
      "[INFO]\tGene DDIT3 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN1 has 0 variants, skipping\n",
      "[INFO]\tGene MROH7 has 0 variants, skipping\n",
      "[INFO]\tGene MROH7-TTC4 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY2 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L3 has 0 variants, skipping\n",
      "[INFO]\tGene TK2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMCC1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT3 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507634 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4422HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4422 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01753 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01755 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01767 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929935 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L5 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L29 has 0 variants, skipping\n",
      "[INFO]\tGene PFKP has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERE has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A12 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB5 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM94 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL43 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01135 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01358 has 0 variants, skipping\n",
      "[INFO]\tGene HSD52 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5B has 0 variants, skipping\n",
      "[INFO]\tGene SBK3 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L4 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD14 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI2-DT has 0 variants, skipping\n",
      "[INFO]\tGene TMEM139-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RCHY1 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01614 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf136 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3H has 0 variants, skipping\n",
      "[INFO]\tGene MIR4711 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926944 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01748 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926964 has 0 variants, skipping\n",
      "[INFO]\tGene NFIA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATC1L has 0 variants, skipping\n",
      "[INFO]\tGene MGC34796 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF20B has 0 variants, skipping\n",
      "[INFO]\tGene HSPB11 has 0 variants, skipping\n",
      "[INFO]\tGene SUGP2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB3 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT2 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ANG has 0 variants, skipping\n",
      "[INFO]\tGene AMFR has 0 variants, skipping\n",
      "[INFO]\tGene MIR3116-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3116-2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP5 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS12 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1D has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K3 has 0 variants, skipping\n",
      "[INFO]\tGene RGS5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A7 has 0 variants, skipping\n",
      "[INFO]\tGene GALR3 has 0 variants, skipping\n",
      "[INFO]\tGene CILP has 0 variants, skipping\n",
      "[INFO]\tGene HGS has 0 variants, skipping\n",
      "[INFO]\tGene STXBP6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM193A has 0 variants, skipping\n",
      "[INFO]\tGene TTLL6 has 0 variants, skipping\n",
      "[INFO]\tGene KHSRP has 0 variants, skipping\n",
      "[INFO]\tGene BCAS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01739 has 0 variants, skipping\n",
      "[INFO]\tGene FAM238C has 0 variants, skipping\n",
      "[INFO]\tGene TRMT11 has 0 variants, skipping\n",
      "[INFO]\tGene TMED2-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR6068 has 0 variants, skipping\n",
      "[INFO]\tGene HOATZ has 0 variants, skipping\n",
      "[INFO]\tGene PTX4 has 0 variants, skipping\n",
      "[INFO]\tGene RPGRIP1L has 0 variants, skipping\n",
      "[INFO]\tGene ESR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR591 has 0 variants, skipping\n",
      "[INFO]\tGene MIR589 has 0 variants, skipping\n",
      "[INFO]\tGene MIR587 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU2L has 0 variants, skipping\n",
      "[INFO]\tGene DOCK8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5196 has 0 variants, skipping\n",
      "[INFO]\tGene TAPBPL has 0 variants, skipping\n",
      "[INFO]\tGene MIB2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL25 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2U has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8R has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4794 has 0 variants, skipping\n",
      "[INFO]\tGene HYDIN has 0 variants, skipping\n",
      "[INFO]\tGene ERAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GNA12 has 0 variants, skipping\n",
      "[INFO]\tGene CDC27 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF11 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01359 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3671 has 0 variants, skipping\n",
      "[INFO]\tGene COLGALT1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM37 has 0 variants, skipping\n",
      "[INFO]\tGene CADPS2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP36 has 0 variants, skipping\n",
      "[INFO]\tGene RCC1 has 0 variants, skipping\n",
      "[INFO]\tGene DPF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GCOM1 has 0 variants, skipping\n",
      "[INFO]\tGene NFKB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3117 has 0 variants, skipping\n",
      "[INFO]\tGene PTK2 has 0 variants, skipping\n",
      "[INFO]\tGene LARP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289561 has 0 variants, skipping\n",
      "[INFO]\tGene HAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGED4 has 0 variants, skipping\n",
      "[INFO]\tGene TTC29 has 0 variants, skipping\n",
      "[INFO]\tGene FZD6 has 0 variants, skipping\n",
      "[INFO]\tGene SLAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM252 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene MON2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC197 has 0 variants, skipping\n",
      "[INFO]\tGene COA1 has 0 variants, skipping\n",
      "[INFO]\tGene TIAM1 has 0 variants, skipping\n",
      "[INFO]\tGene STK32A has 0 variants, skipping\n",
      "[INFO]\tGene RANBP3L has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G1 has 0 variants, skipping\n",
      "[INFO]\tGene ASB7 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H11B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB12 has 0 variants, skipping\n",
      "[INFO]\tGene GNG12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESCO2 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2B has 0 variants, skipping\n",
      "[INFO]\tGene CT45A1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP8 has 0 variants, skipping\n",
      "[INFO]\tGene CEP164 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1262 has 0 variants, skipping\n",
      "[INFO]\tGene CTNND1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF550 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01707 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01758 has 0 variants, skipping\n",
      "[INFO]\tGene PLCD1 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAE has 0 variants, skipping\n",
      "[INFO]\tGene BCAR4 has 0 variants, skipping\n",
      "[INFO]\tGene TAZ has 0 variants, skipping\n",
      "[INFO]\tGene GALNT5 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB has 0 variants, skipping\n",
      "[INFO]\tGene PIN1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC113 has 0 variants, skipping\n",
      "[INFO]\tGene RALGDS has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF1 has 0 variants, skipping\n",
      "[INFO]\tGene COX15 has 0 variants, skipping\n",
      "[INFO]\tGene PISD has 0 variants, skipping\n",
      "[INFO]\tGene AQP7 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1C has 0 variants, skipping\n",
      "[INFO]\tGene KATNAL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01788 has 0 variants, skipping\n",
      "[INFO]\tGene EPN1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F2 has 0 variants, skipping\n",
      "[INFO]\tGene IL6ST has 0 variants, skipping\n",
      "[INFO]\tGene SUMF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD1 has 0 variants, skipping\n",
      "[INFO]\tGene NEGR1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02238 has 0 variants, skipping\n",
      "[INFO]\tGene FAM169A has 0 variants, skipping\n",
      "[INFO]\tGene ERICH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VARS2 has 0 variants, skipping\n",
      "[INFO]\tGene TYW3 has 0 variants, skipping\n",
      "[INFO]\tGene CBY1 has 0 variants, skipping\n",
      "[INFO]\tGene INSC has 0 variants, skipping\n",
      "[INFO]\tGene CFTR has 0 variants, skipping\n",
      "[INFO]\tGene SHMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHA has 0 variants, skipping\n",
      "[INFO]\tGene RRAD has 0 variants, skipping\n",
      "[INFO]\tGene RBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB27B has 0 variants, skipping\n",
      "[INFO]\tGene RAB5A has 0 variants, skipping\n",
      "[INFO]\tGene RAB1A has 0 variants, skipping\n",
      "[INFO]\tGene PSMB8 has 0 variants, skipping\n",
      "[INFO]\tGene DLSTP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45B has 0 variants, skipping\n",
      "[INFO]\tGene SNHG29 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927342 has 0 variants, skipping\n",
      "[INFO]\tGene BMF has 0 variants, skipping\n",
      "[INFO]\tGene TMEM139 has 0 variants, skipping\n",
      "[INFO]\tGene CHD9 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02567 has 0 variants, skipping\n",
      "[INFO]\tGene EFEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7156 has 0 variants, skipping\n",
      "[INFO]\tGene COL11A2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF5 has 0 variants, skipping\n",
      "[INFO]\tGene PRCP has 0 variants, skipping\n",
      "[INFO]\tGene CDC14A has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1C has 0 variants, skipping\n",
      "[INFO]\tGene CYTH1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8T has 0 variants, skipping\n",
      "[INFO]\tGene ATF7-NPFF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF655 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf226 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC3 has 0 variants, skipping\n",
      "[INFO]\tGene MGC27382 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB1 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1C has 0 variants, skipping\n",
      "[INFO]\tGene HPCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MTPAP has 0 variants, skipping\n",
      "[INFO]\tGene FAR2 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf40 has 0 variants, skipping\n",
      "[INFO]\tGene ECT2 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM23 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927434 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF6 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42SE1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAPD1 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB3 has 0 variants, skipping\n",
      "[INFO]\tGene LMCD1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC16 has 0 variants, skipping\n",
      "[INFO]\tGene RMC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A11 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS1B has 0 variants, skipping\n",
      "[INFO]\tGene SCAF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01362 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01361 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01725 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927560 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX5 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS2 has 0 variants, skipping\n",
      "[INFO]\tGene C8B has 0 variants, skipping\n",
      "[INFO]\tGene MYO16 has 0 variants, skipping\n",
      "[INFO]\tGene SATB2 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPD3 has 0 variants, skipping\n",
      "[INFO]\tGene UCKL1 has 0 variants, skipping\n",
      "[INFO]\tGene FASLG has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01461 has 0 variants, skipping\n",
      "[INFO]\tGene MGARP has 0 variants, skipping\n",
      "[INFO]\tGene LNX1 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A3 has 0 variants, skipping\n",
      "[INFO]\tGene COX4I2 has 0 variants, skipping\n",
      "[INFO]\tGene RABGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene POM121 has 0 variants, skipping\n",
      "[INFO]\tGene RARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF879 has 0 variants, skipping\n",
      "[INFO]\tGene POMT1 has 0 variants, skipping\n",
      "[INFO]\tGene MICB has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD has 0 variants, skipping\n",
      "[INFO]\tGene LOC646626 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH2B has 0 variants, skipping\n",
      "[INFO]\tGene NCR1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM11 has 0 variants, skipping\n",
      "[INFO]\tGene RGS13 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIGA2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4A has 0 variants, skipping\n",
      "[INFO]\tGene RPS9 has 0 variants, skipping\n",
      "[INFO]\tGene MBP has 0 variants, skipping\n",
      "[INFO]\tGene CRAT has 0 variants, skipping\n",
      "[INFO]\tGene IFT81 has 0 variants, skipping\n",
      "[INFO]\tGene BBS9 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD11 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH2 has 0 variants, skipping\n",
      "[INFO]\tGene DBI has 0 variants, skipping\n",
      "[INFO]\tGene RCBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7856 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN11 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA3P has 0 variants, skipping\n",
      "[INFO]\tGene SLX4IP has 0 variants, skipping\n",
      "[INFO]\tGene MPP1 has 0 variants, skipping\n",
      "[INFO]\tGene APPL2 has 0 variants, skipping\n",
      "[INFO]\tGene EDA2R has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01140 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01364 has 0 variants, skipping\n",
      "[INFO]\tGene PKN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8A has 0 variants, skipping\n",
      "[INFO]\tGene FBRS has 0 variants, skipping\n",
      "[INFO]\tGene TSFM has 0 variants, skipping\n",
      "[INFO]\tGene PPARGC1B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF512 has 0 variants, skipping\n",
      "[INFO]\tGene KYAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP45 has 0 variants, skipping\n",
      "[INFO]\tGene ST7 has 0 variants, skipping\n",
      "[INFO]\tGene CPAMD8 has 0 variants, skipping\n",
      "[INFO]\tGene EPOP has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR2 has 0 variants, skipping\n",
      "[INFO]\tGene WWTR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR156 has 0 variants, skipping\n",
      "[INFO]\tGene SNTB1 has 0 variants, skipping\n",
      "[INFO]\tGene DCTPP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIO has 0 variants, skipping\n",
      "[INFO]\tGene FAM214B has 0 variants, skipping\n",
      "[INFO]\tGene BRD9 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GBP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8C-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A11 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN8P4 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHIP has 0 variants, skipping\n",
      "[INFO]\tGene PSRC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3G has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC25 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01763 has 0 variants, skipping\n",
      "[INFO]\tGene RFC2 has 0 variants, skipping\n",
      "[INFO]\tGene DIS3L has 0 variants, skipping\n",
      "[INFO]\tGene NOL4 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6 has 0 variants, skipping\n",
      "[INFO]\tGene PEPD has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161A has 0 variants, skipping\n",
      "[INFO]\tGene IL15RA has 0 variants, skipping\n",
      "[INFO]\tGene MIXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513C has 0 variants, skipping\n",
      "[INFO]\tGene PDE9A has 0 variants, skipping\n",
      "[INFO]\tGene PRLR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF664-RFLNA has 0 variants, skipping\n",
      "[INFO]\tGene GC has 0 variants, skipping\n",
      "[INFO]\tGene MUC1 has 0 variants, skipping\n",
      "[INFO]\tGene TSC2 has 0 variants, skipping\n",
      "[INFO]\tGene LTB has 0 variants, skipping\n",
      "[INFO]\tGene LARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADNP has 0 variants, skipping\n",
      "[INFO]\tGene ASXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MED13L has 0 variants, skipping\n",
      "[INFO]\tGene IL32 has 0 variants, skipping\n",
      "[INFO]\tGene RNF170 has 0 variants, skipping\n",
      "[INFO]\tGene PMFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C has 0 variants, skipping\n",
      "[INFO]\tGene ATG2B has 0 variants, skipping\n",
      "[INFO]\tGene DET1 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA66 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL2 has 0 variants, skipping\n",
      "[INFO]\tGene SHOC1 has 0 variants, skipping\n",
      "[INFO]\tGene AGT has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF33 has 0 variants, skipping\n",
      "[INFO]\tGene USP28 has 0 variants, skipping\n",
      "[INFO]\tGene PLCL1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B15 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene ATG4B has 0 variants, skipping\n",
      "[INFO]\tGene RABGGTB has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L3 has 0 variants, skipping\n",
      "[INFO]\tGene ENAH has 0 variants, skipping\n",
      "[INFO]\tGene PACC1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF9 has 0 variants, skipping\n",
      "[INFO]\tGene HAO2 has 0 variants, skipping\n",
      "[INFO]\tGene PQBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACYP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOK4 has 0 variants, skipping\n",
      "[INFO]\tGene REEP6 has 0 variants, skipping\n",
      "[INFO]\tGene AKAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378G has 0 variants, skipping\n",
      "[INFO]\tGene CCN5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM186B has 0 variants, skipping\n",
      "[INFO]\tGene MUC5AC has 0 variants, skipping\n",
      "[INFO]\tGene WNT8A has 0 variants, skipping\n",
      "[INFO]\tGene CABP1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX59 has 0 variants, skipping\n",
      "[INFO]\tGene TTN has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB38 has 0 variants, skipping\n",
      "[INFO]\tGene ZZZ3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928118 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23B has 0 variants, skipping\n",
      "[INFO]\tGene GZF1 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFA has 0 variants, skipping\n",
      "[INFO]\tGene CLIC6 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01760 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01650 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01761 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02607 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01787 has 0 variants, skipping\n",
      "[INFO]\tGene PNCK has 0 variants, skipping\n",
      "[INFO]\tGene SMTNL2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3H has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K19 has 0 variants, skipping\n",
      "[INFO]\tGene CARF has 0 variants, skipping\n",
      "[INFO]\tGene DPYD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01930 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2682 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01776 has 0 variants, skipping\n",
      "[INFO]\tGene GLT8D2 has 0 variants, skipping\n",
      "[INFO]\tGene CHML has 0 variants, skipping\n",
      "[INFO]\tGene GCSAM has 0 variants, skipping\n",
      "[INFO]\tGene DNAH10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129620 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8F has 0 variants, skipping\n",
      "[INFO]\tGene ACSF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01708 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548N has 0 variants, skipping\n",
      "[INFO]\tGene PEX5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF658 has 0 variants, skipping\n",
      "[INFO]\tGene APTR has 0 variants, skipping\n",
      "[INFO]\tGene STX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A7 has 0 variants, skipping\n",
      "[INFO]\tGene TMC7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF37A has 0 variants, skipping\n",
      "[INFO]\tGene MAP1B has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB8 has 0 variants, skipping\n",
      "[INFO]\tGene PI3 has 0 variants, skipping\n",
      "[INFO]\tGene PHKA1 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT13 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL38 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCAD1 has 0 variants, skipping\n",
      "[INFO]\tGene REXO4 has 0 variants, skipping\n",
      "[INFO]\tGene RTCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SORBS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR553 has 0 variants, skipping\n",
      "[INFO]\tGene GRM5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKMY1 has 0 variants, skipping\n",
      "[INFO]\tGene ILK has 0 variants, skipping\n",
      "[INFO]\tGene LINC01349 has 0 variants, skipping\n",
      "[INFO]\tGene SASH1 has 0 variants, skipping\n",
      "[INFO]\tGene ICA1 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A7 has 0 variants, skipping\n",
      "[INFO]\tGene MNT has 0 variants, skipping\n",
      "[INFO]\tGene CCNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MGST1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRB1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNBIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102606465 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928370 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01307 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX2 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA1P5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC32 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B5 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2B has 0 variants, skipping\n",
      "[INFO]\tGene MIA has 0 variants, skipping\n",
      "[INFO]\tGene NCAPH2 has 0 variants, skipping\n",
      "[INFO]\tGene CARD10 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF232 has 0 variants, skipping\n",
      "[INFO]\tGene MYBBP1A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF229 has 0 variants, skipping\n",
      "[INFO]\tGene UBP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2S2 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01348 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP3 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA6 has 0 variants, skipping\n",
      "[INFO]\tGene PHRF1 has 0 variants, skipping\n",
      "[INFO]\tGene EPS15L1 has 0 variants, skipping\n",
      "[INFO]\tGene PRODH2 has 0 variants, skipping\n",
      "[INFO]\tGene TC2N has 0 variants, skipping\n",
      "[INFO]\tGene MSH6 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P4 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L5 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CYBA has 0 variants, skipping\n",
      "[INFO]\tGene COL4A3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF20L1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTK1 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A39 has 0 variants, skipping\n",
      "[INFO]\tGene ATL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN19 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129138 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01676 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01677 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01661 has 0 variants, skipping\n",
      "[INFO]\tGene UFSP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX5 has 0 variants, skipping\n",
      "[INFO]\tGene ELP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7852 has 0 variants, skipping\n",
      "[INFO]\tGene VAV3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRB has 0 variants, skipping\n",
      "[INFO]\tGene GALC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF267 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf54 has 0 variants, skipping\n",
      "[INFO]\tGene NMRAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDR2 has 0 variants, skipping\n",
      "[INFO]\tGene CDO1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42 has 0 variants, skipping\n",
      "[INFO]\tGene CDC25C has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2D has 0 variants, skipping\n",
      "[INFO]\tGene CDC25A has 0 variants, skipping\n",
      "[INFO]\tGene CDK11B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC103 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN4 has 0 variants, skipping\n",
      "[INFO]\tGene CHN1 has 0 variants, skipping\n",
      "[INFO]\tGene SNTG1 has 0 variants, skipping\n",
      "[INFO]\tGene PROCA1 has 0 variants, skipping\n",
      "[INFO]\tGene COX7A2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA42 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFSD has 0 variants, skipping\n",
      "[INFO]\tGene IRAG1 has 0 variants, skipping\n",
      "[INFO]\tGene ERLIN2 has 0 variants, skipping\n",
      "[INFO]\tGene STRADA has 0 variants, skipping\n",
      "[INFO]\tGene ETFDH has 0 variants, skipping\n",
      "[INFO]\tGene ENOX2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2H1 has 0 variants, skipping\n",
      "[INFO]\tGene APOA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC25 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN3 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene APLNR has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAB2B has 0 variants, skipping\n",
      "[INFO]\tGene GCC2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT10 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1H has 0 variants, skipping\n",
      "[INFO]\tGene LGMN has 0 variants, skipping\n",
      "[INFO]\tGene MXRA7 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF92 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD12 has 0 variants, skipping\n",
      "[INFO]\tGene RALYL has 0 variants, skipping\n",
      "[INFO]\tGene LDHB has 0 variants, skipping\n",
      "[INFO]\tGene LAIR2 has 0 variants, skipping\n",
      "[INFO]\tGene ISYNA1 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10-NBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MARF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF692 has 0 variants, skipping\n",
      "[INFO]\tGene INTS8 has 0 variants, skipping\n",
      "[INFO]\tGene DUOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1A has 0 variants, skipping\n",
      "[INFO]\tGene NOX1 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1HG has 0 variants, skipping\n",
      "[INFO]\tGene SIRPD has 0 variants, skipping\n",
      "[INFO]\tGene C20orf85 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT4 has 0 variants, skipping\n",
      "[INFO]\tGene DRAM2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF3 has 0 variants, skipping\n",
      "[INFO]\tGene PTH2 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD2 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR4 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE1 has 0 variants, skipping\n",
      "[INFO]\tGene GMCL1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE5A has 0 variants, skipping\n",
      "[INFO]\tGene LCE3E has 0 variants, skipping\n",
      "[INFO]\tGene LCE3C has 0 variants, skipping\n",
      "[INFO]\tGene LCE3B has 0 variants, skipping\n",
      "[INFO]\tGene CDK20 has 0 variants, skipping\n",
      "[INFO]\tGene LCE3A has 0 variants, skipping\n",
      "[INFO]\tGene LCE2D has 0 variants, skipping\n",
      "[INFO]\tGene LCE2C has 0 variants, skipping\n",
      "[INFO]\tGene LCE2A has 0 variants, skipping\n",
      "[INFO]\tGene ARNT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP21 has 0 variants, skipping\n",
      "[INFO]\tGene CLN6 has 0 variants, skipping\n",
      "[INFO]\tGene NMRK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF416 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf42 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB104A has 0 variants, skipping\n",
      "[INFO]\tGene MIR600 has 0 variants, skipping\n",
      "[INFO]\tGene OPN4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS36 has 0 variants, skipping\n",
      "[INFO]\tGene SEC11C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01397 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02586 has 0 variants, skipping\n",
      "[INFO]\tGene GOSR1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2L has 0 variants, skipping\n",
      "[INFO]\tGene PMAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INO80E has 0 variants, skipping\n",
      "[INFO]\tGene ZWILCH has 0 variants, skipping\n",
      "[INFO]\tGene YJU2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf33 has 0 variants, skipping\n",
      "[INFO]\tGene CYMP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNAR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMUB2 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS2 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT2B has 0 variants, skipping\n",
      "[INFO]\tGene FAS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3G has 0 variants, skipping\n",
      "[INFO]\tGene BZW1 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL5 has 0 variants, skipping\n",
      "[INFO]\tGene AP2A2 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL2 has 0 variants, skipping\n",
      "[INFO]\tGene DENND4A has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A1 has 0 variants, skipping\n",
      "[INFO]\tGene SPOP has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB8 has 0 variants, skipping\n",
      "[INFO]\tGene LRRFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2C has 0 variants, skipping\n",
      "[INFO]\tGene DMGDH has 0 variants, skipping\n",
      "[INFO]\tGene CYP2D7 has 0 variants, skipping\n",
      "[INFO]\tGene PIFO has 0 variants, skipping\n",
      "[INFO]\tGene FAM181A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM2 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RUSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01160 has 0 variants, skipping\n",
      "[INFO]\tGene DHX36 has 0 variants, skipping\n",
      "[INFO]\tGene FLT3LG has 0 variants, skipping\n",
      "[INFO]\tGene INKA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928718 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene DFFB has 0 variants, skipping\n",
      "[INFO]\tGene KCND3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene KCND3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01750 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4256 has 0 variants, skipping\n",
      "[INFO]\tGene CCT5 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F1 has 0 variants, skipping\n",
      "[INFO]\tGene USF3 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAP has 0 variants, skipping\n",
      "[INFO]\tGene PRH1-TAS2R14 has 0 variants, skipping\n",
      "[INFO]\tGene PTBP3 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF11 has 0 variants, skipping\n",
      "[INFO]\tGene TFPT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01356 has 0 variants, skipping\n",
      "[INFO]\tGene SMTN has 0 variants, skipping\n",
      "[INFO]\tGene MIR4739 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00639 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD45C has 0 variants, skipping\n",
      "[INFO]\tGene FMO9P has 0 variants, skipping\n",
      "[INFO]\tGene MBD4 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP57 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643441 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH10 has 0 variants, skipping\n",
      "[INFO]\tGene RBM6 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N4C has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2A has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L7 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC13 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD3 has 0 variants, skipping\n",
      "[INFO]\tGene PHTF1 has 0 variants, skipping\n",
      "[INFO]\tGene APH1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF13B has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC6A has 0 variants, skipping\n",
      "[INFO]\tGene GLS2 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA13 has 0 variants, skipping\n",
      "[INFO]\tGene CHRDL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCHCR1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IRF5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF526 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF21B has 0 variants, skipping\n",
      "[INFO]\tGene GRB7 has 0 variants, skipping\n",
      "[INFO]\tGene CIB2 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD9L has 0 variants, skipping\n",
      "[INFO]\tGene LIPK has 0 variants, skipping\n",
      "[INFO]\tGene CPNE4 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNP1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI2 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI1 has 0 variants, skipping\n",
      "[INFO]\tGene TNNC2 has 0 variants, skipping\n",
      "[INFO]\tGene TNNC1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC3B has 0 variants, skipping\n",
      "[INFO]\tGene CLDN5 has 0 variants, skipping\n",
      "[INFO]\tGene TMPO has 0 variants, skipping\n",
      "[INFO]\tGene TMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN8 has 0 variants, skipping\n",
      "[INFO]\tGene SETD9 has 0 variants, skipping\n",
      "[INFO]\tGene MBD1 has 0 variants, skipping\n",
      "[INFO]\tGene STAM has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP3 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC21 has 0 variants, skipping\n",
      "[INFO]\tGene AFTPH has 0 variants, skipping\n",
      "[INFO]\tGene ZNF542P has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-1HG has 0 variants, skipping\n",
      "[INFO]\tGene TASOR2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB4 has 0 variants, skipping\n",
      "[INFO]\tGene GSTZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1L has 0 variants, skipping\n",
      "[INFO]\tGene AP4S1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02861 has 0 variants, skipping\n",
      "[INFO]\tGene KLK2 has 0 variants, skipping\n",
      "[INFO]\tGene COL25A1 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL4 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB1 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH1 has 0 variants, skipping\n",
      "[INFO]\tGene C2 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB6 has 0 variants, skipping\n",
      "[INFO]\tGene NGF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM26 has 0 variants, skipping\n",
      "[INFO]\tGene FBLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM218 has 0 variants, skipping\n",
      "[INFO]\tGene CROCC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01649 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928977 has 0 variants, skipping\n",
      "[INFO]\tGene RAB5C has 0 variants, skipping\n",
      "[INFO]\tGene RAB5B has 0 variants, skipping\n",
      "[INFO]\tGene TPBG has 0 variants, skipping\n",
      "[INFO]\tGene CYP46A1 has 0 variants, skipping\n",
      "[INFO]\tGene PGRMC1 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE4 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01762 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3B1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM34 has 0 variants, skipping\n",
      "[INFO]\tGene ASB12 has 0 variants, skipping\n",
      "[INFO]\tGene ASB14 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC12 has 0 variants, skipping\n",
      "[INFO]\tGene ATRIP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF783 has 0 variants, skipping\n",
      "[INFO]\tGene SSB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929099 has 0 variants, skipping\n",
      "[INFO]\tGene TP53RK has 0 variants, skipping\n",
      "[INFO]\tGene INTS4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL4L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01525 has 0 variants, skipping\n",
      "[INFO]\tGene NFAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996263 has 0 variants, skipping\n",
      "[INFO]\tGene EHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene GPS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP4 has 0 variants, skipping\n",
      "[INFO]\tGene NEK6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS41 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378933 has 0 variants, skipping\n",
      "[INFO]\tGene IL17REL has 0 variants, skipping\n",
      "[INFO]\tGene WARS2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene WARS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01780 has 0 variants, skipping\n",
      "[INFO]\tGene HAO2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51I2 has 0 variants, skipping\n",
      "[INFO]\tGene HSD3BP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00622 has 0 variants, skipping\n",
      "[INFO]\tGene DAAM2 has 0 variants, skipping\n",
      "[INFO]\tGene ELN has 0 variants, skipping\n",
      "[INFO]\tGene DNM1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC16 has 0 variants, skipping\n",
      "[INFO]\tGene BBS4 has 0 variants, skipping\n",
      "[INFO]\tGene PFN1P2 has 0 variants, skipping\n",
      "[INFO]\tGene RFX5 has 0 variants, skipping\n",
      "[INFO]\tGene RBPJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR128-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR374A has 0 variants, skipping\n",
      "[INFO]\tGene MIR155 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200C has 0 variants, skipping\n",
      "[INFO]\tGene MIR650 has 0 variants, skipping\n",
      "[INFO]\tGene MIR622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR615 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-19 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00623 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-20 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-4 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL3 has 0 variants, skipping\n",
      "[INFO]\tGene PARP6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996318 has 0 variants, skipping\n",
      "[INFO]\tGene OBSL1 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P5 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644634 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1CP has 0 variants, skipping\n",
      "[INFO]\tGene MIEF1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM72C has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101060524 has 0 variants, skipping\n",
      "[INFO]\tGene DRD5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927429 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A2 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-15 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF3 has 0 variants, skipping\n",
      "[INFO]\tGene EML2 has 0 variants, skipping\n",
      "[INFO]\tGene DBNDD2 has 0 variants, skipping\n",
      "[INFO]\tGene PKP4 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC5 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1A has 0 variants, skipping\n",
      "[INFO]\tGene FAM72D has 0 variants, skipping\n",
      "[INFO]\tGene LINC01145 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996740 has 0 variants, skipping\n",
      "[INFO]\tGene FGF13 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-14 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF9 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC19 has 0 variants, skipping\n",
      "[INFO]\tGene GPX4 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO4 has 0 variants, skipping\n",
      "[INFO]\tGene RARG has 0 variants, skipping\n",
      "[INFO]\tGene CCDC74A has 0 variants, skipping\n",
      "[INFO]\tGene ABCB1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF25P has 0 variants, skipping\n",
      "[INFO]\tGene BDKRB2 has 0 variants, skipping\n",
      "[INFO]\tGene HHIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR89A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF888 has 0 variants, skipping\n",
      "[INFO]\tGene NKX3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF266 has 0 variants, skipping\n",
      "[INFO]\tGene PEX5L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6736 has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG1 has 0 variants, skipping\n",
      "[INFO]\tGene APBB1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2D has 0 variants, skipping\n",
      "[INFO]\tGene GSS has 0 variants, skipping\n",
      "[INFO]\tGene NR3C1 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE5 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL3 has 0 variants, skipping\n",
      "[INFO]\tGene GNRHR2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8S has 0 variants, skipping\n",
      "[INFO]\tGene LIX1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOF has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01719 has 0 variants, skipping\n",
      "[INFO]\tGene ARL17B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4P2 has 0 variants, skipping\n",
      "[INFO]\tGene HYDIN2 has 0 variants, skipping\n",
      "[INFO]\tGene SV2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC728989 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-8 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2C has 0 variants, skipping\n",
      "[INFO]\tGene PDIA3P1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW12 has 0 variants, skipping\n",
      "[INFO]\tGene FLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC10 has 0 variants, skipping\n",
      "[INFO]\tGene ZPBP has 0 variants, skipping\n",
      "[INFO]\tGene CD200 has 0 variants, skipping\n",
      "[INFO]\tGene NTRK2 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT2 has 0 variants, skipping\n",
      "[INFO]\tGene FYN has 0 variants, skipping\n",
      "[INFO]\tGene EFR3A has 0 variants, skipping\n",
      "[INFO]\tGene STOML1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF286A has 0 variants, skipping\n",
      "[INFO]\tGene IYD has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L2-SCNM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO2-KLHL23 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A has 0 variants, skipping\n",
      "[INFO]\tGene TYW5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM165 has 0 variants, skipping\n",
      "[INFO]\tGene FUBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCER1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM28 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL5 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HINT1 has 0 variants, skipping\n",
      "[INFO]\tGene STYXL1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO11 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H4 has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ has 0 variants, skipping\n",
      "[INFO]\tGene METTL23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00624 has 0 variants, skipping\n",
      "[INFO]\tGene ERLIN1 has 0 variants, skipping\n",
      "[INFO]\tGene SOST has 0 variants, skipping\n",
      "[INFO]\tGene LY6K has 0 variants, skipping\n",
      "[INFO]\tGene ZNF7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF460 has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD1 has 0 variants, skipping\n",
      "[INFO]\tGene AK4 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED9 has 0 variants, skipping\n",
      "[INFO]\tGene PPARG has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF205-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CKS1B has 0 variants, skipping\n",
      "[INFO]\tGene DGCR5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC-PINT has 0 variants, skipping\n",
      "[INFO]\tGene SNHG12 has 0 variants, skipping\n",
      "[INFO]\tGene PBX3 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM3A has 0 variants, skipping\n",
      "[INFO]\tGene VSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927468 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5087 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6077 has 0 variants, skipping\n",
      "[INFO]\tGene FN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-3 has 0 variants, skipping\n",
      "[INFO]\tGene MZB1 has 0 variants, skipping\n",
      "[INFO]\tGene HK1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT8 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2A has 0 variants, skipping\n",
      "[INFO]\tGene PCCA has 0 variants, skipping\n",
      "[INFO]\tGene PRIMPOL has 0 variants, skipping\n",
      "[INFO]\tGene RNF167 has 0 variants, skipping\n",
      "[INFO]\tGene PNO1 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene GPI has 0 variants, skipping\n",
      "[INFO]\tGene STX3 has 0 variants, skipping\n",
      "[INFO]\tGene USO1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A35 has 0 variants, skipping\n",
      "[INFO]\tGene BMX has 0 variants, skipping\n",
      "[INFO]\tGene SPATA1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD7 has 0 variants, skipping\n",
      "[INFO]\tGene NKTR has 0 variants, skipping\n",
      "[INFO]\tGene PIGG has 0 variants, skipping\n",
      "[INFO]\tGene LOC653513 has 0 variants, skipping\n",
      "[INFO]\tGene DOK1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf47 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN7 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A1 has 0 variants, skipping\n",
      "[INFO]\tGene RND3 has 0 variants, skipping\n",
      "[INFO]\tGene NABP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00869 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF518A has 0 variants, skipping\n",
      "[INFO]\tGene AZIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4F has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS5 has 0 variants, skipping\n",
      "[INFO]\tGene CADM2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE8B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371433 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN6 has 0 variants, skipping\n",
      "[INFO]\tGene ANK3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13C has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL5B has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene APH1A has 0 variants, skipping\n",
      "[INFO]\tGene ZPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene CMPK1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM131A has 0 variants, skipping\n",
      "[INFO]\tGene DSE has 0 variants, skipping\n",
      "[INFO]\tGene KCP has 0 variants, skipping\n",
      "[INFO]\tGene FOXO3 has 0 variants, skipping\n",
      "[INFO]\tGene GATD3A has 0 variants, skipping\n",
      "[INFO]\tGene C16orf87 has 0 variants, skipping\n",
      "[INFO]\tGene DLX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00461 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF224 has 0 variants, skipping\n",
      "[INFO]\tGene MOV10L1 has 0 variants, skipping\n",
      "[INFO]\tGene COPS7A has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DOA has 0 variants, skipping\n",
      "[INFO]\tGene FKBP7 has 0 variants, skipping\n",
      "[INFO]\tGene KAAG1 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6878 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3J has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN25 has 0 variants, skipping\n",
      "[INFO]\tGene OR2F1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP12 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EPC1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL7 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8O has 0 variants, skipping\n",
      "[INFO]\tGene MYH4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF821 has 0 variants, skipping\n",
      "[INFO]\tGene USP48 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3B has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA1 has 0 variants, skipping\n",
      "[INFO]\tGene LARP1B has 0 variants, skipping\n",
      "[INFO]\tGene POMK has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10 has 0 variants, skipping\n",
      "[INFO]\tGene SERPING1 has 0 variants, skipping\n",
      "[INFO]\tGene ENO1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL6 has 0 variants, skipping\n",
      "[INFO]\tGene IQCB1 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L22 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L21 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L20 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L18 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L19 has 0 variants, skipping\n",
      "[INFO]\tGene RNF34 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L17 has 0 variants, skipping\n",
      "[INFO]\tGene BTRC has 0 variants, skipping\n",
      "[INFO]\tGene USP17L13 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L11 has 0 variants, skipping\n",
      "[INFO]\tGene GALK2 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK6 has 0 variants, skipping\n",
      "[INFO]\tGene FAR1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPM has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PITRM1 has 0 variants, skipping\n",
      "[INFO]\tGene TSBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF556 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN9 has 0 variants, skipping\n",
      "[INFO]\tGene CD1A has 0 variants, skipping\n",
      "[INFO]\tGene CCT6A has 0 variants, skipping\n",
      "[INFO]\tGene CCNF has 0 variants, skipping\n",
      "[INFO]\tGene CCND3 has 0 variants, skipping\n",
      "[INFO]\tGene CCND2 has 0 variants, skipping\n",
      "[INFO]\tGene DST has 0 variants, skipping\n",
      "[INFO]\tGene CEP97 has 0 variants, skipping\n",
      "[INFO]\tGene CTSB has 0 variants, skipping\n",
      "[INFO]\tGene CANX has 0 variants, skipping\n",
      "[INFO]\tGene NXPE3 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf173 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724788 has 0 variants, skipping\n",
      "[INFO]\tGene CRYAB has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724770 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724159 has 0 variants, skipping\n",
      "[INFO]\tGene FAM243A has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF687-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK3 has 0 variants, skipping\n",
      "[INFO]\tGene DMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM121B has 0 variants, skipping\n",
      "[INFO]\tGene C17orf99 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT11 has 0 variants, skipping\n",
      "[INFO]\tGene CHST15 has 0 variants, skipping\n",
      "[INFO]\tGene ARNT2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C2 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3B has 0 variants, skipping\n",
      "[INFO]\tGene ESRRG has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1191 has 0 variants, skipping\n",
      "[INFO]\tGene NAV1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAKD has 0 variants, skipping\n",
      "[INFO]\tGene TSEN34 has 0 variants, skipping\n",
      "[INFO]\tGene MIR554 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD5 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf40 has 0 variants, skipping\n",
      "[INFO]\tGene CBSL has 0 variants, skipping\n",
      "[INFO]\tGene BLZF1 has 0 variants, skipping\n",
      "[INFO]\tGene CBS has 0 variants, skipping\n",
      "[INFO]\tGene CCDC66 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB2 has 0 variants, skipping\n",
      "[INFO]\tGene TDRKH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110384692 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3B has 0 variants, skipping\n",
      "[INFO]\tGene AP2S1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT5 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF18P has 0 variants, skipping\n",
      "[INFO]\tGene MYO19 has 0 variants, skipping\n",
      "[INFO]\tGene FLG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRF has 0 variants, skipping\n",
      "[INFO]\tGene RPL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00302 has 0 variants, skipping\n",
      "[INFO]\tGene ADK has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF12 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3A has 0 variants, skipping\n",
      "[INFO]\tGene USP40 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf14 has 0 variants, skipping\n",
      "[INFO]\tGene SETMAR has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928009 has 0 variants, skipping\n",
      "[INFO]\tGene DCHS2 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPPS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928034 has 0 variants, skipping\n",
      "[INFO]\tGene FGD4 has 0 variants, skipping\n",
      "[INFO]\tGene CAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GREB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIB1 has 0 variants, skipping\n",
      "[INFO]\tGene LIPT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8083 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF419 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC343052 has 0 variants, skipping\n",
      "[INFO]\tGene RAB44 has 0 variants, skipping\n",
      "[INFO]\tGene BCS1L has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZPR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6737 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN14 has 0 variants, skipping\n",
      "[INFO]\tGene NEK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3 has 0 variants, skipping\n",
      "[INFO]\tGene OSGEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF44 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01060 has 0 variants, skipping\n",
      "[INFO]\tGene ENKUR has 0 variants, skipping\n",
      "[INFO]\tGene STAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMLHE has 0 variants, skipping\n",
      "[INFO]\tGene FAM78B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5698 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1 has 0 variants, skipping\n",
      "[INFO]\tGene HCLS1 has 0 variants, skipping\n",
      "[INFO]\tGene HDLBP has 0 variants, skipping\n",
      "[INFO]\tGene CYC1 has 0 variants, skipping\n",
      "[INFO]\tGene FLG has 0 variants, skipping\n",
      "[INFO]\tGene FOXO1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2E3 has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF4 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM174C has 0 variants, skipping\n",
      "[INFO]\tGene CDC37L1 has 0 variants, skipping\n",
      "[INFO]\tGene ATG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM3 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf72 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF619 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12E has 0 variants, skipping\n",
      "[INFO]\tGene PRM2 has 0 variants, skipping\n",
      "[INFO]\tGene GPRC6A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA58B has 0 variants, skipping\n",
      "[INFO]\tGene REV1 has 0 variants, skipping\n",
      "[INFO]\tGene IL6R-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAF8 has 0 variants, skipping\n",
      "[INFO]\tGene TEX49 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG6 has 0 variants, skipping\n",
      "[INFO]\tGene HYKK has 0 variants, skipping\n",
      "[INFO]\tGene MKS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF8 has 0 variants, skipping\n",
      "[INFO]\tGene TP73 has 0 variants, skipping\n",
      "[INFO]\tGene GLMP has 0 variants, skipping\n",
      "[INFO]\tGene LARP4 has 0 variants, skipping\n",
      "[INFO]\tGene CDK1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC20B has 0 variants, skipping\n",
      "[INFO]\tGene GLS has 0 variants, skipping\n",
      "[INFO]\tGene EPHX2 has 0 variants, skipping\n",
      "[INFO]\tGene DMPK has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF75A has 0 variants, skipping\n",
      "[INFO]\tGene CPNE1 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE1 has 0 variants, skipping\n",
      "[INFO]\tGene CHFR has 0 variants, skipping\n",
      "[INFO]\tGene NR2C2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF420 has 0 variants, skipping\n",
      "[INFO]\tGene DEDD has 0 variants, skipping\n",
      "[INFO]\tGene ZNF875 has 0 variants, skipping\n",
      "[INFO]\tGene WDR20 has 0 variants, skipping\n",
      "[INFO]\tGene EHMT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6J1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724488 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4258 has 0 variants, skipping\n",
      "[INFO]\tGene NFIL3 has 0 variants, skipping\n",
      "[INFO]\tGene FHIT has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01082 has 0 variants, skipping\n",
      "[INFO]\tGene MINAR2 has 0 variants, skipping\n",
      "[INFO]\tGene MOK has 0 variants, skipping\n",
      "[INFO]\tGene ZNF74 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT has 0 variants, skipping\n",
      "[INFO]\tGene DPP10 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene CADM3 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL5 has 0 variants, skipping\n",
      "[INFO]\tGene CD8B2 has 0 variants, skipping\n",
      "[INFO]\tGene MYO7A has 0 variants, skipping\n",
      "[INFO]\tGene MYO5A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC30 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA4B has 0 variants, skipping\n",
      "[INFO]\tGene OR10G6 has 0 variants, skipping\n",
      "[INFO]\tGene RET has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM126A has 0 variants, skipping\n",
      "[INFO]\tGene AKAP10 has 0 variants, skipping\n",
      "[INFO]\tGene F11R has 0 variants, skipping\n",
      "[INFO]\tGene LINC02751 has 0 variants, skipping\n",
      "[INFO]\tGene HSPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB2 has 0 variants, skipping\n",
      "[INFO]\tGene OAT has 0 variants, skipping\n",
      "[INFO]\tGene SRSF7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM240A has 0 variants, skipping\n",
      "[INFO]\tGene MAS1L has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN15 has 0 variants, skipping\n",
      "[INFO]\tGene CLNK has 0 variants, skipping\n",
      "[INFO]\tGene GNG2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1MT has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF35-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184B has 0 variants, skipping\n",
      "[INFO]\tGene THBS3 has 0 variants, skipping\n",
      "[INFO]\tGene MUSK has 0 variants, skipping\n",
      "[INFO]\tGene PAN2 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1B has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL26 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC14 has 0 variants, skipping\n",
      "[INFO]\tGene THOC3 has 0 variants, skipping\n",
      "[INFO]\tGene CMSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC115 has 0 variants, skipping\n",
      "[INFO]\tGene NAA38 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12C has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX2 has 0 variants, skipping\n",
      "[INFO]\tGene SSH2 has 0 variants, skipping\n",
      "[INFO]\tGene ATG16L2 has 0 variants, skipping\n",
      "[INFO]\tGene IL18BP has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A2 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1C has 0 variants, skipping\n",
      "[INFO]\tGene BCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D5 has 0 variants, skipping\n",
      "[INFO]\tGene OR1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2 has 0 variants, skipping\n",
      "[INFO]\tGene OTX1 has 0 variants, skipping\n",
      "[INFO]\tGene PARPBP has 0 variants, skipping\n",
      "[INFO]\tGene ERI3 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA2 has 0 variants, skipping\n",
      "[INFO]\tGene MOBP has 0 variants, skipping\n",
      "[INFO]\tGene IFFO1 has 0 variants, skipping\n",
      "[INFO]\tGene USP5 has 0 variants, skipping\n",
      "[INFO]\tGene DDR1 has 0 variants, skipping\n",
      "[INFO]\tGene COQ8A has 0 variants, skipping\n",
      "[INFO]\tGene TAF1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR555 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P4 has 0 variants, skipping\n",
      "[INFO]\tGene ASH1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TOP3A has 0 variants, skipping\n",
      "[INFO]\tGene RB1CC1 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRF1L has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG1 has 0 variants, skipping\n",
      "[INFO]\tGene STRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM20 has 0 variants, skipping\n",
      "[INFO]\tGene CASP8AP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC7 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L2 has 0 variants, skipping\n",
      "[INFO]\tGene TSIX has 0 variants, skipping\n",
      "[INFO]\tGene LAS1L has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN5 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01714 has 0 variants, skipping\n",
      "[INFO]\tGene RNF207-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSTO2P has 0 variants, skipping\n",
      "[INFO]\tGene SMAGP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF451 has 0 variants, skipping\n",
      "[INFO]\tGene AMMECR1 has 0 variants, skipping\n",
      "[INFO]\tGene SOHLH1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX23 has 0 variants, skipping\n",
      "[INFO]\tGene TJP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM189A2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP29 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA26A has 0 variants, skipping\n",
      "[INFO]\tGene LALBA has 0 variants, skipping\n",
      "[INFO]\tGene GTDC1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB2 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1A1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP9 has 0 variants, skipping\n",
      "[INFO]\tGene TP53I3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3R2 has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA26B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA4 has 0 variants, skipping\n",
      "[INFO]\tGene EMC8 has 0 variants, skipping\n",
      "[INFO]\tGene MTERF4 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT1A has 0 variants, skipping\n",
      "[INFO]\tGene SP140L has 0 variants, skipping\n",
      "[INFO]\tGene GLT8D1 has 0 variants, skipping\n",
      "[INFO]\tGene SMU1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6738 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2B has 0 variants, skipping\n",
      "[INFO]\tGene IMPG1 has 0 variants, skipping\n",
      "[INFO]\tGene CDIN1 has 0 variants, skipping\n",
      "[INFO]\tGene INF2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX25 has 0 variants, skipping\n",
      "[INFO]\tGene SRPK2 has 0 variants, skipping\n",
      "[INFO]\tGene RACGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF184 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1 has 0 variants, skipping\n",
      "[INFO]\tGene STRA6 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B15 has 0 variants, skipping\n",
      "[INFO]\tGene BRS3 has 0 variants, skipping\n",
      "[INFO]\tGene BRDT has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB44 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC8 has 0 variants, skipping\n",
      "[INFO]\tGene DMAC2 has 0 variants, skipping\n",
      "[INFO]\tGene PHIP has 0 variants, skipping\n",
      "[INFO]\tGene WDR70 has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L has 0 variants, skipping\n",
      "[INFO]\tGene WDR6 has 0 variants, skipping\n",
      "[INFO]\tGene PEX26 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL1A has 0 variants, skipping\n",
      "[INFO]\tGene DAZAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMF1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D26 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4RL2 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf38 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4RL1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO2 has 0 variants, skipping\n",
      "[INFO]\tGene NRBP2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC7 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf51B has 0 variants, skipping\n",
      "[INFO]\tGene CNIH2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A5 has 0 variants, skipping\n",
      "[INFO]\tGene CYP20A1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG2 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK1 has 0 variants, skipping\n",
      "[INFO]\tGene PANDAR has 0 variants, skipping\n",
      "[INFO]\tGene RPL13 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A5 has 0 variants, skipping\n",
      "[INFO]\tGene DUS2 has 0 variants, skipping\n",
      "[INFO]\tGene SIVA1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2CA has 0 variants, skipping\n",
      "[INFO]\tGene DLD has 0 variants, skipping\n",
      "[INFO]\tGene CPOX has 0 variants, skipping\n",
      "[INFO]\tGene DDB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928177 has 0 variants, skipping\n",
      "[INFO]\tGene DHFR has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS12 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene SAP30L has 0 variants, skipping\n",
      "[INFO]\tGene ARMC9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF558 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC1 has 0 variants, skipping\n",
      "[INFO]\tGene CABIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF513 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK2 has 0 variants, skipping\n",
      "[INFO]\tGene SCRN1 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM20 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MMS19 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A2 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI has 0 variants, skipping\n",
      "[INFO]\tGene TFDP2 has 0 variants, skipping\n",
      "[INFO]\tGene TESK1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD3 has 0 variants, skipping\n",
      "[INFO]\tGene CYCSP52 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF595 has 0 variants, skipping\n",
      "[INFO]\tGene IDH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAK has 0 variants, skipping\n",
      "[INFO]\tGene LINC00324 has 0 variants, skipping\n",
      "[INFO]\tGene EWSAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR8 has 0 variants, skipping\n",
      "[INFO]\tGene SMN2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC13 has 0 variants, skipping\n",
      "[INFO]\tGene SEPSECS has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371458 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF160 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01704 has 0 variants, skipping\n",
      "[INFO]\tGene POLD2 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf58 has 0 variants, skipping\n",
      "[INFO]\tGene DHPS has 0 variants, skipping\n",
      "[INFO]\tGene TTC27 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL24 has 0 variants, skipping\n",
      "[INFO]\tGene EVPL has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC28 has 0 variants, skipping\n",
      "[INFO]\tGene MUC3A has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE3 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFF has 0 variants, skipping\n",
      "[INFO]\tGene LSM5 has 0 variants, skipping\n",
      "[INFO]\tGene LSM4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G15 has 0 variants, skipping\n",
      "[INFO]\tGene CADM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCAP31 has 0 variants, skipping\n",
      "[INFO]\tGene REPS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL11A1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S6 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J4 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12G has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161B has 0 variants, skipping\n",
      "[INFO]\tGene LATS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6506 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG28 has 0 variants, skipping\n",
      "[INFO]\tGene NEPRO has 0 variants, skipping\n",
      "[INFO]\tGene MIR4259 has 0 variants, skipping\n",
      "[INFO]\tGene ATG4D has 0 variants, skipping\n",
      "[INFO]\tGene CLTC has 0 variants, skipping\n",
      "[INFO]\tGene PRELID1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN10 has 0 variants, skipping\n",
      "[INFO]\tGene DROSHA has 0 variants, skipping\n",
      "[INFO]\tGene EIF3K has 0 variants, skipping\n",
      "[INFO]\tGene STK39 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729867 has 0 variants, skipping\n",
      "[INFO]\tGene PATE3 has 0 variants, skipping\n",
      "[INFO]\tGene WWOX has 0 variants, skipping\n",
      "[INFO]\tGene FOXP2 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1D has 0 variants, skipping\n",
      "[INFO]\tGene EIF6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01535 has 0 variants, skipping\n",
      "[INFO]\tGene CCND1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287049 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723623 has 0 variants, skipping\n",
      "[INFO]\tGene ACACA has 0 variants, skipping\n",
      "[INFO]\tGene PPRC1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMLG has 0 variants, skipping\n",
      "[INFO]\tGene CAMK4 has 0 variants, skipping\n",
      "[INFO]\tGene CALM2 has 0 variants, skipping\n",
      "[INFO]\tGene CALCR has 0 variants, skipping\n",
      "[INFO]\tGene CALCA has 0 variants, skipping\n",
      "[INFO]\tGene CALB2 has 0 variants, skipping\n",
      "[INFO]\tGene CA5A has 0 variants, skipping\n",
      "[INFO]\tGene CA1 has 0 variants, skipping\n",
      "[INFO]\tGene C9 has 0 variants, skipping\n",
      "[INFO]\tGene C5AR1 has 0 variants, skipping\n",
      "[INFO]\tGene C5 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf147 has 0 variants, skipping\n",
      "[INFO]\tGene IFT20 has 0 variants, skipping\n",
      "[INFO]\tGene CARD16 has 0 variants, skipping\n",
      "[INFO]\tGene CTSA has 0 variants, skipping\n",
      "[INFO]\tGene WDR77 has 0 variants, skipping\n",
      "[INFO]\tGene RAB25 has 0 variants, skipping\n",
      "[INFO]\tGene S100A16 has 0 variants, skipping\n",
      "[INFO]\tGene MMP11 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1C has 0 variants, skipping\n",
      "[INFO]\tGene DPP6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928372 has 0 variants, skipping\n",
      "[INFO]\tGene HERC1 has 0 variants, skipping\n",
      "[INFO]\tGene PPAN has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN2 has 0 variants, skipping\n",
      "[INFO]\tGene BAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL27 has 0 variants, skipping\n",
      "[INFO]\tGene GMPR2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2H has 0 variants, skipping\n",
      "[INFO]\tGene ACSM5 has 0 variants, skipping\n",
      "[INFO]\tGene BBIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIPOL1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMS4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP4 has 0 variants, skipping\n",
      "[INFO]\tGene BCO1 has 0 variants, skipping\n",
      "[INFO]\tGene SPA17 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT7 has 0 variants, skipping\n",
      "[INFO]\tGene CALML5 has 0 variants, skipping\n",
      "[INFO]\tGene COQ3 has 0 variants, skipping\n",
      "[INFO]\tGene SIX4 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC5 has 0 variants, skipping\n",
      "[INFO]\tGene CLPSL1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC8L1 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHB has 0 variants, skipping\n",
      "[INFO]\tGene PRDM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5187 has 0 variants, skipping\n",
      "[INFO]\tGene STING1 has 0 variants, skipping\n",
      "[INFO]\tGene RPGR has 0 variants, skipping\n",
      "[INFO]\tGene HEPH has 0 variants, skipping\n",
      "[INFO]\tGene NXF2 has 0 variants, skipping\n",
      "[INFO]\tGene MRNIP has 0 variants, skipping\n",
      "[INFO]\tGene ZXDC has 0 variants, skipping\n",
      "[INFO]\tGene CILK1 has 0 variants, skipping\n",
      "[INFO]\tGene VSX1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF22 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A45 has 0 variants, skipping\n",
      "[INFO]\tGene MECR has 0 variants, skipping\n",
      "[INFO]\tGene KLK15 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC18 has 0 variants, skipping\n",
      "[INFO]\tGene WEE1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf67 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2012 has 0 variants, skipping\n",
      "[INFO]\tGene RPL31P11 has 0 variants, skipping\n",
      "[INFO]\tGene AMD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF17 has 0 variants, skipping\n",
      "[INFO]\tGene STARD3NL has 0 variants, skipping\n",
      "[INFO]\tGene GATB has 0 variants, skipping\n",
      "[INFO]\tGene CLHC1 has 0 variants, skipping\n",
      "[INFO]\tGene CS has 0 variants, skipping\n",
      "[INFO]\tGene AIP has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4654 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF227 has 0 variants, skipping\n",
      "[INFO]\tGene STRA8 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf50 has 0 variants, skipping\n",
      "[INFO]\tGene USP12 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML2A has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF6 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA13 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM59 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1L1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT10 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC92 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM19 has 0 variants, skipping\n",
      "[INFO]\tGene HBS1L has 0 variants, skipping\n",
      "[INFO]\tGene PMP2 has 0 variants, skipping\n",
      "[INFO]\tGene PMM1 has 0 variants, skipping\n",
      "[INFO]\tGene PML has 0 variants, skipping\n",
      "[INFO]\tGene PMCH has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422212 has 0 variants, skipping\n",
      "[INFO]\tGene LENG8 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP7 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF9 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene LCN12 has 0 variants, skipping\n",
      "[INFO]\tGene RNF180 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9C2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC52-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440700 has 0 variants, skipping\n",
      "[INFO]\tGene OXR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T7 has 0 variants, skipping\n",
      "[INFO]\tGene DDX39B has 0 variants, skipping\n",
      "[INFO]\tGene TMCO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB6 has 0 variants, skipping\n",
      "[INFO]\tGene PART1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01675 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG1 has 0 variants, skipping\n",
      "[INFO]\tGene RLN2 has 0 variants, skipping\n",
      "[INFO]\tGene MTOR has 0 variants, skipping\n",
      "[INFO]\tGene TEX15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01363 has 0 variants, skipping\n",
      "[INFO]\tGene SH3D19 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN3 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB2 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1A1 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP5 has 0 variants, skipping\n",
      "[INFO]\tGene FPGS has 0 variants, skipping\n",
      "[INFO]\tGene ETV1 has 0 variants, skipping\n",
      "[INFO]\tGene RAPSN has 0 variants, skipping\n",
      "[INFO]\tGene RGPD5 has 0 variants, skipping\n",
      "[INFO]\tGene MARK2 has 0 variants, skipping\n",
      "[INFO]\tGene H3-3B has 0 variants, skipping\n",
      "[INFO]\tGene H1-6 has 0 variants, skipping\n",
      "[INFO]\tGene COPS8 has 0 variants, skipping\n",
      "[INFO]\tGene BLCAP has 0 variants, skipping\n",
      "[INFO]\tGene GLO1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD8 has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45G has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3A has 0 variants, skipping\n",
      "[INFO]\tGene JTB has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCQ has 0 variants, skipping\n",
      "[INFO]\tGene GAREM1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO32 has 0 variants, skipping\n",
      "[INFO]\tGene RSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255B2 has 0 variants, skipping\n",
      "[INFO]\tGene IRAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TBXAS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC8 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS11 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR557 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505918 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928565 has 0 variants, skipping\n",
      "[INFO]\tGene TNK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00626 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00970 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928596 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1D has 0 variants, skipping\n",
      "[INFO]\tGene TTC13 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB3BP has 0 variants, skipping\n",
      "[INFO]\tGene RC3H1 has 0 variants, skipping\n",
      "[INFO]\tGene TNR has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A9 has 0 variants, skipping\n",
      "[INFO]\tGene IFT43 has 0 variants, skipping\n",
      "[INFO]\tGene ATF2 has 0 variants, skipping\n",
      "[INFO]\tGene SWI5 has 0 variants, skipping\n",
      "[INFO]\tGene DERPC has 0 variants, skipping\n",
      "[INFO]\tGene LIG1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H14 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF5 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL1 has 0 variants, skipping\n",
      "[INFO]\tGene SV2A has 0 variants, skipping\n",
      "[INFO]\tGene STUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MUTYH has 0 variants, skipping\n",
      "[INFO]\tGene MVP has 0 variants, skipping\n",
      "[INFO]\tGene FAM153CP has 0 variants, skipping\n",
      "[INFO]\tGene CREM has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene RHOG has 0 variants, skipping\n",
      "[INFO]\tGene RHOA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM102 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC40 has 0 variants, skipping\n",
      "[INFO]\tGene TTI2 has 0 variants, skipping\n",
      "[INFO]\tGene EMC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3119-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3119-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01681 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01142 has 0 variants, skipping\n",
      "[INFO]\tGene GORAB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RDM1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-E has 0 variants, skipping\n",
      "[INFO]\tGene MNX1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-B has 0 variants, skipping\n",
      "[INFO]\tGene DEF8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1295A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1295B has 0 variants, skipping\n",
      "[INFO]\tGene VWA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCMT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf153 has 0 variants, skipping\n",
      "[INFO]\tGene NRDC has 0 variants, skipping\n",
      "[INFO]\tGene NASP has 0 variants, skipping\n",
      "[INFO]\tGene TEX28 has 0 variants, skipping\n",
      "[INFO]\tGene CNPPD1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT8 has 0 variants, skipping\n",
      "[INFO]\tGene TWSG1 has 0 variants, skipping\n",
      "[INFO]\tGene JPH1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL9A1 has 0 variants, skipping\n",
      "[INFO]\tGene CLPB has 0 variants, skipping\n",
      "[INFO]\tGene DNM3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene QRFPR has 0 variants, skipping\n",
      "[INFO]\tGene SPATA32 has 0 variants, skipping\n",
      "[INFO]\tGene CDYL2 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA2L has 0 variants, skipping\n",
      "[INFO]\tGene ZG16 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf46 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM266 has 0 variants, skipping\n",
      "[INFO]\tGene TARS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506023 has 0 variants, skipping\n",
      "[INFO]\tGene MPZ has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC8 has 0 variants, skipping\n",
      "[INFO]\tGene DLG4 has 0 variants, skipping\n",
      "[INFO]\tGene MB has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA12 has 0 variants, skipping\n",
      "[INFO]\tGene HELB has 0 variants, skipping\n",
      "[INFO]\tGene RIPK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF430 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRNP has 0 variants, skipping\n",
      "[INFO]\tGene GNAL has 0 variants, skipping\n",
      "[INFO]\tGene AIFM3 has 0 variants, skipping\n",
      "[INFO]\tGene NOP16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01355 has 0 variants, skipping\n",
      "[INFO]\tGene UGP2 has 0 variants, skipping\n",
      "[INFO]\tGene GUCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD81 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD47 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD80 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD79 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD78 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA103 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD77 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP94 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD75 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD74 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF717 has 0 variants, skipping\n",
      "[INFO]\tGene OPALIN has 0 variants, skipping\n",
      "[INFO]\tGene LRIG2-DT has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2 has 0 variants, skipping\n",
      "[INFO]\tGene WARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400499 has 0 variants, skipping\n",
      "[INFO]\tGene DUS4L-BCAP29 has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L has 0 variants, skipping\n",
      "[INFO]\tGene STAC3 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L3B has 0 variants, skipping\n",
      "[INFO]\tGene ITGB4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF205 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1B1 has 0 variants, skipping\n",
      "[INFO]\tGene IMMT has 0 variants, skipping\n",
      "[INFO]\tGene LUZP4 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D19 has 0 variants, skipping\n",
      "[INFO]\tGene FAM160B1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT80 has 0 variants, skipping\n",
      "[INFO]\tGene TOP3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01657 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K4 has 0 variants, skipping\n",
      "[INFO]\tGene PLD3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCG2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB14 has 0 variants, skipping\n",
      "[INFO]\tGene WIPI2 has 0 variants, skipping\n",
      "[INFO]\tGene ICA1L has 0 variants, skipping\n",
      "[INFO]\tGene TTC23 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC130 has 0 variants, skipping\n",
      "[INFO]\tGene GDF15 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC2 has 0 variants, skipping\n",
      "[INFO]\tGene LITAF has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP39 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01645 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01741 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD5 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB4 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EXO1 has 0 variants, skipping\n",
      "[INFO]\tGene MTFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC16 has 0 variants, skipping\n",
      "[INFO]\tGene FTX has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL2P has 0 variants, skipping\n",
      "[INFO]\tGene RASAL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4424 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL6 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD28 has 0 variants, skipping\n",
      "[INFO]\tGene HERC4 has 0 variants, skipping\n",
      "[INFO]\tGene ST14 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2L has 0 variants, skipping\n",
      "[INFO]\tGene SMG7 has 0 variants, skipping\n",
      "[INFO]\tGene FMC1-LUC7L2 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW3 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3F has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC9 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB3 has 0 variants, skipping\n",
      "[INFO]\tGene DMTN has 0 variants, skipping\n",
      "[INFO]\tGene ATG7 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT44 has 0 variants, skipping\n",
      "[INFO]\tGene STH has 0 variants, skipping\n",
      "[INFO]\tGene NKAPL has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF38B has 0 variants, skipping\n",
      "[INFO]\tGene SCML1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB5 has 0 variants, skipping\n",
      "[INFO]\tGene ADD1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3121 has 0 variants, skipping\n",
      "[INFO]\tGene AASDH has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5B has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH4 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1614-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX17 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF6 has 0 variants, skipping\n",
      "[INFO]\tGene DDTL has 0 variants, skipping\n",
      "[INFO]\tGene TDRKH has 0 variants, skipping\n",
      "[INFO]\tGene LCN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01732 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01699 has 0 variants, skipping\n",
      "[INFO]\tGene MITF has 0 variants, skipping\n",
      "[INFO]\tGene WDR44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF461 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL7 has 0 variants, skipping\n",
      "[INFO]\tGene PBRM1 has 0 variants, skipping\n",
      "[INFO]\tGene APOM has 0 variants, skipping\n",
      "[INFO]\tGene LINC01344 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00272 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf65 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK2 has 0 variants, skipping\n",
      "[INFO]\tGene CTPS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01686 has 0 variants, skipping\n",
      "[INFO]\tGene DMXL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01688 has 0 variants, skipping\n",
      "[INFO]\tGene SCN3B has 0 variants, skipping\n",
      "[INFO]\tGene OOEP has 0 variants, skipping\n",
      "[INFO]\tGene BANP has 0 variants, skipping\n",
      "[INFO]\tGene LOC647070 has 0 variants, skipping\n",
      "[INFO]\tGene PDIA6 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC3 has 0 variants, skipping\n",
      "[INFO]\tGene MTERF3 has 0 variants, skipping\n",
      "[INFO]\tGene DERL2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1H has 0 variants, skipping\n",
      "[INFO]\tGene SNX20 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E2A has 0 variants, skipping\n",
      "[INFO]\tGene LAMC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8N has 0 variants, skipping\n",
      "[INFO]\tGene C19orf48 has 0 variants, skipping\n",
      "[INFO]\tGene MICA has 0 variants, skipping\n",
      "[INFO]\tGene TBATA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF43 has 0 variants, skipping\n",
      "[INFO]\tGene MZF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF35 has 0 variants, skipping\n",
      "[INFO]\tGene SMG7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABR has 0 variants, skipping\n",
      "[INFO]\tGene TTTY15 has 0 variants, skipping\n",
      "[INFO]\tGene ABTB2 has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF511 has 0 variants, skipping\n",
      "[INFO]\tGene ISL2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNB1 has 0 variants, skipping\n",
      "[INFO]\tGene UIMC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD3 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD2 has 0 variants, skipping\n",
      "[INFO]\tGene RCC2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM13 has 0 variants, skipping\n",
      "[INFO]\tGene GPR63 has 0 variants, skipping\n",
      "[INFO]\tGene ADTRP has 0 variants, skipping\n",
      "[INFO]\tGene UNC93A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD33 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR2 has 0 variants, skipping\n",
      "[INFO]\tGene MMP17 has 0 variants, skipping\n",
      "[INFO]\tGene RAB4B has 0 variants, skipping\n",
      "[INFO]\tGene HSFX1 has 0 variants, skipping\n",
      "[INFO]\tGene RARB has 0 variants, skipping\n",
      "[INFO]\tGene TAOK2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf80 has 0 variants, skipping\n",
      "[INFO]\tGene MYLPF has 0 variants, skipping\n",
      "[INFO]\tGene LINC01633 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM220 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM5 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC138 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-279B7.1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01350 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR3 has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene DISP1 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CES1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KB1 has 0 variants, skipping\n",
      "[INFO]\tGene RTEL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644090 has 0 variants, skipping\n",
      "[INFO]\tGene PINLYP has 0 variants, skipping\n",
      "[INFO]\tGene EYS has 0 variants, skipping\n",
      "[INFO]\tGene MINDY1 has 0 variants, skipping\n",
      "[INFO]\tGene CIB1 has 0 variants, skipping\n",
      "[INFO]\tGene PACERR has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01036 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01037 has 0 variants, skipping\n",
      "[INFO]\tGene IL27RA has 0 variants, skipping\n",
      "[INFO]\tGene AKAP7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01680 has 0 variants, skipping\n",
      "[INFO]\tGene MIER2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4426 has 0 variants, skipping\n",
      "[INFO]\tGene GNPAT has 0 variants, skipping\n",
      "[INFO]\tGene MAF has 0 variants, skipping\n",
      "[INFO]\tGene LLGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC5A has 0 variants, skipping\n",
      "[INFO]\tGene CDHR3 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNFX1 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA18B has 0 variants, skipping\n",
      "[INFO]\tGene FGFRL1 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA1 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1L has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL has 0 variants, skipping\n",
      "[INFO]\tGene MIR1278 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01031 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01724 has 0 variants, skipping\n",
      "[INFO]\tGene CELSR3 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBIS has 0 variants, skipping\n",
      "[INFO]\tGene KIF1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OBP2A has 0 variants, skipping\n",
      "[INFO]\tGene SEC61A2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1C has 0 variants, skipping\n",
      "[INFO]\tGene SYTL3 has 0 variants, skipping\n",
      "[INFO]\tGene DTNB has 0 variants, skipping\n",
      "[INFO]\tGene GATD3B has 0 variants, skipping\n",
      "[INFO]\tGene ECPAS has 0 variants, skipping\n",
      "[INFO]\tGene DIS3 has 0 variants, skipping\n",
      "[INFO]\tGene BATF3 has 0 variants, skipping\n",
      "[INFO]\tGene PXMP2 has 0 variants, skipping\n",
      "[INFO]\tGene DISC1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB103B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF395 has 0 variants, skipping\n",
      "[INFO]\tGene CYTL1 has 0 variants, skipping\n",
      "[INFO]\tGene MYNN has 0 variants, skipping\n",
      "[INFO]\tGene LENEP has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5D has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5C has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6B has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN7 has 0 variants, skipping\n",
      "[INFO]\tGene MACROH2A2 has 0 variants, skipping\n",
      "[INFO]\tGene MARK1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP10 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF19 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A1HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPL42 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01222 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01221 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00862 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10 has 0 variants, skipping\n",
      "[INFO]\tGene MEIKIN has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CD226 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX2 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene AZIN2 has 0 variants, skipping\n",
      "[INFO]\tGene NOD2 has 0 variants, skipping\n",
      "[INFO]\tGene VBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP4 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEFL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02709 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINI2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6H has 0 variants, skipping\n",
      "[INFO]\tGene THAP5 has 0 variants, skipping\n",
      "[INFO]\tGene DYSF has 0 variants, skipping\n",
      "[INFO]\tGene CNST has 0 variants, skipping\n",
      "[INFO]\tGene LBHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MGST3 has 0 variants, skipping\n",
      "[INFO]\tGene MEOX1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM2 has 0 variants, skipping\n",
      "[INFO]\tGene LRP2 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH2L has 0 variants, skipping\n",
      "[INFO]\tGene MROH3P has 0 variants, skipping\n",
      "[INFO]\tGene NGLY1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO2A has 0 variants, skipping\n",
      "[INFO]\tGene IKBIP has 0 variants, skipping\n",
      "[INFO]\tGene GJD2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD14 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MID2 has 0 variants, skipping\n",
      "[INFO]\tGene CARD9 has 0 variants, skipping\n",
      "[INFO]\tGene BAG6 has 0 variants, skipping\n",
      "[INFO]\tGene COBL has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K8 has 0 variants, skipping\n",
      "[INFO]\tGene SPIB has 0 variants, skipping\n",
      "[INFO]\tGene HMOX2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB40 has 0 variants, skipping\n",
      "[INFO]\tGene AIF1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA3 has 0 variants, skipping\n",
      "[INFO]\tGene PPAN-P2RY11 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10P7 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5191 has 0 variants, skipping\n",
      "[INFO]\tGene HES6 has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TREM1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST12 has 0 variants, skipping\n",
      "[INFO]\tGene WSB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6739 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70H has 0 variants, skipping\n",
      "[INFO]\tGene SEC23B has 0 variants, skipping\n",
      "[INFO]\tGene AMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6740 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B2 has 0 variants, skipping\n",
      "[INFO]\tGene CPTP has 0 variants, skipping\n",
      "[INFO]\tGene ALK has 0 variants, skipping\n",
      "[INFO]\tGene PTPRVP has 0 variants, skipping\n",
      "[INFO]\tGene ELF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCCS has 0 variants, skipping\n",
      "[INFO]\tGene CTSG has 0 variants, skipping\n",
      "[INFO]\tGene GDA has 0 variants, skipping\n",
      "[INFO]\tGene LINC00997 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00673 has 0 variants, skipping\n",
      "[INFO]\tGene SBF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FEZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL5 has 0 variants, skipping\n",
      "[INFO]\tGene FENDRR has 0 variants, skipping\n",
      "[INFO]\tGene MIR4288 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4262 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4257 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4319 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3198-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320E has 0 variants, skipping\n",
      "[INFO]\tGene CPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CPM has 0 variants, skipping\n",
      "[INFO]\tGene CPE has 0 variants, skipping\n",
      "[INFO]\tGene CPB2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4EP has 0 variants, skipping\n",
      "[INFO]\tGene KRTCAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA12 has 0 variants, skipping\n",
      "[INFO]\tGene GSC has 0 variants, skipping\n",
      "[INFO]\tGene SPACA3 has 0 variants, skipping\n",
      "[INFO]\tGene TPPP2 has 0 variants, skipping\n",
      "[INFO]\tGene MALT1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RN has 0 variants, skipping\n",
      "[INFO]\tGene CBR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103B1 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRV1 has 0 variants, skipping\n",
      "[INFO]\tGene F5 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3IP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM3 has 0 variants, skipping\n",
      "[INFO]\tGene GPHN has 0 variants, skipping\n",
      "[INFO]\tGene GRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DACH1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3622A has 0 variants, skipping\n",
      "[INFO]\tGene MIR23C has 0 variants, skipping\n",
      "[INFO]\tGene JDP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3909 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01353 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01136 has 0 variants, skipping\n",
      "[INFO]\tGene GRID1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM168A has 0 variants, skipping\n",
      "[INFO]\tGene PMPCA has 0 variants, skipping\n",
      "[INFO]\tGene KANK1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA77 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00260 has 0 variants, skipping\n",
      "[INFO]\tGene CYLD has 0 variants, skipping\n",
      "[INFO]\tGene ADD3 has 0 variants, skipping\n",
      "[INFO]\tGene AGMO has 0 variants, skipping\n",
      "[INFO]\tGene NEO1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE1B has 0 variants, skipping\n",
      "[INFO]\tGene JMJD7-PLA2G4B has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF275 has 0 variants, skipping\n",
      "[INFO]\tGene ARRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB5 has 0 variants, skipping\n",
      "[INFO]\tGene CALCRL has 0 variants, skipping\n",
      "[INFO]\tGene CCNY has 0 variants, skipping\n",
      "[INFO]\tGene CMTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00303 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS6 has 0 variants, skipping\n",
      "[INFO]\tGene JADE2 has 0 variants, skipping\n",
      "[INFO]\tGene BOC has 0 variants, skipping\n",
      "[INFO]\tGene DDX60L has 0 variants, skipping\n",
      "[INFO]\tGene RABL2B has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B7 has 0 variants, skipping\n",
      "[INFO]\tGene TJAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM20 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM21 has 0 variants, skipping\n",
      "[INFO]\tGene CEP19 has 0 variants, skipping\n",
      "[INFO]\tGene DAB1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO1 has 0 variants, skipping\n",
      "[INFO]\tGene EPDR1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-C has 0 variants, skipping\n",
      "[INFO]\tGene RNF146 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DS3 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1A has 0 variants, skipping\n",
      "[INFO]\tGene RHOH has 0 variants, skipping\n",
      "[INFO]\tGene CDH22 has 0 variants, skipping\n",
      "[INFO]\tGene ARL13B has 0 variants, skipping\n",
      "[INFO]\tGene GMEB1 has 0 variants, skipping\n",
      "[INFO]\tGene PSTPIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG2 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAV has 0 variants, skipping\n",
      "[INFO]\tGene PWWP2A has 0 variants, skipping\n",
      "[INFO]\tGene NCSTN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF334 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725023 has 0 variants, skipping\n",
      "[INFO]\tGene LCORL has 0 variants, skipping\n",
      "[INFO]\tGene PUF60 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284581 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF443 has 0 variants, skipping\n",
      "[INFO]\tGene GPA33 has 0 variants, skipping\n",
      "[INFO]\tGene PRKD3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103021295 has 0 variants, skipping\n",
      "[INFO]\tGene GPR141 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122B has 0 variants, skipping\n",
      "[INFO]\tGene THAP8 has 0 variants, skipping\n",
      "[INFO]\tGene PKN2 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6L has 0 variants, skipping\n",
      "[INFO]\tGene OTUB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDE1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS50 has 0 variants, skipping\n",
      "[INFO]\tGene TANC2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO33 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1C has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6769B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF66 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC2B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD5 has 0 variants, skipping\n",
      "[INFO]\tGene SPHK1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSA has 0 variants, skipping\n",
      "[INFO]\tGene USP6 has 0 variants, skipping\n",
      "[INFO]\tGene RNF10 has 0 variants, skipping\n",
      "[INFO]\tGene BLM has 0 variants, skipping\n",
      "[INFO]\tGene BCKDHB has 0 variants, skipping\n",
      "[INFO]\tGene BCHE has 0 variants, skipping\n",
      "[INFO]\tGene ATP7B has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG7 has 0 variants, skipping\n",
      "[INFO]\tGene URGCP-MRPS24 has 0 variants, skipping\n",
      "[INFO]\tGene MIER1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF195 has 0 variants, skipping\n",
      "[INFO]\tGene LIAS has 0 variants, skipping\n",
      "[INFO]\tGene CAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23A has 0 variants, skipping\n",
      "[INFO]\tGene PDIA5 has 0 variants, skipping\n",
      "[INFO]\tGene TOGARAM1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD3 has 0 variants, skipping\n",
      "[INFO]\tGene BCAS3 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR8 has 0 variants, skipping\n",
      "[INFO]\tGene GIN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDHR2 has 0 variants, skipping\n",
      "[INFO]\tGene SWT1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL6 has 0 variants, skipping\n",
      "[INFO]\tGene GPAM has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST8 has 0 variants, skipping\n",
      "[INFO]\tGene AZI2 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1A has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN10 has 0 variants, skipping\n",
      "[INFO]\tGene SQLE has 0 variants, skipping\n",
      "[INFO]\tGene TIMM10 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS2 has 0 variants, skipping\n",
      "[INFO]\tGene PJVK has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B2CHG has 0 variants, skipping\n",
      "[INFO]\tGene APBB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC148696 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01717 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01774 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01696 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01698 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY3AP has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A22 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPD has 0 variants, skipping\n",
      "[INFO]\tGene MIR5193 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A8 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPA1 has 0 variants, skipping\n",
      "[INFO]\tGene AGO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4260 has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE2 has 0 variants, skipping\n",
      "[INFO]\tGene RB1-DT has 0 variants, skipping\n",
      "[INFO]\tGene BANK1 has 0 variants, skipping\n",
      "[INFO]\tGene DEF6 has 0 variants, skipping\n",
      "[INFO]\tGene BCORL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK14 has 0 variants, skipping\n",
      "[INFO]\tGene MTBP has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A23 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf93 has 0 variants, skipping\n",
      "[INFO]\tGene SDF2L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00595 has 0 variants, skipping\n",
      "[INFO]\tGene PACRGL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF846 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM26 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene APOC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX4 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD5 has 0 variants, skipping\n",
      "[INFO]\tGene KAT14 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FUS has 0 variants, skipping\n",
      "[INFO]\tGene METTL21A has 0 variants, skipping\n",
      "[INFO]\tGene RNF6 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND8 has 0 variants, skipping\n",
      "[INFO]\tGene NIN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01693 has 0 variants, skipping\n",
      "[INFO]\tGene PKM has 0 variants, skipping\n",
      "[INFO]\tGene LOC91548 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH8 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT4 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene LPGAT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA9 has 0 variants, skipping\n",
      "[INFO]\tGene PRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene STK26 has 0 variants, skipping\n",
      "[INFO]\tGene INTS7 has 0 variants, skipping\n",
      "[INFO]\tGene IL31RA has 0 variants, skipping\n",
      "[INFO]\tGene AHCYL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21P28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02608 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGED4B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA16B has 0 variants, skipping\n",
      "[INFO]\tGene CD300A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF382 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY3 has 0 variants, skipping\n",
      "[INFO]\tGene SSX2IP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM176B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS28 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR2 has 0 variants, skipping\n",
      "[INFO]\tGene SACM1L has 0 variants, skipping\n",
      "[INFO]\tGene DEXI has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP200 has 0 variants, skipping\n",
      "[INFO]\tGene REM1 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS5 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN2 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B7 has 0 variants, skipping\n",
      "[INFO]\tGene GCFC2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A6 has 0 variants, skipping\n",
      "[INFO]\tGene ANGEL2 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4C has 0 variants, skipping\n",
      "[INFO]\tGene AKIRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP92 has 0 variants, skipping\n",
      "[INFO]\tGene IFFO2 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6 has 0 variants, skipping\n",
      "[INFO]\tGene ELAPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR155 has 0 variants, skipping\n",
      "[INFO]\tGene UNC45B has 0 variants, skipping\n",
      "[INFO]\tGene CASP7 has 0 variants, skipping\n",
      "[INFO]\tGene RIBC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX17 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHB2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L4 has 0 variants, skipping\n",
      "[INFO]\tGene ARFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KC1 has 0 variants, skipping\n",
      "[INFO]\tGene PROX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00538 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND12 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L2 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL4 has 0 variants, skipping\n",
      "[INFO]\tGene RPL34 has 0 variants, skipping\n",
      "[INFO]\tGene IL12RB2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0040 has 0 variants, skipping\n",
      "[INFO]\tGene CEP131 has 0 variants, skipping\n",
      "[INFO]\tGene NEU1 has 0 variants, skipping\n",
      "[INFO]\tGene NCF2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB2 has 0 variants, skipping\n",
      "[INFO]\tGene CALCOCO2 has 0 variants, skipping\n",
      "[INFO]\tGene AP3S2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS31 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPR has 0 variants, skipping\n",
      "[INFO]\tGene ANO1 has 0 variants, skipping\n",
      "[INFO]\tGene PPEF1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM15 has 0 variants, skipping\n",
      "[INFO]\tGene SGIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITSN2 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf60 has 0 variants, skipping\n",
      "[INFO]\tGene CPA4 has 0 variants, skipping\n",
      "[INFO]\tGene RCBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHN1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT6B1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01653 has 0 variants, skipping\n",
      "[INFO]\tGene KIFAP3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARF1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM41B has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F3 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01710 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1-DT has 0 variants, skipping\n",
      "[INFO]\tGene NUDT17 has 0 variants, skipping\n",
      "[INFO]\tGene STYX has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CA has 0 variants, skipping\n",
      "[INFO]\tGene MIA2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLAL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KTN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506388 has 0 variants, skipping\n",
      "[INFO]\tGene ATF7IP2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8H has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A1 has 0 variants, skipping\n",
      "[INFO]\tGene CBFB has 0 variants, skipping\n",
      "[INFO]\tGene BRMS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36B has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6ATAC35P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01352 has 0 variants, skipping\n",
      "[INFO]\tGene HLX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2D has 0 variants, skipping\n",
      "[INFO]\tGene LINC01655 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02257 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02474 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GYS1 has 0 variants, skipping\n",
      "[INFO]\tGene F3 has 0 variants, skipping\n",
      "[INFO]\tGene GRSF1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPG has 0 variants, skipping\n",
      "[INFO]\tGene PROZ has 0 variants, skipping\n",
      "[INFO]\tGene CAPG has 0 variants, skipping\n",
      "[INFO]\tGene FSBP has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1BP1L has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20H has 0 variants, skipping\n",
      "[INFO]\tGene VDR has 0 variants, skipping\n",
      "[INFO]\tGene FAM177B has 0 variants, skipping\n",
      "[INFO]\tGene IARS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPP8 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf54 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927164 has 0 variants, skipping\n",
      "[INFO]\tGene DAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY10 has 0 variants, skipping\n",
      "[INFO]\tGene NCR3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM164 has 0 variants, skipping\n",
      "[INFO]\tGene ESRP1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP3 has 0 variants, skipping\n",
      "[INFO]\tGene CDK19 has 0 variants, skipping\n",
      "[INFO]\tGene MORN1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLT1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4742 has 0 variants, skipping\n",
      "[INFO]\tGene MDGA2 has 0 variants, skipping\n",
      "[INFO]\tGene LILRB3 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A3 has 0 variants, skipping\n",
      "[INFO]\tGene CLASP1 has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD2L has 0 variants, skipping\n",
      "[INFO]\tGene PYCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MATR3 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN5 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA3 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH1 has 0 variants, skipping\n",
      "[INFO]\tGene BPGM has 0 variants, skipping\n",
      "[INFO]\tGene SREK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6741 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF106 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM216 has 0 variants, skipping\n",
      "[INFO]\tGene MZT2B has 0 variants, skipping\n",
      "[INFO]\tGene TVP23A has 0 variants, skipping\n",
      "[INFO]\tGene ACBD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB18 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC42 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1A2 has 0 variants, skipping\n",
      "[INFO]\tGene GPAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ATPAF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene HVCN1 has 0 variants, skipping\n",
      "[INFO]\tGene USP22 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC4 has 0 variants, skipping\n",
      "[INFO]\tGene SRP19 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKB-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLA has 0 variants, skipping\n",
      "[INFO]\tGene USP17L25 has 0 variants, skipping\n",
      "[INFO]\tGene PDE7A has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K13 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD4 has 0 variants, skipping\n",
      "[INFO]\tGene RNF144A has 0 variants, skipping\n",
      "[INFO]\tGene IGFN1 has 0 variants, skipping\n",
      "[INFO]\tGene LDLR has 0 variants, skipping\n",
      "[INFO]\tGene MLLT10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF678 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF847P has 0 variants, skipping\n",
      "[INFO]\tGene DDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CARNMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC-ROR has 0 variants, skipping\n",
      "[INFO]\tGene HDDC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF345 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5008 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985355 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3620 has 0 variants, skipping\n",
      "[INFO]\tGene RAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A7 has 0 variants, skipping\n",
      "[INFO]\tGene MOG has 0 variants, skipping\n",
      "[INFO]\tGene CFB has 0 variants, skipping\n",
      "[INFO]\tGene SBK2 has 0 variants, skipping\n",
      "[INFO]\tGene PODNL1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4B has 0 variants, skipping\n",
      "[INFO]\tGene IBA57-DT has 0 variants, skipping\n",
      "[INFO]\tGene OBSCN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BECN1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM32 has 0 variants, skipping\n",
      "[INFO]\tGene FHL5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-8 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP36 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF131 has 0 variants, skipping\n",
      "[INFO]\tGene OARD1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM136A has 0 variants, skipping\n",
      "[INFO]\tGene ABCD4 has 0 variants, skipping\n",
      "[INFO]\tGene PNISR has 0 variants, skipping\n",
      "[INFO]\tGene MIR6742 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4666A has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S5 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S6 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S7 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S8 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S10 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S11 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S12 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S13 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S14 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S15 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S16 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S17 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5S9 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP5P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373159 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11A-SCG5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927478 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01682 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01736 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM17 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01737 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF568 has 0 variants, skipping\n",
      "[INFO]\tGene FBF1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C has 0 variants, skipping\n",
      "[INFO]\tGene WASHC3 has 0 variants, skipping\n",
      "[INFO]\tGene NOX4 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927604 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS12 has 0 variants, skipping\n",
      "[INFO]\tGene CASZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3C has 0 variants, skipping\n",
      "[INFO]\tGene CSNKA2IP has 0 variants, skipping\n",
      "[INFO]\tGene SMIM28 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP4B has 0 variants, skipping\n",
      "[INFO]\tGene SCN4B has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF25 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A2 has 0 variants, skipping\n",
      "[INFO]\tGene POGK has 0 variants, skipping\n",
      "[INFO]\tGene F9 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD2P2 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAX-DISC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00582 has 0 variants, skipping\n",
      "[INFO]\tGene APOC1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD2 has 0 variants, skipping\n",
      "[INFO]\tGene PNKD has 0 variants, skipping\n",
      "[INFO]\tGene RWDD3 has 0 variants, skipping\n",
      "[INFO]\tGene SYF2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD2 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN3 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM4 has 0 variants, skipping\n",
      "[INFO]\tGene PSME3 has 0 variants, skipping\n",
      "[INFO]\tGene IFI35 has 0 variants, skipping\n",
      "[INFO]\tGene DISC1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01745 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01744 has 0 variants, skipping\n",
      "[INFO]\tGene EGF has 0 variants, skipping\n",
      "[INFO]\tGene MIR4427 has 0 variants, skipping\n",
      "[INFO]\tGene AMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4671 has 0 variants, skipping\n",
      "[INFO]\tGene COA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A9 has 0 variants, skipping\n",
      "[INFO]\tGene FLNB has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A11 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene A1CF has 0 variants, skipping\n",
      "[INFO]\tGene SH2D6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927787 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01132 has 0 variants, skipping\n",
      "[INFO]\tGene PP2672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927851 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC4 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1L has 0 variants, skipping\n",
      "[INFO]\tGene SNORA14B has 0 variants, skipping\n",
      "[INFO]\tGene LIMS4 has 0 variants, skipping\n",
      "[INFO]\tGene IRF4 has 0 variants, skipping\n",
      "[INFO]\tGene ASCC1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3L has 0 variants, skipping\n",
      "[INFO]\tGene HTR2A has 0 variants, skipping\n",
      "[INFO]\tGene NPR2 has 0 variants, skipping\n",
      "[INFO]\tGene ARSA has 0 variants, skipping\n",
      "[INFO]\tGene FCGBP has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K2 has 0 variants, skipping\n",
      "[INFO]\tGene KTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR89B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4753 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7A has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2LI1 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS4 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND10 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM33 has 0 variants, skipping\n",
      "[INFO]\tGene GMNN has 0 variants, skipping\n",
      "[INFO]\tGene TRIM24 has 0 variants, skipping\n",
      "[INFO]\tGene STMN3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5B has 0 variants, skipping\n",
      "[INFO]\tGene IFT52 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD2 has 0 variants, skipping\n",
      "[INFO]\tGene FCHO1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPIND1 has 0 variants, skipping\n",
      "[INFO]\tGene HBG2 has 0 variants, skipping\n",
      "[INFO]\tGene HADHB has 0 variants, skipping\n",
      "[INFO]\tGene HADHA has 0 variants, skipping\n",
      "[INFO]\tGene MIR1537 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6872 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6778 has 0 variants, skipping\n",
      "[INFO]\tGene MDM1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS3 has 0 variants, skipping\n",
      "[INFO]\tGene GCM2 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CRLF1 has 0 variants, skipping\n",
      "[INFO]\tGene TBRG4 has 0 variants, skipping\n",
      "[INFO]\tGene CCPG1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH1 has 0 variants, skipping\n",
      "[INFO]\tGene ART1 has 0 variants, skipping\n",
      "[INFO]\tGene ARR3 has 0 variants, skipping\n",
      "[INFO]\tGene C1RL has 0 variants, skipping\n",
      "[INFO]\tGene RBP5 has 0 variants, skipping\n",
      "[INFO]\tGene DAXX has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130331 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS7P5 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN2 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1B has 0 variants, skipping\n",
      "[INFO]\tGene PTPN22 has 0 variants, skipping\n",
      "[INFO]\tGene NMT2 has 0 variants, skipping\n",
      "[INFO]\tGene H2AW has 0 variants, skipping\n",
      "[INFO]\tGene MIR190B has 0 variants, skipping\n",
      "[INFO]\tGene TP53 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8Q has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A has 0 variants, skipping\n",
      "[INFO]\tGene NFU1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF431 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR31HG has 0 variants, skipping\n",
      "[INFO]\tGene LAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5W2 has 0 variants, skipping\n",
      "[INFO]\tGene OR11L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3123 has 0 variants, skipping\n",
      "[INFO]\tGene SMN1 has 0 variants, skipping\n",
      "[INFO]\tGene SWAP70 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML2B has 0 variants, skipping\n",
      "[INFO]\tGene PUM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132202 has 0 variants, skipping\n",
      "[INFO]\tGene CES3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01347 has 0 variants, skipping\n",
      "[INFO]\tGene BCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF185 has 0 variants, skipping\n",
      "[INFO]\tGene UNKL has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5L has 0 variants, skipping\n",
      "[INFO]\tGene UBE2V1 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4677 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS7 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS3 has 0 variants, skipping\n",
      "[INFO]\tGene LBR has 0 variants, skipping\n",
      "[INFO]\tGene H3C7 has 0 variants, skipping\n",
      "[INFO]\tGene PSG3 has 0 variants, skipping\n",
      "[INFO]\tGene SSX5 has 0 variants, skipping\n",
      "[INFO]\tGene SSX3 has 0 variants, skipping\n",
      "[INFO]\tGene FABP7 has 0 variants, skipping\n",
      "[INFO]\tGene MEA1 has 0 variants, skipping\n",
      "[INFO]\tGene EAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TPRKB has 0 variants, skipping\n",
      "[INFO]\tGene SPDL1 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA100 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928068 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF26B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01743 has 0 variants, skipping\n",
      "[INFO]\tGene MC2R has 0 variants, skipping\n",
      "[INFO]\tGene CORO1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01341 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf83 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR72 has 0 variants, skipping\n",
      "[INFO]\tGene RNF144B has 0 variants, skipping\n",
      "[INFO]\tGene SPDYA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF438 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1L has 0 variants, skipping\n",
      "[INFO]\tGene MCM10 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA4 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF670-ZNF695 has 0 variants, skipping\n",
      "[INFO]\tGene MICALL2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ39095 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF436 has 0 variants, skipping\n",
      "[INFO]\tGene GPBAR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3916 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD2 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1A has 0 variants, skipping\n",
      "[INFO]\tGene SMIM33 has 0 variants, skipping\n",
      "[INFO]\tGene GCSAML-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SKP2 has 0 variants, skipping\n",
      "[INFO]\tGene HIP1R has 0 variants, skipping\n",
      "[INFO]\tGene ZNF570 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L1P has 0 variants, skipping\n",
      "[INFO]\tGene DPP9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M1P has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A4 has 0 variants, skipping\n",
      "[INFO]\tGene OSM has 0 variants, skipping\n",
      "[INFO]\tGene MTX2 has 0 variants, skipping\n",
      "[INFO]\tGene RDH16 has 0 variants, skipping\n",
      "[INFO]\tGene OPCML has 0 variants, skipping\n",
      "[INFO]\tGene LYPD9P has 0 variants, skipping\n",
      "[INFO]\tGene CDKL4 has 0 variants, skipping\n",
      "[INFO]\tGene EGFR has 0 variants, skipping\n",
      "[INFO]\tGene TMEM183A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3124 has 0 variants, skipping\n",
      "[INFO]\tGene BICDL2 has 0 variants, skipping\n",
      "[INFO]\tGene CSAG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02564 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723376 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DMA has 0 variants, skipping\n",
      "[INFO]\tGene HAX1 has 0 variants, skipping\n",
      "[INFO]\tGene ECI2 has 0 variants, skipping\n",
      "[INFO]\tGene TAB1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAME has 0 variants, skipping\n",
      "[INFO]\tGene TOMM40 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5699 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT1 has 0 variants, skipping\n",
      "[INFO]\tGene GPAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927762 has 0 variants, skipping\n",
      "[INFO]\tGene HRAS has 0 variants, skipping\n",
      "[INFO]\tGene IDI2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNP1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00200 has 0 variants, skipping\n",
      "[INFO]\tGene ADARB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00700 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6072 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00701 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02645 has 0 variants, skipping\n",
      "[INFO]\tGene SPI1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1C has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E2 has 0 variants, skipping\n",
      "[INFO]\tGene LCE1C has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A12 has 0 variants, skipping\n",
      "[INFO]\tGene F11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF138 has 0 variants, skipping\n",
      "[INFO]\tGene NAT1 has 0 variants, skipping\n",
      "[INFO]\tGene HDX has 0 variants, skipping\n",
      "[INFO]\tGene GCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD142 has 0 variants, skipping\n",
      "[INFO]\tGene PACS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLK7 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4A has 0 variants, skipping\n",
      "[INFO]\tGene GRM4 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO3B has 0 variants, skipping\n",
      "[INFO]\tGene WDFY4 has 0 variants, skipping\n",
      "[INFO]\tGene ISG20L2 has 0 variants, skipping\n",
      "[INFO]\tGene DTYMK has 0 variants, skipping\n",
      "[INFO]\tGene USP46-DT has 0 variants, skipping\n",
      "[INFO]\tGene SFTA3 has 0 variants, skipping\n",
      "[INFO]\tGene MCRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL4 has 0 variants, skipping\n",
      "[INFO]\tGene AKT1S1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L1 has 0 variants, skipping\n",
      "[INFO]\tGene GPSM1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2F has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2 has 0 variants, skipping\n",
      "[INFO]\tGene PITRM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02669 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02639 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6078 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927964 has 0 variants, skipping\n",
      "[INFO]\tGene MIR339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00705 has 0 variants, skipping\n",
      "[INFO]\tGene RNF135 has 0 variants, skipping\n",
      "[INFO]\tGene NIT1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1E2 has 0 variants, skipping\n",
      "[INFO]\tGene CYB561D1 has 0 variants, skipping\n",
      "[INFO]\tGene OSTC has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C6P has 0 variants, skipping\n",
      "[INFO]\tGene ITGA6 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C8P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02561 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC2-KCTD14 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF41 has 0 variants, skipping\n",
      "[INFO]\tGene DPH1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB49 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX3 has 0 variants, skipping\n",
      "[INFO]\tGene RGMB has 0 variants, skipping\n",
      "[INFO]\tGene SPIDR has 0 variants, skipping\n",
      "[INFO]\tGene AHNAK has 0 variants, skipping\n",
      "[INFO]\tGene FKBP6 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA5 has 0 variants, skipping\n",
      "[INFO]\tGene ATF7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF15 has 0 variants, skipping\n",
      "[INFO]\tGene WIPI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM118A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX2 has 0 variants, skipping\n",
      "[INFO]\tGene SKA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3155A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3155B has 0 variants, skipping\n",
      "[INFO]\tGene COMMD9 has 0 variants, skipping\n",
      "[INFO]\tGene UGGT1 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD2B has 0 variants, skipping\n",
      "[INFO]\tGene TRMT2B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02649 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02656 has 0 variants, skipping\n",
      "[INFO]\tGene KCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3A1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN15 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIA has 0 variants, skipping\n",
      "[INFO]\tGene PRKCQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02648 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00706 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A7 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD129 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM52B has 0 variants, skipping\n",
      "[INFO]\tGene PLB1 has 0 variants, skipping\n",
      "[INFO]\tGene TLK2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00708 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02676 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928272 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00709 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02670 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-DT has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R1B has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2A has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00710 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC4 has 0 variants, skipping\n",
      "[INFO]\tGene PCMTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANTXR2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF19 has 0 variants, skipping\n",
      "[INFO]\tGene TENT4A has 0 variants, skipping\n",
      "[INFO]\tGene CLTCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCGN has 0 variants, skipping\n",
      "[INFO]\tGene ELOB has 0 variants, skipping\n",
      "[INFO]\tGene SSR3 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PLS3 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2B has 0 variants, skipping\n",
      "[INFO]\tGene NRON has 0 variants, skipping\n",
      "[INFO]\tGene MIR371B has 0 variants, skipping\n",
      "[INFO]\tGene PROSER2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEK has 0 variants, skipping\n",
      "[INFO]\tGene PLD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AK has 0 variants, skipping\n",
      "[INFO]\tGene EVX2 has 0 variants, skipping\n",
      "[INFO]\tGene PHTF2 has 0 variants, skipping\n",
      "[INFO]\tGene SUPV3L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Q has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL3 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83A has 0 variants, skipping\n",
      "[INFO]\tGene NIBAN3 has 0 variants, skipping\n",
      "[INFO]\tGene PLAGL1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2C has 0 variants, skipping\n",
      "[INFO]\tGene FBXL22 has 0 variants, skipping\n",
      "[INFO]\tGene KSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM8 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L11 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB112 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928453 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4293 has 0 variants, skipping\n",
      "[INFO]\tGene AQP8 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC5 has 0 variants, skipping\n",
      "[INFO]\tGene XIAP has 0 variants, skipping\n",
      "[INFO]\tGene BIRC2 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC3 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR6 has 0 variants, skipping\n",
      "[INFO]\tGene DDX51 has 0 variants, skipping\n",
      "[INFO]\tGene LYNX1 has 0 variants, skipping\n",
      "[INFO]\tGene PLRG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR758 has 0 variants, skipping\n",
      "[INFO]\tGene UROC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYRFL has 0 variants, skipping\n",
      "[INFO]\tGene DRC3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3A has 0 variants, skipping\n",
      "[INFO]\tGene CINP has 0 variants, skipping\n",
      "[INFO]\tGene SSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT71 has 0 variants, skipping\n",
      "[INFO]\tGene RNF14 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1CP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1C1 has 0 variants, skipping\n",
      "[INFO]\tGene SPG7 has 0 variants, skipping\n",
      "[INFO]\tGene MORN2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAP30 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM2 has 0 variants, skipping\n",
      "[INFO]\tGene TACC1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf162 has 0 variants, skipping\n",
      "[INFO]\tGene CIRBP has 0 variants, skipping\n",
      "[INFO]\tGene XRRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD46 has 0 variants, skipping\n",
      "[INFO]\tGene LOXHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25C has 0 variants, skipping\n",
      "[INFO]\tGene LANCL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF3 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD65 has 0 variants, skipping\n",
      "[INFO]\tGene FAM240C has 0 variants, skipping\n",
      "[INFO]\tGene RAD21L1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4M has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSGALNACT1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3A has 0 variants, skipping\n",
      "[INFO]\tGene IL1R1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL55 has 0 variants, skipping\n",
      "[INFO]\tGene FAXC has 0 variants, skipping\n",
      "[INFO]\tGene GNB3 has 0 variants, skipping\n",
      "[INFO]\tGene ENO2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928834 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4675 has 0 variants, skipping\n",
      "[INFO]\tGene CST8 has 0 variants, skipping\n",
      "[INFO]\tGene ANKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene NEBL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02643 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6G has 0 variants, skipping\n",
      "[INFO]\tGene PCNA has 0 variants, skipping\n",
      "[INFO]\tGene PDE6D has 0 variants, skipping\n",
      "[INFO]\tGene PCK1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG14 has 0 variants, skipping\n",
      "[INFO]\tGene RPS19BP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130992 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP1 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF5 has 0 variants, skipping\n",
      "[INFO]\tGene SCHIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100499489 has 0 variants, skipping\n",
      "[INFO]\tGene POTEA has 0 variants, skipping\n",
      "[INFO]\tGene DMD has 0 variants, skipping\n",
      "[INFO]\tGene ERLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene WWC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376453 has 0 variants, skipping\n",
      "[INFO]\tGene MAOA has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN4 has 0 variants, skipping\n",
      "[INFO]\tGene VPS16 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A2 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP98 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF264 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR195 has 0 variants, skipping\n",
      "[INFO]\tGene LINCADL has 0 variants, skipping\n",
      "[INFO]\tGene PSMC3IP has 0 variants, skipping\n",
      "[INFO]\tGene MYH10 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA8 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF31 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene WSB1 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1 has 0 variants, skipping\n",
      "[INFO]\tGene OTX2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23 has 0 variants, skipping\n",
      "[INFO]\tGene ULK3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01516 has 0 variants, skipping\n",
      "[INFO]\tGene GPR158-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00836 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929073 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO2 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX has 0 variants, skipping\n",
      "[INFO]\tGene NAA40 has 0 variants, skipping\n",
      "[INFO]\tGene ECE1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929117 has 0 variants, skipping\n",
      "[INFO]\tGene FAM238A has 0 variants, skipping\n",
      "[INFO]\tGene FAM238B has 0 variants, skipping\n",
      "[INFO]\tGene PWWP3A has 0 variants, skipping\n",
      "[INFO]\tGene IRGM has 0 variants, skipping\n",
      "[INFO]\tGene MEF2B has 0 variants, skipping\n",
      "[INFO]\tGene CHAT has 0 variants, skipping\n",
      "[INFO]\tGene PARG has 0 variants, skipping\n",
      "[INFO]\tGene MFSD2A has 0 variants, skipping\n",
      "[INFO]\tGene PEG3 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L1 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF540 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA4 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF484 has 0 variants, skipping\n",
      "[INFO]\tGene GRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD1 has 0 variants, skipping\n",
      "[INFO]\tGene ITCH has 0 variants, skipping\n",
      "[INFO]\tGene ATG10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A18 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR1 has 0 variants, skipping\n",
      "[INFO]\tGene POU1F1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT8 has 0 variants, skipping\n",
      "[INFO]\tGene CYREN has 0 variants, skipping\n",
      "[INFO]\tGene CCDC110 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD4 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7A has 0 variants, skipping\n",
      "[INFO]\tGene UBXN10 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf216 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF684 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR875 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF580 has 0 variants, skipping\n",
      "[INFO]\tGene GLYCTK has 0 variants, skipping\n",
      "[INFO]\tGene VLDLR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD86 has 0 variants, skipping\n",
      "[INFO]\tGene OBI1 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A3 has 0 variants, skipping\n",
      "[INFO]\tGene WFIKKN2 has 0 variants, skipping\n",
      "[INFO]\tGene ADRM1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP8 has 0 variants, skipping\n",
      "[INFO]\tGene XG has 0 variants, skipping\n",
      "[INFO]\tGene CNTN5 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7F2 has 0 variants, skipping\n",
      "[INFO]\tGene RCC1L has 0 variants, skipping\n",
      "[INFO]\tGene TNKS1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR642A has 0 variants, skipping\n",
      "[INFO]\tGene MID1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A6P has 0 variants, skipping\n",
      "[INFO]\tGene ARMC4P1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA5 has 0 variants, skipping\n",
      "[INFO]\tGene RIF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A4 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL4 has 0 variants, skipping\n",
      "[INFO]\tGene SERBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30B has 0 variants, skipping\n",
      "[INFO]\tGene GPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LY9 has 0 variants, skipping\n",
      "[INFO]\tGene CLK2 has 0 variants, skipping\n",
      "[INFO]\tGene FABP3 has 0 variants, skipping\n",
      "[INFO]\tGene ECHS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCS2 has 0 variants, skipping\n",
      "[INFO]\tGene MFNG has 0 variants, skipping\n",
      "[INFO]\tGene SNORD130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8086 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02652 has 0 variants, skipping\n",
      "[INFO]\tGene WAC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01517 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00837 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5A has 0 variants, skipping\n",
      "[INFO]\tGene SVIL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB10 has 0 variants, skipping\n",
      "[INFO]\tGene NRXN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR604 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929279 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7162 has 0 variants, skipping\n",
      "[INFO]\tGene KALRN has 0 variants, skipping\n",
      "[INFO]\tGene SVIL2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02644 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR2 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf56 has 0 variants, skipping\n",
      "[INFO]\tGene PRR30 has 0 variants, skipping\n",
      "[INFO]\tGene RPH3A has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF285 has 0 variants, skipping\n",
      "[INFO]\tGene TECRL has 0 variants, skipping\n",
      "[INFO]\tGene TSPO2 has 0 variants, skipping\n",
      "[INFO]\tGene GBP6 has 0 variants, skipping\n",
      "[INFO]\tGene SCYL2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8J has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ15 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ13 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ10 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNF1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA5 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA3 has 0 variants, skipping\n",
      "[INFO]\tGene CD82 has 0 variants, skipping\n",
      "[INFO]\tGene JUP has 0 variants, skipping\n",
      "[INFO]\tGene JUNB has 0 variants, skipping\n",
      "[INFO]\tGene JUN has 0 variants, skipping\n",
      "[INFO]\tGene JAG2 has 0 variants, skipping\n",
      "[INFO]\tGene IVD has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD16 has 0 variants, skipping\n",
      "[INFO]\tGene RNF19A has 0 variants, skipping\n",
      "[INFO]\tGene PLPP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102031319 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929431 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF7 has 0 variants, skipping\n",
      "[INFO]\tGene MSRB3 has 0 variants, skipping\n",
      "[INFO]\tGene STAC has 0 variants, skipping\n",
      "[INFO]\tGene SSX2 has 0 variants, skipping\n",
      "[INFO]\tGene SSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA86 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMCH1 has 0 variants, skipping\n",
      "[INFO]\tGene IL18RAP has 0 variants, skipping\n",
      "[INFO]\tGene PLIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CD96 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM4B has 0 variants, skipping\n",
      "[INFO]\tGene CHDH has 0 variants, skipping\n",
      "[INFO]\tGene MUC4 has 0 variants, skipping\n",
      "[INFO]\tGene PPT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00838 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18B has 0 variants, skipping\n",
      "[INFO]\tGene IL1RL1 has 0 variants, skipping\n",
      "[INFO]\tGene NLK has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R4 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS21 has 0 variants, skipping\n",
      "[INFO]\tGene USP25 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC4 has 0 variants, skipping\n",
      "[INFO]\tGene NUP153 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFB has 0 variants, skipping\n",
      "[INFO]\tGene VCL has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA1 has 0 variants, skipping\n",
      "[INFO]\tGene VCX has 0 variants, skipping\n",
      "[INFO]\tGene MIR3611 has 0 variants, skipping\n",
      "[INFO]\tGene BEND3 has 0 variants, skipping\n",
      "[INFO]\tGene KIF6 has 0 variants, skipping\n",
      "[INFO]\tGene SYT16 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF18 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A26 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BL has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5C has 0 variants, skipping\n",
      "[INFO]\tGene MREG has 0 variants, skipping\n",
      "[INFO]\tGene CZIB has 0 variants, skipping\n",
      "[INFO]\tGene ULK4 has 0 variants, skipping\n",
      "[INFO]\tGene RIC8B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4683 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT5 has 0 variants, skipping\n",
      "[INFO]\tGene MTIF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D3 has 0 variants, skipping\n",
      "[INFO]\tGene UNK has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B10 has 0 variants, skipping\n",
      "[INFO]\tGene CBFA2T2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRK has 0 variants, skipping\n",
      "[INFO]\tGene SIGIRR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33BP1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD6 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene DDI2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIEN1 has 0 variants, skipping\n",
      "[INFO]\tGene LLPH has 0 variants, skipping\n",
      "[INFO]\tGene GINS4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF6 has 0 variants, skipping\n",
      "[INFO]\tGene IFT122 has 0 variants, skipping\n",
      "[INFO]\tGene DPF3 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00667 has 0 variants, skipping\n",
      "[INFO]\tGene UCA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG17 has 0 variants, skipping\n",
      "[INFO]\tGene PCA3 has 0 variants, skipping\n",
      "[INFO]\tGene HULC has 0 variants, skipping\n",
      "[INFO]\tGene RNU11 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA10 has 0 variants, skipping\n",
      "[INFO]\tGene EMBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX2-OT has 0 variants, skipping\n",
      "[INFO]\tGene HCG22 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf37-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PNMT has 0 variants, skipping\n",
      "[INFO]\tGene UBAC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC190 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN5 has 0 variants, skipping\n",
      "[INFO]\tGene GMDS-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR876 has 0 variants, skipping\n",
      "[INFO]\tGene INTS9 has 0 variants, skipping\n",
      "[INFO]\tGene THEG has 0 variants, skipping\n",
      "[INFO]\tGene IL23A has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A3 has 0 variants, skipping\n",
      "[INFO]\tGene CD320 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH12 has 0 variants, skipping\n",
      "[INFO]\tGene RSF1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD2 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR6 has 0 variants, skipping\n",
      "[INFO]\tGene HUWE1 has 0 variants, skipping\n",
      "[INFO]\tGene COBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B7P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00999 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3BP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441666 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00839 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF37BP has 0 variants, skipping\n",
      "[INFO]\tGene IL6R has 0 variants, skipping\n",
      "[INFO]\tGene ZNF33B has 0 variants, skipping\n",
      "[INFO]\tGene PRPF4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP2 has 0 variants, skipping\n",
      "[INFO]\tGene TXN has 0 variants, skipping\n",
      "[INFO]\tGene MST1R has 0 variants, skipping\n",
      "[INFO]\tGene LOC283028 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378269 has 0 variants, skipping\n",
      "[INFO]\tGene SCN8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02623 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01264 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF782 has 0 variants, skipping\n",
      "[INFO]\tGene ACE has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3J has 0 variants, skipping\n",
      "[INFO]\tGene CCDC58 has 0 variants, skipping\n",
      "[INFO]\tGene TRABD has 0 variants, skipping\n",
      "[INFO]\tGene PIGP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf141 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF32-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02658 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00619 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00840 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00841 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM72-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FADS2 has 0 variants, skipping\n",
      "[INFO]\tGene CD8B has 0 variants, skipping\n",
      "[INFO]\tGene RSU1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724323 has 0 variants, skipping\n",
      "[INFO]\tGene IPCEF1 has 0 variants, skipping\n",
      "[INFO]\tGene PLP2 has 0 variants, skipping\n",
      "[INFO]\tGene PLN has 0 variants, skipping\n",
      "[INFO]\tGene PLIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLB2 has 0 variants, skipping\n",
      "[INFO]\tGene ECE2 has 0 variants, skipping\n",
      "[INFO]\tGene OAS2 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM2 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP4 has 0 variants, skipping\n",
      "[INFO]\tGene SP4 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS35 has 0 variants, skipping\n",
      "[INFO]\tGene STK32B has 0 variants, skipping\n",
      "[INFO]\tGene DUSP22 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC4 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP6 has 0 variants, skipping\n",
      "[INFO]\tGene MT1HL1 has 0 variants, skipping\n",
      "[INFO]\tGene ECSCR has 0 variants, skipping\n",
      "[INFO]\tGene HIP1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PAWR has 0 variants, skipping\n",
      "[INFO]\tGene MCM3 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0319 has 0 variants, skipping\n",
      "[INFO]\tGene TPBGL has 0 variants, skipping\n",
      "[INFO]\tGene MPC1L has 0 variants, skipping\n",
      "[INFO]\tGene MICU1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R17 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN14 has 0 variants, skipping\n",
      "[INFO]\tGene EVPLL has 0 variants, skipping\n",
      "[INFO]\tGene VANGL1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX13 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD1 has 0 variants, skipping\n",
      "[INFO]\tGene PARGP1 has 0 variants, skipping\n",
      "[INFO]\tGene HEMK1 has 0 variants, skipping\n",
      "[INFO]\tGene UBAC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74C-1 has 0 variants, skipping\n",
      "[INFO]\tGene ELOC has 0 variants, skipping\n",
      "[INFO]\tGene COA8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A9 has 0 variants, skipping\n",
      "[INFO]\tGene CAPNS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM1 has 0 variants, skipping\n",
      "[INFO]\tGene ENOSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF302 has 0 variants, skipping\n",
      "[INFO]\tGene ANTXRLP1 has 0 variants, skipping\n",
      "[INFO]\tGene CUTA has 0 variants, skipping\n",
      "[INFO]\tGene SORCS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25BP has 0 variants, skipping\n",
      "[INFO]\tGene CDH12 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1P33 has 0 variants, skipping\n",
      "[INFO]\tGene CRHR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927699 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2P1 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1P2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD2B has 0 variants, skipping\n",
      "[INFO]\tGene SURF4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2I has 0 variants, skipping\n",
      "[INFO]\tGene RNF130 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB6 has 0 variants, skipping\n",
      "[INFO]\tGene GFM2 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF281 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR68 has 0 variants, skipping\n",
      "[INFO]\tGene IFT27 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD35 has 0 variants, skipping\n",
      "[INFO]\tGene M1AP has 0 variants, skipping\n",
      "[INFO]\tGene FUT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107001062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378292 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP6 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC9 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-5 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CTSLP2 has 0 variants, skipping\n",
      "[INFO]\tGene SHLD2P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724593 has 0 variants, skipping\n",
      "[INFO]\tGene HEPACAM2 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP12P has 0 variants, skipping\n",
      "[INFO]\tGene UTS2R has 0 variants, skipping\n",
      "[INFO]\tGene AFF2 has 0 variants, skipping\n",
      "[INFO]\tGene STIM2 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP4 has 0 variants, skipping\n",
      "[INFO]\tGene IGBP1 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT3 has 0 variants, skipping\n",
      "[INFO]\tGene IFRD1 has 0 variants, skipping\n",
      "[INFO]\tGene PELI1 has 0 variants, skipping\n",
      "[INFO]\tGene RCN3 has 0 variants, skipping\n",
      "[INFO]\tGene CBX8 has 0 variants, skipping\n",
      "[INFO]\tGene AMOTL1 has 0 variants, skipping\n",
      "[INFO]\tGene FXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE28 has 0 variants, skipping\n",
      "[INFO]\tGene FAM209A has 0 variants, skipping\n",
      "[INFO]\tGene SPTB has 0 variants, skipping\n",
      "[INFO]\tGene MYH2 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2D has 0 variants, skipping\n",
      "[INFO]\tGene PPA2 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM2 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAP30L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DYRK1A has 0 variants, skipping\n",
      "[INFO]\tGene PFKM has 0 variants, skipping\n",
      "[INFO]\tGene CD36 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP22-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PASK has 0 variants, skipping\n",
      "[INFO]\tGene ADCK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4294 has 0 variants, skipping\n",
      "[INFO]\tGene FAM170B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSLN has 0 variants, skipping\n",
      "[INFO]\tGene STRN4 has 0 variants, skipping\n",
      "[INFO]\tGene MRM2 has 0 variants, skipping\n",
      "[INFO]\tGene NRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf71-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JPH3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC8A has 0 variants, skipping\n",
      "[INFO]\tGene PNLDC1 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKG has 0 variants, skipping\n",
      "[INFO]\tGene FBXO42 has 0 variants, skipping\n",
      "[INFO]\tGene LIN37 has 0 variants, skipping\n",
      "[INFO]\tGene RIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF183 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AB1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG1 has 0 variants, skipping\n",
      "[INFO]\tGene PDIA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCN2A has 0 variants, skipping\n",
      "[INFO]\tGene PRR20C has 0 variants, skipping\n",
      "[INFO]\tGene PRR20B has 0 variants, skipping\n",
      "[INFO]\tGene MYB has 0 variants, skipping\n",
      "[INFO]\tGene LTBP3 has 0 variants, skipping\n",
      "[INFO]\tGene DENND4C has 0 variants, skipping\n",
      "[INFO]\tGene PCBP3 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1A has 0 variants, skipping\n",
      "[INFO]\tGene GFI1B has 0 variants, skipping\n",
      "[INFO]\tGene FAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC31 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9 has 0 variants, skipping\n",
      "[INFO]\tGene RNF13 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLRB1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23B has 0 variants, skipping\n",
      "[INFO]\tGene TSHB has 0 variants, skipping\n",
      "[INFO]\tGene WDFY3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA2 has 0 variants, skipping\n",
      "[INFO]\tGene CYTOR has 0 variants, skipping\n",
      "[INFO]\tGene TBCB has 0 variants, skipping\n",
      "[INFO]\tGene PRDM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193A has 0 variants, skipping\n",
      "[INFO]\tGene MIR136 has 0 variants, skipping\n",
      "[INFO]\tGene MIR190A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74C-2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCD has 0 variants, skipping\n",
      "[INFO]\tGene FAM21EP has 0 variants, skipping\n",
      "[INFO]\tGene FRG2 has 0 variants, skipping\n",
      "[INFO]\tGene DPH2 has 0 variants, skipping\n",
      "[INFO]\tGene ASMTL has 0 variants, skipping\n",
      "[INFO]\tGene PDHB has 0 variants, skipping\n",
      "[INFO]\tGene SP7 has 0 variants, skipping\n",
      "[INFO]\tGene KIF21A has 0 variants, skipping\n",
      "[INFO]\tGene UCHL5 has 0 variants, skipping\n",
      "[INFO]\tGene SGMS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN6 has 0 variants, skipping\n",
      "[INFO]\tGene SPTY2D1OS has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN2 has 0 variants, skipping\n",
      "[INFO]\tGene CCM2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724719 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INAVA has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378305 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM247 has 0 variants, skipping\n",
      "[INFO]\tGene C3AR1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL4A has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF1 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF39 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM51 has 0 variants, skipping\n",
      "[INFO]\tGene NPLOC4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3C has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC12 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH4 has 0 variants, skipping\n",
      "[INFO]\tGene POMT2 has 0 variants, skipping\n",
      "[INFO]\tGene DPP7 has 0 variants, skipping\n",
      "[INFO]\tGene VPREB3 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABT1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6IP has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC8 has 0 variants, skipping\n",
      "[INFO]\tGene GREM1 has 0 variants, skipping\n",
      "[INFO]\tGene IL19 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD3 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC2 has 0 variants, skipping\n",
      "[INFO]\tGene CIDEA has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF860 has 0 variants, skipping\n",
      "[INFO]\tGene RTKN has 0 variants, skipping\n",
      "[INFO]\tGene ARMC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378311 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F1 has 0 variants, skipping\n",
      "[INFO]\tGene UVSSA has 0 variants, skipping\n",
      "[INFO]\tGene MIR3924 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF10 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C2 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133CP has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA1 has 0 variants, skipping\n",
      "[INFO]\tGene AHRR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01553 has 0 variants, skipping\n",
      "[INFO]\tGene GBA3 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF3 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf22 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT2 has 0 variants, skipping\n",
      "[INFO]\tGene MOB1A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC87 has 0 variants, skipping\n",
      "[INFO]\tGene EDEM2 has 0 variants, skipping\n",
      "[INFO]\tGene PANK4 has 0 variants, skipping\n",
      "[INFO]\tGene CER1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB22 has 0 variants, skipping\n",
      "[INFO]\tGene WDR46 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM7 has 0 variants, skipping\n",
      "[INFO]\tGene FCMR has 0 variants, skipping\n",
      "[INFO]\tGene TXNL4B has 0 variants, skipping\n",
      "[INFO]\tGene LMO2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00845 has 0 variants, skipping\n",
      "[INFO]\tGene RELCH has 0 variants, skipping\n",
      "[INFO]\tGene YEATS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MED15 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM3 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23D1 has 0 variants, skipping\n",
      "[INFO]\tGene GLUL has 0 variants, skipping\n",
      "[INFO]\tGene EGR4 has 0 variants, skipping\n",
      "[INFO]\tGene EGR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283045 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3F has 0 variants, skipping\n",
      "[INFO]\tGene ZNF331 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN2L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM235 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP90 has 0 variants, skipping\n",
      "[INFO]\tGene TPGS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR203A has 0 variants, skipping\n",
      "[INFO]\tGene PM20D2 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX6 has 0 variants, skipping\n",
      "[INFO]\tGene BOD1 has 0 variants, skipping\n",
      "[INFO]\tGene BNIPL has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378330 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02671 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01515 has 0 variants, skipping\n",
      "[INFO]\tGene TRAK1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928961 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7151 has 0 variants, skipping\n",
      "[INFO]\tGene PTRH2 has 0 variants, skipping\n",
      "[INFO]\tGene DESI2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS36 has 0 variants, skipping\n",
      "[INFO]\tGene TAF9B has 0 variants, skipping\n",
      "[INFO]\tGene MRPL48 has 0 variants, skipping\n",
      "[INFO]\tGene PPME1 has 0 variants, skipping\n",
      "[INFO]\tGene FMO2 has 0 variants, skipping\n",
      "[INFO]\tGene MYD88 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P5 has 0 variants, skipping\n",
      "[INFO]\tGene ROBO2 has 0 variants, skipping\n",
      "[INFO]\tGene COG6 has 0 variants, skipping\n",
      "[INFO]\tGene XPO5 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9B has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM21 has 0 variants, skipping\n",
      "[INFO]\tGene UCK1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRA1 has 0 variants, skipping\n",
      "[INFO]\tGene GKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP5 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO2 has 0 variants, skipping\n",
      "[INFO]\tGene DNM3 has 0 variants, skipping\n",
      "[INFO]\tGene ANO9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS22 has 0 variants, skipping\n",
      "[INFO]\tGene COQ6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR573 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA1 has 0 variants, skipping\n",
      "[INFO]\tGene ECI2-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01619 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01138 has 0 variants, skipping\n",
      "[INFO]\tGene KRT23 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8D has 0 variants, skipping\n",
      "[INFO]\tGene MRPS2 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1L4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR61 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD98 has 0 variants, skipping\n",
      "[INFO]\tGene H3-4 has 0 variants, skipping\n",
      "[INFO]\tGene CDC7 has 0 variants, skipping\n",
      "[INFO]\tGene AXIN1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM187 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX3 has 0 variants, skipping\n",
      "[INFO]\tGene NAA10 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX2 has 0 variants, skipping\n",
      "[INFO]\tGene NRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYN3 has 0 variants, skipping\n",
      "[INFO]\tGene DPH7 has 0 variants, skipping\n",
      "[INFO]\tGene UCN has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2V2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2N has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2I has 0 variants, skipping\n",
      "[INFO]\tGene CCNG1 has 0 variants, skipping\n",
      "[INFO]\tGene ENO4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928994 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF763 has 0 variants, skipping\n",
      "[INFO]\tGene PALLD has 0 variants, skipping\n",
      "[INFO]\tGene ZNF446 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF806 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKN has 0 variants, skipping\n",
      "[INFO]\tGene EWSR1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG2 has 0 variants, skipping\n",
      "[INFO]\tGene LERFS has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB12 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL7 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15B has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4IP1 has 0 variants, skipping\n",
      "[INFO]\tGene GATM has 0 variants, skipping\n",
      "[INFO]\tGene C17orf113 has 0 variants, skipping\n",
      "[INFO]\tGene MOCS1 has 0 variants, skipping\n",
      "[INFO]\tGene MME has 0 variants, skipping\n",
      "[INFO]\tGene LMLN2 has 0 variants, skipping\n",
      "[INFO]\tGene PVALEF has 0 variants, skipping\n",
      "[INFO]\tGene TTC3 has 0 variants, skipping\n",
      "[INFO]\tGene TYSND1 has 0 variants, skipping\n",
      "[INFO]\tGene ME1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC3A2 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP4 has 0 variants, skipping\n",
      "[INFO]\tGene MDM4 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MEST has 0 variants, skipping\n",
      "[INFO]\tGene MDM2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K3 has 0 variants, skipping\n",
      "[INFO]\tGene MDK has 0 variants, skipping\n",
      "[INFO]\tGene ADAM11 has 0 variants, skipping\n",
      "[INFO]\tGene CD46 has 0 variants, skipping\n",
      "[INFO]\tGene COCH has 0 variants, skipping\n",
      "[INFO]\tGene RHEX has 0 variants, skipping\n",
      "[INFO]\tGene HEXD has 0 variants, skipping\n",
      "[INFO]\tGene KCTD16 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR3A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM10 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02622 has 0 variants, skipping\n",
      "[INFO]\tGene COX19 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH3B2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP47 has 0 variants, skipping\n",
      "[INFO]\tGene RAD17 has 0 variants, skipping\n",
      "[INFO]\tGene MYOF has 0 variants, skipping\n",
      "[INFO]\tGene GNAS has 0 variants, skipping\n",
      "[INFO]\tGene SP110 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN2 has 0 variants, skipping\n",
      "[INFO]\tGene MYL1 has 0 variants, skipping\n",
      "[INFO]\tGene NR6A1 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH23-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2NL has 0 variants, skipping\n",
      "[INFO]\tGene MIR7152 has 0 variants, skipping\n",
      "[INFO]\tGene APLP2 has 0 variants, skipping\n",
      "[INFO]\tGene HES5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM230 has 0 variants, skipping\n",
      "[INFO]\tGene FAM171A1 has 0 variants, skipping\n",
      "[INFO]\tGene THEMIS has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L15 has 0 variants, skipping\n",
      "[INFO]\tGene PAK4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110A has 0 variants, skipping\n",
      "[INFO]\tGene POLDIP2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK10 has 0 variants, skipping\n",
      "[INFO]\tGene CCL28 has 0 variants, skipping\n",
      "[INFO]\tGene HPS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF5 has 0 variants, skipping\n",
      "[INFO]\tGene S100A8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR595 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4676 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC20 has 0 variants, skipping\n",
      "[INFO]\tGene LIPN has 0 variants, skipping\n",
      "[INFO]\tGene AQP12B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF219 has 0 variants, skipping\n",
      "[INFO]\tGene OPRM1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF5A has 0 variants, skipping\n",
      "[INFO]\tGene PRUNE1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11F has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIPL2 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB4 has 0 variants, skipping\n",
      "[INFO]\tGene NTMT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B has 0 variants, skipping\n",
      "[INFO]\tGene LYPD6 has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS54 has 0 variants, skipping\n",
      "[INFO]\tGene RSPRY1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3C has 0 variants, skipping\n",
      "[INFO]\tGene IFNGR2 has 0 variants, skipping\n",
      "[INFO]\tGene RANGRF has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC13 has 0 variants, skipping\n",
      "[INFO]\tGene IL16 has 0 variants, skipping\n",
      "[INFO]\tGene DKC1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC22 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSF1R has 0 variants, skipping\n",
      "[INFO]\tGene PHF19 has 0 variants, skipping\n",
      "[INFO]\tGene CENPJ has 0 variants, skipping\n",
      "[INFO]\tGene ZNF718 has 0 variants, skipping\n",
      "[INFO]\tGene FAM227A has 0 variants, skipping\n",
      "[INFO]\tGene ABLIM3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF121 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P4 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD1P3 has 0 variants, skipping\n",
      "[INFO]\tGene GRB14 has 0 variants, skipping\n",
      "[INFO]\tGene GP5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGB1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2 has 0 variants, skipping\n",
      "[INFO]\tGene GNG4 has 0 variants, skipping\n",
      "[INFO]\tGene GPC3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL32 has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP has 0 variants, skipping\n",
      "[INFO]\tGene CRLF2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL4 has 0 variants, skipping\n",
      "[INFO]\tGene DNM2 has 0 variants, skipping\n",
      "[INFO]\tGene ATL2 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1A has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM69 has 0 variants, skipping\n",
      "[INFO]\tGene ISCU has 0 variants, skipping\n",
      "[INFO]\tGene PEX11A has 0 variants, skipping\n",
      "[INFO]\tGene LRRN3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137C has 0 variants, skipping\n",
      "[INFO]\tGene FOXN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR300 has 0 variants, skipping\n",
      "[INFO]\tGene PANTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A3 has 0 variants, skipping\n",
      "[INFO]\tGene ASZ1 has 0 variants, skipping\n",
      "[INFO]\tGene RGSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4L1 has 0 variants, skipping\n",
      "[INFO]\tGene TWNK has 0 variants, skipping\n",
      "[INFO]\tGene HPSE2 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC13-AS has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN12 has 0 variants, skipping\n",
      "[INFO]\tGene DPY30 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP44 has 0 variants, skipping\n",
      "[INFO]\tGene IRF3 has 0 variants, skipping\n",
      "[INFO]\tGene POLM has 0 variants, skipping\n",
      "[INFO]\tGene MRPL44 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3E has 0 variants, skipping\n",
      "[INFO]\tGene MIR3714 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3690 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR26 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4735 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4715 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4712 has 0 variants, skipping\n",
      "[INFO]\tGene MIR203B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3198-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4701 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4673 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4669 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4665 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4656 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4638 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4632 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378I has 0 variants, skipping\n",
      "[INFO]\tGene MIR4516 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4513 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2392 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4505 has 0 variants, skipping\n",
      "[INFO]\tGene RBM39 has 0 variants, skipping\n",
      "[INFO]\tGene BTG4 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP17 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD11 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P1-BPTFP1-KPNA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene POU6F1 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723439 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1G has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929165 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD172 has 0 variants, skipping\n",
      "[INFO]\tGene FAM219B has 0 variants, skipping\n",
      "[INFO]\tGene NUTM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929234 has 0 variants, skipping\n",
      "[INFO]\tGene MIR606 has 0 variants, skipping\n",
      "[INFO]\tGene LRMDA has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378367 has 0 variants, skipping\n",
      "[INFO]\tGene PKN1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCG has 0 variants, skipping\n",
      "[INFO]\tGene FCRL1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F3 has 0 variants, skipping\n",
      "[INFO]\tGene HGF has 0 variants, skipping\n",
      "[INFO]\tGene HRNR has 0 variants, skipping\n",
      "[INFO]\tGene TIAM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM149A has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L2 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene DLG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBIP has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00856 has 0 variants, skipping\n",
      "[INFO]\tGene MDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM27 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPHL has 0 variants, skipping\n",
      "[INFO]\tGene TRPM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729815 has 0 variants, skipping\n",
      "[INFO]\tGene PXN has 0 variants, skipping\n",
      "[INFO]\tGene USP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02679 has 0 variants, skipping\n",
      "[INFO]\tGene BEND3P3 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P21 has 0 variants, skipping\n",
      "[INFO]\tGene MBL1P has 0 variants, skipping\n",
      "[INFO]\tGene PLS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM254-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM254 has 0 variants, skipping\n",
      "[INFO]\tGene MTCL1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB4 has 0 variants, skipping\n",
      "[INFO]\tGene CETP has 0 variants, skipping\n",
      "[INFO]\tGene FZD5 has 0 variants, skipping\n",
      "[INFO]\tGene CHIT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130698 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CELF2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRDL1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929574 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02655 has 0 variants, skipping\n",
      "[INFO]\tGene DMKN has 0 variants, skipping\n",
      "[INFO]\tGene NRG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02650 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM8 has 0 variants, skipping\n",
      "[INFO]\tGene STOX2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20C has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM19 has 0 variants, skipping\n",
      "[INFO]\tGene RNPEP has 0 variants, skipping\n",
      "[INFO]\tGene AGFG1 has 0 variants, skipping\n",
      "[INFO]\tGene LMBRD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01519 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02647 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929662 has 0 variants, skipping\n",
      "[INFO]\tGene GRID1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX46 has 0 variants, skipping\n",
      "[INFO]\tGene RBM17 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R16A has 0 variants, skipping\n",
      "[INFO]\tGene COX14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP19 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD5 has 0 variants, skipping\n",
      "[INFO]\tGene FATE1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCD2 has 0 variants, skipping\n",
      "[INFO]\tGene EAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP2 has 0 variants, skipping\n",
      "[INFO]\tGene SARNP has 0 variants, skipping\n",
      "[INFO]\tGene DIDO1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00863 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM245A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4678 has 0 variants, skipping\n",
      "[INFO]\tGene HES7 has 0 variants, skipping\n",
      "[INFO]\tGene DHX29 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD11 has 0 variants, skipping\n",
      "[INFO]\tGene CFL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene QPRT has 0 variants, skipping\n",
      "[INFO]\tGene SMIM5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP25 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC4 has 0 variants, skipping\n",
      "[INFO]\tGene AMELY has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-3 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX2 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R5 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4679-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4679-1 has 0 variants, skipping\n",
      "[INFO]\tGene WDPCP has 0 variants, skipping\n",
      "[INFO]\tGene SRRT has 0 variants, skipping\n",
      "[INFO]\tGene FAM219A has 0 variants, skipping\n",
      "[INFO]\tGene ARGFX has 0 variants, skipping\n",
      "[INFO]\tGene MYPOP has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS9C has 0 variants, skipping\n",
      "[INFO]\tGene PRORP has 0 variants, skipping\n",
      "[INFO]\tGene FLJ37201 has 0 variants, skipping\n",
      "[INFO]\tGene TAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01374 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02653 has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_008559 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378430 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00502 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT9P1 has 0 variants, skipping\n",
      "[INFO]\tGene PIANP has 0 variants, skipping\n",
      "[INFO]\tGene HECTD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM65 has 0 variants, skipping\n",
      "[INFO]\tGene ADSL has 0 variants, skipping\n",
      "[INFO]\tGene ADRB3 has 0 variants, skipping\n",
      "[INFO]\tGene ADRB2 has 0 variants, skipping\n",
      "[INFO]\tGene SGCA has 0 variants, skipping\n",
      "[INFO]\tGene ADA has 0 variants, skipping\n",
      "[INFO]\tGene HECTD2 has 0 variants, skipping\n",
      "[INFO]\tGene TNKS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS39 has 0 variants, skipping\n",
      "[INFO]\tGene VAC14 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB130B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14A-ACY1 has 0 variants, skipping\n",
      "[INFO]\tGene MARK2P9 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK1 has 0 variants, skipping\n",
      "[INFO]\tGene RTL8A has 0 variants, skipping\n",
      "[INFO]\tGene PRR22 has 0 variants, skipping\n",
      "[INFO]\tGene AP3M2 has 0 variants, skipping\n",
      "[INFO]\tGene PFN1 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP3 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1C has 0 variants, skipping\n",
      "[INFO]\tGene AATK has 0 variants, skipping\n",
      "[INFO]\tGene PDCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCL1B has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1A has 0 variants, skipping\n",
      "[INFO]\tGene KLK4 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCG1 has 0 variants, skipping\n",
      "[INFO]\tGene OXA1L has 0 variants, skipping\n",
      "[INFO]\tGene OMD has 0 variants, skipping\n",
      "[INFO]\tGene NUCB2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS9D1 has 0 variants, skipping\n",
      "[INFO]\tGene ROR1 has 0 variants, skipping\n",
      "[INFO]\tGene NRF1 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM1 has 0 variants, skipping\n",
      "[INFO]\tGene NME4 has 0 variants, skipping\n",
      "[INFO]\tGene CLOCK has 0 variants, skipping\n",
      "[INFO]\tGene SNU13 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIL1 has 0 variants, skipping\n",
      "[INFO]\tGene IER2 has 0 variants, skipping\n",
      "[INFO]\tGene MINPP1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26A has 0 variants, skipping\n",
      "[INFO]\tGene WTAP has 0 variants, skipping\n",
      "[INFO]\tGene FRA10AC1 has 0 variants, skipping\n",
      "[INFO]\tGene LACC1 has 0 variants, skipping\n",
      "[INFO]\tGene PIPSL has 0 variants, skipping\n",
      "[INFO]\tGene PGPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GP2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF667 has 0 variants, skipping\n",
      "[INFO]\tGene UAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13D has 0 variants, skipping\n",
      "[INFO]\tGene TMX2 has 0 variants, skipping\n",
      "[INFO]\tGene GVQW3 has 0 variants, skipping\n",
      "[INFO]\tGene RBFOX3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC6 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPDL has 0 variants, skipping\n",
      "[INFO]\tGene MOV10 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM34 has 0 variants, skipping\n",
      "[INFO]\tGene KDM1A has 0 variants, skipping\n",
      "[INFO]\tGene ADNP2 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB3 has 0 variants, skipping\n",
      "[INFO]\tGene CALU has 0 variants, skipping\n",
      "[INFO]\tGene RTL5 has 0 variants, skipping\n",
      "[INFO]\tGene SBK1 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR6 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS35L has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC3 has 0 variants, skipping\n",
      "[INFO]\tGene BMP2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX3-2 has 0 variants, skipping\n",
      "[INFO]\tGene BAK1 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE has 0 variants, skipping\n",
      "[INFO]\tGene APBA2 has 0 variants, skipping\n",
      "[INFO]\tGene TEK has 0 variants, skipping\n",
      "[INFO]\tGene MIR3157 has 0 variants, skipping\n",
      "[INFO]\tGene SUGCT has 0 variants, skipping\n",
      "[INFO]\tGene OR9G4 has 0 variants, skipping\n",
      "[INFO]\tGene OR9Q2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M8 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACR has 0 variants, skipping\n",
      "[INFO]\tGene ACLY has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4Y has 0 variants, skipping\n",
      "[INFO]\tGene WLS has 0 variants, skipping\n",
      "[INFO]\tGene BLNK has 0 variants, skipping\n",
      "[INFO]\tGene TEX14 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL3 has 0 variants, skipping\n",
      "[INFO]\tGene MALRD1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN8 has 0 variants, skipping\n",
      "[INFO]\tGene AJAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CX3CL1 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723665 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf50 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP19-SLIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268354 has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L1BP has 0 variants, skipping\n",
      "[INFO]\tGene PRSS47 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LETMD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4787 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4779 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4756 has 0 variants, skipping\n",
      "[INFO]\tGene MIR499B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4741 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4749 has 0 variants, skipping\n",
      "[INFO]\tGene ZBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF630 has 0 variants, skipping\n",
      "[INFO]\tGene NR1D2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMK1 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE7 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE3 has 0 variants, skipping\n",
      "[INFO]\tGene SPZ1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106B has 0 variants, skipping\n",
      "[INFO]\tGene FOXP3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC22 has 0 variants, skipping\n",
      "[INFO]\tGene SPON2 has 0 variants, skipping\n",
      "[INFO]\tGene BDNF has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO1 has 0 variants, skipping\n",
      "[INFO]\tGene FES has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL4 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP4 has 0 variants, skipping\n",
      "[INFO]\tGene ECM2 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-4 has 0 variants, skipping\n",
      "[INFO]\tGene DTX1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00866 has 0 variants, skipping\n",
      "[INFO]\tGene GRAP2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRF1 has 0 variants, skipping\n",
      "[INFO]\tGene COL8A2 has 0 variants, skipping\n",
      "[INFO]\tGene CD80 has 0 variants, skipping\n",
      "[INFO]\tGene CCR8 has 0 variants, skipping\n",
      "[INFO]\tGene CBX2 has 0 variants, skipping\n",
      "[INFO]\tGene RGN has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AY has 0 variants, skipping\n",
      "[INFO]\tGene VCY has 0 variants, skipping\n",
      "[INFO]\tGene CDY1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3F has 0 variants, skipping\n",
      "[INFO]\tGene LGALS7B has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R4 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2D has 0 variants, skipping\n",
      "[INFO]\tGene GRK6 has 0 variants, skipping\n",
      "[INFO]\tGene CD209 has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO2 has 0 variants, skipping\n",
      "[INFO]\tGene MYO18B has 0 variants, skipping\n",
      "[INFO]\tGene SPATA22 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L3 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A14 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf24 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R9B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4685 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-H has 0 variants, skipping\n",
      "[INFO]\tGene STX16 has 0 variants, skipping\n",
      "[INFO]\tGene CCL3L3 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT7L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6507 has 0 variants, skipping\n",
      "[INFO]\tGene SENP8 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE2B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM44 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01475 has 0 variants, skipping\n",
      "[INFO]\tGene EFHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP112 has 0 variants, skipping\n",
      "[INFO]\tGene TNNT2 has 0 variants, skipping\n",
      "[INFO]\tGene RXRB has 0 variants, skipping\n",
      "[INFO]\tGene TSC1 has 0 variants, skipping\n",
      "[INFO]\tGene DNMBP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC4H2 has 0 variants, skipping\n",
      "[INFO]\tGene NENF has 0 variants, skipping\n",
      "[INFO]\tGene RACGAP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM99A has 0 variants, skipping\n",
      "[INFO]\tGene NR1I3 has 0 variants, skipping\n",
      "[INFO]\tGene THAP9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCLL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF343 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22C has 0 variants, skipping\n",
      "[INFO]\tGene ASB11 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR3 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1L has 0 variants, skipping\n",
      "[INFO]\tGene PIGB has 0 variants, skipping\n",
      "[INFO]\tGene CHST10 has 0 variants, skipping\n",
      "[INFO]\tGene STX8 has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA12 has 0 variants, skipping\n",
      "[INFO]\tGene SYBU has 0 variants, skipping\n",
      "[INFO]\tGene GDAP1 has 0 variants, skipping\n",
      "[INFO]\tGene APOL1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1 has 0 variants, skipping\n",
      "[INFO]\tGene OLMALINC has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB13 has 0 variants, skipping\n",
      "[INFO]\tGene GRP has 0 variants, skipping\n",
      "[INFO]\tGene GYPC has 0 variants, skipping\n",
      "[INFO]\tGene EVX1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC45A has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A5 has 0 variants, skipping\n",
      "[INFO]\tGene TLX1NB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01514 has 0 variants, skipping\n",
      "[INFO]\tGene LBX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02681 has 0 variants, skipping\n",
      "[INFO]\tGene YY1AP1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8I2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A3 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene IGFLR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3158-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3158-2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC4 has 0 variants, skipping\n",
      "[INFO]\tGene PDE12 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD2 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA6 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C2 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf15 has 0 variants, skipping\n",
      "[INFO]\tGene RBM46 has 0 variants, skipping\n",
      "[INFO]\tGene CLDND1 has 0 variants, skipping\n",
      "[INFO]\tGene DUX4 has 0 variants, skipping\n",
      "[INFO]\tGene TFF1 has 0 variants, skipping\n",
      "[INFO]\tGene UBD has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP5 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2A has 0 variants, skipping\n",
      "[INFO]\tGene PRDX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD2 has 0 variants, skipping\n",
      "[INFO]\tGene CNBD2 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBID has 0 variants, skipping\n",
      "[INFO]\tGene MGAM has 0 variants, skipping\n",
      "[INFO]\tGene TIRAP has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A4 has 0 variants, skipping\n",
      "[INFO]\tGene TESMIN has 0 variants, skipping\n",
      "[INFO]\tGene HNF4A has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A3 has 0 variants, skipping\n",
      "[INFO]\tGene RPARP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SGK1 has 0 variants, skipping\n",
      "[INFO]\tGene SGSM3 has 0 variants, skipping\n",
      "[INFO]\tGene LY75-CD302 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP42-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY3-GNMT has 0 variants, skipping\n",
      "[INFO]\tGene BORCS7-ASMT has 0 variants, skipping\n",
      "[INFO]\tGene YIPF5 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO38 has 0 variants, skipping\n",
      "[INFO]\tGene CIITA has 0 variants, skipping\n",
      "[INFO]\tGene CADPS has 0 variants, skipping\n",
      "[INFO]\tGene PIGH has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2G has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF12-TNFSF13 has 0 variants, skipping\n",
      "[INFO]\tGene NOB1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP19 has 0 variants, skipping\n",
      "[INFO]\tGene TTC32 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6B has 0 variants, skipping\n",
      "[INFO]\tGene NDOR1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB29 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC74B has 0 variants, skipping\n",
      "[INFO]\tGene R3HCC1L has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2I1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF660-ZNF197 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM69 has 0 variants, skipping\n",
      "[INFO]\tGene VTA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD11 has 0 variants, skipping\n",
      "[INFO]\tGene TRMO has 0 variants, skipping\n",
      "[INFO]\tGene SHISA5 has 0 variants, skipping\n",
      "[INFO]\tGene JKAMP has 0 variants, skipping\n",
      "[INFO]\tGene CCDC174 has 0 variants, skipping\n",
      "[INFO]\tGene GSKIP has 0 variants, skipping\n",
      "[INFO]\tGene OSER1 has 0 variants, skipping\n",
      "[INFO]\tGene COX16 has 0 variants, skipping\n",
      "[INFO]\tGene BRF1 has 0 variants, skipping\n",
      "[INFO]\tGene HACD3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM217B has 0 variants, skipping\n",
      "[INFO]\tGene CDV3 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZB has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IMP4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR7 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT13 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR936 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4482 has 0 variants, skipping\n",
      "[INFO]\tGene HIKESHI has 0 variants, skipping\n",
      "[INFO]\tGene CFAP58-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02620 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0586 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02627 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02624 has 0 variants, skipping\n",
      "[INFO]\tGene SMKR1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRO has 0 variants, skipping\n",
      "[INFO]\tGene CXorf49B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01435 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM177 has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389831 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM135 has 0 variants, skipping\n",
      "[INFO]\tGene NDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM36 has 0 variants, skipping\n",
      "[INFO]\tGene ADD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM52 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF383 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A14 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM166B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4680 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF75D has 0 variants, skipping\n",
      "[INFO]\tGene FXYD4 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP6 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPC has 0 variants, skipping\n",
      "[INFO]\tGene PIK3C2A has 0 variants, skipping\n",
      "[INFO]\tGene PIGR has 0 variants, skipping\n",
      "[INFO]\tGene MIR6715B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6715A has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2GP has 0 variants, skipping\n",
      "[INFO]\tGene TRMT6 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B6 has 0 variants, skipping\n",
      "[INFO]\tGene PELO has 0 variants, skipping\n",
      "[INFO]\tGene COPZ1 has 0 variants, skipping\n",
      "[INFO]\tGene APIP has 0 variants, skipping\n",
      "[INFO]\tGene MRPL4 has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L2 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1 has 0 variants, skipping\n",
      "[INFO]\tGene TASP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0895L has 0 variants, skipping\n",
      "[INFO]\tGene NUDT6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103344931 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA87 has 0 variants, skipping\n",
      "[INFO]\tGene PRR13 has 0 variants, skipping\n",
      "[INFO]\tGene GPNMB has 0 variants, skipping\n",
      "[INFO]\tGene XPO7 has 0 variants, skipping\n",
      "[INFO]\tGene TSNARE1 has 0 variants, skipping\n",
      "[INFO]\tGene CWC27 has 0 variants, skipping\n",
      "[INFO]\tGene DHH has 0 variants, skipping\n",
      "[INFO]\tGene GBA2 has 0 variants, skipping\n",
      "[INFO]\tGene CWC22 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4483 has 0 variants, skipping\n",
      "[INFO]\tGene GBA has 0 variants, skipping\n",
      "[INFO]\tGene LDB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC186 has 0 variants, skipping\n",
      "[INFO]\tGene SFR1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW9 has 0 variants, skipping\n",
      "[INFO]\tGene PSD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLF1 has 0 variants, skipping\n",
      "[INFO]\tGene TCHP has 0 variants, skipping\n",
      "[INFO]\tGene FYTTD1 has 0 variants, skipping\n",
      "[INFO]\tGene RTL6 has 0 variants, skipping\n",
      "[INFO]\tGene MED10 has 0 variants, skipping\n",
      "[INFO]\tGene MRI1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC18 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47E has 0 variants, skipping\n",
      "[INFO]\tGene E2F3 has 0 variants, skipping\n",
      "[INFO]\tGene ELF5 has 0 variants, skipping\n",
      "[INFO]\tGene PLK5 has 0 variants, skipping\n",
      "[INFO]\tGene PARN has 0 variants, skipping\n",
      "[INFO]\tGene NTRK3 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIB has 0 variants, skipping\n",
      "[INFO]\tGene IL5RA has 0 variants, skipping\n",
      "[INFO]\tGene NUSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PELI3 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP3 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene BBS12 has 0 variants, skipping\n",
      "[INFO]\tGene SAA1 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL4 has 0 variants, skipping\n",
      "[INFO]\tGene CLINT1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC14 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4A has 0 variants, skipping\n",
      "[INFO]\tGene FAM53B has 0 variants, skipping\n",
      "[INFO]\tGene TTI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY1 has 0 variants, skipping\n",
      "[INFO]\tGene MLF1 has 0 variants, skipping\n",
      "[INFO]\tGene DCP1A has 0 variants, skipping\n",
      "[INFO]\tGene DIXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A24 has 0 variants, skipping\n",
      "[INFO]\tGene TERB1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM2 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD6 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927692 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02626 has 0 variants, skipping\n",
      "[INFO]\tGene CASKIN1 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSAD has 0 variants, skipping\n",
      "[INFO]\tGene HMCN2 has 0 variants, skipping\n",
      "[INFO]\tGene SYNM has 0 variants, skipping\n",
      "[INFO]\tGene SNORD158 has 0 variants, skipping\n",
      "[INFO]\tGene RAD23A has 0 variants, skipping\n",
      "[INFO]\tGene HSPB3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SRPX has 0 variants, skipping\n",
      "[INFO]\tGene RPL22L1 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf49 has 0 variants, skipping\n",
      "[INFO]\tGene MSL3 has 0 variants, skipping\n",
      "[INFO]\tGene CGB5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3663HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3663 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAS has 0 variants, skipping\n",
      "[INFO]\tGene OXCT1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00867 has 0 variants, skipping\n",
      "[INFO]\tGene KLF8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA19 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T6 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC17 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP13 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB7B has 0 variants, skipping\n",
      "[INFO]\tGene BAHCC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4681 has 0 variants, skipping\n",
      "[INFO]\tGene NFASC has 0 variants, skipping\n",
      "[INFO]\tGene DAZ4 has 0 variants, skipping\n",
      "[INFO]\tGene GNAT3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXRED2 has 0 variants, skipping\n",
      "[INFO]\tGene DGLUCY has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1197 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1306 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302E has 0 variants, skipping\n",
      "[INFO]\tGene SNAPIN has 0 variants, skipping\n",
      "[INFO]\tGene ELDR has 0 variants, skipping\n",
      "[INFO]\tGene MIR663B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1181 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND1 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH5 has 0 variants, skipping\n",
      "[INFO]\tGene ATRAID has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4682 has 0 variants, skipping\n",
      "[INFO]\tGene RBM41 has 0 variants, skipping\n",
      "[INFO]\tGene WDR11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM13A has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A8 has 0 variants, skipping\n",
      "[INFO]\tGene HSH2D has 0 variants, skipping\n",
      "[INFO]\tGene PGBD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB33 has 0 variants, skipping\n",
      "[INFO]\tGene FGB has 0 variants, skipping\n",
      "[INFO]\tGene MMP2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ACP2 has 0 variants, skipping\n",
      "[INFO]\tGene FHL2 has 0 variants, skipping\n",
      "[INFO]\tGene OPA1 has 0 variants, skipping\n",
      "[INFO]\tGene A4GNT has 0 variants, skipping\n",
      "[INFO]\tGene TRO has 0 variants, skipping\n",
      "[INFO]\tGene LOC339685 has 0 variants, skipping\n",
      "[INFO]\tGene MIS18A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC93 has 0 variants, skipping\n",
      "[INFO]\tGene PUS7 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM234B has 0 variants, skipping\n",
      "[INFO]\tGene ATE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASPH has 0 variants, skipping\n",
      "[INFO]\tGene CCL20 has 0 variants, skipping\n",
      "[INFO]\tGene SCO1 has 0 variants, skipping\n",
      "[INFO]\tGene RO60 has 0 variants, skipping\n",
      "[INFO]\tGene INO80 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP8 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3I has 0 variants, skipping\n",
      "[INFO]\tGene PURB has 0 variants, skipping\n",
      "[INFO]\tGene GABRG3 has 0 variants, skipping\n",
      "[INFO]\tGene PSIP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM14 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3F has 0 variants, skipping\n",
      "[INFO]\tGene GK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF670 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC102A has 0 variants, skipping\n",
      "[INFO]\tGene MACIR has 0 variants, skipping\n",
      "[INFO]\tGene UCN2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF502 has 0 variants, skipping\n",
      "[INFO]\tGene PIGS has 0 variants, skipping\n",
      "[INFO]\tGene TIGD7 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF682 has 0 variants, skipping\n",
      "[INFO]\tGene LDHAL6B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF101 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB9 has 0 variants, skipping\n",
      "[INFO]\tGene BMERB1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-4 has 0 variants, skipping\n",
      "[INFO]\tGene BCOR has 0 variants, skipping\n",
      "[INFO]\tGene DEFB123 has 0 variants, skipping\n",
      "[INFO]\tGene PMP22 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A7 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB119 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL1 has 0 variants, skipping\n",
      "[INFO]\tGene UBA1 has 0 variants, skipping\n",
      "[INFO]\tGene SFTA1P has 0 variants, skipping\n",
      "[INFO]\tGene LOC646214 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24B-CUZD1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24B has 0 variants, skipping\n",
      "[INFO]\tGene NSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene CHIC1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM23 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF9 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L has 0 variants, skipping\n",
      "[INFO]\tGene H1-5 has 0 variants, skipping\n",
      "[INFO]\tGene H1-4 has 0 variants, skipping\n",
      "[INFO]\tGene H1-3 has 0 variants, skipping\n",
      "[INFO]\tGene H1-2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-0 has 0 variants, skipping\n",
      "[INFO]\tGene GZMM has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H1 has 0 variants, skipping\n",
      "[INFO]\tGene GSC2 has 0 variants, skipping\n",
      "[INFO]\tGene GRPR has 0 variants, skipping\n",
      "[INFO]\tGene PDIA3 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L2 has 0 variants, skipping\n",
      "[INFO]\tGene NFIB has 0 variants, skipping\n",
      "[INFO]\tGene DSN1 has 0 variants, skipping\n",
      "[INFO]\tGene BHMT2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B5 has 0 variants, skipping\n",
      "[INFO]\tGene MLH3 has 0 variants, skipping\n",
      "[INFO]\tGene NBEA has 0 variants, skipping\n",
      "[INFO]\tGene LIMK2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC62 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4296 has 0 variants, skipping\n",
      "[INFO]\tGene TEX36-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283038 has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1-DT has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1 has 0 variants, skipping\n",
      "[INFO]\tGene STKLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A14 has 0 variants, skipping\n",
      "[INFO]\tGene SENP7 has 0 variants, skipping\n",
      "[INFO]\tGene EDRF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NFILZ has 0 variants, skipping\n",
      "[INFO]\tGene OTOGL has 0 variants, skipping\n",
      "[INFO]\tGene FAM120AOS has 0 variants, skipping\n",
      "[INFO]\tGene MCRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene FANK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00601 has 0 variants, skipping\n",
      "[INFO]\tGene LEPR has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6E has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD4 has 0 variants, skipping\n",
      "[INFO]\tGene ERAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728158 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A4A has 0 variants, skipping\n",
      "[INFO]\tGene HINFP has 0 variants, skipping\n",
      "[INFO]\tGene GRTP1 has 0 variants, skipping\n",
      "[INFO]\tGene TKFC has 0 variants, skipping\n",
      "[INFO]\tGene PRH1 has 0 variants, skipping\n",
      "[INFO]\tGene USP16 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCE1 has 0 variants, skipping\n",
      "[INFO]\tGene CDYL has 0 variants, skipping\n",
      "[INFO]\tGene KIF2A has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ2 has 0 variants, skipping\n",
      "[INFO]\tGene GHRL has 0 variants, skipping\n",
      "[INFO]\tGene ACP6 has 0 variants, skipping\n",
      "[INFO]\tGene TACO1 has 0 variants, skipping\n",
      "[INFO]\tGene AS-PTPRE has 0 variants, skipping\n",
      "[INFO]\tGene MIR455 has 0 variants, skipping\n",
      "[INFO]\tGene MIR532 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01163 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02667 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4297 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf143 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE7P has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01164 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINH1 has 0 variants, skipping\n",
      "[INFO]\tGene XRN1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX2 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC1 has 0 variants, skipping\n",
      "[INFO]\tGene GPM6B has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5L has 0 variants, skipping\n",
      "[INFO]\tGene HP has 0 variants, skipping\n",
      "[INFO]\tGene CBLIF has 0 variants, skipping\n",
      "[INFO]\tGene CNGA2 has 0 variants, skipping\n",
      "[INFO]\tGene DPF1 has 0 variants, skipping\n",
      "[INFO]\tGene MED24 has 0 variants, skipping\n",
      "[INFO]\tGene MTDH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01166 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01167 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01168 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB4 has 0 variants, skipping\n",
      "[INFO]\tGene POTEH has 0 variants, skipping\n",
      "[INFO]\tGene MIR202HG has 0 variants, skipping\n",
      "[INFO]\tGene TBCD has 0 variants, skipping\n",
      "[INFO]\tGene TULP4 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00184 has 0 variants, skipping\n",
      "[INFO]\tGene MPPE1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM35 has 0 variants, skipping\n",
      "[INFO]\tGene NOS1AP has 0 variants, skipping\n",
      "[INFO]\tGene SART3 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3944 has 0 variants, skipping\n",
      "[INFO]\tGene FADS3 has 0 variants, skipping\n",
      "[INFO]\tGene NR1D1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM22 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1C has 0 variants, skipping\n",
      "[INFO]\tGene SCART1 has 0 variants, skipping\n",
      "[INFO]\tGene CUX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC117779438 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1109 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM2 has 0 variants, skipping\n",
      "[INFO]\tGene PDE10A has 0 variants, skipping\n",
      "[INFO]\tGene TUT7 has 0 variants, skipping\n",
      "[INFO]\tGene AGPS has 0 variants, skipping\n",
      "[INFO]\tGene PNMA5 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA8A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01001 has 0 variants, skipping\n",
      "[INFO]\tGene APOH has 0 variants, skipping\n",
      "[INFO]\tGene APOE has 0 variants, skipping\n",
      "[INFO]\tGene APOC3 has 0 variants, skipping\n",
      "[INFO]\tGene AMHR2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPA has 0 variants, skipping\n",
      "[INFO]\tGene DVL1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP5 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF13B has 0 variants, skipping\n",
      "[INFO]\tGene SLC35C2 has 0 variants, skipping\n",
      "[INFO]\tGene ERP44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF365 has 0 variants, skipping\n",
      "[INFO]\tGene CMTR1 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC4 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L24 has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP3 has 0 variants, skipping\n",
      "[INFO]\tGene THEM4 has 0 variants, skipping\n",
      "[INFO]\tGene TGM7 has 0 variants, skipping\n",
      "[INFO]\tGene STAT1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD4 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN2 has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR210HG has 0 variants, skipping\n",
      "[INFO]\tGene LOC143666 has 0 variants, skipping\n",
      "[INFO]\tGene KIF21B has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1B has 0 variants, skipping\n",
      "[INFO]\tGene DCT has 0 variants, skipping\n",
      "[INFO]\tGene CNN3 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A6 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR4 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI3K has 0 variants, skipping\n",
      "[INFO]\tGene ZFR has 0 variants, skipping\n",
      "[INFO]\tGene IER3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene MDH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC171391 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL3 has 0 variants, skipping\n",
      "[INFO]\tGene NNMT has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA52 has 0 variants, skipping\n",
      "[INFO]\tGene MRAS has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D31 has 0 variants, skipping\n",
      "[INFO]\tGene IDE has 0 variants, skipping\n",
      "[INFO]\tGene SMIM21 has 0 variants, skipping\n",
      "[INFO]\tGene CNGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2B has 0 variants, skipping\n",
      "[INFO]\tGene CNGA1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCNKB has 0 variants, skipping\n",
      "[INFO]\tGene CLCN5 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P14 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL9 has 0 variants, skipping\n",
      "[INFO]\tGene RGS8 has 0 variants, skipping\n",
      "[INFO]\tGene ATG14 has 0 variants, skipping\n",
      "[INFO]\tGene WDTC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6744 has 0 variants, skipping\n",
      "[INFO]\tGene TOLLIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBRSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4B has 0 variants, skipping\n",
      "[INFO]\tGene FAM13C has 0 variants, skipping\n",
      "[INFO]\tGene IGSF1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F12 has 0 variants, skipping\n",
      "[INFO]\tGene ANGEL1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCH has 0 variants, skipping\n",
      "[INFO]\tGene MIR4298 has 0 variants, skipping\n",
      "[INFO]\tGene LIPJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR7847 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01150 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCB1 has 0 variants, skipping\n",
      "[INFO]\tGene SFI1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF11 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34C has 0 variants, skipping\n",
      "[INFO]\tGene NKX1-2 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM114 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A15 has 0 variants, skipping\n",
      "[INFO]\tGene NACAD has 0 variants, skipping\n",
      "[INFO]\tGene PLAUR has 0 variants, skipping\n",
      "[INFO]\tGene KRTCAP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF615 has 0 variants, skipping\n",
      "[INFO]\tGene MMP14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD131 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01219 has 0 variants, skipping\n",
      "[INFO]\tGene JAM3 has 0 variants, skipping\n",
      "[INFO]\tGene LPXN has 0 variants, skipping\n",
      "[INFO]\tGene UBAP2L has 0 variants, skipping\n",
      "[INFO]\tGene MET has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF40 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX6 has 0 variants, skipping\n",
      "[INFO]\tGene PSORS1C3 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK4 has 0 variants, skipping\n",
      "[INFO]\tGene CD81-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SUN2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN2 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM5 has 0 variants, skipping\n",
      "[INFO]\tGene FIBCD1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85C has 0 variants, skipping\n",
      "[INFO]\tGene OCA2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1DN has 0 variants, skipping\n",
      "[INFO]\tGene TMEM259 has 0 variants, skipping\n",
      "[INFO]\tGene ACP5 has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF17 has 0 variants, skipping\n",
      "[INFO]\tGene BMP4 has 0 variants, skipping\n",
      "[INFO]\tGene BCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA54 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM100 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1AIP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T2 has 0 variants, skipping\n",
      "[INFO]\tGene REST has 0 variants, skipping\n",
      "[INFO]\tGene ADAR has 0 variants, skipping\n",
      "[INFO]\tGene ENO3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR644A has 0 variants, skipping\n",
      "[INFO]\tGene OR7E12P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927708 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM9 has 0 variants, skipping\n",
      "[INFO]\tGene ABI1 has 0 variants, skipping\n",
      "[INFO]\tGene ART5 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF2 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SOCS3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCSH has 0 variants, skipping\n",
      "[INFO]\tGene DPP4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF3 has 0 variants, skipping\n",
      "[INFO]\tGene CLU has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NRP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM41 has 0 variants, skipping\n",
      "[INFO]\tGene DLST has 0 variants, skipping\n",
      "[INFO]\tGene CHEK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1298 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441204 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A18AS has 0 variants, skipping\n",
      "[INFO]\tGene IGF2 has 0 variants, skipping\n",
      "[INFO]\tGene EZH1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2A has 0 variants, skipping\n",
      "[INFO]\tGene H2AJ has 0 variants, skipping\n",
      "[INFO]\tGene PP2D1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA11 has 0 variants, skipping\n",
      "[INFO]\tGene LPO has 0 variants, skipping\n",
      "[INFO]\tGene MIR105-2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K9 has 0 variants, skipping\n",
      "[INFO]\tGene CALD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4687 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene SYT6 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4D has 0 variants, skipping\n",
      "[INFO]\tGene HAUS4 has 0 variants, skipping\n",
      "[INFO]\tGene PMS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA13 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF6 has 0 variants, skipping\n",
      "[INFO]\tGene HBBP1 has 0 variants, skipping\n",
      "[INFO]\tGene BGLT3 has 0 variants, skipping\n",
      "[INFO]\tGene USP39 has 0 variants, skipping\n",
      "[INFO]\tGene TEX13A has 0 variants, skipping\n",
      "[INFO]\tGene OLFM5P has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD42 has 0 variants, skipping\n",
      "[INFO]\tGene EYA2 has 0 variants, skipping\n",
      "[INFO]\tGene SREBF2 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF536 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM207A has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL7A has 0 variants, skipping\n",
      "[INFO]\tGene ACTL7B has 0 variants, skipping\n",
      "[INFO]\tGene SMR3B has 0 variants, skipping\n",
      "[INFO]\tGene CFHR4 has 0 variants, skipping\n",
      "[INFO]\tGene EDDM3A has 0 variants, skipping\n",
      "[INFO]\tGene FGL2 has 0 variants, skipping\n",
      "[INFO]\tGene NMU has 0 variants, skipping\n",
      "[INFO]\tGene ME3 has 0 variants, skipping\n",
      "[INFO]\tGene CTPS1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf27A has 0 variants, skipping\n",
      "[INFO]\tGene CENPS-CORT has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3L has 0 variants, skipping\n",
      "[INFO]\tGene RPL19 has 0 variants, skipping\n",
      "[INFO]\tGene REEP2 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1I2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF212 has 0 variants, skipping\n",
      "[INFO]\tGene GVINP1 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD1 has 0 variants, skipping\n",
      "[INFO]\tGene AUH has 0 variants, skipping\n",
      "[INFO]\tGene MSR1 has 0 variants, skipping\n",
      "[INFO]\tGene MSN has 0 variants, skipping\n",
      "[INFO]\tGene MSMB has 0 variants, skipping\n",
      "[INFO]\tGene MTAP has 0 variants, skipping\n",
      "[INFO]\tGene TOMM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506258 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC4 has 0 variants, skipping\n",
      "[INFO]\tGene GGT1 has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF2 has 0 variants, skipping\n",
      "[INFO]\tGene PCLAF has 0 variants, skipping\n",
      "[INFO]\tGene JADE3 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC136 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN1 has 0 variants, skipping\n",
      "[INFO]\tGene PMEL has 0 variants, skipping\n",
      "[INFO]\tGene NPL has 0 variants, skipping\n",
      "[INFO]\tGene SYNPR has 0 variants, skipping\n",
      "[INFO]\tGene TESPA1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5E1P has 0 variants, skipping\n",
      "[INFO]\tGene LOC283299 has 0 variants, skipping\n",
      "[INFO]\tGene CASC23 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH11 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2B has 0 variants, skipping\n",
      "[INFO]\tGene TUB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G2 has 0 variants, skipping\n",
      "[INFO]\tGene PBX4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A3 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC102B has 0 variants, skipping\n",
      "[INFO]\tGene DYTN has 0 variants, skipping\n",
      "[INFO]\tGene MEX3A has 0 variants, skipping\n",
      "[INFO]\tGene FMN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA2 has 0 variants, skipping\n",
      "[INFO]\tGene LRP5L has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP299 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724784 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D2A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIM1 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM1 has 0 variants, skipping\n",
      "[INFO]\tGene AUNIP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5691 has 0 variants, skipping\n",
      "[INFO]\tGene KRT8P41 has 0 variants, skipping\n",
      "[INFO]\tGene TP53BP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF257 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA23 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644656 has 0 variants, skipping\n",
      "[INFO]\tGene ZAP70 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23 has 0 variants, skipping\n",
      "[INFO]\tGene COMT has 0 variants, skipping\n",
      "[INFO]\tGene BRCA1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH2 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf204 has 0 variants, skipping\n",
      "[INFO]\tGene AGO3 has 0 variants, skipping\n",
      "[INFO]\tGene URGCP has 0 variants, skipping\n",
      "[INFO]\tGene MED9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928008 has 0 variants, skipping\n",
      "[INFO]\tGene CAND1.11 has 0 variants, skipping\n",
      "[INFO]\tGene RRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF2RA has 0 variants, skipping\n",
      "[INFO]\tGene COMMD7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4485 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf43 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA11 has 0 variants, skipping\n",
      "[INFO]\tGene CRACDL has 0 variants, skipping\n",
      "[INFO]\tGene FREM2 has 0 variants, skipping\n",
      "[INFO]\tGene GADL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12D has 0 variants, skipping\n",
      "[INFO]\tGene C1orf174 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL18 has 0 variants, skipping\n",
      "[INFO]\tGene CYP27A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP26A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP24A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP11A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP7A1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4B1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4A11 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A5 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2J2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2E1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD97 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928053 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL3 has 0 variants, skipping\n",
      "[INFO]\tGene NRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8070 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF99 has 0 variants, skipping\n",
      "[INFO]\tGene OC90 has 0 variants, skipping\n",
      "[INFO]\tGene API5 has 0 variants, skipping\n",
      "[INFO]\tGene POLG has 0 variants, skipping\n",
      "[INFO]\tGene HNF1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02547 has 0 variants, skipping\n",
      "[INFO]\tGene RAD9B has 0 variants, skipping\n",
      "[INFO]\tGene ASAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA8 has 0 variants, skipping\n",
      "[INFO]\tGene CLUH has 0 variants, skipping\n",
      "[INFO]\tGene ACTB has 0 variants, skipping\n",
      "[INFO]\tGene POFUT2 has 0 variants, skipping\n",
      "[INFO]\tGene RPP25L has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRB has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB5 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL6 has 0 variants, skipping\n",
      "[INFO]\tGene DOP1A has 0 variants, skipping\n",
      "[INFO]\tGene EBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene SESN1 has 0 variants, skipping\n",
      "[INFO]\tGene SIAE has 0 variants, skipping\n",
      "[INFO]\tGene PGM3 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF200 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372977 has 0 variants, skipping\n",
      "[INFO]\tGene NPPB has 0 variants, skipping\n",
      "[INFO]\tGene NPAT has 0 variants, skipping\n",
      "[INFO]\tGene NPTX2 has 0 variants, skipping\n",
      "[INFO]\tGene NPTX1 has 0 variants, skipping\n",
      "[INFO]\tGene TCTN1 has 0 variants, skipping\n",
      "[INFO]\tGene CA10 has 0 variants, skipping\n",
      "[INFO]\tGene OPRL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAA80 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267881 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC4 has 0 variants, skipping\n",
      "[INFO]\tGene CUL2 has 0 variants, skipping\n",
      "[INFO]\tGene GFAP has 0 variants, skipping\n",
      "[INFO]\tGene PYDC5 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645188 has 0 variants, skipping\n",
      "[INFO]\tGene EML4 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN15 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN12 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP45 has 0 variants, skipping\n",
      "[INFO]\tGene NARF has 0 variants, skipping\n",
      "[INFO]\tGene MYO1F has 0 variants, skipping\n",
      "[INFO]\tGene BAMBI has 0 variants, skipping\n",
      "[INFO]\tGene DEFA1B has 0 variants, skipping\n",
      "[INFO]\tGene RAC3 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf95 has 0 variants, skipping\n",
      "[INFO]\tGene FDPS has 0 variants, skipping\n",
      "[INFO]\tGene GDE1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507006 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM129 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927168 has 0 variants, skipping\n",
      "[INFO]\tGene ABI3BP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02548 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02545 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02683 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB9 has 0 variants, skipping\n",
      "[INFO]\tGene EOMES has 0 variants, skipping\n",
      "[INFO]\tGene ARMH2 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45A has 0 variants, skipping\n",
      "[INFO]\tGene CTSS has 0 variants, skipping\n",
      "[INFO]\tGene CRABP2 has 0 variants, skipping\n",
      "[INFO]\tGene DHX33 has 0 variants, skipping\n",
      "[INFO]\tGene MIR561 has 0 variants, skipping\n",
      "[INFO]\tGene GET1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D15 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF396 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF432 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7 has 0 variants, skipping\n",
      "[INFO]\tGene CCNE1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF79 has 0 variants, skipping\n",
      "[INFO]\tGene CCNI has 0 variants, skipping\n",
      "[INFO]\tGene NBDY has 0 variants, skipping\n",
      "[INFO]\tGene TRIM16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6073 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY5 has 0 variants, skipping\n",
      "[INFO]\tGene IBTK has 0 variants, skipping\n",
      "[INFO]\tGene SNORD147 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E14P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM6 has 0 variants, skipping\n",
      "[INFO]\tGene RECQL5 has 0 variants, skipping\n",
      "[INFO]\tGene MTARC2 has 0 variants, skipping\n",
      "[INFO]\tGene VMO1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL12B has 0 variants, skipping\n",
      "[INFO]\tGene CTSE has 0 variants, skipping\n",
      "[INFO]\tGene PALM3 has 0 variants, skipping\n",
      "[INFO]\tGene NKIRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRQ has 0 variants, skipping\n",
      "[INFO]\tGene GLI1 has 0 variants, skipping\n",
      "[INFO]\tGene NEU4 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM3 has 0 variants, skipping\n",
      "[INFO]\tGene TLR10 has 0 variants, skipping\n",
      "[INFO]\tGene GSR has 0 variants, skipping\n",
      "[INFO]\tGene MEPCE has 0 variants, skipping\n",
      "[INFO]\tGene COPS4 has 0 variants, skipping\n",
      "[INFO]\tGene AP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF16 has 0 variants, skipping\n",
      "[INFO]\tGene TUNAR has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01611 has 0 variants, skipping\n",
      "[INFO]\tGene THRA1/BTR has 0 variants, skipping\n",
      "[INFO]\tGene ENC1 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG1IP has 0 variants, skipping\n",
      "[INFO]\tGene METAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF623 has 0 variants, skipping\n",
      "[INFO]\tGene SAA3P has 0 variants, skipping\n",
      "[INFO]\tGene FBXL12 has 0 variants, skipping\n",
      "[INFO]\tGene MTIF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3159 has 0 variants, skipping\n",
      "[INFO]\tGene XIRP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFY has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP2 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP2 has 0 variants, skipping\n",
      "[INFO]\tGene DENND5B has 0 variants, skipping\n",
      "[INFO]\tGene STRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF415 has 0 variants, skipping\n",
      "[INFO]\tGene ATG2A has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01094 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00707 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1B has 0 variants, skipping\n",
      "[INFO]\tGene NEK9 has 0 variants, skipping\n",
      "[INFO]\tGene TPP2 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4486 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126784 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB17 has 0 variants, skipping\n",
      "[INFO]\tGene COLQ has 0 variants, skipping\n",
      "[INFO]\tGene LILRA4 has 0 variants, skipping\n",
      "[INFO]\tGene IL36RN has 0 variants, skipping\n",
      "[INFO]\tGene H2AP has 0 variants, skipping\n",
      "[INFO]\tGene HYAL4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4694 has 0 variants, skipping\n",
      "[INFO]\tGene GPSM3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKEF1 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK9 has 0 variants, skipping\n",
      "[INFO]\tGene ARID5A has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM117 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM68 has 0 variants, skipping\n",
      "[INFO]\tGene PREPL has 0 variants, skipping\n",
      "[INFO]\tGene ITM2A has 0 variants, skipping\n",
      "[INFO]\tGene MCFD2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R13L has 0 variants, skipping\n",
      "[INFO]\tGene DHX15 has 0 variants, skipping\n",
      "[INFO]\tGene DHX9 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8054 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14C has 0 variants, skipping\n",
      "[INFO]\tGene ANO3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIO3 has 0 variants, skipping\n",
      "[INFO]\tGene DHODH has 0 variants, skipping\n",
      "[INFO]\tGene DHCR7 has 0 variants, skipping\n",
      "[INFO]\tGene DECR1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK2A has 0 variants, skipping\n",
      "[INFO]\tGene AMOTL2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2A has 0 variants, skipping\n",
      "[INFO]\tGene CDK2AP2 has 0 variants, skipping\n",
      "[INFO]\tGene MLPH has 0 variants, skipping\n",
      "[INFO]\tGene FLYWCH1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8087 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00678 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf86 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02546 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01616 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY4 has 0 variants, skipping\n",
      "[INFO]\tGene CHID1 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF1A has 0 variants, skipping\n",
      "[INFO]\tGene RARA has 0 variants, skipping\n",
      "[INFO]\tGene TKT has 0 variants, skipping\n",
      "[INFO]\tGene PTGS2 has 0 variants, skipping\n",
      "[INFO]\tGene PAX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PAUPAR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506675 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA88 has 0 variants, skipping\n",
      "[INFO]\tGene IDNK has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B3P has 0 variants, skipping\n",
      "[INFO]\tGene CEMIP2 has 0 variants, skipping\n",
      "[INFO]\tGene CEMIP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00294 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM251 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf43 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CADM1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO3-DT has 0 variants, skipping\n",
      "[INFO]\tGene TFG has 0 variants, skipping\n",
      "[INFO]\tGene OAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ1 has 0 variants, skipping\n",
      "[INFO]\tGene DRG1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB7 has 0 variants, skipping\n",
      "[INFO]\tGene MYO9B has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB1 has 0 variants, skipping\n",
      "[INFO]\tGene IREB2 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD17 has 0 variants, skipping\n",
      "[INFO]\tGene E2F8 has 0 variants, skipping\n",
      "[INFO]\tGene NARS2 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN7 has 0 variants, skipping\n",
      "[INFO]\tGene ARG1 has 0 variants, skipping\n",
      "[INFO]\tGene RRNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1343 has 0 variants, skipping\n",
      "[INFO]\tGene GPR34 has 0 variants, skipping\n",
      "[INFO]\tGene CIAPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene DUS1L has 0 variants, skipping\n",
      "[INFO]\tGene SNORD164 has 0 variants, skipping\n",
      "[INFO]\tGene PTGR2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP51A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC124 has 0 variants, skipping\n",
      "[INFO]\tGene ISOC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3973 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928510 has 0 variants, skipping\n",
      "[INFO]\tGene ASPDH has 0 variants, skipping\n",
      "[INFO]\tGene OR2J3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02790 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02762 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01493 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01499 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPKP3 has 0 variants, skipping\n",
      "[INFO]\tGene OR52I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E8 has 0 variants, skipping\n",
      "[INFO]\tGene PJA1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4A has 0 variants, skipping\n",
      "[INFO]\tGene TFB2M has 0 variants, skipping\n",
      "[INFO]\tGene MIR670 has 0 variants, skipping\n",
      "[INFO]\tGene MIR670HG has 0 variants, skipping\n",
      "[INFO]\tGene EVI5 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EYA1 has 0 variants, skipping\n",
      "[INFO]\tGene CD59 has 0 variants, skipping\n",
      "[INFO]\tGene CD44 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP5 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP3 has 0 variants, skipping\n",
      "[INFO]\tGene ORM2 has 0 variants, skipping\n",
      "[INFO]\tGene CIR1 has 0 variants, skipping\n",
      "[INFO]\tGene NBAS has 0 variants, skipping\n",
      "[INFO]\tGene SNCA has 0 variants, skipping\n",
      "[INFO]\tGene CEROX1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD5 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN3 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02685 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02687 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7154 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507384 has 0 variants, skipping\n",
      "[INFO]\tGene COX7B2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD7A has 0 variants, skipping\n",
      "[INFO]\tGene RAET1L has 0 variants, skipping\n",
      "[INFO]\tGene PTPRS has 0 variants, skipping\n",
      "[INFO]\tGene ALPL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM71 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD2 has 0 variants, skipping\n",
      "[INFO]\tGene EHD3 has 0 variants, skipping\n",
      "[INFO]\tGene ERVW-1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2E has 0 variants, skipping\n",
      "[INFO]\tGene SOX8 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A41 has 0 variants, skipping\n",
      "[INFO]\tGene AKIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928894 has 0 variants, skipping\n",
      "[INFO]\tGene NPY5R has 0 variants, skipping\n",
      "[INFO]\tGene RPL12 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL2 has 0 variants, skipping\n",
      "[INFO]\tGene LIPE has 0 variants, skipping\n",
      "[INFO]\tGene LCK has 0 variants, skipping\n",
      "[INFO]\tGene KIF25 has 0 variants, skipping\n",
      "[INFO]\tGene JUND has 0 variants, skipping\n",
      "[INFO]\tGene ITGAD has 0 variants, skipping\n",
      "[INFO]\tGene MIR4688 has 0 variants, skipping\n",
      "[INFO]\tGene LOC408186 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA8 has 0 variants, skipping\n",
      "[INFO]\tGene CNR1 has 0 variants, skipping\n",
      "[INFO]\tGene TYW1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM2 has 0 variants, skipping\n",
      "[INFO]\tGene GMIP has 0 variants, skipping\n",
      "[INFO]\tGene YEATS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIS3L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3160-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3160-2 has 0 variants, skipping\n",
      "[INFO]\tGene ART3 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB1 has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD3 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA8 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2D has 0 variants, skipping\n",
      "[INFO]\tGene SETBP1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN13 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL2 has 0 variants, skipping\n",
      "[INFO]\tGene JAML has 0 variants, skipping\n",
      "[INFO]\tGene CKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene ASB1 has 0 variants, skipping\n",
      "[INFO]\tGene TDP1 has 0 variants, skipping\n",
      "[INFO]\tGene MCC has 0 variants, skipping\n",
      "[INFO]\tGene TIMM8A has 0 variants, skipping\n",
      "[INFO]\tGene HSDL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALCAM has 0 variants, skipping\n",
      "[INFO]\tGene RAET1E has 0 variants, skipping\n",
      "[INFO]\tGene MS4A15 has 0 variants, skipping\n",
      "[INFO]\tGene ATP4A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268355 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SECISBP2 has 0 variants, skipping\n",
      "[INFO]\tGene BRD4 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB4 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D3 has 0 variants, skipping\n",
      "[INFO]\tGene DAG1 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD67 has 0 variants, skipping\n",
      "[INFO]\tGene LRP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AMN1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS4 has 0 variants, skipping\n",
      "[INFO]\tGene JAZF1 has 0 variants, skipping\n",
      "[INFO]\tGene SVOPL has 0 variants, skipping\n",
      "[INFO]\tGene EBF3 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC107 has 0 variants, skipping\n",
      "[INFO]\tGene TEX50 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6745 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G3 has 0 variants, skipping\n",
      "[INFO]\tGene GPC4 has 0 variants, skipping\n",
      "[INFO]\tGene FABP6 has 0 variants, skipping\n",
      "[INFO]\tGene FABP5 has 0 variants, skipping\n",
      "[INFO]\tGene FABP1 has 0 variants, skipping\n",
      "[INFO]\tGene FABP4 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL1 has 0 variants, skipping\n",
      "[INFO]\tGene GRID2 has 0 variants, skipping\n",
      "[INFO]\tGene GPX5 has 0 variants, skipping\n",
      "[INFO]\tGene CHKB has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA1 has 0 variants, skipping\n",
      "[INFO]\tGene CRKL has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3 has 0 variants, skipping\n",
      "[INFO]\tGene CRK has 0 variants, skipping\n",
      "[INFO]\tGene CEBPD has 0 variants, skipping\n",
      "[INFO]\tGene COX6A2 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC5 has 0 variants, skipping\n",
      "[INFO]\tGene IMMP1L has 0 variants, skipping\n",
      "[INFO]\tGene SPRR3 has 0 variants, skipping\n",
      "[INFO]\tGene LUZP2 has 0 variants, skipping\n",
      "[INFO]\tGene SGPL1 has 0 variants, skipping\n",
      "[INFO]\tGene CTLA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCD5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP8 has 0 variants, skipping\n",
      "[INFO]\tGene KMT5B has 0 variants, skipping\n",
      "[INFO]\tGene HTR3B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM196 has 0 variants, skipping\n",
      "[INFO]\tGene RIN1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J1 has 0 variants, skipping\n",
      "[INFO]\tGene PAAF1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1E has 0 variants, skipping\n",
      "[INFO]\tGene CCKBR has 0 variants, skipping\n",
      "[INFO]\tGene FANK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A13 has 0 variants, skipping\n",
      "[INFO]\tGene MYC has 0 variants, skipping\n",
      "[INFO]\tGene MYF6 has 0 variants, skipping\n",
      "[INFO]\tGene XCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPF has 0 variants, skipping\n",
      "[INFO]\tGene SORD has 0 variants, skipping\n",
      "[INFO]\tGene HSF2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS5 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS8 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS3 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1B has 0 variants, skipping\n",
      "[INFO]\tGene DRP2 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR3 has 0 variants, skipping\n",
      "[INFO]\tGene TRDMT1 has 0 variants, skipping\n",
      "[INFO]\tGene GCKR has 0 variants, skipping\n",
      "[INFO]\tGene NCOA7 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14 has 0 variants, skipping\n",
      "[INFO]\tGene NUTF2 has 0 variants, skipping\n",
      "[INFO]\tGene PHYH has 0 variants, skipping\n",
      "[INFO]\tGene SFXN1 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX3 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3161 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376526 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1D1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT1 has 0 variants, skipping\n",
      "[INFO]\tGene MLKL has 0 variants, skipping\n",
      "[INFO]\tGene LOC440040 has 0 variants, skipping\n",
      "[INFO]\tGene TFEB has 0 variants, skipping\n",
      "[INFO]\tGene OR4C45 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646813 has 0 variants, skipping\n",
      "[INFO]\tGene CCNL2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51HP has 0 variants, skipping\n",
      "[INFO]\tGene RAD52 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC31 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E5P has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6128 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AK4P has 0 variants, skipping\n",
      "[INFO]\tGene HELZ has 0 variants, skipping\n",
      "[INFO]\tGene JOSD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF14 has 0 variants, skipping\n",
      "[INFO]\tGene RAB27A has 0 variants, skipping\n",
      "[INFO]\tGene RAB4A has 0 variants, skipping\n",
      "[INFO]\tGene POU4F2 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL4 has 0 variants, skipping\n",
      "[INFO]\tGene DXO has 0 variants, skipping\n",
      "[INFO]\tGene FARP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR122 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L2 has 0 variants, skipping\n",
      "[INFO]\tGene EDN3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC47A2 has 0 variants, skipping\n",
      "[INFO]\tGene TMX2-CTNND1 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GLT1D1 has 0 variants, skipping\n",
      "[INFO]\tGene ATL3 has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST1 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6B has 0 variants, skipping\n",
      "[INFO]\tGene MAGED1 has 0 variants, skipping\n",
      "[INFO]\tGene TRA2B has 0 variants, skipping\n",
      "[INFO]\tGene DYNLL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB4 has 0 variants, skipping\n",
      "[INFO]\tGene CALCOCO1 has 0 variants, skipping\n",
      "[INFO]\tGene TLE3 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP91-CNTF has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283194 has 0 variants, skipping\n",
      "[INFO]\tGene GSTCD has 0 variants, skipping\n",
      "[INFO]\tGene CXCR2 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKB has 0 variants, skipping\n",
      "[INFO]\tGene DUSP23 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111A-DT has 0 variants, skipping\n",
      "[INFO]\tGene AZGP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATR has 0 variants, skipping\n",
      "[INFO]\tGene HARS1 has 0 variants, skipping\n",
      "[INFO]\tGene BID has 0 variants, skipping\n",
      "[INFO]\tGene ROBO4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR87 has 0 variants, skipping\n",
      "[INFO]\tGene OR10V2P has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22A has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE16 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP1 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC8 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7C has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CB has 0 variants, skipping\n",
      "[INFO]\tGene MIR6503 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC6 has 0 variants, skipping\n",
      "[INFO]\tGene DPEP2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7 has 0 variants, skipping\n",
      "[INFO]\tGene MAX has 0 variants, skipping\n",
      "[INFO]\tGene GP6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00301 has 0 variants, skipping\n",
      "[INFO]\tGene FAAH has 0 variants, skipping\n",
      "[INFO]\tGene CDK10 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1A2 has 0 variants, skipping\n",
      "[INFO]\tGene IL21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF233 has 0 variants, skipping\n",
      "[INFO]\tGene SAP130 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4488 has 0 variants, skipping\n",
      "[INFO]\tGene RPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927495 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP0P2 has 0 variants, skipping\n",
      "[INFO]\tGene THSD4 has 0 variants, skipping\n",
      "[INFO]\tGene REV3L has 0 variants, skipping\n",
      "[INFO]\tGene RPL5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF829 has 0 variants, skipping\n",
      "[INFO]\tGene MIR611 has 0 variants, skipping\n",
      "[INFO]\tGene CD28 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6746 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY6-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4725 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K6 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3654 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6747 has 0 variants, skipping\n",
      "[INFO]\tGene PTGR1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf81 has 0 variants, skipping\n",
      "[INFO]\tGene ADA2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM39 has 0 variants, skipping\n",
      "[INFO]\tGene POP1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf98 has 0 variants, skipping\n",
      "[INFO]\tGene SHBG has 0 variants, skipping\n",
      "[INFO]\tGene POMC has 0 variants, skipping\n",
      "[INFO]\tGene PON3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA57 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-3 has 0 variants, skipping\n",
      "[INFO]\tGene CSPG5 has 0 variants, skipping\n",
      "[INFO]\tGene OR52W1 has 0 variants, skipping\n",
      "[INFO]\tGene CLK3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPUL2-BSCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6748 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR2 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6514 has 0 variants, skipping\n",
      "[INFO]\tGene WDR89 has 0 variants, skipping\n",
      "[INFO]\tGene APC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369332 has 0 variants, skipping\n",
      "[INFO]\tGene GLOD5 has 0 variants, skipping\n",
      "[INFO]\tGene ADAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TCF20 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MACC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROGDI has 0 variants, skipping\n",
      "[INFO]\tGene FKBP14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD25 has 0 variants, skipping\n",
      "[INFO]\tGene DANCR has 0 variants, skipping\n",
      "[INFO]\tGene LINC00339 has 0 variants, skipping\n",
      "[INFO]\tGene WT1-AS has 0 variants, skipping\n",
      "[INFO]\tGene PTENP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD7 has 0 variants, skipping\n",
      "[INFO]\tGene COPB2 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L6 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA3 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRQ has 0 variants, skipping\n",
      "[INFO]\tGene ARAP2 has 0 variants, skipping\n",
      "[INFO]\tGene MYEOV has 0 variants, skipping\n",
      "[INFO]\tGene NEU3 has 0 variants, skipping\n",
      "[INFO]\tGene SATL1 has 0 variants, skipping\n",
      "[INFO]\tGene RAG1 has 0 variants, skipping\n",
      "[INFO]\tGene PER3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene MMS22L has 0 variants, skipping\n",
      "[INFO]\tGene MECOM has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF18 has 0 variants, skipping\n",
      "[INFO]\tGene BMP8B has 0 variants, skipping\n",
      "[INFO]\tGene BMP6 has 0 variants, skipping\n",
      "[INFO]\tGene BNC1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR5 has 0 variants, skipping\n",
      "[INFO]\tGene NNAT has 0 variants, skipping\n",
      "[INFO]\tGene PCSK2 has 0 variants, skipping\n",
      "[INFO]\tGene CD40 has 0 variants, skipping\n",
      "[INFO]\tGene RPS29 has 0 variants, skipping\n",
      "[INFO]\tGene CD79A has 0 variants, skipping\n",
      "[INFO]\tGene CD72 has 0 variants, skipping\n",
      "[INFO]\tGene CD69 has 0 variants, skipping\n",
      "[INFO]\tGene CD63 has 0 variants, skipping\n",
      "[INFO]\tGene CD58 has 0 variants, skipping\n",
      "[INFO]\tGene RPL8 has 0 variants, skipping\n",
      "[INFO]\tGene STAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CUL5 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02743 has 0 variants, skipping\n",
      "[INFO]\tGene DBNL has 0 variants, skipping\n",
      "[INFO]\tGene HFM1 has 0 variants, skipping\n",
      "[INFO]\tGene RFLNB has 0 variants, skipping\n",
      "[INFO]\tGene SELENOV has 0 variants, skipping\n",
      "[INFO]\tGene ANKDD1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02735 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-1 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKB has 0 variants, skipping\n",
      "[INFO]\tGene ITGB8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7155 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1237 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM4 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKMY2 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP37 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106C has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A5 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf25 has 0 variants, skipping\n",
      "[INFO]\tGene GID4 has 0 variants, skipping\n",
      "[INFO]\tGene DDA1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT9 has 0 variants, skipping\n",
      "[INFO]\tGene CAMKV has 0 variants, skipping\n",
      "[INFO]\tGene LYNX1-SLURP2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX50 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1B has 0 variants, skipping\n",
      "[INFO]\tGene DBNDD1 has 0 variants, skipping\n",
      "[INFO]\tGene METRN has 0 variants, skipping\n",
      "[INFO]\tGene LRRC61 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE4 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A7 has 0 variants, skipping\n",
      "[INFO]\tGene APC has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-2HG has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL7 has 0 variants, skipping\n",
      "[INFO]\tGene PXDN has 0 variants, skipping\n",
      "[INFO]\tGene SIN3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6750 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6749 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2-SNX15 has 0 variants, skipping\n",
      "[INFO]\tGene SASS6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6879 has 0 variants, skipping\n",
      "[INFO]\tGene CDK18 has 0 variants, skipping\n",
      "[INFO]\tGene LNCOG has 0 variants, skipping\n",
      "[INFO]\tGene DLEU2 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7 has 0 variants, skipping\n",
      "[INFO]\tGene OR5L2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX14 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP27 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL49 has 0 variants, skipping\n",
      "[INFO]\tGene MYRIP has 0 variants, skipping\n",
      "[INFO]\tGene DENND1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC728975 has 0 variants, skipping\n",
      "[INFO]\tGene NTN4 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A20P has 0 variants, skipping\n",
      "[INFO]\tGene PAK5 has 0 variants, skipping\n",
      "[INFO]\tGene RRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene AHCY has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD5 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR5 has 0 variants, skipping\n",
      "[INFO]\tGene BCKDK has 0 variants, skipping\n",
      "[INFO]\tGene TRAIP has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ3 has 0 variants, skipping\n",
      "[INFO]\tGene TXNL4A has 0 variants, skipping\n",
      "[INFO]\tGene DIABLO has 0 variants, skipping\n",
      "[INFO]\tGene RPE has 0 variants, skipping\n",
      "[INFO]\tGene KCNMA1 has 0 variants, skipping\n",
      "[INFO]\tGene BCAP29 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOB has 0 variants, skipping\n",
      "[INFO]\tGene PRL has 0 variants, skipping\n",
      "[INFO]\tGene SYNRG has 0 variants, skipping\n",
      "[INFO]\tGene FBXW7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4690 has 0 variants, skipping\n",
      "[INFO]\tGene RELA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4489 has 0 variants, skipping\n",
      "[INFO]\tGene MIR570 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPLL has 0 variants, skipping\n",
      "[INFO]\tGene UMOD has 0 variants, skipping\n",
      "[INFO]\tGene PCDH1 has 0 variants, skipping\n",
      "[INFO]\tGene AXL has 0 variants, skipping\n",
      "[INFO]\tGene NELFE has 0 variants, skipping\n",
      "[INFO]\tGene RBBP8 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARA3 has 0 variants, skipping\n",
      "[INFO]\tGene U2SURP has 0 variants, skipping\n",
      "[INFO]\tGene APEH has 0 variants, skipping\n",
      "[INFO]\tGene C1S has 0 variants, skipping\n",
      "[INFO]\tGene MTERF2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLPH3L has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2 has 0 variants, skipping\n",
      "[INFO]\tGene LGI2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1S has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2B has 0 variants, skipping\n",
      "[INFO]\tGene MAST2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF654 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13F has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC15 has 0 variants, skipping\n",
      "[INFO]\tGene TBL1XR1 has 0 variants, skipping\n",
      "[INFO]\tGene SF3A1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF8 has 0 variants, skipping\n",
      "[INFO]\tGene TAP1 has 0 variants, skipping\n",
      "[INFO]\tGene STX1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724064 has 0 variants, skipping\n",
      "[INFO]\tGene FLII has 0 variants, skipping\n",
      "[INFO]\tGene DUSP7 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP6 has 0 variants, skipping\n",
      "[INFO]\tGene MFF has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928069 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM6 has 0 variants, skipping\n",
      "[INFO]\tGene PICALM has 0 variants, skipping\n",
      "[INFO]\tGene SF3A2 has 0 variants, skipping\n",
      "[INFO]\tGene IMPG2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF20 has 0 variants, skipping\n",
      "[INFO]\tGene ISLR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A10 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS8 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12D has 0 variants, skipping\n",
      "[INFO]\tGene PEX14 has 0 variants, skipping\n",
      "[INFO]\tGene SETX has 0 variants, skipping\n",
      "[INFO]\tGene RBM14-RBM4 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf56 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AX has 0 variants, skipping\n",
      "[INFO]\tGene EDA has 0 variants, skipping\n",
      "[INFO]\tGene MEGF6 has 0 variants, skipping\n",
      "[INFO]\tGene ECH1 has 0 variants, skipping\n",
      "[INFO]\tGene CELSR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A1 has 0 variants, skipping\n",
      "[INFO]\tGene F8A3 has 0 variants, skipping\n",
      "[INFO]\tGene STOML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3163 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6860 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D4 has 0 variants, skipping\n",
      "[INFO]\tGene TTC21B has 0 variants, skipping\n",
      "[INFO]\tGene LRCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130987 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DL3 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS6 has 0 variants, skipping\n",
      "[INFO]\tGene DOK3 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC11 has 0 variants, skipping\n",
      "[INFO]\tGene SNPH has 0 variants, skipping\n",
      "[INFO]\tGene RFNG has 0 variants, skipping\n",
      "[INFO]\tGene SRSF3 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF2 has 0 variants, skipping\n",
      "[INFO]\tGene HES2 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MRRF has 0 variants, skipping\n",
      "[INFO]\tGene ABCC9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6752 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN3 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNTL1 has 0 variants, skipping\n",
      "[INFO]\tGene MXD1 has 0 variants, skipping\n",
      "[INFO]\tGene SURF2 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2GP has 0 variants, skipping\n",
      "[INFO]\tGene ACADVL has 0 variants, skipping\n",
      "[INFO]\tGene FAM86C2P has 0 variants, skipping\n",
      "[INFO]\tGene FBXO46 has 0 variants, skipping\n",
      "[INFO]\tGene PRPSAP2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR48 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724951 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4691 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1T1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6753 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R42 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO3 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF433 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF48 has 0 variants, skipping\n",
      "[INFO]\tGene DIMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A8 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R18 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM17 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC152 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB2 has 0 variants, skipping\n",
      "[INFO]\tGene AHI1 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL5 has 0 variants, skipping\n",
      "[INFO]\tGene MAJIN has 0 variants, skipping\n",
      "[INFO]\tGene CNOT8 has 0 variants, skipping\n",
      "[INFO]\tGene NUMBL has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP2-CSNK1E has 0 variants, skipping\n",
      "[INFO]\tGene EYA4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF333 has 0 variants, skipping\n",
      "[INFO]\tGene MYOT has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN4 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf202 has 0 variants, skipping\n",
      "[INFO]\tGene LAT has 0 variants, skipping\n",
      "[INFO]\tGene CYB561 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1958 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC92B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC6 has 0 variants, skipping\n",
      "[INFO]\tGene LRCOL1 has 0 variants, skipping\n",
      "[INFO]\tGene TLX1 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO3 has 0 variants, skipping\n",
      "[INFO]\tGene LXN has 0 variants, skipping\n",
      "[INFO]\tGene PAK6 has 0 variants, skipping\n",
      "[INFO]\tGene AMBRA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF850 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A8 has 0 variants, skipping\n",
      "[INFO]\tGene EVA1B has 0 variants, skipping\n",
      "[INFO]\tGene INTS13 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG4 has 0 variants, skipping\n",
      "[INFO]\tGene ALG8 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM8 has 0 variants, skipping\n",
      "[INFO]\tGene NOC4L has 0 variants, skipping\n",
      "[INFO]\tGene KCTD15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM38A has 0 variants, skipping\n",
      "[INFO]\tGene DDX54 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE21 has 0 variants, skipping\n",
      "[INFO]\tGene KXD1 has 0 variants, skipping\n",
      "[INFO]\tGene PVRIG has 0 variants, skipping\n",
      "[INFO]\tGene NABP2 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf26 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3164 has 0 variants, skipping\n",
      "[INFO]\tGene LOC338694 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC157 has 0 variants, skipping\n",
      "[INFO]\tGene OR51J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01488 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH15 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30E has 0 variants, skipping\n",
      "[INFO]\tGene LINC02584 has 0 variants, skipping\n",
      "[INFO]\tGene KLK13 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD3B has 0 variants, skipping\n",
      "[INFO]\tGene YPEL3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC3 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR645 has 0 variants, skipping\n",
      "[INFO]\tGene OGFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02878 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS4 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene CAV2 has 0 variants, skipping\n",
      "[INFO]\tGene CERS1 has 0 variants, skipping\n",
      "[INFO]\tGene MLX has 0 variants, skipping\n",
      "[INFO]\tGene STOM has 0 variants, skipping\n",
      "[INFO]\tGene POP5 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L9P has 0 variants, skipping\n",
      "[INFO]\tGene GNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene KNL1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP12 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN14 has 0 variants, skipping\n",
      "[INFO]\tGene MATK has 0 variants, skipping\n",
      "[INFO]\tGene TAF1C has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128494 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3165 has 0 variants, skipping\n",
      "[INFO]\tGene EML1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP1L has 0 variants, skipping\n",
      "[INFO]\tGene ELK4 has 0 variants, skipping\n",
      "[INFO]\tGene ELANE has 0 variants, skipping\n",
      "[INFO]\tGene GNAQ has 0 variants, skipping\n",
      "[INFO]\tGene CELA1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5A has 0 variants, skipping\n",
      "[INFO]\tGene GNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E has 0 variants, skipping\n",
      "[INFO]\tGene GNA15 has 0 variants, skipping\n",
      "[INFO]\tGene GNA11 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4A2 has 0 variants, skipping\n",
      "[INFO]\tGene EHHADH has 0 variants, skipping\n",
      "[INFO]\tGene GML has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B2 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC4 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC12 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT5H has 0 variants, skipping\n",
      "[INFO]\tGene ALG6 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL27A has 0 variants, skipping\n",
      "[INFO]\tGene ACTA1 has 0 variants, skipping\n",
      "[INFO]\tGene MEPE has 0 variants, skipping\n",
      "[INFO]\tGene GIGYF2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM156A has 0 variants, skipping\n",
      "[INFO]\tGene MIF4GD has 0 variants, skipping\n",
      "[INFO]\tGene DYRK1B has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4692 has 0 variants, skipping\n",
      "[INFO]\tGene SDHB has 0 variants, skipping\n",
      "[INFO]\tGene RBP3 has 0 variants, skipping\n",
      "[INFO]\tGene RARRES1 has 0 variants, skipping\n",
      "[INFO]\tGene RARS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2B has 0 variants, skipping\n",
      "[INFO]\tGene FLOT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287837 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF808 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC8 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP2 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf43 has 0 variants, skipping\n",
      "[INFO]\tGene APEX1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNS has 0 variants, skipping\n",
      "[INFO]\tGene DCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LETM2 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene THSD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11E has 0 variants, skipping\n",
      "[INFO]\tGene OGN has 0 variants, skipping\n",
      "[INFO]\tGene CNPY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF114 has 0 variants, skipping\n",
      "[INFO]\tGene COX6B2 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK1 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GCK has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AL has 0 variants, skipping\n",
      "[INFO]\tGene MIR623 has 0 variants, skipping\n",
      "[INFO]\tGene MIR620 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2B has 0 variants, skipping\n",
      "[INFO]\tGene YWHAG has 0 variants, skipping\n",
      "[INFO]\tGene WBP1 has 0 variants, skipping\n",
      "[INFO]\tGene VAX2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4696 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC3B has 0 variants, skipping\n",
      "[INFO]\tGene ATG16L1 has 0 variants, skipping\n",
      "[INFO]\tGene TOPORS has 0 variants, skipping\n",
      "[INFO]\tGene DKK2 has 0 variants, skipping\n",
      "[INFO]\tGene DKK4 has 0 variants, skipping\n",
      "[INFO]\tGene DKKL1 has 0 variants, skipping\n",
      "[INFO]\tGene BBC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB11 has 0 variants, skipping\n",
      "[INFO]\tGene CACYBP has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN13 has 0 variants, skipping\n",
      "[INFO]\tGene TAF5L has 0 variants, skipping\n",
      "[INFO]\tGene SNORD15A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD15B has 0 variants, skipping\n",
      "[INFO]\tGene AAGAB has 0 variants, skipping\n",
      "[INFO]\tGene RGS7BP has 0 variants, skipping\n",
      "[INFO]\tGene GLE1 has 0 variants, skipping\n",
      "[INFO]\tGene GHDC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369391 has 0 variants, skipping\n",
      "[INFO]\tGene PRIM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM180B has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC283214 has 0 variants, skipping\n",
      "[INFO]\tGene UVRAG-DT has 0 variants, skipping\n",
      "[INFO]\tGene MUCL1 has 0 variants, skipping\n",
      "[INFO]\tGene THAP12 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL5 has 0 variants, skipping\n",
      "[INFO]\tGene ERBIN has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2EP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928837 has 0 variants, skipping\n",
      "[INFO]\tGene TMEFF2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCK has 0 variants, skipping\n",
      "[INFO]\tGene DUSP26 has 0 variants, skipping\n",
      "[INFO]\tGene RPL15 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646029 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3A has 0 variants, skipping\n",
      "[INFO]\tGene CNTN1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL2P-SEC16B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN32 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF746 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928896 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A5 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5579 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4300HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4300 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1B has 0 variants, skipping\n",
      "[INFO]\tGene FDXR has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIDEAS has 0 variants, skipping\n",
      "[INFO]\tGene ATG4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70E has 0 variants, skipping\n",
      "[INFO]\tGene RAB30-DT has 0 variants, skipping\n",
      "[INFO]\tGene GBGT1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506282 has 0 variants, skipping\n",
      "[INFO]\tGene PCF11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC15 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK16 has 0 variants, skipping\n",
      "[INFO]\tGene PHF21A has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2 has 0 variants, skipping\n",
      "[INFO]\tGene IFT88 has 0 variants, skipping\n",
      "[INFO]\tGene CENPK has 0 variants, skipping\n",
      "[INFO]\tGene MDP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX5-GPRASP2 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFA has 0 variants, skipping\n",
      "[INFO]\tGene NAPA has 0 variants, skipping\n",
      "[INFO]\tGene NAPG has 0 variants, skipping\n",
      "[INFO]\tGene SNAP23 has 0 variants, skipping\n",
      "[INFO]\tGene SOHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRS has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF5 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL2 has 0 variants, skipping\n",
      "[INFO]\tGene CT55 has 0 variants, skipping\n",
      "[INFO]\tGene PIGX has 0 variants, skipping\n",
      "[INFO]\tGene C1orf56 has 0 variants, skipping\n",
      "[INFO]\tGene TIPIN has 0 variants, skipping\n",
      "[INFO]\tGene SSH3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS7 has 0 variants, skipping\n",
      "[INFO]\tGene GCM1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB121 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1L2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO7-PAM16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6755 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B14 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B11 has 0 variants, skipping\n",
      "[INFO]\tGene TUG1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK4 has 0 variants, skipping\n",
      "[INFO]\tGene GET1-SH3BGR has 0 variants, skipping\n",
      "[INFO]\tGene OR7E2P has 0 variants, skipping\n",
      "[INFO]\tGene CX3CR1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A21 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02753 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3166 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A10 has 0 variants, skipping\n",
      "[INFO]\tGene GRM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC91 has 0 variants, skipping\n",
      "[INFO]\tGene OR4S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N5 has 0 variants, skipping\n",
      "[INFO]\tGene HELT has 0 variants, skipping\n",
      "[INFO]\tGene RPL28 has 0 variants, skipping\n",
      "[INFO]\tGene ITPK1 has 0 variants, skipping\n",
      "[INFO]\tGene RBKS has 0 variants, skipping\n",
      "[INFO]\tGene LHPP has 0 variants, skipping\n",
      "[INFO]\tGene TRIM53AP has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51EP has 0 variants, skipping\n",
      "[INFO]\tGene FANCL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC81 has 0 variants, skipping\n",
      "[INFO]\tGene DSTN has 0 variants, skipping\n",
      "[INFO]\tGene INTS14 has 0 variants, skipping\n",
      "[INFO]\tGene ETNPPL has 0 variants, skipping\n",
      "[INFO]\tGene DISC1FP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283710 has 0 variants, skipping\n",
      "[INFO]\tGene MICU2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB1 has 0 variants, skipping\n",
      "[INFO]\tGene USF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA9 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA32 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A13 has 0 variants, skipping\n",
      "[INFO]\tGene COX5A has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3R1 has 0 variants, skipping\n",
      "[INFO]\tGene MRO has 0 variants, skipping\n",
      "[INFO]\tGene SSTR5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAST4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548L has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369438 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1A has 0 variants, skipping\n",
      "[INFO]\tGene SRSF8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929295 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129203 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L2 has 0 variants, skipping\n",
      "[INFO]\tGene JRKL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EZR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL4 has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02720 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02728 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02763 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02696 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02759 has 0 variants, skipping\n",
      "[INFO]\tGene CCN1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13I has 0 variants, skipping\n",
      "[INFO]\tGene FKBP9 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB11 has 0 variants, skipping\n",
      "[INFO]\tGene PGR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3920 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A2 has 0 variants, skipping\n",
      "[INFO]\tGene RGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNJL has 0 variants, skipping\n",
      "[INFO]\tGene DCAF17 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR3 has 0 variants, skipping\n",
      "[INFO]\tGene DHDDS has 0 variants, skipping\n",
      "[INFO]\tGene TAF7L has 0 variants, skipping\n",
      "[INFO]\tGene LONRF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928424 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723838 has 0 variants, skipping\n",
      "[INFO]\tGene CEP85 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM265 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC7A has 0 variants, skipping\n",
      "[INFO]\tGene HIBCH has 0 variants, skipping\n",
      "[INFO]\tGene PRR23B has 0 variants, skipping\n",
      "[INFO]\tGene INAFM1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC60 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOV has 0 variants, skipping\n",
      "[INFO]\tGene FLNA has 0 variants, skipping\n",
      "[INFO]\tGene TMX3 has 0 variants, skipping\n",
      "[INFO]\tGene DYDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL32 has 0 variants, skipping\n",
      "[INFO]\tGene NCLN has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB13 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4693 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02552 has 0 variants, skipping\n",
      "[INFO]\tGene MANEAL has 0 variants, skipping\n",
      "[INFO]\tGene SPEF2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC13 has 0 variants, skipping\n",
      "[INFO]\tGene SPICE1 has 0 variants, skipping\n",
      "[INFO]\tGene PORCN has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9 has 0 variants, skipping\n",
      "[INFO]\tGene DLL3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP15 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548J has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B3-SLCO1B7 has 0 variants, skipping\n",
      "[INFO]\tGene STX7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF841 has 0 variants, skipping\n",
      "[INFO]\tGene CASP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene AURKA has 0 variants, skipping\n",
      "[INFO]\tGene DENND1C has 0 variants, skipping\n",
      "[INFO]\tGene RPS20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928535 has 0 variants, skipping\n",
      "[INFO]\tGene GNAO1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT7 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA5 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H7 has 0 variants, skipping\n",
      "[INFO]\tGene OR4Q2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA5L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM273 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730098 has 0 variants, skipping\n",
      "[INFO]\tGene EMG1 has 0 variants, skipping\n",
      "[INFO]\tGene NAPEPLD has 0 variants, skipping\n",
      "[INFO]\tGene HHAT has 0 variants, skipping\n",
      "[INFO]\tGene MIR335 has 0 variants, skipping\n",
      "[INFO]\tGene MIR384 has 0 variants, skipping\n",
      "[INFO]\tGene MIR338 has 0 variants, skipping\n",
      "[INFO]\tGene MIR346 has 0 variants, skipping\n",
      "[INFO]\tGene MIR324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR345 has 0 variants, skipping\n",
      "[INFO]\tGene MIR331 has 0 variants, skipping\n",
      "[INFO]\tGene MIR148B has 0 variants, skipping\n",
      "[INFO]\tGene MIR133B has 0 variants, skipping\n",
      "[INFO]\tGene MIR135B has 0 variants, skipping\n",
      "[INFO]\tGene MIR326 has 0 variants, skipping\n",
      "[INFO]\tGene MIR337 has 0 variants, skipping\n",
      "[INFO]\tGene MIR342 has 0 variants, skipping\n",
      "[INFO]\tGene MIR330 has 0 variants, skipping\n",
      "[INFO]\tGene MIR340 has 0 variants, skipping\n",
      "[INFO]\tGene MIR383 has 0 variants, skipping\n",
      "[INFO]\tGene MIR382 has 0 variants, skipping\n",
      "[INFO]\tGene MIR381 has 0 variants, skipping\n",
      "[INFO]\tGene MIR379 has 0 variants, skipping\n",
      "[INFO]\tGene MIR377 has 0 variants, skipping\n",
      "[INFO]\tGene COL10A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01087 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF84 has 0 variants, skipping\n",
      "[INFO]\tGene DOK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF566 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC181 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP70 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02550 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4491 has 0 variants, skipping\n",
      "[INFO]\tGene IRX4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132078 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728196 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN4 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF4 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA8 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS55 has 0 variants, skipping\n",
      "[INFO]\tGene ADRB1 has 0 variants, skipping\n",
      "[INFO]\tGene PSEN1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2C has 0 variants, skipping\n",
      "[INFO]\tGene CDK16 has 0 variants, skipping\n",
      "[INFO]\tGene S100A5 has 0 variants, skipping\n",
      "[INFO]\tGene S100A4 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT1 has 0 variants, skipping\n",
      "[INFO]\tGene SORT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A4 has 0 variants, skipping\n",
      "[INFO]\tGene S100A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A2 has 0 variants, skipping\n",
      "[INFO]\tGene RYK has 0 variants, skipping\n",
      "[INFO]\tGene WWP2 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT14 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB2-C11orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP2B has 0 variants, skipping\n",
      "[INFO]\tGene CHTOP has 0 variants, skipping\n",
      "[INFO]\tGene BRINP3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM8B has 0 variants, skipping\n",
      "[INFO]\tGene NIF3L1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132686 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928847 has 0 variants, skipping\n",
      "[INFO]\tGene REEP1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C4 has 0 variants, skipping\n",
      "[INFO]\tGene SPTAN1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24OS has 0 variants, skipping\n",
      "[INFO]\tGene NCAM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP4 has 0 variants, skipping\n",
      "[INFO]\tGene PCOTH has 0 variants, skipping\n",
      "[INFO]\tGene CARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF532 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERB has 0 variants, skipping\n",
      "[INFO]\tGene DNAI4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF552 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3B has 0 variants, skipping\n",
      "[INFO]\tGene AGMAT has 0 variants, skipping\n",
      "[INFO]\tGene MMRN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLTM has 0 variants, skipping\n",
      "[INFO]\tGene PTCD2 has 0 variants, skipping\n",
      "[INFO]\tGene REXO5 has 0 variants, skipping\n",
      "[INFO]\tGene PLET1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCG has 0 variants, skipping\n",
      "[INFO]\tGene XPC has 0 variants, skipping\n",
      "[INFO]\tGene WNT11 has 0 variants, skipping\n",
      "[INFO]\tGene WNT7A has 0 variants, skipping\n",
      "[INFO]\tGene VIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC4 has 0 variants, skipping\n",
      "[INFO]\tGene TSN has 0 variants, skipping\n",
      "[INFO]\tGene TRPC6 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF6 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF4 has 0 variants, skipping\n",
      "[INFO]\tGene NETO2 has 0 variants, skipping\n",
      "[INFO]\tGene ALPG has 0 variants, skipping\n",
      "[INFO]\tGene CPVL has 0 variants, skipping\n",
      "[INFO]\tGene PLVAP has 0 variants, skipping\n",
      "[INFO]\tGene SCRT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC6 has 0 variants, skipping\n",
      "[INFO]\tGene EPPK1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF208 has 0 variants, skipping\n",
      "[INFO]\tGene PUS3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB33B has 0 variants, skipping\n",
      "[INFO]\tGene IP6K2 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12H has 0 variants, skipping\n",
      "[INFO]\tGene COPA has 0 variants, skipping\n",
      "[INFO]\tGene SPSB2 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14B has 0 variants, skipping\n",
      "[INFO]\tGene FAM222B has 0 variants, skipping\n",
      "[INFO]\tGene EPHA7 has 0 variants, skipping\n",
      "[INFO]\tGene USP7 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF599 has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132A has 0 variants, skipping\n",
      "[INFO]\tGene TTC12 has 0 variants, skipping\n",
      "[INFO]\tGene HPF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928940 has 0 variants, skipping\n",
      "[INFO]\tGene HNMT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC34 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB4 has 0 variants, skipping\n",
      "[INFO]\tGene HLF has 0 variants, skipping\n",
      "[INFO]\tGene CYP27B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369507 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00900 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205A has 0 variants, skipping\n",
      "[INFO]\tGene PRKRA has 0 variants, skipping\n",
      "[INFO]\tGene TXNIP has 0 variants, skipping\n",
      "[INFO]\tGene LNCOC1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM72B has 0 variants, skipping\n",
      "[INFO]\tGene RSPH9 has 0 variants, skipping\n",
      "[INFO]\tGene BEND6 has 0 variants, skipping\n",
      "[INFO]\tGene PYGO1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF23 has 0 variants, skipping\n",
      "[INFO]\tGene TARS2 has 0 variants, skipping\n",
      "[INFO]\tGene CFC1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652768 has 0 variants, skipping\n",
      "[INFO]\tGene RC3H2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD5 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13B has 0 variants, skipping\n",
      "[INFO]\tGene DHFR2 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2R1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM156B has 0 variants, skipping\n",
      "[INFO]\tGene ELP4 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A31 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC3A has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP28 has 0 variants, skipping\n",
      "[INFO]\tGene TACC2 has 0 variants, skipping\n",
      "[INFO]\tGene CSK has 0 variants, skipping\n",
      "[INFO]\tGene CEP104 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT4-ECE2 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf62 has 0 variants, skipping\n",
      "[INFO]\tGene MGST2 has 0 variants, skipping\n",
      "[INFO]\tGene NPR3 has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6L has 0 variants, skipping\n",
      "[INFO]\tGene CNOT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA9 has 0 variants, skipping\n",
      "[INFO]\tGene CDX2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131626 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF83 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR4 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT1 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929089 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388813 has 0 variants, skipping\n",
      "[INFO]\tGene ECI1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6716 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4492 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF292 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX10 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP64 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN12L has 0 variants, skipping\n",
      "[INFO]\tGene FFAR4 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS13 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LRTM1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71B has 0 variants, skipping\n",
      "[INFO]\tGene DYNLRB2 has 0 variants, skipping\n",
      "[INFO]\tGene SNCG has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14D has 0 variants, skipping\n",
      "[INFO]\tGene TOR4A has 0 variants, skipping\n",
      "[INFO]\tGene TRMT1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D1A has 0 variants, skipping\n",
      "[INFO]\tGene SNRK has 0 variants, skipping\n",
      "[INFO]\tGene C17orf78 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12F has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB21 has 0 variants, skipping\n",
      "[INFO]\tGene DTL has 0 variants, skipping\n",
      "[INFO]\tGene FAAH2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD150 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF589 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6756 has 0 variants, skipping\n",
      "[INFO]\tGene PEMT has 0 variants, skipping\n",
      "[INFO]\tGene CHAD has 0 variants, skipping\n",
      "[INFO]\tGene USP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C8 has 0 variants, skipping\n",
      "[INFO]\tGene SPAST has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724301 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984399 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM4 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC649133 has 0 variants, skipping\n",
      "[INFO]\tGene ETV3L has 0 variants, skipping\n",
      "[INFO]\tGene ZYG11A has 0 variants, skipping\n",
      "[INFO]\tGene GPR179 has 0 variants, skipping\n",
      "[INFO]\tGene SAE1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D3C has 0 variants, skipping\n",
      "[INFO]\tGene TOM1 has 0 variants, skipping\n",
      "[INFO]\tGene TOM1L1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP3 has 0 variants, skipping\n",
      "[INFO]\tGene CHAF1A has 0 variants, skipping\n",
      "[INFO]\tGene PIGK has 0 variants, skipping\n",
      "[INFO]\tGene RNF31 has 0 variants, skipping\n",
      "[INFO]\tGene RGL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929227 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929208 has 0 variants, skipping\n",
      "[INFO]\tGene CXADR has 0 variants, skipping\n",
      "[INFO]\tGene CYP2D6 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL5 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA9 has 0 variants, skipping\n",
      "[INFO]\tGene DMBT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT19 has 0 variants, skipping\n",
      "[INFO]\tGene PDXP-DT has 0 variants, skipping\n",
      "[INFO]\tGene GPR1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LIN9 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D24 has 0 variants, skipping\n",
      "[INFO]\tGene NIPBL has 0 variants, skipping\n",
      "[INFO]\tGene LOC341056 has 0 variants, skipping\n",
      "[INFO]\tGene INPP4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14E has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD14C has 0 variants, skipping\n",
      "[INFO]\tGene COL19A1 has 0 variants, skipping\n",
      "[INFO]\tGene COL16A1 has 0 variants, skipping\n",
      "[INFO]\tGene SCG2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4493 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00210 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HAV1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHD1L has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R10 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R7 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R13 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R8 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD7 has 0 variants, skipping\n",
      "[INFO]\tGene PTK7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929340 has 0 variants, skipping\n",
      "[INFO]\tGene HERC6 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF239 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507283 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD3 has 0 variants, skipping\n",
      "[INFO]\tGene GCLM has 0 variants, skipping\n",
      "[INFO]\tGene BCAR3 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL8 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4 has 0 variants, skipping\n",
      "[INFO]\tGene PHB2 has 0 variants, skipping\n",
      "[INFO]\tGene FMO5 has 0 variants, skipping\n",
      "[INFO]\tGene ARL13A has 0 variants, skipping\n",
      "[INFO]\tGene ANXA7 has 0 variants, skipping\n",
      "[INFO]\tGene PKNOX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX11B has 0 variants, skipping\n",
      "[INFO]\tGene DYRK4 has 0 variants, skipping\n",
      "[INFO]\tGene SIM2 has 0 variants, skipping\n",
      "[INFO]\tGene TARDBP has 0 variants, skipping\n",
      "[INFO]\tGene SIX6 has 0 variants, skipping\n",
      "[INFO]\tGene SHOC2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX42 has 0 variants, skipping\n",
      "[INFO]\tGene STT3A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT8B has 0 variants, skipping\n",
      "[INFO]\tGene XRCC4 has 0 variants, skipping\n",
      "[INFO]\tGene WNT2 has 0 variants, skipping\n",
      "[INFO]\tGene XPO1 has 0 variants, skipping\n",
      "[INFO]\tGene HYMAI has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A25 has 0 variants, skipping\n",
      "[INFO]\tGene EMX2OS has 0 variants, skipping\n",
      "[INFO]\tGene ISG20 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF606 has 0 variants, skipping\n",
      "[INFO]\tGene CEP95 has 0 variants, skipping\n",
      "[INFO]\tGene GLCE has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385B has 0 variants, skipping\n",
      "[INFO]\tGene SMUG1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2R1 has 0 variants, skipping\n",
      "[INFO]\tGene FYCO1 has 0 variants, skipping\n",
      "[INFO]\tGene GSEC has 0 variants, skipping\n",
      "[INFO]\tGene CYP26B1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA3 has 0 variants, skipping\n",
      "[INFO]\tGene SCO2 has 0 variants, skipping\n",
      "[INFO]\tGene GATD1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM7A-DT has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR761 has 0 variants, skipping\n",
      "[INFO]\tGene MTA3 has 0 variants, skipping\n",
      "[INFO]\tGene CTRB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929427 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3167 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02098 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6090 has 0 variants, skipping\n",
      "[INFO]\tGene FLYWCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929538 has 0 variants, skipping\n",
      "[INFO]\tGene CD22 has 0 variants, skipping\n",
      "[INFO]\tGene PRC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR323A has 0 variants, skipping\n",
      "[INFO]\tGene PROM1 has 0 variants, skipping\n",
      "[INFO]\tGene NECAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LUM has 0 variants, skipping\n",
      "[INFO]\tGene C16orf90 has 0 variants, skipping\n",
      "[INFO]\tGene LOC399975 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01395 has 0 variants, skipping\n",
      "[INFO]\tGene EXO5 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R35 has 0 variants, skipping\n",
      "[INFO]\tGene IFNL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF488 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22A has 0 variants, skipping\n",
      "[INFO]\tGene PIH1D1 has 0 variants, skipping\n",
      "[INFO]\tGene RADX has 0 variants, skipping\n",
      "[INFO]\tGene BCL11A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5C has 0 variants, skipping\n",
      "[INFO]\tGene FNBP4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM189A1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM127 has 0 variants, skipping\n",
      "[INFO]\tGene ODR4 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD8 has 0 variants, skipping\n",
      "[INFO]\tGene BORA has 0 variants, skipping\n",
      "[INFO]\tGene SNX22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00167 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF669 has 0 variants, skipping\n",
      "[INFO]\tGene ALKAL2 has 0 variants, skipping\n",
      "[INFO]\tGene BCAS4 has 0 variants, skipping\n",
      "[INFO]\tGene MXI1 has 0 variants, skipping\n",
      "[INFO]\tGene EI24 has 0 variants, skipping\n",
      "[INFO]\tGene USP45 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8052 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02551 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF454 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2 has 0 variants, skipping\n",
      "[INFO]\tGene PCYOX1L has 0 variants, skipping\n",
      "[INFO]\tGene FARS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929653 has 0 variants, skipping\n",
      "[INFO]\tGene NTM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF836 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA4 has 0 variants, skipping\n",
      "[INFO]\tGene NTM-IT has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02724 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4697 has 0 variants, skipping\n",
      "[INFO]\tGene MUC7 has 0 variants, skipping\n",
      "[INFO]\tGene UNC119 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD153 has 0 variants, skipping\n",
      "[INFO]\tGene CGNL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL26L1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC3 has 0 variants, skipping\n",
      "[INFO]\tGene EFNB1 has 0 variants, skipping\n",
      "[INFO]\tGene MSL1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507548 has 0 variants, skipping\n",
      "[INFO]\tGene AGTR2 has 0 variants, skipping\n",
      "[INFO]\tGene AGTR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44 has 0 variants, skipping\n",
      "[INFO]\tGene MTFR1L has 0 variants, skipping\n",
      "[INFO]\tGene TMUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MFRP has 0 variants, skipping\n",
      "[INFO]\tGene AFF1 has 0 variants, skipping\n",
      "[INFO]\tGene ESM1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS27A has 0 variants, skipping\n",
      "[INFO]\tGene ABCA4 has 0 variants, skipping\n",
      "[INFO]\tGene PAH has 0 variants, skipping\n",
      "[INFO]\tGene NRXN2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD8 has 0 variants, skipping\n",
      "[INFO]\tGene CALM1 has 0 variants, skipping\n",
      "[INFO]\tGene TINF2 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL2 has 0 variants, skipping\n",
      "[INFO]\tGene FGFR2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP69B has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATS2 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCB has 0 variants, skipping\n",
      "[INFO]\tGene ABCA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF789 has 0 variants, skipping\n",
      "[INFO]\tGene COG3 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL2 has 0 variants, skipping\n",
      "[INFO]\tGene CALN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A9 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC9 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R3 has 0 variants, skipping\n",
      "[INFO]\tGene GINS3 has 0 variants, skipping\n",
      "[INFO]\tGene PROK2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB4 has 0 variants, skipping\n",
      "[INFO]\tGene MKKS has 0 variants, skipping\n",
      "[INFO]\tGene KLHL9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM107B has 0 variants, skipping\n",
      "[INFO]\tGene GDPD2 has 0 variants, skipping\n",
      "[INFO]\tGene HNF1A has 0 variants, skipping\n",
      "[INFO]\tGene GBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF69 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL2 has 0 variants, skipping\n",
      "[INFO]\tGene WASH8P has 0 variants, skipping\n",
      "[INFO]\tGene PRMT9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC574538 has 0 variants, skipping\n",
      "[INFO]\tGene CLNS1A has 0 variants, skipping\n",
      "[INFO]\tGene TNNT3 has 0 variants, skipping\n",
      "[INFO]\tGene EPN2 has 0 variants, skipping\n",
      "[INFO]\tGene EXPH5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723544 has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369595 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1290 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1285-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1285-1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR5 has 0 variants, skipping\n",
      "[INFO]\tGene ZRSR2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR91 has 0 variants, skipping\n",
      "[INFO]\tGene TCF25 has 0 variants, skipping\n",
      "[INFO]\tGene EFR3B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF410 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F11 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSP1 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00942 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3649 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00940 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene AARD has 0 variants, skipping\n",
      "[INFO]\tGene MYOC has 0 variants, skipping\n",
      "[INFO]\tGene RTCA has 0 variants, skipping\n",
      "[INFO]\tGene PCBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CSTB has 0 variants, skipping\n",
      "[INFO]\tGene COL1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LYVE1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAFD1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL1 has 0 variants, skipping\n",
      "[INFO]\tGene JPT1 has 0 variants, skipping\n",
      "[INFO]\tGene NDEL1 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAMR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN31 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf89 has 0 variants, skipping\n",
      "[INFO]\tGene TCF4 has 0 variants, skipping\n",
      "[INFO]\tGene BCL7B has 0 variants, skipping\n",
      "[INFO]\tGene ANKUB1 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131107 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129307 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3I has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF706 has 0 variants, skipping\n",
      "[INFO]\tGene SCFD1 has 0 variants, skipping\n",
      "[INFO]\tGene KLF13 has 0 variants, skipping\n",
      "[INFO]\tGene GINS2 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1D has 0 variants, skipping\n",
      "[INFO]\tGene SAR1B has 0 variants, skipping\n",
      "[INFO]\tGene PLLP has 0 variants, skipping\n",
      "[INFO]\tGene C1QA has 0 variants, skipping\n",
      "[INFO]\tGene EIF3L has 0 variants, skipping\n",
      "[INFO]\tGene NIP7 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPE has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52L2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CEPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEANC2 has 0 variants, skipping\n",
      "[INFO]\tGene HPGD has 0 variants, skipping\n",
      "[INFO]\tGene DENND4B has 0 variants, skipping\n",
      "[INFO]\tGene FLACC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT3H has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02417 has 0 variants, skipping\n",
      "[INFO]\tGene THCAT155 has 0 variants, skipping\n",
      "[INFO]\tGene THRB has 0 variants, skipping\n",
      "[INFO]\tGene CCND2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PADI4 has 0 variants, skipping\n",
      "[INFO]\tGene NUPR1 has 0 variants, skipping\n",
      "[INFO]\tGene GMEB2 has 0 variants, skipping\n",
      "[INFO]\tGene OSTF1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC33 has 0 variants, skipping\n",
      "[INFO]\tGene ORC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B8 has 0 variants, skipping\n",
      "[INFO]\tGene OR7C2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX1 has 0 variants, skipping\n",
      "[INFO]\tGene CABP4 has 0 variants, skipping\n",
      "[INFO]\tGene ESRRB has 0 variants, skipping\n",
      "[INFO]\tGene FAM43A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02443 has 0 variants, skipping\n",
      "[INFO]\tGene MYL4 has 0 variants, skipping\n",
      "[INFO]\tGene CRPPA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA120 has 0 variants, skipping\n",
      "[INFO]\tGene KYAT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AT4 has 0 variants, skipping\n",
      "[INFO]\tGene LYAR has 0 variants, skipping\n",
      "[INFO]\tGene KIF3A has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA4 has 0 variants, skipping\n",
      "[INFO]\tGene CD27-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK17 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02570 has 0 variants, skipping\n",
      "[INFO]\tGene PARP12 has 0 variants, skipping\n",
      "[INFO]\tGene ACD has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH1 has 0 variants, skipping\n",
      "[INFO]\tGene TCIRG1 has 0 variants, skipping\n",
      "[INFO]\tGene AFG1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC02737 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02746 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02711 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02732 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF778 has 0 variants, skipping\n",
      "[INFO]\tGene MED18 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12B has 0 variants, skipping\n",
      "[INFO]\tGene ACADM has 0 variants, skipping\n",
      "[INFO]\tGene EFTUD2 has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA11 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA3 has 0 variants, skipping\n",
      "[INFO]\tGene TM6SF1 has 0 variants, skipping\n",
      "[INFO]\tGene CHAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MRS2 has 0 variants, skipping\n",
      "[INFO]\tGene KIDINS220 has 0 variants, skipping\n",
      "[INFO]\tGene NIFK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5A has 0 variants, skipping\n",
      "[INFO]\tGene INHBC has 0 variants, skipping\n",
      "[INFO]\tGene TNIK has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369632 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13P5 has 0 variants, skipping\n",
      "[INFO]\tGene DSTNP2 has 0 variants, skipping\n",
      "[INFO]\tGene PROSER3 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU7-1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf57 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200CHG has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA12 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1C has 0 variants, skipping\n",
      "[INFO]\tGene MRPL40 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR4 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT36 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF9 has 0 variants, skipping\n",
      "[INFO]\tGene KRT37 has 0 variants, skipping\n",
      "[INFO]\tGene PEA15 has 0 variants, skipping\n",
      "[INFO]\tGene C1RL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELF4 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAVL3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4G2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4B has 0 variants, skipping\n",
      "[INFO]\tGene FUZ has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD3-TMEFF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A1 has 0 variants, skipping\n",
      "[INFO]\tGene MSANTD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC4-TTLL3 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP85L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF544 has 0 variants, skipping\n",
      "[INFO]\tGene SYT12 has 0 variants, skipping\n",
      "[INFO]\tGene KIF2C has 0 variants, skipping\n",
      "[INFO]\tGene RUSC2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724562 has 0 variants, skipping\n",
      "[INFO]\tGene CA8 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP4 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1P3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM66C has 0 variants, skipping\n",
      "[INFO]\tGene FAM86FP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02449 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00937 has 0 variants, skipping\n",
      "[INFO]\tGene CDH11 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM267 has 0 variants, skipping\n",
      "[INFO]\tGene CMC1 has 0 variants, skipping\n",
      "[INFO]\tGene L3HYPDH has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2A has 0 variants, skipping\n",
      "[INFO]\tGene PTAR1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL22 has 0 variants, skipping\n",
      "[INFO]\tGene MFFP2 has 0 variants, skipping\n",
      "[INFO]\tGene TMED8 has 0 variants, skipping\n",
      "[INFO]\tGene TFB1M has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD1L has 0 variants, skipping\n",
      "[INFO]\tGene WDR45 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9C1 has 0 variants, skipping\n",
      "[INFO]\tGene LY86-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene A2M-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GNB1 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1 has 0 variants, skipping\n",
      "[INFO]\tGene BEST2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB8 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ2A has 0 variants, skipping\n",
      "[INFO]\tGene BAZ1A has 0 variants, skipping\n",
      "[INFO]\tGene PLK4 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15L2 has 0 variants, skipping\n",
      "[INFO]\tGene A2MP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1244-4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00987 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642846 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02367 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928030 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728715 has 0 variants, skipping\n",
      "[INFO]\tGene DDX12P has 0 variants, skipping\n",
      "[INFO]\tGene LOC374443 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646588 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A4 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02470 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4C has 0 variants, skipping\n",
      "[INFO]\tGene GRK2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP32 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD3 has 0 variants, skipping\n",
      "[INFO]\tGene DCTD has 0 variants, skipping\n",
      "[INFO]\tGene ITPA has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2L has 0 variants, skipping\n",
      "[INFO]\tGene PHF8 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTNND2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02617 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L5 has 0 variants, skipping\n",
      "[INFO]\tGene GANC has 0 variants, skipping\n",
      "[INFO]\tGene WASHC2A has 0 variants, skipping\n",
      "[INFO]\tGene ERG has 0 variants, skipping\n",
      "[INFO]\tGene F8A1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP188 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02446 has 0 variants, skipping\n",
      "[INFO]\tGene KLRA1P has 0 variants, skipping\n",
      "[INFO]\tGene CRELD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN2 has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO1R has 0 variants, skipping\n",
      "[INFO]\tGene ZBBX has 0 variants, skipping\n",
      "[INFO]\tGene ZNF81 has 0 variants, skipping\n",
      "[INFO]\tGene PRH1-PRR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440084 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01252 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1A has 0 variants, skipping\n",
      "[INFO]\tGene CCP110 has 0 variants, skipping\n",
      "[INFO]\tGene NAGLU has 0 variants, skipping\n",
      "[INFO]\tGene RPL22 has 0 variants, skipping\n",
      "[INFO]\tGene PDE5A has 0 variants, skipping\n",
      "[INFO]\tGene MIR614 has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR7P1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1H has 0 variants, skipping\n",
      "[INFO]\tGene APOLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF665 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF525 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01559 has 0 variants, skipping\n",
      "[INFO]\tGene MCUR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM39B has 0 variants, skipping\n",
      "[INFO]\tGene PLBD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene QPCTL has 0 variants, skipping\n",
      "[INFO]\tGene SYTL5 has 0 variants, skipping\n",
      "[INFO]\tGene NAPSA has 0 variants, skipping\n",
      "[INFO]\tGene CREB5 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF6 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC9 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT21 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA7 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN5 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01489 has 0 variants, skipping\n",
      "[INFO]\tGene RERG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1R2 has 0 variants, skipping\n",
      "[INFO]\tGene LUZP6 has 0 variants, skipping\n",
      "[INFO]\tGene EXT1 has 0 variants, skipping\n",
      "[INFO]\tGene ETFA has 0 variants, skipping\n",
      "[INFO]\tGene MPV17L has 0 variants, skipping\n",
      "[INFO]\tGene BSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene GASK1B has 0 variants, skipping\n",
      "[INFO]\tGene SMOC2 has 0 variants, skipping\n",
      "[INFO]\tGene UPP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A1 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC8 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf85 has 0 variants, skipping\n",
      "[INFO]\tGene MIR544A has 0 variants, skipping\n",
      "[INFO]\tGene MIR574 has 0 variants, skipping\n",
      "[INFO]\tGene SKP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02378 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3974 has 0 variants, skipping\n",
      "[INFO]\tGene RERGL has 0 variants, skipping\n",
      "[INFO]\tGene GNRH1 has 0 variants, skipping\n",
      "[INFO]\tGene DDC has 0 variants, skipping\n",
      "[INFO]\tGene HTT has 0 variants, skipping\n",
      "[INFO]\tGene RNF157 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02468 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1D has 0 variants, skipping\n",
      "[INFO]\tGene BTAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE3 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD10 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF39 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518C has 0 variants, skipping\n",
      "[INFO]\tGene KCNH5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80E has 0 variants, skipping\n",
      "[INFO]\tGene DMBX1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP21 has 0 variants, skipping\n",
      "[INFO]\tGene CERS5 has 0 variants, skipping\n",
      "[INFO]\tGene CMAS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8A has 0 variants, skipping\n",
      "[INFO]\tGene WT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF483 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369691 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928441 has 0 variants, skipping\n",
      "[INFO]\tGene FERMT3 has 0 variants, skipping\n",
      "[INFO]\tGene MDS2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM10 has 0 variants, skipping\n",
      "[INFO]\tGene NAPB has 0 variants, skipping\n",
      "[INFO]\tGene COA6 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP6 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH1 has 0 variants, skipping\n",
      "[INFO]\tGene EXD2 has 0 variants, skipping\n",
      "[INFO]\tGene DOK2 has 0 variants, skipping\n",
      "[INFO]\tGene CHM has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505841 has 0 variants, skipping\n",
      "[INFO]\tGene SUV39H2 has 0 variants, skipping\n",
      "[INFO]\tGene FUT8 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA3 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG16 has 0 variants, skipping\n",
      "[INFO]\tGene PALM2AKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL6 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNE has 0 variants, skipping\n",
      "[INFO]\tGene POLK has 0 variants, skipping\n",
      "[INFO]\tGene BORCS8-MEF2B has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP10 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D21 has 0 variants, skipping\n",
      "[INFO]\tGene COQ10B has 0 variants, skipping\n",
      "[INFO]\tGene LEO1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4302 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NAB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF593 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf87 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC75B has 0 variants, skipping\n",
      "[INFO]\tGene IFNAR2 has 0 variants, skipping\n",
      "[INFO]\tGene ERVFRD-1 has 0 variants, skipping\n",
      "[INFO]\tGene DUOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP15 has 0 variants, skipping\n",
      "[INFO]\tGene TYMS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF215 has 0 variants, skipping\n",
      "[INFO]\tGene NOD1 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729291 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD5 has 0 variants, skipping\n",
      "[INFO]\tGene ATG9A has 0 variants, skipping\n",
      "[INFO]\tGene CYFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506606 has 0 variants, skipping\n",
      "[INFO]\tGene OVCH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AUTS2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNBL1 has 0 variants, skipping\n",
      "[INFO]\tGene STAB1 has 0 variants, skipping\n",
      "[INFO]\tGene STIL has 0 variants, skipping\n",
      "[INFO]\tGene RTF2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM54 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF341 has 0 variants, skipping\n",
      "[INFO]\tGene NAIP has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645485 has 0 variants, skipping\n",
      "[INFO]\tGene GNA14 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FCRLA has 0 variants, skipping\n",
      "[INFO]\tGene CD84 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH19 has 0 variants, skipping\n",
      "[INFO]\tGene GPRASP2 has 0 variants, skipping\n",
      "[INFO]\tGene OVOS2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP58 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ13224 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02387 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRB has 0 variants, skipping\n",
      "[INFO]\tGene DENND5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOT1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02422 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf74 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF569 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf25 has 0 variants, skipping\n",
      "[INFO]\tGene FAM187B has 0 variants, skipping\n",
      "[INFO]\tGene TTC9B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF583 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF565 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF560 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM91 has 0 variants, skipping\n",
      "[INFO]\tGene BEND7 has 0 variants, skipping\n",
      "[INFO]\tGene LMX1B has 0 variants, skipping\n",
      "[INFO]\tGene LOX has 0 variants, skipping\n",
      "[INFO]\tGene RNF5 has 0 variants, skipping\n",
      "[INFO]\tGene RLN1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD2 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3B has 0 variants, skipping\n",
      "[INFO]\tGene DDX52 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE1 has 0 variants, skipping\n",
      "[INFO]\tGene CRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02471 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAA1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOX1 has 0 variants, skipping\n",
      "[INFO]\tGene BBX has 0 variants, skipping\n",
      "[INFO]\tGene TREX1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2L has 0 variants, skipping\n",
      "[INFO]\tGene PRR4 has 0 variants, skipping\n",
      "[INFO]\tGene SNF8 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP12 has 0 variants, skipping\n",
      "[INFO]\tGene PXMP4 has 0 variants, skipping\n",
      "[INFO]\tGene SP140 has 0 variants, skipping\n",
      "[INFO]\tGene CHP1 has 0 variants, skipping\n",
      "[INFO]\tGene XPOT has 0 variants, skipping\n",
      "[INFO]\tGene HRH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02400 has 0 variants, skipping\n",
      "[INFO]\tGene YAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7851 has 0 variants, skipping\n",
      "[INFO]\tGene IFNGR1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL58 has 0 variants, skipping\n",
      "[INFO]\tGene HGSNAT has 0 variants, skipping\n",
      "[INFO]\tGene GPAT4 has 0 variants, skipping\n",
      "[INFO]\tGene CTSF has 0 variants, skipping\n",
      "[INFO]\tGene MYOM1 has 0 variants, skipping\n",
      "[INFO]\tGene MYH13 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE5 has 0 variants, skipping\n",
      "[INFO]\tGene NOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02402 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02451 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02450 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02461 has 0 variants, skipping\n",
      "[INFO]\tGene GRAPL has 0 variants, skipping\n",
      "[INFO]\tGene NSDHL has 0 variants, skipping\n",
      "[INFO]\tGene DPEP3 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM3 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG4 has 0 variants, skipping\n",
      "[INFO]\tGene GPX3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF397 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABI3 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA8P1 has 0 variants, skipping\n",
      "[INFO]\tGene CAB39 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00938 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF189 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF177 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288798 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4698 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369747 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02156 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf127 has 0 variants, skipping\n",
      "[INFO]\tGene ACY1 has 0 variants, skipping\n",
      "[INFO]\tGene MPPED2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF7 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK1 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3E has 0 variants, skipping\n",
      "[INFO]\tGene INPPL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRG1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTRL has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT24 has 0 variants, skipping\n",
      "[INFO]\tGene LRP10 has 0 variants, skipping\n",
      "[INFO]\tGene PAX7 has 0 variants, skipping\n",
      "[INFO]\tGene UNC50 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC4 has 0 variants, skipping\n",
      "[INFO]\tGene PAX3 has 0 variants, skipping\n",
      "[INFO]\tGene SPCS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10C1 has 0 variants, skipping\n",
      "[INFO]\tGene ZP4 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERG has 0 variants, skipping\n",
      "[INFO]\tGene C6orf47 has 0 variants, skipping\n",
      "[INFO]\tGene HMHB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF7 has 0 variants, skipping\n",
      "[INFO]\tGene GRHL3 has 0 variants, skipping\n",
      "[INFO]\tGene LSM2 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC2 has 0 variants, skipping\n",
      "[INFO]\tGene HAMP has 0 variants, skipping\n",
      "[INFO]\tGene CCAR2 has 0 variants, skipping\n",
      "[INFO]\tGene MAZ has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene GATAD1 has 0 variants, skipping\n",
      "[INFO]\tGene BRINP2 has 0 variants, skipping\n",
      "[INFO]\tGene POLD4 has 0 variants, skipping\n",
      "[INFO]\tGene HES4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40C has 0 variants, skipping\n",
      "[INFO]\tGene AP1M1 has 0 variants, skipping\n",
      "[INFO]\tGene MCHR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928841 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6505 has 0 variants, skipping\n",
      "[INFO]\tGene OR4P4 has 0 variants, skipping\n",
      "[INFO]\tGene AVL9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR1291 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA2B has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P3 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POC5 has 0 variants, skipping\n",
      "[INFO]\tGene RNF24 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS5 has 0 variants, skipping\n",
      "[INFO]\tGene RUBCNL has 0 variants, skipping\n",
      "[INFO]\tGene CERCAM has 0 variants, skipping\n",
      "[INFO]\tGene SETD4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS27 has 0 variants, skipping\n",
      "[INFO]\tGene SIMC1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF220 has 0 variants, skipping\n",
      "[INFO]\tGene SMG8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927267 has 0 variants, skipping\n",
      "[INFO]\tGene COLCA2 has 0 variants, skipping\n",
      "[INFO]\tGene EMC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO18A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100335030 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM159 has 0 variants, skipping\n",
      "[INFO]\tGene REL has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K2 has 0 variants, skipping\n",
      "[INFO]\tGene LARP6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM13B has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B3 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MRM3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL11 has 0 variants, skipping\n",
      "[INFO]\tGene INTS10 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS10 has 0 variants, skipping\n",
      "[INFO]\tGene CEP72 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TBCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT6 has 0 variants, skipping\n",
      "[INFO]\tGene ASPM has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724265 has 0 variants, skipping\n",
      "[INFO]\tGene BCDIN3D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB6 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02396 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02395 has 0 variants, skipping\n",
      "[INFO]\tGene SS18L2 has 0 variants, skipping\n",
      "[INFO]\tGene RSL24D1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL9 has 0 variants, skipping\n",
      "[INFO]\tGene BIN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GPN3 has 0 variants, skipping\n",
      "[INFO]\tGene ARPP21 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB1 has 0 variants, skipping\n",
      "[INFO]\tGene RAI1 has 0 variants, skipping\n",
      "[INFO]\tGene PTER has 0 variants, skipping\n",
      "[INFO]\tGene SMCP has 0 variants, skipping\n",
      "[INFO]\tGene CKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene GRINA has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM87B has 0 variants, skipping\n",
      "[INFO]\tGene SENCR has 0 variants, skipping\n",
      "[INFO]\tGene CCDC169-SOHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A47 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-2 has 0 variants, skipping\n",
      "[INFO]\tGene CLUL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1293 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-11 has 0 variants, skipping\n",
      "[INFO]\tGene MARK3 has 0 variants, skipping\n",
      "[INFO]\tGene PDS5A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM192 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL3 has 0 variants, skipping\n",
      "[INFO]\tGene PLPBP has 0 variants, skipping\n",
      "[INFO]\tGene DUSP10 has 0 variants, skipping\n",
      "[INFO]\tGene FZD10 has 0 variants, skipping\n",
      "[INFO]\tGene DDX20 has 0 variants, skipping\n",
      "[INFO]\tGene NFYC has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA8 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF1 has 0 variants, skipping\n",
      "[INFO]\tGene HSF4 has 0 variants, skipping\n",
      "[INFO]\tGene MPEG1 has 0 variants, skipping\n",
      "[INFO]\tGene PRB1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOP1B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD133 has 0 variants, skipping\n",
      "[INFO]\tGene VSNL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC16 has 0 variants, skipping\n",
      "[INFO]\tGene TYMSOS has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR4 has 0 variants, skipping\n",
      "[INFO]\tGene OSCP1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM2 has 0 variants, skipping\n",
      "[INFO]\tGene SFMBT1 has 0 variants, skipping\n",
      "[INFO]\tGene UMODL1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1B has 0 variants, skipping\n",
      "[INFO]\tGene ACTL6A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA3 has 0 variants, skipping\n",
      "[INFO]\tGene SART1 has 0 variants, skipping\n",
      "[INFO]\tGene MUL1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL24 has 0 variants, skipping\n",
      "[INFO]\tGene RNF128 has 0 variants, skipping\n",
      "[INFO]\tGene CARS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHPF has 0 variants, skipping\n",
      "[INFO]\tGene BLVRA has 0 variants, skipping\n",
      "[INFO]\tGene CHRAC1 has 0 variants, skipping\n",
      "[INFO]\tGene NEMP2 has 0 variants, skipping\n",
      "[INFO]\tGene CCM2 has 0 variants, skipping\n",
      "[INFO]\tGene BAX has 0 variants, skipping\n",
      "[INFO]\tGene BAD has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MPL has 0 variants, skipping\n",
      "[INFO]\tGene MYRF has 0 variants, skipping\n",
      "[INFO]\tGene ERC1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT78 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24B has 0 variants, skipping\n",
      "[INFO]\tGene ARID3B has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13A has 0 variants, skipping\n",
      "[INFO]\tGene APOL2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM9 has 0 variants, skipping\n",
      "[INFO]\tGene GRIFIN has 0 variants, skipping\n",
      "[INFO]\tGene LENG9 has 0 variants, skipping\n",
      "[INFO]\tGene ASNSD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E47P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00592 has 0 variants, skipping\n",
      "[INFO]\tGene KRT7-AS has 0 variants, skipping\n",
      "[INFO]\tGene KRT87P has 0 variants, skipping\n",
      "[INFO]\tGene PCTP has 0 variants, skipping\n",
      "[INFO]\tGene KRT8 has 0 variants, skipping\n",
      "[INFO]\tGene KRT4 has 0 variants, skipping\n",
      "[INFO]\tGene IPO5 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT73-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1G has 0 variants, skipping\n",
      "[INFO]\tGene BAAT has 0 variants, skipping\n",
      "[INFO]\tGene LINGO2 has 0 variants, skipping\n",
      "[INFO]\tGene TANK has 0 variants, skipping\n",
      "[INFO]\tGene ZNF607 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205C has 0 variants, skipping\n",
      "[INFO]\tGene LOC283335 has 0 variants, skipping\n",
      "[INFO]\tGene C4BPA has 0 variants, skipping\n",
      "[INFO]\tGene TSPO has 0 variants, skipping\n",
      "[INFO]\tGene BLVRB has 0 variants, skipping\n",
      "[INFO]\tGene BCKDHA has 0 variants, skipping\n",
      "[INFO]\tGene AMY2A has 0 variants, skipping\n",
      "[INFO]\tGene BDKRB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6757 has 0 variants, skipping\n",
      "[INFO]\tGene MYEF2 has 0 variants, skipping\n",
      "[INFO]\tGene NBN has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3614 has 0 variants, skipping\n",
      "[INFO]\tGene MIR28 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR9 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf62 has 0 variants, skipping\n",
      "[INFO]\tGene ISCA1 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT1L has 0 variants, skipping\n",
      "[INFO]\tGene SHCBP1L has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRNDE has 0 variants, skipping\n",
      "[INFO]\tGene PCSK9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652999 has 0 variants, skipping\n",
      "[INFO]\tGene IMP3 has 0 variants, skipping\n",
      "[INFO]\tGene ANPEP has 0 variants, skipping\n",
      "[INFO]\tGene RRP12 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT4 has 0 variants, skipping\n",
      "[INFO]\tGene CISTR has 0 variants, skipping\n",
      "[INFO]\tGene GNG11 has 0 variants, skipping\n",
      "[INFO]\tGene GMFB has 0 variants, skipping\n",
      "[INFO]\tGene HOXC-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ12825 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240735 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240734 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02381 has 0 variants, skipping\n",
      "[INFO]\tGene TEX38 has 0 variants, skipping\n",
      "[INFO]\tGene FAM186A has 0 variants, skipping\n",
      "[INFO]\tGene NFIA has 0 variants, skipping\n",
      "[INFO]\tGene VPS37A has 0 variants, skipping\n",
      "[INFO]\tGene JAGN1 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724050 has 0 variants, skipping\n",
      "[INFO]\tGene MED13 has 0 variants, skipping\n",
      "[INFO]\tGene THRAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MED12 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf159 has 0 variants, skipping\n",
      "[INFO]\tGene GNRHR has 0 variants, skipping\n",
      "[INFO]\tGene CSF2RB has 0 variants, skipping\n",
      "[INFO]\tGene GM2A has 0 variants, skipping\n",
      "[INFO]\tGene ADAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARA5 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP41 has 0 variants, skipping\n",
      "[INFO]\tGene SREK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP57L1 has 0 variants, skipping\n",
      "[INFO]\tGene RELL2 has 0 variants, skipping\n",
      "[INFO]\tGene COX18 has 0 variants, skipping\n",
      "[INFO]\tGene TCAIM has 0 variants, skipping\n",
      "[INFO]\tGene RABL3 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf38 has 0 variants, skipping\n",
      "[INFO]\tGene FAM126B has 0 variants, skipping\n",
      "[INFO]\tGene RTP5 has 0 variants, skipping\n",
      "[INFO]\tGene CES4A has 0 variants, skipping\n",
      "[INFO]\tGene PRTG has 0 variants, skipping\n",
      "[INFO]\tGene LMBR1L has 0 variants, skipping\n",
      "[INFO]\tGene AKT3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00337 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW3P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM143 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280B has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S1-RDH5 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHIA has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD4 has 0 variants, skipping\n",
      "[INFO]\tGene CORO7 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM198B has 0 variants, skipping\n",
      "[INFO]\tGene POSTN has 0 variants, skipping\n",
      "[INFO]\tGene TMEM158 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT10 has 0 variants, skipping\n",
      "[INFO]\tGene STS has 0 variants, skipping\n",
      "[INFO]\tGene TCOF1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL9 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H13 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3H has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369781 has 0 variants, skipping\n",
      "[INFO]\tGene RBM14 has 0 variants, skipping\n",
      "[INFO]\tGene LTF has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A7 has 0 variants, skipping\n",
      "[INFO]\tGene ACSF2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD16A has 0 variants, skipping\n",
      "[INFO]\tGene TMED1 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD18 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3G has 0 variants, skipping\n",
      "[INFO]\tGene RTN3 has 0 variants, skipping\n",
      "[INFO]\tGene INSL3 has 0 variants, skipping\n",
      "[INFO]\tGene URM1 has 0 variants, skipping\n",
      "[INFO]\tGene DACT3 has 0 variants, skipping\n",
      "[INFO]\tGene KPNB1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAGA has 0 variants, skipping\n",
      "[INFO]\tGene CHI3L1 has 0 variants, skipping\n",
      "[INFO]\tGene CHGA has 0 variants, skipping\n",
      "[INFO]\tGene CHD4 has 0 variants, skipping\n",
      "[INFO]\tGene CHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105C has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4C has 0 variants, skipping\n",
      "[INFO]\tGene DAOA has 0 variants, skipping\n",
      "[INFO]\tGene C7orf57 has 0 variants, skipping\n",
      "[INFO]\tGene AFDN has 0 variants, skipping\n",
      "[INFO]\tGene PTPRC has 0 variants, skipping\n",
      "[INFO]\tGene SNORD59B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD59A has 0 variants, skipping\n",
      "[INFO]\tGene CWC25 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37B has 0 variants, skipping\n",
      "[INFO]\tGene FCN2 has 0 variants, skipping\n",
      "[INFO]\tGene WARS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADARB1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD2 has 0 variants, skipping\n",
      "[INFO]\tGene TACR1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8 has 0 variants, skipping\n",
      "[INFO]\tGene RDH8 has 0 variants, skipping\n",
      "[INFO]\tGene CALY has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN4 has 0 variants, skipping\n",
      "[INFO]\tGene PODXL2 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG11B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354B has 0 variants, skipping\n",
      "[INFO]\tGene TECTB has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC6 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51C has 0 variants, skipping\n",
      "[INFO]\tGene TMEM185A has 0 variants, skipping\n",
      "[INFO]\tGene ARMC6 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene BEAN1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGPS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC63 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID4B has 0 variants, skipping\n",
      "[INFO]\tGene LRP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene HSCB has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPL has 0 variants, skipping\n",
      "[INFO]\tGene ADCK5 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM14 has 0 variants, skipping\n",
      "[INFO]\tGene MCEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene OGFRL1 has 0 variants, skipping\n",
      "[INFO]\tGene FILIP1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCG has 0 variants, skipping\n",
      "[INFO]\tGene NBPF20 has 0 variants, skipping\n",
      "[INFO]\tGene NOMO3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA7 has 0 variants, skipping\n",
      "[INFO]\tGene SAA2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2C1 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf86 has 0 variants, skipping\n",
      "[INFO]\tGene FARP1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGR has 0 variants, skipping\n",
      "[INFO]\tGene LCN10 has 0 variants, skipping\n",
      "[INFO]\tGene CD93 has 0 variants, skipping\n",
      "[INFO]\tGene PBK has 0 variants, skipping\n",
      "[INFO]\tGene CRB1 has 0 variants, skipping\n",
      "[INFO]\tGene FECH has 0 variants, skipping\n",
      "[INFO]\tGene RBL1 has 0 variants, skipping\n",
      "[INFO]\tGene DERL3 has 0 variants, skipping\n",
      "[INFO]\tGene PPWD1 has 0 variants, skipping\n",
      "[INFO]\tGene KIZ has 0 variants, skipping\n",
      "[INFO]\tGene JRK has 0 variants, skipping\n",
      "[INFO]\tGene VPS45 has 0 variants, skipping\n",
      "[INFO]\tGene TIMD4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6758 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF27 has 0 variants, skipping\n",
      "[INFO]\tGene LRP8 has 0 variants, skipping\n",
      "[INFO]\tGene KNSTRN has 0 variants, skipping\n",
      "[INFO]\tGene TP53INP1 has 0 variants, skipping\n",
      "[INFO]\tGene TBL1Y has 0 variants, skipping\n",
      "[INFO]\tGene SAMD3 has 0 variants, skipping\n",
      "[INFO]\tGene NF1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP57 has 0 variants, skipping\n",
      "[INFO]\tGene UBIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT4 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL1 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK2 has 0 variants, skipping\n",
      "[INFO]\tGene SAC3D1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR171 has 0 variants, skipping\n",
      "[INFO]\tGene DGKH has 0 variants, skipping\n",
      "[INFO]\tGene MED21 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927583 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF12 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene UEVLD has 0 variants, skipping\n",
      "[INFO]\tGene LIM2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6759 has 0 variants, skipping\n",
      "[INFO]\tGene PSCA has 0 variants, skipping\n",
      "[INFO]\tGene UBXN8 has 0 variants, skipping\n",
      "[INFO]\tGene REEP5 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283387 has 0 variants, skipping\n",
      "[INFO]\tGene RTL1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB9 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC10A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02403 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02388 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506869 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A18 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP100 has 0 variants, skipping\n",
      "[INFO]\tGene WDR53 has 0 variants, skipping\n",
      "[INFO]\tGene GEN1 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5KL1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF12 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC23 has 0 variants, skipping\n",
      "[INFO]\tGene VPS52 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SELL has 0 variants, skipping\n",
      "[INFO]\tGene RNF123 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN16 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF10-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR6125 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB11 has 0 variants, skipping\n",
      "[INFO]\tGene CARHSP1 has 0 variants, skipping\n",
      "[INFO]\tGene POLE3 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01465 has 0 variants, skipping\n",
      "[INFO]\tGene RXYLT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPN has 0 variants, skipping\n",
      "[INFO]\tGene PTGER2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGDS has 0 variants, skipping\n",
      "[INFO]\tGene PTGDR has 0 variants, skipping\n",
      "[INFO]\tGene PTAFR has 0 variants, skipping\n",
      "[INFO]\tGene SSTR4 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR3 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG2 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR2 has 0 variants, skipping\n",
      "[INFO]\tGene PANK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Z has 0 variants, skipping\n",
      "[INFO]\tGene CLCA1 has 0 variants, skipping\n",
      "[INFO]\tGene AP3S1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02389 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02231 has 0 variants, skipping\n",
      "[INFO]\tGene ERF has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369187 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02454 has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6074 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02425 has 0 variants, skipping\n",
      "[INFO]\tGene LLPH-DT has 0 variants, skipping\n",
      "[INFO]\tGene C12orf4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6502 has 0 variants, skipping\n",
      "[INFO]\tGene LMCD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724421 has 0 variants, skipping\n",
      "[INFO]\tGene DST-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP3L has 0 variants, skipping\n",
      "[INFO]\tGene DAAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02408 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02421 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01479 has 0 variants, skipping\n",
      "[INFO]\tGene HCST has 0 variants, skipping\n",
      "[INFO]\tGene UBL7 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02384 has 0 variants, skipping\n",
      "[INFO]\tGene SCOC has 0 variants, skipping\n",
      "[INFO]\tGene GORAB has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70G has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507250 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2 has 0 variants, skipping\n",
      "[INFO]\tGene SMC1B has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM7 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130075 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND6 has 0 variants, skipping\n",
      "[INFO]\tGene CTSK has 0 variants, skipping\n",
      "[INFO]\tGene MIR1279 has 0 variants, skipping\n",
      "[INFO]\tGene ART4 has 0 variants, skipping\n",
      "[INFO]\tGene TMPPE has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A2 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3913-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3913-2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF487 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC2 has 0 variants, skipping\n",
      "[INFO]\tGene HCRTR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928002 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD12 has 0 variants, skipping\n",
      "[INFO]\tGene PIKFYVE has 0 variants, skipping\n",
      "[INFO]\tGene MORC2 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM243B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A24 has 0 variants, skipping\n",
      "[INFO]\tGene SYT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF641 has 0 variants, skipping\n",
      "[INFO]\tGene GAB4 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNA has 0 variants, skipping\n",
      "[INFO]\tGene PIN4 has 0 variants, skipping\n",
      "[INFO]\tGene PIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CB has 0 variants, skipping\n",
      "[INFO]\tGene MOGS has 0 variants, skipping\n",
      "[INFO]\tGene MIR7977 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM7 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM181 has 0 variants, skipping\n",
      "[INFO]\tGene MRS2P2 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC11 has 0 variants, skipping\n",
      "[INFO]\tGene TRHDE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02444 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM2 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130268 has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX3X has 0 variants, skipping\n",
      "[INFO]\tGene PTPA has 0 variants, skipping\n",
      "[INFO]\tGene OGA has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC10 has 0 variants, skipping\n",
      "[INFO]\tGene TADA1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF510 has 0 variants, skipping\n",
      "[INFO]\tGene EHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene LMBRD2 has 0 variants, skipping\n",
      "[INFO]\tGene CAND1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA4 has 0 variants, skipping\n",
      "[INFO]\tGene SARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD5 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02424 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L3 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1C has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG11 has 0 variants, skipping\n",
      "[INFO]\tGene ELF3 has 0 variants, skipping\n",
      "[INFO]\tGene MVK has 0 variants, skipping\n",
      "[INFO]\tGene RD3 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf97 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01490 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf73 has 0 variants, skipping\n",
      "[INFO]\tGene MIR617 has 0 variants, skipping\n",
      "[INFO]\tGene ITGBL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4699 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM19 has 0 variants, skipping\n",
      "[INFO]\tGene TTC7A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF207 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3D has 0 variants, skipping\n",
      "[INFO]\tGene CANT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2B has 0 variants, skipping\n",
      "[INFO]\tGene GALM has 0 variants, skipping\n",
      "[INFO]\tGene PIH1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM43 has 0 variants, skipping\n",
      "[INFO]\tGene PLD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM45B has 0 variants, skipping\n",
      "[INFO]\tGene IFTAP has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02426 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC59 has 0 variants, skipping\n",
      "[INFO]\tGene LCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF133 has 0 variants, skipping\n",
      "[INFO]\tGene ADAD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9B1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN8 has 0 variants, skipping\n",
      "[INFO]\tGene L1CAM has 0 variants, skipping\n",
      "[INFO]\tGene GDNF has 0 variants, skipping\n",
      "[INFO]\tGene COL5A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369879 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02258 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN9P has 0 variants, skipping\n",
      "[INFO]\tGene MYO1C has 0 variants, skipping\n",
      "[INFO]\tGene H3-2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L22 has 0 variants, skipping\n",
      "[INFO]\tGene STAR has 0 variants, skipping\n",
      "[INFO]\tGene CD14 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02458 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4 has 0 variants, skipping\n",
      "[INFO]\tGene POC1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH4A has 0 variants, skipping\n",
      "[INFO]\tGene MTX3 has 0 variants, skipping\n",
      "[INFO]\tGene MAD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene IMPDH1 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP20 has 0 variants, skipping\n",
      "[INFO]\tGene HMG20A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF23 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02399 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02392 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00615 has 0 variants, skipping\n",
      "[INFO]\tGene SOX21 has 0 variants, skipping\n",
      "[INFO]\tGene MMP23B has 0 variants, skipping\n",
      "[INFO]\tGene DENND3 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928617 has 0 variants, skipping\n",
      "[INFO]\tGene CGB1 has 0 variants, skipping\n",
      "[INFO]\tGene NCDN has 0 variants, skipping\n",
      "[INFO]\tGene LINC02397 has 0 variants, skipping\n",
      "[INFO]\tGene SGPP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02412 has 0 variants, skipping\n",
      "[INFO]\tGene PREB has 0 variants, skipping\n",
      "[INFO]\tGene SOCS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A44 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928731 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369911 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25G has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1A has 0 variants, skipping\n",
      "[INFO]\tGene BEX5 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX6 has 0 variants, skipping\n",
      "[INFO]\tGene ARSL has 0 variants, skipping\n",
      "[INFO]\tGene SPRING1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS48 has 0 variants, skipping\n",
      "[INFO]\tGene CEP83-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5700 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7844 has 0 variants, skipping\n",
      "[INFO]\tGene KRT19P2 has 0 variants, skipping\n",
      "[INFO]\tGene MORN4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4G has 0 variants, skipping\n",
      "[INFO]\tGene FOXP1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB1B has 0 variants, skipping\n",
      "[INFO]\tGene C19orf44 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106A has 0 variants, skipping\n",
      "[INFO]\tGene RABEP1 has 0 variants, skipping\n",
      "[INFO]\tGene NBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1A has 0 variants, skipping\n",
      "[INFO]\tGene CT45A7 has 0 variants, skipping\n",
      "[INFO]\tGene THY1 has 0 variants, skipping\n",
      "[INFO]\tGene PTH2R has 0 variants, skipping\n",
      "[INFO]\tGene SHH has 0 variants, skipping\n",
      "[INFO]\tGene GALNS has 0 variants, skipping\n",
      "[INFO]\tGene AAR2 has 0 variants, skipping\n",
      "[INFO]\tGene SZRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3685 has 0 variants, skipping\n",
      "[INFO]\tGene COPS2 has 0 variants, skipping\n",
      "[INFO]\tGene VIPAS39 has 0 variants, skipping\n",
      "[INFO]\tGene SCAP has 0 variants, skipping\n",
      "[INFO]\tGene ADAT3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF423 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM1P5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF883 has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1C1L has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369921 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02410 has 0 variants, skipping\n",
      "[INFO]\tGene RPP30 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP57 has 0 variants, skipping\n",
      "[INFO]\tGene POLL has 0 variants, skipping\n",
      "[INFO]\tGene ADH6 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC10 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF7 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD2 has 0 variants, skipping\n",
      "[INFO]\tGene TPD52 has 0 variants, skipping\n",
      "[INFO]\tGene CTSV has 0 variants, skipping\n",
      "[INFO]\tGene PAEP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF214 has 0 variants, skipping\n",
      "[INFO]\tGene NXT1 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4495 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4303 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A7P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02453 has 0 variants, skipping\n",
      "[INFO]\tGene INSM2 has 0 variants, skipping\n",
      "[INFO]\tGene HINT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL3 has 0 variants, skipping\n",
      "[INFO]\tGene ACCS has 0 variants, skipping\n",
      "[INFO]\tGene NIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA53 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GAL1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5 has 0 variants, skipping\n",
      "[INFO]\tGene ADAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MTFMT has 0 variants, skipping\n",
      "[INFO]\tGene B9D1 has 0 variants, skipping\n",
      "[INFO]\tGene STAT6 has 0 variants, skipping\n",
      "[INFO]\tGene GPN1 has 0 variants, skipping\n",
      "[INFO]\tGene LONP2 has 0 variants, skipping\n",
      "[INFO]\tGene LRTM2 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10L has 0 variants, skipping\n",
      "[INFO]\tGene TEX33 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC1A has 0 variants, skipping\n",
      "[INFO]\tGene TBCK has 0 variants, skipping\n",
      "[INFO]\tGene CIART has 0 variants, skipping\n",
      "[INFO]\tGene PRKDC has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12B has 0 variants, skipping\n",
      "[INFO]\tGene C1orf198 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928937 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13A has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1827 has 0 variants, skipping\n",
      "[INFO]\tGene EPSTI1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA8 has 0 variants, skipping\n",
      "[INFO]\tGene C4B_2 has 0 variants, skipping\n",
      "[INFO]\tGene CYCS has 0 variants, skipping\n",
      "[INFO]\tGene PKLR has 0 variants, skipping\n",
      "[INFO]\tGene APAF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3D has 0 variants, skipping\n",
      "[INFO]\tGene TRPC4AP has 0 variants, skipping\n",
      "[INFO]\tGene CLLU1 has 0 variants, skipping\n",
      "[INFO]\tGene GRB10 has 0 variants, skipping\n",
      "[INFO]\tGene EMCN has 0 variants, skipping\n",
      "[INFO]\tGene CTAG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AQ has 0 variants, skipping\n",
      "[INFO]\tGene GJB3 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL11 has 0 variants, skipping\n",
      "[INFO]\tGene CHPF2 has 0 variants, skipping\n",
      "[INFO]\tGene TASOR has 0 variants, skipping\n",
      "[INFO]\tGene CCDC153 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB3 has 0 variants, skipping\n",
      "[INFO]\tGene SOX1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS6 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFC2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB10 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB4 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA10 has 0 variants, skipping\n",
      "[INFO]\tGene NEB has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00485 has 0 variants, skipping\n",
      "[INFO]\tGene LSMEM1 has 0 variants, skipping\n",
      "[INFO]\tGene FHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929058 has 0 variants, skipping\n",
      "[INFO]\tGene SEPHS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369945 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02401 has 0 variants, skipping\n",
      "[INFO]\tGene NPC1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRP2 has 0 variants, skipping\n",
      "[INFO]\tGene TREH has 0 variants, skipping\n",
      "[INFO]\tGene TTC41P has 0 variants, skipping\n",
      "[INFO]\tGene MIR3652 has 0 variants, skipping\n",
      "[INFO]\tGene SPAAR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF462 has 0 variants, skipping\n",
      "[INFO]\tGene NWD1 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC5 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM67 has 0 variants, skipping\n",
      "[INFO]\tGene DCK has 0 variants, skipping\n",
      "[INFO]\tGene DBH has 0 variants, skipping\n",
      "[INFO]\tGene DDX25 has 0 variants, skipping\n",
      "[INFO]\tGene LOC414300 has 0 variants, skipping\n",
      "[INFO]\tGene TOX3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF26 has 0 variants, skipping\n",
      "[INFO]\tGene REPS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE27 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC6 has 0 variants, skipping\n",
      "[INFO]\tGene KCCAT198 has 0 variants, skipping\n",
      "[INFO]\tGene CASC18 has 0 variants, skipping\n",
      "[INFO]\tGene ADAL has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287944 has 0 variants, skipping\n",
      "[INFO]\tGene THNSL2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505978 has 0 variants, skipping\n",
      "[INFO]\tGene TST has 0 variants, skipping\n",
      "[INFO]\tGene PSMD11 has 0 variants, skipping\n",
      "[INFO]\tGene WNK2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB30 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY7 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6 has 0 variants, skipping\n",
      "[INFO]\tGene FGR has 0 variants, skipping\n",
      "[INFO]\tGene FGF3 has 0 variants, skipping\n",
      "[INFO]\tGene AQR has 0 variants, skipping\n",
      "[INFO]\tGene SECISBP2L has 0 variants, skipping\n",
      "[INFO]\tGene CCDC178 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49D2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728739 has 0 variants, skipping\n",
      "[INFO]\tGene ETV7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01498 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP251 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B3 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF6 has 0 variants, skipping\n",
      "[INFO]\tGene SORBS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF256 has 0 variants, skipping\n",
      "[INFO]\tGene RCL1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST4 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR6 has 0 variants, skipping\n",
      "[INFO]\tGene USP30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY3 has 0 variants, skipping\n",
      "[INFO]\tGene ALAD has 0 variants, skipping\n",
      "[INFO]\tGene LINC01486 has 0 variants, skipping\n",
      "[INFO]\tGene BET1 has 0 variants, skipping\n",
      "[INFO]\tGene PBXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF225 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF223 has 0 variants, skipping\n",
      "[INFO]\tGene CEP68 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC14 has 0 variants, skipping\n",
      "[INFO]\tGene EVI2A has 0 variants, skipping\n",
      "[INFO]\tGene FAM222A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPST2 has 0 variants, skipping\n",
      "[INFO]\tGene DTHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DAP3 has 0 variants, skipping\n",
      "[INFO]\tGene CERT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM250 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf76 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL8 has 0 variants, skipping\n",
      "[INFO]\tGene OXLD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSACC has 0 variants, skipping\n",
      "[INFO]\tGene ADIG has 0 variants, skipping\n",
      "[INFO]\tGene ODAPH has 0 variants, skipping\n",
      "[INFO]\tGene BOLA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-1HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPS25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4474 has 0 variants, skipping\n",
      "[INFO]\tGene GLTPD2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01638 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01405 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369980 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6760 has 0 variants, skipping\n",
      "[INFO]\tGene CTTN has 0 variants, skipping\n",
      "[INFO]\tGene CRYAA2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6761 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128966 has 0 variants, skipping\n",
      "[INFO]\tGene COG5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3657 has 0 variants, skipping\n",
      "[INFO]\tGene NFXL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6861 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF174 has 0 variants, skipping\n",
      "[INFO]\tGene PTH has 0 variants, skipping\n",
      "[INFO]\tGene THSD7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-1 has 0 variants, skipping\n",
      "[INFO]\tGene DNA2 has 0 variants, skipping\n",
      "[INFO]\tGene CLTA has 0 variants, skipping\n",
      "[INFO]\tGene NABP1-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCEPR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF596 has 0 variants, skipping\n",
      "[INFO]\tGene PSPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7106 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA11 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA4 has 0 variants, skipping\n",
      "[INFO]\tGene EGR3 has 0 variants, skipping\n",
      "[INFO]\tGene SERF2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6762 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF6 has 0 variants, skipping\n",
      "[INFO]\tGene LHX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPF2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02459 has 0 variants, skipping\n",
      "[INFO]\tGene TBX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4472-2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L7 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L30 has 0 variants, skipping\n",
      "[INFO]\tGene UTP14C has 0 variants, skipping\n",
      "[INFO]\tGene TRIB2 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2A has 0 variants, skipping\n",
      "[INFO]\tGene ADAM12 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506551 has 0 variants, skipping\n",
      "[INFO]\tGene CR1 has 0 variants, skipping\n",
      "[INFO]\tGene TESC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11A has 0 variants, skipping\n",
      "[INFO]\tGene KAT5 has 0 variants, skipping\n",
      "[INFO]\tGene METTL21C has 0 variants, skipping\n",
      "[INFO]\tGene SH3KBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS17 has 0 variants, skipping\n",
      "[INFO]\tGene RNMT has 0 variants, skipping\n",
      "[INFO]\tGene MRPS16 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA13 has 0 variants, skipping\n",
      "[INFO]\tGene TPPP3 has 0 variants, skipping\n",
      "[INFO]\tGene THAP4 has 0 variants, skipping\n",
      "[INFO]\tGene HDDC2 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO2B has 0 variants, skipping\n",
      "[INFO]\tGene FCF1 has 0 variants, skipping\n",
      "[INFO]\tGene MED31 has 0 variants, skipping\n",
      "[INFO]\tGene CUTC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02423 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02439 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370024 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00934 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF90 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF772 has 0 variants, skipping\n",
      "[INFO]\tGene TENM1 has 0 variants, skipping\n",
      "[INFO]\tGene BICDL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT4 has 0 variants, skipping\n",
      "[INFO]\tGene RBM10 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF11A has 0 variants, skipping\n",
      "[INFO]\tGene CFAP221 has 0 variants, skipping\n",
      "[INFO]\tGene PUS10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4498 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4-1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB43 has 0 variants, skipping\n",
      "[INFO]\tGene DNTTIP1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAV has 0 variants, skipping\n",
      "[INFO]\tGene SCRIB has 0 variants, skipping\n",
      "[INFO]\tGene GNA13 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4700 has 0 variants, skipping\n",
      "[INFO]\tGene BUD13 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPA-DT has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_009911 has 0 variants, skipping\n",
      "[INFO]\tGene ELK3 has 0 variants, skipping\n",
      "[INFO]\tGene ELK1 has 0 variants, skipping\n",
      "[INFO]\tGene LDB1 has 0 variants, skipping\n",
      "[INFO]\tGene KIN has 0 variants, skipping\n",
      "[INFO]\tGene KIF4A has 0 variants, skipping\n",
      "[INFO]\tGene SHANK2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM2A has 0 variants, skipping\n",
      "[INFO]\tGene YJEFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF577 has 0 variants, skipping\n",
      "[INFO]\tGene FAM111A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF445 has 0 variants, skipping\n",
      "[INFO]\tGene NUP42 has 0 variants, skipping\n",
      "[INFO]\tGene ALOXE3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7107 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6F-LY6G6D has 0 variants, skipping\n",
      "[INFO]\tGene PDE4C has 0 variants, skipping\n",
      "[INFO]\tGene LOXL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP21A1P has 0 variants, skipping\n",
      "[INFO]\tGene GLRA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506691 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK2 has 0 variants, skipping\n",
      "[INFO]\tGene CORO1C has 0 variants, skipping\n",
      "[INFO]\tGene GSDMB has 0 variants, skipping\n",
      "[INFO]\tGene NAA16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101593348 has 0 variants, skipping\n",
      "[INFO]\tGene GGACT has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3B has 0 variants, skipping\n",
      "[INFO]\tGene KHNYN has 0 variants, skipping\n",
      "[INFO]\tGene CLIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF726 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH8 has 0 variants, skipping\n",
      "[INFO]\tGene GON7 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3 has 0 variants, skipping\n",
      "[INFO]\tGene ENKD1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28A has 0 variants, skipping\n",
      "[INFO]\tGene KCND3 has 0 variants, skipping\n",
      "[INFO]\tGene CDON has 0 variants, skipping\n",
      "[INFO]\tGene SF1 has 0 variants, skipping\n",
      "[INFO]\tGene PRUNE2 has 0 variants, skipping\n",
      "[INFO]\tGene PAXX has 0 variants, skipping\n",
      "[INFO]\tGene TBX15 has 0 variants, skipping\n",
      "[INFO]\tGene LAX1 has 0 variants, skipping\n",
      "[INFO]\tGene DDR2 has 0 variants, skipping\n",
      "[INFO]\tGene STK16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4304 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8072 has 0 variants, skipping\n",
      "[INFO]\tGene CWF19L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3908 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G7 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP40 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA9B has 0 variants, skipping\n",
      "[INFO]\tGene GAGE10 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE13 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2B has 0 variants, skipping\n",
      "[INFO]\tGene PAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCR7 has 0 variants, skipping\n",
      "[INFO]\tGene ESR2 has 0 variants, skipping\n",
      "[INFO]\tGene STAMBP has 0 variants, skipping\n",
      "[INFO]\tGene ING4 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB43 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA8 has 0 variants, skipping\n",
      "[INFO]\tGene DHX38 has 0 variants, skipping\n",
      "[INFO]\tGene IKBKE has 0 variants, skipping\n",
      "[INFO]\tGene MIR6880 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5188 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00939 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02359 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02350 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02347 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996671 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00944 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00943 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02372 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02405 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02376 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02411 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02393 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00507 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02441 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02369 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02368 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3612 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128276 has 0 variants, skipping\n",
      "[INFO]\tGene NBEAL1 has 0 variants, skipping\n",
      "[INFO]\tGene AMDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MPIG6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02419 has 0 variants, skipping\n",
      "[INFO]\tGene FZD10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A10 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R21 has 0 variants, skipping\n",
      "[INFO]\tGene POMZP3 has 0 variants, skipping\n",
      "[INFO]\tGene PXT1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR3 has 0 variants, skipping\n",
      "[INFO]\tGene INTS3 has 0 variants, skipping\n",
      "[INFO]\tGene FILIP1L has 0 variants, skipping\n",
      "[INFO]\tGene FAM3C has 0 variants, skipping\n",
      "[INFO]\tGene NSA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF557 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A3 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01257 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02415 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02370 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02414 has 0 variants, skipping\n",
      "[INFO]\tGene METTL5 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF26 has 0 variants, skipping\n",
      "[INFO]\tGene GABRD has 0 variants, skipping\n",
      "[INFO]\tGene SNORA49 has 0 variants, skipping\n",
      "[INFO]\tGene EP400P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02361 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928416 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6763 has 0 variants, skipping\n",
      "[INFO]\tGene SP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN22 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928530 has 0 variants, skipping\n",
      "[INFO]\tGene HMBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF76 has 0 variants, skipping\n",
      "[INFO]\tGene COPS3 has 0 variants, skipping\n",
      "[INFO]\tGene PI16 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3F has 0 variants, skipping\n",
      "[INFO]\tGene CENPL has 0 variants, skipping\n",
      "[INFO]\tGene ZNF135 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC3 has 0 variants, skipping\n",
      "[INFO]\tGene ATF5 has 0 variants, skipping\n",
      "[INFO]\tGene CFD has 0 variants, skipping\n",
      "[INFO]\tGene ERP27 has 0 variants, skipping\n",
      "[INFO]\tGene MPL has 0 variants, skipping\n",
      "[INFO]\tGene MOS has 0 variants, skipping\n",
      "[INFO]\tGene ANXA6 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA5 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA4 has 0 variants, skipping\n",
      "[INFO]\tGene IATPR has 0 variants, skipping\n",
      "[INFO]\tGene PRRX1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF34 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM80 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT6 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B4 has 0 variants, skipping\n",
      "[INFO]\tGene NSMAF has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L4 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf67 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC4L has 0 variants, skipping\n",
      "[INFO]\tGene NDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AWAT1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A6 has 0 variants, skipping\n",
      "[INFO]\tGene PRR3 has 0 variants, skipping\n",
      "[INFO]\tGene AIFM1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYZ has 0 variants, skipping\n",
      "[INFO]\tGene MYO15B has 0 variants, skipping\n",
      "[INFO]\tGene GSG1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM230C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00417 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02481 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR375 has 0 variants, skipping\n",
      "[INFO]\tGene MIR373 has 0 variants, skipping\n",
      "[INFO]\tGene MIR372 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00408 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984132 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928697 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00421 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3D21 has 0 variants, skipping\n",
      "[INFO]\tGene PALB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00350 has 0 variants, skipping\n",
      "[INFO]\tGene ORAI2 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1A has 0 variants, skipping\n",
      "[INFO]\tGene NSUN6 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4C has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3A has 0 variants, skipping\n",
      "[INFO]\tGene SDK1 has 0 variants, skipping\n",
      "[INFO]\tGene OASL has 0 variants, skipping\n",
      "[INFO]\tGene GSPT1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM4 has 0 variants, skipping\n",
      "[INFO]\tGene CCT2 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN5A has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE3 has 0 variants, skipping\n",
      "[INFO]\tGene HARS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR124-2HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC01072 has 0 variants, skipping\n",
      "[INFO]\tGene DNPEP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4499 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6F has 0 variants, skipping\n",
      "[INFO]\tGene OR10V1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AK2 has 0 variants, skipping\n",
      "[INFO]\tGene SREBF1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM76A has 0 variants, skipping\n",
      "[INFO]\tGene OXSM has 0 variants, skipping\n",
      "[INFO]\tGene FGFR1OP2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCR has 0 variants, skipping\n",
      "[INFO]\tGene ACOT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC22 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB16 has 0 variants, skipping\n",
      "[INFO]\tGene ASPA has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-3 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE4 has 0 variants, skipping\n",
      "[INFO]\tGene MELK has 0 variants, skipping\n",
      "[INFO]\tGene STPG3 has 0 variants, skipping\n",
      "[INFO]\tGene BANF2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM38B has 0 variants, skipping\n",
      "[INFO]\tGene C19orf73 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNG has 0 variants, skipping\n",
      "[INFO]\tGene C22orf15 has 0 variants, skipping\n",
      "[INFO]\tGene SFTA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR656 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00367 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01046 has 0 variants, skipping\n",
      "[INFO]\tGene MIPEPP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00539 has 0 variants, skipping\n",
      "[INFO]\tGene GRK6P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00424 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00621 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00327 has 0 variants, skipping\n",
      "[INFO]\tGene UGCG has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1A1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM223 has 0 variants, skipping\n",
      "[INFO]\tGene MND1 has 0 variants, skipping\n",
      "[INFO]\tGene SRP68 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A19P has 0 variants, skipping\n",
      "[INFO]\tGene THTPA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF576 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL15 has 0 variants, skipping\n",
      "[INFO]\tGene CRELD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2276 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00566 has 0 variants, skipping\n",
      "[INFO]\tGene MCMDC2 has 0 variants, skipping\n",
      "[INFO]\tGene NAPRT has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC6 has 0 variants, skipping\n",
      "[INFO]\tGene MMD2 has 0 variants, skipping\n",
      "[INFO]\tGene NDRG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01088 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00463 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01053 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF8 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN3 has 0 variants, skipping\n",
      "[INFO]\tGene STK32C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00415 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY2 has 0 variants, skipping\n",
      "[INFO]\tGene RXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene USP12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP12-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00412 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD102 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA27 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZWINT has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT2 has 0 variants, skipping\n",
      "[INFO]\tGene PLUT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00543 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8 has 0 variants, skipping\n",
      "[INFO]\tGene TBX20 has 0 variants, skipping\n",
      "[INFO]\tGene OLR1 has 0 variants, skipping\n",
      "[INFO]\tGene AMN has 0 variants, skipping\n",
      "[INFO]\tGene PAN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGR has 0 variants, skipping\n",
      "[INFO]\tGene RPS24 has 0 variants, skipping\n",
      "[INFO]\tGene RPS23 has 0 variants, skipping\n",
      "[INFO]\tGene PDE1B has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D2 has 0 variants, skipping\n",
      "[INFO]\tGene URB2 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00297 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00572 has 0 variants, skipping\n",
      "[INFO]\tGene MUCL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00365 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00384 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00426 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01058 has 0 variants, skipping\n",
      "[INFO]\tGene USB1 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP300 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRU has 0 variants, skipping\n",
      "[INFO]\tGene LINC00398 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00545 has 0 variants, skipping\n",
      "[INFO]\tGene TEX26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3GAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5A has 0 variants, skipping\n",
      "[INFO]\tGene EEF1DP3 has 0 variants, skipping\n",
      "[INFO]\tGene FRY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP2 has 0 variants, skipping\n",
      "[INFO]\tGene NKG7 has 0 variants, skipping\n",
      "[INFO]\tGene KRT33A has 0 variants, skipping\n",
      "[INFO]\tGene TCEANC has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG3 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS10P1 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP2L2-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXM1 has 0 variants, skipping\n",
      "[INFO]\tGene STARD13-AS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02344 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02343 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00457 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF780A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00445 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A8 has 0 variants, skipping\n",
      "[INFO]\tGene CHRND has 0 variants, skipping\n",
      "[INFO]\tGene FAM166C has 0 variants, skipping\n",
      "[INFO]\tGene C17orf100 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPB has 0 variants, skipping\n",
      "[INFO]\tGene SPART-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FHL3 has 0 variants, skipping\n",
      "[INFO]\tGene FGL1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM22 has 0 variants, skipping\n",
      "[INFO]\tGene AGGF1 has 0 variants, skipping\n",
      "[INFO]\tGene BSDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5 has 0 variants, skipping\n",
      "[INFO]\tGene USP47 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO34 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN12 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH2 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00547 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPPIN-WFDC6 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC5 has 0 variants, skipping\n",
      "[INFO]\tGene DCN has 0 variants, skipping\n",
      "[INFO]\tGene OS9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00571 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF18 has 0 variants, skipping\n",
      "[INFO]\tGene PERP has 0 variants, skipping\n",
      "[INFO]\tGene PRSS22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00437 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00366 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF15 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP1 has 0 variants, skipping\n",
      "[INFO]\tGene FREM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RUBCN has 0 variants, skipping\n",
      "[INFO]\tGene SBNO2 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene KNTC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4305 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00332 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00548 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00598 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR621 has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3168 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929140 has 0 variants, skipping\n",
      "[INFO]\tGene YME1L1 has 0 variants, skipping\n",
      "[INFO]\tGene P3H3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E37P has 0 variants, skipping\n",
      "[INFO]\tGene HEATR4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5006 has 0 variants, skipping\n",
      "[INFO]\tGene VWA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBASH3B has 0 variants, skipping\n",
      "[INFO]\tGene PF4 has 0 variants, skipping\n",
      "[INFO]\tGene CTTNBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC3 has 0 variants, skipping\n",
      "[INFO]\tGene USH1G has 0 variants, skipping\n",
      "[INFO]\tGene LINC02341 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01050 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00428 has 0 variants, skipping\n",
      "[INFO]\tGene DGKA has 0 variants, skipping\n",
      "[INFO]\tGene PSMC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00400 has 0 variants, skipping\n",
      "[INFO]\tGene NUP58 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF8 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K17 has 0 variants, skipping\n",
      "[INFO]\tGene POTEF has 0 variants, skipping\n",
      "[INFO]\tGene GGT5 has 0 variants, skipping\n",
      "[INFO]\tGene HRCT1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB28 has 0 variants, skipping\n",
      "[INFO]\tGene ENOX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene URB1 has 0 variants, skipping\n",
      "[INFO]\tGene FCHSD2 has 0 variants, skipping\n",
      "[INFO]\tGene WNK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00390 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8079 has 0 variants, skipping\n",
      "[INFO]\tGene SERP2 has 0 variants, skipping\n",
      "[INFO]\tGene PTCRA has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00330 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929259 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD9 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCTA has 0 variants, skipping\n",
      "[INFO]\tGene ABCG4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA31B has 0 variants, skipping\n",
      "[INFO]\tGene CRB3 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN1 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3G has 0 variants, skipping\n",
      "[INFO]\tGene AK5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01055 has 0 variants, skipping\n",
      "[INFO]\tGene ERVMER34-1 has 0 variants, skipping\n",
      "[INFO]\tGene CHST5 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG16 has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS19 has 0 variants, skipping\n",
      "[INFO]\tGene CPB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCEH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00563 has 0 variants, skipping\n",
      "[INFO]\tGene GPR149 has 0 variants, skipping\n",
      "[INFO]\tGene OLFM1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01198 has 0 variants, skipping\n",
      "[INFO]\tGene HTR2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00562 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED3 has 0 variants, skipping\n",
      "[INFO]\tGene ASB17 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A9 has 0 variants, skipping\n",
      "[INFO]\tGene MED4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP10 has 0 variants, skipping\n",
      "[INFO]\tGene BCL9L has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5E has 0 variants, skipping\n",
      "[INFO]\tGene CDH18 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00462 has 0 variants, skipping\n",
      "[INFO]\tGene SPO11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370203 has 0 variants, skipping\n",
      "[INFO]\tGene AMY1B has 0 variants, skipping\n",
      "[INFO]\tGene USP17L3 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF7 has 0 variants, skipping\n",
      "[INFO]\tGene EBPL has 0 variants, skipping\n",
      "[INFO]\tGene RGS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE10P has 0 variants, skipping\n",
      "[INFO]\tGene GAGE7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507412 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00641 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723729 has 0 variants, skipping\n",
      "[INFO]\tGene BBOX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ST13P4 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD7 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY1B2 has 0 variants, skipping\n",
      "[INFO]\tGene C13orf42 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645177 has 0 variants, skipping\n",
      "[INFO]\tGene DAP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5693 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC120 has 0 variants, skipping\n",
      "[INFO]\tGene GPANK1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4703 has 0 variants, skipping\n",
      "[INFO]\tGene ECT2L has 0 variants, skipping\n",
      "[INFO]\tGene PROSER1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL4 has 0 variants, skipping\n",
      "[INFO]\tGene AQP11 has 0 variants, skipping\n",
      "[INFO]\tGene MYPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929657 has 0 variants, skipping\n",
      "[INFO]\tGene PLAA has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A42 has 0 variants, skipping\n",
      "[INFO]\tGene RNPS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS31P5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02333 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND10 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE2P3 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR759 has 0 variants, skipping\n",
      "[INFO]\tGene NAAA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00558 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5007 has 0 variants, skipping\n",
      "[INFO]\tGene AK9 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A6A has 0 variants, skipping\n",
      "[INFO]\tGene NCAPG has 0 variants, skipping\n",
      "[INFO]\tGene KIF13B has 0 variants, skipping\n",
      "[INFO]\tGene EDDM3B has 0 variants, skipping\n",
      "[INFO]\tGene KCNK15 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02338 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00374 has 0 variants, skipping\n",
      "[INFO]\tGene MATN2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR27 has 0 variants, skipping\n",
      "[INFO]\tGene COL8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD2 has 0 variants, skipping\n",
      "[INFO]\tGene AGTRAP has 0 variants, skipping\n",
      "[INFO]\tGene LZTFL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF543 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf66 has 0 variants, skipping\n",
      "[INFO]\tGene KRBA2 has 0 variants, skipping\n",
      "[INFO]\tGene MTNR1B has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00434 has 0 variants, skipping\n",
      "[INFO]\tGene RGL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00378 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02339 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00358 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01075 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00459 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00448 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00376 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00395 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E156P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00355 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01052 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548X2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4704 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH9-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00364 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00550 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00383 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00348 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN17 has 0 variants, skipping\n",
      "[INFO]\tGene MON1B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0754 has 0 variants, skipping\n",
      "[INFO]\tGene MLXIP has 0 variants, skipping\n",
      "[INFO]\tGene ENPP4 has 0 variants, skipping\n",
      "[INFO]\tGene ENDOD1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA6 has 0 variants, skipping\n",
      "[INFO]\tGene FNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDS5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00392 has 0 variants, skipping\n",
      "[INFO]\tGene SRPRB has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00381 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00347 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE11P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01078 has 0 variants, skipping\n",
      "[INFO]\tGene ERO1B has 0 variants, skipping\n",
      "[INFO]\tGene CIP2A has 0 variants, skipping\n",
      "[INFO]\tGene LMO7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF554 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF300 has 0 variants, skipping\n",
      "[INFO]\tGene CEP43 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2H has 0 variants, skipping\n",
      "[INFO]\tGene ANTXRL has 0 variants, skipping\n",
      "[INFO]\tGene PLCH2 has 0 variants, skipping\n",
      "[INFO]\tGene TENT4B has 0 variants, skipping\n",
      "[INFO]\tGene NQO1 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7DN-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00561 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3P11 has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTR has 0 variants, skipping\n",
      "[INFO]\tGene KNCN has 0 variants, skipping\n",
      "[INFO]\tGene SCEL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD4L has 0 variants, skipping\n",
      "[INFO]\tGene MIR3665 has 0 variants, skipping\n",
      "[INFO]\tGene ZIK1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01069 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC780529 has 0 variants, skipping\n",
      "[INFO]\tGene RNF20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00331 has 0 variants, skipping\n",
      "[INFO]\tGene KLK11 has 0 variants, skipping\n",
      "[INFO]\tGene NPM3 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA2B has 0 variants, skipping\n",
      "[INFO]\tGene RBM26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01068 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01038 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00382 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00564 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00333 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00375 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00430 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4500HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC00397 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00373 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00433 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00560 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01047 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00440 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01040 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00353 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00559 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01049 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00410 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00379 has 0 variants, skipping\n",
      "[INFO]\tGene RNF150 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC5-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AS has 0 variants, skipping\n",
      "[INFO]\tGene GPC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00363 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB10 has 0 variants, skipping\n",
      "[INFO]\tGene ECE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927248 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927284 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00557 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A8 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC14 has 0 variants, skipping\n",
      "[INFO]\tGene RPL31 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM6 has 0 variants, skipping\n",
      "[INFO]\tGene HPSE has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13G has 0 variants, skipping\n",
      "[INFO]\tGene CLDN10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC3-DT has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD3 has 0 variants, skipping\n",
      "[INFO]\tGene SPARC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00359 has 0 variants, skipping\n",
      "[INFO]\tGene CRMP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00456 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY3 has 0 variants, skipping\n",
      "[INFO]\tGene STARD9 has 0 variants, skipping\n",
      "[INFO]\tGene SERINC1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP31 has 0 variants, skipping\n",
      "[INFO]\tGene CASKIN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3170 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf201 has 0 variants, skipping\n",
      "[INFO]\tGene NUBPL has 0 variants, skipping\n",
      "[INFO]\tGene STK24-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF6 has 0 variants, skipping\n",
      "[INFO]\tGene GDF7 has 0 variants, skipping\n",
      "[INFO]\tGene ATF4 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB4 has 0 variants, skipping\n",
      "[INFO]\tGene MAT2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF787 has 0 variants, skipping\n",
      "[INFO]\tGene HRH2 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC9 has 0 variants, skipping\n",
      "[INFO]\tGene SC5D has 0 variants, skipping\n",
      "[INFO]\tGene ZNF671 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GLB2 has 0 variants, skipping\n",
      "[INFO]\tGene CEP70 has 0 variants, skipping\n",
      "[INFO]\tGene TEX29 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5G has 0 variants, skipping\n",
      "[INFO]\tGene RHEBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-6 has 0 variants, skipping\n",
      "[INFO]\tGene UCMA has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1C has 0 variants, skipping\n",
      "[INFO]\tGene DARS1 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49C has 0 variants, skipping\n",
      "[INFO]\tGene C4B has 0 variants, skipping\n",
      "[INFO]\tGene PDCD4 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK9-DT has 0 variants, skipping\n",
      "[INFO]\tGene UBAC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FKSG29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01232 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00449 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01039 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927437 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00554 has 0 variants, skipping\n",
      "[INFO]\tGene ATG12 has 0 variants, skipping\n",
      "[INFO]\tGene CACNB1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR150 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13D has 0 variants, skipping\n",
      "[INFO]\tGene ANO6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF780B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF835 has 0 variants, skipping\n",
      "[INFO]\tGene STUM has 0 variants, skipping\n",
      "[INFO]\tGene SMAD2 has 0 variants, skipping\n",
      "[INFO]\tGene RAE1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC25 has 0 variants, skipping\n",
      "[INFO]\tGene AURKC has 0 variants, skipping\n",
      "[INFO]\tGene PAPSS2 has 0 variants, skipping\n",
      "[INFO]\tGene PURG has 0 variants, skipping\n",
      "[INFO]\tGene PCCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF629 has 0 variants, skipping\n",
      "[INFO]\tGene NVL has 0 variants, skipping\n",
      "[INFO]\tGene LOC642414 has 0 variants, skipping\n",
      "[INFO]\tGene XKR7 has 0 variants, skipping\n",
      "[INFO]\tGene NMS has 0 variants, skipping\n",
      "[INFO]\tGene IQCE has 0 variants, skipping\n",
      "[INFO]\tGene PAX4 has 0 variants, skipping\n",
      "[INFO]\tGene NALCN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00411 has 0 variants, skipping\n",
      "[INFO]\tGene CHURC1-FNTB has 0 variants, skipping\n",
      "[INFO]\tGene BUD23 has 0 variants, skipping\n",
      "[INFO]\tGene PIGV has 0 variants, skipping\n",
      "[INFO]\tGene GRXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM3 has 0 variants, skipping\n",
      "[INFO]\tGene MMP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K11 has 0 variants, skipping\n",
      "[INFO]\tGene CTR9 has 0 variants, skipping\n",
      "[INFO]\tGene RNF182 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6M1 has 0 variants, skipping\n",
      "[INFO]\tGene LMTK3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2681 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4705 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF14-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00958 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01116 has 0 variants, skipping\n",
      "[INFO]\tGene MROCKI has 0 variants, skipping\n",
      "[INFO]\tGene LINC01133 has 0 variants, skipping\n",
      "[INFO]\tGene MNX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00974 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1G-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00637 has 0 variants, skipping\n",
      "[INFO]\tGene DNM3OS has 0 variants, skipping\n",
      "[INFO]\tGene WTAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene HOTAIRM1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM147 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM20 has 0 variants, skipping\n",
      "[INFO]\tGene ZAR1L has 0 variants, skipping\n",
      "[INFO]\tGene METTL21EP has 0 variants, skipping\n",
      "[INFO]\tGene DAOA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00343 has 0 variants, skipping\n",
      "[INFO]\tGene SRGN has 0 variants, skipping\n",
      "[INFO]\tGene PREP has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1549L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00551 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00443 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD31B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1267 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene ETV4 has 0 variants, skipping\n",
      "[INFO]\tGene MYO16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00399 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00676 has 0 variants, skipping\n",
      "[INFO]\tGene TAF6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00396 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370361 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF549 has 0 variants, skipping\n",
      "[INFO]\tGene CNEP1R1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC57 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN2 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL6 has 0 variants, skipping\n",
      "[INFO]\tGene PGPEP1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR8073 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS15 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L3 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf71 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF316 has 0 variants, skipping\n",
      "[INFO]\tGene WHRN has 0 variants, skipping\n",
      "[INFO]\tGene LINC00567 has 0 variants, skipping\n",
      "[INFO]\tGene KIF15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00431 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00368 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP7 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF10 has 0 variants, skipping\n",
      "[INFO]\tGene SAXO2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC4 has 0 variants, skipping\n",
      "[INFO]\tGene F8 has 0 variants, skipping\n",
      "[INFO]\tGene F7 has 0 variants, skipping\n",
      "[INFO]\tGene F13A1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101060553 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11A has 0 variants, skipping\n",
      "[INFO]\tGene UPK3BL1 has 0 variants, skipping\n",
      "[INFO]\tGene KREMEN2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX52 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC29 has 0 variants, skipping\n",
      "[INFO]\tGene RFLNA has 0 variants, skipping\n",
      "[INFO]\tGene SMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A2 has 0 variants, skipping\n",
      "[INFO]\tGene GRAMD1B has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP58 has 0 variants, skipping\n",
      "[INFO]\tGene NANOGP8 has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL1 has 0 variants, skipping\n",
      "[INFO]\tGene FBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene H3Y1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD3OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02337 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00354 has 0 variants, skipping\n",
      "[INFO]\tGene SOX1-OT has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506016 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01070 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928730 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01043 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01044 has 0 variants, skipping\n",
      "[INFO]\tGene GLMN has 0 variants, skipping\n",
      "[INFO]\tGene BTD has 0 variants, skipping\n",
      "[INFO]\tGene KIF5C has 0 variants, skipping\n",
      "[INFO]\tGene KIF5B has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ3 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL23 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53BP2 has 0 variants, skipping\n",
      "[INFO]\tGene TNP2 has 0 variants, skipping\n",
      "[INFO]\tGene TIE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFF2 has 0 variants, skipping\n",
      "[INFO]\tGene TECTA has 0 variants, skipping\n",
      "[INFO]\tGene TAL2 has 0 variants, skipping\n",
      "[INFO]\tGene STAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1E1 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB3 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT2A has 0 variants, skipping\n",
      "[INFO]\tGene GPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELF2 has 0 variants, skipping\n",
      "[INFO]\tGene CFI has 0 variants, skipping\n",
      "[INFO]\tGene TMCC2 has 0 variants, skipping\n",
      "[INFO]\tGene F10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD17 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R16B has 0 variants, skipping\n",
      "[INFO]\tGene ACVR1C has 0 variants, skipping\n",
      "[INFO]\tGene FAM120A has 0 variants, skipping\n",
      "[INFO]\tGene RMDN1 has 0 variants, skipping\n",
      "[INFO]\tGene AGTPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA1 has 0 variants, skipping\n",
      "[INFO]\tGene SHF has 0 variants, skipping\n",
      "[INFO]\tGene EME2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288966 has 0 variants, skipping\n",
      "[INFO]\tGene TPO has 0 variants, skipping\n",
      "[INFO]\tGene AMZ2 has 0 variants, skipping\n",
      "[INFO]\tGene GRTP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT5 has 0 variants, skipping\n",
      "[INFO]\tGene KRT2 has 0 variants, skipping\n",
      "[INFO]\tGene TPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC4 has 0 variants, skipping\n",
      "[INFO]\tGene THAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF8 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00552 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00454 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00565 has 0 variants, skipping\n",
      "[INFO]\tGene OSR2 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA2 has 0 variants, skipping\n",
      "[INFO]\tGene NPFF has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L1 has 0 variants, skipping\n",
      "[INFO]\tGene SALL1 has 0 variants, skipping\n",
      "[INFO]\tGene TFIP11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AR has 0 variants, skipping\n",
      "[INFO]\tGene MIR4502 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS3 has 0 variants, skipping\n",
      "[INFO]\tGene GCC1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN1 has 0 variants, skipping\n",
      "[INFO]\tGene EDNRA has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P11 has 0 variants, skipping\n",
      "[INFO]\tGene POT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG1 has 0 variants, skipping\n",
      "[INFO]\tGene NBR2 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG9 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01054 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A9 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC3 has 0 variants, skipping\n",
      "[INFO]\tGene FADD has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100508046 has 0 variants, skipping\n",
      "[INFO]\tGene ACIN1 has 0 variants, skipping\n",
      "[INFO]\tGene XYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P18 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P17 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P22 has 0 variants, skipping\n",
      "[INFO]\tGene POTEG has 0 variants, skipping\n",
      "[INFO]\tGene POTEH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929572 has 0 variants, skipping\n",
      "[INFO]\tGene DCLRE1B has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA79B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD126 has 0 variants, skipping\n",
      "[INFO]\tGene RPPH1 has 0 variants, skipping\n",
      "[INFO]\tGene PGC has 0 variants, skipping\n",
      "[INFO]\tGene LOC254028 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3B has 0 variants, skipping\n",
      "[INFO]\tGene ECRP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929718 has 0 variants, skipping\n",
      "[INFO]\tGene FAIM2 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A5 has 0 variants, skipping\n",
      "[INFO]\tGene TKTL1 has 0 variants, skipping\n",
      "[INFO]\tGene TFEC has 0 variants, skipping\n",
      "[INFO]\tGene RHOQ has 0 variants, skipping\n",
      "[INFO]\tGene SEPHS2 has 0 variants, skipping\n",
      "[INFO]\tGene SNW1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A3 has 0 variants, skipping\n",
      "[INFO]\tGene DKK1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT3 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT2 has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6717 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD9 has 0 variants, skipping\n",
      "[INFO]\tGene YEATS4 has 0 variants, skipping\n",
      "[INFO]\tGene PEX6 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA4 has 0 variants, skipping\n",
      "[INFO]\tGene RPL35A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD9 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAB has 0 variants, skipping\n",
      "[INFO]\tGene ATRN has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B2 has 0 variants, skipping\n",
      "[INFO]\tGene INS has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370401 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02332 has 0 variants, skipping\n",
      "[INFO]\tGene GPR161 has 0 variants, skipping\n",
      "[INFO]\tGene FAM114A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR107 has 0 variants, skipping\n",
      "[INFO]\tGene ORC2 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf39 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G7 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H5 has 0 variants, skipping\n",
      "[INFO]\tGene TERT has 0 variants, skipping\n",
      "[INFO]\tGene DBF4 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RE has 0 variants, skipping\n",
      "[INFO]\tGene PRMT5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL38 has 0 variants, skipping\n",
      "[INFO]\tGene TSPOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene ACACB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926933 has 0 variants, skipping\n",
      "[INFO]\tGene THADA has 0 variants, skipping\n",
      "[INFO]\tGene MIR4707 has 0 variants, skipping\n",
      "[INFO]\tGene LRRN1 has 0 variants, skipping\n",
      "[INFO]\tGene SYTL1 has 0 variants, skipping\n",
      "[INFO]\tGene RPA1 has 0 variants, skipping\n",
      "[INFO]\tGene IL13 has 0 variants, skipping\n",
      "[INFO]\tGene PLCH1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A17 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2B has 0 variants, skipping\n",
      "[INFO]\tGene RTBDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF610 has 0 variants, skipping\n",
      "[INFO]\tGene SPP1 has 0 variants, skipping\n",
      "[INFO]\tGene IAH1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF19B has 0 variants, skipping\n",
      "[INFO]\tGene GAR1 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1B has 0 variants, skipping\n",
      "[INFO]\tGene C8orf37 has 0 variants, skipping\n",
      "[INFO]\tGene SYP has 0 variants, skipping\n",
      "[INFO]\tGene PEF1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf35 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724814 has 0 variants, skipping\n",
      "[INFO]\tGene CST9 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FETUB has 0 variants, skipping\n",
      "[INFO]\tGene GPR160 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3G has 0 variants, skipping\n",
      "[INFO]\tGene ACBD4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC3A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf53 has 0 variants, skipping\n",
      "[INFO]\tGene CENPA has 0 variants, skipping\n",
      "[INFO]\tGene ITGB2 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP10 has 0 variants, skipping\n",
      "[INFO]\tGene CASP9 has 0 variants, skipping\n",
      "[INFO]\tGene GALP has 0 variants, skipping\n",
      "[INFO]\tGene MIR7703 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A4 has 0 variants, skipping\n",
      "[INFO]\tGene POR has 0 variants, skipping\n",
      "[INFO]\tGene IPO4 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf89 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD8-MDP1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F2 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK3 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D5 has 0 variants, skipping\n",
      "[INFO]\tGene RPA3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A1 has 0 variants, skipping\n",
      "[INFO]\tGene MED30 has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2B has 0 variants, skipping\n",
      "[INFO]\tGene INTS6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927045 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF408 has 0 variants, skipping\n",
      "[INFO]\tGene KAZN has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724219 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN3 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02588 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02294 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02293 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4307HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4307 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728755 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00645 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3171 has 0 variants, skipping\n",
      "[INFO]\tGene FOXG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INHBE has 0 variants, skipping\n",
      "[INFO]\tGene LINC02282 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN1 has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3B has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3C has 0 variants, skipping\n",
      "[INFO]\tGene BUB1B-PAK6 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4O has 0 variants, skipping\n",
      "[INFO]\tGene G2E3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NTSR1 has 0 variants, skipping\n",
      "[INFO]\tGene NTRK1 has 0 variants, skipping\n",
      "[INFO]\tGene NTF3 has 0 variants, skipping\n",
      "[INFO]\tGene NT5E has 0 variants, skipping\n",
      "[INFO]\tGene IFT140 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM4A has 0 variants, skipping\n",
      "[INFO]\tGene SETD1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506071 has 0 variants, skipping\n",
      "[INFO]\tGene PDE3B has 0 variants, skipping\n",
      "[INFO]\tGene NCOA5 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND15 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPG has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927124 has 0 variants, skipping\n",
      "[INFO]\tGene CNPY4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02313 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GSDMD has 0 variants, skipping\n",
      "[INFO]\tGene SNORA89 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP5 has 0 variants, skipping\n",
      "[INFO]\tGene MIGA1 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF283 has 0 variants, skipping\n",
      "[INFO]\tGene BCL6B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB12 has 0 variants, skipping\n",
      "[INFO]\tGene SRP54-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGBP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927178 has 0 variants, skipping\n",
      "[INFO]\tGene RGP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP4 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMP1 has 0 variants, skipping\n",
      "[INFO]\tGene TIAL1 has 0 variants, skipping\n",
      "[INFO]\tGene THRSP has 0 variants, skipping\n",
      "[INFO]\tGene THOP1 has 0 variants, skipping\n",
      "[INFO]\tGene THRA has 0 variants, skipping\n",
      "[INFO]\tGene THBS4 has 0 variants, skipping\n",
      "[INFO]\tGene THBS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D2 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB5 has 0 variants, skipping\n",
      "[INFO]\tGene SDK2 has 0 variants, skipping\n",
      "[INFO]\tGene THAP3 has 0 variants, skipping\n",
      "[INFO]\tGene FAH has 0 variants, skipping\n",
      "[INFO]\tGene FANCA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA101B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00609 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1287 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL25 has 0 variants, skipping\n",
      "[INFO]\tGene BEX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF426 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4503 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AG1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00517 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF597 has 0 variants, skipping\n",
      "[INFO]\tGene BEST3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370457 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4G has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF18 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN21 has 0 variants, skipping\n",
      "[INFO]\tGene CDC45 has 0 variants, skipping\n",
      "[INFO]\tGene PLAU has 0 variants, skipping\n",
      "[INFO]\tGene METTL18 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC6 has 0 variants, skipping\n",
      "[INFO]\tGene PLAT has 0 variants, skipping\n",
      "[INFO]\tGene SCIMP has 0 variants, skipping\n",
      "[INFO]\tGene HDGF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF326 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288846 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM24 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1200 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1238 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1236 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1178 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2113 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf40 has 0 variants, skipping\n",
      "[INFO]\tGene NOS3 has 0 variants, skipping\n",
      "[INFO]\tGene SGO2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02315 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1F has 0 variants, skipping\n",
      "[INFO]\tGene CENPW has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02302 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927418 has 0 variants, skipping\n",
      "[INFO]\tGene IL34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD127 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2A has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE8 has 0 variants, skipping\n",
      "[INFO]\tGene COL21A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00871 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548Y has 0 variants, skipping\n",
      "[INFO]\tGene LINC00648 has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL2 has 0 variants, skipping\n",
      "[INFO]\tGene LRR1 has 0 variants, skipping\n",
      "[INFO]\tGene CABLES1 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT5B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB32 has 0 variants, skipping\n",
      "[INFO]\tGene RBAK has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6076 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01588 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01599 has 0 variants, skipping\n",
      "[INFO]\tGene RBM24 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4504 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD1 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1L has 0 variants, skipping\n",
      "[INFO]\tGene TOMM40L has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A1 has 0 variants, skipping\n",
      "[INFO]\tGene ALX3 has 0 variants, skipping\n",
      "[INFO]\tGene NOVA1 has 0 variants, skipping\n",
      "[INFO]\tGene MCAM has 0 variants, skipping\n",
      "[INFO]\tGene KHK has 0 variants, skipping\n",
      "[INFO]\tGene HTR6 has 0 variants, skipping\n",
      "[INFO]\tGene BRD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370489 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6P1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS53 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00519 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00640 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02310 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD6 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ODF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723604 has 0 variants, skipping\n",
      "[INFO]\tGene CD2 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927620 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5580 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D4 has 0 variants, skipping\n",
      "[INFO]\tGene PPARGC1A has 0 variants, skipping\n",
      "[INFO]\tGene ELP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4308 has 0 variants, skipping\n",
      "[INFO]\tGene CHTF8 has 0 variants, skipping\n",
      "[INFO]\tGene TOX2 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN5 has 0 variants, skipping\n",
      "[INFO]\tGene GPS2 has 0 variants, skipping\n",
      "[INFO]\tGene GCSAML has 0 variants, skipping\n",
      "[INFO]\tGene DSG3 has 0 variants, skipping\n",
      "[INFO]\tGene TRRAP has 0 variants, skipping\n",
      "[INFO]\tGene CLEC12A has 0 variants, skipping\n",
      "[INFO]\tGene FAS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ID1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP3 has 0 variants, skipping\n",
      "[INFO]\tGene LIMA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927690 has 0 variants, skipping\n",
      "[INFO]\tGene SIX3 has 0 variants, skipping\n",
      "[INFO]\tGene OTX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCP1B has 0 variants, skipping\n",
      "[INFO]\tGene ASTE1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8E has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H18 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN4 has 0 variants, skipping\n",
      "[INFO]\tGene NOL4L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00216 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4 has 0 variants, skipping\n",
      "[INFO]\tGene USP14 has 0 variants, skipping\n",
      "[INFO]\tGene PRR11 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106A has 0 variants, skipping\n",
      "[INFO]\tGene SIKE1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF703 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP3 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6D has 0 variants, skipping\n",
      "[INFO]\tGene ATRX has 0 variants, skipping\n",
      "[INFO]\tGene PCSK4 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK2 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene CPED1 has 0 variants, skipping\n",
      "[INFO]\tGene NREP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF304 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF571 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01500 has 0 variants, skipping\n",
      "[INFO]\tGene SS18 has 0 variants, skipping\n",
      "[INFO]\tGene NRXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF649 has 0 variants, skipping\n",
      "[INFO]\tGene CPLANE1 has 0 variants, skipping\n",
      "[INFO]\tGene RABGGTA has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5586 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC9 has 0 variants, skipping\n",
      "[INFO]\tGene CDK7 has 0 variants, skipping\n",
      "[INFO]\tGene MYO3A has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02322 has 0 variants, skipping\n",
      "[INFO]\tGene LDHAL6A has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM272 has 0 variants, skipping\n",
      "[INFO]\tGene DDX56 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01303 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ22447 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150A has 0 variants, skipping\n",
      "[INFO]\tGene OPTN has 0 variants, skipping\n",
      "[INFO]\tGene SMC1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00643 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00644 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A7 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A8 has 0 variants, skipping\n",
      "[INFO]\tGene HEXIM1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM1 has 0 variants, skipping\n",
      "[INFO]\tGene CSDC2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF30 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR2 has 0 variants, skipping\n",
      "[INFO]\tGene ERFL has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H1 has 0 variants, skipping\n",
      "[INFO]\tGene CYBC1 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP8 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSP has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOB has 0 variants, skipping\n",
      "[INFO]\tGene ABCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX21P has 0 variants, skipping\n",
      "[INFO]\tGene SLCO5A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723809 has 0 variants, skipping\n",
      "[INFO]\tGene SLC19A2 has 0 variants, skipping\n",
      "[INFO]\tGene UBL3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10A has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7855 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4706 has 0 variants, skipping\n",
      "[INFO]\tGene PRRX2 has 0 variants, skipping\n",
      "[INFO]\tGene PCYOX1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506321 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4708 has 0 variants, skipping\n",
      "[INFO]\tGene HMGXB4 has 0 variants, skipping\n",
      "[INFO]\tGene FUT8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GINS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNN2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-3 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPF has 0 variants, skipping\n",
      "[INFO]\tGene HMGN1 has 0 variants, skipping\n",
      "[INFO]\tGene SEL1L has 0 variants, skipping\n",
      "[INFO]\tGene HDAC1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL23 has 0 variants, skipping\n",
      "[INFO]\tGene SCD has 0 variants, skipping\n",
      "[INFO]\tGene MIR5694 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4B has 0 variants, skipping\n",
      "[INFO]\tGene LRRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX19 has 0 variants, skipping\n",
      "[INFO]\tGene CRYAA has 0 variants, skipping\n",
      "[INFO]\tGene COL5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SCAPER has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996664 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE19 has 0 variants, skipping\n",
      "[INFO]\tGene ERI2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB3 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R3 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289511 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506358 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD169 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC6L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646548 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM21P1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR15 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM20P1 has 0 variants, skipping\n",
      "[INFO]\tGene CSKMT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC158 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928075 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01269 has 0 variants, skipping\n",
      "[INFO]\tGene SMG5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD56B has 0 variants, skipping\n",
      "[INFO]\tGene DDAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145474 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4D has 0 variants, skipping\n",
      "[INFO]\tGene KLHL41 has 0 variants, skipping\n",
      "[INFO]\tGene MYH1 has 0 variants, skipping\n",
      "[INFO]\tGene OLFM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB42 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7843 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF627 has 0 variants, skipping\n",
      "[INFO]\tGene SDF2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN3A has 0 variants, skipping\n",
      "[INFO]\tGene ENPEP has 0 variants, skipping\n",
      "[INFO]\tGene GPRC5A has 0 variants, skipping\n",
      "[INFO]\tGene CSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM245 has 0 variants, skipping\n",
      "[INFO]\tGene TEX48 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928123 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF507 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS6 has 0 variants, skipping\n",
      "[INFO]\tGene GNPDA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928143 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB16 has 0 variants, skipping\n",
      "[INFO]\tGene PRX has 0 variants, skipping\n",
      "[INFO]\tGene HSPA9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02274 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC1 has 0 variants, skipping\n",
      "[INFO]\tGene RAVER2 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAXD has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC11 has 0 variants, skipping\n",
      "[INFO]\tGene ETNK2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM62 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP64 has 0 variants, skipping\n",
      "[INFO]\tGene VPS35 has 0 variants, skipping\n",
      "[INFO]\tGene BEND4 has 0 variants, skipping\n",
      "[INFO]\tGene SORL1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RL2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL5 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL6 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPC has 0 variants, skipping\n",
      "[INFO]\tGene SNRPB2 has 0 variants, skipping\n",
      "[INFO]\tGene SOD3 has 0 variants, skipping\n",
      "[INFO]\tGene SOAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP25 has 0 variants, skipping\n",
      "[INFO]\tGene CCL1 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCE1 has 0 variants, skipping\n",
      "[INFO]\tGene SCTR has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD3 has 0 variants, skipping\n",
      "[INFO]\tGene OGDHL has 0 variants, skipping\n",
      "[INFO]\tGene MAPK12 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX2 has 0 variants, skipping\n",
      "[INFO]\tGene CST4 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM9 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP11B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN10 has 0 variants, skipping\n",
      "[INFO]\tGene ESRP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01220 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724153 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1 has 0 variants, skipping\n",
      "[INFO]\tGene DLG5 has 0 variants, skipping\n",
      "[INFO]\tGene AP2M1 has 0 variants, skipping\n",
      "[INFO]\tGene VASH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RYR3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01629 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF541 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02288 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02289 has 0 variants, skipping\n",
      "[INFO]\tGene WSCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ATMIN has 0 variants, skipping\n",
      "[INFO]\tGene BICD2 has 0 variants, skipping\n",
      "[INFO]\tGene PES1 has 0 variants, skipping\n",
      "[INFO]\tGene COPG2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA1 has 0 variants, skipping\n",
      "[INFO]\tGene MAML1 has 0 variants, skipping\n",
      "[INFO]\tGene KLC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02527 has 0 variants, skipping\n",
      "[INFO]\tGene GLA has 0 variants, skipping\n",
      "[INFO]\tGene TTC6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC198 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX4 has 0 variants, skipping\n",
      "[INFO]\tGene MEG3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370586 has 0 variants, skipping\n",
      "[INFO]\tGene WDR4 has 0 variants, skipping\n",
      "[INFO]\tGene PTEN has 0 variants, skipping\n",
      "[INFO]\tGene BROX has 0 variants, skipping\n",
      "[INFO]\tGene GARS1-DT has 0 variants, skipping\n",
      "[INFO]\tGene DIO2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO22 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01061 has 0 variants, skipping\n",
      "[INFO]\tGene CHASERR has 0 variants, skipping\n",
      "[INFO]\tGene SNORA79 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928504 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02308 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01467 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02311 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02301 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02329 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00911 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02328 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01148 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02330 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01147 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01146 has 0 variants, skipping\n",
      "[INFO]\tGene PDHA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6B has 0 variants, skipping\n",
      "[INFO]\tGene PCBD1 has 0 variants, skipping\n",
      "[INFO]\tGene OCRL has 0 variants, skipping\n",
      "[INFO]\tGene CLVS2 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT8 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAALAD2 has 0 variants, skipping\n",
      "[INFO]\tGene MED6 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC1 has 0 variants, skipping\n",
      "[INFO]\tGene MAFB has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1C has 0 variants, skipping\n",
      "[INFO]\tGene FAM221B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN23 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71F2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-7 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP157 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF2 has 0 variants, skipping\n",
      "[INFO]\tGene DDO has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK1 has 0 variants, skipping\n",
      "[INFO]\tGene WIF1 has 0 variants, skipping\n",
      "[INFO]\tGene UTP11 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKNMT has 0 variants, skipping\n",
      "[INFO]\tGene COQ4 has 0 variants, skipping\n",
      "[INFO]\tGene NOP58 has 0 variants, skipping\n",
      "[INFO]\tGene TMA7 has 0 variants, skipping\n",
      "[INFO]\tGene POMP has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC9 has 0 variants, skipping\n",
      "[INFO]\tGene SSUH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC7 has 0 variants, skipping\n",
      "[INFO]\tGene VEZF1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370619 has 0 variants, skipping\n",
      "[INFO]\tGene RSKR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928909 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370622 has 0 variants, skipping\n",
      "[INFO]\tGene TPRN has 0 variants, skipping\n",
      "[INFO]\tGene GNE has 0 variants, skipping\n",
      "[INFO]\tGene PMVK has 0 variants, skipping\n",
      "[INFO]\tGene YKT6 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID3A has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D1 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D2 has 0 variants, skipping\n",
      "[INFO]\tGene AIPL1 has 0 variants, skipping\n",
      "[INFO]\tGene IL12RB1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP12 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAZ has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11B has 0 variants, skipping\n",
      "[INFO]\tGene DEFB104B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF181 has 0 variants, skipping\n",
      "[INFO]\tGene RFX8 has 0 variants, skipping\n",
      "[INFO]\tGene GATA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A1 has 0 variants, skipping\n",
      "[INFO]\tGene CKMT2 has 0 variants, skipping\n",
      "[INFO]\tGene NRG3 has 0 variants, skipping\n",
      "[INFO]\tGene UPF3A has 0 variants, skipping\n",
      "[INFO]\tGene OR8S1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP6 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf115 has 0 variants, skipping\n",
      "[INFO]\tGene SETD7 has 0 variants, skipping\n",
      "[INFO]\tGene KDM7A has 0 variants, skipping\n",
      "[INFO]\tGene APOL3 has 0 variants, skipping\n",
      "[INFO]\tGene APOL4 has 0 variants, skipping\n",
      "[INFO]\tGene APOL5 has 0 variants, skipping\n",
      "[INFO]\tGene APOL6 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP16 has 0 variants, skipping\n",
      "[INFO]\tGene EEPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ASXL3 has 0 variants, skipping\n",
      "[INFO]\tGene HID1 has 0 variants, skipping\n",
      "[INFO]\tGene CMIP has 0 variants, skipping\n",
      "[INFO]\tGene KRT73 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02287 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD7 has 0 variants, skipping\n",
      "[INFO]\tGene ITPK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC7 has 0 variants, skipping\n",
      "[INFO]\tGene PCNX3 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY4 has 0 variants, skipping\n",
      "[INFO]\tGene UNC79 has 0 variants, skipping\n",
      "[INFO]\tGene AP5M1 has 0 variants, skipping\n",
      "[INFO]\tGene VRTN has 0 variants, skipping\n",
      "[INFO]\tGene UBA6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM181A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RD3L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4506 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC1L has 0 variants, skipping\n",
      "[INFO]\tGene NFYB has 0 variants, skipping\n",
      "[INFO]\tGene CNOT7 has 0 variants, skipping\n",
      "[INFO]\tGene TMOD4 has 0 variants, skipping\n",
      "[INFO]\tGene TBX21 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ14 has 0 variants, skipping\n",
      "[INFO]\tGene RPA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNX12 has 0 variants, skipping\n",
      "[INFO]\tGene GPR132 has 0 variants, skipping\n",
      "[INFO]\tGene STRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG2 has 0 variants, skipping\n",
      "[INFO]\tGene FGD3 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA2 has 0 variants, skipping\n",
      "[INFO]\tGene BRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02767 has 0 variants, skipping\n",
      "[INFO]\tGene PLD5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC112 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA13P has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2B has 0 variants, skipping\n",
      "[INFO]\tGene GSC-DT has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A52 has 0 variants, skipping\n",
      "[INFO]\tGene H4C15 has 0 variants, skipping\n",
      "[INFO]\tGene CDK3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3173 has 0 variants, skipping\n",
      "[INFO]\tGene PTPMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX4 has 0 variants, skipping\n",
      "[INFO]\tGene MED11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02292 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN4 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB131B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02318 has 0 variants, skipping\n",
      "[INFO]\tGene POLD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984703 has 0 variants, skipping\n",
      "[INFO]\tGene DERA has 0 variants, skipping\n",
      "[INFO]\tGene TEX10 has 0 variants, skipping\n",
      "[INFO]\tGene H19 has 0 variants, skipping\n",
      "[INFO]\tGene XACT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730202 has 0 variants, skipping\n",
      "[INFO]\tGene TDGF1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene LAG3 has 0 variants, skipping\n",
      "[INFO]\tGene SOCS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02299 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00618 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02304 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02325 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02291 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02312 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01550 has 0 variants, skipping\n",
      "[INFO]\tGene BCL6 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6764 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A6 has 0 variants, skipping\n",
      "[INFO]\tGene KDM6A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM45 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2C has 0 variants, skipping\n",
      "[INFO]\tGene PTPRE has 0 variants, skipping\n",
      "[INFO]\tGene LINC00523 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA1 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL3 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD8 has 0 variants, skipping\n",
      "[INFO]\tGene LGR5 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH3 has 0 variants, skipping\n",
      "[INFO]\tGene IQCN has 0 variants, skipping\n",
      "[INFO]\tGene SRCIN1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02702 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02745 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD112 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD113-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD114-31 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1193 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376B has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1185-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1185-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR381HG has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF713 has 0 variants, skipping\n",
      "[INFO]\tGene CLECL1 has 0 variants, skipping\n",
      "[INFO]\tGene MEG9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02285 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00524 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02314 has 0 variants, skipping\n",
      "[INFO]\tGene DIO3OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00239 has 0 variants, skipping\n",
      "[INFO]\tGene MKNK1 has 0 variants, skipping\n",
      "[INFO]\tGene DGKG has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A2 has 0 variants, skipping\n",
      "[INFO]\tGene CELF6 has 0 variants, skipping\n",
      "[INFO]\tGene ASMT has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3 has 0 variants, skipping\n",
      "[INFO]\tGene FRS2 has 0 variants, skipping\n",
      "[INFO]\tGene FRS3 has 0 variants, skipping\n",
      "[INFO]\tGene SNURF has 0 variants, skipping\n",
      "[INFO]\tGene NES has 0 variants, skipping\n",
      "[INFO]\tGene CAPN9 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXB1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP8 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO7 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO8 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO5 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD21 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO43 has 0 variants, skipping\n",
      "[INFO]\tGene ANKZF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT3 has 0 variants, skipping\n",
      "[INFO]\tGene MIPEP has 0 variants, skipping\n",
      "[INFO]\tGene YDJC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4309 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02323 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00677 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00605 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378183 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA28 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N2 has 0 variants, skipping\n",
      "[INFO]\tGene NCK2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF28 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0753 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1FNB has 0 variants, skipping\n",
      "[INFO]\tGene ACCSL has 0 variants, skipping\n",
      "[INFO]\tGene OR2V2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CB has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CA has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2B has 0 variants, skipping\n",
      "[INFO]\tGene QKI has 0 variants, skipping\n",
      "[INFO]\tGene SKOR2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02691 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02280 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4710 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC3 has 0 variants, skipping\n",
      "[INFO]\tGene STK38L has 0 variants, skipping\n",
      "[INFO]\tGene NPNT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF26 has 0 variants, skipping\n",
      "[INFO]\tGene RAB34 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00638 has 0 variants, skipping\n",
      "[INFO]\tGene HGC6.3 has 0 variants, skipping\n",
      "[INFO]\tGene ARSB has 0 variants, skipping\n",
      "[INFO]\tGene LINC02298 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6765 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB2 has 0 variants, skipping\n",
      "[INFO]\tGene NDST2 has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23C-CDRT4 has 0 variants, skipping\n",
      "[INFO]\tGene CUZD1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP24-1 has 0 variants, skipping\n",
      "[INFO]\tGene GIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PGP has 0 variants, skipping\n",
      "[INFO]\tGene ZNF732 has 0 variants, skipping\n",
      "[INFO]\tGene PARVG has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507437 has 0 variants, skipping\n",
      "[INFO]\tGene CD19 has 0 variants, skipping\n",
      "[INFO]\tGene PHETA1 has 0 variants, skipping\n",
      "[INFO]\tGene AJUBA has 0 variants, skipping\n",
      "[INFO]\tGene RAI14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370697 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8071-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8071-2 has 0 variants, skipping\n",
      "[INFO]\tGene ELK2AP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4539 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4507 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4538 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4537 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00226 has 0 variants, skipping\n",
      "[INFO]\tGene NEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene C8A has 0 variants, skipping\n",
      "[INFO]\tGene GSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene HBG1 has 0 variants, skipping\n",
      "[INFO]\tGene HBA1 has 0 variants, skipping\n",
      "[INFO]\tGene GDF5 has 0 variants, skipping\n",
      "[INFO]\tGene CRX has 0 variants, skipping\n",
      "[INFO]\tGene WRN has 0 variants, skipping\n",
      "[INFO]\tGene KCTD13 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF3 has 0 variants, skipping\n",
      "[INFO]\tGene FCER2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM170B has 0 variants, skipping\n",
      "[INFO]\tGene C5orf63 has 0 variants, skipping\n",
      "[INFO]\tGene CHD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF813 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F4 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN4 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf19 has 0 variants, skipping\n",
      "[INFO]\tGene HHATL has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D4 has 0 variants, skipping\n",
      "[INFO]\tGene SLFNL1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5B1 has 0 variants, skipping\n",
      "[INFO]\tGene NPPA has 0 variants, skipping\n",
      "[INFO]\tGene AKTIP has 0 variants, skipping\n",
      "[INFO]\tGene TUSC3 has 0 variants, skipping\n",
      "[INFO]\tGene CELF5 has 0 variants, skipping\n",
      "[INFO]\tGene CORO2B has 0 variants, skipping\n",
      "[INFO]\tGene LYZ has 0 variants, skipping\n",
      "[INFO]\tGene LPL has 0 variants, skipping\n",
      "[INFO]\tGene LIPC has 0 variants, skipping\n",
      "[INFO]\tGene LHCGR has 0 variants, skipping\n",
      "[INFO]\tGene SGCB has 0 variants, skipping\n",
      "[INFO]\tGene SGCG has 0 variants, skipping\n",
      "[INFO]\tGene LEP has 0 variants, skipping\n",
      "[INFO]\tGene LCAT has 0 variants, skipping\n",
      "[INFO]\tGene KRT9 has 0 variants, skipping\n",
      "[INFO]\tGene KRT18 has 0 variants, skipping\n",
      "[INFO]\tGene KRT12 has 0 variants, skipping\n",
      "[INFO]\tGene KIT has 0 variants, skipping\n",
      "[INFO]\tGene RBM18 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE8 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2013 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5B2 has 0 variants, skipping\n",
      "[INFO]\tGene KLC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRKD2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA4A has 0 variants, skipping\n",
      "[INFO]\tGene PDK1 has 0 variants, skipping\n",
      "[INFO]\tGene S100A13 has 0 variants, skipping\n",
      "[INFO]\tGene TANC1 has 0 variants, skipping\n",
      "[INFO]\tGene POLI has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene KPTN has 0 variants, skipping\n",
      "[INFO]\tGene CFC1 has 0 variants, skipping\n",
      "[INFO]\tGene DUS4L has 0 variants, skipping\n",
      "[INFO]\tGene CUX2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CNTFR has 0 variants, skipping\n",
      "[INFO]\tGene CNR2 has 0 variants, skipping\n",
      "[INFO]\tGene CCR3 has 0 variants, skipping\n",
      "[INFO]\tGene CHEK2P2 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P9 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3C has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8CP has 0 variants, skipping\n",
      "[INFO]\tGene NBEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF470 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01193 has 0 variants, skipping\n",
      "[INFO]\tGene FAM30C has 0 variants, skipping\n",
      "[INFO]\tGene CXADRP2 has 0 variants, skipping\n",
      "[INFO]\tGene POTEB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02203 has 0 variants, skipping\n",
      "[INFO]\tGene IGHV1OR15-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724760 has 0 variants, skipping\n",
      "[INFO]\tGene IGHV1OR15-3 has 0 variants, skipping\n",
      "[INFO]\tGene PRANCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927079 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1268A has 0 variants, skipping\n",
      "[INFO]\tGene REREP3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8EP has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P7 has 0 variants, skipping\n",
      "[INFO]\tGene RPS21 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8IP has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283683 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM2 has 0 variants, skipping\n",
      "[INFO]\tGene GAPDH has 0 variants, skipping\n",
      "[INFO]\tGene GMCL2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S3B has 0 variants, skipping\n",
      "[INFO]\tGene ABCG8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM275 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH18A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCT4 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC19A has 0 variants, skipping\n",
      "[INFO]\tGene ALPK1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP7 has 0 variants, skipping\n",
      "[INFO]\tGene MMP3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8DP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4509-3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4508 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN4 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN2 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN3 has 0 variants, skipping\n",
      "[INFO]\tGene PWRN1 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOVA2 has 0 variants, skipping\n",
      "[INFO]\tGene CCN3 has 0 variants, skipping\n",
      "[INFO]\tGene NME3 has 0 variants, skipping\n",
      "[INFO]\tGene NME2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A9P has 0 variants, skipping\n",
      "[INFO]\tGene NEXN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G10 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD107 has 0 variants, skipping\n",
      "[INFO]\tGene PWARSN has 0 variants, skipping\n",
      "[INFO]\tGene PWAR5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD64 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD108 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD109B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD109A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-22 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD116-30 has 0 variants, skipping\n",
      "[INFO]\tGene IPW has 0 variants, skipping\n",
      "[INFO]\tGene PWAR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-29 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-36 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-13 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-22 has 0 variants, skipping\n",
      "[INFO]\tGene PWAR4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-23 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-27 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-28 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-32 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-33 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-35 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-37 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-38 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-39 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-40 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-41 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-42 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-44 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-45 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-47 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD115-48 has 0 variants, skipping\n",
      "[INFO]\tGene YPEL2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64B has 0 variants, skipping\n",
      "[INFO]\tGene BHLHA9 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6C has 0 variants, skipping\n",
      "[INFO]\tGene MTF2 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF6 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAM has 0 variants, skipping\n",
      "[INFO]\tGene MOB2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT7 has 0 variants, skipping\n",
      "[INFO]\tGene KLF10 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F1 has 0 variants, skipping\n",
      "[INFO]\tGene NUGGC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF98 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00929 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1M has 0 variants, skipping\n",
      "[INFO]\tGene GPR4 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK1 has 0 variants, skipping\n",
      "[INFO]\tGene ETFBKMT has 0 variants, skipping\n",
      "[INFO]\tGene PHPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF4B has 0 variants, skipping\n",
      "[INFO]\tGene F10 has 0 variants, skipping\n",
      "[INFO]\tGene KLRF2 has 0 variants, skipping\n",
      "[INFO]\tGene YARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928042 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289656 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6IPP2 has 0 variants, skipping\n",
      "[INFO]\tGene VAX1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53C has 0 variants, skipping\n",
      "[INFO]\tGene SEMA7A has 0 variants, skipping\n",
      "[INFO]\tGene KLHL8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF136 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130111 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf62 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D2B has 0 variants, skipping\n",
      "[INFO]\tGene ITPRID2 has 0 variants, skipping\n",
      "[INFO]\tGene COX20 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1B has 0 variants, skipping\n",
      "[INFO]\tGene CHRM4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM1 has 0 variants, skipping\n",
      "[INFO]\tGene CGB3 has 0 variants, skipping\n",
      "[INFO]\tGene CGA has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P1 has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P2 has 0 variants, skipping\n",
      "[INFO]\tGene ULK4P3 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02249 has 0 variants, skipping\n",
      "[INFO]\tGene IER3 has 0 variants, skipping\n",
      "[INFO]\tGene CGREF1 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGA has 0 variants, skipping\n",
      "[INFO]\tGene CGRRF1 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P50 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288203 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02256 has 0 variants, skipping\n",
      "[INFO]\tGene ASH2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369728 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6C has 0 variants, skipping\n",
      "[INFO]\tGene CEP76 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288637 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P10 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02352 has 0 variants, skipping\n",
      "[INFO]\tGene IL33 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5B has 0 variants, skipping\n",
      "[INFO]\tGene COLCA1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF5 has 0 variants, skipping\n",
      "[INFO]\tGene PFKFB2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO25 has 0 variants, skipping\n",
      "[INFO]\tGene PC has 0 variants, skipping\n",
      "[INFO]\tGene FAM209B has 0 variants, skipping\n",
      "[INFO]\tGene TRNP1 has 0 variants, skipping\n",
      "[INFO]\tGene THYN1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR7 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL13 has 0 variants, skipping\n",
      "[INFO]\tGene FAM32A has 0 variants, skipping\n",
      "[INFO]\tGene GCDH has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131315 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO5B has 0 variants, skipping\n",
      "[INFO]\tGene MKRN2OS has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KL1 has 0 variants, skipping\n",
      "[INFO]\tGene CD99L2 has 0 variants, skipping\n",
      "[INFO]\tGene CRISPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene SESN2 has 0 variants, skipping\n",
      "[INFO]\tGene PARP9 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A8 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOO has 0 variants, skipping\n",
      "[INFO]\tGene RAMAC has 0 variants, skipping\n",
      "[INFO]\tGene TEX101 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf68 has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ2 has 0 variants, skipping\n",
      "[INFO]\tGene ERP29 has 0 variants, skipping\n",
      "[INFO]\tGene RING1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1233-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1233-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02252 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928174 has 0 variants, skipping\n",
      "[INFO]\tGene AIFM2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32AP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3942 has 0 variants, skipping\n",
      "[INFO]\tGene DPH6-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4510 has 0 variants, skipping\n",
      "[INFO]\tGene OR52J3 has 0 variants, skipping\n",
      "[INFO]\tGene HEPN1 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145845 has 0 variants, skipping\n",
      "[INFO]\tGene CRYM has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA4 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8063 has 0 variants, skipping\n",
      "[INFO]\tGene PTGS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02345 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01852 has 0 variants, skipping\n",
      "[INFO]\tGene WIZ has 0 variants, skipping\n",
      "[INFO]\tGene SMAD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370941 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF234 has 0 variants, skipping\n",
      "[INFO]\tGene BACE1 has 0 variants, skipping\n",
      "[INFO]\tGene RPAIN has 0 variants, skipping\n",
      "[INFO]\tGene FEZF1 has 0 variants, skipping\n",
      "[INFO]\tGene SRP14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCSER1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31C2 has 0 variants, skipping\n",
      "[INFO]\tGene REEP4 has 0 variants, skipping\n",
      "[INFO]\tGene VCP has 0 variants, skipping\n",
      "[INFO]\tGene MYBL2 has 0 variants, skipping\n",
      "[INFO]\tGene CBL has 0 variants, skipping\n",
      "[INFO]\tGene CBFA2T3 has 0 variants, skipping\n",
      "[INFO]\tGene CALML3 has 0 variants, skipping\n",
      "[INFO]\tGene CALM3 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL42P5 has 0 variants, skipping\n",
      "[INFO]\tGene OR6S1 has 0 variants, skipping\n",
      "[INFO]\tGene TDG has 0 variants, skipping\n",
      "[INFO]\tGene CTIF has 0 variants, skipping\n",
      "[INFO]\tGene RNF40 has 0 variants, skipping\n",
      "[INFO]\tGene SHTN1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD5 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF71 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370943 has 0 variants, skipping\n",
      "[INFO]\tGene UTP15 has 0 variants, skipping\n",
      "[INFO]\tGene PSTPIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR35 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A8 has 0 variants, skipping\n",
      "[INFO]\tGene EARS2 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A30 has 0 variants, skipping\n",
      "[INFO]\tGene HMX1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE5 has 0 variants, skipping\n",
      "[INFO]\tGene QDPR has 0 variants, skipping\n",
      "[INFO]\tGene TDRD15 has 0 variants, skipping\n",
      "[INFO]\tGene RASAL2 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF6 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK3 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4310 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370792 has 0 variants, skipping\n",
      "[INFO]\tGene EHD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD4 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL3 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC14 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf210 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF714 has 0 variants, skipping\n",
      "[INFO]\tGene SPC24 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD10 has 0 variants, skipping\n",
      "[INFO]\tGene ISM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF428 has 0 variants, skipping\n",
      "[INFO]\tGene FAM216B has 0 variants, skipping\n",
      "[INFO]\tGene KRT80 has 0 variants, skipping\n",
      "[INFO]\tGene KRT40 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD4A has 0 variants, skipping\n",
      "[INFO]\tGene SETDB1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRR has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM50B has 0 variants, skipping\n",
      "[INFO]\tGene DAGLA has 0 variants, skipping\n",
      "[INFO]\tGene BMP1 has 0 variants, skipping\n",
      "[INFO]\tGene IDS has 0 variants, skipping\n",
      "[INFO]\tGene MAN2A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA10 has 0 variants, skipping\n",
      "[INFO]\tGene VPS72 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB10 has 0 variants, skipping\n",
      "[INFO]\tGene ETDA has 0 variants, skipping\n",
      "[INFO]\tGene ETDB has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B2 has 0 variants, skipping\n",
      "[INFO]\tGene HRC has 0 variants, skipping\n",
      "[INFO]\tGene HPN has 0 variants, skipping\n",
      "[INFO]\tGene HPD has 0 variants, skipping\n",
      "[INFO]\tGene HOXD10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB5 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB1 has 0 variants, skipping\n",
      "[INFO]\tGene HPCA has 0 variants, skipping\n",
      "[INFO]\tGene HOXA4 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPK has 0 variants, skipping\n",
      "[INFO]\tGene CLIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SP1 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene USP4 has 0 variants, skipping\n",
      "[INFO]\tGene CD83 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF621 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49D1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP20 has 0 variants, skipping\n",
      "[INFO]\tGene SNX10 has 0 variants, skipping\n",
      "[INFO]\tGene CD79B has 0 variants, skipping\n",
      "[INFO]\tGene DAW1 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT112 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO2 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf74 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB1 has 0 variants, skipping\n",
      "[INFO]\tGene LDHC has 0 variants, skipping\n",
      "[INFO]\tGene ASPHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SALL4 has 0 variants, skipping\n",
      "[INFO]\tGene GJC2 has 0 variants, skipping\n",
      "[INFO]\tGene JPH2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63C has 0 variants, skipping\n",
      "[INFO]\tGene C18orf63 has 0 variants, skipping\n",
      "[INFO]\tGene NR2E1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM9B has 0 variants, skipping\n",
      "[INFO]\tGene ATG5 has 0 variants, skipping\n",
      "[INFO]\tGene GPR6 has 0 variants, skipping\n",
      "[INFO]\tGene THEMIS2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC90B has 0 variants, skipping\n",
      "[INFO]\tGene ACSM1 has 0 variants, skipping\n",
      "[INFO]\tGene UROS has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER2 has 0 variants, skipping\n",
      "[INFO]\tGene NCF4 has 0 variants, skipping\n",
      "[INFO]\tGene PEDS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAT1 has 0 variants, skipping\n",
      "[INFO]\tGene STRCP1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER2P1 has 0 variants, skipping\n",
      "[INFO]\tGene CD1D has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP3 has 0 variants, skipping\n",
      "[INFO]\tGene SERF2-C15ORF63 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1282 has 0 variants, skipping\n",
      "[INFO]\tGene MPP5 has 0 variants, skipping\n",
      "[INFO]\tGene PIN4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287387 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100419583 has 0 variants, skipping\n",
      "[INFO]\tGene LIN52 has 0 variants, skipping\n",
      "[INFO]\tGene SORD2P has 0 variants, skipping\n",
      "[INFO]\tGene PLAC8 has 0 variants, skipping\n",
      "[INFO]\tGene IMPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MARK4 has 0 variants, skipping\n",
      "[INFO]\tGene DDX19A has 0 variants, skipping\n",
      "[INFO]\tGene TOP2B has 0 variants, skipping\n",
      "[INFO]\tGene TOP2A has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928414 has 0 variants, skipping\n",
      "[INFO]\tGene CNRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2 has 0 variants, skipping\n",
      "[INFO]\tGene SP6 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN2P46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA41B has 0 variants, skipping\n",
      "[INFO]\tGene C3orf62 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370802 has 0 variants, skipping\n",
      "[INFO]\tGene DCBLD1 has 0 variants, skipping\n",
      "[INFO]\tGene BIVM has 0 variants, skipping\n",
      "[INFO]\tGene GCA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01491 has 0 variants, skipping\n",
      "[INFO]\tGene NEIL3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30A has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-3 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA7 has 0 variants, skipping\n",
      "[INFO]\tGene KIR2DL5A has 0 variants, skipping\n",
      "[INFO]\tGene MRPL57 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN18 has 0 variants, skipping\n",
      "[INFO]\tGene CAPRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MD has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF4P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4716 has 0 variants, skipping\n",
      "[INFO]\tGene EMC3 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P1 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf58 has 0 variants, skipping\n",
      "[INFO]\tGene UXT has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf78 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC150 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA5 has 0 variants, skipping\n",
      "[INFO]\tGene GABPB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSRP3 has 0 variants, skipping\n",
      "[INFO]\tGene SURF6 has 0 variants, skipping\n",
      "[INFO]\tGene GNMT has 0 variants, skipping\n",
      "[INFO]\tGene CELSR1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRD1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF13P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4713HG has 0 variants, skipping\n",
      "[INFO]\tGene ZFR2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D1 has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2C has 0 variants, skipping\n",
      "[INFO]\tGene XPO6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4713 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7973-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7973-1 has 0 variants, skipping\n",
      "[INFO]\tGene PADI2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422556 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D7-LOC100130357 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A9 has 0 variants, skipping\n",
      "[INFO]\tGene GORASP2 has 0 variants, skipping\n",
      "[INFO]\tGene GALK1 has 0 variants, skipping\n",
      "[INFO]\tGene GAA has 0 variants, skipping\n",
      "[INFO]\tGene LINC02490 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370829 has 0 variants, skipping\n",
      "[INFO]\tGene STRN3 has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD2 has 0 variants, skipping\n",
      "[INFO]\tGene MACROH2A1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF4-CCPG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR628 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR1 has 0 variants, skipping\n",
      "[INFO]\tGene LARP4B has 0 variants, skipping\n",
      "[INFO]\tGene TIMP3 has 0 variants, skipping\n",
      "[INFO]\tGene S100G has 0 variants, skipping\n",
      "[INFO]\tGene CH25H has 0 variants, skipping\n",
      "[INFO]\tGene LINC02716 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02800 has 0 variants, skipping\n",
      "[INFO]\tGene MYRF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MMGT1 has 0 variants, skipping\n",
      "[INFO]\tGene OLA1 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG5 has 0 variants, skipping\n",
      "[INFO]\tGene NAA35 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145783 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP17A1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-11 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL17 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13D has 0 variants, skipping\n",
      "[INFO]\tGene LINC00926 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01413 has 0 variants, skipping\n",
      "[INFO]\tGene RTL4 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX31 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283665 has 0 variants, skipping\n",
      "[INFO]\tGene LIPC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90AB4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928725 has 0 variants, skipping\n",
      "[INFO]\tGene SATB1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC1 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD11 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPB has 0 variants, skipping\n",
      "[INFO]\tGene SIGMAR1 has 0 variants, skipping\n",
      "[INFO]\tGene RORA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PHYKPL has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF1 has 0 variants, skipping\n",
      "[INFO]\tGene LNX2 has 0 variants, skipping\n",
      "[INFO]\tGene RORA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02349 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN5 has 0 variants, skipping\n",
      "[INFO]\tGene PSPN has 0 variants, skipping\n",
      "[INFO]\tGene CDC25B has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928907 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984784 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8067 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6085 has 0 variants, skipping\n",
      "[INFO]\tGene MGC15885 has 0 variants, skipping\n",
      "[INFO]\tGene HSPG2 has 0 variants, skipping\n",
      "[INFO]\tGene EP400 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD14A has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR33 has 0 variants, skipping\n",
      "[INFO]\tGene SHFL has 0 variants, skipping\n",
      "[INFO]\tGene MEN1 has 0 variants, skipping\n",
      "[INFO]\tGene TPM1-AS has 0 variants, skipping\n",
      "[INFO]\tGene SGCE has 0 variants, skipping\n",
      "[INFO]\tGene ADAP2 has 0 variants, skipping\n",
      "[INFO]\tGene VEPH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02568 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP91 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9B has 0 variants, skipping\n",
      "[INFO]\tGene GMPPA has 0 variants, skipping\n",
      "[INFO]\tGene USP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR1B has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12 has 0 variants, skipping\n",
      "[INFO]\tGene AVPR1A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH9A1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP4B has 0 variants, skipping\n",
      "[INFO]\tGene PSMD10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928988 has 0 variants, skipping\n",
      "[INFO]\tGene A4GALT has 0 variants, skipping\n",
      "[INFO]\tGene MTERF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A8 has 0 variants, skipping\n",
      "[INFO]\tGene RBP7 has 0 variants, skipping\n",
      "[INFO]\tGene PANX3 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf34 has 0 variants, skipping\n",
      "[INFO]\tGene METAP1D has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD7 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR4 has 0 variants, skipping\n",
      "[INFO]\tGene ATM has 0 variants, skipping\n",
      "[INFO]\tGene MIR1272 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB3 has 0 variants, skipping\n",
      "[INFO]\tGene LGR4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA24B has 0 variants, skipping\n",
      "[INFO]\tGene RNU5A-1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU5B-1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM222 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC23 has 0 variants, skipping\n",
      "[INFO]\tGene ACADS has 0 variants, skipping\n",
      "[INFO]\tGene NAT2 has 0 variants, skipping\n",
      "[INFO]\tGene A2M has 0 variants, skipping\n",
      "[INFO]\tGene MIA3 has 0 variants, skipping\n",
      "[INFO]\tGene SAP18 has 0 variants, skipping\n",
      "[INFO]\tGene ACO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13E has 0 variants, skipping\n",
      "[INFO]\tGene MIR4511 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4311 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A16 has 0 variants, skipping\n",
      "[INFO]\tGene SEC23IP has 0 variants, skipping\n",
      "[INFO]\tGene CRYL1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNB2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA14 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4512 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD16 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD18A has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K1 has 0 variants, skipping\n",
      "[INFO]\tGene AIRE has 0 variants, skipping\n",
      "[INFO]\tGene LINC01169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02206 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723493 has 0 variants, skipping\n",
      "[INFO]\tGene CCNP has 0 variants, skipping\n",
      "[INFO]\tGene COQ8B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0319L has 0 variants, skipping\n",
      "[INFO]\tGene IQCH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AG1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929076 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA7L has 0 variants, skipping\n",
      "[INFO]\tGene WDR62 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4312 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA10 has 0 variants, skipping\n",
      "[INFO]\tGene CDH3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC145694 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC114 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2B has 0 variants, skipping\n",
      "[INFO]\tGene CYP19A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMED2 has 0 variants, skipping\n",
      "[INFO]\tGene PNRC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNT2 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT29 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00593 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02205 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02204 has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA3 has 0 variants, skipping\n",
      "[INFO]\tGene SALRNA2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1A has 0 variants, skipping\n",
      "[INFO]\tGene THSD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THSD4-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRH has 0 variants, skipping\n",
      "[INFO]\tGene PTPRG has 0 variants, skipping\n",
      "[INFO]\tGene TRIM67 has 0 variants, skipping\n",
      "[INFO]\tGene RAB24 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF584 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD10 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB6 has 0 variants, skipping\n",
      "[INFO]\tGene ELOF1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4M1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2BP has 0 variants, skipping\n",
      "[INFO]\tGene LRIF1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1B has 0 variants, skipping\n",
      "[INFO]\tGene DRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ACER3 has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2 has 0 variants, skipping\n",
      "[INFO]\tGene SKA3 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP4 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2C has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2B has 0 variants, skipping\n",
      "[INFO]\tGene TERF1 has 0 variants, skipping\n",
      "[INFO]\tGene TEF has 0 variants, skipping\n",
      "[INFO]\tGene TEC has 0 variants, skipping\n",
      "[INFO]\tGene TEAD3 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD4 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA has 0 variants, skipping\n",
      "[INFO]\tGene TCF21 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA3 has 0 variants, skipping\n",
      "[INFO]\tGene TCEA2 has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L1 has 0 variants, skipping\n",
      "[INFO]\tGene TBP has 0 variants, skipping\n",
      "[INFO]\tGene TBCE has 0 variants, skipping\n",
      "[INFO]\tGene TBCC has 0 variants, skipping\n",
      "[INFO]\tGene TFAM has 0 variants, skipping\n",
      "[INFO]\tGene HEXA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM202-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02259 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3B has 0 variants, skipping\n",
      "[INFO]\tGene GIPC1 has 0 variants, skipping\n",
      "[INFO]\tGene TRUB1 has 0 variants, skipping\n",
      "[INFO]\tGene MASTL has 0 variants, skipping\n",
      "[INFO]\tGene OR5M3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01541 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8B has 0 variants, skipping\n",
      "[INFO]\tGene PIK3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP41 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK10 has 0 variants, skipping\n",
      "[INFO]\tGene HGH1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC4 has 0 variants, skipping\n",
      "[INFO]\tGene NCKIPSD has 0 variants, skipping\n",
      "[INFO]\tGene LOC283731 has 0 variants, skipping\n",
      "[INFO]\tGene CCNI2 has 0 variants, skipping\n",
      "[INFO]\tGene TAPBP has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K7 has 0 variants, skipping\n",
      "[INFO]\tGene TAF9 has 0 variants, skipping\n",
      "[INFO]\tGene TAGLN has 0 variants, skipping\n",
      "[INFO]\tGene TAF4 has 0 variants, skipping\n",
      "[INFO]\tGene TAF2 has 0 variants, skipping\n",
      "[INFO]\tGene TAC1 has 0 variants, skipping\n",
      "[INFO]\tGene TBXT has 0 variants, skipping\n",
      "[INFO]\tGene SYT5 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAP46 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02255 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1260B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3162 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6881 has 0 variants, skipping\n",
      "[INFO]\tGene MCPH1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1A has 0 variants, skipping\n",
      "[INFO]\tGene PKD2L2 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH2 has 0 variants, skipping\n",
      "[INFO]\tGene CES2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR180 has 0 variants, skipping\n",
      "[INFO]\tGene ZACN has 0 variants, skipping\n",
      "[INFO]\tGene MRPL52 has 0 variants, skipping\n",
      "[INFO]\tGene IL21R has 0 variants, skipping\n",
      "[INFO]\tGene LINC00857 has 0 variants, skipping\n",
      "[INFO]\tGene NORAD has 0 variants, skipping\n",
      "[INFO]\tGene LINC01124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6882 has 0 variants, skipping\n",
      "[INFO]\tGene IL22RA2 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf86 has 0 variants, skipping\n",
      "[INFO]\tGene PTDSS2 has 0 variants, skipping\n",
      "[INFO]\tGene ARPP19 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD7 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR631 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP3 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD5 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN5 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT13 has 0 variants, skipping\n",
      "[INFO]\tGene IL6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A2 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4313 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN3 has 0 variants, skipping\n",
      "[INFO]\tGene NACA2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK5 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC137 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929439 has 0 variants, skipping\n",
      "[INFO]\tGene TAX1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TYRO3P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF250 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC2 has 0 variants, skipping\n",
      "[INFO]\tGene JHY has 0 variants, skipping\n",
      "[INFO]\tGene ZNF141 has 0 variants, skipping\n",
      "[INFO]\tGene PAOX has 0 variants, skipping\n",
      "[INFO]\tGene EZR has 0 variants, skipping\n",
      "[INFO]\tGene FAM160A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3713 has 0 variants, skipping\n",
      "[INFO]\tGene CUL4A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88C has 0 variants, skipping\n",
      "[INFO]\tGene STARD13 has 0 variants, skipping\n",
      "[INFO]\tGene UBASH3A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00597 has 0 variants, skipping\n",
      "[INFO]\tGene CHN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929457 has 0 variants, skipping\n",
      "[INFO]\tGene TOLLIP has 0 variants, skipping\n",
      "[INFO]\tGene LIPG has 0 variants, skipping\n",
      "[INFO]\tGene GDAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC91450 has 0 variants, skipping\n",
      "[INFO]\tGene EPGN has 0 variants, skipping\n",
      "[INFO]\tGene WDR47 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54L has 0 variants, skipping\n",
      "[INFO]\tGene LUZP1 has 0 variants, skipping\n",
      "[INFO]\tGene DBF4B has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA2 has 0 variants, skipping\n",
      "[INFO]\tGene IL27 has 0 variants, skipping\n",
      "[INFO]\tGene SPESP1 has 0 variants, skipping\n",
      "[INFO]\tGene GSX1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646938 has 0 variants, skipping\n",
      "[INFO]\tGene CCN6 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RAD9A has 0 variants, skipping\n",
      "[INFO]\tGene COL5A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129540 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMED3 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFS has 0 variants, skipping\n",
      "[INFO]\tGene PRR7 has 0 variants, skipping\n",
      "[INFO]\tGene ST20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM54 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00927 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929586 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5572 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4514 has 0 variants, skipping\n",
      "[INFO]\tGene COX7A2L has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB37 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01583 has 0 variants, skipping\n",
      "[INFO]\tGene DNMT1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R3 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN14 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX16 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS7P1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC73 has 0 variants, skipping\n",
      "[INFO]\tGene VCPKMT has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2P2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724034 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC727751 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L17P has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF749 has 0 variants, skipping\n",
      "[INFO]\tGene INIP has 0 variants, skipping\n",
      "[INFO]\tGene C10orf105 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A6 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A3 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB107B has 0 variants, skipping\n",
      "[INFO]\tGene INCENP has 0 variants, skipping\n",
      "[INFO]\tGene SMARCAL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC338963 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P17 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG21 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA15 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1E has 0 variants, skipping\n",
      "[INFO]\tGene CSN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373289 has 0 variants, skipping\n",
      "[INFO]\tGene GUK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4515 has 0 variants, skipping\n",
      "[INFO]\tGene DUT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF865 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF17 has 0 variants, skipping\n",
      "[INFO]\tGene EFHB has 0 variants, skipping\n",
      "[INFO]\tGene CBLN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM232 has 0 variants, skipping\n",
      "[INFO]\tGene CCNH has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS13 has 0 variants, skipping\n",
      "[INFO]\tGene TCAP has 0 variants, skipping\n",
      "[INFO]\tGene EFL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P41 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440300 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P7 has 0 variants, skipping\n",
      "[INFO]\tGene NBAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103171574 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2E has 0 variants, skipping\n",
      "[INFO]\tGene LINC00460 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L5P has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00933 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL3 has 0 variants, skipping\n",
      "[INFO]\tGene SCAND2P has 0 variants, skipping\n",
      "[INFO]\tGene OR3A2 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D2 has 0 variants, skipping\n",
      "[INFO]\tGene OPHN1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7706 has 0 variants, skipping\n",
      "[INFO]\tGene FMO1 has 0 variants, skipping\n",
      "[INFO]\tGene ORC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1276 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AP has 0 variants, skipping\n",
      "[INFO]\tGene ARID1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01584 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370954 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724452 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF3 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L3 has 0 variants, skipping\n",
      "[INFO]\tGene CLN5 has 0 variants, skipping\n",
      "[INFO]\tGene POLH has 0 variants, skipping\n",
      "[INFO]\tGene NTRK3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP4 has 0 variants, skipping\n",
      "[INFO]\tGene NCK1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SETD6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01586 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3529 has 0 variants, skipping\n",
      "[INFO]\tGene BCO2 has 0 variants, skipping\n",
      "[INFO]\tGene MLEC has 0 variants, skipping\n",
      "[INFO]\tGene MIR6766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-3HG has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00928 has 0 variants, skipping\n",
      "[INFO]\tGene WASF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5ME has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5094 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5009 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3174 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF710-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RESF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNL3L has 0 variants, skipping\n",
      "[INFO]\tGene IDH2-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP3 has 0 variants, skipping\n",
      "[INFO]\tGene CNDP2 has 0 variants, skipping\n",
      "[INFO]\tGene NUP133 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB105A has 0 variants, skipping\n",
      "[INFO]\tGene SND1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB114 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL13P has 0 variants, skipping\n",
      "[INFO]\tGene C1orf52 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAPL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF853 has 0 variants, skipping\n",
      "[INFO]\tGene NETO1 has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1B has 0 variants, skipping\n",
      "[INFO]\tGene CRTC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01585 has 0 variants, skipping\n",
      "[INFO]\tGene OR56A1 has 0 variants, skipping\n",
      "[INFO]\tGene DBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705A has 0 variants, skipping\n",
      "[INFO]\tGene CYP27C1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB7B has 0 variants, skipping\n",
      "[INFO]\tGene CCNQ has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G9 has 0 variants, skipping\n",
      "[INFO]\tGene CYTIP has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SUCO has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP22 has 0 variants, skipping\n",
      "[INFO]\tGene PRC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP126 has 0 variants, skipping\n",
      "[INFO]\tGene ZC2HC1B has 0 variants, skipping\n",
      "[INFO]\tGene VPS33B-DT has 0 variants, skipping\n",
      "[INFO]\tGene NAF1 has 0 variants, skipping\n",
      "[INFO]\tGene CRAT37 has 0 variants, skipping\n",
      "[INFO]\tGene LOC104613533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00930 has 0 variants, skipping\n",
      "[INFO]\tGene ASB9P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02207 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01580 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01581 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440311 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00924 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1L has 0 variants, skipping\n",
      "[INFO]\tGene SPATA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927310 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00923 has 0 variants, skipping\n",
      "[INFO]\tGene NUP62CL has 0 variants, skipping\n",
      "[INFO]\tGene LINC02251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01582 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02351 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4714 has 0 variants, skipping\n",
      "[INFO]\tGene LUNAR1 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH1 has 0 variants, skipping\n",
      "[INFO]\tGene EML6 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN2A has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01213 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF4P has 0 variants, skipping\n",
      "[INFO]\tGene HSP90B2P has 0 variants, skipping\n",
      "[INFO]\tGene HSPA4L has 0 variants, skipping\n",
      "[INFO]\tGene MPDU1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA18 has 0 variants, skipping\n",
      "[INFO]\tGene RAD23B has 0 variants, skipping\n",
      "[INFO]\tGene PRRC1 has 0 variants, skipping\n",
      "[INFO]\tGene AFF3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT85 has 0 variants, skipping\n",
      "[INFO]\tGene DNM1P46 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400464 has 0 variants, skipping\n",
      "[INFO]\tGene RBM25 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA41 has 0 variants, skipping\n",
      "[INFO]\tGene CERS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL2NB has 0 variants, skipping\n",
      "[INFO]\tGene DLAT has 0 variants, skipping\n",
      "[INFO]\tGene PRKXP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723335 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927751 has 0 variants, skipping\n",
      "[INFO]\tGene H1-8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507472 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02348 has 0 variants, skipping\n",
      "[INFO]\tGene FOLH1 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A4E has 0 variants, skipping\n",
      "[INFO]\tGene FAM25E has 0 variants, skipping\n",
      "[INFO]\tGene OR4F13P has 0 variants, skipping\n",
      "[INFO]\tGene FAM138E has 0 variants, skipping\n",
      "[INFO]\tGene WASH3P has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-4 has 0 variants, skipping\n",
      "[INFO]\tGene SP100 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIZ has 0 variants, skipping\n",
      "[INFO]\tGene MRPL9 has 0 variants, skipping\n",
      "[INFO]\tGene ANO3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH19 has 0 variants, skipping\n",
      "[INFO]\tGene SPEGNB has 0 variants, skipping\n",
      "[INFO]\tGene EAPP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM242 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOS has 0 variants, skipping\n",
      "[INFO]\tGene PDP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A10 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3E has 0 variants, skipping\n",
      "[INFO]\tGene NISCH has 0 variants, skipping\n",
      "[INFO]\tGene NT5C1B has 0 variants, skipping\n",
      "[INFO]\tGene RPS14 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA2 has 0 variants, skipping\n",
      "[INFO]\tGene LTA has 0 variants, skipping\n",
      "[INFO]\tGene CD4 has 0 variants, skipping\n",
      "[INFO]\tGene NCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene VRK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389895 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene COA3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH17 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB11 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQA2 has 0 variants, skipping\n",
      "[INFO]\tGene UBB has 0 variants, skipping\n",
      "[INFO]\tGene WWC1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM184A has 0 variants, skipping\n",
      "[INFO]\tGene RWDD4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD4A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC36 has 0 variants, skipping\n",
      "[INFO]\tGene PGK1 has 0 variants, skipping\n",
      "[INFO]\tGene PLG has 0 variants, skipping\n",
      "[INFO]\tGene PGAM2 has 0 variants, skipping\n",
      "[INFO]\tGene DYRK3 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM4 has 0 variants, skipping\n",
      "[INFO]\tGene AOAH has 0 variants, skipping\n",
      "[INFO]\tGene LASP1 has 0 variants, skipping\n",
      "[INFO]\tGene TWF1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT4H1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT1 has 0 variants, skipping\n",
      "[INFO]\tGene SRF has 0 variants, skipping\n",
      "[INFO]\tGene WNT2B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724720 has 0 variants, skipping\n",
      "[INFO]\tGene OR51S1 has 0 variants, skipping\n",
      "[INFO]\tGene CLYBL has 0 variants, skipping\n",
      "[INFO]\tGene SMIM41 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM40 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM38 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2C has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3B has 0 variants, skipping\n",
      "[INFO]\tGene NXF2B has 0 variants, skipping\n",
      "[INFO]\tGene BCL11B has 0 variants, skipping\n",
      "[INFO]\tGene STAG3 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12J has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L6 has 0 variants, skipping\n",
      "[INFO]\tGene WFS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYL2 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B1 has 0 variants, skipping\n",
      "[INFO]\tGene MAT1A has 0 variants, skipping\n",
      "[INFO]\tGene WASF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF317 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D30 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD4 has 0 variants, skipping\n",
      "[INFO]\tGene OSMR has 0 variants, skipping\n",
      "[INFO]\tGene FDX1 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene IRX5 has 0 variants, skipping\n",
      "[INFO]\tGene CHD3 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE18 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA2 has 0 variants, skipping\n",
      "[INFO]\tGene ACSBG2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP5 has 0 variants, skipping\n",
      "[INFO]\tGene GH2 has 0 variants, skipping\n",
      "[INFO]\tGene ETHE1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D13 has 0 variants, skipping\n",
      "[INFO]\tGene OR14A16 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D5 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1LW has 0 variants, skipping\n",
      "[INFO]\tGene CCDC33 has 0 variants, skipping\n",
      "[INFO]\tGene CENPU has 0 variants, skipping\n",
      "[INFO]\tGene PSMA5 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB8 has 0 variants, skipping\n",
      "[INFO]\tGene VPS51 has 0 variants, skipping\n",
      "[INFO]\tGene MYL9 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH1 has 0 variants, skipping\n",
      "[INFO]\tGene TNK2 has 0 variants, skipping\n",
      "[INFO]\tGene PITX2 has 0 variants, skipping\n",
      "[INFO]\tGene EXD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR572 has 0 variants, skipping\n",
      "[INFO]\tGene WASIR2 has 0 variants, skipping\n",
      "[INFO]\tGene PHKG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM234A has 0 variants, skipping\n",
      "[INFO]\tGene GIMD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRTOMT has 0 variants, skipping\n",
      "[INFO]\tGene RNF225 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf27 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA1 has 0 variants, skipping\n",
      "[INFO]\tGene TBRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134368 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W3 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C9 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C13 has 0 variants, skipping\n",
      "[INFO]\tGene OR5A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP37 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00235 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5587 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3176 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11L2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SETDB2-PHF11 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf57 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287175 has 0 variants, skipping\n",
      "[INFO]\tGene COA7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371038 has 0 variants, skipping\n",
      "[INFO]\tGene MDH1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA52 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02825 has 0 variants, skipping\n",
      "[INFO]\tGene STARD6 has 0 variants, skipping\n",
      "[INFO]\tGene RHOQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRPS1 has 0 variants, skipping\n",
      "[INFO]\tGene CD177 has 0 variants, skipping\n",
      "[INFO]\tGene PLXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LMF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG1 has 0 variants, skipping\n",
      "[INFO]\tGene CD55 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD8 has 0 variants, skipping\n",
      "[INFO]\tGene USP21 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM237 has 0 variants, skipping\n",
      "[INFO]\tGene MYL3 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02789 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02714 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371046 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD3 has 0 variants, skipping\n",
      "[INFO]\tGene IGFALS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00254 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02124 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA64 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA78 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC20 has 0 variants, skipping\n",
      "[INFO]\tGene GDF10 has 0 variants, skipping\n",
      "[INFO]\tGene RPL3L has 0 variants, skipping\n",
      "[INFO]\tGene OR8D2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8D1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD11 has 0 variants, skipping\n",
      "[INFO]\tGene PTRH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRCP has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD7 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf16 has 0 variants, skipping\n",
      "[INFO]\tGene EQTN has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D1 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF23 has 0 variants, skipping\n",
      "[INFO]\tGene FGF22 has 0 variants, skipping\n",
      "[INFO]\tGene GDF3 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF117 has 0 variants, skipping\n",
      "[INFO]\tGene CELA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF18 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF56 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511B2 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD60 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4F has 0 variants, skipping\n",
      "[INFO]\tGene TUT4 has 0 variants, skipping\n",
      "[INFO]\tGene FOXRED1 has 0 variants, skipping\n",
      "[INFO]\tGene PPBP has 0 variants, skipping\n",
      "[INFO]\tGene PPAT has 0 variants, skipping\n",
      "[INFO]\tGene POLE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC106660606 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA17P has 0 variants, skipping\n",
      "[INFO]\tGene DMTF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6767 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM53 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6768 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517B has 0 variants, skipping\n",
      "[INFO]\tGene LOC652276 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42627 has 0 variants, skipping\n",
      "[INFO]\tGene ERVK13-1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA3C has 0 variants, skipping\n",
      "[INFO]\tGene NONO has 0 variants, skipping\n",
      "[INFO]\tGene PRSS30P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929613 has 0 variants, skipping\n",
      "[INFO]\tGene LCA5 has 0 variants, skipping\n",
      "[INFO]\tGene HMMR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128770 has 0 variants, skipping\n",
      "[INFO]\tGene MMP25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF213-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NHP2 has 0 variants, skipping\n",
      "[INFO]\tGene MUC12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASP16P has 0 variants, skipping\n",
      "[INFO]\tGene OR1F2P has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD29 has 0 variants, skipping\n",
      "[INFO]\tGene LARP7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00921 has 0 variants, skipping\n",
      "[INFO]\tGene SLC40A1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6126 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E2B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01569 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM236 has 0 variants, skipping\n",
      "[INFO]\tGene SSX4B has 0 variants, skipping\n",
      "[INFO]\tGene PDCD10 has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD5 has 0 variants, skipping\n",
      "[INFO]\tGene MUC6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6769A has 0 variants, skipping\n",
      "[INFO]\tGene IQUB has 0 variants, skipping\n",
      "[INFO]\tGene TAFA5 has 0 variants, skipping\n",
      "[INFO]\tGene SAMM50 has 0 variants, skipping\n",
      "[INFO]\tGene BRI3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13D has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L13 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS40A has 0 variants, skipping\n",
      "[INFO]\tGene AUP1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf84 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF10 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L3 has 0 variants, skipping\n",
      "[INFO]\tGene NAGPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PI4K2A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01570 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8065 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM131 has 0 variants, skipping\n",
      "[INFO]\tGene CASS4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130283 has 0 variants, skipping\n",
      "[INFO]\tGene PPP5C has 0 variants, skipping\n",
      "[INFO]\tGene FBXW10 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01195 has 0 variants, skipping\n",
      "[INFO]\tGene SPX has 0 variants, skipping\n",
      "[INFO]\tGene PM20D1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723753 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01290 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA1A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2K has 0 variants, skipping\n",
      "[INFO]\tGene RNF214 has 0 variants, skipping\n",
      "[INFO]\tGene LTB4R2 has 0 variants, skipping\n",
      "[INFO]\tGene BEGAIN has 0 variants, skipping\n",
      "[INFO]\tGene HOMEZ has 0 variants, skipping\n",
      "[INFO]\tGene ZNF687 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371083 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927131 has 0 variants, skipping\n",
      "[INFO]\tGene INSR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF8 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGC has 0 variants, skipping\n",
      "[INFO]\tGene NFS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF516 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4718 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD1 has 0 variants, skipping\n",
      "[INFO]\tGene ERVV-2 has 0 variants, skipping\n",
      "[INFO]\tGene PDK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02185 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02186 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3L4 has 0 variants, skipping\n",
      "[INFO]\tGene POMGNT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02130 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC11A2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3179-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288162 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6770-3 has 0 variants, skipping\n",
      "[INFO]\tGene DEK has 0 variants, skipping\n",
      "[INFO]\tGene SARDH has 0 variants, skipping\n",
      "[INFO]\tGene C1orf94 has 0 variants, skipping\n",
      "[INFO]\tGene NXNL2 has 0 variants, skipping\n",
      "[INFO]\tGene DUS3L has 0 variants, skipping\n",
      "[INFO]\tGene NECAB1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1972-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1972-2 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene FGF17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505915 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P6-NPIPP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3180-4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100190986 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK4 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK1 has 0 variants, skipping\n",
      "[INFO]\tGene MAS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1C has 0 variants, skipping\n",
      "[INFO]\tGene ATOH1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOX2A has 0 variants, skipping\n",
      "[INFO]\tGene PPM1J has 0 variants, skipping\n",
      "[INFO]\tGene P4HA3 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A12 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf72 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf197 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF652 has 0 variants, skipping\n",
      "[INFO]\tGene CERKL has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723692 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20A has 0 variants, skipping\n",
      "[INFO]\tGene PRECSIT has 0 variants, skipping\n",
      "[INFO]\tGene NOP56 has 0 variants, skipping\n",
      "[INFO]\tGene DOLK has 0 variants, skipping\n",
      "[INFO]\tGene GLTP has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC15 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6P1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf36 has 0 variants, skipping\n",
      "[INFO]\tGene ADM2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723385 has 0 variants, skipping\n",
      "[INFO]\tGene SCN9A has 0 variants, skipping\n",
      "[INFO]\tGene SMARCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN7A has 0 variants, skipping\n",
      "[INFO]\tGene CLEC11A has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB4 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC1 has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1 has 0 variants, skipping\n",
      "[INFO]\tGene GASK1A has 0 variants, skipping\n",
      "[INFO]\tGene MSMO1 has 0 variants, skipping\n",
      "[INFO]\tGene NDP has 0 variants, skipping\n",
      "[INFO]\tGene NCF1 has 0 variants, skipping\n",
      "[INFO]\tGene NFYA has 0 variants, skipping\n",
      "[INFO]\tGene TICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF500 has 0 variants, skipping\n",
      "[INFO]\tGene OTOA has 0 variants, skipping\n",
      "[INFO]\tGene KCNH3 has 0 variants, skipping\n",
      "[INFO]\tGene VAV3 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B4 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC5 has 0 variants, skipping\n",
      "[INFO]\tGene ARVCF has 0 variants, skipping\n",
      "[INFO]\tGene CRYBA2 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ARTN has 0 variants, skipping\n",
      "[INFO]\tGene KRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TAGAP has 0 variants, skipping\n",
      "[INFO]\tGene CIB3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB118 has 0 variants, skipping\n",
      "[INFO]\tGene IP6K3 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT15 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT5 has 0 variants, skipping\n",
      "[INFO]\tGene OR1J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AC2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C3 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX4 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRX3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKH has 0 variants, skipping\n",
      "[INFO]\tGene SCGB3A2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR101 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1A has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4B has 0 variants, skipping\n",
      "[INFO]\tGene ASS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT1 has 0 variants, skipping\n",
      "[INFO]\tGene FBH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29P1 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3680-2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927814 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF211 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200A has 0 variants, skipping\n",
      "[INFO]\tGene OTOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2 has 0 variants, skipping\n",
      "[INFO]\tGene TLE7 has 0 variants, skipping\n",
      "[INFO]\tGene PPOX has 0 variants, skipping\n",
      "[INFO]\tGene BGLAP has 0 variants, skipping\n",
      "[INFO]\tGene PMEPA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440346 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD13B has 0 variants, skipping\n",
      "[INFO]\tGene CD151 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF4 has 0 variants, skipping\n",
      "[INFO]\tGene BICD1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS29 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730183 has 0 variants, skipping\n",
      "[INFO]\tGene PI4K2B has 0 variants, skipping\n",
      "[INFO]\tGene DUSP9 has 0 variants, skipping\n",
      "[INFO]\tGene PSME2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1273H has 0 variants, skipping\n",
      "[INFO]\tGene LINC01567 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA6 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB1 has 0 variants, skipping\n",
      "[INFO]\tGene HASPIN has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP17-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-8 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-5 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP15 has 0 variants, skipping\n",
      "[INFO]\tGene F11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF185 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P1 has 0 variants, skipping\n",
      "[INFO]\tGene RRN3P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC554206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02175 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548W has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE1-DT has 0 variants, skipping\n",
      "[INFO]\tGene IL21R-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NGRN has 0 variants, skipping\n",
      "[INFO]\tGene MYCL has 0 variants, skipping\n",
      "[INFO]\tGene TAAR2 has 0 variants, skipping\n",
      "[INFO]\tGene INTS11 has 0 variants, skipping\n",
      "[INFO]\tGene WAPL has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128079 has 0 variants, skipping\n",
      "[INFO]\tGene WDR3 has 0 variants, skipping\n",
      "[INFO]\tGene VLDLR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF845 has 0 variants, skipping\n",
      "[INFO]\tGene GSAP has 0 variants, skipping\n",
      "[INFO]\tGene DDX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6862-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6862-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPD2 has 0 variants, skipping\n",
      "[INFO]\tGene TBCA has 0 variants, skipping\n",
      "[INFO]\tGene SNRPA has 0 variants, skipping\n",
      "[INFO]\tGene SMS has 0 variants, skipping\n",
      "[INFO]\tGene SULT2B1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP23 has 0 variants, skipping\n",
      "[INFO]\tGene CACFD1 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF559-ZNF177 has 0 variants, skipping\n",
      "[INFO]\tGene GLRB has 0 variants, skipping\n",
      "[INFO]\tGene GHRHR has 0 variants, skipping\n",
      "[INFO]\tGene GART has 0 variants, skipping\n",
      "[INFO]\tGene MIR4721 has 0 variants, skipping\n",
      "[INFO]\tGene HTATSF1 has 0 variants, skipping\n",
      "[INFO]\tGene UGDH has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4517 has 0 variants, skipping\n",
      "[INFO]\tGene PGM1 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC1 has 0 variants, skipping\n",
      "[INFO]\tGene WSCD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29P2 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC606724 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC4 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1A-SULT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1B-SULT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388242 has 0 variants, skipping\n",
      "[INFO]\tGene LOC613038 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR3B has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA7 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA5 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA3 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A40 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG12 has 0 variants, skipping\n",
      "[INFO]\tGene NSFL1C has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA12 has 0 variants, skipping\n",
      "[INFO]\tGene SULF2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIPTOSP has 0 variants, skipping\n",
      "[INFO]\tGene FBXO21 has 0 variants, skipping\n",
      "[INFO]\tGene RBMX has 0 variants, skipping\n",
      "[INFO]\tGene CXorf65 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642361 has 0 variants, skipping\n",
      "[INFO]\tGene MUSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12B has 0 variants, skipping\n",
      "[INFO]\tGene ACER2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM102B has 0 variants, skipping\n",
      "[INFO]\tGene CDRT1 has 0 variants, skipping\n",
      "[INFO]\tGene LCTL has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM has 0 variants, skipping\n",
      "[INFO]\tGene DRG2 has 0 variants, skipping\n",
      "[INFO]\tGene ALS2 has 0 variants, skipping\n",
      "[INFO]\tGene LZTS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928595 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD9 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P5 has 0 variants, skipping\n",
      "[INFO]\tGene IL4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80C has 0 variants, skipping\n",
      "[INFO]\tGene IZUMO2 has 0 variants, skipping\n",
      "[INFO]\tGene AHSA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4518 has 0 variants, skipping\n",
      "[INFO]\tGene TRDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF689 has 0 variants, skipping\n",
      "[INFO]\tGene CASP12 has 0 variants, skipping\n",
      "[INFO]\tGene GSTO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA30 has 0 variants, skipping\n",
      "[INFO]\tGene BLID has 0 variants, skipping\n",
      "[INFO]\tGene COP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEP295NL has 0 variants, skipping\n",
      "[INFO]\tGene HELLS has 0 variants, skipping\n",
      "[INFO]\tGene MIR4519 has 0 variants, skipping\n",
      "[INFO]\tGene MIR762HG has 0 variants, skipping\n",
      "[INFO]\tGene PTPN9 has 0 variants, skipping\n",
      "[INFO]\tGene DDX10 has 0 variants, skipping\n",
      "[INFO]\tGene DDX6 has 0 variants, skipping\n",
      "[INFO]\tGene DMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX5 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH2 has 0 variants, skipping\n",
      "[INFO]\tGene PALM has 0 variants, skipping\n",
      "[INFO]\tGene REG3A has 0 variants, skipping\n",
      "[INFO]\tGene NAV3 has 0 variants, skipping\n",
      "[INFO]\tGene PYCARD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF320 has 0 variants, skipping\n",
      "[INFO]\tGene RDH11 has 0 variants, skipping\n",
      "[INFO]\tGene C1R has 0 variants, skipping\n",
      "[INFO]\tGene ADCY3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02190 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2KP has 0 variants, skipping\n",
      "[INFO]\tGene YBX3P1 has 0 variants, skipping\n",
      "[INFO]\tGene CLUHP3 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P4 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3D has 0 variants, skipping\n",
      "[INFO]\tGene LOC390705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369266 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724135 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A10P has 0 variants, skipping\n",
      "[INFO]\tGene DCTN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC93622 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG8 has 0 variants, skipping\n",
      "[INFO]\tGene SACS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369212 has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929479 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP7P13 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2MP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01566 has 0 variants, skipping\n",
      "[INFO]\tGene FRG2DP has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG3HP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02167 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26P1 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf180 has 0 variants, skipping\n",
      "[INFO]\tGene SLIRP has 0 variants, skipping\n",
      "[INFO]\tGene UPF2 has 0 variants, skipping\n",
      "[INFO]\tGene CST9L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF222 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD148 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02192 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02133 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02134 has 0 variants, skipping\n",
      "[INFO]\tGene CCT8 has 0 variants, skipping\n",
      "[INFO]\tGene PCP2 has 0 variants, skipping\n",
      "[INFO]\tGene ESPNL has 0 variants, skipping\n",
      "[INFO]\tGene ADAM3B has 0 variants, skipping\n",
      "[INFO]\tGene THG1L has 0 variants, skipping\n",
      "[INFO]\tGene PCMTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A36 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6771 has 0 variants, skipping\n",
      "[INFO]\tGene TNNI3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927272 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene SAFB has 0 variants, skipping\n",
      "[INFO]\tGene S100A10 has 0 variants, skipping\n",
      "[INFO]\tGene S100A9 has 0 variants, skipping\n",
      "[INFO]\tGene SLPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR3181 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02168 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AI has 0 variants, skipping\n",
      "[INFO]\tGene LINC02127 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01571 has 0 variants, skipping\n",
      "[INFO]\tGene PPCS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00919 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02180 has 0 variants, skipping\n",
      "[INFO]\tGene CASC22 has 0 variants, skipping\n",
      "[INFO]\tGene CASC16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371267 has 0 variants, skipping\n",
      "[INFO]\tGene RECK has 0 variants, skipping\n",
      "[INFO]\tGene CTAG1A has 0 variants, skipping\n",
      "[INFO]\tGene MS4A6E has 0 variants, skipping\n",
      "[INFO]\tGene LIPH has 0 variants, skipping\n",
      "[INFO]\tGene TENT5D has 0 variants, skipping\n",
      "[INFO]\tGene TMEM123 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF6 has 0 variants, skipping\n",
      "[INFO]\tGene FHAD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723373 has 0 variants, skipping\n",
      "[INFO]\tGene YIF1B has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L12 has 0 variants, skipping\n",
      "[INFO]\tGene CENPN has 0 variants, skipping\n",
      "[INFO]\tGene RDH12 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL14 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf66 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF664 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1L2 has 0 variants, skipping\n",
      "[INFO]\tGene AMDHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CWF19L2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP42 has 0 variants, skipping\n",
      "[INFO]\tGene PIWIL4 has 0 variants, skipping\n",
      "[INFO]\tGene OR51E1 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP3 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3D has 0 variants, skipping\n",
      "[INFO]\tGene FTO-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927480 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA6 has 0 variants, skipping\n",
      "[INFO]\tGene MMP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MED27 has 0 variants, skipping\n",
      "[INFO]\tGene ACY3 has 0 variants, skipping\n",
      "[INFO]\tGene ARAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CES1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CES1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF404 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283856 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL22 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP434H168 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3935 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K13 has 0 variants, skipping\n",
      "[INFO]\tGene OR1I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K14 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A3 has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIB has 0 variants, skipping\n",
      "[INFO]\tGene TDRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1L has 0 variants, skipping\n",
      "[INFO]\tGene ZAR1 has 0 variants, skipping\n",
      "[INFO]\tGene MT1DP has 0 variants, skipping\n",
      "[INFO]\tGene MT1IP has 0 variants, skipping\n",
      "[INFO]\tGene GPR33 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F16 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AE1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A15 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A16 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R13B has 0 variants, skipping\n",
      "[INFO]\tGene STIP1 has 0 variants, skipping\n",
      "[INFO]\tGene RIOX2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB10 has 0 variants, skipping\n",
      "[INFO]\tGene COPG1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL3 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3C has 0 variants, skipping\n",
      "[INFO]\tGene FAM170A has 0 variants, skipping\n",
      "[INFO]\tGene TOX has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD4 has 0 variants, skipping\n",
      "[INFO]\tGene MTUS1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS2 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2B has 0 variants, skipping\n",
      "[INFO]\tGene POLR2A has 0 variants, skipping\n",
      "[INFO]\tGene PNLIP has 0 variants, skipping\n",
      "[INFO]\tGene POGLUT2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR18 has 0 variants, skipping\n",
      "[INFO]\tGene ASB9 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF3 has 0 variants, skipping\n",
      "[INFO]\tGene UXS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6772 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAL has 0 variants, skipping\n",
      "[INFO]\tGene KRT33B has 0 variants, skipping\n",
      "[INFO]\tGene KRT35 has 0 variants, skipping\n",
      "[INFO]\tGene KRT32 has 0 variants, skipping\n",
      "[INFO]\tGene FTSJ1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE5 has 0 variants, skipping\n",
      "[INFO]\tGene KCND2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDCD3 has 0 variants, skipping\n",
      "[INFO]\tGene TREML2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8084 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6783 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6751 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF84-DT has 0 variants, skipping\n",
      "[INFO]\tGene SEH1L has 0 variants, skipping\n",
      "[INFO]\tGene MPST has 0 variants, skipping\n",
      "[INFO]\tGene PAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene PREX2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX21 has 0 variants, skipping\n",
      "[INFO]\tGene MROH8 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA46 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50A has 0 variants, skipping\n",
      "[INFO]\tGene APOOP5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02141 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02126 has 0 variants, skipping\n",
      "[INFO]\tGene LOC647264 has 0 variants, skipping\n",
      "[INFO]\tGene H2BU1 has 0 variants, skipping\n",
      "[INFO]\tGene SYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene BEAN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13C has 0 variants, skipping\n",
      "[INFO]\tGene EFNA4 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR3 has 0 variants, skipping\n",
      "[INFO]\tGene E2F1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID3A has 0 variants, skipping\n",
      "[INFO]\tGene DLX6 has 0 variants, skipping\n",
      "[INFO]\tGene DLX5 has 0 variants, skipping\n",
      "[INFO]\tGene DLX3 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA3 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM3 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1LI2 has 0 variants, skipping\n",
      "[INFO]\tGene NGDN has 0 variants, skipping\n",
      "[INFO]\tGene LOC106699570 has 0 variants, skipping\n",
      "[INFO]\tGene SSH1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F5 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC8 has 0 variants, skipping\n",
      "[INFO]\tGene AK3 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT3 has 0 variants, skipping\n",
      "[INFO]\tGene CHGB has 0 variants, skipping\n",
      "[INFO]\tGene IGSF9 has 0 variants, skipping\n",
      "[INFO]\tGene RGS6 has 0 variants, skipping\n",
      "[INFO]\tGene TINAGL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF213 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF529 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A14 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1614 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA5 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA10 has 0 variants, skipping\n",
      "[INFO]\tGene MEAK7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF253 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-8 has 0 variants, skipping\n",
      "[INFO]\tGene BPY2 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF15 has 0 variants, skipping\n",
      "[INFO]\tGene NR1H2 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELL2 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PUDP has 0 variants, skipping\n",
      "[INFO]\tGene SLAMF9 has 0 variants, skipping\n",
      "[INFO]\tGene OPRPN has 0 variants, skipping\n",
      "[INFO]\tGene VASH2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505942 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984813 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD6B has 0 variants, skipping\n",
      "[INFO]\tGene TECR has 0 variants, skipping\n",
      "[INFO]\tGene HRG has 0 variants, skipping\n",
      "[INFO]\tGene HLCS has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10B has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10C has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF10D has 0 variants, skipping\n",
      "[INFO]\tGene FBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D1 has 0 variants, skipping\n",
      "[INFO]\tGene RTF1 has 0 variants, skipping\n",
      "[INFO]\tGene RNGTT has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ-SCHIP1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A2 has 0 variants, skipping\n",
      "[INFO]\tGene CCL27 has 0 variants, skipping\n",
      "[INFO]\tGene SRCAP has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP2 has 0 variants, skipping\n",
      "[INFO]\tGene CLPX has 0 variants, skipping\n",
      "[INFO]\tGene FTCD has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A4 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD10 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6773 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRD has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A1 has 0 variants, skipping\n",
      "[INFO]\tGene DPM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF418 has 0 variants, skipping\n",
      "[INFO]\tGene EZH2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2W has 0 variants, skipping\n",
      "[INFO]\tGene SPATA48 has 0 variants, skipping\n",
      "[INFO]\tGene WHAMM has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRF has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF5 has 0 variants, skipping\n",
      "[INFO]\tGene HTATIP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA9 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB6 has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL2 has 0 variants, skipping\n",
      "[INFO]\tGene STX18 has 0 variants, skipping\n",
      "[INFO]\tGene PIGW has 0 variants, skipping\n",
      "[INFO]\tGene MIR1538 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A9 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM29 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf58 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13H has 0 variants, skipping\n",
      "[INFO]\tGene HACD4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIP has 0 variants, skipping\n",
      "[INFO]\tGene PXDNL has 0 variants, skipping\n",
      "[INFO]\tGene ADHFE1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA46 has 0 variants, skipping\n",
      "[INFO]\tGene THEM5 has 0 variants, skipping\n",
      "[INFO]\tGene ICE1 has 0 variants, skipping\n",
      "[INFO]\tGene PDXDC2P-NPIPB14P has 0 variants, skipping\n",
      "[INFO]\tGene LOC400541 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM268 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371328 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1P7 has 0 variants, skipping\n",
      "[INFO]\tGene CLGN has 0 variants, skipping\n",
      "[INFO]\tGene SPAG9 has 0 variants, skipping\n",
      "[INFO]\tGene HACD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506083 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf74 has 0 variants, skipping\n",
      "[INFO]\tGene NHSL1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB10 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM270 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK3 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD111B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD111 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4A has 0 variants, skipping\n",
      "[INFO]\tGene TCF15 has 0 variants, skipping\n",
      "[INFO]\tGene SPOCK1 has 0 variants, skipping\n",
      "[INFO]\tGene TBX6 has 0 variants, skipping\n",
      "[INFO]\tGene DAB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC117 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNB has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02136 has 0 variants, skipping\n",
      "[INFO]\tGene CEP120 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371335 has 0 variants, skipping\n",
      "[INFO]\tGene TAT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD71 has 0 variants, skipping\n",
      "[INFO]\tGene IL12B has 0 variants, skipping\n",
      "[INFO]\tGene IL9R has 0 variants, skipping\n",
      "[INFO]\tGene IL7R has 0 variants, skipping\n",
      "[INFO]\tGene IL3RA has 0 variants, skipping\n",
      "[INFO]\tGene ANKFY1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01572 has 0 variants, skipping\n",
      "[INFO]\tGene HCCAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01568 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928035 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5RL has 0 variants, skipping\n",
      "[INFO]\tGene LOC283922 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376772 has 0 variants, skipping\n",
      "[INFO]\tGene DGKZ has 0 variants, skipping\n",
      "[INFO]\tGene ARPC3 has 0 variants, skipping\n",
      "[INFO]\tGene LMLN has 0 variants, skipping\n",
      "[INFO]\tGene UBTF has 0 variants, skipping\n",
      "[INFO]\tGene NSUN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21 has 0 variants, skipping\n",
      "[INFO]\tGene CUBN has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506281 has 0 variants, skipping\n",
      "[INFO]\tGene LEXM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF609 has 0 variants, skipping\n",
      "[INFO]\tGene NRSN2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT34 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ12 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA4 has 0 variants, skipping\n",
      "[INFO]\tGene ILF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM5 has 0 variants, skipping\n",
      "[INFO]\tGene COL20A1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC15 has 0 variants, skipping\n",
      "[INFO]\tGene KATNBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ELMO3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM29 has 0 variants, skipping\n",
      "[INFO]\tGene CCR2 has 0 variants, skipping\n",
      "[INFO]\tGene CASK has 0 variants, skipping\n",
      "[INFO]\tGene CYBRD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4719 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf116 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM15 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D2 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALSL has 0 variants, skipping\n",
      "[INFO]\tGene LTB4R has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2H1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMSN1 has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFTRR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01229 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724084 has 0 variants, skipping\n",
      "[INFO]\tGene CYP7B1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01227 has 0 variants, skipping\n",
      "[INFO]\tGene ARLNC1 has 0 variants, skipping\n",
      "[INFO]\tGene RMST has 0 variants, skipping\n",
      "[INFO]\tGene BBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF1 has 0 variants, skipping\n",
      "[INFO]\tGene LBX1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD1B has 0 variants, skipping\n",
      "[INFO]\tGene DEFA1 has 0 variants, skipping\n",
      "[INFO]\tGene ERV3-1-ZNF117 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT9 has 0 variants, skipping\n",
      "[INFO]\tGene AOC4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928985 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF471 has 0 variants, skipping\n",
      "[INFO]\tGene PYGM has 0 variants, skipping\n",
      "[INFO]\tGene MIR4720 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR7854 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6504 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129617 has 0 variants, skipping\n",
      "[INFO]\tGene IFT46 has 0 variants, skipping\n",
      "[INFO]\tGene MARCOL has 0 variants, skipping\n",
      "[INFO]\tGene RFC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8058 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928417 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724163 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GAL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO3A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC654780 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TARBP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02139 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5093 has 0 variants, skipping\n",
      "[INFO]\tGene PDE11A has 0 variants, skipping\n",
      "[INFO]\tGene SAMD15 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1910 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32E has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928557 has 0 variants, skipping\n",
      "[INFO]\tGene HEXIM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6774 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01081 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02135 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00917 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC4 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ30679 has 0 variants, skipping\n",
      "[INFO]\tGene LMF2 has 0 variants, skipping\n",
      "[INFO]\tGene RMND5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02189 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02188 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02181 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928708 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928682 has 0 variants, skipping\n",
      "[INFO]\tGene PELP1 has 0 variants, skipping\n",
      "[INFO]\tGene FCHO2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928659 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928737 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC88A has 0 variants, skipping\n",
      "[INFO]\tGene ATF7IP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724467 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400553 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128882 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCB has 0 variants, skipping\n",
      "[INFO]\tGene C1orf167-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI1 has 0 variants, skipping\n",
      "[INFO]\tGene RAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4722 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289580 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339059 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF8 has 0 variants, skipping\n",
      "[INFO]\tGene EYA3 has 0 variants, skipping\n",
      "[INFO]\tGene EFCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L2 has 0 variants, skipping\n",
      "[INFO]\tGene TCN1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGIS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927793 has 0 variants, skipping\n",
      "[INFO]\tGene FANCM has 0 variants, skipping\n",
      "[INFO]\tGene HACL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02138 has 0 variants, skipping\n",
      "[INFO]\tGene CISD3 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf185 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371414 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927817 has 0 variants, skipping\n",
      "[INFO]\tGene CSPP1 has 0 variants, skipping\n",
      "[INFO]\tGene SSU72 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM208 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD68 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT18 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM120A has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP7A has 0 variants, skipping\n",
      "[INFO]\tGene GBF1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS9D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF276 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT76 has 0 variants, skipping\n",
      "[INFO]\tGene TGFBRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene AFG3L1P has 0 variants, skipping\n",
      "[INFO]\tGene RPS4X has 0 variants, skipping\n",
      "[INFO]\tGene RPS3A has 0 variants, skipping\n",
      "[INFO]\tGene URAHP has 0 variants, skipping\n",
      "[INFO]\tGene FAM157C has 0 variants, skipping\n",
      "[INFO]\tGene RTTN has 0 variants, skipping\n",
      "[INFO]\tGene MPI has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P3 has 0 variants, skipping\n",
      "[INFO]\tGene SRPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM2 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SRL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC169 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A5 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM4 has 0 variants, skipping\n",
      "[INFO]\tGene GFUS has 0 variants, skipping\n",
      "[INFO]\tGene HPS3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAML1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1B has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B1 has 0 variants, skipping\n",
      "[INFO]\tGene HRH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK6 has 0 variants, skipping\n",
      "[INFO]\tGene PEG10 has 0 variants, skipping\n",
      "[INFO]\tGene ADCYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL2 has 0 variants, skipping\n",
      "[INFO]\tGene CHEK2 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA10 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT16H has 0 variants, skipping\n",
      "[INFO]\tGene ICE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RIOX1 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT12 has 0 variants, skipping\n",
      "[INFO]\tGene MANEA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF322 has 0 variants, skipping\n",
      "[INFO]\tGene YRDC has 0 variants, skipping\n",
      "[INFO]\tGene QTRT2 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP2P1 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD2 has 0 variants, skipping\n",
      "[INFO]\tGene PECAM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723971 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA17 has 0 variants, skipping\n",
      "[INFO]\tGene RP1 has 0 variants, skipping\n",
      "[INFO]\tGene GARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTU2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf49 has 0 variants, skipping\n",
      "[INFO]\tGene BUD31 has 0 variants, skipping\n",
      "[INFO]\tGene MMEL1 has 0 variants, skipping\n",
      "[INFO]\tGene NOSTRIN has 0 variants, skipping\n",
      "[INFO]\tGene UCN3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL53 has 0 variants, skipping\n",
      "[INFO]\tGene WDFY2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP33 has 0 variants, skipping\n",
      "[INFO]\tGene ALPK2 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene FAXDC2 has 0 variants, skipping\n",
      "[INFO]\tGene COG8 has 0 variants, skipping\n",
      "[INFO]\tGene CORIN has 0 variants, skipping\n",
      "[INFO]\tGene UCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02091 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371430 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM244 has 0 variants, skipping\n",
      "[INFO]\tGene EGR2 has 0 variants, skipping\n",
      "[INFO]\tGene GPD2 has 0 variants, skipping\n",
      "[INFO]\tGene DBIL5P has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A9 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPT4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927727 has 0 variants, skipping\n",
      "[INFO]\tGene IL4I1 has 0 variants, skipping\n",
      "[INFO]\tGene RAN has 0 variants, skipping\n",
      "[INFO]\tGene BATF2 has 0 variants, skipping\n",
      "[INFO]\tGene MDFI has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3183 has 0 variants, skipping\n",
      "[INFO]\tGene DOK5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF208 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC116 has 0 variants, skipping\n",
      "[INFO]\tGene TP63 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GZMH has 0 variants, skipping\n",
      "[INFO]\tGene NTAN1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371485 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927839 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD91B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD91A has 0 variants, skipping\n",
      "[INFO]\tGene LOC284009 has 0 variants, skipping\n",
      "[INFO]\tGene INSYN2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF497 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB45 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6776 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371592 has 0 variants, skipping\n",
      "[INFO]\tGene MTA1 has 0 variants, skipping\n",
      "[INFO]\tGene NMI has 0 variants, skipping\n",
      "[INFO]\tGene MTMR4 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927911 has 0 variants, skipping\n",
      "[INFO]\tGene NEK4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288728 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf3 has 0 variants, skipping\n",
      "[INFO]\tGene CPS1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR3A4P has 0 variants, skipping\n",
      "[INFO]\tGene MYZAP has 0 variants, skipping\n",
      "[INFO]\tGene OPA3 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB1 has 0 variants, skipping\n",
      "[INFO]\tGene OIP5 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX5-TAX1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPZOS has 0 variants, skipping\n",
      "[INFO]\tGene FAM189B has 0 variants, skipping\n",
      "[INFO]\tGene PKMYT1 has 0 variants, skipping\n",
      "[INFO]\tGene OTC has 0 variants, skipping\n",
      "[INFO]\tGene PRND has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1C has 0 variants, skipping\n",
      "[INFO]\tGene PICK1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01975 has 0 variants, skipping\n",
      "[INFO]\tGene AMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTC1 has 0 variants, skipping\n",
      "[INFO]\tGene ABL2 has 0 variants, skipping\n",
      "[INFO]\tGene ABL1 has 0 variants, skipping\n",
      "[INFO]\tGene CWH43 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A17 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24C has 0 variants, skipping\n",
      "[INFO]\tGene LGALS4 has 0 variants, skipping\n",
      "[INFO]\tGene IRF6 has 0 variants, skipping\n",
      "[INFO]\tGene GZMA has 0 variants, skipping\n",
      "[INFO]\tGene GPR19 has 0 variants, skipping\n",
      "[INFO]\tGene SFN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01996 has 0 variants, skipping\n",
      "[INFO]\tGene TYMP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101559451 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288069 has 0 variants, skipping\n",
      "[INFO]\tGene CLCC1 has 0 variants, skipping\n",
      "[INFO]\tGene BBS2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K6 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC10 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1B has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1D has 0 variants, skipping\n",
      "[INFO]\tGene RECQL has 0 variants, skipping\n",
      "[INFO]\tGene MEI4 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ2B has 0 variants, skipping\n",
      "[INFO]\tGene MS4A7 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2A1 has 0 variants, skipping\n",
      "[INFO]\tGene POFUT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6864 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6865 has 0 variants, skipping\n",
      "[INFO]\tGene PHOSPHO2 has 0 variants, skipping\n",
      "[INFO]\tGene PTK6 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA1 has 0 variants, skipping\n",
      "[INFO]\tGene NARS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130950 has 0 variants, skipping\n",
      "[INFO]\tGene CPSF4 has 0 variants, skipping\n",
      "[INFO]\tGene LHX6 has 0 variants, skipping\n",
      "[INFO]\tGene POPDC3 has 0 variants, skipping\n",
      "[INFO]\tGene NMBR has 0 variants, skipping\n",
      "[INFO]\tGene LOC339166 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC97 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4520-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4520-2 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15P1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP298 has 0 variants, skipping\n",
      "[INFO]\tGene SNX6 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12P2 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK38 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEK has 0 variants, skipping\n",
      "[INFO]\tGene RNASEK-C17orf49 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR497HG has 0 variants, skipping\n",
      "[INFO]\tGene NEFL has 0 variants, skipping\n",
      "[INFO]\tGene KDM3A has 0 variants, skipping\n",
      "[INFO]\tGene BRICD5 has 0 variants, skipping\n",
      "[INFO]\tGene GJE1 has 0 variants, skipping\n",
      "[INFO]\tGene SURF1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT6H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF711 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA6 has 0 variants, skipping\n",
      "[INFO]\tGene PTHLH has 0 variants, skipping\n",
      "[INFO]\tGene PTBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA4 has 0 variants, skipping\n",
      "[INFO]\tGene NPHS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTPAL has 0 variants, skipping\n",
      "[INFO]\tGene RANBP10 has 0 variants, skipping\n",
      "[INFO]\tGene REG1CP has 0 variants, skipping\n",
      "[INFO]\tGene SNHG6 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL3 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY2 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM256-PLSCR3 has 0 variants, skipping\n",
      "[INFO]\tGene THOC6 has 0 variants, skipping\n",
      "[INFO]\tGene IRX1 has 0 variants, skipping\n",
      "[INFO]\tGene IRX3 has 0 variants, skipping\n",
      "[INFO]\tGene IRX6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM43 has 0 variants, skipping\n",
      "[INFO]\tGene FSD1 has 0 variants, skipping\n",
      "[INFO]\tGene BRCC3 has 0 variants, skipping\n",
      "[INFO]\tGene SECTM1 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP7 has 0 variants, skipping\n",
      "[INFO]\tGene SENP3-EIF4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA48 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA67 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L2 has 0 variants, skipping\n",
      "[INFO]\tGene CA2 has 0 variants, skipping\n",
      "[INFO]\tGene GBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA16 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PATZ1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL2 has 0 variants, skipping\n",
      "[INFO]\tGene FGFBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL29P2 has 0 variants, skipping\n",
      "[INFO]\tGene KHDC3L has 0 variants, skipping\n",
      "[INFO]\tGene H2AB2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTBL2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB20 has 0 variants, skipping\n",
      "[INFO]\tGene BPNT2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2R2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM260 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF1 has 0 variants, skipping\n",
      "[INFO]\tGene OSGEP has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA21 has 0 variants, skipping\n",
      "[INFO]\tGene VPS54 has 0 variants, skipping\n",
      "[INFO]\tGene SLC43A3 has 0 variants, skipping\n",
      "[INFO]\tGene VIP has 0 variants, skipping\n",
      "[INFO]\tGene VIM has 0 variants, skipping\n",
      "[INFO]\tGene VGF has 0 variants, skipping\n",
      "[INFO]\tGene VEGFB has 0 variants, skipping\n",
      "[INFO]\tGene VDAC1 has 0 variants, skipping\n",
      "[INFO]\tGene VASP has 0 variants, skipping\n",
      "[INFO]\tGene UVRAG has 0 variants, skipping\n",
      "[INFO]\tGene USP1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4314 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6883 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4521 has 0 variants, skipping\n",
      "[INFO]\tGene HS1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene XPO4 has 0 variants, skipping\n",
      "[INFO]\tGene CASR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128288 has 0 variants, skipping\n",
      "[INFO]\tGene GZMB has 0 variants, skipping\n",
      "[INFO]\tGene GYG1 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF16 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN11 has 0 variants, skipping\n",
      "[INFO]\tGene RCSD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR150 has 0 variants, skipping\n",
      "[INFO]\tGene MIR149 has 0 variants, skipping\n",
      "[INFO]\tGene MIR146A has 0 variants, skipping\n",
      "[INFO]\tGene MIR9-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR138-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR191 has 0 variants, skipping\n",
      "[INFO]\tGene MIR153-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR152 has 0 variants, skipping\n",
      "[INFO]\tGene MIR145 has 0 variants, skipping\n",
      "[INFO]\tGene MIR144 has 0 variants, skipping\n",
      "[INFO]\tGene MIR143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928266 has 0 variants, skipping\n",
      "[INFO]\tGene YBX3 has 0 variants, skipping\n",
      "[INFO]\tGene CST7 has 0 variants, skipping\n",
      "[INFO]\tGene DGKD has 0 variants, skipping\n",
      "[INFO]\tGene DGKE has 0 variants, skipping\n",
      "[INFO]\tGene LRRC18 has 0 variants, skipping\n",
      "[INFO]\tGene OFD1 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1R1 has 0 variants, skipping\n",
      "[INFO]\tGene UBN1 has 0 variants, skipping\n",
      "[INFO]\tGene STAG1 has 0 variants, skipping\n",
      "[INFO]\tGene EFS has 0 variants, skipping\n",
      "[INFO]\tGene PRSS16 has 0 variants, skipping\n",
      "[INFO]\tGene BOP1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC6B has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC9 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D12 has 0 variants, skipping\n",
      "[INFO]\tGene MAGOH2P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM220-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42SE2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL1 has 0 variants, skipping\n",
      "[INFO]\tGene DTWD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRM has 0 variants, skipping\n",
      "[INFO]\tGene PSMG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00670 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128006 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1269B has 0 variants, skipping\n",
      "[INFO]\tGene TROAP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02093 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H3 has 0 variants, skipping\n",
      "[INFO]\tGene COX10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGC12916 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928475 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT7 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT8 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4731 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM256 has 0 variants, skipping\n",
      "[INFO]\tGene VXN has 0 variants, skipping\n",
      "[INFO]\tGene AKNAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TSGA10IP has 0 variants, skipping\n",
      "[INFO]\tGene CXorf58 has 0 variants, skipping\n",
      "[INFO]\tGene SNX32 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC43 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02087 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD163 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF165 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD49B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD49A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene USP32P1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D27P has 0 variants, skipping\n",
      "[INFO]\tGene SHLD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284191 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02090 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG11A has 0 variants, skipping\n",
      "[INFO]\tGene DEFB103A has 0 variants, skipping\n",
      "[INFO]\tGene C11orf24 has 0 variants, skipping\n",
      "[INFO]\tGene CCL22 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL18 has 0 variants, skipping\n",
      "[INFO]\tGene CCL17 has 0 variants, skipping\n",
      "[INFO]\tGene SNAPC4 has 0 variants, skipping\n",
      "[INFO]\tGene OR6A2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6D has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A12 has 0 variants, skipping\n",
      "[INFO]\tGene SMCR2 has 0 variants, skipping\n",
      "[INFO]\tGene RAI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6777 has 0 variants, skipping\n",
      "[INFO]\tGene DNTT has 0 variants, skipping\n",
      "[INFO]\tGene PRAC2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF32 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF121 has 0 variants, skipping\n",
      "[INFO]\tGene REPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene BZW2 has 0 variants, skipping\n",
      "[INFO]\tGene EGFL7 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02076 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P5 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17P2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P1 has 0 variants, skipping\n",
      "[INFO]\tGene USP32P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144B has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPA has 0 variants, skipping\n",
      "[INFO]\tGene CDH7 has 0 variants, skipping\n",
      "[INFO]\tGene CDC34 has 0 variants, skipping\n",
      "[INFO]\tGene CD81 has 0 variants, skipping\n",
      "[INFO]\tGene CD74 has 0 variants, skipping\n",
      "[INFO]\tGene CCNG2 has 0 variants, skipping\n",
      "[INFO]\tGene CBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX3 has 0 variants, skipping\n",
      "[INFO]\tGene CASP3 has 0 variants, skipping\n",
      "[INFO]\tGene CAMP has 0 variants, skipping\n",
      "[INFO]\tGene CETN2 has 0 variants, skipping\n",
      "[INFO]\tGene CALR has 0 variants, skipping\n",
      "[INFO]\tGene TDO2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3B-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3B-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC79999 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388436 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3C has 0 variants, skipping\n",
      "[INFO]\tGene EPN2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene EPN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PARK7 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACSS1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX2B has 0 variants, skipping\n",
      "[INFO]\tGene FSTL1 has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L6P has 0 variants, skipping\n",
      "[INFO]\tGene SIRT6 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF6 has 0 variants, skipping\n",
      "[INFO]\tGene MGME1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144CP has 0 variants, skipping\n",
      "[INFO]\tGene USP32P3 has 0 variants, skipping\n",
      "[INFO]\tGene PHGR1 has 0 variants, skipping\n",
      "[INFO]\tGene NOS2P3 has 0 variants, skipping\n",
      "[INFO]\tGene ARL4A has 0 variants, skipping\n",
      "[INFO]\tGene KRT16P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02088 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287072 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01563 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGB has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G5C has 0 variants, skipping\n",
      "[INFO]\tGene UBBP4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E5 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ36000 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371703 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4522 has 0 variants, skipping\n",
      "[INFO]\tGene NRTN has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P5 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28B has 0 variants, skipping\n",
      "[INFO]\tGene LEKR1 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMTC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01992 has 0 variants, skipping\n",
      "[INFO]\tGene PYY2 has 0 variants, skipping\n",
      "[INFO]\tGene PPY2P has 0 variants, skipping\n",
      "[INFO]\tGene KRT18P55 has 0 variants, skipping\n",
      "[INFO]\tGene OR6Q1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4723 has 0 variants, skipping\n",
      "[INFO]\tGene SGK2 has 0 variants, skipping\n",
      "[INFO]\tGene COL12A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL45 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4X1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI44L has 0 variants, skipping\n",
      "[INFO]\tGene MMAA has 0 variants, skipping\n",
      "[INFO]\tGene MEF2D has 0 variants, skipping\n",
      "[INFO]\tGene TGM3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC4 has 0 variants, skipping\n",
      "[INFO]\tGene SYNC has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL3 has 0 variants, skipping\n",
      "[INFO]\tGene MC1R has 0 variants, skipping\n",
      "[INFO]\tGene MATN3 has 0 variants, skipping\n",
      "[INFO]\tGene MATN1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP1A has 0 variants, skipping\n",
      "[INFO]\tGene MAN2A1 has 0 variants, skipping\n",
      "[INFO]\tGene HADH has 0 variants, skipping\n",
      "[INFO]\tGene GYG2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD42B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD4A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD42A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD4B has 0 variants, skipping\n",
      "[INFO]\tGene ALPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR451B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4732 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927018 has 0 variants, skipping\n",
      "[INFO]\tGene E2F5 has 0 variants, skipping\n",
      "[INFO]\tGene E2F4 has 0 variants, skipping\n",
      "[INFO]\tGene GATA1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F29 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B21 has 0 variants, skipping\n",
      "[INFO]\tGene CDHR4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4523 has 0 variants, skipping\n",
      "[INFO]\tGene ALDOA has 0 variants, skipping\n",
      "[INFO]\tGene AGL has 0 variants, skipping\n",
      "[INFO]\tGene TSPY1 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF2P1-LRRC37BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1P2 has 0 variants, skipping\n",
      "[INFO]\tGene SUZ12P1 has 0 variants, skipping\n",
      "[INFO]\tGene RRM1 has 0 variants, skipping\n",
      "[INFO]\tGene DPRXP4 has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984974 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646030 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4733 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC166 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10B has 0 variants, skipping\n",
      "[INFO]\tGene FMNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4724 has 0 variants, skipping\n",
      "[INFO]\tGene SYDE1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3GL1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371730 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf67 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC8 has 0 variants, skipping\n",
      "[INFO]\tGene ARGFXP2 has 0 variants, skipping\n",
      "[INFO]\tGene MYL6B has 0 variants, skipping\n",
      "[INFO]\tGene MYH11 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS11 has 0 variants, skipping\n",
      "[INFO]\tGene DNLZ has 0 variants, skipping\n",
      "[INFO]\tGene TTC28 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC3 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR8 has 0 variants, skipping\n",
      "[INFO]\tGene PHGDH has 0 variants, skipping\n",
      "[INFO]\tGene PLK2 has 0 variants, skipping\n",
      "[INFO]\tGene AA06 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01989 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF668 has 0 variants, skipping\n",
      "[INFO]\tGene AGRN has 0 variants, skipping\n",
      "[INFO]\tGene SAMD12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51L3-RFFL has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985033 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD7 has 0 variants, skipping\n",
      "[INFO]\tGene CPA5 has 0 variants, skipping\n",
      "[INFO]\tGene RNF41 has 0 variants, skipping\n",
      "[INFO]\tGene MPRIP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A8P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF534 has 0 variants, skipping\n",
      "[INFO]\tGene CCL15-CCL14 has 0 variants, skipping\n",
      "[INFO]\tGene PBX2 has 0 variants, skipping\n",
      "[INFO]\tGene PBX1 has 0 variants, skipping\n",
      "[INFO]\tGene OCLN has 0 variants, skipping\n",
      "[INFO]\tGene CCL4L1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A4 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC4 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM47 has 0 variants, skipping\n",
      "[INFO]\tGene CERS6 has 0 variants, skipping\n",
      "[INFO]\tGene RPS5 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6 has 0 variants, skipping\n",
      "[INFO]\tGene NPY2R has 0 variants, skipping\n",
      "[INFO]\tGene RRP9 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A6 has 0 variants, skipping\n",
      "[INFO]\tGene KRT75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR943 has 0 variants, skipping\n",
      "[INFO]\tGene MIR940 has 0 variants, skipping\n",
      "[INFO]\tGene LHX1-DT has 0 variants, skipping\n",
      "[INFO]\tGene STK24 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3C has 0 variants, skipping\n",
      "[INFO]\tGene FAT4 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2S1 has 0 variants, skipping\n",
      "[INFO]\tGene PTK2B has 0 variants, skipping\n",
      "[INFO]\tGene EFNB2 has 0 variants, skipping\n",
      "[INFO]\tGene ARGLU1 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT61B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA90 has 0 variants, skipping\n",
      "[INFO]\tGene EPM2A has 0 variants, skipping\n",
      "[INFO]\tGene RNF103 has 0 variants, skipping\n",
      "[INFO]\tGene CFHR2 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAB1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA9 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL33 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA7 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3B has 0 variants, skipping\n",
      "[INFO]\tGene SPAG7 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL14 has 0 variants, skipping\n",
      "[INFO]\tGene APBA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378J has 0 variants, skipping\n",
      "[INFO]\tGene YWHAEP7 has 0 variants, skipping\n",
      "[INFO]\tGene SPAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF284 has 0 variants, skipping\n",
      "[INFO]\tGene ANLN has 0 variants, skipping\n",
      "[INFO]\tGene CBLL1 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5J has 0 variants, skipping\n",
      "[INFO]\tGene STAT5A has 0 variants, skipping\n",
      "[INFO]\tGene SRPRA has 0 variants, skipping\n",
      "[INFO]\tGene OPN5 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371763 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4734 has 0 variants, skipping\n",
      "[INFO]\tGene WDR75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4726 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287808 has 0 variants, skipping\n",
      "[INFO]\tGene DTD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4727 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA21B has 0 variants, skipping\n",
      "[INFO]\tGene GRM3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6779 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00672 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02079 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A11P has 0 variants, skipping\n",
      "[INFO]\tGene RDM1P5 has 0 variants, skipping\n",
      "[INFO]\tGene MTX1 has 0 variants, skipping\n",
      "[INFO]\tGene STAT5B has 0 variants, skipping\n",
      "[INFO]\tGene CORO6 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A5 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3E has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L2 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT11 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC61 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4728 has 0 variants, skipping\n",
      "[INFO]\tGene ST13 has 0 variants, skipping\n",
      "[INFO]\tGene SKAP2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL39 has 0 variants, skipping\n",
      "[INFO]\tGene H2BS1 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CROT has 0 variants, skipping\n",
      "[INFO]\tGene GPR142 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K4 has 0 variants, skipping\n",
      "[INFO]\tGene AMBP has 0 variants, skipping\n",
      "[INFO]\tGene ALPP has 0 variants, skipping\n",
      "[INFO]\tGene HERC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6884 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD124 has 0 variants, skipping\n",
      "[INFO]\tGene VCAN has 0 variants, skipping\n",
      "[INFO]\tGene LMF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6866 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF716 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6867 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE1 has 0 variants, skipping\n",
      "[INFO]\tGene RARA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RDX has 0 variants, skipping\n",
      "[INFO]\tGene CTCF has 0 variants, skipping\n",
      "[INFO]\tGene KLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPA1 has 0 variants, skipping\n",
      "[INFO]\tGene CHUK has 0 variants, skipping\n",
      "[INFO]\tGene PRKCI has 0 variants, skipping\n",
      "[INFO]\tGene HSPA4 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505782 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6510 has 0 variants, skipping\n",
      "[INFO]\tGene UPF3B has 0 variants, skipping\n",
      "[INFO]\tGene KRT42P has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQB1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A14 has 0 variants, skipping\n",
      "[INFO]\tGene IMMP2L has 0 variants, skipping\n",
      "[INFO]\tGene ARNTL2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C3B has 0 variants, skipping\n",
      "[INFO]\tGene SPNS3 has 0 variants, skipping\n",
      "[INFO]\tGene ANK1 has 0 variants, skipping\n",
      "[INFO]\tGene YIF1A has 0 variants, skipping\n",
      "[INFO]\tGene ABO has 0 variants, skipping\n",
      "[INFO]\tGene SMIM7 has 0 variants, skipping\n",
      "[INFO]\tGene GPR146 has 0 variants, skipping\n",
      "[INFO]\tGene MAP11 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL2 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML3 has 0 variants, skipping\n",
      "[INFO]\tGene CDR2L has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R4 has 0 variants, skipping\n",
      "[INFO]\tGene MTRR has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5010 has 0 variants, skipping\n",
      "[INFO]\tGene EVI5L has 0 variants, skipping\n",
      "[INFO]\tGene MRAP has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AD has 0 variants, skipping\n",
      "[INFO]\tGene IL10RA has 0 variants, skipping\n",
      "[INFO]\tGene ADCY9 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY8 has 0 variants, skipping\n",
      "[INFO]\tGene GPHA2 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3C has 0 variants, skipping\n",
      "[INFO]\tGene ACOT12 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5K has 0 variants, skipping\n",
      "[INFO]\tGene MADCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6780A has 0 variants, skipping\n",
      "[INFO]\tGene RAMP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IDI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6781 has 0 variants, skipping\n",
      "[INFO]\tGene MPND has 0 variants, skipping\n",
      "[INFO]\tGene MIR1261 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00671 has 0 variants, skipping\n",
      "[INFO]\tGene DDX41 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES3L-AARSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNTN has 0 variants, skipping\n",
      "[INFO]\tGene NOMO1 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG2 has 0 variants, skipping\n",
      "[INFO]\tGene CRY1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929767 has 0 variants, skipping\n",
      "[INFO]\tGene DTX4 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA4 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC14 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC7 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC11 has 0 variants, skipping\n",
      "[INFO]\tGene EPG5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF146 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC200 has 0 variants, skipping\n",
      "[INFO]\tGene RNU2-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00910 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378A has 0 variants, skipping\n",
      "[INFO]\tGene MIR2117 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO4 has 0 variants, skipping\n",
      "[INFO]\tGene METRNL has 0 variants, skipping\n",
      "[INFO]\tGene FAM76B has 0 variants, skipping\n",
      "[INFO]\tGene C10orf82 has 0 variants, skipping\n",
      "[INFO]\tGene TCP10L has 0 variants, skipping\n",
      "[INFO]\tGene SAMD8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00309 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5D has 0 variants, skipping\n",
      "[INFO]\tGene WFDC12 has 0 variants, skipping\n",
      "[INFO]\tGene RDH14 has 0 variants, skipping\n",
      "[INFO]\tGene HAL has 0 variants, skipping\n",
      "[INFO]\tGene BRSK1 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8 has 0 variants, skipping\n",
      "[INFO]\tGene TOM1L2 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF85 has 0 variants, skipping\n",
      "[INFO]\tGene FAM215A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01976 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf64 has 0 variants, skipping\n",
      "[INFO]\tGene LACRT has 0 variants, skipping\n",
      "[INFO]\tGene ANKS4B has 0 variants, skipping\n",
      "[INFO]\tGene ASB3 has 0 variants, skipping\n",
      "[INFO]\tGene EDARADD has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371789 has 0 variants, skipping\n",
      "[INFO]\tGene ZFX has 0 variants, skipping\n",
      "[INFO]\tGene RPIA has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM217 has 0 variants, skipping\n",
      "[INFO]\tGene WDCP has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6782 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC3A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAF11 has 0 variants, skipping\n",
      "[INFO]\tGene EBP has 0 variants, skipping\n",
      "[INFO]\tGene GNB5 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT2 has 0 variants, skipping\n",
      "[INFO]\tGene AVIL has 0 variants, skipping\n",
      "[INFO]\tGene NFRKB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01180 has 0 variants, skipping\n",
      "[INFO]\tGene COASY has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4C has 0 variants, skipping\n",
      "[INFO]\tGene SINHCAF has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD1 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP2 has 0 variants, skipping\n",
      "[INFO]\tGene NFKB1 has 0 variants, skipping\n",
      "[INFO]\tGene EMX2 has 0 variants, skipping\n",
      "[INFO]\tGene FCGRT has 0 variants, skipping\n",
      "[INFO]\tGene DOT1L has 0 variants, skipping\n",
      "[INFO]\tGene MRPL36 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6784 has 0 variants, skipping\n",
      "[INFO]\tGene DPYD has 0 variants, skipping\n",
      "[INFO]\tGene NACC1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS26B has 0 variants, skipping\n",
      "[INFO]\tGene STX1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371795 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339192 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4315-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4315-2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A4P has 0 variants, skipping\n",
      "[INFO]\tGene ARL17A has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02210 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02210-CRHR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLTP has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPT-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene AQP2 has 0 variants, skipping\n",
      "[INFO]\tGene APRT has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF236 has 0 variants, skipping\n",
      "[INFO]\tGene NSFP1 has 0 variants, skipping\n",
      "[INFO]\tGene WRAP53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF407 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf91 has 0 variants, skipping\n",
      "[INFO]\tGene ASB15 has 0 variants, skipping\n",
      "[INFO]\tGene DCBLD2 has 0 variants, skipping\n",
      "[INFO]\tGene SETSIP has 0 variants, skipping\n",
      "[INFO]\tGene MIR5089 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B4 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC65 has 0 variants, skipping\n",
      "[INFO]\tGene THCAT158 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAE has 0 variants, skipping\n",
      "[INFO]\tGene MRPL45P2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2U1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724532 has 0 variants, skipping\n",
      "[INFO]\tGene SP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEDD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA20 has 0 variants, skipping\n",
      "[INFO]\tGene PEX12 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927166 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG7 has 0 variants, skipping\n",
      "[INFO]\tGene NLE1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3185 has 0 variants, skipping\n",
      "[INFO]\tGene EXT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371814 has 0 variants, skipping\n",
      "[INFO]\tGene AP1G1 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ40194 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6129 has 0 variants, skipping\n",
      "[INFO]\tGene ICOSLG has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724596 has 0 variants, skipping\n",
      "[INFO]\tGene BCL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02075 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288866 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6165 has 0 variants, skipping\n",
      "[INFO]\tGene RIC1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC11A has 0 variants, skipping\n",
      "[INFO]\tGene PICART1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3 has 0 variants, skipping\n",
      "[INFO]\tGene POLD3 has 0 variants, skipping\n",
      "[INFO]\tGene MTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01969 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM92-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND1B has 0 variants, skipping\n",
      "[INFO]\tGene FOSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20E has 0 variants, skipping\n",
      "[INFO]\tGene PRR20D has 0 variants, skipping\n",
      "[INFO]\tGene FN3KRP has 0 variants, skipping\n",
      "[INFO]\tGene NUP43 has 0 variants, skipping\n",
      "[INFO]\tGene TTC9C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371824 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT15 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM19 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY2 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf49 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP3 has 0 variants, skipping\n",
      "[INFO]\tGene SOD2 has 0 variants, skipping\n",
      "[INFO]\tGene DZANK1 has 0 variants, skipping\n",
      "[INFO]\tGene SNX2 has 0 variants, skipping\n",
      "[INFO]\tGene NPY4R2 has 0 variants, skipping\n",
      "[INFO]\tGene LTK has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB4 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB3 has 0 variants, skipping\n",
      "[INFO]\tGene MAL has 0 variants, skipping\n",
      "[INFO]\tGene MAGOH has 0 variants, skipping\n",
      "[INFO]\tGene IARS2 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf116 has 0 variants, skipping\n",
      "[INFO]\tGene RAVER1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL16 has 0 variants, skipping\n",
      "[INFO]\tGene SCN2B has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA3 has 0 variants, skipping\n",
      "[INFO]\tGene PTN has 0 variants, skipping\n",
      "[INFO]\tGene FBLN1 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS8 has 0 variants, skipping\n",
      "[INFO]\tGene CRISP3 has 0 variants, skipping\n",
      "[INFO]\tGene LAMC3 has 0 variants, skipping\n",
      "[INFO]\tGene RPS27L has 0 variants, skipping\n",
      "[INFO]\tGene DPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1I1 has 0 variants, skipping\n",
      "[INFO]\tGene TRA2A has 0 variants, skipping\n",
      "[INFO]\tGene ENOPH1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8059 has 0 variants, skipping\n",
      "[INFO]\tGene TOB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TP53INP2 has 0 variants, skipping\n",
      "[INFO]\tGene RHCE has 0 variants, skipping\n",
      "[INFO]\tGene ARMC3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02071 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02072 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02073 has 0 variants, skipping\n",
      "[INFO]\tGene GJD4 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01982 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02089 has 0 variants, skipping\n",
      "[INFO]\tGene LRATD2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM167A has 0 variants, skipping\n",
      "[INFO]\tGene TCTE3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO36 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC50 has 0 variants, skipping\n",
      "[INFO]\tGene LYG1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R2 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98C has 0 variants, skipping\n",
      "[INFO]\tGene RNF151 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD3 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LB has 0 variants, skipping\n",
      "[INFO]\tGene CLBA1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM9C has 0 variants, skipping\n",
      "[INFO]\tGene ZFP42 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MTVR2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF126P1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1B has 0 variants, skipping\n",
      "[INFO]\tGene BCAS2 has 0 variants, skipping\n",
      "[INFO]\tGene SMNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927557 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927539 has 0 variants, skipping\n",
      "[INFO]\tGene GGT6 has 0 variants, skipping\n",
      "[INFO]\tGene MECP2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927666 has 0 variants, skipping\n",
      "[INFO]\tGene MSX2P1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT14 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ11 has 0 variants, skipping\n",
      "[INFO]\tGene TSPOAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4736 has 0 variants, skipping\n",
      "[INFO]\tGene DES has 0 variants, skipping\n",
      "[INFO]\tGene CENPF has 0 variants, skipping\n",
      "[INFO]\tGene MAGEL2 has 0 variants, skipping\n",
      "[INFO]\tGene FMN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3680-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3670-1 has 0 variants, skipping\n",
      "[INFO]\tGene PBX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D3 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5RAP2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A3 has 0 variants, skipping\n",
      "[INFO]\tGene ABAT has 0 variants, skipping\n",
      "[INFO]\tGene PPM1E has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A5 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK7 has 0 variants, skipping\n",
      "[INFO]\tGene NEMF has 0 variants, skipping\n",
      "[INFO]\tGene STOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH5 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4729 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01476 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF574 has 0 variants, skipping\n",
      "[INFO]\tGene H3-3A has 0 variants, skipping\n",
      "[INFO]\tGene TRIR has 0 variants, skipping\n",
      "[INFO]\tGene KAT7 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P1-DHX40P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNFT1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4737 has 0 variants, skipping\n",
      "[INFO]\tGene ORM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371849 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653653 has 0 variants, skipping\n",
      "[INFO]\tGene CASP4LP has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA20 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01999 has 0 variants, skipping\n",
      "[INFO]\tGene LRATD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM209 has 0 variants, skipping\n",
      "[INFO]\tGene MIR216A has 0 variants, skipping\n",
      "[INFO]\tGene MIR217 has 0 variants, skipping\n",
      "[INFO]\tGene MIR215 has 0 variants, skipping\n",
      "[INFO]\tGene MIR214 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927855 has 0 variants, skipping\n",
      "[INFO]\tGene TBX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3P2 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF676 has 0 variants, skipping\n",
      "[INFO]\tGene ATF6B has 0 variants, skipping\n",
      "[INFO]\tGene TBL1X has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A4 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK1 has 0 variants, skipping\n",
      "[INFO]\tGene PKIB has 0 variants, skipping\n",
      "[INFO]\tGene UHRF2 has 0 variants, skipping\n",
      "[INFO]\tGene PRLHR has 0 variants, skipping\n",
      "[INFO]\tGene GTF2F2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729683 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A31 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC70 has 0 variants, skipping\n",
      "[INFO]\tGene MXD3 has 0 variants, skipping\n",
      "[INFO]\tGene INO80B has 0 variants, skipping\n",
      "[INFO]\tGene SF3B5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL3 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L1 has 0 variants, skipping\n",
      "[INFO]\tGene FCRL4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS15 has 0 variants, skipping\n",
      "[INFO]\tGene RNF17 has 0 variants, skipping\n",
      "[INFO]\tGene TCAM1P has 0 variants, skipping\n",
      "[INFO]\tGene SUOX has 0 variants, skipping\n",
      "[INFO]\tGene TAC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR29-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL11 has 0 variants, skipping\n",
      "[INFO]\tGene SEC16A has 0 variants, skipping\n",
      "[INFO]\tGene SERTM1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG25 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD104 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50C has 0 variants, skipping\n",
      "[INFO]\tGene PPDPF has 0 variants, skipping\n",
      "[INFO]\tGene TNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene TGFA has 0 variants, skipping\n",
      "[INFO]\tGene PCNX1 has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5047 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6080 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM1P1 has 0 variants, skipping\n",
      "[INFO]\tGene AMZ2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507002 has 0 variants, skipping\n",
      "[INFO]\tGene TP53I13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02563 has 0 variants, skipping\n",
      "[INFO]\tGene NALCN has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA7 has 0 variants, skipping\n",
      "[INFO]\tGene EIF5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF6 has 0 variants, skipping\n",
      "[INFO]\tGene CCL19 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC7 has 0 variants, skipping\n",
      "[INFO]\tGene CCL7 has 0 variants, skipping\n",
      "[INFO]\tGene S100B has 0 variants, skipping\n",
      "[INFO]\tGene S100A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928021 has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AA2 has 0 variants, skipping\n",
      "[INFO]\tGene GORASP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA38B has 0 variants, skipping\n",
      "[INFO]\tGene PSG6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00674 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378948 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CSDE1 has 0 variants, skipping\n",
      "[INFO]\tGene UBL5 has 0 variants, skipping\n",
      "[INFO]\tGene SCYL1 has 0 variants, skipping\n",
      "[INFO]\tGene CEMP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO3B has 0 variants, skipping\n",
      "[INFO]\tGene LY6E has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2A has 0 variants, skipping\n",
      "[INFO]\tGene UGT3A1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT3A2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD60 has 0 variants, skipping\n",
      "[INFO]\tGene MIR635 has 0 variants, skipping\n",
      "[INFO]\tGene FEV has 0 variants, skipping\n",
      "[INFO]\tGene LINC01482 has 0 variants, skipping\n",
      "[INFO]\tGene SLA has 0 variants, skipping\n",
      "[INFO]\tGene MON1A has 0 variants, skipping\n",
      "[INFO]\tGene MAN2C1 has 0 variants, skipping\n",
      "[INFO]\tGene HEXA has 0 variants, skipping\n",
      "[INFO]\tGene ABCA9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4524A has 0 variants, skipping\n",
      "[INFO]\tGene PRO1804 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM131L has 0 variants, skipping\n",
      "[INFO]\tGene KCTD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371880 has 0 variants, skipping\n",
      "[INFO]\tGene SCHLAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01497 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01028 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CASC17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01152 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723517 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4530 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02097 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02003 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01128 has 0 variants, skipping\n",
      "[INFO]\tGene HOTAIR has 0 variants, skipping\n",
      "[INFO]\tGene RGPD3 has 0 variants, skipping\n",
      "[INFO]\tGene CARNS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00469 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02092 has 0 variants, skipping\n",
      "[INFO]\tGene MYH14 has 0 variants, skipping\n",
      "[INFO]\tGene MGC16275 has 0 variants, skipping\n",
      "[INFO]\tGene PERM1 has 0 variants, skipping\n",
      "[INFO]\tGene LYST has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130520 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928343 has 0 variants, skipping\n",
      "[INFO]\tGene TBCEL has 0 variants, skipping\n",
      "[INFO]\tGene MSX2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3615 has 0 variants, skipping\n",
      "[INFO]\tGene GRXCR2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM163B has 0 variants, skipping\n",
      "[INFO]\tGene SCX has 0 variants, skipping\n",
      "[INFO]\tGene CPN2 has 0 variants, skipping\n",
      "[INFO]\tGene BICC1 has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1C has 0 variants, skipping\n",
      "[INFO]\tGene NPY1R has 0 variants, skipping\n",
      "[INFO]\tGene HID1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL5 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP7 has 0 variants, skipping\n",
      "[INFO]\tGene QRSL1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM8 has 0 variants, skipping\n",
      "[INFO]\tGene GRAP has 0 variants, skipping\n",
      "[INFO]\tGene KLHL26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287042 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A4 has 0 variants, skipping\n",
      "[INFO]\tGene LCLAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP214 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6785 has 0 variants, skipping\n",
      "[INFO]\tGene AK8 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3D has 0 variants, skipping\n",
      "[INFO]\tGene NBPF7 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene LFNG has 0 variants, skipping\n",
      "[INFO]\tGene LAMC1 has 0 variants, skipping\n",
      "[INFO]\tGene LECT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4738 has 0 variants, skipping\n",
      "[INFO]\tGene MCAT has 0 variants, skipping\n",
      "[INFO]\tGene TEN1-CDK3 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC8 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA5 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1B2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF157-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF587B has 0 variants, skipping\n",
      "[INFO]\tGene MS4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF12 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370579 has 0 variants, skipping\n",
      "[INFO]\tGene BRPF1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD1A has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105274304 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4463 has 0 variants, skipping\n",
      "[INFO]\tGene MIR636 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1A has 0 variants, skipping\n",
      "[INFO]\tGene CERS3 has 0 variants, skipping\n",
      "[INFO]\tGene CHAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00868 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371899 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA16 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6516 has 0 variants, skipping\n",
      "[INFO]\tGene RNF216 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400622 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507351 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01987 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01973 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B8 has 0 variants, skipping\n",
      "[INFO]\tGene AFMID has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01993 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928674 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA21 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN20 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A27 has 0 variants, skipping\n",
      "[INFO]\tGene CHURC1 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD3-BMI1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02078 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01977 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01979 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01978 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF807P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1268B has 0 variants, skipping\n",
      "[INFO]\tGene PRY2 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF1 has 0 variants, skipping\n",
      "[INFO]\tGene DTNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGT1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4730 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3E has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928855 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400627 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1250 has 0 variants, skipping\n",
      "[INFO]\tGene GK has 0 variants, skipping\n",
      "[INFO]\tGene CAND2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF155 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371925 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG16-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00482 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN26 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4740 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3186 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2K has 0 variants, skipping\n",
      "[INFO]\tGene OR10K1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10R2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR95 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370295 has 0 variants, skipping\n",
      "[INFO]\tGene EBI3 has 0 variants, skipping\n",
      "[INFO]\tGene G3BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SBSN has 0 variants, skipping\n",
      "[INFO]\tGene RGS9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6786 has 0 variants, skipping\n",
      "[INFO]\tGene D2HGDH has 0 variants, skipping\n",
      "[INFO]\tGene TMEM168 has 0 variants, skipping\n",
      "[INFO]\tGene NOP14 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN1 has 0 variants, skipping\n",
      "[INFO]\tGene TOR2A has 0 variants, skipping\n",
      "[INFO]\tGene KLHL6 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA6 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF8 has 0 variants, skipping\n",
      "[INFO]\tGene VHL has 0 variants, skipping\n",
      "[INFO]\tGene AFP has 0 variants, skipping\n",
      "[INFO]\tGene SNORD134 has 0 variants, skipping\n",
      "[INFO]\tGene GLP1R has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF2 has 0 variants, skipping\n",
      "[INFO]\tGene TRH has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6787 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01970 has 0 variants, skipping\n",
      "[INFO]\tGene PEX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TFDP1 has 0 variants, skipping\n",
      "[INFO]\tGene TEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4525 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929552 has 0 variants, skipping\n",
      "[INFO]\tGene PSG9 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP87 has 0 variants, skipping\n",
      "[INFO]\tGene WNT4 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR5 has 0 variants, skipping\n",
      "[INFO]\tGene MAEA has 0 variants, skipping\n",
      "[INFO]\tGene GCH1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT6 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR2 has 0 variants, skipping\n",
      "[INFO]\tGene NKX1-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730100 has 0 variants, skipping\n",
      "[INFO]\tGene IGFL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SVIP has 0 variants, skipping\n",
      "[INFO]\tGene MESP1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS33B has 0 variants, skipping\n",
      "[INFO]\tGene SMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene SAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFE3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC39 has 0 variants, skipping\n",
      "[INFO]\tGene ARL16 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1L5 has 0 variants, skipping\n",
      "[INFO]\tGene WAC has 0 variants, skipping\n",
      "[INFO]\tGene DNAH11 has 0 variants, skipping\n",
      "[INFO]\tGene OR6N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C76 has 0 variants, skipping\n",
      "[INFO]\tGene RPRD1A has 0 variants, skipping\n",
      "[INFO]\tGene CUL1 has 0 variants, skipping\n",
      "[INFO]\tGene PCCA-DT has 0 variants, skipping\n",
      "[INFO]\tGene EREG has 0 variants, skipping\n",
      "[INFO]\tGene MSX1 has 0 variants, skipping\n",
      "[INFO]\tGene NMUR1 has 0 variants, skipping\n",
      "[INFO]\tGene MTF1 has 0 variants, skipping\n",
      "[INFO]\tGene MT3 has 0 variants, skipping\n",
      "[INFO]\tGene DEPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM22P has 0 variants, skipping\n",
      "[INFO]\tGene LTV1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD13 has 0 variants, skipping\n",
      "[INFO]\tGene MAEL has 0 variants, skipping\n",
      "[INFO]\tGene LACTB has 0 variants, skipping\n",
      "[INFO]\tGene WDR73 has 0 variants, skipping\n",
      "[INFO]\tGene WDR41 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP62 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF268 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf56 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH4A1 has 0 variants, skipping\n",
      "[INFO]\tGene RCOR3 has 0 variants, skipping\n",
      "[INFO]\tGene SULT2A1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP1 has 0 variants, skipping\n",
      "[INFO]\tGene STX5 has 0 variants, skipping\n",
      "[INFO]\tGene STRN has 0 variants, skipping\n",
      "[INFO]\tGene UBOX5 has 0 variants, skipping\n",
      "[INFO]\tGene COL2A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM186 has 0 variants, skipping\n",
      "[INFO]\tGene MXRA5 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF13 has 0 variants, skipping\n",
      "[INFO]\tGene SPEF1 has 0 variants, skipping\n",
      "[INFO]\tGene MPC2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36 has 0 variants, skipping\n",
      "[INFO]\tGene LILRA5 has 0 variants, skipping\n",
      "[INFO]\tGene ETDC has 0 variants, skipping\n",
      "[INFO]\tGene LRRC27 has 0 variants, skipping\n",
      "[INFO]\tGene FBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene ZGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene NCK1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf42 has 0 variants, skipping\n",
      "[INFO]\tGene TSTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A3 has 0 variants, skipping\n",
      "[INFO]\tGene PHB has 0 variants, skipping\n",
      "[INFO]\tGene PGF has 0 variants, skipping\n",
      "[INFO]\tGene PGD has 0 variants, skipping\n",
      "[INFO]\tGene PGAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PFN2 has 0 variants, skipping\n",
      "[INFO]\tGene TUFM has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985728 has 0 variants, skipping\n",
      "[INFO]\tGene EOGT has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E3 has 0 variants, skipping\n",
      "[INFO]\tGene CAAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2A has 0 variants, skipping\n",
      "[INFO]\tGene RSRC2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L9P has 0 variants, skipping\n",
      "[INFO]\tGene RNF43 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPC has 0 variants, skipping\n",
      "[INFO]\tGene PRELP has 0 variants, skipping\n",
      "[INFO]\tGene HMGN5 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE41 has 0 variants, skipping\n",
      "[INFO]\tGene SKIL has 0 variants, skipping\n",
      "[INFO]\tGene LOC112577516 has 0 variants, skipping\n",
      "[INFO]\tGene RYR2 has 0 variants, skipping\n",
      "[INFO]\tGene STX10 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L4 has 0 variants, skipping\n",
      "[INFO]\tGene NLRX1 has 0 variants, skipping\n",
      "[INFO]\tGene GYPE has 0 variants, skipping\n",
      "[INFO]\tGene ALG14 has 0 variants, skipping\n",
      "[INFO]\tGene MKRN2 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRC1 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN7 has 0 variants, skipping\n",
      "[INFO]\tGene CNMD has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R32 has 0 variants, skipping\n",
      "[INFO]\tGene CDH10 has 0 variants, skipping\n",
      "[INFO]\tGene ALG10B has 0 variants, skipping\n",
      "[INFO]\tGene ZMIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD50 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPB has 0 variants, skipping\n",
      "[INFO]\tGene VANGL2 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR2 has 0 variants, skipping\n",
      "[INFO]\tGene CAV3 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN3 has 0 variants, skipping\n",
      "[INFO]\tGene MAMLD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS15 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP5 has 0 variants, skipping\n",
      "[INFO]\tGene SPEN has 0 variants, skipping\n",
      "[INFO]\tGene EFHD2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A3 has 0 variants, skipping\n",
      "[INFO]\tGene DACT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287036 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM7 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF5 has 0 variants, skipping\n",
      "[INFO]\tGene HJURP has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL1 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK1P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8078 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AD1 has 0 variants, skipping\n",
      "[INFO]\tGene QSOX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01925 has 0 variants, skipping\n",
      "[INFO]\tGene GDPGP1 has 0 variants, skipping\n",
      "[INFO]\tGene ALOX12B has 0 variants, skipping\n",
      "[INFO]\tGene AGRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01904 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00470 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN8 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN7 has 0 variants, skipping\n",
      "[INFO]\tGene CBX3P2 has 0 variants, skipping\n",
      "[INFO]\tGene GAS8 has 0 variants, skipping\n",
      "[INFO]\tGene GLYAT has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01895 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3D has 0 variants, skipping\n",
      "[INFO]\tGene RAB6D has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6718 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01892 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L26 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4E1B has 0 variants, skipping\n",
      "[INFO]\tGene PSTK has 0 variants, skipping\n",
      "[INFO]\tGene NUB1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP4 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3976HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3976 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373311 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR199B has 0 variants, skipping\n",
      "[INFO]\tGene MIR374B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01089 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM221 has 0 variants, skipping\n",
      "[INFO]\tGene MGP has 0 variants, skipping\n",
      "[INFO]\tGene RHBG has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01387 has 0 variants, skipping\n",
      "[INFO]\tGene UOX has 0 variants, skipping\n",
      "[INFO]\tGene TSSK3 has 0 variants, skipping\n",
      "[INFO]\tGene ADD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927188 has 0 variants, skipping\n",
      "[INFO]\tGene PRH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100192426 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF223 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf12 has 0 variants, skipping\n",
      "[INFO]\tGene NTAQ1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AP1S3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01254 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01887 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927410 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM167B has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL3 has 0 variants, skipping\n",
      "[INFO]\tGene TEX51 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6788 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01928 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01255 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7153 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723996 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf61 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371998 has 0 variants, skipping\n",
      "[INFO]\tGene LEPROT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01882 has 0 variants, skipping\n",
      "[INFO]\tGene CENPO has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5190 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4526 has 0 variants, skipping\n",
      "[INFO]\tGene INHA has 0 variants, skipping\n",
      "[INFO]\tGene UPK1A has 0 variants, skipping\n",
      "[INFO]\tGene TRIM50 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD2 has 0 variants, skipping\n",
      "[INFO]\tGene PKIG has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F35P has 0 variants, skipping\n",
      "[INFO]\tGene CXADRP3 has 0 variants, skipping\n",
      "[INFO]\tGene POTEC has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01443 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01444 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644669 has 0 variants, skipping\n",
      "[INFO]\tGene SCEL has 0 variants, skipping\n",
      "[INFO]\tGene PRKD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR133A1HG has 0 variants, skipping\n",
      "[INFO]\tGene SEC24D has 0 variants, skipping\n",
      "[INFO]\tGene LGR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927571 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1F has 0 variants, skipping\n",
      "[INFO]\tGene OR5AP2 has 0 variants, skipping\n",
      "[INFO]\tGene AMT has 0 variants, skipping\n",
      "[INFO]\tGene CDHR5 has 0 variants, skipping\n",
      "[INFO]\tGene RBFA has 0 variants, skipping\n",
      "[INFO]\tGene TMEM241 has 0 variants, skipping\n",
      "[INFO]\tGene RTL8B has 0 variants, skipping\n",
      "[INFO]\tGene RTL8C has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC3 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE3 has 0 variants, skipping\n",
      "[INFO]\tGene RRP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDXK has 0 variants, skipping\n",
      "[INFO]\tGene YARS1 has 0 variants, skipping\n",
      "[INFO]\tGene KMO has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3A has 0 variants, skipping\n",
      "[INFO]\tGene RAB2A has 0 variants, skipping\n",
      "[INFO]\tGene PZP has 0 variants, skipping\n",
      "[INFO]\tGene PYGL has 0 variants, skipping\n",
      "[INFO]\tGene TTC39C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AMER3 has 0 variants, skipping\n",
      "[INFO]\tGene APOC2 has 0 variants, skipping\n",
      "[INFO]\tGene CITED1 has 0 variants, skipping\n",
      "[INFO]\tGene LLGL1 has 0 variants, skipping\n",
      "[INFO]\tGene LBP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01915 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372028 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01894 has 0 variants, skipping\n",
      "[INFO]\tGene NR5A2 has 0 variants, skipping\n",
      "[INFO]\tGene CALHM6 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF13 has 0 variants, skipping\n",
      "[INFO]\tGene OAZ3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01543 has 0 variants, skipping\n",
      "[INFO]\tGene SESN3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8057 has 0 variants, skipping\n",
      "[INFO]\tGene ALPK3 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS2 has 0 variants, skipping\n",
      "[INFO]\tGene AQP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372038 has 0 variants, skipping\n",
      "[INFO]\tGene ADAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAS has 0 variants, skipping\n",
      "[INFO]\tGene SULF1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf105 has 0 variants, skipping\n",
      "[INFO]\tGene DSG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC29A2 has 0 variants, skipping\n",
      "[INFO]\tGene DSG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABRAXAS1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP4 has 0 variants, skipping\n",
      "[INFO]\tGene CEP250 has 0 variants, skipping\n",
      "[INFO]\tGene SNX5 has 0 variants, skipping\n",
      "[INFO]\tGene RELT has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1E has 0 variants, skipping\n",
      "[INFO]\tGene GJD3 has 0 variants, skipping\n",
      "[INFO]\tGene PANK1 has 0 variants, skipping\n",
      "[INFO]\tGene OXER1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOS has 0 variants, skipping\n",
      "[INFO]\tGene HUS1B has 0 variants, skipping\n",
      "[INFO]\tGene RPP38 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H2 has 0 variants, skipping\n",
      "[INFO]\tGene BTN3A2 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS1 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB1I1 has 0 variants, skipping\n",
      "[INFO]\tGene NPM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF271P has 0 variants, skipping\n",
      "[INFO]\tGene KRCC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3975 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3929 has 0 variants, skipping\n",
      "[INFO]\tGene U2AF1 has 0 variants, skipping\n",
      "[INFO]\tGene VARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEM1 has 0 variants, skipping\n",
      "[INFO]\tGene AIMP2 has 0 variants, skipping\n",
      "[INFO]\tGene TYRO3 has 0 variants, skipping\n",
      "[INFO]\tGene TSG101 has 0 variants, skipping\n",
      "[INFO]\tGene SEC16B has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC22 has 0 variants, skipping\n",
      "[INFO]\tGene HROB has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372071 has 0 variants, skipping\n",
      "[INFO]\tGene NAE1 has 0 variants, skipping\n",
      "[INFO]\tGene GJB2 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL1 has 0 variants, skipping\n",
      "[INFO]\tGene ETV5 has 0 variants, skipping\n",
      "[INFO]\tGene ESRRA has 0 variants, skipping\n",
      "[INFO]\tGene ERH has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372069 has 0 variants, skipping\n",
      "[INFO]\tGene SVEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK4 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf18 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG1 has 0 variants, skipping\n",
      "[INFO]\tGene SST has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A1 has 0 variants, skipping\n",
      "[INFO]\tGene PRIM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A4 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3R1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS3 has 0 variants, skipping\n",
      "[INFO]\tGene PENK has 0 variants, skipping\n",
      "[INFO]\tGene TARM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F1 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB14 has 0 variants, skipping\n",
      "[INFO]\tGene DELE1 has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL3 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MAMSTR has 0 variants, skipping\n",
      "[INFO]\tGene C10orf62 has 0 variants, skipping\n",
      "[INFO]\tGene DALRD3 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H3 has 0 variants, skipping\n",
      "[INFO]\tGene APPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL2BP has 0 variants, skipping\n",
      "[INFO]\tGene FITM2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF697 has 0 variants, skipping\n",
      "[INFO]\tGene SYCN has 0 variants, skipping\n",
      "[INFO]\tGene MYO5B has 0 variants, skipping\n",
      "[INFO]\tGene CCRL2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6V1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12A has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4A has 0 variants, skipping\n",
      "[INFO]\tGene PYM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372068 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA111 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4318 has 0 variants, skipping\n",
      "[INFO]\tGene MIR924HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR5583-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5583-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01902 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01477 has 0 variants, skipping\n",
      "[INFO]\tGene KC6 has 0 variants, skipping\n",
      "[INFO]\tGene FKTN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF765-ZNF761 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00907 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01478 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01601 has 0 variants, skipping\n",
      "[INFO]\tGene MCMBP has 0 variants, skipping\n",
      "[INFO]\tGene TESK2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC14 has 0 variants, skipping\n",
      "[INFO]\tGene TARBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF165 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C5 has 0 variants, skipping\n",
      "[INFO]\tGene SET has 0 variants, skipping\n",
      "[INFO]\tGene FAM124B has 0 variants, skipping\n",
      "[INFO]\tGene FAM228B has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0825 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf20 has 0 variants, skipping\n",
      "[INFO]\tGene INS-IGF2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR92 has 0 variants, skipping\n",
      "[INFO]\tGene DHX30 has 0 variants, skipping\n",
      "[INFO]\tGene HAS3 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM5 has 0 variants, skipping\n",
      "[INFO]\tGene HDGFL1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG4 has 0 variants, skipping\n",
      "[INFO]\tGene TAF8 has 0 variants, skipping\n",
      "[INFO]\tGene HINT3 has 0 variants, skipping\n",
      "[INFO]\tGene NUP88 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4527HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4743 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129878 has 0 variants, skipping\n",
      "[INFO]\tGene TCP10L3 has 0 variants, skipping\n",
      "[INFO]\tGene KY has 0 variants, skipping\n",
      "[INFO]\tGene MIR4744 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF3 has 0 variants, skipping\n",
      "[INFO]\tGene SPRN has 0 variants, skipping\n",
      "[INFO]\tGene MIR1539 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF6 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD58B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA17 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG22 has 0 variants, skipping\n",
      "[INFO]\tGene H2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4320 has 0 variants, skipping\n",
      "[INFO]\tGene MYCN has 0 variants, skipping\n",
      "[INFO]\tGene PRG4 has 0 variants, skipping\n",
      "[INFO]\tGene MCCC1 has 0 variants, skipping\n",
      "[INFO]\tGene GFM1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX30 has 0 variants, skipping\n",
      "[INFO]\tGene CDH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01630 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4528 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01919 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01917 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA37 has 0 variants, skipping\n",
      "[INFO]\tGene INPP4B has 0 variants, skipping\n",
      "[INFO]\tGene STX19 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T35 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T27 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS53 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01929 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG4 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A9 has 0 variants, skipping\n",
      "[INFO]\tGene CT47A10 has 0 variants, skipping\n",
      "[INFO]\tGene CASP8 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F3 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM15 has 0 variants, skipping\n",
      "[INFO]\tGene SMC4 has 0 variants, skipping\n",
      "[INFO]\tGene TCF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4529 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01415 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01416 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642484 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01905 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01539 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH1 has 0 variants, skipping\n",
      "[INFO]\tGene NPM1 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02565 has 0 variants, skipping\n",
      "[INFO]\tGene SNX1 has 0 variants, skipping\n",
      "[INFO]\tGene CMPK2 has 0 variants, skipping\n",
      "[INFO]\tGene THEGL has 0 variants, skipping\n",
      "[INFO]\tGene MRPL14 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP3 has 0 variants, skipping\n",
      "[INFO]\tGene UCHL1 has 0 variants, skipping\n",
      "[INFO]\tGene TPH1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF3 has 0 variants, skipping\n",
      "[INFO]\tGene IL4R has 0 variants, skipping\n",
      "[INFO]\tGene PLP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG1 has 0 variants, skipping\n",
      "[INFO]\tGene HECW2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3591 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA108 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01926 has 0 variants, skipping\n",
      "[INFO]\tGene PGAM4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM24A has 0 variants, skipping\n",
      "[INFO]\tGene PSME1 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT12 has 0 variants, skipping\n",
      "[INFO]\tGene OACYLP has 0 variants, skipping\n",
      "[INFO]\tGene ITIH4 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01544 has 0 variants, skipping\n",
      "[INFO]\tGene LSAMP has 0 variants, skipping\n",
      "[INFO]\tGene LRPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP6 has 0 variants, skipping\n",
      "[INFO]\tGene LRP4 has 0 variants, skipping\n",
      "[INFO]\tGene MROH6 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHD1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR139 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2A has 0 variants, skipping\n",
      "[INFO]\tGene RNASE10 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC73 has 0 variants, skipping\n",
      "[INFO]\tGene LRP2BP has 0 variants, skipping\n",
      "[INFO]\tGene ARV1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00515 has 0 variants, skipping\n",
      "[INFO]\tGene PICSAR has 0 variants, skipping\n",
      "[INFO]\tGene TAC3 has 0 variants, skipping\n",
      "[INFO]\tGene MASP1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR14 has 0 variants, skipping\n",
      "[INFO]\tGene WDR54 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10A has 0 variants, skipping\n",
      "[INFO]\tGene NIPSNAP3A has 0 variants, skipping\n",
      "[INFO]\tGene C3orf85 has 0 variants, skipping\n",
      "[INFO]\tGene CDH4 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL2 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP14 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM115 has 0 variants, skipping\n",
      "[INFO]\tGene DAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00305 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01924 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01538 has 0 variants, skipping\n",
      "[INFO]\tGene CSRNP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5011 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643542 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01903 has 0 variants, skipping\n",
      "[INFO]\tGene FAM221A has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372179 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf194 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01909 has 0 variants, skipping\n",
      "[INFO]\tGene LIVAR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01899 has 0 variants, skipping\n",
      "[INFO]\tGene MLH1 has 0 variants, skipping\n",
      "[INFO]\tGene GYPA has 0 variants, skipping\n",
      "[INFO]\tGene LOC541473 has 0 variants, skipping\n",
      "[INFO]\tGene SPEG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AV has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02582 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01922 has 0 variants, skipping\n",
      "[INFO]\tGene GRM2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01898 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01893 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927989 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00683 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01927 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01879 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF236-DT has 0 variants, skipping\n",
      "[INFO]\tGene CA7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01029 has 0 variants, skipping\n",
      "[INFO]\tGene CDK8 has 0 variants, skipping\n",
      "[INFO]\tGene CD164 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R3 has 0 variants, skipping\n",
      "[INFO]\tGene SLURP1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L2B has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR3L has 0 variants, skipping\n",
      "[INFO]\tGene LOC284240 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284241 has 0 variants, skipping\n",
      "[INFO]\tGene TOR1AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF35 has 0 variants, skipping\n",
      "[INFO]\tGene RBFADN has 0 variants, skipping\n",
      "[INFO]\tGene PARD6G-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRSAM1 has 0 variants, skipping\n",
      "[INFO]\tGene WASH5P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01002 has 0 variants, skipping\n",
      "[INFO]\tGene PSG7 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL36 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf72 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN11A has 0 variants, skipping\n",
      "[INFO]\tGene PNP has 0 variants, skipping\n",
      "[INFO]\tGene NF2 has 0 variants, skipping\n",
      "[INFO]\tGene NT5C has 0 variants, skipping\n",
      "[INFO]\tGene MIR4745 has 0 variants, skipping\n",
      "[INFO]\tGene AZU1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3187 has 0 variants, skipping\n",
      "[INFO]\tGene CRHR2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E1 has 0 variants, skipping\n",
      "[INFO]\tGene TFR2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN2 has 0 variants, skipping\n",
      "[INFO]\tGene STMN1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMC2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF503 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3D has 0 variants, skipping\n",
      "[INFO]\tGene BAG2 has 0 variants, skipping\n",
      "[INFO]\tGene BAG3 has 0 variants, skipping\n",
      "[INFO]\tGene PMPCB has 0 variants, skipping\n",
      "[INFO]\tGene CIRBP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHM1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM11B has 0 variants, skipping\n",
      "[INFO]\tGene NRGN has 0 variants, skipping\n",
      "[INFO]\tGene PPP5D1 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1909 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288123 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HIRIP3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR65 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1227 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6789 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4321 has 0 variants, skipping\n",
      "[INFO]\tGene CD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7108 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01775 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7850 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPD has 0 variants, skipping\n",
      "[INFO]\tGene TNRC6A has 0 variants, skipping\n",
      "[INFO]\tGene UTP20 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF19 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2S has 0 variants, skipping\n",
      "[INFO]\tGene LARGE2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996351 has 0 variants, skipping\n",
      "[INFO]\tGene CACTIN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF765 has 0 variants, skipping\n",
      "[INFO]\tGene NMRK2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD37 has 0 variants, skipping\n",
      "[INFO]\tGene MIS12 has 0 variants, skipping\n",
      "[INFO]\tGene STAC2 has 0 variants, skipping\n",
      "[INFO]\tGene LAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD2L has 0 variants, skipping\n",
      "[INFO]\tGene DTX3 has 0 variants, skipping\n",
      "[INFO]\tGene EML3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4746 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B1 has 0 variants, skipping\n",
      "[INFO]\tGene FTCDNL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDY2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-3HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4747 has 0 variants, skipping\n",
      "[INFO]\tGene RFX1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDF2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AP4B1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A8 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB7 has 0 variants, skipping\n",
      "[INFO]\tGene TGDS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF600 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC154 has 0 variants, skipping\n",
      "[INFO]\tGene TTC9 has 0 variants, skipping\n",
      "[INFO]\tGene BICRAL has 0 variants, skipping\n",
      "[INFO]\tGene KRT6B has 0 variants, skipping\n",
      "[INFO]\tGene FEM1C has 0 variants, skipping\n",
      "[INFO]\tGene SMCO4 has 0 variants, skipping\n",
      "[INFO]\tGene HMCES has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF3 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928844 has 0 variants, skipping\n",
      "[INFO]\tGene MCM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128568 has 0 variants, skipping\n",
      "[INFO]\tGene FAM204A has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6885 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6790 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF384 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3940 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM240 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2A1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1F has 0 variants, skipping\n",
      "[INFO]\tGene KMT5A has 0 variants, skipping\n",
      "[INFO]\tGene SMIM34A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6791 has 0 variants, skipping\n",
      "[INFO]\tGene KIF13A has 0 variants, skipping\n",
      "[INFO]\tGene C20orf194 has 0 variants, skipping\n",
      "[INFO]\tGene CD6 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK9 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRE4P has 0 variants, skipping\n",
      "[INFO]\tGene FLJ25758 has 0 variants, skipping\n",
      "[INFO]\tGene CD244 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDPK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF723 has 0 variants, skipping\n",
      "[INFO]\tGene DNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene TECPR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARID4A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC19 has 0 variants, skipping\n",
      "[INFO]\tGene CASD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6792 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0100 has 0 variants, skipping\n",
      "[INFO]\tGene NET1 has 0 variants, skipping\n",
      "[INFO]\tGene COL13A1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCKSL1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4GP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC121 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA2P2 has 0 variants, skipping\n",
      "[INFO]\tGene S100A12 has 0 variants, skipping\n",
      "[INFO]\tGene S100A11 has 0 variants, skipping\n",
      "[INFO]\tGene RTN2 has 0 variants, skipping\n",
      "[INFO]\tGene DLL1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE6 has 0 variants, skipping\n",
      "[INFO]\tGene RHEB has 0 variants, skipping\n",
      "[INFO]\tGene RBL2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP4 has 0 variants, skipping\n",
      "[INFO]\tGene NHLH2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRCAP has 0 variants, skipping\n",
      "[INFO]\tGene NHLH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPNS2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2B has 0 variants, skipping\n",
      "[INFO]\tGene RAB11B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4999 has 0 variants, skipping\n",
      "[INFO]\tGene RNF4 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1SW has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PO has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1E1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP16 has 0 variants, skipping\n",
      "[INFO]\tGene MRC2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL6 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf70 has 0 variants, skipping\n",
      "[INFO]\tGene RAB12 has 0 variants, skipping\n",
      "[INFO]\tGene LACTBL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF426-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF561-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM179 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF812P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505555 has 0 variants, skipping\n",
      "[INFO]\tGene OGG1 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGD has 0 variants, skipping\n",
      "[INFO]\tGene ABRACL has 0 variants, skipping\n",
      "[INFO]\tGene MID1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNE2 has 0 variants, skipping\n",
      "[INFO]\tGene PARD3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5589 has 0 variants, skipping\n",
      "[INFO]\tGene C3P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD105 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD105B has 0 variants, skipping\n",
      "[INFO]\tGene RBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1841 has 0 variants, skipping\n",
      "[INFO]\tGene RGS11 has 0 variants, skipping\n",
      "[INFO]\tGene PSMG4 has 0 variants, skipping\n",
      "[INFO]\tGene ECD has 0 variants, skipping\n",
      "[INFO]\tGene MIR4322 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB7 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF157 has 0 variants, skipping\n",
      "[INFO]\tGene METAP1 has 0 variants, skipping\n",
      "[INFO]\tGene FIGN has 0 variants, skipping\n",
      "[INFO]\tGene TCP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4748 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6793 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL8 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF14 has 0 variants, skipping\n",
      "[INFO]\tGene AK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6886 has 0 variants, skipping\n",
      "[INFO]\tGene FAM227B has 0 variants, skipping\n",
      "[INFO]\tGene ARID2 has 0 variants, skipping\n",
      "[INFO]\tGene STPG4 has 0 variants, skipping\n",
      "[INFO]\tGene TYROBP has 0 variants, skipping\n",
      "[INFO]\tGene TXNRD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372273 has 0 variants, skipping\n",
      "[INFO]\tGene OXTR has 0 variants, skipping\n",
      "[INFO]\tGene GIP has 0 variants, skipping\n",
      "[INFO]\tGene GHSR has 0 variants, skipping\n",
      "[INFO]\tGene GBP2 has 0 variants, skipping\n",
      "[INFO]\tGene FLT3 has 0 variants, skipping\n",
      "[INFO]\tGene VWA3B has 0 variants, skipping\n",
      "[INFO]\tGene AKT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7974 has 0 variants, skipping\n",
      "[INFO]\tGene IFT22 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1E2 has 0 variants, skipping\n",
      "[INFO]\tGene PWP1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1P10 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF833P has 0 variants, skipping\n",
      "[INFO]\tGene CEBPG has 0 variants, skipping\n",
      "[INFO]\tGene SPDEF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF433-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723360 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF625-ZNF20 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289333 has 0 variants, skipping\n",
      "[INFO]\tGene PSAP has 0 variants, skipping\n",
      "[INFO]\tGene PRR36 has 0 variants, skipping\n",
      "[INFO]\tGene CTH has 0 variants, skipping\n",
      "[INFO]\tGene SNAI1 has 0 variants, skipping\n",
      "[INFO]\tGene BPNT1 has 0 variants, skipping\n",
      "[INFO]\tGene IRF9 has 0 variants, skipping\n",
      "[INFO]\tGene PTH1R has 0 variants, skipping\n",
      "[INFO]\tGene L1TD1 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene NUAK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD135 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD41 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00703 has 0 variants, skipping\n",
      "[INFO]\tGene CLCN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5684 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6794 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC19 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC141 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG1 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L12 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5695 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6515 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3177 has 0 variants, skipping\n",
      "[INFO]\tGene CCL11 has 0 variants, skipping\n",
      "[INFO]\tGene CCL5 has 0 variants, skipping\n",
      "[INFO]\tGene EVC2 has 0 variants, skipping\n",
      "[INFO]\tGene FRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM135B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF691 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L5 has 0 variants, skipping\n",
      "[INFO]\tGene RFC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284454 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF679 has 0 variants, skipping\n",
      "[INFO]\tGene EPRS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD5 has 0 variants, skipping\n",
      "[INFO]\tGene MISP3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1199 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507373 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01841 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01842 has 0 variants, skipping\n",
      "[INFO]\tGene MIR639 has 0 variants, skipping\n",
      "[INFO]\tGene SCG3 has 0 variants, skipping\n",
      "[INFO]\tGene AGXT has 0 variants, skipping\n",
      "[INFO]\tGene CLEC17A has 0 variants, skipping\n",
      "[INFO]\tGene CXCR3 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RA has 0 variants, skipping\n",
      "[INFO]\tGene SNORA104 has 0 variants, skipping\n",
      "[INFO]\tGene H3C4 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFB3 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6795 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1470 has 0 variants, skipping\n",
      "[INFO]\tGene LZIC has 0 variants, skipping\n",
      "[INFO]\tGene AGO1 has 0 variants, skipping\n",
      "[INFO]\tGene RAP1GAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F24P has 0 variants, skipping\n",
      "[INFO]\tGene FLI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM229A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00661 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00905 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01855 has 0 variants, skipping\n",
      "[INFO]\tGene FLG2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC183 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02749 has 0 variants, skipping\n",
      "[INFO]\tGene UTP6 has 0 variants, skipping\n",
      "[INFO]\tGene PRAC1 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene NIFK has 0 variants, skipping\n",
      "[INFO]\tGene WNK4 has 0 variants, skipping\n",
      "[INFO]\tGene NNT has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MG has 0 variants, skipping\n",
      "[INFO]\tGene GRK4 has 0 variants, skipping\n",
      "[INFO]\tGene OSGIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00221 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2U1 has 0 variants, skipping\n",
      "[INFO]\tGene GLT6D1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL8 has 0 variants, skipping\n",
      "[INFO]\tGene CDK14 has 0 variants, skipping\n",
      "[INFO]\tGene CENPP has 0 variants, skipping\n",
      "[INFO]\tGene SRP14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA118 has 0 variants, skipping\n",
      "[INFO]\tGene COTL1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNO has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1256 has 0 variants, skipping\n",
      "[INFO]\tGene PKN3 has 0 variants, skipping\n",
      "[INFO]\tGene BISPR has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT77 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CD99 has 0 variants, skipping\n",
      "[INFO]\tGene MIF has 0 variants, skipping\n",
      "[INFO]\tGene MKI67 has 0 variants, skipping\n",
      "[INFO]\tGene MSH3 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2B has 0 variants, skipping\n",
      "[INFO]\tGene OR8J3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC119B has 0 variants, skipping\n",
      "[INFO]\tGene SCRG1 has 0 variants, skipping\n",
      "[INFO]\tGene GABARAP has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA68 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112577592 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725254 has 0 variants, skipping\n",
      "[INFO]\tGene KIFC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729966 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3189 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF514 has 0 variants, skipping\n",
      "[INFO]\tGene PCNX4 has 0 variants, skipping\n",
      "[INFO]\tGene ACADSB has 0 variants, skipping\n",
      "[INFO]\tGene LINC00513 has 0 variants, skipping\n",
      "[INFO]\tGene FAM177A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR18 has 0 variants, skipping\n",
      "[INFO]\tGene ARAF has 0 variants, skipping\n",
      "[INFO]\tGene BTG2 has 0 variants, skipping\n",
      "[INFO]\tGene LAPTM5 has 0 variants, skipping\n",
      "[INFO]\tGene UPK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF175 has 0 variants, skipping\n",
      "[INFO]\tGene EPC2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS38 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM34 has 0 variants, skipping\n",
      "[INFO]\tGene PDYN has 0 variants, skipping\n",
      "[INFO]\tGene FLT4 has 0 variants, skipping\n",
      "[INFO]\tGene INKA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS7 has 0 variants, skipping\n",
      "[INFO]\tGene ARSG has 0 variants, skipping\n",
      "[INFO]\tGene FAM104A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00663 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF826P has 0 variants, skipping\n",
      "[INFO]\tGene RLF has 0 variants, skipping\n",
      "[INFO]\tGene RGS17 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA6L has 0 variants, skipping\n",
      "[INFO]\tGene LINC00664 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB2A has 0 variants, skipping\n",
      "[INFO]\tGene TUBG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400682 has 0 variants, skipping\n",
      "[INFO]\tGene TIFA has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929124 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01233 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01785 has 0 variants, skipping\n",
      "[INFO]\tGene NAGS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01859 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01858 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO5P1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA10 has 0 variants, skipping\n",
      "[INFO]\tGene FEZ1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505851 has 0 variants, skipping\n",
      "[INFO]\tGene FCRLB has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927151 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100420587 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724908 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00906 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724958 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284395 has 0 variants, skipping\n",
      "[INFO]\tGene MRC1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK5 has 0 variants, skipping\n",
      "[INFO]\tGene PTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGOHB has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN2 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01791 has 0 variants, skipping\n",
      "[INFO]\tGene THEG5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01782 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400684 has 0 variants, skipping\n",
      "[INFO]\tGene XKR9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA68B has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA13 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA12 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN2A has 0 variants, skipping\n",
      "[INFO]\tGene ETAA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA11 has 0 variants, skipping\n",
      "[INFO]\tGene DHX40 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2A has 0 variants, skipping\n",
      "[INFO]\tGene LEF1 has 0 variants, skipping\n",
      "[INFO]\tGene FIGLA has 0 variants, skipping\n",
      "[INFO]\tGene VMA21 has 0 variants, skipping\n",
      "[INFO]\tGene CT83 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1B2P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM130 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2B3P has 0 variants, skipping\n",
      "[INFO]\tGene UNC80 has 0 variants, skipping\n",
      "[INFO]\tGene STK25 has 0 variants, skipping\n",
      "[INFO]\tGene APMAP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01801 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00904 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01838 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF30-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BUB3 has 0 variants, skipping\n",
      "[INFO]\tGene LNPEP has 0 variants, skipping\n",
      "[INFO]\tGene SERGEF has 0 variants, skipping\n",
      "[INFO]\tGene HPN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN23 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA5 has 0 variants, skipping\n",
      "[INFO]\tGene UBC has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1549 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE26 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL10 has 0 variants, skipping\n",
      "[INFO]\tGene ING2 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF9 has 0 variants, skipping\n",
      "[INFO]\tGene IL13RA1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00312 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC66 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01531 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PRG3 has 0 variants, skipping\n",
      "[INFO]\tGene NDC80 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL13 has 0 variants, skipping\n",
      "[INFO]\tGene TSSC4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM147-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01766 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF180 has 0 variants, skipping\n",
      "[INFO]\tGene UPK1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDH16 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR4 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA2 has 0 variants, skipping\n",
      "[INFO]\tGene NEFM has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-3 has 0 variants, skipping\n",
      "[INFO]\tGene SNX29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-1 has 0 variants, skipping\n",
      "[INFO]\tGene RRP7A has 0 variants, skipping\n",
      "[INFO]\tGene MMADHC has 0 variants, skipping\n",
      "[INFO]\tGene LINC01529 has 0 variants, skipping\n",
      "[INFO]\tGene PIBF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372383 has 0 variants, skipping\n",
      "[INFO]\tGene SIK1 has 0 variants, skipping\n",
      "[INFO]\tGene TPH2 has 0 variants, skipping\n",
      "[INFO]\tGene MED20 has 0 variants, skipping\n",
      "[INFO]\tGene NKILA has 0 variants, skipping\n",
      "[INFO]\tGene FAT3 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf71 has 0 variants, skipping\n",
      "[INFO]\tGene STK19 has 0 variants, skipping\n",
      "[INFO]\tGene ASMTL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134317 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728752 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF529-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0D1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01534 has 0 variants, skipping\n",
      "[INFO]\tGene STMP1 has 0 variants, skipping\n",
      "[INFO]\tGene DSG4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF790-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ING1 has 0 variants, skipping\n",
      "[INFO]\tGene EPYC has 0 variants, skipping\n",
      "[INFO]\tGene CREBRF has 0 variants, skipping\n",
      "[INFO]\tGene MIR328 has 0 variants, skipping\n",
      "[INFO]\tGene LOC104968399 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376731 has 0 variants, skipping\n",
      "[INFO]\tGene LSS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01391 has 0 variants, skipping\n",
      "[INFO]\tGene IQCJ-SCHIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IRAIN has 0 variants, skipping\n",
      "[INFO]\tGene SPATA19 has 0 variants, skipping\n",
      "[INFO]\tGene PDILT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF793-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF571-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX37 has 0 variants, skipping\n",
      "[INFO]\tGene NANP has 0 variants, skipping\n",
      "[INFO]\tGene TCTEX1D1 has 0 variants, skipping\n",
      "[INFO]\tGene GGN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF585A has 0 variants, skipping\n",
      "[INFO]\tGene DAND5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644554 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD152 has 0 variants, skipping\n",
      "[INFO]\tGene LDB2 has 0 variants, skipping\n",
      "[INFO]\tGene GAPT has 0 variants, skipping\n",
      "[INFO]\tGene SPRED3 has 0 variants, skipping\n",
      "[INFO]\tGene STK11 has 0 variants, skipping\n",
      "[INFO]\tGene SOD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A2 has 0 variants, skipping\n",
      "[INFO]\tGene SHOX has 0 variants, skipping\n",
      "[INFO]\tGene SELE has 0 variants, skipping\n",
      "[INFO]\tGene PSEN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372397 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF330 has 0 variants, skipping\n",
      "[INFO]\tGene CPT1B has 0 variants, skipping\n",
      "[INFO]\tGene CYP2S1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP2 has 0 variants, skipping\n",
      "[INFO]\tGene ROS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA6 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA4 has 0 variants, skipping\n",
      "[INFO]\tGene FMOD has 0 variants, skipping\n",
      "[INFO]\tGene DDB1 has 0 variants, skipping\n",
      "[INFO]\tGene FMO4 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT4 has 0 variants, skipping\n",
      "[INFO]\tGene NCS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA4 has 0 variants, skipping\n",
      "[INFO]\tGene FGF5 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD175 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129935 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS17A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM169 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6719 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2A has 0 variants, skipping\n",
      "[INFO]\tGene UBA7 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC49A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6796 has 0 variants, skipping\n",
      "[INFO]\tGene RALA has 0 variants, skipping\n",
      "[INFO]\tGene PHF2 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN14 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A1 has 0 variants, skipping\n",
      "[INFO]\tGene SSPN has 0 variants, skipping\n",
      "[INFO]\tGene PARP10 has 0 variants, skipping\n",
      "[INFO]\tGene SDS has 0 variants, skipping\n",
      "[INFO]\tGene SF3B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIA-RAB4B has 0 variants, skipping\n",
      "[INFO]\tGene RAB4B-EGLN2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM8 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2T1P has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2G1P has 0 variants, skipping\n",
      "[INFO]\tGene CYP2B7P has 0 variants, skipping\n",
      "[INFO]\tGene NPTN has 0 variants, skipping\n",
      "[INFO]\tGene S100A14 has 0 variants, skipping\n",
      "[INFO]\tGene CLK4 has 0 variants, skipping\n",
      "[INFO]\tGene CLTRN has 0 variants, skipping\n",
      "[INFO]\tGene DECR2 has 0 variants, skipping\n",
      "[INFO]\tGene RHOJ has 0 variants, skipping\n",
      "[INFO]\tGene NUP205 has 0 variants, skipping\n",
      "[INFO]\tGene UPP2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-9 has 0 variants, skipping\n",
      "[INFO]\tGene NINL has 0 variants, skipping\n",
      "[INFO]\tGene AHSP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01480 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A2 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL2A has 0 variants, skipping\n",
      "[INFO]\tGene SNAP29 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFYVE9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505585 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D4A has 0 variants, skipping\n",
      "[INFO]\tGene GFOD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4323 has 0 variants, skipping\n",
      "[INFO]\tGene STX6 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR1 has 0 variants, skipping\n",
      "[INFO]\tGene RWDD2B has 0 variants, skipping\n",
      "[INFO]\tGene OTOF has 0 variants, skipping\n",
      "[INFO]\tGene CLCNKA has 0 variants, skipping\n",
      "[INFO]\tGene ILVBL has 0 variants, skipping\n",
      "[INFO]\tGene MIR8077 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930071 has 0 variants, skipping\n",
      "[INFO]\tGene LIPE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BARHL2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR108 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1D has 0 variants, skipping\n",
      "[INFO]\tGene PSG8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf58 has 0 variants, skipping\n",
      "[INFO]\tGene STAP2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6N2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284344 has 0 variants, skipping\n",
      "[INFO]\tGene PRG1 has 0 variants, skipping\n",
      "[INFO]\tGene TWIST1 has 0 variants, skipping\n",
      "[INFO]\tGene PEX1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAP has 0 variants, skipping\n",
      "[INFO]\tGene CDC73 has 0 variants, skipping\n",
      "[INFO]\tGene NEBL has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A3 has 0 variants, skipping\n",
      "[INFO]\tGene DFFA has 0 variants, skipping\n",
      "[INFO]\tGene ALLC has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505715 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928063 has 0 variants, skipping\n",
      "[INFO]\tGene SNUPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC100379224 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A6 has 0 variants, skipping\n",
      "[INFO]\tGene UQCC1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC47A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184C has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP1L has 0 variants, skipping\n",
      "[INFO]\tGene PTPN6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4531 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7A has 0 variants, skipping\n",
      "[INFO]\tGene MIR8085 has 0 variants, skipping\n",
      "[INFO]\tGene CCR4 has 0 variants, skipping\n",
      "[INFO]\tGene AP1M2 has 0 variants, skipping\n",
      "[INFO]\tGene FSTL3 has 0 variants, skipping\n",
      "[INFO]\tGene PURA has 0 variants, skipping\n",
      "[INFO]\tGene AKAP8 has 0 variants, skipping\n",
      "[INFO]\tGene ZMPSTE24 has 0 variants, skipping\n",
      "[INFO]\tGene RAMP3 has 0 variants, skipping\n",
      "[INFO]\tGene APOC4-APOC2 has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG4 has 0 variants, skipping\n",
      "[INFO]\tGene TESC has 0 variants, skipping\n",
      "[INFO]\tGene RMND1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT57 has 0 variants, skipping\n",
      "[INFO]\tGene FEZF2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR1 has 0 variants, skipping\n",
      "[INFO]\tGene GID8 has 0 variants, skipping\n",
      "[INFO]\tGene TRMU has 0 variants, skipping\n",
      "[INFO]\tGene PAK1IP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM45A has 0 variants, skipping\n",
      "[INFO]\tGene TTC22 has 0 variants, skipping\n",
      "[INFO]\tGene UACA has 0 variants, skipping\n",
      "[INFO]\tGene DDT has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6088 has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF107 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM46 has 0 variants, skipping\n",
      "[INFO]\tGene TCF3 has 0 variants, skipping\n",
      "[INFO]\tGene EML2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AHR has 0 variants, skipping\n",
      "[INFO]\tGene CYP2W1 has 0 variants, skipping\n",
      "[INFO]\tGene CHD7 has 0 variants, skipping\n",
      "[INFO]\tGene DM1-AS has 0 variants, skipping\n",
      "[INFO]\tGene FLOT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF417 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC93429 has 0 variants, skipping\n",
      "[INFO]\tGene VIL1 has 0 variants, skipping\n",
      "[INFO]\tGene AGK has 0 variants, skipping\n",
      "[INFO]\tGene CCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene KIRREL1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F6 has 0 variants, skipping\n",
      "[INFO]\tGene DACT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS34 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-E has 0 variants, skipping\n",
      "[INFO]\tGene USP17L1 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L8 has 0 variants, skipping\n",
      "[INFO]\tGene TENT2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3190 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3191 has 0 variants, skipping\n",
      "[INFO]\tGene MANSC4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP52 has 0 variants, skipping\n",
      "[INFO]\tGene NAPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD23 has 0 variants, skipping\n",
      "[INFO]\tGene NOP53-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PSG5 has 0 variants, skipping\n",
      "[INFO]\tGene PSD has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A12 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A13 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A4 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A5 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A7 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A11 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A9 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A6 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A8 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A10 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-A14 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C4 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-C3 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDY1B has 0 variants, skipping\n",
      "[INFO]\tGene MANBAL has 0 variants, skipping\n",
      "[INFO]\tGene SPAG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP6NL has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D1 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD3 has 0 variants, skipping\n",
      "[INFO]\tGene RALB has 0 variants, skipping\n",
      "[INFO]\tGene CARD8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CST11 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10H3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A16 has 0 variants, skipping\n",
      "[INFO]\tGene OR12D2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX24 has 0 variants, skipping\n",
      "[INFO]\tGene ASF1A has 0 variants, skipping\n",
      "[INFO]\tGene FOCAD has 0 variants, skipping\n",
      "[INFO]\tGene SNORD76 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL3 has 0 variants, skipping\n",
      "[INFO]\tGene MESTIT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01139 has 0 variants, skipping\n",
      "[INFO]\tGene ASPSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PAX8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEC1P has 0 variants, skipping\n",
      "[INFO]\tGene TIMM50 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105447645 has 0 variants, skipping\n",
      "[INFO]\tGene PAG1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO6 has 0 variants, skipping\n",
      "[INFO]\tGene HEMGN has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1G has 0 variants, skipping\n",
      "[INFO]\tGene NUCB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HELQ has 0 variants, skipping\n",
      "[INFO]\tGene SAP25 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF675 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR1 has 0 variants, skipping\n",
      "[INFO]\tGene ACMSD has 0 variants, skipping\n",
      "[INFO]\tGene MIR6798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101059948 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-G2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-G1 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB4 has 0 variants, skipping\n",
      "[INFO]\tGene H4C14 has 0 variants, skipping\n",
      "[INFO]\tGene CHST14 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3B has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB3 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC12 has 0 variants, skipping\n",
      "[INFO]\tGene ZRANB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928295 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AJ1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14C has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOQ has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST1 has 0 variants, skipping\n",
      "[INFO]\tGene KL has 0 variants, skipping\n",
      "[INFO]\tGene SNORD32A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD33 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD34 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD35A has 0 variants, skipping\n",
      "[INFO]\tGene ZIC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD35B has 0 variants, skipping\n",
      "[INFO]\tGene MORC3 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR939 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOI has 0 variants, skipping\n",
      "[INFO]\tGene MIR5088 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG6 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1D has 0 variants, skipping\n",
      "[INFO]\tGene MIR6799 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHJ1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6800 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1E has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A32 has 0 variants, skipping\n",
      "[INFO]\tGene PTOV1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL19 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP29 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4750 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR4751 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP35 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400710 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-B2 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-B1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-D has 0 variants, skipping\n",
      "[INFO]\tGene TOPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMF1 has 0 variants, skipping\n",
      "[INFO]\tGene NAPSB has 0 variants, skipping\n",
      "[INFO]\tGene IQSEC3 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-F has 0 variants, skipping\n",
      "[INFO]\tGene C19orf33 has 0 variants, skipping\n",
      "[INFO]\tGene RMI1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD88C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01869 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3C has 0 variants, skipping\n",
      "[INFO]\tGene TMEM225B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372441 has 0 variants, skipping\n",
      "[INFO]\tGene TLE2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1143 has 0 variants, skipping\n",
      "[INFO]\tGene KLKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM160 has 0 variants, skipping\n",
      "[INFO]\tGene G2E3 has 0 variants, skipping\n",
      "[INFO]\tGene STMN4 has 0 variants, skipping\n",
      "[INFO]\tGene SGO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C6 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC28 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928517 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC17P has 0 variants, skipping\n",
      "[INFO]\tGene MIR8074 has 0 variants, skipping\n",
      "[INFO]\tGene FSHB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01872 has 0 variants, skipping\n",
      "[INFO]\tGene RPS16 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB2 has 0 variants, skipping\n",
      "[INFO]\tGene PHF23 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A6 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP30 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01530 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA6P-AS has 0 variants, skipping\n",
      "[INFO]\tGene MATN4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF649-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF350-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD8 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF616 has 0 variants, skipping\n",
      "[INFO]\tGene CLIC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6801 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM42 has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF528-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DUOX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF137P has 0 variants, skipping\n",
      "[INFO]\tGene MSI1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF321P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF816-ZNF321P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF702P has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1C2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R16 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R4 has 0 variants, skipping\n",
      "[INFO]\tGene RFFL has 0 variants, skipping\n",
      "[INFO]\tGene YLPM1 has 0 variants, skipping\n",
      "[INFO]\tGene SF3A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK4 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B2 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH3 has 0 variants, skipping\n",
      "[INFO]\tGene GNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A27P has 0 variants, skipping\n",
      "[INFO]\tGene RAD51AP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAAP100 has 0 variants, skipping\n",
      "[INFO]\tGene TARS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPM3P9 has 0 variants, skipping\n",
      "[INFO]\tGene CLCF1 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF45 has 0 variants, skipping\n",
      "[INFO]\tGene NAV2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284379 has 0 variants, skipping\n",
      "[INFO]\tGene MIR512-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR515-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519E has 0 variants, skipping\n",
      "[INFO]\tGene COMMD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR523 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518F has 0 variants, skipping\n",
      "[INFO]\tGene MIR521-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518E has 0 variants, skipping\n",
      "[INFO]\tGene MIR518A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR518A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR521-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR522 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1283-2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCA1L has 0 variants, skipping\n",
      "[INFO]\tGene LSMEM2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD54L2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX1 has 0 variants, skipping\n",
      "[INFO]\tGene OVGP1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBP has 0 variants, skipping\n",
      "[INFO]\tGene DENND11 has 0 variants, skipping\n",
      "[INFO]\tGene KRT17 has 0 variants, skipping\n",
      "[INFO]\tGene MYO15A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX5 has 0 variants, skipping\n",
      "[INFO]\tGene XRCC2 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFC has 0 variants, skipping\n",
      "[INFO]\tGene WNT1 has 0 variants, skipping\n",
      "[INFO]\tGene VAV1 has 0 variants, skipping\n",
      "[INFO]\tGene TCHHL1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B28 has 0 variants, skipping\n",
      "[INFO]\tGene FAM99B has 0 variants, skipping\n",
      "[INFO]\tGene DMAC2L has 0 variants, skipping\n",
      "[INFO]\tGene ERCC3 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4752 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC5 has 0 variants, skipping\n",
      "[INFO]\tGene IGDCC4 has 0 variants, skipping\n",
      "[INFO]\tGene BDNF-AS has 0 variants, skipping\n",
      "[INFO]\tGene SMPX has 0 variants, skipping\n",
      "[INFO]\tGene FPGT-TNNI3K has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A1 has 0 variants, skipping\n",
      "[INFO]\tGene LENG8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIR3DX1 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00899 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8061 has 0 variants, skipping\n",
      "[INFO]\tGene CEP44 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINI1 has 0 variants, skipping\n",
      "[INFO]\tGene MICAL3 has 0 variants, skipping\n",
      "[INFO]\tGene LILRP2 has 0 variants, skipping\n",
      "[INFO]\tGene BRD7 has 0 variants, skipping\n",
      "[INFO]\tGene MYLIP has 0 variants, skipping\n",
      "[INFO]\tGene SAP30BP has 0 variants, skipping\n",
      "[INFO]\tGene PYCARD has 0 variants, skipping\n",
      "[INFO]\tGene SGK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928804 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133A has 0 variants, skipping\n",
      "[INFO]\tGene GP1BA has 0 variants, skipping\n",
      "[INFO]\tGene MMP12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00659 has 0 variants, skipping\n",
      "[INFO]\tGene NOL8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR137HG has 0 variants, skipping\n",
      "[INFO]\tGene SCYL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF468 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7975 has 0 variants, skipping\n",
      "[INFO]\tGene PTCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM98 has 0 variants, skipping\n",
      "[INFO]\tGene CST3 has 0 variants, skipping\n",
      "[INFO]\tGene KDSR has 0 variants, skipping\n",
      "[INFO]\tGene DLX4 has 0 variants, skipping\n",
      "[INFO]\tGene FUT5 has 0 variants, skipping\n",
      "[INFO]\tGene FUT4 has 0 variants, skipping\n",
      "[INFO]\tGene FTH1 has 0 variants, skipping\n",
      "[INFO]\tGene FRK has 0 variants, skipping\n",
      "[INFO]\tGene MIR6804 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6803 has 0 variants, skipping\n",
      "[INFO]\tGene CD24 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51P4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A51P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6805 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF683 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD157 has 0 variants, skipping\n",
      "[INFO]\tGene IL7 has 0 variants, skipping\n",
      "[INFO]\tGene HYPK has 0 variants, skipping\n",
      "[INFO]\tGene GNAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SOSTDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF521 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR9 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC5 has 0 variants, skipping\n",
      "[INFO]\tGene NELFB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01864 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF485 has 0 variants, skipping\n",
      "[INFO]\tGene TACR3 has 0 variants, skipping\n",
      "[INFO]\tGene PTGFR has 0 variants, skipping\n",
      "[INFO]\tGene PTGIR has 0 variants, skipping\n",
      "[INFO]\tGene PTGER4 has 0 variants, skipping\n",
      "[INFO]\tGene TACR2 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C2 has 0 variants, skipping\n",
      "[INFO]\tGene SERF1A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2O has 0 variants, skipping\n",
      "[INFO]\tGene TCL1A has 0 variants, skipping\n",
      "[INFO]\tGene PGM5 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF582-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HHLA1 has 0 variants, skipping\n",
      "[INFO]\tGene LIN7C has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF4 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN3 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1522 has 0 variants, skipping\n",
      "[INFO]\tGene PEG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR1B has 0 variants, skipping\n",
      "[INFO]\tGene SPR has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF460-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE4A has 0 variants, skipping\n",
      "[INFO]\tGene DONSON has 0 variants, skipping\n",
      "[INFO]\tGene PLCB3 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM13 has 0 variants, skipping\n",
      "[INFO]\tGene POC1A has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A5 has 0 variants, skipping\n",
      "[INFO]\tGene SYNGR3 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPD1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1AP1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2CP5 has 0 variants, skipping\n",
      "[INFO]\tGene RESP18 has 0 variants, skipping\n",
      "[INFO]\tGene CD48 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128398 has 0 variants, skipping\n",
      "[INFO]\tGene SLC48A1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF2 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene CD109 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP39A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF8-ERVK3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ERVK3-1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372480 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6806 has 0 variants, skipping\n",
      "[INFO]\tGene A1BG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4754 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02560 has 0 variants, skipping\n",
      "[INFO]\tGene NPAS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL18 has 0 variants, skipping\n",
      "[INFO]\tGene MZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPBD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf226 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO2 has 0 variants, skipping\n",
      "[INFO]\tGene RELL1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2M has 0 variants, skipping\n",
      "[INFO]\tGene PPM1N has 0 variants, skipping\n",
      "[INFO]\tGene PCGF3 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA2B has 0 variants, skipping\n",
      "[INFO]\tGene HERPUD1 has 0 variants, skipping\n",
      "[INFO]\tGene TEDC1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT3 has 0 variants, skipping\n",
      "[INFO]\tGene SPTY2D1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN23 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC21 has 0 variants, skipping\n",
      "[INFO]\tGene INO80C has 0 variants, skipping\n",
      "[INFO]\tGene ISCA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM219 has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP82 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705G has 0 variants, skipping\n",
      "[INFO]\tGene C6orf132 has 0 variants, skipping\n",
      "[INFO]\tGene CBY3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD31 has 0 variants, skipping\n",
      "[INFO]\tGene SHISAL2B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD33B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM212 has 0 variants, skipping\n",
      "[INFO]\tGene DAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM255A has 0 variants, skipping\n",
      "[INFO]\tGene PPT1 has 0 variants, skipping\n",
      "[INFO]\tGene GDI2 has 0 variants, skipping\n",
      "[INFO]\tGene CD300LD has 0 variants, skipping\n",
      "[INFO]\tGene TBK1 has 0 variants, skipping\n",
      "[INFO]\tGene YBX1 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01606 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY3 has 0 variants, skipping\n",
      "[INFO]\tGene CLK1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX46 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP11 has 0 variants, skipping\n",
      "[INFO]\tGene PTENP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP16 has 0 variants, skipping\n",
      "[INFO]\tGene JARID2 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01561 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB12 has 0 variants, skipping\n",
      "[INFO]\tGene ATRNL1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CMA1 has 0 variants, skipping\n",
      "[INFO]\tGene DTWD2 has 0 variants, skipping\n",
      "[INFO]\tGene BECN2 has 0 variants, skipping\n",
      "[INFO]\tGene NSF has 0 variants, skipping\n",
      "[INFO]\tGene NCAPG2 has 0 variants, skipping\n",
      "[INFO]\tGene MYOM2 has 0 variants, skipping\n",
      "[INFO]\tGene MTG1 has 0 variants, skipping\n",
      "[INFO]\tGene MTSS2 has 0 variants, skipping\n",
      "[INFO]\tGene RIPPLY1 has 0 variants, skipping\n",
      "[INFO]\tGene FDXACB1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC5 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD6 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM23B-AGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene NOLC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4686 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4651 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4437 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4423 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH9 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2-AS has 0 variants, skipping\n",
      "[INFO]\tGene PKDCC has 0 variants, skipping\n",
      "[INFO]\tGene AP5B1 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3A has 0 variants, skipping\n",
      "[INFO]\tGene CD160 has 0 variants, skipping\n",
      "[INFO]\tGene UPK1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3184 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC7A has 0 variants, skipping\n",
      "[INFO]\tGene CAPN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L has 0 variants, skipping\n",
      "[INFO]\tGene IGF1R has 0 variants, skipping\n",
      "[INFO]\tGene C11orf80 has 0 variants, skipping\n",
      "[INFO]\tGene MIR641 has 0 variants, skipping\n",
      "[INFO]\tGene MIR640 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP69 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAH has 0 variants, skipping\n",
      "[INFO]\tGene WNT10B has 0 variants, skipping\n",
      "[INFO]\tGene YY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF251 has 0 variants, skipping\n",
      "[INFO]\tGene MFHAS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR50 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2L6 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11B has 0 variants, skipping\n",
      "[INFO]\tGene MYH9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR575 has 0 variants, skipping\n",
      "[INFO]\tGene RPLP0 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1324 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1322 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1277 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1269A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1266 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548G has 0 variants, skipping\n",
      "[INFO]\tGene MIR1253 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L1 has 0 variants, skipping\n",
      "[INFO]\tGene H6PD has 0 variants, skipping\n",
      "[INFO]\tGene CMTM5 has 0 variants, skipping\n",
      "[INFO]\tGene SHPK has 0 variants, skipping\n",
      "[INFO]\tGene GPR83 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6C has 0 variants, skipping\n",
      "[INFO]\tGene SUMO3 has 0 variants, skipping\n",
      "[INFO]\tGene YBEY has 0 variants, skipping\n",
      "[INFO]\tGene CFAP54 has 0 variants, skipping\n",
      "[INFO]\tGene MIR487A has 0 variants, skipping\n",
      "[INFO]\tGene MDH1B has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36 has 0 variants, skipping\n",
      "[INFO]\tGene WNT9B has 0 variants, skipping\n",
      "[INFO]\tGene WNT9A has 0 variants, skipping\n",
      "[INFO]\tGene SCGB3A1 has 0 variants, skipping\n",
      "[INFO]\tGene RFT1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF28 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT3 has 0 variants, skipping\n",
      "[INFO]\tGene CDC37 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF22 has 0 variants, skipping\n",
      "[INFO]\tGene CNTF has 0 variants, skipping\n",
      "[INFO]\tGene MIR633 has 0 variants, skipping\n",
      "[INFO]\tGene PAX1 has 0 variants, skipping\n",
      "[INFO]\tGene PA2G4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF625 has 0 variants, skipping\n",
      "[INFO]\tGene ASGR2 has 0 variants, skipping\n",
      "[INFO]\tGene CWC15 has 0 variants, skipping\n",
      "[INFO]\tGene RBM48 has 0 variants, skipping\n",
      "[INFO]\tGene PCSK1N has 0 variants, skipping\n",
      "[INFO]\tGene PRSS50 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS13 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M2 has 0 variants, skipping\n",
      "[INFO]\tGene AMH has 0 variants, skipping\n",
      "[INFO]\tGene NUF2 has 0 variants, skipping\n",
      "[INFO]\tGene COX8A has 0 variants, skipping\n",
      "[INFO]\tGene PLK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD5 has 0 variants, skipping\n",
      "[INFO]\tGene TNKS2 has 0 variants, skipping\n",
      "[INFO]\tGene CDH9 has 0 variants, skipping\n",
      "[INFO]\tGene ALX1 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A3 has 0 variants, skipping\n",
      "[INFO]\tGene DENND5A has 0 variants, skipping\n",
      "[INFO]\tGene MIR618 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT1 has 0 variants, skipping\n",
      "[INFO]\tGene CFP has 0 variants, skipping\n",
      "[INFO]\tGene PF4V1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL2 has 0 variants, skipping\n",
      "[INFO]\tGene PRIMA1 has 0 variants, skipping\n",
      "[INFO]\tGene MVB12B has 0 variants, skipping\n",
      "[INFO]\tGene HELZ2 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF10 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD2 has 0 variants, skipping\n",
      "[INFO]\tGene PPCDC has 0 variants, skipping\n",
      "[INFO]\tGene TICRR has 0 variants, skipping\n",
      "[INFO]\tGene TRIM6 has 0 variants, skipping\n",
      "[INFO]\tGene PROX2 has 0 variants, skipping\n",
      "[INFO]\tGene GALE has 0 variants, skipping\n",
      "[INFO]\tGene IL17C has 0 variants, skipping\n",
      "[INFO]\tGene DCPS has 0 variants, skipping\n",
      "[INFO]\tGene WDR37 has 0 variants, skipping\n",
      "[INFO]\tGene ALG3 has 0 variants, skipping\n",
      "[INFO]\tGene TAB2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP97 has 0 variants, skipping\n",
      "[INFO]\tGene TM2D3 has 0 variants, skipping\n",
      "[INFO]\tGene PPARA has 0 variants, skipping\n",
      "[INFO]\tGene C1orf54 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45GIP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3M has 0 variants, skipping\n",
      "[INFO]\tGene ZNF805 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2C has 0 variants, skipping\n",
      "[INFO]\tGene FAM71E2 has 0 variants, skipping\n",
      "[INFO]\tGene RASL12 has 0 variants, skipping\n",
      "[INFO]\tGene PLCZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF773 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL1 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L2 has 0 variants, skipping\n",
      "[INFO]\tGene COQ7 has 0 variants, skipping\n",
      "[INFO]\tGene LIPM has 0 variants, skipping\n",
      "[INFO]\tGene DAZ2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM204 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1J has 0 variants, skipping\n",
      "[INFO]\tGene IP6K1 has 0 variants, skipping\n",
      "[INFO]\tGene PMF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A7 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH8 has 0 variants, skipping\n",
      "[INFO]\tGene NRL has 0 variants, skipping\n",
      "[INFO]\tGene NRAP has 0 variants, skipping\n",
      "[INFO]\tGene NSMF has 0 variants, skipping\n",
      "[INFO]\tGene ST20-MTHFS has 0 variants, skipping\n",
      "[INFO]\tGene OR6C65 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K4 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K3 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H15 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H14 has 0 variants, skipping\n",
      "[INFO]\tGene SENP5 has 0 variants, skipping\n",
      "[INFO]\tGene UPRT has 0 variants, skipping\n",
      "[INFO]\tGene FBXL4 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL3 has 0 variants, skipping\n",
      "[INFO]\tGene SUFU has 0 variants, skipping\n",
      "[INFO]\tGene NIPA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL20 has 0 variants, skipping\n",
      "[INFO]\tGene NRDE2 has 0 variants, skipping\n",
      "[INFO]\tGene IWS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP7D1 has 0 variants, skipping\n",
      "[INFO]\tGene GPN2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37C has 0 variants, skipping\n",
      "[INFO]\tGene AKIRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM3 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61G has 0 variants, skipping\n",
      "[INFO]\tGene CECR2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP4 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G5 has 0 variants, skipping\n",
      "[INFO]\tGene PKP3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF788P has 0 variants, skipping\n",
      "[INFO]\tGene H2BC3 has 0 variants, skipping\n",
      "[INFO]\tGene CLSPN has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF124 has 0 variants, skipping\n",
      "[INFO]\tGene EBAG9 has 0 variants, skipping\n",
      "[INFO]\tGene MAN2B2 has 0 variants, skipping\n",
      "[INFO]\tGene NDST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN4 has 0 variants, skipping\n",
      "[INFO]\tGene GPX8 has 0 variants, skipping\n",
      "[INFO]\tGene CKAP2L has 0 variants, skipping\n",
      "[INFO]\tGene DYRK2 has 0 variants, skipping\n",
      "[INFO]\tGene RGS14 has 0 variants, skipping\n",
      "[INFO]\tGene ATN1 has 0 variants, skipping\n",
      "[INFO]\tGene PSLNR has 0 variants, skipping\n",
      "[INFO]\tGene AP3D1 has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC106 has 0 variants, skipping\n",
      "[INFO]\tGene FAM216A has 0 variants, skipping\n",
      "[INFO]\tGene PEDS1-UBE2V1 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH3 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS12 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM269 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36 has 0 variants, skipping\n",
      "[INFO]\tGene ARL9 has 0 variants, skipping\n",
      "[INFO]\tGene LTA4H has 0 variants, skipping\n",
      "[INFO]\tGene GTSF1L has 0 variants, skipping\n",
      "[INFO]\tGene FSTL5 has 0 variants, skipping\n",
      "[INFO]\tGene HSPE1-MOB4 has 0 variants, skipping\n",
      "[INFO]\tGene STARD4 has 0 variants, skipping\n",
      "[INFO]\tGene DYNAP has 0 variants, skipping\n",
      "[INFO]\tGene RAB26 has 0 variants, skipping\n",
      "[INFO]\tGene SHD has 0 variants, skipping\n",
      "[INFO]\tGene PHETA2 has 0 variants, skipping\n",
      "[INFO]\tGene HBM has 0 variants, skipping\n",
      "[INFO]\tGene LMNA has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX8 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23D2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM210 has 0 variants, skipping\n",
      "[INFO]\tGene APPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB10 has 0 variants, skipping\n",
      "[INFO]\tGene PGGT1B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP46 has 0 variants, skipping\n",
      "[INFO]\tGene STK10 has 0 variants, skipping\n",
      "[INFO]\tGene MSL2 has 0 variants, skipping\n",
      "[INFO]\tGene RPTOR has 0 variants, skipping\n",
      "[INFO]\tGene TMTC3 has 0 variants, skipping\n",
      "[INFO]\tGene SDR39U1 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2L has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5L has 0 variants, skipping\n",
      "[INFO]\tGene RAB23 has 0 variants, skipping\n",
      "[INFO]\tGene TTC1 has 0 variants, skipping\n",
      "[INFO]\tGene XAF1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC5 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17B has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC11 has 0 variants, skipping\n",
      "[INFO]\tGene DSCC1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf49 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A23 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM109 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD3 has 0 variants, skipping\n",
      "[INFO]\tGene PCK2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236A has 0 variants, skipping\n",
      "[INFO]\tGene AHCYL2 has 0 variants, skipping\n",
      "[INFO]\tGene NXPE1 has 0 variants, skipping\n",
      "[INFO]\tGene CST1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM32 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC58 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT9B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A13 has 0 variants, skipping\n",
      "[INFO]\tGene TEX35 has 0 variants, skipping\n",
      "[INFO]\tGene APOA4 has 0 variants, skipping\n",
      "[INFO]\tGene MMADHC-DT has 0 variants, skipping\n",
      "[INFO]\tGene HAS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00423 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1A3P has 0 variants, skipping\n",
      "[INFO]\tGene OR2L13 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CD86 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLA has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC4 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP6 has 0 variants, skipping\n",
      "[INFO]\tGene TERB2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB35 has 0 variants, skipping\n",
      "[INFO]\tGene IL10RB has 0 variants, skipping\n",
      "[INFO]\tGene NOS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL15 has 0 variants, skipping\n",
      "[INFO]\tGene SPHKAP has 0 variants, skipping\n",
      "[INFO]\tGene SON has 0 variants, skipping\n",
      "[INFO]\tGene ZNF764 has 0 variants, skipping\n",
      "[INFO]\tGene FOXS1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP5 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12 has 0 variants, skipping\n",
      "[INFO]\tGene FGF11 has 0 variants, skipping\n",
      "[INFO]\tGene SMG1 has 0 variants, skipping\n",
      "[INFO]\tGene ELAPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF221 has 0 variants, skipping\n",
      "[INFO]\tGene PADI1 has 0 variants, skipping\n",
      "[INFO]\tGene COMP has 0 variants, skipping\n",
      "[INFO]\tGene COL7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLK has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN3 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP77 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2K has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC9 has 0 variants, skipping\n",
      "[INFO]\tGene JMY has 0 variants, skipping\n",
      "[INFO]\tGene CALHM4 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2B has 0 variants, skipping\n",
      "[INFO]\tGene CDK5 has 0 variants, skipping\n",
      "[INFO]\tGene POLRMT has 0 variants, skipping\n",
      "[INFO]\tGene TOB1 has 0 variants, skipping\n",
      "[INFO]\tGene SPHK2 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC11 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA7 has 0 variants, skipping\n",
      "[INFO]\tGene MPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRE has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRL has 0 variants, skipping\n",
      "[INFO]\tGene PCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFRA has 0 variants, skipping\n",
      "[INFO]\tGene PDE6H has 0 variants, skipping\n",
      "[INFO]\tGene PAX9 has 0 variants, skipping\n",
      "[INFO]\tGene PDE6C has 0 variants, skipping\n",
      "[INFO]\tGene CLIP4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20DC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCP4 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP3CC has 0 variants, skipping\n",
      "[INFO]\tGene G3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene SIK2 has 0 variants, skipping\n",
      "[INFO]\tGene CDA has 0 variants, skipping\n",
      "[INFO]\tGene DOCK3 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12 has 0 variants, skipping\n",
      "[INFO]\tGene MITD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A53 has 0 variants, skipping\n",
      "[INFO]\tGene DDX1 has 0 variants, skipping\n",
      "[INFO]\tGene PPID has 0 variants, skipping\n",
      "[INFO]\tGene IHH has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE4 has 0 variants, skipping\n",
      "[INFO]\tGene AOPEP has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A3 has 0 variants, skipping\n",
      "[INFO]\tGene TONSL has 0 variants, skipping\n",
      "[INFO]\tGene ACO1 has 0 variants, skipping\n",
      "[INFO]\tGene INSM1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA5 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C68 has 0 variants, skipping\n",
      "[INFO]\tGene NSG2 has 0 variants, skipping\n",
      "[INFO]\tGene GLOD4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM64 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC4-KLRK1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU2 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF138 has 0 variants, skipping\n",
      "[INFO]\tGene TULP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM53A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH7A1 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST3 has 0 variants, skipping\n",
      "[INFO]\tGene HOMER1 has 0 variants, skipping\n",
      "[INFO]\tGene TTF1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PIAS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR9 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS18A has 0 variants, skipping\n",
      "[INFO]\tGene CENPS has 0 variants, skipping\n",
      "[INFO]\tGene RBMY2EP has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD27 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440570 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY2 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4E has 0 variants, skipping\n",
      "[INFO]\tGene NOS1 has 0 variants, skipping\n",
      "[INFO]\tGene ESF1 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXN1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS12 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1I has 0 variants, skipping\n",
      "[INFO]\tGene FGF6 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRP1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAL4 has 0 variants, skipping\n",
      "[INFO]\tGene CRTC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00628 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA10 has 0 variants, skipping\n",
      "[INFO]\tGene RETREG1 has 0 variants, skipping\n",
      "[INFO]\tGene OCM has 0 variants, skipping\n",
      "[INFO]\tGene LEFTY1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302B has 0 variants, skipping\n",
      "[INFO]\tGene SAA4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2G2 has 0 variants, skipping\n",
      "[INFO]\tGene ERVH48-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSLTR2 has 0 variants, skipping\n",
      "[INFO]\tGene NAA50 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP28 has 0 variants, skipping\n",
      "[INFO]\tGene TSR2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DRAIC has 0 variants, skipping\n",
      "[INFO]\tGene POC1B has 0 variants, skipping\n",
      "[INFO]\tGene MYCBP has 0 variants, skipping\n",
      "[INFO]\tGene MIR3666 has 0 variants, skipping\n",
      "[INFO]\tGene GLI2 has 0 variants, skipping\n",
      "[INFO]\tGene MORC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01874 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01875 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373352 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01115 has 0 variants, skipping\n",
      "[INFO]\tGene SNTG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS26 has 0 variants, skipping\n",
      "[INFO]\tGene GPR45 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH2 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH3 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR176 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX1 has 0 variants, skipping\n",
      "[INFO]\tGene CA5B has 0 variants, skipping\n",
      "[INFO]\tGene RNF139 has 0 variants, skipping\n",
      "[INFO]\tGene HPS5 has 0 variants, skipping\n",
      "[INFO]\tGene POLG2 has 0 variants, skipping\n",
      "[INFO]\tGene SEC63 has 0 variants, skipping\n",
      "[INFO]\tGene PRAF2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF2 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01250 has 0 variants, skipping\n",
      "[INFO]\tGene MSH2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01304 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373394 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01249 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01248 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01810 has 0 variants, skipping\n",
      "[INFO]\tGene SILC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7158 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400940 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01247 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01246 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7515HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR7515 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01857 has 0 variants, skipping\n",
      "[INFO]\tGene CHORDC1P4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929452 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506274 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929551 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01814 has 0 variants, skipping\n",
      "[INFO]\tGene G6PC has 0 variants, skipping\n",
      "[INFO]\tGene FUCA1 has 0 variants, skipping\n",
      "[INFO]\tGene FTL has 0 variants, skipping\n",
      "[INFO]\tGene FSHR has 0 variants, skipping\n",
      "[INFO]\tGene FXN has 0 variants, skipping\n",
      "[INFO]\tGene FH has 0 variants, skipping\n",
      "[INFO]\tGene MS4A2 has 0 variants, skipping\n",
      "[INFO]\tGene FBN1 has 0 variants, skipping\n",
      "[INFO]\tGene HACD1 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM6 has 0 variants, skipping\n",
      "[INFO]\tGene VAT1L has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A9 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene XK has 0 variants, skipping\n",
      "[INFO]\tGene CLDN9 has 0 variants, skipping\n",
      "[INFO]\tGene HPS6 has 0 variants, skipping\n",
      "[INFO]\tGene HHIPL2 has 0 variants, skipping\n",
      "[INFO]\tGene SHCBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR212 has 0 variants, skipping\n",
      "[INFO]\tGene MIR208B has 0 variants, skipping\n",
      "[INFO]\tGene MIR211 has 0 variants, skipping\n",
      "[INFO]\tGene MIR210 has 0 variants, skipping\n",
      "[INFO]\tGene MIR216B has 0 variants, skipping\n",
      "[INFO]\tGene MIR205 has 0 variants, skipping\n",
      "[INFO]\tGene MIR301B has 0 variants, skipping\n",
      "[INFO]\tGene MIR204 has 0 variants, skipping\n",
      "[INFO]\tGene MIR760 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO2A has 0 variants, skipping\n",
      "[INFO]\tGene DDX59-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCMAP-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00993 has 0 variants, skipping\n",
      "[INFO]\tGene MEIOC has 0 variants, skipping\n",
      "[INFO]\tGene AARSD1 has 0 variants, skipping\n",
      "[INFO]\tGene CD38 has 0 variants, skipping\n",
      "[INFO]\tGene CD37 has 0 variants, skipping\n",
      "[INFO]\tGene CD34 has 0 variants, skipping\n",
      "[INFO]\tGene CD33 has 0 variants, skipping\n",
      "[INFO]\tGene CD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929882 has 0 variants, skipping\n",
      "[INFO]\tGene MYCBPAP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80B has 0 variants, skipping\n",
      "[INFO]\tGene ODC1-DT has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL832P has 0 variants, skipping\n",
      "[INFO]\tGene PDCD11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01954 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ33534 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR3 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR2 has 0 variants, skipping\n",
      "[INFO]\tGene CALB1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00570 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD9 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC57 has 0 variants, skipping\n",
      "[INFO]\tGene NSD1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX17 has 0 variants, skipping\n",
      "[INFO]\tGene RNF25 has 0 variants, skipping\n",
      "[INFO]\tGene NOC3L has 0 variants, skipping\n",
      "[INFO]\tGene RHBDF1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB17 has 0 variants, skipping\n",
      "[INFO]\tGene TTC36 has 0 variants, skipping\n",
      "[INFO]\tGene OTOL1 has 0 variants, skipping\n",
      "[INFO]\tGene HSF5 has 0 variants, skipping\n",
      "[INFO]\tGene A3GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548S has 0 variants, skipping\n",
      "[INFO]\tGene MIR3681HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3681 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3125 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506474 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00276 has 0 variants, skipping\n",
      "[INFO]\tGene RHCG has 0 variants, skipping\n",
      "[INFO]\tGene KCNH7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR151A has 0 variants, skipping\n",
      "[INFO]\tGene GNL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01804 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf98 has 0 variants, skipping\n",
      "[INFO]\tGene ASPG has 0 variants, skipping\n",
      "[INFO]\tGene GP9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO4 has 0 variants, skipping\n",
      "[INFO]\tGene CD274 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01376 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4757 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00954 has 0 variants, skipping\n",
      "[INFO]\tGene DLK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928222 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIQK has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985856 has 0 variants, skipping\n",
      "[INFO]\tGene HS1BP3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene CC2D1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01822 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01884 has 0 variants, skipping\n",
      "[INFO]\tGene BRD1 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC30 has 0 variants, skipping\n",
      "[INFO]\tGene SYNDIG1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A2 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51AP2 has 0 variants, skipping\n",
      "[INFO]\tGene YAP1 has 0 variants, skipping\n",
      "[INFO]\tGene POGZ has 0 variants, skipping\n",
      "[INFO]\tGene CT45A8 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC187 has 0 variants, skipping\n",
      "[INFO]\tGene SMG6 has 0 variants, skipping\n",
      "[INFO]\tGene CNTLN has 0 variants, skipping\n",
      "[INFO]\tGene MUC2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC171 has 0 variants, skipping\n",
      "[INFO]\tGene SKA2 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KA has 0 variants, skipping\n",
      "[INFO]\tGene ASRGL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC27-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01381 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF37 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL4 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT5 has 0 variants, skipping\n",
      "[INFO]\tGene RAB22A has 0 variants, skipping\n",
      "[INFO]\tGene FBXO27 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM4 has 0 variants, skipping\n",
      "[INFO]\tGene ARL5B has 0 variants, skipping\n",
      "[INFO]\tGene SMOX has 0 variants, skipping\n",
      "[INFO]\tGene TRANK1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA3 has 0 variants, skipping\n",
      "[INFO]\tGene LGI1 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf17 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT3 has 0 variants, skipping\n",
      "[INFO]\tGene CPLX1 has 0 variants, skipping\n",
      "[INFO]\tGene UTP14A has 0 variants, skipping\n",
      "[INFO]\tGene CPLX2 has 0 variants, skipping\n",
      "[INFO]\tGene STOML3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31E1 has 0 variants, skipping\n",
      "[INFO]\tGene RBCK1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSLTR1 has 0 variants, skipping\n",
      "[INFO]\tGene DCTN4 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1B1 has 0 variants, skipping\n",
      "[INFO]\tGene HECA has 0 variants, skipping\n",
      "[INFO]\tGene DBR1 has 0 variants, skipping\n",
      "[INFO]\tGene CUTALP has 0 variants, skipping\n",
      "[INFO]\tGene HAUS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL34-DT has 0 variants, skipping\n",
      "[INFO]\tGene TINCR has 0 variants, skipping\n",
      "[INFO]\tGene AGBL5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR1B-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND5 has 0 variants, skipping\n",
      "[INFO]\tGene CDK12 has 0 variants, skipping\n",
      "[INFO]\tGene FOLR1 has 0 variants, skipping\n",
      "[INFO]\tGene PIEZO1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB37 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS2 has 0 variants, skipping\n",
      "[INFO]\tGene SPTSSB has 0 variants, skipping\n",
      "[INFO]\tGene TRIM71 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MC1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C2 has 0 variants, skipping\n",
      "[INFO]\tGene COPS9 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM6-TRIM34 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B7 has 0 variants, skipping\n",
      "[INFO]\tGene CISH has 0 variants, skipping\n",
      "[INFO]\tGene SNX8 has 0 variants, skipping\n",
      "[INFO]\tGene HCFC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZW10 has 0 variants, skipping\n",
      "[INFO]\tGene LRAT has 0 variants, skipping\n",
      "[INFO]\tGene MTA2 has 0 variants, skipping\n",
      "[INFO]\tGene TIAF1 has 0 variants, skipping\n",
      "[INFO]\tGene VAPB has 0 variants, skipping\n",
      "[INFO]\tGene XPR1 has 0 variants, skipping\n",
      "[INFO]\tGene PDE2A has 0 variants, skipping\n",
      "[INFO]\tGene SMAD5 has 0 variants, skipping\n",
      "[INFO]\tGene CNP has 0 variants, skipping\n",
      "[INFO]\tGene ZIC5 has 0 variants, skipping\n",
      "[INFO]\tGene WNT3A has 0 variants, skipping\n",
      "[INFO]\tGene SCRT2 has 0 variants, skipping\n",
      "[INFO]\tGene PSKH2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A16 has 0 variants, skipping\n",
      "[INFO]\tGene CA4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01460 has 0 variants, skipping\n",
      "[INFO]\tGene PNPT1 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM7 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC13 has 0 variants, skipping\n",
      "[INFO]\tGene ADSS2 has 0 variants, skipping\n",
      "[INFO]\tGene AKAP8L has 0 variants, skipping\n",
      "[INFO]\tGene MIR4263 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505736 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505716 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ31356 has 0 variants, skipping\n",
      "[INFO]\tGene MYH8 has 0 variants, skipping\n",
      "[INFO]\tGene MYH6 has 0 variants, skipping\n",
      "[INFO]\tGene MYH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00320 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5701-1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD92 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD53 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD53B has 0 variants, skipping\n",
      "[INFO]\tGene CHMP5 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM144 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374389 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA10B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01936 has 0 variants, skipping\n",
      "[INFO]\tGene CCT3 has 0 variants, skipping\n",
      "[INFO]\tGene FHOD1 has 0 variants, skipping\n",
      "[INFO]\tGene N6AMT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R3B has 0 variants, skipping\n",
      "[INFO]\tGene ACRV1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01946 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF142 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A48-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOK has 0 variants, skipping\n",
      "[INFO]\tGene KAT8 has 0 variants, skipping\n",
      "[INFO]\tGene ESPN has 0 variants, skipping\n",
      "[INFO]\tGene NRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene IL36G has 0 variants, skipping\n",
      "[INFO]\tGene GKN1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR45B has 0 variants, skipping\n",
      "[INFO]\tGene IRGC has 0 variants, skipping\n",
      "[INFO]\tGene CPXM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR558 has 0 variants, skipping\n",
      "[INFO]\tGene BIRC6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4765 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00486 has 0 variants, skipping\n",
      "[INFO]\tGene FAM168B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4430 has 0 variants, skipping\n",
      "[INFO]\tGene CRHBP has 0 variants, skipping\n",
      "[INFO]\tGene CR2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA6 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01317 has 0 variants, skipping\n",
      "[INFO]\tGene MYADML has 0 variants, skipping\n",
      "[INFO]\tGene LINC01318 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01320 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AD has 0 variants, skipping\n",
      "[INFO]\tGene CRIM1-DT has 0 variants, skipping\n",
      "[INFO]\tGene PTPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MED16 has 0 variants, skipping\n",
      "[INFO]\tGene JRKL has 0 variants, skipping\n",
      "[INFO]\tGene EXTL3 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf18 has 0 variants, skipping\n",
      "[INFO]\tGene ERVV-1 has 0 variants, skipping\n",
      "[INFO]\tGene TMC8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM199 has 0 variants, skipping\n",
      "[INFO]\tGene EME1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35G3 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf89 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf77 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF785 has 0 variants, skipping\n",
      "[INFO]\tGene KCNU1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00211 has 0 variants, skipping\n",
      "[INFO]\tGene RMDN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNN3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CYP1B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO5A has 0 variants, skipping\n",
      "[INFO]\tGene GPRASP1 has 0 variants, skipping\n",
      "[INFO]\tGene EHBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01883 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02613 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2D has 0 variants, skipping\n",
      "[INFO]\tGene SMR3A has 0 variants, skipping\n",
      "[INFO]\tGene LOC375196 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K3-DT has 0 variants, skipping\n",
      "[INFO]\tGene AGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene DAZL has 0 variants, skipping\n",
      "[INFO]\tGene ORC1 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT31 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01913 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01914 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01819 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02580 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01126 has 0 variants, skipping\n",
      "[INFO]\tGene RFX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02476 has 0 variants, skipping\n",
      "[INFO]\tGene CSHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01833 has 0 variants, skipping\n",
      "[INFO]\tGene SIX3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01121 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01820 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02583 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01119 has 0 variants, skipping\n",
      "[INFO]\tGene KRAS has 0 variants, skipping\n",
      "[INFO]\tGene HBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520F has 0 variants, skipping\n",
      "[INFO]\tGene VCY1B has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPCAM-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR559 has 0 variants, skipping\n",
      "[INFO]\tGene TBX10 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD23 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM190 has 0 variants, skipping\n",
      "[INFO]\tGene RCE1 has 0 variants, skipping\n",
      "[INFO]\tGene STT3A has 0 variants, skipping\n",
      "[INFO]\tGene C12orf77 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF473 has 0 variants, skipping\n",
      "[INFO]\tGene MMACHC has 0 variants, skipping\n",
      "[INFO]\tGene LINC01867 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3682 has 0 variants, skipping\n",
      "[INFO]\tGene PEBP4 has 0 variants, skipping\n",
      "[INFO]\tGene ACHE has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP32 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE3 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK11 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF622 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC46 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF662 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN7 has 0 variants, skipping\n",
      "[INFO]\tGene AXDND1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C75 has 0 variants, skipping\n",
      "[INFO]\tGene FPGT has 0 variants, skipping\n",
      "[INFO]\tGene FOXB2 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4B has 0 variants, skipping\n",
      "[INFO]\tGene FAM47C has 0 variants, skipping\n",
      "[INFO]\tGene OR4F15 has 0 variants, skipping\n",
      "[INFO]\tGene PRORSD1P has 0 variants, skipping\n",
      "[INFO]\tGene MNDA has 0 variants, skipping\n",
      "[INFO]\tGene MIR217HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC01813 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129434 has 0 variants, skipping\n",
      "[INFO]\tGene SHPRH has 0 variants, skipping\n",
      "[INFO]\tGene USP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01795 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01122 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4432HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4432 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01185 has 0 variants, skipping\n",
      "[INFO]\tGene DBT has 0 variants, skipping\n",
      "[INFO]\tGene CCER2 has 0 variants, skipping\n",
      "[INFO]\tGene VCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339803 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4E2 has 0 variants, skipping\n",
      "[INFO]\tGene AHSA2P has 0 variants, skipping\n",
      "[INFO]\tGene XIRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70B has 0 variants, skipping\n",
      "[INFO]\tGene SPARCL1 has 0 variants, skipping\n",
      "[INFO]\tGene C1D has 0 variants, skipping\n",
      "[INFO]\tGene MIR5192 has 0 variants, skipping\n",
      "[INFO]\tGene KPRP has 0 variants, skipping\n",
      "[INFO]\tGene CHI3L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929372 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf76 has 0 variants, skipping\n",
      "[INFO]\tGene MTCP1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF5 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL13 has 0 variants, skipping\n",
      "[INFO]\tGene ERFE has 0 variants, skipping\n",
      "[INFO]\tGene ABCC4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132215 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP20 has 0 variants, skipping\n",
      "[INFO]\tGene DBIL5P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCT7 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ8 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ4 has 0 variants, skipping\n",
      "[INFO]\tGene XBP1 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4433B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4433A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927402 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4434 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02579 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01800 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02245 has 0 variants, skipping\n",
      "[INFO]\tGene WASHC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4778 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01873 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01798 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01799 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01628 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01828 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01829 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927661 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01812 has 0 variants, skipping\n",
      "[INFO]\tGene GPR143 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD2B has 0 variants, skipping\n",
      "[INFO]\tGene AURKAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CIAO3 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA5 has 0 variants, skipping\n",
      "[INFO]\tGene PLAGL2 has 0 variants, skipping\n",
      "[INFO]\tGene PLAG1 has 0 variants, skipping\n",
      "[INFO]\tGene EFEMP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3126 has 0 variants, skipping\n",
      "[INFO]\tGene PARVB has 0 variants, skipping\n",
      "[INFO]\tGene RNF224 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36C has 0 variants, skipping\n",
      "[INFO]\tGene CLDN8 has 0 variants, skipping\n",
      "[INFO]\tGene HENMT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01816 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf92 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01224 has 0 variants, skipping\n",
      "[INFO]\tGene EXOG has 0 variants, skipping\n",
      "[INFO]\tGene ABCA12 has 0 variants, skipping\n",
      "[INFO]\tGene GFOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01354 has 0 variants, skipping\n",
      "[INFO]\tGene TRHR has 0 variants, skipping\n",
      "[INFO]\tGene TGFA-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LEFTY2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01143 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1B1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E91P has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC9B has 0 variants, skipping\n",
      "[INFO]\tGene TOMM20L has 0 variants, skipping\n",
      "[INFO]\tGene CPT2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL7B has 0 variants, skipping\n",
      "[INFO]\tGene METTL15 has 0 variants, skipping\n",
      "[INFO]\tGene OIT3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47B has 0 variants, skipping\n",
      "[INFO]\tGene SNX31 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF366 has 0 variants, skipping\n",
      "[INFO]\tGene MIER3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM60 has 0 variants, skipping\n",
      "[INFO]\tGene RNF168 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM42 has 0 variants, skipping\n",
      "[INFO]\tGene TEX44 has 0 variants, skipping\n",
      "[INFO]\tGene WBP2NL has 0 variants, skipping\n",
      "[INFO]\tGene LRRN4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC185 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD6L has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF10 has 0 variants, skipping\n",
      "[INFO]\tGene SDE2 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP1 has 0 variants, skipping\n",
      "[INFO]\tGene EXD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF781 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD4 has 0 variants, skipping\n",
      "[INFO]\tGene SPRED1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF709 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf50 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF579 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf65 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A2 has 0 variants, skipping\n",
      "[INFO]\tGene MOSPD2 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1NB has 0 variants, skipping\n",
      "[INFO]\tGene CBLL2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39B has 0 variants, skipping\n",
      "[INFO]\tGene RASEF has 0 variants, skipping\n",
      "[INFO]\tGene C9orf163 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0D2 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL27 has 0 variants, skipping\n",
      "[INFO]\tGene BMT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP48 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM81B has 0 variants, skipping\n",
      "[INFO]\tGene BTNL9 has 0 variants, skipping\n",
      "[INFO]\tGene SRFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf45 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1K has 0 variants, skipping\n",
      "[INFO]\tGene TEX55 has 0 variants, skipping\n",
      "[INFO]\tGene SCFD2 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR941-5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02729 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4497 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4490 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4465 has 0 variants, skipping\n",
      "[INFO]\tGene PLXND1 has 0 variants, skipping\n",
      "[INFO]\tGene DAAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZER1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR564 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4775 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4496 has 0 variants, skipping\n",
      "[INFO]\tGene PRADC1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPSL has 0 variants, skipping\n",
      "[INFO]\tGene JCHAIN has 0 variants, skipping\n",
      "[INFO]\tGene SPATA4 has 0 variants, skipping\n",
      "[INFO]\tGene SCLT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene ALMS1P1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8B has 0 variants, skipping\n",
      "[INFO]\tGene ULK2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf12 has 0 variants, skipping\n",
      "[INFO]\tGene MELTF has 0 variants, skipping\n",
      "[INFO]\tGene PNMA1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB7 has 0 variants, skipping\n",
      "[INFO]\tGene TMCC3 has 0 variants, skipping\n",
      "[INFO]\tGene CD247 has 0 variants, skipping\n",
      "[INFO]\tGene DGUOK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VCX3A has 0 variants, skipping\n",
      "[INFO]\tGene VCX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR297 has 0 variants, skipping\n",
      "[INFO]\tGene ANKLE1 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO4 has 0 variants, skipping\n",
      "[INFO]\tGene AWAT2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS37 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM89 has 0 variants, skipping\n",
      "[INFO]\tGene MISP has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB41 has 0 variants, skipping\n",
      "[INFO]\tGene STX11 has 0 variants, skipping\n",
      "[INFO]\tGene INO80B-WBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SETD5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR889 has 0 variants, skipping\n",
      "[INFO]\tGene MIR147A has 0 variants, skipping\n",
      "[INFO]\tGene MIR139 has 0 variants, skipping\n",
      "[INFO]\tGene MIR892B has 0 variants, skipping\n",
      "[INFO]\tGene MIR874 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450B has 0 variants, skipping\n",
      "[INFO]\tGene MIR199A1 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC has 0 variants, skipping\n",
      "[INFO]\tGene CHORDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01291 has 0 variants, skipping\n",
      "[INFO]\tGene CDK2AP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5000 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927884 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927967 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01851 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-H has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927948 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374820 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCC1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNB3 has 0 variants, skipping\n",
      "[INFO]\tGene DDOST has 0 variants, skipping\n",
      "[INFO]\tGene DCC has 0 variants, skipping\n",
      "[INFO]\tGene MIR4264 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8080 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01815 has 0 variants, skipping\n",
      "[INFO]\tGene DHFRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01809 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA4 has 0 variants, skipping\n",
      "[INFO]\tGene PWWP2B has 0 variants, skipping\n",
      "[INFO]\tGene SEZ6 has 0 variants, skipping\n",
      "[INFO]\tGene HS2ST1 has 0 variants, skipping\n",
      "[INFO]\tGene GBP5 has 0 variants, skipping\n",
      "[INFO]\tGene AFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01964 has 0 variants, skipping\n",
      "[INFO]\tGene TCF7L1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724579 has 0 variants, skipping\n",
      "[INFO]\tGene NBEAL2 has 0 variants, skipping\n",
      "[INFO]\tGene CRISP2 has 0 variants, skipping\n",
      "[INFO]\tGene TCFL5 has 0 variants, skipping\n",
      "[INFO]\tGene BPTF has 0 variants, skipping\n",
      "[INFO]\tGene CDY2B has 0 variants, skipping\n",
      "[INFO]\tGene OSGIN2 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H6 has 0 variants, skipping\n",
      "[INFO]\tGene NGFR has 0 variants, skipping\n",
      "[INFO]\tGene NEK2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL28 has 0 variants, skipping\n",
      "[INFO]\tGene GIPC2 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF586 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R9A has 0 variants, skipping\n",
      "[INFO]\tGene FTSJ3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6071 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284950 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD94 has 0 variants, skipping\n",
      "[INFO]\tGene QSER1 has 0 variants, skipping\n",
      "[INFO]\tGene CA3 has 0 variants, skipping\n",
      "[INFO]\tGene BMI1 has 0 variants, skipping\n",
      "[INFO]\tGene IDH3A has 0 variants, skipping\n",
      "[INFO]\tGene HSF1 has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4771-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4771-2 has 0 variants, skipping\n",
      "[INFO]\tGene ERBB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB561A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4780 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928371 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO2B1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK3-DT has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927050 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P4 has 0 variants, skipping\n",
      "[INFO]\tGene GGT8P has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3BP2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P14 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133920 has 0 variants, skipping\n",
      "[INFO]\tGene FAM95A has 0 variants, skipping\n",
      "[INFO]\tGene LOC442028 has 0 variants, skipping\n",
      "[INFO]\tGene INSRR has 0 variants, skipping\n",
      "[INFO]\tGene C3orf49 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2CP has 0 variants, skipping\n",
      "[INFO]\tGene HYI has 0 variants, skipping\n",
      "[INFO]\tGene MRPS35 has 0 variants, skipping\n",
      "[INFO]\tGene STARD7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA112 has 0 variants, skipping\n",
      "[INFO]\tGene OR4M2 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373496 has 0 variants, skipping\n",
      "[INFO]\tGene MCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN11 has 0 variants, skipping\n",
      "[INFO]\tGene KANSL1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372440 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2B2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC2 has 0 variants, skipping\n",
      "[INFO]\tGene PROSER2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2A has 0 variants, skipping\n",
      "[INFO]\tGene CNNM3-DT has 0 variants, skipping\n",
      "[INFO]\tGene EPAS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927053 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN5 has 0 variants, skipping\n",
      "[INFO]\tGene FTCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506076 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506123 has 0 variants, skipping\n",
      "[INFO]\tGene TGM6 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf92 has 0 variants, skipping\n",
      "[INFO]\tGene OLFML1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10C has 0 variants, skipping\n",
      "[INFO]\tGene LINC02611 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00304 has 0 variants, skipping\n",
      "[INFO]\tGene ILF3-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR662 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX3 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST3B1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO16 has 0 variants, skipping\n",
      "[INFO]\tGene HMBS has 0 variants, skipping\n",
      "[INFO]\tGene TSKU has 0 variants, skipping\n",
      "[INFO]\tGene SMIM18 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF19 has 0 variants, skipping\n",
      "[INFO]\tGene GPRACR has 0 variants, skipping\n",
      "[INFO]\tGene DCLK2 has 0 variants, skipping\n",
      "[INFO]\tGene TCF24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01104 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01849 has 0 variants, skipping\n",
      "[INFO]\tGene AMER1 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL2 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L10 has 0 variants, skipping\n",
      "[INFO]\tGene ALS2CL has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5696 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF394 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01127 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4X has 0 variants, skipping\n",
      "[INFO]\tGene IFT74 has 0 variants, skipping\n",
      "[INFO]\tGene LSM12 has 0 variants, skipping\n",
      "[INFO]\tGene SRP72 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB34 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC43 has 0 variants, skipping\n",
      "[INFO]\tGene CUEDC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4772 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA11 has 0 variants, skipping\n",
      "[INFO]\tGene RETN has 0 variants, skipping\n",
      "[INFO]\tGene FABP2 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01796 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01935 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287010 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01831 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01102 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01103 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01114 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01159 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01918 has 0 variants, skipping\n",
      "[INFO]\tGene NGEF has 0 variants, skipping\n",
      "[INFO]\tGene LOC285000 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLA has 0 variants, skipping\n",
      "[INFO]\tGene OR4K5 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4K15 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F5 has 0 variants, skipping\n",
      "[INFO]\tGene OR13G1 has 0 variants, skipping\n",
      "[INFO]\tGene ILDR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AU has 0 variants, skipping\n",
      "[INFO]\tGene LINC01886 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A38 has 0 variants, skipping\n",
      "[INFO]\tGene ASB6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01594 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1C2P1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QL2 has 0 variants, skipping\n",
      "[INFO]\tGene CABP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM229B has 0 variants, skipping\n",
      "[INFO]\tGene GCC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VTCN1 has 0 variants, skipping\n",
      "[INFO]\tGene CABP2 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA1 has 0 variants, skipping\n",
      "[INFO]\tGene PHC2 has 0 variants, skipping\n",
      "[INFO]\tGene PHC1 has 0 variants, skipping\n",
      "[INFO]\tGene ECM1 has 0 variants, skipping\n",
      "[INFO]\tGene E4F1 has 0 variants, skipping\n",
      "[INFO]\tGene DVL3 has 0 variants, skipping\n",
      "[INFO]\tGene RBMX2 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2J1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00161 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193BHG has 0 variants, skipping\n",
      "[INFO]\tGene MGAT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF22-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4265 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4266 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268219 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4S2 has 0 variants, skipping\n",
      "[INFO]\tGene OR8K5 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM118B has 0 variants, skipping\n",
      "[INFO]\tGene CERS4 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS3-LOC440895 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440895 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288570 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01123 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4267 has 0 variants, skipping\n",
      "[INFO]\tGene TMED5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436B2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF667-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507334 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01106 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD2B has 0 variants, skipping\n",
      "[INFO]\tGene AHNAK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373553 has 0 variants, skipping\n",
      "[INFO]\tGene ALAS1 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD132 has 0 variants, skipping\n",
      "[INFO]\tGene ACOXL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AIF1L has 0 variants, skipping\n",
      "[INFO]\tGene NUDC has 0 variants, skipping\n",
      "[INFO]\tGene NFATC3 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2R has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC13 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF3 has 0 variants, skipping\n",
      "[INFO]\tGene COG4 has 0 variants, skipping\n",
      "[INFO]\tGene PHF20 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373562 has 0 variants, skipping\n",
      "[INFO]\tGene CHAC2 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF8 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42351 has 0 variants, skipping\n",
      "[INFO]\tGene PET117 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC1B has 0 variants, skipping\n",
      "[INFO]\tGene LRWD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A11 has 0 variants, skipping\n",
      "[INFO]\tGene UROD has 0 variants, skipping\n",
      "[INFO]\tGene UMPS has 0 variants, skipping\n",
      "[INFO]\tGene TYR has 0 variants, skipping\n",
      "[INFO]\tGene TTR has 0 variants, skipping\n",
      "[INFO]\tGene TTPA has 0 variants, skipping\n",
      "[INFO]\tGene TSHR has 0 variants, skipping\n",
      "[INFO]\tGene TRIM23 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC12 has 0 variants, skipping\n",
      "[INFO]\tGene CSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene HYOU1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM138B has 0 variants, skipping\n",
      "[INFO]\tGene WASH2P has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP7 has 0 variants, skipping\n",
      "[INFO]\tGene TLR4 has 0 variants, skipping\n",
      "[INFO]\tGene TLR3 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE3 has 0 variants, skipping\n",
      "[INFO]\tGene MFN2 has 0 variants, skipping\n",
      "[INFO]\tGene R3HCC1 has 0 variants, skipping\n",
      "[INFO]\tGene METTL11B has 0 variants, skipping\n",
      "[INFO]\tGene ATXN3L has 0 variants, skipping\n",
      "[INFO]\tGene SCRN2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2B3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4782 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01191 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100499194 has 0 variants, skipping\n",
      "[INFO]\tGene TARP has 0 variants, skipping\n",
      "[INFO]\tGene POTEE has 0 variants, skipping\n",
      "[INFO]\tGene DPP10-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene DPP10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A11 has 0 variants, skipping\n",
      "[INFO]\tGene MDFIC has 0 variants, skipping\n",
      "[INFO]\tGene RNF212B has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985939 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927709 has 0 variants, skipping\n",
      "[INFO]\tGene EVI2B has 0 variants, skipping\n",
      "[INFO]\tGene STEAP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC6 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OGFOD3 has 0 variants, skipping\n",
      "[INFO]\tGene ITGAX has 0 variants, skipping\n",
      "[INFO]\tGene TRIP13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01101 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00908 has 0 variants, skipping\n",
      "[INFO]\tGene CASC15 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL22 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01826 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985820 has 0 variants, skipping\n",
      "[INFO]\tGene KARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01941 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM35B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287896 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS7 has 0 variants, skipping\n",
      "[INFO]\tGene HSBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene BET1L has 0 variants, skipping\n",
      "[INFO]\tGene SYT17 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB11 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT12 has 0 variants, skipping\n",
      "[INFO]\tGene NODAL has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX5 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD3 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4783 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373609 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA4 has 0 variants, skipping\n",
      "[INFO]\tGene MAG has 0 variants, skipping\n",
      "[INFO]\tGene MAFK has 0 variants, skipping\n",
      "[INFO]\tGene MAFG has 0 variants, skipping\n",
      "[INFO]\tGene MAD2L1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCKS has 0 variants, skipping\n",
      "[INFO]\tGene M6PR has 0 variants, skipping\n",
      "[INFO]\tGene EPCAM has 0 variants, skipping\n",
      "[INFO]\tGene TACSTD2 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D1A has 0 variants, skipping\n",
      "[INFO]\tGene LYN has 0 variants, skipping\n",
      "[INFO]\tGene LY75 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB6 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD4 has 0 variants, skipping\n",
      "[INFO]\tGene CRTAM has 0 variants, skipping\n",
      "[INFO]\tGene GPCPD1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RC has 0 variants, skipping\n",
      "[INFO]\tGene PARM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927881 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01854 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02572 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC9P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01856 has 0 variants, skipping\n",
      "[INFO]\tGene HCAR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC13A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAR2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MED15P9 has 0 variants, skipping\n",
      "[INFO]\tGene CEP152 has 0 variants, skipping\n",
      "[INFO]\tGene RAG2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR183 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF844 has 0 variants, skipping\n",
      "[INFO]\tGene CDK4 has 0 variants, skipping\n",
      "[INFO]\tGene CD40LG has 0 variants, skipping\n",
      "[INFO]\tGene FAR2P2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F62P has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A10 has 0 variants, skipping\n",
      "[INFO]\tGene UNG has 0 variants, skipping\n",
      "[INFO]\tGene DBN1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F30P has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK12 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01120 has 0 variants, skipping\n",
      "[INFO]\tGene NOC2LP2 has 0 variants, skipping\n",
      "[INFO]\tGene CTRB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4784 has 0 variants, skipping\n",
      "[INFO]\tGene GINM1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF653 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-4 has 0 variants, skipping\n",
      "[INFO]\tGene POTEKP has 0 variants, skipping\n",
      "[INFO]\tGene IRAG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTF3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF9 has 0 variants, skipping\n",
      "[INFO]\tGene CHD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01945 has 0 variants, skipping\n",
      "[INFO]\tGene FAM201B has 0 variants, skipping\n",
      "[INFO]\tGene KIF4B has 0 variants, skipping\n",
      "[INFO]\tGene USP20 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5IF1 has 0 variants, skipping\n",
      "[INFO]\tGene NME1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCKAP5-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCNT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT5A has 0 variants, skipping\n",
      "[INFO]\tGene PWP2 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40B has 0 variants, skipping\n",
      "[INFO]\tGene MMP8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132E has 0 variants, skipping\n",
      "[INFO]\tGene PRPSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT7B has 0 variants, skipping\n",
      "[INFO]\tGene MIR302C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928273 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01832 has 0 variants, skipping\n",
      "[INFO]\tGene YY1P2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1 has 0 variants, skipping\n",
      "[INFO]\tGene ERN2 has 0 variants, skipping\n",
      "[INFO]\tGene TTYH2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7157 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928386 has 0 variants, skipping\n",
      "[INFO]\tGene MST1L has 0 variants, skipping\n",
      "[INFO]\tGene C5AR2 has 0 variants, skipping\n",
      "[INFO]\tGene APEX2 has 0 variants, skipping\n",
      "[INFO]\tGene FHOD3 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2C has 0 variants, skipping\n",
      "[INFO]\tGene BTK has 0 variants, skipping\n",
      "[INFO]\tGene BRCA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01412 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373656 has 0 variants, skipping\n",
      "[INFO]\tGene TEX41 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1P2 has 0 variants, skipping\n",
      "[INFO]\tGene PROC has 0 variants, skipping\n",
      "[INFO]\tGene PKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TICAM1 has 0 variants, skipping\n",
      "[INFO]\tGene TRNT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM50A has 0 variants, skipping\n",
      "[INFO]\tGene PDE7B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01931 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01817 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01920 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02612 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929319 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4773-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4773-2 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1217 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF561 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF562 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN29 has 0 variants, skipping\n",
      "[INFO]\tGene FAM81A has 0 variants, skipping\n",
      "[INFO]\tGene DCST1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF362 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC27 has 0 variants, skipping\n",
      "[INFO]\tGene CST6 has 0 variants, skipping\n",
      "[INFO]\tGene CST2 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1G2 has 0 variants, skipping\n",
      "[INFO]\tGene CSH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK14 has 0 variants, skipping\n",
      "[INFO]\tGene CRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CREBL2 has 0 variants, skipping\n",
      "[INFO]\tGene CNN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNGA3 has 0 variants, skipping\n",
      "[INFO]\tGene CPN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNGB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR2 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN3 has 0 variants, skipping\n",
      "[INFO]\tGene CCR1 has 0 variants, skipping\n",
      "[INFO]\tGene CLPTM1 has 0 variants, skipping\n",
      "[INFO]\tGene LRP3 has 0 variants, skipping\n",
      "[INFO]\tGene LRP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6X1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM66 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3J has 0 variants, skipping\n",
      "[INFO]\tGene MPPED1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXO6 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB11 has 0 variants, skipping\n",
      "[INFO]\tGene RAPH1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01876 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46875 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3A_B has 0 variants, skipping\n",
      "[INFO]\tGene FAM133DP has 0 variants, skipping\n",
      "[INFO]\tGene CENPX has 0 variants, skipping\n",
      "[INFO]\tGene P3H2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC148-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AXIN2 has 0 variants, skipping\n",
      "[INFO]\tGene USP9Y has 0 variants, skipping\n",
      "[INFO]\tGene PKP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC14B has 0 variants, skipping\n",
      "[INFO]\tGene TENM3 has 0 variants, skipping\n",
      "[INFO]\tGene GRIA3 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6888 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM128 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF337 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL35 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643072 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDD2 has 0 variants, skipping\n",
      "[INFO]\tGene SIDT2 has 0 variants, skipping\n",
      "[INFO]\tGene LETM1 has 0 variants, skipping\n",
      "[INFO]\tGene LDOC1 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA6 has 0 variants, skipping\n",
      "[INFO]\tGene KLK9 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02478 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF737 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4785 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929512 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01806 has 0 variants, skipping\n",
      "[INFO]\tGene AHCTF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A6 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL1 has 0 variants, skipping\n",
      "[INFO]\tGene HEY2 has 0 variants, skipping\n",
      "[INFO]\tGene HEY1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF212 has 0 variants, skipping\n",
      "[INFO]\tGene XRN2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1E1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI3 has 0 variants, skipping\n",
      "[INFO]\tGene CRNKL1 has 0 variants, skipping\n",
      "[INFO]\tGene GOT2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132B has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RITA1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM45 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC2C has 0 variants, skipping\n",
      "[INFO]\tGene CCDC142 has 0 variants, skipping\n",
      "[INFO]\tGene PARM1 has 0 variants, skipping\n",
      "[INFO]\tGene AADAT has 0 variants, skipping\n",
      "[INFO]\tGene TFRC has 0 variants, skipping\n",
      "[INFO]\tGene PADI3 has 0 variants, skipping\n",
      "[INFO]\tGene USPL1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX26 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929532 has 0 variants, skipping\n",
      "[INFO]\tGene NXF3 has 0 variants, skipping\n",
      "[INFO]\tGene EGLN1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR88 has 0 variants, skipping\n",
      "[INFO]\tGene BCAN has 0 variants, skipping\n",
      "[INFO]\tGene PRSS57 has 0 variants, skipping\n",
      "[INFO]\tGene MIR944 has 0 variants, skipping\n",
      "[INFO]\tGene SOX6 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2 has 0 variants, skipping\n",
      "[INFO]\tGene NCR2 has 0 variants, skipping\n",
      "[INFO]\tGene PHACTR3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70F has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929633 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP12 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM2 has 0 variants, skipping\n",
      "[INFO]\tGene ELP5 has 0 variants, skipping\n",
      "[INFO]\tGene GNAZ has 0 variants, skipping\n",
      "[INFO]\tGene ZCWPW1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF358 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD4 has 0 variants, skipping\n",
      "[INFO]\tGene SRBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506124 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf42 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN3 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR5 has 0 variants, skipping\n",
      "[INFO]\tGene TTC21B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724058 has 0 variants, skipping\n",
      "[INFO]\tGene KLKB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF624 has 0 variants, skipping\n",
      "[INFO]\tGene TBL2 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX2 has 0 variants, skipping\n",
      "[INFO]\tGene SCN1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SENP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4774 has 0 variants, skipping\n",
      "[INFO]\tGene CERS6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM61 has 0 variants, skipping\n",
      "[INFO]\tGene MYT1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA13 has 0 variants, skipping\n",
      "[INFO]\tGene BZW1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3K has 0 variants, skipping\n",
      "[INFO]\tGene LRRC7 has 0 variants, skipping\n",
      "[INFO]\tGene MAFG-DT has 0 variants, skipping\n",
      "[INFO]\tGene MYO3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926913 has 0 variants, skipping\n",
      "[INFO]\tGene VDAC3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD26 has 0 variants, skipping\n",
      "[INFO]\tGene DCXR has 0 variants, skipping\n",
      "[INFO]\tGene AK6 has 0 variants, skipping\n",
      "[INFO]\tGene FOXK2 has 0 variants, skipping\n",
      "[INFO]\tGene ONECUT1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXA3 has 0 variants, skipping\n",
      "[INFO]\tGene HUS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRP8 has 0 variants, skipping\n",
      "[INFO]\tGene UFL1 has 0 variants, skipping\n",
      "[INFO]\tGene FEM1B has 0 variants, skipping\n",
      "[INFO]\tGene BFAR has 0 variants, skipping\n",
      "[INFO]\tGene POLR2E has 0 variants, skipping\n",
      "[INFO]\tGene GPR42 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK1 has 0 variants, skipping\n",
      "[INFO]\tGene CKLF has 0 variants, skipping\n",
      "[INFO]\tGene FIG4 has 0 variants, skipping\n",
      "[INFO]\tGene TECPR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A3 has 0 variants, skipping\n",
      "[INFO]\tGene NOL9 has 0 variants, skipping\n",
      "[INFO]\tGene PRKRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DLX2-DT has 0 variants, skipping\n",
      "[INFO]\tGene GPR182 has 0 variants, skipping\n",
      "[INFO]\tGene COPE has 0 variants, skipping\n",
      "[INFO]\tGene RTRAF has 0 variants, skipping\n",
      "[INFO]\tGene SBDS has 0 variants, skipping\n",
      "[INFO]\tGene NMD3 has 0 variants, skipping\n",
      "[INFO]\tGene RAPGEF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R1A has 0 variants, skipping\n",
      "[INFO]\tGene PERCC1 has 0 variants, skipping\n",
      "[INFO]\tGene BMPER has 0 variants, skipping\n",
      "[INFO]\tGene LOC112694756 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01960 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01305 has 0 variants, skipping\n",
      "[INFO]\tGene WDR81 has 0 variants, skipping\n",
      "[INFO]\tGene MROH9 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF4 has 0 variants, skipping\n",
      "[INFO]\tGene HIRA has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGC4 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC74B has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA11 has 0 variants, skipping\n",
      "[INFO]\tGene CYLC1 has 0 variants, skipping\n",
      "[INFO]\tGene CRY2 has 0 variants, skipping\n",
      "[INFO]\tGene TPM4 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401021 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf53 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7704 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01117 has 0 variants, skipping\n",
      "[INFO]\tGene NUP35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4444-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4444-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130691 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6512 has 0 variants, skipping\n",
      "[INFO]\tGene GAP43 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385C has 0 variants, skipping\n",
      "[INFO]\tGene DGKI has 0 variants, skipping\n",
      "[INFO]\tGene OBI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NADSYN1 has 0 variants, skipping\n",
      "[INFO]\tGene BACE2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF6 has 0 variants, skipping\n",
      "[INFO]\tGene CHST2 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT11 has 0 variants, skipping\n",
      "[INFO]\tGene SLC4A1AP has 0 variants, skipping\n",
      "[INFO]\tGene ASF1B has 0 variants, skipping\n",
      "[INFO]\tGene ANTXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR11 has 0 variants, skipping\n",
      "[INFO]\tGene PVR has 0 variants, skipping\n",
      "[INFO]\tGene TRAF1 has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene LYPLA2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC1B has 0 variants, skipping\n",
      "[INFO]\tGene SCLY has 0 variants, skipping\n",
      "[INFO]\tGene MTFP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL51 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL27 has 0 variants, skipping\n",
      "[INFO]\tGene RNF181 has 0 variants, skipping\n",
      "[INFO]\tGene RSC1A1 has 0 variants, skipping\n",
      "[INFO]\tGene RELB has 0 variants, skipping\n",
      "[INFO]\tGene MCL1 has 0 variants, skipping\n",
      "[INFO]\tGene MACC1 has 0 variants, skipping\n",
      "[INFO]\tGene SUMF1 has 0 variants, skipping\n",
      "[INFO]\tGene BUB1 has 0 variants, skipping\n",
      "[INFO]\tGene BST2 has 0 variants, skipping\n",
      "[INFO]\tGene BST1 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD4 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD4 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP3L has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOR2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC28A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H7A has 0 variants, skipping\n",
      "[INFO]\tGene UBE2T has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927055 has 0 variants, skipping\n",
      "[INFO]\tGene SUV39H1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7L1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO9A has 0 variants, skipping\n",
      "[INFO]\tGene EGOT has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4B has 0 variants, skipping\n",
      "[INFO]\tGene ACSM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01934 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM1 has 0 variants, skipping\n",
      "[INFO]\tGene BDP1 has 0 variants, skipping\n",
      "[INFO]\tGene CBX3 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP10 has 0 variants, skipping\n",
      "[INFO]\tGene MYBL1 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1LI1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR99A has 0 variants, skipping\n",
      "[INFO]\tGene CEP162 has 0 variants, skipping\n",
      "[INFO]\tGene CRLS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRADD has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H12A has 0 variants, skipping\n",
      "[INFO]\tGene FBXW4 has 0 variants, skipping\n",
      "[INFO]\tGene FST has 0 variants, skipping\n",
      "[INFO]\tGene MARVELD2 has 0 variants, skipping\n",
      "[INFO]\tGene DLX1 has 0 variants, skipping\n",
      "[INFO]\tGene SIM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AE1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373782 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01473 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN2 has 0 variants, skipping\n",
      "[INFO]\tGene PITPNC1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3GAP2 has 0 variants, skipping\n",
      "[INFO]\tGene QPCT has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01090 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR1245A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1245B has 0 variants, skipping\n",
      "[INFO]\tGene SNORC has 0 variants, skipping\n",
      "[INFO]\tGene ZNF790 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A10 has 0 variants, skipping\n",
      "[INFO]\tGene HSD3B7 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP7 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNB has 0 variants, skipping\n",
      "[INFO]\tGene OSGEPL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP74 has 0 variants, skipping\n",
      "[INFO]\tGene ENG has 0 variants, skipping\n",
      "[INFO]\tGene ZEB2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A18 has 0 variants, skipping\n",
      "[INFO]\tGene ORC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101059915 has 0 variants, skipping\n",
      "[INFO]\tGene NGF has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS8 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS4 has 0 variants, skipping\n",
      "[INFO]\tGene NFX1 has 0 variants, skipping\n",
      "[INFO]\tGene SMDT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01821 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01790 has 0 variants, skipping\n",
      "[INFO]\tGene HECW2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130452 has 0 variants, skipping\n",
      "[INFO]\tGene HAND2 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN3 has 0 variants, skipping\n",
      "[INFO]\tGene GMPPB has 0 variants, skipping\n",
      "[INFO]\tGene NR0B2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMTOR3 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR6 has 0 variants, skipping\n",
      "[INFO]\tGene PIEZO2 has 0 variants, skipping\n",
      "[INFO]\tGene MT1JP has 0 variants, skipping\n",
      "[INFO]\tGene ZFAS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFC has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene GNPDA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA105B has 0 variants, skipping\n",
      "[INFO]\tGene SAMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP4 has 0 variants, skipping\n",
      "[INFO]\tGene WAS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01923 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01877 has 0 variants, skipping\n",
      "[INFO]\tGene PRRG3 has 0 variants, skipping\n",
      "[INFO]\tGene RIN3 has 0 variants, skipping\n",
      "[INFO]\tGene RPL18A has 0 variants, skipping\n",
      "[INFO]\tGene ALDH5A1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM40 has 0 variants, skipping\n",
      "[INFO]\tGene AOX3P-AOX2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01792 has 0 variants, skipping\n",
      "[INFO]\tGene LTO1 has 0 variants, skipping\n",
      "[INFO]\tGene MED19 has 0 variants, skipping\n",
      "[INFO]\tGene ESX1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP5 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene BEND5 has 0 variants, skipping\n",
      "[INFO]\tGene CMKLR1 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375556 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02429 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927278 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM231 has 0 variants, skipping\n",
      "[INFO]\tGene GK5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02664 has 0 variants, skipping\n",
      "[INFO]\tGene CFLAR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCN9 has 0 variants, skipping\n",
      "[INFO]\tGene BMP5 has 0 variants, skipping\n",
      "[INFO]\tGene AQP6 has 0 variants, skipping\n",
      "[INFO]\tGene AQP5 has 0 variants, skipping\n",
      "[INFO]\tGene ARF6 has 0 variants, skipping\n",
      "[INFO]\tGene ARF5 has 0 variants, skipping\n",
      "[INFO]\tGene ARL4D has 0 variants, skipping\n",
      "[INFO]\tGene ARF3 has 0 variants, skipping\n",
      "[INFO]\tGene ARF4 has 0 variants, skipping\n",
      "[INFO]\tGene ARF1 has 0 variants, skipping\n",
      "[INFO]\tGene AREG has 0 variants, skipping\n",
      "[INFO]\tGene ARCN1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA2012-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD70 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD70B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD11B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD11 has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2B has 0 variants, skipping\n",
      "[INFO]\tGene OPN1MW has 0 variants, skipping\n",
      "[INFO]\tGene FUT2 has 0 variants, skipping\n",
      "[INFO]\tGene FGG has 0 variants, skipping\n",
      "[INFO]\tGene FGA has 0 variants, skipping\n",
      "[INFO]\tGene CYP11B2 has 0 variants, skipping\n",
      "[INFO]\tGene FBP1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP11B1 has 0 variants, skipping\n",
      "[INFO]\tGene F2 has 0 variants, skipping\n",
      "[INFO]\tGene KERA has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A3 has 0 variants, skipping\n",
      "[INFO]\tGene RER1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT10 has 0 variants, skipping\n",
      "[INFO]\tGene FAM3B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM11A has 0 variants, skipping\n",
      "[INFO]\tGene PSAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA4 has 0 variants, skipping\n",
      "[INFO]\tGene ALAS2 has 0 variants, skipping\n",
      "[INFO]\tGene HPGDS has 0 variants, skipping\n",
      "[INFO]\tGene MOCS3 has 0 variants, skipping\n",
      "[INFO]\tGene BMP10 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG5 has 0 variants, skipping\n",
      "[INFO]\tGene DHDH has 0 variants, skipping\n",
      "[INFO]\tGene SMPDL3B has 0 variants, skipping\n",
      "[INFO]\tGene SPINK4 has 0 variants, skipping\n",
      "[INFO]\tGene RND1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL2 has 0 variants, skipping\n",
      "[INFO]\tGene VENTX has 0 variants, skipping\n",
      "[INFO]\tGene SRPX2 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT2 has 0 variants, skipping\n",
      "[INFO]\tGene VASH1 has 0 variants, skipping\n",
      "[INFO]\tGene GCSHP3 has 0 variants, skipping\n",
      "[INFO]\tGene RPP40 has 0 variants, skipping\n",
      "[INFO]\tGene HMGCL has 0 variants, skipping\n",
      "[INFO]\tGene SNORD51 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA41 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02731 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P9 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2B has 0 variants, skipping\n",
      "[INFO]\tGene GRIK3 has 0 variants, skipping\n",
      "[INFO]\tGene CIC has 0 variants, skipping\n",
      "[INFO]\tGene MIR3130-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3130-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7845 has 0 variants, skipping\n",
      "[INFO]\tGene MYOSLID-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01802 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-4 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF3 has 0 variants, skipping\n",
      "[INFO]\tGene RICTOR has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507443 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM40 has 0 variants, skipping\n",
      "[INFO]\tGene IDH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM185A has 0 variants, skipping\n",
      "[INFO]\tGene SLC35C1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927960 has 0 variants, skipping\n",
      "[INFO]\tGene RYBP has 0 variants, skipping\n",
      "[INFO]\tGene C7orf33 has 0 variants, skipping\n",
      "[INFO]\tGene PIDD1 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR3 has 0 variants, skipping\n",
      "[INFO]\tGene SYCE2 has 0 variants, skipping\n",
      "[INFO]\tGene URAD has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01878 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4776-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4776-2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01953 has 0 variants, skipping\n",
      "[INFO]\tGene MYCNUT has 0 variants, skipping\n",
      "[INFO]\tGene GATA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4438 has 0 variants, skipping\n",
      "[INFO]\tGene VWC2L-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOX4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70I has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A2 has 0 variants, skipping\n",
      "[INFO]\tGene ASXL2 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD1 has 0 variants, skipping\n",
      "[INFO]\tGene EHF has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724849 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00607 has 0 variants, skipping\n",
      "[INFO]\tGene GPR84 has 0 variants, skipping\n",
      "[INFO]\tGene FSCN3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01963 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01280 has 0 variants, skipping\n",
      "[INFO]\tGene DVL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM133B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373876 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01921 has 0 variants, skipping\n",
      "[INFO]\tGene DIRC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC3 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA115 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6513 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6810 has 0 variants, skipping\n",
      "[INFO]\tGene CATIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD8 has 0 variants, skipping\n",
      "[INFO]\tGene CSTF2T has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D3 has 0 variants, skipping\n",
      "[INFO]\tGene RBAK-RBAKDN has 0 variants, skipping\n",
      "[INFO]\tGene VCX3B has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9500 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00608 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129175 has 0 variants, skipping\n",
      "[INFO]\tGene TERF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A7 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA8 has 0 variants, skipping\n",
      "[INFO]\tGene PTTG2 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3131 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2A has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L9 has 0 variants, skipping\n",
      "[INFO]\tGene AAK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A1 has 0 variants, skipping\n",
      "[INFO]\tGene SPC25 has 0 variants, skipping\n",
      "[INFO]\tGene NEK1 has 0 variants, skipping\n",
      "[INFO]\tGene MVB12A has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02418 has 0 variants, skipping\n",
      "[INFO]\tGene RHBDL2 has 0 variants, skipping\n",
      "[INFO]\tGene NPY4R has 0 variants, skipping\n",
      "[INFO]\tGene MIR3132 has 0 variants, skipping\n",
      "[INFO]\tGene NELL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN30 has 0 variants, skipping\n",
      "[INFO]\tGene FAM236B has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01803 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4268 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3B has 0 variants, skipping\n",
      "[INFO]\tGene PDZK1 has 0 variants, skipping\n",
      "[INFO]\tGene RABL6 has 0 variants, skipping\n",
      "[INFO]\tGene MCTP1 has 0 variants, skipping\n",
      "[INFO]\tGene CT75 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB4Y has 0 variants, skipping\n",
      "[INFO]\tGene LIPF has 0 variants, skipping\n",
      "[INFO]\tGene SOX14 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3F has 0 variants, skipping\n",
      "[INFO]\tGene TPST1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB9 has 0 variants, skipping\n",
      "[INFO]\tGene SGCZ has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC10 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC8 has 0 variants, skipping\n",
      "[INFO]\tGene CDK13 has 0 variants, skipping\n",
      "[INFO]\tGene VIRMA has 0 variants, skipping\n",
      "[INFO]\tGene UBXN11 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4439 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646736 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5702 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF695 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP99 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5703 has 0 variants, skipping\n",
      "[INFO]\tGene PRELID2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2L has 0 variants, skipping\n",
      "[INFO]\tGene LOC729968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01807 has 0 variants, skipping\n",
      "[INFO]\tGene XKR5 has 0 variants, skipping\n",
      "[INFO]\tGene GLP2R has 0 variants, skipping\n",
      "[INFO]\tGene TGM5 has 0 variants, skipping\n",
      "[INFO]\tGene CD163 has 0 variants, skipping\n",
      "[INFO]\tGene TRIP11 has 0 variants, skipping\n",
      "[INFO]\tGene KLF4 has 0 variants, skipping\n",
      "[INFO]\tGene SOS2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM124A has 0 variants, skipping\n",
      "[INFO]\tGene PYCR3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A37 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLM1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD2 has 0 variants, skipping\n",
      "[INFO]\tGene LGSN has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC2 has 0 variants, skipping\n",
      "[INFO]\tGene TBX3 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP3 has 0 variants, skipping\n",
      "[INFO]\tGene BCCIP has 0 variants, skipping\n",
      "[INFO]\tGene COA4 has 0 variants, skipping\n",
      "[INFO]\tGene CEND1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR7 has 0 variants, skipping\n",
      "[INFO]\tGene SCAND1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP62 has 0 variants, skipping\n",
      "[INFO]\tGene TM7SF3 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN11 has 0 variants, skipping\n",
      "[INFO]\tGene MAN2B1 has 0 variants, skipping\n",
      "[INFO]\tGene OMP has 0 variants, skipping\n",
      "[INFO]\tGene PCSK5 has 0 variants, skipping\n",
      "[INFO]\tGene CUL9 has 0 variants, skipping\n",
      "[INFO]\tGene DDN has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01907 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC12 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM74B has 0 variants, skipping\n",
      "[INFO]\tGene MRLN has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHO1 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4777 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA75 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD82 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00471 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC24 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL10 has 0 variants, skipping\n",
      "[INFO]\tGene SEBOX has 0 variants, skipping\n",
      "[INFO]\tGene MIR1471 has 0 variants, skipping\n",
      "[INFO]\tGene MIR562 has 0 variants, skipping\n",
      "[INFO]\tGene ECEL1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC17 has 0 variants, skipping\n",
      "[INFO]\tGene STARD10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5001 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6D has 0 variants, skipping\n",
      "[INFO]\tGene ARL5C has 0 variants, skipping\n",
      "[INFO]\tGene STT3B has 0 variants, skipping\n",
      "[INFO]\tGene POP4 has 0 variants, skipping\n",
      "[INFO]\tGene COX8C has 0 variants, skipping\n",
      "[INFO]\tGene COPRS has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928881 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC151 has 0 variants, skipping\n",
      "[INFO]\tGene RBSN has 0 variants, skipping\n",
      "[INFO]\tGene UMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX3 has 0 variants, skipping\n",
      "[INFO]\tGene NSL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA5 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA6 has 0 variants, skipping\n",
      "[INFO]\tGene ENAM has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4H has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR3 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100286922 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGN has 0 variants, skipping\n",
      "[INFO]\tGene RNF145 has 0 variants, skipping\n",
      "[INFO]\tGene TTC23L has 0 variants, skipping\n",
      "[INFO]\tGene ZMAT2 has 0 variants, skipping\n",
      "[INFO]\tGene MSL3P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01891 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01173 has 0 variants, skipping\n",
      "[INFO]\tGene LONRF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF80 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC17 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF614 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD53 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC93463 has 0 variants, skipping\n",
      "[INFO]\tGene RAD50 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSP2 has 0 variants, skipping\n",
      "[INFO]\tGene PPIF has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL11 has 0 variants, skipping\n",
      "[INFO]\tGene SRI-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COMETT has 0 variants, skipping\n",
      "[INFO]\tGene PLIN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6811 has 0 variants, skipping\n",
      "[INFO]\tGene ACSM2A has 0 variants, skipping\n",
      "[INFO]\tGene MEP1B has 0 variants, skipping\n",
      "[INFO]\tGene TRAK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1B1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2F-SCLY has 0 variants, skipping\n",
      "[INFO]\tGene TM9SF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02610 has 0 variants, skipping\n",
      "[INFO]\tGene SUZ12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01107 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373759 has 0 variants, skipping\n",
      "[INFO]\tGene HPX has 0 variants, skipping\n",
      "[INFO]\tGene LINC01940 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4440 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4441 has 0 variants, skipping\n",
      "[INFO]\tGene MGC16025 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4269 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2467 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC150935 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4786 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A17 has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC18 has 0 variants, skipping\n",
      "[INFO]\tGene SLC23A1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN10-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC285191 has 0 variants, skipping\n",
      "[INFO]\tGene GPT2 has 0 variants, skipping\n",
      "[INFO]\tGene VWA5A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM171 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC18 has 0 variants, skipping\n",
      "[INFO]\tGene RBM15B has 0 variants, skipping\n",
      "[INFO]\tGene SLC9B2 has 0 variants, skipping\n",
      "[INFO]\tGene ORAI1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B3 has 0 variants, skipping\n",
      "[INFO]\tGene HCN4 has 0 variants, skipping\n",
      "[INFO]\tGene FRAT1 has 0 variants, skipping\n",
      "[INFO]\tGene INSL5 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1B has 0 variants, skipping\n",
      "[INFO]\tGene PDK4 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFB has 0 variants, skipping\n",
      "[INFO]\tGene C11orf21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3133 has 0 variants, skipping\n",
      "[INFO]\tGene BAG4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH9 has 0 variants, skipping\n",
      "[INFO]\tGene BOK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN7 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1H has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R8 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3A has 0 variants, skipping\n",
      "[INFO]\tGene PPP1CC has 0 variants, skipping\n",
      "[INFO]\tGene POU3F1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01237 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285097 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01880 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01881 has 0 variants, skipping\n",
      "[INFO]\tGene TCHH has 0 variants, skipping\n",
      "[INFO]\tGene NRSN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372493 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP1A-SDCBP2 has 0 variants, skipping\n",
      "[INFO]\tGene SDCBP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289473 has 0 variants, skipping\n",
      "[INFO]\tGene LBX2 has 0 variants, skipping\n",
      "[INFO]\tGene CBX7 has 0 variants, skipping\n",
      "[INFO]\tGene PDYN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRH has 0 variants, skipping\n",
      "[INFO]\tGene GNLY has 0 variants, skipping\n",
      "[INFO]\tGene SNORD119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1292 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA51 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD56 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD57 has 0 variants, skipping\n",
      "[INFO]\tGene FNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LIME1 has 0 variants, skipping\n",
      "[INFO]\tGene FCAMR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996637 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723730 has 0 variants, skipping\n",
      "[INFO]\tGene OVCH2 has 0 variants, skipping\n",
      "[INFO]\tGene CABCOCO1 has 0 variants, skipping\n",
      "[INFO]\tGene MT2A has 0 variants, skipping\n",
      "[INFO]\tGene UBOX5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L2 has 0 variants, skipping\n",
      "[INFO]\tGene NOM1 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA2 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC16 has 0 variants, skipping\n",
      "[INFO]\tGene JPH4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01730 has 0 variants, skipping\n",
      "[INFO]\tGene NANOGNB has 0 variants, skipping\n",
      "[INFO]\tGene PRNT has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2A has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1B has 0 variants, skipping\n",
      "[INFO]\tGene ADRA1D has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2B has 0 variants, skipping\n",
      "[INFO]\tGene ADH7 has 0 variants, skipping\n",
      "[INFO]\tGene ID2 has 0 variants, skipping\n",
      "[INFO]\tGene PCNA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00658 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643406 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00654 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01729 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA2 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf23 has 0 variants, skipping\n",
      "[INFO]\tGene KIF2B has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14B has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2E has 0 variants, skipping\n",
      "[INFO]\tGene MCM8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAIM has 0 variants, skipping\n",
      "[INFO]\tGene CASC20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01713 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01428 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01751 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01706 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8062 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU105B has 0 variants, skipping\n",
      "[INFO]\tGene GNGT2 has 0 variants, skipping\n",
      "[INFO]\tGene LAMP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CA6 has 0 variants, skipping\n",
      "[INFO]\tGene TAF11 has 0 variants, skipping\n",
      "[INFO]\tGene SNAP25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB10 has 0 variants, skipping\n",
      "[INFO]\tGene PGLS has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C3 has 0 variants, skipping\n",
      "[INFO]\tGene GLUD2 has 0 variants, skipping\n",
      "[INFO]\tGene FRAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6870 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929413 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00687 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01722 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102606466 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01723 has 0 variants, skipping\n",
      "[INFO]\tGene ISM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01605 has 0 variants, skipping\n",
      "[INFO]\tGene THOC7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS25 has 0 variants, skipping\n",
      "[INFO]\tGene BRMS1L has 0 variants, skipping\n",
      "[INFO]\tGene MXRA8 has 0 variants, skipping\n",
      "[INFO]\tGene FMO6P has 0 variants, skipping\n",
      "[INFO]\tGene CCDC82 has 0 variants, skipping\n",
      "[INFO]\tGene ASB13 has 0 variants, skipping\n",
      "[INFO]\tGene SNIP1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC16 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MACROD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC9 has 0 variants, skipping\n",
      "[INFO]\tGene REX1BD has 0 variants, skipping\n",
      "[INFO]\tGene LOC613266 has 0 variants, skipping\n",
      "[INFO]\tGene HEXB has 0 variants, skipping\n",
      "[INFO]\tGene CKM has 0 variants, skipping\n",
      "[INFO]\tGene CKB has 0 variants, skipping\n",
      "[INFO]\tGene CD3G has 0 variants, skipping\n",
      "[INFO]\tGene CLTB has 0 variants, skipping\n",
      "[INFO]\tGene SNORD17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A2 has 0 variants, skipping\n",
      "[INFO]\tGene MOAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEF1 has 0 variants, skipping\n",
      "[INFO]\tGene NPVF has 0 variants, skipping\n",
      "[INFO]\tGene LINC00851 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A11 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf122 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2V1 has 0 variants, skipping\n",
      "[INFO]\tGene FDX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3192 has 0 variants, skipping\n",
      "[INFO]\tGene DAD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00652 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00653 has 0 variants, skipping\n",
      "[INFO]\tGene NID2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRK1 has 0 variants, skipping\n",
      "[INFO]\tGene CASC3 has 0 variants, skipping\n",
      "[INFO]\tGene COG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00237 has 0 variants, skipping\n",
      "[INFO]\tGene AIM2 has 0 variants, skipping\n",
      "[INFO]\tGene KIZ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01727 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01726 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01432 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01427 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284788 has 0 variants, skipping\n",
      "[INFO]\tGene GPR35 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01747 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00656 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01431 has 0 variants, skipping\n",
      "[INFO]\tGene CST13P has 0 variants, skipping\n",
      "[INFO]\tGene PRDM11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01721 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM150B has 0 variants, skipping\n",
      "[INFO]\tGene SYNDIG1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35D2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926889 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SSNA1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASET2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF770 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J4 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF337-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372582 has 0 variants, skipping\n",
      "[INFO]\tGene FAM182B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926935 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01733 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134868 has 0 variants, skipping\n",
      "[INFO]\tGene FAM182A has 0 variants, skipping\n",
      "[INFO]\tGene MIR663AHG has 0 variants, skipping\n",
      "[INFO]\tGene FRG1CP has 0 variants, skipping\n",
      "[INFO]\tGene FRG1DP has 0 variants, skipping\n",
      "[INFO]\tGene FRG2EP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379477 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1EP has 0 variants, skipping\n",
      "[INFO]\tGene FAM242B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4477A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4477B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01597 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1BP has 0 variants, skipping\n",
      "[INFO]\tGene MLLT10P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARMT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUMA1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB122 has 0 variants, skipping\n",
      "[INFO]\tGene ABCB11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00028 has 0 variants, skipping\n",
      "[INFO]\tGene FGF7 has 0 variants, skipping\n",
      "[INFO]\tGene FGF9 has 0 variants, skipping\n",
      "[INFO]\tGene HM13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3193 has 0 variants, skipping\n",
      "[INFO]\tGene ABALON has 0 variants, skipping\n",
      "[INFO]\tGene REC114 has 0 variants, skipping\n",
      "[INFO]\tGene LEUTX has 0 variants, skipping\n",
      "[INFO]\tGene ZNF701 has 0 variants, skipping\n",
      "[INFO]\tGene SPTLC3 has 0 variants, skipping\n",
      "[INFO]\tGene TSC22D1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK7 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY26P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1825 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929698 has 0 variants, skipping\n",
      "[INFO]\tGene NOL4L-DT has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A8 has 0 variants, skipping\n",
      "[INFO]\tGene TERF2IP has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A10 has 0 variants, skipping\n",
      "[INFO]\tGene DLL4 has 0 variants, skipping\n",
      "[INFO]\tGene DPM3 has 0 variants, skipping\n",
      "[INFO]\tGene SGTB has 0 variants, skipping\n",
      "[INFO]\tGene ING3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR27 has 0 variants, skipping\n",
      "[INFO]\tGene DDX49 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA4P has 0 variants, skipping\n",
      "[INFO]\tGene CDH5 has 0 variants, skipping\n",
      "[INFO]\tGene CDX1 has 0 variants, skipping\n",
      "[INFO]\tGene CD52 has 0 variants, skipping\n",
      "[INFO]\tGene FEN1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF341-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RALY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4755 has 0 variants, skipping\n",
      "[INFO]\tGene OR2Y1 has 0 variants, skipping\n",
      "[INFO]\tGene MED8 has 0 variants, skipping\n",
      "[INFO]\tGene BSPRY has 0 variants, skipping\n",
      "[INFO]\tGene FOLR3 has 0 variants, skipping\n",
      "[INFO]\tGene TRPM7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71E1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM16 has 0 variants, skipping\n",
      "[INFO]\tGene HMGB3P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1205 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1229 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4261 has 0 variants, skipping\n",
      "[INFO]\tGene TSKS has 0 variants, skipping\n",
      "[INFO]\tGene AVPI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TADA2A has 0 variants, skipping\n",
      "[INFO]\tGene TDRD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1289-1 has 0 variants, skipping\n",
      "[INFO]\tGene GCNT7 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC3 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD4B has 0 variants, skipping\n",
      "[INFO]\tGene STAM2 has 0 variants, skipping\n",
      "[INFO]\tGene THBD has 0 variants, skipping\n",
      "[INFO]\tGene TGM1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK6 has 0 variants, skipping\n",
      "[INFO]\tGene NTN1 has 0 variants, skipping\n",
      "[INFO]\tGene KAT2A has 0 variants, skipping\n",
      "[INFO]\tGene CHKA has 0 variants, skipping\n",
      "[INFO]\tGene CYFIP2 has 0 variants, skipping\n",
      "[INFO]\tGene MCF2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130373 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA21 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA16 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA14 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA10 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA8 has 0 variants, skipping\n",
      "[INFO]\tGene FNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene LCN8 has 0 variants, skipping\n",
      "[INFO]\tGene GAD1 has 0 variants, skipping\n",
      "[INFO]\tGene SAG has 0 variants, skipping\n",
      "[INFO]\tGene RHO has 0 variants, skipping\n",
      "[INFO]\tGene RYR1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM201 has 0 variants, skipping\n",
      "[INFO]\tGene NR5A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRY has 0 variants, skipping\n",
      "[INFO]\tGene POP7 has 0 variants, skipping\n",
      "[INFO]\tGene RIDA has 0 variants, skipping\n",
      "[INFO]\tGene TIMM17B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00489 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287792 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF626 has 0 variants, skipping\n",
      "[INFO]\tGene SMPDL3A has 0 variants, skipping\n",
      "[INFO]\tGene SUB1 has 0 variants, skipping\n",
      "[INFO]\tGene FASTK has 0 variants, skipping\n",
      "[INFO]\tGene FCRL6 has 0 variants, skipping\n",
      "[INFO]\tGene MSH2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC149684 has 0 variants, skipping\n",
      "[INFO]\tGene TCTN2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71D has 0 variants, skipping\n",
      "[INFO]\tGene DRICH1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA71E has 0 variants, skipping\n",
      "[INFO]\tGene SNORA60 has 0 variants, skipping\n",
      "[INFO]\tGene GAREM2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548O2 has 0 variants, skipping\n",
      "[INFO]\tGene DLEC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01734 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01370 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD154 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128988 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RORA has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6871 has 0 variants, skipping\n",
      "[INFO]\tGene RGPD4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF467 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927159 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32A has 0 variants, skipping\n",
      "[INFO]\tGene WDR43 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN6 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D9 has 0 variants, skipping\n",
      "[INFO]\tGene MT1B has 0 variants, skipping\n",
      "[INFO]\tGene MT1A has 0 variants, skipping\n",
      "[INFO]\tGene ATP9A has 0 variants, skipping\n",
      "[INFO]\tGene GSTM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01728 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D28 has 0 variants, skipping\n",
      "[INFO]\tGene OSER1-DT has 0 variants, skipping\n",
      "[INFO]\tGene FPR3 has 0 variants, skipping\n",
      "[INFO]\tGene FPR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3646 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01430 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01620 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC47 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A20 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD2 has 0 variants, skipping\n",
      "[INFO]\tGene EML4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPGRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01260 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STK4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR12 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F1B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF735 has 0 variants, skipping\n",
      "[INFO]\tGene SYS1-DBNDD2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6812 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372631 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3617 has 0 variants, skipping\n",
      "[INFO]\tGene LACTB2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX264 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM3 has 0 variants, skipping\n",
      "[INFO]\tGene CGGBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRN has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NINJ2 has 0 variants, skipping\n",
      "[INFO]\tGene ETFB has 0 variants, skipping\n",
      "[INFO]\tGene SSR4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372633 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF663P has 0 variants, skipping\n",
      "[INFO]\tGene MKRN7P has 0 variants, skipping\n",
      "[INFO]\tGene PRTFDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5B has 0 variants, skipping\n",
      "[INFO]\tGene B9D2 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD2 has 0 variants, skipping\n",
      "[INFO]\tGene NDFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM121 has 0 variants, skipping\n",
      "[INFO]\tGene RFXAP has 0 variants, skipping\n",
      "[INFO]\tGene REN has 0 variants, skipping\n",
      "[INFO]\tGene HEATR5A has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP5 has 0 variants, skipping\n",
      "[INFO]\tGene ELP6 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC3 has 0 variants, skipping\n",
      "[INFO]\tGene CLPS has 0 variants, skipping\n",
      "[INFO]\tGene CEL has 0 variants, skipping\n",
      "[INFO]\tGene CDH8 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPE has 0 variants, skipping\n",
      "[INFO]\tGene COL6A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131496 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01754 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01522 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01523 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00494 has 0 variants, skipping\n",
      "[INFO]\tGene CSE1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM271 has 0 variants, skipping\n",
      "[INFO]\tGene AP5Z1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD12 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOM has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372653 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHB9 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA9 has 0 variants, skipping\n",
      "[INFO]\tGene SETDB2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01273 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01270 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01271 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA5B has 0 variants, skipping\n",
      "[INFO]\tGene ANTKMT has 0 variants, skipping\n",
      "[INFO]\tGene OSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-5 has 0 variants, skipping\n",
      "[INFO]\tGene ADNP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SYS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF19 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01429 has 0 variants, skipping\n",
      "[INFO]\tGene TREM2 has 0 variants, skipping\n",
      "[INFO]\tGene BARHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC37A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01524 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927770 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372672 has 0 variants, skipping\n",
      "[INFO]\tGene MIR767 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01441 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01440 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01716 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene VEGFD has 0 variants, skipping\n",
      "[INFO]\tGene BMP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4325 has 0 variants, skipping\n",
      "[INFO]\tGene RTP4 has 0 variants, skipping\n",
      "[INFO]\tGene NPFFR1 has 0 variants, skipping\n",
      "[INFO]\tGene TNMD has 0 variants, skipping\n",
      "[INFO]\tGene LRRC4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF563 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01742 has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR1288 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1L-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01711 has 0 variants, skipping\n",
      "[INFO]\tGene STX16-NPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372695 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927932 has 0 variants, skipping\n",
      "[INFO]\tGene ECHDC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIEF2 has 0 variants, skipping\n",
      "[INFO]\tGene CDK15 has 0 variants, skipping\n",
      "[INFO]\tGene SLMO2-ATP5E has 0 variants, skipping\n",
      "[INFO]\tGene PDXP has 0 variants, skipping\n",
      "[INFO]\tGene GDF1 has 0 variants, skipping\n",
      "[INFO]\tGene GNRH2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR646HG has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928048 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4533 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01718 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128310 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1257 has 0 variants, skipping\n",
      "[INFO]\tGene FBLN7 has 0 variants, skipping\n",
      "[INFO]\tGene SBF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4758 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-1HG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR520D has 0 variants, skipping\n",
      "[INFO]\tGene OGFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP21A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA117 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01749 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00029 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01056 has 0 variants, skipping\n",
      "[INFO]\tGene HAR1B has 0 variants, skipping\n",
      "[INFO]\tGene HAR1A has 0 variants, skipping\n",
      "[INFO]\tGene FLJ16779 has 0 variants, skipping\n",
      "[INFO]\tGene NPY has 0 variants, skipping\n",
      "[INFO]\tGene RPLP2 has 0 variants, skipping\n",
      "[INFO]\tGene LHB has 0 variants, skipping\n",
      "[INFO]\tGene NQO2 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4326 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130587 has 0 variants, skipping\n",
      "[INFO]\tGene CDSN has 0 variants, skipping\n",
      "[INFO]\tGene MIR3135B has 0 variants, skipping\n",
      "[INFO]\tGene EMX1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4EBP2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F6 has 0 variants, skipping\n",
      "[INFO]\tGene MHENCR has 0 variants, skipping\n",
      "[INFO]\tGene SPIN1 has 0 variants, skipping\n",
      "[INFO]\tGene RTEL1-TNFRSF6B has 0 variants, skipping\n",
      "[INFO]\tGene FBXO30 has 0 variants, skipping\n",
      "[INFO]\tGene XIRP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6B has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB46-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02764 has 0 variants, skipping\n",
      "[INFO]\tGene GDPD3 has 0 variants, skipping\n",
      "[INFO]\tGene FTHL17 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL32 has 0 variants, skipping\n",
      "[INFO]\tGene CDH20 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS21 has 0 variants, skipping\n",
      "[INFO]\tGene AGXT2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHAC2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHAC1 has 0 variants, skipping\n",
      "[INFO]\tGene FZD8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1914 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34A has 0 variants, skipping\n",
      "[INFO]\tGene UCKL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6813 has 0 variants, skipping\n",
      "[INFO]\tGene PEX7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00266-1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13A1 has 0 variants, skipping\n",
      "[INFO]\tGene STX4 has 0 variants, skipping\n",
      "[INFO]\tGene NPTXR has 0 variants, skipping\n",
      "[INFO]\tGene CBX6 has 0 variants, skipping\n",
      "[INFO]\tGene GCAT has 0 variants, skipping\n",
      "[INFO]\tGene OR2T4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01678 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01669 has 0 variants, skipping\n",
      "[INFO]\tGene TOPAZ1 has 0 variants, skipping\n",
      "[INFO]\tGene FRGCA has 0 variants, skipping\n",
      "[INFO]\tGene LOC106780825 has 0 variants, skipping\n",
      "[INFO]\tGene NKRF has 0 variants, skipping\n",
      "[INFO]\tGene MIR22HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR8069-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8069-2 has 0 variants, skipping\n",
      "[INFO]\tGene ULK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR549A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6724-4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3648-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3648-2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL23-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN2 has 0 variants, skipping\n",
      "[INFO]\tGene GHRLOS has 0 variants, skipping\n",
      "[INFO]\tGene SNHG7 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT4P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379514 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01667 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR5 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD49 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3156-3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01674 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3118-1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F29P has 0 variants, skipping\n",
      "[INFO]\tGene SRSF3P2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3120 has 0 variants, skipping\n",
      "[INFO]\tGene SAMSN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02246 has 0 variants, skipping\n",
      "[INFO]\tGene SCG5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR99AHG has 0 variants, skipping\n",
      "[INFO]\tGene SNORD74B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01549 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf91-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene CHODL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548XHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548X has 0 variants, skipping\n",
      "[INFO]\tGene LINC01683 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02573 has 0 variants, skipping\n",
      "[INFO]\tGene DTD1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS40B has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST6 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM151B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00317 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01425 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01687 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00308 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6130 has 0 variants, skipping\n",
      "[INFO]\tGene D21S2088E has 0 variants, skipping\n",
      "[INFO]\tGene LINC01689 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01684 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01692 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00158 has 0 variants, skipping\n",
      "[INFO]\tGene DHX58 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM48 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548K has 0 variants, skipping\n",
      "[INFO]\tGene PRSS1 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM4 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD9 has 0 variants, skipping\n",
      "[INFO]\tGene BTG3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF720 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4759 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00113 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00314 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01697 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01695 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM156 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf88 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00189 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD19 has 0 variants, skipping\n",
      "[INFO]\tGene CRBN has 0 variants, skipping\n",
      "[INFO]\tGene BACH1-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene BACH1-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene MROH2A has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00307 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4327 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS7 has 0 variants, skipping\n",
      "[INFO]\tGene RAB39A has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F2 has 0 variants, skipping\n",
      "[INFO]\tGene PLXNA3 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH8 has 0 variants, skipping\n",
      "[INFO]\tGene SMAP2 has 0 variants, skipping\n",
      "[INFO]\tGene FRYL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC42 has 0 variants, skipping\n",
      "[INFO]\tGene OR52R1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N4 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52K2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6R1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00159 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80A has 0 variants, skipping\n",
      "[INFO]\tGene URB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE12 has 0 variants, skipping\n",
      "[INFO]\tGene PAXBP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PAXBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C21orf62-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01690 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00945 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01548 has 0 variants, skipping\n",
      "[INFO]\tGene GABPA has 0 variants, skipping\n",
      "[INFO]\tGene IL10RB-DT has 0 variants, skipping\n",
      "[INFO]\tGene TTC21A has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1C1 has 0 variants, skipping\n",
      "[INFO]\tGene AMER2 has 0 variants, skipping\n",
      "[INFO]\tGene RTL3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02525 has 0 variants, skipping\n",
      "[INFO]\tGene MELTF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6501 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL5 has 0 variants, skipping\n",
      "[INFO]\tGene THAP11 has 0 variants, skipping\n",
      "[INFO]\tGene ATP10D has 0 variants, skipping\n",
      "[INFO]\tGene SELENON has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928126 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA9 has 0 variants, skipping\n",
      "[INFO]\tGene EEA1 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00160 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPB has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506403 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928269 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01436 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133286 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF398 has 0 variants, skipping\n",
      "[INFO]\tGene NRN1L has 0 variants, skipping\n",
      "[INFO]\tGene ABCF3 has 0 variants, skipping\n",
      "[INFO]\tGene STRAP has 0 variants, skipping\n",
      "[INFO]\tGene FAM107A has 0 variants, skipping\n",
      "[INFO]\tGene ERG28 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGEF3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00174 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C46 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T10 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T12 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C16 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M7 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M3 has 0 variants, skipping\n",
      "[INFO]\tGene TTC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUT1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB110 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR4 has 0 variants, skipping\n",
      "[INFO]\tGene NAT8L has 0 variants, skipping\n",
      "[INFO]\tGene TRIML1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR10 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E3 has 0 variants, skipping\n",
      "[INFO]\tGene PIGQ has 0 variants, skipping\n",
      "[INFO]\tGene LINC01423 has 0 variants, skipping\n",
      "[INFO]\tGene NCCRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD7 has 0 variants, skipping\n",
      "[INFO]\tGene SPINT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00114 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1755 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928398 has 0 variants, skipping\n",
      "[INFO]\tGene LOC400867 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01700 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3R1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1B has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene BRWD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEPSIN has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6508 has 0 variants, skipping\n",
      "[INFO]\tGene PECR has 0 variants, skipping\n",
      "[INFO]\tGene IMPACT has 0 variants, skipping\n",
      "[INFO]\tGene AMPH has 0 variants, skipping\n",
      "[INFO]\tGene MIR412 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM15 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4760 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00922 has 0 variants, skipping\n",
      "[INFO]\tGene DSCAM-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3197 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1B has 0 variants, skipping\n",
      "[INFO]\tGene PLAC4 has 0 variants, skipping\n",
      "[INFO]\tGene PODXL has 0 variants, skipping\n",
      "[INFO]\tGene GSDME has 0 variants, skipping\n",
      "[INFO]\tGene LINC00111 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00479 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6814 has 0 variants, skipping\n",
      "[INFO]\tGene TRIB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO1 has 0 variants, skipping\n",
      "[INFO]\tGene USP30 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA91 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF295-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDGFA has 0 variants, skipping\n",
      "[INFO]\tGene UMODL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT7 has 0 variants, skipping\n",
      "[INFO]\tGene AOC1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL30 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA3 has 0 variants, skipping\n",
      "[INFO]\tGene FZD2 has 0 variants, skipping\n",
      "[INFO]\tGene FRZB has 0 variants, skipping\n",
      "[INFO]\tGene FPR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930094 has 0 variants, skipping\n",
      "[INFO]\tGene NME6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01671 has 0 variants, skipping\n",
      "[INFO]\tGene DELEC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00271 has 0 variants, skipping\n",
      "[INFO]\tGene PYGO2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPTSSA has 0 variants, skipping\n",
      "[INFO]\tGene DTX3L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF681 has 0 variants, skipping\n",
      "[INFO]\tGene CSTL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01668 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692B has 0 variants, skipping\n",
      "[INFO]\tGene PTPRJ has 0 variants, skipping\n",
      "[INFO]\tGene HECW1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSPL has 0 variants, skipping\n",
      "[INFO]\tGene CCDC173 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00322 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01679 has 0 variants, skipping\n",
      "[INFO]\tGene MYOSLID has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6070 has 0 variants, skipping\n",
      "[INFO]\tGene AATBC has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A1 has 0 variants, skipping\n",
      "[INFO]\tGene ENY2 has 0 variants, skipping\n",
      "[INFO]\tGene KLK6 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGS has 0 variants, skipping\n",
      "[INFO]\tGene TRPM2-AS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC3-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02575 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM42 has 0 variants, skipping\n",
      "[INFO]\tGene PRR15L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01424 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL1 has 0 variants, skipping\n",
      "[INFO]\tGene LMO1 has 0 variants, skipping\n",
      "[INFO]\tGene SAYSD1 has 0 variants, skipping\n",
      "[INFO]\tGene GCNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01547 has 0 variants, skipping\n",
      "[INFO]\tGene SSR4P1 has 0 variants, skipping\n",
      "[INFO]\tGene ETS2 has 0 variants, skipping\n",
      "[INFO]\tGene CALML3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00334 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00316 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene COL18A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CBLC has 0 variants, skipping\n",
      "[INFO]\tGene HBZ has 0 variants, skipping\n",
      "[INFO]\tGene MIR6815 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01694 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM202 has 0 variants, skipping\n",
      "[INFO]\tGene UNCX has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928796 has 0 variants, skipping\n",
      "[INFO]\tGene MTRF1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD159 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD118 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF587 has 0 variants, skipping\n",
      "[INFO]\tGene ATOH8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35B4 has 0 variants, skipping\n",
      "[INFO]\tGene DDIAS has 0 variants, skipping\n",
      "[INFO]\tGene OR1G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E2 has 0 variants, skipping\n",
      "[INFO]\tGene USP24 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C15 has 0 variants, skipping\n",
      "[INFO]\tGene PUS1 has 0 variants, skipping\n",
      "[INFO]\tGene FLVCR2 has 0 variants, skipping\n",
      "[INFO]\tGene TEX53 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A4 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2A-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA1 has 0 variants, skipping\n",
      "[INFO]\tGene COPS7B has 0 variants, skipping\n",
      "[INFO]\tGene ANKDD1B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS30 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF12A has 0 variants, skipping\n",
      "[INFO]\tGene SPG21 has 0 variants, skipping\n",
      "[INFO]\tGene MEX3C has 0 variants, skipping\n",
      "[INFO]\tGene RSRC1 has 0 variants, skipping\n",
      "[INFO]\tGene SCG5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1FP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723769 has 0 variants, skipping\n",
      "[INFO]\tGene LHX1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNYL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD62P1-PARP4P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01665 has 0 variants, skipping\n",
      "[INFO]\tGene HSFY1P1 has 0 variants, skipping\n",
      "[INFO]\tGene CECR7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01664 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CECR3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF611 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC9 has 0 variants, skipping\n",
      "[INFO]\tGene ALYREF has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL6 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM5 has 0 variants, skipping\n",
      "[INFO]\tGene CNIH1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF38 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00528 has 0 variants, skipping\n",
      "[INFO]\tGene MIR648 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01634 has 0 variants, skipping\n",
      "[INFO]\tGene RMI2 has 0 variants, skipping\n",
      "[INFO]\tGene MTCH1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00106 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00243 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GGT3P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725072 has 0 variants, skipping\n",
      "[INFO]\tGene MTARC1 has 0 variants, skipping\n",
      "[INFO]\tGene RUSF1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD3 has 0 variants, skipping\n",
      "[INFO]\tGene HM13 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01311 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128108 has 0 variants, skipping\n",
      "[INFO]\tGene WDR25 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00895 has 0 variants, skipping\n",
      "[INFO]\tGene SLITRK5 has 0 variants, skipping\n",
      "[INFO]\tGene SEPT5-GP1BB has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC5 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPI has 0 variants, skipping\n",
      "[INFO]\tGene MIR4761 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH2 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS16 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL31 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF12 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN2 has 0 variants, skipping\n",
      "[INFO]\tGene PRAG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM120B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA31 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA24 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN18 has 0 variants, skipping\n",
      "[INFO]\tGene NPC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3618 has 0 variants, skipping\n",
      "[INFO]\tGene TMX4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6816 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA77B has 0 variants, skipping\n",
      "[INFO]\tGene LOC284865 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00896 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1286 has 0 variants, skipping\n",
      "[INFO]\tGene SPOUT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM170B has 0 variants, skipping\n",
      "[INFO]\tGene POM121L4P has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191A has 0 variants, skipping\n",
      "[INFO]\tGene TEX36 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01637 has 0 variants, skipping\n",
      "[INFO]\tGene FABP9 has 0 variants, skipping\n",
      "[INFO]\tGene AVEN has 0 variants, skipping\n",
      "[INFO]\tGene THAP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3FP has 0 variants, skipping\n",
      "[INFO]\tGene EIF5AL1 has 0 variants, skipping\n",
      "[INFO]\tGene RRS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR649 has 0 variants, skipping\n",
      "[INFO]\tGene P2RX6P has 0 variants, skipping\n",
      "[INFO]\tGene BCRP2 has 0 variants, skipping\n",
      "[INFO]\tGene FNTA has 0 variants, skipping\n",
      "[INFO]\tGene POM121L8P has 0 variants, skipping\n",
      "[INFO]\tGene RPL37A has 0 variants, skipping\n",
      "[INFO]\tGene RPL37 has 0 variants, skipping\n",
      "[INFO]\tGene PI4KAP2 has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL2 has 0 variants, skipping\n",
      "[INFO]\tGene CT47B1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf72 has 0 variants, skipping\n",
      "[INFO]\tGene SCN4A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01210 has 0 variants, skipping\n",
      "[INFO]\tGene CCR10 has 0 variants, skipping\n",
      "[INFO]\tGene TIMMDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NRN1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1A has 0 variants, skipping\n",
      "[INFO]\tGene PRAMENP has 0 variants, skipping\n",
      "[INFO]\tGene STARD3 has 0 variants, skipping\n",
      "[INFO]\tGene NT5DC4 has 0 variants, skipping\n",
      "[INFO]\tGene LL22NC03-63E9.3 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L1P has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST4 has 0 variants, skipping\n",
      "[INFO]\tGene CD68 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02556 has 0 variants, skipping\n",
      "[INFO]\tGene CES5AP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01659 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT14 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS2 has 0 variants, skipping\n",
      "[INFO]\tGene HIC1 has 0 variants, skipping\n",
      "[INFO]\tGene REG1B has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP11 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1F has 0 variants, skipping\n",
      "[INFO]\tGene NANOG has 0 variants, skipping\n",
      "[INFO]\tGene MRM1 has 0 variants, skipping\n",
      "[INFO]\tGene GSTTP2 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1L has 0 variants, skipping\n",
      "[INFO]\tGene ZNF791 has 0 variants, skipping\n",
      "[INFO]\tGene DICER1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CALR3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF501 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN3 has 0 variants, skipping\n",
      "[INFO]\tGene CENPBD1 has 0 variants, skipping\n",
      "[INFO]\tGene DRC1 has 0 variants, skipping\n",
      "[INFO]\tGene EHD1 has 0 variants, skipping\n",
      "[INFO]\tGene DPEP2NB has 0 variants, skipping\n",
      "[INFO]\tGene POM121L9P has 0 variants, skipping\n",
      "[INFO]\tGene SPECC1L-ADORA2A has 0 variants, skipping\n",
      "[INFO]\tGene B3GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX15 has 0 variants, skipping\n",
      "[INFO]\tGene ADORA2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC14A has 0 variants, skipping\n",
      "[INFO]\tGene BCRP3 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L10P has 0 variants, skipping\n",
      "[INFO]\tGene PRKAB2 has 0 variants, skipping\n",
      "[INFO]\tGene TOP1P2 has 0 variants, skipping\n",
      "[INFO]\tGene CDS2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM7 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC62 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6817 has 0 variants, skipping\n",
      "[INFO]\tGene CSH2 has 0 variants, skipping\n",
      "[INFO]\tGene PHKA2 has 0 variants, skipping\n",
      "[INFO]\tGene MYMX has 0 variants, skipping\n",
      "[INFO]\tGene RASAL3 has 0 variants, skipping\n",
      "[INFO]\tGene COL1A2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122C has 0 variants, skipping\n",
      "[INFO]\tGene LINC00299 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE2E has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507599 has 0 variants, skipping\n",
      "[INFO]\tGene MIR668 has 0 variants, skipping\n",
      "[INFO]\tGene MIATNB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01422 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD66 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02554 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40AL has 0 variants, skipping\n",
      "[INFO]\tGene MIR3199-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3199-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5739 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5D2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32B has 0 variants, skipping\n",
      "[INFO]\tGene IPO7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191B has 0 variants, skipping\n",
      "[INFO]\tGene BOD1L2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD125 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL1S has 0 variants, skipping\n",
      "[INFO]\tGene NID1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP12A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6818 has 0 variants, skipping\n",
      "[INFO]\tGene LIF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC91370 has 0 variants, skipping\n",
      "[INFO]\tGene DUXA has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1656 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105372990 has 0 variants, skipping\n",
      "[INFO]\tGene RAB41 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB136 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB135 has 0 variants, skipping\n",
      "[INFO]\tGene SDC4P has 0 variants, skipping\n",
      "[INFO]\tGene RTKN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3200 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM72 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985544 has 0 variants, skipping\n",
      "[INFO]\tGene MORC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UPB1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF274 has 0 variants, skipping\n",
      "[INFO]\tGene HERC5 has 0 variants, skipping\n",
      "[INFO]\tGene RAB14 has 0 variants, skipping\n",
      "[INFO]\tGene TGM2 has 0 variants, skipping\n",
      "[INFO]\tGene HAND2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01521 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L12 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L10 has 0 variants, skipping\n",
      "[INFO]\tGene CBX5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3R2 has 0 variants, skipping\n",
      "[INFO]\tGene MPZL3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD9 has 0 variants, skipping\n",
      "[INFO]\tGene MACO1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D13 has 0 variants, skipping\n",
      "[INFO]\tGene BBS7 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf106 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02868 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02870 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF414 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02558 has 0 variants, skipping\n",
      "[INFO]\tGene AP1B1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129484 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF3 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL3S has 0 variants, skipping\n",
      "[INFO]\tGene LOC339666 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01640 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4764 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50B has 0 variants, skipping\n",
      "[INFO]\tGene LARGE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01399 has 0 variants, skipping\n",
      "[INFO]\tGene FRRS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR31 has 0 variants, skipping\n",
      "[INFO]\tGene TRPV4 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6069 has 0 variants, skipping\n",
      "[INFO]\tGene PDIK1L has 0 variants, skipping\n",
      "[INFO]\tGene BPIFA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF700 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF5 has 0 variants, skipping\n",
      "[INFO]\tGene DENND2D has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6819 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373021 has 0 variants, skipping\n",
      "[INFO]\tGene NCF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LL22NC01-81G9.3 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA8L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF799 has 0 variants, skipping\n",
      "[INFO]\tGene TMED10 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAQ has 0 variants, skipping\n",
      "[INFO]\tGene CKAP4 has 0 variants, skipping\n",
      "[INFO]\tGene EBNA1BP2 has 0 variants, skipping\n",
      "[INFO]\tGene PKIA has 0 variants, skipping\n",
      "[INFO]\tGene RAB40B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF644 has 0 variants, skipping\n",
      "[INFO]\tGene SYDE2 has 0 variants, skipping\n",
      "[INFO]\tGene ABRAXAS2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM161A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506271 has 0 variants, skipping\n",
      "[INFO]\tGene ACOD1 has 0 variants, skipping\n",
      "[INFO]\tGene FCN3 has 0 variants, skipping\n",
      "[INFO]\tGene AP3B1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2G1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR659 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6820 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4534 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF496 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA92 has 0 variants, skipping\n",
      "[INFO]\tGene TPTEP2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF4 has 0 variants, skipping\n",
      "[INFO]\tGene CSPG4 has 0 variants, skipping\n",
      "[INFO]\tGene CTRL has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373031 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene QARS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC14A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02871 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD83B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD83A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD139 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD43 has 0 variants, skipping\n",
      "[INFO]\tGene HBB has 0 variants, skipping\n",
      "[INFO]\tGene HBD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506472 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC1H1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF3B has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A2 has 0 variants, skipping\n",
      "[INFO]\tGene DIO2 has 0 variants, skipping\n",
      "[INFO]\tGene LPAR3 has 0 variants, skipping\n",
      "[INFO]\tGene STX17 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A4 has 0 variants, skipping\n",
      "[INFO]\tGene DCANP1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf47 has 0 variants, skipping\n",
      "[INFO]\tGene MRTFA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A17 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD140 has 0 variants, skipping\n",
      "[INFO]\tGene KIFBP has 0 variants, skipping\n",
      "[INFO]\tGene TCTN3 has 0 variants, skipping\n",
      "[INFO]\tGene EP300-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6889 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM74 has 0 variants, skipping\n",
      "[INFO]\tGene NATD1 has 0 variants, skipping\n",
      "[INFO]\tGene NADK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373044 has 0 variants, skipping\n",
      "[INFO]\tGene CD1C has 0 variants, skipping\n",
      "[INFO]\tGene CD1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR30C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29C has 0 variants, skipping\n",
      "[INFO]\tGene MIR106B has 0 variants, skipping\n",
      "[INFO]\tGene MIR194-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR371A has 0 variants, skipping\n",
      "[INFO]\tGene MIR370 has 0 variants, skipping\n",
      "[INFO]\tGene MIR369 has 0 variants, skipping\n",
      "[INFO]\tGene MIR376C has 0 variants, skipping\n",
      "[INFO]\tGene MIR367 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302D has 0 variants, skipping\n",
      "[INFO]\tGene MIR365B has 0 variants, skipping\n",
      "[INFO]\tGene MIR365A has 0 variants, skipping\n",
      "[INFO]\tGene MIR363 has 0 variants, skipping\n",
      "[INFO]\tGene MIR362 has 0 variants, skipping\n",
      "[INFO]\tGene MIR361 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR130B has 0 variants, skipping\n",
      "[INFO]\tGene MIR296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR99B has 0 variants, skipping\n",
      "[INFO]\tGene MIR301A has 0 variants, skipping\n",
      "[INFO]\tGene MIR299 has 0 variants, skipping\n",
      "[INFO]\tGene MIR34B has 0 variants, skipping\n",
      "[INFO]\tGene MIR34C has 0 variants, skipping\n",
      "[INFO]\tGene MIR219A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR101-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302A has 0 variants, skipping\n",
      "[INFO]\tGene MIR200A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00634 has 0 variants, skipping\n",
      "[INFO]\tGene SOBP has 0 variants, skipping\n",
      "[INFO]\tGene KIF26B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA6-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929829 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1202 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01315 has 0 variants, skipping\n",
      "[INFO]\tGene SERHL has 0 variants, skipping\n",
      "[INFO]\tGene FAM174B has 0 variants, skipping\n",
      "[INFO]\tGene RRP7BP has 0 variants, skipping\n",
      "[INFO]\tGene RNU12 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB105B has 0 variants, skipping\n",
      "[INFO]\tGene EFL1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT1 has 0 variants, skipping\n",
      "[INFO]\tGene AQP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRUB2 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01639 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01656 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00207 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00229 has 0 variants, skipping\n",
      "[INFO]\tGene SMC5 has 0 variants, skipping\n",
      "[INFO]\tGene DCDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927551 has 0 variants, skipping\n",
      "[INFO]\tGene NUP50-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373064 has 0 variants, skipping\n",
      "[INFO]\tGene IL12A has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2F has 0 variants, skipping\n",
      "[INFO]\tGene ANO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4762 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730668 has 0 variants, skipping\n",
      "[INFO]\tGene PRR34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642648 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7BHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4763 has 0 variants, skipping\n",
      "[INFO]\tGene PHLPP2 has 0 variants, skipping\n",
      "[INFO]\tGene GTSE1-DT has 0 variants, skipping\n",
      "[INFO]\tGene SQSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene CDCP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF776 has 0 variants, skipping\n",
      "[INFO]\tGene OR10K2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01644 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00898 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3201 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284933 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4535 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01310 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP19 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6821 has 0 variants, skipping\n",
      "[INFO]\tGene GPER1 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL16 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9 has 0 variants, skipping\n",
      "[INFO]\tGene GYPB has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG5 has 0 variants, skipping\n",
      "[INFO]\tGene RPS28 has 0 variants, skipping\n",
      "[INFO]\tGene NEK10 has 0 variants, skipping\n",
      "[INFO]\tGene INSIG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM95 has 0 variants, skipping\n",
      "[INFO]\tGene ADSS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC7B has 0 variants, skipping\n",
      "[INFO]\tGene MCCC2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB7A has 0 variants, skipping\n",
      "[INFO]\tGene CHKB-CPT1B has 0 variants, skipping\n",
      "[INFO]\tGene CHKB-DT has 0 variants, skipping\n",
      "[INFO]\tGene TRHDE has 0 variants, skipping\n",
      "[INFO]\tGene G6PD has 0 variants, skipping\n",
      "[INFO]\tGene ABCC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373100 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP82 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF4 has 0 variants, skipping\n",
      "[INFO]\tGene CAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf25 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF4B has 0 variants, skipping\n",
      "[INFO]\tGene IBA57 has 0 variants, skipping\n",
      "[INFO]\tGene PVT1 has 0 variants, skipping\n",
      "[INFO]\tGene JPX has 0 variants, skipping\n",
      "[INFO]\tGene APOO has 0 variants, skipping\n",
      "[INFO]\tGene LBX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDC3 has 0 variants, skipping\n",
      "[INFO]\tGene IPO13 has 0 variants, skipping\n",
      "[INFO]\tGene RPS19 has 0 variants, skipping\n",
      "[INFO]\tGene RERE has 0 variants, skipping\n",
      "[INFO]\tGene RFX7 has 0 variants, skipping\n",
      "[INFO]\tGene FCER1G has 0 variants, skipping\n",
      "[INFO]\tGene LRRC32 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K12 has 0 variants, skipping\n",
      "[INFO]\tGene PLCD4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC391322 has 0 variants, skipping\n",
      "[INFO]\tGene GSTT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEC has 0 variants, skipping\n",
      "[INFO]\tGene ABCC6P2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM72 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10AC1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13H1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02692 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284412 has 0 variants, skipping\n",
      "[INFO]\tGene OR13F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13D1 has 0 variants, skipping\n",
      "[INFO]\tGene VAC14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5A has 0 variants, skipping\n",
      "[INFO]\tGene NCBP3 has 0 variants, skipping\n",
      "[INFO]\tGene TAX1BP3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A14 has 0 variants, skipping\n",
      "[INFO]\tGene CRACR2A has 0 variants, skipping\n",
      "[INFO]\tGene IFIT1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP11C has 0 variants, skipping\n",
      "[INFO]\tGene GRB2 has 0 variants, skipping\n",
      "[INFO]\tGene USP50 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01986 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CHL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01266 has 0 variants, skipping\n",
      "[INFO]\tGene AIMP1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf47 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A3 has 0 variants, skipping\n",
      "[INFO]\tGene AP4E1 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF3 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR934 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4P1 has 0 variants, skipping\n",
      "[INFO]\tGene HMMR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK11 has 0 variants, skipping\n",
      "[INFO]\tGene LNCNEF has 0 variants, skipping\n",
      "[INFO]\tGene ROCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130207 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR1-DT has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D2 has 0 variants, skipping\n",
      "[INFO]\tGene RRAGC has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE40-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4790 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927394 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM5 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A5 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SMC6 has 0 variants, skipping\n",
      "[INFO]\tGene RHOBTB3 has 0 variants, skipping\n",
      "[INFO]\tGene MOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10B has 0 variants, skipping\n",
      "[INFO]\tGene TSLP has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIL has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R26 has 0 variants, skipping\n",
      "[INFO]\tGene CEP350 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJA2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL16 has 0 variants, skipping\n",
      "[INFO]\tGene ABITRAM has 0 variants, skipping\n",
      "[INFO]\tGene RNF125 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD5 has 0 variants, skipping\n",
      "[INFO]\tGene SARS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT3 has 0 variants, skipping\n",
      "[INFO]\tGene EMC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIDECP1 has 0 variants, skipping\n",
      "[INFO]\tGene AP2B1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCRB1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND10P1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF213-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00852 has 0 variants, skipping\n",
      "[INFO]\tGene SPRTN has 0 variants, skipping\n",
      "[INFO]\tGene NFE2 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG5 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6F has 0 variants, skipping\n",
      "[INFO]\tGene XPA has 0 variants, skipping\n",
      "[INFO]\tGene PSMA7 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf70 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378B has 0 variants, skipping\n",
      "[INFO]\tGene ATP2B2-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00606 has 0 variants, skipping\n",
      "[INFO]\tGene ZFP91 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEURL2 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR3 has 0 variants, skipping\n",
      "[INFO]\tGene ANO8 has 0 variants, skipping\n",
      "[INFO]\tGene H3C14 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCD3 has 0 variants, skipping\n",
      "[INFO]\tGene MYO7B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF480 has 0 variants, skipping\n",
      "[INFO]\tGene DHX35 has 0 variants, skipping\n",
      "[INFO]\tGene DUXB has 0 variants, skipping\n",
      "[INFO]\tGene SNORA7A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02022 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF14 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA93 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00620 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01267 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02011 has 0 variants, skipping\n",
      "[INFO]\tGene PSG8 has 0 variants, skipping\n",
      "[INFO]\tGene PGBP has 0 variants, skipping\n",
      "[INFO]\tGene LRGUK has 0 variants, skipping\n",
      "[INFO]\tGene CCDC18 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG4 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS16 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFS7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4270 has 0 variants, skipping\n",
      "[INFO]\tGene BTLA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF134 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF132 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT3 has 0 variants, skipping\n",
      "[INFO]\tGene CALML4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC284930 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00887 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR563 has 0 variants, skipping\n",
      "[INFO]\tGene FAM224A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3129 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00690 has 0 variants, skipping\n",
      "[INFO]\tGene PLCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG11 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD7 has 0 variants, skipping\n",
      "[INFO]\tGene NXF1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB13 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PD has 0 variants, skipping\n",
      "[INFO]\tGene TRIM38 has 0 variants, skipping\n",
      "[INFO]\tGene TADA3 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN4 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB18 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM44 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT1 has 0 variants, skipping\n",
      "[INFO]\tGene PPIH has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A9 has 0 variants, skipping\n",
      "[INFO]\tGene MERTK has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376975 has 0 variants, skipping\n",
      "[INFO]\tGene RERG has 0 variants, skipping\n",
      "[INFO]\tGene TRIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SATB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4791 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3135A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3659HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3065 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927829 has 0 variants, skipping\n",
      "[INFO]\tGene VENTXP7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TK1 has 0 variants, skipping\n",
      "[INFO]\tGene JAM2 has 0 variants, skipping\n",
      "[INFO]\tGene BYSL has 0 variants, skipping\n",
      "[INFO]\tGene SARM1 has 0 variants, skipping\n",
      "[INFO]\tGene H3-5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED6 has 0 variants, skipping\n",
      "[INFO]\tGene THRB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RARB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TDP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4442 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00692 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM239 has 0 variants, skipping\n",
      "[INFO]\tGene SYMPK has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1270 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01981 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00693 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene MBOAT4 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01985 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN9 has 0 variants, skipping\n",
      "[INFO]\tGene PIGN has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AY has 0 variants, skipping\n",
      "[INFO]\tGene PHC3 has 0 variants, skipping\n",
      "[INFO]\tGene SLX1A has 0 variants, skipping\n",
      "[INFO]\tGene EOLA2 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX4 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN12 has 0 variants, skipping\n",
      "[INFO]\tGene PYDC2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A3 has 0 variants, skipping\n",
      "[INFO]\tGene PRF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZGPAT has 0 variants, skipping\n",
      "[INFO]\tGene C7orf65 has 0 variants, skipping\n",
      "[INFO]\tGene RPTN has 0 variants, skipping\n",
      "[INFO]\tGene MTREX has 0 variants, skipping\n",
      "[INFO]\tGene ERAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI3 has 0 variants, skipping\n",
      "[INFO]\tGene MAVS has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928135 has 0 variants, skipping\n",
      "[INFO]\tGene MED25 has 0 variants, skipping\n",
      "[INFO]\tGene AK7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02033 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP3 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705E has 0 variants, skipping\n",
      "[INFO]\tGene TP53AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene A2ML1 has 0 variants, skipping\n",
      "[INFO]\tGene PRORY has 0 variants, skipping\n",
      "[INFO]\tGene TSPY10 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA8M has 0 variants, skipping\n",
      "[INFO]\tGene SPIN2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC63 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM33 has 0 variants, skipping\n",
      "[INFO]\tGene LOC152048 has 0 variants, skipping\n",
      "[INFO]\tGene UTP18 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBX4 has 0 variants, skipping\n",
      "[INFO]\tGene ACAA1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF4ENIF1 has 0 variants, skipping\n",
      "[INFO]\tGene FNBP1L has 0 variants, skipping\n",
      "[INFO]\tGene MLN has 0 variants, skipping\n",
      "[INFO]\tGene MIR100HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC00689 has 0 variants, skipping\n",
      "[INFO]\tGene CASC8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-2HG has 0 variants, skipping\n",
      "[INFO]\tGene FAM225A has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B3P has 0 variants, skipping\n",
      "[INFO]\tGene FBXL19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HNF1A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZEB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NECAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT18 has 0 variants, skipping\n",
      "[INFO]\tGene GP1BB has 0 variants, skipping\n",
      "[INFO]\tGene SRP54 has 0 variants, skipping\n",
      "[INFO]\tGene CDC5L has 0 variants, skipping\n",
      "[INFO]\tGene CD70 has 0 variants, skipping\n",
      "[INFO]\tGene MIR503HG has 0 variants, skipping\n",
      "[INFO]\tGene CASP6 has 0 variants, skipping\n",
      "[INFO]\tGene LTBP2 has 0 variants, skipping\n",
      "[INFO]\tGene RASA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR218-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6822 has 0 variants, skipping\n",
      "[INFO]\tGene PIR has 0 variants, skipping\n",
      "[INFO]\tGene FOXE3 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP43 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA6 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA62 has 0 variants, skipping\n",
      "[INFO]\tGene ADCYAP1R1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA1S has 0 variants, skipping\n",
      "[INFO]\tGene KPNA3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LTBR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354A has 0 variants, skipping\n",
      "[INFO]\tGene PPP2CB has 0 variants, skipping\n",
      "[INFO]\tGene SGSM2 has 0 variants, skipping\n",
      "[INFO]\tGene CASTOR1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB107A has 0 variants, skipping\n",
      "[INFO]\tGene DEFB128 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB116 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB115 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB113 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM8 has 0 variants, skipping\n",
      "[INFO]\tGene GPR39 has 0 variants, skipping\n",
      "[INFO]\tGene GCLC has 0 variants, skipping\n",
      "[INFO]\tGene MIR1231 has 0 variants, skipping\n",
      "[INFO]\tGene CRTAC1 has 0 variants, skipping\n",
      "[INFO]\tGene VIPR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH18 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf131 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928323 has 0 variants, skipping\n",
      "[INFO]\tGene MIR425 has 0 variants, skipping\n",
      "[INFO]\tGene MIR424 has 0 variants, skipping\n",
      "[INFO]\tGene MIR423 has 0 variants, skipping\n",
      "[INFO]\tGene MIR422A has 0 variants, skipping\n",
      "[INFO]\tGene MIR196B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02158 has 0 variants, skipping\n",
      "[INFO]\tGene NUP107 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2D3A has 0 variants, skipping\n",
      "[INFO]\tGene BRWD3 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDF4 has 0 variants, skipping\n",
      "[INFO]\tGene OR7G2 has 0 variants, skipping\n",
      "[INFO]\tGene GDF11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01988 has 0 variants, skipping\n",
      "[INFO]\tGene TNC has 0 variants, skipping\n",
      "[INFO]\tGene HTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20HL1 has 0 variants, skipping\n",
      "[INFO]\tGene SUPT20HL2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF197-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1P-OR7E38P has 0 variants, skipping\n",
      "[INFO]\tGene RBM7 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE1A has 0 variants, skipping\n",
      "[INFO]\tGene H4C9 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR3 has 0 variants, skipping\n",
      "[INFO]\tGene RPF1 has 0 variants, skipping\n",
      "[INFO]\tGene LARS2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LIMD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT11 has 0 variants, skipping\n",
      "[INFO]\tGene TCN2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00458 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02009 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE1L2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA7B has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC11 has 0 variants, skipping\n",
      "[INFO]\tGene GJC1 has 0 variants, skipping\n",
      "[INFO]\tGene CFL1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARB2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS46P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS42P has 0 variants, skipping\n",
      "[INFO]\tGene UNC5B has 0 variants, skipping\n",
      "[INFO]\tGene CRYGD has 0 variants, skipping\n",
      "[INFO]\tGene NRADDP has 0 variants, skipping\n",
      "[INFO]\tGene REC8 has 0 variants, skipping\n",
      "[INFO]\tGene KIF9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51V1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13J has 0 variants, skipping\n",
      "[INFO]\tGene TNKS has 0 variants, skipping\n",
      "[INFO]\tGene IRGQ has 0 variants, skipping\n",
      "[INFO]\tGene SNORD146 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2B has 0 variants, skipping\n",
      "[INFO]\tGene GSTA4 has 0 variants, skipping\n",
      "[INFO]\tGene MFAP5 has 0 variants, skipping\n",
      "[INFO]\tGene FCF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene NPS has 0 variants, skipping\n",
      "[INFO]\tGene KLK3 has 0 variants, skipping\n",
      "[INFO]\tGene CEP126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2115 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF143 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6823 has 0 variants, skipping\n",
      "[INFO]\tGene MIR711 has 0 variants, skipping\n",
      "[INFO]\tGene PNOC has 0 variants, skipping\n",
      "[INFO]\tGene SNORA94 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6824 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4793 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02585 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18B1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAR2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDI1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM166A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF618 has 0 variants, skipping\n",
      "[INFO]\tGene MYADM has 0 variants, skipping\n",
      "[INFO]\tGene CFAP70 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC6B has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR586 has 0 variants, skipping\n",
      "[INFO]\tGene GPR151 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF804A has 0 variants, skipping\n",
      "[INFO]\tGene DND1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1182 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10A has 0 variants, skipping\n",
      "[INFO]\tGene P3R3URF-PIK3R3 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH4 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6890 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN5 has 0 variants, skipping\n",
      "[INFO]\tGene SIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB5 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF4 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN9 has 0 variants, skipping\n",
      "[INFO]\tGene ISY1 has 0 variants, skipping\n",
      "[INFO]\tGene LAMB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene IL18R1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA12 has 0 variants, skipping\n",
      "[INFO]\tGene AOC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4271 has 0 variants, skipping\n",
      "[INFO]\tGene CMC2 has 0 variants, skipping\n",
      "[INFO]\tGene KRBOX1 has 0 variants, skipping\n",
      "[INFO]\tGene SASH3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11A has 0 variants, skipping\n",
      "[INFO]\tGene THSD7A has 0 variants, skipping\n",
      "[INFO]\tGene ARC has 0 variants, skipping\n",
      "[INFO]\tGene RPRD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLCB1 has 0 variants, skipping\n",
      "[INFO]\tGene BSN-DT has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2B has 0 variants, skipping\n",
      "[INFO]\tGene ACTG2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00662 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf88B has 0 variants, skipping\n",
      "[INFO]\tGene RBM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3F-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5787 has 0 variants, skipping\n",
      "[INFO]\tGene PHF11 has 0 variants, skipping\n",
      "[INFO]\tGene CDH17 has 0 variants, skipping\n",
      "[INFO]\tGene BNC2 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD4 has 0 variants, skipping\n",
      "[INFO]\tGene PMF1-BGLAP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC10B has 0 variants, skipping\n",
      "[INFO]\tGene USP27X has 0 variants, skipping\n",
      "[INFO]\tGene NELL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3941 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF818P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00589 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01018 has 0 variants, skipping\n",
      "[INFO]\tGene CARS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM89B has 0 variants, skipping\n",
      "[INFO]\tGene IQCF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D4 has 0 variants, skipping\n",
      "[INFO]\tGene YBX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00696 has 0 variants, skipping\n",
      "[INFO]\tGene PSENEN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929054 has 0 variants, skipping\n",
      "[INFO]\tGene CREBBP has 0 variants, skipping\n",
      "[INFO]\tGene LPIN2 has 0 variants, skipping\n",
      "[INFO]\tGene GLYCTK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA2 has 0 variants, skipping\n",
      "[INFO]\tGene PJA2 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM70 has 0 variants, skipping\n",
      "[INFO]\tGene RAB15 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3C has 0 variants, skipping\n",
      "[INFO]\tGene SMIM27 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf15 has 0 variants, skipping\n",
      "[INFO]\tGene METTL17 has 0 variants, skipping\n",
      "[INFO]\tGene GCGR has 0 variants, skipping\n",
      "[INFO]\tGene GBE1 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD6 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D23 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA8 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L4 has 0 variants, skipping\n",
      "[INFO]\tGene GATA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD136 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD19C has 0 variants, skipping\n",
      "[INFO]\tGene SNORD69 has 0 variants, skipping\n",
      "[INFO]\tGene UFD1 has 0 variants, skipping\n",
      "[INFO]\tGene SSPOP has 0 variants, skipping\n",
      "[INFO]\tGene PLCD3 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTUB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8064 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM43B has 0 variants, skipping\n",
      "[INFO]\tGene IFIT2 has 0 variants, skipping\n",
      "[INFO]\tGene ID4 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG2 has 0 variants, skipping\n",
      "[INFO]\tGene RASD2 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN8OS has 0 variants, skipping\n",
      "[INFO]\tGene ACAD9 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38C has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN11 has 0 variants, skipping\n",
      "[INFO]\tGene TIFAB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF862 has 0 variants, skipping\n",
      "[INFO]\tGene SI has 0 variants, skipping\n",
      "[INFO]\tGene ESRG has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02017 has 0 variants, skipping\n",
      "[INFO]\tGene TXK has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF4 has 0 variants, skipping\n",
      "[INFO]\tGene ERC2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3938 has 0 variants, skipping\n",
      "[INFO]\tGene SHKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene C1QB has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377102 has 0 variants, skipping\n",
      "[INFO]\tGene ARF4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6A-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985115 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1C4 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD1 has 0 variants, skipping\n",
      "[INFO]\tGene DRC7 has 0 variants, skipping\n",
      "[INFO]\tGene SLA2 has 0 variants, skipping\n",
      "[INFO]\tGene CD5L has 0 variants, skipping\n",
      "[INFO]\tGene FLNB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM17 has 0 variants, skipping\n",
      "[INFO]\tGene TUBD1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH11X has 0 variants, skipping\n",
      "[INFO]\tGene ME2 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC6A has 0 variants, skipping\n",
      "[INFO]\tGene CITED2 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf44-SGK3 has 0 variants, skipping\n",
      "[INFO]\tGene CHODL has 0 variants, skipping\n",
      "[INFO]\tGene ZHX1-C8orf76 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA4 has 0 variants, skipping\n",
      "[INFO]\tGene CUL7 has 0 variants, skipping\n",
      "[INFO]\tGene CUL3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339902 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BB has 0 variants, skipping\n",
      "[INFO]\tGene SERINC5 has 0 variants, skipping\n",
      "[INFO]\tGene LMX1A has 0 variants, skipping\n",
      "[INFO]\tGene PTPRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00698 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THOC7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAANT1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00994 has 0 variants, skipping\n",
      "[INFO]\tGene MAP4K5 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRICKLE2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF704 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF10 has 0 variants, skipping\n",
      "[INFO]\tGene FGD1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FZR1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBE1 has 0 variants, skipping\n",
      "[INFO]\tGene IKZF2 has 0 variants, skipping\n",
      "[INFO]\tGene NAGPA has 0 variants, skipping\n",
      "[INFO]\tGene FAM8A1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP15 has 0 variants, skipping\n",
      "[INFO]\tGene MAML3 has 0 variants, skipping\n",
      "[INFO]\tGene PDE11A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377143 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4272 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377146 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3136 has 0 variants, skipping\n",
      "[INFO]\tGene RBM28 has 0 variants, skipping\n",
      "[INFO]\tGene KIF18B has 0 variants, skipping\n",
      "[INFO]\tGene RPE65 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4B has 0 variants, skipping\n",
      "[INFO]\tGene GRHL1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK1 has 0 variants, skipping\n",
      "[INFO]\tGene QRICH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IFT172 has 0 variants, skipping\n",
      "[INFO]\tGene TES has 0 variants, skipping\n",
      "[INFO]\tGene LINC00877 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00870 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377162 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP4 has 0 variants, skipping\n",
      "[INFO]\tGene RNF217 has 0 variants, skipping\n",
      "[INFO]\tGene RPUSD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927296 has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02005 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86DP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02018 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00960 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4273 has 0 variants, skipping\n",
      "[INFO]\tGene MRFAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGA1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02077 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927374 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3923 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02050 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02027 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02008 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00971 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02025 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA95 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5688 has 0 variants, skipping\n",
      "[INFO]\tGene CADM2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02070 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00506 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4795 has 0 variants, skipping\n",
      "[INFO]\tGene SNX18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF880 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00879 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8060 has 0 variants, skipping\n",
      "[INFO]\tGene AACS has 0 variants, skipping\n",
      "[INFO]\tGene GFPT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF837 has 0 variants, skipping\n",
      "[INFO]\tGene FAM30A has 0 variants, skipping\n",
      "[INFO]\tGene NPFFR2 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC645202 has 0 variants, skipping\n",
      "[INFO]\tGene HP09053 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3921 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30CP has 0 variants, skipping\n",
      "[INFO]\tGene SRRM5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR576 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92B has 0 variants, skipping\n",
      "[INFO]\tGene MIR552 has 0 variants, skipping\n",
      "[INFO]\tGene MIR551A has 0 variants, skipping\n",
      "[INFO]\tGene MIR487B has 0 variants, skipping\n",
      "[INFO]\tGene MIR376A2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH6A1 has 0 variants, skipping\n",
      "[INFO]\tGene GFI1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A9 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM172BP has 0 variants, skipping\n",
      "[INFO]\tGene TAPT1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71C has 0 variants, skipping\n",
      "[INFO]\tGene PCNP has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL3P4 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00310 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00265 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02085 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929411 has 0 variants, skipping\n",
      "[INFO]\tGene NPHS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AB has 0 variants, skipping\n",
      "[INFO]\tGene MROH1 has 0 variants, skipping\n",
      "[INFO]\tGene AP3B2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF1 has 0 variants, skipping\n",
      "[INFO]\tGene MARCO has 0 variants, skipping\n",
      "[INFO]\tGene DUBR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929579 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01990 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00635 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00636 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01215 has 0 variants, skipping\n",
      "[INFO]\tGene TAL1 has 0 variants, skipping\n",
      "[INFO]\tGene TULP1 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH2 has 0 variants, skipping\n",
      "[INFO]\tGene RSAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4445 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPT1 has 0 variants, skipping\n",
      "[INFO]\tGene PLAAT1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3J-DT has 0 variants, skipping\n",
      "[INFO]\tGene SMIM22 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF444 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02042 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C8G has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02044 has 0 variants, skipping\n",
      "[INFO]\tGene OR5K1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8076 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF530 has 0 variants, skipping\n",
      "[INFO]\tGene KCNA1 has 0 variants, skipping\n",
      "[INFO]\tGene ANOS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAK3 has 0 variants, skipping\n",
      "[INFO]\tGene MX2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4446 has 0 variants, skipping\n",
      "[INFO]\tGene IL2RG has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B2 has 0 variants, skipping\n",
      "[INFO]\tGene HPRT1 has 0 variants, skipping\n",
      "[INFO]\tGene IDUA has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR890 has 0 variants, skipping\n",
      "[INFO]\tGene OR51F2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA7 has 0 variants, skipping\n",
      "[INFO]\tGene TAT has 0 variants, skipping\n",
      "[INFO]\tGene MIR568 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4796 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB20-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD155 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00903 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4447 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00901 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02024 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374060 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926968 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA5 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP31-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RADIL has 0 variants, skipping\n",
      "[INFO]\tGene CAB39L has 0 variants, skipping\n",
      "[INFO]\tGene FAM193B has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF2 has 0 variants, skipping\n",
      "[INFO]\tGene ROBO1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02049 has 0 variants, skipping\n",
      "[INFO]\tGene MBL2 has 0 variants, skipping\n",
      "[INFO]\tGene LENG1 has 0 variants, skipping\n",
      "[INFO]\tGene NOL12 has 0 variants, skipping\n",
      "[INFO]\tGene GNPTAB has 0 variants, skipping\n",
      "[INFO]\tGene MIR5682 has 0 variants, skipping\n",
      "[INFO]\tGene RLIM has 0 variants, skipping\n",
      "[INFO]\tGene TRPV2 has 0 variants, skipping\n",
      "[INFO]\tGene TELO2 has 0 variants, skipping\n",
      "[INFO]\tGene VAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723582 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH6A has 0 variants, skipping\n",
      "[INFO]\tGene CLPTM1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A10 has 0 variants, skipping\n",
      "[INFO]\tGene WNT5B has 0 variants, skipping\n",
      "[INFO]\tGene OR51E2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB1 has 0 variants, skipping\n",
      "[INFO]\tGene GJA9 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC34 has 0 variants, skipping\n",
      "[INFO]\tGene ILKAP has 0 variants, skipping\n",
      "[INFO]\tGene AKNA has 0 variants, skipping\n",
      "[INFO]\tGene LINC02035 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7110 has 0 variants, skipping\n",
      "[INFO]\tGene MYLK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2 has 0 variants, skipping\n",
      "[INFO]\tGene ADI1 has 0 variants, skipping\n",
      "[INFO]\tGene TG has 0 variants, skipping\n",
      "[INFO]\tGene MYLK-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC189 has 0 variants, skipping\n",
      "[INFO]\tGene METTL4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM11 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4316 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5002 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6083 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM213 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF843 has 0 variants, skipping\n",
      "[INFO]\tGene RNF26 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS24 has 0 variants, skipping\n",
      "[INFO]\tGene ARID3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR544B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5092 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374312 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02614 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86JP has 0 variants, skipping\n",
      "[INFO]\tGene CNBP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf220 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH1L1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC149 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC37-DT has 0 variants, skipping\n",
      "[INFO]\tGene CCDC24 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210P1 has 0 variants, skipping\n",
      "[INFO]\tGene ANO7 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02016 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01471 has 0 variants, skipping\n",
      "[INFO]\tGene LDHA has 0 variants, skipping\n",
      "[INFO]\tGene MIR6825 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7976 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC10B has 0 variants, skipping\n",
      "[INFO]\tGene WFDC13 has 0 variants, skipping\n",
      "[INFO]\tGene SYT3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF5 has 0 variants, skipping\n",
      "[INFO]\tGene RUVBL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SP5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC90246 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01565 has 0 variants, skipping\n",
      "[INFO]\tGene LOC653712 has 0 variants, skipping\n",
      "[INFO]\tGene WIPF2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM200A has 0 variants, skipping\n",
      "[INFO]\tGene LRRC45 has 0 variants, skipping\n",
      "[INFO]\tGene PGA3 has 0 variants, skipping\n",
      "[INFO]\tGene PTGDR2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT5 has 0 variants, skipping\n",
      "[INFO]\tGene MED1 has 0 variants, skipping\n",
      "[INFO]\tGene TREML1 has 0 variants, skipping\n",
      "[INFO]\tGene HLTF has 0 variants, skipping\n",
      "[INFO]\tGene MIR6826 has 0 variants, skipping\n",
      "[INFO]\tGene RPL32P3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA7B has 0 variants, skipping\n",
      "[INFO]\tGene TET3 has 0 variants, skipping\n",
      "[INFO]\tGene TM7SF2 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB131A has 0 variants, skipping\n",
      "[INFO]\tGene TMCC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02014 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86HP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02021 has 0 variants, skipping\n",
      "[INFO]\tGene COL6A4P2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R1 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF18 has 0 variants, skipping\n",
      "[INFO]\tGene MCIDAS has 0 variants, skipping\n",
      "[INFO]\tGene STMND1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14C36 has 0 variants, skipping\n",
      "[INFO]\tGene SSBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM22 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61A1 has 0 variants, skipping\n",
      "[INFO]\tGene HILPDA has 0 variants, skipping\n",
      "[INFO]\tGene NME7 has 0 variants, skipping\n",
      "[INFO]\tGene PYCR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339874 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT16P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA58 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5704 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3-ACAD11 has 0 variants, skipping\n",
      "[INFO]\tGene DPAGT1 has 0 variants, skipping\n",
      "[INFO]\tGene NPHP3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM108-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BFSP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FANCD2OS has 0 variants, skipping\n",
      "[INFO]\tGene LINC02000 has 0 variants, skipping\n",
      "[INFO]\tGene FCHSD1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC10 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02004 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6827 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4788 has 0 variants, skipping\n",
      "[INFO]\tGene LY6L has 0 variants, skipping\n",
      "[INFO]\tGene TCP11X1 has 0 variants, skipping\n",
      "[INFO]\tGene TENM2 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L27 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD14 has 0 variants, skipping\n",
      "[INFO]\tGene ELL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIGZ has 0 variants, skipping\n",
      "[INFO]\tGene TASL has 0 variants, skipping\n",
      "[INFO]\tGene DCSTAMP has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP29-1 has 0 variants, skipping\n",
      "[INFO]\tGene MLNR has 0 variants, skipping\n",
      "[INFO]\tGene GFRA3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR32 has 0 variants, skipping\n",
      "[INFO]\tGene GPLD1 has 0 variants, skipping\n",
      "[INFO]\tGene UNC13A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3677 has 0 variants, skipping\n",
      "[INFO]\tGene BPESC1 has 0 variants, skipping\n",
      "[INFO]\tGene PISRT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMID1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507291 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TTF2 has 0 variants, skipping\n",
      "[INFO]\tGene IRS4 has 0 variants, skipping\n",
      "[INFO]\tGene MORN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02618 has 0 variants, skipping\n",
      "[INFO]\tGene RAD51 has 0 variants, skipping\n",
      "[INFO]\tGene DDX53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF232-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLASRP has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4E has 0 variants, skipping\n",
      "[INFO]\tGene POLR2G has 0 variants, skipping\n",
      "[INFO]\tGene PPL has 0 variants, skipping\n",
      "[INFO]\tGene POU4F3 has 0 variants, skipping\n",
      "[INFO]\tGene GLIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507389 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289361 has 0 variants, skipping\n",
      "[INFO]\tGene EFHC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCHS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCSRLR has 0 variants, skipping\n",
      "[INFO]\tGene FAM178B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02010 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02032 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02046 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN2B has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF263 has 0 variants, skipping\n",
      "[INFO]\tGene HLTF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728392 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene NOL6 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN21 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646903 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374313 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01214 has 0 variants, skipping\n",
      "[INFO]\tGene ESYT1 has 0 variants, skipping\n",
      "[INFO]\tGene NCBP2L has 0 variants, skipping\n",
      "[INFO]\tGene SVIL has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928105 has 0 variants, skipping\n",
      "[INFO]\tGene NKD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5186 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AADACP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928166 has 0 variants, skipping\n",
      "[INFO]\tGene RFX4 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA33 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14EP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02006 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INSIG2 has 0 variants, skipping\n",
      "[INFO]\tGene CBY2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01487 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP61 has 0 variants, skipping\n",
      "[INFO]\tGene PLCH1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TIMELESS has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL21 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNAB1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TIPARP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00886 has 0 variants, skipping\n",
      "[INFO]\tGene PA2G4P4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00880 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02029 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00881 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHC has 0 variants, skipping\n",
      "[INFO]\tGene VPS33A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4458 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996447 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA5 has 0 variants, skipping\n",
      "[INFO]\tGene CTSZ has 0 variants, skipping\n",
      "[INFO]\tGene CTSW has 0 variants, skipping\n",
      "[INFO]\tGene CTSO has 0 variants, skipping\n",
      "[INFO]\tGene MCU has 0 variants, skipping\n",
      "[INFO]\tGene C5orf64 has 0 variants, skipping\n",
      "[INFO]\tGene GPX1 has 0 variants, skipping\n",
      "[INFO]\tGene MOB3C has 0 variants, skipping\n",
      "[INFO]\tGene USP17L2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF2KMT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3919 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG30 has 0 variants, skipping\n",
      "[INFO]\tGene IL12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01100 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM59-IFT80 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC52 has 0 variants, skipping\n",
      "[INFO]\tGene OR9Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR9I1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC55 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM198 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H1 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL10 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA7 has 0 variants, skipping\n",
      "[INFO]\tGene OBP2B has 0 variants, skipping\n",
      "[INFO]\tGene RHOD has 0 variants, skipping\n",
      "[INFO]\tGene TMEM97 has 0 variants, skipping\n",
      "[INFO]\tGene LATS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02067 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01192 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1263 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01324 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01322 has 0 variants, skipping\n",
      "[INFO]\tGene SSR1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374194 has 0 variants, skipping\n",
      "[INFO]\tGene RABEPK has 0 variants, skipping\n",
      "[INFO]\tGene RFX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01327 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN6 has 0 variants, skipping\n",
      "[INFO]\tGene MANSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC77P has 0 variants, skipping\n",
      "[INFO]\tGene EGFEM1P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02082 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF9 has 0 variants, skipping\n",
      "[INFO]\tGene NCMAP has 0 variants, skipping\n",
      "[INFO]\tGene C1orf189 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD6 has 0 variants, skipping\n",
      "[INFO]\tGene CLSTN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128164 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6828 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB134 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR569 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM212-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATIC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02068 has 0 variants, skipping\n",
      "[INFO]\tGene FAM106C has 0 variants, skipping\n",
      "[INFO]\tGene STK3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC59 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO6 has 0 variants, skipping\n",
      "[INFO]\tGene CBWD1 has 0 variants, skipping\n",
      "[INFO]\tGene ASH1L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM176A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR10 has 0 variants, skipping\n",
      "[INFO]\tGene BEX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H15 has 0 variants, skipping\n",
      "[INFO]\tGene USE1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGRKT has 0 variants, skipping\n",
      "[INFO]\tGene CISD1 has 0 variants, skipping\n",
      "[INFO]\tGene ITFG2 has 0 variants, skipping\n",
      "[INFO]\tGene BRK1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2D has 0 variants, skipping\n",
      "[INFO]\tGene NLGN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4789 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01209 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02015 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01014 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928739 has 0 variants, skipping\n",
      "[INFO]\tGene TLR5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-4 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf49 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF823 has 0 variants, skipping\n",
      "[INFO]\tGene GNG14 has 0 variants, skipping\n",
      "[INFO]\tGene PEX5L-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02053 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928882 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724604 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01994 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01995 has 0 variants, skipping\n",
      "[INFO]\tGene STBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00888 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63E has 0 variants, skipping\n",
      "[INFO]\tGene KLHL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EIF2AK2 has 0 variants, skipping\n",
      "[INFO]\tGene LCE6A has 0 variants, skipping\n",
      "[INFO]\tGene FBXO17 has 0 variants, skipping\n",
      "[INFO]\tGene PDE4B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PHOX2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4448 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR3E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OLAH has 0 variants, skipping\n",
      "[INFO]\tGene BRIX1 has 0 variants, skipping\n",
      "[INFO]\tGene BRF2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTSL3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA4 has 0 variants, skipping\n",
      "[INFO]\tGene MEP1A has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L1 has 0 variants, skipping\n",
      "[INFO]\tGene LYL1 has 0 variants, skipping\n",
      "[INFO]\tGene CD180 has 0 variants, skipping\n",
      "[INFO]\tGene BCAM has 0 variants, skipping\n",
      "[INFO]\tGene LPA has 0 variants, skipping\n",
      "[INFO]\tGene LOXL1 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf71 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM13 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf73 has 0 variants, skipping\n",
      "[INFO]\tGene DENR has 0 variants, skipping\n",
      "[INFO]\tGene TLN1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf100 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD66 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY6 has 0 variants, skipping\n",
      "[INFO]\tGene MMAB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01840 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02069 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM39A has 0 variants, skipping\n",
      "[INFO]\tGene EHHADH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A9 has 0 variants, skipping\n",
      "[INFO]\tGene IGF2BP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NMRAL2P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02020 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02052 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1248 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA81 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA63 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02043 has 0 variants, skipping\n",
      "[INFO]\tGene ADIPOQ-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AKT1 has 0 variants, skipping\n",
      "[INFO]\tGene GRK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929106 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRH has 0 variants, skipping\n",
      "[INFO]\tGene ADM has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR410 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131635 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01991 has 0 variants, skipping\n",
      "[INFO]\tGene LPP-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42393 has 0 variants, skipping\n",
      "[INFO]\tGene LPP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF831 has 0 variants, skipping\n",
      "[INFO]\tGene GNG10 has 0 variants, skipping\n",
      "[INFO]\tGene P3H2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAR-I has 0 variants, skipping\n",
      "[INFO]\tGene OSTN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A1 has 0 variants, skipping\n",
      "[INFO]\tGene TENT5C has 0 variants, skipping\n",
      "[INFO]\tGene FOXC1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXF2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXF1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF12-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGC2889 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYS has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK11A has 0 variants, skipping\n",
      "[INFO]\tGene OPA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02038 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02026 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA2P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02028 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02036 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02037 has 0 variants, skipping\n",
      "[INFO]\tGene LCE4A has 0 variants, skipping\n",
      "[INFO]\tGene LCE1F has 0 variants, skipping\n",
      "[INFO]\tGene LCE1E has 0 variants, skipping\n",
      "[INFO]\tGene LINC00884 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WASF3 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC16A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01968 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01972 has 0 variants, skipping\n",
      "[INFO]\tGene GKN2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM61 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3137 has 0 variants, skipping\n",
      "[INFO]\tGene XXYLT1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PROKR2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374297 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12B has 0 variants, skipping\n",
      "[INFO]\tGene WDR83 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01983 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6829 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00885 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19-TCTEX1D2 has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01063 has 0 variants, skipping\n",
      "[INFO]\tGene TAF15 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK6 has 0 variants, skipping\n",
      "[INFO]\tGene POM121C has 0 variants, skipping\n",
      "[INFO]\tGene PCNX2 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN2 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR543 has 0 variants, skipping\n",
      "[INFO]\tGene MIR873 has 0 variants, skipping\n",
      "[INFO]\tGene MIR199A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR196A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR665 has 0 variants, skipping\n",
      "[INFO]\tGene MIR187 has 0 variants, skipping\n",
      "[INFO]\tGene MIR183 has 0 variants, skipping\n",
      "[INFO]\tGene MIR182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR885 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181C has 0 variants, skipping\n",
      "[INFO]\tGene MIR744 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30D has 0 variants, skipping\n",
      "[INFO]\tGene MIR34A has 0 variants, skipping\n",
      "[INFO]\tGene MIR10B has 0 variants, skipping\n",
      "[INFO]\tGene MIR10A has 0 variants, skipping\n",
      "[INFO]\tGene MIR708 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR148A has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR129-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR208A has 0 variants, skipping\n",
      "[INFO]\tGene MTFR2 has 0 variants, skipping\n",
      "[INFO]\tGene PRCC has 0 variants, skipping\n",
      "[INFO]\tGene CCL26 has 0 variants, skipping\n",
      "[INFO]\tGene PKDREJ has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02012 has 0 variants, skipping\n",
      "[INFO]\tGene LOC220729 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR922 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf158 has 0 variants, skipping\n",
      "[INFO]\tGene POLA1 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFR has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984282 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA8 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD7 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB4B has 0 variants, skipping\n",
      "[INFO]\tGene LMLN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18DP has 0 variants, skipping\n",
      "[INFO]\tGene FAM157A has 0 variants, skipping\n",
      "[INFO]\tGene JMJD7 has 0 variants, skipping\n",
      "[INFO]\tGene CLCA4 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf232 has 0 variants, skipping\n",
      "[INFO]\tGene KDM1B has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A7-CYP3A51P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652931 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642131 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK4-TEX40 has 0 variants, skipping\n",
      "[INFO]\tGene FAAHP1 has 0 variants, skipping\n",
      "[INFO]\tGene SQOR has 0 variants, skipping\n",
      "[INFO]\tGene GPT has 0 variants, skipping\n",
      "[INFO]\tGene GPR31 has 0 variants, skipping\n",
      "[INFO]\tGene GRK5 has 0 variants, skipping\n",
      "[INFO]\tGene MAFIP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM269-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF876P has 0 variants, skipping\n",
      "[INFO]\tGene KRT19 has 0 variants, skipping\n",
      "[INFO]\tGene KRT15 has 0 variants, skipping\n",
      "[INFO]\tGene KRT13 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA11P has 0 variants, skipping\n",
      "[INFO]\tGene MTMR11 has 0 variants, skipping\n",
      "[INFO]\tGene AARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374338 has 0 variants, skipping\n",
      "[INFO]\tGene APBA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129917 has 0 variants, skipping\n",
      "[INFO]\tGene ARPC5 has 0 variants, skipping\n",
      "[INFO]\tGene CTF1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM172A has 0 variants, skipping\n",
      "[INFO]\tGene OVCA2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM210B has 0 variants, skipping\n",
      "[INFO]\tGene H2BC11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374344 has 0 variants, skipping\n",
      "[INFO]\tGene TMED11P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130872 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM255B has 0 variants, skipping\n",
      "[INFO]\tGene MED22 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS19 has 0 variants, skipping\n",
      "[INFO]\tGene DQX1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132C has 0 variants, skipping\n",
      "[INFO]\tGene DMAC1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA22 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf120 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4800 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF638 has 0 variants, skipping\n",
      "[INFO]\tGene HPR has 0 variants, skipping\n",
      "[INFO]\tGene NECAB2 has 0 variants, skipping\n",
      "[INFO]\tGene GOLIM4 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B11 has 0 variants, skipping\n",
      "[INFO]\tGene RPL7A has 0 variants, skipping\n",
      "[INFO]\tGene H2BE1 has 0 variants, skipping\n",
      "[INFO]\tGene NOP14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS4Y1 has 0 variants, skipping\n",
      "[INFO]\tGene MAOB has 0 variants, skipping\n",
      "[INFO]\tGene TGFBR2 has 0 variants, skipping\n",
      "[INFO]\tGene TGM4 has 0 variants, skipping\n",
      "[INFO]\tGene HTT-AS has 0 variants, skipping\n",
      "[INFO]\tGene SERPINC1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00955 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02171 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86EP has 0 variants, skipping\n",
      "[INFO]\tGene GCG has 0 variants, skipping\n",
      "[INFO]\tGene STX18-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene STX18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928279 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01396 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928306 has 0 variants, skipping\n",
      "[INFO]\tGene MUS81 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01587 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02495 has 0 variants, skipping\n",
      "[INFO]\tGene MUC5B has 0 variants, skipping\n",
      "[INFO]\tGene XAB2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD7B has 0 variants, skipping\n",
      "[INFO]\tGene LHX9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02482 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP3 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP14P has 0 variants, skipping\n",
      "[INFO]\tGene AGAP13P has 0 variants, skipping\n",
      "[INFO]\tGene CLEC3A has 0 variants, skipping\n",
      "[INFO]\tGene PGM5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129931 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02447 has 0 variants, skipping\n",
      "[INFO]\tGene GRIK5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4798 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4274 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB12 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB11 has 0 variants, skipping\n",
      "[INFO]\tGene PYROXD2 has 0 variants, skipping\n",
      "[INFO]\tGene COX7B has 0 variants, skipping\n",
      "[INFO]\tGene COX7A1 has 0 variants, skipping\n",
      "[INFO]\tGene COX6B1 has 0 variants, skipping\n",
      "[INFO]\tGene COX5B has 0 variants, skipping\n",
      "[INFO]\tGene SLC31A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC31A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02517 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE4 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf88 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L6P has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3138 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02360 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02270 has 0 variants, skipping\n",
      "[INFO]\tGene VHLL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01097 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01096 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5091 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01182 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01085 has 0 variants, skipping\n",
      "[INFO]\tGene CPEB2-DT has 0 variants, skipping\n",
      "[INFO]\tGene GALR1 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE12I has 0 variants, skipping\n",
      "[INFO]\tGene GAGE6 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2A has 0 variants, skipping\n",
      "[INFO]\tGene KRT20 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA6 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA3 has 0 variants, skipping\n",
      "[INFO]\tGene TAPT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STC2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02493 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA75B has 0 variants, skipping\n",
      "[INFO]\tGene KANK4 has 0 variants, skipping\n",
      "[INFO]\tGene KIF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370687 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375131 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370708 has 0 variants, skipping\n",
      "[INFO]\tGene KCNN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLIT2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BAHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7978 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374516 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505912 has 0 variants, skipping\n",
      "[INFO]\tGene KCNV1 has 0 variants, skipping\n",
      "[INFO]\tGene BEX3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC2 has 0 variants, skipping\n",
      "[INFO]\tGene ST20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02194 has 0 variants, skipping\n",
      "[INFO]\tGene PPTC7 has 0 variants, skipping\n",
      "[INFO]\tGene SEPSECS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929161 has 0 variants, skipping\n",
      "[INFO]\tGene ALX4 has 0 variants, skipping\n",
      "[INFO]\tGene ENSA has 0 variants, skipping\n",
      "[INFO]\tGene STIM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF2 has 0 variants, skipping\n",
      "[INFO]\tGene CSTA has 0 variants, skipping\n",
      "[INFO]\tGene CSN3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02261 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4275 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02364 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02472 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02497 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02506 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377651 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02353 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928622 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02484 has 0 variants, skipping\n",
      "[INFO]\tGene LOC439933 has 0 variants, skipping\n",
      "[INFO]\tGene SCNM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255B1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02505 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4801 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf48 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02513 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01258 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02278 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2E has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4D has 0 variants, skipping\n",
      "[INFO]\tGene TMEM229A has 0 variants, skipping\n",
      "[INFO]\tGene DENND2A has 0 variants, skipping\n",
      "[INFO]\tGene FAM184B has 0 variants, skipping\n",
      "[INFO]\tGene PDGFD has 0 variants, skipping\n",
      "[INFO]\tGene MYH7 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf229 has 0 variants, skipping\n",
      "[INFO]\tGene AGA has 0 variants, skipping\n",
      "[INFO]\tGene LOC401127 has 0 variants, skipping\n",
      "[INFO]\tGene UGDH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene INAFM2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC344967 has 0 variants, skipping\n",
      "[INFO]\tGene TMED4 has 0 variants, skipping\n",
      "[INFO]\tGene HDHD2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC159 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4802 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643802 has 0 variants, skipping\n",
      "[INFO]\tGene UCHL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC2 has 0 variants, skipping\n",
      "[INFO]\tGene NEXN has 0 variants, skipping\n",
      "[INFO]\tGene CRTAP has 0 variants, skipping\n",
      "[INFO]\tGene VTI1B has 0 variants, skipping\n",
      "[INFO]\tGene SRPK3 has 0 variants, skipping\n",
      "[INFO]\tGene PLAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4717 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4709 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4660 has 0 variants, skipping\n",
      "[INFO]\tGene FAM20B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374428 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02383 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02475 has 0 variants, skipping\n",
      "[INFO]\tGene LSM7 has 0 variants, skipping\n",
      "[INFO]\tGene VPS28 has 0 variants, skipping\n",
      "[INFO]\tGene SEC31B has 0 variants, skipping\n",
      "[INFO]\tGene METTL1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB8A has 0 variants, skipping\n",
      "[INFO]\tGene MIR8053 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927179 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B1 has 0 variants, skipping\n",
      "[INFO]\tGene TXNDC17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927157 has 0 variants, skipping\n",
      "[INFO]\tGene OCIAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02480 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF350 has 0 variants, skipping\n",
      "[INFO]\tGene GNB4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4449 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA26 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01618 has 0 variants, skipping\n",
      "[INFO]\tGene LNX1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506444 has 0 variants, skipping\n",
      "[INFO]\tGene RPL21P44 has 0 variants, skipping\n",
      "[INFO]\tGene PATL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02283 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02260 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AADAC has 0 variants, skipping\n",
      "[INFO]\tGene CACNG7 has 0 variants, skipping\n",
      "[INFO]\tGene DNAAF5 has 0 variants, skipping\n",
      "[INFO]\tGene SMPD4BP has 0 variants, skipping\n",
      "[INFO]\tGene RGS9BP has 0 variants, skipping\n",
      "[INFO]\tGene UBR7 has 0 variants, skipping\n",
      "[INFO]\tGene CBR1 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA6 has 0 variants, skipping\n",
      "[INFO]\tGene CAV1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXE1 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02494 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02619 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AG1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02271 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF100 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01093 has 0 variants, skipping\n",
      "[INFO]\tGene TRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02232 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927237 has 0 variants, skipping\n",
      "[INFO]\tGene PROX1 has 0 variants, skipping\n",
      "[INFO]\tGene UBA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11GP has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4G has 0 variants, skipping\n",
      "[INFO]\tGene LOC550113 has 0 variants, skipping\n",
      "[INFO]\tGene SYT14P1 has 0 variants, skipping\n",
      "[INFO]\tGene FTLP10 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11BNL has 0 variants, skipping\n",
      "[INFO]\tGene RASL11A has 0 variants, skipping\n",
      "[INFO]\tGene GNS has 0 variants, skipping\n",
      "[INFO]\tGene GOT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377267 has 0 variants, skipping\n",
      "[INFO]\tGene EDC3 has 0 variants, skipping\n",
      "[INFO]\tGene PSG1 has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S2AP has 0 variants, skipping\n",
      "[INFO]\tGene CSN1S2BP has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS2 has 0 variants, skipping\n",
      "[INFO]\tGene THOC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996842 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP8 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL4 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO3 has 0 variants, skipping\n",
      "[INFO]\tGene CBR4 has 0 variants, skipping\n",
      "[INFO]\tGene HAVCR2 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG2 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD14 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ2 has 0 variants, skipping\n",
      "[INFO]\tGene VNN1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02499 has 0 variants, skipping\n",
      "[INFO]\tGene DGKQ has 0 variants, skipping\n",
      "[INFO]\tGene POTEJ has 0 variants, skipping\n",
      "[INFO]\tGene MIR147B has 0 variants, skipping\n",
      "[INFO]\tGene PPBPP2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC1A6 has 0 variants, skipping\n",
      "[INFO]\tGene MTM1 has 0 variants, skipping\n",
      "[INFO]\tGene NUBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02562 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02483 has 0 variants, skipping\n",
      "[INFO]\tGene EPB42 has 0 variants, skipping\n",
      "[INFO]\tGene SYNPO2L has 0 variants, skipping\n",
      "[INFO]\tGene SNX27 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC2 has 0 variants, skipping\n",
      "[INFO]\tGene CRACR2B has 0 variants, skipping\n",
      "[INFO]\tGene NXN has 0 variants, skipping\n",
      "[INFO]\tGene HIF3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4450 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AH has 0 variants, skipping\n",
      "[INFO]\tGene POLR1D has 0 variants, skipping\n",
      "[INFO]\tGene SNORD161 has 0 variants, skipping\n",
      "[INFO]\tGene ZHX2 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG15 has 0 variants, skipping\n",
      "[INFO]\tGene MGC70870 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00989 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT4 has 0 variants, skipping\n",
      "[INFO]\tGene WDR64 has 0 variants, skipping\n",
      "[INFO]\tGene MMP28 has 0 variants, skipping\n",
      "[INFO]\tGene FKRP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00575 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD6 has 0 variants, skipping\n",
      "[INFO]\tGene TIPRL has 0 variants, skipping\n",
      "[INFO]\tGene PYDC1 has 0 variants, skipping\n",
      "[INFO]\tGene APCDD1 has 0 variants, skipping\n",
      "[INFO]\tGene UST has 0 variants, skipping\n",
      "[INFO]\tGene SNORD143 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD144 has 0 variants, skipping\n",
      "[INFO]\tGene EDEM3 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM4 has 0 variants, skipping\n",
      "[INFO]\tGene STON1 has 0 variants, skipping\n",
      "[INFO]\tGene STON1-GTF2A1L has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928978 has 0 variants, skipping\n",
      "[INFO]\tGene ADPGK-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4451 has 0 variants, skipping\n",
      "[INFO]\tGene BTC has 0 variants, skipping\n",
      "[INFO]\tGene PATL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4452 has 0 variants, skipping\n",
      "[INFO]\tGene SUCLA2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB11FIP1 has 0 variants, skipping\n",
      "[INFO]\tGene AFF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996750 has 0 variants, skipping\n",
      "[INFO]\tGene SFTPA2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF169 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1K-DT has 0 variants, skipping\n",
      "[INFO]\tGene PIGY-DT has 0 variants, skipping\n",
      "[INFO]\tGene ARFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene LBHD2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4C has 0 variants, skipping\n",
      "[INFO]\tGene FAM13A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFSWAP has 0 variants, skipping\n",
      "[INFO]\tGene SNCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCPRESS2 has 0 variants, skipping\n",
      "[INFO]\tGene EPOR has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929210 has 0 variants, skipping\n",
      "[INFO]\tGene GAN has 0 variants, skipping\n",
      "[INFO]\tGene LINC02114 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63B has 0 variants, skipping\n",
      "[INFO]\tGene BMPR1B-DT has 0 variants, skipping\n",
      "[INFO]\tGene ALOX5AP has 0 variants, skipping\n",
      "[INFO]\tGene ACOXL has 0 variants, skipping\n",
      "[INFO]\tGene LINC02267 has 0 variants, skipping\n",
      "[INFO]\tGene STPG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNU4ATAC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377342 has 0 variants, skipping\n",
      "[INFO]\tGene OR52M1 has 0 variants, skipping\n",
      "[INFO]\tGene CORO1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3684 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507053 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC78 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC49 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929353 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA101A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01216 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8066 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1255A has 0 variants, skipping\n",
      "[INFO]\tGene FLJ20021 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP7 has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D3 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377621 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2E1 has 0 variants, skipping\n",
      "[INFO]\tGene MLANA has 0 variants, skipping\n",
      "[INFO]\tGene UBA2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF169 has 0 variants, skipping\n",
      "[INFO]\tGene FAM200B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723704 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A10 has 0 variants, skipping\n",
      "[INFO]\tGene OR6Y1 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD12 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA6 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TET2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RRM2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF38-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BPHL has 0 variants, skipping\n",
      "[INFO]\tGene SRA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929577 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP1-GIMAP5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02173 has 0 variants, skipping\n",
      "[INFO]\tGene NMNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSD has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4L2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929621 has 0 variants, skipping\n",
      "[INFO]\tGene MMP9 has 0 variants, skipping\n",
      "[INFO]\tGene DLX2 has 0 variants, skipping\n",
      "[INFO]\tGene HBQ1 has 0 variants, skipping\n",
      "[INFO]\tGene HBE1 has 0 variants, skipping\n",
      "[INFO]\tGene HAS2 has 0 variants, skipping\n",
      "[INFO]\tGene H1-1 has 0 variants, skipping\n",
      "[INFO]\tGene GJA10 has 0 variants, skipping\n",
      "[INFO]\tGene RALY has 0 variants, skipping\n",
      "[INFO]\tGene SLC8B1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC170 has 0 variants, skipping\n",
      "[INFO]\tGene PDZRN4 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7D4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H3 has 0 variants, skipping\n",
      "[INFO]\tGene SPIN4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF729 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-A has 0 variants, skipping\n",
      "[INFO]\tGene EIPR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00526 has 0 variants, skipping\n",
      "[INFO]\tGene FOSB has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF11B has 0 variants, skipping\n",
      "[INFO]\tGene RASSF7 has 0 variants, skipping\n",
      "[INFO]\tGene BIK has 0 variants, skipping\n",
      "[INFO]\tGene PANCR has 0 variants, skipping\n",
      "[INFO]\tGene TMEM52 has 0 variants, skipping\n",
      "[INFO]\tGene PRTN3 has 0 variants, skipping\n",
      "[INFO]\tGene KLK10 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS8 has 0 variants, skipping\n",
      "[INFO]\tGene MANCR has 0 variants, skipping\n",
      "[INFO]\tGene LOC440982 has 0 variants, skipping\n",
      "[INFO]\tGene MKX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3664 has 0 variants, skipping\n",
      "[INFO]\tGene CA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01520 has 0 variants, skipping\n",
      "[INFO]\tGene LNCAROD has 0 variants, skipping\n",
      "[INFO]\tGene ITPRIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0002 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01495 has 0 variants, skipping\n",
      "[INFO]\tGene ETS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507144 has 0 variants, skipping\n",
      "[INFO]\tGene CYYR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02366 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6852 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5588 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf46 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2861 has 0 variants, skipping\n",
      "[INFO]\tGene TMC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5P3 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM79 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR567 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927318 has 0 variants, skipping\n",
      "[INFO]\tGene ILRUN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01284 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1912 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927378 has 0 variants, skipping\n",
      "[INFO]\tGene RP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6511A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6887 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6875 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6868 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6797 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6754 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB24 has 0 variants, skipping\n",
      "[INFO]\tGene TFCP2 has 0 variants, skipping\n",
      "[INFO]\tGene CCL21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF814 has 0 variants, skipping\n",
      "[INFO]\tGene H4-16 has 0 variants, skipping\n",
      "[INFO]\tGene KRT74 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf50 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A48 has 0 variants, skipping\n",
      "[INFO]\tGene RABEP2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMTA1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene CTNNA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CIPC has 0 variants, skipping\n",
      "[INFO]\tGene MIR765 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1243 has 0 variants, skipping\n",
      "[INFO]\tGene DRAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L2 has 0 variants, skipping\n",
      "[INFO]\tGene COLEC10 has 0 variants, skipping\n",
      "[INFO]\tGene PARP4 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM2 has 0 variants, skipping\n",
      "[INFO]\tGene NPC2 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL28 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8082 has 0 variants, skipping\n",
      "[INFO]\tGene TFDP3 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf78 has 0 variants, skipping\n",
      "[INFO]\tGene AMBN has 0 variants, skipping\n",
      "[INFO]\tGene PIPOX has 0 variants, skipping\n",
      "[INFO]\tGene GDF9 has 0 variants, skipping\n",
      "[INFO]\tGene VAV2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC4 has 0 variants, skipping\n",
      "[INFO]\tGene LCN15 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR9 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB3 has 0 variants, skipping\n",
      "[INFO]\tGene RXRG has 0 variants, skipping\n",
      "[INFO]\tGene RP2 has 0 variants, skipping\n",
      "[INFO]\tGene RORB has 0 variants, skipping\n",
      "[INFO]\tGene WWP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF546 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02262 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02263 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01378 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02264 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00488 has 0 variants, skipping\n",
      "[INFO]\tGene CHIAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729218 has 0 variants, skipping\n",
      "[INFO]\tGene METTL14-DT has 0 variants, skipping\n",
      "[INFO]\tGene PAICS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929762 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01365 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02502 has 0 variants, skipping\n",
      "[INFO]\tGene CT45A3 has 0 variants, skipping\n",
      "[INFO]\tGene PP12613 has 0 variants, skipping\n",
      "[INFO]\tGene NPDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES has 0 variants, skipping\n",
      "[INFO]\tGene GMFG has 0 variants, skipping\n",
      "[INFO]\tGene IL21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CETN4P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01091 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02516 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP2L has 0 variants, skipping\n",
      "[INFO]\tGene MIR2054 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA8 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA5 has 0 variants, skipping\n",
      "[INFO]\tGene IFNG has 0 variants, skipping\n",
      "[INFO]\tGene TLN2 has 0 variants, skipping\n",
      "[INFO]\tGene TMIE has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18A has 0 variants, skipping\n",
      "[INFO]\tGene GLIS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCL14 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH10 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02615 has 0 variants, skipping\n",
      "[INFO]\tGene SCARF2 has 0 variants, skipping\n",
      "[INFO]\tGene VSX2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02466 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02465 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02479 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01256 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927359 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02462 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02485 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00613 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02511 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02510 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02172 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00616 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A11-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00499 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377448 has 0 variants, skipping\n",
      "[INFO]\tGene MINDY4B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927516 has 0 variants, skipping\n",
      "[INFO]\tGene SCOC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR51T1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52B6 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E6 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E4 has 0 variants, skipping\n",
      "[INFO]\tGene INSL6 has 0 variants, skipping\n",
      "[INFO]\tGene TNRC18P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02432 has 0 variants, skipping\n",
      "[INFO]\tGene BCLAF3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377623 has 0 variants, skipping\n",
      "[INFO]\tGene FABP12 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM179B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3139 has 0 variants, skipping\n",
      "[INFO]\tGene SMARCA5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377458 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927636 has 0 variants, skipping\n",
      "[INFO]\tGene IGHMBP2 has 0 variants, skipping\n",
      "[INFO]\tGene IGFBP6 has 0 variants, skipping\n",
      "[INFO]\tGene IFNW1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNB1 has 0 variants, skipping\n",
      "[INFO]\tGene HHIP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02266 has 0 variants, skipping\n",
      "[INFO]\tGene IL10 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SMAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01095 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7849 has 0 variants, skipping\n",
      "[INFO]\tGene SPATS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4799 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377480 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02355 has 0 variants, skipping\n",
      "[INFO]\tGene GJA4 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1A2 has 0 variants, skipping\n",
      "[INFO]\tGene EDN2 has 0 variants, skipping\n",
      "[INFO]\tGene GFPT1 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC2 has 0 variants, skipping\n",
      "[INFO]\tGene RALBP1 has 0 variants, skipping\n",
      "[INFO]\tGene UTS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRY has 0 variants, skipping\n",
      "[INFO]\tGene MRPL10 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0B has 0 variants, skipping\n",
      "[INFO]\tGene LOC729558 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD73A has 0 variants, skipping\n",
      "[INFO]\tGene CCNA2 has 0 variants, skipping\n",
      "[INFO]\tGene CBR3 has 0 variants, skipping\n",
      "[INFO]\tGene NAGK has 0 variants, skipping\n",
      "[INFO]\tGene MIR3140 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4453HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4453 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729870 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100419170 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724776 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02433 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM198B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3688-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3688-2 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA12B has 0 variants, skipping\n",
      "[INFO]\tGene GRHPR has 0 variants, skipping\n",
      "[INFO]\tGene FZD4 has 0 variants, skipping\n",
      "[INFO]\tGene GNG3 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM10B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02233 has 0 variants, skipping\n",
      "[INFO]\tGene TTC34 has 0 variants, skipping\n",
      "[INFO]\tGene FMO3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928052 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1C1 has 0 variants, skipping\n",
      "[INFO]\tGene GK3P has 0 variants, skipping\n",
      "[INFO]\tGene MIR578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01179 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928131 has 0 variants, skipping\n",
      "[INFO]\tGene VPREB1 has 0 variants, skipping\n",
      "[INFO]\tGene ISX has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A has 0 variants, skipping\n",
      "[INFO]\tGene GNGT1 has 0 variants, skipping\n",
      "[INFO]\tGene RGL4 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA2 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R30 has 0 variants, skipping\n",
      "[INFO]\tGene GJB1 has 0 variants, skipping\n",
      "[INFO]\tGene REEP3 has 0 variants, skipping\n",
      "[INFO]\tGene DACH2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02275 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02382 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6082 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02174 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441052 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930370 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548T has 0 variants, skipping\n",
      "[INFO]\tGene FAM83G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF727 has 0 variants, skipping\n",
      "[INFO]\tGene AMY2B has 0 variants, skipping\n",
      "[INFO]\tGene NMB has 0 variants, skipping\n",
      "[INFO]\tGene LINC02269 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02268 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4276 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928551 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM225 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928590 has 0 variants, skipping\n",
      "[INFO]\tGene VMAC has 0 variants, skipping\n",
      "[INFO]\tGene RCCD1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM50A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02509 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6L9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01098 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01099 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00290 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02500 has 0 variants, skipping\n",
      "[INFO]\tGene TEMN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TENM3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1305 has 0 variants, skipping\n",
      "[INFO]\tGene EID3 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A11P has 0 variants, skipping\n",
      "[INFO]\tGene WWC2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389247 has 0 variants, skipping\n",
      "[INFO]\tGene CPZ has 0 variants, skipping\n",
      "[INFO]\tGene PPIG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02363 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02362 has 0 variants, skipping\n",
      "[INFO]\tGene ALG14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02427 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02365 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4B has 0 variants, skipping\n",
      "[INFO]\tGene IGFBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2A3 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A3 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA5 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD5 has 0 variants, skipping\n",
      "[INFO]\tGene SLED1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3945HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3945 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4455 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02436 has 0 variants, skipping\n",
      "[INFO]\tGene MAGIX has 0 variants, skipping\n",
      "[INFO]\tGene FAM47E-STBD1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377590 has 0 variants, skipping\n",
      "[INFO]\tGene RNH1 has 0 variants, skipping\n",
      "[INFO]\tGene AMACR has 0 variants, skipping\n",
      "[INFO]\tGene ACSL5 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ38576 has 0 variants, skipping\n",
      "[INFO]\tGene FGF19 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA5 has 0 variants, skipping\n",
      "[INFO]\tGene WDR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR82 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02374 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339975 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02514 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02492 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRD1 has 0 variants, skipping\n",
      "[INFO]\tGene OXGR1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01262 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01596 has 0 variants, skipping\n",
      "[INFO]\tGene DBET has 0 variants, skipping\n",
      "[INFO]\tGene DDX39A has 0 variants, skipping\n",
      "[INFO]\tGene GPR78 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692A2 has 0 variants, skipping\n",
      "[INFO]\tGene CAMSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5591 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT5 has 0 variants, skipping\n",
      "[INFO]\tGene CA12 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6D has 0 variants, skipping\n",
      "[INFO]\tGene PP7080 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4456 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996325 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506688 has 0 variants, skipping\n",
      "[INFO]\tGene SIN3B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4635 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-3080P12.3 has 0 variants, skipping\n",
      "[INFO]\tGene PHF5A has 0 variants, skipping\n",
      "[INFO]\tGene HPDL has 0 variants, skipping\n",
      "[INFO]\tGene RAX2 has 0 variants, skipping\n",
      "[INFO]\tGene XCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1P3 has 0 variants, skipping\n",
      "[INFO]\tGene AURKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4457 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01511 has 0 variants, skipping\n",
      "[INFO]\tGene RIPOR3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6075 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4277 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02116 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2194D22.4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506858 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374620 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01377 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01019 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02162 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01017 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01020 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02121 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2297D10.2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF768 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD55 has 0 variants, skipping\n",
      "[INFO]\tGene LCE1D has 0 variants, skipping\n",
      "[INFO]\tGene LCE1B has 0 variants, skipping\n",
      "[INFO]\tGene LCE1A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02102 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02236 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4278 has 0 variants, skipping\n",
      "[INFO]\tGene LOC442132 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02123 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RD has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2F has 0 variants, skipping\n",
      "[INFO]\tGene CRYZL1 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG5 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA7 has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL7 has 0 variants, skipping\n",
      "[INFO]\tGene CCR5AS has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B12 has 0 variants, skipping\n",
      "[INFO]\tGene OR8A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02226 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4458HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02199 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4636 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG18 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD123 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02112 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02221 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC15 has 0 variants, skipping\n",
      "[INFO]\tGene ROPN1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6131 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02212 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02213 has 0 variants, skipping\n",
      "[INFO]\tGene OCM2 has 0 variants, skipping\n",
      "[INFO]\tGene OAS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01194 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02220 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD170 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD141A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD141B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM12 has 0 variants, skipping\n",
      "[INFO]\tGene DNALI1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130744 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4637 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02149 has 0 variants, skipping\n",
      "[INFO]\tGene CTD-2350J17.1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR887 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401176 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02150 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929524 has 0 variants, skipping\n",
      "[INFO]\tGene BASP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02111 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02217 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02218 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02223 has 0 variants, skipping\n",
      "[INFO]\tGene DAGLB has 0 variants, skipping\n",
      "[INFO]\tGene NCOA3 has 0 variants, skipping\n",
      "[INFO]\tGene CDH18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02241 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02146 has 0 variants, skipping\n",
      "[INFO]\tGene MYADML2 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf1 has 0 variants, skipping\n",
      "[INFO]\tGene SMYD1 has 0 variants, skipping\n",
      "[INFO]\tGene PMCHL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02239 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02228 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02211 has 0 variants, skipping\n",
      "[INFO]\tGene RP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02103 has 0 variants, skipping\n",
      "[INFO]\tGene LSP1P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929645 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02109 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02064 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374704 has 0 variants, skipping\n",
      "[INFO]\tGene ASIC1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4279 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL4 has 0 variants, skipping\n",
      "[INFO]\tGene AAMP has 0 variants, skipping\n",
      "[INFO]\tGene LZTS3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02120 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02160 has 0 variants, skipping\n",
      "[INFO]\tGene PDPN has 0 variants, skipping\n",
      "[INFO]\tGene MAGED2 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf38 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK8IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL8 has 0 variants, skipping\n",
      "[INFO]\tGene RNF114 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2E has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF3-AMACR has 0 variants, skipping\n",
      "[INFO]\tGene SYCE1 has 0 variants, skipping\n",
      "[INFO]\tGene OR6B3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM149B1 has 0 variants, skipping\n",
      "[INFO]\tGene SCN10A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506406 has 0 variants, skipping\n",
      "[INFO]\tGene MIR580 has 0 variants, skipping\n",
      "[INFO]\tGene LTN1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL29 has 0 variants, skipping\n",
      "[INFO]\tGene NIPBL-DT has 0 variants, skipping\n",
      "[INFO]\tGene TRPV5 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB2 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM3 has 0 variants, skipping\n",
      "[INFO]\tGene GPR162 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374727 has 0 variants, skipping\n",
      "[INFO]\tGene GDNF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02110 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02107 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02119 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM-AS4 has 0 variants, skipping\n",
      "[INFO]\tGene EGFLAM-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TULP3 has 0 variants, skipping\n",
      "[INFO]\tGene LIFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3650 has 0 variants, skipping\n",
      "[INFO]\tGene OSMR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01265 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf88 has 0 variants, skipping\n",
      "[INFO]\tGene RORC has 0 variants, skipping\n",
      "[INFO]\tGene ISG15 has 0 variants, skipping\n",
      "[INFO]\tGene MARS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02104 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00603 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD72 has 0 variants, skipping\n",
      "[INFO]\tGene GARRE1 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926960 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD12 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ32255 has 0 variants, skipping\n",
      "[INFO]\tGene LOC648987 has 0 variants, skipping\n",
      "[INFO]\tGene LOC153684 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132356 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506639 has 0 variants, skipping\n",
      "[INFO]\tGene NIM1K has 0 variants, skipping\n",
      "[INFO]\tGene DEFB130A has 0 variants, skipping\n",
      "[INFO]\tGene COL28A1 has 0 variants, skipping\n",
      "[INFO]\tGene PIGY has 0 variants, skipping\n",
      "[INFO]\tGene FGF10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02224 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS30-DT has 0 variants, skipping\n",
      "[INFO]\tGene GSTA3 has 0 variants, skipping\n",
      "[INFO]\tGene STIMATE-MUSTN1 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM21 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02106 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642366 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02118 has 0 variants, skipping\n",
      "[INFO]\tGene TPT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RILP has 0 variants, skipping\n",
      "[INFO]\tGene LOC257396 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02105 has 0 variants, skipping\n",
      "[INFO]\tGene MIR581 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01033 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467080 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467081 has 0 variants, skipping\n",
      "[INFO]\tGene NTN5 has 0 variants, skipping\n",
      "[INFO]\tGene CCN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449C has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR5687 has 0 variants, skipping\n",
      "[INFO]\tGene RNF138P1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51A7 has 0 variants, skipping\n",
      "[INFO]\tGene OR51A2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD50A has 0 variants, skipping\n",
      "[INFO]\tGene PTTG3P has 0 variants, skipping\n",
      "[INFO]\tGene ARSI has 0 variants, skipping\n",
      "[INFO]\tGene MCRS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPIRE1 has 0 variants, skipping\n",
      "[INFO]\tGene NKAIN4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM140 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ31104 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01948 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM22 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0003 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928505 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02225 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02101 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AE2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02108 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM19 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF479 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH7 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD6 has 0 variants, skipping\n",
      "[INFO]\tGene IRX2 has 0 variants, skipping\n",
      "[INFO]\tGene IDI2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXQ1 has 0 variants, skipping\n",
      "[INFO]\tGene CAMK2N2 has 0 variants, skipping\n",
      "[INFO]\tGene GNG8 has 0 variants, skipping\n",
      "[INFO]\tGene DGCR6L has 0 variants, skipping\n",
      "[INFO]\tGene SMIM15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02057 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928651 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf64-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OGDH has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF18 has 0 variants, skipping\n",
      "[INFO]\tGene IPO11-LRRC70 has 0 variants, skipping\n",
      "[INFO]\tGene GABRE has 0 variants, skipping\n",
      "[INFO]\tGene LOC100303749 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02065 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02229 has 0 variants, skipping\n",
      "[INFO]\tGene CCNDBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAST4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PACSIN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02242 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928858 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02219 has 0 variants, skipping\n",
      "[INFO]\tGene HSPE1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02198 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA50D has 0 variants, skipping\n",
      "[INFO]\tGene OLFM4 has 0 variants, skipping\n",
      "[INFO]\tGene IFI44 has 0 variants, skipping\n",
      "[INFO]\tGene DHX34 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A13 has 0 variants, skipping\n",
      "[INFO]\tGene CYP3A7 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2A6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1A2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF3R has 0 variants, skipping\n",
      "[INFO]\tGene CSF2 has 0 variants, skipping\n",
      "[INFO]\tGene CSF1 has 0 variants, skipping\n",
      "[INFO]\tGene CRH has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB4 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB2 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928924 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13B-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13B-1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H2B has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02197 has 0 variants, skipping\n",
      "[INFO]\tGene FAM78A has 0 variants, skipping\n",
      "[INFO]\tGene MICALL1 has 0 variants, skipping\n",
      "[INFO]\tGene IPP has 0 variants, skipping\n",
      "[INFO]\tGene KDM8 has 0 variants, skipping\n",
      "[INFO]\tGene ELF1 has 0 variants, skipping\n",
      "[INFO]\tGene LIN28A has 0 variants, skipping\n",
      "[INFO]\tGene THAP9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441081 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646652 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20G has 0 variants, skipping\n",
      "[INFO]\tGene KIFC2 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL3 has 0 variants, skipping\n",
      "[INFO]\tGene OCLNP1 has 0 variants, skipping\n",
      "[INFO]\tGene PMCHL2 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4803 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102503427 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02056 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4804 has 0 variants, skipping\n",
      "[INFO]\tGene USP8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379030 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD36C has 0 variants, skipping\n",
      "[INFO]\tGene LOC340090 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02230 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01386 has 0 variants, skipping\n",
      "[INFO]\tGene UPK3BL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01335 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01333 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01331 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD110 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD99 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441086 has 0 variants, skipping\n",
      "[INFO]\tGene MYMK has 0 variants, skipping\n",
      "[INFO]\tGene AJM1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP3-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-4 has 0 variants, skipping\n",
      "[INFO]\tGene CGB8 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPH has 0 variants, skipping\n",
      "[INFO]\tGene LOC441087 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929109 has 0 variants, skipping\n",
      "[INFO]\tGene ABCE1 has 0 variants, skipping\n",
      "[INFO]\tGene NCRUPAR has 0 variants, skipping\n",
      "[INFO]\tGene SNORA47 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOG has 0 variants, skipping\n",
      "[INFO]\tGene GABBR2 has 0 variants, skipping\n",
      "[INFO]\tGene KLLN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929154 has 0 variants, skipping\n",
      "[INFO]\tGene ORMDL3 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf16 has 0 variants, skipping\n",
      "[INFO]\tGene SCAMP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC6 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B10 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB7 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP35 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPU has 0 variants, skipping\n",
      "[INFO]\tGene CBWD3 has 0 variants, skipping\n",
      "[INFO]\tGene S100A2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01455 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA6 has 0 variants, skipping\n",
      "[INFO]\tGene OR5R1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR921 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644936 has 0 variants, skipping\n",
      "[INFO]\tGene CRSP8P has 0 variants, skipping\n",
      "[INFO]\tGene GRHL2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01337 has 0 variants, skipping\n",
      "[INFO]\tGene RASGRF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D4 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRH has 0 variants, skipping\n",
      "[INFO]\tGene CKMT2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3977 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01338 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA18 has 0 variants, skipping\n",
      "[INFO]\tGene CABYR has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8A2 has 0 variants, skipping\n",
      "[INFO]\tGene VCAN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF22P has 0 variants, skipping\n",
      "[INFO]\tGene LINC02059 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4280 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929380 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01949 has 0 variants, skipping\n",
      "[INFO]\tGene HJV has 0 variants, skipping\n",
      "[INFO]\tGene LOC644285 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02488 has 0 variants, skipping\n",
      "[INFO]\tGene ARPIN has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM55 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM161B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02060 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MICU3 has 0 variants, skipping\n",
      "[INFO]\tGene PELATON has 0 variants, skipping\n",
      "[INFO]\tGene CDH26 has 0 variants, skipping\n",
      "[INFO]\tGene MEF2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3660 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01339 has 0 variants, skipping\n",
      "[INFO]\tGene LOC731157 has 0 variants, skipping\n",
      "[INFO]\tGene SGSH has 0 variants, skipping\n",
      "[INFO]\tGene PLIN2 has 0 variants, skipping\n",
      "[INFO]\tGene FUT9 has 0 variants, skipping\n",
      "[INFO]\tGene MMP13 has 0 variants, skipping\n",
      "[INFO]\tGene IGDCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00909 has 0 variants, skipping\n",
      "[INFO]\tGene UICLM has 0 variants, skipping\n",
      "[INFO]\tGene MIR2052HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR2277 has 0 variants, skipping\n",
      "[INFO]\tGene ESS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929710 has 0 variants, skipping\n",
      "[INFO]\tGene MIR583 has 0 variants, skipping\n",
      "[INFO]\tGene USP9X has 0 variants, skipping\n",
      "[INFO]\tGene ADAM10 has 0 variants, skipping\n",
      "[INFO]\tGene RCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene NEK7 has 0 variants, skipping\n",
      "[INFO]\tGene ASTL has 0 variants, skipping\n",
      "[INFO]\tGene DEFB108B has 0 variants, skipping\n",
      "[INFO]\tGene RSPH10B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3616 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3609 has 0 variants, skipping\n",
      "[INFO]\tGene TUB has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01340 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02234 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01846 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02062 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289230 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02113 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133050 has 0 variants, skipping\n",
      "[INFO]\tGene COX7C has 0 variants, skipping\n",
      "[INFO]\tGene MIR548P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00492 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00491 has 0 variants, skipping\n",
      "[INFO]\tGene CEP170P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02867 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ1OT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNK2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02609 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643327 has 0 variants, skipping\n",
      "[INFO]\tGene NFYCP2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf21 has 0 variants, skipping\n",
      "[INFO]\tGene AMMECR1L has 0 variants, skipping\n",
      "[INFO]\tGene TMEM47 has 0 variants, skipping\n",
      "[INFO]\tGene SOX7 has 0 variants, skipping\n",
      "[INFO]\tGene RTP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02801 has 0 variants, skipping\n",
      "[INFO]\tGene ACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCY2 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE9 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS5 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02115 has 0 variants, skipping\n",
      "[INFO]\tGene RAB9BP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01950 has 0 variants, skipping\n",
      "[INFO]\tGene KDM6B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01023 has 0 variants, skipping\n",
      "[INFO]\tGene RDH5 has 0 variants, skipping\n",
      "[INFO]\tGene OVAAL has 0 variants, skipping\n",
      "[INFO]\tGene LOC285638 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01848 has 0 variants, skipping\n",
      "[INFO]\tGene CEP290 has 0 variants, skipping\n",
      "[INFO]\tGene SLURP2 has 0 variants, skipping\n",
      "[INFO]\tGene STARD4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK8 has 0 variants, skipping\n",
      "[INFO]\tGene NREP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927023 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02200 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467216 has 0 variants, skipping\n",
      "[INFO]\tGene PCARE has 0 variants, skipping\n",
      "[INFO]\tGene AHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene CIB4 has 0 variants, skipping\n",
      "[INFO]\tGene SFMBT2 has 0 variants, skipping\n",
      "[INFO]\tGene ALDH8A1 has 0 variants, skipping\n",
      "[INFO]\tGene CCK has 0 variants, skipping\n",
      "[INFO]\tGene HAO1 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT2 has 0 variants, skipping\n",
      "[INFO]\tGene AP2A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927078 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01957 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927100 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467217 has 0 variants, skipping\n",
      "[INFO]\tGene METTL7A has 0 variants, skipping\n",
      "[INFO]\tGene DNHD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00649 has 0 variants, skipping\n",
      "[INFO]\tGene MIR325 has 0 variants, skipping\n",
      "[INFO]\tGene MIR129-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR185 has 0 variants, skipping\n",
      "[INFO]\tGene MIR126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR184 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927190 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02214 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00992 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02147 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02148 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02208 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02215 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379143 has 0 variants, skipping\n",
      "[INFO]\tGene SEC61B has 0 variants, skipping\n",
      "[INFO]\tGene CBX1 has 0 variants, skipping\n",
      "[INFO]\tGene AFG3L2 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA0 has 0 variants, skipping\n",
      "[INFO]\tGene GPR75 has 0 variants, skipping\n",
      "[INFO]\tGene KDELR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5706 has 0 variants, skipping\n",
      "[INFO]\tGene SDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467226 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf45 has 0 variants, skipping\n",
      "[INFO]\tGene SHARPIN has 0 variants, skipping\n",
      "[INFO]\tGene MGC32805 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02201 has 0 variants, skipping\n",
      "[INFO]\tGene LGR6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379152 has 0 variants, skipping\n",
      "[INFO]\tGene SEC24A has 0 variants, skipping\n",
      "[INFO]\tGene TAFA1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM182 has 0 variants, skipping\n",
      "[INFO]\tGene SNED1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927421 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02240 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02039 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927488 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXP4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01309 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02169 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01740 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01781 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02428 has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LMNB1-DT has 0 variants, skipping\n",
      "[INFO]\tGene FADS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3IL1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01184 has 0 variants, skipping\n",
      "[INFO]\tGene ENGASE has 0 variants, skipping\n",
      "[INFO]\tGene C16orf96 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4633 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4460 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS19-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BCR has 0 variants, skipping\n",
      "[INFO]\tGene RPL41 has 0 variants, skipping\n",
      "[INFO]\tGene MGMT has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2A2 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT5 has 0 variants, skipping\n",
      "[INFO]\tGene MEIS1 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB2A1 has 0 variants, skipping\n",
      "[INFO]\tGene GPC2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA9 has 0 variants, skipping\n",
      "[INFO]\tGene HIBADH has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB9 has 0 variants, skipping\n",
      "[INFO]\tGene LRCH4 has 0 variants, skipping\n",
      "[INFO]\tGene P4HA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP97D1 has 0 variants, skipping\n",
      "[INFO]\tGene IL11RA has 0 variants, skipping\n",
      "[INFO]\tGene MIR6830 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3936HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3936 has 0 variants, skipping\n",
      "[INFO]\tGene ATPSCKMT has 0 variants, skipping\n",
      "[INFO]\tGene TH2LCRR has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8L1 has 0 variants, skipping\n",
      "[INFO]\tGene LCT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1294 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1208 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379176 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC160 has 0 variants, skipping\n",
      "[INFO]\tGene ADM5 has 0 variants, skipping\n",
      "[INFO]\tGene KLF15 has 0 variants, skipping\n",
      "[INFO]\tGene OAZ2 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1289-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379183 has 0 variants, skipping\n",
      "[INFO]\tGene CDH15 has 0 variants, skipping\n",
      "[INFO]\tGene CDH6 has 0 variants, skipping\n",
      "[INFO]\tGene PLK1 has 0 variants, skipping\n",
      "[INFO]\tGene PITX3 has 0 variants, skipping\n",
      "[INFO]\tGene PROCR has 0 variants, skipping\n",
      "[INFO]\tGene PEX16 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644215 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3661 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546229 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01843 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00544 has 0 variants, skipping\n",
      "[INFO]\tGene GSTM5 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692C1 has 0 variants, skipping\n",
      "[INFO]\tGene FA2H has 0 variants, skipping\n",
      "[INFO]\tGene FBXL21P has 0 variants, skipping\n",
      "[INFO]\tGene TRPC7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PEAK3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP9B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379192 has 0 variants, skipping\n",
      "[INFO]\tGene NPY6R has 0 variants, skipping\n",
      "[INFO]\tGene OR2L3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2L2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2F2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2D3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2AK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130172 has 0 variants, skipping\n",
      "[INFO]\tGene AADACL2 has 0 variants, skipping\n",
      "[INFO]\tGene LONP1 has 0 variants, skipping\n",
      "[INFO]\tGene HERC2P2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD63B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD63 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379194 has 0 variants, skipping\n",
      "[INFO]\tGene IGIP has 0 variants, skipping\n",
      "[INFO]\tGene DIO1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW2 has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL1 has 0 variants, skipping\n",
      "[INFO]\tGene PXYLP1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXK1 has 0 variants, skipping\n",
      "[INFO]\tGene MSS51 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA45 has 0 variants, skipping\n",
      "[INFO]\tGene HES3 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf66 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf53 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74A has 0 variants, skipping\n",
      "[INFO]\tGene CHD5 has 0 variants, skipping\n",
      "[INFO]\tGene TUFT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM126A has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK13 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM220A has 0 variants, skipping\n",
      "[INFO]\tGene PSD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MALINC1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929719 has 0 variants, skipping\n",
      "[INFO]\tGene ANKHD1-EIF4EBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SP8 has 0 variants, skipping\n",
      "[INFO]\tGene DHX57 has 0 variants, skipping\n",
      "[INFO]\tGene GCSH has 0 variants, skipping\n",
      "[INFO]\tGene MIR6831 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3655 has 0 variants, skipping\n",
      "[INFO]\tGene RAF1 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-2 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-3 has 0 variants, skipping\n",
      "[INFO]\tGene VNN3 has 0 variants, skipping\n",
      "[INFO]\tGene JMJD8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926905 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB17P has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB18P has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB19P has 0 variants, skipping\n",
      "[INFO]\tGene KAZALD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM170A has 0 variants, skipping\n",
      "[INFO]\tGene INPP5D has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB8P has 0 variants, skipping\n",
      "[INFO]\tGene RPS2 has 0 variants, skipping\n",
      "[INFO]\tGene RPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD8 has 0 variants, skipping\n",
      "[INFO]\tGene RGS2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BGRL has 0 variants, skipping\n",
      "[INFO]\tGene RGS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABRA has 0 variants, skipping\n",
      "[INFO]\tGene LOC729080 has 0 variants, skipping\n",
      "[INFO]\tGene CLP1 has 0 variants, skipping\n",
      "[INFO]\tGene SPRY4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TH has 0 variants, skipping\n",
      "[INFO]\tGene TGFBI has 0 variants, skipping\n",
      "[INFO]\tGene TAF10 has 0 variants, skipping\n",
      "[INFO]\tGene STK4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01844 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP26-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPO4 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG6 has 0 variants, skipping\n",
      "[INFO]\tGene ILF2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5197 has 0 variants, skipping\n",
      "[INFO]\tGene C13orf46 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723899 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene MED28 has 0 variants, skipping\n",
      "[INFO]\tGene ITPKC has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHO2 has 0 variants, skipping\n",
      "[INFO]\tGene PROS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R2B-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene STK32A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP8B1 has 0 variants, skipping\n",
      "[INFO]\tGene NOTO has 0 variants, skipping\n",
      "[INFO]\tGene JAKMIP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5F1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546294 has 0 variants, skipping\n",
      "[INFO]\tGene NXPH1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC2-DT has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CKLF-CMTM1 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5100 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO28 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1D has 0 variants, skipping\n",
      "[INFO]\tGene POLB has 0 variants, skipping\n",
      "[INFO]\tGene SCCPDH has 0 variants, skipping\n",
      "[INFO]\tGene NUDT13 has 0 variants, skipping\n",
      "[INFO]\tGene CDC40 has 0 variants, skipping\n",
      "[INFO]\tGene PRR5-ARHGAP8 has 0 variants, skipping\n",
      "[INFO]\tGene FKBPL has 0 variants, skipping\n",
      "[INFO]\tGene ITM2B has 0 variants, skipping\n",
      "[INFO]\tGene XPNPEP3 has 0 variants, skipping\n",
      "[INFO]\tGene CHP2 has 0 variants, skipping\n",
      "[INFO]\tGene LY96 has 0 variants, skipping\n",
      "[INFO]\tGene COX6C has 0 variants, skipping\n",
      "[INFO]\tGene COX6A1 has 0 variants, skipping\n",
      "[INFO]\tGene EBLN2 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf131 has 0 variants, skipping\n",
      "[INFO]\tGene BAZ1B has 0 variants, skipping\n",
      "[INFO]\tGene NDST1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE22 has 0 variants, skipping\n",
      "[INFO]\tGene GOT1L1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF572 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF786 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG has 0 variants, skipping\n",
      "[INFO]\tGene WDR93 has 0 variants, skipping\n",
      "[INFO]\tGene HSD11B1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR6499 has 0 variants, skipping\n",
      "[INFO]\tGene CLMAT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100652758 has 0 variants, skipping\n",
      "[INFO]\tGene CENPB has 0 variants, skipping\n",
      "[INFO]\tGene IBSP has 0 variants, skipping\n",
      "[INFO]\tGene SFPQ has 0 variants, skipping\n",
      "[INFO]\tGene LINC01933 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01470 has 0 variants, skipping\n",
      "[INFO]\tGene FGD5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00920 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A12P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01861 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3141 has 0 variants, skipping\n",
      "[INFO]\tGene GPATCH4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378H has 0 variants, skipping\n",
      "[INFO]\tGene C11orf96 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB3 has 0 variants, skipping\n",
      "[INFO]\tGene PUM3 has 0 variants, skipping\n",
      "[INFO]\tGene IL26 has 0 variants, skipping\n",
      "[INFO]\tGene LSG1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF8 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP161 has 0 variants, skipping\n",
      "[INFO]\tGene SETD1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724404 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02227 has 0 variants, skipping\n",
      "[INFO]\tGene SYCP3 has 0 variants, skipping\n",
      "[INFO]\tGene SELPLG has 0 variants, skipping\n",
      "[INFO]\tGene SELP has 0 variants, skipping\n",
      "[INFO]\tGene LINC02202 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01932 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285626 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01847 has 0 variants, skipping\n",
      "[INFO]\tGene CHERP has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF20 has 0 variants, skipping\n",
      "[INFO]\tGene TUBAL3 has 0 variants, skipping\n",
      "[INFO]\tGene CLUAP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02159 has 0 variants, skipping\n",
      "[INFO]\tGene ATG9B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF680 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01202 has 0 variants, skipping\n",
      "[INFO]\tGene BTG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNRPA1 has 0 variants, skipping\n",
      "[INFO]\tGene JOSD2 has 0 variants, skipping\n",
      "[INFO]\tGene MATN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02143 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102546299 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01947 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927908 has 0 variants, skipping\n",
      "[INFO]\tGene CTB-178M22.2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR585 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G1 has 0 variants, skipping\n",
      "[INFO]\tGene HEIH has 0 variants, skipping\n",
      "[INFO]\tGene LINC01187 has 0 variants, skipping\n",
      "[INFO]\tGene CALR4P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128059 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01366 has 0 variants, skipping\n",
      "[INFO]\tGene TRIML2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70J has 0 variants, skipping\n",
      "[INFO]\tGene MIR3912 has 0 variants, skipping\n",
      "[INFO]\tGene ARMH4 has 0 variants, skipping\n",
      "[INFO]\tGene PIM3 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37D has 0 variants, skipping\n",
      "[INFO]\tGene CAD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288254 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01944 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5003 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100268168 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA74B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC179 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8056 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285593 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01942 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01484 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01485 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01411 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4634 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01951 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE2 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE1 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996385 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507387 has 0 variants, skipping\n",
      "[INFO]\tGene LOC643201 has 0 variants, skipping\n",
      "[INFO]\tGene VPS37B has 0 variants, skipping\n",
      "[INFO]\tGene VPS4B has 0 variants, skipping\n",
      "[INFO]\tGene ZFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC16 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E1 has 0 variants, skipping\n",
      "[INFO]\tGene ATOX1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4281 has 0 variants, skipping\n",
      "[INFO]\tGene VKORC1L1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01574 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF12 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYM5 has 0 variants, skipping\n",
      "[INFO]\tGene C6 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP9 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF13 has 0 variants, skipping\n",
      "[INFO]\tGene TAF7 has 0 variants, skipping\n",
      "[INFO]\tGene OR2H2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB5 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC1B has 0 variants, skipping\n",
      "[INFO]\tGene HEBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS18 has 0 variants, skipping\n",
      "[INFO]\tGene LOC202181 has 0 variants, skipping\n",
      "[INFO]\tGene FAM153A has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986794 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728554 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128340 has 0 variants, skipping\n",
      "[INFO]\tGene RHOU has 0 variants, skipping\n",
      "[INFO]\tGene ARL4C has 0 variants, skipping\n",
      "[INFO]\tGene AACSP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL15 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC1 has 0 variants, skipping\n",
      "[INFO]\tGene DOC2B has 0 variants, skipping\n",
      "[INFO]\tGene HOPX has 0 variants, skipping\n",
      "[INFO]\tGene TMEM183B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF747 has 0 variants, skipping\n",
      "[INFO]\tGene CDK6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377763 has 0 variants, skipping\n",
      "[INFO]\tGene RRM2B has 0 variants, skipping\n",
      "[INFO]\tGene STAT4 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE9 has 0 variants, skipping\n",
      "[INFO]\tGene CD163L1 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996419 has 0 variants, skipping\n",
      "[INFO]\tGene GPR137B has 0 variants, skipping\n",
      "[INFO]\tGene LINC02222 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF506 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00847 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2D has 0 variants, skipping\n",
      "[INFO]\tGene RAB33A has 0 variants, skipping\n",
      "[INFO]\tGene CIAO1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A14 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA4C has 0 variants, skipping\n",
      "[INFO]\tGene MIR8089 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01962 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD96A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD95 has 0 variants, skipping\n",
      "[INFO]\tGene CTC-338M12.4 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM52-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf80 has 0 variants, skipping\n",
      "[INFO]\tGene KRT16 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB39 has 0 variants, skipping\n",
      "[INFO]\tGene HIC2 has 0 variants, skipping\n",
      "[INFO]\tGene NOXO1 has 0 variants, skipping\n",
      "[INFO]\tGene IL11 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF728 has 0 variants, skipping\n",
      "[INFO]\tGene MTTP has 0 variants, skipping\n",
      "[INFO]\tGene ECEL1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF469 has 0 variants, skipping\n",
      "[INFO]\tGene COL4A5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929599 has 0 variants, skipping\n",
      "[INFO]\tGene AANAT has 0 variants, skipping\n",
      "[INFO]\tGene RPL26 has 0 variants, skipping\n",
      "[INFO]\tGene CEP295 has 0 variants, skipping\n",
      "[INFO]\tGene ACP7 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS2 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD5 has 0 variants, skipping\n",
      "[INFO]\tGene CD101 has 0 variants, skipping\n",
      "[INFO]\tGene LOXL4 has 0 variants, skipping\n",
      "[INFO]\tGene ARID5B has 0 variants, skipping\n",
      "[INFO]\tGene RNASEH2C has 0 variants, skipping\n",
      "[INFO]\tGene ETV3 has 0 variants, skipping\n",
      "[INFO]\tGene MYBPH has 0 variants, skipping\n",
      "[INFO]\tGene FICD has 0 variants, skipping\n",
      "[INFO]\tGene GPC3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00266-3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285766 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927691 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01622 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6720 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01600 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02521 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4645 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB9P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927730 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01011 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927759 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02880 has 0 variants, skipping\n",
      "[INFO]\tGene RUFY4 has 0 variants, skipping\n",
      "[INFO]\tGene VNN2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130357 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724096 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02533 has 0 variants, skipping\n",
      "[INFO]\tGene KU-MEL-3 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3691 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927972 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927950 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7853 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5683 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf71 has 0 variants, skipping\n",
      "[INFO]\tGene CLMP has 0 variants, skipping\n",
      "[INFO]\tGene CAMKMT has 0 variants, skipping\n",
      "[INFO]\tGene HCN2 has 0 variants, skipping\n",
      "[INFO]\tGene BMPR2 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S5-TXNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene PIP5K1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RTP2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1E1-BLOC1S5 has 0 variants, skipping\n",
      "[INFO]\tGene WEE2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4436B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506207 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5689HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR5689 has 0 variants, skipping\n",
      "[INFO]\tGene RIIAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM14B has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928191 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D16 has 0 variants, skipping\n",
      "[INFO]\tGene POLR3K has 0 variants, skipping\n",
      "[INFO]\tGene FBXO40 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02530 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A42 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ2 has 0 variants, skipping\n",
      "[INFO]\tGene NR3C2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36AL has 0 variants, skipping\n",
      "[INFO]\tGene RPL39 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB7 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD16B has 0 variants, skipping\n",
      "[INFO]\tGene SAMD10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724843 has 0 variants, skipping\n",
      "[INFO]\tGene CIT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01108 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO31 has 0 variants, skipping\n",
      "[INFO]\tGene JARID2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4639 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene S100A7 has 0 variants, skipping\n",
      "[INFO]\tGene SLN has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928491 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DQA1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374952 has 0 variants, skipping\n",
      "[INFO]\tGene DUSP11 has 0 variants, skipping\n",
      "[INFO]\tGene SOAT2 has 0 variants, skipping\n",
      "[INFO]\tGene UTF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF282 has 0 variants, skipping\n",
      "[INFO]\tGene VAPA has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A1HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928519 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374960 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00581 has 0 variants, skipping\n",
      "[INFO]\tGene POLE4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374972 has 0 variants, skipping\n",
      "[INFO]\tGene NRP2 has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CD3E has 0 variants, skipping\n",
      "[INFO]\tGene CD3D has 0 variants, skipping\n",
      "[INFO]\tGene CCKAR has 0 variants, skipping\n",
      "[INFO]\tGene CALCB has 0 variants, skipping\n",
      "[INFO]\tGene CACNG1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC34A3 has 0 variants, skipping\n",
      "[INFO]\tGene NAA20 has 0 variants, skipping\n",
      "[INFO]\tGene HEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT16 has 0 variants, skipping\n",
      "[INFO]\tGene CRLF3 has 0 variants, skipping\n",
      "[INFO]\tGene RASD1 has 0 variants, skipping\n",
      "[INFO]\tGene CMAHP has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928663 has 0 variants, skipping\n",
      "[INFO]\tGene PLOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LRIG3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC96 has 0 variants, skipping\n",
      "[INFO]\tGene FAHD2B has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6 has 0 variants, skipping\n",
      "[INFO]\tGene SMCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN2A3P has 0 variants, skipping\n",
      "[INFO]\tGene C12orf75 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A25 has 0 variants, skipping\n",
      "[INFO]\tGene OR13J1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285819 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374988 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00240 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100270746 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3143 has 0 variants, skipping\n",
      "[INFO]\tGene GPR17 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R10P has 0 variants, skipping\n",
      "[INFO]\tGene ZNF204P has 0 variants, skipping\n",
      "[INFO]\tGene GUF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01012 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131289 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN12P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf61 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF192P1 has 0 variants, skipping\n",
      "[INFO]\tGene TOB2P1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCH has 0 variants, skipping\n",
      "[INFO]\tGene CFAP206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00533 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01623 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01556 has 0 variants, skipping\n",
      "[INFO]\tGene HCG15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129636 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929006 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01015 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD32B has 0 variants, skipping\n",
      "[INFO]\tGene ENDOG has 0 variants, skipping\n",
      "[INFO]\tGene EPHA2 has 0 variants, skipping\n",
      "[INFO]\tGene IFITM4P has 0 variants, skipping\n",
      "[INFO]\tGene HCG4 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-V has 0 variants, skipping\n",
      "[INFO]\tGene HCP5B has 0 variants, skipping\n",
      "[INFO]\tGene GPR3 has 0 variants, skipping\n",
      "[INFO]\tGene HCG4B has 0 variants, skipping\n",
      "[INFO]\tGene HCG9 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRD1ASP has 0 variants, skipping\n",
      "[INFO]\tGene HLA-J has 0 variants, skipping\n",
      "[INFO]\tGene GJB5 has 0 variants, skipping\n",
      "[INFO]\tGene PGA5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM31-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG17 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-L has 0 variants, skipping\n",
      "[INFO]\tGene HCG18 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM39-RPP21 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02569 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB4 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP47 has 0 variants, skipping\n",
      "[INFO]\tGene GNG13 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF581 has 0 variants, skipping\n",
      "[INFO]\tGene KLF3 has 0 variants, skipping\n",
      "[INFO]\tGene MDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IER3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG20 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4640 has 0 variants, skipping\n",
      "[INFO]\tGene HCG21 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM77 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG27 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02571 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6891 has 0 variants, skipping\n",
      "[INFO]\tGene MICA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG26 has 0 variants, skipping\n",
      "[INFO]\tGene MICB-DT has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1G2-DDX39B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD117 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD84 has 0 variants, skipping\n",
      "[INFO]\tGene DDX39B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287329 has 0 variants, skipping\n",
      "[INFO]\tGene KPNA7 has 0 variants, skipping\n",
      "[INFO]\tGene KRT6A has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-9 has 0 variants, skipping\n",
      "[INFO]\tGene HDC has 0 variants, skipping\n",
      "[INFO]\tGene ODC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA38 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6832 has 0 variants, skipping\n",
      "[INFO]\tGene CARD17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4646 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6E has 0 variants, skipping\n",
      "[INFO]\tGene HK3 has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH5-SAPCD1 has 0 variants, skipping\n",
      "[INFO]\tGene SAPCD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRERF1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1G has 0 variants, skipping\n",
      "[INFO]\tGene SNORD48 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD52 has 0 variants, skipping\n",
      "[INFO]\tGene SOGA1 has 0 variants, skipping\n",
      "[INFO]\tGene XDH has 0 variants, skipping\n",
      "[INFO]\tGene C2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR7E24 has 0 variants, skipping\n",
      "[INFO]\tGene LRRIQ1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D9B has 0 variants, skipping\n",
      "[INFO]\tGene SGPP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF688 has 0 variants, skipping\n",
      "[INFO]\tGene MORN5 has 0 variants, skipping\n",
      "[INFO]\tGene TNXA has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10Q1 has 0 variants, skipping\n",
      "[INFO]\tGene MGA has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507547 has 0 variants, skipping\n",
      "[INFO]\tGene PPT2-EGFL8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6721 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6833 has 0 variants, skipping\n",
      "[INFO]\tGene LUC7L3 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG23 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB6 has 0 variants, skipping\n",
      "[INFO]\tGene GAMT has 0 variants, skipping\n",
      "[INFO]\tGene PABPC5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB129 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100294145 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DPB2 has 0 variants, skipping\n",
      "[INFO]\tGene HCG24 has 0 variants, skipping\n",
      "[INFO]\tGene ARSJ has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378979 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT3 has 0 variants, skipping\n",
      "[INFO]\tGene HCG25 has 0 variants, skipping\n",
      "[INFO]\tGene ZP3 has 0 variants, skipping\n",
      "[INFO]\tGene GPBP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene MYG1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM35A has 0 variants, skipping\n",
      "[INFO]\tGene PBOV1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6873 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD62 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6834 has 0 variants, skipping\n",
      "[INFO]\tGene WASL has 0 variants, skipping\n",
      "[INFO]\tGene USP13 has 0 variants, skipping\n",
      "[INFO]\tGene PEX11G has 0 variants, skipping\n",
      "[INFO]\tGene MIR5004 has 0 variants, skipping\n",
      "[INFO]\tGene GGNBP1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC46A2 has 0 variants, skipping\n",
      "[INFO]\tGene MLXIPL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929188 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3934 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3G has 0 variants, skipping\n",
      "[INFO]\tGene SUCLG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR510 has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01016 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7159 has 0 variants, skipping\n",
      "[INFO]\tGene OR5C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52N5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6835 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10-NUDT3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929243 has 0 variants, skipping\n",
      "[INFO]\tGene PATE4 has 0 variants, skipping\n",
      "[INFO]\tGene GPD1 has 0 variants, skipping\n",
      "[INFO]\tGene CDS1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2C has 0 variants, skipping\n",
      "[INFO]\tGene CDK9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7111 has 0 variants, skipping\n",
      "[INFO]\tGene MIR153-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5690 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285847 has 0 variants, skipping\n",
      "[INFO]\tGene CD302 has 0 variants, skipping\n",
      "[INFO]\tGene MKX has 0 variants, skipping\n",
      "[INFO]\tGene TMED9 has 0 variants, skipping\n",
      "[INFO]\tGene OR7A5 has 0 variants, skipping\n",
      "[INFO]\tGene RBM38 has 0 variants, skipping\n",
      "[INFO]\tGene OR2M4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR609 has 0 variants, skipping\n",
      "[INFO]\tGene MIR596 has 0 variants, skipping\n",
      "[INFO]\tGene RHOB has 0 variants, skipping\n",
      "[INFO]\tGene MIR3925 has 0 variants, skipping\n",
      "[INFO]\tGene DINOL has 0 variants, skipping\n",
      "[INFO]\tGene C6orf163 has 0 variants, skipping\n",
      "[INFO]\tGene PCP4L1 has 0 variants, skipping\n",
      "[INFO]\tGene BSG has 0 variants, skipping\n",
      "[INFO]\tGene MSI2 has 0 variants, skipping\n",
      "[INFO]\tGene MZT1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP155 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02520 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4462 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MACF1 has 0 variants, skipping\n",
      "[INFO]\tGene APOD has 0 variants, skipping\n",
      "[INFO]\tGene APOC4 has 0 variants, skipping\n",
      "[INFO]\tGene APOA2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRN2 has 0 variants, skipping\n",
      "[INFO]\tGene MUC12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00951 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929555 has 0 variants, skipping\n",
      "[INFO]\tGene HES1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA11 has 0 variants, skipping\n",
      "[INFO]\tGene HMX2 has 0 variants, skipping\n",
      "[INFO]\tGene HMGN2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A6OS has 0 variants, skipping\n",
      "[INFO]\tGene PDE6A has 0 variants, skipping\n",
      "[INFO]\tGene ADCY10P1 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM2 has 0 variants, skipping\n",
      "[INFO]\tGene TREML3P has 0 variants, skipping\n",
      "[INFO]\tGene IDO1 has 0 variants, skipping\n",
      "[INFO]\tGene TREML5P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01276 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4641 has 0 variants, skipping\n",
      "[INFO]\tGene NPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CNTD1 has 0 variants, skipping\n",
      "[INFO]\tGene CA11 has 0 variants, skipping\n",
      "[INFO]\tGene CA9 has 0 variants, skipping\n",
      "[INFO]\tGene C1QBP has 0 variants, skipping\n",
      "[INFO]\tGene BUB1B has 0 variants, skipping\n",
      "[INFO]\tGene REELD1 has 0 variants, skipping\n",
      "[INFO]\tGene USP11 has 0 variants, skipping\n",
      "[INFO]\tGene GPR25 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR22 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR3 has 0 variants, skipping\n",
      "[INFO]\tGene FFAR1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR20 has 0 variants, skipping\n",
      "[INFO]\tGene GPR37 has 0 variants, skipping\n",
      "[INFO]\tGene GPR15 has 0 variants, skipping\n",
      "[INFO]\tGene GPR12 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0CP3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401261 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT2 has 0 variants, skipping\n",
      "[INFO]\tGene USH1C has 0 variants, skipping\n",
      "[INFO]\tGene FOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene ERMP1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC15 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS20 has 0 variants, skipping\n",
      "[INFO]\tGene LAT2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRTM1 has 0 variants, skipping\n",
      "[INFO]\tGene SRARP has 0 variants, skipping\n",
      "[INFO]\tGene RASGEF1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6780B has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10 has 0 variants, skipping\n",
      "[INFO]\tGene PIGL has 0 variants, skipping\n",
      "[INFO]\tGene GOLPH3 has 0 variants, skipping\n",
      "[INFO]\tGene PBLD has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02537 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01512 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929705 has 0 variants, skipping\n",
      "[INFO]\tGene PPP6C has 0 variants, skipping\n",
      "[INFO]\tGene SYCE3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4647 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD2 has 0 variants, skipping\n",
      "[INFO]\tGene TLL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375075 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF300P1 has 0 variants, skipping\n",
      "[INFO]\tGene FIRRE has 0 variants, skipping\n",
      "[INFO]\tGene TMEM65 has 0 variants, skipping\n",
      "[INFO]\tGene GET4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926915 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926898 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926934 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM18 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF605 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL23 has 0 variants, skipping\n",
      "[INFO]\tGene CPLX3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927020 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927048 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370980 has 0 variants, skipping\n",
      "[INFO]\tGene OR8H2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927082 has 0 variants, skipping\n",
      "[INFO]\tGene LINCMD1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIG7 has 0 variants, skipping\n",
      "[INFO]\tGene TRAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD18 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA7P has 0 variants, skipping\n",
      "[INFO]\tGene RN7SL1 has 0 variants, skipping\n",
      "[INFO]\tGene TPTE has 0 variants, skipping\n",
      "[INFO]\tGene MIR5685 has 0 variants, skipping\n",
      "[INFO]\tGene RPS16P5 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927136 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01564 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927189 has 0 variants, skipping\n",
      "[INFO]\tGene MLIP-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TNS1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSH has 0 variants, skipping\n",
      "[INFO]\tGene CTBS has 0 variants, skipping\n",
      "[INFO]\tGene NCAN has 0 variants, skipping\n",
      "[INFO]\tGene TRIP6 has 0 variants, skipping\n",
      "[INFO]\tGene TPR has 0 variants, skipping\n",
      "[INFO]\tGene ICOS has 0 variants, skipping\n",
      "[INFO]\tGene METTL16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM185B has 0 variants, skipping\n",
      "[INFO]\tGene DSTYK has 0 variants, skipping\n",
      "[INFO]\tGene PCLO has 0 variants, skipping\n",
      "[INFO]\tGene RFESD has 0 variants, skipping\n",
      "[INFO]\tGene FAM160A2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM210A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF451-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506188 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYC has 0 variants, skipping\n",
      "[INFO]\tGene MIR548U has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00680-GUSBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00680 has 0 variants, skipping\n",
      "[INFO]\tGene MPP7 has 0 variants, skipping\n",
      "[INFO]\tGene OPN3 has 0 variants, skipping\n",
      "[INFO]\tGene LEMD3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX58 has 0 variants, skipping\n",
      "[INFO]\tGene SCAT8 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5R1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723883 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02549 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928307 has 0 variants, skipping\n",
      "[INFO]\tGene EVADR has 0 variants, skipping\n",
      "[INFO]\tGene GRM1 has 0 variants, skipping\n",
      "[INFO]\tGene SACS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01626 has 0 variants, skipping\n",
      "[INFO]\tGene ETV6 has 0 variants, skipping\n",
      "[INFO]\tGene GRN has 0 variants, skipping\n",
      "[INFO]\tGene HSPA5 has 0 variants, skipping\n",
      "[INFO]\tGene APCS has 0 variants, skipping\n",
      "[INFO]\tGene APOF has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A6 has 0 variants, skipping\n",
      "[INFO]\tGene PEAR1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AB1 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378949 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP27-1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110C has 0 variants, skipping\n",
      "[INFO]\tGene SH2D7 has 0 variants, skipping\n",
      "[INFO]\tGene LINGO3 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN25 has 0 variants, skipping\n",
      "[INFO]\tGene PIRT has 0 variants, skipping\n",
      "[INFO]\tGene FAM183A has 0 variants, skipping\n",
      "[INFO]\tGene IGLON5 has 0 variants, skipping\n",
      "[INFO]\tGene YAE1 has 0 variants, skipping\n",
      "[INFO]\tGene LYRM9 has 0 variants, skipping\n",
      "[INFO]\tGene SKAP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPH has 0 variants, skipping\n",
      "[INFO]\tGene AMELX has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15B has 0 variants, skipping\n",
      "[INFO]\tGene ALOX15 has 0 variants, skipping\n",
      "[INFO]\tGene PHEX has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNQ5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRMT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928489 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928516 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD156 has 0 variants, skipping\n",
      "[INFO]\tGene RUSC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928540 has 0 variants, skipping\n",
      "[INFO]\tGene PATJ has 0 variants, skipping\n",
      "[INFO]\tGene LINC02540 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100144595 has 0 variants, skipping\n",
      "[INFO]\tGene CCNK has 0 variants, skipping\n",
      "[INFO]\tGene HMGN3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01621 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01526 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02542 has 0 variants, skipping\n",
      "[INFO]\tGene DLK1 has 0 variants, skipping\n",
      "[INFO]\tGene BTN1A1 has 0 variants, skipping\n",
      "[INFO]\tGene BTG1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM21 has 0 variants, skipping\n",
      "[INFO]\tGene AFF4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377879 has 0 variants, skipping\n",
      "[INFO]\tGene TBX18-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02535 has 0 variants, skipping\n",
      "[INFO]\tGene RBMXL1 has 0 variants, skipping\n",
      "[INFO]\tGene ERI1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF1C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K2 has 0 variants, skipping\n",
      "[INFO]\tGene MASP2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL24 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01483 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3195 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC12 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R5 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R4 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01590 has 0 variants, skipping\n",
      "[INFO]\tGene RBP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928911 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928936 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929057 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4464 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4643 has 0 variants, skipping\n",
      "[INFO]\tGene CASC6 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB3 has 0 variants, skipping\n",
      "[INFO]\tGene TSG1 has 0 variants, skipping\n",
      "[INFO]\tGene MANEA-DT has 0 variants, skipping\n",
      "[INFO]\tGene KIF20A has 0 variants, skipping\n",
      "[INFO]\tGene CRAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR642B has 0 variants, skipping\n",
      "[INFO]\tGene MIR676 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3926-1 has 0 variants, skipping\n",
      "[INFO]\tGene DEGS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927314 has 0 variants, skipping\n",
      "[INFO]\tGene HCP5 has 0 variants, skipping\n",
      "[INFO]\tGene OR10T2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D11 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D10 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T33 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T29 has 0 variants, skipping\n",
      "[INFO]\tGene CHRFAM7A has 0 variants, skipping\n",
      "[INFO]\tGene CTHRC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927365 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D18 has 0 variants, skipping\n",
      "[INFO]\tGene CLMN has 0 variants, skipping\n",
      "[INFO]\tGene C1orf195 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN9-DT has 0 variants, skipping\n",
      "[INFO]\tGene FIGNL2-DT has 0 variants, skipping\n",
      "[INFO]\tGene DHX33-DT has 0 variants, skipping\n",
      "[INFO]\tGene MCHR2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD1 has 0 variants, skipping\n",
      "[INFO]\tGene ATPAF2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP1L has 0 variants, skipping\n",
      "[INFO]\tGene SETD2 has 0 variants, skipping\n",
      "[INFO]\tGene PLBD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6869 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6743 has 0 variants, skipping\n",
      "[INFO]\tGene GPX2 has 0 variants, skipping\n",
      "[INFO]\tGene BVES-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIOS has 0 variants, skipping\n",
      "[INFO]\tGene C1orf146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377924 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02526 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02532 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC74A has 0 variants, skipping\n",
      "[INFO]\tGene CNOT6L has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPB has 0 variants, skipping\n",
      "[INFO]\tGene OSTM1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2A has 0 variants, skipping\n",
      "[INFO]\tGene C9orf92 has 0 variants, skipping\n",
      "[INFO]\tGene VSIG10L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00222 has 0 variants, skipping\n",
      "[INFO]\tGene RPL24 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINA3 has 0 variants, skipping\n",
      "[INFO]\tGene ARMC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HIGD2A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC162P has 0 variants, skipping\n",
      "[INFO]\tGene ELP3 has 0 variants, skipping\n",
      "[INFO]\tGene TJP3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEFF1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS20 has 0 variants, skipping\n",
      "[INFO]\tGene TNFSF11 has 0 variants, skipping\n",
      "[INFO]\tGene OR2D2 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA40C has 0 variants, skipping\n",
      "[INFO]\tGene GSTM2P1 has 0 variants, skipping\n",
      "[INFO]\tGene CNKSR1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD166 has 0 variants, skipping\n",
      "[INFO]\tGene STC1 has 0 variants, skipping\n",
      "[INFO]\tGene TXN2 has 0 variants, skipping\n",
      "[INFO]\tGene TRPC5 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAIP1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR135 has 0 variants, skipping\n",
      "[INFO]\tGene NDST4 has 0 variants, skipping\n",
      "[INFO]\tGene NYX has 0 variants, skipping\n",
      "[INFO]\tGene TLNRD1 has 0 variants, skipping\n",
      "[INFO]\tGene GH1 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02518 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02541 has 0 variants, skipping\n",
      "[INFO]\tGene HDAC2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02534 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1P3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1260A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287467 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT8 has 0 variants, skipping\n",
      "[INFO]\tGene IL1RAPL2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL3 has 0 variants, skipping\n",
      "[INFO]\tGene USP18 has 0 variants, skipping\n",
      "[INFO]\tGene APLN has 0 variants, skipping\n",
      "[INFO]\tGene FZD3 has 0 variants, skipping\n",
      "[INFO]\tGene ZUP1 has 0 variants, skipping\n",
      "[INFO]\tGene GREB1L has 0 variants, skipping\n",
      "[INFO]\tGene LPIN3 has 0 variants, skipping\n",
      "[INFO]\tGene IFI6 has 0 variants, skipping\n",
      "[INFO]\tGene ITK has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927919 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377967 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2QL1 has 0 variants, skipping\n",
      "[INFO]\tGene BRD7P3 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOKP3 has 0 variants, skipping\n",
      "[INFO]\tGene GAK has 0 variants, skipping\n",
      "[INFO]\tGene MIR548B has 0 variants, skipping\n",
      "[INFO]\tGene LOC285762 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377975 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3144 has 0 variants, skipping\n",
      "[INFO]\tGene THUMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T3 has 0 variants, skipping\n",
      "[INFO]\tGene CNOT6 has 0 variants, skipping\n",
      "[INFO]\tGene TRDN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM63A has 0 variants, skipping\n",
      "[INFO]\tGene RNF217-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MEFV has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126584 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02523 has 0 variants, skipping\n",
      "[INFO]\tGene NCOA7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCPEP1 has 0 variants, skipping\n",
      "[INFO]\tGene HRH4 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHA2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM12 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM13 has 0 variants, skipping\n",
      "[INFO]\tGene CHST6 has 0 variants, skipping\n",
      "[INFO]\tGene CARD18 has 0 variants, skipping\n",
      "[INFO]\tGene BARX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF708 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA17 has 0 variants, skipping\n",
      "[INFO]\tGene PGAP6 has 0 variants, skipping\n",
      "[INFO]\tGene IL22RA1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR588 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN1 has 0 variants, skipping\n",
      "[INFO]\tGene UBALD2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM169B has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS12 has 0 variants, skipping\n",
      "[INFO]\tGene PPIAL4D has 0 variants, skipping\n",
      "[INFO]\tGene EP300 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM35 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN19 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP11 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT18 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP2 has 0 variants, skipping\n",
      "[INFO]\tGene PKP2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH11Y has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-8 has 0 variants, skipping\n",
      "[INFO]\tGene DPRX has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AJ1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01013 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR3P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD101 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD100 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA33 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00326 has 0 variants, skipping\n",
      "[INFO]\tGene GIPC3 has 0 variants, skipping\n",
      "[INFO]\tGene TARID has 0 variants, skipping\n",
      "[INFO]\tGene LINC01312 has 0 variants, skipping\n",
      "[INFO]\tGene HMGA1P7 has 0 variants, skipping\n",
      "[INFO]\tGene CT69 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01010 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928304 has 0 variants, skipping\n",
      "[INFO]\tGene OR4L1 has 0 variants, skipping\n",
      "[INFO]\tGene SERTM2 has 0 variants, skipping\n",
      "[INFO]\tGene MS4A18 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC69 has 0 variants, skipping\n",
      "[INFO]\tGene NAALADL2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT3 has 0 variants, skipping\n",
      "[INFO]\tGene OR10P1 has 0 variants, skipping\n",
      "[INFO]\tGene PCED1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DSCR9 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA1-1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX43 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF827 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A2 has 0 variants, skipping\n",
      "[INFO]\tGene BSCL2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC644135 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM68 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928429 has 0 variants, skipping\n",
      "[INFO]\tGene NHEG1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723649 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02539 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02528 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3145 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46906 has 0 variants, skipping\n",
      "[INFO]\tGene CRTC3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01625 has 0 variants, skipping\n",
      "[INFO]\tGene FILNC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507477 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103352541 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3668 has 0 variants, skipping\n",
      "[INFO]\tGene SUMO1P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A2 has 0 variants, skipping\n",
      "[INFO]\tGene CT66 has 0 variants, skipping\n",
      "[INFO]\tGene CCT8L2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC153910 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01277 has 0 variants, skipping\n",
      "[INFO]\tGene RSU1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC89 has 0 variants, skipping\n",
      "[INFO]\tGene HEPACAM has 0 variants, skipping\n",
      "[INFO]\tGene DOK6 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF6 has 0 variants, skipping\n",
      "[INFO]\tGene TPSG1 has 0 variants, skipping\n",
      "[INFO]\tGene OTOP3 has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP2 has 0 variants, skipping\n",
      "[INFO]\tGene CRYBB2 has 0 variants, skipping\n",
      "[INFO]\tGene F12 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA98 has 0 variants, skipping\n",
      "[INFO]\tGene TPSAB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928661 has 0 variants, skipping\n",
      "[INFO]\tGene KATNBL1P6 has 0 variants, skipping\n",
      "[INFO]\tGene UST-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2-RAB5IF has 0 variants, skipping\n",
      "[INFO]\tGene TALDO1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS18P9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM131C has 0 variants, skipping\n",
      "[INFO]\tGene LOC645967 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370705 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378052 has 0 variants, skipping\n",
      "[INFO]\tGene RAET1K has 0 variants, skipping\n",
      "[INFO]\tGene MIR505 has 0 variants, skipping\n",
      "[INFO]\tGene MIR503 has 0 variants, skipping\n",
      "[INFO]\tGene MIR502 has 0 variants, skipping\n",
      "[INFO]\tGene MIR501 has 0 variants, skipping\n",
      "[INFO]\tGene MIR500A has 0 variants, skipping\n",
      "[INFO]\tGene MIR499A has 0 variants, skipping\n",
      "[INFO]\tGene MIR519A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520H has 0 variants, skipping\n",
      "[INFO]\tGene MIR518D has 0 variants, skipping\n",
      "[INFO]\tGene MIR520C has 0 variants, skipping\n",
      "[INFO]\tGene MIR518B has 0 variants, skipping\n",
      "[INFO]\tGene MIR520G has 0 variants, skipping\n",
      "[INFO]\tGene MIR520B has 0 variants, skipping\n",
      "[INFO]\tGene MIR525 has 0 variants, skipping\n",
      "[INFO]\tGene MIR519D has 0 variants, skipping\n",
      "[INFO]\tGene MIR519B has 0 variants, skipping\n",
      "[INFO]\tGene MIR524 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526B has 0 variants, skipping\n",
      "[INFO]\tGene MIR520A has 0 variants, skipping\n",
      "[INFO]\tGene MIR519C has 0 variants, skipping\n",
      "[INFO]\tGene MIR515-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR520E has 0 variants, skipping\n",
      "[INFO]\tGene MIR498 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181D has 0 variants, skipping\n",
      "[INFO]\tGene MIR497 has 0 variants, skipping\n",
      "[INFO]\tGene MIR193B has 0 variants, skipping\n",
      "[INFO]\tGene MIR496 has 0 variants, skipping\n",
      "[INFO]\tGene MIR495 has 0 variants, skipping\n",
      "[INFO]\tGene MIR494 has 0 variants, skipping\n",
      "[INFO]\tGene MIR432 has 0 variants, skipping\n",
      "[INFO]\tGene MIR493 has 0 variants, skipping\n",
      "[INFO]\tGene MIR202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR146B has 0 variants, skipping\n",
      "[INFO]\tGene MIR511 has 0 variants, skipping\n",
      "[INFO]\tGene MIR491 has 0 variants, skipping\n",
      "[INFO]\tGene MIR490 has 0 variants, skipping\n",
      "[INFO]\tGene MIR489 has 0 variants, skipping\n",
      "[INFO]\tGene MIR488 has 0 variants, skipping\n",
      "[INFO]\tGene MIR486-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR485 has 0 variants, skipping\n",
      "[INFO]\tGene MIR508 has 0 variants, skipping\n",
      "[INFO]\tGene MIR507 has 0 variants, skipping\n",
      "[INFO]\tGene MIR506 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR504 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723831 has 0 variants, skipping\n",
      "[INFO]\tGene PEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1 has 0 variants, skipping\n",
      "[INFO]\tGene DGKK has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH3 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN2 has 0 variants, skipping\n",
      "[INFO]\tGene DIAPH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TLDC2 has 0 variants, skipping\n",
      "[INFO]\tGene RNLS has 0 variants, skipping\n",
      "[INFO]\tGene MIR1273C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378068 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD28B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4466 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3692 has 0 variants, skipping\n",
      "[INFO]\tGene SYNJ2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7161 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3918 has 0 variants, skipping\n",
      "[INFO]\tGene OSTCP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM178A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929122 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02529 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT3 has 0 variants, skipping\n",
      "[INFO]\tGene USP26 has 0 variants, skipping\n",
      "[INFO]\tGene CACNG8 has 0 variants, skipping\n",
      "[INFO]\tGene SOD2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene OPRK1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA20 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA29 has 0 variants, skipping\n",
      "[INFO]\tGene AIRN has 0 variants, skipping\n",
      "[INFO]\tGene UTRN has 0 variants, skipping\n",
      "[INFO]\tGene LPAL2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM238L has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC8P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4301 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT4-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378098 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS3 has 0 variants, skipping\n",
      "[INFO]\tGene PACRG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZp451B082 has 0 variants, skipping\n",
      "[INFO]\tGene CAHM has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724152 has 0 variants, skipping\n",
      "[INFO]\tGene MEAT6 has 0 variants, skipping\n",
      "[INFO]\tGene FGF16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729681 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100289495 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO47 has 0 variants, skipping\n",
      "[INFO]\tGene RAB19 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1913 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3939 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1A1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378123 has 0 variants, skipping\n",
      "[INFO]\tGene TMCO1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378127 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02538 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02487 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01558 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441179 has 0 variants, skipping\n",
      "[INFO]\tGene PKD2 has 0 variants, skipping\n",
      "[INFO]\tGene POU3F4 has 0 variants, skipping\n",
      "[INFO]\tGene PON2 has 0 variants, skipping\n",
      "[INFO]\tGene KIF25-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378137 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929420 has 0 variants, skipping\n",
      "[INFO]\tGene SP9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929460 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02544 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929523 has 0 variants, skipping\n",
      "[INFO]\tGene FAM153B has 0 variants, skipping\n",
      "[INFO]\tGene OR4A5 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00242 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00574 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131532 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724511 has 0 variants, skipping\n",
      "[INFO]\tGene LOC154449 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285804 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01624 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF816 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4644 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G3 has 0 variants, skipping\n",
      "[INFO]\tGene MMP10 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf203 has 0 variants, skipping\n",
      "[INFO]\tGene NTN3 has 0 variants, skipping\n",
      "[INFO]\tGene NTF4 has 0 variants, skipping\n",
      "[INFO]\tGene NPBWR2 has 0 variants, skipping\n",
      "[INFO]\tGene NPBWR1 has 0 variants, skipping\n",
      "[INFO]\tGene XCR1 has 0 variants, skipping\n",
      "[INFO]\tGene NPRL2 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268269 has 0 variants, skipping\n",
      "[INFO]\tGene RNF111 has 0 variants, skipping\n",
      "[INFO]\tGene TRABD2B has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1E has 0 variants, skipping\n",
      "[INFO]\tGene SHOX2 has 0 variants, skipping\n",
      "[INFO]\tGene GMPR has 0 variants, skipping\n",
      "[INFO]\tGene B4GAT1 has 0 variants, skipping\n",
      "[INFO]\tGene PIM2 has 0 variants, skipping\n",
      "[INFO]\tGene RIPK3 has 0 variants, skipping\n",
      "[INFO]\tGene AOC2 has 0 variants, skipping\n",
      "[INFO]\tGene MNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene MSH4 has 0 variants, skipping\n",
      "[INFO]\tGene MYL10 has 0 variants, skipping\n",
      "[INFO]\tGene GSE1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL9 has 0 variants, skipping\n",
      "[INFO]\tGene MCUB has 0 variants, skipping\n",
      "[INFO]\tGene ACOT7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928436 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35F3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC49A4 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf65 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA12A has 0 variants, skipping\n",
      "[INFO]\tGene C17orf58 has 0 variants, skipping\n",
      "[INFO]\tGene NUCB1 has 0 variants, skipping\n",
      "[INFO]\tGene NTS has 0 variants, skipping\n",
      "[INFO]\tGene CYP4V2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A34 has 0 variants, skipping\n",
      "[INFO]\tGene TSEN54 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC9A has 0 variants, skipping\n",
      "[INFO]\tGene SPRYD4 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP1B has 0 variants, skipping\n",
      "[INFO]\tGene ZP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC24 has 0 variants, skipping\n",
      "[INFO]\tGene PAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM184A has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J3 has 0 variants, skipping\n",
      "[INFO]\tGene RAC2 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene UBA5 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC3L has 0 variants, skipping\n",
      "[INFO]\tGene OR13C4 has 0 variants, skipping\n",
      "[INFO]\tGene MED4 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD34B has 0 variants, skipping\n",
      "[INFO]\tGene SMYD4 has 0 variants, skipping\n",
      "[INFO]\tGene RAB31 has 0 variants, skipping\n",
      "[INFO]\tGene VWDE has 0 variants, skipping\n",
      "[INFO]\tGene FBN2 has 0 variants, skipping\n",
      "[INFO]\tGene DDX47 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA2 has 0 variants, skipping\n",
      "[INFO]\tGene NPSR1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17A has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIE has 0 variants, skipping\n",
      "[INFO]\tGene POU3F2 has 0 variants, skipping\n",
      "[INFO]\tGene MYF5 has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B1 has 0 variants, skipping\n",
      "[INFO]\tGene POU2F3 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHD has 0 variants, skipping\n",
      "[INFO]\tGene HMX3 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO45 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS7C has 0 variants, skipping\n",
      "[INFO]\tGene MSGN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKL2 has 0 variants, skipping\n",
      "[INFO]\tGene UTP4 has 0 variants, skipping\n",
      "[INFO]\tGene SOS1 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L has 0 variants, skipping\n",
      "[INFO]\tGene NOX5 has 0 variants, skipping\n",
      "[INFO]\tGene PALMD has 0 variants, skipping\n",
      "[INFO]\tGene COL6A3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL4 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD10 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15 has 0 variants, skipping\n",
      "[INFO]\tGene TAAR5 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA5A has 0 variants, skipping\n",
      "[INFO]\tGene CETN1 has 0 variants, skipping\n",
      "[INFO]\tGene CDR1 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN1B has 0 variants, skipping\n",
      "[INFO]\tGene TM4SF5 has 0 variants, skipping\n",
      "[INFO]\tGene CPA6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49 has 0 variants, skipping\n",
      "[INFO]\tGene RNF115 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP2A has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF21 has 0 variants, skipping\n",
      "[INFO]\tGene SIT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF16 has 0 variants, skipping\n",
      "[INFO]\tGene SERP1 has 0 variants, skipping\n",
      "[INFO]\tGene IL17B has 0 variants, skipping\n",
      "[INFO]\tGene IL36A has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A1 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0232 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC3L1 has 0 variants, skipping\n",
      "[INFO]\tGene KEL has 0 variants, skipping\n",
      "[INFO]\tGene IAPP has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM125 has 0 variants, skipping\n",
      "[INFO]\tGene IGLL1 has 0 variants, skipping\n",
      "[INFO]\tGene CBARP has 0 variants, skipping\n",
      "[INFO]\tGene CAMK1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN20 has 0 variants, skipping\n",
      "[INFO]\tGene FOXA1 has 0 variants, skipping\n",
      "[INFO]\tGene MVD has 0 variants, skipping\n",
      "[INFO]\tGene FAM43B has 0 variants, skipping\n",
      "[INFO]\tGene NIPAL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDPF1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L3 has 0 variants, skipping\n",
      "[INFO]\tGene C2CD4A has 0 variants, skipping\n",
      "[INFO]\tGene ACSM6 has 0 variants, skipping\n",
      "[INFO]\tGene EFNB3 has 0 variants, skipping\n",
      "[INFO]\tGene EFNA2 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1G has 0 variants, skipping\n",
      "[INFO]\tGene PEX3 has 0 variants, skipping\n",
      "[INFO]\tGene PARP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP22-2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP40 has 0 variants, skipping\n",
      "[INFO]\tGene FAM228A has 0 variants, skipping\n",
      "[INFO]\tGene CDHR1 has 0 variants, skipping\n",
      "[INFO]\tGene ISLR has 0 variants, skipping\n",
      "[INFO]\tGene BLK has 0 variants, skipping\n",
      "[INFO]\tGene TPM2 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC1 has 0 variants, skipping\n",
      "[INFO]\tGene ATCAY has 0 variants, skipping\n",
      "[INFO]\tGene MAP6 has 0 variants, skipping\n",
      "[INFO]\tGene MYL7 has 0 variants, skipping\n",
      "[INFO]\tGene AASDHPPT has 0 variants, skipping\n",
      "[INFO]\tGene CDADC1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAN has 0 variants, skipping\n",
      "[INFO]\tGene AFM has 0 variants, skipping\n",
      "[INFO]\tGene CRISP1 has 0 variants, skipping\n",
      "[INFO]\tGene AEBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TLE5 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1E has 0 variants, skipping\n",
      "[INFO]\tGene GALNT2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY14 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC10 has 0 variants, skipping\n",
      "[INFO]\tGene DCD has 0 variants, skipping\n",
      "[INFO]\tGene MEOX2 has 0 variants, skipping\n",
      "[INFO]\tGene IGF1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC175 has 0 variants, skipping\n",
      "[INFO]\tGene PARP2 has 0 variants, skipping\n",
      "[INFO]\tGene OTUD6A has 0 variants, skipping\n",
      "[INFO]\tGene PPP4R3C has 0 variants, skipping\n",
      "[INFO]\tGene FAM199X has 0 variants, skipping\n",
      "[INFO]\tGene ZNF474 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM207 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS33 has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A9 has 0 variants, skipping\n",
      "[INFO]\tGene RPL3 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF1B has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D3B has 0 variants, skipping\n",
      "[INFO]\tGene GAU1 has 0 variants, skipping\n",
      "[INFO]\tGene NHS has 0 variants, skipping\n",
      "[INFO]\tGene SNHG10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR155HG has 0 variants, skipping\n",
      "[INFO]\tGene RN7SK has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPAB has 0 variants, skipping\n",
      "[INFO]\tGene FAM83B has 0 variants, skipping\n",
      "[INFO]\tGene NANS has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP5-3 has 0 variants, skipping\n",
      "[INFO]\tGene BHMT has 0 variants, skipping\n",
      "[INFO]\tGene BGN has 0 variants, skipping\n",
      "[INFO]\tGene ICAM5 has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO15 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B13 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10-AS has 0 variants, skipping\n",
      "[INFO]\tGene SRMS has 0 variants, skipping\n",
      "[INFO]\tGene SLIT3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL16 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM86A has 0 variants, skipping\n",
      "[INFO]\tGene BEND2 has 0 variants, skipping\n",
      "[INFO]\tGene ENPP6 has 0 variants, skipping\n",
      "[INFO]\tGene ERMAP has 0 variants, skipping\n",
      "[INFO]\tGene P2RX3 has 0 variants, skipping\n",
      "[INFO]\tGene DAO has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3E has 0 variants, skipping\n",
      "[INFO]\tGene UAP1L1 has 0 variants, skipping\n",
      "[INFO]\tGene PLPPR1 has 0 variants, skipping\n",
      "[INFO]\tGene NUP210L has 0 variants, skipping\n",
      "[INFO]\tGene CXCL2 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL3 has 0 variants, skipping\n",
      "[INFO]\tGene RECQL4 has 0 variants, skipping\n",
      "[INFO]\tGene PDAP1 has 0 variants, skipping\n",
      "[INFO]\tGene DARS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IQCC has 0 variants, skipping\n",
      "[INFO]\tGene RFXANK has 0 variants, skipping\n",
      "[INFO]\tGene UQCRHL has 0 variants, skipping\n",
      "[INFO]\tGene BCYRN1 has 0 variants, skipping\n",
      "[INFO]\tGene TERC has 0 variants, skipping\n",
      "[INFO]\tGene XIST has 0 variants, skipping\n",
      "[INFO]\tGene LINC00261 has 0 variants, skipping\n",
      "[INFO]\tGene ASGR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR516A1 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA4 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM2 has 0 variants, skipping\n",
      "[INFO]\tGene FHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF44 has 0 variants, skipping\n",
      "[INFO]\tGene LMOD3 has 0 variants, skipping\n",
      "[INFO]\tGene IFIH1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A7 has 0 variants, skipping\n",
      "[INFO]\tGene AAAS has 0 variants, skipping\n",
      "[INFO]\tGene HOXD1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM3 has 0 variants, skipping\n",
      "[INFO]\tGene PSKH1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R1A has 0 variants, skipping\n",
      "[INFO]\tGene MCM5 has 0 variants, skipping\n",
      "[INFO]\tGene FZD7 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC71 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR2C has 0 variants, skipping\n",
      "[INFO]\tGene MYH15 has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4D has 0 variants, skipping\n",
      "[INFO]\tGene FAM214A has 0 variants, skipping\n",
      "[INFO]\tGene MRPL34 has 0 variants, skipping\n",
      "[INFO]\tGene PAPLN has 0 variants, skipping\n",
      "[INFO]\tGene MIR891B has 0 variants, skipping\n",
      "[INFO]\tGene MIR920 has 0 variants, skipping\n",
      "[INFO]\tGene NR4A2 has 0 variants, skipping\n",
      "[INFO]\tGene ARFGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene NUDT4B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF724 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14A2 has 0 variants, skipping\n",
      "[INFO]\tGene NCR3LG1 has 0 variants, skipping\n",
      "[INFO]\tGene ERN1 has 0 variants, skipping\n",
      "[INFO]\tGene EN1 has 0 variants, skipping\n",
      "[INFO]\tGene EMP3 has 0 variants, skipping\n",
      "[INFO]\tGene EMP2 has 0 variants, skipping\n",
      "[INFO]\tGene CHROMR has 0 variants, skipping\n",
      "[INFO]\tGene SAA2-SAA4 has 0 variants, skipping\n",
      "[INFO]\tGene DEPTOR has 0 variants, skipping\n",
      "[INFO]\tGene ZNF582 has 0 variants, skipping\n",
      "[INFO]\tGene SBNO1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR83OS has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B12 has 0 variants, skipping\n",
      "[INFO]\tGene CPQ has 0 variants, skipping\n",
      "[INFO]\tGene TBX18 has 0 variants, skipping\n",
      "[INFO]\tGene PLA1A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3142HG has 0 variants, skipping\n",
      "[INFO]\tGene RPS27 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX1 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC2 has 0 variants, skipping\n",
      "[INFO]\tGene HAUS3 has 0 variants, skipping\n",
      "[INFO]\tGene LRFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ZBED2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BPL has 0 variants, skipping\n",
      "[INFO]\tGene SUN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM200C has 0 variants, skipping\n",
      "[INFO]\tGene NUPR2 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf144 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL30 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF38 has 0 variants, skipping\n",
      "[INFO]\tGene SAT2 has 0 variants, skipping\n",
      "[INFO]\tGene SYT13 has 0 variants, skipping\n",
      "[INFO]\tGene LAMA2 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM30 has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE has 0 variants, skipping\n",
      "[INFO]\tGene COL17A1 has 0 variants, skipping\n",
      "[INFO]\tGene HSFX2 has 0 variants, skipping\n",
      "[INFO]\tGene USP38 has 0 variants, skipping\n",
      "[INFO]\tGene OR1F12 has 0 variants, skipping\n",
      "[INFO]\tGene SKIDA1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR153 has 0 variants, skipping\n",
      "[INFO]\tGene PLXDC2 has 0 variants, skipping\n",
      "[INFO]\tGene OXSR1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS4 has 0 variants, skipping\n",
      "[INFO]\tGene MYO1E has 0 variants, skipping\n",
      "[INFO]\tGene OR1J4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1B1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF10 has 0 variants, skipping\n",
      "[INFO]\tGene LY86 has 0 variants, skipping\n",
      "[INFO]\tGene MED7 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR1A has 0 variants, skipping\n",
      "[INFO]\tGene ACTR1B has 0 variants, skipping\n",
      "[INFO]\tGene ERV3-1 has 0 variants, skipping\n",
      "[INFO]\tGene ACADL has 0 variants, skipping\n",
      "[INFO]\tGene POLR3D has 0 variants, skipping\n",
      "[INFO]\tGene BMP7 has 0 variants, skipping\n",
      "[INFO]\tGene RBM26 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT4 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL42 has 0 variants, skipping\n",
      "[INFO]\tGene DISP3 has 0 variants, skipping\n",
      "[INFO]\tGene PSD4 has 0 variants, skipping\n",
      "[INFO]\tGene FANCE has 0 variants, skipping\n",
      "[INFO]\tGene SCT has 0 variants, skipping\n",
      "[INFO]\tGene ZNF70 has 0 variants, skipping\n",
      "[INFO]\tGene CFL2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRB2 has 0 variants, skipping\n",
      "[INFO]\tGene PFKL has 0 variants, skipping\n",
      "[INFO]\tGene CFAP73 has 0 variants, skipping\n",
      "[INFO]\tGene NAA30 has 0 variants, skipping\n",
      "[INFO]\tGene PNLIPRP3 has 0 variants, skipping\n",
      "[INFO]\tGene INSL4 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA3 has 0 variants, skipping\n",
      "[INFO]\tGene INPP1 has 0 variants, skipping\n",
      "[INFO]\tGene INHBB has 0 variants, skipping\n",
      "[INFO]\tGene INHBA has 0 variants, skipping\n",
      "[INFO]\tGene ISL1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A4 has 0 variants, skipping\n",
      "[INFO]\tGene HNF4G has 0 variants, skipping\n",
      "[INFO]\tGene WFDC6 has 0 variants, skipping\n",
      "[INFO]\tGene ISM1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A3 has 0 variants, skipping\n",
      "[INFO]\tGene RINL has 0 variants, skipping\n",
      "[INFO]\tGene C3orf84 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCOS has 0 variants, skipping\n",
      "[INFO]\tGene ANHX has 0 variants, skipping\n",
      "[INFO]\tGene DUSP21 has 0 variants, skipping\n",
      "[INFO]\tGene CERS2 has 0 variants, skipping\n",
      "[INFO]\tGene RRP1B has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV3 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFV2 has 0 variants, skipping\n",
      "[INFO]\tGene HCN1 has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN4 has 0 variants, skipping\n",
      "[INFO]\tGene RIOK3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507642 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375115 has 0 variants, skipping\n",
      "[INFO]\tGene LOC442497 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT92 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM163 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83D has 0 variants, skipping\n",
      "[INFO]\tGene NECTIN4 has 0 variants, skipping\n",
      "[INFO]\tGene LBH has 0 variants, skipping\n",
      "[INFO]\tGene OR2A4 has 0 variants, skipping\n",
      "[INFO]\tGene CPLANE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF696 has 0 variants, skipping\n",
      "[INFO]\tGene OR2J2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T1 has 0 variants, skipping\n",
      "[INFO]\tGene STPG1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK13 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf82 has 0 variants, skipping\n",
      "[INFO]\tGene NNT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK6-DT has 0 variants, skipping\n",
      "[INFO]\tGene ESD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128653 has 0 variants, skipping\n",
      "[INFO]\tGene TFAMP1 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4655 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA114 has 0 variants, skipping\n",
      "[INFO]\tGene SOX18 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4648 has 0 variants, skipping\n",
      "[INFO]\tGene COL26A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR593 has 0 variants, skipping\n",
      "[INFO]\tGene MIR590 has 0 variants, skipping\n",
      "[INFO]\tGene MIR584 has 0 variants, skipping\n",
      "[INFO]\tGene MIR582 has 0 variants, skipping\n",
      "[INFO]\tGene MIR571 has 0 variants, skipping\n",
      "[INFO]\tGene MIR579 has 0 variants, skipping\n",
      "[INFO]\tGene MIR551B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129603 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A7 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD165 has 0 variants, skipping\n",
      "[INFO]\tGene CAT has 0 variants, skipping\n",
      "[INFO]\tGene RNF216P1 has 0 variants, skipping\n",
      "[INFO]\tGene RBAKDN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF890P has 0 variants, skipping\n",
      "[INFO]\tGene USP42 has 0 variants, skipping\n",
      "[INFO]\tGene MICOS13 has 0 variants, skipping\n",
      "[INFO]\tGene ANKFN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC221946 has 0 variants, skipping\n",
      "[INFO]\tGene RNF216-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6874 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF815P has 0 variants, skipping\n",
      "[INFO]\tGene SOGA3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A5 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2D has 0 variants, skipping\n",
      "[INFO]\tGene LOH12CR2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB2B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA80D has 0 variants, skipping\n",
      "[INFO]\tGene SV2C has 0 variants, skipping\n",
      "[INFO]\tGene PMS2CL has 0 variants, skipping\n",
      "[INFO]\tGene MIR3683 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131257 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927354 has 0 variants, skipping\n",
      "[INFO]\tGene SPOPL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM211 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927391 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L2 has 0 variants, skipping\n",
      "[INFO]\tGene HESX1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF705D has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505938 has 0 variants, skipping\n",
      "[INFO]\tGene MGC4859 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100131472 has 0 variants, skipping\n",
      "[INFO]\tGene CHPT1 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725191 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF4 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK6 has 0 variants, skipping\n",
      "[INFO]\tGene INMT has 0 variants, skipping\n",
      "[INFO]\tGene DDX18 has 0 variants, skipping\n",
      "[INFO]\tGene KRT38 has 0 variants, skipping\n",
      "[INFO]\tGene LMO4 has 0 variants, skipping\n",
      "[INFO]\tGene BRAP has 0 variants, skipping\n",
      "[INFO]\tGene LZTR1 has 0 variants, skipping\n",
      "[INFO]\tGene SIX2 has 0 variants, skipping\n",
      "[INFO]\tGene SHC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375166 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02587 has 0 variants, skipping\n",
      "[INFO]\tGene CRPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RINT1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF11 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB16 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRFS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCIN has 0 variants, skipping\n",
      "[INFO]\tGene UCHL3 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM4 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf51A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-6 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A7 has 0 variants, skipping\n",
      "[INFO]\tGene CPHXL has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL1B has 0 variants, skipping\n",
      "[INFO]\tGene TEX13D has 0 variants, skipping\n",
      "[INFO]\tGene MIR3146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927668 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J5 has 0 variants, skipping\n",
      "[INFO]\tGene OR10J3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506098 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927769 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927811 has 0 variants, skipping\n",
      "[INFO]\tGene ERMN has 0 variants, skipping\n",
      "[INFO]\tGene RPL23P8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01162 has 0 variants, skipping\n",
      "[INFO]\tGene MEIG1 has 0 variants, skipping\n",
      "[INFO]\tGene MAF1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF7 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC8C has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF1 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf16 has 0 variants, skipping\n",
      "[INFO]\tGene WDR24 has 0 variants, skipping\n",
      "[INFO]\tGene CCL13 has 0 variants, skipping\n",
      "[INFO]\tGene LCP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401312 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG26 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD93 has 0 variants, skipping\n",
      "[INFO]\tGene POC1B-GALNT4 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL7-DT has 0 variants, skipping\n",
      "[INFO]\tGene GAL has 0 variants, skipping\n",
      "[INFO]\tGene GOLT1B has 0 variants, skipping\n",
      "[INFO]\tGene MRPS7 has 0 variants, skipping\n",
      "[INFO]\tGene PAM16 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS23 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS17 has 0 variants, skipping\n",
      "[INFO]\tGene FIS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65C has 0 variants, skipping\n",
      "[INFO]\tGene RPS2P32 has 0 variants, skipping\n",
      "[INFO]\tGene CHST9 has 0 variants, skipping\n",
      "[INFO]\tGene CLK2P1 has 0 variants, skipping\n",
      "[INFO]\tGene NAB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929322 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375304 has 0 variants, skipping\n",
      "[INFO]\tGene NPEPPSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02805 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4295 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0087 has 0 variants, skipping\n",
      "[INFO]\tGene RHOC has 0 variants, skipping\n",
      "[INFO]\tGene BACH2 has 0 variants, skipping\n",
      "[INFO]\tGene OSTN has 0 variants, skipping\n",
      "[INFO]\tGene HOXA10-HOXA9 has 0 variants, skipping\n",
      "[INFO]\tGene PINX1 has 0 variants, skipping\n",
      "[INFO]\tGene EVX1-AS has 0 variants, skipping\n",
      "[INFO]\tGene MAU2 has 0 variants, skipping\n",
      "[INFO]\tGene TNS2 has 0 variants, skipping\n",
      "[INFO]\tGene TSL has 0 variants, skipping\n",
      "[INFO]\tGene EDF1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01297-DUXAP10-NBEAP6 has 0 variants, skipping\n",
      "[INFO]\tGene JAZF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506497 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928168 has 0 variants, skipping\n",
      "[INFO]\tGene HORMAD1 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD4-RWDD3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC646762 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NEAT1 has 0 variants, skipping\n",
      "[INFO]\tGene RSBN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375218 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550B1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP586I1420 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01176 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf48 has 0 variants, skipping\n",
      "[INFO]\tGene INMT-MINDY4 has 0 variants, skipping\n",
      "[INFO]\tGene PYGB has 0 variants, skipping\n",
      "[INFO]\tGene GABRA6 has 0 variants, skipping\n",
      "[INFO]\tGene EPO has 0 variants, skipping\n",
      "[INFO]\tGene GABRA5 has 0 variants, skipping\n",
      "[INFO]\tGene DRD5 has 0 variants, skipping\n",
      "[INFO]\tGene GABRA3 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L1 has 0 variants, skipping\n",
      "[INFO]\tGene GSK3B has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130673 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2D has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR550B2 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L1P2 has 0 variants, skipping\n",
      "[INFO]\tGene RP9P has 0 variants, skipping\n",
      "[INFO]\tGene OR51A4 has 0 variants, skipping\n",
      "[INFO]\tGene GDF6 has 0 variants, skipping\n",
      "[INFO]\tGene KLK5 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB6 has 0 variants, skipping\n",
      "[INFO]\tGene TYW1B has 0 variants, skipping\n",
      "[INFO]\tGene NPSR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401324 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930085 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506725 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928618 has 0 variants, skipping\n",
      "[INFO]\tGene TPP1 has 0 variants, skipping\n",
      "[INFO]\tGene AOAH-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene HIF1AN has 0 variants, skipping\n",
      "[INFO]\tGene ELMO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF19 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB126 has 0 variants, skipping\n",
      "[INFO]\tGene UNC93B1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM183BP has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA20B has 0 variants, skipping\n",
      "[INFO]\tGene POU6F2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01197 has 0 variants, skipping\n",
      "[INFO]\tGene HAGLR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01450 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01449 has 0 variants, skipping\n",
      "[INFO]\tGene INHBA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene IL31 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR2G has 0 variants, skipping\n",
      "[INFO]\tGene LINC01448 has 0 variants, skipping\n",
      "[INFO]\tGene HECW1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3943 has 0 variants, skipping\n",
      "[INFO]\tGene LUARIS has 0 variants, skipping\n",
      "[INFO]\tGene MIR492 has 0 variants, skipping\n",
      "[INFO]\tGene STARD5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF648 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPA has 0 variants, skipping\n",
      "[INFO]\tGene CLDN24 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP4 has 0 variants, skipping\n",
      "[INFO]\tGene GDI1 has 0 variants, skipping\n",
      "[INFO]\tGene CCS has 0 variants, skipping\n",
      "[INFO]\tGene RASA4CP has 0 variants, skipping\n",
      "[INFO]\tGene LINC00957 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6837 has 0 variants, skipping\n",
      "[INFO]\tGene ZNHIT6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6838 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4649 has 0 variants, skipping\n",
      "[INFO]\tGene ARL1 has 0 variants, skipping\n",
      "[INFO]\tGene COQ2 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB8B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD40CL has 0 variants, skipping\n",
      "[INFO]\tGene C4orf33 has 0 variants, skipping\n",
      "[INFO]\tGene RNF112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4657 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM201A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA5B has 0 variants, skipping\n",
      "[INFO]\tGene LOC730234 has 0 variants, skipping\n",
      "[INFO]\tGene MED17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730338 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD151 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01447 has 0 variants, skipping\n",
      "[INFO]\tGene DSC3 has 0 variants, skipping\n",
      "[INFO]\tGene DSC2 has 0 variants, skipping\n",
      "[INFO]\tGene PIGBOS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene COL14A1 has 0 variants, skipping\n",
      "[INFO]\tGene L2HGDH has 0 variants, skipping\n",
      "[INFO]\tGene SLC35E1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100509620 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01446 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01445 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2A-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf34 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT4 has 0 variants, skipping\n",
      "[INFO]\tGene HARBI1 has 0 variants, skipping\n",
      "[INFO]\tGene TIGIT has 0 variants, skipping\n",
      "[INFO]\tGene ZCCHC12 has 0 variants, skipping\n",
      "[INFO]\tGene NEGR1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP9P1 has 0 variants, skipping\n",
      "[INFO]\tGene SLCO4A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT11 has 0 variants, skipping\n",
      "[INFO]\tGene XKRY2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC650226 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100240728 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZp434L192 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928401 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401357 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130849 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4283-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4283-2 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375297 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100653233 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3147 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF733P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724738 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287704 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287834 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF722P has 0 variants, skipping\n",
      "[INFO]\tGene LINC01005 has 0 variants, skipping\n",
      "[INFO]\tGene THEM6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF771 has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN5 has 0 variants, skipping\n",
      "[INFO]\tGene YWHAEP1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128885 has 0 variants, skipping\n",
      "[INFO]\tGene LOC641746 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF273 has 0 variants, skipping\n",
      "[INFO]\tGene CYB5D1 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6P3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22C has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15B-2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA15B-1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS4P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377752 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441242 has 0 variants, skipping\n",
      "[INFO]\tGene INTS4P2 has 0 variants, skipping\n",
      "[INFO]\tGene CCT6P1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA22 has 0 variants, skipping\n",
      "[INFO]\tGene VWA2 has 0 variants, skipping\n",
      "[INFO]\tGene ARSD has 0 variants, skipping\n",
      "[INFO]\tGene ACTL9 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-124K5.4 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-124K5.11 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996437 has 0 variants, skipping\n",
      "[INFO]\tGene OR6F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51I1 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf91 has 0 variants, skipping\n",
      "[INFO]\tGene ARMS2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IRD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02604 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4650-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4650-2 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P4 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01372 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723427 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3914-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3914-2 has 0 variants, skipping\n",
      "[INFO]\tGene OVOL3 has 0 variants, skipping\n",
      "[INFO]\tGene SBDSP1 has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5P2 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100101148 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L3 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P7 has 0 variants, skipping\n",
      "[INFO]\tGene WDR76 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP4 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP1 has 0 variants, skipping\n",
      "[INFO]\tGene NCF1B has 0 variants, skipping\n",
      "[INFO]\tGene EIF4H has 0 variants, skipping\n",
      "[INFO]\tGene GLDC has 0 variants, skipping\n",
      "[INFO]\tGene GLI3 has 0 variants, skipping\n",
      "[INFO]\tGene GJA1 has 0 variants, skipping\n",
      "[INFO]\tGene GIPR has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L7 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L8 has 0 variants, skipping\n",
      "[INFO]\tGene RELN has 0 variants, skipping\n",
      "[INFO]\tGene AQP3 has 0 variants, skipping\n",
      "[INFO]\tGene H1-10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926943 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L2 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P5 has 0 variants, skipping\n",
      "[INFO]\tGene ARID1A has 0 variants, skipping\n",
      "[INFO]\tGene PPIL4 has 0 variants, skipping\n",
      "[INFO]\tGene NCF1C has 0 variants, skipping\n",
      "[INFO]\tGene C15orf54 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14C has 0 variants, skipping\n",
      "[INFO]\tGene NSUN5P1 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P3 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK5 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA14A has 0 variants, skipping\n",
      "[INFO]\tGene PVALB has 0 variants, skipping\n",
      "[INFO]\tGene GJA8 has 0 variants, skipping\n",
      "[INFO]\tGene GJA5 has 0 variants, skipping\n",
      "[INFO]\tGene GFER has 0 variants, skipping\n",
      "[INFO]\tGene GEM has 0 variants, skipping\n",
      "[INFO]\tGene GTF2IP7 has 0 variants, skipping\n",
      "[INFO]\tGene HBA2 has 0 variants, skipping\n",
      "[INFO]\tGene CASQ2 has 0 variants, skipping\n",
      "[INFO]\tGene CASQ1 has 0 variants, skipping\n",
      "[INFO]\tGene FDPSP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKIB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133091 has 0 variants, skipping\n",
      "[INFO]\tGene DTX2P1-UPK3BP1-PMS2P11 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P9 has 0 variants, skipping\n",
      "[INFO]\tGene FAM185BP has 0 variants, skipping\n",
      "[INFO]\tGene EDEM1 has 0 variants, skipping\n",
      "[INFO]\tGene UBE3C has 0 variants, skipping\n",
      "[INFO]\tGene MYLK2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927243 has 0 variants, skipping\n",
      "[INFO]\tGene ING5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF852 has 0 variants, skipping\n",
      "[INFO]\tGene RPL13AP17 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI2-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF113B has 0 variants, skipping\n",
      "[INFO]\tGene SLC51B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01555 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927269 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369146 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128317 has 0 variants, skipping\n",
      "[INFO]\tGene INPP5E has 0 variants, skipping\n",
      "[INFO]\tGene ZFP36L2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00972 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927420 has 0 variants, skipping\n",
      "[INFO]\tGene PTMS has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD1 has 0 variants, skipping\n",
      "[INFO]\tGene MFN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC38A5 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP18 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS3 has 0 variants, skipping\n",
      "[INFO]\tGene TPGS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL5 has 0 variants, skipping\n",
      "[INFO]\tGene DISP2 has 0 variants, skipping\n",
      "[INFO]\tGene MSMP has 0 variants, skipping\n",
      "[INFO]\tGene DSG2 has 0 variants, skipping\n",
      "[INFO]\tGene GABRR2 has 0 variants, skipping\n",
      "[INFO]\tGene DSG1 has 0 variants, skipping\n",
      "[INFO]\tGene GAB1 has 0 variants, skipping\n",
      "[INFO]\tGene DR1 has 0 variants, skipping\n",
      "[INFO]\tGene DPT has 0 variants, skipping\n",
      "[INFO]\tGene ACKR1 has 0 variants, skipping\n",
      "[INFO]\tGene CRIPT has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4A has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP8 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL18 has 0 variants, skipping\n",
      "[INFO]\tGene UCP3 has 0 variants, skipping\n",
      "[INFO]\tGene STEAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927446 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101409256 has 0 variants, skipping\n",
      "[INFO]\tGene LYSMD3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF548 has 0 variants, skipping\n",
      "[INFO]\tGene FERD3L has 0 variants, skipping\n",
      "[INFO]\tGene CYP51A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SAR1A has 0 variants, skipping\n",
      "[INFO]\tGene DMRTC2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR86 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD16 has 0 variants, skipping\n",
      "[INFO]\tGene TMBIM7P has 0 variants, skipping\n",
      "[INFO]\tGene GGNBP2 has 0 variants, skipping\n",
      "[INFO]\tGene LTC4S has 0 variants, skipping\n",
      "[INFO]\tGene IFNL4 has 0 variants, skipping\n",
      "[INFO]\tGene TXNL1 has 0 variants, skipping\n",
      "[INFO]\tGene NDST3 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA4 has 0 variants, skipping\n",
      "[INFO]\tGene PFDN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02145 has 0 variants, skipping\n",
      "[INFO]\tGene COL1A2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF578 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28B has 0 variants, skipping\n",
      "[INFO]\tGene CD2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF186 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A4RG has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA2B1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPA1 has 0 variants, skipping\n",
      "[INFO]\tGene HMOX1 has 0 variants, skipping\n",
      "[INFO]\tGene SRRM4 has 0 variants, skipping\n",
      "[INFO]\tGene CDT1 has 0 variants, skipping\n",
      "[INFO]\tGene CZ1P-ASNS has 0 variants, skipping\n",
      "[INFO]\tGene MIR5692C2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLB has 0 variants, skipping\n",
      "[INFO]\tGene COX10 has 0 variants, skipping\n",
      "[INFO]\tGene CORT has 0 variants, skipping\n",
      "[INFO]\tGene CLIC2 has 0 variants, skipping\n",
      "[INFO]\tGene SH3BP5 has 0 variants, skipping\n",
      "[INFO]\tGene BIN3 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA28 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927550 has 0 variants, skipping\n",
      "[INFO]\tGene MYH16 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL18 has 0 variants, skipping\n",
      "[INFO]\tGene CEP135 has 0 variants, skipping\n",
      "[INFO]\tGene OR1E1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1D4 has 0 variants, skipping\n",
      "[INFO]\tGene NME5 has 0 variants, skipping\n",
      "[INFO]\tGene BCL10 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F8 has 0 variants, skipping\n",
      "[INFO]\tGene AZGP1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375423 has 0 variants, skipping\n",
      "[INFO]\tGene CPPED1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene IMPDH2 has 0 variants, skipping\n",
      "[INFO]\tGene ADRA2C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K14 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA1 has 0 variants, skipping\n",
      "[INFO]\tGene GPAA1 has 0 variants, skipping\n",
      "[INFO]\tGene PDC has 0 variants, skipping\n",
      "[INFO]\tGene GPR52 has 0 variants, skipping\n",
      "[INFO]\tGene GPR55 has 0 variants, skipping\n",
      "[INFO]\tGene TLR1 has 0 variants, skipping\n",
      "[INFO]\tGene TIPARP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM87A has 0 variants, skipping\n",
      "[INFO]\tGene GABRB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6845 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6809 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01672 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL4 has 0 variants, skipping\n",
      "[INFO]\tGene RBM43 has 0 variants, skipping\n",
      "[INFO]\tGene FAM89A has 0 variants, skipping\n",
      "[INFO]\tGene DRAXIN has 0 variants, skipping\n",
      "[INFO]\tGene IGFL1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT8 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF699 has 0 variants, skipping\n",
      "[INFO]\tGene TEX45 has 0 variants, skipping\n",
      "[INFO]\tGene PER3P1 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PMS2P1 has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L5P-PVRIG2P-PILRB has 0 variants, skipping\n",
      "[INFO]\tGene STAG3L5P has 0 variants, skipping\n",
      "[INFO]\tGene PVRIG2P has 0 variants, skipping\n",
      "[INFO]\tGene MIR6840 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1A2 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB46 has 0 variants, skipping\n",
      "[INFO]\tGene PCOLCE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375429 has 0 variants, skipping\n",
      "[INFO]\tGene MOSMO has 0 variants, skipping\n",
      "[INFO]\tGene TRABD2A has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF12 has 0 variants, skipping\n",
      "[INFO]\tGene USP17L15 has 0 variants, skipping\n",
      "[INFO]\tGene SEC22B2P has 0 variants, skipping\n",
      "[INFO]\tGene SNORD50B has 0 variants, skipping\n",
      "[INFO]\tGene GLYCAM1 has 0 variants, skipping\n",
      "[INFO]\tGene ACTG1P25 has 0 variants, skipping\n",
      "[INFO]\tGene PCGEM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4653 has 0 variants, skipping\n",
      "[INFO]\tGene NUP50 has 0 variants, skipping\n",
      "[INFO]\tGene LNCPRESS1 has 0 variants, skipping\n",
      "[INFO]\tGene FREM3 has 0 variants, skipping\n",
      "[INFO]\tGene POTEM has 0 variants, skipping\n",
      "[INFO]\tGene LINC01007 has 0 variants, skipping\n",
      "[INFO]\tGene PGRMC2 has 0 variants, skipping\n",
      "[INFO]\tGene CDIPT has 0 variants, skipping\n",
      "[INFO]\tGene MIR4285 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100630923 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548O has 0 variants, skipping\n",
      "[INFO]\tGene GPSM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5090 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY11 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY4 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY1 has 0 variants, skipping\n",
      "[INFO]\tGene RASA4DP has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506422 has 0 variants, skipping\n",
      "[INFO]\tGene IFIT5 has 0 variants, skipping\n",
      "[INFO]\tGene RPL19P12 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L2P2 has 0 variants, skipping\n",
      "[INFO]\tGene SMOC1 has 0 variants, skipping\n",
      "[INFO]\tGene HYLS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927870 has 0 variants, skipping\n",
      "[INFO]\tGene IPO11 has 0 variants, skipping\n",
      "[INFO]\tGene RASL10A has 0 variants, skipping\n",
      "[INFO]\tGene TAF6L has 0 variants, skipping\n",
      "[INFO]\tGene MYL12A has 0 variants, skipping\n",
      "[INFO]\tGene IVNS1ABP has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01004 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2E-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf52 has 0 variants, skipping\n",
      "[INFO]\tGene CD7 has 0 variants, skipping\n",
      "[INFO]\tGene PRKG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378E has 0 variants, skipping\n",
      "[INFO]\tGene MIR378D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR378D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR151B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00941 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927974 has 0 variants, skipping\n",
      "[INFO]\tGene RASIP1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB6 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP13 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996249 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928012 has 0 variants, skipping\n",
      "[INFO]\tGene RGS21 has 0 variants, skipping\n",
      "[INFO]\tGene HRAT17 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA2 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT7 has 0 variants, skipping\n",
      "[INFO]\tGene PNKP has 0 variants, skipping\n",
      "[INFO]\tGene SFXN4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01393 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01392 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA25B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724434 has 0 variants, skipping\n",
      "[INFO]\tGene COQ9 has 0 variants, skipping\n",
      "[INFO]\tGene ACKR3 has 0 variants, skipping\n",
      "[INFO]\tGene AKR1B10 has 0 variants, skipping\n",
      "[INFO]\tGene FN3K has 0 variants, skipping\n",
      "[INFO]\tGene TSHZ3 has 0 variants, skipping\n",
      "[INFO]\tGene NEFH has 0 variants, skipping\n",
      "[INFO]\tGene ST7-OT4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6132 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene ST7-OT3 has 0 variants, skipping\n",
      "[INFO]\tGene ERCC4 has 0 variants, skipping\n",
      "[INFO]\tGene REXO1 has 0 variants, skipping\n",
      "[INFO]\tGene LRPPRC has 0 variants, skipping\n",
      "[INFO]\tGene CFTR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TEX47 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF449 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM104 has 0 variants, skipping\n",
      "[INFO]\tGene GADD45B has 0 variants, skipping\n",
      "[INFO]\tGene RIMBP3 has 0 variants, skipping\n",
      "[INFO]\tGene SENP3 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB5 has 0 variants, skipping\n",
      "[INFO]\tGene RGS22 has 0 variants, skipping\n",
      "[INFO]\tGene FZD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR433 has 0 variants, skipping\n",
      "[INFO]\tGene HYAL6P has 0 variants, skipping\n",
      "[INFO]\tGene GOPC has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375483 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928211 has 0 variants, skipping\n",
      "[INFO]\tGene IRS2 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3A has 0 variants, skipping\n",
      "[INFO]\tGene POT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928283 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928254 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506682 has 0 variants, skipping\n",
      "[INFO]\tGene SND1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM238 has 0 variants, skipping\n",
      "[INFO]\tGene GET3 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01000 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23C has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF17 has 0 variants, skipping\n",
      "[INFO]\tGene FLNC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB5 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2AIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369535 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TPI1P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC407835 has 0 variants, skipping\n",
      "[INFO]\tGene COL15A1 has 0 variants, skipping\n",
      "[INFO]\tGene CTXND2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128325 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3928 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3922 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375504 has 0 variants, skipping\n",
      "[INFO]\tGene PNPLA3 has 0 variants, skipping\n",
      "[INFO]\tGene WDR82 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM9 has 0 variants, skipping\n",
      "[INFO]\tGene ACSBG1 has 0 variants, skipping\n",
      "[INFO]\tGene COPG2IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM4 has 0 variants, skipping\n",
      "[INFO]\tGene PACS1 has 0 variants, skipping\n",
      "[INFO]\tGene MKLN1-AS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928782 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928807 has 0 variants, skipping\n",
      "[INFO]\tGene UCK2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ40288 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506937 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375512 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6133 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928861 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01230 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM101 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6509 has 0 variants, skipping\n",
      "[INFO]\tGene LOC349160 has 0 variants, skipping\n",
      "[INFO]\tGene FAM229B has 0 variants, skipping\n",
      "[INFO]\tGene LRIT2 has 0 variants, skipping\n",
      "[INFO]\tGene CPA2 has 0 variants, skipping\n",
      "[INFO]\tGene CPA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4468 has 0 variants, skipping\n",
      "[INFO]\tGene GDF2 has 0 variants, skipping\n",
      "[INFO]\tGene LSM8 has 0 variants, skipping\n",
      "[INFO]\tGene CRNN has 0 variants, skipping\n",
      "[INFO]\tGene ACTL6B has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4A has 0 variants, skipping\n",
      "[INFO]\tGene MRTO4 has 0 variants, skipping\n",
      "[INFO]\tGene SRC has 0 variants, skipping\n",
      "[INFO]\tGene RIMS3 has 0 variants, skipping\n",
      "[INFO]\tGene FMC1 has 0 variants, skipping\n",
      "[INFO]\tGene INTS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129148 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene RILPL2 has 0 variants, skipping\n",
      "[INFO]\tGene WEE2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK14 has 0 variants, skipping\n",
      "[INFO]\tGene INCA1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP27 has 0 variants, skipping\n",
      "[INFO]\tGene MOXD2P has 0 variants, skipping\n",
      "[INFO]\tGene TRY2P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE6 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC168 has 0 variants, skipping\n",
      "[INFO]\tGene OR6W1P has 0 variants, skipping\n",
      "[INFO]\tGene ETFRF1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00336 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6892 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TCAF2P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC154761 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A9P has 0 variants, skipping\n",
      "[INFO]\tGene OR2A20P has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF34P has 0 variants, skipping\n",
      "[INFO]\tGene MIR197 has 0 variants, skipping\n",
      "[INFO]\tGene VTRNA2-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR196A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR298 has 0 variants, skipping\n",
      "[INFO]\tGene MIR192 has 0 variants, skipping\n",
      "[INFO]\tGene MIR675 has 0 variants, skipping\n",
      "[INFO]\tGene MIR16-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR107 has 0 variants, skipping\n",
      "[INFO]\tGene MIR106A has 0 variants, skipping\n",
      "[INFO]\tGene MIR103A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR103A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR101-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR100 has 0 variants, skipping\n",
      "[INFO]\tGene MIR98 has 0 variants, skipping\n",
      "[INFO]\tGene MIR96 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR93 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR25 has 0 variants, skipping\n",
      "[INFO]\tGene MIR92A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR33A has 0 variants, skipping\n",
      "[INFO]\tGene MIR24-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR24-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR31 has 0 variants, skipping\n",
      "[INFO]\tGene MIR23A has 0 variants, skipping\n",
      "[INFO]\tGene MIR30A has 0 variants, skipping\n",
      "[INFO]\tGene MIR22 has 0 variants, skipping\n",
      "[INFO]\tGene MIR29A has 0 variants, skipping\n",
      "[INFO]\tGene MIR21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR20A has 0 variants, skipping\n",
      "[INFO]\tGene MIR27A has 0 variants, skipping\n",
      "[INFO]\tGene MIR19B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR19B1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR26B has 0 variants, skipping\n",
      "[INFO]\tGene MIR19A has 0 variants, skipping\n",
      "[INFO]\tGene MIR17 has 0 variants, skipping\n",
      "[INFO]\tGene MIR16-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR15A has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7F1 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7E has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7D has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7B has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7C has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F4 has 0 variants, skipping\n",
      "[INFO]\tGene TAOK1 has 0 variants, skipping\n",
      "[INFO]\tGene BBOF1 has 0 variants, skipping\n",
      "[INFO]\tGene STN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNY5 has 0 variants, skipping\n",
      "[INFO]\tGene RNY4 has 0 variants, skipping\n",
      "[INFO]\tGene RNY3 has 0 variants, skipping\n",
      "[INFO]\tGene RNY1 has 0 variants, skipping\n",
      "[INFO]\tGene RBX1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC155060 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF767P has 0 variants, skipping\n",
      "[INFO]\tGene SLAIN2 has 0 variants, skipping\n",
      "[INFO]\tGene RPRM has 0 variants, skipping\n",
      "[INFO]\tGene SLCO1B3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V0E2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134040 has 0 variants, skipping\n",
      "[INFO]\tGene GPC5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728743 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00996 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK7 has 0 variants, skipping\n",
      "[INFO]\tGene CD300H has 0 variants, skipping\n",
      "[INFO]\tGene OR9A2 has 0 variants, skipping\n",
      "[INFO]\tGene AVP has 0 variants, skipping\n",
      "[INFO]\tGene UBE2D2 has 0 variants, skipping\n",
      "[INFO]\tGene NR1I2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3907 has 0 variants, skipping\n",
      "[INFO]\tGene MRAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LCOR has 0 variants, skipping\n",
      "[INFO]\tGene PRKAG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM3AP has 0 variants, skipping\n",
      "[INFO]\tGene FABP5P3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01003 has 0 variants, skipping\n",
      "[INFO]\tGene ASPN has 0 variants, skipping\n",
      "[INFO]\tGene RIT2 has 0 variants, skipping\n",
      "[INFO]\tGene GRK1 has 0 variants, skipping\n",
      "[INFO]\tGene RGS16 has 0 variants, skipping\n",
      "[INFO]\tGene SHB has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929998 has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene HTR5A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPYSL4 has 0 variants, skipping\n",
      "[INFO]\tGene BLACE has 0 variants, skipping\n",
      "[INFO]\tGene LOC100286906 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506302 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF750 has 0 variants, skipping\n",
      "[INFO]\tGene LOC285889 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01006 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00244 has 0 variants, skipping\n",
      "[INFO]\tGene IL13RA2 has 0 variants, skipping\n",
      "[INFO]\tGene VTN has 0 variants, skipping\n",
      "[INFO]\tGene MIR762 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320B2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1301 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1271 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1296 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320C1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320B1 has 0 variants, skipping\n",
      "[INFO]\tGene MNX1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927914 has 0 variants, skipping\n",
      "[INFO]\tGene FXR2 has 0 variants, skipping\n",
      "[INFO]\tGene DAB2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01022 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5707 has 0 variants, skipping\n",
      "[INFO]\tGene PARVA has 0 variants, skipping\n",
      "[INFO]\tGene UBR4 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS3P2 has 0 variants, skipping\n",
      "[INFO]\tGene FOSL2 has 0 variants, skipping\n",
      "[INFO]\tGene FOS has 0 variants, skipping\n",
      "[INFO]\tGene FOXC2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXG1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL1 has 0 variants, skipping\n",
      "[INFO]\tGene CDK6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NRARP has 0 variants, skipping\n",
      "[INFO]\tGene PMM2 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM28 has 0 variants, skipping\n",
      "[INFO]\tGene SORCS3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMA3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7A2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL23AP53 has 0 variants, skipping\n",
      "[INFO]\tGene FAM87A has 0 variants, skipping\n",
      "[INFO]\tGene DNAH1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401442 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286083 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927752 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3674 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG11 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928058 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD11-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7160 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927815 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287015 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A25 has 0 variants, skipping\n",
      "[INFO]\tGene ILDR2 has 0 variants, skipping\n",
      "[INFO]\tGene MCPH1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8055 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4659A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4659B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT19 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-24F4.2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA8P has 0 variants, skipping\n",
      "[INFO]\tGene DEFA9P has 0 variants, skipping\n",
      "[INFO]\tGene DEFA10P has 0 variants, skipping\n",
      "[INFO]\tGene DEFT1P has 0 variants, skipping\n",
      "[INFO]\tGene DEFT1P2 has 0 variants, skipping\n",
      "[INFO]\tGene DEFA11P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00965 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A7P has 0 variants, skipping\n",
      "[INFO]\tGene FAM66B has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A10P has 0 variants, skipping\n",
      "[INFO]\tGene FAM66E has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM85B has 0 variants, skipping\n",
      "[INFO]\tGene ASDURF has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3I has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC157273 has 0 variants, skipping\n",
      "[INFO]\tGene BBS5 has 0 variants, skipping\n",
      "[INFO]\tGene LINCR-0001 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4286 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723313 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929229 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929269 has 0 variants, skipping\n",
      "[INFO]\tGene TDH has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129129 has 0 variants, skipping\n",
      "[INFO]\tGene FAM167A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00208 has 0 variants, skipping\n",
      "[INFO]\tGene PGA4 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA99 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1973 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1915 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1908 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR320D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1469 has 0 variants, skipping\n",
      "[INFO]\tGene MIR513B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1307 has 0 variants, skipping\n",
      "[INFO]\tGene MIR664A has 0 variants, skipping\n",
      "[INFO]\tGene MIR1281 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1252 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302F has 0 variants, skipping\n",
      "[INFO]\tGene MIR1275 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1265 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1258 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1251 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1249 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1247 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1246 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1304 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1299 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1297 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548E has 0 variants, skipping\n",
      "[INFO]\tGene MIR1207 has 0 variants, skipping\n",
      "[INFO]\tGene ASB18 has 0 variants, skipping\n",
      "[INFO]\tGene PATE2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLA3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM66D has 0 variants, skipping\n",
      "[INFO]\tGene LOC392196 has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A2P has 0 variants, skipping\n",
      "[INFO]\tGene SLC39A5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM85A has 0 variants, skipping\n",
      "[INFO]\tGene FAM66A has 0 variants, skipping\n",
      "[INFO]\tGene LOC649352 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB109A has 0 variants, skipping\n",
      "[INFO]\tGene FAM90A25P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506990 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729732 has 0 variants, skipping\n",
      "[INFO]\tGene CTBP1-DT has 0 variants, skipping\n",
      "[INFO]\tGene TMEM106B has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3926-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC340357 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00681 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725080 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF230 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P20 has 0 variants, skipping\n",
      "[INFO]\tGene SSTR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00685 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF793 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN7L1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548V has 0 variants, skipping\n",
      "[INFO]\tGene MOCS2 has 0 variants, skipping\n",
      "[INFO]\tGene SDR42E2 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K2 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH3 has 0 variants, skipping\n",
      "[INFO]\tGene FBL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929066 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128993 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00536 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRG has 0 variants, skipping\n",
      "[INFO]\tGene TSPY4 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT6 has 0 variants, skipping\n",
      "[INFO]\tGene LZTS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102467222 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02153 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929172 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PCCB has 0 variants, skipping\n",
      "[INFO]\tGene CFAP36 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM26 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A12 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507071 has 0 variants, skipping\n",
      "[INFO]\tGene PSME4 has 0 variants, skipping\n",
      "[INFO]\tGene FAF2 has 0 variants, skipping\n",
      "[INFO]\tGene MDN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN5 has 0 variants, skipping\n",
      "[INFO]\tGene SPP2 has 0 variants, skipping\n",
      "[INFO]\tGene BIN3-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929237 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286059 has 0 variants, skipping\n",
      "[INFO]\tGene LOC254896 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389641 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507156 has 0 variants, skipping\n",
      "[INFO]\tGene RXFP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA25 has 0 variants, skipping\n",
      "[INFO]\tGene VSTM2L has 0 variants, skipping\n",
      "[INFO]\tGene NFIA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929294 has 0 variants, skipping\n",
      "[INFO]\tGene TBX19 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02869 has 0 variants, skipping\n",
      "[INFO]\tGene CCR5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929315 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM178B has 0 variants, skipping\n",
      "[INFO]\tGene TEX13C has 0 variants, skipping\n",
      "[INFO]\tGene MIR6841 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6876 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379331 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF367 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA4 has 0 variants, skipping\n",
      "[INFO]\tGene ELMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene SDAD1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ARPIN-AP3S2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548H4 has 0 variants, skipping\n",
      "[INFO]\tGene EN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6842 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6843 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3622B has 0 variants, skipping\n",
      "[INFO]\tGene LINC01703 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4287 has 0 variants, skipping\n",
      "[INFO]\tGene CD27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450A2 has 0 variants, skipping\n",
      "[INFO]\tGene EXTL3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-12 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHA2 has 0 variants, skipping\n",
      "[INFO]\tGene RBM4B has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF113A has 0 variants, skipping\n",
      "[INFO]\tGene LINC02099 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929470 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02209 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3148 has 0 variants, skipping\n",
      "[INFO]\tGene RBPMS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene REG3G has 0 variants, skipping\n",
      "[INFO]\tGene NRG1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene NRG1-IT3 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf51 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOT has 0 variants, skipping\n",
      "[INFO]\tGene TOB2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC75A has 0 variants, skipping\n",
      "[INFO]\tGene GRASLND has 0 variants, skipping\n",
      "[INFO]\tGene SNORD13 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01288 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929550 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929622 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723701 has 0 variants, skipping\n",
      "[INFO]\tGene GPR75-ASB3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728024 has 0 variants, skipping\n",
      "[INFO]\tGene SYN2 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D10A has 0 variants, skipping\n",
      "[INFO]\tGene RNF5P1 has 0 variants, skipping\n",
      "[INFO]\tGene DPH6 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF26 has 0 variants, skipping\n",
      "[INFO]\tGene CREBZF has 0 variants, skipping\n",
      "[INFO]\tGene ZNF493 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO9 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS3BP has 0 variants, skipping\n",
      "[INFO]\tGene SNORD38D has 0 variants, skipping\n",
      "[INFO]\tGene ADAM5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130964 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AO has 0 variants, skipping\n",
      "[INFO]\tGene SNORD65B has 0 variants, skipping\n",
      "[INFO]\tGene EDDM13 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D16 has 0 variants, skipping\n",
      "[INFO]\tGene KCNS3 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379393 has 0 variants, skipping\n",
      "[INFO]\tGene SCRN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929897 has 0 variants, skipping\n",
      "[INFO]\tGene IER5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC27 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC7 has 0 variants, skipping\n",
      "[INFO]\tGene SF3B3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4469 has 0 variants, skipping\n",
      "[INFO]\tGene ASNSP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00293 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100287846 has 0 variants, skipping\n",
      "[INFO]\tGene SDR9C7 has 0 variants, skipping\n",
      "[INFO]\tGene NPB has 0 variants, skipping\n",
      "[INFO]\tGene BOD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRBA1 has 0 variants, skipping\n",
      "[INFO]\tGene TNS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCG11 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00472 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00520 has 0 variants, skipping\n",
      "[INFO]\tGene RUNX1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00313 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00473 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00691 has 0 variants, skipping\n",
      "[INFO]\tGene SATB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP52 has 0 variants, skipping\n",
      "[INFO]\tGene PSG10P has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3G has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929268 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929217 has 0 variants, skipping\n",
      "[INFO]\tGene PEX13 has 0 variants, skipping\n",
      "[INFO]\tGene PER1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507464 has 0 variants, skipping\n",
      "[INFO]\tGene ARSF has 0 variants, skipping\n",
      "[INFO]\tGene RPS7 has 0 variants, skipping\n",
      "[INFO]\tGene KITLG has 0 variants, skipping\n",
      "[INFO]\tGene SRP9 has 0 variants, skipping\n",
      "[INFO]\tGene SRM has 0 variants, skipping\n",
      "[INFO]\tGene PRB3 has 0 variants, skipping\n",
      "[INFO]\tGene PRB2 has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5B has 0 variants, skipping\n",
      "[INFO]\tGene PPP2R5A has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R3D has 0 variants, skipping\n",
      "[INFO]\tGene PPEF2 has 0 variants, skipping\n",
      "[INFO]\tGene PPARD has 0 variants, skipping\n",
      "[INFO]\tGene POU4F1 has 0 variants, skipping\n",
      "[INFO]\tGene POU3F3 has 0 variants, skipping\n",
      "[INFO]\tGene POU2AF1 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2J has 0 variants, skipping\n",
      "[INFO]\tGene POLR2I has 0 variants, skipping\n",
      "[INFO]\tGene POLE has 0 variants, skipping\n",
      "[INFO]\tGene ACAA2 has 0 variants, skipping\n",
      "[INFO]\tGene TSNAX has 0 variants, skipping\n",
      "[INFO]\tGene PIAS3 has 0 variants, skipping\n",
      "[INFO]\tGene RACK1 has 0 variants, skipping\n",
      "[INFO]\tGene SPON1 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE6 has 0 variants, skipping\n",
      "[INFO]\tGene PCNT has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929341 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01537 has 0 variants, skipping\n",
      "[INFO]\tGene NRIR has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507516 has 0 variants, skipping\n",
      "[INFO]\tGene FEZ2 has 0 variants, skipping\n",
      "[INFO]\tGene HERPUD2 has 0 variants, skipping\n",
      "[INFO]\tGene TOR3A has 0 variants, skipping\n",
      "[INFO]\tGene ROBO3 has 0 variants, skipping\n",
      "[INFO]\tGene RNU105C has 0 variants, skipping\n",
      "[INFO]\tGene SBF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375843 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA1B has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS10 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD54 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA3 has 0 variants, skipping\n",
      "[INFO]\tGene H3C13 has 0 variants, skipping\n",
      "[INFO]\tGene HMSD has 0 variants, skipping\n",
      "[INFO]\tGene SDR16C6P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929415 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286177 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00588 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929488 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286178 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929528 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505501 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01301 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130298 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02155 has 0 variants, skipping\n",
      "[INFO]\tGene YTHDF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR11G2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724612 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01289 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01414 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401463 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00251 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01299 has 0 variants, skipping\n",
      "[INFO]\tGene HR has 0 variants, skipping\n",
      "[INFO]\tGene LINC00967 has 0 variants, skipping\n",
      "[INFO]\tGene RRS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GGH has 0 variants, skipping\n",
      "[INFO]\tGene GMPS has 0 variants, skipping\n",
      "[INFO]\tGene ARL8A has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723655 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723713 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD87 has 0 variants, skipping\n",
      "[INFO]\tGene N4BP1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRA4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724708 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX9 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf34-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM187A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01592 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01603 has 0 variants, skipping\n",
      "[INFO]\tGene RPS10 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT1P3 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7-DT has 0 variants, skipping\n",
      "[INFO]\tGene CCL24 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926892 has 0 variants, skipping\n",
      "[INFO]\tGene LACTB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC392232 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926908 has 0 variants, skipping\n",
      "[INFO]\tGene RDH10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene STAU2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NRAD1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01617 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5681A has 0 variants, skipping\n",
      "[INFO]\tGene MIR5681B has 0 variants, skipping\n",
      "[INFO]\tGene TUSC7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2052 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01111 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3149 has 0 variants, skipping\n",
      "[INFO]\tGene WTIP has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724874 has 0 variants, skipping\n",
      "[INFO]\tGene PKIA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101241902 has 0 variants, skipping\n",
      "[INFO]\tGene PIGO has 0 variants, skipping\n",
      "[INFO]\tGene LINC01607 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927040 has 0 variants, skipping\n",
      "[INFO]\tGene OXCT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389199 has 0 variants, skipping\n",
      "[INFO]\tGene TBKBP1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5708 has 0 variants, skipping\n",
      "[INFO]\tGene IMPA1P1 has 0 variants, skipping\n",
      "[INFO]\tGene AMTN has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723322 has 0 variants, skipping\n",
      "[INFO]\tGene REXO1L2P has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929709 has 0 variants, skipping\n",
      "[INFO]\tGene PXN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00534 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01030 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4661 has 0 variants, skipping\n",
      "[INFO]\tGene LLCFC1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM274P has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724710 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ46284 has 0 variants, skipping\n",
      "[INFO]\tGene CARTPT has 0 variants, skipping\n",
      "[INFO]\tGene NFE2L3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00535 has 0 variants, skipping\n",
      "[INFO]\tGene RBM12B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R14 has 0 variants, skipping\n",
      "[INFO]\tGene CEP128 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100288748 has 0 variants, skipping\n",
      "[INFO]\tGene SLC35A4 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA5 has 0 variants, skipping\n",
      "[INFO]\tGene DTD2 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3HAV1L has 0 variants, skipping\n",
      "[INFO]\tGene C11orf52 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX3 has 0 variants, skipping\n",
      "[INFO]\tGene P4HTM has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G4D has 0 variants, skipping\n",
      "[INFO]\tGene NPW has 0 variants, skipping\n",
      "[INFO]\tGene ARAP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR105-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150BHG has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3150A has 0 variants, skipping\n",
      "[INFO]\tGene LINC01298 has 0 variants, skipping\n",
      "[INFO]\tGene AFDN-DT has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724804 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927066 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3H has 0 variants, skipping\n",
      "[INFO]\tGene ZNF774 has 0 variants, skipping\n",
      "[INFO]\tGene FCER1A has 0 variants, skipping\n",
      "[INFO]\tGene CST5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA72 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN2D has 0 variants, skipping\n",
      "[INFO]\tGene FAM83E has 0 variants, skipping\n",
      "[INFO]\tGene PTS has 0 variants, skipping\n",
      "[INFO]\tGene SAP30 has 0 variants, skipping\n",
      "[INFO]\tGene DPM2 has 0 variants, skipping\n",
      "[INFO]\tGene FGF18 has 0 variants, skipping\n",
      "[INFO]\tGene BANF1 has 0 variants, skipping\n",
      "[INFO]\tGene GALR2 has 0 variants, skipping\n",
      "[INFO]\tGene NMUR2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD77B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4471 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7705 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ42969 has 0 variants, skipping\n",
      "[INFO]\tGene NACA4P has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A20 has 0 variants, skipping\n",
      "[INFO]\tGene BLMH has 0 variants, skipping\n",
      "[INFO]\tGene APOB has 0 variants, skipping\n",
      "[INFO]\tGene LOC104054148 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5680 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC72 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF23 has 0 variants, skipping\n",
      "[INFO]\tGene UBR5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD2 has 0 variants, skipping\n",
      "[INFO]\tGene SNX9 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC13 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf61 has 0 variants, skipping\n",
      "[INFO]\tGene SLC20A1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01181 has 0 variants, skipping\n",
      "[INFO]\tGene BAALC-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3151 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369147 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD173 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375690 has 0 variants, skipping\n",
      "[INFO]\tGene RNF126 has 0 variants, skipping\n",
      "[INFO]\tGene SKOR1 has 0 variants, skipping\n",
      "[INFO]\tGene PRRT2 has 0 variants, skipping\n",
      "[INFO]\tGene STON2 has 0 variants, skipping\n",
      "[INFO]\tGene MIS18BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TMA16 has 0 variants, skipping\n",
      "[INFO]\tGene GMDS has 0 variants, skipping\n",
      "[INFO]\tGene OR52I2 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN12 has 0 variants, skipping\n",
      "[INFO]\tGene UHMK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC52A1 has 0 variants, skipping\n",
      "[INFO]\tGene IPO9 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G2F has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3B has 0 variants, skipping\n",
      "[INFO]\tGene PER2 has 0 variants, skipping\n",
      "[INFO]\tGene MIF-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PFN4 has 0 variants, skipping\n",
      "[INFO]\tGene SFT2D2 has 0 variants, skipping\n",
      "[INFO]\tGene SVBP has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM9 has 0 variants, skipping\n",
      "[INFO]\tGene SPEM1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01608 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01609 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02237 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRA has 0 variants, skipping\n",
      "[INFO]\tGene TUBA1B has 0 variants, skipping\n",
      "[INFO]\tGene SIX1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN31 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3A has 0 variants, skipping\n",
      "[INFO]\tGene S100P has 0 variants, skipping\n",
      "[INFO]\tGene AGFG2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375713 has 0 variants, skipping\n",
      "[INFO]\tGene RAD21-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3610 has 0 variants, skipping\n",
      "[INFO]\tGene MZT2A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD168 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927513 has 0 variants, skipping\n",
      "[INFO]\tGene MAL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AZ has 0 variants, skipping\n",
      "[INFO]\tGene DEFB127 has 0 variants, skipping\n",
      "[INFO]\tGene MAPKAPK5 has 0 variants, skipping\n",
      "[INFO]\tGene TPCN2 has 0 variants, skipping\n",
      "[INFO]\tGene MINAR1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF425 has 0 variants, skipping\n",
      "[INFO]\tGene TMIGD1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM88 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927543 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01151 has 0 variants, skipping\n",
      "[INFO]\tGene ST3GAL1 has 0 variants, skipping\n",
      "[INFO]\tGene DRD4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4663 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AA1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L6-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927588 has 0 variants, skipping\n",
      "[INFO]\tGene RNF139-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PARP14 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6844 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4662B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4662A has 0 variants, skipping\n",
      "[INFO]\tGene LINC00964 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375744 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00861 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927657 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene CASC21 has 0 variants, skipping\n",
      "[INFO]\tGene ST8SIA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM75 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1206 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00824 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00977 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF10 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3686 has 0 variants, skipping\n",
      "[INFO]\tGene FIBP has 0 variants, skipping\n",
      "[INFO]\tGene OR2C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B6 has 0 variants, skipping\n",
      "[INFO]\tGene OR1Q1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1N1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1F1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC146 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS14 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB13 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5194 has 0 variants, skipping\n",
      "[INFO]\tGene ASAP1-IT2 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5MGL has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A11 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH2 has 0 variants, skipping\n",
      "[INFO]\tGene UBR3 has 0 variants, skipping\n",
      "[INFO]\tGene HPYR1 has 0 variants, skipping\n",
      "[INFO]\tGene H4C11 has 0 variants, skipping\n",
      "[INFO]\tGene FDCSP has 0 variants, skipping\n",
      "[INFO]\tGene LIX1 has 0 variants, skipping\n",
      "[INFO]\tGene EID2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM92 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO39 has 0 variants, skipping\n",
      "[INFO]\tGene SBSPON has 0 variants, skipping\n",
      "[INFO]\tGene CILP2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF524 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM174 has 0 variants, skipping\n",
      "[INFO]\tGene POU5F2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF19 has 0 variants, skipping\n",
      "[INFO]\tGene GJB4 has 0 variants, skipping\n",
      "[INFO]\tGene KIF19 has 0 variants, skipping\n",
      "[INFO]\tGene USP43 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7848 has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375773 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927798 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927822 has 0 variants, skipping\n",
      "[INFO]\tGene PRB4 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NCRNA00250 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927845 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01591 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02055 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927915 has 0 variants, skipping\n",
      "[INFO]\tGene TIGAR has 0 variants, skipping\n",
      "[INFO]\tGene PEG13 has 0 variants, skipping\n",
      "[INFO]\tGene RGMB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NOBOX has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ1-DT has 0 variants, skipping\n",
      "[INFO]\tGene GSTCD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF878 has 0 variants, skipping\n",
      "[INFO]\tGene GXYLT2 has 0 variants, skipping\n",
      "[INFO]\tGene DHDDS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L3 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L4 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-9 has 0 variants, skipping\n",
      "[INFO]\tGene RPAP3 has 0 variants, skipping\n",
      "[INFO]\tGene MPV17 has 0 variants, skipping\n",
      "[INFO]\tGene TENM4 has 0 variants, skipping\n",
      "[INFO]\tGene DBX1 has 0 variants, skipping\n",
      "[INFO]\tGene ACOT1 has 0 variants, skipping\n",
      "[INFO]\tGene CYS1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G2P has 0 variants, skipping\n",
      "[INFO]\tGene LYPD4 has 0 variants, skipping\n",
      "[INFO]\tGene GSTP1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376306 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371049 has 0 variants, skipping\n",
      "[INFO]\tGene SLC66A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC01300 has 0 variants, skipping\n",
      "[INFO]\tGene MROH5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4472-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00051 has 0 variants, skipping\n",
      "[INFO]\tGene ACVR2B has 0 variants, skipping\n",
      "[INFO]\tGene ASIC2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928087 has 0 variants, skipping\n",
      "[INFO]\tGene HUNK has 0 variants, skipping\n",
      "[INFO]\tGene LY6E-DT has 0 variants, skipping\n",
      "[INFO]\tGene CDC42P3 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf31 has 0 variants, skipping\n",
      "[INFO]\tGene CRISPLD2 has 0 variants, skipping\n",
      "[INFO]\tGene MAFA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD149 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100310756 has 0 variants, skipping\n",
      "[INFO]\tGene NUBP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT81 has 0 variants, skipping\n",
      "[INFO]\tGene KRT83 has 0 variants, skipping\n",
      "[INFO]\tGene CPB1 has 0 variants, skipping\n",
      "[INFO]\tGene KDM5D has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928160 has 0 variants, skipping\n",
      "[INFO]\tGene SPTA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4664 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5C has 0 variants, skipping\n",
      "[INFO]\tGene GPR173 has 0 variants, skipping\n",
      "[INFO]\tGene WDR5B has 0 variants, skipping\n",
      "[INFO]\tGene SNTG2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6846 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6847 has 0 variants, skipping\n",
      "[INFO]\tGene FAAP24 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7112 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6848 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928764 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928902 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1234 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6849 has 0 variants, skipping\n",
      "[INFO]\tGene TRERNA1 has 0 variants, skipping\n",
      "[INFO]\tGene TLR9 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD3 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF39 has 0 variants, skipping\n",
      "[INFO]\tGene CEP89 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6850 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130027 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf167 has 0 variants, skipping\n",
      "[INFO]\tGene NUTM2A has 0 variants, skipping\n",
      "[INFO]\tGene COQ10A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF252P has 0 variants, skipping\n",
      "[INFO]\tGene TMED10P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF252P-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PPFIA3 has 0 variants, skipping\n",
      "[INFO]\tGene ARL6IP6 has 0 variants, skipping\n",
      "[INFO]\tGene CPA3 has 0 variants, skipping\n",
      "[INFO]\tGene GMNC has 0 variants, skipping\n",
      "[INFO]\tGene TMEM88B has 0 variants, skipping\n",
      "[INFO]\tGene RNF222 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM253 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02303 has 0 variants, skipping\n",
      "[INFO]\tGene SYVN1 has 0 variants, skipping\n",
      "[INFO]\tGene RABIF has 0 variants, skipping\n",
      "[INFO]\tGene RAB6A has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM154 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6A has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC51A has 0 variants, skipping\n",
      "[INFO]\tGene STK36 has 0 variants, skipping\n",
      "[INFO]\tGene SLC18A3 has 0 variants, skipping\n",
      "[INFO]\tGene CHST13 has 0 variants, skipping\n",
      "[INFO]\tGene COL22A1 has 0 variants, skipping\n",
      "[INFO]\tGene NCAPD2 has 0 variants, skipping\n",
      "[INFO]\tGene USP37 has 0 variants, skipping\n",
      "[INFO]\tGene NXT2 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8 has 0 variants, skipping\n",
      "[INFO]\tGene EMC10 has 0 variants, skipping\n",
      "[INFO]\tGene CHRNA10 has 0 variants, skipping\n",
      "[INFO]\tGene HCRT has 0 variants, skipping\n",
      "[INFO]\tGene HAS1 has 0 variants, skipping\n",
      "[INFO]\tGene GUCY2F has 0 variants, skipping\n",
      "[INFO]\tGene C1orf87 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF766 has 0 variants, skipping\n",
      "[INFO]\tGene WFIKKN1 has 0 variants, skipping\n",
      "[INFO]\tGene DLEU7 has 0 variants, skipping\n",
      "[INFO]\tGene PSMB1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPC has 0 variants, skipping\n",
      "[INFO]\tGene OR6T1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01231 has 0 variants, skipping\n",
      "[INFO]\tGene RFX3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GLIS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASCC3 has 0 variants, skipping\n",
      "[INFO]\tGene ZGLP1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC37L1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MMP20 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC2B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM82 has 0 variants, skipping\n",
      "[INFO]\tGene PRR25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC184 has 0 variants, skipping\n",
      "[INFO]\tGene CKS2 has 0 variants, skipping\n",
      "[INFO]\tGene ENTPD3 has 0 variants, skipping\n",
      "[INFO]\tGene GSTO1 has 0 variants, skipping\n",
      "[INFO]\tGene MED26 has 0 variants, skipping\n",
      "[INFO]\tGene TRAPPC8 has 0 variants, skipping\n",
      "[INFO]\tGene FMR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF4L1 has 0 variants, skipping\n",
      "[INFO]\tGene CDNF has 0 variants, skipping\n",
      "[INFO]\tGene OVCH1 has 0 variants, skipping\n",
      "[INFO]\tGene MRFAP1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375972 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRD-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD137 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01235 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00583 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD13 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4A22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389705 has 0 variants, skipping\n",
      "[INFO]\tGene REP15 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE40 has 0 variants, skipping\n",
      "[INFO]\tGene ADPRHL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3152 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928120 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL20 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4473 has 0 variants, skipping\n",
      "[INFO]\tGene FOCAD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA30B has 0 variants, skipping\n",
      "[INFO]\tGene IFNA22P has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM16 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2A-DT has 0 variants, skipping\n",
      "[INFO]\tGene MN1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP15 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01239 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929563 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DMB has 0 variants, skipping\n",
      "[INFO]\tGene LINC01241 has 0 variants, skipping\n",
      "[INFO]\tGene IFT74-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERZ has 0 variants, skipping\n",
      "[INFO]\tGene LINC00032 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01242 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01243 has 0 variants, skipping\n",
      "[INFO]\tGene SYN1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPA13 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZIC2 has 0 variants, skipping\n",
      "[INFO]\tGene GLRX5 has 0 variants, skipping\n",
      "[INFO]\tGene TRIAP1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDSPL2 has 0 variants, skipping\n",
      "[INFO]\tGene UFC1 has 0 variants, skipping\n",
      "[INFO]\tGene USH2A has 0 variants, skipping\n",
      "[INFO]\tGene TENT5A has 0 variants, skipping\n",
      "[INFO]\tGene DDX60 has 0 variants, skipping\n",
      "[INFO]\tGene AGO4 has 0 variants, skipping\n",
      "[INFO]\tGene TET2 has 0 variants, skipping\n",
      "[INFO]\tGene ITLN1 has 0 variants, skipping\n",
      "[INFO]\tGene CNNM3 has 0 variants, skipping\n",
      "[INFO]\tGene BORCS6 has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH4 has 0 variants, skipping\n",
      "[INFO]\tGene RNPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6851 has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01251 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB8 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB125 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD121B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD121A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC83 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGB has 0 variants, skipping\n",
      "[INFO]\tGene CNTFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TRAF3IP1 has 0 variants, skipping\n",
      "[INFO]\tGene CYTH2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM205BP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01118 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB5-DT has 0 variants, skipping\n",
      "[INFO]\tGene ATP8B5P has 0 variants, skipping\n",
      "[INFO]\tGene GPR26 has 0 variants, skipping\n",
      "[INFO]\tGene VRK1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4667 has 0 variants, skipping\n",
      "[INFO]\tGene NOG has 0 variants, skipping\n",
      "[INFO]\tGene BMP15 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101926948 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6853 has 0 variants, skipping\n",
      "[INFO]\tGene GBX1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX5 has 0 variants, skipping\n",
      "[INFO]\tGene ST6GALNAC2 has 0 variants, skipping\n",
      "[INFO]\tGene P3H4 has 0 variants, skipping\n",
      "[INFO]\tGene MXD4 has 0 variants, skipping\n",
      "[INFO]\tGene TBL3 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT10C has 0 variants, skipping\n",
      "[INFO]\tGene WRAP73 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D9 has 0 variants, skipping\n",
      "[INFO]\tGene OR2Z1 has 0 variants, skipping\n",
      "[INFO]\tGene OR2W5 has 0 variants, skipping\n",
      "[INFO]\tGene OR4D6 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM32 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4475 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00477 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4540 has 0 variants, skipping\n",
      "[INFO]\tGene EBLN3P has 0 variants, skipping\n",
      "[INFO]\tGene PLCXD3 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM95C has 0 variants, skipping\n",
      "[INFO]\tGene CLN8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF658B has 0 variants, skipping\n",
      "[INFO]\tGene GLIDR has 0 variants, skipping\n",
      "[INFO]\tGene FGF7P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724238 has 0 variants, skipping\n",
      "[INFO]\tGene LOC554249 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA9B has 0 variants, skipping\n",
      "[INFO]\tGene FAM95B1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379252 has 0 variants, skipping\n",
      "[INFO]\tGene AQP7P3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929583 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724580 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1HP has 0 variants, skipping\n",
      "[INFO]\tGene PGM5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC440896 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R2C has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984035 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM242F has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103908605 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927827 has 0 variants, skipping\n",
      "[INFO]\tGene XLOC_007697 has 0 variants, skipping\n",
      "[INFO]\tGene CD164L2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27C has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379807 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723709 has 0 variants, skipping\n",
      "[INFO]\tGene FGF7P6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01189 has 0 variants, skipping\n",
      "[INFO]\tGene LOC103908604 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132249 has 0 variants, skipping\n",
      "[INFO]\tGene PTGER4P2-CDK2AP2P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996349 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728673 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928381 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928195 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286297 has 0 variants, skipping\n",
      "[INFO]\tGene AQP7P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00537 has 0 variants, skipping\n",
      "[INFO]\tGene FRG1JP has 0 variants, skipping\n",
      "[INFO]\tGene FLJ43315 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00882 has 0 variants, skipping\n",
      "[INFO]\tGene TCL6 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNMB4 has 0 variants, skipping\n",
      "[INFO]\tGene XYLT2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC642929 has 0 variants, skipping\n",
      "[INFO]\tGene FAM74A3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379450 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP3P2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27E3 has 0 variants, skipping\n",
      "[INFO]\tGene FAM27B has 0 variants, skipping\n",
      "[INFO]\tGene MKRN3 has 0 variants, skipping\n",
      "[INFO]\tGene NELFA has 0 variants, skipping\n",
      "[INFO]\tGene LINC01506 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927069 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf135-DT has 0 variants, skipping\n",
      "[INFO]\tGene SMC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene USP46 has 0 variants, skipping\n",
      "[INFO]\tGene PON1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF235 has 0 variants, skipping\n",
      "[INFO]\tGene H4C7 has 0 variants, skipping\n",
      "[INFO]\tGene H4C13 has 0 variants, skipping\n",
      "[INFO]\tGene H4C5 has 0 variants, skipping\n",
      "[INFO]\tGene H4C2 has 0 variants, skipping\n",
      "[INFO]\tGene H4C8 has 0 variants, skipping\n",
      "[INFO]\tGene H4C3 has 0 variants, skipping\n",
      "[INFO]\tGene H4C12 has 0 variants, skipping\n",
      "[INFO]\tGene H4C6 has 0 variants, skipping\n",
      "[INFO]\tGene H4C4 has 0 variants, skipping\n",
      "[INFO]\tGene H4C1 has 0 variants, skipping\n",
      "[INFO]\tGene H3C2 has 0 variants, skipping\n",
      "[INFO]\tGene H3C10 has 0 variants, skipping\n",
      "[INFO]\tGene H3C12 has 0 variants, skipping\n",
      "[INFO]\tGene H3C8 has 0 variants, skipping\n",
      "[INFO]\tGene H3C11 has 0 variants, skipping\n",
      "[INFO]\tGene H3C6 has 0 variants, skipping\n",
      "[INFO]\tGene H3C3 has 0 variants, skipping\n",
      "[INFO]\tGene FARSA has 0 variants, skipping\n",
      "[INFO]\tGene FAP has 0 variants, skipping\n",
      "[INFO]\tGene SOCS6 has 0 variants, skipping\n",
      "[INFO]\tGene S1PR2 has 0 variants, skipping\n",
      "[INFO]\tGene MED14 has 0 variants, skipping\n",
      "[INFO]\tGene STK17B has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G12B has 0 variants, skipping\n",
      "[INFO]\tGene H3C1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC21 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC17 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC4 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC10 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC9 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC6 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC7 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC14 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC15 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC13 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC8 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC20 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC18 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC17 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC4 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC6 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC16 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC15 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC13 has 0 variants, skipping\n",
      "[INFO]\tGene SNN has 0 variants, skipping\n",
      "[INFO]\tGene FZD9 has 0 variants, skipping\n",
      "[INFO]\tGene OBSCN has 0 variants, skipping\n",
      "[INFO]\tGene LINC01504 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01474 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927358 has 0 variants, skipping\n",
      "[INFO]\tGene RORB-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00668 has 0 variants, skipping\n",
      "[INFO]\tGene NFAM1 has 0 variants, skipping\n",
      "[INFO]\tGene XKR3 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L1 has 0 variants, skipping\n",
      "[INFO]\tGene CARNMT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RPSAP9 has 0 variants, skipping\n",
      "[INFO]\tGene GABRP has 0 variants, skipping\n",
      "[INFO]\tGene HIF1A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene VPS13A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTO1 has 0 variants, skipping\n",
      "[INFO]\tGene ALB has 0 variants, skipping\n",
      "[INFO]\tGene NR0B1 has 0 variants, skipping\n",
      "[INFO]\tGene GNA14-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS36 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927450 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE13 has 0 variants, skipping\n",
      "[INFO]\tGene DENND6B has 0 variants, skipping\n",
      "[INFO]\tGene CPXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene SULT1A4 has 0 variants, skipping\n",
      "[INFO]\tGene F8A2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP25-1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01507 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927502 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D5P has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376114 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KA6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927575 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388282 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389765 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927623 has 0 variants, skipping\n",
      "[INFO]\tGene CXCR1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS1RR has 0 variants, skipping\n",
      "[INFO]\tGene LOC440173 has 0 variants, skipping\n",
      "[INFO]\tGene AREL1 has 0 variants, skipping\n",
      "[INFO]\tGene MBTD1 has 0 variants, skipping\n",
      "[INFO]\tGene CTSL3P has 0 variants, skipping\n",
      "[INFO]\tGene CTSLP8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC392364 has 0 variants, skipping\n",
      "[INFO]\tGene LOC497256 has 0 variants, skipping\n",
      "[INFO]\tGene GLDN has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724156 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4289 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3153 has 0 variants, skipping\n",
      "[INFO]\tGene ARSH has 0 variants, skipping\n",
      "[INFO]\tGene ZNF57 has 0 variants, skipping\n",
      "[INFO]\tGene UNQ6494 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927847 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4290HG has 0 variants, skipping\n",
      "[INFO]\tGene MIR4290 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01508 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01501 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK5 has 0 variants, skipping\n",
      "[INFO]\tGene TEX22 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129316 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3910-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3910-2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128076 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00475 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3651 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA84 has 0 variants, skipping\n",
      "[INFO]\tGene ELSPBP1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4670 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128361 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929748 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD19P has 0 variants, skipping\n",
      "[INFO]\tGene LOC642943 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927954 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00702 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4291 has 0 variants, skipping\n",
      "[INFO]\tGene CHSY1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02603 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132077 has 0 variants, skipping\n",
      "[INFO]\tGene PCAT7 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF740 has 0 variants, skipping\n",
      "[INFO]\tGene FYB2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2278 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6081 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3074 has 0 variants, skipping\n",
      "[INFO]\tGene AS3MT has 0 variants, skipping\n",
      "[INFO]\tGene SERAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MYOCD has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507346 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00092 has 0 variants, skipping\n",
      "[INFO]\tGene LOC158435 has 0 variants, skipping\n",
      "[INFO]\tGene LOC158434 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MYO10 has 0 variants, skipping\n",
      "[INFO]\tGene MMD has 0 variants, skipping\n",
      "[INFO]\tGene THAP2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441455 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132781 has 0 variants, skipping\n",
      "[INFO]\tGene PTMAP11 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L1P2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD18CP has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P4-STRA6LP has 0 variants, skipping\n",
      "[INFO]\tGene SUGT1P4-STRA6LP-CCDC180 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R27 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1302-8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC286359 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505774 has 0 variants, skipping\n",
      "[INFO]\tGene PTCSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CMC4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6854 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRB1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1A has 0 variants, skipping\n",
      "[INFO]\tGene NAMA has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928438 has 0 variants, skipping\n",
      "[INFO]\tGene STX17-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR37L1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM246-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00587 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01492 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928523 has 0 variants, skipping\n",
      "[INFO]\tGene SMC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP9-6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376194 has 0 variants, skipping\n",
      "[INFO]\tGene PPY has 0 variants, skipping\n",
      "[INFO]\tGene KRT86 has 0 variants, skipping\n",
      "[INFO]\tGene OR4E1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8081 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01505 has 0 variants, skipping\n",
      "[INFO]\tGene LOC340512 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01509 has 0 variants, skipping\n",
      "[INFO]\tGene NOP9 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4D has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3D has 0 variants, skipping\n",
      "[INFO]\tGene ABCA6 has 0 variants, skipping\n",
      "[INFO]\tGene UBALD1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAP3 has 0 variants, skipping\n",
      "[INFO]\tGene PGK2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3927 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7702 has 0 variants, skipping\n",
      "[INFO]\tGene COPS5 has 0 variants, skipping\n",
      "[INFO]\tGene GCN1 has 0 variants, skipping\n",
      "[INFO]\tGene TDRD9 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC37A5P has 0 variants, skipping\n",
      "[INFO]\tGene TRIM56 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4668 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS58 has 0 variants, skipping\n",
      "[INFO]\tGene RLN3 has 0 variants, skipping\n",
      "[INFO]\tGene IL22 has 0 variants, skipping\n",
      "[INFO]\tGene MIR454 has 0 variants, skipping\n",
      "[INFO]\tGene RAB9A has 0 variants, skipping\n",
      "[INFO]\tGene FAM225B has 0 variants, skipping\n",
      "[INFO]\tGene RAB9B has 0 variants, skipping\n",
      "[INFO]\tGene CFAP298-TCP10L has 0 variants, skipping\n",
      "[INFO]\tGene RBMS2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMG2 has 0 variants, skipping\n",
      "[INFO]\tGene SEMG1 has 0 variants, skipping\n",
      "[INFO]\tGene RBMS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5P2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928748 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928775 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00474 has 0 variants, skipping\n",
      "[INFO]\tGene PAPPA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ASTN2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70C has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928797 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02578 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01613 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC12 has 0 variants, skipping\n",
      "[INFO]\tGene TTC24 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2T34 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf82 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4500 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4484 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4481 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4480 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4476 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4470 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4467 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA1 has 0 variants, skipping\n",
      "[INFO]\tGene FNTB has 0 variants, skipping\n",
      "[INFO]\tGene C14orf28 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6AP1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGDIG has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene ARG2 has 0 variants, skipping\n",
      "[INFO]\tGene GSN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SIPA1L3 has 0 variants, skipping\n",
      "[INFO]\tGene H3P6 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY2FP has 0 variants, skipping\n",
      "[INFO]\tGene MIR4478 has 0 variants, skipping\n",
      "[INFO]\tGene OR1C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1A2 has 0 variants, skipping\n",
      "[INFO]\tGene NUFIP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD90 has 0 variants, skipping\n",
      "[INFO]\tGene RCN1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR600HG has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR526A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7150 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505588 has 0 variants, skipping\n",
      "[INFO]\tGene LOC613206 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129034 has 0 variants, skipping\n",
      "[INFO]\tGene MIR181A2HG has 0 variants, skipping\n",
      "[INFO]\tGene AHCTF1 has 0 variants, skipping\n",
      "[INFO]\tGene NOXRED1 has 0 variants, skipping\n",
      "[INFO]\tGene AMOT has 0 variants, skipping\n",
      "[INFO]\tGene XKRX has 0 variants, skipping\n",
      "[INFO]\tGene TMEM215 has 0 variants, skipping\n",
      "[INFO]\tGene CDC20 has 0 variants, skipping\n",
      "[INFO]\tGene CDC6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376271 has 0 variants, skipping\n",
      "[INFO]\tGene IRAK3 has 0 variants, skipping\n",
      "[INFO]\tGene KIF16B has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L2-PABPN1 has 0 variants, skipping\n",
      "[INFO]\tGene COA5 has 0 variants, skipping\n",
      "[INFO]\tGene CISD2 has 0 variants, skipping\n",
      "[INFO]\tGene CDC23 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC51145 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1183 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929116 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A15 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA65 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB15 has 0 variants, skipping\n",
      "[INFO]\tGene ZADH2 has 0 variants, skipping\n",
      "[INFO]\tGene EVC has 0 variants, skipping\n",
      "[INFO]\tGene GAST has 0 variants, skipping\n",
      "[INFO]\tGene DRD1 has 0 variants, skipping\n",
      "[INFO]\tGene FOLR2 has 0 variants, skipping\n",
      "[INFO]\tGene NOL7 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3911 has 0 variants, skipping\n",
      "[INFO]\tGene PSMC3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM49B has 0 variants, skipping\n",
      "[INFO]\tGene SMTNL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3960 has 0 variants, skipping\n",
      "[INFO]\tGene ALG12 has 0 variants, skipping\n",
      "[INFO]\tGene R3HDM4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC126 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723566 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4672 has 0 variants, skipping\n",
      "[INFO]\tGene RASL11B has 0 variants, skipping\n",
      "[INFO]\tGene CUEDC2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGES2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHF1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP24-AS1-EDEM2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3154 has 0 variants, skipping\n",
      "[INFO]\tGene NTSR2 has 0 variants, skipping\n",
      "[INFO]\tGene P3R3URF has 0 variants, skipping\n",
      "[INFO]\tGene EPX has 0 variants, skipping\n",
      "[INFO]\tGene MYBPC3 has 0 variants, skipping\n",
      "[INFO]\tGene MMUT has 0 variants, skipping\n",
      "[INFO]\tGene MPO has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A14 has 0 variants, skipping\n",
      "[INFO]\tGene TTC30B has 0 variants, skipping\n",
      "[INFO]\tGene MEI1 has 0 variants, skipping\n",
      "[INFO]\tGene ENTHD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF347 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506100 has 0 variants, skipping\n",
      "[INFO]\tGene DLG1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00665 has 0 variants, skipping\n",
      "[INFO]\tGene TTN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf57 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00466 has 0 variants, skipping\n",
      "[INFO]\tGene DDRGK1 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC2 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF2 has 0 variants, skipping\n",
      "[INFO]\tGene PLSCR5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH6 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929331 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01503 has 0 variants, skipping\n",
      "[INFO]\tGene NPR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR603 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD5 has 0 variants, skipping\n",
      "[INFO]\tGene PRRX2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C9 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2C19 has 0 variants, skipping\n",
      "[INFO]\tGene CYP2B6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6855 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100272217 has 0 variants, skipping\n",
      "[INFO]\tGene APOBEC3A has 0 variants, skipping\n",
      "[INFO]\tGene MIR6856 has 0 variants, skipping\n",
      "[INFO]\tGene POM121L2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3662 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3613 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3606 has 0 variants, skipping\n",
      "[INFO]\tGene KIFC1 has 0 variants, skipping\n",
      "[INFO]\tGene BCL9 has 0 variants, skipping\n",
      "[INFO]\tGene COPZ2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOA2 has 0 variants, skipping\n",
      "[INFO]\tGene GTSE1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD62A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD62B has 0 variants, skipping\n",
      "[INFO]\tGene LSM6 has 0 variants, skipping\n",
      "[INFO]\tGene RTL9 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF385D has 0 variants, skipping\n",
      "[INFO]\tGene ADGB has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AW has 0 variants, skipping\n",
      "[INFO]\tGene WNT10A has 0 variants, skipping\n",
      "[INFO]\tGene CCDC68 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6877 has 0 variants, skipping\n",
      "[INFO]\tGene CELP has 0 variants, skipping\n",
      "[INFO]\tGene RTCB has 0 variants, skipping\n",
      "[INFO]\tGene SNORD24 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36B has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36A has 0 variants, skipping\n",
      "[INFO]\tGene SNORD36C has 0 variants, skipping\n",
      "[INFO]\tGene TLX2 has 0 variants, skipping\n",
      "[INFO]\tGene DBH-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130548 has 0 variants, skipping\n",
      "[INFO]\tGene RNU6ATAC has 0 variants, skipping\n",
      "[INFO]\tGene LINC02247 has 0 variants, skipping\n",
      "[INFO]\tGene COL5A1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101448202 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689A has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689D1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689B has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689D2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689E has 0 variants, skipping\n",
      "[INFO]\tGene MIR3689F has 0 variants, skipping\n",
      "[INFO]\tGene LOC401557 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R26-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928525 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01502 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A6 has 0 variants, skipping\n",
      "[INFO]\tGene OR1S2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1N2 has 0 variants, skipping\n",
      "[INFO]\tGene OR1M1 has 0 variants, skipping\n",
      "[INFO]\tGene DKFZP434A062 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf68 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3 has 0 variants, skipping\n",
      "[INFO]\tGene RIC8A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4674 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01451 has 0 variants, skipping\n",
      "[INFO]\tGene HSPC324 has 0 variants, skipping\n",
      "[INFO]\tGene BRI3BP has 0 variants, skipping\n",
      "[INFO]\tGene MBTPS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA17B has 0 variants, skipping\n",
      "[INFO]\tGene SNORA17A has 0 variants, skipping\n",
      "[INFO]\tGene LOC100128593 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6722 has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ9 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM25 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC183-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4292 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF613 has 0 variants, skipping\n",
      "[INFO]\tGene CREG1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4479 has 0 variants, skipping\n",
      "[INFO]\tGene ERGIC3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB8B has 0 variants, skipping\n",
      "[INFO]\tGene MAN1B1-DT has 0 variants, skipping\n",
      "[INFO]\tGene NOMO2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3621 has 0 variants, skipping\n",
      "[INFO]\tGene SNTB2 has 0 variants, skipping\n",
      "[INFO]\tGene STPG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GCSIR has 0 variants, skipping\n",
      "[INFO]\tGene LDC1P has 0 variants, skipping\n",
      "[INFO]\tGene MIR7114 has 0 variants, skipping\n",
      "[INFO]\tGene MEG8 has 0 variants, skipping\n",
      "[INFO]\tGene PRR27 has 0 variants, skipping\n",
      "[INFO]\tGene KRT39 has 0 variants, skipping\n",
      "[INFO]\tGene RNF166 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXC11 has 0 variants, skipping\n",
      "[INFO]\tGene DSPP has 0 variants, skipping\n",
      "[INFO]\tGene TMEM258 has 0 variants, skipping\n",
      "[INFO]\tGene LOC651337 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133077 has 0 variants, skipping\n",
      "[INFO]\tGene ATP6V1C2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376331 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928786 has 0 variants, skipping\n",
      "[INFO]\tGene TUBBP5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928932 has 0 variants, skipping\n",
      "[INFO]\tGene FAM157B has 0 variants, skipping\n",
      "[INFO]\tGene DDX28 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100233156 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389834 has 0 variants, skipping\n",
      "[INFO]\tGene LOC283788 has 0 variants, skipping\n",
      "[INFO]\tGene MIAT has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A8P has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R15B has 0 variants, skipping\n",
      "[INFO]\tGene DIRC3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505874 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT6 has 0 variants, skipping\n",
      "[INFO]\tGene XGY2 has 0 variants, skipping\n",
      "[INFO]\tGene TYRP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01546 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA48B has 0 variants, skipping\n",
      "[INFO]\tGene PRKX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389906 has 0 variants, skipping\n",
      "[INFO]\tGene FAM239B has 0 variants, skipping\n",
      "[INFO]\tGene FAM239A has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928201 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373156 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4770 has 0 variants, skipping\n",
      "[INFO]\tGene ZKSCAN2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR651 has 0 variants, skipping\n",
      "[INFO]\tGene NUP160 has 0 variants, skipping\n",
      "[INFO]\tGene CTXND1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM31 has 0 variants, skipping\n",
      "[INFO]\tGene TPRXL has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AX has 0 variants, skipping\n",
      "[INFO]\tGene TLR8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02154 has 0 variants, skipping\n",
      "[INFO]\tGene GS1-600G8.3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01203 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6086 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK2 has 0 variants, skipping\n",
      "[INFO]\tGene SMURF2 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD3 has 0 variants, skipping\n",
      "[INFO]\tGene NUCKS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD6 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E4P has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB8B has 0 variants, skipping\n",
      "[INFO]\tGene SH2D4B has 0 variants, skipping\n",
      "[INFO]\tGene RRAS has 0 variants, skipping\n",
      "[INFO]\tGene RANBP2 has 0 variants, skipping\n",
      "[INFO]\tGene PIR-FIGF has 0 variants, skipping\n",
      "[INFO]\tGene GS1-594A7.3 has 0 variants, skipping\n",
      "[INFO]\tGene INE2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AM has 0 variants, skipping\n",
      "[INFO]\tGene TMEM151A has 0 variants, skipping\n",
      "[INFO]\tGene MIR4768 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928389 has 0 variants, skipping\n",
      "[INFO]\tGene NHS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5D14 has 0 variants, skipping\n",
      "[INFO]\tGene OR14I1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14K1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01456 has 0 variants, skipping\n",
      "[INFO]\tGene PPEF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCNN1G has 0 variants, skipping\n",
      "[INFO]\tGene PHKA2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CACTIN has 0 variants, skipping\n",
      "[INFO]\tGene LOC729609 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA2 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA9L has 0 variants, skipping\n",
      "[INFO]\tGene EIF1AX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT9 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM25A has 0 variants, skipping\n",
      "[INFO]\tGene PHEX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD1-AS has 0 variants, skipping\n",
      "[INFO]\tGene ZFX-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene JAG1 has 0 variants, skipping\n",
      "[INFO]\tGene PCYT1B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SCARNA23 has 0 variants, skipping\n",
      "[INFO]\tGene VENTXP1 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR6134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4666B has 0 variants, skipping\n",
      "[INFO]\tGene CBWD6 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB5 has 0 variants, skipping\n",
      "[INFO]\tGene PRKACG has 0 variants, skipping\n",
      "[INFO]\tGene TAB3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC729930 has 0 variants, skipping\n",
      "[INFO]\tGene RAB32 has 0 variants, skipping\n",
      "[INFO]\tGene COPS6 has 0 variants, skipping\n",
      "[INFO]\tGene UQCR11 has 0 variants, skipping\n",
      "[INFO]\tGene ADIRF has 0 variants, skipping\n",
      "[INFO]\tGene FRG2C has 0 variants, skipping\n",
      "[INFO]\tGene MIR3915 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548F5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928627 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AJ2 has 0 variants, skipping\n",
      "[INFO]\tGene ASAH2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR15B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1-2 has 0 variants, skipping\n",
      "[INFO]\tGene FHAD1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MID1IP1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01281 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01282 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3937 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1587 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927476 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02601 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2110 has 0 variants, skipping\n",
      "[INFO]\tGene FAM102A has 0 variants, skipping\n",
      "[INFO]\tGene PINCR has 0 variants, skipping\n",
      "[INFO]\tGene MIR643 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01204 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02595 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01186 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF674-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01545 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP7 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK2A3 has 0 variants, skipping\n",
      "[INFO]\tGene SETD3 has 0 variants, skipping\n",
      "[INFO]\tGene BATF has 0 variants, skipping\n",
      "[INFO]\tGene AGR2 has 0 variants, skipping\n",
      "[INFO]\tGene INE1 has 0 variants, skipping\n",
      "[INFO]\tGene PYROXD1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11C has 0 variants, skipping\n",
      "[INFO]\tGene LINC01560 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4769 has 0 variants, skipping\n",
      "[INFO]\tGene UXT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CXXC1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF630-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX6P has 0 variants, skipping\n",
      "[INFO]\tGene VPS4A has 0 variants, skipping\n",
      "[INFO]\tGene SSX9P has 0 variants, skipping\n",
      "[INFO]\tGene GSTA2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927635 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC3 has 0 variants, skipping\n",
      "[INFO]\tGene EIF3H has 0 variants, skipping\n",
      "[INFO]\tGene EIF3G has 0 variants, skipping\n",
      "[INFO]\tGene EIF3F has 0 variants, skipping\n",
      "[INFO]\tGene ERRFI1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC86 has 0 variants, skipping\n",
      "[INFO]\tGene MIR380 has 0 variants, skipping\n",
      "[INFO]\tGene SYP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FLICR has 0 variants, skipping\n",
      "[INFO]\tGene FAM236C has 0 variants, skipping\n",
      "[INFO]\tGene HTD2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM86B1 has 0 variants, skipping\n",
      "[INFO]\tGene USP27X-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TBILA has 0 variants, skipping\n",
      "[INFO]\tGene MIR500B has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL5 has 0 variants, skipping\n",
      "[INFO]\tGene ERICH6B has 0 variants, skipping\n",
      "[INFO]\tGene TMEM31 has 0 variants, skipping\n",
      "[INFO]\tGene TCTE1 has 0 variants, skipping\n",
      "[INFO]\tGene SHROOM4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01496 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2E2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2C2AP has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11D has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11E has 0 variants, skipping\n",
      "[INFO]\tGene MIR8088 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE1B has 0 variants, skipping\n",
      "[INFO]\tGene SSX8P has 0 variants, skipping\n",
      "[INFO]\tGene SDSL has 0 variants, skipping\n",
      "[INFO]\tGene KANTR has 0 variants, skipping\n",
      "[INFO]\tGene PAQR8 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6895 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6894 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6857 has 0 variants, skipping\n",
      "[INFO]\tGene DBP has 0 variants, skipping\n",
      "[INFO]\tGene MORC4 has 0 variants, skipping\n",
      "[INFO]\tGene CDX4 has 0 variants, skipping\n",
      "[INFO]\tGene RAD21 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA11G has 0 variants, skipping\n",
      "[INFO]\tGene METTL3 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC38 has 0 variants, skipping\n",
      "[INFO]\tGene SHE has 0 variants, skipping\n",
      "[INFO]\tGene SNORA109 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100421746 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4536-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4536-2 has 0 variants, skipping\n",
      "[INFO]\tGene ACTR5 has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT2 has 0 variants, skipping\n",
      "[INFO]\tGene GRPEL2 has 0 variants, skipping\n",
      "[INFO]\tGene IQCF1 has 0 variants, skipping\n",
      "[INFO]\tGene UQCRBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RALGAPA1P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01278 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS14 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF9-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene FUBP3 has 0 variants, skipping\n",
      "[INFO]\tGene FRMD8P1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00269 has 0 variants, skipping\n",
      "[INFO]\tGene ARL5A has 0 variants, skipping\n",
      "[INFO]\tGene RFK has 0 variants, skipping\n",
      "[INFO]\tGene DLG3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3E has 0 variants, skipping\n",
      "[INFO]\tGene BEST4 has 0 variants, skipping\n",
      "[INFO]\tGene SNX33 has 0 variants, skipping\n",
      "[INFO]\tGene C20orf96 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL34 has 0 variants, skipping\n",
      "[INFO]\tGene ADH5 has 0 variants, skipping\n",
      "[INFO]\tGene ADH4 has 0 variants, skipping\n",
      "[INFO]\tGene ADH1A has 0 variants, skipping\n",
      "[INFO]\tGene FAM161B has 0 variants, skipping\n",
      "[INFO]\tGene AKAP4 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF12 has 0 variants, skipping\n",
      "[INFO]\tGene CFH has 0 variants, skipping\n",
      "[INFO]\tGene UFSP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51F1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51D1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51B6 has 0 variants, skipping\n",
      "[INFO]\tGene CAVIN4 has 0 variants, skipping\n",
      "[INFO]\tGene INGX has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129291 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132741 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00891 has 0 variants, skipping\n",
      "[INFO]\tGene RPS26P11 has 0 variants, skipping\n",
      "[INFO]\tGene TSPEAR has 0 variants, skipping\n",
      "[INFO]\tGene PHKA1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM226B has 0 variants, skipping\n",
      "[INFO]\tGene FAM226A has 0 variants, skipping\n",
      "[INFO]\tGene P2RX7 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC1L2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K4P1 has 0 variants, skipping\n",
      "[INFO]\tGene UBR1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR374C has 0 variants, skipping\n",
      "[INFO]\tGene APOBR has 0 variants, skipping\n",
      "[INFO]\tGene TTC3P1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR325HG has 0 variants, skipping\n",
      "[INFO]\tGene PNMA6A has 0 variants, skipping\n",
      "[INFO]\tGene LSM10 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF21 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4328 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1B2P has 0 variants, skipping\n",
      "[INFO]\tGene ANKLE2 has 0 variants, skipping\n",
      "[INFO]\tGene ZZEF1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548I4 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2DNL has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928128 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1321 has 0 variants, skipping\n",
      "[INFO]\tGene PLCG2 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548M has 0 variants, skipping\n",
      "[INFO]\tGene BRDTP1 has 0 variants, skipping\n",
      "[INFO]\tGene MSTN has 0 variants, skipping\n",
      "[INFO]\tGene GCHFR has 0 variants, skipping\n",
      "[INFO]\tGene GATA6 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G1B has 0 variants, skipping\n",
      "[INFO]\tGene RPS25 has 0 variants, skipping\n",
      "[INFO]\tGene DNTTIP2 has 0 variants, skipping\n",
      "[INFO]\tGene UBTD1 has 0 variants, skipping\n",
      "[INFO]\tGene VCPIP1 has 0 variants, skipping\n",
      "[INFO]\tGene NAA25 has 0 variants, skipping\n",
      "[INFO]\tGene WWC2 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14 has 0 variants, skipping\n",
      "[INFO]\tGene BAIAP2L2 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD19 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM233 has 0 variants, skipping\n",
      "[INFO]\tGene RPL36A-HNRNPH2 has 0 variants, skipping\n",
      "[INFO]\tGene FOXJ1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC45A3 has 0 variants, skipping\n",
      "[INFO]\tGene HBEGF has 0 variants, skipping\n",
      "[INFO]\tGene NXF5 has 0 variants, skipping\n",
      "[INFO]\tGene SLC6A1 has 0 variants, skipping\n",
      "[INFO]\tGene NXF4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00630 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02589 has 0 variants, skipping\n",
      "[INFO]\tGene GPIHBP1 has 0 variants, skipping\n",
      "[INFO]\tGene HSD17B6 has 0 variants, skipping\n",
      "[INFO]\tGene MORF4L2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB11 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM162A has 0 variants, skipping\n",
      "[INFO]\tGene LOC286437 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA3 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548AN has 0 variants, skipping\n",
      "[INFO]\tGene NT5M has 0 variants, skipping\n",
      "[INFO]\tGene ZNF620 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MPHOSPH10 has 0 variants, skipping\n",
      "[INFO]\tGene EIF1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928335 has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1B has 0 variants, skipping\n",
      "[INFO]\tGene FAU has 0 variants, skipping\n",
      "[INFO]\tGene GTF3A has 0 variants, skipping\n",
      "[INFO]\tGene GTF2F1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AZ1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AX has 0 variants, skipping\n",
      "[INFO]\tGene LINC02864 has 0 variants, skipping\n",
      "[INFO]\tGene BCAR3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1303 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1226 has 0 variants, skipping\n",
      "[INFO]\tGene FAM180A has 0 variants, skipping\n",
      "[INFO]\tGene MIR764 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1283-1 has 0 variants, skipping\n",
      "[INFO]\tGene GJB7 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf34 has 0 variants, skipping\n",
      "[INFO]\tGene NRROS has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928358 has 0 variants, skipping\n",
      "[INFO]\tGene PCNAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD96B has 0 variants, skipping\n",
      "[INFO]\tGene TDGF1P3 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL3 has 0 variants, skipping\n",
      "[INFO]\tGene CYBB has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA6A has 0 variants, skipping\n",
      "[INFO]\tGene MYSM1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC441155 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF154 has 0 variants, skipping\n",
      "[INFO]\tGene PSAPL1 has 0 variants, skipping\n",
      "[INFO]\tGene ACVRL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4329 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928437 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA35 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1264 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1911 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA35B has 0 variants, skipping\n",
      "[INFO]\tGene PLS3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DANT2 has 0 variants, skipping\n",
      "[INFO]\tGene DANT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100126447 has 0 variants, skipping\n",
      "[INFO]\tGene IL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01285 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928336 has 0 variants, skipping\n",
      "[INFO]\tGene GPC1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXN2 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA69 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1P1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01402 has 0 variants, skipping\n",
      "[INFO]\tGene NKAPP1 has 0 variants, skipping\n",
      "[INFO]\tGene C3 has 0 variants, skipping\n",
      "[INFO]\tGene STMN2 has 0 variants, skipping\n",
      "[INFO]\tGene SKP1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM31 has 0 variants, skipping\n",
      "[INFO]\tGene IVL has 0 variants, skipping\n",
      "[INFO]\tGene KCNA10 has 0 variants, skipping\n",
      "[INFO]\tGene PRKX has 0 variants, skipping\n",
      "[INFO]\tGene DMWD has 0 variants, skipping\n",
      "[INFO]\tGene DEFB4A has 0 variants, skipping\n",
      "[INFO]\tGene SYCP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3672 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928402 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928495 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM74 has 0 variants, skipping\n",
      "[INFO]\tGene PRAM1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBD2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf17 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373338 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01201 has 0 variants, skipping\n",
      "[INFO]\tGene ITPR2 has 0 variants, skipping\n",
      "[INFO]\tGene GRM6 has 0 variants, skipping\n",
      "[INFO]\tGene RAP2C-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HGFAC has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC5 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5R2 has 0 variants, skipping\n",
      "[INFO]\tGene BPIFC has 0 variants, skipping\n",
      "[INFO]\tGene LINC00629 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN32 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK15 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL3 has 0 variants, skipping\n",
      "[INFO]\tGene F13B has 0 variants, skipping\n",
      "[INFO]\tGene GZMK has 0 variants, skipping\n",
      "[INFO]\tGene TBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105370362 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02243 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM191C has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100506790 has 0 variants, skipping\n",
      "[INFO]\tGene INTS6L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF103-CHMP3 has 0 variants, skipping\n",
      "[INFO]\tGene SAGE2P has 0 variants, skipping\n",
      "[INFO]\tGene SSR2 has 0 variants, skipping\n",
      "[INFO]\tGene USP29 has 0 variants, skipping\n",
      "[INFO]\tGene RFTN1 has 0 variants, skipping\n",
      "[INFO]\tGene SMCO3 has 0 variants, skipping\n",
      "[INFO]\tGene PLIN5 has 0 variants, skipping\n",
      "[INFO]\tGene SRRD has 0 variants, skipping\n",
      "[INFO]\tGene LINC00892 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC8 has 0 variants, skipping\n",
      "[INFO]\tGene MYOZ1 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD61 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A1P1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF592 has 0 variants, skipping\n",
      "[INFO]\tGene S100A6 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD4 has 0 variants, skipping\n",
      "[INFO]\tGene BRINP1 has 0 variants, skipping\n",
      "[INFO]\tGene CRYGA has 0 variants, skipping\n",
      "[INFO]\tGene ATP11B has 0 variants, skipping\n",
      "[INFO]\tGene LOC728660 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00632 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXA2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR892C has 0 variants, skipping\n",
      "[INFO]\tGene MIR892A has 0 variants, skipping\n",
      "[INFO]\tGene MIR891A has 0 variants, skipping\n",
      "[INFO]\tGene PRDX2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR517C has 0 variants, skipping\n",
      "[INFO]\tGene MIR509-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514A3 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB4X has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG3 has 0 variants, skipping\n",
      "[INFO]\tGene MST1 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL5 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00893 has 0 variants, skipping\n",
      "[INFO]\tGene NFKBIA has 0 variants, skipping\n",
      "[INFO]\tGene LINC00850 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCOLN1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00894 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2114 has 0 variants, skipping\n",
      "[INFO]\tGene CPNE8 has 0 variants, skipping\n",
      "[INFO]\tGene EPPIN has 0 variants, skipping\n",
      "[INFO]\tGene MIR4330 has 0 variants, skipping\n",
      "[INFO]\tGene GPR50-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377213 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA9 has 0 variants, skipping\n",
      "[INFO]\tGene IL17F has 0 variants, skipping\n",
      "[INFO]\tGene CSAG4 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf49 has 0 variants, skipping\n",
      "[INFO]\tGene TINAG has 0 variants, skipping\n",
      "[INFO]\tGene LSM3 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM29 has 0 variants, skipping\n",
      "[INFO]\tGene PROB1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373378 has 0 variants, skipping\n",
      "[INFO]\tGene USP10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105373383 has 0 variants, skipping\n",
      "[INFO]\tGene LCA10 has 0 variants, skipping\n",
      "[INFO]\tGene KDM4E has 0 variants, skipping\n",
      "[INFO]\tGene HCFC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3202-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3202-2 has 0 variants, skipping\n",
      "[INFO]\tGene BOLL has 0 variants, skipping\n",
      "[INFO]\tGene OR2AG2 has 0 variants, skipping\n",
      "[INFO]\tGene BSX has 0 variants, skipping\n",
      "[INFO]\tGene SNORA70 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D2B has 0 variants, skipping\n",
      "[INFO]\tGene CH17-340M24.3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6858 has 0 variants, skipping\n",
      "[INFO]\tGene RFPL4AL1 has 0 variants, skipping\n",
      "[INFO]\tGene POTEI has 0 variants, skipping\n",
      "[INFO]\tGene FAM223A has 0 variants, skipping\n",
      "[INFO]\tGene FAM223B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF354C has 0 variants, skipping\n",
      "[INFO]\tGene CXXC1 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS3 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA36A has 0 variants, skipping\n",
      "[INFO]\tGene SNORA56 has 0 variants, skipping\n",
      "[INFO]\tGene CDK5R1 has 0 variants, skipping\n",
      "[INFO]\tGene KAT2B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1184-3 has 0 variants, skipping\n",
      "[INFO]\tGene TMLHE-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927830 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100129697 has 0 variants, skipping\n",
      "[INFO]\tGene CYRIA has 0 variants, skipping\n",
      "[INFO]\tGene TTC37 has 0 variants, skipping\n",
      "[INFO]\tGene MIR545 has 0 variants, skipping\n",
      "[INFO]\tGene MIR539 has 0 variants, skipping\n",
      "[INFO]\tGene ZFY-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00278 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY23 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY23B has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y9 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00280 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY1 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY1B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY2 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY2B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY21 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY21B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY7 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY7B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY8 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY8B has 0 variants, skipping\n",
      "[INFO]\tGene PRKY has 0 variants, skipping\n",
      "[INFO]\tGene TTTY16 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY12 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00279 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY18 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY19 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY11 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY20 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y8 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y6 has 0 variants, skipping\n",
      "[INFO]\tGene UBR2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y3 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY22 has 0 variants, skipping\n",
      "[INFO]\tGene GYG2P1 has 0 variants, skipping\n",
      "[INFO]\tGene OMG has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene THRB-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AU1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4X2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4X1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN12 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf32 has 0 variants, skipping\n",
      "[INFO]\tGene GALNTL6 has 0 variants, skipping\n",
      "[INFO]\tGene CYP1B1 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF33 has 0 variants, skipping\n",
      "[INFO]\tGene BABAM2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02882 has 0 variants, skipping\n",
      "[INFO]\tGene GIHCG has 0 variants, skipping\n",
      "[INFO]\tGene FBXO30-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00858 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00311 has 0 variants, skipping\n",
      "[INFO]\tGene PURPL has 0 variants, skipping\n",
      "[INFO]\tGene LINC01426 has 0 variants, skipping\n",
      "[INFO]\tGene ACTE1P has 0 variants, skipping\n",
      "[INFO]\tGene MIR550A3 has 0 variants, skipping\n",
      "[INFO]\tGene ZEB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM30A-DT has 0 variants, skipping\n",
      "[INFO]\tGene RAD51-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00963 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4454 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4421 has 0 variants, skipping\n",
      "[INFO]\tGene DCST1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4766 has 0 variants, skipping\n",
      "[INFO]\tGene MIR219B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4658 has 0 variants, skipping\n",
      "[INFO]\tGene MIR638 has 0 variants, skipping\n",
      "[INFO]\tGene MIR608 has 0 variants, skipping\n",
      "[INFO]\tGene MIR607 has 0 variants, skipping\n",
      "[INFO]\tGene MIR599 has 0 variants, skipping\n",
      "[INFO]\tGene MIR597 has 0 variants, skipping\n",
      "[INFO]\tGene FCGR1B has 0 variants, skipping\n",
      "[INFO]\tGene MIR592 has 0 variants, skipping\n",
      "[INFO]\tGene MIR18A has 0 variants, skipping\n",
      "[INFO]\tGene MIR770 has 0 variants, skipping\n",
      "[INFO]\tGene MIR541 has 0 variants, skipping\n",
      "[INFO]\tGene MIR888 has 0 variants, skipping\n",
      "[INFO]\tGene MIR198 has 0 variants, skipping\n",
      "[INFO]\tGene MIR877 has 0 variants, skipping\n",
      "[INFO]\tGene MIR127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR154 has 0 variants, skipping\n",
      "[INFO]\tGene TGFB2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN4Y-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM41AY1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM41AY2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM224B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY9A has 0 variants, skipping\n",
      "[INFO]\tGene TTTY9B has 0 variants, skipping\n",
      "[INFO]\tGene OTULINL has 0 variants, skipping\n",
      "[INFO]\tGene BCORP1 has 0 variants, skipping\n",
      "[INFO]\tGene TXLNGY has 0 variants, skipping\n",
      "[INFO]\tGene TTTY10 has 0 variants, skipping\n",
      "[INFO]\tGene SKI has 0 variants, skipping\n",
      "[INFO]\tGene TTTY13 has 0 variants, skipping\n",
      "[INFO]\tGene FBN3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929148 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY6 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY6B has 0 variants, skipping\n",
      "[INFO]\tGene LORICRIN has 0 variants, skipping\n",
      "[INFO]\tGene TTTY5 has 0 variants, skipping\n",
      "[INFO]\tGene MAST1 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17A has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17C has 0 variants, skipping\n",
      "[INFO]\tGene TTTY17B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4C has 0 variants, skipping\n",
      "[INFO]\tGene TTTY4B has 0 variants, skipping\n",
      "[INFO]\tGene TTTY3 has 0 variants, skipping\n",
      "[INFO]\tGene TTTY3B has 0 variants, skipping\n",
      "[INFO]\tGene CSPG4P1Y has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P2Y has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P3Y has 0 variants, skipping\n",
      "[INFO]\tGene AARS2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P2-AGAP9 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1P4-AGAP5 has 0 variants, skipping\n",
      "[INFO]\tGene TTC28-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG20 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00205 has 0 variants, skipping\n",
      "[INFO]\tGene CASC2 has 0 variants, skipping\n",
      "[INFO]\tGene MBNL1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01554 has 0 variants, skipping\n",
      "[INFO]\tGene AGAP2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TARDBPP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02809 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLECL1 has 0 variants, skipping\n",
      "[INFO]\tGene MFFP3 has 0 variants, skipping\n",
      "[INFO]\tGene PAFAH1B2P2 has 0 variants, skipping\n",
      "[INFO]\tGene NR2F1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene WDR86-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCAT2 has 0 variants, skipping\n",
      "[INFO]\tGene PRNCR1 has 0 variants, skipping\n",
      "[INFO]\tGene PLGLB1 has 0 variants, skipping\n",
      "[INFO]\tGene ACP3 has 0 variants, skipping\n",
      "[INFO]\tGene NEU2 has 0 variants, skipping\n",
      "[INFO]\tGene NCL has 0 variants, skipping\n",
      "[INFO]\tGene CLDN34 has 0 variants, skipping\n",
      "[INFO]\tGene C19orf81 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996720 has 0 variants, skipping\n",
      "[INFO]\tGene RAB3B has 0 variants, skipping\n",
      "[INFO]\tGene RARRES2 has 0 variants, skipping\n",
      "[INFO]\tGene RASA1 has 0 variants, skipping\n",
      "[INFO]\tGene BTBD18 has 0 variants, skipping\n",
      "[INFO]\tGene NWD2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986453 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK2 has 0 variants, skipping\n",
      "[INFO]\tGene REXO2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM155B has 0 variants, skipping\n",
      "[INFO]\tGene AGAP10P has 0 variants, skipping\n",
      "[INFO]\tGene PARGP1-AGAP4 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPM has 0 variants, skipping\n",
      "[INFO]\tGene TMEM175 has 0 variants, skipping\n",
      "[INFO]\tGene HTRA3 has 0 variants, skipping\n",
      "[INFO]\tGene MAP2K7 has 0 variants, skipping\n",
      "[INFO]\tGene GRIP2 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf38 has 0 variants, skipping\n",
      "[INFO]\tGene APELA has 0 variants, skipping\n",
      "[INFO]\tGene UPF1 has 0 variants, skipping\n",
      "[INFO]\tGene FALEC has 0 variants, skipping\n",
      "[INFO]\tGene PAGE2B has 0 variants, skipping\n",
      "[INFO]\tGene EFNA3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369199 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371215 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1G has 0 variants, skipping\n",
      "[INFO]\tGene ZNF286B has 0 variants, skipping\n",
      "[INFO]\tGene SENP1 has 0 variants, skipping\n",
      "[INFO]\tGene MAPRE3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CA5BP1 has 0 variants, skipping\n",
      "[INFO]\tGene SYNE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724020 has 0 variants, skipping\n",
      "[INFO]\tGene FER1L4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02883 has 0 variants, skipping\n",
      "[INFO]\tGene CASC19 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927789 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984208 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369209 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS45P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS43P has 0 variants, skipping\n",
      "[INFO]\tGene PRSS44P has 0 variants, skipping\n",
      "[INFO]\tGene BLACAT1 has 0 variants, skipping\n",
      "[INFO]\tGene OR13C2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR206 has 0 variants, skipping\n",
      "[INFO]\tGene MIR629 has 0 variants, skipping\n",
      "[INFO]\tGene SIRLNT has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1P4 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf95 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927245 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105379443 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724904 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996643 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927824 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102723534 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369201 has 0 variants, skipping\n",
      "[INFO]\tGene SLC24A5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339260 has 0 variants, skipping\n",
      "[INFO]\tGene FGD6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM214 has 0 variants, skipping\n",
      "[INFO]\tGene CALML6 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM64C has 0 variants, skipping\n",
      "[INFO]\tGene YOD1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPA3 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724378 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378577 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984236 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984265 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371031 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376917 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100996413 has 0 variants, skipping\n",
      "[INFO]\tGene POF1B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374378 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378230 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377777 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985184 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985246 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984341 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107984507 has 0 variants, skipping\n",
      "[INFO]\tGene SLC44A3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985911 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986163 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986876 has 0 variants, skipping\n",
      "[INFO]\tGene ABTB1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP38-DT has 0 variants, skipping\n",
      "[INFO]\tGene C17orf102 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1915HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02688 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02783 has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB44-DT has 0 variants, skipping\n",
      "[INFO]\tGene GTSCR1 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf46 has 0 variants, skipping\n",
      "[INFO]\tGene PDSS2 has 0 variants, skipping\n",
      "[INFO]\tGene NAT14 has 0 variants, skipping\n",
      "[INFO]\tGene ATAD3A has 0 variants, skipping\n",
      "[INFO]\tGene ARL8B has 0 variants, skipping\n",
      "[INFO]\tGene NPPC has 0 variants, skipping\n",
      "[INFO]\tGene ODF1 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2 has 0 variants, skipping\n",
      "[INFO]\tGene ABCC1 has 0 variants, skipping\n",
      "[INFO]\tGene WBP11 has 0 variants, skipping\n",
      "[INFO]\tGene TUBGCP6 has 0 variants, skipping\n",
      "[INFO]\tGene PAIP2B has 0 variants, skipping\n",
      "[INFO]\tGene SNHG31 has 0 variants, skipping\n",
      "[INFO]\tGene KDM3B has 0 variants, skipping\n",
      "[INFO]\tGene N4BP3 has 0 variants, skipping\n",
      "[INFO]\tGene ZFAND2A-DT has 0 variants, skipping\n",
      "[INFO]\tGene PDCD1LG2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM99 has 0 variants, skipping\n",
      "[INFO]\tGene DOCK8-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02693 has 0 variants, skipping\n",
      "[INFO]\tGene GAB2 has 0 variants, skipping\n",
      "[INFO]\tGene EEFSEC has 0 variants, skipping\n",
      "[INFO]\tGene ATG101 has 0 variants, skipping\n",
      "[INFO]\tGene MIIP has 0 variants, skipping\n",
      "[INFO]\tGene SPCS3 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A1 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD12 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD11 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD4 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf223 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112543491 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL46 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRL4 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTA1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP25 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR935 has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC32A1 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA2P11 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF8L2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K20 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF287 has 0 variants, skipping\n",
      "[INFO]\tGene TM6SF2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SAXO1 has 0 variants, skipping\n",
      "[INFO]\tGene GRID2IP has 0 variants, skipping\n",
      "[INFO]\tGene PFN3 has 0 variants, skipping\n",
      "[INFO]\tGene ESPL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTDSS1 has 0 variants, skipping\n",
      "[INFO]\tGene STYXL2 has 0 variants, skipping\n",
      "[INFO]\tGene NOX3 has 0 variants, skipping\n",
      "[INFO]\tGene CD207 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G3 has 0 variants, skipping\n",
      "[INFO]\tGene G0S2 has 0 variants, skipping\n",
      "[INFO]\tGene BICRA has 0 variants, skipping\n",
      "[INFO]\tGene NOP53 has 0 variants, skipping\n",
      "[INFO]\tGene GPKOW has 0 variants, skipping\n",
      "[INFO]\tGene GPX7 has 0 variants, skipping\n",
      "[INFO]\tGene BRPF3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF777 has 0 variants, skipping\n",
      "[INFO]\tGene DESI1 has 0 variants, skipping\n",
      "[INFO]\tGene RUNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene PDX1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNRG has 0 variants, skipping\n",
      "[INFO]\tGene ZNF335 has 0 variants, skipping\n",
      "[INFO]\tGene PCIF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF277 has 0 variants, skipping\n",
      "[INFO]\tGene TNN has 0 variants, skipping\n",
      "[INFO]\tGene CHTF18 has 0 variants, skipping\n",
      "[INFO]\tGene TMSB15A has 0 variants, skipping\n",
      "[INFO]\tGene ZBED8 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRB4 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA2 has 0 variants, skipping\n",
      "[INFO]\tGene HSP90B1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPYL1 has 0 variants, skipping\n",
      "[INFO]\tGene HNRNPCL1 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12L2 has 0 variants, skipping\n",
      "[INFO]\tGene NHSL2 has 0 variants, skipping\n",
      "[INFO]\tGene NEXMIF has 0 variants, skipping\n",
      "[INFO]\tGene MILR1 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G6C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF674 has 0 variants, skipping\n",
      "[INFO]\tGene KATNIP has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1D has 0 variants, skipping\n",
      "[INFO]\tGene PRM1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR32 has 0 variants, skipping\n",
      "[INFO]\tGene CDCA3 has 0 variants, skipping\n",
      "[INFO]\tGene TSPY8 has 0 variants, skipping\n",
      "[INFO]\tGene FAF1 has 0 variants, skipping\n",
      "[INFO]\tGene PYY has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC17 has 0 variants, skipping\n",
      "[INFO]\tGene PABPN1 has 0 variants, skipping\n",
      "[INFO]\tGene SH2B2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP9 has 0 variants, skipping\n",
      "[INFO]\tGene KREMEN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS4Y2 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL35 has 0 variants, skipping\n",
      "[INFO]\tGene KATNAL1 has 0 variants, skipping\n",
      "[INFO]\tGene PDHA2 has 0 variants, skipping\n",
      "[INFO]\tGene PRKCE has 0 variants, skipping\n",
      "[INFO]\tGene PDCL has 0 variants, skipping\n",
      "[INFO]\tGene MLF2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF319 has 0 variants, skipping\n",
      "[INFO]\tGene H1-9P has 0 variants, skipping\n",
      "[INFO]\tGene CDC14C has 0 variants, skipping\n",
      "[INFO]\tGene ZNF646 has 0 variants, skipping\n",
      "[INFO]\tGene USP34 has 0 variants, skipping\n",
      "[INFO]\tGene HTR1A has 0 variants, skipping\n",
      "[INFO]\tGene HOXD13 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA13 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB4 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB3 has 0 variants, skipping\n",
      "[INFO]\tGene EPHB1 has 0 variants, skipping\n",
      "[INFO]\tGene CTDNEP1 has 0 variants, skipping\n",
      "[INFO]\tGene PHYHIPL has 0 variants, skipping\n",
      "[INFO]\tGene THOC2 has 0 variants, skipping\n",
      "[INFO]\tGene ELL has 0 variants, skipping\n",
      "[INFO]\tGene CFHR5 has 0 variants, skipping\n",
      "[INFO]\tGene ACER1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC114841035 has 0 variants, skipping\n",
      "[INFO]\tGene HOGA1 has 0 variants, skipping\n",
      "[INFO]\tGene PRR23A has 0 variants, skipping\n",
      "[INFO]\tGene INSYN1 has 0 variants, skipping\n",
      "[INFO]\tGene ACAT2 has 0 variants, skipping\n",
      "[INFO]\tGene TRMT61A has 0 variants, skipping\n",
      "[INFO]\tGene ZASP has 0 variants, skipping\n",
      "[INFO]\tGene FCN1 has 0 variants, skipping\n",
      "[INFO]\tGene TYK2 has 0 variants, skipping\n",
      "[INFO]\tGene LY6G5B has 0 variants, skipping\n",
      "[INFO]\tGene RBM8A has 0 variants, skipping\n",
      "[INFO]\tGene FBLN5 has 0 variants, skipping\n",
      "[INFO]\tGene CYLC2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA2 has 0 variants, skipping\n",
      "[INFO]\tGene HIVEP2 has 0 variants, skipping\n",
      "[INFO]\tGene TNS3 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D1 has 0 variants, skipping\n",
      "[INFO]\tGene STXBP3 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CTRC has 0 variants, skipping\n",
      "[INFO]\tGene RIBC2 has 0 variants, skipping\n",
      "[INFO]\tGene IRF2BP1 has 0 variants, skipping\n",
      "[INFO]\tGene HTATSF1P2 has 0 variants, skipping\n",
      "[INFO]\tGene PTGFRN has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD13 has 0 variants, skipping\n",
      "[INFO]\tGene UBAP1L has 0 variants, skipping\n",
      "[INFO]\tGene CT62 has 0 variants, skipping\n",
      "[INFO]\tGene MGAM2 has 0 variants, skipping\n",
      "[INFO]\tGene OST4 has 0 variants, skipping\n",
      "[INFO]\tGene IL24 has 0 variants, skipping\n",
      "[INFO]\tGene MAST3 has 0 variants, skipping\n",
      "[INFO]\tGene SLC8A2 has 0 variants, skipping\n",
      "[INFO]\tGene METTL2B has 0 variants, skipping\n",
      "[INFO]\tGene PTRHD1 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11L1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD24 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF721 has 0 variants, skipping\n",
      "[INFO]\tGene CITED4 has 0 variants, skipping\n",
      "[INFO]\tGene SGCD has 0 variants, skipping\n",
      "[INFO]\tGene SPRED2 has 0 variants, skipping\n",
      "[INFO]\tGene LEPROTL1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L6 has 0 variants, skipping\n",
      "[INFO]\tGene FCSK has 0 variants, skipping\n",
      "[INFO]\tGene RSL1D1 has 0 variants, skipping\n",
      "[INFO]\tGene GIMAP2 has 0 variants, skipping\n",
      "[INFO]\tGene NOC2L has 0 variants, skipping\n",
      "[INFO]\tGene UBE2E2 has 0 variants, skipping\n",
      "[INFO]\tGene RAB36 has 0 variants, skipping\n",
      "[INFO]\tGene RAB40A has 0 variants, skipping\n",
      "[INFO]\tGene ITLN2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB5 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD4 has 0 variants, skipping\n",
      "[INFO]\tGene FBXO41 has 0 variants, skipping\n",
      "[INFO]\tGene GLB1L3 has 0 variants, skipping\n",
      "[INFO]\tGene HEPHL1 has 0 variants, skipping\n",
      "[INFO]\tGene KLF12 has 0 variants, skipping\n",
      "[INFO]\tGene CLVS1 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf62 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S5 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP16-1 has 0 variants, skipping\n",
      "[INFO]\tGene LNCTAM34A has 0 variants, skipping\n",
      "[INFO]\tGene MESP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF555 has 0 variants, skipping\n",
      "[INFO]\tGene NACC2 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP20DC has 0 variants, skipping\n",
      "[INFO]\tGene MCM4 has 0 variants, skipping\n",
      "[INFO]\tGene HAND1 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP1L has 0 variants, skipping\n",
      "[INFO]\tGene TRPM5 has 0 variants, skipping\n",
      "[INFO]\tGene TFCP2L1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC167 has 0 variants, skipping\n",
      "[INFO]\tGene FLRT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985524 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNV2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC30A7 has 0 variants, skipping\n",
      "[INFO]\tGene DRGX has 0 variants, skipping\n",
      "[INFO]\tGene RNF227 has 0 variants, skipping\n",
      "[INFO]\tGene TVP23C has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R11 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD4 has 0 variants, skipping\n",
      "[INFO]\tGene HLX has 0 variants, skipping\n",
      "[INFO]\tGene GYS2 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK12 has 0 variants, skipping\n",
      "[INFO]\tGene GJA3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK13 has 0 variants, skipping\n",
      "[INFO]\tGene CGB2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC19 has 0 variants, skipping\n",
      "[INFO]\tGene MOCOS has 0 variants, skipping\n",
      "[INFO]\tGene OR5H6 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C74 has 0 variants, skipping\n",
      "[INFO]\tGene REG4 has 0 variants, skipping\n",
      "[INFO]\tGene BPI has 0 variants, skipping\n",
      "[INFO]\tGene RPL17-C18orf32 has 0 variants, skipping\n",
      "[INFO]\tGene FIZ1 has 0 variants, skipping\n",
      "[INFO]\tGene ALG10 has 0 variants, skipping\n",
      "[INFO]\tGene NGB has 0 variants, skipping\n",
      "[INFO]\tGene PELI2 has 0 variants, skipping\n",
      "[INFO]\tGene APOA5 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB106B has 0 variants, skipping\n",
      "[INFO]\tGene ZNF77 has 0 variants, skipping\n",
      "[INFO]\tGene CENPT has 0 variants, skipping\n",
      "[INFO]\tGene NYNRIN has 0 variants, skipping\n",
      "[INFO]\tGene GPR157 has 0 variants, skipping\n",
      "[INFO]\tGene TOE1 has 0 variants, skipping\n",
      "[INFO]\tGene CCNB3 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL41 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS6 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM132D has 0 variants, skipping\n",
      "[INFO]\tGene TEN1 has 0 variants, skipping\n",
      "[INFO]\tGene SCAI has 0 variants, skipping\n",
      "[INFO]\tGene TAMALIN has 0 variants, skipping\n",
      "[INFO]\tGene C10orf95 has 0 variants, skipping\n",
      "[INFO]\tGene PLA2G12A has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13C has 0 variants, skipping\n",
      "[INFO]\tGene PDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5H1 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC12 has 0 variants, skipping\n",
      "[INFO]\tGene CAPZA3 has 0 variants, skipping\n",
      "[INFO]\tGene RASL10B has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT9 has 0 variants, skipping\n",
      "[INFO]\tGene CASP4 has 0 variants, skipping\n",
      "[INFO]\tGene GRIN3A has 0 variants, skipping\n",
      "[INFO]\tGene AGAP11 has 0 variants, skipping\n",
      "[INFO]\tGene PCDH20 has 0 variants, skipping\n",
      "[INFO]\tGene CDCP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWA1 has 0 variants, skipping\n",
      "[INFO]\tGene AIDA has 0 variants, skipping\n",
      "[INFO]\tGene KIF26A has 0 variants, skipping\n",
      "[INFO]\tGene OR56A5 has 0 variants, skipping\n",
      "[INFO]\tGene POLR2C has 0 variants, skipping\n",
      "[INFO]\tGene GAL3ST3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM60 has 0 variants, skipping\n",
      "[INFO]\tGene MT4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT82 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP2-2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM141 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINE1 has 0 variants, skipping\n",
      "[INFO]\tGene IL9 has 0 variants, skipping\n",
      "[INFO]\tGene IL3 has 0 variants, skipping\n",
      "[INFO]\tGene C7 has 0 variants, skipping\n",
      "[INFO]\tGene IL2 has 0 variants, skipping\n",
      "[INFO]\tGene IL15 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL8 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM263 has 0 variants, skipping\n",
      "[INFO]\tGene CYHR1 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf107 has 0 variants, skipping\n",
      "[INFO]\tGene TTBK1 has 0 variants, skipping\n",
      "[INFO]\tGene NTNG2 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF594 has 0 variants, skipping\n",
      "[INFO]\tGene TCEAL7 has 0 variants, skipping\n",
      "[INFO]\tGene OR52Z1 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2B has 0 variants, skipping\n",
      "[INFO]\tGene NSD3 has 0 variants, skipping\n",
      "[INFO]\tGene LRP11 has 0 variants, skipping\n",
      "[INFO]\tGene FAM222A has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE2 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS41 has 0 variants, skipping\n",
      "[INFO]\tGene IK has 0 variants, skipping\n",
      "[INFO]\tGene TSR3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987394 has 0 variants, skipping\n",
      "[INFO]\tGene RAB33B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HEAT2 has 0 variants, skipping\n",
      "[INFO]\tGene NIT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAP6D1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM110D has 0 variants, skipping\n",
      "[INFO]\tGene OR5B3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4B1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G3 has 0 variants, skipping\n",
      "[INFO]\tGene H3C15 has 0 variants, skipping\n",
      "[INFO]\tGene RNF141 has 0 variants, skipping\n",
      "[INFO]\tGene LRP1B has 0 variants, skipping\n",
      "[INFO]\tGene GSDMC has 0 variants, skipping\n",
      "[INFO]\tGene INA has 0 variants, skipping\n",
      "[INFO]\tGene C6orf99 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD1 has 0 variants, skipping\n",
      "[INFO]\tGene FMNL3 has 0 variants, skipping\n",
      "[INFO]\tGene CELA3A has 0 variants, skipping\n",
      "[INFO]\tGene NAMPT has 0 variants, skipping\n",
      "[INFO]\tGene INTS5 has 0 variants, skipping\n",
      "[INFO]\tGene TET1 has 0 variants, skipping\n",
      "[INFO]\tGene ITGB3 has 0 variants, skipping\n",
      "[INFO]\tGene TBX5 has 0 variants, skipping\n",
      "[INFO]\tGene ICAM1 has 0 variants, skipping\n",
      "[INFO]\tGene HK2 has 0 variants, skipping\n",
      "[INFO]\tGene HTN3 has 0 variants, skipping\n",
      "[INFO]\tGene HGD has 0 variants, skipping\n",
      "[INFO]\tGene BCL2 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB12 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC10 has 0 variants, skipping\n",
      "[INFO]\tGene FBXL17 has 0 variants, skipping\n",
      "[INFO]\tGene RPS18 has 0 variants, skipping\n",
      "[INFO]\tGene SUDS3 has 0 variants, skipping\n",
      "[INFO]\tGene FSTL4 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130449 has 0 variants, skipping\n",
      "[INFO]\tGene XKR6 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02054 has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2LY has 0 variants, skipping\n",
      "[INFO]\tGene LINC02860 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf44 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf66 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928093 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK1C has 0 variants, skipping\n",
      "[INFO]\tGene B3GALT6 has 0 variants, skipping\n",
      "[INFO]\tGene ZSWIM1 has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS7 has 0 variants, skipping\n",
      "[INFO]\tGene UBTFL1 has 0 variants, skipping\n",
      "[INFO]\tGene NFATC2IP has 0 variants, skipping\n",
      "[INFO]\tGene DDX24 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP1B has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL47 has 0 variants, skipping\n",
      "[INFO]\tGene COL27A1 has 0 variants, skipping\n",
      "[INFO]\tGene NPIPB8 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF14 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEE2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEC3 has 0 variants, skipping\n",
      "[INFO]\tGene MPLKIP has 0 variants, skipping\n",
      "[INFO]\tGene TMC1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R14B has 0 variants, skipping\n",
      "[INFO]\tGene GRIN3B has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13B has 0 variants, skipping\n",
      "[INFO]\tGene SH2D1B has 0 variants, skipping\n",
      "[INFO]\tGene FAM167A has 0 variants, skipping\n",
      "[INFO]\tGene TAAR8 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD30A has 0 variants, skipping\n",
      "[INFO]\tGene RPRD1B has 0 variants, skipping\n",
      "[INFO]\tGene ABHD17C has 0 variants, skipping\n",
      "[INFO]\tGene CLDN22 has 0 variants, skipping\n",
      "[INFO]\tGene SHANK3 has 0 variants, skipping\n",
      "[INFO]\tGene METTL14 has 0 variants, skipping\n",
      "[INFO]\tGene MED12L has 0 variants, skipping\n",
      "[INFO]\tGene EFHC2 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH2 has 0 variants, skipping\n",
      "[INFO]\tGene CD248 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A28 has 0 variants, skipping\n",
      "[INFO]\tGene QTRT1 has 0 variants, skipping\n",
      "[INFO]\tGene TFPI2 has 0 variants, skipping\n",
      "[INFO]\tGene ARAP3 has 0 variants, skipping\n",
      "[INFO]\tGene C1orf21 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF672 has 0 variants, skipping\n",
      "[INFO]\tGene THNSL1 has 0 variants, skipping\n",
      "[INFO]\tGene RTN4R has 0 variants, skipping\n",
      "[INFO]\tGene RTL10 has 0 variants, skipping\n",
      "[INFO]\tGene TMX1 has 0 variants, skipping\n",
      "[INFO]\tGene WNT3 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC177 has 0 variants, skipping\n",
      "[INFO]\tGene OR6K3 has 0 variants, skipping\n",
      "[INFO]\tGene FNDC4 has 0 variants, skipping\n",
      "[INFO]\tGene MTLN has 0 variants, skipping\n",
      "[INFO]\tGene MFSD14A has 0 variants, skipping\n",
      "[INFO]\tGene CCL15 has 0 variants, skipping\n",
      "[INFO]\tGene ACP4 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTB1 has 0 variants, skipping\n",
      "[INFO]\tGene MPP4 has 0 variants, skipping\n",
      "[INFO]\tGene OR2B2 has 0 variants, skipping\n",
      "[INFO]\tGene SUCNR1 has 0 variants, skipping\n",
      "[INFO]\tGene MUC13 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT84 has 0 variants, skipping\n",
      "[INFO]\tGene PYURF has 0 variants, skipping\n",
      "[INFO]\tGene BSND has 0 variants, skipping\n",
      "[INFO]\tGene NAA15 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2J2 has 0 variants, skipping\n",
      "[INFO]\tGene MOGAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LCMT2 has 0 variants, skipping\n",
      "[INFO]\tGene NMT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPATC1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINK7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-4 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6B has 0 variants, skipping\n",
      "[INFO]\tGene GNPTG has 0 variants, skipping\n",
      "[INFO]\tGene LYZL1 has 0 variants, skipping\n",
      "[INFO]\tGene BOK has 0 variants, skipping\n",
      "[INFO]\tGene MAP1LC3A has 0 variants, skipping\n",
      "[INFO]\tGene RNF215 has 0 variants, skipping\n",
      "[INFO]\tGene RPS15A has 0 variants, skipping\n",
      "[INFO]\tGene P4HB has 0 variants, skipping\n",
      "[INFO]\tGene RPS13 has 0 variants, skipping\n",
      "[INFO]\tGene RPS12 has 0 variants, skipping\n",
      "[INFO]\tGene OXT has 0 variants, skipping\n",
      "[INFO]\tGene RPS11 has 0 variants, skipping\n",
      "[INFO]\tGene RPS8 has 0 variants, skipping\n",
      "[INFO]\tGene OPRD1 has 0 variants, skipping\n",
      "[INFO]\tGene RHOV has 0 variants, skipping\n",
      "[INFO]\tGene LNCRNA-ATB has 0 variants, skipping\n",
      "[INFO]\tGene OR56A3 has 0 variants, skipping\n",
      "[INFO]\tGene OR4C12 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M11 has 0 variants, skipping\n",
      "[INFO]\tGene OR9K2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4N4 has 0 variants, skipping\n",
      "[INFO]\tGene OR4F17 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L3 has 0 variants, skipping\n",
      "[INFO]\tGene RNF187 has 0 variants, skipping\n",
      "[INFO]\tGene LELP1 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER1 has 0 variants, skipping\n",
      "[INFO]\tGene STYK1 has 0 variants, skipping\n",
      "[INFO]\tGene PDLIM1 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K15 has 0 variants, skipping\n",
      "[INFO]\tGene TLCD3A has 0 variants, skipping\n",
      "[INFO]\tGene MFSD13A has 0 variants, skipping\n",
      "[INFO]\tGene RNF122 has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC11 has 0 variants, skipping\n",
      "[INFO]\tGene AGBL2 has 0 variants, skipping\n",
      "[INFO]\tGene NHEJ1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB6C has 0 variants, skipping\n",
      "[INFO]\tGene CEP192 has 0 variants, skipping\n",
      "[INFO]\tGene LINP1 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf69 has 0 variants, skipping\n",
      "[INFO]\tGene LOC388780 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf178 has 0 variants, skipping\n",
      "[INFO]\tGene LOC339862 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC140 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf91 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC28A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507507 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf87 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100132813 has 0 variants, skipping\n",
      "[INFO]\tGene LOC401478 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375787 has 0 variants, skipping\n",
      "[INFO]\tGene PRAMEF2 has 0 variants, skipping\n",
      "[INFO]\tGene CENPH has 0 variants, skipping\n",
      "[INFO]\tGene ACTR8 has 0 variants, skipping\n",
      "[INFO]\tGene KRI1 has 0 variants, skipping\n",
      "[INFO]\tGene RANBP17 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf33 has 0 variants, skipping\n",
      "[INFO]\tGene ISOC1 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHGB5 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN4 has 0 variants, skipping\n",
      "[INFO]\tGene TCIM has 0 variants, skipping\n",
      "[INFO]\tGene HOXB9 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R9 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB8 has 0 variants, skipping\n",
      "[INFO]\tGene HOXB4 has 0 variants, skipping\n",
      "[INFO]\tGene GPR87 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA6 has 0 variants, skipping\n",
      "[INFO]\tGene IFNA1 has 0 variants, skipping\n",
      "[INFO]\tGene HTR5A has 0 variants, skipping\n",
      "[INFO]\tGene CELA2A has 0 variants, skipping\n",
      "[INFO]\tGene TBX2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF628 has 0 variants, skipping\n",
      "[INFO]\tGene CABS1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD13A has 0 variants, skipping\n",
      "[INFO]\tGene ZYG11B has 0 variants, skipping\n",
      "[INFO]\tGene BBS1 has 0 variants, skipping\n",
      "[INFO]\tGene LHX4 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf119 has 0 variants, skipping\n",
      "[INFO]\tGene GNB1L has 0 variants, skipping\n",
      "[INFO]\tGene LOC102725021 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268173 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP15 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR1 has 0 variants, skipping\n",
      "[INFO]\tGene GNG7 has 0 variants, skipping\n",
      "[INFO]\tGene CREB3L1 has 0 variants, skipping\n",
      "[INFO]\tGene ADCK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF486 has 0 variants, skipping\n",
      "[INFO]\tGene MAL2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A13 has 0 variants, skipping\n",
      "[INFO]\tGene ZIM3 has 0 variants, skipping\n",
      "[INFO]\tGene FMNL2 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK2 has 0 variants, skipping\n",
      "[INFO]\tGene ELFN2 has 0 variants, skipping\n",
      "[INFO]\tGene ESCO1 has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHG4B has 0 variants, skipping\n",
      "[INFO]\tGene XKR4 has 0 variants, skipping\n",
      "[INFO]\tGene MBD6 has 0 variants, skipping\n",
      "[INFO]\tGene RHPN1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL29 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP1-3 has 0 variants, skipping\n",
      "[INFO]\tGene PSG2 has 0 variants, skipping\n",
      "[INFO]\tGene NECAB3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF93 has 0 variants, skipping\n",
      "[INFO]\tGene KIF18A has 0 variants, skipping\n",
      "[INFO]\tGene CABLES2 has 0 variants, skipping\n",
      "[INFO]\tGene NUAK2 has 0 variants, skipping\n",
      "[INFO]\tGene ASB16-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BANCR has 0 variants, skipping\n",
      "[INFO]\tGene ADAMTS17 has 0 variants, skipping\n",
      "[INFO]\tGene DNER has 0 variants, skipping\n",
      "[INFO]\tGene PROKR1 has 0 variants, skipping\n",
      "[INFO]\tGene ESAM has 0 variants, skipping\n",
      "[INFO]\tGene TGIF2LX has 0 variants, skipping\n",
      "[INFO]\tGene ARX has 0 variants, skipping\n",
      "[INFO]\tGene SMPD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268117 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268114 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268068 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268140 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267983 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB7 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB4 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC4 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB2 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB15 has 0 variants, skipping\n",
      "[INFO]\tGene RHOF has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB14 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB13 has 0 variants, skipping\n",
      "[INFO]\tGene CCL25 has 0 variants, skipping\n",
      "[INFO]\tGene MYO5C has 0 variants, skipping\n",
      "[INFO]\tGene ABCG5 has 0 variants, skipping\n",
      "[INFO]\tGene GOLM2 has 0 variants, skipping\n",
      "[INFO]\tGene SBNO1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MMP25 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT17 has 0 variants, skipping\n",
      "[INFO]\tGene CDH24 has 0 variants, skipping\n",
      "[INFO]\tGene SCP2D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3142 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2909 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01793 has 0 variants, skipping\n",
      "[INFO]\tGene TEDC2 has 0 variants, skipping\n",
      "[INFO]\tGene GATA5 has 0 variants, skipping\n",
      "[INFO]\tGene PIGU has 0 variants, skipping\n",
      "[INFO]\tGene STAB2 has 0 variants, skipping\n",
      "[INFO]\tGene OPLAH has 0 variants, skipping\n",
      "[INFO]\tGene RNASEL has 0 variants, skipping\n",
      "[INFO]\tGene PPA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL12 has 0 variants, skipping\n",
      "[INFO]\tGene FAM117A has 0 variants, skipping\n",
      "[INFO]\tGene RAI2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02877 has 0 variants, skipping\n",
      "[INFO]\tGene DCXR-DT has 0 variants, skipping\n",
      "[INFO]\tGene COL25A1-DT has 0 variants, skipping\n",
      "[INFO]\tGene GAPLINC has 0 variants, skipping\n",
      "[INFO]\tGene THRIL has 0 variants, skipping\n",
      "[INFO]\tGene C5orf67 has 0 variants, skipping\n",
      "[INFO]\tGene GDF5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARIH2OS has 0 variants, skipping\n",
      "[INFO]\tGene RAB1B has 0 variants, skipping\n",
      "[INFO]\tGene ARPC5L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP4-6 has 0 variants, skipping\n",
      "[INFO]\tGene IPPK has 0 variants, skipping\n",
      "[INFO]\tGene ATG4C has 0 variants, skipping\n",
      "[INFO]\tGene MEDAG has 0 variants, skipping\n",
      "[INFO]\tGene C8orf76 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12136 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12132 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12133 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11399 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1843 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12116 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3085 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9851 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10392 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10526 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9898 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10527 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3059 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9902-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9902-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9718 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12121 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10393 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12135 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11181 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9901 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12128 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11401 has 0 variants, skipping\n",
      "[INFO]\tGene MIR548BC has 0 variants, skipping\n",
      "[INFO]\tGene MIR10226 has 0 variants, skipping\n",
      "[INFO]\tGene MAGI1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene BLOC1S4 has 0 variants, skipping\n",
      "[INFO]\tGene MNS1 has 0 variants, skipping\n",
      "[INFO]\tGene GPD1L has 0 variants, skipping\n",
      "[INFO]\tGene MIR10394 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9986 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9899 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12122 has 0 variants, skipping\n",
      "[INFO]\tGene VGLL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10396B has 0 variants, skipping\n",
      "[INFO]\tGene MIR10396A has 0 variants, skipping\n",
      "[INFO]\tGene MIR12114 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12127 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9900 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6529 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12124 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12113 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12115 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10397 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10522 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12130 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12125 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10523 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12118 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9983 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10398 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10524 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12131 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10525 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12119 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10399 has 0 variants, skipping\n",
      "[INFO]\tGene MIR11400 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC71 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12123 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9903 has 0 variants, skipping\n",
      "[INFO]\tGene MIR10400 has 0 variants, skipping\n",
      "[INFO]\tGene GHITM has 0 variants, skipping\n",
      "[INFO]\tGene UQCRQ has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12117 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12126 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12129 has 0 variants, skipping\n",
      "[INFO]\tGene MIR9985 has 0 variants, skipping\n",
      "[INFO]\tGene MIR12120 has 0 variants, skipping\n",
      "[INFO]\tGene DDIT4 has 0 variants, skipping\n",
      "[INFO]\tGene UGT1A7 has 0 variants, skipping\n",
      "[INFO]\tGene CA5BP1-CA5B has 0 variants, skipping\n",
      "[INFO]\tGene TEX12 has 0 variants, skipping\n",
      "[INFO]\tGene TEX13B has 0 variants, skipping\n",
      "[INFO]\tGene WDR55 has 0 variants, skipping\n",
      "[INFO]\tGene MYOPARR has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267871 has 0 variants, skipping\n",
      "[INFO]\tGene ABCF2-H2BE1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144NL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268124 has 0 variants, skipping\n",
      "[INFO]\tGene PHAX has 0 variants, skipping\n",
      "[INFO]\tGene GPR158 has 0 variants, skipping\n",
      "[INFO]\tGene ZFHX4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC8 has 0 variants, skipping\n",
      "[INFO]\tGene TREX2 has 0 variants, skipping\n",
      "[INFO]\tGene MC3R has 0 variants, skipping\n",
      "[INFO]\tGene CHST7 has 0 variants, skipping\n",
      "[INFO]\tGene GSK3A has 0 variants, skipping\n",
      "[INFO]\tGene TRPV1 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK6 has 0 variants, skipping\n",
      "[INFO]\tGene IFI27L2 has 0 variants, skipping\n",
      "[INFO]\tGene GPR61 has 0 variants, skipping\n",
      "[INFO]\tGene HMCN1 has 0 variants, skipping\n",
      "[INFO]\tGene BRIP1 has 0 variants, skipping\n",
      "[INFO]\tGene DMRTA2 has 0 variants, skipping\n",
      "[INFO]\tGene OTP has 0 variants, skipping\n",
      "[INFO]\tGene SEMA6B has 0 variants, skipping\n",
      "[INFO]\tGene OR8D4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR8J1 has 0 variants, skipping\n",
      "[INFO]\tGene FXYD7 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXB1 has 0 variants, skipping\n",
      "[INFO]\tGene PARD6G has 0 variants, skipping\n",
      "[INFO]\tGene MAK16 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF2 has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H8 has 0 variants, skipping\n",
      "[INFO]\tGene NAA11 has 0 variants, skipping\n",
      "[INFO]\tGene KMT5C has 0 variants, skipping\n",
      "[INFO]\tGene IFNLR1 has 0 variants, skipping\n",
      "[INFO]\tGene MPV17L2 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM51 has 0 variants, skipping\n",
      "[INFO]\tGene PCGF1 has 0 variants, skipping\n",
      "[INFO]\tGene FUT10 has 0 variants, skipping\n",
      "[INFO]\tGene DHX37 has 0 variants, skipping\n",
      "[INFO]\tGene CNDP1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM167B has 0 variants, skipping\n",
      "[INFO]\tGene EPN3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP23 has 0 variants, skipping\n",
      "[INFO]\tGene EEF1AKMT4 has 0 variants, skipping\n",
      "[INFO]\tGene CAPNS2 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL35 has 0 variants, skipping\n",
      "[INFO]\tGene GALNT6 has 0 variants, skipping\n",
      "[INFO]\tGene DHX32 has 0 variants, skipping\n",
      "[INFO]\tGene RETSAT has 0 variants, skipping\n",
      "[INFO]\tGene MRPL21 has 0 variants, skipping\n",
      "[INFO]\tGene XKR8 has 0 variants, skipping\n",
      "[INFO]\tGene PTCD3 has 0 variants, skipping\n",
      "[INFO]\tGene SEC14L5 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0408 has 0 variants, skipping\n",
      "[INFO]\tGene RAB21 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D22B has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL2 has 0 variants, skipping\n",
      "[INFO]\tGene FASTKD5 has 0 variants, skipping\n",
      "[INFO]\tGene LMAN1L has 0 variants, skipping\n",
      "[INFO]\tGene SAV1 has 0 variants, skipping\n",
      "[INFO]\tGene HAPLN2 has 0 variants, skipping\n",
      "[INFO]\tGene UCP1 has 0 variants, skipping\n",
      "[INFO]\tGene RMND5A has 0 variants, skipping\n",
      "[INFO]\tGene OSBPL11 has 0 variants, skipping\n",
      "[INFO]\tGene EPS8L2 has 0 variants, skipping\n",
      "[INFO]\tGene AEN has 0 variants, skipping\n",
      "[INFO]\tGene CERK has 0 variants, skipping\n",
      "[INFO]\tGene ACE2 has 0 variants, skipping\n",
      "[INFO]\tGene ADAM30 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC12 has 0 variants, skipping\n",
      "[INFO]\tGene ACTL8 has 0 variants, skipping\n",
      "[INFO]\tGene CHD6 has 0 variants, skipping\n",
      "[INFO]\tGene FRMPD3 has 0 variants, skipping\n",
      "[INFO]\tGene MAML2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF528 has 0 variants, skipping\n",
      "[INFO]\tGene CLIP2 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXD has 0 variants, skipping\n",
      "[INFO]\tGene PROK1 has 0 variants, skipping\n",
      "[INFO]\tGene CYSTM1 has 0 variants, skipping\n",
      "[INFO]\tGene ECRG4 has 0 variants, skipping\n",
      "[INFO]\tGene PINK1 has 0 variants, skipping\n",
      "[INFO]\tGene HOOK3 has 0 variants, skipping\n",
      "[INFO]\tGene EFCAB7 has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K21 has 0 variants, skipping\n",
      "[INFO]\tGene MRPS26 has 0 variants, skipping\n",
      "[INFO]\tGene YIPF4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC30 has 0 variants, skipping\n",
      "[INFO]\tGene NICN1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A33 has 0 variants, skipping\n",
      "[INFO]\tGene COQ5 has 0 variants, skipping\n",
      "[INFO]\tGene NOA1 has 0 variants, skipping\n",
      "[INFO]\tGene FKBP10 has 0 variants, skipping\n",
      "[INFO]\tGene AASS has 0 variants, skipping\n",
      "[INFO]\tGene ZNF527 has 0 variants, skipping\n",
      "[INFO]\tGene SPIRE2 has 0 variants, skipping\n",
      "[INFO]\tGene C18orf15 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf15 has 0 variants, skipping\n",
      "[INFO]\tGene A1BG has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN18 has 0 variants, skipping\n",
      "[INFO]\tGene RGS18 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE2 has 0 variants, skipping\n",
      "[INFO]\tGene XAGE5 has 0 variants, skipping\n",
      "[INFO]\tGene CNTNAP5 has 0 variants, skipping\n",
      "[INFO]\tGene S100Z has 0 variants, skipping\n",
      "[INFO]\tGene ZNF71-SMIM17 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP3 has 0 variants, skipping\n",
      "[INFO]\tGene PAGR1 has 0 variants, skipping\n",
      "[INFO]\tGene VILL has 0 variants, skipping\n",
      "[INFO]\tGene PRM3 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf75 has 0 variants, skipping\n",
      "[INFO]\tGene SLX4 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB133 has 0 variants, skipping\n",
      "[INFO]\tGene TEAD1 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL17 has 0 variants, skipping\n",
      "[INFO]\tGene DAB2IP has 0 variants, skipping\n",
      "[INFO]\tGene KISS1R has 0 variants, skipping\n",
      "[INFO]\tGene USP32 has 0 variants, skipping\n",
      "[INFO]\tGene IL1F10 has 0 variants, skipping\n",
      "[INFO]\tGene MCEE has 0 variants, skipping\n",
      "[INFO]\tGene CCDC54 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM63 has 0 variants, skipping\n",
      "[INFO]\tGene CARD6 has 0 variants, skipping\n",
      "[INFO]\tGene MIR30B has 0 variants, skipping\n",
      "[INFO]\tGene MIR27B has 0 variants, skipping\n",
      "[INFO]\tGene MIR23B has 0 variants, skipping\n",
      "[INFO]\tGene LINC00102 has 0 variants, skipping\n",
      "[INFO]\tGene BACE1-AS has 0 variants, skipping\n",
      "[INFO]\tGene TAMALIN-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HCAR1 has 0 variants, skipping\n",
      "[INFO]\tGene LCE3D has 0 variants, skipping\n",
      "[INFO]\tGene RETNLB has 0 variants, skipping\n",
      "[INFO]\tGene OIP5-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene KCMF1 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R37 has 0 variants, skipping\n",
      "[INFO]\tGene UGGT2 has 0 variants, skipping\n",
      "[INFO]\tGene PDF has 0 variants, skipping\n",
      "[INFO]\tGene RAB38 has 0 variants, skipping\n",
      "[INFO]\tGene EDAR has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLG has 0 variants, skipping\n",
      "[INFO]\tGene ANKRA2 has 0 variants, skipping\n",
      "[INFO]\tGene FRY has 0 variants, skipping\n",
      "[INFO]\tGene TPSB2 has 0 variants, skipping\n",
      "[INFO]\tGene DIPK2B has 0 variants, skipping\n",
      "[INFO]\tGene BBS10 has 0 variants, skipping\n",
      "[INFO]\tGene TEFM has 0 variants, skipping\n",
      "[INFO]\tGene SLBP has 0 variants, skipping\n",
      "[INFO]\tGene GEMIN6 has 0 variants, skipping\n",
      "[INFO]\tGene PLBD1 has 0 variants, skipping\n",
      "[INFO]\tGene SPANXC has 0 variants, skipping\n",
      "[INFO]\tGene EBF2 has 0 variants, skipping\n",
      "[INFO]\tGene MYCT1 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB1 has 0 variants, skipping\n",
      "[INFO]\tGene PGGHG has 0 variants, skipping\n",
      "[INFO]\tGene PTMA has 0 variants, skipping\n",
      "[INFO]\tGene NYAP2 has 0 variants, skipping\n",
      "[INFO]\tGene EMILIN3 has 0 variants, skipping\n",
      "[INFO]\tGene DYNC2I2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02872 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf55 has 0 variants, skipping\n",
      "[INFO]\tGene FAM218A has 0 variants, skipping\n",
      "[INFO]\tGene BIRC8 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00540 has 0 variants, skipping\n",
      "[INFO]\tGene MIR186 has 0 variants, skipping\n",
      "[INFO]\tGene COMMD4P2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF329 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHF2 has 0 variants, skipping\n",
      "[INFO]\tGene SHISA7 has 0 variants, skipping\n",
      "[INFO]\tGene HEATR5B has 0 variants, skipping\n",
      "[INFO]\tGene APBB1IP has 0 variants, skipping\n",
      "[INFO]\tGene TANGO6 has 0 variants, skipping\n",
      "[INFO]\tGene PITHD1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM47 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL50 has 0 variants, skipping\n",
      "[INFO]\tGene USP53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF830 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD40 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD2 has 0 variants, skipping\n",
      "[INFO]\tGene TSSK1B has 0 variants, skipping\n",
      "[INFO]\tGene VPS18 has 0 variants, skipping\n",
      "[INFO]\tGene CDKAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD63 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L5 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L4 has 0 variants, skipping\n",
      "[INFO]\tGene MTRNR2L3 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC5 has 0 variants, skipping\n",
      "[INFO]\tGene ULBP2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD39 has 0 variants, skipping\n",
      "[INFO]\tGene TLR8 has 0 variants, skipping\n",
      "[INFO]\tGene ARMCX1 has 0 variants, skipping\n",
      "[INFO]\tGene HKDC1 has 0 variants, skipping\n",
      "[INFO]\tGene NAT16 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01720 has 0 variants, skipping\n",
      "[INFO]\tGene SUSD5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985946 has 0 variants, skipping\n",
      "[INFO]\tGene PRDM14 has 0 variants, skipping\n",
      "[INFO]\tGene CNTN2 has 0 variants, skipping\n",
      "[INFO]\tGene EPM2AIP1 has 0 variants, skipping\n",
      "[INFO]\tGene CEBPZ has 0 variants, skipping\n",
      "[INFO]\tGene MSRB2 has 0 variants, skipping\n",
      "[INFO]\tGene GTPBP6 has 0 variants, skipping\n",
      "[INFO]\tGene TPSD1 has 0 variants, skipping\n",
      "[INFO]\tGene MLYCD has 0 variants, skipping\n",
      "[INFO]\tGene RABGAP1 has 0 variants, skipping\n",
      "[INFO]\tGene GAGE8 has 0 variants, skipping\n",
      "[INFO]\tGene HAAO has 0 variants, skipping\n",
      "[INFO]\tGene SLC22A15 has 0 variants, skipping\n",
      "[INFO]\tGene KLRC2 has 0 variants, skipping\n",
      "[INFO]\tGene KLRB1 has 0 variants, skipping\n",
      "[INFO]\tGene KLK1 has 0 variants, skipping\n",
      "[INFO]\tGene KISS1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF3C has 0 variants, skipping\n",
      "[INFO]\tGene KDR has 0 variants, skipping\n",
      "[INFO]\tGene SPTBN2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC69 has 0 variants, skipping\n",
      "[INFO]\tGene LRIT1 has 0 variants, skipping\n",
      "[INFO]\tGene HABP4 has 0 variants, skipping\n",
      "[INFO]\tGene SOX3 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN15 has 0 variants, skipping\n",
      "[INFO]\tGene SMO has 0 variants, skipping\n",
      "[INFO]\tGene SLCO2A1 has 0 variants, skipping\n",
      "[INFO]\tGene SRSF4 has 0 variants, skipping\n",
      "[INFO]\tGene CCL8 has 0 variants, skipping\n",
      "[INFO]\tGene OR5I1 has 0 variants, skipping\n",
      "[INFO]\tGene MTHFD2 has 0 variants, skipping\n",
      "[INFO]\tGene VAMP5 has 0 variants, skipping\n",
      "[INFO]\tGene COG1 has 0 variants, skipping\n",
      "[INFO]\tGene SVOP has 0 variants, skipping\n",
      "[INFO]\tGene PIP4P2 has 0 variants, skipping\n",
      "[INFO]\tGene FEM1A has 0 variants, skipping\n",
      "[INFO]\tGene ELAC1 has 0 variants, skipping\n",
      "[INFO]\tGene DHTKD1 has 0 variants, skipping\n",
      "[INFO]\tGene CTTNBP2NL has 0 variants, skipping\n",
      "[INFO]\tGene ADARB2 has 0 variants, skipping\n",
      "[INFO]\tGene MYH7B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Z has 0 variants, skipping\n",
      "[INFO]\tGene TIGD3 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD4 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEH1 has 0 variants, skipping\n",
      "[INFO]\tGene RGCC has 0 variants, skipping\n",
      "[INFO]\tGene SOCS7 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN4 has 0 variants, skipping\n",
      "[INFO]\tGene SPN has 0 variants, skipping\n",
      "[INFO]\tGene LOC150051 has 0 variants, skipping\n",
      "[INFO]\tGene CRABP1 has 0 variants, skipping\n",
      "[INFO]\tGene SERTAD2 has 0 variants, skipping\n",
      "[INFO]\tGene BMS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPCS2 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL4 has 0 variants, skipping\n",
      "[INFO]\tGene FANCF has 0 variants, skipping\n",
      "[INFO]\tGene ANAPC1 has 0 variants, skipping\n",
      "[INFO]\tGene COIL has 0 variants, skipping\n",
      "[INFO]\tGene STK17A has 0 variants, skipping\n",
      "[INFO]\tGene PRKCA has 0 variants, skipping\n",
      "[INFO]\tGene WDR87BP has 0 variants, skipping\n",
      "[INFO]\tGene PARTICL has 0 variants, skipping\n",
      "[INFO]\tGene SH3PXD2A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DMRT3 has 0 variants, skipping\n",
      "[INFO]\tGene TNF has 0 variants, skipping\n",
      "[INFO]\tGene C5orf47 has 0 variants, skipping\n",
      "[INFO]\tGene SLC11A1 has 0 variants, skipping\n",
      "[INFO]\tGene IL1B has 0 variants, skipping\n",
      "[INFO]\tGene IL1A has 0 variants, skipping\n",
      "[INFO]\tGene PRRT1B has 0 variants, skipping\n",
      "[INFO]\tGene ICMT has 0 variants, skipping\n",
      "[INFO]\tGene PFDN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANP32D has 0 variants, skipping\n",
      "[INFO]\tGene PFAS has 0 variants, skipping\n",
      "[INFO]\tGene ANP32C has 0 variants, skipping\n",
      "[INFO]\tGene PREX1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42BPG has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB17 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF891 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF9B has 0 variants, skipping\n",
      "[INFO]\tGene NUDT14 has 0 variants, skipping\n",
      "[INFO]\tGene TSTD3 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS56 has 0 variants, skipping\n",
      "[INFO]\tGene ROCK1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD61 has 0 variants, skipping\n",
      "[INFO]\tGene EPB41L4A-DT has 0 variants, skipping\n",
      "[INFO]\tGene KCNJ18 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC25-GNG10 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA4 has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA6 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROD1 has 0 variants, skipping\n",
      "[INFO]\tGene NDN has 0 variants, skipping\n",
      "[INFO]\tGene CEACAM6 has 0 variants, skipping\n",
      "[INFO]\tGene MYOG has 0 variants, skipping\n",
      "[INFO]\tGene MYOD1 has 0 variants, skipping\n",
      "[INFO]\tGene LYPD3 has 0 variants, skipping\n",
      "[INFO]\tGene UTP25 has 0 variants, skipping\n",
      "[INFO]\tGene ACAD8 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG1 has 0 variants, skipping\n",
      "[INFO]\tGene SDC4 has 0 variants, skipping\n",
      "[INFO]\tGene SP2 has 0 variants, skipping\n",
      "[INFO]\tGene SOX11 has 0 variants, skipping\n",
      "[INFO]\tGene SDC2 has 0 variants, skipping\n",
      "[INFO]\tGene SNTA1 has 0 variants, skipping\n",
      "[INFO]\tGene SOX4 has 0 variants, skipping\n",
      "[INFO]\tGene SOX2 has 0 variants, skipping\n",
      "[INFO]\tGene WDR12 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC5 has 0 variants, skipping\n",
      "[INFO]\tGene OTOR has 0 variants, skipping\n",
      "[INFO]\tGene C1GALT1 has 0 variants, skipping\n",
      "[INFO]\tGene EMC7 has 0 variants, skipping\n",
      "[INFO]\tGene THAP10 has 0 variants, skipping\n",
      "[INFO]\tGene PAPOLB has 0 variants, skipping\n",
      "[INFO]\tGene NDUFA4L2 has 0 variants, skipping\n",
      "[INFO]\tGene BDH2 has 0 variants, skipping\n",
      "[INFO]\tGene GRIPAP1 has 0 variants, skipping\n",
      "[INFO]\tGene AGPAT4 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01615 has 0 variants, skipping\n",
      "[INFO]\tGene LUADT1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505502 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF9 has 0 variants, skipping\n",
      "[INFO]\tGene SMC3 has 0 variants, skipping\n",
      "[INFO]\tGene PAPSS1 has 0 variants, skipping\n",
      "[INFO]\tGene CHAF1B has 0 variants, skipping\n",
      "[INFO]\tGene RND2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC6 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGEF5 has 0 variants, skipping\n",
      "[INFO]\tGene MALL has 0 variants, skipping\n",
      "[INFO]\tGene YES1 has 0 variants, skipping\n",
      "[INFO]\tGene BCL2L10 has 0 variants, skipping\n",
      "[INFO]\tGene ASB15-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MSRB1 has 0 variants, skipping\n",
      "[INFO]\tGene AOX1 has 0 variants, skipping\n",
      "[INFO]\tGene CCBE1 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRF1 has 0 variants, skipping\n",
      "[INFO]\tGene CLC has 0 variants, skipping\n",
      "[INFO]\tGene IMPA2 has 0 variants, skipping\n",
      "[INFO]\tGene HOXD9 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS2 has 0 variants, skipping\n",
      "[INFO]\tGene BMP2K has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H7B has 0 variants, skipping\n",
      "[INFO]\tGene WDR5 has 0 variants, skipping\n",
      "[INFO]\tGene MIOX has 0 variants, skipping\n",
      "[INFO]\tGene TRIM44 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q1 has 0 variants, skipping\n",
      "[INFO]\tGene CELA3B has 0 variants, skipping\n",
      "[INFO]\tGene MMRN1 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDA1 has 0 variants, skipping\n",
      "[INFO]\tGene PAXIP1 has 0 variants, skipping\n",
      "[INFO]\tGene ATF6 has 0 variants, skipping\n",
      "[INFO]\tGene OGFR has 0 variants, skipping\n",
      "[INFO]\tGene FTHL18 has 0 variants, skipping\n",
      "[INFO]\tGene LDLRAD2 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf97 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAB1 has 0 variants, skipping\n",
      "[INFO]\tGene PRKAA2 has 0 variants, skipping\n",
      "[INFO]\tGene KHDRBS3 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf31 has 0 variants, skipping\n",
      "[INFO]\tGene PANX1 has 0 variants, skipping\n",
      "[INFO]\tGene CXorf66 has 0 variants, skipping\n",
      "[INFO]\tGene OR9G9 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U9 has 0 variants, skipping\n",
      "[INFO]\tGene OR8U8 has 0 variants, skipping\n",
      "[INFO]\tGene OR2G6 has 0 variants, skipping\n",
      "[INFO]\tGene OR11H12 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST4 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM203 has 0 variants, skipping\n",
      "[INFO]\tGene TNFRSF13C has 0 variants, skipping\n",
      "[INFO]\tGene RBM33 has 0 variants, skipping\n",
      "[INFO]\tGene TENT5B has 0 variants, skipping\n",
      "[INFO]\tGene GBP4 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB3 has 0 variants, skipping\n",
      "[INFO]\tGene DCLK3 has 0 variants, skipping\n",
      "[INFO]\tGene DLGAP1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene BSN has 0 variants, skipping\n",
      "[INFO]\tGene SULT4A1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF324 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF318 has 0 variants, skipping\n",
      "[INFO]\tGene FJX1 has 0 variants, skipping\n",
      "[INFO]\tGene EID1 has 0 variants, skipping\n",
      "[INFO]\tGene FRRS1L has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A11 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R15A has 0 variants, skipping\n",
      "[INFO]\tGene EDC4 has 0 variants, skipping\n",
      "[INFO]\tGene HEYL has 0 variants, skipping\n",
      "[INFO]\tGene FOXH1 has 0 variants, skipping\n",
      "[INFO]\tGene RPS6KB2 has 0 variants, skipping\n",
      "[INFO]\tGene BNIP3 has 0 variants, skipping\n",
      "[INFO]\tGene BDH1 has 0 variants, skipping\n",
      "[INFO]\tGene F2RL3 has 0 variants, skipping\n",
      "[INFO]\tGene B2M has 0 variants, skipping\n",
      "[INFO]\tGene FASN has 0 variants, skipping\n",
      "[INFO]\tGene ZNF730 has 0 variants, skipping\n",
      "[INFO]\tGene TCP11X2 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC40 has 0 variants, skipping\n",
      "[INFO]\tGene MTMR10 has 0 variants, skipping\n",
      "[INFO]\tGene PNRC2 has 0 variants, skipping\n",
      "[INFO]\tGene INO80D has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH5 has 0 variants, skipping\n",
      "[INFO]\tGene UHRF1BP1 has 0 variants, skipping\n",
      "[INFO]\tGene TBC1D8B has 0 variants, skipping\n",
      "[INFO]\tGene OR11H4 has 0 variants, skipping\n",
      "[INFO]\tGene OR10Z1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10X1 has 0 variants, skipping\n",
      "[INFO]\tGene EMD has 0 variants, skipping\n",
      "[INFO]\tGene TOR1A has 0 variants, skipping\n",
      "[INFO]\tGene UBFD1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF21 has 0 variants, skipping\n",
      "[INFO]\tGene ABCA7 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-DRA has 0 variants, skipping\n",
      "[INFO]\tGene SMG9 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW5 has 0 variants, skipping\n",
      "[INFO]\tGene MYDGF has 0 variants, skipping\n",
      "[INFO]\tGene TMEM145 has 0 variants, skipping\n",
      "[INFO]\tGene CTAG1B has 0 variants, skipping\n",
      "[INFO]\tGene PIP4K2B has 0 variants, skipping\n",
      "[INFO]\tGene FUT7 has 0 variants, skipping\n",
      "[INFO]\tGene VWF has 0 variants, skipping\n",
      "[INFO]\tGene SCAF1 has 0 variants, skipping\n",
      "[INFO]\tGene LRG1 has 0 variants, skipping\n",
      "[INFO]\tGene LEAP2 has 0 variants, skipping\n",
      "[INFO]\tGene RPL39L has 0 variants, skipping\n",
      "[INFO]\tGene UBQLN1 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf29 has 0 variants, skipping\n",
      "[INFO]\tGene KCNB1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC17A1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC15A1 has 0 variants, skipping\n",
      "[INFO]\tGene PILRA has 0 variants, skipping\n",
      "[INFO]\tGene RAX has 0 variants, skipping\n",
      "[INFO]\tGene VSIR has 0 variants, skipping\n",
      "[INFO]\tGene ACRBP has 0 variants, skipping\n",
      "[INFO]\tGene CNFN has 0 variants, skipping\n",
      "[INFO]\tGene ACTRT3 has 0 variants, skipping\n",
      "[INFO]\tGene MICALCL has 0 variants, skipping\n",
      "[INFO]\tGene TNS4 has 0 variants, skipping\n",
      "[INFO]\tGene PRPF38A has 0 variants, skipping\n",
      "[INFO]\tGene TIGD5 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC12 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHE23 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02694 has 0 variants, skipping\n",
      "[INFO]\tGene IL25 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY12 has 0 variants, skipping\n",
      "[INFO]\tGene SBF2 has 0 variants, skipping\n",
      "[INFO]\tGene SUN5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1204 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1203 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1228 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02885 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00052 has 0 variants, skipping\n",
      "[INFO]\tGene MIR409 has 0 variants, skipping\n",
      "[INFO]\tGene MIR451A has 0 variants, skipping\n",
      "[INFO]\tGene MIR329-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR329-1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR431 has 0 variants, skipping\n",
      "[INFO]\tGene MIR450A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR449A has 0 variants, skipping\n",
      "[INFO]\tGene MIR429 has 0 variants, skipping\n",
      "[INFO]\tGene MIR448 has 0 variants, skipping\n",
      "[INFO]\tGene MIR18B has 0 variants, skipping\n",
      "[INFO]\tGene MIR20B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4524B has 0 variants, skipping\n",
      "[INFO]\tGene MIR1180 has 0 variants, skipping\n",
      "[INFO]\tGene MIR452 has 0 variants, skipping\n",
      "[INFO]\tGene WBP11P1 has 0 variants, skipping\n",
      "[INFO]\tGene TCERG1L has 0 variants, skipping\n",
      "[INFO]\tGene MALAT1 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A2P1 has 0 variants, skipping\n",
      "[INFO]\tGene DAPK1-IT1 has 0 variants, skipping\n",
      "[INFO]\tGene PCBP2-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene MFF-DT has 0 variants, skipping\n",
      "[INFO]\tGene SNHG8 has 0 variants, skipping\n",
      "[INFO]\tGene ZPLD2P has 0 variants, skipping\n",
      "[INFO]\tGene MECOM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM135A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DPP4-DT has 0 variants, skipping\n",
      "[INFO]\tGene HOXC5 has 0 variants, skipping\n",
      "[INFO]\tGene PHC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PDC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01287 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01170 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1671-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00511 has 0 variants, skipping\n",
      "[INFO]\tGene TP53TG1 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR7 has 0 variants, skipping\n",
      "[INFO]\tGene FTH1P3 has 0 variants, skipping\n",
      "[INFO]\tGene GBAP1 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene TMPO-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RNVU1-2A has 0 variants, skipping\n",
      "[INFO]\tGene DLEU1 has 0 variants, skipping\n",
      "[INFO]\tGene GAS5 has 0 variants, skipping\n",
      "[INFO]\tGene MIR933 has 0 variants, skipping\n",
      "[INFO]\tGene MIR924 has 0 variants, skipping\n",
      "[INFO]\tGene TSSC2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG19 has 0 variants, skipping\n",
      "[INFO]\tGene WSPAR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01589 has 0 variants, skipping\n",
      "[INFO]\tGene SMILR has 0 variants, skipping\n",
      "[INFO]\tGene WFDC21P has 0 variants, skipping\n",
      "[INFO]\tGene MIR1179 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2053 has 0 variants, skipping\n",
      "[INFO]\tGene MIR1976 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3064 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC8 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8485 has 0 variants, skipping\n",
      "[INFO]\tGene MIR577 has 0 variants, skipping\n",
      "[INFO]\tGene MIR769 has 0 variants, skipping\n",
      "[INFO]\tGene MIR660 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4767 has 0 variants, skipping\n",
      "[INFO]\tGene WWC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene APOA1-AS has 0 variants, skipping\n",
      "[INFO]\tGene ZDHHC11B has 0 variants, skipping\n",
      "[INFO]\tGene MIR718 has 0 variants, skipping\n",
      "[INFO]\tGene CLLU1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FGF13-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00525 has 0 variants, skipping\n",
      "[INFO]\tGene IL6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC730101 has 0 variants, skipping\n",
      "[INFO]\tGene CXCL17 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00682 has 0 variants, skipping\n",
      "[INFO]\tGene GLCCI1-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR2116 has 0 variants, skipping\n",
      "[INFO]\tGene KLRK1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene PLD6 has 0 variants, skipping\n",
      "[INFO]\tGene KLF3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MTPN has 0 variants, skipping\n",
      "[INFO]\tGene MGAT4FP has 0 variants, skipping\n",
      "[INFO]\tGene MIR556 has 0 variants, skipping\n",
      "[INFO]\tGene CYMP has 0 variants, skipping\n",
      "[INFO]\tGene DNMT3L-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00467 has 0 variants, skipping\n",
      "[INFO]\tGene WDR88 has 0 variants, skipping\n",
      "[INFO]\tGene MIR7109 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6893 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD44-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02605 has 0 variants, skipping\n",
      "[INFO]\tGene HPAT5 has 0 variants, skipping\n",
      "[INFO]\tGene THORLNC has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7I has 0 variants, skipping\n",
      "[INFO]\tGene MIRLET7G has 0 variants, skipping\n",
      "[INFO]\tGene MIR1225 has 0 variants, skipping\n",
      "[INFO]\tGene MIR200B has 0 variants, skipping\n",
      "[INFO]\tGene MIR942 has 0 variants, skipping\n",
      "[INFO]\tGene MIR224 has 0 variants, skipping\n",
      "[INFO]\tGene MIR223 has 0 variants, skipping\n",
      "[INFO]\tGene MIR222 has 0 variants, skipping\n",
      "[INFO]\tGene MIR221 has 0 variants, skipping\n",
      "[INFO]\tGene MIR938 has 0 variants, skipping\n",
      "[INFO]\tGene MIR219A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR134 has 0 variants, skipping\n",
      "[INFO]\tGene MIR188 has 0 variants, skipping\n",
      "[INFO]\tGene GNAS-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MCM3AP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6124 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD89 has 0 variants, skipping\n",
      "[INFO]\tGene RMRP has 0 variants, skipping\n",
      "[INFO]\tGene SNHG5 has 0 variants, skipping\n",
      "[INFO]\tGene DSEL has 0 variants, skipping\n",
      "[INFO]\tGene THBS4-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC20P has 0 variants, skipping\n",
      "[INFO]\tGene LOC100421372 has 0 variants, skipping\n",
      "[INFO]\tGene NCOR1P4 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00518 has 0 variants, skipping\n",
      "[INFO]\tGene BNC2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene TSPAN10 has 0 variants, skipping\n",
      "[INFO]\tGene PARAL1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4435-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4652 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3978 has 0 variants, skipping\n",
      "[INFO]\tGene NLRC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5571 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00514 has 0 variants, skipping\n",
      "[INFO]\tGene OR52H1 has 0 variants, skipping\n",
      "[INFO]\tGene USP51 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3178 has 0 variants, skipping\n",
      "[INFO]\tGene PAQR9 has 0 variants, skipping\n",
      "[INFO]\tGene H2BW4P has 0 variants, skipping\n",
      "[INFO]\tGene MRPS9-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4443 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4429 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4428 has 0 variants, skipping\n",
      "[INFO]\tGene GAS6-DT has 0 variants, skipping\n",
      "[INFO]\tGene MIR3182 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3175 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf36 has 0 variants, skipping\n",
      "[INFO]\tGene C15orf32 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf177 has 0 variants, skipping\n",
      "[INFO]\tGene IRF1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NEUROG2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene H3P4 has 0 variants, skipping\n",
      "[INFO]\tGene H2BP2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR937 has 0 variants, skipping\n",
      "[INFO]\tGene MIR218-2 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00976 has 0 variants, skipping\n",
      "[INFO]\tGene MAPK1IP1L has 0 variants, skipping\n",
      "[INFO]\tGene PDIA2 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC85B has 0 variants, skipping\n",
      "[INFO]\tGene EGFR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HSPB6 has 0 variants, skipping\n",
      "[INFO]\tGene JSRP1 has 0 variants, skipping\n",
      "[INFO]\tGene H2AB3 has 0 variants, skipping\n",
      "[INFO]\tGene PIP4P1 has 0 variants, skipping\n",
      "[INFO]\tGene CAPN13 has 0 variants, skipping\n",
      "[INFO]\tGene JPT2 has 0 variants, skipping\n",
      "[INFO]\tGene IL23R has 0 variants, skipping\n",
      "[INFO]\tGene GPR62 has 0 variants, skipping\n",
      "[INFO]\tGene ASB16 has 0 variants, skipping\n",
      "[INFO]\tGene SPSB4 has 0 variants, skipping\n",
      "[INFO]\tGene OR1K1 has 0 variants, skipping\n",
      "[INFO]\tGene STK35 has 0 variants, skipping\n",
      "[INFO]\tGene RBBP8NL has 0 variants, skipping\n",
      "[INFO]\tGene GTSF1 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf78 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD22 has 0 variants, skipping\n",
      "[INFO]\tGene COMTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF519 has 0 variants, skipping\n",
      "[INFO]\tGene SH3RF1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF492 has 0 variants, skipping\n",
      "[INFO]\tGene CABP5 has 0 variants, skipping\n",
      "[INFO]\tGene STAMBPL1 has 0 variants, skipping\n",
      "[INFO]\tGene USP35 has 0 variants, skipping\n",
      "[INFO]\tGene NLGN2 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL14 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L2 has 0 variants, skipping\n",
      "[INFO]\tGene DCST2 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO1 has 0 variants, skipping\n",
      "[INFO]\tGene MYORG has 0 variants, skipping\n",
      "[INFO]\tGene TAF3 has 0 variants, skipping\n",
      "[INFO]\tGene RASSF4 has 0 variants, skipping\n",
      "[INFO]\tGene FUCA2 has 0 variants, skipping\n",
      "[INFO]\tGene KLF16 has 0 variants, skipping\n",
      "[INFO]\tGene NLN has 0 variants, skipping\n",
      "[INFO]\tGene CAPN12 has 0 variants, skipping\n",
      "[INFO]\tGene KASH5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB5 has 0 variants, skipping\n",
      "[INFO]\tGene SERPINB9 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3CA has 0 variants, skipping\n",
      "[INFO]\tGene CACNG3 has 0 variants, skipping\n",
      "[INFO]\tGene WNT6 has 0 variants, skipping\n",
      "[INFO]\tGene WFDC10A has 0 variants, skipping\n",
      "[INFO]\tGene DPH3P1 has 0 variants, skipping\n",
      "[INFO]\tGene ROMO1 has 0 variants, skipping\n",
      "[INFO]\tGene RPL10L has 0 variants, skipping\n",
      "[INFO]\tGene SSC4D has 0 variants, skipping\n",
      "[INFO]\tGene B3GAT2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF280A has 0 variants, skipping\n",
      "[INFO]\tGene SRXN1 has 0 variants, skipping\n",
      "[INFO]\tGene NRSN1 has 0 variants, skipping\n",
      "[INFO]\tGene OCSTAMP has 0 variants, skipping\n",
      "[INFO]\tGene GCNT4 has 0 variants, skipping\n",
      "[INFO]\tGene ATP11AUN has 0 variants, skipping\n",
      "[INFO]\tGene KBTBD7 has 0 variants, skipping\n",
      "[INFO]\tGene TKTL2 has 0 variants, skipping\n",
      "[INFO]\tGene FSCB has 0 variants, skipping\n",
      "[INFO]\tGene C10orf126 has 0 variants, skipping\n",
      "[INFO]\tGene LMO7DN has 0 variants, skipping\n",
      "[INFO]\tGene LINC00452 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105376353 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105378479 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728485 has 0 variants, skipping\n",
      "[INFO]\tGene PCSEAT has 0 variants, skipping\n",
      "[INFO]\tGene LINC02874 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985770 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02881 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ45513 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02876 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100134391 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100130370 has 0 variants, skipping\n",
      "[INFO]\tGene LOC403312 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100505549 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927322 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02873 has 0 variants, skipping\n",
      "[INFO]\tGene IL20RB has 0 variants, skipping\n",
      "[INFO]\tGene ZNF516-DT has 0 variants, skipping\n",
      "[INFO]\tGene NAXE has 0 variants, skipping\n",
      "[INFO]\tGene SMCR8 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371566 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929141 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724624 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929950 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101929578 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371910 has 0 variants, skipping\n",
      "[INFO]\tGene DPP9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GLRA4 has 0 variants, skipping\n",
      "[INFO]\tGene HHIP has 0 variants, skipping\n",
      "[INFO]\tGene GREM2 has 0 variants, skipping\n",
      "[INFO]\tGene SAFB2 has 0 variants, skipping\n",
      "[INFO]\tGene DZIP3 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01297 has 0 variants, skipping\n",
      "[INFO]\tGene RS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROM1 has 0 variants, skipping\n",
      "[INFO]\tGene RLBP1 has 0 variants, skipping\n",
      "[INFO]\tGene RHAG has 0 variants, skipping\n",
      "[INFO]\tGene PRPH2 has 0 variants, skipping\n",
      "[INFO]\tGene RB1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR514B has 0 variants, skipping\n",
      "[INFO]\tGene MIR4282 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2355 has 0 variants, skipping\n",
      "[INFO]\tGene MIR4299 has 0 variants, skipping\n",
      "[INFO]\tGene ADO has 0 variants, skipping\n",
      "[INFO]\tGene POMGNT2 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEA10-MAGEA5 has 0 variants, skipping\n",
      "[INFO]\tGene CRCT1 has 0 variants, skipping\n",
      "[INFO]\tGene AATF has 0 variants, skipping\n",
      "[INFO]\tGene FAM50B has 0 variants, skipping\n",
      "[INFO]\tGene LMOD1 has 0 variants, skipping\n",
      "[INFO]\tGene CLDN17 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP4 has 0 variants, skipping\n",
      "[INFO]\tGene CD2AP has 0 variants, skipping\n",
      "[INFO]\tGene NOCT has 0 variants, skipping\n",
      "[INFO]\tGene CASP14 has 0 variants, skipping\n",
      "[INFO]\tGene CA14 has 0 variants, skipping\n",
      "[INFO]\tGene TPX2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM59L has 0 variants, skipping\n",
      "[INFO]\tGene CHIC2 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL2 has 0 variants, skipping\n",
      "[INFO]\tGene ARHGAP10 has 0 variants, skipping\n",
      "[INFO]\tGene PANK3 has 0 variants, skipping\n",
      "[INFO]\tGene SRD5A3 has 0 variants, skipping\n",
      "[INFO]\tGene HECTD3 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP6 has 0 variants, skipping\n",
      "[INFO]\tGene OCEL1 has 0 variants, skipping\n",
      "[INFO]\tGene SH3TC2 has 0 variants, skipping\n",
      "[INFO]\tGene TNFAIP8L2 has 0 variants, skipping\n",
      "[INFO]\tGene NDNF has 0 variants, skipping\n",
      "[INFO]\tGene SNRNP25 has 0 variants, skipping\n",
      "[INFO]\tGene IL17RB has 0 variants, skipping\n",
      "[INFO]\tGene ADGRA3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF296 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-3 has 0 variants, skipping\n",
      "[INFO]\tGene C17orf50 has 0 variants, skipping\n",
      "[INFO]\tGene PRR35 has 0 variants, skipping\n",
      "[INFO]\tGene SSMEM1 has 0 variants, skipping\n",
      "[INFO]\tGene SDHAF4 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC127 has 0 variants, skipping\n",
      "[INFO]\tGene OSR1 has 0 variants, skipping\n",
      "[INFO]\tGene CCSAP has 0 variants, skipping\n",
      "[INFO]\tGene LRRC25 has 0 variants, skipping\n",
      "[INFO]\tGene ZG16B has 0 variants, skipping\n",
      "[INFO]\tGene IFI27L1 has 0 variants, skipping\n",
      "[INFO]\tGene RNASE11 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA7 has 0 variants, skipping\n",
      "[INFO]\tGene DDIT4L has 0 variants, skipping\n",
      "[INFO]\tGene OMA1 has 0 variants, skipping\n",
      "[INFO]\tGene ATP5PB has 0 variants, skipping\n",
      "[INFO]\tGene ATP5F1B has 0 variants, skipping\n",
      "[INFO]\tGene COMMD3 has 0 variants, skipping\n",
      "[INFO]\tGene AKR7A3 has 0 variants, skipping\n",
      "[INFO]\tGene MIP has 0 variants, skipping\n",
      "[INFO]\tGene POLA2 has 0 variants, skipping\n",
      "[INFO]\tGene PNN has 0 variants, skipping\n",
      "[INFO]\tGene CD300C has 0 variants, skipping\n",
      "[INFO]\tGene TRIM58 has 0 variants, skipping\n",
      "[INFO]\tGene POLR1A has 0 variants, skipping\n",
      "[INFO]\tGene KATNB1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBA3C has 0 variants, skipping\n",
      "[INFO]\tGene MAP3K5 has 0 variants, skipping\n",
      "[INFO]\tGene CDC123 has 0 variants, skipping\n",
      "[INFO]\tGene MLLT1 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST2 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST3A1 has 0 variants, skipping\n",
      "[INFO]\tGene MTNR1A has 0 variants, skipping\n",
      "[INFO]\tGene TLR6 has 0 variants, skipping\n",
      "[INFO]\tGene GGA2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOH has 0 variants, skipping\n",
      "[INFO]\tGene TBR1 has 0 variants, skipping\n",
      "[INFO]\tGene INTU has 0 variants, skipping\n",
      "[INFO]\tGene CRACD has 0 variants, skipping\n",
      "[INFO]\tGene KIAA1210 has 0 variants, skipping\n",
      "[INFO]\tGene PRR12 has 0 variants, skipping\n",
      "[INFO]\tGene USP31 has 0 variants, skipping\n",
      "[INFO]\tGene PLEKHH1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF490 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF512B has 0 variants, skipping\n",
      "[INFO]\tGene GATAD2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC47 has 0 variants, skipping\n",
      "[INFO]\tGene PNMA8B has 0 variants, skipping\n",
      "[INFO]\tGene KCNS2 has 0 variants, skipping\n",
      "[INFO]\tGene ERLNC1 has 0 variants, skipping\n",
      "[INFO]\tGene SIRPB3P has 0 variants, skipping\n",
      "[INFO]\tGene ZNNT1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP5 has 0 variants, skipping\n",
      "[INFO]\tGene UBQLNL has 0 variants, skipping\n",
      "[INFO]\tGene UBLCP1 has 0 variants, skipping\n",
      "[INFO]\tGene TTC39A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP8 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP9 has 0 variants, skipping\n",
      "[INFO]\tGene MIR570HG has 0 variants, skipping\n",
      "[INFO]\tGene HOXA-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01419 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01410 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD3-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02273 has 0 variants, skipping\n",
      "[INFO]\tGene NALT1 has 0 variants, skipping\n",
      "[INFO]\tGene MINCR has 0 variants, skipping\n",
      "[INFO]\tGene LINC01518 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2A has 0 variants, skipping\n",
      "[INFO]\tGene SOX9-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00342 has 0 variants, skipping\n",
      "[INFO]\tGene LUCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF13P has 0 variants, skipping\n",
      "[INFO]\tGene LINC00968 has 0 variants, skipping\n",
      "[INFO]\tGene DYNLL2 has 0 variants, skipping\n",
      "[INFO]\tGene RSAD2 has 0 variants, skipping\n",
      "[INFO]\tGene CDKN2AIPNL has 0 variants, skipping\n",
      "[INFO]\tGene TMEM41A has 0 variants, skipping\n",
      "[INFO]\tGene SDR42E1 has 0 variants, skipping\n",
      "[INFO]\tGene PIGM has 0 variants, skipping\n",
      "[INFO]\tGene ZBTB47 has 0 variants, skipping\n",
      "[INFO]\tGene GPHB5 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJB7 has 0 variants, skipping\n",
      "[INFO]\tGene DIRAS1 has 0 variants, skipping\n",
      "[INFO]\tGene DHRSX has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A12 has 0 variants, skipping\n",
      "[INFO]\tGene ATOH7 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT7 has 0 variants, skipping\n",
      "[INFO]\tGene CTU1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM11 has 0 variants, skipping\n",
      "[INFO]\tGene MBD3L1 has 0 variants, skipping\n",
      "[INFO]\tGene CSNK1A1L has 0 variants, skipping\n",
      "[INFO]\tGene LINC02875 has 0 variants, skipping\n",
      "[INFO]\tGene GUSBP17 has 0 variants, skipping\n",
      "[INFO]\tGene MUC5B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00842 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01433 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P2 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00163 has 0 variants, skipping\n",
      "[INFO]\tGene TP73-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene GATC has 0 variants, skipping\n",
      "[INFO]\tGene MSC-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CD99P1 has 0 variants, skipping\n",
      "[INFO]\tGene MTURN has 0 variants, skipping\n",
      "[INFO]\tGene ASPRV1 has 0 variants, skipping\n",
      "[INFO]\tGene TAB3 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C1 has 0 variants, skipping\n",
      "[INFO]\tGene OR56B4 has 0 variants, skipping\n",
      "[INFO]\tGene OR56A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR56B1 has 0 variants, skipping\n",
      "[INFO]\tGene RFC4 has 0 variants, skipping\n",
      "[INFO]\tGene MALSU1 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD12 has 0 variants, skipping\n",
      "[INFO]\tGene ARL11 has 0 variants, skipping\n",
      "[INFO]\tGene NXNL1 has 0 variants, skipping\n",
      "[INFO]\tGene GLI4 has 0 variants, skipping\n",
      "[INFO]\tGene ZMYND19 has 0 variants, skipping\n",
      "[INFO]\tGene NUS1 has 0 variants, skipping\n",
      "[INFO]\tGene SPINDOC has 0 variants, skipping\n",
      "[INFO]\tGene CDAN1 has 0 variants, skipping\n",
      "[INFO]\tGene CHADL has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP19-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP7-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP6-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRT26 has 0 variants, skipping\n",
      "[INFO]\tGene GJC3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT27 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L3 has 0 variants, skipping\n",
      "[INFO]\tGene KRT28 has 0 variants, skipping\n",
      "[INFO]\tGene KRT25 has 0 variants, skipping\n",
      "[INFO]\tGene ERAS has 0 variants, skipping\n",
      "[INFO]\tGene LRRC70 has 0 variants, skipping\n",
      "[INFO]\tGene EXOSC8 has 0 variants, skipping\n",
      "[INFO]\tGene ITGA1 has 0 variants, skipping\n",
      "[INFO]\tGene CMTM7 has 0 variants, skipping\n",
      "[INFO]\tGene CD300E has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R42 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC39 has 0 variants, skipping\n",
      "[INFO]\tGene GATA6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CERNA1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5590 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5582 has 0 variants, skipping\n",
      "[INFO]\tGene MIR5581 has 0 variants, skipping\n",
      "[INFO]\tGene MIR664B has 0 variants, skipping\n",
      "[INFO]\tGene MIR5195 has 0 variants, skipping\n",
      "[INFO]\tGene WAKMAR2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTCH2NLR has 0 variants, skipping\n",
      "[INFO]\tGene IFNG-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CENATAC has 0 variants, skipping\n",
      "[INFO]\tGene CASC9 has 0 variants, skipping\n",
      "[INFO]\tGene DUXAP10 has 0 variants, skipping\n",
      "[INFO]\tGene GSX2 has 0 variants, skipping\n",
      "[INFO]\tGene OAF has 0 variants, skipping\n",
      "[INFO]\tGene MB21D2 has 0 variants, skipping\n",
      "[INFO]\tGene NOTUM has 0 variants, skipping\n",
      "[INFO]\tGene R3HDML has 0 variants, skipping\n",
      "[INFO]\tGene SCP2D1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR119 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF12L1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83C has 0 variants, skipping\n",
      "[INFO]\tGene FOXL3 has 0 variants, skipping\n",
      "[INFO]\tGene CADM4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM241B has 0 variants, skipping\n",
      "[INFO]\tGene SLC25A43 has 0 variants, skipping\n",
      "[INFO]\tGene WDR90 has 0 variants, skipping\n",
      "[INFO]\tGene C2orf69 has 0 variants, skipping\n",
      "[INFO]\tGene STRC has 0 variants, skipping\n",
      "[INFO]\tGene ZFPM1 has 0 variants, skipping\n",
      "[INFO]\tGene PHF13 has 0 variants, skipping\n",
      "[INFO]\tGene CACUL1 has 0 variants, skipping\n",
      "[INFO]\tGene TAF1L has 0 variants, skipping\n",
      "[INFO]\tGene NAP1L5 has 0 variants, skipping\n",
      "[INFO]\tGene LIX1L has 0 variants, skipping\n",
      "[INFO]\tGene TTL has 0 variants, skipping\n",
      "[INFO]\tGene CALHM5 has 0 variants, skipping\n",
      "[INFO]\tGene GSTA5 has 0 variants, skipping\n",
      "[INFO]\tGene ADGRG4 has 0 variants, skipping\n",
      "[INFO]\tGene CREG2 has 0 variants, skipping\n",
      "[INFO]\tGene H2BC1 has 0 variants, skipping\n",
      "[INFO]\tGene KMT2C has 0 variants, skipping\n",
      "[INFO]\tGene RYR3-DT has 0 variants, skipping\n",
      "[INFO]\tGene CASC11 has 0 variants, skipping\n",
      "[INFO]\tGene ZAN has 0 variants, skipping\n",
      "[INFO]\tGene LSAMP-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene HNF4A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SFRP1 has 0 variants, skipping\n",
      "[INFO]\tGene SELENOW has 0 variants, skipping\n",
      "[INFO]\tGene REG1A has 0 variants, skipping\n",
      "[INFO]\tGene RENBP has 0 variants, skipping\n",
      "[INFO]\tGene MYCNOS has 0 variants, skipping\n",
      "[INFO]\tGene LINC01939 has 0 variants, skipping\n",
      "[INFO]\tGene SNORA119 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD138 has 0 variants, skipping\n",
      "[INFO]\tGene SNORD3F has 0 variants, skipping\n",
      "[INFO]\tGene HAGLROS has 0 variants, skipping\n",
      "[INFO]\tGene STK11IP has 0 variants, skipping\n",
      "[INFO]\tGene SGMS1 has 0 variants, skipping\n",
      "[INFO]\tGene RAB42 has 0 variants, skipping\n",
      "[INFO]\tGene TADA2B has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF2 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A8 has 0 variants, skipping\n",
      "[INFO]\tGene PATE1 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene GASAL1 has 0 variants, skipping\n",
      "[INFO]\tGene PTPRT-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene RBM38-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GNT6 has 0 variants, skipping\n",
      "[INFO]\tGene PKHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXL3-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM122A has 0 variants, skipping\n",
      "[INFO]\tGene ZFC3H1 has 0 variants, skipping\n",
      "[INFO]\tGene NSMCE1 has 0 variants, skipping\n",
      "[INFO]\tGene C6orf118 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF564 has 0 variants, skipping\n",
      "[INFO]\tGene FOLH1B has 0 variants, skipping\n",
      "[INFO]\tGene PRXL2C has 0 variants, skipping\n",
      "[INFO]\tGene RTP1 has 0 variants, skipping\n",
      "[INFO]\tGene OR4A47 has 0 variants, skipping\n",
      "[INFO]\tGene OR6C70 has 0 variants, skipping\n",
      "[INFO]\tGene SLC7A13 has 0 variants, skipping\n",
      "[INFO]\tGene DPPA2 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf31 has 0 variants, skipping\n",
      "[INFO]\tGene CMBL has 0 variants, skipping\n",
      "[INFO]\tGene CYGB has 0 variants, skipping\n",
      "[INFO]\tGene ALKBH3 has 0 variants, skipping\n",
      "[INFO]\tGene LGI3 has 0 variants, skipping\n",
      "[INFO]\tGene GAS2L2 has 0 variants, skipping\n",
      "[INFO]\tGene LGI4 has 0 variants, skipping\n",
      "[INFO]\tGene RHOXF1 has 0 variants, skipping\n",
      "[INFO]\tGene CDC26 has 0 variants, skipping\n",
      "[INFO]\tGene TEX261 has 0 variants, skipping\n",
      "[INFO]\tGene KCNH8 has 0 variants, skipping\n",
      "[INFO]\tGene RFTN2 has 0 variants, skipping\n",
      "[INFO]\tGene HIPK4 has 0 variants, skipping\n",
      "[INFO]\tGene DHRS13 has 0 variants, skipping\n",
      "[INFO]\tGene SLFN13 has 0 variants, skipping\n",
      "[INFO]\tGene TMED6 has 0 variants, skipping\n",
      "[INFO]\tGene TEKT5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF440 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF491 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF441 has 0 variants, skipping\n",
      "[INFO]\tGene CLDND2 has 0 variants, skipping\n",
      "[INFO]\tGene KRT222 has 0 variants, skipping\n",
      "[INFO]\tGene SAMD1 has 0 variants, skipping\n",
      "[INFO]\tGene OTULIN has 0 variants, skipping\n",
      "[INFO]\tGene CARMIL3 has 0 variants, skipping\n",
      "[INFO]\tGene DCAF15 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM29 has 0 variants, skipping\n",
      "[INFO]\tGene SAAL1 has 0 variants, skipping\n",
      "[INFO]\tGene ID2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene EMSLR has 0 variants, skipping\n",
      "[INFO]\tGene LINC02888 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02889 has 0 variants, skipping\n",
      "[INFO]\tGene SAMMSON has 0 variants, skipping\n",
      "[INFO]\tGene B4GALT1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene VIM-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCAT1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00844 has 0 variants, skipping\n",
      "[INFO]\tGene MIR486-2 has 0 variants, skipping\n",
      "[INFO]\tGene MIR8075 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6863 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6807 has 0 variants, skipping\n",
      "[INFO]\tGene MIR6089 has 0 variants, skipping\n",
      "[INFO]\tGene CARMN has 0 variants, skipping\n",
      "[INFO]\tGene CRB2 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100133315 has 0 variants, skipping\n",
      "[INFO]\tGene LGALS1 has 0 variants, skipping\n",
      "[INFO]\tGene GTF3C6 has 0 variants, skipping\n",
      "[INFO]\tGene MARCHF9 has 0 variants, skipping\n",
      "[INFO]\tGene MARS2 has 0 variants, skipping\n",
      "[INFO]\tGene SGF29 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3679 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3678 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3667 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3659 has 0 variants, skipping\n",
      "[INFO]\tGene MIR3619 has 0 variants, skipping\n",
      "[INFO]\tGene PGBD3 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC1 has 0 variants, skipping\n",
      "[INFO]\tGene TEDDM1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01579 has 0 variants, skipping\n",
      "[INFO]\tGene WASIR1 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD45 has 0 variants, skipping\n",
      "[INFO]\tGene KTI12 has 0 variants, skipping\n",
      "[INFO]\tGene LSM14B has 0 variants, skipping\n",
      "[INFO]\tGene IFNL1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNL2 has 0 variants, skipping\n",
      "[INFO]\tGene OR4Q3 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE2 has 0 variants, skipping\n",
      "[INFO]\tGene CACNA2D4 has 0 variants, skipping\n",
      "[INFO]\tGene KCNG4 has 0 variants, skipping\n",
      "[INFO]\tGene MRPL54 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE1 has 0 variants, skipping\n",
      "[INFO]\tGene TFAP2D has 0 variants, skipping\n",
      "[INFO]\tGene SUGP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP22-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP21-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP20-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRT6C has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R36 has 0 variants, skipping\n",
      "[INFO]\tGene SEPTIN7P13 has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724728 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375800 has 0 variants, skipping\n",
      "[INFO]\tGene COL23A1 has 0 variants, skipping\n",
      "[INFO]\tGene SSX7 has 0 variants, skipping\n",
      "[INFO]\tGene DCUN1D3 has 0 variants, skipping\n",
      "[INFO]\tGene KLF17 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4F22 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC105 has 0 variants, skipping\n",
      "[INFO]\tGene EPHX4 has 0 variants, skipping\n",
      "[INFO]\tGene UNC5CL has 0 variants, skipping\n",
      "[INFO]\tGene FAM217A has 0 variants, skipping\n",
      "[INFO]\tGene NYAP1 has 0 variants, skipping\n",
      "[INFO]\tGene RSPH10B has 0 variants, skipping\n",
      "[INFO]\tGene CGAS has 0 variants, skipping\n",
      "[INFO]\tGene VASN has 0 variants, skipping\n",
      "[INFO]\tGene FAM83F has 0 variants, skipping\n",
      "[INFO]\tGene ZBED6CL has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC7B has 0 variants, skipping\n",
      "[INFO]\tGene GLCCI1 has 0 variants, skipping\n",
      "[INFO]\tGene KIF12 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf42 has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB6 has 0 variants, skipping\n",
      "[INFO]\tGene REM2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71D has 0 variants, skipping\n",
      "[INFO]\tGene RNF152 has 0 variants, skipping\n",
      "[INFO]\tGene RFX6 has 0 variants, skipping\n",
      "[INFO]\tGene ANKS6 has 0 variants, skipping\n",
      "[INFO]\tGene KLHDC8B has 0 variants, skipping\n",
      "[INFO]\tGene APLF has 0 variants, skipping\n",
      "[INFO]\tGene CNBD1 has 0 variants, skipping\n",
      "[INFO]\tGene GABRG1 has 0 variants, skipping\n",
      "[INFO]\tGene SCUBE1 has 0 variants, skipping\n",
      "[INFO]\tGene PASD1 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM117B has 0 variants, skipping\n",
      "[INFO]\tGene FGD2 has 0 variants, skipping\n",
      "[INFO]\tGene PRR14L has 0 variants, skipping\n",
      "[INFO]\tGene CT47A11 has 0 variants, skipping\n",
      "[INFO]\tGene LMNTD2 has 0 variants, skipping\n",
      "[INFO]\tGene PGM2L1 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF22 has 0 variants, skipping\n",
      "[INFO]\tGene TRARG1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM59 has 0 variants, skipping\n",
      "[INFO]\tGene SPRR4 has 0 variants, skipping\n",
      "[INFO]\tGene CPO has 0 variants, skipping\n",
      "[INFO]\tGene ATRIP-TREX1 has 0 variants, skipping\n",
      "[INFO]\tGene DIRC1 has 0 variants, skipping\n",
      "[INFO]\tGene LSM11 has 0 variants, skipping\n",
      "[INFO]\tGene TTBK2 has 0 variants, skipping\n",
      "[INFO]\tGene MROH2B has 0 variants, skipping\n",
      "[INFO]\tGene UBN2 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD52 has 0 variants, skipping\n",
      "[INFO]\tGene MYHAS has 0 variants, skipping\n",
      "[INFO]\tGene LINC00407 has 0 variants, skipping\n",
      "[INFO]\tGene LINC00319 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC9A9 has 0 variants, skipping\n",
      "[INFO]\tGene FSIP2 has 0 variants, skipping\n",
      "[INFO]\tGene RNF149 has 0 variants, skipping\n",
      "[INFO]\tGene RIMKLA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF547 has 0 variants, skipping\n",
      "[INFO]\tGene TAS1R2 has 0 variants, skipping\n",
      "[INFO]\tGene IPMK has 0 variants, skipping\n",
      "[INFO]\tGene LOC284898 has 0 variants, skipping\n",
      "[INFO]\tGene CDC42EP1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB106A has 0 variants, skipping\n",
      "[INFO]\tGene PARS2 has 0 variants, skipping\n",
      "[INFO]\tGene UBTD2 has 0 variants, skipping\n",
      "[INFO]\tGene TTC30A has 0 variants, skipping\n",
      "[INFO]\tGene ZNF585B has 0 variants, skipping\n",
      "[INFO]\tGene SYT11 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4C has 0 variants, skipping\n",
      "[INFO]\tGene ORAI3 has 0 variants, skipping\n",
      "[INFO]\tGene ELOVL3 has 0 variants, skipping\n",
      "[INFO]\tGene PIK3AP1 has 0 variants, skipping\n",
      "[INFO]\tGene CLRN3 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf45 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14EP has 0 variants, skipping\n",
      "[INFO]\tGene C12orf54 has 0 variants, skipping\n",
      "[INFO]\tGene SPIC has 0 variants, skipping\n",
      "[INFO]\tGene ACOT4 has 0 variants, skipping\n",
      "[INFO]\tGene EID2B has 0 variants, skipping\n",
      "[INFO]\tGene MAB21L3 has 0 variants, skipping\n",
      "[INFO]\tGene MYOM3 has 0 variants, skipping\n",
      "[INFO]\tGene VN1R3 has 0 variants, skipping\n",
      "[INFO]\tGene C14orf39 has 0 variants, skipping\n",
      "[INFO]\tGene POTED has 0 variants, skipping\n",
      "[INFO]\tGene SPEM2 has 0 variants, skipping\n",
      "[INFO]\tGene OLIG3 has 0 variants, skipping\n",
      "[INFO]\tGene TCF23 has 0 variants, skipping\n",
      "[INFO]\tGene KRT79 has 0 variants, skipping\n",
      "[INFO]\tGene FTMT has 0 variants, skipping\n",
      "[INFO]\tGene FAM171B has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R60 has 0 variants, skipping\n",
      "[INFO]\tGene STX12 has 0 variants, skipping\n",
      "[INFO]\tGene KLC3 has 0 variants, skipping\n",
      "[INFO]\tGene MIDN has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-2 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY13 has 0 variants, skipping\n",
      "[INFO]\tGene IFNE has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R20 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R50 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R19 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R46 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R45 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R31 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R43 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R41 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R40 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R39 has 0 variants, skipping\n",
      "[INFO]\tGene MT1M has 0 variants, skipping\n",
      "[INFO]\tGene S100A7A has 0 variants, skipping\n",
      "[INFO]\tGene NLRP14 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP9 has 0 variants, skipping\n",
      "[INFO]\tGene TAS2R38 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF800 has 0 variants, skipping\n",
      "[INFO]\tGene AGR3 has 0 variants, skipping\n",
      "[INFO]\tGene CHMP4B has 0 variants, skipping\n",
      "[INFO]\tGene PKHD1L1 has 0 variants, skipping\n",
      "[INFO]\tGene SYT9 has 0 variants, skipping\n",
      "[INFO]\tGene LYG2 has 0 variants, skipping\n",
      "[INFO]\tGene KLB has 0 variants, skipping\n",
      "[INFO]\tGene FAM151A has 0 variants, skipping\n",
      "[INFO]\tGene PRR18 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf51 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf54 has 0 variants, skipping\n",
      "[INFO]\tGene PRPS1L1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM181B has 0 variants, skipping\n",
      "[INFO]\tGene CCDC71L has 0 variants, skipping\n",
      "[INFO]\tGene OR7D2 has 0 variants, skipping\n",
      "[INFO]\tGene SPPL2C has 0 variants, skipping\n",
      "[INFO]\tGene ODF3L1 has 0 variants, skipping\n",
      "[INFO]\tGene EXOC8 has 0 variants, skipping\n",
      "[INFO]\tGene SIX5 has 0 variants, skipping\n",
      "[INFO]\tGene C12orf60 has 0 variants, skipping\n",
      "[INFO]\tGene SOWAHA has 0 variants, skipping\n",
      "[INFO]\tGene ZNF792 has 0 variants, skipping\n",
      "[INFO]\tGene SWSAP1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP11-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP8-1 has 0 variants, skipping\n",
      "[INFO]\tGene CHSY3 has 0 variants, skipping\n",
      "[INFO]\tGene PAN3 has 0 variants, skipping\n",
      "[INFO]\tGene KDF1 has 0 variants, skipping\n",
      "[INFO]\tGene PDCL2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM241A has 0 variants, skipping\n",
      "[INFO]\tGene TRAM1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL40 has 0 variants, skipping\n",
      "[INFO]\tGene CR1L has 0 variants, skipping\n",
      "[INFO]\tGene LSINCT5 has 0 variants, skipping\n",
      "[INFO]\tGene TALAM1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR205HG has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER3 has 0 variants, skipping\n",
      "[INFO]\tGene OTOP1 has 0 variants, skipping\n",
      "[INFO]\tGene TUBB8 has 0 variants, skipping\n",
      "[INFO]\tGene PPM1G has 0 variants, skipping\n",
      "[INFO]\tGene SESTD1 has 0 variants, skipping\n",
      "[INFO]\tGene ANGPTL5 has 0 variants, skipping\n",
      "[INFO]\tGene GGTLC1 has 0 variants, skipping\n",
      "[INFO]\tGene SNAI3 has 0 variants, skipping\n",
      "[INFO]\tGene PILRB has 0 variants, skipping\n",
      "[INFO]\tGene IQGAP3 has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GSDMA has 0 variants, skipping\n",
      "[INFO]\tGene NEK8 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A5 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF598 has 0 variants, skipping\n",
      "[INFO]\tGene PTF1A has 0 variants, skipping\n",
      "[INFO]\tGene OTOP2 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD2 has 0 variants, skipping\n",
      "[INFO]\tGene CYP4Z1 has 0 variants, skipping\n",
      "[INFO]\tGene P2RY8 has 0 variants, skipping\n",
      "[INFO]\tGene FADS6 has 0 variants, skipping\n",
      "[INFO]\tGene TONSL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3B-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ROR1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L16 has 0 variants, skipping\n",
      "[INFO]\tGene FARSA-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01259 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01074 has 0 variants, skipping\n",
      "[INFO]\tGene LIMS1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene C5-OT1 has 0 variants, skipping\n",
      "[INFO]\tGene UC.134 has 0 variants, skipping\n",
      "[INFO]\tGene PNKY has 0 variants, skipping\n",
      "[INFO]\tGene DDX11L17 has 0 variants, skipping\n",
      "[INFO]\tGene L3MBTL2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105369306 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105371855 has 0 variants, skipping\n",
      "[INFO]\tGene SCTR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100507551 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101930421 has 0 variants, skipping\n",
      "[INFO]\tGene LOC100422781 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105374042 has 0 variants, skipping\n",
      "[INFO]\tGene HHATL-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR2117HG has 0 variants, skipping\n",
      "[INFO]\tGene LINC02041 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091777 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091776 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01980 has 0 variants, skipping\n",
      "[INFO]\tGene LOC110091768 has 0 variants, skipping\n",
      "[INFO]\tGene PPP1R12A-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ATXN2-AS has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD20A4-ANKRD20A20P has 0 variants, skipping\n",
      "[INFO]\tGene PPIP5K1P1-CATSPER2 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P4-NPIPA8 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P5-LOC105376752 has 0 variants, skipping\n",
      "[INFO]\tGene PKD1P3-NPIPA1 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG14 has 0 variants, skipping\n",
      "[INFO]\tGene PWAR6 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN3 has 0 variants, skipping\n",
      "[INFO]\tGene RNA18SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA5-8SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA28SN4 has 0 variants, skipping\n",
      "[INFO]\tGene RNA45SN4 has 0 variants, skipping\n",
      "[INFO]\tGene MIR302CHG has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP1 has 0 variants, skipping\n",
      "[INFO]\tGene VWCE has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE6 has 0 variants, skipping\n",
      "[INFO]\tGene MASCRNA has 0 variants, skipping\n",
      "[INFO]\tGene PDE4DIPP2 has 0 variants, skipping\n",
      "[INFO]\tGene ACBD7-DCLRE1CP1 has 0 variants, skipping\n",
      "[INFO]\tGene LINC01149 has 0 variants, skipping\n",
      "[INFO]\tGene MORC1-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene ARSD-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene BARX1-DT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00402 has 0 variants, skipping\n",
      "[INFO]\tGene LOC108783654 has 0 variants, skipping\n",
      "[INFO]\tGene LOC108783645 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPERD has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP15-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP13-3 has 0 variants, skipping\n",
      "[INFO]\tGene DMBT1L1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP23-1 has 0 variants, skipping\n",
      "[INFO]\tGene KCNK18 has 0 variants, skipping\n",
      "[INFO]\tGene TMED7 has 0 variants, skipping\n",
      "[INFO]\tGene BMP8A has 0 variants, skipping\n",
      "[INFO]\tGene POLN has 0 variants, skipping\n",
      "[INFO]\tGene H1-7 has 0 variants, skipping\n",
      "[INFO]\tGene DPY19L4 has 0 variants, skipping\n",
      "[INFO]\tGene SLC36A2 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA12 has 0 variants, skipping\n",
      "[INFO]\tGene ANKRD37 has 0 variants, skipping\n",
      "[INFO]\tGene METTL2A has 0 variants, skipping\n",
      "[INFO]\tGene TMEM119 has 0 variants, skipping\n",
      "[INFO]\tGene FOXR1 has 0 variants, skipping\n",
      "[INFO]\tGene ASPHD1 has 0 variants, skipping\n",
      "[INFO]\tGene CENPV has 0 variants, skipping\n",
      "[INFO]\tGene CRTC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBXN2A has 0 variants, skipping\n",
      "[INFO]\tGene ZNRF4 has 0 variants, skipping\n",
      "[INFO]\tGene BCDIN3D has 0 variants, skipping\n",
      "[INFO]\tGene C17orf64 has 0 variants, skipping\n",
      "[INFO]\tGene DNAJC24 has 0 variants, skipping\n",
      "[INFO]\tGene BAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene QSOX2 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-10 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-2 has 0 variants, skipping\n",
      "[INFO]\tGene OGT has 0 variants, skipping\n",
      "[INFO]\tGene CPLX4 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF804B has 0 variants, skipping\n",
      "[INFO]\tGene DEUP1 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4A has 0 variants, skipping\n",
      "[INFO]\tGene HOTTIP has 0 variants, skipping\n",
      "[INFO]\tGene LINC01234 has 0 variants, skipping\n",
      "[INFO]\tGene ANKK1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC5A12 has 0 variants, skipping\n",
      "[INFO]\tGene CIBAR1P2 has 0 variants, skipping\n",
      "[INFO]\tGene GPX6 has 0 variants, skipping\n",
      "[INFO]\tGene FAM98B has 0 variants, skipping\n",
      "[INFO]\tGene KSR2 has 0 variants, skipping\n",
      "[INFO]\tGene LYZL2 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM37 has 0 variants, skipping\n",
      "[INFO]\tGene EML5 has 0 variants, skipping\n",
      "[INFO]\tGene ARRDC4 has 0 variants, skipping\n",
      "[INFO]\tGene CYP26C1 has 0 variants, skipping\n",
      "[INFO]\tGene PXDC1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC10 has 0 variants, skipping\n",
      "[INFO]\tGene SLC16A13 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM10L2A has 0 variants, skipping\n",
      "[INFO]\tGene CHCHD1 has 0 variants, skipping\n",
      "[INFO]\tGene FIBIN has 0 variants, skipping\n",
      "[INFO]\tGene CAPRIN1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF784 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM81 has 0 variants, skipping\n",
      "[INFO]\tGene RPRML has 0 variants, skipping\n",
      "[INFO]\tGene FITM1 has 0 variants, skipping\n",
      "[INFO]\tGene E2F7 has 0 variants, skipping\n",
      "[INFO]\tGene LURAP1L has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP26-1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM47A has 0 variants, skipping\n",
      "[INFO]\tGene EZHIP has 0 variants, skipping\n",
      "[INFO]\tGene MBLAC1 has 0 variants, skipping\n",
      "[INFO]\tGene MBLAC2 has 0 variants, skipping\n",
      "[INFO]\tGene UBL4B has 0 variants, skipping\n",
      "[INFO]\tGene LRRN4CL has 0 variants, skipping\n",
      "[INFO]\tGene IER5L has 0 variants, skipping\n",
      "[INFO]\tGene IQCF2 has 0 variants, skipping\n",
      "[INFO]\tGene ASCL4 has 0 variants, skipping\n",
      "[INFO]\tGene GGPS1 has 0 variants, skipping\n",
      "[INFO]\tGene LDHD has 0 variants, skipping\n",
      "[INFO]\tGene MDGA1 has 0 variants, skipping\n",
      "[INFO]\tGene LPCAT4 has 0 variants, skipping\n",
      "[INFO]\tGene HS3ST5 has 0 variants, skipping\n",
      "[INFO]\tGene CMYA5 has 0 variants, skipping\n",
      "[INFO]\tGene FAM71A has 0 variants, skipping\n",
      "[INFO]\tGene CRYBG3 has 0 variants, skipping\n",
      "[INFO]\tGene COG7 has 0 variants, skipping\n",
      "[INFO]\tGene PHLPP1 has 0 variants, skipping\n",
      "[INFO]\tGene B3GLCT has 0 variants, skipping\n",
      "[INFO]\tGene KIF24 has 0 variants, skipping\n",
      "[INFO]\tGene IDO2 has 0 variants, skipping\n",
      "[INFO]\tGene C16orf74 has 0 variants, skipping\n",
      "[INFO]\tGene FAM151B has 0 variants, skipping\n",
      "[INFO]\tGene LYPD2 has 0 variants, skipping\n",
      "[INFO]\tGene OR2K2 has 0 variants, skipping\n",
      "[INFO]\tGene SPACA5 has 0 variants, skipping\n",
      "[INFO]\tGene SNHG32 has 0 variants, skipping\n",
      "[INFO]\tGene SIGLEC10-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene SYPL2 has 0 variants, skipping\n",
      "[INFO]\tGene CDRT15P3 has 0 variants, skipping\n",
      "[INFO]\tGene TDRG1 has 0 variants, skipping\n",
      "[INFO]\tGene BIVM-ERCC5 has 0 variants, skipping\n",
      "[INFO]\tGene CTXN1 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107986192 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267991 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987142 has 0 variants, skipping\n",
      "[INFO]\tGene YY2 has 0 variants, skipping\n",
      "[INFO]\tGene MAMDC4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR34 has 0 variants, skipping\n",
      "[INFO]\tGene OR10A4 has 0 variants, skipping\n",
      "[INFO]\tGene LMOD2 has 0 variants, skipping\n",
      "[INFO]\tGene GTF2H5 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM105 has 0 variants, skipping\n",
      "[INFO]\tGene C5orf46 has 0 variants, skipping\n",
      "[INFO]\tGene GPR152 has 0 variants, skipping\n",
      "[INFO]\tGene SPAG17 has 0 variants, skipping\n",
      "[INFO]\tGene SCGB1D4 has 0 variants, skipping\n",
      "[INFO]\tGene GOLGA1 has 0 variants, skipping\n",
      "[INFO]\tGene SALL3 has 0 variants, skipping\n",
      "[INFO]\tGene RAB39B has 0 variants, skipping\n",
      "[INFO]\tGene OR10W1 has 0 variants, skipping\n",
      "[INFO]\tGene C10orf99 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375303 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf139 has 0 variants, skipping\n",
      "[INFO]\tGene PTCHD4 has 0 variants, skipping\n",
      "[INFO]\tGene LCNL1 has 0 variants, skipping\n",
      "[INFO]\tGene DEFB132 has 0 variants, skipping\n",
      "[INFO]\tGene TEX19 has 0 variants, skipping\n",
      "[INFO]\tGene NAIF1 has 0 variants, skipping\n",
      "[INFO]\tGene SLC10A6 has 0 variants, skipping\n",
      "[INFO]\tGene GNPNAT1 has 0 variants, skipping\n",
      "[INFO]\tGene BTNL3 has 0 variants, skipping\n",
      "[INFO]\tGene OR2C3 has 0 variants, skipping\n",
      "[INFO]\tGene CCZ1B has 0 variants, skipping\n",
      "[INFO]\tGene RNF148 has 0 variants, skipping\n",
      "[INFO]\tGene LRRC56 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02817 has 0 variants, skipping\n",
      "[INFO]\tGene B4GALNT3 has 0 variants, skipping\n",
      "[INFO]\tGene DIP2B has 0 variants, skipping\n",
      "[INFO]\tGene MUC19 has 0 variants, skipping\n",
      "[INFO]\tGene FOXI2 has 0 variants, skipping\n",
      "[INFO]\tGene FLJ44635 has 0 variants, skipping\n",
      "[INFO]\tGene PADI6 has 0 variants, skipping\n",
      "[INFO]\tGene C1QTNF8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC2A7 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA31D3 has 0 variants, skipping\n",
      "[INFO]\tGene ALKAL1 has 0 variants, skipping\n",
      "[INFO]\tGene GFRAL has 0 variants, skipping\n",
      "[INFO]\tGene GBP7 has 0 variants, skipping\n",
      "[INFO]\tGene TEX43 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS11F has 0 variants, skipping\n",
      "[INFO]\tGene RNF207 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF324B has 0 variants, skipping\n",
      "[INFO]\tGene IGFL3 has 0 variants, skipping\n",
      "[INFO]\tGene CPXM2 has 0 variants, skipping\n",
      "[INFO]\tGene ABHD15 has 0 variants, skipping\n",
      "[INFO]\tGene CATSPER4 has 0 variants, skipping\n",
      "[INFO]\tGene QRFP has 0 variants, skipping\n",
      "[INFO]\tGene TREML4 has 0 variants, skipping\n",
      "[INFO]\tGene UTS2B has 0 variants, skipping\n",
      "[INFO]\tGene ARSK has 0 variants, skipping\n",
      "[INFO]\tGene FLJ20712 has 0 variants, skipping\n",
      "[INFO]\tGene LINC02591 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18C has 0 variants, skipping\n",
      "[INFO]\tGene GALNT10 has 0 variants, skipping\n",
      "[INFO]\tGene KLHL17 has 0 variants, skipping\n",
      "[INFO]\tGene H2AC2P has 0 variants, skipping\n",
      "[INFO]\tGene KCTD8 has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN3 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM17 has 0 variants, skipping\n",
      "[INFO]\tGene KCTD4 has 0 variants, skipping\n",
      "[INFO]\tGene KLRG2 has 0 variants, skipping\n",
      "[INFO]\tGene FAM174A has 0 variants, skipping\n",
      "[INFO]\tGene LRIT3 has 0 variants, skipping\n",
      "[INFO]\tGene ATP13A5 has 0 variants, skipping\n",
      "[INFO]\tGene CTAGE4 has 0 variants, skipping\n",
      "[INFO]\tGene PRR20A has 0 variants, skipping\n",
      "[INFO]\tGene EMB has 0 variants, skipping\n",
      "[INFO]\tGene GOLT1A has 0 variants, skipping\n",
      "[INFO]\tGene FOXR2 has 0 variants, skipping\n",
      "[INFO]\tGene APOOL has 0 variants, skipping\n",
      "[INFO]\tGene LONRF2 has 0 variants, skipping\n",
      "[INFO]\tGene RSBN1L has 0 variants, skipping\n",
      "[INFO]\tGene NRK has 0 variants, skipping\n",
      "[INFO]\tGene FAM171A2 has 0 variants, skipping\n",
      "[INFO]\tGene KANK3 has 0 variants, skipping\n",
      "[INFO]\tGene TPRX1 has 0 variants, skipping\n",
      "[INFO]\tGene NKPD1 has 0 variants, skipping\n",
      "[INFO]\tGene FAM83H has 0 variants, skipping\n",
      "[INFO]\tGene TPRG1 has 0 variants, skipping\n",
      "[INFO]\tGene CA13 has 0 variants, skipping\n",
      "[INFO]\tGene AMIGO3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-12 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-4 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP12-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-3 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-8 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-5 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-11 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-1 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-7 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-9 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-6 has 0 variants, skipping\n",
      "[INFO]\tGene KRTAP10-4 has 0 variants, skipping\n",
      "[INFO]\tGene ZC3H6 has 0 variants, skipping\n",
      "[INFO]\tGene SLC27A1 has 0 variants, skipping\n",
      "[INFO]\tGene LRRK2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF775 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD8 has 0 variants, skipping\n",
      "[INFO]\tGene OOSP2 has 0 variants, skipping\n",
      "[INFO]\tGene LVRN has 0 variants, skipping\n",
      "[INFO]\tGene MAGEB18 has 0 variants, skipping\n",
      "[INFO]\tGene FAT1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM73 has 0 variants, skipping\n",
      "[INFO]\tGene MRGPRD has 0 variants, skipping\n",
      "[INFO]\tGene CYSRT1 has 0 variants, skipping\n",
      "[INFO]\tGene RNF175 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF660 has 0 variants, skipping\n",
      "[INFO]\tGene MIR655 has 0 variants, skipping\n",
      "[INFO]\tGene LOC114841040 has 0 variants, skipping\n",
      "[INFO]\tGene DGAT2L6 has 0 variants, skipping\n",
      "[INFO]\tGene ITIH6 has 0 variants, skipping\n",
      "[INFO]\tGene C11orf53 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF710 has 0 variants, skipping\n",
      "[INFO]\tGene KIF7 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC172 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC2 has 0 variants, skipping\n",
      "[INFO]\tGene STIMATE has 0 variants, skipping\n",
      "[INFO]\tGene DPPA3 has 0 variants, skipping\n",
      "[INFO]\tGene PRR19 has 0 variants, skipping\n",
      "[INFO]\tGene POLQ has 0 variants, skipping\n",
      "[INFO]\tGene LHFPL3 has 0 variants, skipping\n",
      "[INFO]\tGene SIAH3 has 0 variants, skipping\n",
      "[INFO]\tGene PHLDB3 has 0 variants, skipping\n",
      "[INFO]\tGene ENHO has 0 variants, skipping\n",
      "[INFO]\tGene KLHL30 has 0 variants, skipping\n",
      "[INFO]\tGene NHLRC1 has 0 variants, skipping\n",
      "[INFO]\tGene FUNDC1 has 0 variants, skipping\n",
      "[INFO]\tGene TMEM86B has 0 variants, skipping\n",
      "[INFO]\tGene NELFCD has 0 variants, skipping\n",
      "[INFO]\tGene NDUFAF3 has 0 variants, skipping\n",
      "[INFO]\tGene CARM1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4L3 has 0 variants, skipping\n",
      "[INFO]\tGene AQP12A has 0 variants, skipping\n",
      "[INFO]\tGene SYT10 has 0 variants, skipping\n",
      "[INFO]\tGene LDLR-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC80 has 0 variants, skipping\n",
      "[INFO]\tGene NANOS1 has 0 variants, skipping\n",
      "[INFO]\tGene LNC-LBCS has 0 variants, skipping\n",
      "[INFO]\tGene UST-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene COSMOC has 0 variants, skipping\n",
      "[INFO]\tGene LASTR has 0 variants, skipping\n",
      "[INFO]\tGene FAM230D has 0 variants, skipping\n",
      "[INFO]\tGene FAM230J has 0 variants, skipping\n",
      "[INFO]\tGene FAM230A has 0 variants, skipping\n",
      "[INFO]\tGene FAM230E has 0 variants, skipping\n",
      "[INFO]\tGene FAM230F has 0 variants, skipping\n",
      "[INFO]\tGene FAM230G has 0 variants, skipping\n",
      "[INFO]\tGene FAM230B has 0 variants, skipping\n",
      "[INFO]\tGene FAM230H has 0 variants, skipping\n",
      "[INFO]\tGene FAM230I has 0 variants, skipping\n",
      "[INFO]\tGene HSPB8 has 0 variants, skipping\n",
      "[INFO]\tGene GAPDHS has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-8 has 0 variants, skipping\n",
      "[INFO]\tGene OPTC has 0 variants, skipping\n",
      "[INFO]\tGene CLEC4E has 0 variants, skipping\n",
      "[INFO]\tGene GUCA1ANB has 0 variants, skipping\n",
      "[INFO]\tGene HCAR3 has 0 variants, skipping\n",
      "[INFO]\tGene MCM6 has 0 variants, skipping\n",
      "[INFO]\tGene LAGE3 has 0 variants, skipping\n",
      "[INFO]\tGene MC5R has 0 variants, skipping\n",
      "[INFO]\tGene CLPP has 0 variants, skipping\n",
      "[INFO]\tGene MC4R has 0 variants, skipping\n",
      "[INFO]\tGene MAT2A has 0 variants, skipping\n",
      "[INFO]\tGene MANF has 0 variants, skipping\n",
      "[INFO]\tGene MANBA has 0 variants, skipping\n",
      "[INFO]\tGene MAN1A1 has 0 variants, skipping\n",
      "[INFO]\tGene MIR466 has 0 variants, skipping\n",
      "[INFO]\tGene GHET1 has 0 variants, skipping\n",
      "[INFO]\tGene GACAT3 has 0 variants, skipping\n",
      "[INFO]\tGene MHRT has 0 variants, skipping\n",
      "[INFO]\tGene LINC00504 has 0 variants, skipping\n",
      "[INFO]\tGene MAILR has 0 variants, skipping\n",
      "[INFO]\tGene USH2A-AS2 has 0 variants, skipping\n",
      "[INFO]\tGene PRDX6-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene FOXCUT has 0 variants, skipping\n",
      "[INFO]\tGene LINC01048 has 0 variants, skipping\n",
      "[INFO]\tGene TBCEL-TECTA has 0 variants, skipping\n",
      "[INFO]\tGene LOC116435278 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC144NL has 0 variants, skipping\n",
      "[INFO]\tGene LOC112267934 has 0 variants, skipping\n",
      "[INFO]\tGene LOC115308161 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf129 has 0 variants, skipping\n",
      "[INFO]\tGene UBE2Q2L has 0 variants, skipping\n",
      "[INFO]\tGene NUDT3 has 0 variants, skipping\n",
      "[INFO]\tGene H3Y2 has 0 variants, skipping\n",
      "[INFO]\tGene MEMO1 has 0 variants, skipping\n",
      "[INFO]\tGene TRIM67-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFAP53 has 0 variants, skipping\n",
      "[INFO]\tGene GLYATL2 has 0 variants, skipping\n",
      "[INFO]\tGene FBXW7-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene NFE4 has 0 variants, skipping\n",
      "[INFO]\tGene FAM197Y2 has 0 variants, skipping\n",
      "[INFO]\tGene PDCD6-AHRR has 0 variants, skipping\n",
      "[INFO]\tGene SREBF2-AS1 has 0 variants, skipping\n",
      "[INFO]\tGene CFDP1 has 0 variants, skipping\n",
      "[INFO]\tGene IFI30 has 0 variants, skipping\n",
      "[INFO]\tGene TIMM17A has 0 variants, skipping\n",
      "[INFO]\tGene TACC3 has 0 variants, skipping\n",
      "[INFO]\tGene HMG20B has 0 variants, skipping\n",
      "[INFO]\tGene LOC107987389 has 0 variants, skipping\n",
      "[INFO]\tGene LNCDAT has 0 variants, skipping\n",
      "[INFO]\tGene SLC26A10 has 0 variants, skipping\n",
      "[INFO]\tGene LCA5L has 0 variants, skipping\n",
      "[INFO]\tGene TFF3 has 0 variants, skipping\n",
      "[INFO]\tGene C22orf39 has 0 variants, skipping\n",
      "[INFO]\tGene MCHR1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC134 has 0 variants, skipping\n",
      "[INFO]\tGene KIAA0930 has 0 variants, skipping\n",
      "[INFO]\tGene TTC38 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3B has 0 variants, skipping\n",
      "[INFO]\tGene UBE2FP1 has 0 variants, skipping\n",
      "[INFO]\tGene RPP14 has 0 variants, skipping\n",
      "[INFO]\tGene CD47 has 0 variants, skipping\n",
      "[INFO]\tGene CD200R1L has 0 variants, skipping\n",
      "[INFO]\tGene POPDC2 has 0 variants, skipping\n",
      "[INFO]\tGene COX17 has 0 variants, skipping\n",
      "[INFO]\tGene SLC12A8 has 0 variants, skipping\n",
      "[INFO]\tGene SLC41A3 has 0 variants, skipping\n",
      "[INFO]\tGene C3orf80 has 0 variants, skipping\n",
      "[INFO]\tGene NSG1 has 0 variants, skipping\n",
      "[INFO]\tGene UGT2B17 has 0 variants, skipping\n",
      "[INFO]\tGene BMP3 has 0 variants, skipping\n",
      "[INFO]\tGene TIGD2 has 0 variants, skipping\n",
      "[INFO]\tGene C4orf54 has 0 variants, skipping\n",
      "[INFO]\tGene SMIM43 has 0 variants, skipping\n",
      "[INFO]\tGene ACSL1 has 0 variants, skipping\n",
      "[INFO]\tGene DNAH5 has 0 variants, skipping\n",
      "[INFO]\tGene LOC728178 has 0 variants, skipping\n",
      "[INFO]\tGene ANXA2R has 0 variants, skipping\n",
      "[INFO]\tGene CCDC192 has 0 variants, skipping\n",
      "[INFO]\tGene SPATA24 has 0 variants, skipping\n",
      "[INFO]\tGene PCDHB3 has 0 variants, skipping\n",
      "[INFO]\tGene PROP1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF311 has 0 variants, skipping\n",
      "[INFO]\tGene HLA-G has 0 variants, skipping\n",
      "[INFO]\tGene TOMM6 has 0 variants, skipping\n",
      "[INFO]\tGene HCRTR2 has 0 variants, skipping\n",
      "[INFO]\tGene MFSD4B has 0 variants, skipping\n",
      "[INFO]\tGene LOC102724646 has 0 variants, skipping\n",
      "[INFO]\tGene CCN2 has 0 variants, skipping\n",
      "[INFO]\tGene THBS2 has 0 variants, skipping\n",
      "[INFO]\tGene AMZ1 has 0 variants, skipping\n",
      "[INFO]\tGene CARD11 has 0 variants, skipping\n",
      "[INFO]\tGene TOMM7 has 0 variants, skipping\n",
      "[INFO]\tGene HOXA3 has 0 variants, skipping\n",
      "[INFO]\tGene ITPRID1 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE21 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE7P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE8 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE11 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE15 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE9 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE13 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE14 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE10P has 0 variants, skipping\n",
      "[INFO]\tGene CLDN4 has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE12P has 0 variants, skipping\n",
      "[INFO]\tGene SPDYE17 has 0 variants, skipping\n",
      "[INFO]\tGene SEMA3D has 0 variants, skipping\n",
      "[INFO]\tGene FAM237B has 0 variants, skipping\n",
      "[INFO]\tGene LRRD1 has 0 variants, skipping\n",
      "[INFO]\tGene BHLHA15 has 0 variants, skipping\n",
      "[INFO]\tGene SYPL1 has 0 variants, skipping\n",
      "[INFO]\tGene GPR85 has 0 variants, skipping\n",
      "[INFO]\tGene C7orf77 has 0 variants, skipping\n",
      "[INFO]\tGene TNPO3 has 0 variants, skipping\n",
      "[INFO]\tGene LOC389602 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101927506 has 0 variants, skipping\n",
      "[INFO]\tGene TDRP has 0 variants, skipping\n",
      "[INFO]\tGene LOC105377785 has 0 variants, skipping\n",
      "[INFO]\tGene LOC101928095 has 0 variants, skipping\n",
      "[INFO]\tGene NKX2-6 has 0 variants, skipping\n",
      "[INFO]\tGene C8orf86 has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375938 has 0 variants, skipping\n",
      "[INFO]\tGene MSC has 0 variants, skipping\n",
      "[INFO]\tGene VPS13B has 0 variants, skipping\n",
      "[INFO]\tGene LOC105375683 has 0 variants, skipping\n",
      "[INFO]\tGene IQANK1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF517 has 0 variants, skipping\n",
      "[INFO]\tGene FOXD4 has 0 variants, skipping\n",
      "[INFO]\tGene TUSC1 has 0 variants, skipping\n",
      "[INFO]\tGene CCDC180 has 0 variants, skipping\n",
      "[INFO]\tGene C9orf152 has 0 variants, skipping\n",
      "[INFO]\tGene GGTA1 has 0 variants, skipping\n",
      "[INFO]\tGene TTLL11 has 0 variants, skipping\n",
      "[INFO]\tGene GPR21 has 0 variants, skipping\n",
      "[INFO]\tGene PRRC2B has 0 variants, skipping\n",
      "[INFO]\tGene LRRC26 has 0 variants, skipping\n",
      "[INFO]\tGene MED14OS has 0 variants, skipping\n",
      "[INFO]\tGene CENPVL3 has 0 variants, skipping\n",
      "[INFO]\tGene ZXDB has 0 variants, skipping\n",
      "[INFO]\tGene ZXDA has 0 variants, skipping\n",
      "[INFO]\tGene GPR174 has 0 variants, skipping\n",
      "[INFO]\tGene PDZD4 has 0 variants, skipping\n",
      "[INFO]\tGene RBMY1F has 0 variants, skipping\n",
      "[INFO]\tGene LRRC41 has 0 variants, skipping\n",
      "[INFO]\tGene PGLYRP3 has 0 variants, skipping\n",
      "[INFO]\tGene MR1 has 0 variants, skipping\n",
      "[INFO]\tGene IL20 has 0 variants, skipping\n",
      "[INFO]\tGene HS6ST3 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF839 has 0 variants, skipping\n",
      "[INFO]\tGene TEX9 has 0 variants, skipping\n",
      "[INFO]\tGene MEGF11 has 0 variants, skipping\n",
      "[INFO]\tGene PEAK1 has 0 variants, skipping\n",
      "[INFO]\tGene USP36 has 0 variants, skipping\n",
      "[INFO]\tGene IRS1 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF217 has 0 variants, skipping\n",
      "[INFO]\tGene IFRD2 has 0 variants, skipping\n",
      "[INFO]\tGene ALG1L has 0 variants, skipping\n",
      "[INFO]\tGene GRK7 has 0 variants, skipping\n",
      "[INFO]\tGene IGSF10 has 0 variants, skipping\n",
      "[INFO]\tGene NKX6-1 has 0 variants, skipping\n",
      "[INFO]\tGene ARL14EPL has 0 variants, skipping\n",
      "[INFO]\tGene AADACL4 has 0 variants, skipping\n",
      "[INFO]\tGene MAN1C1 has 0 variants, skipping\n",
      "[INFO]\tGene TAFA3 has 0 variants, skipping\n",
      "[INFO]\tGene CLEC18B has 0 variants, skipping\n",
      "[INFO]\tGene GFY has 0 variants, skipping\n",
      "[INFO]\tGene LSR has 0 variants, skipping\n",
      "[INFO]\tGene IL37 has 0 variants, skipping\n",
      "[INFO]\tGene TMC2 has 0 variants, skipping\n",
      "[INFO]\tGene ASIP has 0 variants, skipping\n",
      "[INFO]\tGene CT45A10 has 0 variants, skipping\n",
      "[INFO]\tGene LOC112268136 has 0 variants, skipping\n",
      "[INFO]\tGene SRGAP2B has 0 variants, skipping\n",
      "[INFO]\tGene FAM72A has 0 variants, skipping\n",
      "[INFO]\tGene IL17D has 0 variants, skipping\n",
      "[INFO]\tGene MTMR6 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF439 has 0 variants, skipping\n",
      "[INFO]\tGene CGB7 has 0 variants, skipping\n",
      "[INFO]\tGene FAM237A has 0 variants, skipping\n",
      "[INFO]\tGene LAP3 has 0 variants, skipping\n",
      "[INFO]\tGene PTP4A1 has 0 variants, skipping\n",
      "[INFO]\tGene GHRH has 0 variants, skipping\n",
      "[INFO]\tGene GPRIN2 has 0 variants, skipping\n",
      "[INFO]\tGene LANCL2 has 0 variants, skipping\n",
      "[INFO]\tGene ECMXP has 0 variants, skipping\n",
      "[INFO]\tGene NBPF15 has 0 variants, skipping\n",
      "[INFO]\tGene CNGB3 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF11 has 0 variants, skipping\n",
      "[INFO]\tGene PRSS33 has 0 variants, skipping\n",
      "[INFO]\tGene NLRP11 has 0 variants, skipping\n",
      "[INFO]\tGene DDX27 has 0 variants, skipping\n",
      "[INFO]\tGene NBPF6 has 0 variants, skipping\n",
      "[INFO]\tGene DNASE2B has 0 variants, skipping\n",
      "[INFO]\tGene RBM15 has 0 variants, skipping\n",
      "[INFO]\tGene ODF3L2 has 0 variants, skipping\n",
      "[INFO]\tGene ZNF677 has 0 variants, skipping\n",
      "[INFO]\tGene STARD7 has 0 variants, skipping\n",
      "[INFO]\tGene LOC107985532 has 0 variants, skipping\n",
      "[INFO]\tGene ODAM has 0 variants, skipping\n",
      "[INFO]\tGene ZNF608 has 0 variants, skipping\n",
      "[INFO]\tGene OR2J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR14J1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5V1 has 0 variants, skipping\n",
      "[INFO]\tGene OR11A1 has 0 variants, skipping\n",
      "[INFO]\tGene SMLR1 has 0 variants, skipping\n",
      "[INFO]\tGene OR9A4 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A1 has 0 variants, skipping\n",
      "[INFO]\tGene IFNK has 0 variants, skipping\n",
      "[INFO]\tGene MAP10 has 0 variants, skipping\n",
      "[INFO]\tGene OR52E5 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T2 has 0 variants, skipping\n",
      "[INFO]\tGene OR5AN1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10G8 has 0 variants, skipping\n",
      "[INFO]\tGene OR8B3 has 0 variants, skipping\n",
      "[INFO]\tGene PSMD8 has 0 variants, skipping\n",
      "[INFO]\tGene ZSCAN5B has 0 variants, skipping\n",
      "[INFO]\tGene OR2A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A12 has 0 variants, skipping\n",
      "[INFO]\tGene OR2A14 has 0 variants, skipping\n",
      "[INFO]\tGene OR1L6 has 0 variants, skipping\n",
      "[INFO]\tGene TMPRSS9 has 0 variants, skipping\n",
      "[INFO]\tGene LOC390877 has 0 variants, skipping\n",
      "[INFO]\tGene FAT2 has 0 variants, skipping\n",
      "[INFO]\tGene OR52K1 has 0 variants, skipping\n",
      "[INFO]\tGene OR51H1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5T1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5M1 has 0 variants, skipping\n",
      "[INFO]\tGene OR8G5 has 0 variants, skipping\n",
      "[INFO]\tGene PABPN1L has 0 variants, skipping\n",
      "[INFO]\tGene WDR36 has 0 variants, skipping\n",
      "[INFO]\tGene OR51G2 has 0 variants, skipping\n",
      "[INFO]\tGene OR51L1 has 0 variants, skipping\n",
      "[INFO]\tGene OR52A5 has 0 variants, skipping\n",
      "[INFO]\tGene OR9G1 has 0 variants, skipping\n",
      "[INFO]\tGene OR1S1 has 0 variants, skipping\n",
      "[INFO]\tGene OR5B12 has 0 variants, skipping\n",
      "[INFO]\tGene OR5A1 has 0 variants, skipping\n",
      "[INFO]\tGene OR10D3 has 0 variants, skipping\n",
      "[INFO]\tGene SIRT4 has 0 variants, skipping\n",
      "[INFO]\tGene PABPC3 has 0 variants, skipping\n",
      "[INFO]\tGene KCNE4 has 0 variants, skipping\n",
      "[INFO]\tAnalyzed [ 15 ] variants from [ 28307 ] genes/regions\n",
      "[INFO]\tAnalysis ends at: Fri Feb  5 14:50:27 2021\n",
      "[INFO]\tAnalysis took 1 seconds\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "export PATH=$PATH:~/bin/executable\n",
    "rvtest --noweb --inVcf RIC3_CODING_KB.vcf.gz --pheno covariateFile.txt --pheno-name PHENO --covar covariateFile.txt --covar-name SEX,AGE,PC1,PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10 --kernel skat,skato --burden cmc,zeggini,mb,fp,cmcWald --geneFile refFlat.txt --freqUpper 0.03 --out AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download files:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMC.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  321.0 B/  321.0 B]                                                \n",
      "Operation completed over 1 objects/321.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMCWald.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  3.6 KiB/  3.6 KiB]                                                \n",
      "Operation completed over 1 objects/3.6 KiB.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMC.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  318.0 B/  318.0 B]                                                \n",
      "Operation completed over 1 objects/318.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMCWald.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  3.6 KiB/  3.6 KiB]                                                \n",
      "Operation completed over 1 objects/3.6 KiB.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Fp.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  304.0 B/  304.0 B]                                                \n",
      "Operation completed over 1 objects/304.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.log [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  1.2 MiB/  1.2 MiB]                                                \n",
      "Operation completed over 1 objects/1.2 MiB.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.MadsonBrowning.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  378.0 B/  378.0 B]                                                \n",
      "Operation completed over 1 objects/378.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Skat.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  403.0 B/  403.0 B]                                                \n",
      "Operation completed over 1 objects/403.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.SkatO.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  319.0 B/  319.0 B]                                                \n",
      "Operation completed over 1 objects/319.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Zeggini.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  302.0 B/  302.0 B]                                                \n",
      "Operation completed over 1 objects/302.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Fp.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  305.0 B/  305.0 B]                                                \n",
      "Operation completed over 1 objects/305.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.log [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  1.2 MiB/  1.2 MiB]                                                \n",
      "Operation completed over 1 objects/1.2 MiB.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.MadsonBrowning.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  380.0 B/  380.0 B]                                                \n",
      "Operation completed over 1 objects/380.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Skat.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  403.0 B/  403.0 B]                                                \n",
      "Operation completed over 1 objects/403.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.SkatO.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  320.0 B/  320.0 B]                                                \n",
      "Operation completed over 1 objects/320.0 B.                                      \n",
      "Copying file://./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Zeggini.assoc [Content-Type=application/octet-stream]...\n",
      "/ [1 files][  305.0 B/  305.0 B]                                                \n",
      "Operation completed over 1 objects/305.0 B.                                      \n"
     ]
    }
   ],
   "source": [
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMC.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMCWald.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMC.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMCWald.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Fp.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.log gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.MadsonBrowning.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Skat.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.SkatO.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Zeggini.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Fp.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.log gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.MadsonBrowning.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Skat.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.SkatO.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf\n",
    "!gsutil cp ./AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Zeggini.assoc gs://fc-secure-d273a1b3-b576-476f-995b-1b251495c9bf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Remove non-needed files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "=1.13.0\n",
      "afri_add.txt\n",
      "AFRICA_confirmation_plot.pdf\n",
      "afrio.txt\n",
      "afri.txt\n",
      "after_gender3.bed\n",
      "after_gender3.bim\n",
      "after_gender3.fam\n",
      "after_gender3.hh\n",
      "after_gender3.log\n",
      "after_gender4.bed\n",
      "after_gender4.bim\n",
      "after_gender4.fam\n",
      "after_gender4.hh\n",
      "after_gender4.log\n",
      "all_outliers.txt\n",
      "AMPPD_after_gender.bed\n",
      "AMPPD_after_gender.bim\n",
      "AMPPD_after_gender.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap.bed\n",
      "AMPPD_after_gender_heterozyg_hapmap.bim\n",
      "AMPPD_after_gender_heterozyg_hapmap.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap.log\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.bed\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.bim\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat.log\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bed\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bim\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind.log\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bed\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bim\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.log\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bed\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bim\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.fam\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.log\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.hh\n",
      "AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.log\n",
      "AMPPD_after_gender.hh\n",
      "AMPPD_after_gender.log\n",
      "AMPPD_after_heterozyg.bed\n",
      "AMPPD_after_heterozyg.bim\n",
      "AMPPD_after_heterozyg.fam\n",
      "AMPPD_after_heterozyg.hh\n",
      "AMPPD_after_heterozyg.log\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMC.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.CMCWald.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMC.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.CMCWald.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Fp.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.log\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.MadsonBrowning.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Skat.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.SkatO.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB_CODING.Zeggini.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Fp.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.log\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.MadsonBrowning.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Skat.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.SkatO.assoc\n",
      "AMP_PD_BURDEN.RIC3.WGS.maf003_KB.Zeggini.assoc\n",
      "AMPPD_call_rate.bed\n",
      "AMPPD_call_rate.bim\n",
      "AMPPD_call_rate.fam\n",
      "AMPPD_call_rate.hh\n",
      "AMPPD_call_rate.log\n",
      "AMPPD_FILTERED.bed\n",
      "AMPPD_FILTERED.bim\n",
      "AMPPD_FILTERED.fam\n",
      "AMPPD_FILTERED.RIC3_Annovar\n",
      "AMPPD_FILTERED.RIC3.bed\n",
      "AMPPD_FILTERED.RIC3.bim\n",
      "AMPPD_FILTERED.RIC3.fam\n",
      "AMPPD_FILTERED.RIC3.log\n",
      "AMPPD_FILTERED.RIC3.vcf.gz\n",
      "AMPPD_FILTERED.RIC3.vcf.gz.tbi\n",
      "AMPPD_updated_pheno.bed\n",
      "AMPPD_updated_pheno.bim\n",
      "AMPPD_updated_pheno.fam\n",
      "AMPPD_updated_pheno.hh\n",
      "AMPPD_updated_pheno.log\n",
      "asia_add.txt\n",
      "ASIA_confirmation_plot.pdf\n",
      "asiao.txt\n",
      "asia.txt\n",
      "call_rates.hh\n",
      "call_rates.imiss\n",
      "call_rates.lmiss\n",
      "call_rates.log\n",
      "covariateFile.txt\n",
      "eur_add.txt\n",
      "EURO_confirmation_plot.pdf\n",
      "euro.txt\n",
      "eur.txt\n",
      "EXAMPLE_UNIMPUTED.bed\n",
      "EXAMPLE_UNIMPUTED.bim\n",
      "EXAMPLE_UNIMPUTED.fam\n",
      "EXAMPLE_UNIMPUTED.log\n",
      "EXAMPLE_UNIMPUTED.PCA.eigenval\n",
      "EXAMPLE_UNIMPUTED.PCA.eigenvec\n",
      "EXAMPLE_UNIMPUTED.PCA.log\n",
      "exclusion_regions_hg38.txt\n",
      "facets\n",
      "gender_check1.hh\n",
      "gender_check1.log\n",
      "gender_check1.sexcheck\n",
      "gender_check2.hh\n",
      "gender_check2.log\n",
      "gender_check2.sexcheck\n",
      "GENDER_FAILURES.txt\n",
      "GRM_matrix_0125.grm.bin\n",
      "GRM_matrix_0125.grm.id\n",
      "GRM_matrix_0125.grm.N.bin\n",
      "GRM_matrix_0125.log\n",
      "GRM_matrix.grm.bin\n",
      "GRM_matrix.grm.id\n",
      "GRM_matrix.grm.N.bin\n",
      "GRM_matrix.log\n",
      "HAPLOTYPE_TEST_MISSING_SNPS.txt\n",
      "hapmap3_bin_snplis.bed\n",
      "hapmap3_bin_snplis.bim\n",
      "hapmap3_bin_snplis.fam\n",
      "hapmap3_bin_snplis.hh\n",
      "hapmap3_bin_snplis.log\n",
      "hapmap3_bin_snplis-merge.missnp\n",
      "HAPMAP_GRCh38.bed\n",
      "HAPMAP_GRCh38.bim\n",
      "HAPMAP_GRCh38.fam\n",
      "hapmap_outliers33.txt\n",
      "header.txt\n",
      "HETEROZYGOSITY_DATA.txt\n",
      "HETEROZYGOSITY_OUTLIERS.txt\n",
      "HWE_snps.hh\n",
      "HWE_snps.log\n",
      "HWE_snps.snplist\n",
      "IDs_after_relatedness_filter.txt\n",
      "IDs_before_relatedness_filter.txt\n",
      "missing_hap.hh\n",
      "missing_hap.log\n",
      "missing_hap.missing.hap\n",
      "missing_haps_1E4_final.txt\n",
      "missing_haps_1E4.txt\n",
      "MISSINGNESS_SNPS.txt\n",
      "missing_snps_1E4.txt\n",
      "missing_snps.hh\n",
      "missing_snps.log\n",
      "missing_snps.missing\n",
      "mixed_race_confirmation_plot.pdf\n",
      "new_samples2.txt\n",
      "new_samples_add.txt\n",
      "new_samples.txt\n",
      "outliers1.txt\n",
      "outliers2.txt\n",
      "pca.bed\n",
      "pca.bim\n",
      "pca.eigenval\n",
      "pca.eigenvec\n",
      "pca.eigenvec2\n",
      "pca.fam\n",
      "PCA_filtered_africans.txt\n",
      "PCA_filtered_asians.txt\n",
      "PCA_filtered_europeans.txt\n",
      "PCA_filtered_mixed_race.txt\n",
      "PCA_in_R.R\n",
      "pca.log\n",
      "plink_test_covs.tab\n",
      "plink_test_pheno.tab\n",
      "problems1.txt\n",
      "problems2.txt\n",
      "prune.bed\n",
      "prune.bim\n",
      "pruned_data.bed\n",
      "pruned_data.bim\n",
      "pruned_data.fam\n",
      "pruned_data.hh\n",
      "pruned_data.log\n",
      "prunedHet.het\n",
      "prunedHet.hh\n",
      "prunedHet.log\n",
      "prune.fam\n",
      "prune.log\n",
      "prune.prune.in\n",
      "prune.prune.out\n",
      "pruning.hh\n",
      "pruning.log\n",
      "pruning.prune.in\n",
      "pruning.prune.out\n",
      "Py3 - Clinical - load a CSV from Cloud Storage.ipynb\n",
      "Py3 - Clinical - load a table from BigQuery.ipynb\n",
      "Py3 - Query a saved Data Explorer cohort.ipynb\n",
      "Py3 - Saving Notebook Results.ipynb\n",
      "Py3 - Start Here.ipynb\n",
      "Py3 - WGS -  Query Variants.ipynb\n",
      "Py3 - WGS - Start Here.ipynb\n",
      "raw_hapmap_plot.pdf\n",
      "refFlat.txt\n",
      "RIC3_CODING_KB.log\n",
      "RIC3_CODING_KB.vcf.gz\n",
      "RIC3_CODING_KB.vcf.gz.tbi\n",
      "RIC3-Py-cleaned.ipynb\n",
      "RIC3-Py.ipynb\n",
      "RIC3-Py_New.ipynb\n",
      "RIC3-R.ipynb\n",
      "RIC3_WGS_AMP_PD.assoc.fisher\n",
      "RIC3_WGS_AMP_PD.log\n",
      "RIC3_WGS_AMP_PD_logreg.assoc.logistic\n",
      "RIC3_WGS_AMP_PD_logreg.log\n",
      "RIC3_WGS.annovar.hg38_multianno.txt\n",
      "RIC3.WGS.trimmed.annotation_KB.coding.variants.txt\n",
      "RIC3.WGS.trimmed.annotation_KB.SNPs.txt\n",
      "RIC3.WGS.trimmed.annotation_KB.txt\n",
      "R - Start Here.ipynb\n",
      "samples_to_remove.txt\n",
      "VARIANT_TEST_MISSING_SNPS.txt\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "rm after_gender3.bed\n",
    "rm after_gender3.bim\n",
    "rm after_gender3.fam\n",
    "rm after_gender3.hh\n",
    "rm after_gender3.log\n",
    "rm after_gender4.bed\n",
    "rm after_gender4.bim\n",
    "rm after_gender4.fam\n",
    "rm after_gender4.hh\n",
    "rm after_gender4.log\n",
    "rm all_outliers.txt\n",
    "rm AMPPD_after_gender.bed\n",
    "rm AMPPD_after_gender.bim\n",
    "rm AMPPD_after_gender.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap.bed\n",
    "rm AMPPD_after_gender_heterozyg_hapmap.bim\n",
    "rm AMPPD_after_gender_heterozyg_hapmap.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap.log\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat.bed\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat.bim\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat.log\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bed\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind.bim\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind.log\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bed\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.bim\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing1.log\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bed\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.bim\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.fam\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing2.log\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.hh\n",
    "rm AMPPD_after_gender_heterozyg_hapmap_pihat_mind_missing3.log\n",
    "rm AMPPD_after_gender.hh\n",
    "rm AMPPD_after_gender.log\n",
    "rm AMPPD_after_heterozyg.bed\n",
    "rm AMPPD_after_heterozyg.bim\n",
    "rm AMPPD_after_heterozyg.fam\n",
    "rm AMPPD_after_heterozyg.hh\n",
    "rm AMPPD_after_heterozyg.log\n",
    "rm AMPPD_call_rate.bed\n",
    "rm AMPPD_call_rate.bim\n",
    "rm AMPPD_call_rate.fam\n",
    "rm AMPPD_call_rate.hh\n",
    "rm AMPPD_call_rate.log\n",
    "rm AMPPD_updated_pheno.bed\n",
    "rm AMPPD_updated_pheno.bim\n",
    "rm AMPPD_updated_pheno.fam\n",
    "rm AMPPD_updated_pheno.hh\n",
    "rm AMPPD_updated_pheno.log\n",
    "rm call_rates.hh\n",
    "rm call_rates.imiss\n",
    "rm call_rates.lmiss\n",
    "rm call_rates.log\n",
    "rm EXAMPLE_UNIMPUTED.bed\n",
    "rm EXAMPLE_UNIMPUTED.bim\n",
    "rm EXAMPLE_UNIMPUTED.fam\n",
    "rm EXAMPLE_UNIMPUTED.log\n",
    "rm gender_check1.hh\n",
    "rm gender_check1.log\n",
    "rm gender_check1.sexcheck\n",
    "rm gender_check2.hh\n",
    "rm gender_check2.log\n",
    "rm gender_check2.sexcheck\n",
    "rm GENDER_FAILURES.txt\n",
    "rm GRM_matrix_0125.grm.bin\n",
    "rm GRM_matrix_0125.grm.id\n",
    "rm GRM_matrix_0125.grm.N.bin\n",
    "rm GRM_matrix_0125.log\n",
    "rm GRM_matrix.grm.bin\n",
    "rm GRM_matrix.grm.id\n",
    "rm GRM_matrix.grm.N.bin\n",
    "rm GRM_matrix.log\n",
    "rm HAPLOTYPE_TEST_MISSING_SNPS.txt\n",
    "rm hapmap3_bin_snplis.bed\n",
    "rm hapmap3_bin_snplis.bim\n",
    "rm hapmap3_bin_snplis.fam\n",
    "rm hapmap3_bin_snplis.hh\n",
    "rm hapmap3_bin_snplis.log\n",
    "rm hapmap3_bin_snplis-merge.missnp\n",
    "rm hapmap_outliers33.txt\n",
    "rm header.txt\n",
    "rm HETEROZYGOSITY_DATA.txt\n",
    "rm HETEROZYGOSITY_OUTLIERS.txt\n",
    "rm HWE_snps.hh\n",
    "rm HWE_snps.log\n",
    "rm HWE_snps.snplist\n",
    "rm IDs_after_relatedness_filter.txt\n",
    "rm IDs_before_relatedness_filter.txt\n",
    "rm missing_hap.hh\n",
    "rm missing_hap.log\n",
    "rm missing_hap.missing.hap\n",
    "rm missing_haps_1E4_final.txt\n",
    "rm missing_haps_1E4.txt\n",
    "rm MISSINGNESS_SNPS.txt\n",
    "rm missing_snps_1E4.txt\n",
    "rm missing_snps.hh\n",
    "rm missing_snps.log\n",
    "rm missing_snps.missing\n",
    "rm mixed_race_confirmation_plot.pdf\n",
    "rm new_samples2.txt\n",
    "rm new_samples_add.txt\n",
    "rm new_samples.txt\n",
    "rm outliers1.txt\n",
    "rm outliers2.txt\n",
    "rm pca.bed\n",
    "rm pca.bim\n",
    "rm pca.fam\n",
    "rm pca.log\n",
    "rm problems1.txt\n",
    "rm problems2.txt\n",
    "rm prune.bed\n",
    "rm prune.bim\n",
    "rm pruned_data.bed\n",
    "rm pruned_data.bim\n",
    "rm pruned_data.fam\n",
    "rm pruned_data.hh\n",
    "rm pruned_data.log\n",
    "rm prunedHet.het\n",
    "rm prunedHet.hh\n",
    "rm prunedHet.log\n",
    "rm prune.fam\n",
    "rm prune.log\n",
    "rm prune.prune.in\n",
    "rm prune.prune.out\n",
    "rm pruning.hh\n",
    "rm pruning.log\n",
    "rm pruning.prune.in\n",
    "rm pruning.prune.out\n",
    "rm samples_to_remove.txt\n",
    "rm VARIANT_TEST_MISSING_SNPS.txt"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
